{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "anaconda-cloud": {},
    "celltoolbar": "Create Assignment",
    "colab": {
      "name": "Session12_ProteinDesign.ipynb",
      "provenance": [],
      "collapsed_sections": []
    },
    "kernelspec": {
      "display_name": "Python 3",
      "language": "python",
      "name": "python3"
    },
    "language_info": {
      "codemirror_mode": {
        "name": "ipython",
        "version": 3
      },
      "file_extension": ".py",
      "mimetype": "text/x-python",
      "name": "python",
      "nbconvert_exporter": "python",
      "pygments_lexer": "ipython3",
      "version": "3.7.7"
    },
    "toc": {
      "base_numbering": 1,
      "nav_menu": {},
      "number_sections": true,
      "sideBar": true,
      "skip_h1_title": false,
      "title_cell": "Table of Contents",
      "title_sidebar": "Contents",
      "toc_cell": false,
      "toc_position": {},
      "toc_section_display": true,
      "toc_window_display": false
    },
    "widgets": {
      "application/vnd.jupyter.widget-state+json": {
        "3c8a0274c508449da448f6b4d261772c": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "VBoxModel",
          "state": {
            "_view_name": "VBoxView",
            "_dom_classes": [
              "widget-interact"
            ],
            "_model_name": "VBoxModel",
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "box_style": "",
            "layout": "IPY_MODEL_8a44a0ec76ce4394bbb2dd6dbcb257d9",
            "_model_module": "@jupyter-widgets/controls",
            "children": [
              "IPY_MODEL_532e4bfede5a4e899c40e47bb1357fd8",
              "IPY_MODEL_6a42e493f4cb4d478acb1172b8f113e6"
            ]
          }
        },
        "8a44a0ec76ce4394bbb2dd6dbcb257d9": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "532e4bfede5a4e899c40e47bb1357fd8": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "IntSliderModel",
          "state": {
            "_view_name": "IntSliderView",
            "style": "IPY_MODEL_f701f8f9e4b94d419572b84530cb7745",
            "_dom_classes": [],
            "description": "Decoys",
            "step": 1,
            "_model_name": "IntSliderModel",
            "orientation": "horizontal",
            "max": 1,
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "value": 1,
            "_view_count": null,
            "disabled": false,
            "_view_module_version": "1.5.0",
            "min": 0,
            "continuous_update": false,
            "readout_format": "d",
            "description_tooltip": null,
            "readout": true,
            "_model_module": "@jupyter-widgets/controls",
            "layout": "IPY_MODEL_0dfa943289ba4764887d5807751eb8c1"
          }
        },
        "6a42e493f4cb4d478acb1172b8f113e6": {
          "model_module": "@jupyter-widgets/output",
          "model_name": "OutputModel",
          "state": {
            "_view_name": "OutputView",
            "msg_id": "",
            "_dom_classes": [],
            "_model_name": "OutputModel",
            "outputs": [
              {
                "output_type": "display_data",
                "metadata": {
                  "tags": []
                },
                "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_16234209828572564\"  style=\"position: relative; width: 1200px; height: 800px\">\n        <p id=\"3dmolwarning_16234209828572564\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n      resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n}\n\nvar viewer_16234209828572564 = null;\nvar warn = document.getElementById(\"3dmolwarning_16234209828572564\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_16234209828572564 = $3Dmol.createViewer($(\"#3dmolviewer_16234209828572564\"),{backgroundColor:\"white\"});\nviewer_16234209828572564.zoomTo();\n\tviewer_16234209828572564.addModels(\"HEADER                                            11-JUN-21   XXXX              \\nEXPDTA    THEORETICAL MODEL                                                     \\nREMARK 220                                                                      \\nREMARK 220 EXPERIMENTAL DETAILS                                                 \\nREMARK 220  EXPERIMENT TYPE                : THEORETICAL MODELLING              \\nREMARK 220  DATE OF DATA COLLECTION        : 11-JUN-21                          \\nREMARK 220                                                                      \\nREMARK 220 REMARK: MODEL GENERATED BY ROSETTA                                   \\nREMARK 220  VERSION 2020.50.post.dev+970.commits.3700df14560                    \\nATOM      1  N   ALA A   1       5.257  -0.401 -16.284  1.00  0.00           N  \\nATOM      2  CA  ALA A   1       4.555   0.567 -15.451  1.00  0.00           C  \\nATOM      3  C   ALA A   1       3.845  -0.118 -14.290  1.00  0.00           C  \\nATOM      4  O   ALA A   1       3.851   0.380 -13.165  1.00  0.00           O  \\nATOM      5  CB  ALA A   1       3.560   1.361 -16.284  1.00  0.00           C  \\nATOM      6 1H   ALA A   1       5.713   0.075 -17.036  1.00  0.00           H  \\nATOM      7 2H   ALA A   1       5.938  -0.883 -15.733  1.00  0.00           H  \\nATOM      8 3H   ALA A   1       4.602  -1.060 -16.655  1.00  0.00           H  \\nATOM      9  HA  ALA A   1       5.292   1.253 -15.033  1.00  0.00           H  \\nATOM     10 1HB  ALA A   1       3.043   2.079 -15.648  1.00  0.00           H  \\nATOM     11 2HB  ALA A   1       4.090   1.893 -17.075  1.00  0.00           H  \\nATOM     12 3HB  ALA A   1       2.833   0.682 -16.728  1.00  0.00           H  \\nATOM     13  N   ALA A   2       3.234  -1.265 -14.577  1.00  0.00           N  \\nATOM     14  CA  ALA A   2       2.515  -2.020 -13.558  1.00  0.00           C  \\nATOM     15  C   ALA A   2       3.461  -2.528 -12.478  1.00  0.00           C  \\nATOM     16  O   ALA A   2       3.137  -2.495 -11.291  1.00  0.00           O  \\nATOM     17  CB  ALA A   2       1.763  -3.182 -14.192  1.00  0.00           C  \\nATOM     18  H   ALA A   2       3.271  -1.619 -15.522  1.00  0.00           H  \\nATOM     19  HA  ALA A   2       1.796  -1.351 -13.085  1.00  0.00           H  \\nATOM     20 1HB  ALA A   2       1.231  -3.736 -13.419  1.00  0.00           H  \\nATOM     21 2HB  ALA A   2       1.049  -2.799 -14.921  1.00  0.00           H  \\nATOM     22 3HB  ALA A   2       2.471  -3.843 -14.690  1.00  0.00           H  \\nATOM     23  N   ALA A   3       4.631  -2.997 -12.895  1.00  0.00           N  \\nATOM     24  CA  ALA A   3       5.624  -3.512 -11.959  1.00  0.00           C  \\nATOM     25  C   ALA A   3       6.135  -2.411 -11.038  1.00  0.00           C  \\nATOM     26  O   ALA A   3       6.321  -2.628  -9.841  1.00  0.00           O  \\nATOM     27  CB  ALA A   3       6.781  -4.151 -12.712  1.00  0.00           C  \\nATOM     28  H   ALA A   3       4.837  -2.997 -13.884  1.00  0.00           H  \\nATOM     29  HA  ALA A   3       5.144  -4.270 -11.340  1.00  0.00           H  \\nATOM     30 1HB  ALA A   3       7.514  -4.530 -12.000  1.00  0.00           H  \\nATOM     31 2HB  ALA A   3       6.408  -4.974 -13.322  1.00  0.00           H  \\nATOM     32 3HB  ALA A   3       7.251  -3.408 -13.355  1.00  0.00           H  \\nATOM     33  N   ALA A   4       6.360  -1.230 -11.598  1.00  0.00           N  \\nATOM     34  CA  ALA A   4       6.847  -0.092 -10.827  1.00  0.00           C  \\nATOM     35  C   ALA A   4       5.823   0.349  -9.789  1.00  0.00           C  \\nATOM     36  O   ALA A   4       6.175   0.676  -8.656  1.00  0.00           O  \\nATOM     37  CB  ALA A   4       7.194   1.065 -11.753  1.00  0.00           C  \\nATOM     38  H   ALA A   4       6.191  -1.114 -12.586  1.00  0.00           H  \\nATOM     39  HA  ALA A   4       7.748  -0.402 -10.297  1.00  0.00           H  \\nATOM     40 1HB  ALA A   4       7.556   1.907 -11.163  1.00  0.00           H  \\nATOM     41 2HB  ALA A   4       7.970   0.752 -12.452  1.00  0.00           H  \\nATOM     42 3HB  ALA A   4       6.306   1.366 -12.307  1.00  0.00           H  \\nATOM     43  N   ALA A   5       4.555   0.355 -10.186  1.00  0.00           N  \\nATOM     44  CA  ALA A   5       3.473   0.757  -9.294  1.00  0.00           C  \\nATOM     45  C   ALA A   5       3.337  -0.209  -8.124  1.00  0.00           C  \\nATOM     46  O   ALA A   5       3.119   0.207  -6.986  1.00  0.00           O  \\nATOM     47  CB  ALA A   5       2.162   0.848 -10.060  1.00  0.00           C  \\nATOM     48  H   ALA A   5       4.333   0.074 -11.130  1.00  0.00           H  \\nATOM     49  HA  ALA A   5       3.715   1.740  -8.890  1.00  0.00           H  \\nATOM     50 1HB  ALA A   5       1.364   1.149  -9.381  1.00  0.00           H  \\nATOM     51 2HB  ALA A   5       2.257   1.585 -10.858  1.00  0.00           H  \\nATOM     52 3HB  ALA A   5       1.924  -0.123 -10.491  1.00  0.00           H  \\nATOM     53  N   ALA A   6       3.466  -1.500  -8.416  1.00  0.00           N  \\nATOM     54  CA  ALA A   6       3.354  -2.527  -7.387  1.00  0.00           C  \\nATOM     55  C   ALA A   6       4.476  -2.407  -6.364  1.00  0.00           C  \\nATOM     56  O   ALA A   6       4.254  -2.560  -5.163  1.00  0.00           O  \\nATOM     57  CB  ALA A   6       3.358  -3.912  -8.017  1.00  0.00           C  \\nATOM     58  H   ALA A   6       3.647  -1.773  -9.371  1.00  0.00           H  \\nATOM     59  HA  ALA A   6       2.409  -2.380  -6.864  1.00  0.00           H  \\nATOM     60 1HB  ALA A   6       3.274  -4.667  -7.237  1.00  0.00           H  \\nATOM     61 2HB  ALA A   6       2.516  -4.002  -8.703  1.00  0.00           H  \\nATOM     62 3HB  ALA A   6       4.288  -4.059  -8.565  1.00  0.00           H  \\nATOM     63  N   ALA A   7       5.684  -2.134  -6.843  1.00  0.00           N  \\nATOM     64  CA  ALA A   7       6.841  -1.992  -5.967  1.00  0.00           C  \\nATOM     65  C   ALA A   7       6.688  -0.792  -5.043  1.00  0.00           C  \\nATOM     66  O   ALA A   7       7.026  -0.859  -3.861  1.00  0.00           O  \\nATOM     67  CB  ALA A   7       8.115  -1.868  -6.789  1.00  0.00           C  \\nATOM     68  H   ALA A   7       5.804  -2.021  -7.840  1.00  0.00           H  \\nATOM     69  HA  ALA A   7       6.906  -2.886  -5.346  1.00  0.00           H  \\nATOM     70 1HB  ALA A   7       8.970  -1.763  -6.121  1.00  0.00           H  \\nATOM     71 2HB  ALA A   7       8.241  -2.761  -7.402  1.00  0.00           H  \\nATOM     72 3HB  ALA A   7       8.049  -0.993  -7.433  1.00  0.00           H  \\nATOM     73  N   ALA A   8       6.177   0.307  -5.583  1.00  0.00           N  \\nATOM     74  CA  ALA A   8       5.975   1.526  -4.809  1.00  0.00           C  \\nATOM     75  C   ALA A   8       4.937   1.318  -3.714  1.00  0.00           C  \\nATOM     76  O   ALA A   8       5.101   1.794  -2.591  1.00  0.00           O  \\nATOM     77  CB  ALA A   8       5.559   2.671  -5.722  1.00  0.00           C  \\nATOM     78  H   ALA A   8       5.920   0.301  -6.560  1.00  0.00           H  \\nATOM     79  HA  ALA A   8       6.920   1.783  -4.329  1.00  0.00           H  \\nATOM     80 1HB  ALA A   8       5.412   3.574  -5.129  1.00  0.00           H  \\nATOM     81 2HB  ALA A   8       6.338   2.846  -6.464  1.00  0.00           H  \\nATOM     82 3HB  ALA A   8       4.630   2.413  -6.227  1.00  0.00           H  \\nATOM     83  N   ALA A   9       3.869   0.603  -4.053  1.00  0.00           N  \\nATOM     84  CA  ALA A   9       2.797   0.330  -3.102  1.00  0.00           C  \\nATOM     85  C   ALA A   9       3.291  -0.533  -1.949  1.00  0.00           C  \\nATOM     86  O   ALA A   9       2.935  -0.304  -0.793  1.00  0.00           O  \\nATOM     87  CB  ALA A   9       1.626  -0.342  -3.803  1.00  0.00           C  \\nATOM     88  H   ALA A   9       3.797   0.240  -4.992  1.00  0.00           H  \\nATOM     89  HA  ALA A   9       2.462   1.281  -2.688  1.00  0.00           H  \\nATOM     90 1HB  ALA A   9       0.834  -0.540  -3.080  1.00  0.00           H  \\nATOM     91 2HB  ALA A   9       1.246   0.313  -4.587  1.00  0.00           H  \\nATOM     92 3HB  ALA A   9       1.956  -1.282  -4.244  1.00  0.00           H  \\nATOM     93  N   ALA A  10       4.112  -1.528  -2.273  1.00  0.00           N  \\nATOM     94  CA  ALA A  10       4.652  -2.428  -1.261  1.00  0.00           C  \\nATOM     95  C   ALA A  10       5.562  -1.684  -0.292  1.00  0.00           C  \\nATOM     96  O   ALA A  10       5.526  -1.924   0.915  1.00  0.00           O  \\nATOM     97  CB  ALA A  10       5.406  -3.574  -1.920  1.00  0.00           C  \\nATOM     98  H   ALA A  10       4.365  -1.660  -3.242  1.00  0.00           H  \\nATOM     99  HA  ALA A  10       3.818  -2.836  -0.691  1.00  0.00           H  \\nATOM    100 1HB  ALA A  10       5.804  -4.237  -1.152  1.00  0.00           H  \\nATOM    101 2HB  ALA A  10       4.728  -4.132  -2.566  1.00  0.00           H  \\nATOM    102 3HB  ALA A  10       6.227  -3.175  -2.515  1.00  0.00           H  \\nATOM    103  N   ALA A  11       6.377  -0.781  -0.821  1.00  0.00           N  \\nATOM    104  CA  ALA A  11       7.294   0.000   0.000  1.00  0.00           C  \\nATOM    105  C   ALA A  11       6.537   0.915   0.954  1.00  0.00           C  \\nATOM    106  O   ALA A  11       6.914   1.065   2.116  1.00  0.00           O  \\nATOM    107  CB  ALA A  11       8.232   0.813  -0.880  1.00  0.00           C  \\nATOM    108  H   ALA A  11       6.362  -0.630  -1.820  1.00  0.00           H  \\nATOM    109  HA  ALA A  11       7.885  -0.692   0.600  1.00  0.00           H  \\nATOM    110 1HB  ALA A  11       8.911   1.391  -0.253  1.00  0.00           H  \\nATOM    111 2HB  ALA A  11       8.809   0.141  -1.516  1.00  0.00           H  \\nATOM    112 3HB  ALA A  11       7.650   1.490  -1.504  1.00  0.00           H  \\nATOM    113  N   ALA A  12       5.468   1.526   0.457  1.00  0.00           N  \\nATOM    114  CA  ALA A  12       4.652   2.428   1.261  1.00  0.00           C  \\nATOM    115  C   ALA A  12       3.970   1.684   2.402  1.00  0.00           C  \\nATOM    116  O   ALA A  12       3.894   2.185   3.524  1.00  0.00           O  \\nATOM    117  CB  ALA A  12       3.617   3.124   0.390  1.00  0.00           C  \\nATOM    118  H   ALA A  12       5.211   1.362  -0.506  1.00  0.00           H  \\nATOM    119  HA  ALA A  12       5.309   3.180   1.698  1.00  0.00           H  \\nATOM    120 1HB  ALA A  12       3.015   3.794   1.005  1.00  0.00           H  \\nATOM    121 2HB  ALA A  12       4.122   3.699  -0.386  1.00  0.00           H  \\nATOM    122 3HB  ALA A  12       2.971   2.379  -0.072  1.00  0.00           H  \\nATOM    123  N   ALA A  13       3.478   0.486   2.104  1.00  0.00           N  \\nATOM    124  CA  ALA A  13       2.797  -0.330   3.102  1.00  0.00           C  \\nATOM    125  C   ALA A  13       3.749  -0.746   4.216  1.00  0.00           C  \\nATOM    126  O   ALA A  13       3.385  -0.740   5.392  1.00  0.00           O  \\nATOM    127  CB  ALA A  13       2.178  -1.558   2.451  1.00  0.00           C  \\nATOM    128  H   ALA A  13       3.578   0.133   1.163  1.00  0.00           H  \\nATOM    129  HA  ALA A  13       2.004   0.271   3.548  1.00  0.00           H  \\nATOM    130 1HB  ALA A  13       1.673  -2.157   3.209  1.00  0.00           H  \\nATOM    131 2HB  ALA A  13       1.456  -1.244   1.697  1.00  0.00           H  \\nATOM    132 3HB  ALA A  13       2.959  -2.152   1.980  1.00  0.00           H  \\nATOM    133  N   ALA A  14       4.971  -1.107   3.838  1.00  0.00           N  \\nATOM    134  CA  ALA A  14       5.975  -1.526   4.809  1.00  0.00           C  \\nATOM    135  C   ALA A  14       6.352  -0.382   5.742  1.00  0.00           C  \\nATOM    136  O   ALA A  14       6.518  -0.579   6.945  1.00  0.00           O  \\nATOM    137  CB  ALA A  14       7.211  -2.057   4.097  1.00  0.00           C  \\nATOM    138  H   ALA A  14       5.208  -1.089   2.857  1.00  0.00           H  \\nATOM    139  HA  ALA A  14       5.547  -2.324   5.416  1.00  0.00           H  \\nATOM    140 1HB  ALA A  14       7.951  -2.366   4.835  1.00  0.00           H  \\nATOM    141 2HB  ALA A  14       6.936  -2.912   3.479  1.00  0.00           H  \\nATOM    142 3HB  ALA A  14       7.632  -1.275   3.468  1.00  0.00           H  \\nATOM    143  N   ALA A  15       6.487   0.814   5.185  1.00  0.00           N  \\nATOM    144  CA  ALA A  15       6.841   1.992   5.967  1.00  0.00           C  \\nATOM    145  C   ALA A  15       5.746   2.337   6.969  1.00  0.00           C  \\nATOM    146  O   ALA A  15       6.029   2.696   8.112  1.00  0.00           O  \\nATOM    147  CB  ALA A  15       7.110   3.176   5.050  1.00  0.00           C  \\nATOM    148  H   ALA A  15       6.340   0.913   4.190  1.00  0.00           H  \\nATOM    149  HA  ALA A  15       7.748   1.766   6.528  1.00  0.00           H  \\nATOM    150 1HB  ALA A  15       7.373   4.048   5.649  1.00  0.00           H  \\nATOM    151 2HB  ALA A  15       7.934   2.936   4.378  1.00  0.00           H  \\nATOM    152 3HB  ALA A  15       6.218   3.394   4.465  1.00  0.00           H  \\nATOM    153  N   ALA A  16       4.497   2.227   6.533  1.00  0.00           N  \\nATOM    154  CA  ALA A  16       3.354   2.527   7.387  1.00  0.00           C  \\nATOM    155  C   ALA A  16       3.268   1.552   8.554  1.00  0.00           C  \\nATOM    156  O   ALA A  16       2.975   1.945   9.683  1.00  0.00           O  \\nATOM    157  CB  ALA A  16       2.066   2.498   6.577  1.00  0.00           C  \\nATOM    158  H   ALA A  16       4.332   1.928   5.582  1.00  0.00           H  \\nATOM    159  HA  ALA A  16       3.491   3.527   7.797  1.00  0.00           H  \\nATOM    160 1HB  ALA A  16       1.221   2.723   7.229  1.00  0.00           H  \\nATOM    161 2HB  ALA A  16       2.119   3.241   5.782  1.00  0.00           H  \\nATOM    162 3HB  ALA A  16       1.933   1.509   6.141  1.00  0.00           H  \\nATOM    163  N   ALA A  17       3.525   0.278   8.269  1.00  0.00           N  \\nATOM    164  CA  ALA A  17       3.473  -0.757   9.294  1.00  0.00           C  \\nATOM    165  C   ALA A  17       4.545  -0.537  10.354  1.00  0.00           C  \\nATOM    166  O   ALA A  17       4.298  -0.711  11.547  1.00  0.00           O  \\nATOM    167  CB  ALA A  17       3.626  -2.134   8.665  1.00  0.00           C  \\nATOM    168  H   ALA A  17       3.762   0.024   7.320  1.00  0.00           H  \\nATOM    169  HA  ALA A  17       2.502  -0.699   9.785  1.00  0.00           H  \\nATOM    170 1HB  ALA A  17       3.586  -2.896   9.444  1.00  0.00           H  \\nATOM    171 2HB  ALA A  17       2.819  -2.300   7.952  1.00  0.00           H  \\nATOM    172 3HB  ALA A  17       4.583  -2.194   8.149  1.00  0.00           H  \\nATOM    173  N   ALA A  18       5.738  -0.152   9.913  1.00  0.00           N  \\nATOM    174  CA  ALA A  18       6.847   0.092  10.827  1.00  0.00           C  \\nATOM    175  C   ALA A  18       6.554   1.271  11.746  1.00  0.00           C  \\nATOM    176  O   ALA A  18       6.858   1.231  12.939  1.00  0.00           O  \\nATOM    177  CB  ALA A  18       8.132   0.335  10.049  1.00  0.00           C  \\nATOM    178  H   ALA A  18       5.879  -0.027   8.921  1.00  0.00           H  \\nATOM    179  HA  ALA A  18       6.974  -0.794  11.450  1.00  0.00           H  \\nATOM    180 1HB  ALA A  18       8.951   0.516  10.745  1.00  0.00           H  \\nATOM    181 2HB  ALA A  18       8.360  -0.540   9.440  1.00  0.00           H  \\nATOM    182 3HB  ALA A  18       8.007   1.203   9.403  1.00  0.00           H  \\nATOM    183  N   ALA A  19       5.963   2.320  11.191  1.00  0.00           N  \\nATOM    184  CA  ALA A  19       5.624   3.512  11.959  1.00  0.00           C  \\nATOM    185  C   ALA A  19       4.573   3.204  13.018  1.00  0.00           C  \\nATOM    186  O   ALA A  19       4.656   3.688  14.147  1.00  0.00           O  \\nATOM    187  CB  ALA A  19       5.134   4.617  11.034  1.00  0.00           C  \\nATOM    188  H   ALA A  19       5.740   2.294  10.206  1.00  0.00           H  \\nATOM    189  HA  ALA A  19       6.524   3.852  12.470  1.00  0.00           H  \\nATOM    190 1HB  ALA A  19       4.885   5.500  11.623  1.00  0.00           H  \\nATOM    191 2HB  ALA A  19       5.918   4.867  10.319  1.00  0.00           H  \\nATOM    192 3HB  ALA A  19       4.250   4.277  10.499  1.00  0.00           H  \\nATOM    193  N   ALA A  20       3.587   2.396  12.645  1.00  0.00           N  \\nATOM    194  CA  ALA A  20       2.515   2.020  13.558  1.00  0.00           C  \\nATOM    195  C   ALA A  20       3.047   1.199  14.725  1.00  0.00           C  \\nATOM    196  O   ALA A  20       4.035   0.477  14.589  1.00  0.00           O  \\nATOM    197  OXT ALA A  20       2.502   1.251  15.793  1.00  0.00           O  \\nATOM    198  CB  ALA A  20       1.434   1.246  12.817  1.00  0.00           C  \\nATOM    199  H   ALA A  20       3.580   2.031  11.703  1.00  0.00           H  \\nATOM    200  HA  ALA A  20       2.080   2.934  13.964  1.00  0.00           H  \\nATOM    201 1HB  ALA A  20       0.640   0.973  13.513  1.00  0.00           H  \\nATOM    202 2HB  ALA A  20       1.022   1.868  12.022  1.00  0.00           H  \\nATOM    203 3HB  ALA A  20       1.864   0.344  12.385  1.00  0.00           H  \\nTER                                                                             \\nATOM    205  N   ALA B  21      -5.257   0.409 -16.284  1.00  0.00           N  \\nATOM    206  CA  ALA B  21      -4.555  -0.560 -15.451  1.00  0.00           C  \\nATOM    207  C   ALA B  21      -3.845   0.124 -14.290  1.00  0.00           C  \\nATOM    208  O   ALA B  21      -3.852  -0.374 -13.165  1.00  0.00           O  \\nATOM    209  CB  ALA B  21      -3.562  -1.356 -16.284  1.00  0.00           C  \\nATOM    210 1H   ALA B  21      -5.713  -0.066 -17.037  1.00  0.00           H  \\nATOM    211 2H   ALA B  21      -5.938   0.891 -15.733  1.00  0.00           H  \\nATOM    212 3H   ALA B  21      -4.602   1.067 -16.654  1.00  0.00           H  \\nATOM    213  HA  ALA B  21      -5.294  -1.244 -15.033  1.00  0.00           H  \\nATOM    214 1HB  ALA B  21      -3.046  -2.075 -15.648  1.00  0.00           H  \\nATOM    215 2HB  ALA B  21      -4.093  -1.886 -17.075  1.00  0.00           H  \\nATOM    216 3HB  ALA B  21      -2.835  -0.678 -16.728  1.00  0.00           H  \\nATOM    217  N   ALA B  22      -3.232   1.270 -14.577  1.00  0.00           N  \\nATOM    218  CA  ALA B  22      -2.511   2.024 -13.558  1.00  0.00           C  \\nATOM    219  C   ALA B  22      -3.457   2.533 -12.478  1.00  0.00           C  \\nATOM    220  O   ALA B  22      -3.133   2.500 -11.291  1.00  0.00           O  \\nATOM    221  CB  ALA B  22      -1.758   3.185 -14.192  1.00  0.00           C  \\nATOM    222  H   ALA B  22      -3.268   1.624 -15.522  1.00  0.00           H  \\nATOM    223  HA  ALA B  22      -1.793   1.354 -13.085  1.00  0.00           H  \\nATOM    224 1HB  ALA B  22      -1.225   3.738 -13.419  1.00  0.00           H  \\nATOM    225 2HB  ALA B  22      -1.045   2.801 -14.921  1.00  0.00           H  \\nATOM    226 3HB  ALA B  22      -2.464   3.847 -14.690  1.00  0.00           H  \\nATOM    227  N   ALA B  23      -4.626   3.005 -12.895  1.00  0.00           N  \\nATOM    228  CA  ALA B  23      -5.618   3.521 -11.959  1.00  0.00           C  \\nATOM    229  C   ALA B  23      -6.131   2.421 -11.038  1.00  0.00           C  \\nATOM    230  O   ALA B  23      -6.317   2.638  -9.841  1.00  0.00           O  \\nATOM    231  CB  ALA B  23      -6.775   4.161 -12.712  1.00  0.00           C  \\nATOM    232  H   ALA B  23      -4.832   3.005 -13.884  1.00  0.00           H  \\nATOM    233  HA  ALA B  23      -5.137   4.278 -11.340  1.00  0.00           H  \\nATOM    234 1HB  ALA B  23      -7.507   4.542 -12.000  1.00  0.00           H  \\nATOM    235 2HB  ALA B  23      -6.400   4.984 -13.322  1.00  0.00           H  \\nATOM    236 3HB  ALA B  23      -7.246   3.419 -13.355  1.00  0.00           H  \\nATOM    237  N   ALA B  24      -6.358   1.240 -11.598  1.00  0.00           N  \\nATOM    238  CA  ALA B  24      -6.847   0.103 -10.827  1.00  0.00           C  \\nATOM    239  C   ALA B  24      -5.824  -0.340  -9.789  1.00  0.00           C  \\nATOM    240  O   ALA B  24      -6.176  -0.666  -8.656  1.00  0.00           O  \\nATOM    241  CB  ALA B  24      -7.196  -1.053 -11.753  1.00  0.00           C  \\nATOM    242  H   ALA B  24      -6.189   1.124 -12.586  1.00  0.00           H  \\nATOM    243  HA  ALA B  24      -7.747   0.415 -10.297  1.00  0.00           H  \\nATOM    244 1HB  ALA B  24      -7.559  -1.895 -11.163  1.00  0.00           H  \\nATOM    245 2HB  ALA B  24      -7.971  -0.739 -12.452  1.00  0.00           H  \\nATOM    246 3HB  ALA B  24      -6.309  -1.356 -12.307  1.00  0.00           H  \\nATOM    247  N   ALA B  25      -4.556  -0.348 -10.186  1.00  0.00           N  \\nATOM    248  CA  ALA B  25      -3.475  -0.751  -9.294  1.00  0.00           C  \\nATOM    249  C   ALA B  25      -3.337   0.214  -8.124  1.00  0.00           C  \\nATOM    250  O   ALA B  25      -3.120  -0.202  -6.986  1.00  0.00           O  \\nATOM    251  CB  ALA B  25      -2.163  -0.845 -10.060  1.00  0.00           C  \\nATOM    252  H   ALA B  25      -4.333  -0.067 -11.130  1.00  0.00           H  \\nATOM    253  HA  ALA B  25      -3.718  -1.734  -8.890  1.00  0.00           H  \\nATOM    254 1HB  ALA B  25      -1.366  -1.147  -9.381  1.00  0.00           H  \\nATOM    255 2HB  ALA B  25      -2.259  -1.582 -10.858  1.00  0.00           H  \\nATOM    256 3HB  ALA B  25      -1.924   0.126 -10.491  1.00  0.00           H  \\nATOM    257  N   ALA B  26      -3.464   1.505  -8.416  1.00  0.00           N  \\nATOM    258  CA  ALA B  26      -3.350   2.532  -7.387  1.00  0.00           C  \\nATOM    259  C   ALA B  26      -4.472   2.414  -6.364  1.00  0.00           C  \\nATOM    260  O   ALA B  26      -4.250   2.566  -5.163  1.00  0.00           O  \\nATOM    261  CB  ALA B  26      -3.352   3.917  -8.017  1.00  0.00           C  \\nATOM    262  H   ALA B  26      -3.644   1.779  -9.371  1.00  0.00           H  \\nATOM    263  HA  ALA B  26      -2.405   2.384  -6.864  1.00  0.00           H  \\nATOM    264 1HB  ALA B  26      -3.266   4.673  -7.237  1.00  0.00           H  \\nATOM    265 2HB  ALA B  26      -2.509   4.006  -8.703  1.00  0.00           H  \\nATOM    266 3HB  ALA B  26      -4.282   4.065  -8.565  1.00  0.00           H  \\nATOM    267  N   ALA B  27      -5.680   2.143  -6.843  1.00  0.00           N  \\nATOM    268  CA  ALA B  27      -6.837   2.003  -5.967  1.00  0.00           C  \\nATOM    269  C   ALA B  27      -6.687   0.802  -5.043  1.00  0.00           C  \\nATOM    270  O   ALA B  27      -7.025   0.870  -3.861  1.00  0.00           O  \\nATOM    271  CB  ALA B  27      -8.112   1.881  -6.789  1.00  0.00           C  \\nATOM    272  H   ALA B  27      -5.800   2.030  -7.840  1.00  0.00           H  \\nATOM    273  HA  ALA B  27      -6.902   2.897  -5.346  1.00  0.00           H  \\nATOM    274 1HB  ALA B  27      -8.967   1.778  -6.121  1.00  0.00           H  \\nATOM    275 2HB  ALA B  27      -8.237   2.774  -7.402  1.00  0.00           H  \\nATOM    276 3HB  ALA B  27      -8.048   1.006  -7.433  1.00  0.00           H  \\nATOM    277  N   ALA B  28      -6.178  -0.297  -5.583  1.00  0.00           N  \\nATOM    278  CA  ALA B  28      -5.978  -1.517  -4.809  1.00  0.00           C  \\nATOM    279  C   ALA B  28      -4.939  -1.310  -3.714  1.00  0.00           C  \\nATOM    280  O   ALA B  28      -5.104  -1.786  -2.591  1.00  0.00           O  \\nATOM    281  CB  ALA B  28      -5.563  -2.662  -5.722  1.00  0.00           C  \\nATOM    282  H   ALA B  28      -5.921  -0.292  -6.560  1.00  0.00           H  \\nATOM    283  HA  ALA B  28      -6.923  -1.772  -4.329  1.00  0.00           H  \\nATOM    284 1HB  ALA B  28      -5.418  -3.565  -5.129  1.00  0.00           H  \\nATOM    285 2HB  ALA B  28      -6.343  -2.836  -6.464  1.00  0.00           H  \\nATOM    286 3HB  ALA B  28      -4.633  -2.406  -6.227  1.00  0.00           H  \\nATOM    287  N   ALA B  29      -3.870  -0.597  -4.053  1.00  0.00           N  \\nATOM    288  CA  ALA B  29      -2.798  -0.326  -3.102  1.00  0.00           C  \\nATOM    289  C   ALA B  29      -3.290   0.539  -1.949  1.00  0.00           C  \\nATOM    290  O   ALA B  29      -2.935   0.309  -0.793  1.00  0.00           O  \\nATOM    291  CB  ALA B  29      -1.625   0.345  -3.803  1.00  0.00           C  \\nATOM    292  H   ALA B  29      -3.798  -0.234  -4.992  1.00  0.00           H  \\nATOM    293  HA  ALA B  29      -2.464  -1.277  -2.688  1.00  0.00           H  \\nATOM    294 1HB  ALA B  29      -0.833   0.541  -3.080  1.00  0.00           H  \\nATOM    295 2HB  ALA B  29      -1.247  -0.311  -4.587  1.00  0.00           H  \\nATOM    296 3HB  ALA B  29      -1.954   1.285  -4.244  1.00  0.00           H  \\nATOM    297  N   ALA B  30      -4.109   1.534  -2.273  1.00  0.00           N  \\nATOM    298  CA  ALA B  30      -4.648   2.435  -1.261  1.00  0.00           C  \\nATOM    299  C   ALA B  30      -5.559   1.693  -0.292  1.00  0.00           C  \\nATOM    300  O   ALA B  30      -5.523   1.933   0.915  1.00  0.00           O  \\nATOM    301  CB  ALA B  30      -5.400   3.583  -1.920  1.00  0.00           C  \\nATOM    302  H   ALA B  30      -4.363   1.667  -3.242  1.00  0.00           H  \\nATOM    303  HA  ALA B  30      -3.813   2.842  -0.691  1.00  0.00           H  \\nATOM    304 1HB  ALA B  30      -5.797   4.246  -1.152  1.00  0.00           H  \\nATOM    305 2HB  ALA B  30      -4.721   4.139  -2.566  1.00  0.00           H  \\nATOM    306 3HB  ALA B  30      -6.222   3.185  -2.515  1.00  0.00           H  \\nATOM    307  N   ALA B  31      -6.376   0.791  -0.821  1.00  0.00           N  \\nATOM    308  CA  ALA B  31      -7.294   0.012   0.000  1.00  0.00           C  \\nATOM    309  C   ALA B  31      -6.539  -0.905   0.954  1.00  0.00           C  \\nATOM    310  O   ALA B  31      -6.916  -1.054   2.116  1.00  0.00           O  \\nATOM    311  CB  ALA B  31      -8.233  -0.800  -0.880  1.00  0.00           C  \\nATOM    312  H   ALA B  31      -6.361   0.640  -1.820  1.00  0.00           H  \\nATOM    313  HA  ALA B  31      -7.884   0.705   0.600  1.00  0.00           H  \\nATOM    314 1HB  ALA B  31      -8.913  -1.377  -0.253  1.00  0.00           H  \\nATOM    315 2HB  ALA B  31      -8.809  -0.127  -1.516  1.00  0.00           H  \\nATOM    316 3HB  ALA B  31      -7.652  -1.478  -1.504  1.00  0.00           H  \\nATOM    317  N   ALA B  32      -5.471  -1.517   0.457  1.00  0.00           N  \\nATOM    318  CA  ALA B  32      -4.656  -2.420   1.261  1.00  0.00           C  \\nATOM    319  C   ALA B  32      -3.973  -1.678   2.402  1.00  0.00           C  \\nATOM    320  O   ALA B  32      -3.897  -2.179   3.524  1.00  0.00           O  \\nATOM    321  CB  ALA B  32      -3.622  -3.118   0.390  1.00  0.00           C  \\nATOM    322  H   ALA B  32      -5.213  -1.354  -0.506  1.00  0.00           H  \\nATOM    323  HA  ALA B  32      -5.314  -3.172   1.698  1.00  0.00           H  \\nATOM    324 1HB  ALA B  32      -3.021  -3.789   1.005  1.00  0.00           H  \\nATOM    325 2HB  ALA B  32      -4.128  -3.692  -0.386  1.00  0.00           H  \\nATOM    326 3HB  ALA B  32      -2.975  -2.374  -0.072  1.00  0.00           H  \\nATOM    327  N   ALA B  33      -3.478  -0.480   2.104  1.00  0.00           N  \\nATOM    328  CA  ALA B  33      -2.797   0.335   3.102  1.00  0.00           C  \\nATOM    329  C   ALA B  33      -3.748   0.752   4.216  1.00  0.00           C  \\nATOM    330  O   ALA B  33      -3.384   0.745   5.392  1.00  0.00           O  \\nATOM    331  CB  ALA B  33      -2.175   1.561   2.451  1.00  0.00           C  \\nATOM    332  H   ALA B  33      -3.578  -0.127   1.163  1.00  0.00           H  \\nATOM    333  HA  ALA B  33      -2.005  -0.268   3.548  1.00  0.00           H  \\nATOM    334 1HB  ALA B  33      -1.670   2.160   3.209  1.00  0.00           H  \\nATOM    335 2HB  ALA B  33      -1.454   1.247   1.697  1.00  0.00           H  \\nATOM    336 3HB  ALA B  33      -2.955   2.157   1.980  1.00  0.00           H  \\nATOM    337  N   ALA B  34      -4.970   1.115   3.838  1.00  0.00           N  \\nATOM    338  CA  ALA B  34      -5.973   1.536   4.809  1.00  0.00           C  \\nATOM    339  C   ALA B  34      -6.351   0.392   5.742  1.00  0.00           C  \\nATOM    340  O   ALA B  34      -6.517   0.589   6.945  1.00  0.00           O  \\nATOM    341  CB  ALA B  34      -7.208   2.068   4.097  1.00  0.00           C  \\nATOM    342  H   ALA B  34      -5.207   1.098   2.857  1.00  0.00           H  \\nATOM    343  HA  ALA B  34      -5.544   2.333   5.416  1.00  0.00           H  \\nATOM    344 1HB  ALA B  34      -7.948   2.379   4.835  1.00  0.00           H  \\nATOM    345 2HB  ALA B  34      -6.932   2.923   3.479  1.00  0.00           H  \\nATOM    346 3HB  ALA B  34      -7.630   1.287   3.468  1.00  0.00           H  \\nATOM    347  N   ALA B  35      -6.488  -0.804   5.185  1.00  0.00           N  \\nATOM    348  CA  ALA B  35      -6.844  -1.981   5.967  1.00  0.00           C  \\nATOM    349  C   ALA B  35      -5.750  -2.328   6.969  1.00  0.00           C  \\nATOM    350  O   ALA B  35      -6.033  -2.686   8.112  1.00  0.00           O  \\nATOM    351  CB  ALA B  35      -7.115  -3.165   5.050  1.00  0.00           C  \\nATOM    352  H   ALA B  35      -6.341  -0.903   4.190  1.00  0.00           H  \\nATOM    353  HA  ALA B  35      -7.751  -1.754   6.528  1.00  0.00           H  \\nATOM    354 1HB  ALA B  35      -7.380  -4.036   5.649  1.00  0.00           H  \\nATOM    355 2HB  ALA B  35      -7.939  -2.923   4.378  1.00  0.00           H  \\nATOM    356 3HB  ALA B  35      -6.223  -3.384   4.465  1.00  0.00           H  \\nATOM    357  N   ALA B  36      -4.501  -2.220   6.533  1.00  0.00           N  \\nATOM    358  CA  ALA B  36      -3.358  -2.521   7.387  1.00  0.00           C  \\nATOM    359  C   ALA B  36      -3.270  -1.547   8.554  1.00  0.00           C  \\nATOM    360  O   ALA B  36      -2.978  -1.940   9.683  1.00  0.00           O  \\nATOM    361  CB  ALA B  36      -2.070  -2.494   6.577  1.00  0.00           C  \\nATOM    362  H   ALA B  36      -4.336  -1.921   5.582  1.00  0.00           H  \\nATOM    363  HA  ALA B  36      -3.496  -3.522   7.797  1.00  0.00           H  \\nATOM    364 1HB  ALA B  36      -1.226  -2.721   7.229  1.00  0.00           H  \\nATOM    365 2HB  ALA B  36      -2.124  -3.238   5.782  1.00  0.00           H  \\nATOM    366 3HB  ALA B  36      -1.935  -1.506   6.141  1.00  0.00           H  \\nATOM    367  N   ALA B  37      -3.526  -0.273   8.269  1.00  0.00           N  \\nATOM    368  CA  ALA B  37      -3.472   0.762   9.294  1.00  0.00           C  \\nATOM    369  C   ALA B  37      -4.545   0.544  10.354  1.00  0.00           C  \\nATOM    370  O   ALA B  37      -4.297   0.718  11.547  1.00  0.00           O  \\nATOM    371  CB  ALA B  37      -3.623   2.140   8.665  1.00  0.00           C  \\nATOM    372  H   ALA B  37      -3.762  -0.018   7.320  1.00  0.00           H  \\nATOM    373  HA  ALA B  37      -2.501   0.703   9.785  1.00  0.00           H  \\nATOM    374 1HB  ALA B  37      -3.581   2.902   9.444  1.00  0.00           H  \\nATOM    375 2HB  ALA B  37      -2.815   2.305   7.952  1.00  0.00           H  \\nATOM    376 3HB  ALA B  37      -4.580   2.201   8.149  1.00  0.00           H  \\nATOM    377  N   ALA B  38      -5.738   0.161   9.913  1.00  0.00           N  \\nATOM    378  CA  ALA B  38      -6.847  -0.081  10.827  1.00  0.00           C  \\nATOM    379  C   ALA B  38      -6.556  -1.260  11.746  1.00  0.00           C  \\nATOM    380  O   ALA B  38      -6.860  -1.220  12.939  1.00  0.00           O  \\nATOM    381  CB  ALA B  38      -8.132  -0.323  10.049  1.00  0.00           C  \\nATOM    382  H   ALA B  38      -5.879   0.036   8.921  1.00  0.00           H  \\nATOM    383  HA  ALA B  38      -6.973   0.805  11.450  1.00  0.00           H  \\nATOM    384 1HB  ALA B  38      -8.951  -0.502  10.745  1.00  0.00           H  \\nATOM    385 2HB  ALA B  38      -8.359   0.553   9.440  1.00  0.00           H  \\nATOM    386 3HB  ALA B  38      -8.008  -1.191   9.403  1.00  0.00           H  \\nATOM    387  N   ALA B  39      -5.966  -2.310  11.191  1.00  0.00           N  \\nATOM    388  CA  ALA B  39      -5.629  -3.503  11.959  1.00  0.00           C  \\nATOM    389  C   ALA B  39      -4.578  -3.197  13.018  1.00  0.00           C  \\nATOM    390  O   ALA B  39      -4.662  -3.681  14.147  1.00  0.00           O  \\nATOM    391  CB  ALA B  39      -5.142  -4.609  11.034  1.00  0.00           C  \\nATOM    392  H   ALA B  39      -5.743  -2.285  10.206  1.00  0.00           H  \\nATOM    393  HA  ALA B  39      -6.530  -3.842  12.470  1.00  0.00           H  \\nATOM    394 1HB  ALA B  39      -4.894  -5.492  11.623  1.00  0.00           H  \\nATOM    395 2HB  ALA B  39      -5.926  -4.857  10.319  1.00  0.00           H  \\nATOM    396 3HB  ALA B  39      -4.256  -4.270  10.499  1.00  0.00           H  \\nATOM    397  N   ALA B  40      -3.591  -2.390  12.645  1.00  0.00           N  \\nATOM    398  CA  ALA B  40      -2.518  -2.016  13.558  1.00  0.00           C  \\nATOM    399  C   ALA B  40      -3.049  -1.194  14.725  1.00  0.00           C  \\nATOM    400  O   ALA B  40      -4.036  -0.470  14.589  1.00  0.00           O  \\nATOM    401  OXT ALA B  40      -2.505  -1.247  15.793  1.00  0.00           O  \\nATOM    402  CB  ALA B  40      -1.436  -1.244  12.817  1.00  0.00           C  \\nATOM    403  H   ALA B  40      -3.583  -2.026  11.703  1.00  0.00           H  \\nATOM    404  HA  ALA B  40      -2.084  -2.930  13.964  1.00  0.00           H  \\nATOM    405 1HB  ALA B  40      -0.642  -0.972  13.513  1.00  0.00           H  \\nATOM    406 2HB  ALA B  40      -1.024  -1.867  12.022  1.00  0.00           H  \\nATOM    407 3HB  ALA B  40      -1.864  -0.341  12.385  1.00  0.00           H  \\nTER                                                                             \\nREMARK PDBinfo-LABEL:    1 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    2 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    3 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    4 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    5 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    6 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    7 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    8 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    9 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   10 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   11 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   12 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   13 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   14 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   15 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   16 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   17 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   18 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   19 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   20 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   21 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   22 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   23 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   24 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   25 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   26 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   27 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   28 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   29 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   30 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   31 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   32 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   33 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   34 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   35 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   36 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   37 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   38 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   39 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   40 PEPTIDE_STUB_EXTENSION\\n\\n\",\"pdb\");\n\tviewer_16234209828572564.zoomTo();\n\tviewer_16234209828572564.setStyle({\"cartoon\": {\"color\": \"spectrum\"}, \"stick\": {\"colorscheme\": \"blackCarbon\", \"radius\": \"0.1\"}});\nviewer_16234209828572564.render();\n});\n</script>",
                "text/html": "<div id=\"3dmolviewer_16234209828572564\"  style=\"position: relative; width: 1200px; height: 800px\">\n        <p id=\"3dmolwarning_16234209828572564\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n      resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n}\n\nvar viewer_16234209828572564 = null;\nvar warn = document.getElementById(\"3dmolwarning_16234209828572564\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_16234209828572564 = $3Dmol.createViewer($(\"#3dmolviewer_16234209828572564\"),{backgroundColor:\"white\"});\nviewer_16234209828572564.zoomTo();\n\tviewer_16234209828572564.addModels(\"HEADER                                            11-JUN-21   XXXX              \\nEXPDTA    THEORETICAL MODEL                                                     \\nREMARK 220                                                                      \\nREMARK 220 EXPERIMENTAL DETAILS                                                 \\nREMARK 220  EXPERIMENT TYPE                : THEORETICAL MODELLING              \\nREMARK 220  DATE OF DATA COLLECTION        : 11-JUN-21                          \\nREMARK 220                                                                      \\nREMARK 220 REMARK: MODEL GENERATED BY ROSETTA                                   \\nREMARK 220  VERSION 2020.50.post.dev+970.commits.3700df14560                    \\nATOM      1  N   ALA A   1       5.257  -0.401 -16.284  1.00  0.00           N  \\nATOM      2  CA  ALA A   1       4.555   0.567 -15.451  1.00  0.00           C  \\nATOM      3  C   ALA A   1       3.845  -0.118 -14.290  1.00  0.00           C  \\nATOM      4  O   ALA A   1       3.851   0.380 -13.165  1.00  0.00           O  \\nATOM      5  CB  ALA A   1       3.560   1.361 -16.284  1.00  0.00           C  \\nATOM      6 1H   ALA A   1       5.713   0.075 -17.036  1.00  0.00           H  \\nATOM      7 2H   ALA A   1       5.938  -0.883 -15.733  1.00  0.00           H  \\nATOM      8 3H   ALA A   1       4.602  -1.060 -16.655  1.00  0.00           H  \\nATOM      9  HA  ALA A   1       5.292   1.253 -15.033  1.00  0.00           H  \\nATOM     10 1HB  ALA A   1       3.043   2.079 -15.648  1.00  0.00           H  \\nATOM     11 2HB  ALA A   1       4.090   1.893 -17.075  1.00  0.00           H  \\nATOM     12 3HB  ALA A   1       2.833   0.682 -16.728  1.00  0.00           H  \\nATOM     13  N   ALA A   2       3.234  -1.265 -14.577  1.00  0.00           N  \\nATOM     14  CA  ALA A   2       2.515  -2.020 -13.558  1.00  0.00           C  \\nATOM     15  C   ALA A   2       3.461  -2.528 -12.478  1.00  0.00           C  \\nATOM     16  O   ALA A   2       3.137  -2.495 -11.291  1.00  0.00           O  \\nATOM     17  CB  ALA A   2       1.763  -3.182 -14.192  1.00  0.00           C  \\nATOM     18  H   ALA A   2       3.271  -1.619 -15.522  1.00  0.00           H  \\nATOM     19  HA  ALA A   2       1.796  -1.351 -13.085  1.00  0.00           H  \\nATOM     20 1HB  ALA A   2       1.231  -3.736 -13.419  1.00  0.00           H  \\nATOM     21 2HB  ALA A   2       1.049  -2.799 -14.921  1.00  0.00           H  \\nATOM     22 3HB  ALA A   2       2.471  -3.843 -14.690  1.00  0.00           H  \\nATOM     23  N   ALA A   3       4.631  -2.997 -12.895  1.00  0.00           N  \\nATOM     24  CA  ALA A   3       5.624  -3.512 -11.959  1.00  0.00           C  \\nATOM     25  C   ALA A   3       6.135  -2.411 -11.038  1.00  0.00           C  \\nATOM     26  O   ALA A   3       6.321  -2.628  -9.841  1.00  0.00           O  \\nATOM     27  CB  ALA A   3       6.781  -4.151 -12.712  1.00  0.00           C  \\nATOM     28  H   ALA A   3       4.837  -2.997 -13.884  1.00  0.00           H  \\nATOM     29  HA  ALA A   3       5.144  -4.270 -11.340  1.00  0.00           H  \\nATOM     30 1HB  ALA A   3       7.514  -4.530 -12.000  1.00  0.00           H  \\nATOM     31 2HB  ALA A   3       6.408  -4.974 -13.322  1.00  0.00           H  \\nATOM     32 3HB  ALA A   3       7.251  -3.408 -13.355  1.00  0.00           H  \\nATOM     33  N   ALA A   4       6.360  -1.230 -11.598  1.00  0.00           N  \\nATOM     34  CA  ALA A   4       6.847  -0.092 -10.827  1.00  0.00           C  \\nATOM     35  C   ALA A   4       5.823   0.349  -9.789  1.00  0.00           C  \\nATOM     36  O   ALA A   4       6.175   0.676  -8.656  1.00  0.00           O  \\nATOM     37  CB  ALA A   4       7.194   1.065 -11.753  1.00  0.00           C  \\nATOM     38  H   ALA A   4       6.191  -1.114 -12.586  1.00  0.00           H  \\nATOM     39  HA  ALA A   4       7.748  -0.402 -10.297  1.00  0.00           H  \\nATOM     40 1HB  ALA A   4       7.556   1.907 -11.163  1.00  0.00           H  \\nATOM     41 2HB  ALA A   4       7.970   0.752 -12.452  1.00  0.00           H  \\nATOM     42 3HB  ALA A   4       6.306   1.366 -12.307  1.00  0.00           H  \\nATOM     43  N   ALA A   5       4.555   0.355 -10.186  1.00  0.00           N  \\nATOM     44  CA  ALA A   5       3.473   0.757  -9.294  1.00  0.00           C  \\nATOM     45  C   ALA A   5       3.337  -0.209  -8.124  1.00  0.00           C  \\nATOM     46  O   ALA A   5       3.119   0.207  -6.986  1.00  0.00           O  \\nATOM     47  CB  ALA A   5       2.162   0.848 -10.060  1.00  0.00           C  \\nATOM     48  H   ALA A   5       4.333   0.074 -11.130  1.00  0.00           H  \\nATOM     49  HA  ALA A   5       3.715   1.740  -8.890  1.00  0.00           H  \\nATOM     50 1HB  ALA A   5       1.364   1.149  -9.381  1.00  0.00           H  \\nATOM     51 2HB  ALA A   5       2.257   1.585 -10.858  1.00  0.00           H  \\nATOM     52 3HB  ALA A   5       1.924  -0.123 -10.491  1.00  0.00           H  \\nATOM     53  N   ALA A   6       3.466  -1.500  -8.416  1.00  0.00           N  \\nATOM     54  CA  ALA A   6       3.354  -2.527  -7.387  1.00  0.00           C  \\nATOM     55  C   ALA A   6       4.476  -2.407  -6.364  1.00  0.00           C  \\nATOM     56  O   ALA A   6       4.254  -2.560  -5.163  1.00  0.00           O  \\nATOM     57  CB  ALA A   6       3.358  -3.912  -8.017  1.00  0.00           C  \\nATOM     58  H   ALA A   6       3.647  -1.773  -9.371  1.00  0.00           H  \\nATOM     59  HA  ALA A   6       2.409  -2.380  -6.864  1.00  0.00           H  \\nATOM     60 1HB  ALA A   6       3.274  -4.667  -7.237  1.00  0.00           H  \\nATOM     61 2HB  ALA A   6       2.516  -4.002  -8.703  1.00  0.00           H  \\nATOM     62 3HB  ALA A   6       4.288  -4.059  -8.565  1.00  0.00           H  \\nATOM     63  N   ALA A   7       5.684  -2.134  -6.843  1.00  0.00           N  \\nATOM     64  CA  ALA A   7       6.841  -1.992  -5.967  1.00  0.00           C  \\nATOM     65  C   ALA A   7       6.688  -0.792  -5.043  1.00  0.00           C  \\nATOM     66  O   ALA A   7       7.026  -0.859  -3.861  1.00  0.00           O  \\nATOM     67  CB  ALA A   7       8.115  -1.868  -6.789  1.00  0.00           C  \\nATOM     68  H   ALA A   7       5.804  -2.021  -7.840  1.00  0.00           H  \\nATOM     69  HA  ALA A   7       6.906  -2.886  -5.346  1.00  0.00           H  \\nATOM     70 1HB  ALA A   7       8.970  -1.763  -6.121  1.00  0.00           H  \\nATOM     71 2HB  ALA A   7       8.241  -2.761  -7.402  1.00  0.00           H  \\nATOM     72 3HB  ALA A   7       8.049  -0.993  -7.433  1.00  0.00           H  \\nATOM     73  N   ALA A   8       6.177   0.307  -5.583  1.00  0.00           N  \\nATOM     74  CA  ALA A   8       5.975   1.526  -4.809  1.00  0.00           C  \\nATOM     75  C   ALA A   8       4.937   1.318  -3.714  1.00  0.00           C  \\nATOM     76  O   ALA A   8       5.101   1.794  -2.591  1.00  0.00           O  \\nATOM     77  CB  ALA A   8       5.559   2.671  -5.722  1.00  0.00           C  \\nATOM     78  H   ALA A   8       5.920   0.301  -6.560  1.00  0.00           H  \\nATOM     79  HA  ALA A   8       6.920   1.783  -4.329  1.00  0.00           H  \\nATOM     80 1HB  ALA A   8       5.412   3.574  -5.129  1.00  0.00           H  \\nATOM     81 2HB  ALA A   8       6.338   2.846  -6.464  1.00  0.00           H  \\nATOM     82 3HB  ALA A   8       4.630   2.413  -6.227  1.00  0.00           H  \\nATOM     83  N   ALA A   9       3.869   0.603  -4.053  1.00  0.00           N  \\nATOM     84  CA  ALA A   9       2.797   0.330  -3.102  1.00  0.00           C  \\nATOM     85  C   ALA A   9       3.291  -0.533  -1.949  1.00  0.00           C  \\nATOM     86  O   ALA A   9       2.935  -0.304  -0.793  1.00  0.00           O  \\nATOM     87  CB  ALA A   9       1.626  -0.342  -3.803  1.00  0.00           C  \\nATOM     88  H   ALA A   9       3.797   0.240  -4.992  1.00  0.00           H  \\nATOM     89  HA  ALA A   9       2.462   1.281  -2.688  1.00  0.00           H  \\nATOM     90 1HB  ALA A   9       0.834  -0.540  -3.080  1.00  0.00           H  \\nATOM     91 2HB  ALA A   9       1.246   0.313  -4.587  1.00  0.00           H  \\nATOM     92 3HB  ALA A   9       1.956  -1.282  -4.244  1.00  0.00           H  \\nATOM     93  N   ALA A  10       4.112  -1.528  -2.273  1.00  0.00           N  \\nATOM     94  CA  ALA A  10       4.652  -2.428  -1.261  1.00  0.00           C  \\nATOM     95  C   ALA A  10       5.562  -1.684  -0.292  1.00  0.00           C  \\nATOM     96  O   ALA A  10       5.526  -1.924   0.915  1.00  0.00           O  \\nATOM     97  CB  ALA A  10       5.406  -3.574  -1.920  1.00  0.00           C  \\nATOM     98  H   ALA A  10       4.365  -1.660  -3.242  1.00  0.00           H  \\nATOM     99  HA  ALA A  10       3.818  -2.836  -0.691  1.00  0.00           H  \\nATOM    100 1HB  ALA A  10       5.804  -4.237  -1.152  1.00  0.00           H  \\nATOM    101 2HB  ALA A  10       4.728  -4.132  -2.566  1.00  0.00           H  \\nATOM    102 3HB  ALA A  10       6.227  -3.175  -2.515  1.00  0.00           H  \\nATOM    103  N   ALA A  11       6.377  -0.781  -0.821  1.00  0.00           N  \\nATOM    104  CA  ALA A  11       7.294   0.000   0.000  1.00  0.00           C  \\nATOM    105  C   ALA A  11       6.537   0.915   0.954  1.00  0.00           C  \\nATOM    106  O   ALA A  11       6.914   1.065   2.116  1.00  0.00           O  \\nATOM    107  CB  ALA A  11       8.232   0.813  -0.880  1.00  0.00           C  \\nATOM    108  H   ALA A  11       6.362  -0.630  -1.820  1.00  0.00           H  \\nATOM    109  HA  ALA A  11       7.885  -0.692   0.600  1.00  0.00           H  \\nATOM    110 1HB  ALA A  11       8.911   1.391  -0.253  1.00  0.00           H  \\nATOM    111 2HB  ALA A  11       8.809   0.141  -1.516  1.00  0.00           H  \\nATOM    112 3HB  ALA A  11       7.650   1.490  -1.504  1.00  0.00           H  \\nATOM    113  N   ALA A  12       5.468   1.526   0.457  1.00  0.00           N  \\nATOM    114  CA  ALA A  12       4.652   2.428   1.261  1.00  0.00           C  \\nATOM    115  C   ALA A  12       3.970   1.684   2.402  1.00  0.00           C  \\nATOM    116  O   ALA A  12       3.894   2.185   3.524  1.00  0.00           O  \\nATOM    117  CB  ALA A  12       3.617   3.124   0.390  1.00  0.00           C  \\nATOM    118  H   ALA A  12       5.211   1.362  -0.506  1.00  0.00           H  \\nATOM    119  HA  ALA A  12       5.309   3.180   1.698  1.00  0.00           H  \\nATOM    120 1HB  ALA A  12       3.015   3.794   1.005  1.00  0.00           H  \\nATOM    121 2HB  ALA A  12       4.122   3.699  -0.386  1.00  0.00           H  \\nATOM    122 3HB  ALA A  12       2.971   2.379  -0.072  1.00  0.00           H  \\nATOM    123  N   ALA A  13       3.478   0.486   2.104  1.00  0.00           N  \\nATOM    124  CA  ALA A  13       2.797  -0.330   3.102  1.00  0.00           C  \\nATOM    125  C   ALA A  13       3.749  -0.746   4.216  1.00  0.00           C  \\nATOM    126  O   ALA A  13       3.385  -0.740   5.392  1.00  0.00           O  \\nATOM    127  CB  ALA A  13       2.178  -1.558   2.451  1.00  0.00           C  \\nATOM    128  H   ALA A  13       3.578   0.133   1.163  1.00  0.00           H  \\nATOM    129  HA  ALA A  13       2.004   0.271   3.548  1.00  0.00           H  \\nATOM    130 1HB  ALA A  13       1.673  -2.157   3.209  1.00  0.00           H  \\nATOM    131 2HB  ALA A  13       1.456  -1.244   1.697  1.00  0.00           H  \\nATOM    132 3HB  ALA A  13       2.959  -2.152   1.980  1.00  0.00           H  \\nATOM    133  N   ALA A  14       4.971  -1.107   3.838  1.00  0.00           N  \\nATOM    134  CA  ALA A  14       5.975  -1.526   4.809  1.00  0.00           C  \\nATOM    135  C   ALA A  14       6.352  -0.382   5.742  1.00  0.00           C  \\nATOM    136  O   ALA A  14       6.518  -0.579   6.945  1.00  0.00           O  \\nATOM    137  CB  ALA A  14       7.211  -2.057   4.097  1.00  0.00           C  \\nATOM    138  H   ALA A  14       5.208  -1.089   2.857  1.00  0.00           H  \\nATOM    139  HA  ALA A  14       5.547  -2.324   5.416  1.00  0.00           H  \\nATOM    140 1HB  ALA A  14       7.951  -2.366   4.835  1.00  0.00           H  \\nATOM    141 2HB  ALA A  14       6.936  -2.912   3.479  1.00  0.00           H  \\nATOM    142 3HB  ALA A  14       7.632  -1.275   3.468  1.00  0.00           H  \\nATOM    143  N   ALA A  15       6.487   0.814   5.185  1.00  0.00           N  \\nATOM    144  CA  ALA A  15       6.841   1.992   5.967  1.00  0.00           C  \\nATOM    145  C   ALA A  15       5.746   2.337   6.969  1.00  0.00           C  \\nATOM    146  O   ALA A  15       6.029   2.696   8.112  1.00  0.00           O  \\nATOM    147  CB  ALA A  15       7.110   3.176   5.050  1.00  0.00           C  \\nATOM    148  H   ALA A  15       6.340   0.913   4.190  1.00  0.00           H  \\nATOM    149  HA  ALA A  15       7.748   1.766   6.528  1.00  0.00           H  \\nATOM    150 1HB  ALA A  15       7.373   4.048   5.649  1.00  0.00           H  \\nATOM    151 2HB  ALA A  15       7.934   2.936   4.378  1.00  0.00           H  \\nATOM    152 3HB  ALA A  15       6.218   3.394   4.465  1.00  0.00           H  \\nATOM    153  N   ALA A  16       4.497   2.227   6.533  1.00  0.00           N  \\nATOM    154  CA  ALA A  16       3.354   2.527   7.387  1.00  0.00           C  \\nATOM    155  C   ALA A  16       3.268   1.552   8.554  1.00  0.00           C  \\nATOM    156  O   ALA A  16       2.975   1.945   9.683  1.00  0.00           O  \\nATOM    157  CB  ALA A  16       2.066   2.498   6.577  1.00  0.00           C  \\nATOM    158  H   ALA A  16       4.332   1.928   5.582  1.00  0.00           H  \\nATOM    159  HA  ALA A  16       3.491   3.527   7.797  1.00  0.00           H  \\nATOM    160 1HB  ALA A  16       1.221   2.723   7.229  1.00  0.00           H  \\nATOM    161 2HB  ALA A  16       2.119   3.241   5.782  1.00  0.00           H  \\nATOM    162 3HB  ALA A  16       1.933   1.509   6.141  1.00  0.00           H  \\nATOM    163  N   ALA A  17       3.525   0.278   8.269  1.00  0.00           N  \\nATOM    164  CA  ALA A  17       3.473  -0.757   9.294  1.00  0.00           C  \\nATOM    165  C   ALA A  17       4.545  -0.537  10.354  1.00  0.00           C  \\nATOM    166  O   ALA A  17       4.298  -0.711  11.547  1.00  0.00           O  \\nATOM    167  CB  ALA A  17       3.626  -2.134   8.665  1.00  0.00           C  \\nATOM    168  H   ALA A  17       3.762   0.024   7.320  1.00  0.00           H  \\nATOM    169  HA  ALA A  17       2.502  -0.699   9.785  1.00  0.00           H  \\nATOM    170 1HB  ALA A  17       3.586  -2.896   9.444  1.00  0.00           H  \\nATOM    171 2HB  ALA A  17       2.819  -2.300   7.952  1.00  0.00           H  \\nATOM    172 3HB  ALA A  17       4.583  -2.194   8.149  1.00  0.00           H  \\nATOM    173  N   ALA A  18       5.738  -0.152   9.913  1.00  0.00           N  \\nATOM    174  CA  ALA A  18       6.847   0.092  10.827  1.00  0.00           C  \\nATOM    175  C   ALA A  18       6.554   1.271  11.746  1.00  0.00           C  \\nATOM    176  O   ALA A  18       6.858   1.231  12.939  1.00  0.00           O  \\nATOM    177  CB  ALA A  18       8.132   0.335  10.049  1.00  0.00           C  \\nATOM    178  H   ALA A  18       5.879  -0.027   8.921  1.00  0.00           H  \\nATOM    179  HA  ALA A  18       6.974  -0.794  11.450  1.00  0.00           H  \\nATOM    180 1HB  ALA A  18       8.951   0.516  10.745  1.00  0.00           H  \\nATOM    181 2HB  ALA A  18       8.360  -0.540   9.440  1.00  0.00           H  \\nATOM    182 3HB  ALA A  18       8.007   1.203   9.403  1.00  0.00           H  \\nATOM    183  N   ALA A  19       5.963   2.320  11.191  1.00  0.00           N  \\nATOM    184  CA  ALA A  19       5.624   3.512  11.959  1.00  0.00           C  \\nATOM    185  C   ALA A  19       4.573   3.204  13.018  1.00  0.00           C  \\nATOM    186  O   ALA A  19       4.656   3.688  14.147  1.00  0.00           O  \\nATOM    187  CB  ALA A  19       5.134   4.617  11.034  1.00  0.00           C  \\nATOM    188  H   ALA A  19       5.740   2.294  10.206  1.00  0.00           H  \\nATOM    189  HA  ALA A  19       6.524   3.852  12.470  1.00  0.00           H  \\nATOM    190 1HB  ALA A  19       4.885   5.500  11.623  1.00  0.00           H  \\nATOM    191 2HB  ALA A  19       5.918   4.867  10.319  1.00  0.00           H  \\nATOM    192 3HB  ALA A  19       4.250   4.277  10.499  1.00  0.00           H  \\nATOM    193  N   ALA A  20       3.587   2.396  12.645  1.00  0.00           N  \\nATOM    194  CA  ALA A  20       2.515   2.020  13.558  1.00  0.00           C  \\nATOM    195  C   ALA A  20       3.047   1.199  14.725  1.00  0.00           C  \\nATOM    196  O   ALA A  20       4.035   0.477  14.589  1.00  0.00           O  \\nATOM    197  OXT ALA A  20       2.502   1.251  15.793  1.00  0.00           O  \\nATOM    198  CB  ALA A  20       1.434   1.246  12.817  1.00  0.00           C  \\nATOM    199  H   ALA A  20       3.580   2.031  11.703  1.00  0.00           H  \\nATOM    200  HA  ALA A  20       2.080   2.934  13.964  1.00  0.00           H  \\nATOM    201 1HB  ALA A  20       0.640   0.973  13.513  1.00  0.00           H  \\nATOM    202 2HB  ALA A  20       1.022   1.868  12.022  1.00  0.00           H  \\nATOM    203 3HB  ALA A  20       1.864   0.344  12.385  1.00  0.00           H  \\nTER                                                                             \\nATOM    205  N   ALA B  21      -5.257   0.409 -16.284  1.00  0.00           N  \\nATOM    206  CA  ALA B  21      -4.555  -0.560 -15.451  1.00  0.00           C  \\nATOM    207  C   ALA B  21      -3.845   0.124 -14.290  1.00  0.00           C  \\nATOM    208  O   ALA B  21      -3.852  -0.374 -13.165  1.00  0.00           O  \\nATOM    209  CB  ALA B  21      -3.562  -1.356 -16.284  1.00  0.00           C  \\nATOM    210 1H   ALA B  21      -5.713  -0.066 -17.037  1.00  0.00           H  \\nATOM    211 2H   ALA B  21      -5.938   0.891 -15.733  1.00  0.00           H  \\nATOM    212 3H   ALA B  21      -4.602   1.067 -16.654  1.00  0.00           H  \\nATOM    213  HA  ALA B  21      -5.294  -1.244 -15.033  1.00  0.00           H  \\nATOM    214 1HB  ALA B  21      -3.046  -2.075 -15.648  1.00  0.00           H  \\nATOM    215 2HB  ALA B  21      -4.093  -1.886 -17.075  1.00  0.00           H  \\nATOM    216 3HB  ALA B  21      -2.835  -0.678 -16.728  1.00  0.00           H  \\nATOM    217  N   ALA B  22      -3.232   1.270 -14.577  1.00  0.00           N  \\nATOM    218  CA  ALA B  22      -2.511   2.024 -13.558  1.00  0.00           C  \\nATOM    219  C   ALA B  22      -3.457   2.533 -12.478  1.00  0.00           C  \\nATOM    220  O   ALA B  22      -3.133   2.500 -11.291  1.00  0.00           O  \\nATOM    221  CB  ALA B  22      -1.758   3.185 -14.192  1.00  0.00           C  \\nATOM    222  H   ALA B  22      -3.268   1.624 -15.522  1.00  0.00           H  \\nATOM    223  HA  ALA B  22      -1.793   1.354 -13.085  1.00  0.00           H  \\nATOM    224 1HB  ALA B  22      -1.225   3.738 -13.419  1.00  0.00           H  \\nATOM    225 2HB  ALA B  22      -1.045   2.801 -14.921  1.00  0.00           H  \\nATOM    226 3HB  ALA B  22      -2.464   3.847 -14.690  1.00  0.00           H  \\nATOM    227  N   ALA B  23      -4.626   3.005 -12.895  1.00  0.00           N  \\nATOM    228  CA  ALA B  23      -5.618   3.521 -11.959  1.00  0.00           C  \\nATOM    229  C   ALA B  23      -6.131   2.421 -11.038  1.00  0.00           C  \\nATOM    230  O   ALA B  23      -6.317   2.638  -9.841  1.00  0.00           O  \\nATOM    231  CB  ALA B  23      -6.775   4.161 -12.712  1.00  0.00           C  \\nATOM    232  H   ALA B  23      -4.832   3.005 -13.884  1.00  0.00           H  \\nATOM    233  HA  ALA B  23      -5.137   4.278 -11.340  1.00  0.00           H  \\nATOM    234 1HB  ALA B  23      -7.507   4.542 -12.000  1.00  0.00           H  \\nATOM    235 2HB  ALA B  23      -6.400   4.984 -13.322  1.00  0.00           H  \\nATOM    236 3HB  ALA B  23      -7.246   3.419 -13.355  1.00  0.00           H  \\nATOM    237  N   ALA B  24      -6.358   1.240 -11.598  1.00  0.00           N  \\nATOM    238  CA  ALA B  24      -6.847   0.103 -10.827  1.00  0.00           C  \\nATOM    239  C   ALA B  24      -5.824  -0.340  -9.789  1.00  0.00           C  \\nATOM    240  O   ALA B  24      -6.176  -0.666  -8.656  1.00  0.00           O  \\nATOM    241  CB  ALA B  24      -7.196  -1.053 -11.753  1.00  0.00           C  \\nATOM    242  H   ALA B  24      -6.189   1.124 -12.586  1.00  0.00           H  \\nATOM    243  HA  ALA B  24      -7.747   0.415 -10.297  1.00  0.00           H  \\nATOM    244 1HB  ALA B  24      -7.559  -1.895 -11.163  1.00  0.00           H  \\nATOM    245 2HB  ALA B  24      -7.971  -0.739 -12.452  1.00  0.00           H  \\nATOM    246 3HB  ALA B  24      -6.309  -1.356 -12.307  1.00  0.00           H  \\nATOM    247  N   ALA B  25      -4.556  -0.348 -10.186  1.00  0.00           N  \\nATOM    248  CA  ALA B  25      -3.475  -0.751  -9.294  1.00  0.00           C  \\nATOM    249  C   ALA B  25      -3.337   0.214  -8.124  1.00  0.00           C  \\nATOM    250  O   ALA B  25      -3.120  -0.202  -6.986  1.00  0.00           O  \\nATOM    251  CB  ALA B  25      -2.163  -0.845 -10.060  1.00  0.00           C  \\nATOM    252  H   ALA B  25      -4.333  -0.067 -11.130  1.00  0.00           H  \\nATOM    253  HA  ALA B  25      -3.718  -1.734  -8.890  1.00  0.00           H  \\nATOM    254 1HB  ALA B  25      -1.366  -1.147  -9.381  1.00  0.00           H  \\nATOM    255 2HB  ALA B  25      -2.259  -1.582 -10.858  1.00  0.00           H  \\nATOM    256 3HB  ALA B  25      -1.924   0.126 -10.491  1.00  0.00           H  \\nATOM    257  N   ALA B  26      -3.464   1.505  -8.416  1.00  0.00           N  \\nATOM    258  CA  ALA B  26      -3.350   2.532  -7.387  1.00  0.00           C  \\nATOM    259  C   ALA B  26      -4.472   2.414  -6.364  1.00  0.00           C  \\nATOM    260  O   ALA B  26      -4.250   2.566  -5.163  1.00  0.00           O  \\nATOM    261  CB  ALA B  26      -3.352   3.917  -8.017  1.00  0.00           C  \\nATOM    262  H   ALA B  26      -3.644   1.779  -9.371  1.00  0.00           H  \\nATOM    263  HA  ALA B  26      -2.405   2.384  -6.864  1.00  0.00           H  \\nATOM    264 1HB  ALA B  26      -3.266   4.673  -7.237  1.00  0.00           H  \\nATOM    265 2HB  ALA B  26      -2.509   4.006  -8.703  1.00  0.00           H  \\nATOM    266 3HB  ALA B  26      -4.282   4.065  -8.565  1.00  0.00           H  \\nATOM    267  N   ALA B  27      -5.680   2.143  -6.843  1.00  0.00           N  \\nATOM    268  CA  ALA B  27      -6.837   2.003  -5.967  1.00  0.00           C  \\nATOM    269  C   ALA B  27      -6.687   0.802  -5.043  1.00  0.00           C  \\nATOM    270  O   ALA B  27      -7.025   0.870  -3.861  1.00  0.00           O  \\nATOM    271  CB  ALA B  27      -8.112   1.881  -6.789  1.00  0.00           C  \\nATOM    272  H   ALA B  27      -5.800   2.030  -7.840  1.00  0.00           H  \\nATOM    273  HA  ALA B  27      -6.902   2.897  -5.346  1.00  0.00           H  \\nATOM    274 1HB  ALA B  27      -8.967   1.778  -6.121  1.00  0.00           H  \\nATOM    275 2HB  ALA B  27      -8.237   2.774  -7.402  1.00  0.00           H  \\nATOM    276 3HB  ALA B  27      -8.048   1.006  -7.433  1.00  0.00           H  \\nATOM    277  N   ALA B  28      -6.178  -0.297  -5.583  1.00  0.00           N  \\nATOM    278  CA  ALA B  28      -5.978  -1.517  -4.809  1.00  0.00           C  \\nATOM    279  C   ALA B  28      -4.939  -1.310  -3.714  1.00  0.00           C  \\nATOM    280  O   ALA B  28      -5.104  -1.786  -2.591  1.00  0.00           O  \\nATOM    281  CB  ALA B  28      -5.563  -2.662  -5.722  1.00  0.00           C  \\nATOM    282  H   ALA B  28      -5.921  -0.292  -6.560  1.00  0.00           H  \\nATOM    283  HA  ALA B  28      -6.923  -1.772  -4.329  1.00  0.00           H  \\nATOM    284 1HB  ALA B  28      -5.418  -3.565  -5.129  1.00  0.00           H  \\nATOM    285 2HB  ALA B  28      -6.343  -2.836  -6.464  1.00  0.00           H  \\nATOM    286 3HB  ALA B  28      -4.633  -2.406  -6.227  1.00  0.00           H  \\nATOM    287  N   ALA B  29      -3.870  -0.597  -4.053  1.00  0.00           N  \\nATOM    288  CA  ALA B  29      -2.798  -0.326  -3.102  1.00  0.00           C  \\nATOM    289  C   ALA B  29      -3.290   0.539  -1.949  1.00  0.00           C  \\nATOM    290  O   ALA B  29      -2.935   0.309  -0.793  1.00  0.00           O  \\nATOM    291  CB  ALA B  29      -1.625   0.345  -3.803  1.00  0.00           C  \\nATOM    292  H   ALA B  29      -3.798  -0.234  -4.992  1.00  0.00           H  \\nATOM    293  HA  ALA B  29      -2.464  -1.277  -2.688  1.00  0.00           H  \\nATOM    294 1HB  ALA B  29      -0.833   0.541  -3.080  1.00  0.00           H  \\nATOM    295 2HB  ALA B  29      -1.247  -0.311  -4.587  1.00  0.00           H  \\nATOM    296 3HB  ALA B  29      -1.954   1.285  -4.244  1.00  0.00           H  \\nATOM    297  N   ALA B  30      -4.109   1.534  -2.273  1.00  0.00           N  \\nATOM    298  CA  ALA B  30      -4.648   2.435  -1.261  1.00  0.00           C  \\nATOM    299  C   ALA B  30      -5.559   1.693  -0.292  1.00  0.00           C  \\nATOM    300  O   ALA B  30      -5.523   1.933   0.915  1.00  0.00           O  \\nATOM    301  CB  ALA B  30      -5.400   3.583  -1.920  1.00  0.00           C  \\nATOM    302  H   ALA B  30      -4.363   1.667  -3.242  1.00  0.00           H  \\nATOM    303  HA  ALA B  30      -3.813   2.842  -0.691  1.00  0.00           H  \\nATOM    304 1HB  ALA B  30      -5.797   4.246  -1.152  1.00  0.00           H  \\nATOM    305 2HB  ALA B  30      -4.721   4.139  -2.566  1.00  0.00           H  \\nATOM    306 3HB  ALA B  30      -6.222   3.185  -2.515  1.00  0.00           H  \\nATOM    307  N   ALA B  31      -6.376   0.791  -0.821  1.00  0.00           N  \\nATOM    308  CA  ALA B  31      -7.294   0.012   0.000  1.00  0.00           C  \\nATOM    309  C   ALA B  31      -6.539  -0.905   0.954  1.00  0.00           C  \\nATOM    310  O   ALA B  31      -6.916  -1.054   2.116  1.00  0.00           O  \\nATOM    311  CB  ALA B  31      -8.233  -0.800  -0.880  1.00  0.00           C  \\nATOM    312  H   ALA B  31      -6.361   0.640  -1.820  1.00  0.00           H  \\nATOM    313  HA  ALA B  31      -7.884   0.705   0.600  1.00  0.00           H  \\nATOM    314 1HB  ALA B  31      -8.913  -1.377  -0.253  1.00  0.00           H  \\nATOM    315 2HB  ALA B  31      -8.809  -0.127  -1.516  1.00  0.00           H  \\nATOM    316 3HB  ALA B  31      -7.652  -1.478  -1.504  1.00  0.00           H  \\nATOM    317  N   ALA B  32      -5.471  -1.517   0.457  1.00  0.00           N  \\nATOM    318  CA  ALA B  32      -4.656  -2.420   1.261  1.00  0.00           C  \\nATOM    319  C   ALA B  32      -3.973  -1.678   2.402  1.00  0.00           C  \\nATOM    320  O   ALA B  32      -3.897  -2.179   3.524  1.00  0.00           O  \\nATOM    321  CB  ALA B  32      -3.622  -3.118   0.390  1.00  0.00           C  \\nATOM    322  H   ALA B  32      -5.213  -1.354  -0.506  1.00  0.00           H  \\nATOM    323  HA  ALA B  32      -5.314  -3.172   1.698  1.00  0.00           H  \\nATOM    324 1HB  ALA B  32      -3.021  -3.789   1.005  1.00  0.00           H  \\nATOM    325 2HB  ALA B  32      -4.128  -3.692  -0.386  1.00  0.00           H  \\nATOM    326 3HB  ALA B  32      -2.975  -2.374  -0.072  1.00  0.00           H  \\nATOM    327  N   ALA B  33      -3.478  -0.480   2.104  1.00  0.00           N  \\nATOM    328  CA  ALA B  33      -2.797   0.335   3.102  1.00  0.00           C  \\nATOM    329  C   ALA B  33      -3.748   0.752   4.216  1.00  0.00           C  \\nATOM    330  O   ALA B  33      -3.384   0.745   5.392  1.00  0.00           O  \\nATOM    331  CB  ALA B  33      -2.175   1.561   2.451  1.00  0.00           C  \\nATOM    332  H   ALA B  33      -3.578  -0.127   1.163  1.00  0.00           H  \\nATOM    333  HA  ALA B  33      -2.005  -0.268   3.548  1.00  0.00           H  \\nATOM    334 1HB  ALA B  33      -1.670   2.160   3.209  1.00  0.00           H  \\nATOM    335 2HB  ALA B  33      -1.454   1.247   1.697  1.00  0.00           H  \\nATOM    336 3HB  ALA B  33      -2.955   2.157   1.980  1.00  0.00           H  \\nATOM    337  N   ALA B  34      -4.970   1.115   3.838  1.00  0.00           N  \\nATOM    338  CA  ALA B  34      -5.973   1.536   4.809  1.00  0.00           C  \\nATOM    339  C   ALA B  34      -6.351   0.392   5.742  1.00  0.00           C  \\nATOM    340  O   ALA B  34      -6.517   0.589   6.945  1.00  0.00           O  \\nATOM    341  CB  ALA B  34      -7.208   2.068   4.097  1.00  0.00           C  \\nATOM    342  H   ALA B  34      -5.207   1.098   2.857  1.00  0.00           H  \\nATOM    343  HA  ALA B  34      -5.544   2.333   5.416  1.00  0.00           H  \\nATOM    344 1HB  ALA B  34      -7.948   2.379   4.835  1.00  0.00           H  \\nATOM    345 2HB  ALA B  34      -6.932   2.923   3.479  1.00  0.00           H  \\nATOM    346 3HB  ALA B  34      -7.630   1.287   3.468  1.00  0.00           H  \\nATOM    347  N   ALA B  35      -6.488  -0.804   5.185  1.00  0.00           N  \\nATOM    348  CA  ALA B  35      -6.844  -1.981   5.967  1.00  0.00           C  \\nATOM    349  C   ALA B  35      -5.750  -2.328   6.969  1.00  0.00           C  \\nATOM    350  O   ALA B  35      -6.033  -2.686   8.112  1.00  0.00           O  \\nATOM    351  CB  ALA B  35      -7.115  -3.165   5.050  1.00  0.00           C  \\nATOM    352  H   ALA B  35      -6.341  -0.903   4.190  1.00  0.00           H  \\nATOM    353  HA  ALA B  35      -7.751  -1.754   6.528  1.00  0.00           H  \\nATOM    354 1HB  ALA B  35      -7.380  -4.036   5.649  1.00  0.00           H  \\nATOM    355 2HB  ALA B  35      -7.939  -2.923   4.378  1.00  0.00           H  \\nATOM    356 3HB  ALA B  35      -6.223  -3.384   4.465  1.00  0.00           H  \\nATOM    357  N   ALA B  36      -4.501  -2.220   6.533  1.00  0.00           N  \\nATOM    358  CA  ALA B  36      -3.358  -2.521   7.387  1.00  0.00           C  \\nATOM    359  C   ALA B  36      -3.270  -1.547   8.554  1.00  0.00           C  \\nATOM    360  O   ALA B  36      -2.978  -1.940   9.683  1.00  0.00           O  \\nATOM    361  CB  ALA B  36      -2.070  -2.494   6.577  1.00  0.00           C  \\nATOM    362  H   ALA B  36      -4.336  -1.921   5.582  1.00  0.00           H  \\nATOM    363  HA  ALA B  36      -3.496  -3.522   7.797  1.00  0.00           H  \\nATOM    364 1HB  ALA B  36      -1.226  -2.721   7.229  1.00  0.00           H  \\nATOM    365 2HB  ALA B  36      -2.124  -3.238   5.782  1.00  0.00           H  \\nATOM    366 3HB  ALA B  36      -1.935  -1.506   6.141  1.00  0.00           H  \\nATOM    367  N   ALA B  37      -3.526  -0.273   8.269  1.00  0.00           N  \\nATOM    368  CA  ALA B  37      -3.472   0.762   9.294  1.00  0.00           C  \\nATOM    369  C   ALA B  37      -4.545   0.544  10.354  1.00  0.00           C  \\nATOM    370  O   ALA B  37      -4.297   0.718  11.547  1.00  0.00           O  \\nATOM    371  CB  ALA B  37      -3.623   2.140   8.665  1.00  0.00           C  \\nATOM    372  H   ALA B  37      -3.762  -0.018   7.320  1.00  0.00           H  \\nATOM    373  HA  ALA B  37      -2.501   0.703   9.785  1.00  0.00           H  \\nATOM    374 1HB  ALA B  37      -3.581   2.902   9.444  1.00  0.00           H  \\nATOM    375 2HB  ALA B  37      -2.815   2.305   7.952  1.00  0.00           H  \\nATOM    376 3HB  ALA B  37      -4.580   2.201   8.149  1.00  0.00           H  \\nATOM    377  N   ALA B  38      -5.738   0.161   9.913  1.00  0.00           N  \\nATOM    378  CA  ALA B  38      -6.847  -0.081  10.827  1.00  0.00           C  \\nATOM    379  C   ALA B  38      -6.556  -1.260  11.746  1.00  0.00           C  \\nATOM    380  O   ALA B  38      -6.860  -1.220  12.939  1.00  0.00           O  \\nATOM    381  CB  ALA B  38      -8.132  -0.323  10.049  1.00  0.00           C  \\nATOM    382  H   ALA B  38      -5.879   0.036   8.921  1.00  0.00           H  \\nATOM    383  HA  ALA B  38      -6.973   0.805  11.450  1.00  0.00           H  \\nATOM    384 1HB  ALA B  38      -8.951  -0.502  10.745  1.00  0.00           H  \\nATOM    385 2HB  ALA B  38      -8.359   0.553   9.440  1.00  0.00           H  \\nATOM    386 3HB  ALA B  38      -8.008  -1.191   9.403  1.00  0.00           H  \\nATOM    387  N   ALA B  39      -5.966  -2.310  11.191  1.00  0.00           N  \\nATOM    388  CA  ALA B  39      -5.629  -3.503  11.959  1.00  0.00           C  \\nATOM    389  C   ALA B  39      -4.578  -3.197  13.018  1.00  0.00           C  \\nATOM    390  O   ALA B  39      -4.662  -3.681  14.147  1.00  0.00           O  \\nATOM    391  CB  ALA B  39      -5.142  -4.609  11.034  1.00  0.00           C  \\nATOM    392  H   ALA B  39      -5.743  -2.285  10.206  1.00  0.00           H  \\nATOM    393  HA  ALA B  39      -6.530  -3.842  12.470  1.00  0.00           H  \\nATOM    394 1HB  ALA B  39      -4.894  -5.492  11.623  1.00  0.00           H  \\nATOM    395 2HB  ALA B  39      -5.926  -4.857  10.319  1.00  0.00           H  \\nATOM    396 3HB  ALA B  39      -4.256  -4.270  10.499  1.00  0.00           H  \\nATOM    397  N   ALA B  40      -3.591  -2.390  12.645  1.00  0.00           N  \\nATOM    398  CA  ALA B  40      -2.518  -2.016  13.558  1.00  0.00           C  \\nATOM    399  C   ALA B  40      -3.049  -1.194  14.725  1.00  0.00           C  \\nATOM    400  O   ALA B  40      -4.036  -0.470  14.589  1.00  0.00           O  \\nATOM    401  OXT ALA B  40      -2.505  -1.247  15.793  1.00  0.00           O  \\nATOM    402  CB  ALA B  40      -1.436  -1.244  12.817  1.00  0.00           C  \\nATOM    403  H   ALA B  40      -3.583  -2.026  11.703  1.00  0.00           H  \\nATOM    404  HA  ALA B  40      -2.084  -2.930  13.964  1.00  0.00           H  \\nATOM    405 1HB  ALA B  40      -0.642  -0.972  13.513  1.00  0.00           H  \\nATOM    406 2HB  ALA B  40      -1.024  -1.867  12.022  1.00  0.00           H  \\nATOM    407 3HB  ALA B  40      -1.864  -0.341  12.385  1.00  0.00           H  \\nTER                                                                             \\nREMARK PDBinfo-LABEL:    1 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    2 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    3 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    4 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    5 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    6 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    7 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    8 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    9 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   10 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   11 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   12 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   13 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   14 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   15 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   16 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   17 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   18 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   19 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   20 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   21 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   22 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   23 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   24 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   25 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   26 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   27 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   28 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   29 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   30 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   31 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   32 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   33 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   34 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   35 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   36 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   37 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   38 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   39 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   40 PEPTIDE_STUB_EXTENSION\\n\\n\",\"pdb\");\n\tviewer_16234209828572564.zoomTo();\n\tviewer_16234209828572564.setStyle({\"cartoon\": {\"color\": \"spectrum\"}, \"stick\": {\"colorscheme\": \"blackCarbon\", \"radius\": \"0.1\"}});\nviewer_16234209828572564.render();\n});\n</script>"
              }
            ],
            "_view_module": "@jupyter-widgets/output",
            "_model_module_version": "1.0.0",
            "_view_count": null,
            "_view_module_version": "1.0.0",
            "layout": "IPY_MODEL_45c97a73e3f94334aa04b0a6a15c866a",
            "_model_module": "@jupyter-widgets/output"
          }
        },
        "f701f8f9e4b94d419572b84530cb7745": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "SliderStyleModel",
          "state": {
            "_view_name": "StyleView",
            "handle_color": null,
            "_model_name": "SliderStyleModel",
            "description_width": "",
            "_view_module": "@jupyter-widgets/base",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.2.0",
            "_model_module": "@jupyter-widgets/controls"
          }
        },
        "0dfa943289ba4764887d5807751eb8c1": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "45c97a73e3f94334aa04b0a6a15c866a": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        }
      }
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "qlQwlDs_mdkr"
      },
      "source": [
        "# Packing & Design\n",
        "\n",
        "Rosetta uses a Monte Carlo optimization routine to pack side chains using a library of conformations, or rotamers. This operation can be used for side-chain packing for operations like refinement or for designing optimal sequences. This workshop will examine both capabilities.\n",
        "\n",
        "We will also cover many more ways to do protein design within Rosetta including parametric, denovo, and hydrogen-bond based design.  All of these can be useful tools for protein engineering.  \n",
        "\n",
        "### Suggested readings\n",
        "1. J. Desmet et al., The dead-end elimination theorem and its use in protein side-chain positioning, *Nature* 356, 539-543 (1992).\n",
        "2. B. Kuhlman & D. Baker, Native protein structures are close to optimal for their structures, *PNAS* 97, 10383, 2000.\n",
        "3. Denovo paper\n",
        "4. HbNet\n",
        "5. Andrew's packing paper\n",
        "6. Denovo design\n",
        "7. Parametric design paper"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "knjLQDtamdkt"
      },
      "source": [
        "**Chapter contributors:**\n",
        "\n",
        "- Jared Adolf-Bryfogle (Scripps; Institute for Protein Innovation)\n",
        "- Jason C. Klima (University of Washington; Lyell Immunopharma)\n",
        "- Jack Maguire (University of North Carolina)\n",
        "- Kathy Le (Johns Hopkins University); parts of this chapter were adapted from the [PyRosetta book](https://www.amazon.com/PyRosetta-Interactive-Platform-Structure-Prediction-ebook/dp/B01N21DRY8) (J. J. Gray, S. Chaudhury, S. Lyskov, J. Labonte)."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "6bc0DjS2mdku"
      },
      "source": [
        "# Side Chain Conformations and Dunbrack Energies\n",
        "Keywords: phi(), psi(), energies(), residue_total_energies()\n",
        "\n",
        "# References\n",
        "1. Original Rotamer paper\n",
        "2. New rotamer paper\n",
        "\n",
        "# Recommended Resources\n",
        "1. Rotamer Library Homepage: http://dunbrack.fccc.edu/bbdep2010/\n",
        "2. Rotamer Library Interactive Overview: http://dunbrack.fccc.edu/bbdep2010/ImagesMovies.php"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "f74ScB2vmdku"
      },
      "source": [
        "**Only run one of the next two cells.** If you are using Sergey's \"wheel install,\" then you should run the second one; if you're using the \"minsizerel\" install, use the first one."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "A_-iYe74mdku",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "1f02ba58-6005-4683-8f5a-83d041440aeb"
      },
      "source": [
        "# Notebook setup\n",
        "import sys\n",
        "if 'google.colab' in sys.modules:\n",
        "    !pip install pyrosettacolabsetup\n",
        "    import pyrosettacolabsetup\n",
        "    pyrosettacolabsetup.setup()\n",
        "    print (\"Notebook is set for PyRosetta use in Colab.  Have fun!\")\n"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Requirement already satisfied: pyrosettacolabsetup in /usr/local/lib/python3.7/dist-packages (0.6)\n",
            "Drive already mounted at /content/google_drive; to attempt to forcibly remount, call drive.mount(\"/content/google_drive\", force_remount=True).\n",
            "Notebook is set for PyRosetta use in Colab.  Have fun!\n",
            "Notebook is set for PyRosetta use in Colab.  Have fun!\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "PlXPz5Frmdkv",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "6e8009a1-ec46-460c-9b55-bf355992901a"
      },
      "source": [
        "# Wheel install setup. This takes ~1.5 minutes each time you start a new notebook or do a \"factory reset\".\n",
        "\n",
        "# Mounting Google Drive and add it to Python sys path  \n",
        "\n",
        "google_drive_mount_point = '/content/google_drive'\n",
        "\n",
        "import os, sys, time\n",
        "\n",
        "if 'google.colab' in sys.modules:\n",
        "    from google.colab import drive\n",
        "    drive.mount(google_drive_mount_point)\n",
        "\n",
        "if not os.getenv(\"DEBUG\"):\n",
        "    google_drive = google_drive_mount_point + '/My Drive'\n",
        "\n",
        "if not os.getenv(\"DEBUG\"):\n",
        "    # installing PyRosetta\n",
        "    if sys.version_info.major != 3 or sys.version_info.minor != 7:\n",
        "        print('Need Python-3.7 to run!')\n",
        "        sys.exit(1)\n",
        "\n",
        "    # upload PyRosetta Linux WHEEL package into your google drive and put it into /PyRosetta dir\n",
        "    # or alternatively you can download PyRosetta directly from GrayLab web site (but this might take some time!)\n",
        "    #!mkdir $notebook_path/PyRosetta\n",
        "    #!cd $notebook_path/PyRosetta && wget --user USERNAME --password PASSWORD https://graylab.jhu.edu/download/PyRosetta4/archive/release/PyRosetta4.Release.python37.ubuntu.wheel/latest.html   \n",
        "\n",
        "    pyrosetta_distr_path = google_drive + '/PyRosetta' \n",
        "    \n",
        "    # finding path to wheel package, if multiple packages is found take first one\n",
        "    # replace this with `wheel_path = pyrosetta_distr_path + /<wheel-file-name>.whl` if you want to use particular whl file\n",
        "    wheel_path = pyrosetta_distr_path + '/' + [ f for f in os.listdir(pyrosetta_distr_path) if f.endswith('.whl')][0]\n",
        "    \n",
        "    print(f'Using PyRosetta wheel package: {wheel_path}')\n",
        "\n",
        "    !pip3 install '{wheel_path}'"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Drive already mounted at /content/google_drive; to attempt to forcibly remount, call drive.mount(\"/content/google_drive\", force_remount=True).\n",
            "Using PyRosetta wheel package: /content/google_drive/My Drive/PyRosetta/pyrosetta-2020.50.post0.dev0+970.commits.3700df14560-cp37-cp37m-linux_x86_64.whl\n",
            "Requirement already satisfied: pyrosetta==2020.50.post0.dev0+970.commits.3700df14560 from file:///content/google_drive/My%20Drive/PyRosetta/pyrosetta-2020.50.post0.dev0%2B970.commits.3700df14560-cp37-cp37m-linux_x86_64.whl in /usr/local/lib/python3.7/dist-packages (2020.50.post0.dev0+970.commits.3700df14560)\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "fEwZViBMmdkv"
      },
      "source": [
        "*Warning*: This notebook uses `pyrosetta.distributed.viewer` code, which runs in `jupyter notebook` and might not run if you're using `jupyterlab`."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "9AH1IVAmmdkw"
      },
      "source": [
        "Now, the next couple of steps are a little confusing. First, you're going to have to `pip install` the `py3Dmol` package. (You have to pip install it every time you intend to use the py3Dmol viewer.) Next, you have to \"restart runtime\" (Select the \"Runtime\" menu at the top, and then select \"Restart Runtime\" -- caution, do **not** select \"Factory reset runtime\"). Finally, you have to _rerun_ whichever cell at the top of the notebook that you first ran. If you spent a minute and a half running the wheel install above, fear not: it is nearly instantaneous the second time you run that cell."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "_93Aovuemdkw",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "0fe30252-a2ae-4662-9b7c-d6177614f384"
      },
      "source": [
        "#step 1: pip install py3Dmol\n",
        "!pip install py3Dmol --user"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Requirement already satisfied: py3Dmol in /root/.local/lib/python3.7/site-packages (0.9.1)\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "ElkWghW4mdkw"
      },
      "source": [
        "Step 2: restart runtime\n",
        "\n",
        "Step 3: re-run either the minsizerel install or the wheel install cells from the top of the notebook.\n",
        "\n",
        "OK. After those three steps, we're ready to use the viewer; we just won't use it right away."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "252cxDkXmdkx",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 17
        },
        "outputId": "43513449-0785-475a-89da-a61184fda399"
      },
      "source": [
        "import logging\n",
        "logging.basicConfig(level=logging.INFO)\n",
        "import pyrosetta\n",
        "import pyrosetta.toolbox\n",
        "\n",
        "from IPython.core.display import display, HTML\n",
        "display(HTML(\"<style>.container { width:100% !important; }</style>\"))"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/html": [
              "<style>.container { width:100% !important; }</style>"
            ],
            "text/plain": [
              "<IPython.core.display.HTML object>"
            ]
          },
          "metadata": {
            "tags": []
          }
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "7YRE-HD5mdkx"
      },
      "source": [
        "Begin by moving to the directory with your pdb files and loading cetuximab from 1YY8.clean.pdb used in previous workshops.\n",
        "\n",
        "```\n",
        "cd google_drive/My\\ Drive/pyrbc_202103_notebooks/\n",
        "pose = pose_from_pdb(\"inputs/1YY8.clean.pdb\")\n",
        "start_pose = Pose()\n",
        "start_pose.assign(pose)\n",
        "```"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "CDvTWTddmdky",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "43bd808c-5e1a-4e49-f340-8557d5913fea"
      },
      "source": [
        "#put your CD command here\n",
        "%cd google_drive/My\\ Drive/temp_pyrbc_202103_notebooks/"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "/content/google_drive/My Drive/temp_pyrbc_202103_notebooks\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "RXEkZAjUmdky",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "37d2e22e-6439-401d-9e4d-4ca8d6fa609c"
      },
      "source": [
        "from pyrosetta import *\n",
        "from pyrosetta.teaching import *\n",
        "init()"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:pyrosetta.rosetta:Found rosetta database at: /usr/local/lib/python3.7/dist-packages/pyrosetta/database; using it....\n",
            "INFO:pyrosetta.rosetta:PyRosetta-4 2021 [Rosetta PyRosetta4.MinSizeRel.python37.ubuntu 2020.50.post.dev+970.commits.3700df145608444753aabbec9c4681ec9b21f74b 2021-02-24T13:24:53] retrieved from: http://www.pyrosetta.org\n",
            "(C) Copyright Rosetta Commons Member Institutions. Created in JHU by Sergey Lyskov and PyRosetta Team.\n",
            "INFO:rosetta:core.init: Checking for fconfig files in pwd and ./rosetta/flags\n",
            "INFO:rosetta:core.init: Rosetta version: PyRosetta4.MinSizeRel.python37.ubuntu r17132 2020.50.post.dev+970.commits.3700df14560 3700df145608444753aabbec9c4681ec9b21f74b http://www.pyrosetta.org 2021-02-24T13:24:53\n",
            "INFO:rosetta:core.init: command: PyRosetta -ex1 -ex2aro -database /usr/local/lib/python3.7/dist-packages/pyrosetta/database\n",
            "INFO:rosetta:basic.random.init_random_generator: 'RNG device' seed mode, using '/dev/urandom', seed=-1228256242 seed_offset=0 real_seed=-1228256242\n",
            "INFO:rosetta:basic.random.init_random_generator: RandomGenerator:init: Normal mode, seed=-1228256242 RG_type=mt19937\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "stream",
          "text": [
            "PyRosetta-4 2021 [Rosetta PyRosetta4.MinSizeRel.python37.ubuntu 2020.50.post.dev+970.commits.3700df145608444753aabbec9c4681ec9b21f74b 2021-02-24T13:24:53] retrieved from: http://www.pyrosetta.org\n",
            "(C) Copyright Rosetta Commons Member Institutions. Created in JHU by Sergey Lyskov and PyRosetta Team.\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "OJTsxcVRmdkz",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "a06b2c5a-5102-461c-f90d-051352b8cf4e"
      },
      "source": [
        "pose = pose_from_pdb(\"inputs/1YY8.clean.pdb\")\n",
        "start_pose = Pose()\n",
        "start_pose.assign(pose)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:core.chemical.GlobalResidueTypeSet: Finished initializing fa_standard residue type set.  Created 985 residue types\n",
            "INFO:rosetta:core.chemical.GlobalResidueTypeSet: Total time to initialize 1.07859 seconds.\n",
            "INFO:rosetta:core.import_pose.import_pose: File 'inputs/1YY8.clean.pdb' automatically determined to be of type PDB\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  CG  on residue ARG 18\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  CD  on residue ARG 18\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  NE  on residue ARG 18\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  CZ  on residue ARG 18\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  NH1 on residue ARG 18\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  NH2 on residue ARG 18\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  CG  on residue GLN:NtermProteinFull 214\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  CD  on residue GLN:NtermProteinFull 214\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  OE1 on residue GLN:NtermProteinFull 214\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  NE2 on residue GLN:NtermProteinFull 214\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  CG  on residue ARG 452\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  CD  on residue ARG 452\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  NE  on residue ARG 452\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  CZ  on residue ARG 452\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  NH1 on residue ARG 452\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  NH2 on residue ARG 452\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  CG  on residue GLN:NtermProteinFull 648\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  CD  on residue GLN:NtermProteinFull 648\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  OE1 on residue GLN:NtermProteinFull 648\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  NE2 on residue GLN:NtermProteinFull 648\n",
            "INFO:rosetta:core.conformation.Conformation: Found disulfide between residues 23 88\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 23 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 88 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 23 CYD\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 88 CYD\n",
            "INFO:rosetta:core.conformation.Conformation: Found disulfide between residues 134 194\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 134 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 194 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 134 CYD\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 194 CYD\n",
            "INFO:rosetta:core.conformation.Conformation: Found disulfide between residues 235 308\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 235 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 308 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 235 CYD\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 308 CYD\n",
            "INFO:rosetta:core.conformation.Conformation: Found disulfide between residues 359 415\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 359 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 415 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 359 CYD\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 415 CYD\n",
            "INFO:rosetta:core.conformation.Conformation: Found disulfide between residues 457 522\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 457 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 522 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 457 CYD\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 522 CYD\n",
            "INFO:rosetta:core.conformation.Conformation: Found disulfide between residues 568 628\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 568 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 628 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 568 CYD\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 628 CYD\n",
            "INFO:rosetta:core.conformation.Conformation: Found disulfide between residues 669 742\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 669 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 742 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 669 CYD\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 742 CYD\n",
            "INFO:rosetta:core.conformation.Conformation: Found disulfide between residues 793 849\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 793 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 849 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 793 CYD\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 849 CYD\n",
            "INFO:rosetta:core.pack.pack_missing_sidechains: packing residue number 18 because of missing atom number 6 atom name  CG\n",
            "INFO:rosetta:core.pack.pack_missing_sidechains: packing residue number 214 because of missing atom number 6 atom name  CG\n",
            "INFO:rosetta:core.pack.pack_missing_sidechains: packing residue number 452 because of missing atom number 6 atom name  CG\n",
            "INFO:rosetta:core.pack.pack_missing_sidechains: packing residue number 648 because of missing atom number 6 atom name  CG\n",
            "INFO:rosetta:core.pack.task: Packer task: initialize from command line()\n",
            "INFO:rosetta:core.scoring.ScoreFunctionFactory: SCOREFUNCTION: ref2015\n",
            "INFO:rosetta:core.scoring.etable: Starting energy table calculation\n",
            "INFO:rosetta:core.scoring.etable: smooth_etable: changing atr/rep split to bottom of energy well\n",
            "INFO:rosetta:core.scoring.etable: smooth_etable: spline smoothing lj etables (maxdis = 6)\n",
            "INFO:rosetta:core.scoring.etable: smooth_etable: spline smoothing solvation etables (max_dis = 6)\n",
            "INFO:rosetta:core.scoring.etable: Finished calculating energy tables.\n",
            "INFO:rosetta:basic.io.database: Database file opened: scoring/score_functions/hbonds/ref2015_params/HBPoly1D.csv\n",
            "INFO:rosetta:basic.io.database: Database file opened: scoring/score_functions/hbonds/ref2015_params/HBFadeIntervals.csv\n",
            "INFO:rosetta:basic.io.database: Database file opened: scoring/score_functions/hbonds/ref2015_params/HBEval.csv\n",
            "INFO:rosetta:basic.io.database: Database file opened: scoring/score_functions/hbonds/ref2015_params/DonStrength.csv\n",
            "INFO:rosetta:basic.io.database: Database file opened: scoring/score_functions/hbonds/ref2015_params/AccStrength.csv\n",
            "INFO:rosetta:basic.io.database: Database file opened: scoring/score_functions/rama/fd/all.ramaProb\n",
            "INFO:rosetta:basic.io.database: Database file opened: scoring/score_functions/rama/fd/prepro.ramaProb\n",
            "INFO:rosetta:basic.io.database: Database file opened: scoring/score_functions/omega/omega_ppdep.all.txt\n",
            "INFO:rosetta:basic.io.database: Database file opened: scoring/score_functions/omega/omega_ppdep.gly.txt\n",
            "INFO:rosetta:basic.io.database: Database file opened: scoring/score_functions/omega/omega_ppdep.pro.txt\n",
            "INFO:rosetta:basic.io.database: Database file opened: scoring/score_functions/omega/omega_ppdep.valile.txt\n",
            "INFO:rosetta:basic.io.database: Database file opened: scoring/score_functions/P_AA_pp/P_AA\n",
            "INFO:rosetta:basic.io.database: Database file opened: scoring/score_functions/P_AA_pp/P_AA_n\n",
            "INFO:rosetta:core.scoring.P_AA: shapovalov_lib::shap_p_aa_pp_smooth_level of 1( aka low_smooth ) got activated.\n",
            "INFO:rosetta:basic.io.database: Database file opened: scoring/score_functions/P_AA_pp/shapovalov/10deg/kappa131/a20.prop\n",
            "INFO:rosetta:basic.io.database: Database file opened: scoring/score_functions/elec_cp_reps.dat\n",
            "INFO:rosetta:core.scoring.elec.util: Read 40 countpair representative atoms\n",
            "INFO:rosetta:core.pack.dunbrack.RotamerLibrary: shapovalov_lib_fixes_enable option is true.\n",
            "INFO:rosetta:core.pack.dunbrack.RotamerLibrary: shapovalov_lib::shap_dun10_smooth_level of 1( aka lowest_smooth ) got activated.\n",
            "INFO:rosetta:core.pack.dunbrack.RotamerLibrary: Binary rotamer library selected: /usr/local/lib/python3.7/dist-packages/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/Dunbrack10.lib.bin\n",
            "INFO:rosetta:core.pack.dunbrack.RotamerLibrary: Using Dunbrack library binary file '/usr/local/lib/python3.7/dist-packages/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/Dunbrack10.lib.bin'.\n",
            "INFO:rosetta:core.pack.dunbrack.RotamerLibrary: Dunbrack 2010 library took 0.347156 seconds to load from binary\n",
            "INFO:rosetta:core.pack.pack_rotamers: built 85 rotamers at 4 positions.\n",
            "INFO:rosetta:core.pack.interaction_graph.interaction_graph_factory: Instantiating DensePDInteractionGraph\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "<pyrosetta.rosetta.core.pose.Pose at 0x7fe680fb56b0>"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 7
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "YAX3TOZFmdkz"
      },
      "source": [
        "__Question:__ What are the , , and  angles of residue K49?"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "XPQtqMG8mdkz",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "9a6f974c-6d3b-4f33-d204-09fa51e4af58"
      },
      "source": [
        "print(pose.residue(49).name())\n",
        "print(\"Phi: %.5f\\nPsi: %.5f\\n\" %(pose.phi(49), pose.psi(49)))\n",
        "print(\"Chi 1: %.5f\\nChi 2: %.5f\\nChi 3: %.5f\\nChi 4: %.5f\" %(pose.chi(1, 49), pose.chi(2, 49), pose.chi(3, 49), pose.chi(4, 49)))"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "LYS\n",
            "Phi: -122.41752\n",
            "Psi: 153.08267\n",
            "\n",
            "Chi 1: 68.12458\n",
            "Chi 2: -169.18576\n",
            "Chi 3: -175.37028\n",
            "Chi 4: -169.58806\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "Y92mQ_F2mdkz"
      },
      "source": [
        "Score your pose with the standard full-atom score function. What is the energy of K49? Note the Dunbrack energy component (`fa_dun`), which represents the side-chain conformational probability given phi/psi (Ie backbone-dependant). \n",
        "\n",
        "Any energy can be converted to a probability.  Use this energy (E) to calculate the approximate probability of the rotamer (p): p=e^(-E)\n",
        "```\n",
        "scorefxn = get_fa_scorefxn()\n",
        "scorefxn(pose)\n",
        "energies = pose.energies()\n",
        "print(energies.residue_total_energies(49))\n",
        "print(energies.residue_total_energies(49)[pyrosetta.rosetta.core.scoring.fa_dun])\n",
        "```"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "nbgrader": {
          "grade": true,
          "grade_id": "cell-f210fdda6b087d77",
          "locked": false,
          "points": 0,
          "schema_version": 3,
          "solution": true
        },
        "id": "pG6u3l-Bmdkz",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "9ccc7047-5885-4c44-8f10-9306d650dd13"
      },
      "source": [
        "scorefxn = get_fa_scorefxn()\n",
        "scorefxn(pose)\n",
        "energies = pose.energies()\n",
        "print(energies.residue_total_energies(49))\n",
        "print(energies.residue_total_energies(49)[pyrosetta.rosetta.core.scoring.fa_dun])"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:core.scoring.ScoreFunctionFactory: SCOREFUNCTION: ref2015\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "stream",
          "text": [
            "( fa_atr; -8.83354) ( fa_rep; 2.78019) ( fa_sol; 5.92847) ( fa_intra_atr; -1.68168) ( fa_intra_rep; 2.47533) ( fa_intra_sol; 1.20218) ( fa_intra_atr_xover4; -0.52776) ( fa_intra_rep_xover4; 0.191981) ( fa_intra_sol_xover4; 0.292855) ( fa_intra_atr_nonprotein; 0) ( fa_intra_rep_nonprotein; 0) ( fa_intra_sol_nonprotein; 0) ( fa_intra_RNA_base_phos_atr; 0) ( fa_intra_RNA_base_phos_rep; 0) ( fa_intra_RNA_base_phos_sol; 0) ( fa_atr_dummy; 0) ( fa_rep_dummy; 0) ( fa_sol_dummy; 0) ( fa_vdw_tinker; 0) ( lk_hack; 0) ( lk_ball; 0) ( lk_ball_wtd; 0.321463) ( lk_ball_iso; 0) ( lk_ball_bridge; 0) ( lk_ball_bridge_uncpl; 0) ( coarse_fa_atr; 0) ( coarse_fa_rep; 0) ( coarse_fa_sol; 0) ( coarse_beadlj; 0) ( mm_lj_intra_rep; 0) ( mm_lj_intra_atr; 0) ( mm_lj_inter_rep; 0) ( mm_lj_inter_atr; 0) ( mm_twist; 0) ( mm_bend; 0) ( mm_stretch; 0) ( lk_costheta; 0) ( lk_polar; 0) ( lk_nonpolar; 0) ( lk_polar_intra_RNA; 0) ( lk_nonpolar_intra_RNA; 0) ( fa_elec; -2.75782) ( fa_elec_bb_bb; 0) ( fa_elec_bb_sc; 0) ( fa_elec_sc_sc; 0) ( fa_intra_elec; 0) ( h2o_hbond; 0) ( dna_dr; 0) ( dna_bp; 0) ( dna_bs; 0) ( dna_ref; 0) ( peptide_bond; 0) ( pcs; 0) ( pcsTs1; 0) ( pcsTs2; 0) ( pcsTs3; 0) ( pcsTs4; 0) ( pcs2; 0) ( fastsaxs; 0) ( saxs_score; 0) ( saxs_cen_score; 0) ( saxs_fa_score; 0) ( pddf_score; 0) ( fiberdiffraction; 0) ( fiberdiffractiondens; 0) ( epr_deer_score; 0) ( fa_mbenv; 0) ( fa_mbsolv; 0) ( fa_elec_rna_phos_phos; 0) ( fa_elec_rna_phos_sugr; 0) ( fa_elec_rna_phos_base; 0) ( fa_elec_rna_sugr_sugr; 0) ( fa_elec_rna_sugr_base; 0) ( fa_elec_rna_base_base; 0) ( fa_elec_rna_phos_phos_fast; 0) ( fa_elec_rna_phos_sugr_fast; 0) ( fa_elec_rna_phos_base_fast; 0) ( fa_elec_rna_sugr_sugr_fast; 0) ( fa_elec_rna_sugr_base_fast; 0) ( fa_elec_rna_base_base_fast; 0) ( fa_elec_aro_aro; 0) ( fa_elec_aro_all; 0) ( hack_aro; 0) ( rna_fa_atr_base; 0) ( rna_fa_rep_base; 0) ( rna_data_backbone; 0) ( ch_bond; 0) ( ch_bond_bb_bb; 0) ( ch_bond_sc_sc; 0) ( ch_bond_bb_sc; 0) ( pro_close; 0) ( rama2b; 0) ( vdw; 0) ( cenpack; 0) ( cenpack_smooth; 0) ( cen_hb; 0) ( hybrid_vdw; 0) ( gauss; 0) ( rna_vdw; 0) ( rnp_vdw; 0) ( rna_base_backbone; 0) ( rna_backbone_backbone; 0) ( rna_repulsive; 0) ( rna_base_pair_pairwise; 0) ( rna_base_axis_pairwise; 0) ( rna_base_stagger_pairwise; 0) ( rna_base_stack_pairwise; 0) ( rna_base_stack_axis_pairwise; 0) ( rna_data_base; 0) ( rna_base_pair; 0) ( rna_base_axis; 0) ( rna_base_stagger; 0) ( rna_base_stack; 0) ( rna_base_stack_axis; 0) ( rnp_base_pair; 0) ( rnp_stack; 0) ( rnp_stack_xy; 0) ( rnp_pair; 0) ( rnp_pair_dist; 0) ( rnp_aa_to_rna_backbone; 0) ( rna_mg_point; 0) ( rna_mg_point_indirect; 0) ( mg; 0) ( mg_lig; 0) ( mg_sol; 0) ( mg_ref; 0) ( hoh_ref; 0) ( rna_torsion; 0) ( rna_torsion_sc; 0) ( rna_suite; 0) ( rna_jr_suite; 0) ( suiteness_bonus; 0) ( tna_suite; 0) ( rna_sugar_close; 0) ( fa_stack; 0) ( fa_stack_lower; 0) ( fa_stack_upper; 0) ( fa_stack_aro; 0) ( fa_stack_ext; 0) ( fa_stack_sol; 0) ( fa_stack_lr; 0) ( fa_stack_rna_protein; 0) ( fa_stack_rna_protein_aro; 0) ( ss_contact_worst; 0) ( stack_elec; 0) ( stack_elec_base_base; 0) ( stack_elec_base_bb; 0) ( alignment; 0) ( dna_bb_torsion; 0) ( dna_sugar_close; 0) ( dna_base_distance; 0) ( geom_sol_fast; 0) ( geom_sol_fast_intra_RNA; 0) ( fa_cust_pair_dist; 0) ( custom_atom_pair; 0) ( orbitals_hpol_bb; 0) ( pci_cation_pi; 0) ( pci_pi_pi; 0) ( pci_salt_bridge; 0) ( pci_hbond; 0) ( arg_cation_pi; 0) ( PyRosettaTwoBodyContextIndepenedentEnergy_first; 0) ( PyRosettaTwoBodyContextIndepenedentEnergy_163; 0) ( PyRosettaTwoBodyContextIndepenedentEnergy_164; 0) ( PyRosettaTwoBodyContextIndepenedentEnergy_165; 0) ( PyRosettaTwoBodyContextIndepenedentEnergy_166; 0) ( PyRosettaTwoBodyContextIndepenedentEnergy_167; 0) ( PyRosettaTwoBodyContextIndepenedentEnergy_168; 0) ( PyRosettaTwoBodyContextIndepenedentEnergy_169; 0) ( PyRosettaTwoBodyContextIndepenedentEnergy_170; 0) ( PyRosettaTwoBodyContextIndepenedentEnergy_171; 0) ( PyRosettaTwoBodyContextIndepenedentEnergy_last; 0) ( python; 0) ( fa_pair; 0) ( fa_pair_aro_aro; 0) ( fa_pair_aro_pol; 0) ( fa_pair_pol_pol; 0) ( hbond_sr_bb; 0) ( hbond_lr_bb; 0) ( hbond_bb_sc; 0) ( hbond_sr_bb_sc; 0) ( hbond_lr_bb_sc; 0) ( hbond_sc; 0) ( hbond_intra; 0) ( hbond_wat; 0) ( wat_entropy; 0) ( hbond; 0) ( npd_hbond_sr_bb; 0) ( npd_hbond_lr_bb; 0) ( npd_hbond_bb_sc; 0) ( npd_hbond_sr_bb_sc; 0) ( npd_hbond_lr_bb_sc; 0) ( npd_hbond_sc; 0) ( npd_hbond_intra; 0) ( npd_hbond; 0) ( fa_grpelec; 0) ( PyRosettaTwoBodyContextDependentEnergy_first; 0) ( PyRosettaTwoBodyContextDependentEnergy_198; 0) ( PyRosettaTwoBodyContextDependentEnergy_199; 0) ( PyRosettaTwoBodyContextDependentEnergy_200; 0) ( PyRosettaTwoBodyContextDependentEnergy_201; 0) ( PyRosettaTwoBodyContextDependentEnergy_202; 0) ( PyRosettaTwoBodyContextDependentEnergy_203; 0) ( PyRosettaTwoBodyContextDependentEnergy_204; 0) ( PyRosettaTwoBodyContextDependentEnergy_205; 0) ( PyRosettaTwoBodyContextDependentEnergy_206; 0) ( PyRosettaTwoBodyContextDependentEnergy_last; 0) ( interface_dd_pair; 0) ( geom_sol; 0) ( geom_sol_intra_RNA; 0) ( occ_sol_fitted; 0) ( occ_sol_fitted_onebody; 0) ( occ_sol_exact; 0) ( cen_rot_pair; 0) ( cen_rot_pair_ang; 0) ( cen_rot_pair_dih; 0) ( pair; 0) ( cen_pair_smooth; 0) ( Mpair; 0) ( mp_pair; 0) ( fa_mpsolv; 0) ( suck; 0) ( rna_rg; 0) ( rna_motif; 0) ( facts_elec; 0) ( facts_solv; 0) ( facts_sasa; 0) ( motif_dock; 0) ( pointwater; 0) ( goap; 0) ( goap_dist; 0) ( goap_angle; 0) ( approximate_buried_unsat_penalty; 0) ( interchain_pair; 0) ( interchain_vdw; 0) ( gb_elec; 0) ( multipole_elec; 0) ( fa_sasa; 0) ( dslf_ss_dst; 0) ( dslf_cs_ang; 0) ( dslf_ss_dih; 0) ( dslf_ca_dih; 0) ( dslf_cbs_ds; 0) ( dslf_fa13; 0) ( dslfc_cen_dst; 0) ( dslfc_cb_dst; 0) ( dslfc_ang; 0) ( dslfc_cb_dih; 0) ( dslfc_bb_dih; 0) ( dslfc_rot; 0) ( dslfc_trans; 0) ( dslfc_RT; 0) ( atom_pair_constraint; 0) ( base_pair_constraint; 0) ( coarse_chainbreak_constraint; 0) ( constant_constraint; 0) ( coordinate_constraint; 0) ( angle_constraint; 0) ( dihedral_constraint; 0) ( big_bin_constraint; 0) ( dunbrack_constraint; 0) ( site_constraint; 0) ( metalhash_constraint; 0) ( metalbinding_constraint; 0) ( rna_stub_coord_hack; 0) ( bond_geometry; 0) ( rna_bond_geometry; 0) ( Hpol_bond_geometry; 0) ( rama; 0) ( omega; 0.0116849) ( fa_dun; 3.97212) ( fa_dun_dev; 1.07017) ( fa_dun_rot; 2.90195) ( fa_dun_semi; 0) ( cen_rot_dun; 0) ( dna_chi; 0) ( p_aa_pp; 0.0112208) ( p_aa_ss; 0) ( yhh_planarity; 0) ( hxl_tors; 0) ( h2o_intra; 0) ( ref; -0.71458) ( ref_nc; 0) ( seqdep_ref; 0) ( nmer_ref; 0) ( nmer_pssm; 0) ( nmer_svm; 0) ( envsmooth; 0) ( e_pH; 0) ( rna_bulge; 0) ( dna_dihedral_bb; 0) ( dna_dihedral_chi; 0) ( dna_dihedral_sugar; 0) ( sugar_bb; 0) ( free_suite; 0) ( free_2HOprime; 0) ( free_side_chain; 0) ( free_base; 0) ( free_res; 0) ( free_dof; 0) ( intermol; 0) ( other_pose; 0) ( special_rot; 0) ( PB_elec; 0) ( cen_env_smooth; 0) ( cbeta_smooth; 0) ( cen_rot_env; 0) ( cen_rot_cbeta; 0) ( env; 0) ( cbeta; 0) ( DFIRE; 0) ( Menv; 0) ( Mcbeta; 0) ( Menv_non_helix; 0) ( Menv_termini; 0) ( Menv_tm_proj; 0) ( Mlipo; 0) ( rg; 0) ( rg_local; 0) ( co; 0) ( hs_pair; 0) ( ss_pair; 0) ( rsigma; 0) ( sheet; 0) ( burial_v2; 0) ( burial; 0) ( abego; 0) ( covalent_labeling; 0) ( covalent_labeling_fa; 0) ( hrf_ms_labeling; 0) ( hrf_dynamics; 0) ( rnp_env; 0) ( loop_close; 0) ( missing_res; 0) ( mp_env; 0) ( mp_cbeta; 0) ( mp_lipo; 0) ( mp_termini; 0) ( mp_nonhelix; 0) ( mp_tmproj; 0) ( fa_mpenv; 0) ( fa_mpenv_smooth; 0) ( fa_water_to_bilayer; 0) ( mp_res_lipo; 0) ( span_ins; 0) ( mp_span_ang; 0) ( mp_helicality; 0) ( natbias_ss; 0) ( natbias_hs; 0) ( natbias_hh; 0) ( natbias_stwist; 0) ( aa_cmp; 0) ( dock_ens_conf; 0) ( csa; 0) ( dc; 0) ( rdc; 0) ( rdc_segments; 0) ( rdc_rohl; 0) ( nmr_pcs; 0) ( nmr_rdc; 0) ( nmr_pre; 0) ( cen_pair_motifs; 0) ( cen_pair_motif_degree; 0) ( holes; 0) ( holes_decoy; 0) ( holes_resl; 0) ( holes_min; 0) ( holes_min_mean; 0) ( rna_chem_shift; 0) ( rna_chem_map; 0) ( rna_chem_map_lores; 0) ( rna_partition; 0) ( dab_sasa; 0) ( dab_sev; 0) ( sa; 0) ( d2h_sa; 0) ( ProQM; 0) ( ProQ; 0) ( interchain_env; 0) ( interchain_contact; 0) ( chainbreak; 0) ( linear_chainbreak; 0) ( overlap_chainbreak; 0) ( distance_chainbreak; 0) ( dof_constraint; 0) ( branch_conn; 0) ( linear_branch_conn; 0) ( rama_prepro; -0.0498918) ( paa_abego3; 0) ( cart_bonded; 0) ( cart_bonded_angle; 0) ( cart_bonded_length; 0) ( cart_bonded_ring; 0) ( cart_bonded_torsion; 0) ( cart_bonded_proper; 0) ( cart_bonded_improper; 0) ( gen_bonded; 0) ( gen_bonded_bond; 0) ( gen_bonded_angle; 0) ( gen_bonded_torsion; 0) ( gen_bonded_improper; 0) ( neigh_vect; 0) ( neigh_count; 0) ( neigh_vect_raw; 0) ( symE_bonus; 0) ( sym_lig; 0) ( mhc_epitope; 0) ( pack_stat; 0) ( rms_energy; 0) ( res_type_constraint; 0) ( res_type_linking_constraint; 0) ( pocket_constraint; 0) ( backbone_stub_constraint; 0) ( backbone_stub_linear_constraint; 0) ( surface; 0) ( p_aa; 0) ( unfolded; 0) ( split_unfolded_two_body; 0) ( fa_atr_ref; 0) ( fa_rep_ref; 0) ( fa_sol_ref; 0) ( fa_elec_ref; 0) ( hbond_ref; 0) ( dslf_fa13_ref; 0) ( fa_intra_atr_ref; 0) ( fa_intra_rep_ref; 0) ( fa_intra_sol_ref; 0) ( pro_close_ref; 0) ( fa_dun_ref; 0) ( fa_dun_dev_ref; 0) ( fa_dun_rot_ref; 0) ( fa_dun_semi_ref; 0) ( rama_ref; 0) ( p_aa_pp_ref; 0) ( omega_ref; 0) ( mm_lj_intra_rep_ref; 0) ( mm_lj_intra_atr_ref; 0) ( mm_twist_ref; 0) ( elec_dens_fast; 0) ( elec_dens_window; 0) ( elec_dens_whole_structure_ca; 0) ( elec_dens_whole_structure_allatom; 0) ( elec_dens_atomwise; 0) ( grid_vdw; 0) ( xtal_ml; 0) ( xtal_rwork; 0) ( xtal_rfree; 0) ( hpatch; 0) ( Menv_smooth; 0) ( wat_desolv; 0) ( ring_close; 0) ( aa_repeat; 0) ( aa_composition; 0) ( aspartimide_penalty; 0) ( hbnet; 0) ( buried_unsatisfied_penalty; 0) ( netcharge; 0) ( voids_penalty; 0) ( sap_constraint; 0) ( dna_env; 0) ( dna_pair; 0) ( dump_trajectory; 0) ( PyRosettaEnergy_first; 0) ( PyRosettaEnergy_464; 0) ( PyRosettaEnergy_465; 0) ( PyRosettaEnergy_466; 0) ( PyRosettaEnergy_467; 0) ( PyRosettaEnergy_468; 0) ( PyRosettaEnergy_469; 0) ( PyRosettaEnergy_470; 0) ( PyRosettaEnergy_471; 0) ( PyRosettaEnergy_472; 0) ( PyRosettaEnergy_last; 0) ( sidechain_neighbors; 0) ( target_clash; 0) ( membrane_span_constraint; 0) ( membrane_span_term_z_constraint; 0) ( aromatic_restraint; 0) ( total_score; -1.45222) ( dummy_score_type; 0) \n",
            "3.9721243480599346\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "xsi5IbaBmdkz"
      },
      "source": [
        "Use `pose.set_chi(<i>, <res_num>, <chi>)` to set the side chain of residue 49 to the all-anti conformation. (Here, `i` is the  index, and `chi` is the new torsion angle in degrees.) Re-score the pose and note the Dunbrack energy."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "nbgrader": {
          "grade": true,
          "grade_id": "cell-74a711eb920136be",
          "locked": false,
          "points": 0,
          "schema_version": 3,
          "solution": true
        },
        "id": "rpqHX2TWmdk0",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "7cfacb75-bcac-488e-8872-2e26524bd5fc"
      },
      "source": [
        "for i in range(1, 5):\n",
        "    pose.set_chi(i, 49, 180)\n",
        "    \n",
        "scorefxn = get_fa_scorefxn()\n",
        "scorefxn(pose)\n",
        "energies = pose.energies()\n",
        "print(energies.residue_total_energies(49))\n",
        "print(energies.residue_total_energies(49)[pyrosetta.rosetta.core.scoring.fa_dun])"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:core.scoring.ScoreFunctionFactory: SCOREFUNCTION: ref2015\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "stream",
          "text": [
            "( fa_atr; -11.6375) ( fa_rep; 789.493) ( fa_sol; 11.6773) ( fa_intra_atr; -1.44441) ( fa_intra_rep; 1.80557) ( fa_intra_sol; 0.785201) ( fa_intra_atr_xover4; -0.343391) ( fa_intra_rep_xover4; 0.112709) ( fa_intra_sol_xover4; 0.105435) ( fa_intra_atr_nonprotein; 0) ( fa_intra_rep_nonprotein; 0) ( fa_intra_sol_nonprotein; 0) ( fa_intra_RNA_base_phos_atr; 0) ( fa_intra_RNA_base_phos_rep; 0) ( fa_intra_RNA_base_phos_sol; 0) ( fa_atr_dummy; 0) ( fa_rep_dummy; 0) ( fa_sol_dummy; 0) ( fa_vdw_tinker; 0) ( lk_hack; 0) ( lk_ball; 0) ( lk_ball_wtd; 0.302965) ( lk_ball_iso; 0) ( lk_ball_bridge; 0) ( lk_ball_bridge_uncpl; 0) ( coarse_fa_atr; 0) ( coarse_fa_rep; 0) ( coarse_fa_sol; 0) ( coarse_beadlj; 0) ( mm_lj_intra_rep; 0) ( mm_lj_intra_atr; 0) ( mm_lj_inter_rep; 0) ( mm_lj_inter_atr; 0) ( mm_twist; 0) ( mm_bend; 0) ( mm_stretch; 0) ( lk_costheta; 0) ( lk_polar; 0) ( lk_nonpolar; 0) ( lk_polar_intra_RNA; 0) ( lk_nonpolar_intra_RNA; 0) ( fa_elec; -9.02391) ( fa_elec_bb_bb; 0) ( fa_elec_bb_sc; 0) ( fa_elec_sc_sc; 0) ( fa_intra_elec; 0) ( h2o_hbond; 0) ( dna_dr; 0) ( dna_bp; 0) ( dna_bs; 0) ( dna_ref; 0) ( peptide_bond; 0) ( pcs; 0) ( pcsTs1; 0) ( pcsTs2; 0) ( pcsTs3; 0) ( pcsTs4; 0) ( pcs2; 0) ( fastsaxs; 0) ( saxs_score; 0) ( saxs_cen_score; 0) ( saxs_fa_score; 0) ( pddf_score; 0) ( fiberdiffraction; 0) ( fiberdiffractiondens; 0) ( epr_deer_score; 0) ( fa_mbenv; 0) ( fa_mbsolv; 0) ( fa_elec_rna_phos_phos; 0) ( fa_elec_rna_phos_sugr; 0) ( fa_elec_rna_phos_base; 0) ( fa_elec_rna_sugr_sugr; 0) ( fa_elec_rna_sugr_base; 0) ( fa_elec_rna_base_base; 0) ( fa_elec_rna_phos_phos_fast; 0) ( fa_elec_rna_phos_sugr_fast; 0) ( fa_elec_rna_phos_base_fast; 0) ( fa_elec_rna_sugr_sugr_fast; 0) ( fa_elec_rna_sugr_base_fast; 0) ( fa_elec_rna_base_base_fast; 0) ( fa_elec_aro_aro; 0) ( fa_elec_aro_all; 0) ( hack_aro; 0) ( rna_fa_atr_base; 0) ( rna_fa_rep_base; 0) ( rna_data_backbone; 0) ( ch_bond; 0) ( ch_bond_bb_bb; 0) ( ch_bond_sc_sc; 0) ( ch_bond_bb_sc; 0) ( pro_close; 0) ( rama2b; 0) ( vdw; 0) ( cenpack; 0) ( cenpack_smooth; 0) ( cen_hb; 0) ( hybrid_vdw; 0) ( gauss; 0) ( rna_vdw; 0) ( rnp_vdw; 0) ( rna_base_backbone; 0) ( rna_backbone_backbone; 0) ( rna_repulsive; 0) ( rna_base_pair_pairwise; 0) ( rna_base_axis_pairwise; 0) ( rna_base_stagger_pairwise; 0) ( rna_base_stack_pairwise; 0) ( rna_base_stack_axis_pairwise; 0) ( rna_data_base; 0) ( rna_base_pair; 0) ( rna_base_axis; 0) ( rna_base_stagger; 0) ( rna_base_stack; 0) ( rna_base_stack_axis; 0) ( rnp_base_pair; 0) ( rnp_stack; 0) ( rnp_stack_xy; 0) ( rnp_pair; 0) ( rnp_pair_dist; 0) ( rnp_aa_to_rna_backbone; 0) ( rna_mg_point; 0) ( rna_mg_point_indirect; 0) ( mg; 0) ( mg_lig; 0) ( mg_sol; 0) ( mg_ref; 0) ( hoh_ref; 0) ( rna_torsion; 0) ( rna_torsion_sc; 0) ( rna_suite; 0) ( rna_jr_suite; 0) ( suiteness_bonus; 0) ( tna_suite; 0) ( rna_sugar_close; 0) ( fa_stack; 0) ( fa_stack_lower; 0) ( fa_stack_upper; 0) ( fa_stack_aro; 0) ( fa_stack_ext; 0) ( fa_stack_sol; 0) ( fa_stack_lr; 0) ( fa_stack_rna_protein; 0) ( fa_stack_rna_protein_aro; 0) ( ss_contact_worst; 0) ( stack_elec; 0) ( stack_elec_base_base; 0) ( stack_elec_base_bb; 0) ( alignment; 0) ( dna_bb_torsion; 0) ( dna_sugar_close; 0) ( dna_base_distance; 0) ( geom_sol_fast; 0) ( geom_sol_fast_intra_RNA; 0) ( fa_cust_pair_dist; 0) ( custom_atom_pair; 0) ( orbitals_hpol_bb; 0) ( pci_cation_pi; 0) ( pci_pi_pi; 0) ( pci_salt_bridge; 0) ( pci_hbond; 0) ( arg_cation_pi; 0) ( PyRosettaTwoBodyContextIndepenedentEnergy_first; 0) ( PyRosettaTwoBodyContextIndepenedentEnergy_163; 0) ( PyRosettaTwoBodyContextIndepenedentEnergy_164; 0) ( PyRosettaTwoBodyContextIndepenedentEnergy_165; 0) ( PyRosettaTwoBodyContextIndepenedentEnergy_166; 0) ( PyRosettaTwoBodyContextIndepenedentEnergy_167; 0) ( PyRosettaTwoBodyContextIndepenedentEnergy_168; 0) ( PyRosettaTwoBodyContextIndepenedentEnergy_169; 0) ( PyRosettaTwoBodyContextIndepenedentEnergy_170; 0) ( PyRosettaTwoBodyContextIndepenedentEnergy_171; 0) ( PyRosettaTwoBodyContextIndepenedentEnergy_last; 0) ( python; 0) ( fa_pair; 0) ( fa_pair_aro_aro; 0) ( fa_pair_aro_pol; 0) ( fa_pair_pol_pol; 0) ( hbond_sr_bb; 0) ( hbond_lr_bb; 0) ( hbond_bb_sc; 0) ( hbond_sr_bb_sc; 0) ( hbond_lr_bb_sc; 0) ( hbond_sc; -0.274355) ( hbond_intra; 0) ( hbond_wat; 0) ( wat_entropy; 0) ( hbond; 0) ( npd_hbond_sr_bb; 0) ( npd_hbond_lr_bb; 0) ( npd_hbond_bb_sc; 0) ( npd_hbond_sr_bb_sc; 0) ( npd_hbond_lr_bb_sc; 0) ( npd_hbond_sc; 0) ( npd_hbond_intra; 0) ( npd_hbond; 0) ( fa_grpelec; 0) ( PyRosettaTwoBodyContextDependentEnergy_first; 0) ( PyRosettaTwoBodyContextDependentEnergy_198; 0) ( PyRosettaTwoBodyContextDependentEnergy_199; 0) ( PyRosettaTwoBodyContextDependentEnergy_200; 0) ( PyRosettaTwoBodyContextDependentEnergy_201; 0) ( PyRosettaTwoBodyContextDependentEnergy_202; 0) ( PyRosettaTwoBodyContextDependentEnergy_203; 0) ( PyRosettaTwoBodyContextDependentEnergy_204; 0) ( PyRosettaTwoBodyContextDependentEnergy_205; 0) ( PyRosettaTwoBodyContextDependentEnergy_206; 0) ( PyRosettaTwoBodyContextDependentEnergy_last; 0) ( interface_dd_pair; 0) ( geom_sol; 0) ( geom_sol_intra_RNA; 0) ( occ_sol_fitted; 0) ( occ_sol_fitted_onebody; 0) ( occ_sol_exact; 0) ( cen_rot_pair; 0) ( cen_rot_pair_ang; 0) ( cen_rot_pair_dih; 0) ( pair; 0) ( cen_pair_smooth; 0) ( Mpair; 0) ( mp_pair; 0) ( fa_mpsolv; 0) ( suck; 0) ( rna_rg; 0) ( rna_motif; 0) ( facts_elec; 0) ( facts_solv; 0) ( facts_sasa; 0) ( motif_dock; 0) ( pointwater; 0) ( goap; 0) ( goap_dist; 0) ( goap_angle; 0) ( approximate_buried_unsat_penalty; 0) ( interchain_pair; 0) ( interchain_vdw; 0) ( gb_elec; 0) ( multipole_elec; 0) ( fa_sasa; 0) ( dslf_ss_dst; 0) ( dslf_cs_ang; 0) ( dslf_ss_dih; 0) ( dslf_ca_dih; 0) ( dslf_cbs_ds; 0) ( dslf_fa13; 0) ( dslfc_cen_dst; 0) ( dslfc_cb_dst; 0) ( dslfc_ang; 0) ( dslfc_cb_dih; 0) ( dslfc_bb_dih; 0) ( dslfc_rot; 0) ( dslfc_trans; 0) ( dslfc_RT; 0) ( atom_pair_constraint; 0) ( base_pair_constraint; 0) ( coarse_chainbreak_constraint; 0) ( constant_constraint; 0) ( coordinate_constraint; 0) ( angle_constraint; 0) ( dihedral_constraint; 0) ( big_bin_constraint; 0) ( dunbrack_constraint; 0) ( site_constraint; 0) ( metalhash_constraint; 0) ( metalbinding_constraint; 0) ( rna_stub_coord_hack; 0) ( bond_geometry; 0) ( rna_bond_geometry; 0) ( Hpol_bond_geometry; 0) ( rama; 0) ( omega; 0.0116849) ( fa_dun; 3.79108) ( fa_dun_dev; 0.030004) ( fa_dun_rot; 3.76108) ( fa_dun_semi; 0) ( cen_rot_dun; 0) ( dna_chi; 0) ( p_aa_pp; 0.0112208) ( p_aa_ss; 0) ( yhh_planarity; 0) ( hxl_tors; 0) ( h2o_intra; 0) ( ref; -0.71458) ( ref_nc; 0) ( seqdep_ref; 0) ( nmer_ref; 0) ( nmer_pssm; 0) ( nmer_svm; 0) ( envsmooth; 0) ( e_pH; 0) ( rna_bulge; 0) ( dna_dihedral_bb; 0) ( dna_dihedral_chi; 0) ( dna_dihedral_sugar; 0) ( sugar_bb; 0) ( free_suite; 0) ( free_2HOprime; 0) ( free_side_chain; 0) ( free_base; 0) ( free_res; 0) ( free_dof; 0) ( intermol; 0) ( other_pose; 0) ( special_rot; 0) ( PB_elec; 0) ( cen_env_smooth; 0) ( cbeta_smooth; 0) ( cen_rot_env; 0) ( cen_rot_cbeta; 0) ( env; 0) ( cbeta; 0) ( DFIRE; 0) ( Menv; 0) ( Mcbeta; 0) ( Menv_non_helix; 0) ( Menv_termini; 0) ( Menv_tm_proj; 0) ( Mlipo; 0) ( rg; 0) ( rg_local; 0) ( co; 0) ( hs_pair; 0) ( ss_pair; 0) ( rsigma; 0) ( sheet; 0) ( burial_v2; 0) ( burial; 0) ( abego; 0) ( covalent_labeling; 0) ( covalent_labeling_fa; 0) ( hrf_ms_labeling; 0) ( hrf_dynamics; 0) ( rnp_env; 0) ( loop_close; 0) ( missing_res; 0) ( mp_env; 0) ( mp_cbeta; 0) ( mp_lipo; 0) ( mp_termini; 0) ( mp_nonhelix; 0) ( mp_tmproj; 0) ( fa_mpenv; 0) ( fa_mpenv_smooth; 0) ( fa_water_to_bilayer; 0) ( mp_res_lipo; 0) ( span_ins; 0) ( mp_span_ang; 0) ( mp_helicality; 0) ( natbias_ss; 0) ( natbias_hs; 0) ( natbias_hh; 0) ( natbias_stwist; 0) ( aa_cmp; 0) ( dock_ens_conf; 0) ( csa; 0) ( dc; 0) ( rdc; 0) ( rdc_segments; 0) ( rdc_rohl; 0) ( nmr_pcs; 0) ( nmr_rdc; 0) ( nmr_pre; 0) ( cen_pair_motifs; 0) ( cen_pair_motif_degree; 0) ( holes; 0) ( holes_decoy; 0) ( holes_resl; 0) ( holes_min; 0) ( holes_min_mean; 0) ( rna_chem_shift; 0) ( rna_chem_map; 0) ( rna_chem_map_lores; 0) ( rna_partition; 0) ( dab_sasa; 0) ( dab_sev; 0) ( sa; 0) ( d2h_sa; 0) ( ProQM; 0) ( ProQ; 0) ( interchain_env; 0) ( interchain_contact; 0) ( chainbreak; 0) ( linear_chainbreak; 0) ( overlap_chainbreak; 0) ( distance_chainbreak; 0) ( dof_constraint; 0) ( branch_conn; 0) ( linear_branch_conn; 0) ( rama_prepro; -0.0498918) ( paa_abego3; 0) ( cart_bonded; 0) ( cart_bonded_angle; 0) ( cart_bonded_length; 0) ( cart_bonded_ring; 0) ( cart_bonded_torsion; 0) ( cart_bonded_proper; 0) ( cart_bonded_improper; 0) ( gen_bonded; 0) ( gen_bonded_bond; 0) ( gen_bonded_angle; 0) ( gen_bonded_torsion; 0) ( gen_bonded_improper; 0) ( neigh_vect; 0) ( neigh_count; 0) ( neigh_vect_raw; 0) ( symE_bonus; 0) ( sym_lig; 0) ( mhc_epitope; 0) ( pack_stat; 0) ( rms_energy; 0) ( res_type_constraint; 0) ( res_type_linking_constraint; 0) ( pocket_constraint; 0) ( backbone_stub_constraint; 0) ( backbone_stub_linear_constraint; 0) ( surface; 0) ( p_aa; 0) ( unfolded; 0) ( split_unfolded_two_body; 0) ( fa_atr_ref; 0) ( fa_rep_ref; 0) ( fa_sol_ref; 0) ( fa_elec_ref; 0) ( hbond_ref; 0) ( dslf_fa13_ref; 0) ( fa_intra_atr_ref; 0) ( fa_intra_rep_ref; 0) ( fa_intra_sol_ref; 0) ( pro_close_ref; 0) ( fa_dun_ref; 0) ( fa_dun_dev_ref; 0) ( fa_dun_rot_ref; 0) ( fa_dun_semi_ref; 0) ( rama_ref; 0) ( p_aa_pp_ref; 0) ( omega_ref; 0) ( mm_lj_intra_rep_ref; 0) ( mm_lj_intra_atr_ref; 0) ( mm_twist_ref; 0) ( elec_dens_fast; 0) ( elec_dens_window; 0) ( elec_dens_whole_structure_ca; 0) ( elec_dens_whole_structure_allatom; 0) ( elec_dens_atomwise; 0) ( grid_vdw; 0) ( xtal_ml; 0) ( xtal_rwork; 0) ( xtal_rfree; 0) ( hpatch; 0) ( Menv_smooth; 0) ( wat_desolv; 0) ( ring_close; 0) ( aa_repeat; 0) ( aa_composition; 0) ( aspartimide_penalty; 0) ( hbnet; 0) ( buried_unsatisfied_penalty; 0) ( netcharge; 0) ( voids_penalty; 0) ( sap_constraint; 0) ( dna_env; 0) ( dna_pair; 0) ( dump_trajectory; 0) ( PyRosettaEnergy_first; 0) ( PyRosettaEnergy_464; 0) ( PyRosettaEnergy_465; 0) ( PyRosettaEnergy_466; 0) ( PyRosettaEnergy_467; 0) ( PyRosettaEnergy_468; 0) ( PyRosettaEnergy_469; 0) ( PyRosettaEnergy_470; 0) ( PyRosettaEnergy_471; 0) ( PyRosettaEnergy_472; 0) ( PyRosettaEnergy_last; 0) ( sidechain_neighbors; 0) ( target_clash; 0) ( membrane_span_constraint; 0) ( membrane_span_term_z_constraint; 0) ( aromatic_restraint; 0) ( total_score; 427.308) ( dummy_score_type; 0) \n",
            "3.7910821714280245\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "OSXAFRoPmdk3"
      },
      "source": [
        "# Protein Design with a Resfile and FastRelax\n",
        "\n",
        "Keywords: FastDesign, FastRelax, ResfileCommandOperation, Resfile, ResidueSelector, MoveMapFactory, TaskFactory, TaskOperation, NoRepackDisulfides, IncludeCurrent, ReadResfile, conf2pdb_chain(), pose_from_rcsb(), create_score_function(), CA_rmsd()"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "_QTF-gFrmdk3"
      },
      "source": [
        "## Overview\n",
        "\n",
        "In this Workshop, we will learn the classic way to design proteins, but in the same breath introduce the concept of design using a flexible backbone protocol.  \n",
        "\n",
        "This protocol, is essentially design during FastRelax.  A separate class, FastDesign, has a bit more options for design, but essentially, they are the same.  \n",
        "\n",
        "Many modern designs have used this FastDesign/RelaxedDesign protocol - including many Science papers from the Baker lab and the RosettaAntibodyDesign (RAbD) protocol that we will cover in another tutorial. \n",
        "\n",
        "Before this workshop, you should read about the resfile syntax here: https://www.rosettacommons.org/docs/latest/rosetta_basics/file_types/resfiles"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "71XJqZmcmdk4"
      },
      "source": [
        "### Initialize PyRosetta"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "VNZUuHAvmdk4",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "10bb2267-b6f5-4687-be41-b2f5bc878ed8"
      },
      "source": [
        "pyrosetta.init(\"-ignore_unrecognized_res 1 -ex1 -ex2aro -detect_disulf 0\")"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:pyrosetta.rosetta:Found rosetta database at: /usr/local/lib/python3.7/dist-packages/pyrosetta/database; using it....\n",
            "INFO:pyrosetta.rosetta:PyRosetta-4 2021 [Rosetta PyRosetta4.MinSizeRel.python37.ubuntu 2020.50.post.dev+970.commits.3700df145608444753aabbec9c4681ec9b21f74b 2021-02-24T13:24:53] retrieved from: http://www.pyrosetta.org\n",
            "(C) Copyright Rosetta Commons Member Institutions. Created in JHU by Sergey Lyskov and PyRosetta Team.\n",
            "INFO:rosetta:core.init: Checking for fconfig files in pwd and ./rosetta/flags\n",
            "INFO:rosetta:core.init: Rosetta version: PyRosetta4.MinSizeRel.python37.ubuntu r17132 2020.50.post.dev+970.commits.3700df14560 3700df145608444753aabbec9c4681ec9b21f74b http://www.pyrosetta.org 2021-02-24T13:24:53\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "stream",
          "text": [
            "PyRosetta-4 2021 [Rosetta PyRosetta4.MinSizeRel.python37.ubuntu 2020.50.post.dev+970.commits.3700df145608444753aabbec9c4681ec9b21f74b 2021-02-24T13:24:53] retrieved from: http://www.pyrosetta.org\n",
            "(C) Copyright Rosetta Commons Member Institutions. Created in JHU by Sergey Lyskov and PyRosetta Team.\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:core.init: command: PyRosetta -ignore_unrecognized_res 1 -ex1 -ex2aro -detect_disulf 0 -database /usr/local/lib/python3.7/dist-packages/pyrosetta/database\n",
            "INFO:rosetta:basic.random.init_random_generator: 'RNG device' seed mode, using '/dev/urandom', seed=-1778875097 seed_offset=0 real_seed=-1778875097\n",
            "INFO:rosetta:basic.random.init_random_generator: RandomGenerator:init: Normal mode, seed=-1778875097 RG_type=mt19937\n"
          ],
          "name": "stderr"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "LGvobJhomdk4"
      },
      "source": [
        " For this tutorial, let's use the well-studied native protein crambin from PDB ID 1AB1 (http://www.rcsb.org/structure/1AB1).\n",
        " \n",
        " Setup the input pose and scorefunction:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "CSJZ80Otmdk4",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "6a6eee49-019f-47e7-d79a-fd50a09d3e93"
      },
      "source": [
        "start_pose = pyrosetta.toolbox.rcsb.pose_from_rcsb(\"1AB1\", ATOM=True, CRYS=False)\n",
        "pose = start_pose.clone()\n",
        "scorefxn = pyrosetta.create_score_function(\"ref2015_cart.wts\")"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:core.import_pose.import_pose: File '1AB1.clean.pdb' automatically determined to be of type PDB\n",
            "INFO:rosetta:core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] discarding 3 atoms at position 1 in file 1AB1.clean.pdb. Best match rsd_type:  THR:NtermProteinFull\n",
            "INFO:rosetta:core.energy_methods.CartesianBondedEnergy: Initializing IdealParametersDatabase with default Ks=300 , 80 , 80 , 10 , 80\n",
            "INFO:rosetta:basic.io.database: Database file opened: scoring/score_functions/bondlength_bondangle/default-lengths.txt\n",
            "INFO:rosetta:core.energy_methods.CartesianBondedEnergy: Read 759 bb-independent lengths.\n",
            "INFO:rosetta:basic.io.database: Database file opened: scoring/score_functions/bondlength_bondangle/default-angles.txt\n",
            "INFO:rosetta:core.energy_methods.CartesianBondedEnergy: Read 1434 bb-independent angles.\n",
            "INFO:rosetta:basic.io.database: Database file opened: scoring/score_functions/bondlength_bondangle/default-torsions.txt\n",
            "INFO:rosetta:core.energy_methods.CartesianBondedEnergy: Read 1 bb-independent torsions.\n",
            "INFO:rosetta:basic.io.database: Database file opened: scoring/score_functions/bondlength_bondangle/default-improper.txt\n",
            "INFO:rosetta:core.energy_methods.CartesianBondedEnergy: Read 529 bb-independent improper tors.\n"
          ],
          "name": "stderr"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "xdXC5N0pmdk5"
      },
      "source": [
        " Make of list of which residues are cysteine:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "FAqCHvM4mdk5",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "143c4964-c273-4811-80fe-d2d6ed2b9198"
      },
      "source": [
        "cys_res = []\n",
        "for i, aa in enumerate(start_pose.sequence(), start=1):\n",
        "    if aa == \"C\":\n",
        "        cys_res.append(i)\n",
        "print(cys_res)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "[3, 4, 16, 26, 32, 40]\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "3WQzoG4_mdk5"
      },
      "source": [
        "Inspect `start_pose` using the `PyMolMover` or `dump_pdb()`"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "RMhjH2Pbmdk5"
      },
      "source": [
        "## Design strategy:\n",
        "\n",
        "Design away the cysteine residues (i.e. disulfide bonds) using a resfile, allowing all side-chains to re-pack and all backbone and side-chain torsions to minimize using the `FastRelax` mover.\n",
        "\n",
        " Read more about resfile file structure at https://www.rosettacommons.org/docs/latest/rosetta_basics/file_types/resfiles"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "fyvQ2LsBmdk5"
      },
      "source": [
        "To write a resfile, we need to know which chain to mutate.\n",
        "\n",
        "We can see that the pose consists of only chain \"A\" by printing the `pose.pdb_info()` object:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "mPZmngrwmdk6",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "4ed58046-cf0e-47e5-ca4f-d9e43ef40308"
      },
      "source": [
        "print(pose.pdb_info())"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "PDB file name: 1AB1.clean.pdb\n",
            " Pose Range  Chain    PDB Range  |   #Residues         #Atoms\n",
            "\n",
            "0001 -- 0046    A 0001  -- 0046  |   0046 residues;    00649 atoms\n",
            "                           TOTAL |   0046 residues;    00649 atoms\n",
            "\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "dHjoOXsnmdk6"
      },
      "source": [
        " More programmatically, we could find which chains are in the `pose` using `pyrosetta.rosetta.core.pose.conf2pdb_chain(pose)` which returns a `pyrosetta.rosetta.std.map_unsigned_long_char` object which is iterable."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "09wDzccWmdk6",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "57c82dab-b2c3-41a4-fc9c-69d78bcbd392"
      },
      "source": [
        "print(pyrosetta.rosetta.core.pose.conf2pdb_chain(pose))"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "map_unsigned_long_char{1: A}\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "kCzXpBBUmdk6",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "65ab343c-7c22-43c7-9ce3-c050ac16928e"
      },
      "source": [
        "for k, v in pyrosetta.rosetta.core.pose.conf2pdb_chain(pose).items():\n",
        "    print(v)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "A\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "PskkULv5mdk7"
      },
      "source": [
        " So we could write a resfile to disc indicating design specifications to mutate only the cysteine residues in chain \"A\".  Use the syntax described below, and save your resfile in this directory as `redes_cys.resfile`.\n",
        " \n",
        " https://www.rosettacommons.org/docs/latest/rosetta_basics/file_types/resfiles"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "KwBxqG75mdk7"
      },
      "source": [
        "First, create a directory in your pyrbc_202103_notebooks directory named \"outputs,\" and then write your resfile to that directory"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "nbgrader": {
          "grade": true,
          "grade_id": "cell-4dfc4f43e2031e10",
          "locked": false,
          "points": 0,
          "schema_version": 3,
          "solution": true
        },
        "id": "rv0WDANNmdk7",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "0bf9a7e2-c692-46ec-f88e-c305497ef622"
      },
      "source": [
        "resfile = \"./outputs/redes_cys.resfile\"  # this is the file you should write to; we will read from this file later\n",
        "\n",
        "with open(resfile, \"w\") as f:\n",
        "  f.write(\"NATAA\\n\")\n",
        "  f.write(\"start\\n\")\n",
        "\n",
        "  for i in cys_res:\n",
        "    f.write(\"{0} {1} ALLAAxc\\n\".format(i, pyrosetta.rosetta.core.pose.conf2pdb_chain(pose)[1]))\n",
        "\n",
        "!cat {resfile}"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "NATAA\n",
            "start\n",
            "3 A ALLAAxc\n",
            "4 A ALLAAxc\n",
            "16 A ALLAAxc\n",
            "26 A ALLAAxc\n",
            "32 A ALLAAxc\n",
            "40 A ALLAAxc\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "VBlj6aoemdk7"
      },
      "source": [
        "Note that we don't necessarily need a resfile to use resfile commands.  We can now do this in an intuitive way through code and `ResidueSelectors` using the `ResfileCommandOperation`.  The main docs for the XML interface are available below, however the code-level interface is extremely similar.  Use the ? to get more info on this.  The operation is located in `pyrosetta.rosetta.core.pack.task.operation` as we saw this location in the previous tutorial.\n",
        "\n",
        "https://www.rosettacommons.org/docs/latest/scripting_documentation/RosettaScripts/TaskOperations/taskoperations_pages/ResfileCommandOperation"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "miVXoH5imdk7"
      },
      "source": [
        " Now we can setup the TaskOperations for the `FastRelax` mover. These tell `FastRelax` which residues to design or repack during the packer steps in `FastRelax`.  You should be familiar with this from the previous tutorial\n",
        " \n",
        "We use the `IncludeCurrent` to include the current rotamer of from the crystal structure during packing."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "kUsDY474mdk7",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "997da8bf-5271-45f2-a076-c9e196f4967c"
      },
      "source": [
        "# The task factory accepts all the task operations\n",
        "tf = pyrosetta.rosetta.core.pack.task.TaskFactory()\n",
        "\n",
        "# These are pretty standard\n",
        "tf.push_back(pyrosetta.rosetta.core.pack.task.operation.InitializeFromCommandline())\n",
        "tf.push_back(pyrosetta.rosetta.core.pack.task.operation.IncludeCurrent())\n",
        "tf.push_back(pyrosetta.rosetta.core.pack.task.operation.NoRepackDisulfides())\n",
        "\n",
        "# Include the resfile\n",
        "tf.push_back(pyrosetta.rosetta.core.pack.task.operation.ReadResfile(resfile))\n",
        "\n",
        "# Convert the task factory into a PackerTask to take a look at it\n",
        "packer_task = tf.create_task_and_apply_taskoperations(pose)\n",
        "# View the PackerTask\n",
        "print(packer_task)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:core.pack.task: Packer task: initialize from command line()\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "stream",
          "text": [
            "#Packer_Task\n",
            "\n",
            "Threads to request: ALL AVAILABLE\n",
            "\n",
            "resid\tpack?\tdesign?\tallowed_aas\n",
            "1\tTRUE\tFALSE\tTHR:NtermProteinFull\n",
            "2\tTRUE\tFALSE\tTHR\n",
            "3\tTRUE\tTRUE\tALA,ASP,GLU,PHE,GLY,HIS,HIS_D,ILE,LYS,LEU,MET,ASN,PRO,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "4\tTRUE\tTRUE\tALA,ASP,GLU,PHE,GLY,HIS,HIS_D,ILE,LYS,LEU,MET,ASN,PRO,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "5\tTRUE\tFALSE\tPRO\n",
            "6\tTRUE\tFALSE\tSER\n",
            "7\tTRUE\tFALSE\tILE\n",
            "8\tTRUE\tFALSE\tVAL\n",
            "9\tTRUE\tFALSE\tALA\n",
            "10\tTRUE\tFALSE\tARG\n",
            "11\tTRUE\tFALSE\tSER\n",
            "12\tTRUE\tFALSE\tASN\n",
            "13\tTRUE\tFALSE\tPHE\n",
            "14\tTRUE\tFALSE\tASN\n",
            "15\tTRUE\tFALSE\tVAL\n",
            "16\tTRUE\tTRUE\tALA,ASP,GLU,PHE,GLY,HIS,HIS_D,ILE,LYS,LEU,MET,ASN,PRO,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "17\tTRUE\tFALSE\tARG\n",
            "18\tTRUE\tFALSE\tLEU\n",
            "19\tTRUE\tFALSE\tPRO\n",
            "20\tTRUE\tFALSE\tGLY\n",
            "21\tTRUE\tFALSE\tTHR\n",
            "22\tTRUE\tFALSE\tSER\n",
            "23\tTRUE\tFALSE\tGLU\n",
            "24\tTRUE\tFALSE\tALA\n",
            "25\tTRUE\tFALSE\tILE\n",
            "26\tTRUE\tTRUE\tALA,ASP,GLU,PHE,GLY,HIS,HIS_D,ILE,LYS,LEU,MET,ASN,PRO,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "27\tTRUE\tFALSE\tALA\n",
            "28\tTRUE\tFALSE\tTHR\n",
            "29\tTRUE\tFALSE\tTYR\n",
            "30\tTRUE\tFALSE\tTHR\n",
            "31\tTRUE\tFALSE\tGLY\n",
            "32\tTRUE\tTRUE\tALA,ASP,GLU,PHE,GLY,HIS,HIS_D,ILE,LYS,LEU,MET,ASN,PRO,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "33\tTRUE\tFALSE\tILE\n",
            "34\tTRUE\tFALSE\tILE\n",
            "35\tTRUE\tFALSE\tILE\n",
            "36\tTRUE\tFALSE\tPRO\n",
            "37\tTRUE\tFALSE\tGLY\n",
            "38\tTRUE\tFALSE\tALA\n",
            "39\tTRUE\tFALSE\tTHR\n",
            "40\tTRUE\tTRUE\tALA,ASP,GLU,PHE,GLY,HIS,HIS_D,ILE,LYS,LEU,MET,ASN,PRO,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "41\tTRUE\tFALSE\tPRO\n",
            "42\tTRUE\tFALSE\tGLY\n",
            "43\tTRUE\tFALSE\tASP\n",
            "44\tTRUE\tFALSE\tTYR\n",
            "45\tTRUE\tFALSE\tALA\n",
            "46\tTRUE\tFALSE\tASN:CtermProteinFull\n",
            "\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "7PO6IeA0mdk8"
      },
      "source": [
        " The PackerTask looks as intended! \n",
        "\n",
        " Now we can set up a `MoveMap` or a `MoveMapFactory` to specify which torsions are free to minimize during the minimization steps of the `FastDesign` mover"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "SPZwMGyBmdk8"
      },
      "source": [
        "# Set up a MoveMapFactory\n",
        "mmf = pyrosetta.rosetta.core.select.movemap.MoveMapFactory()\n",
        "mmf.all_bb(setting=True)\n",
        "mmf.all_bondangles(setting=True)\n",
        "mmf.all_bondlengths(setting=True)\n",
        "mmf.all_chi(setting=True)\n",
        "mmf.all_jumps(setting=True)\n",
        "mmf.set_cartesian(setting=True)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "Ce-JuRwPmdk8"
      },
      "source": [
        "# Set up a MoveMap\n",
        "# mm = pyrosetta.rosetta.core.kinematics.MoveMap()\n",
        "# mm.set_bb(True)\n",
        "# mm.set_chi(True)\n",
        "# mm.set_jump(True)\n",
        "\n",
        "# If needed, you could turn off bb and chi torsions for individual residues like this:\n",
        "\n",
        "# vector1 of true/false for each residue in the pose\n",
        "# subset_to_minimize = do_something_set.apply(pose)\n",
        "\n",
        "# for i in range(1, pose.size() + 1):\n",
        "#     if (not subset_to_minimize[i]):\n",
        "#         mm.set_bb(i, False)\n",
        "#         mm.set_chi(i, False)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "a5RkDdIumdk8"
      },
      "source": [
        " Because some Movers only take as input a `MoveMap`, for backwards-compatibility one could generate a `MoveMap` from a `MoveMapFactory` using the `MoveMapFactory` function `create_movemap_from_pose(pose)`"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "zQqjzkVJmdk9"
      },
      "source": [
        " Now let's double-check some more `pose` information to verify that we are ready for `FastRelax`:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "MydLTimNmdk-",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "2831a3c8-13e5-41a0-f510-a2855cfafffd"
      },
      "source": [
        "display_pose = pyrosetta.rosetta.protocols.fold_from_loops.movers.DisplayPoseLabelsMover()\n",
        "display_pose.tasks(tf)\n",
        "display_pose.movemap_factory(mmf)\n",
        "display_pose.apply(pose)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:protocols.DsspMover: LEELLLHHHHHHHHHHHLLLLLHHHHHHHHLLEELLLLLLLLLLLL\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover: SEQUENCE       TTCCPSIVARSNFNVCRLPGTSEAICATYTGCIIIPGATCPGDYAN\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover: STRUCTURE      LEELLLHHHHHHHHHHHLLLLLHHHHHHHHLLEELLLLLLLLLLLL\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover: MVMP_BB        **********************************************\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover: MVMP_CHI       **********************************************\n",
            "INFO:rosetta:core.pack.task: Packer task: initialize from command line()\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover: DESIGN         --**-----------*---------*-----*-------*------\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover: REPACK         **********************************************\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover: FOLDTREE       *********************************************C\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover:\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover: FOLD_TREE  EDGE 1 46 -1\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover: T[THR:NtermProteinFull]TCCPSIVARSNFNVCRLPGTSEAICATYTGCIIIPGATCPGDYAN[ASN:CtermProteinFull]\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover: PDB file name: 1AB1.clean.pdb\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover:  Pose Range  Chain    PDB Range  |   #Residues         #Atoms\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover: 0001 -- 0046    A 0001  -- 0046  |   0046 residues;    00649 atoms\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover:                            TOTAL |   0046 residues;    00649 atoms\n"
          ],
          "name": "stderr"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "YjSHB1dLmdk-"
      },
      "source": [
        " Setting up `FastRelax` prints the default `relaxscript`, showing the `ramp_repack_min` settings with the following assignments:\n",
        ">ramp_repack_min [scale:fa_rep] [min_tolerance] [coord_cst_weight]"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "W0YYYfoXmdk-",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "ce221db4-bc60-42d3-9928-740dc3df15a3"
      },
      "source": [
        "fr = pyrosetta.rosetta.protocols.relax.FastRelax(scorefxn_in=scorefxn, standard_repeats=1)\n",
        "fr.cartesian(True)\n",
        "fr.set_task_factory(tf)\n",
        "fr.set_movemap_factory(mmf)\n",
        "fr.min_type(\"lbfgs_armijo_nonmonotone\") # For non-Cartesian scorefunctions, use \"dfpmin_armijo_nonmonotone\"\n",
        "\n",
        "#Note that this min_type is automatically set when you set the cartesian option. \n",
        "#  But it is good to be aware of this - as not all protocols will do this for you.\n",
        "\n",
        "#fr.set_movemap(mm) # Could have optionally specified a MoveMap instead of MoveMapFactory\n",
        "#fr.minimize_bond_angles(True) # If not using MoveMapFactory, could specify bond angle minimization here\n",
        "#fr.minimize_bond_lengths(True) # If not using MoveMapFactory, could specify bond length minimization here"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:protocols.relax.RelaxScriptManager: Reading relax scripts list from database.\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: Looking for MonomerRelax2019.txt\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: ================== Reading script file: /usr/local/lib/python3.7/dist-packages/pyrosetta/database/sampling/relax_scripts/MonomerRelax2019.txt ==================\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: repeat %%nrepeats%%\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: coord_cst_weight 1.0\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: scale:fa_rep 0.040\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: repack\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: scale:fa_rep 0.051\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: min 0.01\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: coord_cst_weight 0.5\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: scale:fa_rep 0.265\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: repack\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: scale:fa_rep 0.280\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: min 0.01\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: coord_cst_weight 0.0\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: scale:fa_rep 0.559\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: repack\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: scale:fa_rep 0.581\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: min 0.01\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: coord_cst_weight 0.0\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: scale:fa_rep 1\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: repack\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: min 0.00001\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: accept_to_best\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: endrepeat\n"
          ],
          "name": "stderr"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "XmWk1gM1mdk-"
      },
      "source": [
        "For recommendations on setting `fr.min_type()` for the scorefunction being used, see: https://www.rosettacommons.org/docs/latest/rosetta_basics/structural_concepts/minimization-overview#recommendations"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "j8JiHOxOmdk-"
      },
      "source": [
        " Run Fast(Design)! Note: this takes ~1min 31s"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "nbgrader": {
          "grade": true,
          "grade_id": "cell-5bcc2bfb4357035d",
          "locked": false,
          "points": 0,
          "schema_version": 3,
          "solution": true
        },
        "id": "oWzUiO30mdk_",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "f11a5b64-3d71-43b3-fc59-36856280bb99"
      },
      "source": [
        "fr.apply(pose)\n",
        "pose.dump_pdb(\"outputs/1ab1_resfile_relaxed.pdb\")"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (46)\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: repeat  1274.78  0  0  0.55\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: coord_cst_weight  1274.78  0  0  0.55\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: scale:fa_rep  406.204  0  0  0.022\n",
            "INFO:rosetta:core.pack.task: Packer task: initialize from command line()\n",
            "INFO:rosetta:core.pack.pack_rotamers: built 5671 rotamers at 46 positions.\n",
            "INFO:rosetta:core.pack.interaction_graph.interaction_graph_factory: Instantiating PDInteractionGraph\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: repack  75.6883  0  0  0.022\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: scale:fa_rep  83.4309  0  0  0.02805\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: min  -176.221  0.716057  0.716057  0.02805\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: coord_cst_weight  -176.221  0.716057  0.716057  0.02805\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: scale:fa_rep  -72.423  0.716057  0.716057  0.14575\n",
            "INFO:rosetta:core.pack.task: Packer task: initialize from command line()\n",
            "INFO:rosetta:core.pack.pack_rotamers: built 3808 rotamers at 46 positions.\n",
            "INFO:rosetta:core.pack.interaction_graph.interaction_graph_factory: Instantiating PDInteractionGraph\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: repack  -94.8177  0.716057  0.716057  0.14575\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: scale:fa_rep  -92.1913  0.716057  0.716057  0.154\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: min  -129.317  0.641728  0.641728  0.154\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: coord_cst_weight  -129.317  0.641728  0.641728  0.154\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: scale:fa_rep  -108.495  0.641728  0.641728  0.30745\n",
            "INFO:rosetta:core.pack.task: Packer task: initialize from command line()\n",
            "INFO:rosetta:core.pack.pack_rotamers: built 2884 rotamers at 46 positions.\n",
            "INFO:rosetta:core.pack.interaction_graph.interaction_graph_factory: Instantiating PDInteractionGraph\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: repack  -108.873  0.641728  0.641728  0.30745\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: scale:fa_rep  -107.213  0.641728  0.641728  0.31955\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: min  -117.524  0.572089  0.572089  0.31955\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: coord_cst_weight  -117.524  0.572089  0.572089  0.31955\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: scale:fa_rep  -100.693  0.572089  0.572089  0.55\n",
            "INFO:rosetta:core.pack.task: Packer task: initialize from command line()\n",
            "INFO:rosetta:core.pack.pack_rotamers: built 2756 rotamers at 46 positions.\n",
            "INFO:rosetta:core.pack.interaction_graph.interaction_graph_factory: Instantiating PDInteractionGraph\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: repack  -100.693  0.572089  0.572089  0.55\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: min  -104.366  0.537479  0.537479  0.55\n",
            "INFO:rosetta:protocols.relax.FastRelax: MRP: 0  -104.366  -104.366  0.537479  0.537479\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: accept_to_best  -104.366  0.537479  0.537479  0.55\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: endrepeat  -104.366  0.537479  0.537479  0.55\n",
            "INFO:rosetta:protocols::checkpoint: Deleting checkpoints of FastRelax\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "True"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 23
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "p3goJ2dOmdk_"
      },
      "source": [
        "### Analysis\n",
        "\n",
        "Inspect the resulting design!"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "aOCMS_Vmmdk_"
      },
      "source": [
        " By how many Angstroms RMSD did the backbone C atoms move?"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "kNTYHvJmmdk_",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "b9a85c53-713d-4ad0-dd8f-6da424f463d5"
      },
      "source": [
        "pyrosetta.rosetta.core.scoring.CA_rmsd(start_pose, pose)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "0.5374792814254761"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 24
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "7x2uIlD5mdk_"
      },
      "source": [
        "What is the delta `total_score` from `start_pose` to `pose`? Why is it large?"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "k_F9OeaCmdk_",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "0e803655-02d3-40d8-cfa2-d8ab422d9bd2"
      },
      "source": [
        "delta_total_score = scorefxn(pose) - scorefxn(start_pose)\n",
        "print(delta_total_score)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (46)\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "stream",
          "text": [
            "-1379.15070574072\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "K20LvAUpmdk_"
      },
      "source": [
        " What is the per-residue energy difference for each mutated position between `start_pose` and `pose`?"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "nbgrader": {
          "grade": true,
          "grade_id": "cell-32cfb702cb53c564",
          "locked": false,
          "points": 0,
          "schema_version": 3,
          "solution": true
        },
        "id": "yIZHlnFomdlA",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "43baee32-44c1-4056-979d-87bb882481f7"
      },
      "source": [
        "for i in cys_res:\n",
        "  print(\"Residue #: \", i)\n",
        "  print(\"Relaxed pose: \", pyrosetta.rosetta.protocols.relax.get_per_residue_scores(pose, pyrosetta.rosetta.core.scoring.ScoreType.total_score)[i - 1])\n",
        "  print(\"Start pose: \", pyrosetta.rosetta.protocols.relax.get_per_residue_scores(start_pose, pyrosetta.rosetta.core.scoring.ScoreType.total_score)[i - 1])"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Residue #:  3\n",
            "Relaxed pose:  -2.198366153044414\n",
            "Start pose:  9.949274702677952\n",
            "Residue #:  4\n",
            "Relaxed pose:  -2.6002366356597775\n",
            "Start pose:  161.06299751955535\n",
            "Residue #:  16\n",
            "Relaxed pose:  0.04939967402494855\n",
            "Start pose:  35.16917408036165\n",
            "Residue #:  26\n",
            "Relaxed pose:  -1.4184857949233842\n",
            "Start pose:  6.059158875174292\n",
            "Residue #:  32\n",
            "Relaxed pose:  -1.304546479338188\n",
            "Start pose:  5.015792828193076\n",
            "Residue #:  40\n",
            "Relaxed pose:  2.127620519893319\n",
            "Start pose:  8.369850772545249\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "lgo6LMgQmdlA"
      },
      "source": [
        "# Protein Design 2\n",
        "\n",
        "Keywords: FastDesign, FastRelax, ResidueSelector, TaskFactory, TaskOperation, pose_from_rcsb, get_residues_from_subset, ResidueNameSelector, ChainSelector, ResiduePropertySelector, NotResidueSelector, AndResidueSelector, RestrictToRepackingRLT, RestrictAbsentCanonicalAASRLT, NoRepackDisulfides, OperateOnResidueSubset\n",
        "\n",
        "## Overview\n",
        "\n",
        "Here, we will expand upon our knowledge of ResidueSelectors and TaskOperations and how to use them.  We can once again use FastRelax or FastDesign to do design.  Since we don't need any additional tools offered by FastDesign, we will use FastRelax.  I like to call it RelaxedDesign. :)"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "we9Zj2FemdlA",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 17
        },
        "outputId": "3df43cbf-9390-4a8c-f136-bb499351ed1a"
      },
      "source": [
        "import logging\n",
        "logging.basicConfig(level=logging.INFO)\n",
        "import os\n",
        "import pyrosetta\n",
        "import pyrosetta.toolbox\n",
        "import site\n",
        "\n",
        "from IPython.core.display import display, HTML\n",
        "display(HTML(\"<style>.container { width:100% !important; }</style>\"))"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/html": [
              "<style>.container { width:100% !important; }</style>"
            ],
            "text/plain": [
              "<IPython.core.display.HTML object>"
            ]
          },
          "metadata": {
            "tags": []
          }
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "wsGr6tZbmdlA"
      },
      "source": [
        "## Initialize PyRosetta "
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "dBAiuXscmdlA",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "9a429ed9-5cc5-46bb-cea5-5b63c4a627a6"
      },
      "source": [
        "pyrosetta.init(\"-ignore_unrecognized_res 1 -ex1 -ex2aro\")"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:pyrosetta.rosetta:Found rosetta database at: /usr/local/lib/python3.7/dist-packages/pyrosetta/database; using it....\n",
            "INFO:pyrosetta.rosetta:PyRosetta-4 2021 [Rosetta PyRosetta4.MinSizeRel.python37.ubuntu 2020.50.post.dev+970.commits.3700df145608444753aabbec9c4681ec9b21f74b 2021-02-24T13:24:53] retrieved from: http://www.pyrosetta.org\n",
            "(C) Copyright Rosetta Commons Member Institutions. Created in JHU by Sergey Lyskov and PyRosetta Team.\n",
            "INFO:rosetta:core.init: Checking for fconfig files in pwd and ./rosetta/flags\n",
            "INFO:rosetta:core.init: Rosetta version: PyRosetta4.MinSizeRel.python37.ubuntu r17132 2020.50.post.dev+970.commits.3700df14560 3700df145608444753aabbec9c4681ec9b21f74b http://www.pyrosetta.org 2021-02-24T13:24:53\n",
            "INFO:rosetta:core.init: command: PyRosetta -ignore_unrecognized_res 1 -ex1 -ex2aro -database /usr/local/lib/python3.7/dist-packages/pyrosetta/database\n",
            "INFO:rosetta:basic.random.init_random_generator: 'RNG device' seed mode, using '/dev/urandom', seed=-118602821 seed_offset=0 real_seed=-118602821\n",
            "INFO:rosetta:basic.random.init_random_generator: RandomGenerator:init: Normal mode, seed=-118602821 RG_type=mt19937\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "stream",
          "text": [
            "PyRosetta-4 2021 [Rosetta PyRosetta4.MinSizeRel.python37.ubuntu 2020.50.post.dev+970.commits.3700df145608444753aabbec9c4681ec9b21f74b 2021-02-24T13:24:53] retrieved from: http://www.pyrosetta.org\n",
            "(C) Copyright Rosetta Commons Member Institutions. Created in JHU by Sergey Lyskov and PyRosetta Team.\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "M9DASRcRmdlA"
      },
      "source": [
        " For this tutorial, let's again use the native protein crambin from PDB ID 1AB1 (http://www.rcsb.org/structure/1AB1)\n",
        " Setup the input pose and scorefunction:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "S3WE8XNWmdlA",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "5439b7f8-6778-478f-ad6e-b5ac9cd4a313"
      },
      "source": [
        "start_pose = pyrosetta.toolbox.rcsb.pose_from_rcsb(\"1AB1\", ATOM=True, CRYS=False)\n",
        "pose = start_pose.clone()\n",
        "scorefxn = pyrosetta.create_score_function(\"ref2015_cart.wts\")"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:core.import_pose.import_pose: File '1AB1.clean.pdb' automatically determined to be of type PDB\n",
            "INFO:rosetta:core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] discarding 3 atoms at position 1 in file 1AB1.clean.pdb. Best match rsd_type:  THR:NtermProteinFull\n",
            "INFO:rosetta:core.conformation.Conformation: Found disulfide between residues 3 40\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 3 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 40 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 3 CYD\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 40 CYD\n",
            "INFO:rosetta:core.conformation.Conformation: Found disulfide between residues 4 32\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 4 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 32 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 4 CYD\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 32 CYD\n",
            "INFO:rosetta:core.conformation.Conformation: Found disulfide between residues 16 26\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 16 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 26 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 16 CYD\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 26 CYD\n"
          ],
          "name": "stderr"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "tRFz5xPjmdlA"
      },
      "source": [
        " To list the cysteine residues in crambin, now we use ResidueSelectors:\n",
        " Note difference in residue names \"CYS:disulfide\" vs. \"CYS\"\n",
        " \n",
        " Note that we could also use the `ResiduePropertySelector` and use the `DISULFIDE_BONDED` property to grab these."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "b-PbM9-6mdlB",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "fd4930f7-bb67-4800-e6c0-5df2d6d1f048"
      },
      "source": [
        "disulfide_res = pyrosetta.rosetta.core.select.residue_selector.ResidueNameSelector(\"CYS:disulfide\")\n",
        "print(pyrosetta.rosetta.core.select.get_residues_from_subset(disulfide_res.apply(pose)))"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "vector1_unsigned_long[3, 4, 16, 26, 32, 40]\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "DenIWMEnmdlB"
      },
      "source": [
        "You could inspect the starting pose using the `PyMolMover` or `dump_pdb()`. However, by default, disulfide bonds are not drawn even if they exist in PyRosetta."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "i38w0dYYmdlB"
      },
      "source": [
        "You can also use this shortcut to view a pose in `py3Dmol` which recognizes disulfide bonds in PyRosetta:\n",
        " \n",
        "Copy the file `visualization.py` from the Session12 directory into the pyrbc_202103_notebooks directory. Next, run the `visualization.py` script, which will define a function `view_pose`:\n",
        "\n",
        "````\n",
        "%run ./visualization.py\n",
        "```"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "BAuUkGzxmdlB",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 17
        },
        "outputId": "fe0fd06a-884c-4219-c07f-5b5c7b5d1a13"
      },
      "source": [
        "%run ./Sessions/Session12/visualization.py"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/html": [
              "<style>.container { width:100% !important; }</style>"
            ],
            "text/plain": [
              "<IPython.core.display.HTML object>"
            ]
          },
          "metadata": {
            "tags": []
          }
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "u5sVDnE2mdlB"
      },
      "source": [
        "Now that `view_pose` has been defined, run it!\n",
        "```\n",
        "view_pose(pose)\n",
        "```"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "-BzmBjSGmdlB",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 817
        },
        "outputId": "d884fe07-c7ad-4515-ee64-340900003f82"
      },
      "source": [
        "view_pose(pose)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_16234199980179648\"  style=\"position: relative; width: 1200px; height: 800px\">\n        <p id=\"3dmolwarning_16234199980179648\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n      resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n}\n\nvar viewer_16234199980179648 = null;\nvar warn = document.getElementById(\"3dmolwarning_16234199980179648\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_16234199980179648 = $3Dmol.createViewer($(\"#3dmolviewer_16234199980179648\"),{backgroundColor:\"white\"});\nviewer_16234199980179648.zoomTo();\n\tviewer_16234199980179648.addModels(\"HEADER                                            11-JUN-21   XXXX              \\nEXPDTA    THEORETICAL MODEL                                                     \\nREMARK 220                                                                      \\nREMARK 220 EXPERIMENTAL DETAILS                                                 \\nREMARK 220  EXPERIMENT TYPE                : THEORETICAL MODELLING              \\nREMARK 220  DATE OF DATA COLLECTION        : 11-JUN-21                          \\nREMARK 220                                                                      \\nREMARK 220 REMARK: MODEL GENERATED BY ROSETTA                                   \\nREMARK 220  VERSION 2020.50.post.dev+970.commits.3700df14560                    \\nSSBOND     CYS A    3    CYS A   40                                       2.03  \\nSSBOND     CYS A    4    CYS A   32                                       2.04  \\nSSBOND     CYS A   16    CYS A   26                                       2.03  \\nATOM      1  N   THR A   1      16.964  13.974   3.550  1.00  7.40           N  \\nATOM      2  CA  THR A   1      16.911  12.715   4.354  1.00  4.88           C  \\nATOM      3  C   THR A   1      15.617  12.680   5.067  1.00  3.24           C  \\nATOM      4  O   THR A   1      15.135  13.741   5.535  1.00  6.32           O  \\nATOM      5  CB  THR A   1      18.087  12.705   5.331  1.00  8.64           C  \\nATOM      6  OG1 THR A   1      19.254  12.847   4.529  1.00  7.85           O  \\nATOM      7  CG2 THR A   1      18.131  11.440   6.179  1.00  8.70           C  \\nATOM      8 1H   THR A   1      17.833  14.019   3.058  1.00  0.00           H  \\nATOM      9 2H   THR A   1      16.209  13.983   2.894  1.00  0.00           H  \\nATOM     10 3H   THR A   1      16.885  14.763   4.159  1.00  0.00           H  \\nATOM     11  HA  THR A   1      17.003  11.966   3.702  1.00 11.94           H  \\nATOM     12  HB  THR A   1      18.004  13.512   5.933  1.00 36.57           H  \\nATOM     13  HG1 THR A   1      19.001  12.910   3.605  1.00  0.00           H  \\nATOM     14 1HG2 THR A   1      17.274  11.326   6.700  1.00  4.78           H  \\nATOM     15 2HG2 THR A   1      18.199  10.654   5.571  1.00  5.04           H  \\nATOM     16 3HG2 THR A   1      18.888  11.445   6.792  1.00  8.71           H  \\nATOM     17  N   THR A   2      15.050  11.498   5.182  1.00  3.54           N  \\nATOM     18  CA  THR A   2      13.800  11.354   5.944  1.00  2.98           C  \\nATOM     19  C   THR A   2      14.118  10.664   7.274  1.00  2.92           C  \\nATOM     20  O   THR A   2      15.013   9.787   7.376  1.00  3.70           O  \\nATOM     21  CB  THR A   2      12.784  10.610   5.099  1.00  4.35           C  \\nATOM     22  OG1 THR A   2      13.364   9.360   4.756  1.00  5.60           O  \\nATOM     23  CG2 THR A   2      12.451  11.323   3.797  1.00  7.65           C  \\nATOM     24  H   THR A   2      15.500  10.728   5.006  1.00  1.74           H  \\nATOM     25  HA  THR A   2      13.437  12.241   6.128  1.00  4.09           H  \\nATOM     26  HB  THR A   2      11.947  10.457   5.635  1.00  1.11           H  \\nATOM     27  HG1 THR A   2      14.242   9.299   5.140  1.00  0.00           H  \\nATOM     28 1HG2 THR A   2      13.241  11.856   3.510  1.00  1.00           H  \\nATOM     29 2HG2 THR A   2      12.211  10.692   3.092  1.00  5.54           H  \\nATOM     30 3HG2 THR A   2      11.693  11.979   3.928  1.00  9.26           H  \\nATOM     31  N   CYS A   3      13.427  11.137   8.281  1.00  2.67           N  \\nATOM     32  CA  CYS A   3      13.573  10.632   9.683  1.00  3.33           C  \\nATOM     33  C   CYS A   3      12.203  10.369  10.192  1.00  2.48           C  \\nATOM     34  O   CYS A   3      11.304  11.202   9.989  1.00  3.56           O  \\nATOM     35  CB  CYS A   3      14.290  11.646  10.572  1.00  3.72           C  \\nATOM     36  SG  CYS A   3      15.776  12.403   9.909  1.00  3.50           S  \\nATOM     37  H   CYS A   3      12.694  11.667   8.175  1.00  2.61           H  \\nATOM     38  HA  CYS A   3      14.136   9.848   9.663  1.00  1.99           H  \\nATOM     39 1HB  CYS A   3      13.666  12.380  10.778  1.00  2.36           H  \\nATOM     40 2HB  CYS A   3      14.534  11.231  11.427  1.00  5.69           H  \\nATOM     41  N   CYS A   4      12.008   9.241  10.913  1.00  2.21           N  \\nATOM     42  CA  CYS A   4      10.680   8.906  11.348  1.00  2.79           C  \\nATOM     43  C   CYS A   4      10.653   8.718  12.890  1.00  1.95           C  \\nATOM     44  O   CYS A   4      11.636   8.232  13.477  1.00  2.71           O  \\nATOM     45  CB  CYS A   4      10.146   7.657  10.637  1.00  1.88           C  \\nATOM     46  SG  CYS A   4       9.807   7.893   8.888  1.00  2.64           S  \\nATOM     47  H   CYS A   4      12.680   8.639  10.972  1.00  3.79           H  \\nATOM     48  HA  CYS A   4      10.040   9.609  11.131  1.00  2.07           H  \\nATOM     49 1HB  CYS A   4      10.807   6.942  10.710  1.00  1.00           H  \\nATOM     50 2HB  CYS A   4       9.320   7.356  11.060  1.00  1.15           H  \\nATOM     51  N   PRO A   5       9.523   9.074  13.514  1.00  1.98           N  \\nATOM     52  CA  PRO A   5       9.452   9.028  14.967  1.00  2.21           C  \\nATOM     53  C   PRO A   5       9.332   7.648  15.601  1.00  3.11           C  \\nATOM     54  O   PRO A   5       9.619   7.492  16.798  1.00  3.37           O  \\nATOM     55  CB  PRO A   5       8.215   9.866  15.291  1.00  3.11           C  \\nATOM     56  CG  PRO A   5       7.324   9.692  14.078  1.00  2.85           C  \\nATOM     57  CD  PRO A   5       8.335   9.701  12.896  1.00  2.30           C  \\nATOM     58  HA  PRO A   5      10.250   9.418  15.361  1.00  1.18           H  \\nATOM     59 1HB  PRO A   5       7.788   9.653  16.052  1.00  1.00           H  \\nATOM     60 2HB  PRO A   5       8.454  10.786  15.422  1.00  9.12           H  \\nATOM     61 1HG  PRO A   5       6.940   8.954  14.085  1.00  1.00           H  \\nATOM     62 2HG  PRO A   5       6.694  10.413  13.945  1.00  2.23           H  \\nATOM     63 1HD  PRO A   5       7.970   9.189  12.161  1.00 21.74           H  \\nATOM     64 2HD  PRO A   5       8.555  10.587  12.559  1.00  1.09           H  \\nATOM     65  N   SER A   6       8.843   6.697  14.807  1.00  2.50           N  \\nATOM     66  CA  SER A   6       8.667   5.337  15.289  1.00  2.10           C  \\nATOM     67  C   SER A   6       8.826   4.406  14.080  1.00  2.51           C  \\nATOM     68  O   SER A   6       8.772   4.843  12.923  1.00  2.59           O  \\nATOM     69  CB  SER A   6       7.336   5.164  15.936  1.00  2.47           C  \\nATOM     70  OG  SER A   6       6.301   5.230  14.949  1.00  3.00           O  \\nATOM     71  H   SER A   6       8.533   6.896  13.977  1.00 17.98           H  \\nATOM     72  HA  SER A   6       9.345   5.117  15.952  1.00  1.96           H  \\nATOM     73 1HB  SER A   6       7.270   4.271  16.360  1.00  3.42           H  \\nATOM     74 2HB  SER A   6       7.164   5.850  16.638  1.00  1.18           H  \\nATOM     75  HG  SER A   6       6.745   5.372  14.109  1.00  0.00           H  \\nATOM     76  N   ILE A   7       8.861   3.112  14.380  1.00  2.83           N  \\nATOM     77  CA  ILE A   7       8.948   2.068  13.346  1.00  3.68           C  \\nATOM     78  C   ILE A   7       7.725   2.088  12.438  1.00  3.09           C  \\nATOM     79  O   ILE A   7       7.890   2.069  11.213  1.00  3.82           O  \\nATOM     80  CB  ILE A   7       9.139   0.695  13.989  1.00  6.17           C  \\nATOM     81  CG1 ILE A   7      10.455   0.525  14.728  1.00  6.83           C  \\nATOM     82  CG2 ILE A   7       9.072  -0.400  12.934  1.00  8.69           C  \\nATOM     83  CD1 ILE A   7      11.656   0.274  13.824  1.00  5.54           C  \\nATOM     84  H   ILE A   7       8.752   2.816  15.234  1.00  7.72           H  \\nATOM     85  HA  ILE A   7       9.748   2.237  12.802  1.00  1.69           H  \\nATOM     86  HB  ILE A   7       8.388   0.571  14.620  1.00  4.71           H  \\nATOM     87 1HG1 ILE A   7      10.612   1.356  15.237  1.00 12.19           H  \\nATOM     88 2HG1 ILE A   7      10.381  -0.243  15.348  1.00 24.95           H  \\nATOM     89 1HG2 ILE A   7       9.562  -0.104  12.124  1.00 14.52           H  \\nATOM     90 2HG2 ILE A   7       9.482  -1.221  13.309  1.00 14.28           H  \\nATOM     91 3HG2 ILE A   7       8.126  -0.576  12.705  1.00  1.48           H  \\nATOM     92 1HD1 ILE A   7      11.782   1.046  13.200  1.00  1.00           H  \\nATOM     93 2HD1 ILE A   7      12.457   0.142  14.402  1.00  8.13           H  \\nATOM     94 3HD1 ILE A   7      11.483  -0.567  13.323  1.00 24.48           H  \\nATOM     95  N   VAL A   8       6.557   2.085  13.039  1.00  3.25           N  \\nATOM     96  CA  VAL A   8       5.374   2.026  12.194  1.00  3.81           C  \\nATOM     97  C   VAL A   8       5.190   3.318  11.351  1.00  4.01           C  \\nATOM     98  O   VAL A   8       4.676   3.227  10.212  1.00  2.78           O  \\nATOM     99  CB  VAL A   8       4.063   1.695  12.923  1.00  3.19           C  \\nATOM    100  CG1 VAL A   8       3.361   2.862  13.554  1.00  5.80           C  \\nATOM    101  CG2 VAL A   8       3.102   1.021  11.942  1.00  5.67           C  \\nATOM    102  H   VAL A   8       6.474   1.972  13.932  1.00  2.36           H  \\nATOM    103  HA  VAL A   8       5.473   1.286  11.550  1.00 15.45           H  \\nATOM    104  HB  VAL A   8       4.315   1.048  13.631  1.00 27.87           H  \\nATOM    105 1HG1 VAL A   8       3.965   3.600  13.787  1.00 27.46           H  \\nATOM    106 2HG1 VAL A   8       2.666   3.237  12.964  1.00  1.00           H  \\nATOM    107 3HG1 VAL A   8       2.901   2.607  14.387  1.00  6.74           H  \\nATOM    108 1HG2 VAL A   8       3.216   1.388  11.044  1.00  1.28           H  \\nATOM    109 2HG2 VAL A   8       3.216   0.057  11.891  1.00 21.28           H  \\nATOM    110 3HG2 VAL A   8       2.177   1.203  12.180  1.00 22.95           H  \\nATOM    111  N   ALA A   9       5.683   4.414  11.850  1.00  3.23           N  \\nATOM    112  CA  ALA A   9       5.645   5.648  11.058  1.00  3.33           C  \\nATOM    113  C   ALA A   9       6.474   5.468   9.778  1.00  1.97           C  \\nATOM    114  O   ALA A   9       6.047   5.861   8.681  1.00  2.71           O  \\nATOM    115  CB  ALA A   9       6.063   6.882  11.859  1.00  3.85           C  \\nATOM    116  H   ALA A   9       6.199   4.406  12.607  1.00 22.78           H  \\nATOM    117  HA  ALA A   9       4.709   5.811  10.788  1.00  3.33           H  \\nATOM    118 1HB  ALA A   9       6.971   6.766  12.210  1.00  1.12           H  \\nATOM    119 2HB  ALA A   9       6.053   7.680  11.282  1.00  1.16           H  \\nATOM    120 3HB  ALA A   9       5.445   7.021  12.608  1.00  1.00           H  \\nATOM    121  N   ARG A  10       7.641   4.855   9.956  1.00  2.20           N  \\nATOM    122  CA  ARG A  10       8.491   4.521   8.779  1.00  1.73           C  \\nATOM    123  C   ARG A  10       7.794   3.557   7.840  1.00  2.06           C  \\nATOM    124  O   ARG A  10       7.806   3.748   6.612  1.00  2.32           O  \\nATOM    125  CB  ARG A  10       9.861   3.977   9.199  1.00  2.14           C  \\nATOM    126  CG  ARG A  10      10.749   3.510   8.050  1.00  2.44           C  \\nATOM    127  CD  ARG A  10      11.276   4.598   7.171  1.00  2.31           C  \\nATOM    128  NE  ARG A  10      12.142   5.532   7.882  1.00  2.71           N  \\nATOM    129  CZ  ARG A  10      12.638   6.655   7.370  1.00  3.08           C  \\nATOM    130  NH1 ARG A  10      12.387   7.019   6.119  1.00  3.91           N  \\nATOM    131  NH2 ARG A  10      13.420   7.420   8.126  1.00  3.38           N  \\nATOM    132  H   ARG A  10       7.873   4.485  10.738  1.00  7.53           H  \\nATOM    133  HA  ARG A  10       8.669   5.364   8.308  1.00  8.15           H  \\nATOM    134 1HB  ARG A  10      10.301   4.680   9.733  1.00  2.38           H  \\nATOM    135 2HB  ARG A  10       9.693   3.242   9.822  1.00  1.98           H  \\nATOM    136 1HG  ARG A  10      11.519   3.022   8.417  1.00  1.25           H  \\nATOM    137 2HG  ARG A  10      10.304   2.860   7.511  1.00  1.31           H  \\nATOM    138 1HD  ARG A  10      11.805   4.156   6.478  1.00  3.82           H  \\nATOM    139 2HD  ARG A  10      10.579   5.047   6.697  1.00  1.18           H  \\nATOM    140  HE  ARG A  10      12.375   5.339   8.753  1.00  1.00           H  \\nATOM    141 1HH1 ARG A  10      11.851   6.541   5.553  1.00 12.22           H  \\nATOM    142 2HH1 ARG A  10      12.759   7.802   5.792  1.00  1.26           H  \\nATOM    143 1HH2 ARG A  10      13.640   7.201   8.979  1.00  1.34           H  \\nATOM    144 2HH2 ARG A  10      13.784   8.179   7.761  1.00 17.63           H  \\nATOM    145  N   SER A  11       7.224   2.499   8.390  1.00  2.47           N  \\nATOM    146  CA  SER A  11       6.524   1.546   7.525  1.00  2.77           C  \\nATOM    147  C   SER A  11       5.418   2.215   6.717  1.00  2.00           C  \\nATOM    148  O   SER A  11       5.249   1.974   5.505  1.00  2.68           O  \\nATOM    149  CB  SER A  11       5.977   0.404   8.334  1.00  2.72           C  \\nATOM    150  OG  SER A  11       7.066  -0.247   9.003  1.00  3.23           O  \\nATOM    151  H   SER A  11       7.339   2.296   9.280  1.00 21.06           H  \\nATOM    152  HA  SER A  11       7.175   1.150   6.905  1.00  1.58           H  \\nATOM    153 1HB  SER A  11       5.369   0.745   9.033  1.00  2.23           H  \\nATOM    154 2HB  SER A  11       5.485  -0.247   7.774  1.00  1.56           H  \\nATOM    155  HG  SER A  11       7.858   0.232   8.747  1.00  0.00           H  \\nATOM    156  N   ASN A  12       4.655   3.072   7.407  1.00  2.39           N  \\nATOM    157  CA  ASN A  12       3.604   3.824   6.745  1.00  2.38           C  \\nATOM    158  C   ASN A  12       4.150   4.797   5.700  1.00  2.09           C  \\nATOM    159  O   ASN A  12       3.537   4.954   4.628  1.00  2.54           O  \\nATOM    160  CB  ASN A  12       2.768   4.573   7.765  1.00  3.19           C  \\nATOM    161  CG  ASN A  12       1.895   3.666   8.614  1.00  5.38           C  \\nATOM    162  OD1 ASN A  12       1.605   2.542   8.213  1.00  6.27           O  \\nATOM    163  ND2 ASN A  12       1.482   4.163   9.783  1.00  5.55           N  \\nATOM    164  H   ASN A  12       4.736   3.147   8.305  1.00  8.30           H  \\nATOM    165  HA  ASN A  12       2.959   3.230   6.331  1.00  2.71           H  \\nATOM    166 1HB  ASN A  12       3.353   5.091   8.370  1.00  1.00           H  \\nATOM    167 2HB  ASN A  12       2.186   5.242   7.314  1.00  1.00           H  \\nATOM    168 1HD2 ASN A  12       1.713   5.009  10.034  1.00  1.00           H  \\nATOM    169 2HD2 ASN A  12       0.940   3.649  10.331  1.00 11.50           H  \\nATOM    170  N   PHE A  13       5.248   5.433   6.024  1.00  2.52           N  \\nATOM    171  CA  PHE A  13       5.942   6.331   5.086  1.00  1.68           C  \\nATOM    172  C   PHE A  13       6.258   5.590   3.779  1.00  1.95           C  \\nATOM    173  O   PHE A  13       5.990   6.074   2.638  1.00  2.30           O  \\nATOM    174  CB  PHE A  13       7.170   6.951   5.716  1.00  2.27           C  \\nATOM    175  CG  PHE A  13       7.812   8.001   4.863  1.00  1.94           C  \\nATOM    176  CD1 PHE A  13       7.423   9.337   4.990  1.00  3.65           C  \\nATOM    177  CD2 PHE A  13       8.826   7.669   3.984  1.00  3.68           C  \\nATOM    178  CE1 PHE A  13       8.024  10.325   4.205  1.00  4.79           C  \\nATOM    179  CE2 PHE A  13       9.461   8.635   3.204  1.00  5.85           C  \\nATOM    180  CZ  PHE A  13       9.046   9.963   3.317  1.00  6.30           C  \\nATOM    181  H   PHE A  13       5.614   5.362   6.858  1.00  1.58           H  \\nATOM    182  HA  PHE A  13       5.390   7.102   4.897  1.00  1.00           H  \\nATOM    183 1HB  PHE A  13       6.910   7.314   6.586  1.00  6.31           H  \\nATOM    184 2HB  PHE A  13       7.815   6.217   5.906  1.00  2.30           H  \\nATOM    185  HD1 PHE A  13       6.699   9.562   5.599  1.00  8.04           H  \\nATOM    186  HD2 PHE A  13       9.111   6.770   3.938  1.00  2.35           H  \\nATOM    187  HE1 PHE A  13       7.767  11.213   4.252  1.00 22.70           H  \\nATOM    188  HE2 PHE A  13      10.160   8.394   2.609  1.00  1.33           H  \\nATOM    189  HZ  PHE A  13       9.474  10.632   2.775  1.00  7.02           H  \\nATOM    190  N   ASN A  14       6.888   4.403   3.944  1.00  2.75           N  \\nATOM    191  CA  ASN A  14       7.263   3.624   2.761  1.00  2.04           C  \\nATOM    192  C   ASN A  14       6.099   3.183   1.920  1.00  1.90           C  \\nATOM    193  O   ASN A  14       6.203   3.176   0.665  1.00  2.45           O  \\nATOM    194  CB  ASN A  14       8.154   2.450   3.201  1.00  2.97           C  \\nATOM    195  CG  ASN A  14       9.510   2.894   3.674  1.00  3.01           C  \\nATOM    196  OD1 ASN A  14      10.013   3.962   3.263  1.00  3.90           O  \\nATOM    197  ND2 ASN A  14      10.138   2.083   4.511  1.00  3.39           N  \\nATOM    198  H   ASN A  14       7.355   4.260   4.708  1.00  1.17           H  \\nATOM    199  HA  ASN A  14       7.900   4.143   2.227  1.00 30.34           H  \\nATOM    200 1HB  ASN A  14       7.698   1.961   3.927  1.00 23.66           H  \\nATOM    201 2HB  ASN A  14       8.246   1.808   2.454  1.00  1.18           H  \\nATOM    202 1HD2 ASN A  14       9.753   1.308   4.801  1.00  2.48           H  \\nATOM    203 2HD2 ASN A  14      10.989   2.306   4.831  1.00  7.08           H  \\nATOM    204  N   VAL A  15       4.956   2.890   2.535  1.00  1.68           N  \\nATOM    205  CA  VAL A  15       3.744   2.572   1.751  1.00  2.09           C  \\nATOM    206  C   VAL A  15       3.246   3.822   1.015  1.00  2.32           C  \\nATOM    207  O   VAL A  15       2.904   3.792  -0.174  1.00  2.08           O  \\nATOM    208  CB  VAL A  15       2.674   1.873   2.594  1.00  2.71           C  \\nATOM    209  CG1 VAL A  15       1.357   1.810   1.870  1.00  4.19           C  \\nATOM    210  CG2 VAL A  15       3.118   0.473   2.998  1.00  3.20           C  \\nATOM    211  H   VAL A  15       4.866   3.040   3.420  1.00  5.75           H  \\nATOM    212  HA  VAL A  15       3.994   1.902   1.075  1.00  3.09           H  \\nATOM    213  HB  VAL A  15       2.562   2.435   3.404  1.00  2.45           H  \\nATOM    214 1HG1 VAL A  15       1.419   2.162   0.956  1.00  7.84           H  \\nATOM    215 2HG1 VAL A  15       1.028   0.885   1.792  1.00  2.23           H  \\nATOM    216 3HG1 VAL A  15       0.646   2.315   2.326  1.00  2.26           H  \\nATOM    217 1HG2 VAL A  15       3.990   0.446   3.427  1.00  7.04           H  \\nATOM    218 2HG2 VAL A  15       2.475   0.062   3.605  1.00  1.00           H  \\nATOM    219 3HG2 VAL A  15       3.147  -0.120   2.221  1.00  1.52           H  \\nATOM    220  N   CYS A  16       3.172   4.941   1.753  1.00  2.12           N  \\nATOM    221  CA  CYS A  16       2.775   6.205   1.138  1.00  1.65           C  \\nATOM    222  C   CYS A  16       3.606   6.530  -0.126  1.00  2.23           C  \\nATOM    223  O   CYS A  16       3.056   7.047  -1.122  1.00  2.18           O  \\nATOM    224  CB  CYS A  16       2.834   7.281   2.220  1.00  3.44           C  \\nATOM    225  SG  CYS A  16       2.485   8.959   1.654  1.00  2.74           S  \\nATOM    226  H   CYS A  16       3.511   4.955   2.595  1.00 30.84           H  \\nATOM    227  HA  CYS A  16       1.836   6.163   0.874  1.00  1.34           H  \\nATOM    228 1HB  CYS A  16       2.177   7.080   2.916  1.00  1.04           H  \\nATOM    229 2HB  CYS A  16       3.720   7.302   2.635  1.00  1.00           H  \\nATOM    230  N   ARG A  17       4.874   6.209  -0.068  1.00  2.13           N  \\nATOM    231  CA  ARG A  17       5.779   6.511  -1.184  1.00  2.57           C  \\nATOM    232  C   ARG A  17       5.580   5.626  -2.405  1.00  1.88           C  \\nATOM    233  O   ARG A  17       5.943   6.032  -3.507  1.00  3.12           O  \\nATOM    234  CB  ARG A  17       7.237   6.427  -0.699  1.00  2.14           C  \\nATOM    235  CG  ARG A  17       7.626   7.598   0.194  1.00  2.71           C  \\nATOM    236  CD  ARG A  17       7.882   8.874  -0.572  1.00  2.03           C  \\nATOM    237  NE  ARG A  17       9.145   8.817  -1.260  1.00  2.43           N  \\nATOM    238  CZ  ARG A  17       9.512   9.558  -2.317  1.00  2.20           C  \\nATOM    239  NH1 ARG A  17       8.637  10.366  -2.892  1.00  3.42           N  \\nATOM    240  NH2 ARG A  17      10.761   9.510  -2.753  1.00  2.49           N  \\nATOM    241  H   ARG A  17       5.263   5.942   0.716  1.00 12.39           H  \\nATOM    242  HA  ARG A  17       5.646   7.450  -1.445  1.00  3.45           H  \\nATOM    243 1HB  ARG A  17       7.345   5.563  -0.250  1.00 15.45           H  \\nATOM    244 2HB  ARG A  17       7.798   6.398  -1.494  1.00  1.00           H  \\nATOM    245 1HG  ARG A  17       6.912   7.782   0.833  1.00  1.00           H  \\nATOM    246 2HG  ARG A  17       8.387   7.405   0.724  1.00 11.97           H  \\nATOM    247 1HD  ARG A  17       7.154   8.938  -1.218  1.00  8.63           H  \\nATOM    248 2HD  ARG A  17       7.784   9.636  -0.008  1.00  3.94           H  \\nATOM    249  HE  ARG A  17       9.836   8.316  -0.877  1.00  1.00           H  \\nATOM    250 1HH1 ARG A  17       7.770  10.464  -2.632  1.00  1.21           H  \\nATOM    251 2HH1 ARG A  17       8.917  10.885  -3.617  1.00  2.39           H  \\nATOM    252 1HH2 ARG A  17      11.383   8.965  -2.375  1.00  3.74           H  \\nATOM    253 2HH2 ARG A  17      10.997  10.026  -3.469  1.00 11.02           H  \\nATOM    254  N   LEU A  18       4.966   4.437  -2.194  1.00  2.15           N  \\nATOM    255  CA  LEU A  18       4.915   3.480  -3.298  1.00  2.36           C  \\nATOM    256  C   LEU A  18       4.267   4.041  -4.578  1.00  1.99           C  \\nATOM    257  O   LEU A  18       4.804   3.777  -5.697  1.00  2.44           O  \\nATOM    258  CB  LEU A  18       4.135   2.222  -2.884  1.00  2.44           C  \\nATOM    259  CG  LEU A  18       4.897   1.269  -1.951  1.00  2.99           C  \\nATOM    260  CD1 LEU A  18       3.911   0.200  -1.510  1.00  4.76           C  \\nATOM    261  CD2 LEU A  18       6.146   0.688  -2.568  1.00  3.90           C  \\nATOM    262  H   LEU A  18       4.847   4.150  -1.350  1.00  3.14           H  \\nATOM    263  HA  LEU A  18       5.834   3.214  -3.530  1.00  1.11           H  \\nATOM    264 1HB  LEU A  18       3.313   2.488  -2.403  1.00  1.00           H  \\nATOM    265 2HB  LEU A  18       3.894   1.710  -3.701  1.00 21.31           H  \\nATOM    266  HG  LEU A  18       5.181   1.783  -1.152  1.00  3.60           H  \\nATOM    267 1HD1 LEU A  18       3.135   0.640  -1.094  1.00  1.30           H  \\nATOM    268 2HD1 LEU A  18       3.629  -0.312  -2.303  1.00  1.27           H  \\nATOM    269 3HD1 LEU A  18       4.361  -0.391  -0.856  1.00  1.53           H  \\nATOM    270 1HD2 LEU A  18       5.940   0.327  -3.489  1.00 10.88           H  \\nATOM    271 2HD2 LEU A  18       6.851   1.385  -2.671  1.00  2.80           H  \\nATOM    272 3HD2 LEU A  18       6.518  -0.054  -2.033  1.00 22.90           H  \\nATOM    273  N   PRO A  19       3.191   4.774  -4.549  1.00  2.11           N  \\nATOM    274  CA  PRO A  19       2.632   5.337  -5.774  1.00  2.38           C  \\nATOM    275  C   PRO A  19       3.380   6.530  -6.329  1.00  1.98           C  \\nATOM    276  O   PRO A  19       3.036   7.025  -7.419  1.00  3.72           O  \\nATOM    277  CB  PRO A  19       1.214   5.747  -5.417  1.00  2.26           C  \\nATOM    278  CG  PRO A  19       1.231   5.945  -3.911  1.00  3.82           C  \\nATOM    279  CD  PRO A  19       2.217   4.874  -3.434  1.00  2.16           C  \\nATOM    280  HA  PRO A  19       2.649   4.676  -6.487  1.00  1.84           H  \\nATOM    281 1HB  PRO A  19       0.889   6.473  -5.838  1.00  2.18           H  \\nATOM    282 2HB  PRO A  19       0.582   5.061  -5.661  1.00  1.31           H  \\nATOM    283 1HG  PRO A  19       1.520   6.702  -3.717  1.00 19.37           H  \\nATOM    284 2HG  PRO A  19       0.361   5.796  -3.501  1.00  1.64           H  \\nATOM    285 1HD  PRO A  19       2.640   5.153  -2.600  1.00 17.92           H  \\nATOM    286 2HD  PRO A  19       1.833   3.989  -3.257  1.00 34.59           H  \\nATOM    287  N   GLY A  20       4.385   7.010  -5.587  1.00  2.12           N  \\nATOM    288  CA  GLY A  20       5.215   8.141  -5.998  1.00  2.37           C  \\nATOM    289  C   GLY A  20       4.881   9.460  -5.286  1.00  2.40           C  \\nATOM    290  O   GLY A  20       5.399  10.501  -5.709  1.00  3.44           O  \\nATOM    291  H   GLY A  20       4.689   6.515  -4.879  1.00  1.78           H  \\nATOM    292 1HA  GLY A  20       6.157   7.928  -5.814  1.00  1.87           H  \\nATOM    293 2HA  GLY A  20       5.149   8.261  -6.986  1.00  1.04           H  \\nATOM    294  N   THR A  21       4.025   9.398  -4.254  1.00  2.24           N  \\nATOM    295  CA  THR A  21       3.668  10.548  -3.433  1.00  2.10           C  \\nATOM    296  C   THR A  21       4.937  11.268  -2.988  1.00  1.65           C  \\nATOM    297  O   THR A  21       5.886  10.632  -2.523  1.00  2.81           O  \\nATOM    298  CB  THR A  21       2.929  10.063  -2.210  1.00  1.72           C  \\nATOM    299  OG1 THR A  21       1.921   9.122  -2.580  1.00  1.91           O  \\nATOM    300  CG2 THR A  21       2.250  11.223  -1.487  1.00  2.22           C  \\nATOM    301  H   THR A  21       3.597   8.608  -4.094  1.00  4.76           H  \\nATOM    302  HA  THR A  21       3.095  11.155  -3.926  1.00  1.47           H  \\nATOM    303  HB  THR A  21       3.575   9.611  -1.579  1.00  1.00           H  \\nATOM    304  HG1 THR A  21       1.932   9.001  -3.533  1.00  0.00           H  \\nATOM    305 1HG2 THR A  21       1.730  11.750  -2.149  1.00  1.02           H  \\nATOM    306 2HG2 THR A  21       1.653  10.901  -0.785  1.00  2.24           H  \\nATOM    307 3HG2 THR A  21       2.934  11.846  -1.086  1.00  1.00           H  \\nATOM    308  N   SER A  22       4.885  12.600  -3.031  1.00  2.62           N  \\nATOM    309  CA  SER A  22       5.988  13.420  -2.521  1.00  1.76           C  \\nATOM    310  C   SER A  22       6.355  13.048  -1.102  1.00  2.62           C  \\nATOM    311  O   SER A  22       5.495  12.762  -0.268  1.00  3.10           O  \\nATOM    312  CB  SER A  22       5.591  14.892  -2.580  1.00  2.04           C  \\nATOM    313  OG  SER A  22       6.644  15.731  -2.154  1.00  4.09           O  \\nATOM    314  H   SER A  22       4.097  13.030  -3.197  1.00  1.63           H  \\nATOM    315  HA  SER A  22       6.760  13.340  -3.107  1.00  1.66           H  \\nATOM    316 1HB  SER A  22       5.378  15.113  -3.521  1.00 10.38           H  \\nATOM    317 2HB  SER A  22       4.774  15.043  -2.033  1.00 16.82           H  \\nATOM    318  HG  SER A  22       7.372  15.147  -1.926  1.00  0.00           H  \\nATOM    319  N   GLU A  23       7.655  13.108  -0.829  1.00  2.23           N  \\nATOM    320  CA  GLU A  23       8.085  12.974   0.585  1.00  2.20           C  \\nATOM    321  C   GLU A  23       7.375  13.950   1.526  1.00  2.73           C  \\nATOM    322  O   GLU A  23       7.052  13.560   2.662  1.00  2.78           O  \\nATOM    323  CB  GLU A  23       9.591  13.115   0.753  1.00  2.94           C  \\nATOM    324  CG  GLU A  23      10.326  11.958   0.104  1.00  3.57           C  \\nATOM    325  CD  GLU A  23      11.788  11.947   0.391  1.00  5.57           C  \\nATOM    326  OE1 GLU A  23      12.410  12.936   0.725  1.00  6.10           O  \\nATOM    327  OE2 GLU A  23      12.444  10.818   0.157  1.00  7.69           O  \\nATOM    328  H   GLU A  23       8.264  13.424  -1.411  1.00  1.00           H  \\nATOM    329  HA  GLU A  23       7.825  12.056   0.867  1.00  2.75           H  \\nATOM    330 1HB  GLU A  23       9.898  13.956   0.326  1.00  1.83           H  \\nATOM    331 2HB  GLU A  23       9.790  13.168   1.704  1.00  5.20           H  \\nATOM    332 1HG  GLU A  23       9.967  11.080   0.350  1.00  1.36           H  \\nATOM    333 2HG  GLU A  23      10.219  12.029  -0.876  1.00 16.53           H  \\nATOM    334  N   ALA A  24       7.148  15.182   1.059  1.00  2.21           N  \\nATOM    335  CA  ALA A  24       6.537  16.162   1.966  1.00  2.07           C  \\nATOM    336  C   ALA A  24       5.137  15.776   2.367  1.00  2.28           C  \\nATOM    337  O   ALA A  24       4.735  15.971   3.532  1.00  2.52           O  \\nATOM    338  CB  ALA A  24       6.579  17.557   1.315  1.00  3.18           C  \\nATOM    339  H   ALA A  24       7.609  15.473   0.337  1.00  1.36           H  \\nATOM    340  HA  ALA A  24       7.104  16.239   2.768  1.00  2.14           H  \\nATOM    341 1HB  ALA A  24       7.503  17.798   1.109  1.00  1.76           H  \\nATOM    342 2HB  ALA A  24       6.051  17.549   0.491  1.00  1.58           H  \\nATOM    343 3HB  ALA A  24       6.192  18.215   1.933  1.00  3.08           H  \\nATOM    344  N   ILE A  25       4.329  15.256   1.435  1.00  2.04           N  \\nATOM    345  CA  ILE A  25       3.005  14.798   1.771  1.00  2.09           C  \\nATOM    346  C   ILE A  25       3.062  13.560   2.694  1.00  2.09           C  \\nATOM    347  O   ILE A  25       2.311  13.417   3.660  1.00  2.42           O  \\nATOM    348  CB  ILE A  25       2.177  14.515   0.507  1.00  3.02           C  \\nATOM    349  CG1 ILE A  25       1.901  15.860  -0.176  1.00  3.40           C  \\nATOM    350  CG2 ILE A  25       0.920  13.713   0.808  1.00  2.72           C  \\nATOM    351  CD1 ILE A  25       1.370  15.821  -1.605  1.00  3.13           C  \\nATOM    352  H   ILE A  25       4.711  14.990   0.617  1.00  8.00           H  \\nATOM    353  HA  ILE A  25       2.514  15.498   2.260  1.00  1.84           H  \\nATOM    354  HB  ILE A  25       2.759  13.978  -0.085  1.00  1.14           H  \\nATOM    355 1HG1 ILE A  25       1.235  16.320   0.391  1.00  1.24           H  \\nATOM    356 2HG1 ILE A  25       2.747  16.375  -0.186  1.00  1.00           H  \\nATOM    357 1HG2 ILE A  25       0.488  13.470  -0.050  1.00  1.00           H  \\nATOM    358 2HG2 ILE A  25       0.310  14.252   1.372  1.00  1.00           H  \\nATOM    359 3HG2 ILE A  25       1.169  12.886   1.298  1.00  6.18           H  \\nATOM    360 1HD1 ILE A  25       0.627  15.155  -1.675  1.00  7.94           H  \\nATOM    361 2HD1 ILE A  25       1.063  16.736  -1.848  1.00 25.31           H  \\nATOM    362 3HD1 ILE A  25       2.130  15.571  -2.197  1.00 16.42           H  \\nATOM    363  N   CYS A  26       4.013  12.678   2.354  1.00  2.38           N  \\nATOM    364  CA  CYS A  26       4.083  11.486   3.180  1.00  2.10           C  \\nATOM    365  C   CYS A  26       4.529  11.821   4.621  1.00  1.46           C  \\nATOM    366  O   CYS A  26       4.174  11.111   5.587  1.00  2.51           O  \\nATOM    367  CB  CYS A  26       4.981  10.374   2.571  1.00  2.25           C  \\nATOM    368  SG  CYS A  26       4.275   9.703   1.042  1.00  2.45           S  \\nATOM    369  H   CYS A  26       4.462  12.733   1.584  1.00  1.89           H  \\nATOM    370  HA  CYS A  26       3.205  11.059   3.236  1.00  2.97           H  \\nATOM    371 1HB  CYS A  26       5.836  10.756   2.341  1.00  1.46           H  \\nATOM    372 2HB  CYS A  26       5.078   9.648   3.192  1.00  1.67           H  \\nATOM    373  N   ALA A  27       5.362  12.851   4.739  1.00  2.20           N  \\nATOM    374  CA  ALA A  27       5.812  13.342   6.051  1.00  2.56           C  \\nATOM    375  C   ALA A  27       4.649  13.816   6.909  1.00  1.87           C  \\nATOM    376  O   ALA A  27       4.466  13.361   8.057  1.00  2.86           O  \\nATOM    377  CB  ALA A  27       6.883  14.439   5.887  1.00  2.10           C  \\nATOM    378  H   ALA A  27       5.770  13.200   4.004  1.00  1.04           H  \\nATOM    379  HA  ALA A  27       6.271  12.606   6.515  1.00  1.47           H  \\nATOM    380 1HB  ALA A  27       6.531  15.159   5.330  1.00  1.47           H  \\nATOM    381 2HB  ALA A  27       7.124  14.794   6.768  1.00  1.55           H  \\nATOM    382 3HB  ALA A  27       7.676  14.054   5.458  1.00  1.00           H  \\nATOM    383  N   THR A  28       3.820  14.730   6.379  1.00  2.39           N  \\nATOM    384  CA  THR A  28       2.751  15.196   7.191  1.00  2.24           C  \\nATOM    385  C   THR A  28       1.685  14.093   7.443  1.00  2.35           C  \\nATOM    386  O   THR A  28       0.965  14.087   8.454  1.00  3.22           O  \\nATOM    387  CB  THR A  28       2.037  16.445   6.621  1.00  2.62           C  \\nATOM    388  OG1 THR A  28       1.697  16.153   5.271  1.00  2.56           O  \\nATOM    389  CG2 THR A  28       2.928  17.660   6.674  1.00  3.71           C  \\nATOM    390  H   THR A  28       3.960  15.014   5.530  1.00  2.03           H  \\nATOM    391  HA  THR A  28       3.070  15.477   8.082  1.00  2.02           H  \\nATOM    392  HB  THR A  28       1.200  16.584   7.156  1.00  1.00           H  \\nATOM    393  HG1 THR A  28       1.991  15.265   5.052  1.00  0.00           H  \\nATOM    394 1HG2 THR A  28       3.371  17.763   7.580  1.00  1.14           H  \\nATOM    395 2HG2 THR A  28       3.678  17.552   6.020  1.00  1.13           H  \\nATOM    396 3HG2 THR A  28       2.447  18.486   6.464  1.00 13.35           H  \\nATOM    397  N   TYR A  29       1.652  13.136   6.523  1.00  2.97           N  \\nATOM    398  CA  TYR A  29       0.695  12.058   6.670  1.00  2.59           C  \\nATOM    399  C   TYR A  29       1.071  11.120   7.832  1.00  2.85           C  \\nATOM    400  O   TYR A  29       0.170  10.575   8.488  1.00  3.15           O  \\nATOM    401  CB  TYR A  29       0.568  11.279   5.338  1.00  3.41           C  \\nATOM    402  CG  TYR A  29      -0.535  10.247   5.310  1.00  3.14           C  \\nATOM    403  CD1 TYR A  29      -1.811  10.543   5.777  1.00  3.50           C  \\nATOM    404  CD2 TYR A  29      -0.271   8.941   4.895  1.00  6.74           C  \\nATOM    405  CE1 TYR A  29      -2.832   9.604   5.690  1.00  3.76           C  \\nATOM    406  CE2 TYR A  29      -1.266   7.971   4.862  1.00  8.08           C  \\nATOM    407  CZ  TYR A  29      -2.548   8.313   5.238  1.00  6.33           C  \\nATOM    408  OH  TYR A  29      -3.511   7.313   5.238  1.00  9.80           O  \\nATOM    409  H   TYR A  29       2.293  13.075   5.881  1.00  1.04           H  \\nATOM    410  HA  TYR A  29      -0.198  12.415   6.845  1.00  1.50           H  \\nATOM    411 1HB  TYR A  29       0.405  11.951   4.631  1.00  5.15           H  \\nATOM    412 2HB  TYR A  29       1.448  10.860   5.142  1.00  1.10           H  \\nATOM    413  HD1 TYR A  29      -2.042  11.466   6.048  1.00  8.36           H  \\nATOM    414  HD2 TYR A  29       0.648   8.702   4.596  1.00  1.00           H  \\nATOM    415  HE1 TYR A  29      -3.746   9.823   6.001  1.00  1.24           H  \\nATOM    416  HE2 TYR A  29      -1.073   7.056   4.551  1.00 18.09           H  \\nATOM    417  HH  TYR A  29      -3.112   6.486   4.959  1.00  0.00           H  \\nATOM    418  N   THR A  30       2.379  10.893   7.997  1.00  2.88           N  \\nATOM    419  CA  THR A  30       2.870   9.886   8.916  1.00  3.09           C  \\nATOM    420  C   THR A  30       3.519  10.362  10.215  1.00  4.08           C  \\nATOM    421  O   THR A  30       3.687   9.566  11.146  1.00  4.42           O  \\nATOM    422  CB  THR A  30       3.932   8.990   8.199  1.00  3.09           C  \\nATOM    423  OG1 THR A  30       5.025   9.841   7.826  1.00  3.41           O  \\nATOM    424  CG2 THR A  30       3.343   8.274   7.009  1.00  4.83           C  \\nATOM    425  H   THR A  30       2.978  11.293   7.441  1.00  1.44           H  \\nATOM    426  HA  THR A  30       2.146   9.260   9.144  1.00  1.00           H  \\nATOM    427  HB  THR A  30       4.258   8.337   8.891  1.00  1.14           H  \\nATOM    428  HG1 THR A  30       4.839  10.741   8.105  1.00  0.00           H  \\nATOM    429 1HG2 THR A  30       2.905   8.936   6.403  1.00  1.00           H  \\nATOM    430 2HG2 THR A  30       4.020   7.783   6.499  1.00  1.04           H  \\nATOM    431 3HG2 THR A  30       2.614   7.626   7.284  1.00 13.17           H  \\nATOM    432  N   GLY A  31       3.930  11.629  10.199  1.00  3.24           N  \\nATOM    433  CA  GLY A  31       4.730  12.160  11.322  1.00  3.98           C  \\nATOM    434  C   GLY A  31       6.206  12.086  11.070  1.00  2.94           C  \\nATOM    435  O   GLY A  31       6.967  12.580  11.911  1.00  2.85           O  \\nATOM    436  H   GLY A  31       3.978  12.098   9.423  1.00 14.25           H  \\nATOM    437 1HA  GLY A  31       4.454  13.092  11.475  1.00 27.75           H  \\nATOM    438 2HA  GLY A  31       4.465  11.665  12.153  1.00  9.02           H  \\nATOM    439  N   CYS A  32       6.607  11.472   9.956  1.00  2.65           N  \\nATOM    440  CA  CYS A  32       8.008  11.553   9.581  1.00  2.49           C  \\nATOM    441  C   CYS A  32       8.345  13.003   9.174  1.00  2.45           C  \\nATOM    442  O   CYS A  32       7.446  13.847   9.004  1.00  3.38           O  \\nATOM    443  CB  CYS A  32       8.340  10.554   8.464  1.00  2.49           C  \\nATOM    444  SG  CYS A  32       7.990   8.827   8.866  1.00  2.48           S  \\nATOM    445  H   CYS A  32       5.999  11.210   9.337  1.00  8.40           H  \\nATOM    446  HA  CYS A  32       8.581  11.301  10.324  1.00 29.65           H  \\nATOM    447 1HB  CYS A  32       7.818  10.769   7.672  1.00  1.37           H  \\nATOM    448 2HB  CYS A  32       9.288  10.604   8.239  1.00  2.63           H  \\nATOM    449  N   ILE A  33       9.637  13.252   9.040  1.00  3.37           N  \\nATOM    450  CA  ILE A  33      10.148  14.569   8.687  1.00  2.86           C  \\nATOM    451  C   ILE A  33      11.209  14.452   7.608  1.00  3.09           C  \\nATOM    452  O   ILE A  33      11.839  13.391   7.388  1.00  3.22           O  \\nATOM    453  CB  ILE A  33      10.742  15.300   9.921  1.00  3.08           C  \\nATOM    454  CG1 ILE A  33      11.967  14.602  10.463  1.00  3.57           C  \\nATOM    455  CG2 ILE A  33       9.643  15.432  10.953  1.00  4.22           C  \\nATOM    456  CD1 ILE A  33      12.646  15.383  11.593  1.00  4.24           C  \\nATOM    457  H   ILE A  33      10.253  12.576   9.072  1.00  5.52           H  \\nATOM    458  HA  ILE A  33       9.418  15.138   8.376  1.00 14.18           H  \\nATOM    459  HB  ILE A  33      10.983  16.201   9.590  1.00 38.65           H  \\nATOM    460 1HG1 ILE A  33      11.684  13.729  10.820  1.00 19.10           H  \\nATOM    461 2HG1 ILE A  33      12.634  14.483   9.746  1.00  1.06           H  \\nATOM    462 1HG2 ILE A  33       8.844  15.851  10.538  1.00  2.10           H  \\nATOM    463 2HG2 ILE A  33       9.394  14.537  11.302  1.00  2.11           H  \\nATOM    464 3HG2 ILE A  33       9.967  16.013  11.691  1.00  4.03           H  \\nATOM    465 1HD1 ILE A  33      12.743  16.320  11.280  1.00 34.82           H  \\nATOM    466 2HD1 ILE A  33      12.088  15.331  12.419  1.00  1.05           H  \\nATOM    467 3HD1 ILE A  33      13.549  14.992  11.729  1.00  1.31           H  \\nATOM    468  N   ILE A  34      11.426  15.582   6.911  1.00  3.27           N  \\nATOM    469  CA  ILE A  34      12.486  15.737   5.910  1.00  3.42           C  \\nATOM    470  C   ILE A  34      13.470  16.738   6.437  1.00  2.87           C  \\nATOM    471  O   ILE A  34      13.033  17.834   6.853  1.00  4.13           O  \\nATOM    472  CB  ILE A  34      11.966  16.250   4.571  1.00  4.16           C  \\nATOM    473  CG1 ILE A  34      10.864  15.373   3.994  1.00  4.74           C  \\nATOM    474  CG2 ILE A  34      13.097  16.354   3.565  1.00  6.10           C  \\nATOM    475  CD1 ILE A  34      10.036  16.027   2.890  1.00  5.45           C  \\nATOM    476  H   ILE A  34      10.829  16.269   6.963  1.00 16.22           H  \\nATOM    477  HA  ILE A  34      12.909  14.874   5.749  1.00  1.00           H  \\nATOM    478  HB  ILE A  34      11.607  17.158   4.736  1.00 12.04           H  \\nATOM    479 1HG1 ILE A  34      10.259  15.143   4.740  1.00  1.49           H  \\nATOM    480 2HG1 ILE A  34      11.267  14.553   3.619  1.00 10.87           H  \\nATOM    481 1HG2 ILE A  34      13.601  15.499   3.541  1.00  1.00           H  \\nATOM    482 2HG2 ILE A  34      12.709  16.552   2.673  1.00 19.94           H  \\nATOM    483 3HG2 ILE A  34      13.707  17.089   3.826  1.00  2.47           H  \\nATOM    484 1HD1 ILE A  34      10.637  16.377   2.174  1.00  1.82           H  \\nATOM    485 2HD1 ILE A  34       9.402  15.345   2.536  1.00  9.03           H  \\nATOM    486 3HD1 ILE A  34       9.512  16.760   3.311  1.00 19.11           H  \\nATOM    487  N   ILE A  35      14.749  16.368   6.445  1.00  2.75           N  \\nATOM    488  CA  ILE A  35      15.789  17.309   6.862  1.00  2.90           C  \\nATOM    489  C   ILE A  35      16.776  17.474   5.716  1.00  3.20           C  \\nATOM    490  O   ILE A  35      16.949  16.528   4.919  1.00  4.31           O  \\nATOM    491  CB  ILE A  35      16.508  16.857   8.149  1.00  4.73           C  \\nATOM    492  CG1 ILE A  35      17.203  15.509   7.996  1.00  4.32           C  \\nATOM    493  CG2 ILE A  35      15.525  16.833   9.308  1.00  5.81           C  \\nATOM    494  CD1 ILE A  35      18.138  15.189   9.165  1.00  6.90           C  \\nATOM    495  H   ILE A  35      15.012  15.628   5.997  1.00  3.21           H  \\nATOM    496  HA  ILE A  35      15.381  18.173   7.068  1.00  2.09           H  \\nATOM    497  HB  ILE A  35      17.191  17.550   8.330  1.00  1.53           H  \\nATOM    498 1HG1 ILE A  35      16.501  14.824   7.958  1.00 14.79           H  \\nATOM    499 2HG1 ILE A  35      17.741  15.514   7.174  1.00  1.00           H  \\nATOM    500 1HG2 ILE A  35      15.061  17.707   9.364  1.00  3.70           H  \\nATOM    501 2HG2 ILE A  35      14.855  16.116   9.157  1.00 21.37           H  \\nATOM    502 3HG2 ILE A  35      16.027  16.670  10.146  1.00  5.66           H  \\nATOM    503 1HD1 ILE A  35      17.785  15.603  10.001  1.00  6.46           H  \\nATOM    504 2HD1 ILE A  35      18.205  14.200   9.242  1.00  4.53           H  \\nATOM    505 3HD1 ILE A  35      19.032  15.554   8.930  1.00 16.60           H  \\nATOM    506  N   PRO A  36      17.500  18.590   5.692  1.00  4.28           N  \\nATOM    507  CA  PRO A  36      18.486  18.750   4.634  1.00  3.76           C  \\nATOM    508  C   PRO A  36      19.749  17.969   4.860  1.00  4.79           C  \\nATOM    509  O   PRO A  36      20.493  17.725   3.899  1.00  6.10           O  \\nATOM    510  CB  PRO A  36      18.674  20.267   4.581  1.00  7.70           C  \\nATOM    511  CG  PRO A  36      18.258  20.827   5.923  1.00  6.81           C  \\nATOM    512  CD  PRO A  36      17.153  19.872   6.422  1.00  4.73           C  \\nATOM    513  HA  PRO A  36      18.146  18.411   3.790  1.00  7.56           H  \\nATOM    514 1HB  PRO A  36      19.503  20.541   4.350  1.00  9.51           H  \\nATOM    515 2HB  PRO A  36      18.132  20.641   3.872  1.00  3.67           H  \\nATOM    516 1HG  PRO A  36      18.894  20.806   6.466  1.00  4.56           H  \\nATOM    517 2HG  PRO A  36      17.889  21.725   5.870  1.00  9.72           H  \\nATOM    518 1HD  PRO A  36      17.212  19.754   7.374  1.00  1.00           H  \\nATOM    519 2HD  PRO A  36      16.244  20.115   6.193  1.00  9.27           H  \\nATOM    520  N   GLY A  37      20.048  17.720   6.144  1.00  5.98           N  \\nATOM    521  CA  GLY A  37      21.260  16.950   6.470  1.00  7.34           C  \\nATOM    522  C   GLY A  37      20.950  15.447   6.407  1.00  5.86           C  \\nATOM    523  O   GLY A  37      19.910  15.051   5.860  1.00  6.60           O  \\nATOM    524  H   GLY A  37      19.445  17.850   6.793  1.00  1.75           H  \\nATOM    525 1HA  GLY A  37      21.955  17.153   5.821  1.00  1.00           H  \\nATOM    526 2HA  GLY A  37      21.591  17.207   7.362  1.00  2.90           H  \\nATOM    527  N   ALA A  38      21.921  14.699   6.948  1.00  4.09           N  \\nATOM    528  CA  ALA A  38      21.796  13.225   6.966  1.00  4.89           C  \\nATOM    529  C   ALA A  38      21.760  12.594   8.339  1.00  5.34           C  \\nATOM    530  O   ALA A  38      21.731  11.356   8.416  1.00  7.20           O  \\nATOM    531  CB  ALA A  38      22.990  12.671   6.193  1.00  6.50           C  \\nATOM    532  H   ALA A  38      22.766  15.014   7.023  1.00  7.54           H  \\nATOM    533  HA  ALA A  38      20.997  12.980   6.460  1.00 26.11           H  \\nATOM    534 1HB  ALA A  38      23.103  13.161   5.348  1.00  4.37           H  \\nATOM    535 2HB  ALA A  38      23.813  12.768   6.720  1.00  1.21           H  \\nATOM    536 3HB  ALA A  38      22.858  11.717   5.994  1.00  6.32           H  \\nATOM    537  N   THR A  39      21.731  13.436   9.377  1.00  4.59           N  \\nATOM    538  CA  THR A  39      21.844  12.956  10.763  1.00  4.87           C  \\nATOM    539  C   THR A  39      20.537  13.181  11.479  1.00  4.29           C  \\nATOM    540  O   THR A  39      20.228  14.281  11.997  1.00  7.12           O  \\nATOM    541  CB  THR A  39      23.065  13.641  11.462  1.00  5.05           C  \\nATOM    542  OG1 THR A  39      24.184  13.459  10.655  1.00  5.24           O  \\nATOM    543  CG2 THR A  39      23.335  13.039  12.827  1.00  8.58           C  \\nATOM    544  H   THR A  39      21.871  14.322   9.243  1.00 43.17           H  \\nATOM    545  HA  THR A  39      22.069  12.007  10.755  1.00  1.15           H  \\nATOM    546  HB  THR A  39      22.826  14.615  11.549  1.00 28.76           H  \\nATOM    547  HG1 THR A  39      23.936  12.961   9.872  1.00  0.00           H  \\nATOM    548 1HG2 THR A  39      22.520  13.099  13.423  1.00  6.58           H  \\nATOM    549 2HG2 THR A  39      23.529  12.066  12.719  1.00  5.71           H  \\nATOM    550 3HG2 THR A  39      24.091  13.467  13.273  1.00  6.17           H  \\nATOM    551  N   CYS A  40      19.716  12.124  11.531  1.00  5.13           N  \\nATOM    552  CA  CYS A  40      18.443  12.197  12.212  1.00  3.45           C  \\nATOM    553  C   CYS A  40      18.632  12.283  13.732  1.00  3.56           C  \\nATOM    554  O   CYS A  40      19.477  11.575  14.277  1.00  5.42           O  \\nATOM    555  CB  CYS A  40      17.565  10.987  11.858  1.00  3.80           C  \\nATOM    556  SG  CYS A  40      17.122  10.899  10.106  1.00  3.85           S  \\nATOM    557  H   CYS A  40      19.959  11.330  11.159  1.00 16.91           H  \\nATOM    558  HA  CYS A  40      17.930  12.969  11.914  1.00 30.53           H  \\nATOM    559 1HB  CYS A  40      18.043  10.169  12.069  1.00  2.47           H  \\nATOM    560 2HB  CYS A  40      16.732  11.014  12.359  1.00 15.26           H  \\nATOM    561  N   PRO A  41      17.806  13.078  14.390  1.00  4.30           N  \\nATOM    562  CA  PRO A  41      17.904  13.208  15.859  1.00  5.14           C  \\nATOM    563  C   PRO A  41      17.335  12.004  16.593  1.00  4.23           C  \\nATOM    564  O   PRO A  41      16.580  11.235  16.009  1.00  3.67           O  \\nATOM    565  CB  PRO A  41      17.165  14.509  16.118  1.00  6.36           C  \\nATOM    566  CG  PRO A  41      16.041  14.509  15.075  1.00  6.31           C  \\nATOM    567  CD  PRO A  41      16.711  13.937  13.818  1.00  4.81           C  \\nATOM    568  HA  PRO A  41      18.844  13.274  16.105  1.00  1.94           H  \\nATOM    569 1HB  PRO A  41      16.831  14.630  16.940  1.00  8.14           H  \\nATOM    570 2HB  PRO A  41      17.734  15.269  15.990  1.00  5.84           H  \\nATOM    571 1HG  PRO A  41      15.427  14.006  15.323  1.00  4.02           H  \\nATOM    572 2HG  PRO A  41      15.712  15.400  14.892  1.00 37.89           H  \\nATOM    573 1HD  PRO A  41      16.082  13.407  13.307  1.00  2.11           H  \\nATOM    574 2HD  PRO A  41      17.110  14.583  13.210  1.00 10.45           H  \\nATOM    575  N   GLY A  42      17.621  11.960  17.887  1.00  4.53           N  \\nATOM    576  CA  GLY A  42      17.243  10.798  18.724  1.00  4.51           C  \\nATOM    577  C   GLY A  42      15.754  10.567  18.845  1.00  4.66           C  \\nATOM    578  O   GLY A  42      15.313   9.427  19.147  1.00  5.28           O  \\nATOM    579  H   GLY A  42      18.241  12.514  18.260  1.00 17.19           H  \\nATOM    580 1HA  GLY A  42      17.664  10.001  18.327  1.00  1.42           H  \\nATOM    581 2HA  GLY A  42      17.673  10.912  19.619  1.00  1.00           H  \\nATOM    582  N   ASP A  43      14.965  11.636  18.653  1.00  4.28           N  \\nATOM    583  CA  ASP A  43      13.516  11.493  18.710  1.00  3.50           C  \\nATOM    584  C   ASP A  43      12.833  11.234  17.331  1.00  2.35           C  \\nATOM    585  O   ASP A  43      11.608  11.086  17.287  1.00  3.20           O  \\nATOM    586  CB  ASP A  43      12.831  12.689  19.444  1.00  3.86           C  \\nATOM    587  CG  ASP A  43      13.113  14.044  18.872  1.00  3.26           C  \\nATOM    588  OD1 ASP A  43      14.026  14.181  18.117  1.00  4.70           O  \\nATOM    589  OD2 ASP A  43      12.260  14.976  19.236  1.00  5.33           O  \\nATOM    590  H   ASP A  43      15.316  12.387  18.296  1.00  2.13           H  \\nATOM    591  HA  ASP A  43      13.280  10.739  19.286  1.00  2.21           H  \\nATOM    592 1HB  ASP A  43      11.861  12.536  19.487  1.00  1.02           H  \\nATOM    593 2HB  ASP A  43      13.136  12.666  20.371  1.00 10.83           H  \\nATOM    594  N   TYR A  44      13.678  11.149  16.321  1.00  2.34           N  \\nATOM    595  CA  TYR A  44      13.245  10.686  14.994  1.00  2.19           C  \\nATOM    596  C   TYR A  44      14.250   9.639  14.507  1.00  3.17           C  \\nATOM    597  O   TYR A  44      14.936   9.825  13.497  1.00  3.57           O  \\nATOM    598  CB  TYR A  44      13.133  11.837  13.995  1.00  3.35           C  \\nATOM    599  CG  TYR A  44      11.969  12.752  14.283  1.00  3.59           C  \\nATOM    600  CD1 TYR A  44      12.059  13.805  15.217  1.00  2.83           C  \\nATOM    601  CD2 TYR A  44      10.739  12.534  13.671  1.00  2.97           C  \\nATOM    602  CE1 TYR A  44      10.974  14.618  15.484  1.00  3.79           C  \\nATOM    603  CE2 TYR A  44       9.634  13.344  13.940  1.00  3.59           C  \\nATOM    604  CZ  TYR A  44       9.743  14.374  14.873  1.00  4.28           C  \\nATOM    605  OH  TYR A  44       8.665  15.141  15.119  1.00  3.42           O  \\nATOM    606  H   TYR A  44      14.557  11.342  16.414  1.00  2.53           H  \\nATOM    607  HA  TYR A  44      12.366  10.278  15.066  1.00  1.14           H  \\nATOM    608 1HB  TYR A  44      13.975  12.353  14.021  1.00  1.00           H  \\nATOM    609 2HB  TYR A  44      13.056  11.460  13.073  1.00  1.87           H  \\nATOM    610  HD1 TYR A  44      12.921  13.967  15.665  1.00  1.96           H  \\nATOM    611  HD2 TYR A  44      10.637  11.796  13.023  1.00 34.95           H  \\nATOM    612  HE1 TYR A  44      11.064  15.351  16.144  1.00 17.91           H  \\nATOM    613  HE2 TYR A  44       8.766  13.182  13.502  1.00  2.99           H  \\nATOM    614  HH  TYR A  44       7.931  14.843  14.577  1.00  0.00           H  \\nATOM    615  N   ALA A  45      14.315   8.586  15.319  1.00  2.68           N  \\nATOM    616  CA  ALA A  45      15.390   7.610  15.230  1.00  2.63           C  \\nATOM    617  C   ALA A  45      15.141   6.463  14.236  1.00  2.97           C  \\nATOM    618  O   ALA A  45      16.082   5.694  13.983  1.00  3.46           O  \\nATOM    619  CB  ALA A  45      15.628   7.011  16.601  1.00  3.77           C  \\nATOM    620  H   ALA A  45      13.626   8.372  15.878  1.00  1.01           H  \\nATOM    621  HA  ALA A  45      16.222   8.060  14.964  1.00  1.22           H  \\nATOM    622 1HB  ALA A  45      14.814   6.570  16.937  1.00  1.00           H  \\nATOM    623 2HB  ALA A  45      16.349   6.339  16.572  1.00  1.05           H  \\nATOM    624 3HB  ALA A  45      15.882   7.710  17.248  1.00  7.30           H  \\nATOM    625  N   ASN A  46      13.954   6.416  13.695  1.00  3.26           N  \\nATOM    626  CA  ASN A  46      13.473   5.311  12.853  1.00  2.90           C  \\nATOM    627  C   ASN A  46      13.234   5.730  11.404  1.00  2.96           C  \\nATOM    628  O   ASN A  46      13.687   6.853  11.033  1.00  3.39           O  \\nATOM    629  OXT ASN A  46      12.614   4.965  10.671  1.00  3.85           O  \\nATOM    630  CB  ASN A  46      12.231   4.667  13.478  1.00  3.58           C  \\nATOM    631  CG  ASN A  46      12.521   4.154  14.875  1.00  3.07           C  \\nATOM    632  OD1 ASN A  46      11.776   4.485  15.792  1.00  7.12           O  \\nATOM    633  ND2 ASN A  46      13.505   3.283  14.993  1.00  5.43           N  \\nATOM    634  H   ASN A  46      13.323   7.061  13.882  1.00  2.79           H  \\nATOM    635  HA  ASN A  46      14.165   4.602  12.878  1.00  1.84           H  \\nATOM    636 1HB  ASN A  46      11.503   5.323  13.522  1.00  1.01           H  \\nATOM    637 2HB  ASN A  46      11.913   3.922  12.913  1.00  3.08           H  \\nATOM    638 1HD2 ASN A  46      14.024   3.051  14.273  1.00  5.31           H  \\nATOM    639 2HD2 ASN A  46      13.719   2.934  15.831  1.00 17.33           H  \\nTER                                                                             \\n\",\"pdb\");\n\tviewer_16234199980179648.setStyle({\"cartoon\": {\"color\": \"spectrum\"}, \"stick\": {\"colorscheme\": \"lightgreyCarbon\", \"radius\": 0.25}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 16.885193180887107, \"y\": 14.76286018676573, \"z\": 4.15949904921767}, \"end\": {\"x\": 16.949, \"y\": 16.528, \"z\": 4.919}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 15.012, \"y\": 15.628, \"z\": 5.997}, \"end\": {\"x\": 15.135, \"y\": 13.741, \"z\": 5.535}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.759, \"y\": 7.802, \"z\": 5.792}, \"end\": {\"x\": 13.364, \"y\": 9.36, \"z\": 4.756}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.784, \"y\": 8.179, \"z\": 7.761}, \"end\": {\"x\": 15.013, \"y\": 9.787, \"z\": 7.376}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.694, \"y\": 11.667, \"z\": 8.175}, \"end\": {\"x\": 11.839, \"y\": 13.391, \"z\": 7.388}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 10.253, \"y\": 12.576, \"z\": 9.072}, \"end\": {\"x\": 11.304, \"y\": 11.202, \"z\": 9.989}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.68, \"y\": 8.639, \"z\": 10.972}, \"end\": {\"x\": 13.687, \"y\": 6.853, \"z\": 11.033}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.323, \"y\": 7.061, \"z\": 13.882}, \"end\": {\"x\": 11.636, \"y\": 8.232, \"z\": 13.477}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.873, \"y\": 4.485, \"z\": 10.738}, \"end\": {\"x\": 8.772, \"y\": 4.843, \"z\": 12.923}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.339, \"y\": 2.296, \"z\": 9.28}, \"end\": {\"x\": 7.89, \"y\": 2.069, \"z\": 11.213}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.736, \"y\": 3.147, \"z\": 8.305}, \"end\": {\"x\": 4.676, \"y\": 3.227, \"z\": 10.212}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.614, \"y\": 5.362, \"z\": 6.858}, \"end\": {\"x\": 6.047, \"y\": 5.861, \"z\": 8.681}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.759, \"y\": 7.802, \"z\": 5.792}, \"end\": {\"x\": 13.364, \"y\": 9.36, \"z\": 4.756}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.784, \"y\": 8.179, \"z\": 7.761}, \"end\": {\"x\": 15.013, \"y\": 9.787, \"z\": 7.376}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.873, \"y\": 4.485, \"z\": 10.738}, \"end\": {\"x\": 8.772, \"y\": 4.843, \"z\": 12.923}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.355, \"y\": 4.26, \"z\": 4.708}, \"end\": {\"x\": 7.806, \"y\": 3.748, \"z\": 6.612}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.375, \"y\": 5.339, \"z\": 8.753}, \"end\": {\"x\": 12.614, \"y\": 4.965, \"z\": 10.671}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.64, \"y\": 7.201, \"z\": 8.979}, \"end\": {\"x\": 13.687, \"y\": 6.853, \"z\": 11.033}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.339, \"y\": 2.296, \"z\": 9.28}, \"end\": {\"x\": 7.89, \"y\": 2.069, \"z\": 11.213}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.866, \"y\": 3.04, \"z\": 3.42}, \"end\": {\"x\": 5.249, \"y\": 1.974, \"z\": 5.505}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.736, \"y\": 3.147, \"z\": 8.305}, \"end\": {\"x\": 4.676, \"y\": 3.227, \"z\": 10.212}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.511, \"y\": 4.955, \"z\": 2.595}, \"end\": {\"x\": 3.537, \"y\": 4.954, \"z\": 4.628}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.614, \"y\": 5.362, \"z\": 6.858}, \"end\": {\"x\": 6.047, \"y\": 5.861, \"z\": 8.681}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.263, \"y\": 5.942, \"z\": 0.716}, \"end\": {\"x\": 5.99, \"y\": 6.074, \"z\": 2.638}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.355, \"y\": 4.26, \"z\": 4.708}, \"end\": {\"x\": 7.806, \"y\": 3.748, \"z\": 6.612}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.866, \"y\": 3.04, \"z\": 3.42}, \"end\": {\"x\": 5.249, \"y\": 1.974, \"z\": 5.505}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.511, \"y\": 4.955, \"z\": 2.595}, \"end\": {\"x\": 3.537, \"y\": 4.954, \"z\": 4.628}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.263, \"y\": 5.942, \"z\": 0.716}, \"end\": {\"x\": 5.99, \"y\": 6.074, \"z\": 2.638}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.689, \"y\": 6.515, \"z\": -4.879}, \"end\": {\"x\": 5.943, \"y\": 6.032, \"z\": -3.507}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.77, \"y\": 10.464, \"z\": -2.632}, \"end\": {\"x\": 5.886, \"y\": 10.632, \"z\": -2.523}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.689, \"y\": 6.515, \"z\": -4.879}, \"end\": {\"x\": 5.943, \"y\": 6.032, \"z\": -3.507}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.77, \"y\": 10.464, \"z\": -2.632}, \"end\": {\"x\": 5.886, \"y\": 10.632, \"z\": -2.523}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.462, \"y\": 12.733, \"z\": 1.584}, \"end\": {\"x\": 5.495, \"y\": 12.762, \"z\": -0.268}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.77, \"y\": 13.2, \"z\": 4.004}, \"end\": {\"x\": 7.052, \"y\": 13.56, \"z\": 2.662}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.293, \"y\": 13.075, \"z\": 5.881}, \"end\": {\"x\": 2.311, \"y\": 13.417, \"z\": 3.66}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.462, \"y\": 12.733, \"z\": 1.584}, \"end\": {\"x\": 5.495, \"y\": 12.762, \"z\": -0.268}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.978, \"y\": 11.293, \"z\": 7.441}, \"end\": {\"x\": 4.174, \"y\": 11.111, \"z\": 5.587}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.77, \"y\": 13.2, \"z\": 4.004}, \"end\": {\"x\": 7.052, \"y\": 13.56, \"z\": 2.662}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.978, \"y\": 12.098, \"z\": 9.423}, \"end\": {\"x\": 4.466, \"y\": 13.361, \"z\": 8.057}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.293, \"y\": 13.075, \"z\": 5.881}, \"end\": {\"x\": 2.311, \"y\": 13.417, \"z\": 3.66}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.978, \"y\": 11.293, \"z\": 7.441}, \"end\": {\"x\": 4.174, \"y\": 11.111, \"z\": 5.587}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.999, \"y\": 11.21, \"z\": 9.337}, \"end\": {\"x\": 5.025, \"y\": 9.841, \"z\": 7.826}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.978, \"y\": 12.098, \"z\": 9.423}, \"end\": {\"x\": 4.466, \"y\": 13.361, \"z\": 8.057}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.999, \"y\": 11.21, \"z\": 9.337}, \"end\": {\"x\": 5.025, \"y\": 9.841, \"z\": 7.826}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.694, \"y\": 11.667, \"z\": 8.175}, \"end\": {\"x\": 11.839, \"y\": 13.391, \"z\": 7.388}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 10.253, \"y\": 12.576, \"z\": 9.072}, \"end\": {\"x\": 11.304, \"y\": 11.202, \"z\": 9.989}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 16.885193180887107, \"y\": 14.76286018676573, \"z\": 4.15949904921767}, \"end\": {\"x\": 16.949, \"y\": 16.528, \"z\": 4.919}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 15.012, \"y\": 15.628, \"z\": 5.997}, \"end\": {\"x\": 15.135, \"y\": 13.741, \"z\": 5.535}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 14.557, \"y\": 11.342, \"z\": 16.414}, \"end\": {\"x\": 16.58, \"y\": 11.235, \"z\": 16.009}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 14.557, \"y\": 11.342, \"z\": 16.414}, \"end\": {\"x\": 16.58, \"y\": 11.235, \"z\": 16.009}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.68, \"y\": 8.639, \"z\": 10.972}, \"end\": {\"x\": 13.687, \"y\": 6.853, \"z\": 11.033}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.323, \"y\": 7.061, \"z\": 13.882}, \"end\": {\"x\": 11.636, \"y\": 8.232, \"z\": 13.477}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.375, \"y\": 5.339, \"z\": 8.753}, \"end\": {\"x\": 12.614, \"y\": 4.965, \"z\": 10.671}});\n\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.64, \"y\": 7.201, \"z\": 8.979}, \"end\": {\"x\": 13.687, \"y\": 6.853, \"z\": 11.033}});\n\tviewer_16234199980179648.addCylinder({\"radius\": 0.25, \"color\": \"gold\", \"fromCap\": true, \"toCap\": true, \"start\": {\"x\": 15.776, \"y\": 12.403, \"z\": 9.909}, \"end\": {\"x\": 17.122, \"y\": 10.899, \"z\": 10.106}});\n\tviewer_16234199980179648.addCylinder({\"radius\": 0.25, \"color\": \"gold\", \"fromCap\": true, \"toCap\": true, \"start\": {\"x\": 9.807, \"y\": 7.893, \"z\": 8.888}, \"end\": {\"x\": 7.99, \"y\": 8.827, \"z\": 8.866}});\n\tviewer_16234199980179648.addCylinder({\"radius\": 0.25, \"color\": \"gold\", \"fromCap\": true, \"toCap\": true, \"start\": {\"x\": 2.485, \"y\": 8.959, \"z\": 1.654}, \"end\": {\"x\": 4.275, \"y\": 9.703, \"z\": 1.042}});\n\tviewer_16234199980179648.addCylinder({\"radius\": 0.25, \"color\": \"gold\", \"fromCap\": true, \"toCap\": true, \"start\": {\"x\": 4.275, \"y\": 9.703, \"z\": 1.042}, \"end\": {\"x\": 2.485, \"y\": 8.959, \"z\": 1.654}});\n\tviewer_16234199980179648.addCylinder({\"radius\": 0.25, \"color\": \"gold\", \"fromCap\": true, \"toCap\": true, \"start\": {\"x\": 7.99, \"y\": 8.827, \"z\": 8.866}, \"end\": {\"x\": 9.807, \"y\": 7.893, \"z\": 8.888}});\n\tviewer_16234199980179648.addCylinder({\"radius\": 0.25, \"color\": \"gold\", \"fromCap\": true, \"toCap\": true, \"start\": {\"x\": 17.122, \"y\": 10.899, \"z\": 10.106}, \"end\": {\"x\": 15.776, \"y\": 12.403, \"z\": 9.909}});\n\tviewer_16234199980179648.zoomTo();\nviewer_16234199980179648.render();\n});\n</script>",
            "text/html": [
              "<div id=\"3dmolviewer_16234199980179648\"  style=\"position: relative; width: 1200px; height: 800px\">\n",
              "        <p id=\"3dmolwarning_16234199980179648\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n",
              "        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n",
              "        </div>\n",
              "<script>\n",
              "\n",
              "var loadScriptAsync = function(uri){\n",
              "  return new Promise((resolve, reject) => {\n",
              "    var tag = document.createElement('script');\n",
              "    tag.src = uri;\n",
              "    tag.async = true;\n",
              "    tag.onload = () => {\n",
              "      resolve();\n",
              "    };\n",
              "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
              "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
              "});\n",
              "};\n",
              "\n",
              "if(typeof $3Dmolpromise === 'undefined') {\n",
              "$3Dmolpromise = null;\n",
              "  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n",
              "}\n",
              "\n",
              "var viewer_16234199980179648 = null;\n",
              "var warn = document.getElementById(\"3dmolwarning_16234199980179648\");\n",
              "if(warn) {\n",
              "    warn.parentNode.removeChild(warn);\n",
              "}\n",
              "$3Dmolpromise.then(function() {\n",
              "viewer_16234199980179648 = $3Dmol.createViewer($(\"#3dmolviewer_16234199980179648\"),{backgroundColor:\"white\"});\n",
              "viewer_16234199980179648.zoomTo();\n",
              "\tviewer_16234199980179648.addModels(\"HEADER                                            11-JUN-21   XXXX              \\nEXPDTA    THEORETICAL MODEL                                                     \\nREMARK 220                                                                      \\nREMARK 220 EXPERIMENTAL DETAILS                                                 \\nREMARK 220  EXPERIMENT TYPE                : THEORETICAL MODELLING              \\nREMARK 220  DATE OF DATA COLLECTION        : 11-JUN-21                          \\nREMARK 220                                                                      \\nREMARK 220 REMARK: MODEL GENERATED BY ROSETTA                                   \\nREMARK 220  VERSION 2020.50.post.dev+970.commits.3700df14560                    \\nSSBOND     CYS A    3    CYS A   40                                       2.03  \\nSSBOND     CYS A    4    CYS A   32                                       2.04  \\nSSBOND     CYS A   16    CYS A   26                                       2.03  \\nATOM      1  N   THR A   1      16.964  13.974   3.550  1.00  7.40           N  \\nATOM      2  CA  THR A   1      16.911  12.715   4.354  1.00  4.88           C  \\nATOM      3  C   THR A   1      15.617  12.680   5.067  1.00  3.24           C  \\nATOM      4  O   THR A   1      15.135  13.741   5.535  1.00  6.32           O  \\nATOM      5  CB  THR A   1      18.087  12.705   5.331  1.00  8.64           C  \\nATOM      6  OG1 THR A   1      19.254  12.847   4.529  1.00  7.85           O  \\nATOM      7  CG2 THR A   1      18.131  11.440   6.179  1.00  8.70           C  \\nATOM      8 1H   THR A   1      17.833  14.019   3.058  1.00  0.00           H  \\nATOM      9 2H   THR A   1      16.209  13.983   2.894  1.00  0.00           H  \\nATOM     10 3H   THR A   1      16.885  14.763   4.159  1.00  0.00           H  \\nATOM     11  HA  THR A   1      17.003  11.966   3.702  1.00 11.94           H  \\nATOM     12  HB  THR A   1      18.004  13.512   5.933  1.00 36.57           H  \\nATOM     13  HG1 THR A   1      19.001  12.910   3.605  1.00  0.00           H  \\nATOM     14 1HG2 THR A   1      17.274  11.326   6.700  1.00  4.78           H  \\nATOM     15 2HG2 THR A   1      18.199  10.654   5.571  1.00  5.04           H  \\nATOM     16 3HG2 THR A   1      18.888  11.445   6.792  1.00  8.71           H  \\nATOM     17  N   THR A   2      15.050  11.498   5.182  1.00  3.54           N  \\nATOM     18  CA  THR A   2      13.800  11.354   5.944  1.00  2.98           C  \\nATOM     19  C   THR A   2      14.118  10.664   7.274  1.00  2.92           C  \\nATOM     20  O   THR A   2      15.013   9.787   7.376  1.00  3.70           O  \\nATOM     21  CB  THR A   2      12.784  10.610   5.099  1.00  4.35           C  \\nATOM     22  OG1 THR A   2      13.364   9.360   4.756  1.00  5.60           O  \\nATOM     23  CG2 THR A   2      12.451  11.323   3.797  1.00  7.65           C  \\nATOM     24  H   THR A   2      15.500  10.728   5.006  1.00  1.74           H  \\nATOM     25  HA  THR A   2      13.437  12.241   6.128  1.00  4.09           H  \\nATOM     26  HB  THR A   2      11.947  10.457   5.635  1.00  1.11           H  \\nATOM     27  HG1 THR A   2      14.242   9.299   5.140  1.00  0.00           H  \\nATOM     28 1HG2 THR A   2      13.241  11.856   3.510  1.00  1.00           H  \\nATOM     29 2HG2 THR A   2      12.211  10.692   3.092  1.00  5.54           H  \\nATOM     30 3HG2 THR A   2      11.693  11.979   3.928  1.00  9.26           H  \\nATOM     31  N   CYS A   3      13.427  11.137   8.281  1.00  2.67           N  \\nATOM     32  CA  CYS A   3      13.573  10.632   9.683  1.00  3.33           C  \\nATOM     33  C   CYS A   3      12.203  10.369  10.192  1.00  2.48           C  \\nATOM     34  O   CYS A   3      11.304  11.202   9.989  1.00  3.56           O  \\nATOM     35  CB  CYS A   3      14.290  11.646  10.572  1.00  3.72           C  \\nATOM     36  SG  CYS A   3      15.776  12.403   9.909  1.00  3.50           S  \\nATOM     37  H   CYS A   3      12.694  11.667   8.175  1.00  2.61           H  \\nATOM     38  HA  CYS A   3      14.136   9.848   9.663  1.00  1.99           H  \\nATOM     39 1HB  CYS A   3      13.666  12.380  10.778  1.00  2.36           H  \\nATOM     40 2HB  CYS A   3      14.534  11.231  11.427  1.00  5.69           H  \\nATOM     41  N   CYS A   4      12.008   9.241  10.913  1.00  2.21           N  \\nATOM     42  CA  CYS A   4      10.680   8.906  11.348  1.00  2.79           C  \\nATOM     43  C   CYS A   4      10.653   8.718  12.890  1.00  1.95           C  \\nATOM     44  O   CYS A   4      11.636   8.232  13.477  1.00  2.71           O  \\nATOM     45  CB  CYS A   4      10.146   7.657  10.637  1.00  1.88           C  \\nATOM     46  SG  CYS A   4       9.807   7.893   8.888  1.00  2.64           S  \\nATOM     47  H   CYS A   4      12.680   8.639  10.972  1.00  3.79           H  \\nATOM     48  HA  CYS A   4      10.040   9.609  11.131  1.00  2.07           H  \\nATOM     49 1HB  CYS A   4      10.807   6.942  10.710  1.00  1.00           H  \\nATOM     50 2HB  CYS A   4       9.320   7.356  11.060  1.00  1.15           H  \\nATOM     51  N   PRO A   5       9.523   9.074  13.514  1.00  1.98           N  \\nATOM     52  CA  PRO A   5       9.452   9.028  14.967  1.00  2.21           C  \\nATOM     53  C   PRO A   5       9.332   7.648  15.601  1.00  3.11           C  \\nATOM     54  O   PRO A   5       9.619   7.492  16.798  1.00  3.37           O  \\nATOM     55  CB  PRO A   5       8.215   9.866  15.291  1.00  3.11           C  \\nATOM     56  CG  PRO A   5       7.324   9.692  14.078  1.00  2.85           C  \\nATOM     57  CD  PRO A   5       8.335   9.701  12.896  1.00  2.30           C  \\nATOM     58  HA  PRO A   5      10.250   9.418  15.361  1.00  1.18           H  \\nATOM     59 1HB  PRO A   5       7.788   9.653  16.052  1.00  1.00           H  \\nATOM     60 2HB  PRO A   5       8.454  10.786  15.422  1.00  9.12           H  \\nATOM     61 1HG  PRO A   5       6.940   8.954  14.085  1.00  1.00           H  \\nATOM     62 2HG  PRO A   5       6.694  10.413  13.945  1.00  2.23           H  \\nATOM     63 1HD  PRO A   5       7.970   9.189  12.161  1.00 21.74           H  \\nATOM     64 2HD  PRO A   5       8.555  10.587  12.559  1.00  1.09           H  \\nATOM     65  N   SER A   6       8.843   6.697  14.807  1.00  2.50           N  \\nATOM     66  CA  SER A   6       8.667   5.337  15.289  1.00  2.10           C  \\nATOM     67  C   SER A   6       8.826   4.406  14.080  1.00  2.51           C  \\nATOM     68  O   SER A   6       8.772   4.843  12.923  1.00  2.59           O  \\nATOM     69  CB  SER A   6       7.336   5.164  15.936  1.00  2.47           C  \\nATOM     70  OG  SER A   6       6.301   5.230  14.949  1.00  3.00           O  \\nATOM     71  H   SER A   6       8.533   6.896  13.977  1.00 17.98           H  \\nATOM     72  HA  SER A   6       9.345   5.117  15.952  1.00  1.96           H  \\nATOM     73 1HB  SER A   6       7.270   4.271  16.360  1.00  3.42           H  \\nATOM     74 2HB  SER A   6       7.164   5.850  16.638  1.00  1.18           H  \\nATOM     75  HG  SER A   6       6.745   5.372  14.109  1.00  0.00           H  \\nATOM     76  N   ILE A   7       8.861   3.112  14.380  1.00  2.83           N  \\nATOM     77  CA  ILE A   7       8.948   2.068  13.346  1.00  3.68           C  \\nATOM     78  C   ILE A   7       7.725   2.088  12.438  1.00  3.09           C  \\nATOM     79  O   ILE A   7       7.890   2.069  11.213  1.00  3.82           O  \\nATOM     80  CB  ILE A   7       9.139   0.695  13.989  1.00  6.17           C  \\nATOM     81  CG1 ILE A   7      10.455   0.525  14.728  1.00  6.83           C  \\nATOM     82  CG2 ILE A   7       9.072  -0.400  12.934  1.00  8.69           C  \\nATOM     83  CD1 ILE A   7      11.656   0.274  13.824  1.00  5.54           C  \\nATOM     84  H   ILE A   7       8.752   2.816  15.234  1.00  7.72           H  \\nATOM     85  HA  ILE A   7       9.748   2.237  12.802  1.00  1.69           H  \\nATOM     86  HB  ILE A   7       8.388   0.571  14.620  1.00  4.71           H  \\nATOM     87 1HG1 ILE A   7      10.612   1.356  15.237  1.00 12.19           H  \\nATOM     88 2HG1 ILE A   7      10.381  -0.243  15.348  1.00 24.95           H  \\nATOM     89 1HG2 ILE A   7       9.562  -0.104  12.124  1.00 14.52           H  \\nATOM     90 2HG2 ILE A   7       9.482  -1.221  13.309  1.00 14.28           H  \\nATOM     91 3HG2 ILE A   7       8.126  -0.576  12.705  1.00  1.48           H  \\nATOM     92 1HD1 ILE A   7      11.782   1.046  13.200  1.00  1.00           H  \\nATOM     93 2HD1 ILE A   7      12.457   0.142  14.402  1.00  8.13           H  \\nATOM     94 3HD1 ILE A   7      11.483  -0.567  13.323  1.00 24.48           H  \\nATOM     95  N   VAL A   8       6.557   2.085  13.039  1.00  3.25           N  \\nATOM     96  CA  VAL A   8       5.374   2.026  12.194  1.00  3.81           C  \\nATOM     97  C   VAL A   8       5.190   3.318  11.351  1.00  4.01           C  \\nATOM     98  O   VAL A   8       4.676   3.227  10.212  1.00  2.78           O  \\nATOM     99  CB  VAL A   8       4.063   1.695  12.923  1.00  3.19           C  \\nATOM    100  CG1 VAL A   8       3.361   2.862  13.554  1.00  5.80           C  \\nATOM    101  CG2 VAL A   8       3.102   1.021  11.942  1.00  5.67           C  \\nATOM    102  H   VAL A   8       6.474   1.972  13.932  1.00  2.36           H  \\nATOM    103  HA  VAL A   8       5.473   1.286  11.550  1.00 15.45           H  \\nATOM    104  HB  VAL A   8       4.315   1.048  13.631  1.00 27.87           H  \\nATOM    105 1HG1 VAL A   8       3.965   3.600  13.787  1.00 27.46           H  \\nATOM    106 2HG1 VAL A   8       2.666   3.237  12.964  1.00  1.00           H  \\nATOM    107 3HG1 VAL A   8       2.901   2.607  14.387  1.00  6.74           H  \\nATOM    108 1HG2 VAL A   8       3.216   1.388  11.044  1.00  1.28           H  \\nATOM    109 2HG2 VAL A   8       3.216   0.057  11.891  1.00 21.28           H  \\nATOM    110 3HG2 VAL A   8       2.177   1.203  12.180  1.00 22.95           H  \\nATOM    111  N   ALA A   9       5.683   4.414  11.850  1.00  3.23           N  \\nATOM    112  CA  ALA A   9       5.645   5.648  11.058  1.00  3.33           C  \\nATOM    113  C   ALA A   9       6.474   5.468   9.778  1.00  1.97           C  \\nATOM    114  O   ALA A   9       6.047   5.861   8.681  1.00  2.71           O  \\nATOM    115  CB  ALA A   9       6.063   6.882  11.859  1.00  3.85           C  \\nATOM    116  H   ALA A   9       6.199   4.406  12.607  1.00 22.78           H  \\nATOM    117  HA  ALA A   9       4.709   5.811  10.788  1.00  3.33           H  \\nATOM    118 1HB  ALA A   9       6.971   6.766  12.210  1.00  1.12           H  \\nATOM    119 2HB  ALA A   9       6.053   7.680  11.282  1.00  1.16           H  \\nATOM    120 3HB  ALA A   9       5.445   7.021  12.608  1.00  1.00           H  \\nATOM    121  N   ARG A  10       7.641   4.855   9.956  1.00  2.20           N  \\nATOM    122  CA  ARG A  10       8.491   4.521   8.779  1.00  1.73           C  \\nATOM    123  C   ARG A  10       7.794   3.557   7.840  1.00  2.06           C  \\nATOM    124  O   ARG A  10       7.806   3.748   6.612  1.00  2.32           O  \\nATOM    125  CB  ARG A  10       9.861   3.977   9.199  1.00  2.14           C  \\nATOM    126  CG  ARG A  10      10.749   3.510   8.050  1.00  2.44           C  \\nATOM    127  CD  ARG A  10      11.276   4.598   7.171  1.00  2.31           C  \\nATOM    128  NE  ARG A  10      12.142   5.532   7.882  1.00  2.71           N  \\nATOM    129  CZ  ARG A  10      12.638   6.655   7.370  1.00  3.08           C  \\nATOM    130  NH1 ARG A  10      12.387   7.019   6.119  1.00  3.91           N  \\nATOM    131  NH2 ARG A  10      13.420   7.420   8.126  1.00  3.38           N  \\nATOM    132  H   ARG A  10       7.873   4.485  10.738  1.00  7.53           H  \\nATOM    133  HA  ARG A  10       8.669   5.364   8.308  1.00  8.15           H  \\nATOM    134 1HB  ARG A  10      10.301   4.680   9.733  1.00  2.38           H  \\nATOM    135 2HB  ARG A  10       9.693   3.242   9.822  1.00  1.98           H  \\nATOM    136 1HG  ARG A  10      11.519   3.022   8.417  1.00  1.25           H  \\nATOM    137 2HG  ARG A  10      10.304   2.860   7.511  1.00  1.31           H  \\nATOM    138 1HD  ARG A  10      11.805   4.156   6.478  1.00  3.82           H  \\nATOM    139 2HD  ARG A  10      10.579   5.047   6.697  1.00  1.18           H  \\nATOM    140  HE  ARG A  10      12.375   5.339   8.753  1.00  1.00           H  \\nATOM    141 1HH1 ARG A  10      11.851   6.541   5.553  1.00 12.22           H  \\nATOM    142 2HH1 ARG A  10      12.759   7.802   5.792  1.00  1.26           H  \\nATOM    143 1HH2 ARG A  10      13.640   7.201   8.979  1.00  1.34           H  \\nATOM    144 2HH2 ARG A  10      13.784   8.179   7.761  1.00 17.63           H  \\nATOM    145  N   SER A  11       7.224   2.499   8.390  1.00  2.47           N  \\nATOM    146  CA  SER A  11       6.524   1.546   7.525  1.00  2.77           C  \\nATOM    147  C   SER A  11       5.418   2.215   6.717  1.00  2.00           C  \\nATOM    148  O   SER A  11       5.249   1.974   5.505  1.00  2.68           O  \\nATOM    149  CB  SER A  11       5.977   0.404   8.334  1.00  2.72           C  \\nATOM    150  OG  SER A  11       7.066  -0.247   9.003  1.00  3.23           O  \\nATOM    151  H   SER A  11       7.339   2.296   9.280  1.00 21.06           H  \\nATOM    152  HA  SER A  11       7.175   1.150   6.905  1.00  1.58           H  \\nATOM    153 1HB  SER A  11       5.369   0.745   9.033  1.00  2.23           H  \\nATOM    154 2HB  SER A  11       5.485  -0.247   7.774  1.00  1.56           H  \\nATOM    155  HG  SER A  11       7.858   0.232   8.747  1.00  0.00           H  \\nATOM    156  N   ASN A  12       4.655   3.072   7.407  1.00  2.39           N  \\nATOM    157  CA  ASN A  12       3.604   3.824   6.745  1.00  2.38           C  \\nATOM    158  C   ASN A  12       4.150   4.797   5.700  1.00  2.09           C  \\nATOM    159  O   ASN A  12       3.537   4.954   4.628  1.00  2.54           O  \\nATOM    160  CB  ASN A  12       2.768   4.573   7.765  1.00  3.19           C  \\nATOM    161  CG  ASN A  12       1.895   3.666   8.614  1.00  5.38           C  \\nATOM    162  OD1 ASN A  12       1.605   2.542   8.213  1.00  6.27           O  \\nATOM    163  ND2 ASN A  12       1.482   4.163   9.783  1.00  5.55           N  \\nATOM    164  H   ASN A  12       4.736   3.147   8.305  1.00  8.30           H  \\nATOM    165  HA  ASN A  12       2.959   3.230   6.331  1.00  2.71           H  \\nATOM    166 1HB  ASN A  12       3.353   5.091   8.370  1.00  1.00           H  \\nATOM    167 2HB  ASN A  12       2.186   5.242   7.314  1.00  1.00           H  \\nATOM    168 1HD2 ASN A  12       1.713   5.009  10.034  1.00  1.00           H  \\nATOM    169 2HD2 ASN A  12       0.940   3.649  10.331  1.00 11.50           H  \\nATOM    170  N   PHE A  13       5.248   5.433   6.024  1.00  2.52           N  \\nATOM    171  CA  PHE A  13       5.942   6.331   5.086  1.00  1.68           C  \\nATOM    172  C   PHE A  13       6.258   5.590   3.779  1.00  1.95           C  \\nATOM    173  O   PHE A  13       5.990   6.074   2.638  1.00  2.30           O  \\nATOM    174  CB  PHE A  13       7.170   6.951   5.716  1.00  2.27           C  \\nATOM    175  CG  PHE A  13       7.812   8.001   4.863  1.00  1.94           C  \\nATOM    176  CD1 PHE A  13       7.423   9.337   4.990  1.00  3.65           C  \\nATOM    177  CD2 PHE A  13       8.826   7.669   3.984  1.00  3.68           C  \\nATOM    178  CE1 PHE A  13       8.024  10.325   4.205  1.00  4.79           C  \\nATOM    179  CE2 PHE A  13       9.461   8.635   3.204  1.00  5.85           C  \\nATOM    180  CZ  PHE A  13       9.046   9.963   3.317  1.00  6.30           C  \\nATOM    181  H   PHE A  13       5.614   5.362   6.858  1.00  1.58           H  \\nATOM    182  HA  PHE A  13       5.390   7.102   4.897  1.00  1.00           H  \\nATOM    183 1HB  PHE A  13       6.910   7.314   6.586  1.00  6.31           H  \\nATOM    184 2HB  PHE A  13       7.815   6.217   5.906  1.00  2.30           H  \\nATOM    185  HD1 PHE A  13       6.699   9.562   5.599  1.00  8.04           H  \\nATOM    186  HD2 PHE A  13       9.111   6.770   3.938  1.00  2.35           H  \\nATOM    187  HE1 PHE A  13       7.767  11.213   4.252  1.00 22.70           H  \\nATOM    188  HE2 PHE A  13      10.160   8.394   2.609  1.00  1.33           H  \\nATOM    189  HZ  PHE A  13       9.474  10.632   2.775  1.00  7.02           H  \\nATOM    190  N   ASN A  14       6.888   4.403   3.944  1.00  2.75           N  \\nATOM    191  CA  ASN A  14       7.263   3.624   2.761  1.00  2.04           C  \\nATOM    192  C   ASN A  14       6.099   3.183   1.920  1.00  1.90           C  \\nATOM    193  O   ASN A  14       6.203   3.176   0.665  1.00  2.45           O  \\nATOM    194  CB  ASN A  14       8.154   2.450   3.201  1.00  2.97           C  \\nATOM    195  CG  ASN A  14       9.510   2.894   3.674  1.00  3.01           C  \\nATOM    196  OD1 ASN A  14      10.013   3.962   3.263  1.00  3.90           O  \\nATOM    197  ND2 ASN A  14      10.138   2.083   4.511  1.00  3.39           N  \\nATOM    198  H   ASN A  14       7.355   4.260   4.708  1.00  1.17           H  \\nATOM    199  HA  ASN A  14       7.900   4.143   2.227  1.00 30.34           H  \\nATOM    200 1HB  ASN A  14       7.698   1.961   3.927  1.00 23.66           H  \\nATOM    201 2HB  ASN A  14       8.246   1.808   2.454  1.00  1.18           H  \\nATOM    202 1HD2 ASN A  14       9.753   1.308   4.801  1.00  2.48           H  \\nATOM    203 2HD2 ASN A  14      10.989   2.306   4.831  1.00  7.08           H  \\nATOM    204  N   VAL A  15       4.956   2.890   2.535  1.00  1.68           N  \\nATOM    205  CA  VAL A  15       3.744   2.572   1.751  1.00  2.09           C  \\nATOM    206  C   VAL A  15       3.246   3.822   1.015  1.00  2.32           C  \\nATOM    207  O   VAL A  15       2.904   3.792  -0.174  1.00  2.08           O  \\nATOM    208  CB  VAL A  15       2.674   1.873   2.594  1.00  2.71           C  \\nATOM    209  CG1 VAL A  15       1.357   1.810   1.870  1.00  4.19           C  \\nATOM    210  CG2 VAL A  15       3.118   0.473   2.998  1.00  3.20           C  \\nATOM    211  H   VAL A  15       4.866   3.040   3.420  1.00  5.75           H  \\nATOM    212  HA  VAL A  15       3.994   1.902   1.075  1.00  3.09           H  \\nATOM    213  HB  VAL A  15       2.562   2.435   3.404  1.00  2.45           H  \\nATOM    214 1HG1 VAL A  15       1.419   2.162   0.956  1.00  7.84           H  \\nATOM    215 2HG1 VAL A  15       1.028   0.885   1.792  1.00  2.23           H  \\nATOM    216 3HG1 VAL A  15       0.646   2.315   2.326  1.00  2.26           H  \\nATOM    217 1HG2 VAL A  15       3.990   0.446   3.427  1.00  7.04           H  \\nATOM    218 2HG2 VAL A  15       2.475   0.062   3.605  1.00  1.00           H  \\nATOM    219 3HG2 VAL A  15       3.147  -0.120   2.221  1.00  1.52           H  \\nATOM    220  N   CYS A  16       3.172   4.941   1.753  1.00  2.12           N  \\nATOM    221  CA  CYS A  16       2.775   6.205   1.138  1.00  1.65           C  \\nATOM    222  C   CYS A  16       3.606   6.530  -0.126  1.00  2.23           C  \\nATOM    223  O   CYS A  16       3.056   7.047  -1.122  1.00  2.18           O  \\nATOM    224  CB  CYS A  16       2.834   7.281   2.220  1.00  3.44           C  \\nATOM    225  SG  CYS A  16       2.485   8.959   1.654  1.00  2.74           S  \\nATOM    226  H   CYS A  16       3.511   4.955   2.595  1.00 30.84           H  \\nATOM    227  HA  CYS A  16       1.836   6.163   0.874  1.00  1.34           H  \\nATOM    228 1HB  CYS A  16       2.177   7.080   2.916  1.00  1.04           H  \\nATOM    229 2HB  CYS A  16       3.720   7.302   2.635  1.00  1.00           H  \\nATOM    230  N   ARG A  17       4.874   6.209  -0.068  1.00  2.13           N  \\nATOM    231  CA  ARG A  17       5.779   6.511  -1.184  1.00  2.57           C  \\nATOM    232  C   ARG A  17       5.580   5.626  -2.405  1.00  1.88           C  \\nATOM    233  O   ARG A  17       5.943   6.032  -3.507  1.00  3.12           O  \\nATOM    234  CB  ARG A  17       7.237   6.427  -0.699  1.00  2.14           C  \\nATOM    235  CG  ARG A  17       7.626   7.598   0.194  1.00  2.71           C  \\nATOM    236  CD  ARG A  17       7.882   8.874  -0.572  1.00  2.03           C  \\nATOM    237  NE  ARG A  17       9.145   8.817  -1.260  1.00  2.43           N  \\nATOM    238  CZ  ARG A  17       9.512   9.558  -2.317  1.00  2.20           C  \\nATOM    239  NH1 ARG A  17       8.637  10.366  -2.892  1.00  3.42           N  \\nATOM    240  NH2 ARG A  17      10.761   9.510  -2.753  1.00  2.49           N  \\nATOM    241  H   ARG A  17       5.263   5.942   0.716  1.00 12.39           H  \\nATOM    242  HA  ARG A  17       5.646   7.450  -1.445  1.00  3.45           H  \\nATOM    243 1HB  ARG A  17       7.345   5.563  -0.250  1.00 15.45           H  \\nATOM    244 2HB  ARG A  17       7.798   6.398  -1.494  1.00  1.00           H  \\nATOM    245 1HG  ARG A  17       6.912   7.782   0.833  1.00  1.00           H  \\nATOM    246 2HG  ARG A  17       8.387   7.405   0.724  1.00 11.97           H  \\nATOM    247 1HD  ARG A  17       7.154   8.938  -1.218  1.00  8.63           H  \\nATOM    248 2HD  ARG A  17       7.784   9.636  -0.008  1.00  3.94           H  \\nATOM    249  HE  ARG A  17       9.836   8.316  -0.877  1.00  1.00           H  \\nATOM    250 1HH1 ARG A  17       7.770  10.464  -2.632  1.00  1.21           H  \\nATOM    251 2HH1 ARG A  17       8.917  10.885  -3.617  1.00  2.39           H  \\nATOM    252 1HH2 ARG A  17      11.383   8.965  -2.375  1.00  3.74           H  \\nATOM    253 2HH2 ARG A  17      10.997  10.026  -3.469  1.00 11.02           H  \\nATOM    254  N   LEU A  18       4.966   4.437  -2.194  1.00  2.15           N  \\nATOM    255  CA  LEU A  18       4.915   3.480  -3.298  1.00  2.36           C  \\nATOM    256  C   LEU A  18       4.267   4.041  -4.578  1.00  1.99           C  \\nATOM    257  O   LEU A  18       4.804   3.777  -5.697  1.00  2.44           O  \\nATOM    258  CB  LEU A  18       4.135   2.222  -2.884  1.00  2.44           C  \\nATOM    259  CG  LEU A  18       4.897   1.269  -1.951  1.00  2.99           C  \\nATOM    260  CD1 LEU A  18       3.911   0.200  -1.510  1.00  4.76           C  \\nATOM    261  CD2 LEU A  18       6.146   0.688  -2.568  1.00  3.90           C  \\nATOM    262  H   LEU A  18       4.847   4.150  -1.350  1.00  3.14           H  \\nATOM    263  HA  LEU A  18       5.834   3.214  -3.530  1.00  1.11           H  \\nATOM    264 1HB  LEU A  18       3.313   2.488  -2.403  1.00  1.00           H  \\nATOM    265 2HB  LEU A  18       3.894   1.710  -3.701  1.00 21.31           H  \\nATOM    266  HG  LEU A  18       5.181   1.783  -1.152  1.00  3.60           H  \\nATOM    267 1HD1 LEU A  18       3.135   0.640  -1.094  1.00  1.30           H  \\nATOM    268 2HD1 LEU A  18       3.629  -0.312  -2.303  1.00  1.27           H  \\nATOM    269 3HD1 LEU A  18       4.361  -0.391  -0.856  1.00  1.53           H  \\nATOM    270 1HD2 LEU A  18       5.940   0.327  -3.489  1.00 10.88           H  \\nATOM    271 2HD2 LEU A  18       6.851   1.385  -2.671  1.00  2.80           H  \\nATOM    272 3HD2 LEU A  18       6.518  -0.054  -2.033  1.00 22.90           H  \\nATOM    273  N   PRO A  19       3.191   4.774  -4.549  1.00  2.11           N  \\nATOM    274  CA  PRO A  19       2.632   5.337  -5.774  1.00  2.38           C  \\nATOM    275  C   PRO A  19       3.380   6.530  -6.329  1.00  1.98           C  \\nATOM    276  O   PRO A  19       3.036   7.025  -7.419  1.00  3.72           O  \\nATOM    277  CB  PRO A  19       1.214   5.747  -5.417  1.00  2.26           C  \\nATOM    278  CG  PRO A  19       1.231   5.945  -3.911  1.00  3.82           C  \\nATOM    279  CD  PRO A  19       2.217   4.874  -3.434  1.00  2.16           C  \\nATOM    280  HA  PRO A  19       2.649   4.676  -6.487  1.00  1.84           H  \\nATOM    281 1HB  PRO A  19       0.889   6.473  -5.838  1.00  2.18           H  \\nATOM    282 2HB  PRO A  19       0.582   5.061  -5.661  1.00  1.31           H  \\nATOM    283 1HG  PRO A  19       1.520   6.702  -3.717  1.00 19.37           H  \\nATOM    284 2HG  PRO A  19       0.361   5.796  -3.501  1.00  1.64           H  \\nATOM    285 1HD  PRO A  19       2.640   5.153  -2.600  1.00 17.92           H  \\nATOM    286 2HD  PRO A  19       1.833   3.989  -3.257  1.00 34.59           H  \\nATOM    287  N   GLY A  20       4.385   7.010  -5.587  1.00  2.12           N  \\nATOM    288  CA  GLY A  20       5.215   8.141  -5.998  1.00  2.37           C  \\nATOM    289  C   GLY A  20       4.881   9.460  -5.286  1.00  2.40           C  \\nATOM    290  O   GLY A  20       5.399  10.501  -5.709  1.00  3.44           O  \\nATOM    291  H   GLY A  20       4.689   6.515  -4.879  1.00  1.78           H  \\nATOM    292 1HA  GLY A  20       6.157   7.928  -5.814  1.00  1.87           H  \\nATOM    293 2HA  GLY A  20       5.149   8.261  -6.986  1.00  1.04           H  \\nATOM    294  N   THR A  21       4.025   9.398  -4.254  1.00  2.24           N  \\nATOM    295  CA  THR A  21       3.668  10.548  -3.433  1.00  2.10           C  \\nATOM    296  C   THR A  21       4.937  11.268  -2.988  1.00  1.65           C  \\nATOM    297  O   THR A  21       5.886  10.632  -2.523  1.00  2.81           O  \\nATOM    298  CB  THR A  21       2.929  10.063  -2.210  1.00  1.72           C  \\nATOM    299  OG1 THR A  21       1.921   9.122  -2.580  1.00  1.91           O  \\nATOM    300  CG2 THR A  21       2.250  11.223  -1.487  1.00  2.22           C  \\nATOM    301  H   THR A  21       3.597   8.608  -4.094  1.00  4.76           H  \\nATOM    302  HA  THR A  21       3.095  11.155  -3.926  1.00  1.47           H  \\nATOM    303  HB  THR A  21       3.575   9.611  -1.579  1.00  1.00           H  \\nATOM    304  HG1 THR A  21       1.932   9.001  -3.533  1.00  0.00           H  \\nATOM    305 1HG2 THR A  21       1.730  11.750  -2.149  1.00  1.02           H  \\nATOM    306 2HG2 THR A  21       1.653  10.901  -0.785  1.00  2.24           H  \\nATOM    307 3HG2 THR A  21       2.934  11.846  -1.086  1.00  1.00           H  \\nATOM    308  N   SER A  22       4.885  12.600  -3.031  1.00  2.62           N  \\nATOM    309  CA  SER A  22       5.988  13.420  -2.521  1.00  1.76           C  \\nATOM    310  C   SER A  22       6.355  13.048  -1.102  1.00  2.62           C  \\nATOM    311  O   SER A  22       5.495  12.762  -0.268  1.00  3.10           O  \\nATOM    312  CB  SER A  22       5.591  14.892  -2.580  1.00  2.04           C  \\nATOM    313  OG  SER A  22       6.644  15.731  -2.154  1.00  4.09           O  \\nATOM    314  H   SER A  22       4.097  13.030  -3.197  1.00  1.63           H  \\nATOM    315  HA  SER A  22       6.760  13.340  -3.107  1.00  1.66           H  \\nATOM    316 1HB  SER A  22       5.378  15.113  -3.521  1.00 10.38           H  \\nATOM    317 2HB  SER A  22       4.774  15.043  -2.033  1.00 16.82           H  \\nATOM    318  HG  SER A  22       7.372  15.147  -1.926  1.00  0.00           H  \\nATOM    319  N   GLU A  23       7.655  13.108  -0.829  1.00  2.23           N  \\nATOM    320  CA  GLU A  23       8.085  12.974   0.585  1.00  2.20           C  \\nATOM    321  C   GLU A  23       7.375  13.950   1.526  1.00  2.73           C  \\nATOM    322  O   GLU A  23       7.052  13.560   2.662  1.00  2.78           O  \\nATOM    323  CB  GLU A  23       9.591  13.115   0.753  1.00  2.94           C  \\nATOM    324  CG  GLU A  23      10.326  11.958   0.104  1.00  3.57           C  \\nATOM    325  CD  GLU A  23      11.788  11.947   0.391  1.00  5.57           C  \\nATOM    326  OE1 GLU A  23      12.410  12.936   0.725  1.00  6.10           O  \\nATOM    327  OE2 GLU A  23      12.444  10.818   0.157  1.00  7.69           O  \\nATOM    328  H   GLU A  23       8.264  13.424  -1.411  1.00  1.00           H  \\nATOM    329  HA  GLU A  23       7.825  12.056   0.867  1.00  2.75           H  \\nATOM    330 1HB  GLU A  23       9.898  13.956   0.326  1.00  1.83           H  \\nATOM    331 2HB  GLU A  23       9.790  13.168   1.704  1.00  5.20           H  \\nATOM    332 1HG  GLU A  23       9.967  11.080   0.350  1.00  1.36           H  \\nATOM    333 2HG  GLU A  23      10.219  12.029  -0.876  1.00 16.53           H  \\nATOM    334  N   ALA A  24       7.148  15.182   1.059  1.00  2.21           N  \\nATOM    335  CA  ALA A  24       6.537  16.162   1.966  1.00  2.07           C  \\nATOM    336  C   ALA A  24       5.137  15.776   2.367  1.00  2.28           C  \\nATOM    337  O   ALA A  24       4.735  15.971   3.532  1.00  2.52           O  \\nATOM    338  CB  ALA A  24       6.579  17.557   1.315  1.00  3.18           C  \\nATOM    339  H   ALA A  24       7.609  15.473   0.337  1.00  1.36           H  \\nATOM    340  HA  ALA A  24       7.104  16.239   2.768  1.00  2.14           H  \\nATOM    341 1HB  ALA A  24       7.503  17.798   1.109  1.00  1.76           H  \\nATOM    342 2HB  ALA A  24       6.051  17.549   0.491  1.00  1.58           H  \\nATOM    343 3HB  ALA A  24       6.192  18.215   1.933  1.00  3.08           H  \\nATOM    344  N   ILE A  25       4.329  15.256   1.435  1.00  2.04           N  \\nATOM    345  CA  ILE A  25       3.005  14.798   1.771  1.00  2.09           C  \\nATOM    346  C   ILE A  25       3.062  13.560   2.694  1.00  2.09           C  \\nATOM    347  O   ILE A  25       2.311  13.417   3.660  1.00  2.42           O  \\nATOM    348  CB  ILE A  25       2.177  14.515   0.507  1.00  3.02           C  \\nATOM    349  CG1 ILE A  25       1.901  15.860  -0.176  1.00  3.40           C  \\nATOM    350  CG2 ILE A  25       0.920  13.713   0.808  1.00  2.72           C  \\nATOM    351  CD1 ILE A  25       1.370  15.821  -1.605  1.00  3.13           C  \\nATOM    352  H   ILE A  25       4.711  14.990   0.617  1.00  8.00           H  \\nATOM    353  HA  ILE A  25       2.514  15.498   2.260  1.00  1.84           H  \\nATOM    354  HB  ILE A  25       2.759  13.978  -0.085  1.00  1.14           H  \\nATOM    355 1HG1 ILE A  25       1.235  16.320   0.391  1.00  1.24           H  \\nATOM    356 2HG1 ILE A  25       2.747  16.375  -0.186  1.00  1.00           H  \\nATOM    357 1HG2 ILE A  25       0.488  13.470  -0.050  1.00  1.00           H  \\nATOM    358 2HG2 ILE A  25       0.310  14.252   1.372  1.00  1.00           H  \\nATOM    359 3HG2 ILE A  25       1.169  12.886   1.298  1.00  6.18           H  \\nATOM    360 1HD1 ILE A  25       0.627  15.155  -1.675  1.00  7.94           H  \\nATOM    361 2HD1 ILE A  25       1.063  16.736  -1.848  1.00 25.31           H  \\nATOM    362 3HD1 ILE A  25       2.130  15.571  -2.197  1.00 16.42           H  \\nATOM    363  N   CYS A  26       4.013  12.678   2.354  1.00  2.38           N  \\nATOM    364  CA  CYS A  26       4.083  11.486   3.180  1.00  2.10           C  \\nATOM    365  C   CYS A  26       4.529  11.821   4.621  1.00  1.46           C  \\nATOM    366  O   CYS A  26       4.174  11.111   5.587  1.00  2.51           O  \\nATOM    367  CB  CYS A  26       4.981  10.374   2.571  1.00  2.25           C  \\nATOM    368  SG  CYS A  26       4.275   9.703   1.042  1.00  2.45           S  \\nATOM    369  H   CYS A  26       4.462  12.733   1.584  1.00  1.89           H  \\nATOM    370  HA  CYS A  26       3.205  11.059   3.236  1.00  2.97           H  \\nATOM    371 1HB  CYS A  26       5.836  10.756   2.341  1.00  1.46           H  \\nATOM    372 2HB  CYS A  26       5.078   9.648   3.192  1.00  1.67           H  \\nATOM    373  N   ALA A  27       5.362  12.851   4.739  1.00  2.20           N  \\nATOM    374  CA  ALA A  27       5.812  13.342   6.051  1.00  2.56           C  \\nATOM    375  C   ALA A  27       4.649  13.816   6.909  1.00  1.87           C  \\nATOM    376  O   ALA A  27       4.466  13.361   8.057  1.00  2.86           O  \\nATOM    377  CB  ALA A  27       6.883  14.439   5.887  1.00  2.10           C  \\nATOM    378  H   ALA A  27       5.770  13.200   4.004  1.00  1.04           H  \\nATOM    379  HA  ALA A  27       6.271  12.606   6.515  1.00  1.47           H  \\nATOM    380 1HB  ALA A  27       6.531  15.159   5.330  1.00  1.47           H  \\nATOM    381 2HB  ALA A  27       7.124  14.794   6.768  1.00  1.55           H  \\nATOM    382 3HB  ALA A  27       7.676  14.054   5.458  1.00  1.00           H  \\nATOM    383  N   THR A  28       3.820  14.730   6.379  1.00  2.39           N  \\nATOM    384  CA  THR A  28       2.751  15.196   7.191  1.00  2.24           C  \\nATOM    385  C   THR A  28       1.685  14.093   7.443  1.00  2.35           C  \\nATOM    386  O   THR A  28       0.965  14.087   8.454  1.00  3.22           O  \\nATOM    387  CB  THR A  28       2.037  16.445   6.621  1.00  2.62           C  \\nATOM    388  OG1 THR A  28       1.697  16.153   5.271  1.00  2.56           O  \\nATOM    389  CG2 THR A  28       2.928  17.660   6.674  1.00  3.71           C  \\nATOM    390  H   THR A  28       3.960  15.014   5.530  1.00  2.03           H  \\nATOM    391  HA  THR A  28       3.070  15.477   8.082  1.00  2.02           H  \\nATOM    392  HB  THR A  28       1.200  16.584   7.156  1.00  1.00           H  \\nATOM    393  HG1 THR A  28       1.991  15.265   5.052  1.00  0.00           H  \\nATOM    394 1HG2 THR A  28       3.371  17.763   7.580  1.00  1.14           H  \\nATOM    395 2HG2 THR A  28       3.678  17.552   6.020  1.00  1.13           H  \\nATOM    396 3HG2 THR A  28       2.447  18.486   6.464  1.00 13.35           H  \\nATOM    397  N   TYR A  29       1.652  13.136   6.523  1.00  2.97           N  \\nATOM    398  CA  TYR A  29       0.695  12.058   6.670  1.00  2.59           C  \\nATOM    399  C   TYR A  29       1.071  11.120   7.832  1.00  2.85           C  \\nATOM    400  O   TYR A  29       0.170  10.575   8.488  1.00  3.15           O  \\nATOM    401  CB  TYR A  29       0.568  11.279   5.338  1.00  3.41           C  \\nATOM    402  CG  TYR A  29      -0.535  10.247   5.310  1.00  3.14           C  \\nATOM    403  CD1 TYR A  29      -1.811  10.543   5.777  1.00  3.50           C  \\nATOM    404  CD2 TYR A  29      -0.271   8.941   4.895  1.00  6.74           C  \\nATOM    405  CE1 TYR A  29      -2.832   9.604   5.690  1.00  3.76           C  \\nATOM    406  CE2 TYR A  29      -1.266   7.971   4.862  1.00  8.08           C  \\nATOM    407  CZ  TYR A  29      -2.548   8.313   5.238  1.00  6.33           C  \\nATOM    408  OH  TYR A  29      -3.511   7.313   5.238  1.00  9.80           O  \\nATOM    409  H   TYR A  29       2.293  13.075   5.881  1.00  1.04           H  \\nATOM    410  HA  TYR A  29      -0.198  12.415   6.845  1.00  1.50           H  \\nATOM    411 1HB  TYR A  29       0.405  11.951   4.631  1.00  5.15           H  \\nATOM    412 2HB  TYR A  29       1.448  10.860   5.142  1.00  1.10           H  \\nATOM    413  HD1 TYR A  29      -2.042  11.466   6.048  1.00  8.36           H  \\nATOM    414  HD2 TYR A  29       0.648   8.702   4.596  1.00  1.00           H  \\nATOM    415  HE1 TYR A  29      -3.746   9.823   6.001  1.00  1.24           H  \\nATOM    416  HE2 TYR A  29      -1.073   7.056   4.551  1.00 18.09           H  \\nATOM    417  HH  TYR A  29      -3.112   6.486   4.959  1.00  0.00           H  \\nATOM    418  N   THR A  30       2.379  10.893   7.997  1.00  2.88           N  \\nATOM    419  CA  THR A  30       2.870   9.886   8.916  1.00  3.09           C  \\nATOM    420  C   THR A  30       3.519  10.362  10.215  1.00  4.08           C  \\nATOM    421  O   THR A  30       3.687   9.566  11.146  1.00  4.42           O  \\nATOM    422  CB  THR A  30       3.932   8.990   8.199  1.00  3.09           C  \\nATOM    423  OG1 THR A  30       5.025   9.841   7.826  1.00  3.41           O  \\nATOM    424  CG2 THR A  30       3.343   8.274   7.009  1.00  4.83           C  \\nATOM    425  H   THR A  30       2.978  11.293   7.441  1.00  1.44           H  \\nATOM    426  HA  THR A  30       2.146   9.260   9.144  1.00  1.00           H  \\nATOM    427  HB  THR A  30       4.258   8.337   8.891  1.00  1.14           H  \\nATOM    428  HG1 THR A  30       4.839  10.741   8.105  1.00  0.00           H  \\nATOM    429 1HG2 THR A  30       2.905   8.936   6.403  1.00  1.00           H  \\nATOM    430 2HG2 THR A  30       4.020   7.783   6.499  1.00  1.04           H  \\nATOM    431 3HG2 THR A  30       2.614   7.626   7.284  1.00 13.17           H  \\nATOM    432  N   GLY A  31       3.930  11.629  10.199  1.00  3.24           N  \\nATOM    433  CA  GLY A  31       4.730  12.160  11.322  1.00  3.98           C  \\nATOM    434  C   GLY A  31       6.206  12.086  11.070  1.00  2.94           C  \\nATOM    435  O   GLY A  31       6.967  12.580  11.911  1.00  2.85           O  \\nATOM    436  H   GLY A  31       3.978  12.098   9.423  1.00 14.25           H  \\nATOM    437 1HA  GLY A  31       4.454  13.092  11.475  1.00 27.75           H  \\nATOM    438 2HA  GLY A  31       4.465  11.665  12.153  1.00  9.02           H  \\nATOM    439  N   CYS A  32       6.607  11.472   9.956  1.00  2.65           N  \\nATOM    440  CA  CYS A  32       8.008  11.553   9.581  1.00  2.49           C  \\nATOM    441  C   CYS A  32       8.345  13.003   9.174  1.00  2.45           C  \\nATOM    442  O   CYS A  32       7.446  13.847   9.004  1.00  3.38           O  \\nATOM    443  CB  CYS A  32       8.340  10.554   8.464  1.00  2.49           C  \\nATOM    444  SG  CYS A  32       7.990   8.827   8.866  1.00  2.48           S  \\nATOM    445  H   CYS A  32       5.999  11.210   9.337  1.00  8.40           H  \\nATOM    446  HA  CYS A  32       8.581  11.301  10.324  1.00 29.65           H  \\nATOM    447 1HB  CYS A  32       7.818  10.769   7.672  1.00  1.37           H  \\nATOM    448 2HB  CYS A  32       9.288  10.604   8.239  1.00  2.63           H  \\nATOM    449  N   ILE A  33       9.637  13.252   9.040  1.00  3.37           N  \\nATOM    450  CA  ILE A  33      10.148  14.569   8.687  1.00  2.86           C  \\nATOM    451  C   ILE A  33      11.209  14.452   7.608  1.00  3.09           C  \\nATOM    452  O   ILE A  33      11.839  13.391   7.388  1.00  3.22           O  \\nATOM    453  CB  ILE A  33      10.742  15.300   9.921  1.00  3.08           C  \\nATOM    454  CG1 ILE A  33      11.967  14.602  10.463  1.00  3.57           C  \\nATOM    455  CG2 ILE A  33       9.643  15.432  10.953  1.00  4.22           C  \\nATOM    456  CD1 ILE A  33      12.646  15.383  11.593  1.00  4.24           C  \\nATOM    457  H   ILE A  33      10.253  12.576   9.072  1.00  5.52           H  \\nATOM    458  HA  ILE A  33       9.418  15.138   8.376  1.00 14.18           H  \\nATOM    459  HB  ILE A  33      10.983  16.201   9.590  1.00 38.65           H  \\nATOM    460 1HG1 ILE A  33      11.684  13.729  10.820  1.00 19.10           H  \\nATOM    461 2HG1 ILE A  33      12.634  14.483   9.746  1.00  1.06           H  \\nATOM    462 1HG2 ILE A  33       8.844  15.851  10.538  1.00  2.10           H  \\nATOM    463 2HG2 ILE A  33       9.394  14.537  11.302  1.00  2.11           H  \\nATOM    464 3HG2 ILE A  33       9.967  16.013  11.691  1.00  4.03           H  \\nATOM    465 1HD1 ILE A  33      12.743  16.320  11.280  1.00 34.82           H  \\nATOM    466 2HD1 ILE A  33      12.088  15.331  12.419  1.00  1.05           H  \\nATOM    467 3HD1 ILE A  33      13.549  14.992  11.729  1.00  1.31           H  \\nATOM    468  N   ILE A  34      11.426  15.582   6.911  1.00  3.27           N  \\nATOM    469  CA  ILE A  34      12.486  15.737   5.910  1.00  3.42           C  \\nATOM    470  C   ILE A  34      13.470  16.738   6.437  1.00  2.87           C  \\nATOM    471  O   ILE A  34      13.033  17.834   6.853  1.00  4.13           O  \\nATOM    472  CB  ILE A  34      11.966  16.250   4.571  1.00  4.16           C  \\nATOM    473  CG1 ILE A  34      10.864  15.373   3.994  1.00  4.74           C  \\nATOM    474  CG2 ILE A  34      13.097  16.354   3.565  1.00  6.10           C  \\nATOM    475  CD1 ILE A  34      10.036  16.027   2.890  1.00  5.45           C  \\nATOM    476  H   ILE A  34      10.829  16.269   6.963  1.00 16.22           H  \\nATOM    477  HA  ILE A  34      12.909  14.874   5.749  1.00  1.00           H  \\nATOM    478  HB  ILE A  34      11.607  17.158   4.736  1.00 12.04           H  \\nATOM    479 1HG1 ILE A  34      10.259  15.143   4.740  1.00  1.49           H  \\nATOM    480 2HG1 ILE A  34      11.267  14.553   3.619  1.00 10.87           H  \\nATOM    481 1HG2 ILE A  34      13.601  15.499   3.541  1.00  1.00           H  \\nATOM    482 2HG2 ILE A  34      12.709  16.552   2.673  1.00 19.94           H  \\nATOM    483 3HG2 ILE A  34      13.707  17.089   3.826  1.00  2.47           H  \\nATOM    484 1HD1 ILE A  34      10.637  16.377   2.174  1.00  1.82           H  \\nATOM    485 2HD1 ILE A  34       9.402  15.345   2.536  1.00  9.03           H  \\nATOM    486 3HD1 ILE A  34       9.512  16.760   3.311  1.00 19.11           H  \\nATOM    487  N   ILE A  35      14.749  16.368   6.445  1.00  2.75           N  \\nATOM    488  CA  ILE A  35      15.789  17.309   6.862  1.00  2.90           C  \\nATOM    489  C   ILE A  35      16.776  17.474   5.716  1.00  3.20           C  \\nATOM    490  O   ILE A  35      16.949  16.528   4.919  1.00  4.31           O  \\nATOM    491  CB  ILE A  35      16.508  16.857   8.149  1.00  4.73           C  \\nATOM    492  CG1 ILE A  35      17.203  15.509   7.996  1.00  4.32           C  \\nATOM    493  CG2 ILE A  35      15.525  16.833   9.308  1.00  5.81           C  \\nATOM    494  CD1 ILE A  35      18.138  15.189   9.165  1.00  6.90           C  \\nATOM    495  H   ILE A  35      15.012  15.628   5.997  1.00  3.21           H  \\nATOM    496  HA  ILE A  35      15.381  18.173   7.068  1.00  2.09           H  \\nATOM    497  HB  ILE A  35      17.191  17.550   8.330  1.00  1.53           H  \\nATOM    498 1HG1 ILE A  35      16.501  14.824   7.958  1.00 14.79           H  \\nATOM    499 2HG1 ILE A  35      17.741  15.514   7.174  1.00  1.00           H  \\nATOM    500 1HG2 ILE A  35      15.061  17.707   9.364  1.00  3.70           H  \\nATOM    501 2HG2 ILE A  35      14.855  16.116   9.157  1.00 21.37           H  \\nATOM    502 3HG2 ILE A  35      16.027  16.670  10.146  1.00  5.66           H  \\nATOM    503 1HD1 ILE A  35      17.785  15.603  10.001  1.00  6.46           H  \\nATOM    504 2HD1 ILE A  35      18.205  14.200   9.242  1.00  4.53           H  \\nATOM    505 3HD1 ILE A  35      19.032  15.554   8.930  1.00 16.60           H  \\nATOM    506  N   PRO A  36      17.500  18.590   5.692  1.00  4.28           N  \\nATOM    507  CA  PRO A  36      18.486  18.750   4.634  1.00  3.76           C  \\nATOM    508  C   PRO A  36      19.749  17.969   4.860  1.00  4.79           C  \\nATOM    509  O   PRO A  36      20.493  17.725   3.899  1.00  6.10           O  \\nATOM    510  CB  PRO A  36      18.674  20.267   4.581  1.00  7.70           C  \\nATOM    511  CG  PRO A  36      18.258  20.827   5.923  1.00  6.81           C  \\nATOM    512  CD  PRO A  36      17.153  19.872   6.422  1.00  4.73           C  \\nATOM    513  HA  PRO A  36      18.146  18.411   3.790  1.00  7.56           H  \\nATOM    514 1HB  PRO A  36      19.503  20.541   4.350  1.00  9.51           H  \\nATOM    515 2HB  PRO A  36      18.132  20.641   3.872  1.00  3.67           H  \\nATOM    516 1HG  PRO A  36      18.894  20.806   6.466  1.00  4.56           H  \\nATOM    517 2HG  PRO A  36      17.889  21.725   5.870  1.00  9.72           H  \\nATOM    518 1HD  PRO A  36      17.212  19.754   7.374  1.00  1.00           H  \\nATOM    519 2HD  PRO A  36      16.244  20.115   6.193  1.00  9.27           H  \\nATOM    520  N   GLY A  37      20.048  17.720   6.144  1.00  5.98           N  \\nATOM    521  CA  GLY A  37      21.260  16.950   6.470  1.00  7.34           C  \\nATOM    522  C   GLY A  37      20.950  15.447   6.407  1.00  5.86           C  \\nATOM    523  O   GLY A  37      19.910  15.051   5.860  1.00  6.60           O  \\nATOM    524  H   GLY A  37      19.445  17.850   6.793  1.00  1.75           H  \\nATOM    525 1HA  GLY A  37      21.955  17.153   5.821  1.00  1.00           H  \\nATOM    526 2HA  GLY A  37      21.591  17.207   7.362  1.00  2.90           H  \\nATOM    527  N   ALA A  38      21.921  14.699   6.948  1.00  4.09           N  \\nATOM    528  CA  ALA A  38      21.796  13.225   6.966  1.00  4.89           C  \\nATOM    529  C   ALA A  38      21.760  12.594   8.339  1.00  5.34           C  \\nATOM    530  O   ALA A  38      21.731  11.356   8.416  1.00  7.20           O  \\nATOM    531  CB  ALA A  38      22.990  12.671   6.193  1.00  6.50           C  \\nATOM    532  H   ALA A  38      22.766  15.014   7.023  1.00  7.54           H  \\nATOM    533  HA  ALA A  38      20.997  12.980   6.460  1.00 26.11           H  \\nATOM    534 1HB  ALA A  38      23.103  13.161   5.348  1.00  4.37           H  \\nATOM    535 2HB  ALA A  38      23.813  12.768   6.720  1.00  1.21           H  \\nATOM    536 3HB  ALA A  38      22.858  11.717   5.994  1.00  6.32           H  \\nATOM    537  N   THR A  39      21.731  13.436   9.377  1.00  4.59           N  \\nATOM    538  CA  THR A  39      21.844  12.956  10.763  1.00  4.87           C  \\nATOM    539  C   THR A  39      20.537  13.181  11.479  1.00  4.29           C  \\nATOM    540  O   THR A  39      20.228  14.281  11.997  1.00  7.12           O  \\nATOM    541  CB  THR A  39      23.065  13.641  11.462  1.00  5.05           C  \\nATOM    542  OG1 THR A  39      24.184  13.459  10.655  1.00  5.24           O  \\nATOM    543  CG2 THR A  39      23.335  13.039  12.827  1.00  8.58           C  \\nATOM    544  H   THR A  39      21.871  14.322   9.243  1.00 43.17           H  \\nATOM    545  HA  THR A  39      22.069  12.007  10.755  1.00  1.15           H  \\nATOM    546  HB  THR A  39      22.826  14.615  11.549  1.00 28.76           H  \\nATOM    547  HG1 THR A  39      23.936  12.961   9.872  1.00  0.00           H  \\nATOM    548 1HG2 THR A  39      22.520  13.099  13.423  1.00  6.58           H  \\nATOM    549 2HG2 THR A  39      23.529  12.066  12.719  1.00  5.71           H  \\nATOM    550 3HG2 THR A  39      24.091  13.467  13.273  1.00  6.17           H  \\nATOM    551  N   CYS A  40      19.716  12.124  11.531  1.00  5.13           N  \\nATOM    552  CA  CYS A  40      18.443  12.197  12.212  1.00  3.45           C  \\nATOM    553  C   CYS A  40      18.632  12.283  13.732  1.00  3.56           C  \\nATOM    554  O   CYS A  40      19.477  11.575  14.277  1.00  5.42           O  \\nATOM    555  CB  CYS A  40      17.565  10.987  11.858  1.00  3.80           C  \\nATOM    556  SG  CYS A  40      17.122  10.899  10.106  1.00  3.85           S  \\nATOM    557  H   CYS A  40      19.959  11.330  11.159  1.00 16.91           H  \\nATOM    558  HA  CYS A  40      17.930  12.969  11.914  1.00 30.53           H  \\nATOM    559 1HB  CYS A  40      18.043  10.169  12.069  1.00  2.47           H  \\nATOM    560 2HB  CYS A  40      16.732  11.014  12.359  1.00 15.26           H  \\nATOM    561  N   PRO A  41      17.806  13.078  14.390  1.00  4.30           N  \\nATOM    562  CA  PRO A  41      17.904  13.208  15.859  1.00  5.14           C  \\nATOM    563  C   PRO A  41      17.335  12.004  16.593  1.00  4.23           C  \\nATOM    564  O   PRO A  41      16.580  11.235  16.009  1.00  3.67           O  \\nATOM    565  CB  PRO A  41      17.165  14.509  16.118  1.00  6.36           C  \\nATOM    566  CG  PRO A  41      16.041  14.509  15.075  1.00  6.31           C  \\nATOM    567  CD  PRO A  41      16.711  13.937  13.818  1.00  4.81           C  \\nATOM    568  HA  PRO A  41      18.844  13.274  16.105  1.00  1.94           H  \\nATOM    569 1HB  PRO A  41      16.831  14.630  16.940  1.00  8.14           H  \\nATOM    570 2HB  PRO A  41      17.734  15.269  15.990  1.00  5.84           H  \\nATOM    571 1HG  PRO A  41      15.427  14.006  15.323  1.00  4.02           H  \\nATOM    572 2HG  PRO A  41      15.712  15.400  14.892  1.00 37.89           H  \\nATOM    573 1HD  PRO A  41      16.082  13.407  13.307  1.00  2.11           H  \\nATOM    574 2HD  PRO A  41      17.110  14.583  13.210  1.00 10.45           H  \\nATOM    575  N   GLY A  42      17.621  11.960  17.887  1.00  4.53           N  \\nATOM    576  CA  GLY A  42      17.243  10.798  18.724  1.00  4.51           C  \\nATOM    577  C   GLY A  42      15.754  10.567  18.845  1.00  4.66           C  \\nATOM    578  O   GLY A  42      15.313   9.427  19.147  1.00  5.28           O  \\nATOM    579  H   GLY A  42      18.241  12.514  18.260  1.00 17.19           H  \\nATOM    580 1HA  GLY A  42      17.664  10.001  18.327  1.00  1.42           H  \\nATOM    581 2HA  GLY A  42      17.673  10.912  19.619  1.00  1.00           H  \\nATOM    582  N   ASP A  43      14.965  11.636  18.653  1.00  4.28           N  \\nATOM    583  CA  ASP A  43      13.516  11.493  18.710  1.00  3.50           C  \\nATOM    584  C   ASP A  43      12.833  11.234  17.331  1.00  2.35           C  \\nATOM    585  O   ASP A  43      11.608  11.086  17.287  1.00  3.20           O  \\nATOM    586  CB  ASP A  43      12.831  12.689  19.444  1.00  3.86           C  \\nATOM    587  CG  ASP A  43      13.113  14.044  18.872  1.00  3.26           C  \\nATOM    588  OD1 ASP A  43      14.026  14.181  18.117  1.00  4.70           O  \\nATOM    589  OD2 ASP A  43      12.260  14.976  19.236  1.00  5.33           O  \\nATOM    590  H   ASP A  43      15.316  12.387  18.296  1.00  2.13           H  \\nATOM    591  HA  ASP A  43      13.280  10.739  19.286  1.00  2.21           H  \\nATOM    592 1HB  ASP A  43      11.861  12.536  19.487  1.00  1.02           H  \\nATOM    593 2HB  ASP A  43      13.136  12.666  20.371  1.00 10.83           H  \\nATOM    594  N   TYR A  44      13.678  11.149  16.321  1.00  2.34           N  \\nATOM    595  CA  TYR A  44      13.245  10.686  14.994  1.00  2.19           C  \\nATOM    596  C   TYR A  44      14.250   9.639  14.507  1.00  3.17           C  \\nATOM    597  O   TYR A  44      14.936   9.825  13.497  1.00  3.57           O  \\nATOM    598  CB  TYR A  44      13.133  11.837  13.995  1.00  3.35           C  \\nATOM    599  CG  TYR A  44      11.969  12.752  14.283  1.00  3.59           C  \\nATOM    600  CD1 TYR A  44      12.059  13.805  15.217  1.00  2.83           C  \\nATOM    601  CD2 TYR A  44      10.739  12.534  13.671  1.00  2.97           C  \\nATOM    602  CE1 TYR A  44      10.974  14.618  15.484  1.00  3.79           C  \\nATOM    603  CE2 TYR A  44       9.634  13.344  13.940  1.00  3.59           C  \\nATOM    604  CZ  TYR A  44       9.743  14.374  14.873  1.00  4.28           C  \\nATOM    605  OH  TYR A  44       8.665  15.141  15.119  1.00  3.42           O  \\nATOM    606  H   TYR A  44      14.557  11.342  16.414  1.00  2.53           H  \\nATOM    607  HA  TYR A  44      12.366  10.278  15.066  1.00  1.14           H  \\nATOM    608 1HB  TYR A  44      13.975  12.353  14.021  1.00  1.00           H  \\nATOM    609 2HB  TYR A  44      13.056  11.460  13.073  1.00  1.87           H  \\nATOM    610  HD1 TYR A  44      12.921  13.967  15.665  1.00  1.96           H  \\nATOM    611  HD2 TYR A  44      10.637  11.796  13.023  1.00 34.95           H  \\nATOM    612  HE1 TYR A  44      11.064  15.351  16.144  1.00 17.91           H  \\nATOM    613  HE2 TYR A  44       8.766  13.182  13.502  1.00  2.99           H  \\nATOM    614  HH  TYR A  44       7.931  14.843  14.577  1.00  0.00           H  \\nATOM    615  N   ALA A  45      14.315   8.586  15.319  1.00  2.68           N  \\nATOM    616  CA  ALA A  45      15.390   7.610  15.230  1.00  2.63           C  \\nATOM    617  C   ALA A  45      15.141   6.463  14.236  1.00  2.97           C  \\nATOM    618  O   ALA A  45      16.082   5.694  13.983  1.00  3.46           O  \\nATOM    619  CB  ALA A  45      15.628   7.011  16.601  1.00  3.77           C  \\nATOM    620  H   ALA A  45      13.626   8.372  15.878  1.00  1.01           H  \\nATOM    621  HA  ALA A  45      16.222   8.060  14.964  1.00  1.22           H  \\nATOM    622 1HB  ALA A  45      14.814   6.570  16.937  1.00  1.00           H  \\nATOM    623 2HB  ALA A  45      16.349   6.339  16.572  1.00  1.05           H  \\nATOM    624 3HB  ALA A  45      15.882   7.710  17.248  1.00  7.30           H  \\nATOM    625  N   ASN A  46      13.954   6.416  13.695  1.00  3.26           N  \\nATOM    626  CA  ASN A  46      13.473   5.311  12.853  1.00  2.90           C  \\nATOM    627  C   ASN A  46      13.234   5.730  11.404  1.00  2.96           C  \\nATOM    628  O   ASN A  46      13.687   6.853  11.033  1.00  3.39           O  \\nATOM    629  OXT ASN A  46      12.614   4.965  10.671  1.00  3.85           O  \\nATOM    630  CB  ASN A  46      12.231   4.667  13.478  1.00  3.58           C  \\nATOM    631  CG  ASN A  46      12.521   4.154  14.875  1.00  3.07           C  \\nATOM    632  OD1 ASN A  46      11.776   4.485  15.792  1.00  7.12           O  \\nATOM    633  ND2 ASN A  46      13.505   3.283  14.993  1.00  5.43           N  \\nATOM    634  H   ASN A  46      13.323   7.061  13.882  1.00  2.79           H  \\nATOM    635  HA  ASN A  46      14.165   4.602  12.878  1.00  1.84           H  \\nATOM    636 1HB  ASN A  46      11.503   5.323  13.522  1.00  1.01           H  \\nATOM    637 2HB  ASN A  46      11.913   3.922  12.913  1.00  3.08           H  \\nATOM    638 1HD2 ASN A  46      14.024   3.051  14.273  1.00  5.31           H  \\nATOM    639 2HD2 ASN A  46      13.719   2.934  15.831  1.00 17.33           H  \\nTER                                                                             \\n\",\"pdb\");\n",
              "\tviewer_16234199980179648.setStyle({\"cartoon\": {\"color\": \"spectrum\"}, \"stick\": {\"colorscheme\": \"lightgreyCarbon\", \"radius\": 0.25}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 16.885193180887107, \"y\": 14.76286018676573, \"z\": 4.15949904921767}, \"end\": {\"x\": 16.949, \"y\": 16.528, \"z\": 4.919}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 15.012, \"y\": 15.628, \"z\": 5.997}, \"end\": {\"x\": 15.135, \"y\": 13.741, \"z\": 5.535}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.759, \"y\": 7.802, \"z\": 5.792}, \"end\": {\"x\": 13.364, \"y\": 9.36, \"z\": 4.756}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.784, \"y\": 8.179, \"z\": 7.761}, \"end\": {\"x\": 15.013, \"y\": 9.787, \"z\": 7.376}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.694, \"y\": 11.667, \"z\": 8.175}, \"end\": {\"x\": 11.839, \"y\": 13.391, \"z\": 7.388}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 10.253, \"y\": 12.576, \"z\": 9.072}, \"end\": {\"x\": 11.304, \"y\": 11.202, \"z\": 9.989}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.68, \"y\": 8.639, \"z\": 10.972}, \"end\": {\"x\": 13.687, \"y\": 6.853, \"z\": 11.033}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.323, \"y\": 7.061, \"z\": 13.882}, \"end\": {\"x\": 11.636, \"y\": 8.232, \"z\": 13.477}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.873, \"y\": 4.485, \"z\": 10.738}, \"end\": {\"x\": 8.772, \"y\": 4.843, \"z\": 12.923}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.339, \"y\": 2.296, \"z\": 9.28}, \"end\": {\"x\": 7.89, \"y\": 2.069, \"z\": 11.213}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.736, \"y\": 3.147, \"z\": 8.305}, \"end\": {\"x\": 4.676, \"y\": 3.227, \"z\": 10.212}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.614, \"y\": 5.362, \"z\": 6.858}, \"end\": {\"x\": 6.047, \"y\": 5.861, \"z\": 8.681}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.759, \"y\": 7.802, \"z\": 5.792}, \"end\": {\"x\": 13.364, \"y\": 9.36, \"z\": 4.756}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.784, \"y\": 8.179, \"z\": 7.761}, \"end\": {\"x\": 15.013, \"y\": 9.787, \"z\": 7.376}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.873, \"y\": 4.485, \"z\": 10.738}, \"end\": {\"x\": 8.772, \"y\": 4.843, \"z\": 12.923}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.355, \"y\": 4.26, \"z\": 4.708}, \"end\": {\"x\": 7.806, \"y\": 3.748, \"z\": 6.612}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.375, \"y\": 5.339, \"z\": 8.753}, \"end\": {\"x\": 12.614, \"y\": 4.965, \"z\": 10.671}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.64, \"y\": 7.201, \"z\": 8.979}, \"end\": {\"x\": 13.687, \"y\": 6.853, \"z\": 11.033}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.339, \"y\": 2.296, \"z\": 9.28}, \"end\": {\"x\": 7.89, \"y\": 2.069, \"z\": 11.213}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.866, \"y\": 3.04, \"z\": 3.42}, \"end\": {\"x\": 5.249, \"y\": 1.974, \"z\": 5.505}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.736, \"y\": 3.147, \"z\": 8.305}, \"end\": {\"x\": 4.676, \"y\": 3.227, \"z\": 10.212}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.511, \"y\": 4.955, \"z\": 2.595}, \"end\": {\"x\": 3.537, \"y\": 4.954, \"z\": 4.628}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.614, \"y\": 5.362, \"z\": 6.858}, \"end\": {\"x\": 6.047, \"y\": 5.861, \"z\": 8.681}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.263, \"y\": 5.942, \"z\": 0.716}, \"end\": {\"x\": 5.99, \"y\": 6.074, \"z\": 2.638}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.355, \"y\": 4.26, \"z\": 4.708}, \"end\": {\"x\": 7.806, \"y\": 3.748, \"z\": 6.612}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.866, \"y\": 3.04, \"z\": 3.42}, \"end\": {\"x\": 5.249, \"y\": 1.974, \"z\": 5.505}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.511, \"y\": 4.955, \"z\": 2.595}, \"end\": {\"x\": 3.537, \"y\": 4.954, \"z\": 4.628}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.263, \"y\": 5.942, \"z\": 0.716}, \"end\": {\"x\": 5.99, \"y\": 6.074, \"z\": 2.638}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.689, \"y\": 6.515, \"z\": -4.879}, \"end\": {\"x\": 5.943, \"y\": 6.032, \"z\": -3.507}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.77, \"y\": 10.464, \"z\": -2.632}, \"end\": {\"x\": 5.886, \"y\": 10.632, \"z\": -2.523}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.689, \"y\": 6.515, \"z\": -4.879}, \"end\": {\"x\": 5.943, \"y\": 6.032, \"z\": -3.507}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.77, \"y\": 10.464, \"z\": -2.632}, \"end\": {\"x\": 5.886, \"y\": 10.632, \"z\": -2.523}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.462, \"y\": 12.733, \"z\": 1.584}, \"end\": {\"x\": 5.495, \"y\": 12.762, \"z\": -0.268}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.77, \"y\": 13.2, \"z\": 4.004}, \"end\": {\"x\": 7.052, \"y\": 13.56, \"z\": 2.662}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.293, \"y\": 13.075, \"z\": 5.881}, \"end\": {\"x\": 2.311, \"y\": 13.417, \"z\": 3.66}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.462, \"y\": 12.733, \"z\": 1.584}, \"end\": {\"x\": 5.495, \"y\": 12.762, \"z\": -0.268}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.978, \"y\": 11.293, \"z\": 7.441}, \"end\": {\"x\": 4.174, \"y\": 11.111, \"z\": 5.587}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.77, \"y\": 13.2, \"z\": 4.004}, \"end\": {\"x\": 7.052, \"y\": 13.56, \"z\": 2.662}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.978, \"y\": 12.098, \"z\": 9.423}, \"end\": {\"x\": 4.466, \"y\": 13.361, \"z\": 8.057}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.293, \"y\": 13.075, \"z\": 5.881}, \"end\": {\"x\": 2.311, \"y\": 13.417, \"z\": 3.66}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.978, \"y\": 11.293, \"z\": 7.441}, \"end\": {\"x\": 4.174, \"y\": 11.111, \"z\": 5.587}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.999, \"y\": 11.21, \"z\": 9.337}, \"end\": {\"x\": 5.025, \"y\": 9.841, \"z\": 7.826}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.978, \"y\": 12.098, \"z\": 9.423}, \"end\": {\"x\": 4.466, \"y\": 13.361, \"z\": 8.057}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.999, \"y\": 11.21, \"z\": 9.337}, \"end\": {\"x\": 5.025, \"y\": 9.841, \"z\": 7.826}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.694, \"y\": 11.667, \"z\": 8.175}, \"end\": {\"x\": 11.839, \"y\": 13.391, \"z\": 7.388}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 10.253, \"y\": 12.576, \"z\": 9.072}, \"end\": {\"x\": 11.304, \"y\": 11.202, \"z\": 9.989}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 16.885193180887107, \"y\": 14.76286018676573, \"z\": 4.15949904921767}, \"end\": {\"x\": 16.949, \"y\": 16.528, \"z\": 4.919}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 15.012, \"y\": 15.628, \"z\": 5.997}, \"end\": {\"x\": 15.135, \"y\": 13.741, \"z\": 5.535}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 14.557, \"y\": 11.342, \"z\": 16.414}, \"end\": {\"x\": 16.58, \"y\": 11.235, \"z\": 16.009}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 14.557, \"y\": 11.342, \"z\": 16.414}, \"end\": {\"x\": 16.58, \"y\": 11.235, \"z\": 16.009}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.68, \"y\": 8.639, \"z\": 10.972}, \"end\": {\"x\": 13.687, \"y\": 6.853, \"z\": 11.033}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.323, \"y\": 7.061, \"z\": 13.882}, \"end\": {\"x\": 11.636, \"y\": 8.232, \"z\": 13.477}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.375, \"y\": 5.339, \"z\": 8.753}, \"end\": {\"x\": 12.614, \"y\": 4.965, \"z\": 10.671}});\n",
              "\tviewer_16234199980179648.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.64, \"y\": 7.201, \"z\": 8.979}, \"end\": {\"x\": 13.687, \"y\": 6.853, \"z\": 11.033}});\n",
              "\tviewer_16234199980179648.addCylinder({\"radius\": 0.25, \"color\": \"gold\", \"fromCap\": true, \"toCap\": true, \"start\": {\"x\": 15.776, \"y\": 12.403, \"z\": 9.909}, \"end\": {\"x\": 17.122, \"y\": 10.899, \"z\": 10.106}});\n",
              "\tviewer_16234199980179648.addCylinder({\"radius\": 0.25, \"color\": \"gold\", \"fromCap\": true, \"toCap\": true, \"start\": {\"x\": 9.807, \"y\": 7.893, \"z\": 8.888}, \"end\": {\"x\": 7.99, \"y\": 8.827, \"z\": 8.866}});\n",
              "\tviewer_16234199980179648.addCylinder({\"radius\": 0.25, \"color\": \"gold\", \"fromCap\": true, \"toCap\": true, \"start\": {\"x\": 2.485, \"y\": 8.959, \"z\": 1.654}, \"end\": {\"x\": 4.275, \"y\": 9.703, \"z\": 1.042}});\n",
              "\tviewer_16234199980179648.addCylinder({\"radius\": 0.25, \"color\": \"gold\", \"fromCap\": true, \"toCap\": true, \"start\": {\"x\": 4.275, \"y\": 9.703, \"z\": 1.042}, \"end\": {\"x\": 2.485, \"y\": 8.959, \"z\": 1.654}});\n",
              "\tviewer_16234199980179648.addCylinder({\"radius\": 0.25, \"color\": \"gold\", \"fromCap\": true, \"toCap\": true, \"start\": {\"x\": 7.99, \"y\": 8.827, \"z\": 8.866}, \"end\": {\"x\": 9.807, \"y\": 7.893, \"z\": 8.888}});\n",
              "\tviewer_16234199980179648.addCylinder({\"radius\": 0.25, \"color\": \"gold\", \"fromCap\": true, \"toCap\": true, \"start\": {\"x\": 17.122, \"y\": 10.899, \"z\": 10.106}, \"end\": {\"x\": 15.776, \"y\": 12.403, \"z\": 9.909}});\n",
              "\tviewer_16234199980179648.zoomTo();\n",
              "viewer_16234199980179648.render();\n",
              "});\n",
              "</script>"
            ]
          },
          "metadata": {
            "tags": []
          }
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "-PH2voFdmdlB"
      },
      "source": [
        "## Design strategy\n",
        "\n",
        "Design away the cysteine residues (i.e. disulfide bonds) using ResidueSelectors and TaskOperations, allowing all side-chains to re-pack and all backbone and side-chain torsions to minimize using the `FastDesign` mover."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "lcXdOvLCmdlD"
      },
      "source": [
        " Note: because we did not include the `-detect_disulfide 0` command line flag in `pyrosetta.init()`, in order to mutate disulfide bonded cysteine residues using packing steps in the `FastDesign` mover, first we need to break the disulfide bonds:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "2VDHfU8LmdlD"
      },
      "source": [
        "for i in pyrosetta.rosetta.core.select.get_residues_from_subset(disulfide_res.apply(pose)):\n",
        "    for j in pyrosetta.rosetta.core.select.get_residues_from_subset(disulfide_res.apply(pose)):\n",
        "        if pyrosetta.rosetta.core.conformation.is_disulfide_bond(pose.conformation(), i, j):\n",
        "            pyrosetta.rosetta.core.conformation.break_disulfide(pose.conformation(), i, j)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "XWPckPT8mdlD"
      },
      "source": [
        " Now we can declare a `ResidueSelector` for cysteine residues:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "JkzCua09mdlD",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "900a4596-494a-4e31-8af1-c3b1e6136fac"
      },
      "source": [
        "cys_res = pyrosetta.rosetta.core.select.residue_selector.ResidueNameSelector(\"CYS\")\n",
        "print(pyrosetta.rosetta.core.select.get_residues_from_subset(cys_res.apply(pose)))"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "vector1_unsigned_long[3, 4, 16, 26, 32, 40]\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "lY5ixybOmdlE"
      },
      "source": [
        " Note: the pose consists of only chain \"A\""
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "J8XKaa5SmdlE",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "60c46bf5-9c98-49cf-f20b-55465660e7b5"
      },
      "source": [
        "print(pose.pdb_info())"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "PDB file name: 1AB1.clean.pdb\n",
            " Pose Range  Chain    PDB Range  |   #Residues         #Atoms\n",
            "\n",
            "0001 -- 0046    A 0001  -- 0046  |   0046 residues;    00649 atoms\n",
            "                           TOTAL |   0046 residues;    00649 atoms\n",
            "\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "yo_IqW9gmdlE"
      },
      "source": [
        " Although the pose contains only chain A, if the pose contained other chains we could make the selection to only select chain A and cysteine residues using an `AndResidueSelector`. We will use this ResidueSelector with `RestrictAbsentCanonicalAASRLT` (restrict absent canonical amino acids residue level task operation)."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "2dWrfwVjmdlE",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "5c3569a3-bdb0-40ee-a2d2-d811d7ac7460"
      },
      "source": [
        "chain_A = pyrosetta.rosetta.core.select.residue_selector.ChainSelector(\"A\")\n",
        "print(pyrosetta.rosetta.core.select.get_residues_from_subset(chain_A.apply(pose)))\n",
        "chain_A_cys_res = pyrosetta.rosetta.core.select.residue_selector.AndResidueSelector(selector1=cys_res, selector2=chain_A)\n",
        "print(pyrosetta.rosetta.core.select.get_residues_from_subset(chain_A_cys_res.apply(pose)))"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "vector1_unsigned_long[1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46]\n",
            "vector1_unsigned_long[3, 4, 16, 26, 32, 40]\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "5JaDBFnnmdlE"
      },
      "source": [
        " Specify a `NotResidueSelector` selecting everything except the `chain_A_cys_res` ResidueSelector to use with a `RestrictToRepackingRLT` (restrict to repacking residue level task operation):"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "U2HdTHF_mdlE",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "16ad89c0-ca79-4b68-d44b-aa0df51a8ea1"
      },
      "source": [
        "not_chain_A_cys_res = pyrosetta.rosetta.core.select.residue_selector.NotResidueSelector(chain_A_cys_res)\n",
        "print(pyrosetta.rosetta.core.select.get_residues_from_subset(not_chain_A_cys_res.apply(pose)))"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "vector1_unsigned_long[1, 2, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 17, 18, 19, 20, 21, 22, 23, 24, 25, 27, 28, 29, 30, 31, 33, 34, 35, 36, 37, 38, 39, 41, 42, 43, 44, 45, 46]\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "RpiVLPcqmdlF"
      },
      "source": [
        " Quickly view the `not_chain_A_cys_res` ResidueSelector:"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "f1PCLhvXmdlF"
      },
      "source": [
        " Now we can set up the TaskOperations for `FastDesign`:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "rAO22lT9mdlF",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "55bd1fcf-b70c-45d9-93de-f8339dd283d0"
      },
      "source": [
        "# The task factory accepts all the task operations\n",
        "tf = pyrosetta.rosetta.core.pack.task.TaskFactory()\n",
        "\n",
        "# These are pretty standard\n",
        "tf.push_back(pyrosetta.rosetta.core.pack.task.operation.InitializeFromCommandline())\n",
        "tf.push_back(pyrosetta.rosetta.core.pack.task.operation.IncludeCurrent())\n",
        "tf.push_back(pyrosetta.rosetta.core.pack.task.operation.NoRepackDisulfides())\n",
        "\n",
        "\n",
        "# Disable design (i.e. repack only) on not_chain_A_cys_res\n",
        "tf.push_back(pyrosetta.rosetta.core.pack.task.operation.OperateOnResidueSubset(\n",
        "    pyrosetta.rosetta.core.pack.task.operation.RestrictToRepackingRLT(), not_chain_A_cys_res))\n",
        "\n",
        "# Enable design on chain_A_cys_res\n",
        "aa_to_design = pyrosetta.rosetta.core.pack.task.operation.RestrictAbsentCanonicalAASRLT()\n",
        "aa_to_design.aas_to_keep(\"ADEFGHIKLMNPQRSTVWY\")\n",
        "tf.push_back(pyrosetta.rosetta.core.pack.task.operation.OperateOnResidueSubset(\n",
        "    aa_to_design, chain_A_cys_res))\n",
        "\n",
        "# Convert the task factory into a PackerTask\n",
        "packer_task = tf.create_task_and_apply_taskoperations(pose)\n",
        "# View the PackerTask\n",
        "print(packer_task)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:core.pack.task: Packer task: initialize from command line()\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "stream",
          "text": [
            "#Packer_Task\n",
            "\n",
            "Threads to request: ALL AVAILABLE\n",
            "\n",
            "resid\tpack?\tdesign?\tallowed_aas\n",
            "1\tTRUE\tFALSE\tTHR:NtermProteinFull\n",
            "2\tTRUE\tFALSE\tTHR\n",
            "3\tTRUE\tTRUE\tALA,ASP,GLU,PHE,GLY,HIS,HIS_D,ILE,LYS,LEU,MET,ASN,PRO,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "4\tTRUE\tTRUE\tALA,ASP,GLU,PHE,GLY,HIS,HIS_D,ILE,LYS,LEU,MET,ASN,PRO,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "5\tTRUE\tFALSE\tPRO\n",
            "6\tTRUE\tFALSE\tSER\n",
            "7\tTRUE\tFALSE\tILE\n",
            "8\tTRUE\tFALSE\tVAL\n",
            "9\tTRUE\tFALSE\tALA\n",
            "10\tTRUE\tFALSE\tARG\n",
            "11\tTRUE\tFALSE\tSER\n",
            "12\tTRUE\tFALSE\tASN\n",
            "13\tTRUE\tFALSE\tPHE\n",
            "14\tTRUE\tFALSE\tASN\n",
            "15\tTRUE\tFALSE\tVAL\n",
            "16\tTRUE\tTRUE\tALA,ASP,GLU,PHE,GLY,HIS,HIS_D,ILE,LYS,LEU,MET,ASN,PRO,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "17\tTRUE\tFALSE\tARG\n",
            "18\tTRUE\tFALSE\tLEU\n",
            "19\tTRUE\tFALSE\tPRO\n",
            "20\tTRUE\tFALSE\tGLY\n",
            "21\tTRUE\tFALSE\tTHR\n",
            "22\tTRUE\tFALSE\tSER\n",
            "23\tTRUE\tFALSE\tGLU\n",
            "24\tTRUE\tFALSE\tALA\n",
            "25\tTRUE\tFALSE\tILE\n",
            "26\tTRUE\tTRUE\tALA,ASP,GLU,PHE,GLY,HIS,HIS_D,ILE,LYS,LEU,MET,ASN,PRO,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "27\tTRUE\tFALSE\tALA\n",
            "28\tTRUE\tFALSE\tTHR\n",
            "29\tTRUE\tFALSE\tTYR\n",
            "30\tTRUE\tFALSE\tTHR\n",
            "31\tTRUE\tFALSE\tGLY\n",
            "32\tTRUE\tTRUE\tALA,ASP,GLU,PHE,GLY,HIS,HIS_D,ILE,LYS,LEU,MET,ASN,PRO,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "33\tTRUE\tFALSE\tILE\n",
            "34\tTRUE\tFALSE\tILE\n",
            "35\tTRUE\tFALSE\tILE\n",
            "36\tTRUE\tFALSE\tPRO\n",
            "37\tTRUE\tFALSE\tGLY\n",
            "38\tTRUE\tFALSE\tALA\n",
            "39\tTRUE\tFALSE\tTHR\n",
            "40\tTRUE\tTRUE\tALA,ASP,GLU,PHE,GLY,HIS,HIS_D,ILE,LYS,LEU,MET,ASN,PRO,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "41\tTRUE\tFALSE\tPRO\n",
            "42\tTRUE\tFALSE\tGLY\n",
            "43\tTRUE\tFALSE\tASP\n",
            "44\tTRUE\tFALSE\tTYR\n",
            "45\tTRUE\tFALSE\tALA\n",
            "46\tTRUE\tFALSE\tASN:CtermProteinFull\n",
            "\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "LNHOpMXZmdlF"
      },
      "source": [
        " This time we will set up a `MoveMap` to establish which torsions are free to minimize"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "uOiwd0DsmdlF"
      },
      "source": [
        "mm = pyrosetta.rosetta.core.kinematics.MoveMap()\n",
        "mm.set_bb(True)\n",
        "mm.set_chi(True)\n",
        "mm.set_jump(True)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "N_G7ve-OmdlF"
      },
      "source": [
        "Set up `FastDesign`"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "Te4z_0BomdlF",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "73e5ed27-7778-4ee2-9182-04b140cca96b"
      },
      "source": [
        "rel_design = pyrosetta.rosetta.protocols.relax.FastRelax(scorefxn_in=scorefxn, standard_repeats=1, script_file=\"MonomerDesign2019\")\n",
        "rel_design.cartesian(True)\n",
        "rel_design.set_task_factory(tf)\n",
        "rel_design.set_movemap(mm)\n",
        "rel_design.minimize_bond_angles(True)\n",
        "rel_design.minimize_bond_lengths(True)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:protocols.relax.RelaxScriptManager: Looking for MonomerDesign2019.txt\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: ================== Reading script file: /usr/local/lib/python3.7/dist-packages/pyrosetta/database/sampling/relax_scripts/MonomerDesign2019.txt ==================\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: repeat %%nrepeats%%\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: coord_cst_weight 1.0\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: scale:fa_rep 0.059\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: repack\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: scale:fa_rep 0.092\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: min 0.01\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: coord_cst_weight 0.5\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: scale:fa_rep 0.280\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: repack\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: scale:fa_rep 0.323\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: min 0.01\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: coord_cst_weight 0.0\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: scale:fa_rep 0.568\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: repack\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: scale:fa_rep 0.633\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: min 0.01\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: coord_cst_weight 0.0\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: scale:fa_rep 1\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: repack\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: min 0.00001\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: accept_to_best\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: endrepeat\n"
          ],
          "name": "stderr"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "ZCs8uwcMmdlG"
      },
      "source": [
        " Run the `FastDesign` mover. Note: this takes ~39.6 s"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "nbgrader": {
          "grade": true,
          "grade_id": "cell-0276fa644308dd5b",
          "locked": false,
          "points": 0,
          "schema_version": 3,
          "solution": true
        },
        "id": "8jaoYq6SmdlG",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "4ba8fdf8-843f-4ec6-c7db-cf22e1088a4d"
      },
      "source": [
        "rel_design.apply(pose)\n",
        "pose.dump_pdb(\"outputs/1ab1_res_selector_re1.pdb\")"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (46)\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: repeat  1274.78  0  0  0.55\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: coord_cst_weight  1274.78  0  0  0.55\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: scale:fa_rep  423.395  0  0  0.03245\n",
            "INFO:rosetta:core.pack.task: Packer task: initialize from command line()\n",
            "INFO:rosetta:core.pack.pack_rotamers: built 5220 rotamers at 46 positions.\n",
            "INFO:rosetta:core.pack.interaction_graph.interaction_graph_factory: Instantiating PDInteractionGraph\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: repack  80.8061  0  0  0.03245\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: scale:fa_rep  84.7937  0  0  0.0506\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: min  -159.089  1.05967  1.05967  0.0506\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: coord_cst_weight  -159.089  1.05967  1.05967  0.0506\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: scale:fa_rep  -114.304  1.05967  1.05967  0.154\n",
            "INFO:rosetta:core.pack.task: Packer task: initialize from command line()\n",
            "INFO:rosetta:core.pack.pack_rotamers: built 898 rotamers at 46 positions.\n",
            "INFO:rosetta:core.pack.interaction_graph.interaction_graph_factory: Instantiating DensePDInteractionGraph\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: repack  -115.063  1.05967  1.05967  0.154\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: scale:fa_rep  -105.006  1.05967  1.05967  0.17765\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: min  -132.662  0.97338  0.97338  0.17765\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: coord_cst_weight  -132.662  0.97338  0.97338  0.17765\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: scale:fa_rep  -115.096  0.97338  0.97338  0.3124\n",
            "INFO:rosetta:core.pack.task: Packer task: initialize from command line()\n",
            "INFO:rosetta:core.pack.pack_rotamers: built 804 rotamers at 46 positions.\n",
            "INFO:rosetta:core.pack.interaction_graph.interaction_graph_factory: Instantiating DensePDInteractionGraph\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: repack  -115.209  0.97338  0.97338  0.3124\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: scale:fa_rep  -110.6  0.97338  0.97338  0.34815\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: min  -117.786  0.944433  0.944433  0.34815\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: coord_cst_weight  -117.786  0.944433  0.944433  0.34815\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: scale:fa_rep  -102.76  0.944433  0.944433  0.55\n",
            "INFO:rosetta:core.pack.task: Packer task: initialize from command line()\n",
            "INFO:rosetta:core.pack.pack_rotamers: built 758 rotamers at 46 positions.\n",
            "INFO:rosetta:core.pack.interaction_graph.interaction_graph_factory: Instantiating DensePDInteractionGraph\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: repack  -102.76  0.944433  0.944433  0.55\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: min  -105.664  0.909208  0.909208  0.55\n",
            "INFO:rosetta:protocols.relax.FastRelax: MRP: 0  -105.664  -105.664  0.909208  0.909208\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: accept_to_best  -105.664  0.909208  0.909208  0.55\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: endrepeat  -105.664  0.909208  0.909208  0.55\n",
            "INFO:rosetta:protocols::checkpoint: Deleting checkpoints of FastRelax\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "True"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 41
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "uTotEs9CmdlG"
      },
      "source": [
        " Inspect the resulting design!"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "AabCEWt7mdlI",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 817
        },
        "outputId": "f32f4a83-37dd-4593-8ebb-e9c6a89f08c8"
      },
      "source": [
        "view_pose(pose)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_16234200519329896\"  style=\"position: relative; width: 1200px; height: 800px\">\n        <p id=\"3dmolwarning_16234200519329896\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n      resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n}\n\nvar viewer_16234200519329896 = null;\nvar warn = document.getElementById(\"3dmolwarning_16234200519329896\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_16234200519329896 = $3Dmol.createViewer($(\"#3dmolviewer_16234200519329896\"),{backgroundColor:\"white\"});\nviewer_16234200519329896.zoomTo();\n\tviewer_16234200519329896.addModels(\"HEADER                                            11-JUN-21   XXXX              \\nEXPDTA    THEORETICAL MODEL                                                     \\nREMARK 220                                                                      \\nREMARK 220 EXPERIMENTAL DETAILS                                                 \\nREMARK 220  EXPERIMENT TYPE                : THEORETICAL MODELLING              \\nREMARK 220  DATE OF DATA COLLECTION        : 11-JUN-21                          \\nREMARK 220                                                                      \\nREMARK 220 REMARK: MODEL GENERATED BY ROSETTA                                   \\nREMARK 220  VERSION 2020.50.post.dev+970.commits.3700df14560                    \\nATOM      1  N   THR A   1      15.647  13.699   3.148  1.00  0.00           N  \\nATOM      2  CA  THR A   1      15.488  12.506   3.975  1.00  0.00           C  \\nATOM      3  C   THR A   1      14.363  12.693   4.978  1.00  0.00           C  \\nATOM      4  O   THR A   1      14.179  13.779   5.533  1.00  0.00           O  \\nATOM      5  CB  THR A   1      16.785  12.142   4.735  1.00  0.00           C  \\nATOM      6  OG1 THR A   1      17.839  11.889   3.812  1.00  0.00           O  \\nATOM      7  CG2 THR A   1      16.604  10.909   5.632  1.00  0.00           C  \\nATOM      8 1H   THR A   1      16.291  13.500   2.397  1.00  0.00           H  \\nATOM      9 2H   THR A   1      14.749  13.954   2.756  1.00  0.00           H  \\nATOM     10 3H   THR A   1      16.006  14.478   3.704  1.00  0.00           H  \\nATOM     11  HA  THR A   1      15.225  11.670   3.327  1.00  0.00           H  \\nATOM     12  HB  THR A   1      17.066  12.959   5.351  1.00  0.00           H  \\nATOM     13  HG1 THR A   1      18.502  12.610   3.948  1.00  0.00           H  \\nATOM     14 1HG2 THR A   1      17.539  10.699   6.147  1.00  0.00           H  \\nATOM     15 2HG2 THR A   1      15.836  11.095   6.373  1.00  0.00           H  \\nATOM     16 3HG2 THR A   1      16.325  10.052   5.026  1.00  0.00           H  \\nATOM     17  N   THR A   2      13.565  11.653   5.184  1.00  0.00           N  \\nATOM     18  CA  THR A   2      12.505  11.759   6.176  1.00  0.00           C  \\nATOM     19  C   THR A   2      12.937  11.020   7.437  1.00  0.00           C  \\nATOM     20  O   THR A   2      13.469   9.911   7.359  1.00  0.00           O  \\nATOM     21  CB  THR A   2      11.176  11.194   5.677  1.00  0.00           C  \\nATOM     22  OG1 THR A   2      10.783  11.892   4.501  1.00  0.00           O  \\nATOM     23  CG2 THR A   2      10.104  11.417   6.747  1.00  0.00           C  \\nATOM     24  H   THR A   2      13.711  10.782   4.690  1.00  0.00           H  \\nATOM     25  HA  THR A   2      12.354  12.802   6.428  1.00  0.00           H  \\nATOM     26  HB  THR A   2      11.279  10.136   5.458  1.00  0.00           H  \\nATOM     27  HG1 THR A   2      10.683  12.816   4.701  1.00  0.00           H  \\nATOM     28 1HG2 THR A   2       9.152  11.036   6.400  1.00  0.00           H  \\nATOM     29 2HG2 THR A   2      10.375  10.910   7.663  1.00  0.00           H  \\nATOM     30 3HG2 THR A   2      10.019  12.477   6.947  1.00  0.00           H  \\nATOM     31  N   ALA A   3      12.768  11.671   8.580  1.00  0.00           N  \\nATOM     32  CA  ALA A   3      13.096  11.108   9.887  1.00  0.00           C  \\nATOM     33  C   ALA A   3      11.791  10.725  10.596  1.00  0.00           C  \\nATOM     34  O   ALA A   3      10.803  11.466  10.493  1.00  0.00           O  \\nATOM     35  CB  ALA A   3      13.914  12.124  10.662  1.00  0.00           C  \\nATOM     36  H   ALA A   3      12.351  12.598   8.528  1.00  0.00           H  \\nATOM     37  HA  ALA A   3      13.681  10.196   9.750  1.00  0.00           H  \\nATOM     38 1HB  ALA A   3      14.171  11.731  11.611  1.00  0.00           H  \\nATOM     39 2HB  ALA A   3      14.822  12.353  10.107  1.00  0.00           H  \\nATOM     40 3HB  ALA A   3      13.349  13.011  10.792  1.00  0.00           H  \\nATOM     41  N   PHE A   4      11.767   9.610  11.354  1.00  0.00           N  \\nATOM     42  CA  PHE A   4      10.499   9.194  11.963  1.00  0.00           C  \\nATOM     43  C   PHE A   4      10.493   9.036  13.489  1.00  0.00           C  \\nATOM     44  O   PHE A   4      11.491   8.646  14.096  1.00  0.00           O  \\nATOM     45  CB  PHE A   4      10.118   7.851  11.368  1.00  0.00           C  \\nATOM     46  CG  PHE A   4      10.012   7.928   9.943  1.00  0.00           C  \\nATOM     47  CD1 PHE A   4      11.123   7.631   9.200  1.00  0.00           C  \\nATOM     48  CD2 PHE A   4       8.869   8.334   9.317  1.00  0.00           C  \\nATOM     49  CE1 PHE A   4      11.111   7.719   7.865  1.00  0.00           C  \\nATOM     50  CE2 PHE A   4       8.852   8.427   7.973  1.00  0.00           C  \\nATOM     51  CZ  PHE A   4       9.996   8.110   7.248  1.00  0.00           C  \\nATOM     52  H   PHE A   4      12.595   9.009  11.455  1.00  0.00           H  \\nATOM     53  HA  PHE A   4       9.762   9.932  11.671  1.00  0.00           H  \\nATOM     54 1HB  PHE A   4      10.877   7.111  11.610  1.00  0.00           H  \\nATOM     55 2HB  PHE A   4       9.200   7.496  11.765  1.00  0.00           H  \\nATOM     56  HD1 PHE A   4      12.036   7.321   9.718  1.00  0.00           H  \\nATOM     57  HD2 PHE A   4       7.985   8.589   9.898  1.00  0.00           H  \\nATOM     58  HE1 PHE A   4      12.009   7.485   7.291  1.00  0.00           H  \\nATOM     59  HE2 PHE A   4       7.947   8.760   7.469  1.00  0.00           H  \\nATOM     60  HZ  PHE A   4      10.016   8.188   6.188  1.00  0.00           H  \\nATOM     61  N   PRO A   5       9.339   9.273  14.148  1.00  0.00           N  \\nATOM     62  CA  PRO A   5       9.126   9.049  15.565  1.00  0.00           C  \\nATOM     63  C   PRO A   5       9.049   7.556  15.899  1.00  0.00           C  \\nATOM     64  O   PRO A   5       9.195   7.158  17.056  1.00  0.00           O  \\nATOM     65  CB  PRO A   5       7.779   9.737  15.808  1.00  0.00           C  \\nATOM     66  CG  PRO A   5       7.059   9.660  14.475  1.00  0.00           C  \\nATOM     67  CD  PRO A   5       8.132   9.794  13.439  1.00  0.00           C  \\nATOM     68  HA  PRO A   5       9.930   9.538  16.137  1.00  0.00           H  \\nATOM     69 1HB  PRO A   5       7.238   9.222  16.613  1.00  0.00           H  \\nATOM     70 2HB  PRO A   5       7.945  10.774  16.142  1.00  0.00           H  \\nATOM     71 1HG  PRO A   5       6.513   8.706  14.394  1.00  0.00           H  \\nATOM     72 2HG  PRO A   5       6.305  10.464  14.398  1.00  0.00           H  \\nATOM     73 1HD  PRO A   5       7.850   9.192  12.574  1.00  0.00           H  \\nATOM     74 2HD  PRO A   5       8.236  10.838  13.196  1.00  0.00           H  \\nATOM     75  N   SER A   6       8.820   6.723  14.880  1.00  0.00           N  \\nATOM     76  CA  SER A   6       8.676   5.290  15.093  1.00  0.00           C  \\nATOM     77  C   SER A   6       8.954   4.462  13.851  1.00  0.00           C  \\nATOM     78  O   SER A   6       8.808   4.935  12.722  1.00  0.00           O  \\nATOM     79  CB  SER A   6       7.269   4.976  15.570  1.00  0.00           C  \\nATOM     80  OG  SER A   6       6.303   5.237  14.569  1.00  0.00           O  \\nATOM     81  H   SER A   6       8.723   7.089  13.950  1.00  0.00           H  \\nATOM     82  HA  SER A   6       9.378   4.986  15.867  1.00  0.00           H  \\nATOM     83 1HB  SER A   6       7.208   3.932  15.874  1.00  0.00           H  \\nATOM     84 2HB  SER A   6       7.050   5.581  16.449  1.00  0.00           H  \\nATOM     85  HG  SER A   6       5.454   5.274  15.032  1.00  0.00           H  \\nATOM     86  N   ILE A   7       9.218   3.181  14.077  1.00  0.00           N  \\nATOM     87  CA  ILE A   7       9.383   2.207  13.009  1.00  0.00           C  \\nATOM     88  C   ILE A   7       8.113   2.079  12.192  1.00  0.00           C  \\nATOM     89  O   ILE A   7       8.159   1.930  10.968  1.00  0.00           O  \\nATOM     90  CB  ILE A   7       9.818   0.852  13.557  1.00  0.00           C  \\nATOM     91  CG1 ILE A   7      11.253   0.981  14.087  1.00  0.00           C  \\nATOM     92  CG2 ILE A   7       9.697  -0.200  12.478  1.00  0.00           C  \\nATOM     93  CD1 ILE A   7      11.719  -0.195  14.888  1.00  0.00           C  \\nATOM     94  H   ILE A   7       9.326   2.872  15.034  1.00  0.00           H  \\nATOM     95  HA  ILE A   7      10.175   2.560  12.348  1.00  0.00           H  \\nATOM     96  HB  ILE A   7       9.184   0.576  14.398  1.00  0.00           H  \\nATOM     97 1HG1 ILE A   7      11.927   1.116  13.242  1.00  0.00           H  \\nATOM     98 2HG1 ILE A   7      11.315   1.868  14.718  1.00  0.00           H  \\nATOM     99 1HG2 ILE A   7      10.008  -1.166  12.874  1.00  0.00           H  \\nATOM    100 2HG2 ILE A   7       8.661  -0.262  12.145  1.00  0.00           H  \\nATOM    101 3HG2 ILE A   7      10.327   0.070  11.641  1.00  0.00           H  \\nATOM    102 1HD1 ILE A   7      12.742  -0.019  15.225  1.00  0.00           H  \\nATOM    103 2HD1 ILE A   7      11.069  -0.326  15.753  1.00  0.00           H  \\nATOM    104 3HD1 ILE A   7      11.689  -1.091  14.271  1.00  0.00           H  \\nATOM    105  N   VAL A   8       6.971   2.100  12.869  1.00  0.00           N  \\nATOM    106  CA  VAL A   8       5.707   2.008  12.168  1.00  0.00           C  \\nATOM    107  C   VAL A   8       5.553   3.190  11.224  1.00  0.00           C  \\nATOM    108  O   VAL A   8       5.143   3.017  10.075  1.00  0.00           O  \\nATOM    109  CB  VAL A   8       4.538   1.992  13.151  1.00  0.00           C  \\nATOM    110  CG1 VAL A   8       3.232   2.104  12.380  1.00  0.00           C  \\nATOM    111  CG2 VAL A   8       4.595   0.712  13.967  1.00  0.00           C  \\nATOM    112  H   VAL A   8       6.988   2.209  13.871  1.00  0.00           H  \\nATOM    113  HA  VAL A   8       5.693   1.085  11.589  1.00  0.00           H  \\nATOM    114  HB  VAL A   8       4.607   2.854  13.814  1.00  0.00           H  \\nATOM    115 1HG1 VAL A   8       2.395   2.103  13.077  1.00  0.00           H  \\nATOM    116 2HG1 VAL A   8       3.227   3.033  11.810  1.00  0.00           H  \\nATOM    117 3HG1 VAL A   8       3.136   1.260  11.697  1.00  0.00           H  \\nATOM    118 1HG2 VAL A   8       3.768   0.693  14.676  1.00  0.00           H  \\nATOM    119 2HG2 VAL A   8       4.523  -0.147  13.300  1.00  0.00           H  \\nATOM    120 3HG2 VAL A   8       5.540   0.672  14.510  1.00  0.00           H  \\nATOM    121  N   ALA A   9       5.891   4.398  11.689  1.00  0.00           N  \\nATOM    122  CA  ALA A   9       5.799   5.566  10.822  1.00  0.00           C  \\nATOM    123  C   ALA A   9       6.696   5.377   9.595  1.00  0.00           C  \\nATOM    124  O   ALA A   9       6.280   5.653   8.463  1.00  0.00           O  \\nATOM    125  CB  ALA A   9       6.191   6.806  11.609  1.00  0.00           C  \\nATOM    126  H   ALA A   9       6.211   4.520  12.650  1.00  0.00           H  \\nATOM    127  HA  ALA A   9       4.770   5.666  10.477  1.00  0.00           H  \\nATOM    128 1HB  ALA A   9       7.191   6.689  11.957  1.00  0.00           H  \\nATOM    129 2HB  ALA A   9       6.126   7.690  11.005  1.00  0.00           H  \\nATOM    130 3HB  ALA A   9       5.529   6.916  12.463  1.00  0.00           H  \\nATOM    131  N   ARG A  10       7.897   4.825   9.805  1.00  0.00           N  \\nATOM    132  CA  ARG A  10       8.807   4.540   8.697  1.00  0.00           C  \\nATOM    133  C   ARG A  10       8.183   3.546   7.714  1.00  0.00           C  \\nATOM    134  O   ARG A  10       8.249   3.722   6.493  1.00  0.00           O  \\nATOM    135  CB  ARG A  10      10.132   3.994   9.201  1.00  0.00           C  \\nATOM    136  CG  ARG A  10      11.166   3.741   8.114  1.00  0.00           C  \\nATOM    137  CD  ARG A  10      12.432   3.174   8.668  1.00  0.00           C  \\nATOM    138  NE  ARG A  10      12.260   1.816   9.187  1.00  0.00           N  \\nATOM    139  CZ  ARG A  10      13.096   1.219  10.064  1.00  0.00           C  \\nATOM    140  NH1 ARG A  10      14.145   1.871  10.517  1.00  0.00           N  \\nATOM    141  NH2 ARG A  10      12.867  -0.023  10.461  1.00  0.00           N  \\nATOM    142  H   ARG A  10       8.197   4.633  10.763  1.00  0.00           H  \\nATOM    143  HA  ARG A  10       9.002   5.466   8.167  1.00  0.00           H  \\nATOM    144 1HB  ARG A  10      10.562   4.690   9.927  1.00  0.00           H  \\nATOM    145 2HB  ARG A  10       9.971   3.057   9.714  1.00  0.00           H  \\nATOM    146 1HG  ARG A  10      10.759   3.036   7.388  1.00  0.00           H  \\nATOM    147 2HG  ARG A  10      11.400   4.681   7.616  1.00  0.00           H  \\nATOM    148 1HD  ARG A  10      13.189   3.147   7.886  1.00  0.00           H  \\nATOM    149 2HD  ARG A  10      12.778   3.806   9.481  1.00  0.00           H  \\nATOM    150  HE  ARG A  10      11.464   1.287   8.858  1.00  0.00           H  \\nATOM    151 1HH1 ARG A  10      14.329   2.818  10.200  1.00  0.00           H  \\nATOM    152 2HH1 ARG A  10      14.773   1.427  11.171  1.00  0.00           H  \\nATOM    153 1HH2 ARG A  10      12.071  -0.530  10.094  1.00  0.00           H  \\nATOM    154 2HH2 ARG A  10      13.496  -0.469  11.113  1.00  0.00           H  \\nATOM    155  N   SER A  11       7.567   2.489   8.242  1.00  0.00           N  \\nATOM    156  CA  SER A  11       6.954   1.486   7.384  1.00  0.00           C  \\nATOM    157  C   SER A  11       5.818   2.100   6.567  1.00  0.00           C  \\nATOM    158  O   SER A  11       5.696   1.842   5.360  1.00  0.00           O  \\nATOM    159  CB  SER A  11       6.432   0.337   8.227  1.00  0.00           C  \\nATOM    160  OG  SER A  11       7.491  -0.344   8.855  1.00  0.00           O  \\nATOM    161  H   SER A  11       7.547   2.371   9.254  1.00  0.00           H  \\nATOM    162  HA  SER A  11       7.710   1.108   6.698  1.00  0.00           H  \\nATOM    163 1HB  SER A  11       5.747   0.722   8.980  1.00  0.00           H  \\nATOM    164 2HB  SER A  11       5.874  -0.350   7.596  1.00  0.00           H  \\nATOM    165  HG  SER A  11       7.883   0.298   9.474  1.00  0.00           H  \\nATOM    166  N   ASN A  12       5.014   2.955   7.208  1.00  0.00           N  \\nATOM    167  CA  ASN A  12       3.918   3.607   6.513  1.00  0.00           C  \\nATOM    168  C   ASN A  12       4.467   4.478   5.395  1.00  0.00           C  \\nATOM    169  O   ASN A  12       3.898   4.527   4.305  1.00  0.00           O  \\nATOM    170  CB  ASN A  12       3.089   4.443   7.472  1.00  0.00           C  \\nATOM    171  CG  ASN A  12       2.223   3.627   8.393  1.00  0.00           C  \\nATOM    172  OD1 ASN A  12       1.939   2.449   8.142  1.00  0.00           O  \\nATOM    173  ND2 ASN A  12       1.780   4.246   9.461  1.00  0.00           N  \\nATOM    174  H   ASN A  12       5.156   3.128   8.204  1.00  0.00           H  \\nATOM    175  HA  ASN A  12       3.285   2.844   6.060  1.00  0.00           H  \\nATOM    176 1HB  ASN A  12       3.761   5.043   8.085  1.00  0.00           H  \\nATOM    177 2HB  ASN A  12       2.461   5.129   6.907  1.00  0.00           H  \\nATOM    178 1HD2 ASN A  12       1.187   3.764  10.106  1.00  0.00           H  \\nATOM    179 2HD2 ASN A  12       2.028   5.199   9.625  1.00  0.00           H  \\nATOM    180  N   PHE A  13       5.603   5.130   5.648  1.00  0.00           N  \\nATOM    181  CA  PHE A  13       6.234   6.006   4.673  1.00  0.00           C  \\nATOM    182  C   PHE A  13       6.537   5.237   3.407  1.00  0.00           C  \\nATOM    183  O   PHE A  13       6.213   5.674   2.298  1.00  0.00           O  \\nATOM    184  CB  PHE A  13       7.544   6.546   5.225  1.00  0.00           C  \\nATOM    185  CG  PHE A  13       8.254   7.538   4.398  1.00  0.00           C  \\nATOM    186  CD1 PHE A  13       7.936   8.874   4.494  1.00  0.00           C  \\nATOM    187  CD2 PHE A  13       9.263   7.158   3.553  1.00  0.00           C  \\nATOM    188  CE1 PHE A  13       8.591   9.813   3.761  1.00  0.00           C  \\nATOM    189  CE2 PHE A  13       9.937   8.097   2.820  1.00  0.00           C  \\nATOM    190  CZ  PHE A  13       9.595   9.426   2.930  1.00  0.00           C  \\nATOM    191  H   PHE A  13       6.019   5.058   6.578  1.00  0.00           H  \\nATOM    192  HA  PHE A  13       5.556   6.820   4.450  1.00  0.00           H  \\nATOM    193 1HB  PHE A  13       7.371   6.952   6.195  1.00  0.00           H  \\nATOM    194 2HB  PHE A  13       8.224   5.735   5.352  1.00  0.00           H  \\nATOM    195  HD1 PHE A  13       7.160   9.174   5.188  1.00  0.00           H  \\nATOM    196  HD2 PHE A  13       9.532   6.099   3.479  1.00  0.00           H  \\nATOM    197  HE1 PHE A  13       8.330  10.874   3.852  1.00  0.00           H  \\nATOM    198  HE2 PHE A  13      10.747   7.793   2.158  1.00  0.00           H  \\nATOM    199  HZ  PHE A  13      10.133  10.171   2.369  1.00  0.00           H  \\nATOM    200  N   ASN A  14       7.153   4.071   3.578  1.00  0.00           N  \\nATOM    201  CA  ASN A  14       7.549   3.257   2.444  1.00  0.00           C  \\nATOM    202  C   ASN A  14       6.343   2.857   1.597  1.00  0.00           C  \\nATOM    203  O   ASN A  14       6.426   2.815   0.369  1.00  0.00           O  \\nATOM    204  CB  ASN A  14       8.314   2.052   2.940  1.00  0.00           C  \\nATOM    205  CG  ASN A  14       9.696   2.431   3.421  1.00  0.00           C  \\nATOM    206  OD1 ASN A  14      10.240   3.475   3.036  1.00  0.00           O  \\nATOM    207  ND2 ASN A  14      10.270   1.605   4.258  1.00  0.00           N  \\nATOM    208  H   ASN A  14       7.381   3.769   4.530  1.00  0.00           H  \\nATOM    209  HA  ASN A  14       8.208   3.851   1.812  1.00  0.00           H  \\nATOM    210 1HB  ASN A  14       7.768   1.591   3.765  1.00  0.00           H  \\nATOM    211 2HB  ASN A  14       8.397   1.314   2.144  1.00  0.00           H  \\nATOM    212 1HD2 ASN A  14      11.184   1.805   4.611  1.00  0.00           H  \\nATOM    213 2HD2 ASN A  14       9.794   0.775   4.545  1.00  0.00           H  \\nATOM    214  N   VAL A  15       5.203   2.620   2.242  1.00  0.00           N  \\nATOM    215  CA  VAL A  15       3.987   2.288   1.508  1.00  0.00           C  \\nATOM    216  C   VAL A  15       3.423   3.527   0.798  1.00  0.00           C  \\nATOM    217  O   VAL A  15       3.070   3.495  -0.382  1.00  0.00           O  \\nATOM    218  CB  VAL A  15       2.929   1.717   2.457  1.00  0.00           C  \\nATOM    219  CG1 VAL A  15       1.625   1.516   1.714  1.00  0.00           C  \\nATOM    220  CG2 VAL A  15       3.438   0.415   3.031  1.00  0.00           C  \\nATOM    221  H   VAL A  15       5.199   2.642   3.266  1.00  0.00           H  \\nATOM    222  HA  VAL A  15       4.231   1.532   0.765  1.00  0.00           H  \\nATOM    223  HB  VAL A  15       2.748   2.427   3.265  1.00  0.00           H  \\nATOM    224 1HG1 VAL A  15       0.876   1.119   2.398  1.00  0.00           H  \\nATOM    225 2HG1 VAL A  15       1.283   2.472   1.317  1.00  0.00           H  \\nATOM    226 3HG1 VAL A  15       1.777   0.814   0.896  1.00  0.00           H  \\nATOM    227 1HG2 VAL A  15       2.699   0.005   3.718  1.00  0.00           H  \\nATOM    228 2HG2 VAL A  15       3.618  -0.292   2.223  1.00  0.00           H  \\nATOM    229 3HG2 VAL A  15       4.369   0.599   3.567  1.00  0.00           H  \\nATOM    230  N   MET A  16       3.375   4.639   1.516  1.00  0.00           N  \\nATOM    231  CA  MET A  16       2.872   5.903   0.998  1.00  0.00           C  \\nATOM    232  C   MET A  16       3.658   6.461  -0.185  1.00  0.00           C  \\nATOM    233  O   MET A  16       3.107   7.197  -1.001  1.00  0.00           O  \\nATOM    234  CB  MET A  16       2.875   6.907   2.108  1.00  0.00           C  \\nATOM    235  CG  MET A  16       1.828   6.770   3.143  1.00  0.00           C  \\nATOM    236  SD  MET A  16       0.210   7.042   2.476  1.00  0.00           S  \\nATOM    237  CE  MET A  16       0.417   8.784   2.039  1.00  0.00           C  \\nATOM    238  H   MET A  16       3.693   4.609   2.483  1.00  0.00           H  \\nATOM    239  HA  MET A  16       1.848   5.744   0.666  1.00  0.00           H  \\nATOM    240 1HB  MET A  16       3.826   6.834   2.612  1.00  0.00           H  \\nATOM    241 2HB  MET A  16       2.782   7.880   1.687  1.00  0.00           H  \\nATOM    242 1HG  MET A  16       1.858   5.777   3.583  1.00  0.00           H  \\nATOM    243 2HG  MET A  16       2.020   7.493   3.913  1.00  0.00           H  \\nATOM    244 1HE  MET A  16      -0.506   9.162   1.603  1.00  0.00           H  \\nATOM    245 2HE  MET A  16       0.661   9.357   2.924  1.00  0.00           H  \\nATOM    246 3HE  MET A  16       1.225   8.882   1.316  1.00  0.00           H  \\nATOM    247  N   ARG A  17       4.934   6.092  -0.300  1.00  0.00           N  \\nATOM    248  CA  ARG A  17       5.775   6.523  -1.414  1.00  0.00           C  \\nATOM    249  C   ARG A  17       5.593   5.671  -2.677  1.00  0.00           C  \\nATOM    250  O   ARG A  17       6.136   6.004  -3.735  1.00  0.00           O  \\nATOM    251  CB  ARG A  17       7.255   6.457  -1.051  1.00  0.00           C  \\nATOM    252  CG  ARG A  17       7.781   7.426   0.001  1.00  0.00           C  \\nATOM    253  CD  ARG A  17       7.827   8.821  -0.493  1.00  0.00           C  \\nATOM    254  NE  ARG A  17       8.612   8.973  -1.732  1.00  0.00           N  \\nATOM    255  CZ  ARG A  17       9.960   9.074  -1.816  1.00  0.00           C  \\nATOM    256  NH1 ARG A  17      10.701   8.988  -0.745  1.00  0.00           N  \\nATOM    257  NH2 ARG A  17      10.537   9.274  -2.990  1.00  0.00           N  \\nATOM    258  H   ARG A  17       5.347   5.530   0.444  1.00  0.00           H  \\nATOM    259  HA  ARG A  17       5.521   7.557  -1.650  1.00  0.00           H  \\nATOM    260 1HB  ARG A  17       7.471   5.452  -0.679  1.00  0.00           H  \\nATOM    261 2HB  ARG A  17       7.848   6.603  -1.952  1.00  0.00           H  \\nATOM    262 1HG  ARG A  17       7.126   7.418   0.866  1.00  0.00           H  \\nATOM    263 2HG  ARG A  17       8.780   7.123   0.302  1.00  0.00           H  \\nATOM    264 1HD  ARG A  17       6.821   9.149  -0.704  1.00  0.00           H  \\nATOM    265 2HD  ARG A  17       8.258   9.468   0.267  1.00  0.00           H  \\nATOM    266  HE  ARG A  17       8.081   9.070  -2.593  1.00  0.00           H  \\nATOM    267 1HH1 ARG A  17      10.269   8.853   0.154  1.00  0.00           H  \\nATOM    268 2HH1 ARG A  17      11.714   9.135  -0.802  1.00  0.00           H  \\nATOM    269 1HH2 ARG A  17       9.974   9.348  -3.824  1.00  0.00           H  \\nATOM    270 2HH2 ARG A  17      11.543   9.357  -3.045  1.00  0.00           H  \\nATOM    271  N   LEU A  18       4.814   4.590  -2.605  1.00  0.00           N  \\nATOM    272  CA  LEU A  18       4.682   3.706  -3.758  1.00  0.00           C  \\nATOM    273  C   LEU A  18       4.174   4.367  -5.059  1.00  0.00           C  \\nATOM    274  O   LEU A  18       4.684   4.021  -6.122  1.00  0.00           O  \\nATOM    275  CB  LEU A  18       3.809   2.497  -3.388  1.00  0.00           C  \\nATOM    276  CG  LEU A  18       4.463   1.516  -2.409  1.00  0.00           C  \\nATOM    277  CD1 LEU A  18       3.440   0.482  -1.963  1.00  0.00           C  \\nATOM    278  CD2 LEU A  18       5.646   0.864  -3.074  1.00  0.00           C  \\nATOM    279  H   LEU A  18       4.331   4.344  -1.738  1.00  0.00           H  \\nATOM    280  HA  LEU A  18       5.677   3.315  -3.969  1.00  0.00           H  \\nATOM    281 1HB  LEU A  18       2.902   2.837  -2.913  1.00  0.00           H  \\nATOM    282 2HB  LEU A  18       3.547   1.954  -4.294  1.00  0.00           H  \\nATOM    283  HG  LEU A  18       4.809   2.056  -1.535  1.00  0.00           H  \\nATOM    284 1HD1 LEU A  18       3.902  -0.205  -1.256  1.00  0.00           H  \\nATOM    285 2HD1 LEU A  18       2.608   0.996  -1.478  1.00  0.00           H  \\nATOM    286 3HD1 LEU A  18       3.076  -0.071  -2.826  1.00  0.00           H  \\nATOM    287 1HD2 LEU A  18       6.118   0.176  -2.374  1.00  0.00           H  \\nATOM    288 2HD2 LEU A  18       5.315   0.318  -3.956  1.00  0.00           H  \\nATOM    289 3HD2 LEU A  18       6.362   1.632  -3.368  1.00  0.00           H  \\nATOM    290  N   PRO A  19       3.192   5.303  -5.057  1.00  0.00           N  \\nATOM    291  CA  PRO A  19       2.719   5.988  -6.243  1.00  0.00           C  \\nATOM    292  C   PRO A  19       3.553   7.224  -6.618  1.00  0.00           C  \\nATOM    293  O   PRO A  19       3.158   7.979  -7.507  1.00  0.00           O  \\nATOM    294  CB  PRO A  19       1.287   6.351  -5.850  1.00  0.00           C  \\nATOM    295  CG  PRO A  19       1.374   6.632  -4.385  1.00  0.00           C  \\nATOM    296  CD  PRO A  19       2.400   5.644  -3.847  1.00  0.00           C  \\nATOM    297  HA  PRO A  19       2.713   5.275  -7.080  1.00  0.00           H  \\nATOM    298 1HB  PRO A  19       0.949   7.223  -6.429  1.00  0.00           H  \\nATOM    299 2HB  PRO A  19       0.606   5.519  -6.092  1.00  0.00           H  \\nATOM    300 1HG  PRO A  19       1.685   7.672  -4.230  1.00  0.00           H  \\nATOM    301 2HG  PRO A  19       0.389   6.523  -3.912  1.00  0.00           H  \\nATOM    302 1HD  PRO A  19       2.971   6.203  -3.147  1.00  0.00           H  \\nATOM    303 2HD  PRO A  19       1.913   4.777  -3.404  1.00  0.00           H  \\nATOM    304  N   GLY A  20       4.685   7.463  -5.932  1.00  0.00           N  \\nATOM    305  CA  GLY A  20       5.496   8.650  -6.213  1.00  0.00           C  \\nATOM    306  C   GLY A  20       5.162   9.901  -5.383  1.00  0.00           C  \\nATOM    307  O   GLY A  20       5.639  10.993  -5.695  1.00  0.00           O  \\nATOM    308  H   GLY A  20       5.023   6.826  -5.211  1.00  0.00           H  \\nATOM    309 1HA  GLY A  20       6.542   8.392  -6.048  1.00  0.00           H  \\nATOM    310 2HA  GLY A  20       5.408   8.892  -7.271  1.00  0.00           H  \\nATOM    311  N   THR A  21       4.359   9.751  -4.334  1.00  0.00           N  \\nATOM    312  CA  THR A  21       3.981  10.875  -3.468  1.00  0.00           C  \\nATOM    313  C   THR A  21       5.235  11.541  -2.877  1.00  0.00           C  \\nATOM    314  O   THR A  21       6.147  10.849  -2.425  1.00  0.00           O  \\nATOM    315  CB  THR A  21       3.047  10.419  -2.325  1.00  0.00           C  \\nATOM    316  OG1 THR A  21       1.872   9.810  -2.868  1.00  0.00           O  \\nATOM    317  CG2 THR A  21       2.615  11.610  -1.484  1.00  0.00           C  \\nATOM    318  H   THR A  21       3.999   8.829  -4.135  1.00  0.00           H  \\nATOM    319  HA  THR A  21       3.454  11.618  -4.069  1.00  0.00           H  \\nATOM    320  HB  THR A  21       3.571   9.695  -1.702  1.00  0.00           H  \\nATOM    321  HG1 THR A  21       1.392   9.361  -2.161  1.00  0.00           H  \\nATOM    322 1HG2 THR A  21       1.960  11.273  -0.682  1.00  0.00           H  \\nATOM    323 2HG2 THR A  21       3.479  12.087  -1.063  1.00  0.00           H  \\nATOM    324 3HG2 THR A  21       2.084  12.325  -2.108  1.00  0.00           H  \\nATOM    325  N   SER A  22       5.290  12.877  -2.905  1.00  0.00           N  \\nATOM    326  CA  SER A  22       6.451  13.619  -2.400  1.00  0.00           C  \\nATOM    327  C   SER A  22       6.744  13.323  -0.933  1.00  0.00           C  \\nATOM    328  O   SER A  22       5.828  13.113  -0.132  1.00  0.00           O  \\nATOM    329  CB  SER A  22       6.242  15.112  -2.555  1.00  0.00           C  \\nATOM    330  OG  SER A  22       6.108  15.468  -3.899  1.00  0.00           O  \\nATOM    331  H   SER A  22       4.512  13.384  -3.301  1.00  0.00           H  \\nATOM    332  HA  SER A  22       7.323  13.322  -2.988  1.00  0.00           H  \\nATOM    333 1HB  SER A  22       5.369  15.421  -2.000  1.00  0.00           H  \\nATOM    334 2HB  SER A  22       7.095  15.635  -2.126  1.00  0.00           H  \\nATOM    335  HG  SER A  22       6.873  15.100  -4.351  1.00  0.00           H  \\nATOM    336  N   GLU A  23       8.030  13.373  -0.570  1.00  0.00           N  \\nATOM    337  CA  GLU A  23       8.465  13.069   0.792  1.00  0.00           C  \\nATOM    338  C   GLU A  23       7.865  13.983   1.841  1.00  0.00           C  \\nATOM    339  O   GLU A  23       7.515  13.535   2.926  1.00  0.00           O  \\nATOM    340  CB  GLU A  23       9.994  13.168   0.884  1.00  0.00           C  \\nATOM    341  CG  GLU A  23      10.696  12.085   0.124  1.00  0.00           C  \\nATOM    342  CD  GLU A  23      12.231  12.094   0.173  1.00  0.00           C  \\nATOM    343  OE1 GLU A  23      12.814  13.075   0.594  1.00  0.00           O  \\nATOM    344  OE2 GLU A  23      12.815  11.078  -0.200  1.00  0.00           O  \\nATOM    345  H   GLU A  23       8.728  13.579  -1.272  1.00  0.00           H  \\nATOM    346  HA  GLU A  23       8.163  12.045   1.022  1.00  0.00           H  \\nATOM    347 1HB  GLU A  23      10.327  14.133   0.504  1.00  0.00           H  \\nATOM    348 2HB  GLU A  23      10.300  13.099   1.929  1.00  0.00           H  \\nATOM    349 1HG  GLU A  23      10.342  11.177   0.523  1.00  0.00           H  \\nATOM    350 2HG  GLU A  23      10.373  12.134  -0.914  1.00  0.00           H  \\nATOM    351  N   ALA A  24       7.729  15.270   1.536  1.00  0.00           N  \\nATOM    352  CA  ALA A  24       7.152  16.187   2.516  1.00  0.00           C  \\nATOM    353  C   ALA A  24       5.712  15.807   2.843  1.00  0.00           C  \\nATOM    354  O   ALA A  24       5.262  15.904   3.987  1.00  0.00           O  \\nATOM    355  CB  ALA A  24       7.210  17.607   1.998  1.00  0.00           C  \\nATOM    356  H   ALA A  24       8.032  15.605   0.634  1.00  0.00           H  \\nATOM    357  HA  ALA A  24       7.729  16.116   3.432  1.00  0.00           H  \\nATOM    358 1HB  ALA A  24       8.246  17.876   1.794  1.00  0.00           H  \\nATOM    359 2HB  ALA A  24       6.625  17.684   1.081  1.00  0.00           H  \\nATOM    360 3HB  ALA A  24       6.799  18.280   2.747  1.00  0.00           H  \\nATOM    361  N   ILE A  25       4.975  15.365   1.827  1.00  0.00           N  \\nATOM    362  CA  ILE A  25       3.579  15.019   2.021  1.00  0.00           C  \\nATOM    363  C   ILE A  25       3.477  13.775   2.864  1.00  0.00           C  \\nATOM    364  O   ILE A  25       2.746  13.725   3.868  1.00  0.00           O  \\nATOM    365  CB  ILE A  25       2.887  14.726   0.683  1.00  0.00           C  \\nATOM    366  CG1 ILE A  25       2.810  15.971  -0.170  1.00  0.00           C  \\nATOM    367  CG2 ILE A  25       1.506  14.136   0.926  1.00  0.00           C  \\nATOM    368  CD1 ILE A  25       2.390  15.671  -1.595  1.00  0.00           C  \\nATOM    369  H   ILE A  25       5.392  15.265   0.912  1.00  0.00           H  \\nATOM    370  HA  ILE A  25       3.074  15.838   2.530  1.00  0.00           H  \\nATOM    371  HB  ILE A  25       3.482  14.016   0.135  1.00  0.00           H  \\nATOM    372 1HG1 ILE A  25       2.101  16.670   0.269  1.00  0.00           H  \\nATOM    373 2HG1 ILE A  25       3.793  16.442  -0.194  1.00  0.00           H  \\nATOM    374 1HG2 ILE A  25       1.036  13.910  -0.028  1.00  0.00           H  \\nATOM    375 2HG2 ILE A  25       1.598  13.222   1.510  1.00  0.00           H  \\nATOM    376 3HG2 ILE A  25       0.895  14.855   1.471  1.00  0.00           H  \\nATOM    377 1HD1 ILE A  25       2.362  16.597  -2.169  1.00  0.00           H  \\nATOM    378 2HD1 ILE A  25       3.098  14.986  -2.048  1.00  0.00           H  \\nATOM    379 3HD1 ILE A  25       1.402  15.216  -1.593  1.00  0.00           H  \\nATOM    380  N   VAL A  26       4.254  12.776   2.473  1.00  0.00           N  \\nATOM    381  CA  VAL A  26       4.197  11.517   3.158  1.00  0.00           C  \\nATOM    382  C   VAL A  26       4.693  11.662   4.579  1.00  0.00           C  \\nATOM    383  O   VAL A  26       4.135  11.066   5.499  1.00  0.00           O  \\nATOM    384  CB  VAL A  26       5.006  10.444   2.474  1.00  0.00           C  \\nATOM    385  CG1 VAL A  26       4.940   9.292   3.328  1.00  0.00           C  \\nATOM    386  CG2 VAL A  26       4.474  10.178   1.123  1.00  0.00           C  \\nATOM    387  H   VAL A  26       4.862  12.892   1.659  1.00  0.00           H  \\nATOM    388  HA  VAL A  26       3.159  11.189   3.181  1.00  0.00           H  \\nATOM    389  HB  VAL A  26       6.052  10.747   2.400  1.00  0.00           H  \\nATOM    390 1HG1 VAL A  26       5.524   8.486   2.884  1.00  0.00           H  \\nATOM    391 2HG1 VAL A  26       5.316   9.551   4.290  1.00  0.00           H  \\nATOM    392 3HG1 VAL A  26       3.916   9.003   3.437  1.00  0.00           H  \\nATOM    393 1HG2 VAL A  26       5.037   9.388   0.653  1.00  0.00           H  \\nATOM    394 2HG2 VAL A  26       3.440   9.897   1.183  1.00  0.00           H  \\nATOM    395 3HG2 VAL A  26       4.569  11.075   0.551  1.00  0.00           H  \\nATOM    396  N   ALA A  27       5.743  12.444   4.787  1.00  0.00           N  \\nATOM    397  CA  ALA A  27       6.242  12.598   6.128  1.00  0.00           C  \\nATOM    398  C   ALA A  27       5.134  13.083   7.049  1.00  0.00           C  \\nATOM    399  O   ALA A  27       4.968  12.512   8.135  1.00  0.00           O  \\nATOM    400  CB  ALA A  27       7.391  13.579   6.145  1.00  0.00           C  \\nATOM    401  H   ALA A  27       6.209  12.921   4.021  1.00  0.00           H  \\nATOM    402  HA  ALA A  27       6.590  11.627   6.479  1.00  0.00           H  \\nATOM    403 1HB  ALA A  27       7.046  14.551   5.787  1.00  0.00           H  \\nATOM    404 2HB  ALA A  27       7.776  13.683   7.154  1.00  0.00           H  \\nATOM    405 3HB  ALA A  27       8.168  13.213   5.491  1.00  0.00           H  \\nATOM    406  N   THR A  28       4.295  14.042   6.599  1.00  0.00           N  \\nATOM    407  CA  THR A  28       3.250  14.514   7.504  1.00  0.00           C  \\nATOM    408  C   THR A  28       2.202  13.431   7.751  1.00  0.00           C  \\nATOM    409  O   THR A  28       1.665  13.332   8.856  1.00  0.00           O  \\nATOM    410  CB  THR A  28       2.568  15.798   6.980  1.00  0.00           C  \\nATOM    411  OG1 THR A  28       1.935  15.557   5.707  1.00  0.00           O  \\nATOM    412  CG2 THR A  28       3.608  16.891   6.830  1.00  0.00           C  \\nATOM    413  H   THR A  28       4.437  14.483   5.684  1.00  0.00           H  \\nATOM    414  HA  THR A  28       3.714  14.756   8.460  1.00  0.00           H  \\nATOM    415  HB  THR A  28       1.813  16.124   7.689  1.00  0.00           H  \\nATOM    416  HG1 THR A  28       2.480  14.959   5.154  1.00  0.00           H  \\nATOM    417 1HG2 THR A  28       3.132  17.801   6.468  1.00  0.00           H  \\nATOM    418 2HG2 THR A  28       4.075  17.082   7.795  1.00  0.00           H  \\nATOM    419 3HG2 THR A  28       4.362  16.575   6.126  1.00  0.00           H  \\nATOM    420  N   TYR A  29       1.975  12.564   6.759  1.00  0.00           N  \\nATOM    421  CA  TYR A  29       1.053  11.434   6.906  1.00  0.00           C  \\nATOM    422  C   TYR A  29       1.501  10.552   8.067  1.00  0.00           C  \\nATOM    423  O   TYR A  29       0.703  10.137   8.911  1.00  0.00           O  \\nATOM    424  CB  TYR A  29       0.988  10.576   5.638  1.00  0.00           C  \\nATOM    425  CG  TYR A  29       0.072   9.382   5.757  1.00  0.00           C  \\nATOM    426  CD1 TYR A  29      -1.285   9.514   5.534  1.00  0.00           C  \\nATOM    427  CD2 TYR A  29       0.600   8.149   6.111  1.00  0.00           C  \\nATOM    428  CE1 TYR A  29      -2.108   8.410   5.646  1.00  0.00           C  \\nATOM    429  CE2 TYR A  29      -0.214   7.050   6.226  1.00  0.00           C  \\nATOM    430  CZ  TYR A  29      -1.566   7.174   5.991  1.00  0.00           C  \\nATOM    431  OH  TYR A  29      -2.386   6.074   6.101  1.00  0.00           O  \\nATOM    432  H   TYR A  29       2.436  12.721   5.860  1.00  0.00           H  \\nATOM    433  HA  TYR A  29       0.060  11.818   7.134  1.00  0.00           H  \\nATOM    434 1HB  TYR A  29       0.660  11.187   4.797  1.00  0.00           H  \\nATOM    435 2HB  TYR A  29       1.959  10.196   5.397  1.00  0.00           H  \\nATOM    436  HD1 TYR A  29      -1.702  10.483   5.264  1.00  0.00           H  \\nATOM    437  HD2 TYR A  29       1.662   8.052   6.292  1.00  0.00           H  \\nATOM    438  HE1 TYR A  29      -3.177   8.508   5.464  1.00  0.00           H  \\nATOM    439  HE2 TYR A  29       0.208   6.084   6.498  1.00  0.00           H  \\nATOM    440  HH  TYR A  29      -3.289   6.329   5.896  1.00  0.00           H  \\nATOM    441  N   THR A  30       2.801  10.243   8.061  1.00  0.00           N  \\nATOM    442  CA  THR A  30       3.420   9.347   9.028  1.00  0.00           C  \\nATOM    443  C   THR A  30       3.776  10.037  10.349  1.00  0.00           C  \\nATOM    444  O   THR A  30       4.029   9.367  11.353  1.00  0.00           O  \\nATOM    445  CB  THR A  30       4.711   8.772   8.436  1.00  0.00           C  \\nATOM    446  OG1 THR A  30       5.644   9.838   8.231  1.00  0.00           O  \\nATOM    447  CG2 THR A  30       4.424   8.113   7.125  1.00  0.00           C  \\nATOM    448  H   THR A  30       3.376  10.641   7.314  1.00  0.00           H  \\nATOM    449  HA  THR A  30       2.725   8.536   9.238  1.00  0.00           H  \\nATOM    450  HB  THR A  30       5.131   8.048   9.093  1.00  0.00           H  \\nATOM    451  HG1 THR A  30       5.160  10.626   7.911  1.00  0.00           H  \\nATOM    452 1HG2 THR A  30       5.335   7.734   6.733  1.00  0.00           H  \\nATOM    453 2HG2 THR A  30       3.726   7.305   7.273  1.00  0.00           H  \\nATOM    454 3HG2 THR A  30       4.004   8.825   6.435  1.00  0.00           H  \\nATOM    455  N   GLY A  31       3.788  11.371  10.352  1.00  0.00           N  \\nATOM    456  CA  GLY A  31       4.147  12.151  11.530  1.00  0.00           C  \\nATOM    457  C   GLY A  31       5.652  12.409  11.637  1.00  0.00           C  \\nATOM    458  O   GLY A  31       6.154  12.740  12.716  1.00  0.00           O  \\nATOM    459  H   GLY A  31       3.591  11.867   9.489  1.00  0.00           H  \\nATOM    460 1HA  GLY A  31       3.620  13.104  11.486  1.00  0.00           H  \\nATOM    461 2HA  GLY A  31       3.799  11.636  12.422  1.00  0.00           H  \\nATOM    462  N   GLY A  32       6.379  12.202  10.541  1.00  0.00           N  \\nATOM    463  CA  GLY A  32       7.819  12.411  10.519  1.00  0.00           C  \\nATOM    464  C   GLY A  32       8.160  13.822  10.076  1.00  0.00           C  \\nATOM    465  O   GLY A  32       7.276  14.668   9.905  1.00  0.00           O  \\nATOM    466  H   GLY A  32       5.905  11.950   9.678  1.00  0.00           H  \\nATOM    467 1HA  GLY A  32       8.240  12.238  11.502  1.00  0.00           H  \\nATOM    468 2HA  GLY A  32       8.282  11.692   9.843  1.00  0.00           H  \\nATOM    469  N   ILE A  33       9.451  14.068   9.902  1.00  0.00           N  \\nATOM    470  CA  ILE A  33       9.948  15.378   9.491  1.00  0.00           C  \\nATOM    471  C   ILE A  33      10.922  15.260   8.343  1.00  0.00           C  \\nATOM    472  O   ILE A  33      11.547  14.219   8.164  1.00  0.00           O  \\nATOM    473  CB  ILE A  33      10.666  16.099  10.639  1.00  0.00           C  \\nATOM    474  CG1 ILE A  33      11.854  15.283  11.033  1.00  0.00           C  \\nATOM    475  CG2 ILE A  33       9.736  16.301  11.816  1.00  0.00           C  \\nATOM    476  CD1 ILE A  33      12.778  15.958  11.979  1.00  0.00           C  \\nATOM    477  H   ILE A  33      10.097  13.296  10.073  1.00  0.00           H  \\nATOM    478  HA  ILE A  33       9.106  15.986   9.162  1.00  0.00           H  \\nATOM    479  HB  ILE A  33      11.030  17.066  10.289  1.00  0.00           H  \\nATOM    480 1HG1 ILE A  33      11.497  14.361  11.496  1.00  0.00           H  \\nATOM    481 2HG1 ILE A  33      12.407  15.028  10.134  1.00  0.00           H  \\nATOM    482 1HG2 ILE A  33      10.267  16.810  12.620  1.00  0.00           H  \\nATOM    483 2HG2 ILE A  33       8.882  16.901  11.508  1.00  0.00           H  \\nATOM    484 3HG2 ILE A  33       9.389  15.343  12.166  1.00  0.00           H  \\nATOM    485 1HD1 ILE A  33      13.602  15.289  12.206  1.00  0.00           H  \\nATOM    486 2HD1 ILE A  33      13.163  16.869  11.518  1.00  0.00           H  \\nATOM    487 3HD1 ILE A  33      12.251  16.213  12.886  1.00  0.00           H  \\nATOM    488  N   ILE A  34      11.118  16.354   7.621  1.00  0.00           N  \\nATOM    489  CA  ILE A  34      12.112  16.364   6.558  1.00  0.00           C  \\nATOM    490  C   ILE A  34      13.400  17.017   7.039  1.00  0.00           C  \\nATOM    491  O   ILE A  34      13.376  18.111   7.608  1.00  0.00           O  \\nATOM    492  CB  ILE A  34      11.597  17.104   5.313  1.00  0.00           C  \\nATOM    493  CG1 ILE A  34      10.282  16.491   4.854  1.00  0.00           C  \\nATOM    494  CG2 ILE A  34      12.635  16.997   4.196  1.00  0.00           C  \\nATOM    495  CD1 ILE A  34      10.408  15.044   4.535  1.00  0.00           C  \\nATOM    496  H   ILE A  34      10.575  17.184   7.808  1.00  0.00           H  \\nATOM    497  HA  ILE A  34      12.340  15.340   6.276  1.00  0.00           H  \\nATOM    498  HB  ILE A  34      11.419  18.150   5.549  1.00  0.00           H  \\nATOM    499 1HG1 ILE A  34       9.534  16.610   5.638  1.00  0.00           H  \\nATOM    500 2HG1 ILE A  34       9.940  17.014   3.962  1.00  0.00           H  \\nATOM    501 1HG2 ILE A  34      12.271  17.515   3.312  1.00  0.00           H  \\nATOM    502 2HG2 ILE A  34      13.572  17.450   4.517  1.00  0.00           H  \\nATOM    503 3HG2 ILE A  34      12.804  15.944   3.960  1.00  0.00           H  \\nATOM    504 1HD1 ILE A  34       9.459  14.654   4.212  1.00  0.00           H  \\nATOM    505 2HD1 ILE A  34      11.141  14.905   3.740  1.00  0.00           H  \\nATOM    506 3HD1 ILE A  34      10.735  14.522   5.426  1.00  0.00           H  \\nATOM    507  N   ILE A  35      14.519  16.337   6.816  1.00  0.00           N  \\nATOM    508  CA  ILE A  35      15.810  16.851   7.227  1.00  0.00           C  \\nATOM    509  C   ILE A  35      16.700  16.975   5.976  1.00  0.00           C  \\nATOM    510  O   ILE A  35      16.536  16.204   5.019  1.00  0.00           O  \\nATOM    511  CB  ILE A  35      16.462  15.897   8.261  1.00  0.00           C  \\nATOM    512  CG1 ILE A  35      16.719  14.555   7.625  1.00  0.00           C  \\nATOM    513  CG2 ILE A  35      15.529  15.722   9.455  1.00  0.00           C  \\nATOM    514  CD1 ILE A  35      17.534  13.612   8.455  1.00  0.00           C  \\nATOM    515  H   ILE A  35      14.454  15.435   6.349  1.00  0.00           H  \\nATOM    516  HA  ILE A  35      15.652  17.821   7.684  1.00  0.00           H  \\nATOM    517  HB  ILE A  35      17.411  16.292   8.600  1.00  0.00           H  \\nATOM    518 1HG1 ILE A  35      15.765  14.081   7.419  1.00  0.00           H  \\nATOM    519 2HG1 ILE A  35      17.240  14.726   6.688  1.00  0.00           H  \\nATOM    520 1HG2 ILE A  35      15.981  15.048  10.173  1.00  0.00           H  \\nATOM    521 2HG2 ILE A  35      15.348  16.688   9.924  1.00  0.00           H  \\nATOM    522 3HG2 ILE A  35      14.583  15.301   9.117  1.00  0.00           H  \\nATOM    523 1HD1 ILE A  35      17.675  12.673   7.923  1.00  0.00           H  \\nATOM    524 2HD1 ILE A  35      18.505  14.058   8.660  1.00  0.00           H  \\nATOM    525 3HD1 ILE A  35      17.017  13.428   9.374  1.00  0.00           H  \\nATOM    526  N   PRO A  36      17.627  17.956   5.935  1.00  0.00           N  \\nATOM    527  CA  PRO A  36      18.580  18.198   4.856  1.00  0.00           C  \\nATOM    528  C   PRO A  36      19.686  17.150   4.738  1.00  0.00           C  \\nATOM    529  O   PRO A  36      20.373  17.080   3.718  1.00  0.00           O  \\nATOM    530  CB  PRO A  36      19.165  19.565   5.235  1.00  0.00           C  \\nATOM    531  CG  PRO A  36      19.011  19.655   6.735  1.00  0.00           C  \\nATOM    532  CD  PRO A  36      17.718  18.937   7.042  1.00  0.00           C  \\nATOM    533  HA  PRO A  36      18.027  18.268   3.908  1.00  0.00           H  \\nATOM    534 1HB  PRO A  36      20.217  19.616   4.915  1.00  0.00           H  \\nATOM    535 2HB  PRO A  36      18.630  20.362   4.700  1.00  0.00           H  \\nATOM    536 1HG  PRO A  36      19.879  19.190   7.227  1.00  0.00           H  \\nATOM    537 2HG  PRO A  36      18.997  20.709   7.051  1.00  0.00           H  \\nATOM    538 1HD  PRO A  36      17.816  18.455   8.017  1.00  0.00           H  \\nATOM    539 2HD  PRO A  36      16.873  19.645   7.005  1.00  0.00           H  \\nATOM    540  N   GLY A  37      19.884  16.363   5.791  1.00  0.00           N  \\nATOM    541  CA  GLY A  37      20.938  15.361   5.801  1.00  0.00           C  \\nATOM    542  C   GLY A  37      20.388  13.961   5.586  1.00  0.00           C  \\nATOM    543  O   GLY A  37      19.339  13.776   4.962  1.00  0.00           O  \\nATOM    544  H   GLY A  37      19.299  16.457   6.605  1.00  0.00           H  \\nATOM    545 1HA  GLY A  37      21.670  15.591   5.027  1.00  0.00           H  \\nATOM    546 2HA  GLY A  37      21.460  15.404   6.756  1.00  0.00           H  \\nATOM    547  N   ALA A  38      21.129  12.974   6.090  1.00  0.00           N  \\nATOM    548  CA  ALA A  38      20.737  11.571   6.001  1.00  0.00           C  \\nATOM    549  C   ALA A  38      20.597  10.963   7.397  1.00  0.00           C  \\nATOM    550  O   ALA A  38      20.168   9.818   7.546  1.00  0.00           O  \\nATOM    551  CB  ALA A  38      21.751  10.789   5.189  1.00  0.00           C  \\nATOM    552  H   ALA A  38      21.996  13.207   6.549  1.00  0.00           H  \\nATOM    553  HA  ALA A  38      19.769  11.512   5.511  1.00  0.00           H  \\nATOM    554 1HB  ALA A  38      21.823  11.215   4.188  1.00  0.00           H  \\nATOM    555 2HB  ALA A  38      22.724  10.838   5.679  1.00  0.00           H  \\nATOM    556 3HB  ALA A  38      21.431   9.751   5.119  1.00  0.00           H  \\nATOM    557  N   THR A  39      20.994  11.724   8.415  1.00  0.00           N  \\nATOM    558  CA  THR A  39      20.983  11.244   9.790  1.00  0.00           C  \\nATOM    559  C   THR A  39      19.890  11.957  10.597  1.00  0.00           C  \\nATOM    560  O   THR A  39      19.898  13.187  10.660  1.00  0.00           O  \\nATOM    561  CB  THR A  39      22.346  11.477  10.469  1.00  0.00           C  \\nATOM    562  OG1 THR A  39      23.368  10.764   9.757  1.00  0.00           O  \\nATOM    563  CG2 THR A  39      22.307  10.996  11.911  1.00  0.00           C  \\nATOM    564  H   THR A  39      21.310  12.661   8.232  1.00  0.00           H  \\nATOM    565  HA  THR A  39      20.817  10.176   9.753  1.00  0.00           H  \\nATOM    566  HB  THR A  39      22.578  12.541  10.457  1.00  0.00           H  \\nATOM    567  HG1 THR A  39      23.082   9.857   9.613  1.00  0.00           H  \\nATOM    568 1HG2 THR A  39      23.273  11.171  12.379  1.00  0.00           H  \\nATOM    569 2HG2 THR A  39      21.535  11.541  12.459  1.00  0.00           H  \\nATOM    570 3HG2 THR A  39      22.083   9.934  11.936  1.00  0.00           H  \\nATOM    571  N   PRO A  40      18.925  11.234  11.190  1.00  0.00           N  \\nATOM    572  CA  PRO A  40      17.825  11.780  11.957  1.00  0.00           C  \\nATOM    573  C   PRO A  40      18.265  12.313  13.323  1.00  0.00           C  \\nATOM    574  O   PRO A  40      19.285  11.867  13.849  1.00  0.00           O  \\nATOM    575  CB  PRO A  40      16.923  10.567  12.116  1.00  0.00           C  \\nATOM    576  CG  PRO A  40      17.831   9.383  12.072  1.00  0.00           C  \\nATOM    577  CD  PRO A  40      18.915   9.756  11.100  1.00  0.00           C  \\nATOM    578  HA  PRO A  40      17.347  12.550  11.352  1.00  0.00           H  \\nATOM    579 1HB  PRO A  40      16.415  10.650  13.067  1.00  0.00           H  \\nATOM    580 2HB  PRO A  40      16.163  10.534  11.339  1.00  0.00           H  \\nATOM    581 1HG  PRO A  40      18.213   9.168  13.081  1.00  0.00           H  \\nATOM    582 2HG  PRO A  40      17.256   8.489  11.764  1.00  0.00           H  \\nATOM    583 1HD  PRO A  40      19.845   9.316  11.471  1.00  0.00           H  \\nATOM    584 2HD  PRO A  40      18.660   9.424  10.076  1.00  0.00           H  \\nATOM    585  N   PRO A  41      17.505  13.252  13.918  1.00  0.00           N  \\nATOM    586  CA  PRO A  41      17.631  13.747  15.278  1.00  0.00           C  \\nATOM    587  C   PRO A  41      17.093  12.715  16.260  1.00  0.00           C  \\nATOM    588  O   PRO A  41      16.301  11.858  15.881  1.00  0.00           O  \\nATOM    589  CB  PRO A  41      16.806  15.033  15.249  1.00  0.00           C  \\nATOM    590  CG  PRO A  41      15.758  14.777  14.221  1.00  0.00           C  \\nATOM    591  CD  PRO A  41      16.435  13.918  13.155  1.00  0.00           C  \\nATOM    592  HA  PRO A  41      18.690  13.962  15.483  1.00  0.00           H  \\nATOM    593 1HB  PRO A  41      16.382  15.231  16.244  1.00  0.00           H  \\nATOM    594 2HB  PRO A  41      17.452  15.889  14.999  1.00  0.00           H  \\nATOM    595 1HG  PRO A  41      14.929  14.278  14.704  1.00  0.00           H  \\nATOM    596 2HG  PRO A  41      15.374  15.730  13.828  1.00  0.00           H  \\nATOM    597 1HD  PRO A  41      15.713  13.192  12.776  1.00  0.00           H  \\nATOM    598 2HD  PRO A  41      16.861  14.551  12.363  1.00  0.00           H  \\nATOM    599  N   GLY A  42      17.456  12.845  17.532  1.00  0.00           N  \\nATOM    600  CA  GLY A  42      17.002  11.933  18.591  1.00  0.00           C  \\nATOM    601  C   GLY A  42      15.482  11.781  18.752  1.00  0.00           C  \\nATOM    602  O   GLY A  42      15.010  10.737  19.199  1.00  0.00           O  \\nATOM    603  H   GLY A  42      18.100  13.583  17.780  1.00  0.00           H  \\nATOM    604 1HA  GLY A  42      17.437  10.950  18.407  1.00  0.00           H  \\nATOM    605 2HA  GLY A  42      17.419  12.271  19.539  1.00  0.00           H  \\nATOM    606  N   ASP A  43      14.715  12.804  18.387  1.00  0.00           N  \\nATOM    607  CA  ASP A  43      13.258  12.757  18.530  1.00  0.00           C  \\nATOM    608  C   ASP A  43      12.573  12.058  17.355  1.00  0.00           C  \\nATOM    609  O   ASP A  43      11.394  11.707  17.424  1.00  0.00           O  \\nATOM    610  CB  ASP A  43      12.715  14.177  18.649  1.00  0.00           C  \\nATOM    611  CG  ASP A  43      13.181  14.869  19.914  1.00  0.00           C  \\nATOM    612  OD1 ASP A  43      13.225  14.238  20.943  1.00  0.00           O  \\nATOM    613  OD2 ASP A  43      13.515  16.022  19.834  1.00  0.00           O  \\nATOM    614  H   ASP A  43      15.155  13.633  18.019  1.00  0.00           H  \\nATOM    615  HA  ASP A  43      13.019  12.209  19.441  1.00  0.00           H  \\nATOM    616 1HB  ASP A  43      13.044  14.761  17.793  1.00  0.00           H  \\nATOM    617 2HB  ASP A  43      11.623  14.155  18.631  1.00  0.00           H  \\nATOM    618  N   TYR A  44      13.307  11.925  16.260  1.00  0.00           N  \\nATOM    619  CA  TYR A  44      12.835  11.340  15.020  1.00  0.00           C  \\nATOM    620  C   TYR A  44      13.868  10.331  14.562  1.00  0.00           C  \\nATOM    621  O   TYR A  44      14.138  10.191  13.378  1.00  0.00           O  \\nATOM    622  CB  TYR A  44      12.554  12.414  13.978  1.00  0.00           C  \\nATOM    623  CG  TYR A  44      11.478  13.338  14.397  1.00  0.00           C  \\nATOM    624  CD1 TYR A  44      11.765  14.522  15.047  1.00  0.00           C  \\nATOM    625  CD2 TYR A  44      10.189  12.991  14.127  1.00  0.00           C  \\nATOM    626  CE1 TYR A  44      10.742  15.354  15.435  1.00  0.00           C  \\nATOM    627  CE2 TYR A  44       9.156  13.808  14.507  1.00  0.00           C  \\nATOM    628  CZ  TYR A  44       9.430  14.995  15.165  1.00  0.00           C  \\nATOM    629  OH  TYR A  44       8.404  15.829  15.552  1.00  0.00           O  \\nATOM    630  H   TYR A  44      14.289  12.189  16.286  1.00  0.00           H  \\nATOM    631  HA  TYR A  44      11.910  10.809  15.219  1.00  0.00           H  \\nATOM    632 1HB  TYR A  44      13.436  12.975  13.756  1.00  0.00           H  \\nATOM    633 2HB  TYR A  44      12.232  11.940  13.072  1.00  0.00           H  \\nATOM    634  HD1 TYR A  44      12.791  14.798  15.258  1.00  0.00           H  \\nATOM    635  HD2 TYR A  44       9.995  12.067  13.611  1.00  0.00           H  \\nATOM    636  HE1 TYR A  44      10.963  16.288  15.951  1.00  0.00           H  \\nATOM    637  HE2 TYR A  44       8.125  13.523  14.287  1.00  0.00           H  \\nATOM    638  HH  TYR A  44       7.565  15.461  15.261  1.00  0.00           H  \\nATOM    639  N   ALA A  45      14.462   9.650  15.530  1.00  0.00           N  \\nATOM    640  CA  ALA A  45      15.554   8.695  15.353  1.00  0.00           C  \\nATOM    641  C   ALA A  45      15.270   7.484  14.448  1.00  0.00           C  \\nATOM    642  O   ALA A  45      16.221   6.813  14.034  1.00  0.00           O  \\nATOM    643  CB  ALA A  45      15.967   8.173  16.712  1.00  0.00           C  \\nATOM    644  H   ALA A  45      14.152   9.835  16.475  1.00  0.00           H  \\nATOM    645  HA  ALA A  45      16.389   9.244  14.924  1.00  0.00           H  \\nATOM    646 1HB  ALA A  45      15.136   7.626  17.154  1.00  0.00           H  \\nATOM    647 2HB  ALA A  45      16.823   7.510  16.601  1.00  0.00           H  \\nATOM    648 3HB  ALA A  45      16.234   9.009  17.353  1.00  0.00           H  \\nATOM    649  N   ASN A  46      14.002   7.127  14.237  1.00  0.00           N  \\nATOM    650  CA  ASN A  46      13.667   5.941  13.464  1.00  0.00           C  \\nATOM    651  C   ASN A  46      13.528   6.245  11.977  1.00  0.00           C  \\nATOM    652  O   ASN A  46      13.710   7.387  11.554  1.00  0.00           O  \\nATOM    653  OXT ASN A  46      12.938   5.436  11.266  1.00  0.00           O  \\nATOM    654  CB  ASN A  46      12.413   5.294  14.026  1.00  0.00           C  \\nATOM    655  CG  ASN A  46      12.660   4.675  15.393  1.00  0.00           C  \\nATOM    656  OD1 ASN A  46      13.450   3.734  15.502  1.00  0.00           O  \\nATOM    657  ND2 ASN A  46      12.025   5.180  16.425  1.00  0.00           N  \\nATOM    658  H   ASN A  46      13.238   7.723  14.553  1.00  0.00           H  \\nATOM    659  HA  ASN A  46      14.485   5.227  13.566  1.00  0.00           H  \\nATOM    660 1HB  ASN A  46      11.627   6.043  14.113  1.00  0.00           H  \\nATOM    661 2HB  ASN A  46      12.057   4.521  13.342  1.00  0.00           H  \\nATOM    662 1HD2 ASN A  46      12.173   4.800  17.339  1.00  0.00           H  \\nATOM    663 2HD2 ASN A  46      11.387   5.949  16.312  1.00  0.00           H  \\nTER                                                                             \\n# All scores below are weighted scores, not raw scores.\\n#BEGIN_POSE_ENERGIES_TABLE \\nlabel fa_atr fa_rep fa_sol fa_intra_rep fa_intra_sol_xover4 lk_ball_wtd fa_elec hbond_sr_bb hbond_lr_bb hbond_bb_sc hbond_sc dslf_fa13 omega fa_dun p_aa_pp yhh_planarity ref rama_prepro cart_bonded total\\nweights 1 0.55 1 0.005 1 1 1 1 1 1 1 1.25 0.4 0.7 0.6 0.625 1 0.45 0.5 NA\\npose -239.681 27.9079 136.559 0.59997 5.62287 -9.99474 -58.6334 -17.569 -7.84279 -2.9582 -0.78388 0 0.023 24.7556 -10.6105 0.00105 27.0504 -2.94564 22.8353 -105.664\\nTHR:NtermProteinFull_1 -4.79798 0.60793 4.13562 0.00741 0.04888 -0.11143 -4.70664 0 0 -0.47959 0 0 0.04264 0.23002 0 0 1.15175 0 0.63112 -3.24026\\nTHR_2 -6.39905 1.08915 1.24092 0.00838 0.05099 -0.60355 -0.79124 0 0 0 0 0 -0.02779 0.11746 -0.1214 0 1.15175 0.04595 0.78322 -3.4552\\nALA_3 -6.30334 0.8635 1.6266 0.0013 0 0.04833 -2.70902 0 0 0 0 0 0.21287 0 0.22575 0 1.32468 0.21682 0.45506 -4.03745\\nPHE_4 -12.2748 1.93026 4.62065 0.03516 0.36123 -0.62862 -2.20739 0 0 0 0 0 0.05691 1.56711 -0.30546 0 1.21829 0.1879 1.40681 -4.03195\\nPRO_5 -4.54223 0.56048 1.52888 0.00269 0.04856 -0.06109 -0.09651 0 0 0 0 0 -0.03956 0.09224 -0.08767 0 -1.64321 -0.02588 1.24109 -3.02222\\nSER_6 -5.24858 0.38989 4.61534 0.00158 0.07909 -0.15974 -1.4134 0 0 -0.99951 0 0 0.02908 0.08866 -0.51955 0 -0.28969 -0.18665 0.31622 -3.29725\\nILE_7 -4.72735 0.22409 3.74211 0.03085 0.07811 -0.35474 -1.2327 0 0 0 0 0 -0.03234 0.20264 -0.23094 0 2.30374 -0.21123 0.28451 0.07675\\nVAL_8 -4.26499 0.27565 3.69259 0.02377 0.05467 -0.38457 -0.72467 0 0 0 0 0 -0.0095 -0.0228 -0.2447 0 2.64269 -0.13365 0.18987 1.09433\\nALA_9 -6.67405 0.55093 3.58245 0.00137 0 -0.19011 -1.42914 0 0 -0.99951 0 0 0.11897 0 -0.26765 0 1.32468 -0.25674 0.19895 -4.03984\\nARG_10 -7.65705 0.65417 6.31609 0.01601 0.48214 -0.50806 -2.51068 0 0 0 0 0 -0.01533 2.00846 -0.05239 0 -0.09474 -0.31608 0.17364 -1.50381\\nSER_11 -4.58252 0.28401 4.89523 0.00137 0.02305 -0.2291 -2.40015 0 0 0 0 0 0.02222 0.41317 0.29757 0 -0.28969 -0.13957 0.11247 -1.59195\\nASN_12 -6.09048 0.35179 4.80044 0.00531 0.24678 -0.50172 -2.36605 0 0 0 0 0 -0.03288 1.0132 0.3692 0 -1.34026 0.0101 0.12799 -3.40658\\nPHE_13 -11.6626 2.20987 4.43707 0.02526 0.22704 0.09823 -2.39608 0 0 0 0 0 -0.00964 1.6896 -0.40919 0 1.21829 0.0183 0.31151 -4.24234\\nASN_14 -4.66564 0.16522 4.22592 0.00714 0.26372 -0.41959 -1.07191 0 0 0 0 0 0.08978 1.07143 0.24519 0 -1.34026 -0.01213 0.31899 -1.12214\\nVAL_15 -4.85903 0.45389 3.23337 0.02144 0.05208 -0.07805 -1.50959 0 0 0 0 0 -0.02716 -0.02537 -0.37638 0 2.64269 -0.01645 0.15515 -0.33344\\nMET_16 -7.41252 1.05483 2.70399 0.02917 0.10701 -0.30827 -1.31194 0 0 0 0 0 0.02655 1.1676 -0.14045 0 1.65735 -0.14825 0.38368 -2.19125\\nARG_17 -7.80978 0.85075 5.77442 0.01822 0.7668 -0.2075 -3.34978 0 0 0 -0.39194 0 -0.02145 2.46001 -0.04389 0 -0.09474 -0.38363 0.67463 -1.75786\\nLEU_18 -3.49255 0.74052 2.38712 0.02547 0.09136 0.04185 -0.45351 0 0 0 0 0 0.08137 0.37468 -0.23355 0 1.66147 0.63211 1.60363 3.45998\\nPRO_19 -2.8163 0.77704 1.86779 0.00358 0.06274 -0.17089 -0.03888 0 0 0 0 0 -0.0022 0.03408 0.08172 0 -1.64321 1.09721 1.53491 0.78758\\nGLY_20 -1.59292 0.1145 1.79118 7e-05 0 -0.19162 -0.82887 0 0 0 0 0 -0.15114 0 -1.47729 0 0.79816 -0.08469 0.4502 -1.17242\\nTHR_21 -4.56763 0.51111 2.4891 0.00746 0.05044 -0.22961 -0.95018 0 0 0 0 0 -0.04231 0.07541 0.05806 0 1.15175 -0.47811 0.23404 -1.69045\\nSER_22 -3.73858 0.31765 2.86265 0.00139 0.02399 -0.12762 -1.22401 0 0 0 0 0 0.06894 0.72925 -0.08403 0 -0.28969 -0.37752 0.15921 -1.67837\\nGLU_23 -5.51906 0.60802 4.80064 0.0063 0.23794 -0.26465 -2.54018 0 0 0 -0.39194 0 -0.01842 2.59976 -0.26247 0 -2.72453 -0.48516 0.30784 -3.64592\\nALA_24 -3.31787 0.28076 2.14888 0.00143 0 -0.22111 -0.32028 0 0 0 0 0 -0.04903 0 -0.26573 0 1.32468 -0.52533 0.27632 -0.66727\\nILE_25 -5.37464 0.4076 3.27691 0.03382 0.06859 -0.19911 -1.44121 0 0 0 0 0 -0.0197 0.10925 -0.4943 0 2.30374 -0.19509 0.24394 -1.28019\\nVAL_26 -9.72156 1.88111 2.56664 0.05711 0.0571 -0.26881 -2.04992 0 0 0 0 0 -0.05866 0.12725 -0.18584 0 2.64269 -0.05295 0.67004 -4.33579\\nALA_27 -7.04065 0.52963 4.16336 0.00159 0 -0.25099 -3.14613 0 0 0 0 0 -0.03622 0 -0.2653 0 1.32468 -0.23788 0.77287 -4.18505\\nTHR_28 -3.71605 0.33337 4.1083 0.00639 0.05256 -0.29067 -1.16545 0 0 0 0 0 0.21134 0.27754 0.12474 0 1.15175 -0.17233 0.4994 1.4209\\nTYR_29 -5.53987 0.61722 2.44116 0.02379 0.32507 -0.19533 -0.19117 0 0 0 0 0 0.01248 1.44884 -0.3283 9e-05 0.58223 -0.04814 0.33743 -0.5145\\nTHR_30 -7.1055 0.64172 3.61121 0.01029 0.06735 -0.05932 -1.48751 0 0 0 0 0 -0.0325 0.07636 -0.17353 0 1.15175 -0.23161 0.36079 -3.1705\\nGLY_31 -2.50353 0.09509 2.47581 7e-05 0 -0.18295 -0.80438 0 0 0 0 0 -0.14799 0 -1.45282 0 0.79816 -0.52026 0.29643 -1.94637\\nGLY_32 -3.86213 0.28419 2.09185 0.00012 0 -0.17529 -1.0729 0 0 0 0 0 -0.10239 0 -0.71375 0 0.79816 -0.27498 0.1054 -2.92173\\nILE_33 -6.82019 0.69331 1.51806 0.04351 0.13497 -0.08665 -1.82862 0 0 0 0 0 -0.00034 0.05471 0.00698 0 2.30374 -0.10673 0.19412 -3.89314\\nILE_34 -5.18863 0.52976 0.64377 0.04611 0.08526 -0.46831 -0.47882 0 0 0 0 0 -0.0499 0.86321 -0.66459 0 2.30374 -0.36471 0.22339 -2.51972\\nILE_35 -7.31409 0.90159 2.32237 0.03629 0.13438 -0.21033 -0.94397 0 0 0 0 0 -0.04555 0.31837 -0.25504 0 2.30374 -0.09405 0.70919 -2.1371\\nPRO_36 -1.2947 0.33064 0.56957 0.00269 0.04736 -0.02819 0.32108 0 0 0 0 0 -0.05022 0.10708 -0.15972 0 -1.64321 0.06412 0.78758 -0.94592\\nGLY_37 -2.32084 0.25487 2.01547 6e-05 0 -0.11421 -0.61489 0 0 -0.47959 0 0 0.0168 0 -1.12306 0 0.79816 0.07419 0.1763 -1.31673\\nALA_38 -1.76735 0.11067 0.86479 0.00161 0 -0.21834 0.2358 0 0 0 0 0 -0.05349 0 0.68551 0 1.32468 0.57294 0.06127 1.81809\\nTHR_39 -2.20251 0.47512 1.04592 0.00993 0.05961 -0.23234 -0.02617 0 0 0 0 0 -0.05202 0.02039 0.01602 0 1.15175 0.51783 0.662 1.44555\\nPRO_40 -4.36009 1.08448 1.2951 0.00384 0.11208 -0.03211 -0.37615 0 0 0 0 0 0.1557 0.07114 -0.88599 0 -1.64321 -0.09628 1.2631 -3.4084\\nPRO_41 -4.63389 0.63915 2.47373 0.00373 0.11431 0.02771 -0.69746 0 0 0 0 0 0.08613 0.04043 -0.86656 0 -1.64321 -0.29746 0.7624 -3.99099\\nGLY_42 -1.56787 0.08378 1.31747 0.00011 0 -0.19545 0.08907 0 0 0 0 0 -0.14491 0 0.40313 0 0.79816 0.00639 0.29984 1.08971\\nASP_43 -2.69948 0.19853 1.86724 0.00404 0.32211 -0.2322 0.08917 0 0 0 0 0 -0.03526 1.94329 -0.26492 0 -2.14574 -0.0499 0.2981 -0.70503\\nTYR_44 -10.2549 1.47306 3.21825 0.02508 0.30454 -0.1777 -1.15412 0 0 0 0 0 0.11381 1.5232 -0.00916 0.00096 0.58223 0.24552 0.4482 -3.66101\\nALA_45 -3.2744 0.11016 2.50247 0.00203 0 -0.21655 -0.39047 0 0 0 0 0 -0.01468 0 -0.15877 0 1.32468 0.14911 0.75518 0.78876\\nASN:CtermProteinFull_46 -6.10153 0.36685 6.65043 0.00563 0.38095 -0.42435 -2.91642 0 0 0 0 0 0 1.88789 0 0 -1.34026 -0.26068 0.57724 -1.17426\\n#END_POSE_ENERGIES_TABLE \\n\\n\",\"pdb\");\n\tviewer_16234200519329896.setStyle({\"cartoon\": {\"color\": \"spectrum\"}, \"stick\": {\"colorscheme\": \"lightgreyCarbon\", \"radius\": 0.25}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.501930166384962, \"y\": 12.609923315803297, \"z\": 3.948209828627384}, \"end\": {\"x\": 19.33886416435858, \"y\": 13.775721176988824, \"z\": 4.962085272711633}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 16.00563702599574, \"y\": 14.477987473439045, \"z\": 3.7037402242218587}, \"end\": {\"x\": 16.535533321636706, \"y\": 16.20405075041262, \"z\": 5.019148295057891}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 14.453818090014181, \"y\": 15.434611231043343, \"z\": 6.349324753819337}, \"end\": {\"x\": 14.179465000973828, \"y\": 13.778989413197973, \"z\": 5.532955025358115}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.351100348025618, \"y\": 12.598023300746231, \"z\": 8.527723408668292}, \"end\": {\"x\": 11.547272748069226, \"y\": 14.218615922922377, \"z\": 8.164396736622226}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 10.096697320524056, \"y\": 13.295568434092688, \"z\": 10.072931479257942}, \"end\": {\"x\": 10.80342294160081, \"y\": 11.466100909156166, \"z\": 10.49330002896156}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.59549519498949, \"y\": 9.008544830703977, \"z\": 11.455433196786293}, \"end\": {\"x\": 13.710100834998277, \"y\": 7.3870436072749515, \"z\": 11.554301851447404}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.238356733146379, \"y\": 7.723051866041844, \"z\": 14.552938624809517}, \"end\": {\"x\": 11.491047492098945, \"y\": 8.64602251061033, \"z\": 14.096052446841924}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 8.197173327953104, \"y\": 4.6332944015185715, \"z\": 10.762786537460512}, \"end\": {\"x\": 8.808120309283371, \"y\": 4.935041958074698, \"z\": 12.722034046986833}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.210948116188301, \"y\": 4.520199195000023, \"z\": 12.650397650346607}, \"end\": {\"x\": 6.302684649659714, \"y\": 5.237340866075713, \"z\": 14.568610348032506}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.546892955767251, \"y\": 2.3711895398141625, \"z\": 9.254443513459494}, \"end\": {\"x\": 8.159013670956877, \"y\": 1.9302431908245523, \"z\": 10.968386151214002}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.156152049209645, \"y\": 3.127992703602933, \"z\": 8.203808250307834}, \"end\": {\"x\": 5.1433693312028534, \"y\": 3.0166457614765503, \"z\": 10.075172492683675}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.210948116188301, \"y\": 4.520199195000023, \"z\": 12.650397650346607}, \"end\": {\"x\": 6.302684649659714, \"y\": 5.237340866075713, \"z\": 14.568610348032506}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.019005799996025, \"y\": 5.0575726863878545, \"z\": 6.577620964912224}, \"end\": {\"x\": 6.280447155272698, \"y\": 5.65314491415779, \"z\": 8.46296558615851}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 8.197173327953104, \"y\": 4.6332944015185715, \"z\": 10.762786537460512}, \"end\": {\"x\": 8.808120309283371, \"y\": 4.935041958074698, \"z\": 12.722034046986833}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.3807705257937215, \"y\": 3.769478936476843, \"z\": 4.530495247582149}, \"end\": {\"x\": 8.24871827373643, \"y\": 3.7219948443813866, \"z\": 6.492939911646457}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.546892955767251, \"y\": 2.3711895398141625, \"z\": 9.254443513459494}, \"end\": {\"x\": 8.159013670956877, \"y\": 1.9302431908245523, \"z\": 10.968386151214002}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.19875848395078, \"y\": 2.6415947617638884, \"z\": 3.2663536851364374}, \"end\": {\"x\": 5.696416772571791, \"y\": 1.8423117566352694, \"z\": 5.360285718561732}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.156152049209645, \"y\": 3.127992703602933, \"z\": 8.203808250307834}, \"end\": {\"x\": 5.1433693312028534, \"y\": 3.0166457614765503, \"z\": 10.075172492683675}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.6934330554077053, \"y\": 4.609270113824714, \"z\": 2.482657476301151}, \"end\": {\"x\": 3.8979135945848835, \"y\": 4.526709865498515, \"z\": 4.30498053288776}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.019005799996025, \"y\": 5.0575726863878545, \"z\": 6.577620964912224}, \"end\": {\"x\": 6.280447155272698, \"y\": 5.65314491415779, \"z\": 8.46296558615851}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.346908707324643, \"y\": 5.529708684458337, \"z\": 0.44429949173639477}, \"end\": {\"x\": 6.213399362919355, \"y\": 5.673851700784857, \"z\": 2.2983121281981505}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.3807705257937215, \"y\": 3.769478936476843, \"z\": 4.530495247582149}, \"end\": {\"x\": 8.24871827373643, \"y\": 3.7219948443813866, \"z\": 6.492939911646457}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.19875848395078, \"y\": 2.6415947617638884, \"z\": 3.2663536851364374}, \"end\": {\"x\": 5.696416772571791, \"y\": 1.8423117566352694, \"z\": 5.360285718561732}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.330573852435505, \"y\": 4.344187216389852, \"z\": -1.7381592484159094}, \"end\": {\"x\": 3.069899911110412, \"y\": 3.4951769499568277, \"z\": -0.38192375119158256}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.6934330554077053, \"y\": 4.609270113824714, \"z\": 2.482657476301151}, \"end\": {\"x\": 3.8979135945848835, \"y\": 4.526709865498515, \"z\": 4.30498053288776}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.346908707324643, \"y\": 5.529708684458337, \"z\": 0.44429949173639477}, \"end\": {\"x\": 6.213399362919355, \"y\": 5.673851700784857, \"z\": 2.2983121281981505}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 11.714263394535879, \"y\": 9.135126074135785, \"z\": -0.8018482166163853}, \"end\": {\"x\": 12.815426590298328, \"y\": 11.07814054174171, \"z\": -0.19954070205677193}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.023462724566651, \"y\": 6.825863514454596, \"z\": -5.211017881721877}, \"end\": {\"x\": 6.135787313971983, \"y\": 6.003921036650385, \"z\": -3.734850396379906}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.330573852435505, \"y\": 4.344187216389852, \"z\": -1.7381592484159094}, \"end\": {\"x\": 3.069899911110412, \"y\": 3.4951769499568277, \"z\": -0.38192375119158256}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.023462724566651, \"y\": 6.825863514454596, \"z\": -5.211017881721877}, \"end\": {\"x\": 6.135787313971983, \"y\": 6.003921036650385, \"z\": -3.734850396379906}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.862379554998098, \"y\": 12.891872348709098, \"z\": 1.6594546089172375}, \"end\": {\"x\": 5.827793289861441, \"y\": 13.112819923791411, \"z\": -0.13165692648088542}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 11.714263394535879, \"y\": 9.135126074135785, \"z\": -0.8018482166163853}, \"end\": {\"x\": 12.815426590298328, \"y\": 11.07814054174171, \"z\": -0.19954070205677193}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.2091876908773855, \"y\": 12.921111980322193, \"z\": 4.021058262322262}, \"end\": {\"x\": 7.514505168757632, \"y\": 13.535036636617333, \"z\": 2.926400951321996}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.436597608212043, \"y\": 14.483327567659321, \"z\": 5.683944002497221}, \"end\": {\"x\": 5.262080139661818, \"y\": 15.904394590696, \"z\": 3.9870451708961543}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.4360859573422253, \"y\": 12.720569482855412, \"z\": 5.860476285426446}, \"end\": {\"x\": 2.745797214264521, \"y\": 13.724906417266116, \"z\": 3.8681656343546997}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.480113706512622, \"y\": 14.958817449329434, \"z\": 5.153903962366791}, \"end\": {\"x\": 2.745797214264521, \"y\": 13.724906417266116, \"z\": 3.8681656343546997}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.862379554998098, \"y\": 12.891872348709098, \"z\": 1.6594546089172375}, \"end\": {\"x\": 5.827793289861441, \"y\": 13.112819923791411, \"z\": -0.13165692648088542}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.3757807705989387, \"y\": 10.641189297552101, \"z\": 7.314445152594147}, \"end\": {\"x\": 4.134825898978926, \"y\": 11.065818669659016, \"z\": 5.499067699985038}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.2091876908773855, \"y\": 12.921111980322193, \"z\": 4.021058262322262}, \"end\": {\"x\": 7.514505168757632, \"y\": 13.535036636617333, \"z\": 2.926400951321996}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.905176103973061, \"y\": 11.950208225084916, \"z\": 9.678030140476736}, \"end\": {\"x\": 4.968165842984307, \"y\": 12.512466509072583, \"z\": 8.134583619562187}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.5906811155751894, \"y\": 11.867479904865563, \"z\": 9.489357147186416}, \"end\": {\"x\": 4.968165842984307, \"y\": 12.512466509072583, \"z\": 8.134583619562187}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.160076179676509, \"y\": 10.626161725774526, \"z\": 7.911375251744044}, \"end\": {\"x\": 4.968165842984307, \"y\": 12.512466509072583, \"z\": 8.134583619562187}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.436597608212043, \"y\": 14.483327567659321, \"z\": 5.683944002497221}, \"end\": {\"x\": 5.262080139661818, \"y\": 15.904394590696, \"z\": 3.9870451708961543}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.480113706512622, \"y\": 14.958817449329434, \"z\": 5.153903962366791}, \"end\": {\"x\": 2.745797214264521, \"y\": 13.724906417266116, \"z\": 3.8681656343546997}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.4360859573422253, \"y\": 12.720569482855412, \"z\": 5.860476285426446}, \"end\": {\"x\": 2.745797214264521, \"y\": 13.724906417266116, \"z\": 3.8681656343546997}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.3757807705989387, \"y\": 10.641189297552101, \"z\": 7.314445152594147}, \"end\": {\"x\": 4.134825898978926, \"y\": 11.065818669659016, \"z\": 5.499067699985038}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.160076179676509, \"y\": 10.626161725774526, \"z\": 7.911375251744044}, \"end\": {\"x\": 4.968165842984307, \"y\": 12.512466509072583, \"z\": 8.134583619562187}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.5906811155751894, \"y\": 11.867479904865563, \"z\": 9.489357147186416}, \"end\": {\"x\": 4.968165842984307, \"y\": 12.512466509072583, \"z\": 8.134583619562187}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.905176103973061, \"y\": 11.950208225084916, \"z\": 9.678030140476736}, \"end\": {\"x\": 4.968165842984307, \"y\": 12.512466509072583, \"z\": 8.134583619562187}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.351100348025618, \"y\": 12.598023300746231, \"z\": 8.527723408668292}, \"end\": {\"x\": 11.547272748069226, \"y\": 14.218615922922377, \"z\": 8.164396736622226}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 10.096697320524056, \"y\": 13.295568434092688, \"z\": 10.072931479257942}, \"end\": {\"x\": 10.80342294160081, \"y\": 11.466100909156166, \"z\": 10.49330002896156}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 16.00563702599574, \"y\": 14.477987473439045, \"z\": 3.7037402242218587}, \"end\": {\"x\": 16.535533321636706, \"y\": 16.20405075041262, \"z\": 5.019148295057891}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 14.453818090014181, \"y\": 15.434611231043343, \"z\": 6.349324753819337}, \"end\": {\"x\": 14.179465000973828, \"y\": 13.778989413197973, \"z\": 5.532955025358115}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.501930166384962, \"y\": 12.609923315803297, \"z\": 3.948209828627384}, \"end\": {\"x\": 19.33886416435858, \"y\": 13.775721176988824, \"z\": 4.962085272711633}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 14.288839344741703, \"y\": 12.188847934629763, \"z\": 16.28592787383205}, \"end\": {\"x\": 16.30057380264282, \"y\": 11.857780663235769, \"z\": 15.88123415008611}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 14.288839344741703, \"y\": 12.188847934629763, \"z\": 16.28592787383205}, \"end\": {\"x\": 16.30057380264282, \"y\": 11.857780663235769, \"z\": 15.88123415008611}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.59549519498949, \"y\": 9.008544830703977, \"z\": 11.455433196786293}, \"end\": {\"x\": 13.710100834998277, \"y\": 7.3870436072749515, \"z\": 11.554301851447404}});\n\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.238356733146379, \"y\": 7.723051866041844, \"z\": 14.552938624809517}, \"end\": {\"x\": 11.491047492098945, \"y\": 8.64602251061033, \"z\": 14.096052446841924}});\n\tviewer_16234200519329896.zoomTo();\nviewer_16234200519329896.render();\n});\n</script>",
            "text/html": [
              "<div id=\"3dmolviewer_16234200519329896\"  style=\"position: relative; width: 1200px; height: 800px\">\n",
              "        <p id=\"3dmolwarning_16234200519329896\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n",
              "        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n",
              "        </div>\n",
              "<script>\n",
              "\n",
              "var loadScriptAsync = function(uri){\n",
              "  return new Promise((resolve, reject) => {\n",
              "    var tag = document.createElement('script');\n",
              "    tag.src = uri;\n",
              "    tag.async = true;\n",
              "    tag.onload = () => {\n",
              "      resolve();\n",
              "    };\n",
              "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
              "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
              "});\n",
              "};\n",
              "\n",
              "if(typeof $3Dmolpromise === 'undefined') {\n",
              "$3Dmolpromise = null;\n",
              "  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n",
              "}\n",
              "\n",
              "var viewer_16234200519329896 = null;\n",
              "var warn = document.getElementById(\"3dmolwarning_16234200519329896\");\n",
              "if(warn) {\n",
              "    warn.parentNode.removeChild(warn);\n",
              "}\n",
              "$3Dmolpromise.then(function() {\n",
              "viewer_16234200519329896 = $3Dmol.createViewer($(\"#3dmolviewer_16234200519329896\"),{backgroundColor:\"white\"});\n",
              "viewer_16234200519329896.zoomTo();\n",
              "\tviewer_16234200519329896.addModels(\"HEADER                                            11-JUN-21   XXXX              \\nEXPDTA    THEORETICAL MODEL                                                     \\nREMARK 220                                                                      \\nREMARK 220 EXPERIMENTAL DETAILS                                                 \\nREMARK 220  EXPERIMENT TYPE                : THEORETICAL MODELLING              \\nREMARK 220  DATE OF DATA COLLECTION        : 11-JUN-21                          \\nREMARK 220                                                                      \\nREMARK 220 REMARK: MODEL GENERATED BY ROSETTA                                   \\nREMARK 220  VERSION 2020.50.post.dev+970.commits.3700df14560                    \\nATOM      1  N   THR A   1      15.647  13.699   3.148  1.00  0.00           N  \\nATOM      2  CA  THR A   1      15.488  12.506   3.975  1.00  0.00           C  \\nATOM      3  C   THR A   1      14.363  12.693   4.978  1.00  0.00           C  \\nATOM      4  O   THR A   1      14.179  13.779   5.533  1.00  0.00           O  \\nATOM      5  CB  THR A   1      16.785  12.142   4.735  1.00  0.00           C  \\nATOM      6  OG1 THR A   1      17.839  11.889   3.812  1.00  0.00           O  \\nATOM      7  CG2 THR A   1      16.604  10.909   5.632  1.00  0.00           C  \\nATOM      8 1H   THR A   1      16.291  13.500   2.397  1.00  0.00           H  \\nATOM      9 2H   THR A   1      14.749  13.954   2.756  1.00  0.00           H  \\nATOM     10 3H   THR A   1      16.006  14.478   3.704  1.00  0.00           H  \\nATOM     11  HA  THR A   1      15.225  11.670   3.327  1.00  0.00           H  \\nATOM     12  HB  THR A   1      17.066  12.959   5.351  1.00  0.00           H  \\nATOM     13  HG1 THR A   1      18.502  12.610   3.948  1.00  0.00           H  \\nATOM     14 1HG2 THR A   1      17.539  10.699   6.147  1.00  0.00           H  \\nATOM     15 2HG2 THR A   1      15.836  11.095   6.373  1.00  0.00           H  \\nATOM     16 3HG2 THR A   1      16.325  10.052   5.026  1.00  0.00           H  \\nATOM     17  N   THR A   2      13.565  11.653   5.184  1.00  0.00           N  \\nATOM     18  CA  THR A   2      12.505  11.759   6.176  1.00  0.00           C  \\nATOM     19  C   THR A   2      12.937  11.020   7.437  1.00  0.00           C  \\nATOM     20  O   THR A   2      13.469   9.911   7.359  1.00  0.00           O  \\nATOM     21  CB  THR A   2      11.176  11.194   5.677  1.00  0.00           C  \\nATOM     22  OG1 THR A   2      10.783  11.892   4.501  1.00  0.00           O  \\nATOM     23  CG2 THR A   2      10.104  11.417   6.747  1.00  0.00           C  \\nATOM     24  H   THR A   2      13.711  10.782   4.690  1.00  0.00           H  \\nATOM     25  HA  THR A   2      12.354  12.802   6.428  1.00  0.00           H  \\nATOM     26  HB  THR A   2      11.279  10.136   5.458  1.00  0.00           H  \\nATOM     27  HG1 THR A   2      10.683  12.816   4.701  1.00  0.00           H  \\nATOM     28 1HG2 THR A   2       9.152  11.036   6.400  1.00  0.00           H  \\nATOM     29 2HG2 THR A   2      10.375  10.910   7.663  1.00  0.00           H  \\nATOM     30 3HG2 THR A   2      10.019  12.477   6.947  1.00  0.00           H  \\nATOM     31  N   ALA A   3      12.768  11.671   8.580  1.00  0.00           N  \\nATOM     32  CA  ALA A   3      13.096  11.108   9.887  1.00  0.00           C  \\nATOM     33  C   ALA A   3      11.791  10.725  10.596  1.00  0.00           C  \\nATOM     34  O   ALA A   3      10.803  11.466  10.493  1.00  0.00           O  \\nATOM     35  CB  ALA A   3      13.914  12.124  10.662  1.00  0.00           C  \\nATOM     36  H   ALA A   3      12.351  12.598   8.528  1.00  0.00           H  \\nATOM     37  HA  ALA A   3      13.681  10.196   9.750  1.00  0.00           H  \\nATOM     38 1HB  ALA A   3      14.171  11.731  11.611  1.00  0.00           H  \\nATOM     39 2HB  ALA A   3      14.822  12.353  10.107  1.00  0.00           H  \\nATOM     40 3HB  ALA A   3      13.349  13.011  10.792  1.00  0.00           H  \\nATOM     41  N   PHE A   4      11.767   9.610  11.354  1.00  0.00           N  \\nATOM     42  CA  PHE A   4      10.499   9.194  11.963  1.00  0.00           C  \\nATOM     43  C   PHE A   4      10.493   9.036  13.489  1.00  0.00           C  \\nATOM     44  O   PHE A   4      11.491   8.646  14.096  1.00  0.00           O  \\nATOM     45  CB  PHE A   4      10.118   7.851  11.368  1.00  0.00           C  \\nATOM     46  CG  PHE A   4      10.012   7.928   9.943  1.00  0.00           C  \\nATOM     47  CD1 PHE A   4      11.123   7.631   9.200  1.00  0.00           C  \\nATOM     48  CD2 PHE A   4       8.869   8.334   9.317  1.00  0.00           C  \\nATOM     49  CE1 PHE A   4      11.111   7.719   7.865  1.00  0.00           C  \\nATOM     50  CE2 PHE A   4       8.852   8.427   7.973  1.00  0.00           C  \\nATOM     51  CZ  PHE A   4       9.996   8.110   7.248  1.00  0.00           C  \\nATOM     52  H   PHE A   4      12.595   9.009  11.455  1.00  0.00           H  \\nATOM     53  HA  PHE A   4       9.762   9.932  11.671  1.00  0.00           H  \\nATOM     54 1HB  PHE A   4      10.877   7.111  11.610  1.00  0.00           H  \\nATOM     55 2HB  PHE A   4       9.200   7.496  11.765  1.00  0.00           H  \\nATOM     56  HD1 PHE A   4      12.036   7.321   9.718  1.00  0.00           H  \\nATOM     57  HD2 PHE A   4       7.985   8.589   9.898  1.00  0.00           H  \\nATOM     58  HE1 PHE A   4      12.009   7.485   7.291  1.00  0.00           H  \\nATOM     59  HE2 PHE A   4       7.947   8.760   7.469  1.00  0.00           H  \\nATOM     60  HZ  PHE A   4      10.016   8.188   6.188  1.00  0.00           H  \\nATOM     61  N   PRO A   5       9.339   9.273  14.148  1.00  0.00           N  \\nATOM     62  CA  PRO A   5       9.126   9.049  15.565  1.00  0.00           C  \\nATOM     63  C   PRO A   5       9.049   7.556  15.899  1.00  0.00           C  \\nATOM     64  O   PRO A   5       9.195   7.158  17.056  1.00  0.00           O  \\nATOM     65  CB  PRO A   5       7.779   9.737  15.808  1.00  0.00           C  \\nATOM     66  CG  PRO A   5       7.059   9.660  14.475  1.00  0.00           C  \\nATOM     67  CD  PRO A   5       8.132   9.794  13.439  1.00  0.00           C  \\nATOM     68  HA  PRO A   5       9.930   9.538  16.137  1.00  0.00           H  \\nATOM     69 1HB  PRO A   5       7.238   9.222  16.613  1.00  0.00           H  \\nATOM     70 2HB  PRO A   5       7.945  10.774  16.142  1.00  0.00           H  \\nATOM     71 1HG  PRO A   5       6.513   8.706  14.394  1.00  0.00           H  \\nATOM     72 2HG  PRO A   5       6.305  10.464  14.398  1.00  0.00           H  \\nATOM     73 1HD  PRO A   5       7.850   9.192  12.574  1.00  0.00           H  \\nATOM     74 2HD  PRO A   5       8.236  10.838  13.196  1.00  0.00           H  \\nATOM     75  N   SER A   6       8.820   6.723  14.880  1.00  0.00           N  \\nATOM     76  CA  SER A   6       8.676   5.290  15.093  1.00  0.00           C  \\nATOM     77  C   SER A   6       8.954   4.462  13.851  1.00  0.00           C  \\nATOM     78  O   SER A   6       8.808   4.935  12.722  1.00  0.00           O  \\nATOM     79  CB  SER A   6       7.269   4.976  15.570  1.00  0.00           C  \\nATOM     80  OG  SER A   6       6.303   5.237  14.569  1.00  0.00           O  \\nATOM     81  H   SER A   6       8.723   7.089  13.950  1.00  0.00           H  \\nATOM     82  HA  SER A   6       9.378   4.986  15.867  1.00  0.00           H  \\nATOM     83 1HB  SER A   6       7.208   3.932  15.874  1.00  0.00           H  \\nATOM     84 2HB  SER A   6       7.050   5.581  16.449  1.00  0.00           H  \\nATOM     85  HG  SER A   6       5.454   5.274  15.032  1.00  0.00           H  \\nATOM     86  N   ILE A   7       9.218   3.181  14.077  1.00  0.00           N  \\nATOM     87  CA  ILE A   7       9.383   2.207  13.009  1.00  0.00           C  \\nATOM     88  C   ILE A   7       8.113   2.079  12.192  1.00  0.00           C  \\nATOM     89  O   ILE A   7       8.159   1.930  10.968  1.00  0.00           O  \\nATOM     90  CB  ILE A   7       9.818   0.852  13.557  1.00  0.00           C  \\nATOM     91  CG1 ILE A   7      11.253   0.981  14.087  1.00  0.00           C  \\nATOM     92  CG2 ILE A   7       9.697  -0.200  12.478  1.00  0.00           C  \\nATOM     93  CD1 ILE A   7      11.719  -0.195  14.888  1.00  0.00           C  \\nATOM     94  H   ILE A   7       9.326   2.872  15.034  1.00  0.00           H  \\nATOM     95  HA  ILE A   7      10.175   2.560  12.348  1.00  0.00           H  \\nATOM     96  HB  ILE A   7       9.184   0.576  14.398  1.00  0.00           H  \\nATOM     97 1HG1 ILE A   7      11.927   1.116  13.242  1.00  0.00           H  \\nATOM     98 2HG1 ILE A   7      11.315   1.868  14.718  1.00  0.00           H  \\nATOM     99 1HG2 ILE A   7      10.008  -1.166  12.874  1.00  0.00           H  \\nATOM    100 2HG2 ILE A   7       8.661  -0.262  12.145  1.00  0.00           H  \\nATOM    101 3HG2 ILE A   7      10.327   0.070  11.641  1.00  0.00           H  \\nATOM    102 1HD1 ILE A   7      12.742  -0.019  15.225  1.00  0.00           H  \\nATOM    103 2HD1 ILE A   7      11.069  -0.326  15.753  1.00  0.00           H  \\nATOM    104 3HD1 ILE A   7      11.689  -1.091  14.271  1.00  0.00           H  \\nATOM    105  N   VAL A   8       6.971   2.100  12.869  1.00  0.00           N  \\nATOM    106  CA  VAL A   8       5.707   2.008  12.168  1.00  0.00           C  \\nATOM    107  C   VAL A   8       5.553   3.190  11.224  1.00  0.00           C  \\nATOM    108  O   VAL A   8       5.143   3.017  10.075  1.00  0.00           O  \\nATOM    109  CB  VAL A   8       4.538   1.992  13.151  1.00  0.00           C  \\nATOM    110  CG1 VAL A   8       3.232   2.104  12.380  1.00  0.00           C  \\nATOM    111  CG2 VAL A   8       4.595   0.712  13.967  1.00  0.00           C  \\nATOM    112  H   VAL A   8       6.988   2.209  13.871  1.00  0.00           H  \\nATOM    113  HA  VAL A   8       5.693   1.085  11.589  1.00  0.00           H  \\nATOM    114  HB  VAL A   8       4.607   2.854  13.814  1.00  0.00           H  \\nATOM    115 1HG1 VAL A   8       2.395   2.103  13.077  1.00  0.00           H  \\nATOM    116 2HG1 VAL A   8       3.227   3.033  11.810  1.00  0.00           H  \\nATOM    117 3HG1 VAL A   8       3.136   1.260  11.697  1.00  0.00           H  \\nATOM    118 1HG2 VAL A   8       3.768   0.693  14.676  1.00  0.00           H  \\nATOM    119 2HG2 VAL A   8       4.523  -0.147  13.300  1.00  0.00           H  \\nATOM    120 3HG2 VAL A   8       5.540   0.672  14.510  1.00  0.00           H  \\nATOM    121  N   ALA A   9       5.891   4.398  11.689  1.00  0.00           N  \\nATOM    122  CA  ALA A   9       5.799   5.566  10.822  1.00  0.00           C  \\nATOM    123  C   ALA A   9       6.696   5.377   9.595  1.00  0.00           C  \\nATOM    124  O   ALA A   9       6.280   5.653   8.463  1.00  0.00           O  \\nATOM    125  CB  ALA A   9       6.191   6.806  11.609  1.00  0.00           C  \\nATOM    126  H   ALA A   9       6.211   4.520  12.650  1.00  0.00           H  \\nATOM    127  HA  ALA A   9       4.770   5.666  10.477  1.00  0.00           H  \\nATOM    128 1HB  ALA A   9       7.191   6.689  11.957  1.00  0.00           H  \\nATOM    129 2HB  ALA A   9       6.126   7.690  11.005  1.00  0.00           H  \\nATOM    130 3HB  ALA A   9       5.529   6.916  12.463  1.00  0.00           H  \\nATOM    131  N   ARG A  10       7.897   4.825   9.805  1.00  0.00           N  \\nATOM    132  CA  ARG A  10       8.807   4.540   8.697  1.00  0.00           C  \\nATOM    133  C   ARG A  10       8.183   3.546   7.714  1.00  0.00           C  \\nATOM    134  O   ARG A  10       8.249   3.722   6.493  1.00  0.00           O  \\nATOM    135  CB  ARG A  10      10.132   3.994   9.201  1.00  0.00           C  \\nATOM    136  CG  ARG A  10      11.166   3.741   8.114  1.00  0.00           C  \\nATOM    137  CD  ARG A  10      12.432   3.174   8.668  1.00  0.00           C  \\nATOM    138  NE  ARG A  10      12.260   1.816   9.187  1.00  0.00           N  \\nATOM    139  CZ  ARG A  10      13.096   1.219  10.064  1.00  0.00           C  \\nATOM    140  NH1 ARG A  10      14.145   1.871  10.517  1.00  0.00           N  \\nATOM    141  NH2 ARG A  10      12.867  -0.023  10.461  1.00  0.00           N  \\nATOM    142  H   ARG A  10       8.197   4.633  10.763  1.00  0.00           H  \\nATOM    143  HA  ARG A  10       9.002   5.466   8.167  1.00  0.00           H  \\nATOM    144 1HB  ARG A  10      10.562   4.690   9.927  1.00  0.00           H  \\nATOM    145 2HB  ARG A  10       9.971   3.057   9.714  1.00  0.00           H  \\nATOM    146 1HG  ARG A  10      10.759   3.036   7.388  1.00  0.00           H  \\nATOM    147 2HG  ARG A  10      11.400   4.681   7.616  1.00  0.00           H  \\nATOM    148 1HD  ARG A  10      13.189   3.147   7.886  1.00  0.00           H  \\nATOM    149 2HD  ARG A  10      12.778   3.806   9.481  1.00  0.00           H  \\nATOM    150  HE  ARG A  10      11.464   1.287   8.858  1.00  0.00           H  \\nATOM    151 1HH1 ARG A  10      14.329   2.818  10.200  1.00  0.00           H  \\nATOM    152 2HH1 ARG A  10      14.773   1.427  11.171  1.00  0.00           H  \\nATOM    153 1HH2 ARG A  10      12.071  -0.530  10.094  1.00  0.00           H  \\nATOM    154 2HH2 ARG A  10      13.496  -0.469  11.113  1.00  0.00           H  \\nATOM    155  N   SER A  11       7.567   2.489   8.242  1.00  0.00           N  \\nATOM    156  CA  SER A  11       6.954   1.486   7.384  1.00  0.00           C  \\nATOM    157  C   SER A  11       5.818   2.100   6.567  1.00  0.00           C  \\nATOM    158  O   SER A  11       5.696   1.842   5.360  1.00  0.00           O  \\nATOM    159  CB  SER A  11       6.432   0.337   8.227  1.00  0.00           C  \\nATOM    160  OG  SER A  11       7.491  -0.344   8.855  1.00  0.00           O  \\nATOM    161  H   SER A  11       7.547   2.371   9.254  1.00  0.00           H  \\nATOM    162  HA  SER A  11       7.710   1.108   6.698  1.00  0.00           H  \\nATOM    163 1HB  SER A  11       5.747   0.722   8.980  1.00  0.00           H  \\nATOM    164 2HB  SER A  11       5.874  -0.350   7.596  1.00  0.00           H  \\nATOM    165  HG  SER A  11       7.883   0.298   9.474  1.00  0.00           H  \\nATOM    166  N   ASN A  12       5.014   2.955   7.208  1.00  0.00           N  \\nATOM    167  CA  ASN A  12       3.918   3.607   6.513  1.00  0.00           C  \\nATOM    168  C   ASN A  12       4.467   4.478   5.395  1.00  0.00           C  \\nATOM    169  O   ASN A  12       3.898   4.527   4.305  1.00  0.00           O  \\nATOM    170  CB  ASN A  12       3.089   4.443   7.472  1.00  0.00           C  \\nATOM    171  CG  ASN A  12       2.223   3.627   8.393  1.00  0.00           C  \\nATOM    172  OD1 ASN A  12       1.939   2.449   8.142  1.00  0.00           O  \\nATOM    173  ND2 ASN A  12       1.780   4.246   9.461  1.00  0.00           N  \\nATOM    174  H   ASN A  12       5.156   3.128   8.204  1.00  0.00           H  \\nATOM    175  HA  ASN A  12       3.285   2.844   6.060  1.00  0.00           H  \\nATOM    176 1HB  ASN A  12       3.761   5.043   8.085  1.00  0.00           H  \\nATOM    177 2HB  ASN A  12       2.461   5.129   6.907  1.00  0.00           H  \\nATOM    178 1HD2 ASN A  12       1.187   3.764  10.106  1.00  0.00           H  \\nATOM    179 2HD2 ASN A  12       2.028   5.199   9.625  1.00  0.00           H  \\nATOM    180  N   PHE A  13       5.603   5.130   5.648  1.00  0.00           N  \\nATOM    181  CA  PHE A  13       6.234   6.006   4.673  1.00  0.00           C  \\nATOM    182  C   PHE A  13       6.537   5.237   3.407  1.00  0.00           C  \\nATOM    183  O   PHE A  13       6.213   5.674   2.298  1.00  0.00           O  \\nATOM    184  CB  PHE A  13       7.544   6.546   5.225  1.00  0.00           C  \\nATOM    185  CG  PHE A  13       8.254   7.538   4.398  1.00  0.00           C  \\nATOM    186  CD1 PHE A  13       7.936   8.874   4.494  1.00  0.00           C  \\nATOM    187  CD2 PHE A  13       9.263   7.158   3.553  1.00  0.00           C  \\nATOM    188  CE1 PHE A  13       8.591   9.813   3.761  1.00  0.00           C  \\nATOM    189  CE2 PHE A  13       9.937   8.097   2.820  1.00  0.00           C  \\nATOM    190  CZ  PHE A  13       9.595   9.426   2.930  1.00  0.00           C  \\nATOM    191  H   PHE A  13       6.019   5.058   6.578  1.00  0.00           H  \\nATOM    192  HA  PHE A  13       5.556   6.820   4.450  1.00  0.00           H  \\nATOM    193 1HB  PHE A  13       7.371   6.952   6.195  1.00  0.00           H  \\nATOM    194 2HB  PHE A  13       8.224   5.735   5.352  1.00  0.00           H  \\nATOM    195  HD1 PHE A  13       7.160   9.174   5.188  1.00  0.00           H  \\nATOM    196  HD2 PHE A  13       9.532   6.099   3.479  1.00  0.00           H  \\nATOM    197  HE1 PHE A  13       8.330  10.874   3.852  1.00  0.00           H  \\nATOM    198  HE2 PHE A  13      10.747   7.793   2.158  1.00  0.00           H  \\nATOM    199  HZ  PHE A  13      10.133  10.171   2.369  1.00  0.00           H  \\nATOM    200  N   ASN A  14       7.153   4.071   3.578  1.00  0.00           N  \\nATOM    201  CA  ASN A  14       7.549   3.257   2.444  1.00  0.00           C  \\nATOM    202  C   ASN A  14       6.343   2.857   1.597  1.00  0.00           C  \\nATOM    203  O   ASN A  14       6.426   2.815   0.369  1.00  0.00           O  \\nATOM    204  CB  ASN A  14       8.314   2.052   2.940  1.00  0.00           C  \\nATOM    205  CG  ASN A  14       9.696   2.431   3.421  1.00  0.00           C  \\nATOM    206  OD1 ASN A  14      10.240   3.475   3.036  1.00  0.00           O  \\nATOM    207  ND2 ASN A  14      10.270   1.605   4.258  1.00  0.00           N  \\nATOM    208  H   ASN A  14       7.381   3.769   4.530  1.00  0.00           H  \\nATOM    209  HA  ASN A  14       8.208   3.851   1.812  1.00  0.00           H  \\nATOM    210 1HB  ASN A  14       7.768   1.591   3.765  1.00  0.00           H  \\nATOM    211 2HB  ASN A  14       8.397   1.314   2.144  1.00  0.00           H  \\nATOM    212 1HD2 ASN A  14      11.184   1.805   4.611  1.00  0.00           H  \\nATOM    213 2HD2 ASN A  14       9.794   0.775   4.545  1.00  0.00           H  \\nATOM    214  N   VAL A  15       5.203   2.620   2.242  1.00  0.00           N  \\nATOM    215  CA  VAL A  15       3.987   2.288   1.508  1.00  0.00           C  \\nATOM    216  C   VAL A  15       3.423   3.527   0.798  1.00  0.00           C  \\nATOM    217  O   VAL A  15       3.070   3.495  -0.382  1.00  0.00           O  \\nATOM    218  CB  VAL A  15       2.929   1.717   2.457  1.00  0.00           C  \\nATOM    219  CG1 VAL A  15       1.625   1.516   1.714  1.00  0.00           C  \\nATOM    220  CG2 VAL A  15       3.438   0.415   3.031  1.00  0.00           C  \\nATOM    221  H   VAL A  15       5.199   2.642   3.266  1.00  0.00           H  \\nATOM    222  HA  VAL A  15       4.231   1.532   0.765  1.00  0.00           H  \\nATOM    223  HB  VAL A  15       2.748   2.427   3.265  1.00  0.00           H  \\nATOM    224 1HG1 VAL A  15       0.876   1.119   2.398  1.00  0.00           H  \\nATOM    225 2HG1 VAL A  15       1.283   2.472   1.317  1.00  0.00           H  \\nATOM    226 3HG1 VAL A  15       1.777   0.814   0.896  1.00  0.00           H  \\nATOM    227 1HG2 VAL A  15       2.699   0.005   3.718  1.00  0.00           H  \\nATOM    228 2HG2 VAL A  15       3.618  -0.292   2.223  1.00  0.00           H  \\nATOM    229 3HG2 VAL A  15       4.369   0.599   3.567  1.00  0.00           H  \\nATOM    230  N   MET A  16       3.375   4.639   1.516  1.00  0.00           N  \\nATOM    231  CA  MET A  16       2.872   5.903   0.998  1.00  0.00           C  \\nATOM    232  C   MET A  16       3.658   6.461  -0.185  1.00  0.00           C  \\nATOM    233  O   MET A  16       3.107   7.197  -1.001  1.00  0.00           O  \\nATOM    234  CB  MET A  16       2.875   6.907   2.108  1.00  0.00           C  \\nATOM    235  CG  MET A  16       1.828   6.770   3.143  1.00  0.00           C  \\nATOM    236  SD  MET A  16       0.210   7.042   2.476  1.00  0.00           S  \\nATOM    237  CE  MET A  16       0.417   8.784   2.039  1.00  0.00           C  \\nATOM    238  H   MET A  16       3.693   4.609   2.483  1.00  0.00           H  \\nATOM    239  HA  MET A  16       1.848   5.744   0.666  1.00  0.00           H  \\nATOM    240 1HB  MET A  16       3.826   6.834   2.612  1.00  0.00           H  \\nATOM    241 2HB  MET A  16       2.782   7.880   1.687  1.00  0.00           H  \\nATOM    242 1HG  MET A  16       1.858   5.777   3.583  1.00  0.00           H  \\nATOM    243 2HG  MET A  16       2.020   7.493   3.913  1.00  0.00           H  \\nATOM    244 1HE  MET A  16      -0.506   9.162   1.603  1.00  0.00           H  \\nATOM    245 2HE  MET A  16       0.661   9.357   2.924  1.00  0.00           H  \\nATOM    246 3HE  MET A  16       1.225   8.882   1.316  1.00  0.00           H  \\nATOM    247  N   ARG A  17       4.934   6.092  -0.300  1.00  0.00           N  \\nATOM    248  CA  ARG A  17       5.775   6.523  -1.414  1.00  0.00           C  \\nATOM    249  C   ARG A  17       5.593   5.671  -2.677  1.00  0.00           C  \\nATOM    250  O   ARG A  17       6.136   6.004  -3.735  1.00  0.00           O  \\nATOM    251  CB  ARG A  17       7.255   6.457  -1.051  1.00  0.00           C  \\nATOM    252  CG  ARG A  17       7.781   7.426   0.001  1.00  0.00           C  \\nATOM    253  CD  ARG A  17       7.827   8.821  -0.493  1.00  0.00           C  \\nATOM    254  NE  ARG A  17       8.612   8.973  -1.732  1.00  0.00           N  \\nATOM    255  CZ  ARG A  17       9.960   9.074  -1.816  1.00  0.00           C  \\nATOM    256  NH1 ARG A  17      10.701   8.988  -0.745  1.00  0.00           N  \\nATOM    257  NH2 ARG A  17      10.537   9.274  -2.990  1.00  0.00           N  \\nATOM    258  H   ARG A  17       5.347   5.530   0.444  1.00  0.00           H  \\nATOM    259  HA  ARG A  17       5.521   7.557  -1.650  1.00  0.00           H  \\nATOM    260 1HB  ARG A  17       7.471   5.452  -0.679  1.00  0.00           H  \\nATOM    261 2HB  ARG A  17       7.848   6.603  -1.952  1.00  0.00           H  \\nATOM    262 1HG  ARG A  17       7.126   7.418   0.866  1.00  0.00           H  \\nATOM    263 2HG  ARG A  17       8.780   7.123   0.302  1.00  0.00           H  \\nATOM    264 1HD  ARG A  17       6.821   9.149  -0.704  1.00  0.00           H  \\nATOM    265 2HD  ARG A  17       8.258   9.468   0.267  1.00  0.00           H  \\nATOM    266  HE  ARG A  17       8.081   9.070  -2.593  1.00  0.00           H  \\nATOM    267 1HH1 ARG A  17      10.269   8.853   0.154  1.00  0.00           H  \\nATOM    268 2HH1 ARG A  17      11.714   9.135  -0.802  1.00  0.00           H  \\nATOM    269 1HH2 ARG A  17       9.974   9.348  -3.824  1.00  0.00           H  \\nATOM    270 2HH2 ARG A  17      11.543   9.357  -3.045  1.00  0.00           H  \\nATOM    271  N   LEU A  18       4.814   4.590  -2.605  1.00  0.00           N  \\nATOM    272  CA  LEU A  18       4.682   3.706  -3.758  1.00  0.00           C  \\nATOM    273  C   LEU A  18       4.174   4.367  -5.059  1.00  0.00           C  \\nATOM    274  O   LEU A  18       4.684   4.021  -6.122  1.00  0.00           O  \\nATOM    275  CB  LEU A  18       3.809   2.497  -3.388  1.00  0.00           C  \\nATOM    276  CG  LEU A  18       4.463   1.516  -2.409  1.00  0.00           C  \\nATOM    277  CD1 LEU A  18       3.440   0.482  -1.963  1.00  0.00           C  \\nATOM    278  CD2 LEU A  18       5.646   0.864  -3.074  1.00  0.00           C  \\nATOM    279  H   LEU A  18       4.331   4.344  -1.738  1.00  0.00           H  \\nATOM    280  HA  LEU A  18       5.677   3.315  -3.969  1.00  0.00           H  \\nATOM    281 1HB  LEU A  18       2.902   2.837  -2.913  1.00  0.00           H  \\nATOM    282 2HB  LEU A  18       3.547   1.954  -4.294  1.00  0.00           H  \\nATOM    283  HG  LEU A  18       4.809   2.056  -1.535  1.00  0.00           H  \\nATOM    284 1HD1 LEU A  18       3.902  -0.205  -1.256  1.00  0.00           H  \\nATOM    285 2HD1 LEU A  18       2.608   0.996  -1.478  1.00  0.00           H  \\nATOM    286 3HD1 LEU A  18       3.076  -0.071  -2.826  1.00  0.00           H  \\nATOM    287 1HD2 LEU A  18       6.118   0.176  -2.374  1.00  0.00           H  \\nATOM    288 2HD2 LEU A  18       5.315   0.318  -3.956  1.00  0.00           H  \\nATOM    289 3HD2 LEU A  18       6.362   1.632  -3.368  1.00  0.00           H  \\nATOM    290  N   PRO A  19       3.192   5.303  -5.057  1.00  0.00           N  \\nATOM    291  CA  PRO A  19       2.719   5.988  -6.243  1.00  0.00           C  \\nATOM    292  C   PRO A  19       3.553   7.224  -6.618  1.00  0.00           C  \\nATOM    293  O   PRO A  19       3.158   7.979  -7.507  1.00  0.00           O  \\nATOM    294  CB  PRO A  19       1.287   6.351  -5.850  1.00  0.00           C  \\nATOM    295  CG  PRO A  19       1.374   6.632  -4.385  1.00  0.00           C  \\nATOM    296  CD  PRO A  19       2.400   5.644  -3.847  1.00  0.00           C  \\nATOM    297  HA  PRO A  19       2.713   5.275  -7.080  1.00  0.00           H  \\nATOM    298 1HB  PRO A  19       0.949   7.223  -6.429  1.00  0.00           H  \\nATOM    299 2HB  PRO A  19       0.606   5.519  -6.092  1.00  0.00           H  \\nATOM    300 1HG  PRO A  19       1.685   7.672  -4.230  1.00  0.00           H  \\nATOM    301 2HG  PRO A  19       0.389   6.523  -3.912  1.00  0.00           H  \\nATOM    302 1HD  PRO A  19       2.971   6.203  -3.147  1.00  0.00           H  \\nATOM    303 2HD  PRO A  19       1.913   4.777  -3.404  1.00  0.00           H  \\nATOM    304  N   GLY A  20       4.685   7.463  -5.932  1.00  0.00           N  \\nATOM    305  CA  GLY A  20       5.496   8.650  -6.213  1.00  0.00           C  \\nATOM    306  C   GLY A  20       5.162   9.901  -5.383  1.00  0.00           C  \\nATOM    307  O   GLY A  20       5.639  10.993  -5.695  1.00  0.00           O  \\nATOM    308  H   GLY A  20       5.023   6.826  -5.211  1.00  0.00           H  \\nATOM    309 1HA  GLY A  20       6.542   8.392  -6.048  1.00  0.00           H  \\nATOM    310 2HA  GLY A  20       5.408   8.892  -7.271  1.00  0.00           H  \\nATOM    311  N   THR A  21       4.359   9.751  -4.334  1.00  0.00           N  \\nATOM    312  CA  THR A  21       3.981  10.875  -3.468  1.00  0.00           C  \\nATOM    313  C   THR A  21       5.235  11.541  -2.877  1.00  0.00           C  \\nATOM    314  O   THR A  21       6.147  10.849  -2.425  1.00  0.00           O  \\nATOM    315  CB  THR A  21       3.047  10.419  -2.325  1.00  0.00           C  \\nATOM    316  OG1 THR A  21       1.872   9.810  -2.868  1.00  0.00           O  \\nATOM    317  CG2 THR A  21       2.615  11.610  -1.484  1.00  0.00           C  \\nATOM    318  H   THR A  21       3.999   8.829  -4.135  1.00  0.00           H  \\nATOM    319  HA  THR A  21       3.454  11.618  -4.069  1.00  0.00           H  \\nATOM    320  HB  THR A  21       3.571   9.695  -1.702  1.00  0.00           H  \\nATOM    321  HG1 THR A  21       1.392   9.361  -2.161  1.00  0.00           H  \\nATOM    322 1HG2 THR A  21       1.960  11.273  -0.682  1.00  0.00           H  \\nATOM    323 2HG2 THR A  21       3.479  12.087  -1.063  1.00  0.00           H  \\nATOM    324 3HG2 THR A  21       2.084  12.325  -2.108  1.00  0.00           H  \\nATOM    325  N   SER A  22       5.290  12.877  -2.905  1.00  0.00           N  \\nATOM    326  CA  SER A  22       6.451  13.619  -2.400  1.00  0.00           C  \\nATOM    327  C   SER A  22       6.744  13.323  -0.933  1.00  0.00           C  \\nATOM    328  O   SER A  22       5.828  13.113  -0.132  1.00  0.00           O  \\nATOM    329  CB  SER A  22       6.242  15.112  -2.555  1.00  0.00           C  \\nATOM    330  OG  SER A  22       6.108  15.468  -3.899  1.00  0.00           O  \\nATOM    331  H   SER A  22       4.512  13.384  -3.301  1.00  0.00           H  \\nATOM    332  HA  SER A  22       7.323  13.322  -2.988  1.00  0.00           H  \\nATOM    333 1HB  SER A  22       5.369  15.421  -2.000  1.00  0.00           H  \\nATOM    334 2HB  SER A  22       7.095  15.635  -2.126  1.00  0.00           H  \\nATOM    335  HG  SER A  22       6.873  15.100  -4.351  1.00  0.00           H  \\nATOM    336  N   GLU A  23       8.030  13.373  -0.570  1.00  0.00           N  \\nATOM    337  CA  GLU A  23       8.465  13.069   0.792  1.00  0.00           C  \\nATOM    338  C   GLU A  23       7.865  13.983   1.841  1.00  0.00           C  \\nATOM    339  O   GLU A  23       7.515  13.535   2.926  1.00  0.00           O  \\nATOM    340  CB  GLU A  23       9.994  13.168   0.884  1.00  0.00           C  \\nATOM    341  CG  GLU A  23      10.696  12.085   0.124  1.00  0.00           C  \\nATOM    342  CD  GLU A  23      12.231  12.094   0.173  1.00  0.00           C  \\nATOM    343  OE1 GLU A  23      12.814  13.075   0.594  1.00  0.00           O  \\nATOM    344  OE2 GLU A  23      12.815  11.078  -0.200  1.00  0.00           O  \\nATOM    345  H   GLU A  23       8.728  13.579  -1.272  1.00  0.00           H  \\nATOM    346  HA  GLU A  23       8.163  12.045   1.022  1.00  0.00           H  \\nATOM    347 1HB  GLU A  23      10.327  14.133   0.504  1.00  0.00           H  \\nATOM    348 2HB  GLU A  23      10.300  13.099   1.929  1.00  0.00           H  \\nATOM    349 1HG  GLU A  23      10.342  11.177   0.523  1.00  0.00           H  \\nATOM    350 2HG  GLU A  23      10.373  12.134  -0.914  1.00  0.00           H  \\nATOM    351  N   ALA A  24       7.729  15.270   1.536  1.00  0.00           N  \\nATOM    352  CA  ALA A  24       7.152  16.187   2.516  1.00  0.00           C  \\nATOM    353  C   ALA A  24       5.712  15.807   2.843  1.00  0.00           C  \\nATOM    354  O   ALA A  24       5.262  15.904   3.987  1.00  0.00           O  \\nATOM    355  CB  ALA A  24       7.210  17.607   1.998  1.00  0.00           C  \\nATOM    356  H   ALA A  24       8.032  15.605   0.634  1.00  0.00           H  \\nATOM    357  HA  ALA A  24       7.729  16.116   3.432  1.00  0.00           H  \\nATOM    358 1HB  ALA A  24       8.246  17.876   1.794  1.00  0.00           H  \\nATOM    359 2HB  ALA A  24       6.625  17.684   1.081  1.00  0.00           H  \\nATOM    360 3HB  ALA A  24       6.799  18.280   2.747  1.00  0.00           H  \\nATOM    361  N   ILE A  25       4.975  15.365   1.827  1.00  0.00           N  \\nATOM    362  CA  ILE A  25       3.579  15.019   2.021  1.00  0.00           C  \\nATOM    363  C   ILE A  25       3.477  13.775   2.864  1.00  0.00           C  \\nATOM    364  O   ILE A  25       2.746  13.725   3.868  1.00  0.00           O  \\nATOM    365  CB  ILE A  25       2.887  14.726   0.683  1.00  0.00           C  \\nATOM    366  CG1 ILE A  25       2.810  15.971  -0.170  1.00  0.00           C  \\nATOM    367  CG2 ILE A  25       1.506  14.136   0.926  1.00  0.00           C  \\nATOM    368  CD1 ILE A  25       2.390  15.671  -1.595  1.00  0.00           C  \\nATOM    369  H   ILE A  25       5.392  15.265   0.912  1.00  0.00           H  \\nATOM    370  HA  ILE A  25       3.074  15.838   2.530  1.00  0.00           H  \\nATOM    371  HB  ILE A  25       3.482  14.016   0.135  1.00  0.00           H  \\nATOM    372 1HG1 ILE A  25       2.101  16.670   0.269  1.00  0.00           H  \\nATOM    373 2HG1 ILE A  25       3.793  16.442  -0.194  1.00  0.00           H  \\nATOM    374 1HG2 ILE A  25       1.036  13.910  -0.028  1.00  0.00           H  \\nATOM    375 2HG2 ILE A  25       1.598  13.222   1.510  1.00  0.00           H  \\nATOM    376 3HG2 ILE A  25       0.895  14.855   1.471  1.00  0.00           H  \\nATOM    377 1HD1 ILE A  25       2.362  16.597  -2.169  1.00  0.00           H  \\nATOM    378 2HD1 ILE A  25       3.098  14.986  -2.048  1.00  0.00           H  \\nATOM    379 3HD1 ILE A  25       1.402  15.216  -1.593  1.00  0.00           H  \\nATOM    380  N   VAL A  26       4.254  12.776   2.473  1.00  0.00           N  \\nATOM    381  CA  VAL A  26       4.197  11.517   3.158  1.00  0.00           C  \\nATOM    382  C   VAL A  26       4.693  11.662   4.579  1.00  0.00           C  \\nATOM    383  O   VAL A  26       4.135  11.066   5.499  1.00  0.00           O  \\nATOM    384  CB  VAL A  26       5.006  10.444   2.474  1.00  0.00           C  \\nATOM    385  CG1 VAL A  26       4.940   9.292   3.328  1.00  0.00           C  \\nATOM    386  CG2 VAL A  26       4.474  10.178   1.123  1.00  0.00           C  \\nATOM    387  H   VAL A  26       4.862  12.892   1.659  1.00  0.00           H  \\nATOM    388  HA  VAL A  26       3.159  11.189   3.181  1.00  0.00           H  \\nATOM    389  HB  VAL A  26       6.052  10.747   2.400  1.00  0.00           H  \\nATOM    390 1HG1 VAL A  26       5.524   8.486   2.884  1.00  0.00           H  \\nATOM    391 2HG1 VAL A  26       5.316   9.551   4.290  1.00  0.00           H  \\nATOM    392 3HG1 VAL A  26       3.916   9.003   3.437  1.00  0.00           H  \\nATOM    393 1HG2 VAL A  26       5.037   9.388   0.653  1.00  0.00           H  \\nATOM    394 2HG2 VAL A  26       3.440   9.897   1.183  1.00  0.00           H  \\nATOM    395 3HG2 VAL A  26       4.569  11.075   0.551  1.00  0.00           H  \\nATOM    396  N   ALA A  27       5.743  12.444   4.787  1.00  0.00           N  \\nATOM    397  CA  ALA A  27       6.242  12.598   6.128  1.00  0.00           C  \\nATOM    398  C   ALA A  27       5.134  13.083   7.049  1.00  0.00           C  \\nATOM    399  O   ALA A  27       4.968  12.512   8.135  1.00  0.00           O  \\nATOM    400  CB  ALA A  27       7.391  13.579   6.145  1.00  0.00           C  \\nATOM    401  H   ALA A  27       6.209  12.921   4.021  1.00  0.00           H  \\nATOM    402  HA  ALA A  27       6.590  11.627   6.479  1.00  0.00           H  \\nATOM    403 1HB  ALA A  27       7.046  14.551   5.787  1.00  0.00           H  \\nATOM    404 2HB  ALA A  27       7.776  13.683   7.154  1.00  0.00           H  \\nATOM    405 3HB  ALA A  27       8.168  13.213   5.491  1.00  0.00           H  \\nATOM    406  N   THR A  28       4.295  14.042   6.599  1.00  0.00           N  \\nATOM    407  CA  THR A  28       3.250  14.514   7.504  1.00  0.00           C  \\nATOM    408  C   THR A  28       2.202  13.431   7.751  1.00  0.00           C  \\nATOM    409  O   THR A  28       1.665  13.332   8.856  1.00  0.00           O  \\nATOM    410  CB  THR A  28       2.568  15.798   6.980  1.00  0.00           C  \\nATOM    411  OG1 THR A  28       1.935  15.557   5.707  1.00  0.00           O  \\nATOM    412  CG2 THR A  28       3.608  16.891   6.830  1.00  0.00           C  \\nATOM    413  H   THR A  28       4.437  14.483   5.684  1.00  0.00           H  \\nATOM    414  HA  THR A  28       3.714  14.756   8.460  1.00  0.00           H  \\nATOM    415  HB  THR A  28       1.813  16.124   7.689  1.00  0.00           H  \\nATOM    416  HG1 THR A  28       2.480  14.959   5.154  1.00  0.00           H  \\nATOM    417 1HG2 THR A  28       3.132  17.801   6.468  1.00  0.00           H  \\nATOM    418 2HG2 THR A  28       4.075  17.082   7.795  1.00  0.00           H  \\nATOM    419 3HG2 THR A  28       4.362  16.575   6.126  1.00  0.00           H  \\nATOM    420  N   TYR A  29       1.975  12.564   6.759  1.00  0.00           N  \\nATOM    421  CA  TYR A  29       1.053  11.434   6.906  1.00  0.00           C  \\nATOM    422  C   TYR A  29       1.501  10.552   8.067  1.00  0.00           C  \\nATOM    423  O   TYR A  29       0.703  10.137   8.911  1.00  0.00           O  \\nATOM    424  CB  TYR A  29       0.988  10.576   5.638  1.00  0.00           C  \\nATOM    425  CG  TYR A  29       0.072   9.382   5.757  1.00  0.00           C  \\nATOM    426  CD1 TYR A  29      -1.285   9.514   5.534  1.00  0.00           C  \\nATOM    427  CD2 TYR A  29       0.600   8.149   6.111  1.00  0.00           C  \\nATOM    428  CE1 TYR A  29      -2.108   8.410   5.646  1.00  0.00           C  \\nATOM    429  CE2 TYR A  29      -0.214   7.050   6.226  1.00  0.00           C  \\nATOM    430  CZ  TYR A  29      -1.566   7.174   5.991  1.00  0.00           C  \\nATOM    431  OH  TYR A  29      -2.386   6.074   6.101  1.00  0.00           O  \\nATOM    432  H   TYR A  29       2.436  12.721   5.860  1.00  0.00           H  \\nATOM    433  HA  TYR A  29       0.060  11.818   7.134  1.00  0.00           H  \\nATOM    434 1HB  TYR A  29       0.660  11.187   4.797  1.00  0.00           H  \\nATOM    435 2HB  TYR A  29       1.959  10.196   5.397  1.00  0.00           H  \\nATOM    436  HD1 TYR A  29      -1.702  10.483   5.264  1.00  0.00           H  \\nATOM    437  HD2 TYR A  29       1.662   8.052   6.292  1.00  0.00           H  \\nATOM    438  HE1 TYR A  29      -3.177   8.508   5.464  1.00  0.00           H  \\nATOM    439  HE2 TYR A  29       0.208   6.084   6.498  1.00  0.00           H  \\nATOM    440  HH  TYR A  29      -3.289   6.329   5.896  1.00  0.00           H  \\nATOM    441  N   THR A  30       2.801  10.243   8.061  1.00  0.00           N  \\nATOM    442  CA  THR A  30       3.420   9.347   9.028  1.00  0.00           C  \\nATOM    443  C   THR A  30       3.776  10.037  10.349  1.00  0.00           C  \\nATOM    444  O   THR A  30       4.029   9.367  11.353  1.00  0.00           O  \\nATOM    445  CB  THR A  30       4.711   8.772   8.436  1.00  0.00           C  \\nATOM    446  OG1 THR A  30       5.644   9.838   8.231  1.00  0.00           O  \\nATOM    447  CG2 THR A  30       4.424   8.113   7.125  1.00  0.00           C  \\nATOM    448  H   THR A  30       3.376  10.641   7.314  1.00  0.00           H  \\nATOM    449  HA  THR A  30       2.725   8.536   9.238  1.00  0.00           H  \\nATOM    450  HB  THR A  30       5.131   8.048   9.093  1.00  0.00           H  \\nATOM    451  HG1 THR A  30       5.160  10.626   7.911  1.00  0.00           H  \\nATOM    452 1HG2 THR A  30       5.335   7.734   6.733  1.00  0.00           H  \\nATOM    453 2HG2 THR A  30       3.726   7.305   7.273  1.00  0.00           H  \\nATOM    454 3HG2 THR A  30       4.004   8.825   6.435  1.00  0.00           H  \\nATOM    455  N   GLY A  31       3.788  11.371  10.352  1.00  0.00           N  \\nATOM    456  CA  GLY A  31       4.147  12.151  11.530  1.00  0.00           C  \\nATOM    457  C   GLY A  31       5.652  12.409  11.637  1.00  0.00           C  \\nATOM    458  O   GLY A  31       6.154  12.740  12.716  1.00  0.00           O  \\nATOM    459  H   GLY A  31       3.591  11.867   9.489  1.00  0.00           H  \\nATOM    460 1HA  GLY A  31       3.620  13.104  11.486  1.00  0.00           H  \\nATOM    461 2HA  GLY A  31       3.799  11.636  12.422  1.00  0.00           H  \\nATOM    462  N   GLY A  32       6.379  12.202  10.541  1.00  0.00           N  \\nATOM    463  CA  GLY A  32       7.819  12.411  10.519  1.00  0.00           C  \\nATOM    464  C   GLY A  32       8.160  13.822  10.076  1.00  0.00           C  \\nATOM    465  O   GLY A  32       7.276  14.668   9.905  1.00  0.00           O  \\nATOM    466  H   GLY A  32       5.905  11.950   9.678  1.00  0.00           H  \\nATOM    467 1HA  GLY A  32       8.240  12.238  11.502  1.00  0.00           H  \\nATOM    468 2HA  GLY A  32       8.282  11.692   9.843  1.00  0.00           H  \\nATOM    469  N   ILE A  33       9.451  14.068   9.902  1.00  0.00           N  \\nATOM    470  CA  ILE A  33       9.948  15.378   9.491  1.00  0.00           C  \\nATOM    471  C   ILE A  33      10.922  15.260   8.343  1.00  0.00           C  \\nATOM    472  O   ILE A  33      11.547  14.219   8.164  1.00  0.00           O  \\nATOM    473  CB  ILE A  33      10.666  16.099  10.639  1.00  0.00           C  \\nATOM    474  CG1 ILE A  33      11.854  15.283  11.033  1.00  0.00           C  \\nATOM    475  CG2 ILE A  33       9.736  16.301  11.816  1.00  0.00           C  \\nATOM    476  CD1 ILE A  33      12.778  15.958  11.979  1.00  0.00           C  \\nATOM    477  H   ILE A  33      10.097  13.296  10.073  1.00  0.00           H  \\nATOM    478  HA  ILE A  33       9.106  15.986   9.162  1.00  0.00           H  \\nATOM    479  HB  ILE A  33      11.030  17.066  10.289  1.00  0.00           H  \\nATOM    480 1HG1 ILE A  33      11.497  14.361  11.496  1.00  0.00           H  \\nATOM    481 2HG1 ILE A  33      12.407  15.028  10.134  1.00  0.00           H  \\nATOM    482 1HG2 ILE A  33      10.267  16.810  12.620  1.00  0.00           H  \\nATOM    483 2HG2 ILE A  33       8.882  16.901  11.508  1.00  0.00           H  \\nATOM    484 3HG2 ILE A  33       9.389  15.343  12.166  1.00  0.00           H  \\nATOM    485 1HD1 ILE A  33      13.602  15.289  12.206  1.00  0.00           H  \\nATOM    486 2HD1 ILE A  33      13.163  16.869  11.518  1.00  0.00           H  \\nATOM    487 3HD1 ILE A  33      12.251  16.213  12.886  1.00  0.00           H  \\nATOM    488  N   ILE A  34      11.118  16.354   7.621  1.00  0.00           N  \\nATOM    489  CA  ILE A  34      12.112  16.364   6.558  1.00  0.00           C  \\nATOM    490  C   ILE A  34      13.400  17.017   7.039  1.00  0.00           C  \\nATOM    491  O   ILE A  34      13.376  18.111   7.608  1.00  0.00           O  \\nATOM    492  CB  ILE A  34      11.597  17.104   5.313  1.00  0.00           C  \\nATOM    493  CG1 ILE A  34      10.282  16.491   4.854  1.00  0.00           C  \\nATOM    494  CG2 ILE A  34      12.635  16.997   4.196  1.00  0.00           C  \\nATOM    495  CD1 ILE A  34      10.408  15.044   4.535  1.00  0.00           C  \\nATOM    496  H   ILE A  34      10.575  17.184   7.808  1.00  0.00           H  \\nATOM    497  HA  ILE A  34      12.340  15.340   6.276  1.00  0.00           H  \\nATOM    498  HB  ILE A  34      11.419  18.150   5.549  1.00  0.00           H  \\nATOM    499 1HG1 ILE A  34       9.534  16.610   5.638  1.00  0.00           H  \\nATOM    500 2HG1 ILE A  34       9.940  17.014   3.962  1.00  0.00           H  \\nATOM    501 1HG2 ILE A  34      12.271  17.515   3.312  1.00  0.00           H  \\nATOM    502 2HG2 ILE A  34      13.572  17.450   4.517  1.00  0.00           H  \\nATOM    503 3HG2 ILE A  34      12.804  15.944   3.960  1.00  0.00           H  \\nATOM    504 1HD1 ILE A  34       9.459  14.654   4.212  1.00  0.00           H  \\nATOM    505 2HD1 ILE A  34      11.141  14.905   3.740  1.00  0.00           H  \\nATOM    506 3HD1 ILE A  34      10.735  14.522   5.426  1.00  0.00           H  \\nATOM    507  N   ILE A  35      14.519  16.337   6.816  1.00  0.00           N  \\nATOM    508  CA  ILE A  35      15.810  16.851   7.227  1.00  0.00           C  \\nATOM    509  C   ILE A  35      16.700  16.975   5.976  1.00  0.00           C  \\nATOM    510  O   ILE A  35      16.536  16.204   5.019  1.00  0.00           O  \\nATOM    511  CB  ILE A  35      16.462  15.897   8.261  1.00  0.00           C  \\nATOM    512  CG1 ILE A  35      16.719  14.555   7.625  1.00  0.00           C  \\nATOM    513  CG2 ILE A  35      15.529  15.722   9.455  1.00  0.00           C  \\nATOM    514  CD1 ILE A  35      17.534  13.612   8.455  1.00  0.00           C  \\nATOM    515  H   ILE A  35      14.454  15.435   6.349  1.00  0.00           H  \\nATOM    516  HA  ILE A  35      15.652  17.821   7.684  1.00  0.00           H  \\nATOM    517  HB  ILE A  35      17.411  16.292   8.600  1.00  0.00           H  \\nATOM    518 1HG1 ILE A  35      15.765  14.081   7.419  1.00  0.00           H  \\nATOM    519 2HG1 ILE A  35      17.240  14.726   6.688  1.00  0.00           H  \\nATOM    520 1HG2 ILE A  35      15.981  15.048  10.173  1.00  0.00           H  \\nATOM    521 2HG2 ILE A  35      15.348  16.688   9.924  1.00  0.00           H  \\nATOM    522 3HG2 ILE A  35      14.583  15.301   9.117  1.00  0.00           H  \\nATOM    523 1HD1 ILE A  35      17.675  12.673   7.923  1.00  0.00           H  \\nATOM    524 2HD1 ILE A  35      18.505  14.058   8.660  1.00  0.00           H  \\nATOM    525 3HD1 ILE A  35      17.017  13.428   9.374  1.00  0.00           H  \\nATOM    526  N   PRO A  36      17.627  17.956   5.935  1.00  0.00           N  \\nATOM    527  CA  PRO A  36      18.580  18.198   4.856  1.00  0.00           C  \\nATOM    528  C   PRO A  36      19.686  17.150   4.738  1.00  0.00           C  \\nATOM    529  O   PRO A  36      20.373  17.080   3.718  1.00  0.00           O  \\nATOM    530  CB  PRO A  36      19.165  19.565   5.235  1.00  0.00           C  \\nATOM    531  CG  PRO A  36      19.011  19.655   6.735  1.00  0.00           C  \\nATOM    532  CD  PRO A  36      17.718  18.937   7.042  1.00  0.00           C  \\nATOM    533  HA  PRO A  36      18.027  18.268   3.908  1.00  0.00           H  \\nATOM    534 1HB  PRO A  36      20.217  19.616   4.915  1.00  0.00           H  \\nATOM    535 2HB  PRO A  36      18.630  20.362   4.700  1.00  0.00           H  \\nATOM    536 1HG  PRO A  36      19.879  19.190   7.227  1.00  0.00           H  \\nATOM    537 2HG  PRO A  36      18.997  20.709   7.051  1.00  0.00           H  \\nATOM    538 1HD  PRO A  36      17.816  18.455   8.017  1.00  0.00           H  \\nATOM    539 2HD  PRO A  36      16.873  19.645   7.005  1.00  0.00           H  \\nATOM    540  N   GLY A  37      19.884  16.363   5.791  1.00  0.00           N  \\nATOM    541  CA  GLY A  37      20.938  15.361   5.801  1.00  0.00           C  \\nATOM    542  C   GLY A  37      20.388  13.961   5.586  1.00  0.00           C  \\nATOM    543  O   GLY A  37      19.339  13.776   4.962  1.00  0.00           O  \\nATOM    544  H   GLY A  37      19.299  16.457   6.605  1.00  0.00           H  \\nATOM    545 1HA  GLY A  37      21.670  15.591   5.027  1.00  0.00           H  \\nATOM    546 2HA  GLY A  37      21.460  15.404   6.756  1.00  0.00           H  \\nATOM    547  N   ALA A  38      21.129  12.974   6.090  1.00  0.00           N  \\nATOM    548  CA  ALA A  38      20.737  11.571   6.001  1.00  0.00           C  \\nATOM    549  C   ALA A  38      20.597  10.963   7.397  1.00  0.00           C  \\nATOM    550  O   ALA A  38      20.168   9.818   7.546  1.00  0.00           O  \\nATOM    551  CB  ALA A  38      21.751  10.789   5.189  1.00  0.00           C  \\nATOM    552  H   ALA A  38      21.996  13.207   6.549  1.00  0.00           H  \\nATOM    553  HA  ALA A  38      19.769  11.512   5.511  1.00  0.00           H  \\nATOM    554 1HB  ALA A  38      21.823  11.215   4.188  1.00  0.00           H  \\nATOM    555 2HB  ALA A  38      22.724  10.838   5.679  1.00  0.00           H  \\nATOM    556 3HB  ALA A  38      21.431   9.751   5.119  1.00  0.00           H  \\nATOM    557  N   THR A  39      20.994  11.724   8.415  1.00  0.00           N  \\nATOM    558  CA  THR A  39      20.983  11.244   9.790  1.00  0.00           C  \\nATOM    559  C   THR A  39      19.890  11.957  10.597  1.00  0.00           C  \\nATOM    560  O   THR A  39      19.898  13.187  10.660  1.00  0.00           O  \\nATOM    561  CB  THR A  39      22.346  11.477  10.469  1.00  0.00           C  \\nATOM    562  OG1 THR A  39      23.368  10.764   9.757  1.00  0.00           O  \\nATOM    563  CG2 THR A  39      22.307  10.996  11.911  1.00  0.00           C  \\nATOM    564  H   THR A  39      21.310  12.661   8.232  1.00  0.00           H  \\nATOM    565  HA  THR A  39      20.817  10.176   9.753  1.00  0.00           H  \\nATOM    566  HB  THR A  39      22.578  12.541  10.457  1.00  0.00           H  \\nATOM    567  HG1 THR A  39      23.082   9.857   9.613  1.00  0.00           H  \\nATOM    568 1HG2 THR A  39      23.273  11.171  12.379  1.00  0.00           H  \\nATOM    569 2HG2 THR A  39      21.535  11.541  12.459  1.00  0.00           H  \\nATOM    570 3HG2 THR A  39      22.083   9.934  11.936  1.00  0.00           H  \\nATOM    571  N   PRO A  40      18.925  11.234  11.190  1.00  0.00           N  \\nATOM    572  CA  PRO A  40      17.825  11.780  11.957  1.00  0.00           C  \\nATOM    573  C   PRO A  40      18.265  12.313  13.323  1.00  0.00           C  \\nATOM    574  O   PRO A  40      19.285  11.867  13.849  1.00  0.00           O  \\nATOM    575  CB  PRO A  40      16.923  10.567  12.116  1.00  0.00           C  \\nATOM    576  CG  PRO A  40      17.831   9.383  12.072  1.00  0.00           C  \\nATOM    577  CD  PRO A  40      18.915   9.756  11.100  1.00  0.00           C  \\nATOM    578  HA  PRO A  40      17.347  12.550  11.352  1.00  0.00           H  \\nATOM    579 1HB  PRO A  40      16.415  10.650  13.067  1.00  0.00           H  \\nATOM    580 2HB  PRO A  40      16.163  10.534  11.339  1.00  0.00           H  \\nATOM    581 1HG  PRO A  40      18.213   9.168  13.081  1.00  0.00           H  \\nATOM    582 2HG  PRO A  40      17.256   8.489  11.764  1.00  0.00           H  \\nATOM    583 1HD  PRO A  40      19.845   9.316  11.471  1.00  0.00           H  \\nATOM    584 2HD  PRO A  40      18.660   9.424  10.076  1.00  0.00           H  \\nATOM    585  N   PRO A  41      17.505  13.252  13.918  1.00  0.00           N  \\nATOM    586  CA  PRO A  41      17.631  13.747  15.278  1.00  0.00           C  \\nATOM    587  C   PRO A  41      17.093  12.715  16.260  1.00  0.00           C  \\nATOM    588  O   PRO A  41      16.301  11.858  15.881  1.00  0.00           O  \\nATOM    589  CB  PRO A  41      16.806  15.033  15.249  1.00  0.00           C  \\nATOM    590  CG  PRO A  41      15.758  14.777  14.221  1.00  0.00           C  \\nATOM    591  CD  PRO A  41      16.435  13.918  13.155  1.00  0.00           C  \\nATOM    592  HA  PRO A  41      18.690  13.962  15.483  1.00  0.00           H  \\nATOM    593 1HB  PRO A  41      16.382  15.231  16.244  1.00  0.00           H  \\nATOM    594 2HB  PRO A  41      17.452  15.889  14.999  1.00  0.00           H  \\nATOM    595 1HG  PRO A  41      14.929  14.278  14.704  1.00  0.00           H  \\nATOM    596 2HG  PRO A  41      15.374  15.730  13.828  1.00  0.00           H  \\nATOM    597 1HD  PRO A  41      15.713  13.192  12.776  1.00  0.00           H  \\nATOM    598 2HD  PRO A  41      16.861  14.551  12.363  1.00  0.00           H  \\nATOM    599  N   GLY A  42      17.456  12.845  17.532  1.00  0.00           N  \\nATOM    600  CA  GLY A  42      17.002  11.933  18.591  1.00  0.00           C  \\nATOM    601  C   GLY A  42      15.482  11.781  18.752  1.00  0.00           C  \\nATOM    602  O   GLY A  42      15.010  10.737  19.199  1.00  0.00           O  \\nATOM    603  H   GLY A  42      18.100  13.583  17.780  1.00  0.00           H  \\nATOM    604 1HA  GLY A  42      17.437  10.950  18.407  1.00  0.00           H  \\nATOM    605 2HA  GLY A  42      17.419  12.271  19.539  1.00  0.00           H  \\nATOM    606  N   ASP A  43      14.715  12.804  18.387  1.00  0.00           N  \\nATOM    607  CA  ASP A  43      13.258  12.757  18.530  1.00  0.00           C  \\nATOM    608  C   ASP A  43      12.573  12.058  17.355  1.00  0.00           C  \\nATOM    609  O   ASP A  43      11.394  11.707  17.424  1.00  0.00           O  \\nATOM    610  CB  ASP A  43      12.715  14.177  18.649  1.00  0.00           C  \\nATOM    611  CG  ASP A  43      13.181  14.869  19.914  1.00  0.00           C  \\nATOM    612  OD1 ASP A  43      13.225  14.238  20.943  1.00  0.00           O  \\nATOM    613  OD2 ASP A  43      13.515  16.022  19.834  1.00  0.00           O  \\nATOM    614  H   ASP A  43      15.155  13.633  18.019  1.00  0.00           H  \\nATOM    615  HA  ASP A  43      13.019  12.209  19.441  1.00  0.00           H  \\nATOM    616 1HB  ASP A  43      13.044  14.761  17.793  1.00  0.00           H  \\nATOM    617 2HB  ASP A  43      11.623  14.155  18.631  1.00  0.00           H  \\nATOM    618  N   TYR A  44      13.307  11.925  16.260  1.00  0.00           N  \\nATOM    619  CA  TYR A  44      12.835  11.340  15.020  1.00  0.00           C  \\nATOM    620  C   TYR A  44      13.868  10.331  14.562  1.00  0.00           C  \\nATOM    621  O   TYR A  44      14.138  10.191  13.378  1.00  0.00           O  \\nATOM    622  CB  TYR A  44      12.554  12.414  13.978  1.00  0.00           C  \\nATOM    623  CG  TYR A  44      11.478  13.338  14.397  1.00  0.00           C  \\nATOM    624  CD1 TYR A  44      11.765  14.522  15.047  1.00  0.00           C  \\nATOM    625  CD2 TYR A  44      10.189  12.991  14.127  1.00  0.00           C  \\nATOM    626  CE1 TYR A  44      10.742  15.354  15.435  1.00  0.00           C  \\nATOM    627  CE2 TYR A  44       9.156  13.808  14.507  1.00  0.00           C  \\nATOM    628  CZ  TYR A  44       9.430  14.995  15.165  1.00  0.00           C  \\nATOM    629  OH  TYR A  44       8.404  15.829  15.552  1.00  0.00           O  \\nATOM    630  H   TYR A  44      14.289  12.189  16.286  1.00  0.00           H  \\nATOM    631  HA  TYR A  44      11.910  10.809  15.219  1.00  0.00           H  \\nATOM    632 1HB  TYR A  44      13.436  12.975  13.756  1.00  0.00           H  \\nATOM    633 2HB  TYR A  44      12.232  11.940  13.072  1.00  0.00           H  \\nATOM    634  HD1 TYR A  44      12.791  14.798  15.258  1.00  0.00           H  \\nATOM    635  HD2 TYR A  44       9.995  12.067  13.611  1.00  0.00           H  \\nATOM    636  HE1 TYR A  44      10.963  16.288  15.951  1.00  0.00           H  \\nATOM    637  HE2 TYR A  44       8.125  13.523  14.287  1.00  0.00           H  \\nATOM    638  HH  TYR A  44       7.565  15.461  15.261  1.00  0.00           H  \\nATOM    639  N   ALA A  45      14.462   9.650  15.530  1.00  0.00           N  \\nATOM    640  CA  ALA A  45      15.554   8.695  15.353  1.00  0.00           C  \\nATOM    641  C   ALA A  45      15.270   7.484  14.448  1.00  0.00           C  \\nATOM    642  O   ALA A  45      16.221   6.813  14.034  1.00  0.00           O  \\nATOM    643  CB  ALA A  45      15.967   8.173  16.712  1.00  0.00           C  \\nATOM    644  H   ALA A  45      14.152   9.835  16.475  1.00  0.00           H  \\nATOM    645  HA  ALA A  45      16.389   9.244  14.924  1.00  0.00           H  \\nATOM    646 1HB  ALA A  45      15.136   7.626  17.154  1.00  0.00           H  \\nATOM    647 2HB  ALA A  45      16.823   7.510  16.601  1.00  0.00           H  \\nATOM    648 3HB  ALA A  45      16.234   9.009  17.353  1.00  0.00           H  \\nATOM    649  N   ASN A  46      14.002   7.127  14.237  1.00  0.00           N  \\nATOM    650  CA  ASN A  46      13.667   5.941  13.464  1.00  0.00           C  \\nATOM    651  C   ASN A  46      13.528   6.245  11.977  1.00  0.00           C  \\nATOM    652  O   ASN A  46      13.710   7.387  11.554  1.00  0.00           O  \\nATOM    653  OXT ASN A  46      12.938   5.436  11.266  1.00  0.00           O  \\nATOM    654  CB  ASN A  46      12.413   5.294  14.026  1.00  0.00           C  \\nATOM    655  CG  ASN A  46      12.660   4.675  15.393  1.00  0.00           C  \\nATOM    656  OD1 ASN A  46      13.450   3.734  15.502  1.00  0.00           O  \\nATOM    657  ND2 ASN A  46      12.025   5.180  16.425  1.00  0.00           N  \\nATOM    658  H   ASN A  46      13.238   7.723  14.553  1.00  0.00           H  \\nATOM    659  HA  ASN A  46      14.485   5.227  13.566  1.00  0.00           H  \\nATOM    660 1HB  ASN A  46      11.627   6.043  14.113  1.00  0.00           H  \\nATOM    661 2HB  ASN A  46      12.057   4.521  13.342  1.00  0.00           H  \\nATOM    662 1HD2 ASN A  46      12.173   4.800  17.339  1.00  0.00           H  \\nATOM    663 2HD2 ASN A  46      11.387   5.949  16.312  1.00  0.00           H  \\nTER                                                                             \\n# All scores below are weighted scores, not raw scores.\\n#BEGIN_POSE_ENERGIES_TABLE \\nlabel fa_atr fa_rep fa_sol fa_intra_rep fa_intra_sol_xover4 lk_ball_wtd fa_elec hbond_sr_bb hbond_lr_bb hbond_bb_sc hbond_sc dslf_fa13 omega fa_dun p_aa_pp yhh_planarity ref rama_prepro cart_bonded total\\nweights 1 0.55 1 0.005 1 1 1 1 1 1 1 1.25 0.4 0.7 0.6 0.625 1 0.45 0.5 NA\\npose -239.681 27.9079 136.559 0.59997 5.62287 -9.99474 -58.6334 -17.569 -7.84279 -2.9582 -0.78388 0 0.023 24.7556 -10.6105 0.00105 27.0504 -2.94564 22.8353 -105.664\\nTHR:NtermProteinFull_1 -4.79798 0.60793 4.13562 0.00741 0.04888 -0.11143 -4.70664 0 0 -0.47959 0 0 0.04264 0.23002 0 0 1.15175 0 0.63112 -3.24026\\nTHR_2 -6.39905 1.08915 1.24092 0.00838 0.05099 -0.60355 -0.79124 0 0 0 0 0 -0.02779 0.11746 -0.1214 0 1.15175 0.04595 0.78322 -3.4552\\nALA_3 -6.30334 0.8635 1.6266 0.0013 0 0.04833 -2.70902 0 0 0 0 0 0.21287 0 0.22575 0 1.32468 0.21682 0.45506 -4.03745\\nPHE_4 -12.2748 1.93026 4.62065 0.03516 0.36123 -0.62862 -2.20739 0 0 0 0 0 0.05691 1.56711 -0.30546 0 1.21829 0.1879 1.40681 -4.03195\\nPRO_5 -4.54223 0.56048 1.52888 0.00269 0.04856 -0.06109 -0.09651 0 0 0 0 0 -0.03956 0.09224 -0.08767 0 -1.64321 -0.02588 1.24109 -3.02222\\nSER_6 -5.24858 0.38989 4.61534 0.00158 0.07909 -0.15974 -1.4134 0 0 -0.99951 0 0 0.02908 0.08866 -0.51955 0 -0.28969 -0.18665 0.31622 -3.29725\\nILE_7 -4.72735 0.22409 3.74211 0.03085 0.07811 -0.35474 -1.2327 0 0 0 0 0 -0.03234 0.20264 -0.23094 0 2.30374 -0.21123 0.28451 0.07675\\nVAL_8 -4.26499 0.27565 3.69259 0.02377 0.05467 -0.38457 -0.72467 0 0 0 0 0 -0.0095 -0.0228 -0.2447 0 2.64269 -0.13365 0.18987 1.09433\\nALA_9 -6.67405 0.55093 3.58245 0.00137 0 -0.19011 -1.42914 0 0 -0.99951 0 0 0.11897 0 -0.26765 0 1.32468 -0.25674 0.19895 -4.03984\\nARG_10 -7.65705 0.65417 6.31609 0.01601 0.48214 -0.50806 -2.51068 0 0 0 0 0 -0.01533 2.00846 -0.05239 0 -0.09474 -0.31608 0.17364 -1.50381\\nSER_11 -4.58252 0.28401 4.89523 0.00137 0.02305 -0.2291 -2.40015 0 0 0 0 0 0.02222 0.41317 0.29757 0 -0.28969 -0.13957 0.11247 -1.59195\\nASN_12 -6.09048 0.35179 4.80044 0.00531 0.24678 -0.50172 -2.36605 0 0 0 0 0 -0.03288 1.0132 0.3692 0 -1.34026 0.0101 0.12799 -3.40658\\nPHE_13 -11.6626 2.20987 4.43707 0.02526 0.22704 0.09823 -2.39608 0 0 0 0 0 -0.00964 1.6896 -0.40919 0 1.21829 0.0183 0.31151 -4.24234\\nASN_14 -4.66564 0.16522 4.22592 0.00714 0.26372 -0.41959 -1.07191 0 0 0 0 0 0.08978 1.07143 0.24519 0 -1.34026 -0.01213 0.31899 -1.12214\\nVAL_15 -4.85903 0.45389 3.23337 0.02144 0.05208 -0.07805 -1.50959 0 0 0 0 0 -0.02716 -0.02537 -0.37638 0 2.64269 -0.01645 0.15515 -0.33344\\nMET_16 -7.41252 1.05483 2.70399 0.02917 0.10701 -0.30827 -1.31194 0 0 0 0 0 0.02655 1.1676 -0.14045 0 1.65735 -0.14825 0.38368 -2.19125\\nARG_17 -7.80978 0.85075 5.77442 0.01822 0.7668 -0.2075 -3.34978 0 0 0 -0.39194 0 -0.02145 2.46001 -0.04389 0 -0.09474 -0.38363 0.67463 -1.75786\\nLEU_18 -3.49255 0.74052 2.38712 0.02547 0.09136 0.04185 -0.45351 0 0 0 0 0 0.08137 0.37468 -0.23355 0 1.66147 0.63211 1.60363 3.45998\\nPRO_19 -2.8163 0.77704 1.86779 0.00358 0.06274 -0.17089 -0.03888 0 0 0 0 0 -0.0022 0.03408 0.08172 0 -1.64321 1.09721 1.53491 0.78758\\nGLY_20 -1.59292 0.1145 1.79118 7e-05 0 -0.19162 -0.82887 0 0 0 0 0 -0.15114 0 -1.47729 0 0.79816 -0.08469 0.4502 -1.17242\\nTHR_21 -4.56763 0.51111 2.4891 0.00746 0.05044 -0.22961 -0.95018 0 0 0 0 0 -0.04231 0.07541 0.05806 0 1.15175 -0.47811 0.23404 -1.69045\\nSER_22 -3.73858 0.31765 2.86265 0.00139 0.02399 -0.12762 -1.22401 0 0 0 0 0 0.06894 0.72925 -0.08403 0 -0.28969 -0.37752 0.15921 -1.67837\\nGLU_23 -5.51906 0.60802 4.80064 0.0063 0.23794 -0.26465 -2.54018 0 0 0 -0.39194 0 -0.01842 2.59976 -0.26247 0 -2.72453 -0.48516 0.30784 -3.64592\\nALA_24 -3.31787 0.28076 2.14888 0.00143 0 -0.22111 -0.32028 0 0 0 0 0 -0.04903 0 -0.26573 0 1.32468 -0.52533 0.27632 -0.66727\\nILE_25 -5.37464 0.4076 3.27691 0.03382 0.06859 -0.19911 -1.44121 0 0 0 0 0 -0.0197 0.10925 -0.4943 0 2.30374 -0.19509 0.24394 -1.28019\\nVAL_26 -9.72156 1.88111 2.56664 0.05711 0.0571 -0.26881 -2.04992 0 0 0 0 0 -0.05866 0.12725 -0.18584 0 2.64269 -0.05295 0.67004 -4.33579\\nALA_27 -7.04065 0.52963 4.16336 0.00159 0 -0.25099 -3.14613 0 0 0 0 0 -0.03622 0 -0.2653 0 1.32468 -0.23788 0.77287 -4.18505\\nTHR_28 -3.71605 0.33337 4.1083 0.00639 0.05256 -0.29067 -1.16545 0 0 0 0 0 0.21134 0.27754 0.12474 0 1.15175 -0.17233 0.4994 1.4209\\nTYR_29 -5.53987 0.61722 2.44116 0.02379 0.32507 -0.19533 -0.19117 0 0 0 0 0 0.01248 1.44884 -0.3283 9e-05 0.58223 -0.04814 0.33743 -0.5145\\nTHR_30 -7.1055 0.64172 3.61121 0.01029 0.06735 -0.05932 -1.48751 0 0 0 0 0 -0.0325 0.07636 -0.17353 0 1.15175 -0.23161 0.36079 -3.1705\\nGLY_31 -2.50353 0.09509 2.47581 7e-05 0 -0.18295 -0.80438 0 0 0 0 0 -0.14799 0 -1.45282 0 0.79816 -0.52026 0.29643 -1.94637\\nGLY_32 -3.86213 0.28419 2.09185 0.00012 0 -0.17529 -1.0729 0 0 0 0 0 -0.10239 0 -0.71375 0 0.79816 -0.27498 0.1054 -2.92173\\nILE_33 -6.82019 0.69331 1.51806 0.04351 0.13497 -0.08665 -1.82862 0 0 0 0 0 -0.00034 0.05471 0.00698 0 2.30374 -0.10673 0.19412 -3.89314\\nILE_34 -5.18863 0.52976 0.64377 0.04611 0.08526 -0.46831 -0.47882 0 0 0 0 0 -0.0499 0.86321 -0.66459 0 2.30374 -0.36471 0.22339 -2.51972\\nILE_35 -7.31409 0.90159 2.32237 0.03629 0.13438 -0.21033 -0.94397 0 0 0 0 0 -0.04555 0.31837 -0.25504 0 2.30374 -0.09405 0.70919 -2.1371\\nPRO_36 -1.2947 0.33064 0.56957 0.00269 0.04736 -0.02819 0.32108 0 0 0 0 0 -0.05022 0.10708 -0.15972 0 -1.64321 0.06412 0.78758 -0.94592\\nGLY_37 -2.32084 0.25487 2.01547 6e-05 0 -0.11421 -0.61489 0 0 -0.47959 0 0 0.0168 0 -1.12306 0 0.79816 0.07419 0.1763 -1.31673\\nALA_38 -1.76735 0.11067 0.86479 0.00161 0 -0.21834 0.2358 0 0 0 0 0 -0.05349 0 0.68551 0 1.32468 0.57294 0.06127 1.81809\\nTHR_39 -2.20251 0.47512 1.04592 0.00993 0.05961 -0.23234 -0.02617 0 0 0 0 0 -0.05202 0.02039 0.01602 0 1.15175 0.51783 0.662 1.44555\\nPRO_40 -4.36009 1.08448 1.2951 0.00384 0.11208 -0.03211 -0.37615 0 0 0 0 0 0.1557 0.07114 -0.88599 0 -1.64321 -0.09628 1.2631 -3.4084\\nPRO_41 -4.63389 0.63915 2.47373 0.00373 0.11431 0.02771 -0.69746 0 0 0 0 0 0.08613 0.04043 -0.86656 0 -1.64321 -0.29746 0.7624 -3.99099\\nGLY_42 -1.56787 0.08378 1.31747 0.00011 0 -0.19545 0.08907 0 0 0 0 0 -0.14491 0 0.40313 0 0.79816 0.00639 0.29984 1.08971\\nASP_43 -2.69948 0.19853 1.86724 0.00404 0.32211 -0.2322 0.08917 0 0 0 0 0 -0.03526 1.94329 -0.26492 0 -2.14574 -0.0499 0.2981 -0.70503\\nTYR_44 -10.2549 1.47306 3.21825 0.02508 0.30454 -0.1777 -1.15412 0 0 0 0 0 0.11381 1.5232 -0.00916 0.00096 0.58223 0.24552 0.4482 -3.66101\\nALA_45 -3.2744 0.11016 2.50247 0.00203 0 -0.21655 -0.39047 0 0 0 0 0 -0.01468 0 -0.15877 0 1.32468 0.14911 0.75518 0.78876\\nASN:CtermProteinFull_46 -6.10153 0.36685 6.65043 0.00563 0.38095 -0.42435 -2.91642 0 0 0 0 0 0 1.88789 0 0 -1.34026 -0.26068 0.57724 -1.17426\\n#END_POSE_ENERGIES_TABLE \\n\\n\",\"pdb\");\n",
              "\tviewer_16234200519329896.setStyle({\"cartoon\": {\"color\": \"spectrum\"}, \"stick\": {\"colorscheme\": \"lightgreyCarbon\", \"radius\": 0.25}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.501930166384962, \"y\": 12.609923315803297, \"z\": 3.948209828627384}, \"end\": {\"x\": 19.33886416435858, \"y\": 13.775721176988824, \"z\": 4.962085272711633}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 16.00563702599574, \"y\": 14.477987473439045, \"z\": 3.7037402242218587}, \"end\": {\"x\": 16.535533321636706, \"y\": 16.20405075041262, \"z\": 5.019148295057891}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 14.453818090014181, \"y\": 15.434611231043343, \"z\": 6.349324753819337}, \"end\": {\"x\": 14.179465000973828, \"y\": 13.778989413197973, \"z\": 5.532955025358115}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.351100348025618, \"y\": 12.598023300746231, \"z\": 8.527723408668292}, \"end\": {\"x\": 11.547272748069226, \"y\": 14.218615922922377, \"z\": 8.164396736622226}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 10.096697320524056, \"y\": 13.295568434092688, \"z\": 10.072931479257942}, \"end\": {\"x\": 10.80342294160081, \"y\": 11.466100909156166, \"z\": 10.49330002896156}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.59549519498949, \"y\": 9.008544830703977, \"z\": 11.455433196786293}, \"end\": {\"x\": 13.710100834998277, \"y\": 7.3870436072749515, \"z\": 11.554301851447404}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.238356733146379, \"y\": 7.723051866041844, \"z\": 14.552938624809517}, \"end\": {\"x\": 11.491047492098945, \"y\": 8.64602251061033, \"z\": 14.096052446841924}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 8.197173327953104, \"y\": 4.6332944015185715, \"z\": 10.762786537460512}, \"end\": {\"x\": 8.808120309283371, \"y\": 4.935041958074698, \"z\": 12.722034046986833}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.210948116188301, \"y\": 4.520199195000023, \"z\": 12.650397650346607}, \"end\": {\"x\": 6.302684649659714, \"y\": 5.237340866075713, \"z\": 14.568610348032506}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.546892955767251, \"y\": 2.3711895398141625, \"z\": 9.254443513459494}, \"end\": {\"x\": 8.159013670956877, \"y\": 1.9302431908245523, \"z\": 10.968386151214002}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.156152049209645, \"y\": 3.127992703602933, \"z\": 8.203808250307834}, \"end\": {\"x\": 5.1433693312028534, \"y\": 3.0166457614765503, \"z\": 10.075172492683675}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.210948116188301, \"y\": 4.520199195000023, \"z\": 12.650397650346607}, \"end\": {\"x\": 6.302684649659714, \"y\": 5.237340866075713, \"z\": 14.568610348032506}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.019005799996025, \"y\": 5.0575726863878545, \"z\": 6.577620964912224}, \"end\": {\"x\": 6.280447155272698, \"y\": 5.65314491415779, \"z\": 8.46296558615851}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 8.197173327953104, \"y\": 4.6332944015185715, \"z\": 10.762786537460512}, \"end\": {\"x\": 8.808120309283371, \"y\": 4.935041958074698, \"z\": 12.722034046986833}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.3807705257937215, \"y\": 3.769478936476843, \"z\": 4.530495247582149}, \"end\": {\"x\": 8.24871827373643, \"y\": 3.7219948443813866, \"z\": 6.492939911646457}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.546892955767251, \"y\": 2.3711895398141625, \"z\": 9.254443513459494}, \"end\": {\"x\": 8.159013670956877, \"y\": 1.9302431908245523, \"z\": 10.968386151214002}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.19875848395078, \"y\": 2.6415947617638884, \"z\": 3.2663536851364374}, \"end\": {\"x\": 5.696416772571791, \"y\": 1.8423117566352694, \"z\": 5.360285718561732}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.156152049209645, \"y\": 3.127992703602933, \"z\": 8.203808250307834}, \"end\": {\"x\": 5.1433693312028534, \"y\": 3.0166457614765503, \"z\": 10.075172492683675}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.6934330554077053, \"y\": 4.609270113824714, \"z\": 2.482657476301151}, \"end\": {\"x\": 3.8979135945848835, \"y\": 4.526709865498515, \"z\": 4.30498053288776}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.019005799996025, \"y\": 5.0575726863878545, \"z\": 6.577620964912224}, \"end\": {\"x\": 6.280447155272698, \"y\": 5.65314491415779, \"z\": 8.46296558615851}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.346908707324643, \"y\": 5.529708684458337, \"z\": 0.44429949173639477}, \"end\": {\"x\": 6.213399362919355, \"y\": 5.673851700784857, \"z\": 2.2983121281981505}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.3807705257937215, \"y\": 3.769478936476843, \"z\": 4.530495247582149}, \"end\": {\"x\": 8.24871827373643, \"y\": 3.7219948443813866, \"z\": 6.492939911646457}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.19875848395078, \"y\": 2.6415947617638884, \"z\": 3.2663536851364374}, \"end\": {\"x\": 5.696416772571791, \"y\": 1.8423117566352694, \"z\": 5.360285718561732}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.330573852435505, \"y\": 4.344187216389852, \"z\": -1.7381592484159094}, \"end\": {\"x\": 3.069899911110412, \"y\": 3.4951769499568277, \"z\": -0.38192375119158256}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.6934330554077053, \"y\": 4.609270113824714, \"z\": 2.482657476301151}, \"end\": {\"x\": 3.8979135945848835, \"y\": 4.526709865498515, \"z\": 4.30498053288776}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.346908707324643, \"y\": 5.529708684458337, \"z\": 0.44429949173639477}, \"end\": {\"x\": 6.213399362919355, \"y\": 5.673851700784857, \"z\": 2.2983121281981505}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 11.714263394535879, \"y\": 9.135126074135785, \"z\": -0.8018482166163853}, \"end\": {\"x\": 12.815426590298328, \"y\": 11.07814054174171, \"z\": -0.19954070205677193}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.023462724566651, \"y\": 6.825863514454596, \"z\": -5.211017881721877}, \"end\": {\"x\": 6.135787313971983, \"y\": 6.003921036650385, \"z\": -3.734850396379906}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.330573852435505, \"y\": 4.344187216389852, \"z\": -1.7381592484159094}, \"end\": {\"x\": 3.069899911110412, \"y\": 3.4951769499568277, \"z\": -0.38192375119158256}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.023462724566651, \"y\": 6.825863514454596, \"z\": -5.211017881721877}, \"end\": {\"x\": 6.135787313971983, \"y\": 6.003921036650385, \"z\": -3.734850396379906}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.862379554998098, \"y\": 12.891872348709098, \"z\": 1.6594546089172375}, \"end\": {\"x\": 5.827793289861441, \"y\": 13.112819923791411, \"z\": -0.13165692648088542}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 11.714263394535879, \"y\": 9.135126074135785, \"z\": -0.8018482166163853}, \"end\": {\"x\": 12.815426590298328, \"y\": 11.07814054174171, \"z\": -0.19954070205677193}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.2091876908773855, \"y\": 12.921111980322193, \"z\": 4.021058262322262}, \"end\": {\"x\": 7.514505168757632, \"y\": 13.535036636617333, \"z\": 2.926400951321996}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.436597608212043, \"y\": 14.483327567659321, \"z\": 5.683944002497221}, \"end\": {\"x\": 5.262080139661818, \"y\": 15.904394590696, \"z\": 3.9870451708961543}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.4360859573422253, \"y\": 12.720569482855412, \"z\": 5.860476285426446}, \"end\": {\"x\": 2.745797214264521, \"y\": 13.724906417266116, \"z\": 3.8681656343546997}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.480113706512622, \"y\": 14.958817449329434, \"z\": 5.153903962366791}, \"end\": {\"x\": 2.745797214264521, \"y\": 13.724906417266116, \"z\": 3.8681656343546997}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.862379554998098, \"y\": 12.891872348709098, \"z\": 1.6594546089172375}, \"end\": {\"x\": 5.827793289861441, \"y\": 13.112819923791411, \"z\": -0.13165692648088542}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.3757807705989387, \"y\": 10.641189297552101, \"z\": 7.314445152594147}, \"end\": {\"x\": 4.134825898978926, \"y\": 11.065818669659016, \"z\": 5.499067699985038}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.2091876908773855, \"y\": 12.921111980322193, \"z\": 4.021058262322262}, \"end\": {\"x\": 7.514505168757632, \"y\": 13.535036636617333, \"z\": 2.926400951321996}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.905176103973061, \"y\": 11.950208225084916, \"z\": 9.678030140476736}, \"end\": {\"x\": 4.968165842984307, \"y\": 12.512466509072583, \"z\": 8.134583619562187}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.5906811155751894, \"y\": 11.867479904865563, \"z\": 9.489357147186416}, \"end\": {\"x\": 4.968165842984307, \"y\": 12.512466509072583, \"z\": 8.134583619562187}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.160076179676509, \"y\": 10.626161725774526, \"z\": 7.911375251744044}, \"end\": {\"x\": 4.968165842984307, \"y\": 12.512466509072583, \"z\": 8.134583619562187}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.436597608212043, \"y\": 14.483327567659321, \"z\": 5.683944002497221}, \"end\": {\"x\": 5.262080139661818, \"y\": 15.904394590696, \"z\": 3.9870451708961543}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.480113706512622, \"y\": 14.958817449329434, \"z\": 5.153903962366791}, \"end\": {\"x\": 2.745797214264521, \"y\": 13.724906417266116, \"z\": 3.8681656343546997}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.4360859573422253, \"y\": 12.720569482855412, \"z\": 5.860476285426446}, \"end\": {\"x\": 2.745797214264521, \"y\": 13.724906417266116, \"z\": 3.8681656343546997}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.3757807705989387, \"y\": 10.641189297552101, \"z\": 7.314445152594147}, \"end\": {\"x\": 4.134825898978926, \"y\": 11.065818669659016, \"z\": 5.499067699985038}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.160076179676509, \"y\": 10.626161725774526, \"z\": 7.911375251744044}, \"end\": {\"x\": 4.968165842984307, \"y\": 12.512466509072583, \"z\": 8.134583619562187}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.5906811155751894, \"y\": 11.867479904865563, \"z\": 9.489357147186416}, \"end\": {\"x\": 4.968165842984307, \"y\": 12.512466509072583, \"z\": 8.134583619562187}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.905176103973061, \"y\": 11.950208225084916, \"z\": 9.678030140476736}, \"end\": {\"x\": 4.968165842984307, \"y\": 12.512466509072583, \"z\": 8.134583619562187}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.351100348025618, \"y\": 12.598023300746231, \"z\": 8.527723408668292}, \"end\": {\"x\": 11.547272748069226, \"y\": 14.218615922922377, \"z\": 8.164396736622226}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 10.096697320524056, \"y\": 13.295568434092688, \"z\": 10.072931479257942}, \"end\": {\"x\": 10.80342294160081, \"y\": 11.466100909156166, \"z\": 10.49330002896156}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 16.00563702599574, \"y\": 14.477987473439045, \"z\": 3.7037402242218587}, \"end\": {\"x\": 16.535533321636706, \"y\": 16.20405075041262, \"z\": 5.019148295057891}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 14.453818090014181, \"y\": 15.434611231043343, \"z\": 6.349324753819337}, \"end\": {\"x\": 14.179465000973828, \"y\": 13.778989413197973, \"z\": 5.532955025358115}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.501930166384962, \"y\": 12.609923315803297, \"z\": 3.948209828627384}, \"end\": {\"x\": 19.33886416435858, \"y\": 13.775721176988824, \"z\": 4.962085272711633}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 14.288839344741703, \"y\": 12.188847934629763, \"z\": 16.28592787383205}, \"end\": {\"x\": 16.30057380264282, \"y\": 11.857780663235769, \"z\": 15.88123415008611}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 14.288839344741703, \"y\": 12.188847934629763, \"z\": 16.28592787383205}, \"end\": {\"x\": 16.30057380264282, \"y\": 11.857780663235769, \"z\": 15.88123415008611}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.59549519498949, \"y\": 9.008544830703977, \"z\": 11.455433196786293}, \"end\": {\"x\": 13.710100834998277, \"y\": 7.3870436072749515, \"z\": 11.554301851447404}});\n",
              "\tviewer_16234200519329896.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.238356733146379, \"y\": 7.723051866041844, \"z\": 14.552938624809517}, \"end\": {\"x\": 11.491047492098945, \"y\": 8.64602251061033, \"z\": 14.096052446841924}});\n",
              "\tviewer_16234200519329896.zoomTo();\n",
              "viewer_16234200519329896.render();\n",
              "});\n",
              "</script>"
            ]
          },
          "metadata": {
            "tags": []
          }
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "HETbY3TpmdlI"
      },
      "source": [
        "## *Design Challenge*:\n",
        "\n",
        "### Copy and modify this script to:\n",
        " - Repack all residues and allow all degrees of freedom to minimize (i.e. FastRelax) `start_pose` prior to downstream design. Use the resulting `start_pose` for analysis.\n",
        " - Re-design the cysteine residues in `start_pose` to allow only hydrophobic residues excluding glycine and proline\n",
        " - Only re-pack residues within a 6 Angstroms radius around the neighborhood of each cysteine residue (C-C distance)\n",
        "   - Note this will require using:\n",
        "   - The NeighborhoodResidueSelector: `pyrosetta.rosetta.core.select.residue_selector.NeighborhoodResidueSelector(selector, distance, include_focus_in_subset)`\n",
        "   - The prevent repacking residue level task operation: `pyrosetta.rosetta.core.pack.task.operation.PreventRepackingRLT()`\n",
        " - Minimize only the side-chains (chi degrees of freedom) being re-packed or designed\n",
        " - Allow the backbone torsions to minimize, except for the first, last and any proline phi/psi angles\n",
        " \n",
        "## *Extra Challenge!*\n",
        "\n",
        " Instead of using `pyrosetta.rosetta.core.pack.task.operation.OperateOnResidueSubset` TaskOperations, use `pyrosetta.rosetta.protocols.task_operations.ResfileCommandOperation` TaskOperations by applying resfile-style commands to ResidueSelectors. Example:\n",
        " > repack_only_task_operation = pyrosetta.rosetta.protocols.task_operations.ResfileCommandOperation()   \n",
        " > repack_only_task_operation.set_command(\"POLAR EMPTY NC R2 NC T6 NC OP5\")   \n",
        " > repack_only_task_operation.set_residue_selector(selector)   \n",
        " > tf.push_back(repack_only_task_operation)"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "oyy4MDq9mdlJ"
      },
      "source": [
        " By how many Angstroms RMSD did the backbone C atoms move?"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "0hDoABTvmdlJ",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "9ed48f7c-ad81-48fa-cc17-2509f1eb3816"
      },
      "source": [
        "pyrosetta.rosetta.core.scoring.CA_rmsd(start_pose, pose)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "0.9092081189155579"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 43
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "4B6QuG3WmdlJ"
      },
      "source": [
        " What is the delta `total_score` from `start_pose` to `pose`?"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "BOMdL0ucmdlJ",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "b47f804d-adc1-4649-f139-1f1771cb7577"
      },
      "source": [
        "delta_total_score = scorefxn(pose) - scorefxn(start_pose)\n",
        "print(delta_total_score)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (46)\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "stream",
          "text": [
            "-506.33418627927233\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "jQCBjqOemdlJ"
      },
      "source": [
        " What is the per-residue energy difference for each mutated position between `start_pose` and `pose`?"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "Ae9gfV9smdlJ",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 266
        },
        "outputId": "8ba42828-b839-4d2e-ab2c-648650a151ec"
      },
      "source": [
        "# # for loop through cys_res\n",
        "# ## per-residue energy: pyrosetta.rosetta.protocols.relax.get_per_residue_scores(pose, pyrosetta.rosetta.core.scoring.ScoreType.total_score)\n",
        "# ## find per residue energy in pose and original pose\n",
        "# ## subtraction between original pose and pose\n",
        "# delta_total_score = scorefxn(pose) - scorefxn(start_pose)\n",
        "# print(delta_total_score)\n",
        "# for i in cys_res:\n",
        "#     pose_total_score = pyrosetta.rosetta.protocols.relax.get_per_residue_scores(pose, pyrosetta.rosetta.core.scoring.ScoreType.total_score)[i - 1]\n",
        "#     start_pose_total_score = pyrosetta.rosetta.protocols.relax.get_per_residue_scores(start_pose, pyrosetta.rosetta.core.scoring.ScoreType.total_score)[i - 1]\n",
        "#     print(\"The delta total_score for residue {} between start_pose and pose is {}\".format(i, pose_total_score - start_pose_total_score))"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "-506.33418627927233\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "error",
          "ename": "TypeError",
          "evalue": "ignored",
          "traceback": [
            "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
            "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
            "\u001b[0;32m<ipython-input-45-e20d874a16ca>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      5\u001b[0m \u001b[0mdelta_total_score\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mscorefxn\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mpose\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m-\u001b[0m \u001b[0mscorefxn\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mstart_pose\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      6\u001b[0m \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdelta_total_score\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 7\u001b[0;31m \u001b[0;32mfor\u001b[0m \u001b[0mi\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mcys_res\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      8\u001b[0m     \u001b[0mpose_total_score\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mpyrosetta\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrosetta\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mprotocols\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrelax\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mget_per_residue_scores\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mpose\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mpyrosetta\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrosetta\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mcore\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mscoring\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mScoreType\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mtotal_score\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mi\u001b[0m \u001b[0;34m-\u001b[0m \u001b[0;36m1\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      9\u001b[0m     \u001b[0mstart_pose_total_score\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mpyrosetta\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrosetta\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mprotocols\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrelax\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mget_per_residue_scores\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mstart_pose\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mpyrosetta\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrosetta\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mcore\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mscoring\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mScoreType\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mtotal_score\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mi\u001b[0m \u001b[0;34m-\u001b[0m \u001b[0;36m1\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;31mTypeError\u001b[0m: 'pyrosetta.rosetta.core.select.residue_selector.ResidueNameSelector' object is not iterable"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "HpfCxT0HmdlJ"
      },
      "source": [
        "# HBNet Before Design\n",
        "\n",
        "Keywords: HBNet, OperateOnResidueSubset, getPoseExtraScore, InterGroupInterfaceByVectorSelector, ChainSelector, PreventRepackingRLT, RestrictToRepackingRLT, OperateOnResidueSubset, ResiduePDBInfoHasLabelSelector, PackRotamersMover"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "GROxpJ_dmdlJ"
      },
      "source": [
        "Sometimes in Rosetta we want to run implicit multistage design. That is, we want to optimize one conformation while implicitly modeling another (either negatively or positively). There are many ways to accomplish this depending on your interests. In this section we will look at HBNet, a tool for explicitly designing hydrogen bond networks.\n",
        "\n",
        "One negative-design approach is to implicitly model binding specificity. Designing a complicated network of hydrogen bonds at one interface will implicitly destabilize other interfaces. Hydrogen bonds are so sensitive to geometry that competing interfaces are unlikely to be able to \"satisfy\" the network well enough to remain competetive.\n",
        "\n",
        "The previous example can also be viewed through the implicit positive-design lens as well. We often find that Rosetta designs very hydrophobic interfaces (especially with newer score functions). Running HBNet before the traditional design protocols can boost the polar residue concentration of your interface in exchange for a small cost packing quality. In other words, we can implicitly stabilize the unbound state by running HBNet, but it might mildly destabilize the bound state.\n",
        "\n",
        "Our experience shows that it is useful to run both with and without HBNet, depending on your design case. It is possible that the default design protocol handles your implicits states well enough. When that fails, though, there is not much to do to fix it other than to run pre-design protocols like HBNet. An added benefit of HBNet is that it can provide \"seeds\" for packing, which can influence design diversity if nothing else."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "KEO67RsrmdlK",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "520d1270-9b23-4770-acd1-79c26d97c836"
      },
      "source": [
        "# From previous section:\n",
        "from pyrosetta import *\n",
        "from pyrosetta.teaching import *\n",
        "pyrosetta.init(\"-mute core -mute basic\")\n",
        "print( \"init complete\" )"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:pyrosetta.rosetta:Found rosetta database at: /usr/local/lib/python3.7/dist-packages/pyrosetta/database; using it....\n",
            "INFO:pyrosetta.rosetta:PyRosetta-4 2021 [Rosetta PyRosetta4.MinSizeRel.python37.ubuntu 2020.50.post.dev+970.commits.3700df145608444753aabbec9c4681ec9b21f74b 2021-02-24T13:24:53] retrieved from: http://www.pyrosetta.org\n",
            "(C) Copyright Rosetta Commons Member Institutions. Created in JHU by Sergey Lyskov and PyRosetta Team.\n",
            "INFO:rosetta:core.init: Checking for fconfig files in pwd and ./rosetta/flags\n",
            "INFO:rosetta:core.init: Rosetta version: PyRosetta4.MinSizeRel.python37.ubuntu r17132 2020.50.post.dev+970.commits.3700df14560 3700df145608444753aabbec9c4681ec9b21f74b http://www.pyrosetta.org 2021-02-24T13:24:53\n",
            "INFO:rosetta:core.init: command: PyRosetta -mute core -mute basic -database /usr/local/lib/python3.7/dist-packages/pyrosetta/database\n",
            "INFO:rosetta:basic.random.init_random_generator: 'RNG device' seed mode, using '/dev/urandom', seed=-2104333995 seed_offset=0 real_seed=-2104333995\n",
            "INFO:rosetta:basic.random.init_random_generator: RandomGenerator:init: Normal mode, seed=-2104333995 RG_type=mt19937\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "stream",
          "text": [
            "PyRosetta-4 2021 [Rosetta PyRosetta4.MinSizeRel.python37.ubuntu 2020.50.post.dev+970.commits.3700df145608444753aabbec9c4681ec9b21f74b 2021-02-24T13:24:53] retrieved from: http://www.pyrosetta.org\n",
            "(C) Copyright Rosetta Commons Member Institutions. Created in JHU by Sergey Lyskov and PyRosetta Team.\n",
            "init complete\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "R94ROpF0mdlK"
      },
      "source": [
        "We prepare for HBNet the same way that we prepare for packing. We setup the pose and score function as before..."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "nbgrader": {
          "grade": true,
          "grade_id": "cell-c34d0bd1a81b4a7d",
          "locked": false,
          "points": 0,
          "schema_version": 3,
          "solution": true
        },
        "id": "0z5VLhKbmdlK",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "adee3404-1075-427b-a0ef-05584a668b72"
      },
      "source": [
        "pose = pose_from_pdb(\"inputs/hbnet_example.pdb\")\n",
        "start_pose = Pose()\n",
        "start_pose.assign(pose)\n",
        "scorefxn = get_fa_scorefxn()\n",
        "print( \"setup complete\" )"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:core.import_pose.import_pose: File 'inputs/hbnet_example.pdb' automatically determined to be of type PDB\n",
            "INFO:rosetta:core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] PDB reader is ignoring atom  CD  in residue 131 D.  Pass flag -ignore_zero_occupancy false to change this behavior\n",
            "INFO:rosetta:core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] PDB reader is ignoring atom  CZ  in residue 141 D.  Pass flag -ignore_zero_occupancy false to change this behavior\n",
            "INFO:rosetta:core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] PDB reader is ignoring atom  CD  in residue 144 D.  Pass flag -ignore_zero_occupancy false to change this behavior\n",
            "INFO:rosetta:core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] PDB reader is ignoring atom  CG  in residue 174 D.  Pass flag -ignore_zero_occupancy false to change this behavior\n",
            "INFO:rosetta:core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] PDB reader is ignoring atom  CD  in residue 212 D.  Pass flag -ignore_zero_occupancy false to change this behavior\n",
            "INFO:rosetta:core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] PDB reader is ignoring atom  OD1 in residue 77 E.  Pass flag -ignore_zero_occupancy false to change this behavior\n",
            "INFO:rosetta:core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] PDB reader is ignoring atom  CD  in residue 187 E.  Pass flag -ignore_zero_occupancy false to change this behavior\n",
            "INFO:rosetta:core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] PDB reader is ignoring atom  CD  in residue 191 E.  Pass flag -ignore_zero_occupancy false to change this behavior\n",
            "INFO:rosetta:core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] PDB reader is ignoring atom  CD  in residue 217 E.  Pass flag -ignore_zero_occupancy false to change this behavior\n",
            "INFO:rosetta:core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] PDB reader is ignoring atom  CZ  in residue 217 E.  Pass flag -ignore_zero_occupancy false to change this behavior\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  CD  on residue GLN 117\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  CZ  on residue ARG 127\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  CD  on residue LYS 130\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  CG  on residue ASP 160\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  CD  on residue GLU 198\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  OD1 on residue ASN 269\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  CD  on residue GLN 378\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  CD  on residue GLU 382\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  CD  on residue ARG 408\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  CZ  on residue ARG 408\n",
            "INFO:rosetta:core.conformation.Conformation: Found disulfide between residues 21 90\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 21 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 90 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 21 CYD\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 90 CYD\n",
            "INFO:rosetta:core.conformation.Conformation: Found disulfide between residues 137 187\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 137 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 187 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 137 CYD\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 187 CYD\n",
            "INFO:rosetta:core.conformation.Conformation: Found disulfide between residues 162 374\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 162 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 374 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 162 CYD\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 374 CYD\n",
            "INFO:rosetta:core.conformation.Conformation: Found disulfide between residues 227 295\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 227 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 295 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 227 CYD\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 295 CYD\n",
            "INFO:rosetta:core.conformation.Conformation: Found disulfide between residues 348 413\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 348 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 413 CYS\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 348 CYD\n",
            "INFO:rosetta:core.conformation.Conformation: current variant for 413 CYD\n",
            "INFO:rosetta:core.pack.pack_missing_sidechains: packing residue number 117 because of missing atom number 7 atom name  CD\n",
            "INFO:rosetta:core.pack.pack_missing_sidechains: packing residue number 127 because of missing atom number 9 atom name  CZ\n",
            "INFO:rosetta:core.pack.pack_missing_sidechains: packing residue number 130 because of missing atom number 7 atom name  CD\n",
            "INFO:rosetta:core.pack.pack_missing_sidechains: packing residue number 160 because of missing atom number 6 atom name  CG\n",
            "INFO:rosetta:core.pack.pack_missing_sidechains: packing residue number 198 because of missing atom number 7 atom name  CD\n",
            "INFO:rosetta:core.pack.pack_missing_sidechains: packing residue number 269 because of missing atom number 7 atom name  OD1\n",
            "INFO:rosetta:core.pack.pack_missing_sidechains: packing residue number 378 because of missing atom number 7 atom name  CD\n",
            "INFO:rosetta:core.pack.pack_missing_sidechains: packing residue number 382 because of missing atom number 7 atom name  CD\n",
            "INFO:rosetta:core.pack.pack_missing_sidechains: packing residue number 408 because of missing atom number 7 atom name  CD\n",
            "INFO:rosetta:core.pack.task: Packer task: initialize from command line()\n",
            "INFO:rosetta:core.scoring.ScoreFunctionFactory: SCOREFUNCTION: ref2015\n",
            "INFO:rosetta:core.pack.pack_rotamers: built 167 rotamers at 9 positions.\n",
            "INFO:rosetta:core.pack.interaction_graph.interaction_graph_factory: Instantiating DensePDInteractionGraph\n",
            "INFO:rosetta:core.scoring.ScoreFunctionFactory: SCOREFUNCTION: ref2015\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "stream",
          "text": [
            "setup complete\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "Z8T097UAmdlK"
      },
      "source": [
        "Just like before, you can edit the resfile to your own personal specifications. Alternatively, you can use task operations to automate the process. Let's use task operations to fix all residues not at the interface."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "-0BH9v9smdlK"
      },
      "source": [
        "## Setting Designable Residues:\n",
        "\n",
        "Create a new task for design\n"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "nbgrader": {
          "grade": true,
          "grade_id": "cell-1754b479780ad2e4",
          "locked": false,
          "points": 0,
          "schema_version": 3,
          "solution": true
        },
        "id": "RXXSDPbcmdlL",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "b7d36ab7-fc1b-4d4b-b635-a17b729ddac5"
      },
      "source": [
        "from pyrosetta.rosetta.core.select.residue_selector import InterGroupInterfaceByVectorSelector, ChainSelector, NotResidueSelector\n",
        "\n",
        "chain1 = ChainSelector( \"1\" ) #selects the first chain\n",
        "chain2 = ChainSelector( \"2\" ) #selects the second chain\n",
        "\n",
        "interface_selector = InterGroupInterfaceByVectorSelector( chain1, chain2 );#selects residues at the interface\n",
        "not_interface_selector = NotResidueSelector( interface_selector ); #selects residues not at the interface\n",
        "\n",
        "from pyrosetta.rosetta.core.pack.task.operation import PreventRepackingRLT, RestrictToRepackingRLT, OperateOnResidueSubset\n",
        "\n",
        "#prevent non interface residues from repacking/designing\n",
        "fix_non_interface = OperateOnResidueSubset( PreventRepackingRLT(), not_interface_selector )\n",
        "\n",
        "#perhaps we are performing one-sided design and do not want to make mutations on chain 2:\n",
        "no_mutation_chain2 = OperateOnResidueSubset( RestrictToRepackingRLT(), chain2 )\n",
        "\n",
        "from pyrosetta.rosetta.core.pack.task import TaskFactory\n",
        "task_factory = TaskFactory()\n",
        "task_factory.push_back( fix_non_interface )\n",
        "task_factory.push_back( no_mutation_chain2 )\n",
        "\n",
        "task_design = task_factory.create_task_and_apply_taskoperations( pose )\n",
        "print( \"Num residues: \", pose.size() )\n",
        "print( \"Num packable residues: \", task_design.num_to_be_packed() ) # this includes the ones being designed\n",
        "\n",
        "num_designable = 0\n",
        "for i in range( 1, pose.size() + 1 ):\n",
        "    if( task_design.design_residue( i ) ):\n",
        "        num_designable += 1;\n",
        "print( \"Num designable residues: \", num_designable )"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Num residues:  454\n",
            "Num packable residues:  116\n",
            "Num designable residues:  53\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "-2x2hlAEmdlL"
      },
      "source": [
        "## Running HBNet\n",
        "\n",
        "This is an interface case so we will use HBNetStapleInterface.  We will use both the code-level interface, and the XML interface as an introduction to this functionality.  The XML interface to PyRosetta will be covered more in later workshops.  "
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "nbgrader": {
          "grade": true,
          "grade_id": "cell-9e50dd452de9681a",
          "locked": false,
          "points": 0,
          "schema_version": 3,
          "solution": true
        },
        "id": "-NLZygPzmdlL",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "f6ddb99e-67dd-47f3-86ab-53c067ba8378"
      },
      "source": [
        "from pyrosetta.rosetta.protocols.hbnet import HBNetStapleInterface\n",
        "\n",
        "#This is similar to the clone operation, \n",
        "# but instead of copying the original pose, we copy the dtat of start pose INTO the pose\n",
        "\n",
        "pose.assign(start_pose)\n",
        "\n",
        "# There are two ways to setup a mover:\n",
        "#   (1) by creating a blank instance of that mover and individually setting any non-default features\n",
        "#   (2) by passing an XML string, similar to rosetta_scripts\n",
        "# We will be using option 2 because it provides a more consistent interface to the movers\n",
        "# Plus, the store_network_scores_in_pose feature is only available using option 2 for versions of PyRosetta older than September-ish 2019\n",
        "setup_using_string = True #Option 2\n",
        "\n",
        "if setup_using_string:\n",
        "    hbnet = pyrosetta.rosetta.protocols.rosetta_scripts.XmlObjects.create_from_string(\"\"\"\n",
        "    <MOVERS>\n",
        "    <HBNetStapleInterface name=\"hbnet\" monte_carlo=\"true\" store_network_scores_in_pose=\"true\"/>\n",
        "    </MOVERS>\n",
        "    \"\"\").get_mover(\"hbnet\")\n",
        "\n",
        "    #monte_carlo=\"true\" is highly recommended, especially for large systems like asymmetric interfaces\n",
        "    #see PMID: 29652499\n",
        "else:\n",
        "    hbnet = HBNetStapleInterface()\n",
        "\n",
        "    #This is highly recommended, especially for large systems like asymmetric interfaces\n",
        "    #see PMID: 29652499\n",
        "    hbnet.set_monte_carlo_branch( True )\n",
        "\n",
        "    #set_monte_carlo_seed_must_be_buried does two things:\n",
        "    #(1) speeds us up by decreasing the sample space\n",
        "    #(2) ensures that our final hbond network will be at least partially buried\n",
        "    #hbnet.set_monte_carlo_seed_must_be_buried( True )\n",
        "    #commenting this out just to give us more results\n",
        "\n",
        "#You can toggle the verbosity here if you are interested\n",
        "#hbnet.verbose( True )\n",
        "\n",
        "#We can normallly leave this as the default\n",
        "#making it smaller now to let it run faster\n",
        "hbnet.set_total_num_mc_runs( 1000 )\n",
        "#Could have been done with\n",
        "#<HBNetStapleInterface name=\"hbnet\" monte_carlo=\"true\" total_num_mc_runs=\"1000\" store_network_scores_in_pose=\"true\"/>\n",
        "\n",
        "hbnet.task_factory( task_factory )\n",
        "#alternatively:\n",
        "#hbnet.set_task( task_design )\n",
        "\n",
        "hbnet.set_score_function( scorefxn )\n",
        "hbnet.apply(pose)\n",
        "\n",
        "print( \"Change in score\", scorefxn(pose) - scorefxn(start_pose) )\n"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Generating XML Schema for rosetta_scripts...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Initializing schema validator...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Validating input script...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Parsed script:\n",
            "<ROSETTASCRIPTS>\n",
            "\t<MOVERS>\n",
            "\t\t<HBNetStapleInterface monte_carlo=\"true\" name=\"hbnet\" store_network_scores_in_pose=\"true\"/>\n",
            "\t</MOVERS>\n",
            "\t<PROTOCOLS/>\n",
            "</ROSETTASCRIPTS>\n",
            "INFO:rosetta:core.scoring.ScoreFunctionFactory: SCOREFUNCTION: ref2015\n",
            "INFO:rosetta:core.scoring.etable: Starting energy table calculation\n",
            "INFO:rosetta:core.scoring.etable: smooth_etable: changing atr/rep split to bottom of energy well\n",
            "INFO:rosetta:core.scoring.etable: smooth_etable: spline smoothing lj etables (maxdis = 6)\n",
            "INFO:rosetta:core.scoring.etable: smooth_etable: spline smoothing solvation etables (max_dis = 6)\n",
            "INFO:rosetta:core.scoring.etable: Finished calculating energy tables.\n",
            "INFO:rosetta:basic.io.database: Database file opened: scoring/score_functions/PairEPotential/pdb_pair_stats_fine\n",
            "INFO:rosetta:basic.io.database: Database file opened: scoring/score_functions/InterchainPotential/interchain_env_log.txt\n",
            "INFO:rosetta:basic.io.database: Database file opened: scoring/score_functions/InterchainPotential/interchain_pair_log.txt\n",
            "INFO:rosetta:basic.io.database: Database file opened: scoring/score_functions/EnvPairPotential/env_log.txt\n",
            "INFO:rosetta:basic.io.database: Database file opened: scoring/score_functions/EnvPairPotential/cbeta_den.txt\n",
            "INFO:rosetta:basic.io.database: Database file opened: scoring/score_functions/EnvPairPotential/pair_log.txt\n",
            "INFO:rosetta:basic.io.database: Database file opened: scoring/score_functions/EnvPairPotential/cenpack_log.txt\n",
            "INFO:rosetta:basic.io.database: Database file opened: scoring/score_functions/rama/shapovalov/kappa25/all.ramaProb\n",
            "INFO:rosetta:core.scoring.ScoreFunctionFactory: SCOREFUNCTION: ref2015\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"hbnet\" of type HBNetStapleInterface\n",
            "INFO:rosetta:protocols.rosetta_scripts.ParsedProtocol: ParsedProtocol mover with the following settings\n",
            "INFO:rosetta:protocols.hbnet.HBNet:  Creating packer task based on specified task operations...\n",
            "INFO:rosetta:protocols.hbnet.HBNet:  Creating packer task based on specified task operations...\n",
            "INFO:rosetta:protocols.hbnet.HBNet:  built 9732 rotamers at 84 positions.\n",
            "INFO:rosetta:protocols.hbnet.HBNet:  IG: 1544332 bytes\n",
            "INFO:rosetta:protocols.hbnet.HBNet:  ==================================================================\n",
            "INFO:rosetta:protocols.hbnet.HBNet:  ============     PERFORMING H-BOND NETWORK DESIGN     ============\n",
            "INFO:rosetta:protocols.hbnet.HBNet:  ==================================================================\n",
            "INFO:rosetta:protocols.hbnet.HBNet: starting monte carlo branching protocol\n",
            "INFO:rosetta:protocols.hbnet.HBNet: \t10% done with branching\n",
            "INFO:rosetta:protocols.hbnet.HBNet: \t20% done with branching\n",
            "INFO:rosetta:protocols.hbnet.HBNet: \t30% done with branching\n",
            "INFO:rosetta:protocols.hbnet.HBNet: \t40% done with branching\n",
            "INFO:rosetta:protocols.hbnet.HBNet: \t50% done with branching\n",
            "INFO:rosetta:protocols.hbnet.HBNet: \t60% done with branching\n",
            "INFO:rosetta:protocols.hbnet.HBNet: \t70% done with branching\n",
            "INFO:rosetta:protocols.hbnet.HBNet: \t80% done with branching\n",
            "INFO:rosetta:protocols.hbnet.HBNet: \t90% done with branching\n",
            "INFO:rosetta:protocols.hbnet.HBNet: \t100% done with branching\n",
            "INFO:rosetta:protocols.hbnet.HBNet: Monte carlo branching protocol found 240 networks.\n",
            "INFO:rosetta:protocols.hbnet.HBNet:  NUMBER OF H-BOND NETWORKS AFTER BRANCH: 172\n",
            "INFO:rosetta:protocols.hbnet.HBNet:  NUMBER OF NETWORKS AFTER REMOVE REPLICATES = 168\n",
            "INFO:rosetta:protocols.hbnet.HBNet:  NUMBER OF NETWORKS AFTER SELECTION = 6\n",
            "INFO:rosetta:protocols.hbnet.HBNet:  NUMBER OF NETWORKS AFTER REMOVE_REP = 6\n",
            "INFO:rosetta:protocols.hbnet.HBNet:  Designed these new networks that meet your criteria:\n",
            "INFO:rosetta:protocols.hbnet.HBNet: \t#HBNet_rank \t residues \t size \t score \t num_hbonds \t percent_hbond_capacity \t num_unsat_Hpol \tinterf_hbs\n",
            "INFO:rosetta:protocols.hbnet.HBNet: \t#network_1\tD_Y_155,D_R_180,D_E_185,D_R_187,E_R_207,backbone\t5\t-0.650077\t8\t0.714286\t0\t3\n",
            "INFO:rosetta:protocols.hbnet.HBNet: \t#network_2\tD_Y_155,D_R_180,D_E_185,E_R_207\t4\t-0.746147\t5\t0.625\t0\t3\n",
            "INFO:rosetta:protocols.hbnet.HBNet: \t#network_3\tD_T_154,D_S_189,E_R_207\t3\t3.54027\t3\t0.555556\t0\t1\n",
            "INFO:rosetta:protocols.hbnet.HBNet: \t#network_4\tD_Y_155,D_E_185,E_R_207\t3\t-1.72747\t3\t0.545455\t0\t3\n",
            "INFO:rosetta:protocols.hbnet.HBNet: \t#network_5\tD_K_33,D_K_50,D_E_105,D_Q_116,E_R_107\t5\t4.56103\t5\t0.526316\t0\t2\n",
            "INFO:rosetta:protocols.hbnet.HBNet: \t#network_6\tD_R_136,D_E_155,E_K_152\t3\t0.070937\t3\t0.5\t0\t1\n",
            "INFO:rosetta:protocols.evaluation.ChiWellRmsdEvaluatorCreator: Evaluation Creator active ...\n",
            "INFO:rosetta:protocols.hbnet.HBNet:\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "stream",
          "text": [
            "Change in score 1209.278530664904\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "Gm7iLXi4mdlL"
      },
      "source": [
        "Wait, my score is terrible.\n",
        "\n",
        "__Question:__ Why?"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "jppaoQyOmdlM"
      },
      "source": [
        "# Because it's more precise this way"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "9zCyfLEvmdlM"
      },
      "source": [
        "## Finishing Design:\n",
        "\n",
        "Well of course the score is terrible, the pose is dense with clashes. We had 116 packable residues and only assigned states to 5 of them. The other 111 residues are still in their input conformations and likely clash with the 5 we just assigned."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "nUHzkee0mdlM"
      },
      "source": [
        "We need to run the packer (either using PackRotamersMover or FastDesign) but we don't want to overwrite the residues we just assigned with HBNet. The trick here is to select the residues with \"HBNet\" labels and fix them."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "nbgrader": {
          "grade": true,
          "grade_id": "cell-ecdd3a3881511698",
          "locked": false,
          "points": 0,
          "schema_version": 3,
          "solution": true
        },
        "id": "V4ipUJJBmdlM",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "23b93d32-f676-4bd6-cf9c-03347c4a83a3"
      },
      "source": [
        "from pyrosetta.rosetta.core.select.residue_selector import ResiduePDBInfoHasLabelSelector\n",
        "\n",
        "#prevent hbnet residues from repacking/designing\n",
        "hbnet_selector = ResiduePDBInfoHasLabelSelector( \"HBNet\" )\n",
        "fix_hbnet = OperateOnResidueSubset( PreventRepackingRLT(), hbnet_selector )\n",
        "task_factory.push_back( fix_hbnet ) #recycling the same factory as before, just adding a new operation\n",
        "task_design2 = task_factory.create_task_and_apply_taskoperations( pose )\n",
        "\n",
        "#sanity check\n",
        "num_hbnet_residues = 0\n",
        "for x in hbnet_selector.apply( pose ):\n",
        "    if x:\n",
        "        num_hbnet_residues += 1\n",
        "print( \"Num HBNet Residues\", num_hbnet_residues )\n",
        "\n",
        "#this is unrelated to the narrative but I highly recommend using the linear memory interaction graph whenever performing design. It's a huge speedup\n",
        "#it does not seem to matter for the scope here, but it will when you start using extra chi sampling (-ex1, -ex2)\n",
        "task_design2.or_linmem_ig( True )\n",
        "\n",
        "from pyrosetta.rosetta.protocols.minimization_packing import PackRotamersMover\n",
        "pack_mover = PackRotamersMover( scorefxn, task_design2 )\n",
        "pack_mover.apply( pose )\n",
        "print( \"Change in score\", scorefxn(pose) - scorefxn(start_pose) )"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Num HBNet Residues 5\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:core.pack.pack_rotamers: built 12444 rotamers at 110 positions.\n",
            "INFO:rosetta:core.pack.interaction_graph.interaction_graph_factory: Instantiating LinearMemoryInteractionGraph\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "stream",
          "text": [
            "Change in score 158.48651751526\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "oqbYMoAcmdlM"
      },
      "source": [
        "## We made it...?\n",
        "\n",
        "The change in score is a better, but still positive. One great thing about HBNet is that it can return multiple poses. Each one is slightly worse than the previous by HBNet's standards but might design into something better. Let's try a few to see if they help."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "Ly6tZbXomdlM",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "08d9ac89-e8e6-45d4-ac7b-443ad448eae5"
      },
      "source": [
        "#there were 10 (or so) networks total, but let's just try the next 5\n",
        "#this might take a few minutes...\n",
        "if not os.getenv('DEBUG'): #Adding this line to decrease runtime on the testing server\n",
        "    for x in range(0,5):\n",
        "        extra_pose = hbnet.get_additional_output()\n",
        "        if extra_pose is None:\n",
        "            break\n",
        "        task_design3 = task_factory.create_task_and_apply_taskoperations( extra_pose )\n",
        "        task_design3.or_linmem_ig( True )\n",
        "        pack_mover = PackRotamersMover( scorefxn, task_design3 )\n",
        "        pack_mover.apply( extra_pose )\n",
        "        print( \"Change in score\", scorefxn(extra_pose) - scorefxn(start_pose) )"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:protocols.hbnet.HBNet:\n",
            "INFO:rosetta:core.pack.pack_rotamers: built 12619 rotamers at 111 positions.\n",
            "INFO:rosetta:core.pack.interaction_graph.interaction_graph_factory: Instantiating LinearMemoryInteractionGraph\n",
            "INFO:rosetta:protocols.hbnet.HBNet:\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "stream",
          "text": [
            "Change in score 118.64912865170936\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:core.pack.pack_rotamers: built 12854 rotamers at 113 positions.\n",
            "INFO:rosetta:core.pack.interaction_graph.interaction_graph_factory: Instantiating LinearMemoryInteractionGraph\n",
            "INFO:rosetta:protocols.hbnet.HBNet:\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "stream",
          "text": [
            "Change in score 24.276594525971575\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:core.pack.pack_rotamers: built 12879 rotamers at 113 positions.\n",
            "INFO:rosetta:core.pack.interaction_graph.interaction_graph_factory: Instantiating LinearMemoryInteractionGraph\n",
            "INFO:rosetta:protocols.hbnet.HBNet:\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "stream",
          "text": [
            "Change in score 149.1720235493416\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:core.pack.pack_rotamers: built 11858 rotamers at 111 positions.\n",
            "INFO:rosetta:core.pack.interaction_graph.interaction_graph_factory: Instantiating LinearMemoryInteractionGraph\n",
            "INFO:rosetta:protocols.hbnet.HBNet:\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "stream",
          "text": [
            "Change in score -44.815447481188414\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:core.pack.pack_rotamers: built 12826 rotamers at 113 positions.\n",
            "INFO:rosetta:core.pack.interaction_graph.interaction_graph_factory: Instantiating LinearMemoryInteractionGraph\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "stream",
          "text": [
            "Change in score -51.83529961721587\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "IJlg7oCWmdlN"
      },
      "source": [
        "## But wait, there's more??\n",
        "\n",
        "Great! We found a few results that designed to me more stable than the input pose (-60, -45, and -31 REU)!\n",
        "\n",
        "The main score function is not the only way to evaluate these networks. HBNet also adds its own score terms. These are useful for sorting/filtering decoys before running expensive packing calculations."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "v3dY-VvcmdlN",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "48c9f682-949f-4aee-b731-43ed06e33f24"
      },
      "source": [
        "from pyrosetta.rosetta.core.pose import hasPoseExtraScore, getPoseExtraScore\n",
        "\n",
        "if hasPoseExtraScore( pose, \"HBNet_NumUnsatHpol\" ):\n",
        "    #All 3 of these metrics are explained in more detail in Maguire, Boyken, et al. (see second reference below)\n",
        "\n",
        "    #NumUnsatHpol is HBNet's primary sorting metric, it counts the number of polar hydrogen atoms that are unsatisfied (buried and not forming hbonds). We know that there are no heavy (non-hydrogen) unsatisfied atoms because HBNet filters those out by default. Lower is better\n",
        "    print( \"HBNet_NumUnsatHpol\", getPoseExtraScore( pose, \"HBNet_NumUnsatHpol\" ) )\n",
        "\n",
        "    #HBNet's secondary sorting metric. 1.0 if every polar atom in the network is forming the maximum number of hbonds. Higher is better\n",
        "    print( \"HBNet_Saturation\", getPoseExtraScore( pose, \"HBNet_Saturation\" ) )\n",
        "\n",
        "    #HBNet's tertiary sorting metric. A little complicated but lower is better.\n",
        "    print( \"HBNet_Score\", getPoseExtraScore( pose, \"HBNet_Score\" ) )\n",
        "else:\n",
        "    print( \"Somebody go bug a developer to enable this feature for PyRosetta\" )"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "HBNet_NumUnsatHpol 0.0\n",
            "HBNet_Saturation 0.7142857313156128\n",
            "HBNet_Score -0.6500769257545471\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "C2-l2kQ3mdlN"
      },
      "source": [
        "## Advice for using this in the wild\n",
        "\n",
        "#### ex1 ex2\n",
        "\n",
        "HBonds are very sensitive to sidechain sampling resolution. I highly recommend using -ex1 and -ex2. You can do this by adding:\n",
        "```py\n",
        "ex1ex2 = ExtraRotamersGeneric()\n",
        "ex1ex2.ex1( True )\n",
        "ex1ex2.ex2( True )\n",
        "task_factory.push_back( ex1ex2 )\n",
        "```\n",
        "\n",
        "#### get_additional_output\n",
        "\n",
        "As we saw in the exercise, the first result out of HBNet does not always wind up being the best. Try designing with a few results from `hbnet.get_additional_output()` to get more coverage of the design space. For the commandline users reading this, this functionality can also be accessed via `multistage_rosetta_scripts` or the `MultiplePoseMover` in `rosetta_scripts`. See the `rosetta_scripts_scripts` repository for examples.\n",
        "\n",
        "#### set_monte_carlo_seed_must_be_buried\n",
        "\n",
        "I highly recommend playing with the `set_monte_carlo_seed_must_be_buried` mentioned above. Without it, HBNet tends to just design many surface networks that nobody really cares about."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "4b85xzpymdlN"
      },
      "source": [
        "## Thought Question\n",
        "\n",
        "The energy of HBNet+Design is often less favorable that the energy after an equivalent design run without HBNet. Why do people still use HBNet?\n",
        "\n",
        "\n",
        "## References\n",
        "\n",
        "\n",
        "- Boyken SE, Chen Z, Groves B, et al. De novo design of protein homo-oligomers with modular hydrogen-bond network-mediated specificity. Science. 2016;352(6286):680687. doi:10.1126/science.aad8865\n",
        "\n",
        "\n",
        "- Maguire JB, Boyken SE, Baker D, Kuhlman B. Rapid Sampling of Hydrogen Bond Networks for Computational Protein Design. J Chem Theory Comput. 2018;14(5):27512760. doi:10.1021/acs.jctc.8b00033"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "Y1FHFC7XmdlN"
      },
      "source": [
        ""
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "SJoQItW8mdlO"
      },
      "source": [
        "# *De Novo* Parametric Backbone Design\n",
        "\n",
        "Keywords: Parametric, Bundle, BundleGridSampler, MakeBundle\n",
        "\n",
        "## Overview\n",
        "\n",
        "In this workshop, we will give examples for how to do Parametric Protein Design using the RosettaScripts interface to Rosetta written. The code you will be using from Rosetta was written by Vikram Mulligan.  The BundleGridSampler and MakeBundle movers are available through code in PyRosetta, but because of the many number of options to it, it is a bit easier to work with these in XML. We will use this opportunity to do see how easy it is to run XML in PyRosetta, but we will cover the code-level interface as well in this workshop.  \n",
        "\n",
        "This is some cool stuff. Enjoy. :)\n",
        "\n",
        "Please refer to the docs for more info.\n",
        "\n",
        "https://www.rosettacommons.org/docs/latest/scripting_documentation/RosettaScripts/Movers/Movers-RosettaScripts\n",
        "\n",
        "https://www.rosettacommons.org/docs/latest/scripting_documentation/RosettaScripts/Movers/movers_pages/MakeBundleMover\n",
        "\n",
        "https://www.rosettacommons.org/docs/latest/scripting_documentation/RosettaScripts/Movers/movers_pages/BundleGridSamplerMover"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "ywQSUSKemdlO",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 17
        },
        "outputId": "e018a998-2e58-4bec-8717-85e37a48a635"
      },
      "source": [
        "import logging\n",
        "logging.basicConfig(level=logging.INFO)\n",
        "import pyrosetta\n",
        "\n",
        "from IPython.core.display import display, HTML\n",
        "display(HTML(\"<style>.container { width:100% !important; }</style>\"))"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/html": [
              "<style>.container { width:100% !important; }</style>"
            ],
            "text/plain": [
              "<IPython.core.display.HTML object>"
            ]
          },
          "metadata": {
            "tags": []
          }
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "TK3gD24qmdlO",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "b6b2612b-0b85-49ae-9d4c-bc35a6e655eb"
      },
      "source": [
        "pyrosetta.init(\"-mute all\")"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:pyrosetta.rosetta:Found rosetta database at: /usr/local/lib/python3.7/dist-packages/pyrosetta/database; using it....\n",
            "INFO:pyrosetta.rosetta:PyRosetta-4 2021 [Rosetta PyRosetta4.MinSizeRel.python37.ubuntu 2020.50.post.dev+970.commits.3700df145608444753aabbec9c4681ec9b21f74b 2021-02-24T13:24:53] retrieved from: http://www.pyrosetta.org\n",
            "(C) Copyright Rosetta Commons Member Institutions. Created in JHU by Sergey Lyskov and PyRosetta Team.\n",
            "INFO:rosetta:core.init: Checking for fconfig files in pwd and ./rosetta/flags\n",
            "INFO:rosetta:core.init: Rosetta version: PyRosetta4.MinSizeRel.python37.ubuntu r17132 2020.50.post.dev+970.commits.3700df14560 3700df145608444753aabbec9c4681ec9b21f74b http://www.pyrosetta.org 2021-02-24T13:24:53\n",
            "INFO:rosetta:core.init: command: PyRosetta -mute all -database /usr/local/lib/python3.7/dist-packages/pyrosetta/database\n",
            "INFO:rosetta:basic.random.init_random_generator: 'RNG device' seed mode, using '/dev/urandom', seed=1786069850 seed_offset=0 real_seed=1786069850\n",
            "INFO:rosetta:basic.random.init_random_generator: RandomGenerator:init: Normal mode, seed=1786069850 RG_type=mt19937\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "stream",
          "text": [
            "PyRosetta-4 2021 [Rosetta PyRosetta4.MinSizeRel.python37.ubuntu 2020.50.post.dev+970.commits.3700df145608444753aabbec9c4681ec9b21f74b 2021-02-24T13:24:53] retrieved from: http://www.pyrosetta.org\n",
            "(C) Copyright Rosetta Commons Member Institutions. Created in JHU by Sergey Lyskov and PyRosetta Team.\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "Ytf4rLsUmdlO"
      },
      "source": [
        "## Overview\n",
        "\n",
        "Parametric sampling means changing specific angles of groups of bundles (coiled-coils) in order to sample different conformations.  The equations that are used for Parametric sampling were originally described by Crick and are called Crick equations. \n",
        "\n",
        "These equations were reintroduced to the scientific community through William F Degrado's seminal paper, \"Probing designability via a generalized model of helical bundle geometry\"\n",
        "\n",
        "Since the puplication of that paper, researchers have used this method to create bundles of all sorts.  Fibers, membrane proteins, and even working antiporters!  \n",
        "\n",
        "\n",
        "## Main Rosetta References (in order of use of tools)\n",
        "\n",
        "- Dang B, Wu H, Mulligan VK, Mravic M, Wu Y, Lemmin T, Ford A, Silva DA, Baker D, DeGrado WF.\n",
        "\n",
        "   **\"De novo design of covalently constrained mesosize protein scaffolds with unique tertiary structures.\"**\n",
        "   \n",
        "   *Proc Natl Acad Sci U S A*. 2017 Oct 10;114(41):10852-10857.\n",
        "   https://www.ncbi.nlm.nih.gov/pubmed/28973862\n",
        "   \n",
        "   \n",
        "- Lu P,Min D, DiMaio F, Wei KY, Vahey MD, Boyken SE, Mulligan, et al.\n",
        "\n",
        "   **\"Accurate computational design of multipass transmembrane proteins.\"**\n",
        "   \n",
        "   *Science*. 2018 Mar 2;359(6379):1042-1046. \n",
        "   https://www.ncbi.nlm.nih.gov/pubmed/29496880\n",
        "   \n",
        "   \n",
        "- Chen Z, Boyken SE, Jia M, Busch F, Flores-Solis D, Mulligan, et al.\n",
        "\n",
        "   **\"Programmable design of orthogonal protein heterodimers.\"**\n",
        "   \n",
        "   *Nature*. 2019 Jan;565(7737):106-111. \n",
        "   https://www.ncbi.nlm.nih.gov/pubmed/30568301\n",
        "   \n",
        "   \n",
        "- Langan RA Boyken SE, Ng AH, Samson JA, Dods G, Mulligan, et al.\n",
        "\n",
        "  **\"De novo design of bioactive protein switches.\"**\n",
        "  \n",
        "  *Nature*. 2019 Aug;572(7768):205-210.\n",
        "  https://www.ncbi.nlm.nih.gov/pubmed/31341284\n",
        "  \n",
        "  \n",
        "## Parameter tips (From Vikram himself)\n",
        "\n",
        "- Most bundles would have r0 from 3 to 10 A (depending on the helix count).\n",
        "- Omega0 should range from about -2.5 degrees to 2.5 degrees.\n",
        "- Delta_omega1 can be anywhere from 0 to 360.  (It's just the roll of the helix about its own axis).\n",
        "- Delta_z0, delta_z1 or delta_t can be anything (you're just sliding the helix up and down).  Typically, you'd just sample around a few Angstroms, though.\n",
        "- r1, omega1, etc. are all minor helix parameters that are tied to the secondary structure.  We don't normally vary them."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "YUCE-fJSmdlO"
      },
      "source": [
        "  \n",
        "## Creating multiple Helices\n",
        "The Helix lines tell the sampler to create a helix and how to do so. We set them at 90 degree rotations through this line, but we could add additional parameters as well.  These parameters are VERY WELL documented on the link above, so make sure to use that:\n",
        "\n",
        "**delta_omega0**: Rotation of a helix about the z-axis, stored in radians.\n",
        "\n",
        "### Changing r0:\n",
        "\n",
        "Here, we will sample on the r0 axis.  To make this a bit easier, we will set dump_pdbs to 1. You can also put this code in a loop and sample on that as well, writing each to pymol.  You would set r0 instead of r0_max and min.\n",
        "\n",
        "**r0**: Major helix radius, in Angstroms."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "4M4JYTKRmdlO",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "2f3ca22a-e968-4992-a0bc-ba3072065bb2"
      },
      "source": [
        "pose = pyrosetta.rosetta.core.pose.Pose()\n",
        "pyrosetta.rosetta.protocols.rosetta_scripts.XmlObjects.create_from_string(\"\"\"\n",
        "<SCOREFXNS>\n",
        "  <ScoreFunction name=\"sfxn1\" weights=\"ref2015\"/>\n",
        "</SCOREFXNS>\n",
        "<MOVERS>\n",
        "  <BundleGridSampler name=\"bgs1\"\n",
        "                     helix_length=\"20\"\n",
        "                     scorefxn=\"sfxn1\"\n",
        "                     r0_min=\"1.0\"\n",
        "                     r0_max=\"10.0\"\n",
        "                     r0_samples=\"10\"\n",
        "                     omega0=\"0.05\"\n",
        "                     delta_omega0=\"0\"\n",
        "                     delta_omega1=\"0\"\n",
        "                     delta_t=\"0\"\n",
        "                     dump_pdbs=\"1\">\n",
        "  <Helix/>\n",
        "  <Helix delta_omega0=\"3.14\" r0_copies_helix=\"1\"/>\n",
        "</BundleGridSampler>\n",
        "</MOVERS>\n",
        "\"\"\").get_mover(\"bgs1\").apply(pose)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Generating XML Schema for rosetta_scripts...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Initializing schema validator...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Validating input script...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Parsed script:\n",
            "<ROSETTASCRIPTS>\n",
            "\t<SCOREFXNS>\n",
            "\t\t<ScoreFunction name=\"sfxn1\" weights=\"ref2015\"/>\n",
            "\t</SCOREFXNS>\n",
            "\t<MOVERS>\n",
            "\t\t<BundleGridSampler delta_omega0=\"0\" delta_omega1=\"0\" delta_t=\"0\" dump_pdbs=\"1\" helix_length=\"20\" name=\"bgs1\" omega0=\"0.05\" r0_max=\"10.0\" r0_min=\"1.0\" r0_samples=\"10\" scorefxn=\"sfxn1\">\n",
            "\t\t\t<Helix/>\n",
            "\t\t\t<Helix delta_omega0=\"3.14\" r0_copies_helix=\"1\"/>\n",
            "\t\t</BundleGridSampler>\n",
            "\t</MOVERS>\n",
            "\t<PROTOCOLS/>\n",
            "</ROSETTASCRIPTS>\n",
            "INFO:rosetta:core.scoring.ScoreFunctionFactory: SCOREFUNCTION: ref2015\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"bgs1\" of type BundleGridSampler\n",
            "INFO:rosetta:protocols.rosetta_scripts.ParsedProtocol: ParsedProtocol mover with the following settings\n"
          ],
          "name": "stderr"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 817
        },
        "id": "7NsrURWEYTmd",
        "outputId": "89e57b40-b5aa-41a4-dde7-6ea2ebc2f7bc"
      },
      "source": [
        "view_pose(pose)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_16234205772799664\"  style=\"position: relative; width: 1200px; height: 800px\">\n        <p id=\"3dmolwarning_16234205772799664\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n      resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n}\n\nvar viewer_16234205772799664 = null;\nvar warn = document.getElementById(\"3dmolwarning_16234205772799664\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_16234205772799664 = $3Dmol.createViewer($(\"#3dmolviewer_16234205772799664\"),{backgroundColor:\"white\"});\nviewer_16234205772799664.zoomTo();\n\tviewer_16234205772799664.addModels(\"HEADER                                            11-JUN-21   XXXX              \\nEXPDTA    THEORETICAL MODEL                                                     \\nREMARK 220                                                                      \\nREMARK 220 EXPERIMENTAL DETAILS                                                 \\nREMARK 220  EXPERIMENT TYPE                : THEORETICAL MODELLING              \\nREMARK 220  DATE OF DATA COLLECTION        : 11-JUN-21                          \\nREMARK 220                                                                      \\nREMARK 220 REMARK: MODEL GENERATED BY ROSETTA                                   \\nREMARK 220  VERSION 2020.50.post.dev+970.commits.3700df14560                    \\nREMARK 250 ----------------------------------------                             \\nREMARK 250 CRICK PARAMETER INFORMATION                                          \\nREMARK 250 Bundle symmetry: 0 (not symmetric)                                   \\nREMARK 250 Bundle symmetry copies: 0                                            \\nREMARK 250 Number of helices per copy: 2                                        \\nREMARK 250 ---HELIX 1 PARAMETERS:---                                            \\nREMARK 250  PARAMETERS THAT ARE TYPICALLY SAMPLED:                              \\nREMARK 250    Major radius (r0,Angstroms): 5.00000000                           \\nREMARK 250    Major twist (omega0,radians/residue): 0.05000000                  \\nREMARK 250    Rotation about z-axis (delta_omega0,radians): 0.00000000          \\nREMARK 250    Roll about helix axis (delta_omega1,radians): 0.00000000          \\nREMARK 250    Registry shift (delta_t,residues): 0.00000000                     \\nREMARK 250    Major axial offset (z0_offset,Angstroms): 0.00000000              \\nREMARK 250    Minor axial offset (z1_offset,vert. Angstroms): 0.000000          \\nREMARK 250    Lateral squash (epsilon,dimensionless): 1.00000000                \\nREMARK 250  PARAMETERS THAT ARE NOT TYPICALLY SAMPLED:                          \\nREMARK 250    Residues/repeat (residues_per_repeat,dimensionless): 1            \\nREMARK 250    Repeat unit offset (repeating_unit_offset,residues): 0            \\nREMARK 250    Atoms/residue (atoms_per_residue,dimensionless):                  \\nREMARK 250      [3]                                                             \\nREMARK 250    Minor radius (r1_peratom,Angstroms):                              \\nREMARK 250      [1.53673520, 2.29431130, 1.73316244]                            \\nREMARK 250    Minor twist (omega1,radians/residue): 1.72180480                  \\nREMARK 250    Minor rise (z1,Angstroms/residue): 1.53104401                     \\nREMARK 250    Atom angular offset (delta_omega1_peratom,radians):               \\nREMARK 250      [-0.46010617, 0.00000000, 0.48012932]                           \\nREMARK 250    Atom axial offset (delta_z1_peratom,Angstroms):                   \\nREMARK 250      [-0.90470326, 0.00000000, 1.05169636]                           \\nREMARK 250    Invert helix (invert,Boolean): FALSE                              \\nREMARK 250    Dihedrals set (set_dihedrals,Boolean): TRUE                       \\nREMARK 250    Nonideal bond angles (set_bondangles,Boolean): TRUE               \\nREMARK 250    Nonideal bond lengths (set_bondlengths,Boolean): TRUE             \\nREMARK 250 ---HELIX 2 PARAMETERS:---                                            \\nREMARK 250  PARAMETERS THAT ARE TYPICALLY SAMPLED:                              \\nREMARK 250    Major radius (r0,Angstroms): 5.00000000                           \\nREMARK 250    Major twist (omega0,radians/residue): 0.05000000                  \\nREMARK 250    Rotation about z-axis (delta_omega0,radians): 3.14000000          \\nREMARK 250    Roll about helix axis (delta_omega1,radians): 0.00000000          \\nREMARK 250    Registry shift (delta_t,residues): 0.00000000                     \\nREMARK 250    Major axial offset (z0_offset,Angstroms): 0.00000000              \\nREMARK 250    Minor axial offset (z1_offset,vert. Angstroms): 0.000000          \\nREMARK 250    Lateral squash (epsilon,dimensionless): 1.00000000                \\nREMARK 250  PARAMETERS THAT ARE NOT TYPICALLY SAMPLED:                          \\nREMARK 250    Residues/repeat (residues_per_repeat,dimensionless): 1            \\nREMARK 250    Repeat unit offset (repeating_unit_offset,residues): 0            \\nREMARK 250    Atoms/residue (atoms_per_residue,dimensionless):                  \\nREMARK 250      [3]                                                             \\nREMARK 250    Minor radius (r1_peratom,Angstroms):                              \\nREMARK 250      [1.53673520, 2.29431130, 1.73316244]                            \\nREMARK 250    Minor twist (omega1,radians/residue): 1.72180480                  \\nREMARK 250    Minor rise (z1,Angstroms/residue): 1.53104401                     \\nREMARK 250    Atom angular offset (delta_omega1_peratom,radians):               \\nREMARK 250      [-0.46010617, 0.00000000, 0.48012932]                           \\nREMARK 250    Atom axial offset (delta_z1_peratom,Angstroms):                   \\nREMARK 250      [-0.90470326, 0.00000000, 1.05169636]                           \\nREMARK 250    Invert helix (invert,Boolean): FALSE                              \\nREMARK 250    Dihedrals set (set_dihedrals,Boolean): TRUE                       \\nREMARK 250    Nonideal bond angles (set_bondangles,Boolean): TRUE               \\nREMARK 250    Nonideal bond lengths (set_bondlengths,Boolean): TRUE             \\nREMARK 250 ----------------------------------------                             \\nATOM      1  N   ALA A   1       4.905  -1.129 -16.247  1.00  0.00           N  \\nATOM      2  CA  ALA A   1       4.236  -0.130 -15.422  1.00  0.00           C  \\nATOM      3  C   ALA A   1       3.572  -0.771 -14.210  1.00  0.00           C  \\nATOM      4  O   ALA A   1       3.624  -0.232 -13.105  1.00  0.00           O  \\nATOM      5  CB  ALA A   1       3.210   0.635 -16.245  1.00  0.00           C  \\nATOM      6  H   ALA A   1       4.539  -2.070 -16.274  1.00  0.00           H  \\nATOM      7  HA  ALA A   1       4.990   0.569 -15.059  1.00  0.00           H  \\nATOM      8 1HB  ALA A   1       2.719   1.377 -15.615  1.00  0.00           H  \\nATOM      9 2HB  ALA A   1       3.708   1.136 -17.074  1.00  0.00           H  \\nATOM     10 3HB  ALA A   1       2.466  -0.058 -16.634  1.00  0.00           H  \\nATOM     11  N   ALA A   2       2.950  -1.926 -14.437  1.00  0.00           N  \\nATOM     12  CA  ALA A   2       2.267  -2.640 -13.364  1.00  0.00           C  \\nATOM     13  C   ALA A   2       3.251  -3.107 -12.300  1.00  0.00           C  \\nATOM     14  O   ALA A   2       2.971  -3.027 -11.104  1.00  0.00           O  \\nATOM     15  CB  ALA A   2       1.493  -3.824 -13.925  1.00  0.00           C  \\nATOM     16  H   ALA A   2       2.954  -2.312 -15.370  1.00  0.00           H  \\nATOM     17  HA  ALA A   2       1.566  -1.952 -12.892  1.00  0.00           H  \\nATOM     18 1HB  ALA A   2       0.989  -4.347 -13.112  1.00  0.00           H  \\nATOM     19 2HB  ALA A   2       0.753  -3.469 -14.642  1.00  0.00           H  \\nATOM     20 3HB  ALA A   2       2.182  -4.506 -14.422  1.00  0.00           H  \\nATOM     21  N   ALA A   3       4.406  -3.594 -12.737  1.00  0.00           N  \\nATOM     22  CA  ALA A   3       5.428  -4.072 -11.813  1.00  0.00           C  \\nATOM     23  C   ALA A   3       5.969  -2.935 -10.955  1.00  0.00           C  \\nATOM     24  O   ALA A   3       6.195  -3.102  -9.757  1.00  0.00           O  \\nATOM     25  CB  ALA A   3       6.560  -4.743 -12.576  1.00  0.00           C  \\nATOM     26  H   ALA A   3       4.578  -3.635 -13.731  1.00  0.00           H  \\nATOM     27  HA  ALA A   3       4.968  -4.803 -11.148  1.00  0.00           H  \\nATOM     28 1HB  ALA A   3       7.315  -5.094 -11.873  1.00  0.00           H  \\nATOM     29 2HB  ALA A   3       6.168  -5.590 -13.139  1.00  0.00           H  \\nATOM     30 3HB  ALA A   3       7.010  -4.028 -13.264  1.00  0.00           H  \\nATOM     31  N   ALA A   4       6.178  -1.778 -11.565  1.00  0.00           N  \\nATOM     32  CA  ALA A   4       6.687  -0.609 -10.858  1.00  0.00           C  \\nATOM     33  C   ALA A   4       5.693  -0.122  -9.812  1.00  0.00           C  \\nATOM     34  O   ALA A   4       6.078   0.253  -8.704  1.00  0.00           O  \\nATOM     35  CB  ALA A   4       7.008   0.506 -11.843  1.00  0.00           C  \\nATOM     36  H   ALA A   4       5.979  -1.700 -12.552  1.00  0.00           H  \\nATOM     37  HA  ALA A   4       7.602  -0.897 -10.341  1.00  0.00           H  \\nATOM     38 1HB  ALA A   4       7.387   1.373 -11.300  1.00  0.00           H  \\nATOM     39 2HB  ALA A   4       7.763   0.162 -12.549  1.00  0.00           H  \\nATOM     40 3HB  ALA A   4       6.105   0.784 -12.384  1.00  0.00           H  \\nATOM     41  N   ALA A   5       4.415  -0.131 -10.175  1.00  0.00           N  \\nATOM     42  CA  ALA A   5       3.356   0.311  -9.276  1.00  0.00           C  \\nATOM     43  C   ALA A   5       3.248  -0.600  -8.061  1.00  0.00           C  \\nATOM     44  O   ALA A   5       3.060  -0.133  -6.938  1.00  0.00           O  \\nATOM     45  CB  ALA A   5       2.026   0.370 -10.013  1.00  0.00           C  \\nATOM     46  H   ALA A   5       4.169  -0.454 -11.100  1.00  0.00           H  \\nATOM     47  HA  ALA A   5       3.608   1.311  -8.923  1.00  0.00           H  \\nATOM     48 1HB  ALA A   5       1.245   0.702  -9.329  1.00  0.00           H  \\nATOM     49 2HB  ALA A   5       2.102   1.071 -10.845  1.00  0.00           H  \\nATOM     50 3HB  ALA A   5       1.778  -0.619 -10.394  1.00  0.00           H  \\nATOM     51  N   ALA A   6       3.370  -1.904  -8.302  1.00  0.00           N  \\nATOM     52  CA  ALA A   6       3.279  -2.882  -7.225  1.00  0.00           C  \\nATOM     53  C   ALA A   6       4.422  -2.716  -6.232  1.00  0.00           C  \\nATOM     54  O   ALA A   6       4.225  -2.813  -5.021  1.00  0.00           O  \\nATOM     55  CB  ALA A   6       3.270  -4.294  -7.790  1.00  0.00           C  \\nATOM     56  H   ALA A   6       3.529  -2.216  -9.250  1.00  0.00           H  \\nATOM     57  HA  ALA A   6       2.345  -2.710  -6.690  1.00  0.00           H  \\nATOM     58 1HB  ALA A   6       3.201  -5.013  -6.974  1.00  0.00           H  \\nATOM     59 2HB  ALA A   6       2.413  -4.415  -8.453  1.00  0.00           H  \\nATOM     60 3HB  ALA A   6       4.188  -4.468  -8.349  1.00  0.00           H  \\nATOM     61  N   ALA A   7       5.620  -2.466  -6.744  1.00  0.00           N  \\nATOM     62  CA  ALA A   7       6.791  -2.285  -5.895  1.00  0.00           C  \\nATOM     63  C   ALA A   7       6.654  -1.041  -5.026  1.00  0.00           C  \\nATOM     64  O   ALA A   7       7.012  -1.052  -3.848  1.00  0.00           O  \\nATOM     65  CB  ALA A   7       8.052  -2.201  -6.742  1.00  0.00           C  \\nATOM     66  H   ALA A   7       5.722  -2.401  -7.746  1.00  0.00           H  \\nATOM     67  HA  ALA A   7       6.867  -3.148  -5.233  1.00  0.00           H  \\nATOM     68 1HB  ALA A   7       8.918  -2.065  -6.094  1.00  0.00           H  \\nATOM     69 2HB  ALA A   7       8.168  -3.122  -7.314  1.00  0.00           H  \\nATOM     70 3HB  ALA A   7       7.975  -1.357  -7.426  1.00  0.00           H  \\nATOM     71  N   ALA A   8       6.136   0.031  -5.605  1.00  0.00           N  \\nATOM     72  CA  ALA A   8       5.944   1.287  -4.889  1.00  0.00           C  \\nATOM     73  C   ALA A   8       4.919   1.132  -3.772  1.00  0.00           C  \\nATOM     74  O   ALA A   8       5.098   1.662  -2.675  1.00  0.00           O  \\nATOM     75  CB  ALA A   8       5.516   2.386  -5.851  1.00  0.00           C  \\nATOM     76  H   ALA A   8       5.863  -0.018  -6.577  1.00  0.00           H  \\nATOM     77  HA  ALA A   8       6.894   1.566  -4.434  1.00  0.00           H  \\nATOM     78 1HB  ALA A   8       5.377   3.316  -5.301  1.00  0.00           H  \\nATOM     79 2HB  ALA A   8       6.286   2.524  -6.610  1.00  0.00           H  \\nATOM     80 3HB  ALA A   8       4.580   2.104  -6.331  1.00  0.00           H  \\nATOM     81  N   ALA A   9       3.847   0.402  -4.068  1.00  0.00           N  \\nATOM     82  CA  ALA A   9       2.784   0.177  -3.096  1.00  0.00           C  \\nATOM     83  C   ALA A   9       3.287  -0.629  -1.906  1.00  0.00           C  \\nATOM     84  O   ALA A   9       2.942  -0.343  -0.759  1.00  0.00           O  \\nATOM     85  CB  ALA A   9       1.607  -0.529  -3.753  1.00  0.00           C  \\nATOM     86  H   ALA A   9       3.769  -0.006  -4.988  1.00  0.00           H  \\nATOM     87  HA  ALA A   9       2.452   1.147  -2.726  1.00  0.00           H  \\nATOM     88 1HB  ALA A   9       0.821  -0.690  -3.015  1.00  0.00           H  \\nATOM     89 2HB  ALA A   9       1.221   0.087  -4.566  1.00  0.00           H  \\nATOM     90 3HB  ALA A   9       1.934  -1.488  -4.150  1.00  0.00           H  \\nATOM     91  N   ALA A  10       4.105  -1.639  -2.191  1.00  0.00           N  \\nATOM     92  CA  ALA A  10       4.650  -2.487  -1.138  1.00  0.00           C  \\nATOM     93  C   ALA A  10       5.564  -1.697  -0.210  1.00  0.00           C  \\nATOM     94  O   ALA A  10       5.534  -1.877   1.007  1.00  0.00           O  \\nATOM     95  CB  ALA A  10       5.401  -3.665  -1.742  1.00  0.00           C  \\nATOM     96  H   ALA A  10       4.351  -1.816  -3.154  1.00  0.00           H  \\nATOM     97  HA  ALA A  10       3.818  -2.867  -0.544  1.00  0.00           H  \\nATOM     98 1HB  ALA A  10       5.802  -4.289  -0.943  1.00  0.00           H  \\nATOM     99 2HB  ALA A  10       4.720  -4.254  -2.356  1.00  0.00           H  \\nATOM    100 3HB  ALA A  10       6.219  -3.296  -2.359  1.00  0.00           H  \\nATOM    101  N   ALA A  11       6.377  -0.821  -0.781  1.00  0.00           N  \\nATOM    102  CA  ALA A  11       7.294   0.000   0.000  1.00  0.00           C  \\nATOM    103  C   ALA A  11       6.537   0.962   0.907  1.00  0.00           C  \\nATOM    104  O   ALA A  11       6.914   1.169   2.060  1.00  0.00           O  \\nATOM    105  CB  ALA A  11       8.232   0.768  -0.920  1.00  0.00           C  \\nATOM    106  H   ALA A  11       6.362  -0.718  -1.786  1.00  0.00           H  \\nATOM    107  HA  ALA A  11       7.885  -0.662   0.634  1.00  0.00           H  \\nATOM    108 1HB  ALA A  11       8.911   1.376  -0.322  1.00  0.00           H  \\nATOM    109 2HB  ALA A  11       8.809   0.065  -1.521  1.00  0.00           H  \\nATOM    110 3HB  ALA A  11       7.650   1.413  -1.576  1.00  0.00           H  \\nATOM    111  N   ALA A  12       5.470   1.547   0.383  1.00  0.00           N  \\nATOM    112  CA  ALA A  12       4.650   2.487   1.138  1.00  0.00           C  \\nATOM    113  C   ALA A  12       3.963   1.801   2.312  1.00  0.00           C  \\nATOM    114  O   ALA A  12       3.883   2.357   3.407  1.00  0.00           O  \\nATOM    115  CB  ALA A  12       3.618   3.139   0.229  1.00  0.00           C  \\nATOM    116  H   ALA A  12       5.212   1.337  -0.571  1.00  0.00           H  \\nATOM    117  HA  ALA A  12       5.305   3.261   1.540  1.00  0.00           H  \\nATOM    118 1HB  ALA A  12       3.014   3.839   0.807  1.00  0.00           H  \\nATOM    119 2HB  ALA A  12       4.126   3.675  -0.573  1.00  0.00           H  \\nATOM    120 3HB  ALA A  12       2.975   2.372  -0.199  1.00  0.00           H  \\nATOM    121  N   ALA A  13       3.473   0.589   2.064  1.00  0.00           N  \\nATOM    122  CA  ALA A  13       2.784  -0.177   3.096  1.00  0.00           C  \\nATOM    123  C   ALA A  13       3.727  -0.537   4.237  1.00  0.00           C  \\nATOM    124  O   ALA A  13       3.353  -0.473   5.408  1.00  0.00           O  \\nATOM    125  CB  ALA A  13       2.170  -1.436   2.501  1.00  0.00           C  \\nATOM    126  H   ALA A  13       3.581   0.193   1.142  1.00  0.00           H  \\nATOM    127  HA  ALA A  13       1.987   0.445   3.505  1.00  0.00           H  \\nATOM    128 1HB  ALA A  13       1.659  -1.997   3.283  1.00  0.00           H  \\nATOM    129 2HB  ALA A  13       1.454  -1.160   1.726  1.00  0.00           H  \\nATOM    130 3HB  ALA A  13       2.955  -2.052   2.066  1.00  0.00           H  \\nATOM    131  N   ALA A  14       4.952  -0.915   3.887  1.00  0.00           N  \\nATOM    132  CA  ALA A  14       5.944  -1.287   4.889  1.00  0.00           C  \\nATOM    133  C   ALA A  14       6.308  -0.098   5.770  1.00  0.00           C  \\nATOM    134  O   ALA A  14       6.460  -0.236   6.983  1.00  0.00           O  \\nATOM    135  CB  ALA A  14       7.188  -1.850   4.219  1.00  0.00           C  \\nATOM    136  H   ALA A  14       5.197  -0.945   2.907  1.00  0.00           H  \\nATOM    137  HA  ALA A  14       5.508  -2.055   5.528  1.00  0.00           H  \\nATOM    138 1HB  ALA A  14       7.919  -2.123   4.981  1.00  0.00           H  \\nATOM    139 2HB  ALA A  14       6.921  -2.734   3.640  1.00  0.00           H  \\nATOM    140 3HB  ALA A  14       7.617  -1.099   3.558  1.00  0.00           H  \\nATOM    141  N   ALA A  15       6.450   1.070   5.163  1.00  0.00           N  \\nATOM    142  CA  ALA A  15       6.791   2.285   5.895  1.00  0.00           C  \\nATOM    143  C   ALA A  15       5.680   2.676   6.861  1.00  0.00           C  \\nATOM    144  O   ALA A  15       5.944   3.088   7.990  1.00  0.00           O  \\nATOM    145  CB  ALA A  15       7.076   3.424   4.927  1.00  0.00           C  \\nATOM    146  H   ALA A  15       6.319   1.125   4.164  1.00  0.00           H  \\nATOM    147  HA  ALA A  15       7.689   2.087   6.481  1.00  0.00           H  \\nATOM    148 1HB  ALA A  15       7.329   4.323   5.488  1.00  0.00           H  \\nATOM    149 2HB  ALA A  15       7.911   3.152   4.281  1.00  0.00           H  \\nATOM    150 3HB  ALA A  15       6.193   3.612   4.318  1.00  0.00           H  \\nATOM    151  N   ALA A  16       4.440   2.544   6.409  1.00  0.00           N  \\nATOM    152  CA  ALA A  16       3.279   2.882   7.225  1.00  0.00           C  \\nATOM    153  C   ALA A  16       3.169   1.962   8.434  1.00  0.00           C  \\nATOM    154  O   ALA A  16       2.854   2.406   9.537  1.00  0.00           O  \\nATOM    155  CB  ALA A  16       2.008   2.814   6.391  1.00  0.00           C  \\nATOM    156  H   ALA A  16       4.291   2.200   5.472  1.00  0.00           H  \\nATOM    157  HA  ALA A  16       3.407   3.901   7.591  1.00  0.00           H  \\nATOM    158 1HB  ALA A  16       1.150   3.068   7.014  1.00  0.00           H  \\nATOM    159 2HB  ALA A  16       2.077   3.520   5.563  1.00  0.00           H  \\nATOM    160 3HB  ALA A  16       1.884   1.806   5.998  1.00  0.00           H  \\nATOM    161  N   ALA A  17       3.433   0.677   8.207  1.00  0.00           N  \\nATOM    162  CA  ALA A  17       3.356  -0.311   9.276  1.00  0.00           C  \\nATOM    163  C   ALA A  17       4.402  -0.042  10.351  1.00  0.00           C  \\nATOM    164  O   ALA A  17       4.125  -0.164  11.544  1.00  0.00           O  \\nATOM    165  CB  ALA A  17       3.525  -1.715   8.714  1.00  0.00           C  \\nATOM    166  H   ALA A  17       3.691   0.384   7.275  1.00  0.00           H  \\nATOM    167  HA  ALA A  17       2.373  -0.233   9.740  1.00  0.00           H  \\nATOM    168 1HB  ALA A  17       3.465  -2.442   9.524  1.00  0.00           H  \\nATOM    169 2HB  ALA A  17       2.735  -1.914   7.989  1.00  0.00           H  \\nATOM    170 3HB  ALA A  17       4.495  -1.797   8.225  1.00  0.00           H  \\nATOM    171  N   ALA A  18       5.604   0.324   9.924  1.00  0.00           N  \\nATOM    172  CA  ALA A  18       6.687   0.609  10.858  1.00  0.00           C  \\nATOM    173  C   ALA A  18       6.367   1.826  11.717  1.00  0.00           C  \\nATOM    174  O   ALA A  18       6.637   1.838  12.918  1.00  0.00           O  \\nATOM    175  CB  ALA A  18       7.993   0.821  10.107  1.00  0.00           C  \\nATOM    176  H   ALA A  18       5.770   0.405   8.931  1.00  0.00           H  \\nATOM    177  HA  ALA A  18       6.797  -0.249  11.522  1.00  0.00           H  \\nATOM    178 1HB  ALA A  18       8.792   1.033  10.818  1.00  0.00           H  \\nATOM    179 2HB  ALA A  18       8.239  -0.080   9.543  1.00  0.00           H  \\nATOM    180 3HB  ALA A  18       7.886   1.659   9.421  1.00  0.00           H  \\nATOM    181  N   ALA A  19       5.792   2.850  11.106  1.00  0.00           N  \\nATOM    182  CA  ALA A  19       5.428   4.072  11.813  1.00  0.00           C  \\nATOM    183  C   ALA A  19       4.344   3.806  12.850  1.00  0.00           C  \\nATOM    184  O   ALA A  19       4.390   4.336  13.960  1.00  0.00           O  \\nATOM    185  CB  ALA A  19       4.968   5.137  10.828  1.00  0.00           C  \\nATOM    186  H   ALA A  19       5.597   2.786  10.117  1.00  0.00           H  \\nATOM    187  HA  ALA A  19       6.311   4.435  12.339  1.00  0.00           H  \\nATOM    188 1HB  ALA A  19       4.700   6.043  11.372  1.00  0.00           H  \\nATOM    189 2HB  ALA A  19       5.775   5.358  10.129  1.00  0.00           H  \\nATOM    190 3HB  ALA A  19       4.102   4.774  10.279  1.00  0.00           H  \\nATOM    191  N   ALA A  20       3.372   2.982  12.476  1.00  0.00           N  \\nATOM    192  CA  ALA A  20       2.267   2.640  13.364  1.00  0.00           C  \\nATOM    193  C   ALA A  20       2.758   1.866  14.581  1.00  0.00           C  \\nATOM    194  O   ALA A  20       3.750   1.141  14.509  1.00  0.00           O  \\nATOM    195  CB  ALA A  20       1.214   1.836  12.616  1.00  0.00           C  \\nATOM    196  H   ALA A  20       3.397   2.579  11.550  1.00  0.00           H  \\nATOM    197  HA  ALA A  20       1.818   3.568  13.718  1.00  0.00           H  \\nATOM    198 1HB  ALA A  20       0.396   1.588  13.292  1.00  0.00           H  \\nATOM    199 2HB  ALA A  20       0.831   2.425  11.783  1.00  0.00           H  \\nATOM    200 3HB  ALA A  20       1.660   0.918  12.235  1.00  0.00           H  \\nTER                                                                             \\nATOM    202  N   ALA B  21      -4.903   1.137 -16.247  1.00  0.00           N  \\nATOM    203  CA  ALA B  21      -4.236   0.137 -15.422  1.00  0.00           C  \\nATOM    204  C   ALA B  21      -3.571   0.777 -14.210  1.00  0.00           C  \\nATOM    205  O   ALA B  21      -3.624   0.238 -13.105  1.00  0.00           O  \\nATOM    206  CB  ALA B  21      -3.211  -0.630 -16.245  1.00  0.00           C  \\nATOM    207  H   ALA B  21      -4.535   2.077 -16.274  1.00  0.00           H  \\nATOM    208  HA  ALA B  21      -4.991  -0.561 -15.059  1.00  0.00           H  \\nATOM    209 1HB  ALA B  21      -2.721  -1.373 -15.615  1.00  0.00           H  \\nATOM    210 2HB  ALA B  21      -3.710  -1.130 -17.074  1.00  0.00           H  \\nATOM    211 3HB  ALA B  21      -2.466   0.062 -16.634  1.00  0.00           H  \\nATOM    212  N   ALA B  22      -2.947   1.931 -14.437  1.00  0.00           N  \\nATOM    213  CA  ALA B  22      -2.263   2.644 -13.364  1.00  0.00           C  \\nATOM    214  C   ALA B  22      -3.247   3.112 -12.300  1.00  0.00           C  \\nATOM    215  O   ALA B  22      -2.966   3.032 -11.104  1.00  0.00           O  \\nATOM    216  CB  ALA B  22      -1.487   3.827 -13.925  1.00  0.00           C  \\nATOM    217  H   ALA B  22      -2.950   2.317 -15.370  1.00  0.00           H  \\nATOM    218  HA  ALA B  22      -1.563   1.954 -12.892  1.00  0.00           H  \\nATOM    219 1HB  ALA B  22      -0.982   4.348 -13.112  1.00  0.00           H  \\nATOM    220 2HB  ALA B  22      -0.748   3.470 -14.642  1.00  0.00           H  \\nATOM    221 3HB  ALA B  22      -2.174   4.509 -14.422  1.00  0.00           H  \\nATOM    222  N   ALA B  23      -4.400   3.601 -12.737  1.00  0.00           N  \\nATOM    223  CA  ALA B  23      -5.422   4.080 -11.813  1.00  0.00           C  \\nATOM    224  C   ALA B  23      -5.964   2.945 -10.955  1.00  0.00           C  \\nATOM    225  O   ALA B  23      -6.190   3.112  -9.757  1.00  0.00           O  \\nATOM    226  CB  ALA B  23      -6.553   4.754 -12.576  1.00  0.00           C  \\nATOM    227  H   ALA B  23      -4.572   3.642 -13.731  1.00  0.00           H  \\nATOM    228  HA  ALA B  23      -4.961   4.811 -11.148  1.00  0.00           H  \\nATOM    229 1HB  ALA B  23      -7.307   5.106 -11.873  1.00  0.00           H  \\nATOM    230 2HB  ALA B  23      -6.159   5.600 -13.139  1.00  0.00           H  \\nATOM    231 3HB  ALA B  23      -7.003   4.039 -13.264  1.00  0.00           H  \\nATOM    232  N   ALA B  24      -6.175   1.788 -11.565  1.00  0.00           N  \\nATOM    233  CA  ALA B  24      -6.686   0.620 -10.858  1.00  0.00           C  \\nATOM    234  C   ALA B  24      -5.693   0.131  -9.812  1.00  0.00           C  \\nATOM    235  O   ALA B  24      -6.078  -0.243  -8.704  1.00  0.00           O  \\nATOM    236  CB  ALA B  24      -7.008  -0.495 -11.843  1.00  0.00           C  \\nATOM    237  H   ALA B  24      -5.977   1.709 -12.552  1.00  0.00           H  \\nATOM    238  HA  ALA B  24      -7.601   0.910 -10.341  1.00  0.00           H  \\nATOM    239 1HB  ALA B  24      -7.389  -1.361 -11.300  1.00  0.00           H  \\nATOM    240 2HB  ALA B  24      -7.763  -0.150 -12.549  1.00  0.00           H  \\nATOM    241 3HB  ALA B  24      -6.106  -0.775 -12.384  1.00  0.00           H  \\nATOM    242  N   ALA B  25      -4.415   0.138 -10.175  1.00  0.00           N  \\nATOM    243  CA  ALA B  25      -3.357  -0.306  -9.276  1.00  0.00           C  \\nATOM    244  C   ALA B  25      -3.247   0.605  -8.061  1.00  0.00           C  \\nATOM    245  O   ALA B  25      -3.059   0.138  -6.938  1.00  0.00           O  \\nATOM    246  CB  ALA B  25      -2.027  -0.367 -10.013  1.00  0.00           C  \\nATOM    247  H   ALA B  25      -4.169   0.461 -11.100  1.00  0.00           H  \\nATOM    248  HA  ALA B  25      -3.610  -1.306  -8.923  1.00  0.00           H  \\nATOM    249 1HB  ALA B  25      -1.246  -0.700  -9.329  1.00  0.00           H  \\nATOM    250 2HB  ALA B  25      -2.103  -1.067 -10.845  1.00  0.00           H  \\nATOM    251 3HB  ALA B  25      -1.777   0.622 -10.394  1.00  0.00           H  \\nATOM    252  N   ALA B  26      -3.367   1.909  -8.302  1.00  0.00           N  \\nATOM    253  CA  ALA B  26      -3.274   2.887  -7.225  1.00  0.00           C  \\nATOM    254  C   ALA B  26      -4.418   2.723  -6.232  1.00  0.00           C  \\nATOM    255  O   ALA B  26      -4.220   2.820  -5.021  1.00  0.00           O  \\nATOM    256  CB  ALA B  26      -3.263   4.300  -7.790  1.00  0.00           C  \\nATOM    257  H   ALA B  26      -3.525   2.222  -9.250  1.00  0.00           H  \\nATOM    258  HA  ALA B  26      -2.340   2.714  -6.690  1.00  0.00           H  \\nATOM    259 1HB  ALA B  26      -3.193   5.018  -6.974  1.00  0.00           H  \\nATOM    260 2HB  ALA B  26      -2.406   4.419  -8.453  1.00  0.00           H  \\nATOM    261 3HB  ALA B  26      -4.181   4.474  -8.349  1.00  0.00           H  \\nATOM    262  N   ALA B  27      -5.616   2.475  -6.744  1.00  0.00           N  \\nATOM    263  CA  ALA B  27      -6.787   2.295  -5.895  1.00  0.00           C  \\nATOM    264  C   ALA B  27      -6.652   1.052  -5.026  1.00  0.00           C  \\nATOM    265  O   ALA B  27      -7.010   1.063  -3.848  1.00  0.00           O  \\nATOM    266  CB  ALA B  27      -8.049   2.214  -6.742  1.00  0.00           C  \\nATOM    267  H   ALA B  27      -5.719   2.410  -7.746  1.00  0.00           H  \\nATOM    268  HA  ALA B  27      -6.862   3.158  -5.233  1.00  0.00           H  \\nATOM    269 1HB  ALA B  27      -8.915   2.080  -6.094  1.00  0.00           H  \\nATOM    270 2HB  ALA B  27      -8.163   3.135  -7.314  1.00  0.00           H  \\nATOM    271 3HB  ALA B  27      -7.973   1.370  -7.426  1.00  0.00           H  \\nATOM    272  N   ALA B  28      -6.136  -0.022  -5.605  1.00  0.00           N  \\nATOM    273  CA  ALA B  28      -5.946  -1.277  -4.889  1.00  0.00           C  \\nATOM    274  C   ALA B  28      -4.921  -1.125  -3.772  1.00  0.00           C  \\nATOM    275  O   ALA B  28      -5.101  -1.654  -2.675  1.00  0.00           O  \\nATOM    276  CB  ALA B  28      -5.520  -2.377  -5.851  1.00  0.00           C  \\nATOM    277  H   ALA B  28      -5.863   0.027  -6.577  1.00  0.00           H  \\nATOM    278  HA  ALA B  28      -6.897  -1.555  -4.434  1.00  0.00           H  \\nATOM    279 1HB  ALA B  28      -5.382  -3.308  -5.301  1.00  0.00           H  \\nATOM    280 2HB  ALA B  28      -6.290  -2.514  -6.610  1.00  0.00           H  \\nATOM    281 3HB  ALA B  28      -4.584  -2.097  -6.331  1.00  0.00           H  \\nATOM    282  N   ALA B  29      -3.848  -0.396  -4.068  1.00  0.00           N  \\nATOM    283  CA  ALA B  29      -2.784  -0.173  -3.096  1.00  0.00           C  \\nATOM    284  C   ALA B  29      -3.286   0.634  -1.906  1.00  0.00           C  \\nATOM    285  O   ALA B  29      -2.941   0.347  -0.759  1.00  0.00           O  \\nATOM    286  CB  ALA B  29      -1.606   0.531  -3.753  1.00  0.00           C  \\nATOM    287  H   ALA B  29      -3.769   0.012  -4.988  1.00  0.00           H  \\nATOM    288  HA  ALA B  29      -2.454  -1.143  -2.726  1.00  0.00           H  \\nATOM    289 1HB  ALA B  29      -0.820   0.691  -3.015  1.00  0.00           H  \\nATOM    290 2HB  ALA B  29      -1.221  -0.085  -4.566  1.00  0.00           H  \\nATOM    291 3HB  ALA B  29      -1.931   1.492  -4.150  1.00  0.00           H  \\nATOM    292  N   ALA B  30      -4.103   1.645  -2.191  1.00  0.00           N  \\nATOM    293  CA  ALA B  30      -4.646   2.495  -1.138  1.00  0.00           C  \\nATOM    294  C   ALA B  30      -5.561   1.706  -0.210  1.00  0.00           C  \\nATOM    295  O   ALA B  30      -5.531   1.885   1.007  1.00  0.00           O  \\nATOM    296  CB  ALA B  30      -5.395   3.674  -1.742  1.00  0.00           C  \\nATOM    297  H   ALA B  30      -4.348   1.823  -3.154  1.00  0.00           H  \\nATOM    298  HA  ALA B  30      -3.814   2.873  -0.544  1.00  0.00           H  \\nATOM    299 1HB  ALA B  30      -5.795   4.299  -0.943  1.00  0.00           H  \\nATOM    300 2HB  ALA B  30      -4.713   4.262  -2.356  1.00  0.00           H  \\nATOM    301 3HB  ALA B  30      -6.214   3.306  -2.359  1.00  0.00           H  \\nATOM    302  N   ALA B  31      -6.376   0.831  -0.781  1.00  0.00           N  \\nATOM    303  CA  ALA B  31      -7.294   0.012   0.000  1.00  0.00           C  \\nATOM    304  C   ALA B  31      -6.539  -0.951   0.907  1.00  0.00           C  \\nATOM    305  O   ALA B  31      -6.916  -1.158   2.060  1.00  0.00           O  \\nATOM    306  CB  ALA B  31      -8.233  -0.755  -0.920  1.00  0.00           C  \\nATOM    307  H   ALA B  31      -6.361   0.728  -1.786  1.00  0.00           H  \\nATOM    308  HA  ALA B  31      -7.884   0.674   0.634  1.00  0.00           H  \\nATOM    309 1HB  ALA B  31      -8.913  -1.362  -0.322  1.00  0.00           H  \\nATOM    310 2HB  ALA B  31      -8.809  -0.051  -1.521  1.00  0.00           H  \\nATOM    311 3HB  ALA B  31      -7.652  -1.401  -1.576  1.00  0.00           H  \\nATOM    312  N   ALA B  32      -5.472  -1.538   0.383  1.00  0.00           N  \\nATOM    313  CA  ALA B  32      -4.654  -2.480   1.138  1.00  0.00           C  \\nATOM    314  C   ALA B  32      -3.966  -1.795   2.312  1.00  0.00           C  \\nATOM    315  O   ALA B  32      -3.886  -2.351   3.407  1.00  0.00           O  \\nATOM    316  CB  ALA B  32      -3.623  -3.133   0.229  1.00  0.00           C  \\nATOM    317  H   ALA B  32      -5.214  -1.329  -0.571  1.00  0.00           H  \\nATOM    318  HA  ALA B  32      -5.310  -3.252   1.540  1.00  0.00           H  \\nATOM    319 1HB  ALA B  32      -3.020  -3.834   0.807  1.00  0.00           H  \\nATOM    320 2HB  ALA B  32      -4.132  -3.669  -0.573  1.00  0.00           H  \\nATOM    321 3HB  ALA B  32      -2.978  -2.367  -0.199  1.00  0.00           H  \\nATOM    322  N   ALA B  33      -3.474  -0.583   2.064  1.00  0.00           N  \\nATOM    323  CA  ALA B  33      -2.784   0.182   3.096  1.00  0.00           C  \\nATOM    324  C   ALA B  33      -3.726   0.543   4.237  1.00  0.00           C  \\nATOM    325  O   ALA B  33      -3.352   0.478   5.408  1.00  0.00           O  \\nATOM    326  CB  ALA B  33      -2.168   1.439   2.501  1.00  0.00           C  \\nATOM    327  H   ALA B  33      -3.581  -0.187   1.142  1.00  0.00           H  \\nATOM    328  HA  ALA B  33      -1.988  -0.442   3.505  1.00  0.00           H  \\nATOM    329 1HB  ALA B  33      -1.656   2.000   3.283  1.00  0.00           H  \\nATOM    330 2HB  ALA B  33      -1.452   1.162   1.726  1.00  0.00           H  \\nATOM    331 3HB  ALA B  33      -2.952   2.057   2.066  1.00  0.00           H  \\nATOM    332  N   ALA B  34      -4.951   0.923   3.887  1.00  0.00           N  \\nATOM    333  CA  ALA B  34      -5.942   1.296   4.889  1.00  0.00           C  \\nATOM    334  C   ALA B  34      -6.308   0.109   5.770  1.00  0.00           C  \\nATOM    335  O   ALA B  34      -6.459   0.247   6.983  1.00  0.00           O  \\nATOM    336  CB  ALA B  34      -7.185   1.862   4.219  1.00  0.00           C  \\nATOM    337  H   ALA B  34      -5.195   0.953   2.907  1.00  0.00           H  \\nATOM    338  HA  ALA B  34      -5.505   2.064   5.528  1.00  0.00           H  \\nATOM    339 1HB  ALA B  34      -7.916   2.135   4.981  1.00  0.00           H  \\nATOM    340 2HB  ALA B  34      -6.917   2.745   3.640  1.00  0.00           H  \\nATOM    341 3HB  ALA B  34      -7.615   1.111   3.558  1.00  0.00           H  \\nATOM    342  N   ALA B  35      -6.452  -1.060   5.163  1.00  0.00           N  \\nATOM    343  CA  ALA B  35      -6.795  -2.274   5.895  1.00  0.00           C  \\nATOM    344  C   ALA B  35      -5.685  -2.667   6.861  1.00  0.00           C  \\nATOM    345  O   ALA B  35      -5.949  -3.079   7.990  1.00  0.00           O  \\nATOM    346  CB  ALA B  35      -7.081  -3.412   4.927  1.00  0.00           C  \\nATOM    347  H   ALA B  35      -6.321  -1.115   4.164  1.00  0.00           H  \\nATOM    348  HA  ALA B  35      -7.692  -2.074   6.481  1.00  0.00           H  \\nATOM    349 1HB  ALA B  35      -7.335  -4.312   5.488  1.00  0.00           H  \\nATOM    350 2HB  ALA B  35      -7.916  -3.139   4.281  1.00  0.00           H  \\nATOM    351 3HB  ALA B  35      -6.199  -3.602   4.318  1.00  0.00           H  \\nATOM    352  N   ALA B  36      -4.444  -2.537   6.409  1.00  0.00           N  \\nATOM    353  CA  ALA B  36      -3.283  -2.876   7.225  1.00  0.00           C  \\nATOM    354  C   ALA B  36      -3.172  -1.957   8.434  1.00  0.00           C  \\nATOM    355  O   ALA B  36      -2.857  -2.402   9.537  1.00  0.00           O  \\nATOM    356  CB  ALA B  36      -2.012  -2.810   6.391  1.00  0.00           C  \\nATOM    357  H   ALA B  36      -4.295  -2.193   5.472  1.00  0.00           H  \\nATOM    358  HA  ALA B  36      -3.413  -3.895   7.591  1.00  0.00           H  \\nATOM    359 1HB  ALA B  36      -1.155  -3.066   7.014  1.00  0.00           H  \\nATOM    360 2HB  ALA B  36      -2.083  -3.516   5.563  1.00  0.00           H  \\nATOM    361 3HB  ALA B  36      -1.887  -1.803   5.998  1.00  0.00           H  \\nATOM    362  N   ALA B  37      -3.434  -0.671   8.207  1.00  0.00           N  \\nATOM    363  CA  ALA B  37      -3.356   0.317   9.276  1.00  0.00           C  \\nATOM    364  C   ALA B  37      -4.402   0.049  10.351  1.00  0.00           C  \\nATOM    365  O   ALA B  37      -4.124   0.170  11.544  1.00  0.00           O  \\nATOM    366  CB  ALA B  37      -3.522   1.721   8.714  1.00  0.00           C  \\nATOM    367  H   ALA B  37      -3.692  -0.378   7.275  1.00  0.00           H  \\nATOM    368  HA  ALA B  37      -2.372   0.237   9.740  1.00  0.00           H  \\nATOM    369 1HB  ALA B  37      -3.462   2.447   9.524  1.00  0.00           H  \\nATOM    370 2HB  ALA B  37      -2.732   1.919   7.989  1.00  0.00           H  \\nATOM    371 3HB  ALA B  37      -4.492   1.804   8.225  1.00  0.00           H  \\nATOM    372  N   ALA B  38      -5.605  -0.315   9.924  1.00  0.00           N  \\nATOM    373  CA  ALA B  38      -6.688  -0.598  10.858  1.00  0.00           C  \\nATOM    374  C   ALA B  38      -6.370  -1.816  11.717  1.00  0.00           C  \\nATOM    375  O   ALA B  38      -6.640  -1.828  12.918  1.00  0.00           O  \\nATOM    376  CB  ALA B  38      -7.995  -0.808  10.107  1.00  0.00           C  \\nATOM    377  H   ALA B  38      -5.771  -0.396   8.931  1.00  0.00           H  \\nATOM    378  HA  ALA B  38      -6.796   0.260  11.522  1.00  0.00           H  \\nATOM    379 1HB  ALA B  38      -8.793  -1.019  10.818  1.00  0.00           H  \\nATOM    380 2HB  ALA B  38      -8.239   0.093   9.543  1.00  0.00           H  \\nATOM    381 3HB  ALA B  38      -7.889  -1.647   9.421  1.00  0.00           H  \\nATOM    382  N   ALA B  39      -5.797  -2.841  11.106  1.00  0.00           N  \\nATOM    383  CA  ALA B  39      -5.435  -4.063  11.813  1.00  0.00           C  \\nATOM    384  C   ALA B  39      -4.350  -3.799  12.850  1.00  0.00           C  \\nATOM    385  O   ALA B  39      -4.397  -4.329  13.960  1.00  0.00           O  \\nATOM    386  CB  ALA B  39      -4.977  -5.129  10.828  1.00  0.00           C  \\nATOM    387  H   ALA B  39      -5.602  -2.777  10.117  1.00  0.00           H  \\nATOM    388  HA  ALA B  39      -6.318  -4.424  12.339  1.00  0.00           H  \\nATOM    389 1HB  ALA B  39      -4.710  -6.035  11.372  1.00  0.00           H  \\nATOM    390 2HB  ALA B  39      -5.783  -5.349  10.129  1.00  0.00           H  \\nATOM    391 3HB  ALA B  39      -4.109  -4.767  10.279  1.00  0.00           H  \\nATOM    392  N   ALA B  40      -3.377  -2.976  12.476  1.00  0.00           N  \\nATOM    393  CA  ALA B  40      -2.272  -2.636  13.364  1.00  0.00           C  \\nATOM    394  C   ALA B  40      -2.761  -1.862  14.581  1.00  0.00           C  \\nATOM    395  O   ALA B  40      -3.752  -1.135  14.509  1.00  0.00           O  \\nATOM    396  CB  ALA B  40      -1.217  -1.834  12.616  1.00  0.00           C  \\nATOM    397  H   ALA B  40      -3.401  -2.574  11.550  1.00  0.00           H  \\nATOM    398  HA  ALA B  40      -1.823  -3.565  13.718  1.00  0.00           H  \\nATOM    399 1HB  ALA B  40      -0.399  -1.588  13.292  1.00  0.00           H  \\nATOM    400 2HB  ALA B  40      -0.835  -2.424  11.783  1.00  0.00           H  \\nATOM    401 3HB  ALA B  40      -1.662  -0.916  12.235  1.00  0.00           H  \\nTER                                                                             \\nREMARK PDBinfo-LABEL:    1 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    2 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    3 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    4 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    5 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    6 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    7 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    8 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    9 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   10 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   11 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   12 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   13 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   14 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   15 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   16 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   17 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   18 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   19 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   20 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   21 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   22 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   23 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   24 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   25 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   26 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   27 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   28 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   29 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   30 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   31 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   32 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   33 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   34 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   35 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   36 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   37 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   38 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   39 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   40 PEPTIDE_STUB_EXTENSION\\n\\n# All scores below are weighted scores, not raw scores.\\n#BEGIN_POSE_ENERGIES_TABLE \\nlabel fa_atr fa_rep fa_sol fa_intra_rep fa_intra_sol_xover4 lk_ball_wtd fa_elec pro_close hbond_sr_bb hbond_lr_bb hbond_bb_sc hbond_sc dslf_fa13 omega fa_dun p_aa_pp yhh_planarity ref rama_prepro total\\nweights 1 0.55 1 0.005 1 1 1 1.25 1 1 1 1 1.25 0.4 0.7 0.6 0.625 1 0.45 NA\\npose -124.985 5.62822 96.5176 0.05425 0 -0.6028 -43.6441 0 -32.8556 0 0 0 0 -0.24588 0 -5.44976 0 52.9872 2.68022 -49.9154\\nALA_1 -1.80275 0.07143 1.41204 0.00134 0 0.00599 -0.31538 0 0 0 0 0 0 0.00417 0 -0.18334 0 1.32468 3.03052 3.54869\\nALA_2 -2.10827 0.08843 1.56674 0.00136 0 -0.00292 -0.13122 0 0 0 0 0 0 -0.00607 0 -0.15076 0 1.32468 2.89416 3.47614\\nALA_3 -2.44439 0.13223 1.97708 0.00136 0 -0.08229 -0.35542 0 0 0 0 0 0 0.00203 0 -0.14614 0 1.32468 -0.26991 0.13922\\nALA_4 -3.0948 0.13402 2.35461 0.00136 0 -0.11288 -0.82869 0 0 0 0 0 0 -0.01138 0 -0.13507 0 1.32468 -0.25864 -0.62678\\nALA_5 -3.79731 0.13678 2.56908 0.00136 0 0.0137 -1.57053 0 0 0 0 0 0 -0.01566 0 -0.14201 0 1.32468 -0.25484 -1.73475\\nALA_6 -3.53308 0.13659 2.79582 0.00136 0 0.00224 -1.63865 0 0 0 0 0 0 -0.00227 0 -0.15161 0 1.32468 -0.2669 -1.33182\\nALA_7 -3.41839 0.13406 2.73029 0.00136 0 -0.01663 -1.51714 0 0 0 0 0 0 -8e-05 0 -0.14278 0 1.32468 -0.26824 -1.17288\\nALA_8 -3.4928 0.13454 2.7638 0.00136 0 -0.00245 -1.60424 0 0 0 0 0 0 -0.01424 0 -0.13494 0 1.32468 -0.25594 -1.28021\\nALA_9 -4.00975 0.4738 2.50406 0.00136 0 0.00655 -1.70848 0 0 0 0 0 0 -0.01355 0 -0.14535 0 1.32468 -0.25705 -1.82373\\nALA_10 -3.49324 0.13599 2.77639 0.00136 0 -0.00416 -1.59628 0 0 0 0 0 0 0.00075 0 -0.15111 0 1.32468 -0.26916 -1.27479\\nALA_11 -3.4184 0.13376 2.72783 0.00136 0 -0.01635 -1.51938 0 0 0 0 0 0 -0.00339 0 -0.1395 0 1.32468 -0.26557 -1.17496\\nALA_12 -3.5301 0.13523 2.78299 0.00136 0 0.00382 -1.63899 0 0 0 0 0 0 -0.01602 0 -0.13614 0 1.32468 -0.25422 -1.32739\\nALA_13 -3.7136 0.13731 2.72719 0.00136 0 0.01263 -1.6871 0 0 0 0 0 0 -0.0104 0 -0.14835 0 1.32468 -0.26006 -1.61634\\nALA_14 -3.46027 0.13531 2.75753 0.00136 0 -0.00996 -1.55976 0 0 0 0 0 0 0.00236 0 -0.14934 0 1.32468 -0.27027 -1.22836\\nALA_15 -3.43178 0.13372 2.73283 0.00136 0 -0.01384 -1.53558 0 0 0 0 0 0 -0.00724 0 -0.13682 0 1.32468 -0.26234 -1.19502\\nALA_16 -3.57458 0.13598 2.78348 0.00136 0 0.00198 -1.53111 0 0 0 0 0 0 -0.01658 0 -0.13847 0 1.32468 -0.25375 -1.26702\\nALA_17 -3.2829 0.13709 2.56966 0.00136 0 -0.05073 -0.2939 0 0 0 0 0 0 -0.00661 0 -0.15054 0 1.32468 -0.26339 -0.01529\\nALA_18 -2.56628 0.13236 2.22509 0.00136 0 -0.03508 -0.13929 0 0 0 0 0 0 0.00219 0 -0.14658 0 1.32468 -0.27005 0.52841\\nALA_19 -2.08267 0.08654 1.68194 0.00136 0 -0.05463 -0.64274 0 0 0 0 0 0 -0.01092 0 -0.13519 0 1.32468 -0.25906 -0.0907\\nALA_20 -2.23697 0.06895 1.82034 0.00128 0 0.05361 -0.00817 0 0 0 0 0 0 0 0 0.03916 0 1.32468 -0.1252 0.93768\\nALA_21 -1.8026 0.07143 1.41204 0.00134 0 0.00599 -0.31532 0 0 0 0 0 0 0.00417 0 -0.18334 0 1.32468 3.03052 3.54891\\nALA_22 -2.10858 0.08843 1.56675 0.00136 0 -0.0029 -0.13138 0 0 0 0 0 0 -0.00607 0 -0.15076 0 1.32468 2.89416 3.47569\\nALA_23 -2.44439 0.13223 1.97708 0.00136 0 -0.08229 -0.35542 0 0 0 0 0 0 0.00203 0 -0.14614 0 1.32468 -0.26991 0.13922\\nALA_24 -3.0948 0.13402 2.35461 0.00136 0 -0.11288 -0.82869 0 0 0 0 0 0 -0.01138 0 -0.13507 0 1.32468 -0.25864 -0.62678\\nALA_25 -3.79708 0.13678 2.56904 0.00136 0 0.01369 -1.57044 0 0 0 0 0 0 -0.01566 0 -0.14201 0 1.32468 -0.25484 -1.73449\\nALA_26 -3.53315 0.13659 2.79584 0.00136 0 0.00224 -1.63864 0 0 0 0 0 0 -0.00227 0 -0.15161 0 1.32468 -0.2669 -1.33185\\nALA_27 -3.41839 0.13406 2.73029 0.00136 0 -0.01663 -1.51714 0 0 0 0 0 0 -8e-05 0 -0.14278 0 1.32468 -0.26824 -1.17288\\nALA_28 -3.49279 0.13454 2.7638 0.00136 0 -0.00245 -1.60424 0 0 0 0 0 0 -0.01424 0 -0.13494 0 1.32468 -0.25594 -1.28021\\nALA_29 -4.00975 0.4738 2.50407 0.00136 0 0.00654 -1.70844 0 0 0 0 0 0 -0.01355 0 -0.14535 0 1.32468 -0.25705 -1.82369\\nALA_30 -3.49324 0.13599 2.77639 0.00136 0 -0.00416 -1.59628 0 0 0 0 0 0 0.00075 0 -0.15111 0 1.32468 -0.26916 -1.27478\\nALA_31 -3.4184 0.13376 2.72783 0.00136 0 -0.01635 -1.51938 0 0 0 0 0 0 -0.00339 0 -0.1395 0 1.32468 -0.26557 -1.17496\\nALA_32 -3.53001 0.13523 2.78299 0.00136 0 0.00382 -1.63905 0 0 0 0 0 0 -0.01602 0 -0.13614 0 1.32468 -0.25422 -1.32736\\nALA_33 -3.71385 0.13731 2.72708 0.00136 0 0.01262 -1.68717 0 0 0 0 0 0 -0.0104 0 -0.14835 0 1.32468 -0.26006 -1.61679\\nALA_34 -3.46027 0.13531 2.75753 0.00136 0 -0.00996 -1.55976 0 0 0 0 0 0 0.00236 0 -0.14934 0 1.32468 -0.27027 -1.22836\\nALA_35 -3.43178 0.13372 2.73283 0.00136 0 -0.01384 -1.53558 0 0 0 0 0 0 -0.00724 0 -0.13682 0 1.32468 -0.26234 -1.19502\\nALA_36 -3.57427 0.13598 2.7836 0.00136 0 0.00199 -1.53097 0 0 0 0 0 0 -0.01658 0 -0.13847 0 1.32468 -0.25375 -1.26644\\nALA_37 -3.28325 0.13709 2.56966 0.00136 0 -0.05069 -0.2941 0 0 0 0 0 0 -0.00661 0 -0.15054 0 1.32468 -0.26339 -0.0158\\nALA_38 -2.56628 0.13236 2.22509 0.00136 0 -0.03508 -0.13929 0 0 0 0 0 0 0.00219 0 -0.14658 0 1.32468 -0.27005 0.52841\\nALA_39 -2.08267 0.08654 1.68194 0.00136 0 -0.05463 -0.64274 0 0 0 0 0 0 -0.01092 0 -0.13519 0 1.32468 -0.25906 -0.0907\\nALA_40 -2.23677 0.06895 1.82034 0.00128 0 0.05357 -0.00804 0 0 0 0 0 0 0 0 0.03916 0 1.32468 -0.1252 0.93797\\n#END_POSE_ENERGIES_TABLE \\n\\n\",\"pdb\");\n\tviewer_16234205772799664.setStyle({\"cartoon\": {\"color\": \"spectrum\"}, \"stick\": {\"colorscheme\": \"lightgreyCarbon\", \"radius\": 0.25}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.1694070184482745, \"y\": -0.4541068423003337, \"z\": -11.10005319597299}, \"end\": {\"x\": 3.623957999316543, \"y\": -0.23215724975842678, \"z\": -13.10478854478874}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.528978416652365, \"y\": -2.2161298185313845, \"z\": -9.249510730505522}, \"end\": {\"x\": 2.9709219889056273, \"y\": -3.0272069890931306, \"z\": -11.103541580684825}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.722409097721679, \"y\": -2.4006909988148957, \"z\": -7.74632289083335}, \"end\": {\"x\": 6.194595719580915, \"y\": -3.102453653609407, \"z\": -9.756683941833414}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.862918823601143, \"y\": -0.017668114966811584, \"z\": -6.576599708065375}, \"end\": {\"x\": 6.077661961305986, \"y\": 0.2525134088090799, \"z\": -8.703679669723572}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.1694070184482745, \"y\": -0.4541068423003337, \"z\": -11.10005319597299}, \"end\": {\"x\": 3.623957999316543, \"y\": -0.23215724975842678, \"z\": -13.10478854478874}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.768740928672904, \"y\": -0.006165444174080655, \"z\": -4.988195022043259}, \"end\": {\"x\": 3.0596556317676042, \"y\": -0.13343538704308533, \"z\": -6.937586558831088}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.528978416652365, \"y\": -2.2161298185313845, \"z\": -9.249510730505522}, \"end\": {\"x\": 2.9709219889056273, \"y\": -3.0272069890931306, \"z\": -11.103541580684825}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.350865039096124, \"y\": -1.816146038409083, \"z\": -3.1541222768941193}, \"end\": {\"x\": 4.2245533256789525, \"y\": -2.81288740276295, \"z\": -5.020807605138654}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.722409097721679, \"y\": -2.4006909988148957, \"z\": -7.74632289083335}, \"end\": {\"x\": 6.194595719580915, \"y\": -3.102453653609407, \"z\": -9.756683941833414}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.361959675612962, \"y\": -0.7180357066980767, \"z\": -1.785769501948773}, \"end\": {\"x\": 7.012136132444223, \"y\": -1.05218965541963, \"z\": -3.848207825636483}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.862918823601143, \"y\": -0.017668114966811584, \"z\": -6.576599708065375}, \"end\": {\"x\": 6.077661961305986, \"y\": 0.2525134088090799, \"z\": -8.703679669723572}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.21168513535431, \"y\": 1.3371856675336933, \"z\": -0.570709242096436}, \"end\": {\"x\": 5.097920365540859, \"y\": 1.6616815865127768, \"z\": -2.675488710644366}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.768740928672904, \"y\": -0.006165444174080655, \"z\": -4.988195022043259}, \"end\": {\"x\": 3.0596556317676042, \"y\": -0.13343538704308533, \"z\": -6.937586558831088}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.581005696446443, \"y\": 0.1930338767981878, \"z\": 1.1415148453845125}, \"end\": {\"x\": 2.9418542979692983, \"y\": -0.3425786662103132, \"z\": -0.7593325720500322}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.350865039096124, \"y\": -1.816146038409083, \"z\": -3.1541222768941193}, \"end\": {\"x\": 4.2245533256789525, \"y\": -2.81288740276295, \"z\": -5.020807605138654}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.196586250314018, \"y\": -0.9451565883448323, \"z\": 2.907276604469157}, \"end\": {\"x\": 5.533612426158804, \"y\": -1.8765794817407608, \"z\": 1.007461142399106}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.361959675612962, \"y\": -0.7180357066980767, \"z\": -1.785769501948773}, \"end\": {\"x\": 7.012136132444223, \"y\": -1.05218965541963, \"z\": -3.848207825636483}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.319020315978136, \"y\": 1.1248187700230572, \"z\": 4.1635460268324405}, \"end\": {\"x\": 6.9139807428777535, \"y\": 1.1688620923581077, \"z\": 2.0603133068719353}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.21168513535431, \"y\": 1.3371856675336933, \"z\": -0.570709242096436}, \"end\": {\"x\": 5.097920365540859, \"y\": 1.6616815865127768, \"z\": -2.675488710644366}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.291149579572995, \"y\": 2.1996717461271413, \"z\": 5.471519231572035}, \"end\": {\"x\": 3.8826020331512883, \"y\": 2.3568814675441545, \"z\": 3.4070479224982586}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.581005696446443, \"y\": 0.1930338767981878, \"z\": 1.1415148453845125}, \"end\": {\"x\": 2.9418542979692983, \"y\": -0.3425786662103132, \"z\": -0.7593325720500322}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.691129997859202, \"y\": 0.38425138737586323, \"z\": 7.2752128023647185}, \"end\": {\"x\": 3.3527476851495357, \"y\": -0.47293926000680636, \"z\": 5.408025014604342}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.196586250314018, \"y\": -0.9451565883448323, \"z\": 2.907276604469157}, \"end\": {\"x\": 5.533612426158804, \"y\": -1.8765794817407608, \"z\": 1.007461142399106}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.770439890709007, \"y\": 0.40513764239083705, \"z\": 8.931294042709522}, \"end\": {\"x\": 6.459691830801482, \"y\": -0.23646867699845442, \"z\": 6.983482895456959}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.319020315978136, \"y\": 1.1248187700230572, \"z\": 4.1635460268324405}, \"end\": {\"x\": 6.9139807428777535, \"y\": 1.1688620923581077, \"z\": 2.0603133068719353}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.597498887444632, \"y\": 2.786258740530911, \"z\": 10.116802138605896}, \"end\": {\"x\": 5.943612297271183, \"y\": 3.0882230798153936, \"z\": 7.990349725508292}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.291149579572995, \"y\": 2.1996717461271413, \"z\": 5.471519231572035}, \"end\": {\"x\": 3.8826020331512883, \"y\": 2.3568814675441545, \"z\": 3.4070479224982586}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.396939295644632, \"y\": 2.5791351114309555, \"z\": 11.549812839786894}, \"end\": {\"x\": 2.853664452793035, \"y\": 2.406462801205477, \"z\": 9.537425076063954}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.691129997859202, \"y\": 0.38425138737586323, \"z\": 7.2752128023647185}, \"end\": {\"x\": 3.3527476851495357, \"y\": -0.47293926000680636, \"z\": 5.408025014604342}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.770439890709007, \"y\": 0.40513764239083705, \"z\": 8.931294042709522}, \"end\": {\"x\": 6.459691830801482, \"y\": -0.23646867699845442, \"z\": 6.983482895456959}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.597498887444632, \"y\": 2.786258740530911, \"z\": 10.116802138605896}, \"end\": {\"x\": 5.943612297271183, \"y\": 3.0882230798153936, \"z\": 7.990349725508292}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.396939295644632, \"y\": 2.5791351114309555, \"z\": 11.549812839786894}, \"end\": {\"x\": 2.853664452793035, \"y\": 2.406462801205477, \"z\": 9.537425076063954}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.168678495917393, \"y\": 0.46074668461708435, \"z\": -11.100053195972988}, \"end\": {\"x\": -3.6235836572295064, \"y\": 0.23792866259661516, \"z\": -13.104788544788734}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.5254444153274243, \"y\": 2.2217474456424706, \"z\": -9.249510730505524}, \"end\": {\"x\": -2.9660969309270975, \"y\": 3.031934797340165, \"z\": -11.103541580684825}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.718578372626965, \"y\": 2.4098017656234356, \"z\": -7.746322890833351}, \"end\": {\"x\": -6.189646731286077, \"y\": 3.112315559792127, \"z\": -9.756683941833412}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.86288324864783, \"y\": 0.02700568732236141, \"z\": -6.576599708065375}, \"end\": {\"x\": -6.078056419391683, \"y\": -0.242833482505183, \"z\": -8.703679669723567}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.168678495917393, \"y\": 0.46074668461708435, \"z\": -11.100053195972988}, \"end\": {\"x\": -3.6235836572295064, \"y\": 0.23792866259661516, \"z\": -13.104788544788734}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.7687263294699216, \"y\": 0.012167732586157769, \"z\": -4.988195022043261}, \"end\": {\"x\": -3.0594392350322845, \"y\": 0.13830818727604366, \"z\": -6.937586558831089}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.5254444153274243, \"y\": 2.2217474456424706, \"z\": -9.249510730505524}, \"end\": {\"x\": -2.9660969309270975, \"y\": 3.031934797340165, \"z\": -11.103541580684825}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.3479670307289755, \"y\": 1.8230731529348416, \"z\": -3.1541222768941224}, \"end\": {\"x\": -4.2200680144685485, \"y\": 2.8196120824303246, \"z\": -5.020807605138655}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.718578372626965, \"y\": 2.4098017656234356, \"z\": -7.746322890833351}, \"end\": {\"x\": -6.189646731286077, \"y\": 3.112315559792127, \"z\": -9.756683941833412}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.360808025252289, \"y\": 0.7281671896651079, \"z\": -1.7857695019487734}, \"end\": {\"x\": -7.01045146622167, \"y\": 1.0633562200186129, \"z\": -3.8482078256364836}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.86288324864783, \"y\": 0.02700568732236141, \"z\": -6.576599708065375}, \"end\": {\"x\": -6.078056419391683, \"y\": -0.242833482505183, \"z\": -8.703679669723567}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.213808198170849, \"y\": -1.3288835660872982, \"z\": -0.5707092420964386}, \"end\": {\"x\": -5.100560382013878, \"y\": -1.6535602613095794, \"z\": -2.675488710644367}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.7687263294699216, \"y\": 0.012167732586157769, \"z\": -4.988195022043261}, \"end\": {\"x\": -3.0594392350322845, \"y\": 0.13830818727604366, \"z\": -6.937586558831089}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.5813085907223874, \"y\": -0.18733033281222844, \"z\": 1.1415148453845099}, \"end\": {\"x\": -2.9413049579846318, \"y\": 0.3472635845547798, \"z\": -0.7593325720500358}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.3479670307289755, \"y\": 1.8230731529348416, \"z\": -3.1541222768941224}, \"end\": {\"x\": -4.2200680144685485, \"y\": 2.8196120824303246, \"z\": -5.020807605138655}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.195074353229818, \"y\": 0.9534317478761036, \"z\": 2.907276604469158}, \"end\": {\"x\": -5.530616668245941, \"y\": 1.8853902256959216, \"z\": 1.0074611423991024}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.360808025252289, \"y\": 0.7281671896651079, \"z\": -1.7857695019487734}, \"end\": {\"x\": -7.01045146622167, \"y\": 1.0633562200186129, \"z\": -3.8482078256364836}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.3208037476332795, \"y\": -1.1147533373108025, \"z\": 4.1635460268324405}, \"end\": {\"x\": -6.915833565686739, \"y\": -1.1578490383278288, \"z\": 2.060313306871933}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.213808198170849, \"y\": -1.3288835660872982, \"z\": -0.5707092420964386}, \"end\": {\"x\": -5.100560382013878, \"y\": -1.6535602613095794, \"z\": -2.675488710644367}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.294647450847934, \"y\": -2.1928346444510507, \"z\": 5.471519231572033}, \"end\": {\"x\": -3.886350803097142, \"y\": -2.350694840924633, \"z\": 3.4070479224982586}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.5813085907223874, \"y\": -0.18733033281222844, \"z\": 1.1415148453845099}, \"end\": {\"x\": -2.9413049579846318, \"y\": 0.3472635845547798, \"z\": -0.7593325720500358}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.6917372955934393, \"y\": -0.37837221108418234, \"z\": 7.27521280236472}, \"end\": {\"x\": -3.3519902048602006, \"y\": 0.47827842356997363, \"z\": 5.408025014604343}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.195074353229818, \"y\": 0.9534317478761036, \"z\": 2.907276604469158}, \"end\": {\"x\": -5.530616668245941, \"y\": 1.8853902256959216, \"z\": 1.0074611423991024}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.771077815866737, \"y\": -0.39594682064457354, \"z\": 8.931294042709524}, \"end\": {\"x\": -6.459307025624145, \"y\": 0.24675642412567778, \"z\": 6.983482895456962}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.3208037476332795, \"y\": -1.1147533373108025, \"z\": 4.1635460268324405}, \"end\": {\"x\": -6.915833565686739, \"y\": -1.1578490383278288, \"z\": 2.060313306871933}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.601929331400129, \"y\": -2.7773403338675617, \"z\": 10.116802138605893}, \"end\": {\"x\": -5.948523226646209, \"y\": -3.078753051647397, \"z\": 7.990349725508294}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.294647450847934, \"y\": -2.1928346444510507, \"z\": 5.471519231572033}, \"end\": {\"x\": -3.886350803097142, \"y\": -2.350694840924633, \"z\": 3.4070479224982586}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.4010426544572967, \"y\": -2.573721695108579, \"z\": 11.549812839786892}, \"end\": {\"x\": -2.857493493567745, \"y\": -2.4019148521415623, \"z\": 9.537425076063956}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.6917372955934393, \"y\": -0.37837221108418234, \"z\": 7.27521280236472}, \"end\": {\"x\": -3.3519902048602006, \"y\": 0.47827842356997363, \"z\": 5.408025014604343}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.771077815866737, \"y\": -0.39594682064457354, \"z\": 8.931294042709524}, \"end\": {\"x\": -6.459307025624145, \"y\": 0.24675642412567778, \"z\": 6.983482895456962}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.601929331400129, \"y\": -2.7773403338675617, \"z\": 10.116802138605893}, \"end\": {\"x\": -5.948523226646209, \"y\": -3.078753051647397, \"z\": 7.990349725508294}});\n\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.4010426544572967, \"y\": -2.573721695108579, \"z\": 11.549812839786892}, \"end\": {\"x\": -2.857493493567745, \"y\": -2.4019148521415623, \"z\": 9.537425076063956}});\n\tviewer_16234205772799664.zoomTo();\nviewer_16234205772799664.render();\n});\n</script>",
            "text/html": [
              "<div id=\"3dmolviewer_16234205772799664\"  style=\"position: relative; width: 1200px; height: 800px\">\n",
              "        <p id=\"3dmolwarning_16234205772799664\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n",
              "        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n",
              "        </div>\n",
              "<script>\n",
              "\n",
              "var loadScriptAsync = function(uri){\n",
              "  return new Promise((resolve, reject) => {\n",
              "    var tag = document.createElement('script');\n",
              "    tag.src = uri;\n",
              "    tag.async = true;\n",
              "    tag.onload = () => {\n",
              "      resolve();\n",
              "    };\n",
              "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
              "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
              "});\n",
              "};\n",
              "\n",
              "if(typeof $3Dmolpromise === 'undefined') {\n",
              "$3Dmolpromise = null;\n",
              "  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n",
              "}\n",
              "\n",
              "var viewer_16234205772799664 = null;\n",
              "var warn = document.getElementById(\"3dmolwarning_16234205772799664\");\n",
              "if(warn) {\n",
              "    warn.parentNode.removeChild(warn);\n",
              "}\n",
              "$3Dmolpromise.then(function() {\n",
              "viewer_16234205772799664 = $3Dmol.createViewer($(\"#3dmolviewer_16234205772799664\"),{backgroundColor:\"white\"});\n",
              "viewer_16234205772799664.zoomTo();\n",
              "\tviewer_16234205772799664.addModels(\"HEADER                                            11-JUN-21   XXXX              \\nEXPDTA    THEORETICAL MODEL                                                     \\nREMARK 220                                                                      \\nREMARK 220 EXPERIMENTAL DETAILS                                                 \\nREMARK 220  EXPERIMENT TYPE                : THEORETICAL MODELLING              \\nREMARK 220  DATE OF DATA COLLECTION        : 11-JUN-21                          \\nREMARK 220                                                                      \\nREMARK 220 REMARK: MODEL GENERATED BY ROSETTA                                   \\nREMARK 220  VERSION 2020.50.post.dev+970.commits.3700df14560                    \\nREMARK 250 ----------------------------------------                             \\nREMARK 250 CRICK PARAMETER INFORMATION                                          \\nREMARK 250 Bundle symmetry: 0 (not symmetric)                                   \\nREMARK 250 Bundle symmetry copies: 0                                            \\nREMARK 250 Number of helices per copy: 2                                        \\nREMARK 250 ---HELIX 1 PARAMETERS:---                                            \\nREMARK 250  PARAMETERS THAT ARE TYPICALLY SAMPLED:                              \\nREMARK 250    Major radius (r0,Angstroms): 5.00000000                           \\nREMARK 250    Major twist (omega0,radians/residue): 0.05000000                  \\nREMARK 250    Rotation about z-axis (delta_omega0,radians): 0.00000000          \\nREMARK 250    Roll about helix axis (delta_omega1,radians): 0.00000000          \\nREMARK 250    Registry shift (delta_t,residues): 0.00000000                     \\nREMARK 250    Major axial offset (z0_offset,Angstroms): 0.00000000              \\nREMARK 250    Minor axial offset (z1_offset,vert. Angstroms): 0.000000          \\nREMARK 250    Lateral squash (epsilon,dimensionless): 1.00000000                \\nREMARK 250  PARAMETERS THAT ARE NOT TYPICALLY SAMPLED:                          \\nREMARK 250    Residues/repeat (residues_per_repeat,dimensionless): 1            \\nREMARK 250    Repeat unit offset (repeating_unit_offset,residues): 0            \\nREMARK 250    Atoms/residue (atoms_per_residue,dimensionless):                  \\nREMARK 250      [3]                                                             \\nREMARK 250    Minor radius (r1_peratom,Angstroms):                              \\nREMARK 250      [1.53673520, 2.29431130, 1.73316244]                            \\nREMARK 250    Minor twist (omega1,radians/residue): 1.72180480                  \\nREMARK 250    Minor rise (z1,Angstroms/residue): 1.53104401                     \\nREMARK 250    Atom angular offset (delta_omega1_peratom,radians):               \\nREMARK 250      [-0.46010617, 0.00000000, 0.48012932]                           \\nREMARK 250    Atom axial offset (delta_z1_peratom,Angstroms):                   \\nREMARK 250      [-0.90470326, 0.00000000, 1.05169636]                           \\nREMARK 250    Invert helix (invert,Boolean): FALSE                              \\nREMARK 250    Dihedrals set (set_dihedrals,Boolean): TRUE                       \\nREMARK 250    Nonideal bond angles (set_bondangles,Boolean): TRUE               \\nREMARK 250    Nonideal bond lengths (set_bondlengths,Boolean): TRUE             \\nREMARK 250 ---HELIX 2 PARAMETERS:---                                            \\nREMARK 250  PARAMETERS THAT ARE TYPICALLY SAMPLED:                              \\nREMARK 250    Major radius (r0,Angstroms): 5.00000000                           \\nREMARK 250    Major twist (omega0,radians/residue): 0.05000000                  \\nREMARK 250    Rotation about z-axis (delta_omega0,radians): 3.14000000          \\nREMARK 250    Roll about helix axis (delta_omega1,radians): 0.00000000          \\nREMARK 250    Registry shift (delta_t,residues): 0.00000000                     \\nREMARK 250    Major axial offset (z0_offset,Angstroms): 0.00000000              \\nREMARK 250    Minor axial offset (z1_offset,vert. Angstroms): 0.000000          \\nREMARK 250    Lateral squash (epsilon,dimensionless): 1.00000000                \\nREMARK 250  PARAMETERS THAT ARE NOT TYPICALLY SAMPLED:                          \\nREMARK 250    Residues/repeat (residues_per_repeat,dimensionless): 1            \\nREMARK 250    Repeat unit offset (repeating_unit_offset,residues): 0            \\nREMARK 250    Atoms/residue (atoms_per_residue,dimensionless):                  \\nREMARK 250      [3]                                                             \\nREMARK 250    Minor radius (r1_peratom,Angstroms):                              \\nREMARK 250      [1.53673520, 2.29431130, 1.73316244]                            \\nREMARK 250    Minor twist (omega1,radians/residue): 1.72180480                  \\nREMARK 250    Minor rise (z1,Angstroms/residue): 1.53104401                     \\nREMARK 250    Atom angular offset (delta_omega1_peratom,radians):               \\nREMARK 250      [-0.46010617, 0.00000000, 0.48012932]                           \\nREMARK 250    Atom axial offset (delta_z1_peratom,Angstroms):                   \\nREMARK 250      [-0.90470326, 0.00000000, 1.05169636]                           \\nREMARK 250    Invert helix (invert,Boolean): FALSE                              \\nREMARK 250    Dihedrals set (set_dihedrals,Boolean): TRUE                       \\nREMARK 250    Nonideal bond angles (set_bondangles,Boolean): TRUE               \\nREMARK 250    Nonideal bond lengths (set_bondlengths,Boolean): TRUE             \\nREMARK 250 ----------------------------------------                             \\nATOM      1  N   ALA A   1       4.905  -1.129 -16.247  1.00  0.00           N  \\nATOM      2  CA  ALA A   1       4.236  -0.130 -15.422  1.00  0.00           C  \\nATOM      3  C   ALA A   1       3.572  -0.771 -14.210  1.00  0.00           C  \\nATOM      4  O   ALA A   1       3.624  -0.232 -13.105  1.00  0.00           O  \\nATOM      5  CB  ALA A   1       3.210   0.635 -16.245  1.00  0.00           C  \\nATOM      6  H   ALA A   1       4.539  -2.070 -16.274  1.00  0.00           H  \\nATOM      7  HA  ALA A   1       4.990   0.569 -15.059  1.00  0.00           H  \\nATOM      8 1HB  ALA A   1       2.719   1.377 -15.615  1.00  0.00           H  \\nATOM      9 2HB  ALA A   1       3.708   1.136 -17.074  1.00  0.00           H  \\nATOM     10 3HB  ALA A   1       2.466  -0.058 -16.634  1.00  0.00           H  \\nATOM     11  N   ALA A   2       2.950  -1.926 -14.437  1.00  0.00           N  \\nATOM     12  CA  ALA A   2       2.267  -2.640 -13.364  1.00  0.00           C  \\nATOM     13  C   ALA A   2       3.251  -3.107 -12.300  1.00  0.00           C  \\nATOM     14  O   ALA A   2       2.971  -3.027 -11.104  1.00  0.00           O  \\nATOM     15  CB  ALA A   2       1.493  -3.824 -13.925  1.00  0.00           C  \\nATOM     16  H   ALA A   2       2.954  -2.312 -15.370  1.00  0.00           H  \\nATOM     17  HA  ALA A   2       1.566  -1.952 -12.892  1.00  0.00           H  \\nATOM     18 1HB  ALA A   2       0.989  -4.347 -13.112  1.00  0.00           H  \\nATOM     19 2HB  ALA A   2       0.753  -3.469 -14.642  1.00  0.00           H  \\nATOM     20 3HB  ALA A   2       2.182  -4.506 -14.422  1.00  0.00           H  \\nATOM     21  N   ALA A   3       4.406  -3.594 -12.737  1.00  0.00           N  \\nATOM     22  CA  ALA A   3       5.428  -4.072 -11.813  1.00  0.00           C  \\nATOM     23  C   ALA A   3       5.969  -2.935 -10.955  1.00  0.00           C  \\nATOM     24  O   ALA A   3       6.195  -3.102  -9.757  1.00  0.00           O  \\nATOM     25  CB  ALA A   3       6.560  -4.743 -12.576  1.00  0.00           C  \\nATOM     26  H   ALA A   3       4.578  -3.635 -13.731  1.00  0.00           H  \\nATOM     27  HA  ALA A   3       4.968  -4.803 -11.148  1.00  0.00           H  \\nATOM     28 1HB  ALA A   3       7.315  -5.094 -11.873  1.00  0.00           H  \\nATOM     29 2HB  ALA A   3       6.168  -5.590 -13.139  1.00  0.00           H  \\nATOM     30 3HB  ALA A   3       7.010  -4.028 -13.264  1.00  0.00           H  \\nATOM     31  N   ALA A   4       6.178  -1.778 -11.565  1.00  0.00           N  \\nATOM     32  CA  ALA A   4       6.687  -0.609 -10.858  1.00  0.00           C  \\nATOM     33  C   ALA A   4       5.693  -0.122  -9.812  1.00  0.00           C  \\nATOM     34  O   ALA A   4       6.078   0.253  -8.704  1.00  0.00           O  \\nATOM     35  CB  ALA A   4       7.008   0.506 -11.843  1.00  0.00           C  \\nATOM     36  H   ALA A   4       5.979  -1.700 -12.552  1.00  0.00           H  \\nATOM     37  HA  ALA A   4       7.602  -0.897 -10.341  1.00  0.00           H  \\nATOM     38 1HB  ALA A   4       7.387   1.373 -11.300  1.00  0.00           H  \\nATOM     39 2HB  ALA A   4       7.763   0.162 -12.549  1.00  0.00           H  \\nATOM     40 3HB  ALA A   4       6.105   0.784 -12.384  1.00  0.00           H  \\nATOM     41  N   ALA A   5       4.415  -0.131 -10.175  1.00  0.00           N  \\nATOM     42  CA  ALA A   5       3.356   0.311  -9.276  1.00  0.00           C  \\nATOM     43  C   ALA A   5       3.248  -0.600  -8.061  1.00  0.00           C  \\nATOM     44  O   ALA A   5       3.060  -0.133  -6.938  1.00  0.00           O  \\nATOM     45  CB  ALA A   5       2.026   0.370 -10.013  1.00  0.00           C  \\nATOM     46  H   ALA A   5       4.169  -0.454 -11.100  1.00  0.00           H  \\nATOM     47  HA  ALA A   5       3.608   1.311  -8.923  1.00  0.00           H  \\nATOM     48 1HB  ALA A   5       1.245   0.702  -9.329  1.00  0.00           H  \\nATOM     49 2HB  ALA A   5       2.102   1.071 -10.845  1.00  0.00           H  \\nATOM     50 3HB  ALA A   5       1.778  -0.619 -10.394  1.00  0.00           H  \\nATOM     51  N   ALA A   6       3.370  -1.904  -8.302  1.00  0.00           N  \\nATOM     52  CA  ALA A   6       3.279  -2.882  -7.225  1.00  0.00           C  \\nATOM     53  C   ALA A   6       4.422  -2.716  -6.232  1.00  0.00           C  \\nATOM     54  O   ALA A   6       4.225  -2.813  -5.021  1.00  0.00           O  \\nATOM     55  CB  ALA A   6       3.270  -4.294  -7.790  1.00  0.00           C  \\nATOM     56  H   ALA A   6       3.529  -2.216  -9.250  1.00  0.00           H  \\nATOM     57  HA  ALA A   6       2.345  -2.710  -6.690  1.00  0.00           H  \\nATOM     58 1HB  ALA A   6       3.201  -5.013  -6.974  1.00  0.00           H  \\nATOM     59 2HB  ALA A   6       2.413  -4.415  -8.453  1.00  0.00           H  \\nATOM     60 3HB  ALA A   6       4.188  -4.468  -8.349  1.00  0.00           H  \\nATOM     61  N   ALA A   7       5.620  -2.466  -6.744  1.00  0.00           N  \\nATOM     62  CA  ALA A   7       6.791  -2.285  -5.895  1.00  0.00           C  \\nATOM     63  C   ALA A   7       6.654  -1.041  -5.026  1.00  0.00           C  \\nATOM     64  O   ALA A   7       7.012  -1.052  -3.848  1.00  0.00           O  \\nATOM     65  CB  ALA A   7       8.052  -2.201  -6.742  1.00  0.00           C  \\nATOM     66  H   ALA A   7       5.722  -2.401  -7.746  1.00  0.00           H  \\nATOM     67  HA  ALA A   7       6.867  -3.148  -5.233  1.00  0.00           H  \\nATOM     68 1HB  ALA A   7       8.918  -2.065  -6.094  1.00  0.00           H  \\nATOM     69 2HB  ALA A   7       8.168  -3.122  -7.314  1.00  0.00           H  \\nATOM     70 3HB  ALA A   7       7.975  -1.357  -7.426  1.00  0.00           H  \\nATOM     71  N   ALA A   8       6.136   0.031  -5.605  1.00  0.00           N  \\nATOM     72  CA  ALA A   8       5.944   1.287  -4.889  1.00  0.00           C  \\nATOM     73  C   ALA A   8       4.919   1.132  -3.772  1.00  0.00           C  \\nATOM     74  O   ALA A   8       5.098   1.662  -2.675  1.00  0.00           O  \\nATOM     75  CB  ALA A   8       5.516   2.386  -5.851  1.00  0.00           C  \\nATOM     76  H   ALA A   8       5.863  -0.018  -6.577  1.00  0.00           H  \\nATOM     77  HA  ALA A   8       6.894   1.566  -4.434  1.00  0.00           H  \\nATOM     78 1HB  ALA A   8       5.377   3.316  -5.301  1.00  0.00           H  \\nATOM     79 2HB  ALA A   8       6.286   2.524  -6.610  1.00  0.00           H  \\nATOM     80 3HB  ALA A   8       4.580   2.104  -6.331  1.00  0.00           H  \\nATOM     81  N   ALA A   9       3.847   0.402  -4.068  1.00  0.00           N  \\nATOM     82  CA  ALA A   9       2.784   0.177  -3.096  1.00  0.00           C  \\nATOM     83  C   ALA A   9       3.287  -0.629  -1.906  1.00  0.00           C  \\nATOM     84  O   ALA A   9       2.942  -0.343  -0.759  1.00  0.00           O  \\nATOM     85  CB  ALA A   9       1.607  -0.529  -3.753  1.00  0.00           C  \\nATOM     86  H   ALA A   9       3.769  -0.006  -4.988  1.00  0.00           H  \\nATOM     87  HA  ALA A   9       2.452   1.147  -2.726  1.00  0.00           H  \\nATOM     88 1HB  ALA A   9       0.821  -0.690  -3.015  1.00  0.00           H  \\nATOM     89 2HB  ALA A   9       1.221   0.087  -4.566  1.00  0.00           H  \\nATOM     90 3HB  ALA A   9       1.934  -1.488  -4.150  1.00  0.00           H  \\nATOM     91  N   ALA A  10       4.105  -1.639  -2.191  1.00  0.00           N  \\nATOM     92  CA  ALA A  10       4.650  -2.487  -1.138  1.00  0.00           C  \\nATOM     93  C   ALA A  10       5.564  -1.697  -0.210  1.00  0.00           C  \\nATOM     94  O   ALA A  10       5.534  -1.877   1.007  1.00  0.00           O  \\nATOM     95  CB  ALA A  10       5.401  -3.665  -1.742  1.00  0.00           C  \\nATOM     96  H   ALA A  10       4.351  -1.816  -3.154  1.00  0.00           H  \\nATOM     97  HA  ALA A  10       3.818  -2.867  -0.544  1.00  0.00           H  \\nATOM     98 1HB  ALA A  10       5.802  -4.289  -0.943  1.00  0.00           H  \\nATOM     99 2HB  ALA A  10       4.720  -4.254  -2.356  1.00  0.00           H  \\nATOM    100 3HB  ALA A  10       6.219  -3.296  -2.359  1.00  0.00           H  \\nATOM    101  N   ALA A  11       6.377  -0.821  -0.781  1.00  0.00           N  \\nATOM    102  CA  ALA A  11       7.294   0.000   0.000  1.00  0.00           C  \\nATOM    103  C   ALA A  11       6.537   0.962   0.907  1.00  0.00           C  \\nATOM    104  O   ALA A  11       6.914   1.169   2.060  1.00  0.00           O  \\nATOM    105  CB  ALA A  11       8.232   0.768  -0.920  1.00  0.00           C  \\nATOM    106  H   ALA A  11       6.362  -0.718  -1.786  1.00  0.00           H  \\nATOM    107  HA  ALA A  11       7.885  -0.662   0.634  1.00  0.00           H  \\nATOM    108 1HB  ALA A  11       8.911   1.376  -0.322  1.00  0.00           H  \\nATOM    109 2HB  ALA A  11       8.809   0.065  -1.521  1.00  0.00           H  \\nATOM    110 3HB  ALA A  11       7.650   1.413  -1.576  1.00  0.00           H  \\nATOM    111  N   ALA A  12       5.470   1.547   0.383  1.00  0.00           N  \\nATOM    112  CA  ALA A  12       4.650   2.487   1.138  1.00  0.00           C  \\nATOM    113  C   ALA A  12       3.963   1.801   2.312  1.00  0.00           C  \\nATOM    114  O   ALA A  12       3.883   2.357   3.407  1.00  0.00           O  \\nATOM    115  CB  ALA A  12       3.618   3.139   0.229  1.00  0.00           C  \\nATOM    116  H   ALA A  12       5.212   1.337  -0.571  1.00  0.00           H  \\nATOM    117  HA  ALA A  12       5.305   3.261   1.540  1.00  0.00           H  \\nATOM    118 1HB  ALA A  12       3.014   3.839   0.807  1.00  0.00           H  \\nATOM    119 2HB  ALA A  12       4.126   3.675  -0.573  1.00  0.00           H  \\nATOM    120 3HB  ALA A  12       2.975   2.372  -0.199  1.00  0.00           H  \\nATOM    121  N   ALA A  13       3.473   0.589   2.064  1.00  0.00           N  \\nATOM    122  CA  ALA A  13       2.784  -0.177   3.096  1.00  0.00           C  \\nATOM    123  C   ALA A  13       3.727  -0.537   4.237  1.00  0.00           C  \\nATOM    124  O   ALA A  13       3.353  -0.473   5.408  1.00  0.00           O  \\nATOM    125  CB  ALA A  13       2.170  -1.436   2.501  1.00  0.00           C  \\nATOM    126  H   ALA A  13       3.581   0.193   1.142  1.00  0.00           H  \\nATOM    127  HA  ALA A  13       1.987   0.445   3.505  1.00  0.00           H  \\nATOM    128 1HB  ALA A  13       1.659  -1.997   3.283  1.00  0.00           H  \\nATOM    129 2HB  ALA A  13       1.454  -1.160   1.726  1.00  0.00           H  \\nATOM    130 3HB  ALA A  13       2.955  -2.052   2.066  1.00  0.00           H  \\nATOM    131  N   ALA A  14       4.952  -0.915   3.887  1.00  0.00           N  \\nATOM    132  CA  ALA A  14       5.944  -1.287   4.889  1.00  0.00           C  \\nATOM    133  C   ALA A  14       6.308  -0.098   5.770  1.00  0.00           C  \\nATOM    134  O   ALA A  14       6.460  -0.236   6.983  1.00  0.00           O  \\nATOM    135  CB  ALA A  14       7.188  -1.850   4.219  1.00  0.00           C  \\nATOM    136  H   ALA A  14       5.197  -0.945   2.907  1.00  0.00           H  \\nATOM    137  HA  ALA A  14       5.508  -2.055   5.528  1.00  0.00           H  \\nATOM    138 1HB  ALA A  14       7.919  -2.123   4.981  1.00  0.00           H  \\nATOM    139 2HB  ALA A  14       6.921  -2.734   3.640  1.00  0.00           H  \\nATOM    140 3HB  ALA A  14       7.617  -1.099   3.558  1.00  0.00           H  \\nATOM    141  N   ALA A  15       6.450   1.070   5.163  1.00  0.00           N  \\nATOM    142  CA  ALA A  15       6.791   2.285   5.895  1.00  0.00           C  \\nATOM    143  C   ALA A  15       5.680   2.676   6.861  1.00  0.00           C  \\nATOM    144  O   ALA A  15       5.944   3.088   7.990  1.00  0.00           O  \\nATOM    145  CB  ALA A  15       7.076   3.424   4.927  1.00  0.00           C  \\nATOM    146  H   ALA A  15       6.319   1.125   4.164  1.00  0.00           H  \\nATOM    147  HA  ALA A  15       7.689   2.087   6.481  1.00  0.00           H  \\nATOM    148 1HB  ALA A  15       7.329   4.323   5.488  1.00  0.00           H  \\nATOM    149 2HB  ALA A  15       7.911   3.152   4.281  1.00  0.00           H  \\nATOM    150 3HB  ALA A  15       6.193   3.612   4.318  1.00  0.00           H  \\nATOM    151  N   ALA A  16       4.440   2.544   6.409  1.00  0.00           N  \\nATOM    152  CA  ALA A  16       3.279   2.882   7.225  1.00  0.00           C  \\nATOM    153  C   ALA A  16       3.169   1.962   8.434  1.00  0.00           C  \\nATOM    154  O   ALA A  16       2.854   2.406   9.537  1.00  0.00           O  \\nATOM    155  CB  ALA A  16       2.008   2.814   6.391  1.00  0.00           C  \\nATOM    156  H   ALA A  16       4.291   2.200   5.472  1.00  0.00           H  \\nATOM    157  HA  ALA A  16       3.407   3.901   7.591  1.00  0.00           H  \\nATOM    158 1HB  ALA A  16       1.150   3.068   7.014  1.00  0.00           H  \\nATOM    159 2HB  ALA A  16       2.077   3.520   5.563  1.00  0.00           H  \\nATOM    160 3HB  ALA A  16       1.884   1.806   5.998  1.00  0.00           H  \\nATOM    161  N   ALA A  17       3.433   0.677   8.207  1.00  0.00           N  \\nATOM    162  CA  ALA A  17       3.356  -0.311   9.276  1.00  0.00           C  \\nATOM    163  C   ALA A  17       4.402  -0.042  10.351  1.00  0.00           C  \\nATOM    164  O   ALA A  17       4.125  -0.164  11.544  1.00  0.00           O  \\nATOM    165  CB  ALA A  17       3.525  -1.715   8.714  1.00  0.00           C  \\nATOM    166  H   ALA A  17       3.691   0.384   7.275  1.00  0.00           H  \\nATOM    167  HA  ALA A  17       2.373  -0.233   9.740  1.00  0.00           H  \\nATOM    168 1HB  ALA A  17       3.465  -2.442   9.524  1.00  0.00           H  \\nATOM    169 2HB  ALA A  17       2.735  -1.914   7.989  1.00  0.00           H  \\nATOM    170 3HB  ALA A  17       4.495  -1.797   8.225  1.00  0.00           H  \\nATOM    171  N   ALA A  18       5.604   0.324   9.924  1.00  0.00           N  \\nATOM    172  CA  ALA A  18       6.687   0.609  10.858  1.00  0.00           C  \\nATOM    173  C   ALA A  18       6.367   1.826  11.717  1.00  0.00           C  \\nATOM    174  O   ALA A  18       6.637   1.838  12.918  1.00  0.00           O  \\nATOM    175  CB  ALA A  18       7.993   0.821  10.107  1.00  0.00           C  \\nATOM    176  H   ALA A  18       5.770   0.405   8.931  1.00  0.00           H  \\nATOM    177  HA  ALA A  18       6.797  -0.249  11.522  1.00  0.00           H  \\nATOM    178 1HB  ALA A  18       8.792   1.033  10.818  1.00  0.00           H  \\nATOM    179 2HB  ALA A  18       8.239  -0.080   9.543  1.00  0.00           H  \\nATOM    180 3HB  ALA A  18       7.886   1.659   9.421  1.00  0.00           H  \\nATOM    181  N   ALA A  19       5.792   2.850  11.106  1.00  0.00           N  \\nATOM    182  CA  ALA A  19       5.428   4.072  11.813  1.00  0.00           C  \\nATOM    183  C   ALA A  19       4.344   3.806  12.850  1.00  0.00           C  \\nATOM    184  O   ALA A  19       4.390   4.336  13.960  1.00  0.00           O  \\nATOM    185  CB  ALA A  19       4.968   5.137  10.828  1.00  0.00           C  \\nATOM    186  H   ALA A  19       5.597   2.786  10.117  1.00  0.00           H  \\nATOM    187  HA  ALA A  19       6.311   4.435  12.339  1.00  0.00           H  \\nATOM    188 1HB  ALA A  19       4.700   6.043  11.372  1.00  0.00           H  \\nATOM    189 2HB  ALA A  19       5.775   5.358  10.129  1.00  0.00           H  \\nATOM    190 3HB  ALA A  19       4.102   4.774  10.279  1.00  0.00           H  \\nATOM    191  N   ALA A  20       3.372   2.982  12.476  1.00  0.00           N  \\nATOM    192  CA  ALA A  20       2.267   2.640  13.364  1.00  0.00           C  \\nATOM    193  C   ALA A  20       2.758   1.866  14.581  1.00  0.00           C  \\nATOM    194  O   ALA A  20       3.750   1.141  14.509  1.00  0.00           O  \\nATOM    195  CB  ALA A  20       1.214   1.836  12.616  1.00  0.00           C  \\nATOM    196  H   ALA A  20       3.397   2.579  11.550  1.00  0.00           H  \\nATOM    197  HA  ALA A  20       1.818   3.568  13.718  1.00  0.00           H  \\nATOM    198 1HB  ALA A  20       0.396   1.588  13.292  1.00  0.00           H  \\nATOM    199 2HB  ALA A  20       0.831   2.425  11.783  1.00  0.00           H  \\nATOM    200 3HB  ALA A  20       1.660   0.918  12.235  1.00  0.00           H  \\nTER                                                                             \\nATOM    202  N   ALA B  21      -4.903   1.137 -16.247  1.00  0.00           N  \\nATOM    203  CA  ALA B  21      -4.236   0.137 -15.422  1.00  0.00           C  \\nATOM    204  C   ALA B  21      -3.571   0.777 -14.210  1.00  0.00           C  \\nATOM    205  O   ALA B  21      -3.624   0.238 -13.105  1.00  0.00           O  \\nATOM    206  CB  ALA B  21      -3.211  -0.630 -16.245  1.00  0.00           C  \\nATOM    207  H   ALA B  21      -4.535   2.077 -16.274  1.00  0.00           H  \\nATOM    208  HA  ALA B  21      -4.991  -0.561 -15.059  1.00  0.00           H  \\nATOM    209 1HB  ALA B  21      -2.721  -1.373 -15.615  1.00  0.00           H  \\nATOM    210 2HB  ALA B  21      -3.710  -1.130 -17.074  1.00  0.00           H  \\nATOM    211 3HB  ALA B  21      -2.466   0.062 -16.634  1.00  0.00           H  \\nATOM    212  N   ALA B  22      -2.947   1.931 -14.437  1.00  0.00           N  \\nATOM    213  CA  ALA B  22      -2.263   2.644 -13.364  1.00  0.00           C  \\nATOM    214  C   ALA B  22      -3.247   3.112 -12.300  1.00  0.00           C  \\nATOM    215  O   ALA B  22      -2.966   3.032 -11.104  1.00  0.00           O  \\nATOM    216  CB  ALA B  22      -1.487   3.827 -13.925  1.00  0.00           C  \\nATOM    217  H   ALA B  22      -2.950   2.317 -15.370  1.00  0.00           H  \\nATOM    218  HA  ALA B  22      -1.563   1.954 -12.892  1.00  0.00           H  \\nATOM    219 1HB  ALA B  22      -0.982   4.348 -13.112  1.00  0.00           H  \\nATOM    220 2HB  ALA B  22      -0.748   3.470 -14.642  1.00  0.00           H  \\nATOM    221 3HB  ALA B  22      -2.174   4.509 -14.422  1.00  0.00           H  \\nATOM    222  N   ALA B  23      -4.400   3.601 -12.737  1.00  0.00           N  \\nATOM    223  CA  ALA B  23      -5.422   4.080 -11.813  1.00  0.00           C  \\nATOM    224  C   ALA B  23      -5.964   2.945 -10.955  1.00  0.00           C  \\nATOM    225  O   ALA B  23      -6.190   3.112  -9.757  1.00  0.00           O  \\nATOM    226  CB  ALA B  23      -6.553   4.754 -12.576  1.00  0.00           C  \\nATOM    227  H   ALA B  23      -4.572   3.642 -13.731  1.00  0.00           H  \\nATOM    228  HA  ALA B  23      -4.961   4.811 -11.148  1.00  0.00           H  \\nATOM    229 1HB  ALA B  23      -7.307   5.106 -11.873  1.00  0.00           H  \\nATOM    230 2HB  ALA B  23      -6.159   5.600 -13.139  1.00  0.00           H  \\nATOM    231 3HB  ALA B  23      -7.003   4.039 -13.264  1.00  0.00           H  \\nATOM    232  N   ALA B  24      -6.175   1.788 -11.565  1.00  0.00           N  \\nATOM    233  CA  ALA B  24      -6.686   0.620 -10.858  1.00  0.00           C  \\nATOM    234  C   ALA B  24      -5.693   0.131  -9.812  1.00  0.00           C  \\nATOM    235  O   ALA B  24      -6.078  -0.243  -8.704  1.00  0.00           O  \\nATOM    236  CB  ALA B  24      -7.008  -0.495 -11.843  1.00  0.00           C  \\nATOM    237  H   ALA B  24      -5.977   1.709 -12.552  1.00  0.00           H  \\nATOM    238  HA  ALA B  24      -7.601   0.910 -10.341  1.00  0.00           H  \\nATOM    239 1HB  ALA B  24      -7.389  -1.361 -11.300  1.00  0.00           H  \\nATOM    240 2HB  ALA B  24      -7.763  -0.150 -12.549  1.00  0.00           H  \\nATOM    241 3HB  ALA B  24      -6.106  -0.775 -12.384  1.00  0.00           H  \\nATOM    242  N   ALA B  25      -4.415   0.138 -10.175  1.00  0.00           N  \\nATOM    243  CA  ALA B  25      -3.357  -0.306  -9.276  1.00  0.00           C  \\nATOM    244  C   ALA B  25      -3.247   0.605  -8.061  1.00  0.00           C  \\nATOM    245  O   ALA B  25      -3.059   0.138  -6.938  1.00  0.00           O  \\nATOM    246  CB  ALA B  25      -2.027  -0.367 -10.013  1.00  0.00           C  \\nATOM    247  H   ALA B  25      -4.169   0.461 -11.100  1.00  0.00           H  \\nATOM    248  HA  ALA B  25      -3.610  -1.306  -8.923  1.00  0.00           H  \\nATOM    249 1HB  ALA B  25      -1.246  -0.700  -9.329  1.00  0.00           H  \\nATOM    250 2HB  ALA B  25      -2.103  -1.067 -10.845  1.00  0.00           H  \\nATOM    251 3HB  ALA B  25      -1.777   0.622 -10.394  1.00  0.00           H  \\nATOM    252  N   ALA B  26      -3.367   1.909  -8.302  1.00  0.00           N  \\nATOM    253  CA  ALA B  26      -3.274   2.887  -7.225  1.00  0.00           C  \\nATOM    254  C   ALA B  26      -4.418   2.723  -6.232  1.00  0.00           C  \\nATOM    255  O   ALA B  26      -4.220   2.820  -5.021  1.00  0.00           O  \\nATOM    256  CB  ALA B  26      -3.263   4.300  -7.790  1.00  0.00           C  \\nATOM    257  H   ALA B  26      -3.525   2.222  -9.250  1.00  0.00           H  \\nATOM    258  HA  ALA B  26      -2.340   2.714  -6.690  1.00  0.00           H  \\nATOM    259 1HB  ALA B  26      -3.193   5.018  -6.974  1.00  0.00           H  \\nATOM    260 2HB  ALA B  26      -2.406   4.419  -8.453  1.00  0.00           H  \\nATOM    261 3HB  ALA B  26      -4.181   4.474  -8.349  1.00  0.00           H  \\nATOM    262  N   ALA B  27      -5.616   2.475  -6.744  1.00  0.00           N  \\nATOM    263  CA  ALA B  27      -6.787   2.295  -5.895  1.00  0.00           C  \\nATOM    264  C   ALA B  27      -6.652   1.052  -5.026  1.00  0.00           C  \\nATOM    265  O   ALA B  27      -7.010   1.063  -3.848  1.00  0.00           O  \\nATOM    266  CB  ALA B  27      -8.049   2.214  -6.742  1.00  0.00           C  \\nATOM    267  H   ALA B  27      -5.719   2.410  -7.746  1.00  0.00           H  \\nATOM    268  HA  ALA B  27      -6.862   3.158  -5.233  1.00  0.00           H  \\nATOM    269 1HB  ALA B  27      -8.915   2.080  -6.094  1.00  0.00           H  \\nATOM    270 2HB  ALA B  27      -8.163   3.135  -7.314  1.00  0.00           H  \\nATOM    271 3HB  ALA B  27      -7.973   1.370  -7.426  1.00  0.00           H  \\nATOM    272  N   ALA B  28      -6.136  -0.022  -5.605  1.00  0.00           N  \\nATOM    273  CA  ALA B  28      -5.946  -1.277  -4.889  1.00  0.00           C  \\nATOM    274  C   ALA B  28      -4.921  -1.125  -3.772  1.00  0.00           C  \\nATOM    275  O   ALA B  28      -5.101  -1.654  -2.675  1.00  0.00           O  \\nATOM    276  CB  ALA B  28      -5.520  -2.377  -5.851  1.00  0.00           C  \\nATOM    277  H   ALA B  28      -5.863   0.027  -6.577  1.00  0.00           H  \\nATOM    278  HA  ALA B  28      -6.897  -1.555  -4.434  1.00  0.00           H  \\nATOM    279 1HB  ALA B  28      -5.382  -3.308  -5.301  1.00  0.00           H  \\nATOM    280 2HB  ALA B  28      -6.290  -2.514  -6.610  1.00  0.00           H  \\nATOM    281 3HB  ALA B  28      -4.584  -2.097  -6.331  1.00  0.00           H  \\nATOM    282  N   ALA B  29      -3.848  -0.396  -4.068  1.00  0.00           N  \\nATOM    283  CA  ALA B  29      -2.784  -0.173  -3.096  1.00  0.00           C  \\nATOM    284  C   ALA B  29      -3.286   0.634  -1.906  1.00  0.00           C  \\nATOM    285  O   ALA B  29      -2.941   0.347  -0.759  1.00  0.00           O  \\nATOM    286  CB  ALA B  29      -1.606   0.531  -3.753  1.00  0.00           C  \\nATOM    287  H   ALA B  29      -3.769   0.012  -4.988  1.00  0.00           H  \\nATOM    288  HA  ALA B  29      -2.454  -1.143  -2.726  1.00  0.00           H  \\nATOM    289 1HB  ALA B  29      -0.820   0.691  -3.015  1.00  0.00           H  \\nATOM    290 2HB  ALA B  29      -1.221  -0.085  -4.566  1.00  0.00           H  \\nATOM    291 3HB  ALA B  29      -1.931   1.492  -4.150  1.00  0.00           H  \\nATOM    292  N   ALA B  30      -4.103   1.645  -2.191  1.00  0.00           N  \\nATOM    293  CA  ALA B  30      -4.646   2.495  -1.138  1.00  0.00           C  \\nATOM    294  C   ALA B  30      -5.561   1.706  -0.210  1.00  0.00           C  \\nATOM    295  O   ALA B  30      -5.531   1.885   1.007  1.00  0.00           O  \\nATOM    296  CB  ALA B  30      -5.395   3.674  -1.742  1.00  0.00           C  \\nATOM    297  H   ALA B  30      -4.348   1.823  -3.154  1.00  0.00           H  \\nATOM    298  HA  ALA B  30      -3.814   2.873  -0.544  1.00  0.00           H  \\nATOM    299 1HB  ALA B  30      -5.795   4.299  -0.943  1.00  0.00           H  \\nATOM    300 2HB  ALA B  30      -4.713   4.262  -2.356  1.00  0.00           H  \\nATOM    301 3HB  ALA B  30      -6.214   3.306  -2.359  1.00  0.00           H  \\nATOM    302  N   ALA B  31      -6.376   0.831  -0.781  1.00  0.00           N  \\nATOM    303  CA  ALA B  31      -7.294   0.012   0.000  1.00  0.00           C  \\nATOM    304  C   ALA B  31      -6.539  -0.951   0.907  1.00  0.00           C  \\nATOM    305  O   ALA B  31      -6.916  -1.158   2.060  1.00  0.00           O  \\nATOM    306  CB  ALA B  31      -8.233  -0.755  -0.920  1.00  0.00           C  \\nATOM    307  H   ALA B  31      -6.361   0.728  -1.786  1.00  0.00           H  \\nATOM    308  HA  ALA B  31      -7.884   0.674   0.634  1.00  0.00           H  \\nATOM    309 1HB  ALA B  31      -8.913  -1.362  -0.322  1.00  0.00           H  \\nATOM    310 2HB  ALA B  31      -8.809  -0.051  -1.521  1.00  0.00           H  \\nATOM    311 3HB  ALA B  31      -7.652  -1.401  -1.576  1.00  0.00           H  \\nATOM    312  N   ALA B  32      -5.472  -1.538   0.383  1.00  0.00           N  \\nATOM    313  CA  ALA B  32      -4.654  -2.480   1.138  1.00  0.00           C  \\nATOM    314  C   ALA B  32      -3.966  -1.795   2.312  1.00  0.00           C  \\nATOM    315  O   ALA B  32      -3.886  -2.351   3.407  1.00  0.00           O  \\nATOM    316  CB  ALA B  32      -3.623  -3.133   0.229  1.00  0.00           C  \\nATOM    317  H   ALA B  32      -5.214  -1.329  -0.571  1.00  0.00           H  \\nATOM    318  HA  ALA B  32      -5.310  -3.252   1.540  1.00  0.00           H  \\nATOM    319 1HB  ALA B  32      -3.020  -3.834   0.807  1.00  0.00           H  \\nATOM    320 2HB  ALA B  32      -4.132  -3.669  -0.573  1.00  0.00           H  \\nATOM    321 3HB  ALA B  32      -2.978  -2.367  -0.199  1.00  0.00           H  \\nATOM    322  N   ALA B  33      -3.474  -0.583   2.064  1.00  0.00           N  \\nATOM    323  CA  ALA B  33      -2.784   0.182   3.096  1.00  0.00           C  \\nATOM    324  C   ALA B  33      -3.726   0.543   4.237  1.00  0.00           C  \\nATOM    325  O   ALA B  33      -3.352   0.478   5.408  1.00  0.00           O  \\nATOM    326  CB  ALA B  33      -2.168   1.439   2.501  1.00  0.00           C  \\nATOM    327  H   ALA B  33      -3.581  -0.187   1.142  1.00  0.00           H  \\nATOM    328  HA  ALA B  33      -1.988  -0.442   3.505  1.00  0.00           H  \\nATOM    329 1HB  ALA B  33      -1.656   2.000   3.283  1.00  0.00           H  \\nATOM    330 2HB  ALA B  33      -1.452   1.162   1.726  1.00  0.00           H  \\nATOM    331 3HB  ALA B  33      -2.952   2.057   2.066  1.00  0.00           H  \\nATOM    332  N   ALA B  34      -4.951   0.923   3.887  1.00  0.00           N  \\nATOM    333  CA  ALA B  34      -5.942   1.296   4.889  1.00  0.00           C  \\nATOM    334  C   ALA B  34      -6.308   0.109   5.770  1.00  0.00           C  \\nATOM    335  O   ALA B  34      -6.459   0.247   6.983  1.00  0.00           O  \\nATOM    336  CB  ALA B  34      -7.185   1.862   4.219  1.00  0.00           C  \\nATOM    337  H   ALA B  34      -5.195   0.953   2.907  1.00  0.00           H  \\nATOM    338  HA  ALA B  34      -5.505   2.064   5.528  1.00  0.00           H  \\nATOM    339 1HB  ALA B  34      -7.916   2.135   4.981  1.00  0.00           H  \\nATOM    340 2HB  ALA B  34      -6.917   2.745   3.640  1.00  0.00           H  \\nATOM    341 3HB  ALA B  34      -7.615   1.111   3.558  1.00  0.00           H  \\nATOM    342  N   ALA B  35      -6.452  -1.060   5.163  1.00  0.00           N  \\nATOM    343  CA  ALA B  35      -6.795  -2.274   5.895  1.00  0.00           C  \\nATOM    344  C   ALA B  35      -5.685  -2.667   6.861  1.00  0.00           C  \\nATOM    345  O   ALA B  35      -5.949  -3.079   7.990  1.00  0.00           O  \\nATOM    346  CB  ALA B  35      -7.081  -3.412   4.927  1.00  0.00           C  \\nATOM    347  H   ALA B  35      -6.321  -1.115   4.164  1.00  0.00           H  \\nATOM    348  HA  ALA B  35      -7.692  -2.074   6.481  1.00  0.00           H  \\nATOM    349 1HB  ALA B  35      -7.335  -4.312   5.488  1.00  0.00           H  \\nATOM    350 2HB  ALA B  35      -7.916  -3.139   4.281  1.00  0.00           H  \\nATOM    351 3HB  ALA B  35      -6.199  -3.602   4.318  1.00  0.00           H  \\nATOM    352  N   ALA B  36      -4.444  -2.537   6.409  1.00  0.00           N  \\nATOM    353  CA  ALA B  36      -3.283  -2.876   7.225  1.00  0.00           C  \\nATOM    354  C   ALA B  36      -3.172  -1.957   8.434  1.00  0.00           C  \\nATOM    355  O   ALA B  36      -2.857  -2.402   9.537  1.00  0.00           O  \\nATOM    356  CB  ALA B  36      -2.012  -2.810   6.391  1.00  0.00           C  \\nATOM    357  H   ALA B  36      -4.295  -2.193   5.472  1.00  0.00           H  \\nATOM    358  HA  ALA B  36      -3.413  -3.895   7.591  1.00  0.00           H  \\nATOM    359 1HB  ALA B  36      -1.155  -3.066   7.014  1.00  0.00           H  \\nATOM    360 2HB  ALA B  36      -2.083  -3.516   5.563  1.00  0.00           H  \\nATOM    361 3HB  ALA B  36      -1.887  -1.803   5.998  1.00  0.00           H  \\nATOM    362  N   ALA B  37      -3.434  -0.671   8.207  1.00  0.00           N  \\nATOM    363  CA  ALA B  37      -3.356   0.317   9.276  1.00  0.00           C  \\nATOM    364  C   ALA B  37      -4.402   0.049  10.351  1.00  0.00           C  \\nATOM    365  O   ALA B  37      -4.124   0.170  11.544  1.00  0.00           O  \\nATOM    366  CB  ALA B  37      -3.522   1.721   8.714  1.00  0.00           C  \\nATOM    367  H   ALA B  37      -3.692  -0.378   7.275  1.00  0.00           H  \\nATOM    368  HA  ALA B  37      -2.372   0.237   9.740  1.00  0.00           H  \\nATOM    369 1HB  ALA B  37      -3.462   2.447   9.524  1.00  0.00           H  \\nATOM    370 2HB  ALA B  37      -2.732   1.919   7.989  1.00  0.00           H  \\nATOM    371 3HB  ALA B  37      -4.492   1.804   8.225  1.00  0.00           H  \\nATOM    372  N   ALA B  38      -5.605  -0.315   9.924  1.00  0.00           N  \\nATOM    373  CA  ALA B  38      -6.688  -0.598  10.858  1.00  0.00           C  \\nATOM    374  C   ALA B  38      -6.370  -1.816  11.717  1.00  0.00           C  \\nATOM    375  O   ALA B  38      -6.640  -1.828  12.918  1.00  0.00           O  \\nATOM    376  CB  ALA B  38      -7.995  -0.808  10.107  1.00  0.00           C  \\nATOM    377  H   ALA B  38      -5.771  -0.396   8.931  1.00  0.00           H  \\nATOM    378  HA  ALA B  38      -6.796   0.260  11.522  1.00  0.00           H  \\nATOM    379 1HB  ALA B  38      -8.793  -1.019  10.818  1.00  0.00           H  \\nATOM    380 2HB  ALA B  38      -8.239   0.093   9.543  1.00  0.00           H  \\nATOM    381 3HB  ALA B  38      -7.889  -1.647   9.421  1.00  0.00           H  \\nATOM    382  N   ALA B  39      -5.797  -2.841  11.106  1.00  0.00           N  \\nATOM    383  CA  ALA B  39      -5.435  -4.063  11.813  1.00  0.00           C  \\nATOM    384  C   ALA B  39      -4.350  -3.799  12.850  1.00  0.00           C  \\nATOM    385  O   ALA B  39      -4.397  -4.329  13.960  1.00  0.00           O  \\nATOM    386  CB  ALA B  39      -4.977  -5.129  10.828  1.00  0.00           C  \\nATOM    387  H   ALA B  39      -5.602  -2.777  10.117  1.00  0.00           H  \\nATOM    388  HA  ALA B  39      -6.318  -4.424  12.339  1.00  0.00           H  \\nATOM    389 1HB  ALA B  39      -4.710  -6.035  11.372  1.00  0.00           H  \\nATOM    390 2HB  ALA B  39      -5.783  -5.349  10.129  1.00  0.00           H  \\nATOM    391 3HB  ALA B  39      -4.109  -4.767  10.279  1.00  0.00           H  \\nATOM    392  N   ALA B  40      -3.377  -2.976  12.476  1.00  0.00           N  \\nATOM    393  CA  ALA B  40      -2.272  -2.636  13.364  1.00  0.00           C  \\nATOM    394  C   ALA B  40      -2.761  -1.862  14.581  1.00  0.00           C  \\nATOM    395  O   ALA B  40      -3.752  -1.135  14.509  1.00  0.00           O  \\nATOM    396  CB  ALA B  40      -1.217  -1.834  12.616  1.00  0.00           C  \\nATOM    397  H   ALA B  40      -3.401  -2.574  11.550  1.00  0.00           H  \\nATOM    398  HA  ALA B  40      -1.823  -3.565  13.718  1.00  0.00           H  \\nATOM    399 1HB  ALA B  40      -0.399  -1.588  13.292  1.00  0.00           H  \\nATOM    400 2HB  ALA B  40      -0.835  -2.424  11.783  1.00  0.00           H  \\nATOM    401 3HB  ALA B  40      -1.662  -0.916  12.235  1.00  0.00           H  \\nTER                                                                             \\nREMARK PDBinfo-LABEL:    1 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    2 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    3 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    4 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    5 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    6 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    7 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    8 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    9 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   10 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   11 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   12 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   13 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   14 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   15 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   16 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   17 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   18 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   19 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   20 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   21 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   22 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   23 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   24 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   25 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   26 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   27 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   28 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   29 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   30 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   31 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   32 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   33 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   34 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   35 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   36 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   37 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   38 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   39 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   40 PEPTIDE_STUB_EXTENSION\\n\\n# All scores below are weighted scores, not raw scores.\\n#BEGIN_POSE_ENERGIES_TABLE \\nlabel fa_atr fa_rep fa_sol fa_intra_rep fa_intra_sol_xover4 lk_ball_wtd fa_elec pro_close hbond_sr_bb hbond_lr_bb hbond_bb_sc hbond_sc dslf_fa13 omega fa_dun p_aa_pp yhh_planarity ref rama_prepro total\\nweights 1 0.55 1 0.005 1 1 1 1.25 1 1 1 1 1.25 0.4 0.7 0.6 0.625 1 0.45 NA\\npose -124.985 5.62822 96.5176 0.05425 0 -0.6028 -43.6441 0 -32.8556 0 0 0 0 -0.24588 0 -5.44976 0 52.9872 2.68022 -49.9154\\nALA_1 -1.80275 0.07143 1.41204 0.00134 0 0.00599 -0.31538 0 0 0 0 0 0 0.00417 0 -0.18334 0 1.32468 3.03052 3.54869\\nALA_2 -2.10827 0.08843 1.56674 0.00136 0 -0.00292 -0.13122 0 0 0 0 0 0 -0.00607 0 -0.15076 0 1.32468 2.89416 3.47614\\nALA_3 -2.44439 0.13223 1.97708 0.00136 0 -0.08229 -0.35542 0 0 0 0 0 0 0.00203 0 -0.14614 0 1.32468 -0.26991 0.13922\\nALA_4 -3.0948 0.13402 2.35461 0.00136 0 -0.11288 -0.82869 0 0 0 0 0 0 -0.01138 0 -0.13507 0 1.32468 -0.25864 -0.62678\\nALA_5 -3.79731 0.13678 2.56908 0.00136 0 0.0137 -1.57053 0 0 0 0 0 0 -0.01566 0 -0.14201 0 1.32468 -0.25484 -1.73475\\nALA_6 -3.53308 0.13659 2.79582 0.00136 0 0.00224 -1.63865 0 0 0 0 0 0 -0.00227 0 -0.15161 0 1.32468 -0.2669 -1.33182\\nALA_7 -3.41839 0.13406 2.73029 0.00136 0 -0.01663 -1.51714 0 0 0 0 0 0 -8e-05 0 -0.14278 0 1.32468 -0.26824 -1.17288\\nALA_8 -3.4928 0.13454 2.7638 0.00136 0 -0.00245 -1.60424 0 0 0 0 0 0 -0.01424 0 -0.13494 0 1.32468 -0.25594 -1.28021\\nALA_9 -4.00975 0.4738 2.50406 0.00136 0 0.00655 -1.70848 0 0 0 0 0 0 -0.01355 0 -0.14535 0 1.32468 -0.25705 -1.82373\\nALA_10 -3.49324 0.13599 2.77639 0.00136 0 -0.00416 -1.59628 0 0 0 0 0 0 0.00075 0 -0.15111 0 1.32468 -0.26916 -1.27479\\nALA_11 -3.4184 0.13376 2.72783 0.00136 0 -0.01635 -1.51938 0 0 0 0 0 0 -0.00339 0 -0.1395 0 1.32468 -0.26557 -1.17496\\nALA_12 -3.5301 0.13523 2.78299 0.00136 0 0.00382 -1.63899 0 0 0 0 0 0 -0.01602 0 -0.13614 0 1.32468 -0.25422 -1.32739\\nALA_13 -3.7136 0.13731 2.72719 0.00136 0 0.01263 -1.6871 0 0 0 0 0 0 -0.0104 0 -0.14835 0 1.32468 -0.26006 -1.61634\\nALA_14 -3.46027 0.13531 2.75753 0.00136 0 -0.00996 -1.55976 0 0 0 0 0 0 0.00236 0 -0.14934 0 1.32468 -0.27027 -1.22836\\nALA_15 -3.43178 0.13372 2.73283 0.00136 0 -0.01384 -1.53558 0 0 0 0 0 0 -0.00724 0 -0.13682 0 1.32468 -0.26234 -1.19502\\nALA_16 -3.57458 0.13598 2.78348 0.00136 0 0.00198 -1.53111 0 0 0 0 0 0 -0.01658 0 -0.13847 0 1.32468 -0.25375 -1.26702\\nALA_17 -3.2829 0.13709 2.56966 0.00136 0 -0.05073 -0.2939 0 0 0 0 0 0 -0.00661 0 -0.15054 0 1.32468 -0.26339 -0.01529\\nALA_18 -2.56628 0.13236 2.22509 0.00136 0 -0.03508 -0.13929 0 0 0 0 0 0 0.00219 0 -0.14658 0 1.32468 -0.27005 0.52841\\nALA_19 -2.08267 0.08654 1.68194 0.00136 0 -0.05463 -0.64274 0 0 0 0 0 0 -0.01092 0 -0.13519 0 1.32468 -0.25906 -0.0907\\nALA_20 -2.23697 0.06895 1.82034 0.00128 0 0.05361 -0.00817 0 0 0 0 0 0 0 0 0.03916 0 1.32468 -0.1252 0.93768\\nALA_21 -1.8026 0.07143 1.41204 0.00134 0 0.00599 -0.31532 0 0 0 0 0 0 0.00417 0 -0.18334 0 1.32468 3.03052 3.54891\\nALA_22 -2.10858 0.08843 1.56675 0.00136 0 -0.0029 -0.13138 0 0 0 0 0 0 -0.00607 0 -0.15076 0 1.32468 2.89416 3.47569\\nALA_23 -2.44439 0.13223 1.97708 0.00136 0 -0.08229 -0.35542 0 0 0 0 0 0 0.00203 0 -0.14614 0 1.32468 -0.26991 0.13922\\nALA_24 -3.0948 0.13402 2.35461 0.00136 0 -0.11288 -0.82869 0 0 0 0 0 0 -0.01138 0 -0.13507 0 1.32468 -0.25864 -0.62678\\nALA_25 -3.79708 0.13678 2.56904 0.00136 0 0.01369 -1.57044 0 0 0 0 0 0 -0.01566 0 -0.14201 0 1.32468 -0.25484 -1.73449\\nALA_26 -3.53315 0.13659 2.79584 0.00136 0 0.00224 -1.63864 0 0 0 0 0 0 -0.00227 0 -0.15161 0 1.32468 -0.2669 -1.33185\\nALA_27 -3.41839 0.13406 2.73029 0.00136 0 -0.01663 -1.51714 0 0 0 0 0 0 -8e-05 0 -0.14278 0 1.32468 -0.26824 -1.17288\\nALA_28 -3.49279 0.13454 2.7638 0.00136 0 -0.00245 -1.60424 0 0 0 0 0 0 -0.01424 0 -0.13494 0 1.32468 -0.25594 -1.28021\\nALA_29 -4.00975 0.4738 2.50407 0.00136 0 0.00654 -1.70844 0 0 0 0 0 0 -0.01355 0 -0.14535 0 1.32468 -0.25705 -1.82369\\nALA_30 -3.49324 0.13599 2.77639 0.00136 0 -0.00416 -1.59628 0 0 0 0 0 0 0.00075 0 -0.15111 0 1.32468 -0.26916 -1.27478\\nALA_31 -3.4184 0.13376 2.72783 0.00136 0 -0.01635 -1.51938 0 0 0 0 0 0 -0.00339 0 -0.1395 0 1.32468 -0.26557 -1.17496\\nALA_32 -3.53001 0.13523 2.78299 0.00136 0 0.00382 -1.63905 0 0 0 0 0 0 -0.01602 0 -0.13614 0 1.32468 -0.25422 -1.32736\\nALA_33 -3.71385 0.13731 2.72708 0.00136 0 0.01262 -1.68717 0 0 0 0 0 0 -0.0104 0 -0.14835 0 1.32468 -0.26006 -1.61679\\nALA_34 -3.46027 0.13531 2.75753 0.00136 0 -0.00996 -1.55976 0 0 0 0 0 0 0.00236 0 -0.14934 0 1.32468 -0.27027 -1.22836\\nALA_35 -3.43178 0.13372 2.73283 0.00136 0 -0.01384 -1.53558 0 0 0 0 0 0 -0.00724 0 -0.13682 0 1.32468 -0.26234 -1.19502\\nALA_36 -3.57427 0.13598 2.7836 0.00136 0 0.00199 -1.53097 0 0 0 0 0 0 -0.01658 0 -0.13847 0 1.32468 -0.25375 -1.26644\\nALA_37 -3.28325 0.13709 2.56966 0.00136 0 -0.05069 -0.2941 0 0 0 0 0 0 -0.00661 0 -0.15054 0 1.32468 -0.26339 -0.0158\\nALA_38 -2.56628 0.13236 2.22509 0.00136 0 -0.03508 -0.13929 0 0 0 0 0 0 0.00219 0 -0.14658 0 1.32468 -0.27005 0.52841\\nALA_39 -2.08267 0.08654 1.68194 0.00136 0 -0.05463 -0.64274 0 0 0 0 0 0 -0.01092 0 -0.13519 0 1.32468 -0.25906 -0.0907\\nALA_40 -2.23677 0.06895 1.82034 0.00128 0 0.05357 -0.00804 0 0 0 0 0 0 0 0 0.03916 0 1.32468 -0.1252 0.93797\\n#END_POSE_ENERGIES_TABLE \\n\\n\",\"pdb\");\n",
              "\tviewer_16234205772799664.setStyle({\"cartoon\": {\"color\": \"spectrum\"}, \"stick\": {\"colorscheme\": \"lightgreyCarbon\", \"radius\": 0.25}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.1694070184482745, \"y\": -0.4541068423003337, \"z\": -11.10005319597299}, \"end\": {\"x\": 3.623957999316543, \"y\": -0.23215724975842678, \"z\": -13.10478854478874}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.528978416652365, \"y\": -2.2161298185313845, \"z\": -9.249510730505522}, \"end\": {\"x\": 2.9709219889056273, \"y\": -3.0272069890931306, \"z\": -11.103541580684825}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.722409097721679, \"y\": -2.4006909988148957, \"z\": -7.74632289083335}, \"end\": {\"x\": 6.194595719580915, \"y\": -3.102453653609407, \"z\": -9.756683941833414}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.862918823601143, \"y\": -0.017668114966811584, \"z\": -6.576599708065375}, \"end\": {\"x\": 6.077661961305986, \"y\": 0.2525134088090799, \"z\": -8.703679669723572}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.1694070184482745, \"y\": -0.4541068423003337, \"z\": -11.10005319597299}, \"end\": {\"x\": 3.623957999316543, \"y\": -0.23215724975842678, \"z\": -13.10478854478874}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.768740928672904, \"y\": -0.006165444174080655, \"z\": -4.988195022043259}, \"end\": {\"x\": 3.0596556317676042, \"y\": -0.13343538704308533, \"z\": -6.937586558831088}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.528978416652365, \"y\": -2.2161298185313845, \"z\": -9.249510730505522}, \"end\": {\"x\": 2.9709219889056273, \"y\": -3.0272069890931306, \"z\": -11.103541580684825}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.350865039096124, \"y\": -1.816146038409083, \"z\": -3.1541222768941193}, \"end\": {\"x\": 4.2245533256789525, \"y\": -2.81288740276295, \"z\": -5.020807605138654}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.722409097721679, \"y\": -2.4006909988148957, \"z\": -7.74632289083335}, \"end\": {\"x\": 6.194595719580915, \"y\": -3.102453653609407, \"z\": -9.756683941833414}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.361959675612962, \"y\": -0.7180357066980767, \"z\": -1.785769501948773}, \"end\": {\"x\": 7.012136132444223, \"y\": -1.05218965541963, \"z\": -3.848207825636483}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.862918823601143, \"y\": -0.017668114966811584, \"z\": -6.576599708065375}, \"end\": {\"x\": 6.077661961305986, \"y\": 0.2525134088090799, \"z\": -8.703679669723572}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.21168513535431, \"y\": 1.3371856675336933, \"z\": -0.570709242096436}, \"end\": {\"x\": 5.097920365540859, \"y\": 1.6616815865127768, \"z\": -2.675488710644366}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.768740928672904, \"y\": -0.006165444174080655, \"z\": -4.988195022043259}, \"end\": {\"x\": 3.0596556317676042, \"y\": -0.13343538704308533, \"z\": -6.937586558831088}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.581005696446443, \"y\": 0.1930338767981878, \"z\": 1.1415148453845125}, \"end\": {\"x\": 2.9418542979692983, \"y\": -0.3425786662103132, \"z\": -0.7593325720500322}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.350865039096124, \"y\": -1.816146038409083, \"z\": -3.1541222768941193}, \"end\": {\"x\": 4.2245533256789525, \"y\": -2.81288740276295, \"z\": -5.020807605138654}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.196586250314018, \"y\": -0.9451565883448323, \"z\": 2.907276604469157}, \"end\": {\"x\": 5.533612426158804, \"y\": -1.8765794817407608, \"z\": 1.007461142399106}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.361959675612962, \"y\": -0.7180357066980767, \"z\": -1.785769501948773}, \"end\": {\"x\": 7.012136132444223, \"y\": -1.05218965541963, \"z\": -3.848207825636483}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.319020315978136, \"y\": 1.1248187700230572, \"z\": 4.1635460268324405}, \"end\": {\"x\": 6.9139807428777535, \"y\": 1.1688620923581077, \"z\": 2.0603133068719353}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.21168513535431, \"y\": 1.3371856675336933, \"z\": -0.570709242096436}, \"end\": {\"x\": 5.097920365540859, \"y\": 1.6616815865127768, \"z\": -2.675488710644366}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.291149579572995, \"y\": 2.1996717461271413, \"z\": 5.471519231572035}, \"end\": {\"x\": 3.8826020331512883, \"y\": 2.3568814675441545, \"z\": 3.4070479224982586}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.581005696446443, \"y\": 0.1930338767981878, \"z\": 1.1415148453845125}, \"end\": {\"x\": 2.9418542979692983, \"y\": -0.3425786662103132, \"z\": -0.7593325720500322}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.691129997859202, \"y\": 0.38425138737586323, \"z\": 7.2752128023647185}, \"end\": {\"x\": 3.3527476851495357, \"y\": -0.47293926000680636, \"z\": 5.408025014604342}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.196586250314018, \"y\": -0.9451565883448323, \"z\": 2.907276604469157}, \"end\": {\"x\": 5.533612426158804, \"y\": -1.8765794817407608, \"z\": 1.007461142399106}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.770439890709007, \"y\": 0.40513764239083705, \"z\": 8.931294042709522}, \"end\": {\"x\": 6.459691830801482, \"y\": -0.23646867699845442, \"z\": 6.983482895456959}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.319020315978136, \"y\": 1.1248187700230572, \"z\": 4.1635460268324405}, \"end\": {\"x\": 6.9139807428777535, \"y\": 1.1688620923581077, \"z\": 2.0603133068719353}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.597498887444632, \"y\": 2.786258740530911, \"z\": 10.116802138605896}, \"end\": {\"x\": 5.943612297271183, \"y\": 3.0882230798153936, \"z\": 7.990349725508292}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.291149579572995, \"y\": 2.1996717461271413, \"z\": 5.471519231572035}, \"end\": {\"x\": 3.8826020331512883, \"y\": 2.3568814675441545, \"z\": 3.4070479224982586}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.396939295644632, \"y\": 2.5791351114309555, \"z\": 11.549812839786894}, \"end\": {\"x\": 2.853664452793035, \"y\": 2.406462801205477, \"z\": 9.537425076063954}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.691129997859202, \"y\": 0.38425138737586323, \"z\": 7.2752128023647185}, \"end\": {\"x\": 3.3527476851495357, \"y\": -0.47293926000680636, \"z\": 5.408025014604342}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.770439890709007, \"y\": 0.40513764239083705, \"z\": 8.931294042709522}, \"end\": {\"x\": 6.459691830801482, \"y\": -0.23646867699845442, \"z\": 6.983482895456959}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.597498887444632, \"y\": 2.786258740530911, \"z\": 10.116802138605896}, \"end\": {\"x\": 5.943612297271183, \"y\": 3.0882230798153936, \"z\": 7.990349725508292}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.396939295644632, \"y\": 2.5791351114309555, \"z\": 11.549812839786894}, \"end\": {\"x\": 2.853664452793035, \"y\": 2.406462801205477, \"z\": 9.537425076063954}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.168678495917393, \"y\": 0.46074668461708435, \"z\": -11.100053195972988}, \"end\": {\"x\": -3.6235836572295064, \"y\": 0.23792866259661516, \"z\": -13.104788544788734}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.5254444153274243, \"y\": 2.2217474456424706, \"z\": -9.249510730505524}, \"end\": {\"x\": -2.9660969309270975, \"y\": 3.031934797340165, \"z\": -11.103541580684825}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.718578372626965, \"y\": 2.4098017656234356, \"z\": -7.746322890833351}, \"end\": {\"x\": -6.189646731286077, \"y\": 3.112315559792127, \"z\": -9.756683941833412}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.86288324864783, \"y\": 0.02700568732236141, \"z\": -6.576599708065375}, \"end\": {\"x\": -6.078056419391683, \"y\": -0.242833482505183, \"z\": -8.703679669723567}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.168678495917393, \"y\": 0.46074668461708435, \"z\": -11.100053195972988}, \"end\": {\"x\": -3.6235836572295064, \"y\": 0.23792866259661516, \"z\": -13.104788544788734}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.7687263294699216, \"y\": 0.012167732586157769, \"z\": -4.988195022043261}, \"end\": {\"x\": -3.0594392350322845, \"y\": 0.13830818727604366, \"z\": -6.937586558831089}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.5254444153274243, \"y\": 2.2217474456424706, \"z\": -9.249510730505524}, \"end\": {\"x\": -2.9660969309270975, \"y\": 3.031934797340165, \"z\": -11.103541580684825}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.3479670307289755, \"y\": 1.8230731529348416, \"z\": -3.1541222768941224}, \"end\": {\"x\": -4.2200680144685485, \"y\": 2.8196120824303246, \"z\": -5.020807605138655}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.718578372626965, \"y\": 2.4098017656234356, \"z\": -7.746322890833351}, \"end\": {\"x\": -6.189646731286077, \"y\": 3.112315559792127, \"z\": -9.756683941833412}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.360808025252289, \"y\": 0.7281671896651079, \"z\": -1.7857695019487734}, \"end\": {\"x\": -7.01045146622167, \"y\": 1.0633562200186129, \"z\": -3.8482078256364836}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.86288324864783, \"y\": 0.02700568732236141, \"z\": -6.576599708065375}, \"end\": {\"x\": -6.078056419391683, \"y\": -0.242833482505183, \"z\": -8.703679669723567}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.213808198170849, \"y\": -1.3288835660872982, \"z\": -0.5707092420964386}, \"end\": {\"x\": -5.100560382013878, \"y\": -1.6535602613095794, \"z\": -2.675488710644367}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.7687263294699216, \"y\": 0.012167732586157769, \"z\": -4.988195022043261}, \"end\": {\"x\": -3.0594392350322845, \"y\": 0.13830818727604366, \"z\": -6.937586558831089}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.5813085907223874, \"y\": -0.18733033281222844, \"z\": 1.1415148453845099}, \"end\": {\"x\": -2.9413049579846318, \"y\": 0.3472635845547798, \"z\": -0.7593325720500358}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.3479670307289755, \"y\": 1.8230731529348416, \"z\": -3.1541222768941224}, \"end\": {\"x\": -4.2200680144685485, \"y\": 2.8196120824303246, \"z\": -5.020807605138655}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.195074353229818, \"y\": 0.9534317478761036, \"z\": 2.907276604469158}, \"end\": {\"x\": -5.530616668245941, \"y\": 1.8853902256959216, \"z\": 1.0074611423991024}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.360808025252289, \"y\": 0.7281671896651079, \"z\": -1.7857695019487734}, \"end\": {\"x\": -7.01045146622167, \"y\": 1.0633562200186129, \"z\": -3.8482078256364836}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.3208037476332795, \"y\": -1.1147533373108025, \"z\": 4.1635460268324405}, \"end\": {\"x\": -6.915833565686739, \"y\": -1.1578490383278288, \"z\": 2.060313306871933}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.213808198170849, \"y\": -1.3288835660872982, \"z\": -0.5707092420964386}, \"end\": {\"x\": -5.100560382013878, \"y\": -1.6535602613095794, \"z\": -2.675488710644367}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.294647450847934, \"y\": -2.1928346444510507, \"z\": 5.471519231572033}, \"end\": {\"x\": -3.886350803097142, \"y\": -2.350694840924633, \"z\": 3.4070479224982586}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.5813085907223874, \"y\": -0.18733033281222844, \"z\": 1.1415148453845099}, \"end\": {\"x\": -2.9413049579846318, \"y\": 0.3472635845547798, \"z\": -0.7593325720500358}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.6917372955934393, \"y\": -0.37837221108418234, \"z\": 7.27521280236472}, \"end\": {\"x\": -3.3519902048602006, \"y\": 0.47827842356997363, \"z\": 5.408025014604343}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.195074353229818, \"y\": 0.9534317478761036, \"z\": 2.907276604469158}, \"end\": {\"x\": -5.530616668245941, \"y\": 1.8853902256959216, \"z\": 1.0074611423991024}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.771077815866737, \"y\": -0.39594682064457354, \"z\": 8.931294042709524}, \"end\": {\"x\": -6.459307025624145, \"y\": 0.24675642412567778, \"z\": 6.983482895456962}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.3208037476332795, \"y\": -1.1147533373108025, \"z\": 4.1635460268324405}, \"end\": {\"x\": -6.915833565686739, \"y\": -1.1578490383278288, \"z\": 2.060313306871933}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.601929331400129, \"y\": -2.7773403338675617, \"z\": 10.116802138605893}, \"end\": {\"x\": -5.948523226646209, \"y\": -3.078753051647397, \"z\": 7.990349725508294}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.294647450847934, \"y\": -2.1928346444510507, \"z\": 5.471519231572033}, \"end\": {\"x\": -3.886350803097142, \"y\": -2.350694840924633, \"z\": 3.4070479224982586}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.4010426544572967, \"y\": -2.573721695108579, \"z\": 11.549812839786892}, \"end\": {\"x\": -2.857493493567745, \"y\": -2.4019148521415623, \"z\": 9.537425076063956}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.6917372955934393, \"y\": -0.37837221108418234, \"z\": 7.27521280236472}, \"end\": {\"x\": -3.3519902048602006, \"y\": 0.47827842356997363, \"z\": 5.408025014604343}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.771077815866737, \"y\": -0.39594682064457354, \"z\": 8.931294042709524}, \"end\": {\"x\": -6.459307025624145, \"y\": 0.24675642412567778, \"z\": 6.983482895456962}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.601929331400129, \"y\": -2.7773403338675617, \"z\": 10.116802138605893}, \"end\": {\"x\": -5.948523226646209, \"y\": -3.078753051647397, \"z\": 7.990349725508294}});\n",
              "\tviewer_16234205772799664.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.4010426544572967, \"y\": -2.573721695108579, \"z\": 11.549812839786892}, \"end\": {\"x\": -2.857493493567745, \"y\": -2.4019148521415623, \"z\": 9.537425076063956}});\n",
              "\tviewer_16234205772799664.zoomTo();\n",
              "viewer_16234205772799664.render();\n",
              "});\n",
              "</script>"
            ]
          },
          "metadata": {
            "tags": []
          }
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "405bKySXmdlP"
      },
      "source": [
        "Ok, so now lets have a look at this - it is quite fast as we are simply manipulating the backbone and doing doing any sequence design (yet).  Which axis changed?  Lets try changing some other values!\n",
        "Note that if we have dump_pdbs on - we will be overwriting them, so make sure to move the current ones to another directory if you "
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "1l-irfmFmdlP"
      },
      "source": [
        "## Changing omega0\n",
        "\n",
        "This time, we will change the parameters a bit through code and observe them by directly dumping them or through the PyMolMover.\n",
        "Instead of the BundleGridSampler, we will use the MakeBundle mover.\n",
        "\n",
        "Note the use of Python3's awesome f-strings here, and the speed.  Because we need to parse the xml object and create a new mover, pose, etc. each time, this is much slower than the sampler - but it does allow us a bit more flexibility to use python the way it was intended.  Once again, the `MakeBundle` mover does not have an adaquite code-level interface to allow us to set variables the traditional way. If you end up writing C++ Rosetta code, make sure to always keep the XML and code-level interfaces feature-complete!\n",
        "\n",
        "**omega0**: Major helix twist per residue, stored in radians."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "XgtI_ivXmdlP",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "e8baa346-caa2-4905-86c9-1c062df59fbb"
      },
      "source": [
        "import math\n",
        "\n",
        "\n",
        "for i in range(0, 32, 2):\n",
        "\n",
        "    i_rad = math.radians(i)\n",
        "    pose = pyrosetta.rosetta.core.pose.Pose()\n",
        "\n",
        "    sampler_string = f'''\n",
        "    <SCOREFXNS>\n",
        "        <ScoreFunction name=\"sfxn1\" weights=\"ref2015\"/>\n",
        "    </SCOREFXNS>\n",
        "    <MOVERS>\n",
        "        <MakeBundle name=\"bgs1\"\n",
        "                     helix_length=\"20\"\n",
        "                     r0=\"5\"\n",
        "                     omega0=\"{i_rad}\"\n",
        "                     delta_omega0=\"0\"\n",
        "                     delta_omega1=\"0\"\n",
        "                     delta_t=\"0\">\n",
        "        <Helix/>\n",
        "        <Helix delta_omega0=\"3.14\" r0_copies_helix=\"1\"/>\n",
        "        \n",
        "        </MakeBundle>\n",
        "    </MOVERS>'''\n",
        "    pyrosetta.rosetta.protocols.rosetta_scripts.XmlObjects.create_from_string(sampler_string).get_mover(\"bgs1\").apply(pose)\n",
        "    pose.dump_pdb('outputs/omega0_'+str(i)+'.pdb')"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Generating XML Schema for rosetta_scripts...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Initializing schema validator...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Validating input script...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Parsed script:\n",
            "<ROSETTASCRIPTS>\n",
            "\t<SCOREFXNS>\n",
            "\t\t<ScoreFunction name=\"sfxn1\" weights=\"ref2015\"/>\n",
            "\t</SCOREFXNS>\n",
            "\t<MOVERS>\n",
            "\t\t<MakeBundle delta_omega0=\"0\" delta_omega1=\"0\" delta_t=\"0\" helix_length=\"20\" name=\"bgs1\" omega0=\"0.0\" r0=\"5\">\n",
            "\t\t\t<Helix/>\n",
            "\t\t\t<Helix delta_omega0=\"3.14\" r0_copies_helix=\"1\"/>\n",
            "\t\t</MakeBundle>\n",
            "\t</MOVERS>\n",
            "\t<PROTOCOLS/>\n",
            "</ROSETTASCRIPTS>\n",
            "INFO:rosetta:core.scoring.ScoreFunctionFactory: SCOREFUNCTION: ref2015\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"bgs1\" of type MakeBundle\n",
            "INFO:rosetta:protocols.rosetta_scripts.ParsedProtocol: ParsedProtocol mover with the following settings\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Generating XML Schema for rosetta_scripts...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Initializing schema validator...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Validating input script...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Parsed script:\n",
            "<ROSETTASCRIPTS>\n",
            "\t<SCOREFXNS>\n",
            "\t\t<ScoreFunction name=\"sfxn1\" weights=\"ref2015\"/>\n",
            "\t</SCOREFXNS>\n",
            "\t<MOVERS>\n",
            "\t\t<MakeBundle delta_omega0=\"0\" delta_omega1=\"0\" delta_t=\"0\" helix_length=\"20\" name=\"bgs1\" omega0=\"0.03490658503988659\" r0=\"5\">\n",
            "\t\t\t<Helix/>\n",
            "\t\t\t<Helix delta_omega0=\"3.14\" r0_copies_helix=\"1\"/>\n",
            "\t\t</MakeBundle>\n",
            "\t</MOVERS>\n",
            "\t<PROTOCOLS/>\n",
            "</ROSETTASCRIPTS>\n",
            "INFO:rosetta:core.scoring.ScoreFunctionFactory: SCOREFUNCTION: ref2015\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"bgs1\" of type MakeBundle\n",
            "INFO:rosetta:protocols.rosetta_scripts.ParsedProtocol: ParsedProtocol mover with the following settings\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Generating XML Schema for rosetta_scripts...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Initializing schema validator...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Validating input script...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Parsed script:\n",
            "<ROSETTASCRIPTS>\n",
            "\t<SCOREFXNS>\n",
            "\t\t<ScoreFunction name=\"sfxn1\" weights=\"ref2015\"/>\n",
            "\t</SCOREFXNS>\n",
            "\t<MOVERS>\n",
            "\t\t<MakeBundle delta_omega0=\"0\" delta_omega1=\"0\" delta_t=\"0\" helix_length=\"20\" name=\"bgs1\" omega0=\"0.06981317007977318\" r0=\"5\">\n",
            "\t\t\t<Helix/>\n",
            "\t\t\t<Helix delta_omega0=\"3.14\" r0_copies_helix=\"1\"/>\n",
            "\t\t</MakeBundle>\n",
            "\t</MOVERS>\n",
            "\t<PROTOCOLS/>\n",
            "</ROSETTASCRIPTS>\n",
            "INFO:rosetta:core.scoring.ScoreFunctionFactory: SCOREFUNCTION: ref2015\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"bgs1\" of type MakeBundle\n",
            "INFO:rosetta:protocols.rosetta_scripts.ParsedProtocol: ParsedProtocol mover with the following settings\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Generating XML Schema for rosetta_scripts...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Initializing schema validator...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Validating input script...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Parsed script:\n",
            "<ROSETTASCRIPTS>\n",
            "\t<SCOREFXNS>\n",
            "\t\t<ScoreFunction name=\"sfxn1\" weights=\"ref2015\"/>\n",
            "\t</SCOREFXNS>\n",
            "\t<MOVERS>\n",
            "\t\t<MakeBundle delta_omega0=\"0\" delta_omega1=\"0\" delta_t=\"0\" helix_length=\"20\" name=\"bgs1\" omega0=\"0.10471975511965978\" r0=\"5\">\n",
            "\t\t\t<Helix/>\n",
            "\t\t\t<Helix delta_omega0=\"3.14\" r0_copies_helix=\"1\"/>\n",
            "\t\t</MakeBundle>\n",
            "\t</MOVERS>\n",
            "\t<PROTOCOLS/>\n",
            "</ROSETTASCRIPTS>\n",
            "INFO:rosetta:core.scoring.ScoreFunctionFactory: SCOREFUNCTION: ref2015\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"bgs1\" of type MakeBundle\n",
            "INFO:rosetta:protocols.rosetta_scripts.ParsedProtocol: ParsedProtocol mover with the following settings\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Generating XML Schema for rosetta_scripts...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Initializing schema validator...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Validating input script...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Parsed script:\n",
            "<ROSETTASCRIPTS>\n",
            "\t<SCOREFXNS>\n",
            "\t\t<ScoreFunction name=\"sfxn1\" weights=\"ref2015\"/>\n",
            "\t</SCOREFXNS>\n",
            "\t<MOVERS>\n",
            "\t\t<MakeBundle delta_omega0=\"0\" delta_omega1=\"0\" delta_t=\"0\" helix_length=\"20\" name=\"bgs1\" omega0=\"0.13962634015954636\" r0=\"5\">\n",
            "\t\t\t<Helix/>\n",
            "\t\t\t<Helix delta_omega0=\"3.14\" r0_copies_helix=\"1\"/>\n",
            "\t\t</MakeBundle>\n",
            "\t</MOVERS>\n",
            "\t<PROTOCOLS/>\n",
            "</ROSETTASCRIPTS>\n",
            "INFO:rosetta:core.scoring.ScoreFunctionFactory: SCOREFUNCTION: ref2015\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"bgs1\" of type MakeBundle\n",
            "INFO:rosetta:protocols.rosetta_scripts.ParsedProtocol: ParsedProtocol mover with the following settings\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Generating XML Schema for rosetta_scripts...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Initializing schema validator...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Validating input script...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Parsed script:\n",
            "<ROSETTASCRIPTS>\n",
            "\t<SCOREFXNS>\n",
            "\t\t<ScoreFunction name=\"sfxn1\" weights=\"ref2015\"/>\n",
            "\t</SCOREFXNS>\n",
            "\t<MOVERS>\n",
            "\t\t<MakeBundle delta_omega0=\"0\" delta_omega1=\"0\" delta_t=\"0\" helix_length=\"20\" name=\"bgs1\" omega0=\"0.17453292519943295\" r0=\"5\">\n",
            "\t\t\t<Helix/>\n",
            "\t\t\t<Helix delta_omega0=\"3.14\" r0_copies_helix=\"1\"/>\n",
            "\t\t</MakeBundle>\n",
            "\t</MOVERS>\n",
            "\t<PROTOCOLS/>\n",
            "</ROSETTASCRIPTS>\n",
            "INFO:rosetta:core.scoring.ScoreFunctionFactory: SCOREFUNCTION: ref2015\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"bgs1\" of type MakeBundle\n",
            "INFO:rosetta:protocols.rosetta_scripts.ParsedProtocol: ParsedProtocol mover with the following settings\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Generating XML Schema for rosetta_scripts...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Initializing schema validator...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Validating input script...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Parsed script:\n",
            "<ROSETTASCRIPTS>\n",
            "\t<SCOREFXNS>\n",
            "\t\t<ScoreFunction name=\"sfxn1\" weights=\"ref2015\"/>\n",
            "\t</SCOREFXNS>\n",
            "\t<MOVERS>\n",
            "\t\t<MakeBundle delta_omega0=\"0\" delta_omega1=\"0\" delta_t=\"0\" helix_length=\"20\" name=\"bgs1\" omega0=\"0.20943951023931956\" r0=\"5\">\n",
            "\t\t\t<Helix/>\n",
            "\t\t\t<Helix delta_omega0=\"3.14\" r0_copies_helix=\"1\"/>\n",
            "\t\t</MakeBundle>\n",
            "\t</MOVERS>\n",
            "\t<PROTOCOLS/>\n",
            "</ROSETTASCRIPTS>\n",
            "INFO:rosetta:core.scoring.ScoreFunctionFactory: SCOREFUNCTION: ref2015\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"bgs1\" of type MakeBundle\n",
            "INFO:rosetta:protocols.rosetta_scripts.ParsedProtocol: ParsedProtocol mover with the following settings\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Generating XML Schema for rosetta_scripts...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Initializing schema validator...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Validating input script...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Parsed script:\n",
            "<ROSETTASCRIPTS>\n",
            "\t<SCOREFXNS>\n",
            "\t\t<ScoreFunction name=\"sfxn1\" weights=\"ref2015\"/>\n",
            "\t</SCOREFXNS>\n",
            "\t<MOVERS>\n",
            "\t\t<MakeBundle delta_omega0=\"0\" delta_omega1=\"0\" delta_t=\"0\" helix_length=\"20\" name=\"bgs1\" omega0=\"0.24434609527920614\" r0=\"5\">\n",
            "\t\t\t<Helix/>\n",
            "\t\t\t<Helix delta_omega0=\"3.14\" r0_copies_helix=\"1\"/>\n",
            "\t\t</MakeBundle>\n",
            "\t</MOVERS>\n",
            "\t<PROTOCOLS/>\n",
            "</ROSETTASCRIPTS>\n",
            "INFO:rosetta:core.scoring.ScoreFunctionFactory: SCOREFUNCTION: ref2015\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"bgs1\" of type MakeBundle\n",
            "INFO:rosetta:protocols.rosetta_scripts.ParsedProtocol: ParsedProtocol mover with the following settings\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Generating XML Schema for rosetta_scripts...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Initializing schema validator...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Validating input script...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Parsed script:\n",
            "<ROSETTASCRIPTS>\n",
            "\t<SCOREFXNS>\n",
            "\t\t<ScoreFunction name=\"sfxn1\" weights=\"ref2015\"/>\n",
            "\t</SCOREFXNS>\n",
            "\t<MOVERS>\n",
            "\t\t<MakeBundle delta_omega0=\"0\" delta_omega1=\"0\" delta_t=\"0\" helix_length=\"20\" name=\"bgs1\" omega0=\"0.2792526803190927\" r0=\"5\">\n",
            "\t\t\t<Helix/>\n",
            "\t\t\t<Helix delta_omega0=\"3.14\" r0_copies_helix=\"1\"/>\n",
            "\t\t</MakeBundle>\n",
            "\t</MOVERS>\n",
            "\t<PROTOCOLS/>\n",
            "</ROSETTASCRIPTS>\n",
            "INFO:rosetta:core.scoring.ScoreFunctionFactory: SCOREFUNCTION: ref2015\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"bgs1\" of type MakeBundle\n",
            "INFO:rosetta:protocols.rosetta_scripts.ParsedProtocol: ParsedProtocol mover with the following settings\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Generating XML Schema for rosetta_scripts...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Initializing schema validator...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Validating input script...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Parsed script:\n",
            "<ROSETTASCRIPTS>\n",
            "\t<SCOREFXNS>\n",
            "\t\t<ScoreFunction name=\"sfxn1\" weights=\"ref2015\"/>\n",
            "\t</SCOREFXNS>\n",
            "\t<MOVERS>\n",
            "\t\t<MakeBundle delta_omega0=\"0\" delta_omega1=\"0\" delta_t=\"0\" helix_length=\"20\" name=\"bgs1\" omega0=\"0.3141592653589793\" r0=\"5\">\n",
            "\t\t\t<Helix/>\n",
            "\t\t\t<Helix delta_omega0=\"3.14\" r0_copies_helix=\"1\"/>\n",
            "\t\t</MakeBundle>\n",
            "\t</MOVERS>\n",
            "\t<PROTOCOLS/>\n",
            "</ROSETTASCRIPTS>\n",
            "INFO:rosetta:core.scoring.ScoreFunctionFactory: SCOREFUNCTION: ref2015\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"bgs1\" of type MakeBundle\n",
            "INFO:rosetta:protocols.rosetta_scripts.ParsedProtocol: ParsedProtocol mover with the following settings\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Generating XML Schema for rosetta_scripts...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Initializing schema validator...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Validating input script...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Parsed script:\n",
            "<ROSETTASCRIPTS>\n",
            "\t<SCOREFXNS>\n",
            "\t\t<ScoreFunction name=\"sfxn1\" weights=\"ref2015\"/>\n",
            "\t</SCOREFXNS>\n",
            "\t<MOVERS>\n",
            "\t\t<MakeBundle delta_omega0=\"0\" delta_omega1=\"0\" delta_t=\"0\" helix_length=\"20\" name=\"bgs1\" omega0=\"0.3490658503988659\" r0=\"5\">\n",
            "\t\t\t<Helix/>\n",
            "\t\t\t<Helix delta_omega0=\"3.14\" r0_copies_helix=\"1\"/>\n",
            "\t\t</MakeBundle>\n",
            "\t</MOVERS>\n",
            "\t<PROTOCOLS/>\n",
            "</ROSETTASCRIPTS>\n",
            "INFO:rosetta:core.scoring.ScoreFunctionFactory: SCOREFUNCTION: ref2015\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"bgs1\" of type MakeBundle\n",
            "INFO:rosetta:protocols.rosetta_scripts.ParsedProtocol: ParsedProtocol mover with the following settings\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Generating XML Schema for rosetta_scripts...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Initializing schema validator...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Validating input script...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Parsed script:\n",
            "<ROSETTASCRIPTS>\n",
            "\t<SCOREFXNS>\n",
            "\t\t<ScoreFunction name=\"sfxn1\" weights=\"ref2015\"/>\n",
            "\t</SCOREFXNS>\n",
            "\t<MOVERS>\n",
            "\t\t<MakeBundle delta_omega0=\"0\" delta_omega1=\"0\" delta_t=\"0\" helix_length=\"20\" name=\"bgs1\" omega0=\"0.3839724354387525\" r0=\"5\">\n",
            "\t\t\t<Helix/>\n",
            "\t\t\t<Helix delta_omega0=\"3.14\" r0_copies_helix=\"1\"/>\n",
            "\t\t</MakeBundle>\n",
            "\t</MOVERS>\n",
            "\t<PROTOCOLS/>\n",
            "</ROSETTASCRIPTS>\n",
            "INFO:rosetta:core.scoring.ScoreFunctionFactory: SCOREFUNCTION: ref2015\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"bgs1\" of type MakeBundle\n",
            "INFO:rosetta:protocols.rosetta_scripts.ParsedProtocol: ParsedProtocol mover with the following settings\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Generating XML Schema for rosetta_scripts...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Initializing schema validator...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Validating input script...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Parsed script:\n",
            "<ROSETTASCRIPTS>\n",
            "\t<SCOREFXNS>\n",
            "\t\t<ScoreFunction name=\"sfxn1\" weights=\"ref2015\"/>\n",
            "\t</SCOREFXNS>\n",
            "\t<MOVERS>\n",
            "\t\t<MakeBundle delta_omega0=\"0\" delta_omega1=\"0\" delta_t=\"0\" helix_length=\"20\" name=\"bgs1\" omega0=\"0.4188790204786391\" r0=\"5\">\n",
            "\t\t\t<Helix/>\n",
            "\t\t\t<Helix delta_omega0=\"3.14\" r0_copies_helix=\"1\"/>\n",
            "\t\t</MakeBundle>\n",
            "\t</MOVERS>\n",
            "\t<PROTOCOLS/>\n",
            "</ROSETTASCRIPTS>\n",
            "INFO:rosetta:core.scoring.ScoreFunctionFactory: SCOREFUNCTION: ref2015\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"bgs1\" of type MakeBundle\n",
            "INFO:rosetta:protocols.rosetta_scripts.ParsedProtocol: ParsedProtocol mover with the following settings\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Generating XML Schema for rosetta_scripts...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Initializing schema validator...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Validating input script...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Parsed script:\n",
            "<ROSETTASCRIPTS>\n",
            "\t<SCOREFXNS>\n",
            "\t\t<ScoreFunction name=\"sfxn1\" weights=\"ref2015\"/>\n",
            "\t</SCOREFXNS>\n",
            "\t<MOVERS>\n",
            "\t\t<MakeBundle delta_omega0=\"0\" delta_omega1=\"0\" delta_t=\"0\" helix_length=\"20\" name=\"bgs1\" omega0=\"0.4537856055185257\" r0=\"5\">\n",
            "\t\t\t<Helix/>\n",
            "\t\t\t<Helix delta_omega0=\"3.14\" r0_copies_helix=\"1\"/>\n",
            "\t\t</MakeBundle>\n",
            "\t</MOVERS>\n",
            "\t<PROTOCOLS/>\n",
            "</ROSETTASCRIPTS>\n",
            "INFO:rosetta:core.scoring.ScoreFunctionFactory: SCOREFUNCTION: ref2015\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"bgs1\" of type MakeBundle\n",
            "INFO:rosetta:protocols.rosetta_scripts.ParsedProtocol: ParsedProtocol mover with the following settings\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Generating XML Schema for rosetta_scripts...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Initializing schema validator...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Validating input script...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Parsed script:\n",
            "<ROSETTASCRIPTS>\n",
            "\t<SCOREFXNS>\n",
            "\t\t<ScoreFunction name=\"sfxn1\" weights=\"ref2015\"/>\n",
            "\t</SCOREFXNS>\n",
            "\t<MOVERS>\n",
            "\t\t<MakeBundle delta_omega0=\"0\" delta_omega1=\"0\" delta_t=\"0\" helix_length=\"20\" name=\"bgs1\" omega0=\"0.4886921905584123\" r0=\"5\">\n",
            "\t\t\t<Helix/>\n",
            "\t\t\t<Helix delta_omega0=\"3.14\" r0_copies_helix=\"1\"/>\n",
            "\t\t</MakeBundle>\n",
            "\t</MOVERS>\n",
            "\t<PROTOCOLS/>\n",
            "</ROSETTASCRIPTS>\n",
            "INFO:rosetta:core.scoring.ScoreFunctionFactory: SCOREFUNCTION: ref2015\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"bgs1\" of type MakeBundle\n",
            "INFO:rosetta:protocols.rosetta_scripts.ParsedProtocol: ParsedProtocol mover with the following settings\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Generating XML Schema for rosetta_scripts...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Initializing schema validator...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Validating input script...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Parsed script:\n",
            "<ROSETTASCRIPTS>\n",
            "\t<SCOREFXNS>\n",
            "\t\t<ScoreFunction name=\"sfxn1\" weights=\"ref2015\"/>\n",
            "\t</SCOREFXNS>\n",
            "\t<MOVERS>\n",
            "\t\t<MakeBundle delta_omega0=\"0\" delta_omega1=\"0\" delta_t=\"0\" helix_length=\"20\" name=\"bgs1\" omega0=\"0.5235987755982988\" r0=\"5\">\n",
            "\t\t\t<Helix/>\n",
            "\t\t\t<Helix delta_omega0=\"3.14\" r0_copies_helix=\"1\"/>\n",
            "\t\t</MakeBundle>\n",
            "\t</MOVERS>\n",
            "\t<PROTOCOLS/>\n",
            "</ROSETTASCRIPTS>\n",
            "INFO:rosetta:core.scoring.ScoreFunctionFactory: SCOREFUNCTION: ref2015\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"bgs1\" of type MakeBundle\n",
            "INFO:rosetta:protocols.rosetta_scripts.ParsedProtocol: ParsedProtocol mover with the following settings\n"
          ],
          "name": "stderr"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "MQOtRxiemdlP"
      },
      "source": [
        "What is the maximum value of omega0?  Which PDBs are empty??"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 968,
          "referenced_widgets": [
            "3c8a0274c508449da448f6b4d261772c",
            "8a44a0ec76ce4394bbb2dd6dbcb257d9",
            "532e4bfede5a4e899c40e47bb1357fd8",
            "6a42e493f4cb4d478acb1172b8f113e6",
            "f701f8f9e4b94d419572b84530cb7745",
            "0dfa943289ba4764887d5807751eb8c1",
            "45c97a73e3f94334aa04b0a6a15c866a"
          ]
        },
        "id": "GGCvMzAQbU3F",
        "outputId": "7222a0cc-ce1f-4e6b-85af-55762da6a7f0"
      },
      "source": [
        "## iterate through all PDBs you've created and send them to viewer\n",
        "import pyrosetta.distributed.viewer as viewer\n",
        "omega_pose = pyrosetta.pose_from_pdb('outputs/omega0_0.pdb')\n",
        "omega_pose_2 = pyrosetta.pose_from_pdb('outputs/omega0_2.pdb')\n",
        "view = viewer.init([omega_pose, omega_pose_1]) + viewer.setStyle()\n",
        "view()"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:core.import_pose.import_pose: File 'outputs/omega0_0.pdb' automatically determined to be of type PDB\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  OXT on residue ALA:CtermProteinFull 20\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  OXT on residue ALA:CtermProteinFull 40\n",
            "INFO:rosetta:core.import_pose.import_pose: File 'outputs/omega0_2.pdb' automatically determined to be of type PDB\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  OXT on residue ALA:CtermProteinFull 20\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  OXT on residue ALA:CtermProteinFull 40\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "display_data",
          "data": {
            "text/html": [
              "<script>$('.output_scroll').removeClass('output_scroll')</script>"
            ],
            "text/plain": [
              "<IPython.core.display.HTML object>"
            ]
          },
          "metadata": {
            "tags": []
          }
        },
        {
          "output_type": "display_data",
          "data": {
            "text/html": [
              "<script>$('.output_scroll').removeClass('output_scroll')</script>"
            ],
            "text/plain": [
              "<IPython.core.display.HTML object>"
            ]
          },
          "metadata": {
            "tags": []
          }
        },
        {
          "output_type": "display_data",
          "data": {
            "application/vnd.jupyter.widget-view+json": {
              "model_id": "3c8a0274c508449da448f6b4d261772c",
              "version_minor": 0,
              "version_major": 2
            },
            "text/plain": [
              "interactive(children=(IntSlider(value=0, continuous_update=False, description='Decoys', max=1), Output()), _do"
            ]
          },
          "metadata": {
            "tags": []
          }
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "<function pyrosetta.distributed.viewer.core.Viewer.show.<locals>.view>"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 62
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "MflI7KkkmdlP"
      },
      "source": [
        "## MakeBundle Mover in PyRosetta\n",
        "\n",
        "Ok, we have an idea for how to use code to create some helical backbones.  Lets try this through code instead of RosettaScripts.\n",
        "\n",
        "Note that the BPC_xx are `ENUMS` as we have seen before and are imported when we import all of the helical bundle namespace.  \n",
        "Also, if you call `mb.set_use_degrees(True)` before creating helices, you won't need the `math.radians()` call."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "l-AMjR2rmdlP",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "c1b486f4-f2a9-4c46-fc42-a7db001306cb"
      },
      "source": [
        "from pyrosetta.rosetta.protocols.helical_bundle import *\n",
        "import math\n",
        "\n",
        "pose = pyrosetta.rosetta.core.pose.Pose()\n",
        "mb = MakeBundle()\n",
        "\n",
        "mb.add_helix()\n",
        "mb.add_helix()\n",
        "mb.helix(2).calculator_op().real_parameter(BPC_delta_omega0).set_value(3.14)\n",
        "\n",
        "for i in range(1, 3):\n",
        "    mb.helix(i).set_helix_length(20)\n",
        "    mb.helix(i).calculator_op().real_parameter(BPC_r0).set_value(5)\n",
        "    mb.helix(i).calculator_op().real_parameter(BPC_omega0).set_value(math.radians(12))\n",
        "\n",
        "mb.apply(pose)\n",
        "pose.dump_pdb(\"outputs/manual_make_bundle.pdb\")\n",
        "    \n",
        "    "
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n",
            "INFO:rosetta:basic.io.database: Database file opened: protocol_data/crick_parameters/alpha_helix.crick_params\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "True"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 63
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 817
        },
        "id": "41JcL9WAb5CC",
        "outputId": "417537d3-499d-47c9-cc50-4d1a7065ee7d"
      },
      "source": [
        "view = viewer.init(pose) + viewer.setStyle()\n",
        "view()"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_16234210748847926\"  style=\"position: relative; width: 1200px; height: 800px\">\n        <p id=\"3dmolwarning_16234210748847926\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n      resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n}\n\nvar viewer_16234210748847926 = null;\nvar warn = document.getElementById(\"3dmolwarning_16234210748847926\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_16234210748847926 = $3Dmol.createViewer($(\"#3dmolviewer_16234210748847926\"),{backgroundColor:\"white\"});\nviewer_16234210748847926.zoomTo();\n\tviewer_16234210748847926.addModels(\"HEADER                                            11-JUN-21   XXXX              \\nEXPDTA    THEORETICAL MODEL                                                     \\nREMARK 220                                                                      \\nREMARK 220 EXPERIMENTAL DETAILS                                                 \\nREMARK 220  EXPERIMENT TYPE                : THEORETICAL MODELLING              \\nREMARK 220  DATE OF DATA COLLECTION        : 11-JUN-21                          \\nREMARK 220                                                                      \\nREMARK 220 REMARK: MODEL GENERATED BY ROSETTA                                   \\nREMARK 220  VERSION 2020.50.post.dev+970.commits.3700df14560                    \\nREMARK 250 ----------------------------------------                             \\nREMARK 250 CRICK PARAMETER INFORMATION                                          \\nREMARK 250 Bundle symmetry: 0 (not symmetric)                                   \\nREMARK 250 Bundle symmetry copies: 0                                            \\nREMARK 250 Number of helices per copy: 2                                        \\nREMARK 250 ---HELIX 1 PARAMETERS:---                                            \\nREMARK 250  PARAMETERS THAT ARE TYPICALLY SAMPLED:                              \\nREMARK 250    Major radius (r0,Angstroms): 5.00000000                           \\nREMARK 250    Major twist (omega0,radians/residue): 0.20943951                  \\nREMARK 250    Rotation about z-axis (delta_omega0,radians): 0.00000000          \\nREMARK 250    Roll about helix axis (delta_omega1,radians): 0.00000000          \\nREMARK 250    Registry shift (delta_t,residues): 0.00000000                     \\nREMARK 250    Major axial offset (z0_offset,Angstroms): 0.00000000              \\nREMARK 250    Minor axial offset (z1_offset,vert. Angstroms): 0.000000          \\nREMARK 250    Lateral squash (epsilon,dimensionless): 1.00000000                \\nREMARK 250  PARAMETERS THAT ARE NOT TYPICALLY SAMPLED:                          \\nREMARK 250    Residues/repeat (residues_per_repeat,dimensionless): 1            \\nREMARK 250    Repeat unit offset (repeating_unit_offset,residues): 0            \\nREMARK 250    Atoms/residue (atoms_per_residue,dimensionless):                  \\nREMARK 250      [3]                                                             \\nREMARK 250    Minor radius (r1_peratom,Angstroms):                              \\nREMARK 250      [1.53673520, 2.29431130, 1.73316244]                            \\nREMARK 250    Minor twist (omega1,radians/residue): 1.72180480                  \\nREMARK 250    Minor rise (z1,Angstroms/residue): 1.53104401                     \\nREMARK 250    Atom angular offset (delta_omega1_peratom,radians):               \\nREMARK 250      [-0.46010617, 0.00000000, 0.48012932]                           \\nREMARK 250    Atom axial offset (delta_z1_peratom,Angstroms):                   \\nREMARK 250      [-0.90470326, 0.00000000, 1.05169636]                           \\nREMARK 250    Invert helix (invert,Boolean): FALSE                              \\nREMARK 250    Dihedrals set (set_dihedrals,Boolean): TRUE                       \\nREMARK 250    Nonideal bond angles (set_bondangles,Boolean): TRUE               \\nREMARK 250    Nonideal bond lengths (set_bondlengths,Boolean): TRUE             \\nREMARK 250 ---HELIX 2 PARAMETERS:---                                            \\nREMARK 250  PARAMETERS THAT ARE TYPICALLY SAMPLED:                              \\nREMARK 250    Major radius (r0,Angstroms): 5.00000000                           \\nREMARK 250    Major twist (omega0,radians/residue): 0.20943951                  \\nREMARK 250    Rotation about z-axis (delta_omega0,radians): 3.14000000          \\nREMARK 250    Roll about helix axis (delta_omega1,radians): 0.00000000          \\nREMARK 250    Registry shift (delta_t,residues): 0.00000000                     \\nREMARK 250    Major axial offset (z0_offset,Angstroms): 0.00000000              \\nREMARK 250    Minor axial offset (z1_offset,vert. Angstroms): 0.000000          \\nREMARK 250    Lateral squash (epsilon,dimensionless): 1.00000000                \\nREMARK 250  PARAMETERS THAT ARE NOT TYPICALLY SAMPLED:                          \\nREMARK 250    Residues/repeat (residues_per_repeat,dimensionless): 1            \\nREMARK 250    Repeat unit offset (repeating_unit_offset,residues): 0            \\nREMARK 250    Atoms/residue (atoms_per_residue,dimensionless):                  \\nREMARK 250      [3]                                                             \\nREMARK 250    Minor radius (r1_peratom,Angstroms):                              \\nREMARK 250      [1.53673520, 2.29431130, 1.73316244]                            \\nREMARK 250    Minor twist (omega1,radians/residue): 1.72180480                  \\nREMARK 250    Minor rise (z1,Angstroms/residue): 1.53104401                     \\nREMARK 250    Atom angular offset (delta_omega1_peratom,radians):               \\nREMARK 250      [-0.46010617, 0.00000000, 0.48012932]                           \\nREMARK 250    Atom axial offset (delta_z1_peratom,Angstroms):                   \\nREMARK 250      [-0.90470326, 0.00000000, 1.05169636]                           \\nREMARK 250    Invert helix (invert,Boolean): FALSE                              \\nREMARK 250    Dihedrals set (set_dihedrals,Boolean): TRUE                       \\nREMARK 250    Nonideal bond angles (set_bondangles,Boolean): TRUE               \\nREMARK 250    Nonideal bond lengths (set_bondlengths,Boolean): TRUE             \\nREMARK 250 ----------------------------------------                             \\nATOM      1  N   ALA A   1      -2.394  -2.631 -11.699  1.00  0.00           N  \\nATOM      2  CA  ALA A   1      -2.343  -2.211 -10.303  1.00  0.00           C  \\nATOM      3  C   ALA A   1      -2.414  -3.409  -9.365  1.00  0.00           C  \\nATOM      4  O   ALA A   1      -1.699  -3.470  -8.365  1.00  0.00           O  \\nATOM      5  CB  ALA A   1      -3.471  -1.235 -10.002  1.00  0.00           C  \\nATOM      6  H   ALA A   1      -2.900  -3.470 -11.942  1.00  0.00           H  \\nATOM      7  HA  ALA A   1      -1.389  -1.711 -10.135  1.00  0.00           H  \\nATOM      8 1HB  ALA A   1      -3.419  -0.931  -8.956  1.00  0.00           H  \\nATOM      9 2HB  ALA A   1      -3.372  -0.357 -10.640  1.00  0.00           H  \\nATOM     10 3HB  ALA A   1      -4.428  -1.717 -10.193  1.00  0.00           H  \\nATOM     11  N   ALA A   2      -3.229  -4.314  -9.663  1.00  0.00           N  \\nATOM     12  CA  ALA A   2      -3.278  -5.403  -8.695  1.00  0.00           C  \\nATOM     13  C   ALA A   2      -1.957  -6.161  -8.650  1.00  0.00           C  \\nATOM     14  O   ALA A   2      -1.487  -6.545  -7.580  1.00  0.00           O  \\nATOM     15  CB  ALA A   2      -4.421  -6.353  -9.022  1.00  0.00           C  \\nATOM     16  H   ALA A   2      -3.768  -4.231 -10.513  1.00  0.00           H  \\nATOM     17  HA  ALA A   2      -3.449  -4.972  -7.709  1.00  0.00           H  \\nATOM     18 1HB  ALA A   2      -4.445  -7.160  -8.291  1.00  0.00           H  \\nATOM     19 2HB  ALA A   2      -5.365  -5.809  -8.993  1.00  0.00           H  \\nATOM     20 3HB  ALA A   2      -4.274  -6.769 -10.017  1.00  0.00           H  \\nATOM     21  N   ALA A   3      -1.290  -6.437  -9.603  1.00  0.00           N  \\nATOM     22  CA  ALA A   3       0.013  -7.088  -9.642  1.00  0.00           C  \\nATOM     23  C   ALA A   3       1.108  -6.161  -9.131  1.00  0.00           C  \\nATOM     24  O   ALA A   3       1.988  -6.580  -8.378  1.00  0.00           O  \\nATOM     25  CB  ALA A   3       0.331  -7.552 -11.056  1.00  0.00           C  \\nATOM     26  H   ALA A   3      -1.633  -6.209 -10.526  1.00  0.00           H  \\nATOM     27  HA  ALA A   3      -0.022  -7.958  -8.985  1.00  0.00           H  \\nATOM     28 1HB  ALA A   3       1.308  -8.036 -11.070  1.00  0.00           H  \\nATOM     29 2HB  ALA A   3      -0.429  -8.260 -11.387  1.00  0.00           H  \\nATOM     30 3HB  ALA A   3       0.343  -6.694 -11.726  1.00  0.00           H  \\nATOM     31  N   ALA A   4       1.029  -4.998  -9.528  1.00  0.00           N  \\nATOM     32  CA  ALA A   4       1.955  -3.905  -9.258  1.00  0.00           C  \\nATOM     33  C   ALA A   4       1.890  -3.476  -7.798  1.00  0.00           C  \\nATOM     34  O   ALA A   4       2.918  -3.228  -7.167  1.00  0.00           O  \\nATOM     35  CB  ALA A   4       1.659  -2.723 -10.170  1.00  0.00           C  \\nATOM     36  H   ALA A   4       0.249  -4.729 -10.110  1.00  0.00           H  \\nATOM     37  HA  ALA A   4       2.966  -4.260  -9.457  1.00  0.00           H  \\nATOM     38 1HB  ALA A   4       2.358  -1.914  -9.957  1.00  0.00           H  \\nATOM     39 2HB  ALA A   4       1.766  -3.030 -11.210  1.00  0.00           H  \\nATOM     40 3HB  ALA A   4       0.641  -2.377  -9.998  1.00  0.00           H  \\nATOM     41  N   ALA A   5       0.601  -3.456  -7.476  1.00  0.00           N  \\nATOM     42  CA  ALA A   5       0.333  -2.884  -6.162  1.00  0.00           C  \\nATOM     43  C   ALA A   5       0.980  -3.710  -5.059  1.00  0.00           C  \\nATOM     44  O   ALA A   5       1.548  -3.164  -4.113  1.00  0.00           O  \\nATOM     45  CB  ALA A   5      -1.167  -2.773  -5.929  1.00  0.00           C  \\nATOM     46  H   ALA A   5      -0.075  -3.821  -8.132  1.00  0.00           H  \\nATOM     47  HA  ALA A   5       0.769  -1.885  -6.132  1.00  0.00           H  \\nATOM     48 1HB  ALA A   5      -1.352  -2.344  -4.944  1.00  0.00           H  \\nATOM     49 2HB  ALA A   5      -1.608  -2.131  -6.693  1.00  0.00           H  \\nATOM     50 3HB  ALA A   5      -1.618  -3.763  -5.984  1.00  0.00           H  \\nATOM     51  N   ALA A   6       0.878  -4.970  -5.208  1.00  0.00           N  \\nATOM     52  CA  ALA A   6       1.442  -5.704  -4.082  1.00  0.00           C  \\nATOM     53  C   ALA A   6       2.943  -5.471  -3.968  1.00  0.00           C  \\nATOM     54  O   ALA A   6       3.477  -5.325  -2.868  1.00  0.00           O  \\nATOM     55  CB  ALA A   6       1.147  -7.190  -4.217  1.00  0.00           C  \\nATOM     56  H   ALA A   6       0.442  -5.332  -6.044  1.00  0.00           H  \\nATOM     57  HA  ALA A   6       0.977  -5.333  -3.169  1.00  0.00           H  \\nATOM     58 1HB  ALA A   6       1.574  -7.725  -3.368  1.00  0.00           H  \\nATOM     59 2HB  ALA A   6       0.068  -7.347  -4.238  1.00  0.00           H  \\nATOM     60 3HB  ALA A   6       1.586  -7.566  -5.140  1.00  0.00           H  \\nATOM     61  N   ALA A   7       3.701  -5.411  -4.892  1.00  0.00           N  \\nATOM     62  CA  ALA A   7       5.127  -5.115  -4.831  1.00  0.00           C  \\nATOM     63  C   ALA A   7       5.372  -3.654  -4.477  1.00  0.00           C  \\nATOM     64  O   ALA A   7       6.251  -3.339  -3.675  1.00  0.00           O  \\nATOM     65  CB  ALA A   7       5.796  -5.457  -6.154  1.00  0.00           C  \\nATOM     66  H   ALA A   7       3.353  -5.577  -5.825  1.00  0.00           H  \\nATOM     67  HA  ALA A   7       5.567  -5.728  -4.044  1.00  0.00           H  \\nATOM     68 1HB  ALA A   7       6.861  -5.231  -6.093  1.00  0.00           H  \\nATOM     69 2HB  ALA A   7       5.662  -6.518  -6.366  1.00  0.00           H  \\nATOM     70 3HB  ALA A   7       5.345  -4.869  -6.952  1.00  0.00           H  \\nATOM     71  N   ALA A   8       4.663  -2.823  -5.019  1.00  0.00           N  \\nATOM     72  CA  ALA A   8       4.690  -1.370  -4.896  1.00  0.00           C  \\nATOM     73  C   ALA A   8       4.202  -0.925  -3.523  1.00  0.00           C  \\nATOM     74  O   ALA A   8       4.783  -0.032  -2.907  1.00  0.00           O  \\nATOM     75  CB  ALA A   8       3.848  -0.730  -5.990  1.00  0.00           C  \\nATOM     76  H   ALA A   8       3.940  -3.132  -5.653  1.00  0.00           H  \\nATOM     77  HA  ALA A   8       5.723  -1.040  -5.007  1.00  0.00           H  \\nATOM     78 1HB  ALA A   8       3.878   0.355  -5.884  1.00  0.00           H  \\nATOM     79 2HB  ALA A   8       4.245  -1.011  -6.966  1.00  0.00           H  \\nATOM     80 3HB  ALA A   8       2.819  -1.074  -5.904  1.00  0.00           H  \\nATOM     81  N   ALA A   9       3.148  -1.666  -3.248  1.00  0.00           N  \\nATOM     82  CA  ALA A   9       2.407  -1.285  -2.051  1.00  0.00           C  \\nATOM     83  C   ALA A   9       3.262  -1.443  -0.800  1.00  0.00           C  \\nATOM     84  O   ALA A   9       3.244  -0.590   0.087  1.00  0.00           O  \\nATOM     85  CB  ALA A   9       1.135  -2.112  -1.930  1.00  0.00           C  \\nATOM     86  H   ALA A   9       2.913  -2.435  -3.859  1.00  0.00           H  \\nATOM     87  HA  ALA A   9       2.136  -0.233  -2.140  1.00  0.00           H  \\nATOM     88 1HB  ALA A   9       0.593  -1.816  -1.031  1.00  0.00           H  \\nATOM     89 2HB  ALA A   9       0.507  -1.944  -2.805  1.00  0.00           H  \\nATOM     90 3HB  ALA A   9       1.393  -3.168  -1.866  1.00  0.00           H  \\nATOM     91  N   ALA A  10       3.960  -2.529  -0.809  1.00  0.00           N  \\nATOM     92  CA  ALA A  10       4.674  -2.702   0.450  1.00  0.00           C  \\nATOM     93  C   ALA A  10       5.684  -1.582   0.668  1.00  0.00           C  \\nATOM     94  O   ALA A  10       5.842  -1.084   1.783  1.00  0.00           O  \\nATOM     95  CB  ALA A  10       5.370  -4.055   0.483  1.00  0.00           C  \\nATOM     96  H   ALA A  10       3.946  -3.116  -1.631  1.00  0.00           H  \\nATOM     97  HA  ALA A  10       3.947  -2.658   1.260  1.00  0.00           H  \\nATOM     98 1HB  ALA A  10       5.899  -4.169   1.429  1.00  0.00           H  \\nATOM     99 2HB  ALA A  10       4.628  -4.848   0.385  1.00  0.00           H  \\nATOM    100 3HB  ALA A  10       6.080  -4.119  -0.339  1.00  0.00           H  \\nATOM    101  N   ALA A  11       6.377  -1.117  -0.193  1.00  0.00           N  \\nATOM    102  CA  ALA A  11       7.294  -0.000  -0.000  1.00  0.00           C  \\nATOM    103  C   ALA A  11       6.537   1.303   0.220  1.00  0.00           C  \\nATOM    104  O   ALA A  11       6.905   2.110   1.074  1.00  0.00           O  \\nATOM    105  CB  ALA A  11       8.232   0.128  -1.192  1.00  0.00           C  \\nATOM    106  H   ALA A  11       6.338  -1.510  -1.123  1.00  0.00           H  \\nATOM    107  HA  ALA A  11       7.885  -0.198   0.894  1.00  0.00           H  \\nATOM    108 1HB  ALA A  11       8.911   0.965  -1.032  1.00  0.00           H  \\nATOM    109 2HB  ALA A  11       8.809  -0.791  -1.301  1.00  0.00           H  \\nATOM    110 3HB  ALA A  11       7.650   0.299  -2.096  1.00  0.00           H  \\nATOM    111  N   ALA A  12       5.562   1.545  -0.456  1.00  0.00           N  \\nATOM    112  CA  ALA A  12       4.674   2.702  -0.450  1.00  0.00           C  \\nATOM    113  C   ALA A  12       3.807   2.724   0.803  1.00  0.00           C  \\nATOM    114  O   ALA A  12       3.616   3.773   1.418  1.00  0.00           O  \\nATOM    115  CB  ALA A  12       3.802   2.707  -1.697  1.00  0.00           C  \\nATOM    116  H   ALA A  12       5.268   0.875  -1.152  1.00  0.00           H  \\nATOM    117  HA  ALA A  12       5.290   3.601  -0.446  1.00  0.00           H  \\nATOM    118 1HB  ALA A  12       3.145   3.577  -1.677  1.00  0.00           H  \\nATOM    119 2HB  ALA A  12       4.435   2.750  -2.583  1.00  0.00           H  \\nATOM    120 3HB  ALA A  12       3.202   1.800  -1.724  1.00  0.00           H  \\nATOM    121  N   ALA A  13       3.394   1.503   1.000  1.00  0.00           N  \\nATOM    122  CA  ALA A  13       2.407   1.285   2.051  1.00  0.00           C  \\nATOM    123  C   ALA A  13       2.978   1.621   3.422  1.00  0.00           C  \\nATOM    124  O   ALA A  13       2.307   2.237   4.251  1.00  0.00           O  \\nATOM    125  CB  ALA A  13       1.912  -0.154   2.022  1.00  0.00           C  \\nATOM    126  H   ALA A  13       3.766   0.759   0.428  1.00  0.00           H  \\nATOM    127  HA  ALA A  13       1.564   1.952   1.867  1.00  0.00           H  \\nATOM    128 1HB  ALA A  13       1.175  -0.301   2.812  1.00  0.00           H  \\nATOM    129 2HB  ALA A  13       1.453  -0.362   1.056  1.00  0.00           H  \\nATOM    130 3HB  ALA A  13       2.751  -0.830   2.179  1.00  0.00           H  \\nATOM    131  N   ALA A  14       4.201   1.172   3.537  1.00  0.00           N  \\nATOM    132  CA  ALA A  14       4.690   1.370   4.896  1.00  0.00           C  \\nATOM    133  C   ALA A  14       4.752   2.851   5.249  1.00  0.00           C  \\nATOM    134  O   ALA A  14       4.410   3.248   6.363  1.00  0.00           O  \\nATOM    135  CB  ALA A  14       6.059   0.729   5.065  1.00  0.00           C  \\nATOM    136  H   ALA A  14       4.652   0.763   2.731  1.00  0.00           H  \\nATOM    137  HA  ALA A  14       3.989   0.893   5.581  1.00  0.00           H  \\nATOM    138 1HB  ALA A  14       6.411   0.885   6.085  1.00  0.00           H  \\nATOM    139 2HB  ALA A  14       5.987  -0.341   4.866  1.00  0.00           H  \\nATOM    140 3HB  ALA A  14       6.762   1.181   4.367  1.00  0.00           H  \\nATOM    141  N   ALA A  15       5.129   3.700   4.480  1.00  0.00           N  \\nATOM    142  CA  ALA A  15       5.127   5.115   4.831  1.00  0.00           C  \\nATOM    143  C   ALA A  15       3.707   5.653   4.946  1.00  0.00           C  \\nATOM    144  O   ALA A  15       3.396   6.425   5.853  1.00  0.00           O  \\nATOM    145  CB  ALA A  15       5.915   5.917   3.805  1.00  0.00           C  \\nATOM    146  H   ALA A  15       5.456   3.425   3.565  1.00  0.00           H  \\nATOM    147  HA  ALA A  15       5.605   5.223   5.805  1.00  0.00           H  \\nATOM    148 1HB  ALA A  15       5.904   6.971   4.081  1.00  0.00           H  \\nATOM    149 2HB  ALA A  15       6.944   5.559   3.776  1.00  0.00           H  \\nATOM    150 3HB  ALA A  15       5.461   5.795   2.823  1.00  0.00           H  \\nATOM    151  N   ALA A  16       2.852   5.339   4.157  1.00  0.00           N  \\nATOM    152  CA  ALA A  16       1.442   5.704   4.082  1.00  0.00           C  \\nATOM    153  C   ALA A  16       0.639   5.030   5.187  1.00  0.00           C  \\nATOM    154  O   ALA A  16      -0.215   5.655   5.815  1.00  0.00           O  \\nATOM    155  CB  ALA A  16       0.873   5.340   2.718  1.00  0.00           C  \\nATOM    156  H   ALA A  16       3.089   4.721   3.395  1.00  0.00           H  \\nATOM    157  HA  ALA A  16       1.363   6.782   4.221  1.00  0.00           H  \\nATOM    158 1HB  ALA A  16      -0.180   5.618   2.677  1.00  0.00           H  \\nATOM    159 2HB  ALA A  16       1.420   5.875   1.941  1.00  0.00           H  \\nATOM    160 3HB  ALA A  16       0.971   4.268   2.558  1.00  0.00           H  \\nATOM    161  N   ALA A  17       1.034   3.808   5.280  1.00  0.00           N  \\nATOM    162  CA  ALA A  17       0.333   2.884   6.162  1.00  0.00           C  \\nATOM    163  C   ALA A  17       0.485   3.293   7.622  1.00  0.00           C  \\nATOM    164  O   ALA A  17      -0.473   3.245   8.393  1.00  0.00           O  \\nATOM    165  CB  ALA A  17       0.841   1.464   5.955  1.00  0.00           C  \\nATOM    166  H   ALA A  17       1.829   3.505   4.735  1.00  0.00           H  \\nATOM    167  HA  ALA A  17      -0.729   2.918   5.915  1.00  0.00           H  \\nATOM    168 1HB  ALA A  17       0.308   0.786   6.621  1.00  0.00           H  \\nATOM    169 2HB  ALA A  17       0.672   1.165   4.921  1.00  0.00           H  \\nATOM    170 3HB  ALA A  17       1.907   1.424   6.174  1.00  0.00           H  \\nATOM    171  N   ALA A  18       1.735   3.665   7.837  1.00  0.00           N  \\nATOM    172  CA  ALA A  18       1.955   3.905   9.258  1.00  0.00           C  \\nATOM    173  C   ALA A  18       1.074   5.038   9.768  1.00  0.00           C  \\nATOM    174  O   ALA A  18       0.522   4.963  10.865  1.00  0.00           O  \\nATOM    175  CB  ALA A  18       3.421   4.213   9.523  1.00  0.00           C  \\nATOM    176  H   ALA A  18       2.367   3.741   7.053  1.00  0.00           H  \\nATOM    177  HA  ALA A  18       1.681   3.000   9.800  1.00  0.00           H  \\nATOM    178 1HB  ALA A  18       3.569   4.390  10.589  1.00  0.00           H  \\nATOM    179 2HB  ALA A  18       4.033   3.369   9.207  1.00  0.00           H  \\nATOM    180 3HB  ALA A  18       3.712   5.101   8.965  1.00  0.00           H  \\nATOM    181  N   ALA A  19       0.889   6.048   9.116  1.00  0.00           N  \\nATOM    182  CA  ALA A  19       0.013   7.088   9.642  1.00  0.00           C  \\nATOM    183  C   ALA A  19      -1.435   6.619   9.687  1.00  0.00           C  \\nATOM    184  O   ALA A  19      -2.156   6.892  10.647  1.00  0.00           O  \\nATOM    185  CB  ALA A  19       0.133   8.354   8.806  1.00  0.00           C  \\nATOM    186  H   ALA A  19       1.345   6.146   8.221  1.00  0.00           H  \\nATOM    187  HA  ALA A  19       0.325   7.309  10.663  1.00  0.00           H  \\nATOM    188 1HB  ALA A  19      -0.526   9.122   9.212  1.00  0.00           H  \\nATOM    189 2HB  ALA A  19       1.163   8.710   8.831  1.00  0.00           H  \\nATOM    190 3HB  ALA A  19      -0.151   8.139   7.778  1.00  0.00           H  \\nATOM    191  N   ALA A  20      -1.941   5.974   8.812  1.00  0.00           N  \\nATOM    192  CA  ALA A  20      -3.278   5.403   8.695  1.00  0.00           C  \\nATOM    193  C   ALA A  20      -3.455   4.218   9.635  1.00  0.00           C  \\nATOM    194  O   ALA A  20      -2.497   3.509   9.944  1.00  0.00           O  \\nATOM    195  CB  ALA A  20      -3.551   4.984   7.258  1.00  0.00           C  \\nATOM    196  H   ALA A  20      -1.401   5.754   7.987  1.00  0.00           H  \\nATOM    197  HA  ALA A  20      -3.999   6.168   8.982  1.00  0.00           H  \\nATOM    198 1HB  ALA A  20      -4.552   4.560   7.187  1.00  0.00           H  \\nATOM    199 2HB  ALA A  20      -3.478   5.853   6.605  1.00  0.00           H  \\nATOM    200 3HB  ALA A  20      -2.819   4.238   6.952  1.00  0.00           H  \\nTER                                                                             \\nATOM    202  N   ALA B  21       2.398   2.627 -11.699  1.00  0.00           N  \\nATOM    203  CA  ALA B  21       2.346   2.207 -10.303  1.00  0.00           C  \\nATOM    204  C   ALA B  21       2.420   3.405  -9.365  1.00  0.00           C  \\nATOM    205  O   ALA B  21       1.705   3.467  -8.365  1.00  0.00           O  \\nATOM    206  CB  ALA B  21       3.473   1.230 -10.002  1.00  0.00           C  \\nATOM    207  H   ALA B  21       2.905   3.466 -11.942  1.00  0.00           H  \\nATOM    208  HA  ALA B  21       1.392   1.709 -10.135  1.00  0.00           H  \\nATOM    209 1HB  ALA B  21       3.421   0.926  -8.956  1.00  0.00           H  \\nATOM    210 2HB  ALA B  21       3.373   0.352 -10.640  1.00  0.00           H  \\nATOM    211 3HB  ALA B  21       4.431   1.709 -10.193  1.00  0.00           H  \\nATOM    212  N   ALA B  22       3.236   4.309  -9.663  1.00  0.00           N  \\nATOM    213  CA  ALA B  22       3.286   5.398  -8.695  1.00  0.00           C  \\nATOM    214  C   ALA B  22       1.967   6.158  -8.650  1.00  0.00           C  \\nATOM    215  O   ALA B  22       1.497   6.543  -7.580  1.00  0.00           O  \\nATOM    216  CB  ALA B  22       4.431   6.346  -9.022  1.00  0.00           C  \\nATOM    217  H   ALA B  22       3.775   4.225 -10.513  1.00  0.00           H  \\nATOM    218  HA  ALA B  22       3.457   4.966  -7.709  1.00  0.00           H  \\nATOM    219 1HB  ALA B  22       4.456   7.153  -8.291  1.00  0.00           H  \\nATOM    220 2HB  ALA B  22       5.375   5.800  -8.993  1.00  0.00           H  \\nATOM    221 3HB  ALA B  22       4.284   6.763 -10.017  1.00  0.00           H  \\nATOM    222  N   ALA B  23       1.301   6.435  -9.603  1.00  0.00           N  \\nATOM    223  CA  ALA B  23      -0.002   7.088  -9.642  1.00  0.00           C  \\nATOM    224  C   ALA B  23      -1.098   6.163  -9.131  1.00  0.00           C  \\nATOM    225  O   ALA B  23      -1.977   6.583  -8.378  1.00  0.00           O  \\nATOM    226  CB  ALA B  23      -0.319   7.553 -11.056  1.00  0.00           C  \\nATOM    227  H   ALA B  23       1.643   6.207 -10.526  1.00  0.00           H  \\nATOM    228  HA  ALA B  23       0.035   7.958  -8.985  1.00  0.00           H  \\nATOM    229 1HB  ALA B  23      -1.295   8.039 -11.070  1.00  0.00           H  \\nATOM    230 2HB  ALA B  23       0.442   8.259 -11.387  1.00  0.00           H  \\nATOM    231 3HB  ALA B  23      -0.333   6.694 -11.726  1.00  0.00           H  \\nATOM    232  N   ALA B  24      -1.021   5.000  -9.528  1.00  0.00           N  \\nATOM    233  CA  ALA B  24      -1.948   3.908  -9.258  1.00  0.00           C  \\nATOM    234  C   ALA B  24      -1.884   3.479  -7.798  1.00  0.00           C  \\nATOM    235  O   ALA B  24      -2.912   3.233  -7.167  1.00  0.00           O  \\nATOM    236  CB  ALA B  24      -1.654   2.726 -10.170  1.00  0.00           C  \\nATOM    237  H   ALA B  24      -0.241   4.730 -10.110  1.00  0.00           H  \\nATOM    238  HA  ALA B  24      -2.959   4.264  -9.457  1.00  0.00           H  \\nATOM    239 1HB  ALA B  24      -2.355   1.918  -9.957  1.00  0.00           H  \\nATOM    240 2HB  ALA B  24      -1.761   3.033 -11.210  1.00  0.00           H  \\nATOM    241 3HB  ALA B  24      -0.637   2.378  -9.998  1.00  0.00           H  \\nATOM    242  N   ALA B  25      -0.596   3.457  -7.476  1.00  0.00           N  \\nATOM    243  CA  ALA B  25      -0.328   2.884  -6.162  1.00  0.00           C  \\nATOM    244  C   ALA B  25      -0.974   3.712  -5.059  1.00  0.00           C  \\nATOM    245  O   ALA B  25      -1.543   3.166  -4.113  1.00  0.00           O  \\nATOM    246  CB  ALA B  25       1.172   2.771  -5.929  1.00  0.00           C  \\nATOM    247  H   ALA B  25       0.081   3.821  -8.132  1.00  0.00           H  \\nATOM    248  HA  ALA B  25      -0.766   1.887  -6.132  1.00  0.00           H  \\nATOM    249 1HB  ALA B  25       1.355   2.342  -4.944  1.00  0.00           H  \\nATOM    250 2HB  ALA B  25       1.611   2.129  -6.693  1.00  0.00           H  \\nATOM    251 3HB  ALA B  25       1.624   3.760  -5.984  1.00  0.00           H  \\nATOM    252  N   ALA B  26      -0.871   4.971  -5.208  1.00  0.00           N  \\nATOM    253  CA  ALA B  26      -1.433   5.706  -4.082  1.00  0.00           C  \\nATOM    254  C   ALA B  26      -2.935   5.475  -3.968  1.00  0.00           C  \\nATOM    255  O   ALA B  26      -3.469   5.331  -2.868  1.00  0.00           O  \\nATOM    256  CB  ALA B  26      -1.135   7.192  -4.217  1.00  0.00           C  \\nATOM    257  H   ALA B  26      -0.434   5.333  -6.044  1.00  0.00           H  \\nATOM    258  HA  ALA B  26      -0.968   5.334  -3.169  1.00  0.00           H  \\nATOM    259 1HB  ALA B  26      -1.562   7.727  -3.368  1.00  0.00           H  \\nATOM    260 2HB  ALA B  26      -0.057   7.347  -4.238  1.00  0.00           H  \\nATOM    261 3HB  ALA B  26      -1.574   7.568  -5.140  1.00  0.00           H  \\nATOM    262  N   ALA B  27      -3.692   5.417  -4.892  1.00  0.00           N  \\nATOM    263  CA  ALA B  27      -5.119   5.123  -4.831  1.00  0.00           C  \\nATOM    264  C   ALA B  27      -5.366   3.663  -4.477  1.00  0.00           C  \\nATOM    265  O   ALA B  27      -6.246   3.349  -3.675  1.00  0.00           O  \\nATOM    266  CB  ALA B  27      -5.787   5.467  -6.154  1.00  0.00           C  \\nATOM    267  H   ALA B  27      -3.344   5.582  -5.825  1.00  0.00           H  \\nATOM    268  HA  ALA B  27      -5.558   5.737  -4.044  1.00  0.00           H  \\nATOM    269 1HB  ALA B  27      -6.852   5.242  -6.093  1.00  0.00           H  \\nATOM    270 2HB  ALA B  27      -5.652   6.527  -6.366  1.00  0.00           H  \\nATOM    271 3HB  ALA B  27      -5.338   4.878  -6.952  1.00  0.00           H  \\nATOM    272  N   ALA B  28      -4.659   2.830  -5.019  1.00  0.00           N  \\nATOM    273  CA  ALA B  28      -4.688   1.378  -4.896  1.00  0.00           C  \\nATOM    274  C   ALA B  28      -4.201   0.932  -3.523  1.00  0.00           C  \\nATOM    275  O   ALA B  28      -4.783   0.039  -2.907  1.00  0.00           O  \\nATOM    276  CB  ALA B  28      -3.847   0.736  -5.990  1.00  0.00           C  \\nATOM    277  H   ALA B  28      -3.935   3.138  -5.653  1.00  0.00           H  \\nATOM    278  HA  ALA B  28      -5.721   1.049  -5.007  1.00  0.00           H  \\nATOM    279 1HB  ALA B  28      -3.879  -0.348  -5.884  1.00  0.00           H  \\nATOM    280 2HB  ALA B  28      -4.243   1.017  -6.966  1.00  0.00           H  \\nATOM    281 3HB  ALA B  28      -2.817   1.078  -5.904  1.00  0.00           H  \\nATOM    282  N   ALA B  29      -3.145   1.671  -3.248  1.00  0.00           N  \\nATOM    283  CA  ALA B  29      -2.405   1.289  -2.051  1.00  0.00           C  \\nATOM    284  C   ALA B  29      -3.259   1.448  -0.800  1.00  0.00           C  \\nATOM    285  O   ALA B  29      -3.243   0.595   0.087  1.00  0.00           O  \\nATOM    286  CB  ALA B  29      -1.132   2.114  -1.930  1.00  0.00           C  \\nATOM    287  H   ALA B  29      -2.909   2.439  -3.859  1.00  0.00           H  \\nATOM    288  HA  ALA B  29      -2.136   0.236  -2.140  1.00  0.00           H  \\nATOM    289 1HB  ALA B  29      -0.590   1.817  -1.031  1.00  0.00           H  \\nATOM    290 2HB  ALA B  29      -0.504   1.944  -2.805  1.00  0.00           H  \\nATOM    291 3HB  ALA B  29      -1.388   3.170  -1.866  1.00  0.00           H  \\nATOM    292  N   ALA B  30      -3.956   2.536  -0.809  1.00  0.00           N  \\nATOM    293  CA  ALA B  30      -4.670   2.709   0.450  1.00  0.00           C  \\nATOM    294  C   ALA B  30      -5.682   1.591   0.668  1.00  0.00           C  \\nATOM    295  O   ALA B  30      -5.840   1.094   1.783  1.00  0.00           O  \\nATOM    296  CB  ALA B  30      -5.363   4.064   0.483  1.00  0.00           C  \\nATOM    297  H   ALA B  30      -3.941   3.122  -1.631  1.00  0.00           H  \\nATOM    298  HA  ALA B  30      -3.943   2.665   1.260  1.00  0.00           H  \\nATOM    299 1HB  ALA B  30      -5.892   4.178   1.429  1.00  0.00           H  \\nATOM    300 2HB  ALA B  30      -4.620   4.855   0.385  1.00  0.00           H  \\nATOM    301 3HB  ALA B  30      -6.073   4.129  -0.339  1.00  0.00           H  \\nATOM    302  N   ALA B  31      -6.375   1.127  -0.193  1.00  0.00           N  \\nATOM    303  CA  ALA B  31      -7.294   0.012  -0.000  1.00  0.00           C  \\nATOM    304  C   ALA B  31      -6.539  -1.293   0.220  1.00  0.00           C  \\nATOM    305  O   ALA B  31      -6.909  -2.099   1.074  1.00  0.00           O  \\nATOM    306  CB  ALA B  31      -8.232  -0.114  -1.192  1.00  0.00           C  \\nATOM    307  H   ALA B  31      -6.335   1.520  -1.123  1.00  0.00           H  \\nATOM    308  HA  ALA B  31      -7.885   0.211   0.894  1.00  0.00           H  \\nATOM    309 1HB  ALA B  31      -8.912  -0.951  -1.032  1.00  0.00           H  \\nATOM    310 2HB  ALA B  31      -8.808   0.805  -1.301  1.00  0.00           H  \\nATOM    311 3HB  ALA B  31      -7.650  -0.287  -2.096  1.00  0.00           H  \\nATOM    312  N   ALA B  32      -5.564  -1.536  -0.456  1.00  0.00           N  \\nATOM    313  CA  ALA B  32      -4.679  -2.694  -0.450  1.00  0.00           C  \\nATOM    314  C   ALA B  32      -3.812  -2.718   0.803  1.00  0.00           C  \\nATOM    315  O   ALA B  32      -3.622  -3.767   1.418  1.00  0.00           O  \\nATOM    316  CB  ALA B  32      -3.807  -2.701  -1.697  1.00  0.00           C  \\nATOM    317  H   ALA B  32      -5.269  -0.867  -1.152  1.00  0.00           H  \\nATOM    318  HA  ALA B  32      -5.296  -3.593  -0.446  1.00  0.00           H  \\nATOM    319 1HB  ALA B  32      -3.151  -3.572  -1.677  1.00  0.00           H  \\nATOM    320 2HB  ALA B  32      -4.440  -2.743  -2.583  1.00  0.00           H  \\nATOM    321 3HB  ALA B  32      -3.205  -1.795  -1.724  1.00  0.00           H  \\nATOM    322  N   ALA B  33      -3.396  -1.498   1.000  1.00  0.00           N  \\nATOM    323  CA  ALA B  33      -2.409  -1.281   2.051  1.00  0.00           C  \\nATOM    324  C   ALA B  33      -2.980  -1.617   3.422  1.00  0.00           C  \\nATOM    325  O   ALA B  33      -2.311  -2.233   4.251  1.00  0.00           O  \\nATOM    326  CB  ALA B  33      -1.912   0.157   2.022  1.00  0.00           C  \\nATOM    327  H   ALA B  33      -3.767  -0.753   0.428  1.00  0.00           H  \\nATOM    328  HA  ALA B  33      -1.567  -1.949   1.867  1.00  0.00           H  \\nATOM    329 1HB  ALA B  33      -1.175   0.303   2.812  1.00  0.00           H  \\nATOM    330 2HB  ALA B  33      -1.453   0.364   1.056  1.00  0.00           H  \\nATOM    331 3HB  ALA B  33      -2.750   0.834   2.179  1.00  0.00           H  \\nATOM    332  N   ALA B  34      -4.203  -1.166   3.537  1.00  0.00           N  \\nATOM    333  CA  ALA B  34      -4.692  -1.363   4.896  1.00  0.00           C  \\nATOM    334  C   ALA B  34      -4.757  -2.843   5.249  1.00  0.00           C  \\nATOM    335  O   ALA B  34      -4.416  -3.241   6.363  1.00  0.00           O  \\nATOM    336  CB  ALA B  34      -6.061  -0.719   5.065  1.00  0.00           C  \\nATOM    337  H   ALA B  34      -4.654  -0.755   2.731  1.00  0.00           H  \\nATOM    338  HA  ALA B  34      -3.991  -0.887   5.581  1.00  0.00           H  \\nATOM    339 1HB  ALA B  34      -6.412  -0.875   6.085  1.00  0.00           H  \\nATOM    340 2HB  ALA B  34      -5.987   0.350   4.866  1.00  0.00           H  \\nATOM    341 3HB  ALA B  34      -6.763  -1.170   4.367  1.00  0.00           H  \\nATOM    342  N   ALA B  35      -5.135  -3.692   4.480  1.00  0.00           N  \\nATOM    343  CA  ALA B  35      -5.135  -5.107   4.831  1.00  0.00           C  \\nATOM    344  C   ALA B  35      -3.716  -5.647   4.946  1.00  0.00           C  \\nATOM    345  O   ALA B  35      -3.406  -6.419   5.853  1.00  0.00           O  \\nATOM    346  CB  ALA B  35      -5.924  -5.907   3.805  1.00  0.00           C  \\nATOM    347  H   ALA B  35      -5.461  -3.416   3.565  1.00  0.00           H  \\nATOM    348  HA  ALA B  35      -5.613  -5.214   5.805  1.00  0.00           H  \\nATOM    349 1HB  ALA B  35      -5.915  -6.962   4.081  1.00  0.00           H  \\nATOM    350 2HB  ALA B  35      -6.953  -5.548   3.776  1.00  0.00           H  \\nATOM    351 3HB  ALA B  35      -5.470  -5.786   2.823  1.00  0.00           H  \\nATOM    352  N   ALA B  36      -2.861  -5.335   4.157  1.00  0.00           N  \\nATOM    353  CA  ALA B  36      -1.451  -5.701   4.082  1.00  0.00           C  \\nATOM    354  C   ALA B  36      -0.647  -5.029   5.187  1.00  0.00           C  \\nATOM    355  O   ALA B  36       0.206  -5.655   5.815  1.00  0.00           O  \\nATOM    356  CB  ALA B  36      -0.882  -5.339   2.718  1.00  0.00           C  \\nATOM    357  H   ALA B  36      -3.097  -4.716   3.395  1.00  0.00           H  \\nATOM    358  HA  ALA B  36      -1.374  -6.780   4.221  1.00  0.00           H  \\nATOM    359 1HB  ALA B  36       0.171  -5.619   2.677  1.00  0.00           H  \\nATOM    360 2HB  ALA B  36      -1.429  -5.872   1.941  1.00  0.00           H  \\nATOM    361 3HB  ALA B  36      -0.978  -4.266   2.558  1.00  0.00           H  \\nATOM    362  N   ALA B  37      -1.040  -3.807   5.280  1.00  0.00           N  \\nATOM    363  CA  ALA B  37      -0.337  -2.883   6.162  1.00  0.00           C  \\nATOM    364  C   ALA B  37      -0.490  -3.292   7.622  1.00  0.00           C  \\nATOM    365  O   ALA B  37       0.468  -3.245   8.393  1.00  0.00           O  \\nATOM    366  CB  ALA B  37      -0.843  -1.463   5.955  1.00  0.00           C  \\nATOM    367  H   ALA B  37      -1.834  -3.502   4.735  1.00  0.00           H  \\nATOM    368  HA  ALA B  37       0.724  -2.919   5.915  1.00  0.00           H  \\nATOM    369 1HB  ALA B  37      -0.309  -0.785   6.621  1.00  0.00           H  \\nATOM    370 2HB  ALA B  37      -0.674  -1.164   4.921  1.00  0.00           H  \\nATOM    371 3HB  ALA B  37      -1.909  -1.421   6.174  1.00  0.00           H  \\nATOM    372  N   ALA B  38      -1.741  -3.662   7.837  1.00  0.00           N  \\nATOM    373  CA  ALA B  38      -1.961  -3.902   9.258  1.00  0.00           C  \\nATOM    374  C   ALA B  38      -1.082  -5.037   9.768  1.00  0.00           C  \\nATOM    375  O   ALA B  38      -0.529  -4.962  10.865  1.00  0.00           O  \\nATOM    376  CB  ALA B  38      -3.428  -4.208   9.523  1.00  0.00           C  \\nATOM    377  H   ALA B  38      -2.373  -3.737   7.053  1.00  0.00           H  \\nATOM    378  HA  ALA B  38      -1.685  -2.997   9.800  1.00  0.00           H  \\nATOM    379 1HB  ALA B  38      -3.576  -4.384  10.589  1.00  0.00           H  \\nATOM    380 2HB  ALA B  38      -4.039  -3.362   9.207  1.00  0.00           H  \\nATOM    381 3HB  ALA B  38      -3.720  -5.096   8.965  1.00  0.00           H  \\nATOM    382  N   ALA B  39      -0.898  -6.046   9.116  1.00  0.00           N  \\nATOM    383  CA  ALA B  39      -0.025  -7.088   9.642  1.00  0.00           C  \\nATOM    384  C   ALA B  39       1.425  -6.622   9.687  1.00  0.00           C  \\nATOM    385  O   ALA B  39       2.145  -6.895  10.647  1.00  0.00           O  \\nATOM    386  CB  ALA B  39      -0.147  -8.354   8.806  1.00  0.00           C  \\nATOM    387  H   ALA B  39      -1.355  -6.144   8.221  1.00  0.00           H  \\nATOM    388  HA  ALA B  39      -0.337  -7.308  10.663  1.00  0.00           H  \\nATOM    389 1HB  ALA B  39       0.511  -9.123   9.212  1.00  0.00           H  \\nATOM    390 2HB  ALA B  39      -1.177  -8.709   8.831  1.00  0.00           H  \\nATOM    391 3HB  ALA B  39       0.138  -8.139   7.778  1.00  0.00           H  \\nATOM    392  N   ALA B  40       1.932  -5.977   8.812  1.00  0.00           N  \\nATOM    393  CA  ALA B  40       3.269  -5.408   8.695  1.00  0.00           C  \\nATOM    394  C   ALA B  40       3.448  -4.224   9.635  1.00  0.00           C  \\nATOM    395  O   ALA B  40       2.491  -3.513   9.944  1.00  0.00           O  \\nATOM    396  CB  ALA B  40       3.543  -4.989   7.258  1.00  0.00           C  \\nATOM    397  H   ALA B  40       1.392  -5.757   7.987  1.00  0.00           H  \\nATOM    398  HA  ALA B  40       3.989  -6.175   8.982  1.00  0.00           H  \\nATOM    399 1HB  ALA B  40       4.545  -4.567   7.187  1.00  0.00           H  \\nATOM    400 2HB  ALA B  40       3.468  -5.859   6.605  1.00  0.00           H  \\nATOM    401 3HB  ALA B  40       2.812  -4.243   6.952  1.00  0.00           H  \\nTER                                                                             \\nREMARK PDBinfo-LABEL:    1 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    2 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    3 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    4 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    5 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    6 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    7 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    8 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    9 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   10 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   11 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   12 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   13 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   14 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   15 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   16 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   17 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   18 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   19 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   20 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   21 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   22 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   23 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   24 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   25 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   26 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   27 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   28 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   29 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   30 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   31 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   32 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   33 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   34 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   35 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   36 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   37 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   38 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   39 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   40 PEPTIDE_STUB_EXTENSION\\n\\n\",\"pdb\");\n\tviewer_16234210748847926.zoomTo();\n\tviewer_16234210748847926.setStyle({\"cartoon\": {\"color\": \"spectrum\"}, \"stick\": {\"colorscheme\": \"blackCarbon\", \"radius\": \"0.1\"}});\nviewer_16234210748847926.render();\n});\n</script>",
            "text/html": [
              "<div id=\"3dmolviewer_16234210748847926\"  style=\"position: relative; width: 1200px; height: 800px\">\n",
              "        <p id=\"3dmolwarning_16234210748847926\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n",
              "        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n",
              "        </div>\n",
              "<script>\n",
              "\n",
              "var loadScriptAsync = function(uri){\n",
              "  return new Promise((resolve, reject) => {\n",
              "    var tag = document.createElement('script');\n",
              "    tag.src = uri;\n",
              "    tag.async = true;\n",
              "    tag.onload = () => {\n",
              "      resolve();\n",
              "    };\n",
              "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
              "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
              "});\n",
              "};\n",
              "\n",
              "if(typeof $3Dmolpromise === 'undefined') {\n",
              "$3Dmolpromise = null;\n",
              "  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n",
              "}\n",
              "\n",
              "var viewer_16234210748847926 = null;\n",
              "var warn = document.getElementById(\"3dmolwarning_16234210748847926\");\n",
              "if(warn) {\n",
              "    warn.parentNode.removeChild(warn);\n",
              "}\n",
              "$3Dmolpromise.then(function() {\n",
              "viewer_16234210748847926 = $3Dmol.createViewer($(\"#3dmolviewer_16234210748847926\"),{backgroundColor:\"white\"});\n",
              "viewer_16234210748847926.zoomTo();\n",
              "\tviewer_16234210748847926.addModels(\"HEADER                                            11-JUN-21   XXXX              \\nEXPDTA    THEORETICAL MODEL                                                     \\nREMARK 220                                                                      \\nREMARK 220 EXPERIMENTAL DETAILS                                                 \\nREMARK 220  EXPERIMENT TYPE                : THEORETICAL MODELLING              \\nREMARK 220  DATE OF DATA COLLECTION        : 11-JUN-21                          \\nREMARK 220                                                                      \\nREMARK 220 REMARK: MODEL GENERATED BY ROSETTA                                   \\nREMARK 220  VERSION 2020.50.post.dev+970.commits.3700df14560                    \\nREMARK 250 ----------------------------------------                             \\nREMARK 250 CRICK PARAMETER INFORMATION                                          \\nREMARK 250 Bundle symmetry: 0 (not symmetric)                                   \\nREMARK 250 Bundle symmetry copies: 0                                            \\nREMARK 250 Number of helices per copy: 2                                        \\nREMARK 250 ---HELIX 1 PARAMETERS:---                                            \\nREMARK 250  PARAMETERS THAT ARE TYPICALLY SAMPLED:                              \\nREMARK 250    Major radius (r0,Angstroms): 5.00000000                           \\nREMARK 250    Major twist (omega0,radians/residue): 0.20943951                  \\nREMARK 250    Rotation about z-axis (delta_omega0,radians): 0.00000000          \\nREMARK 250    Roll about helix axis (delta_omega1,radians): 0.00000000          \\nREMARK 250    Registry shift (delta_t,residues): 0.00000000                     \\nREMARK 250    Major axial offset (z0_offset,Angstroms): 0.00000000              \\nREMARK 250    Minor axial offset (z1_offset,vert. Angstroms): 0.000000          \\nREMARK 250    Lateral squash (epsilon,dimensionless): 1.00000000                \\nREMARK 250  PARAMETERS THAT ARE NOT TYPICALLY SAMPLED:                          \\nREMARK 250    Residues/repeat (residues_per_repeat,dimensionless): 1            \\nREMARK 250    Repeat unit offset (repeating_unit_offset,residues): 0            \\nREMARK 250    Atoms/residue (atoms_per_residue,dimensionless):                  \\nREMARK 250      [3]                                                             \\nREMARK 250    Minor radius (r1_peratom,Angstroms):                              \\nREMARK 250      [1.53673520, 2.29431130, 1.73316244]                            \\nREMARK 250    Minor twist (omega1,radians/residue): 1.72180480                  \\nREMARK 250    Minor rise (z1,Angstroms/residue): 1.53104401                     \\nREMARK 250    Atom angular offset (delta_omega1_peratom,radians):               \\nREMARK 250      [-0.46010617, 0.00000000, 0.48012932]                           \\nREMARK 250    Atom axial offset (delta_z1_peratom,Angstroms):                   \\nREMARK 250      [-0.90470326, 0.00000000, 1.05169636]                           \\nREMARK 250    Invert helix (invert,Boolean): FALSE                              \\nREMARK 250    Dihedrals set (set_dihedrals,Boolean): TRUE                       \\nREMARK 250    Nonideal bond angles (set_bondangles,Boolean): TRUE               \\nREMARK 250    Nonideal bond lengths (set_bondlengths,Boolean): TRUE             \\nREMARK 250 ---HELIX 2 PARAMETERS:---                                            \\nREMARK 250  PARAMETERS THAT ARE TYPICALLY SAMPLED:                              \\nREMARK 250    Major radius (r0,Angstroms): 5.00000000                           \\nREMARK 250    Major twist (omega0,radians/residue): 0.20943951                  \\nREMARK 250    Rotation about z-axis (delta_omega0,radians): 3.14000000          \\nREMARK 250    Roll about helix axis (delta_omega1,radians): 0.00000000          \\nREMARK 250    Registry shift (delta_t,residues): 0.00000000                     \\nREMARK 250    Major axial offset (z0_offset,Angstroms): 0.00000000              \\nREMARK 250    Minor axial offset (z1_offset,vert. Angstroms): 0.000000          \\nREMARK 250    Lateral squash (epsilon,dimensionless): 1.00000000                \\nREMARK 250  PARAMETERS THAT ARE NOT TYPICALLY SAMPLED:                          \\nREMARK 250    Residues/repeat (residues_per_repeat,dimensionless): 1            \\nREMARK 250    Repeat unit offset (repeating_unit_offset,residues): 0            \\nREMARK 250    Atoms/residue (atoms_per_residue,dimensionless):                  \\nREMARK 250      [3]                                                             \\nREMARK 250    Minor radius (r1_peratom,Angstroms):                              \\nREMARK 250      [1.53673520, 2.29431130, 1.73316244]                            \\nREMARK 250    Minor twist (omega1,radians/residue): 1.72180480                  \\nREMARK 250    Minor rise (z1,Angstroms/residue): 1.53104401                     \\nREMARK 250    Atom angular offset (delta_omega1_peratom,radians):               \\nREMARK 250      [-0.46010617, 0.00000000, 0.48012932]                           \\nREMARK 250    Atom axial offset (delta_z1_peratom,Angstroms):                   \\nREMARK 250      [-0.90470326, 0.00000000, 1.05169636]                           \\nREMARK 250    Invert helix (invert,Boolean): FALSE                              \\nREMARK 250    Dihedrals set (set_dihedrals,Boolean): TRUE                       \\nREMARK 250    Nonideal bond angles (set_bondangles,Boolean): TRUE               \\nREMARK 250    Nonideal bond lengths (set_bondlengths,Boolean): TRUE             \\nREMARK 250 ----------------------------------------                             \\nATOM      1  N   ALA A   1      -2.394  -2.631 -11.699  1.00  0.00           N  \\nATOM      2  CA  ALA A   1      -2.343  -2.211 -10.303  1.00  0.00           C  \\nATOM      3  C   ALA A   1      -2.414  -3.409  -9.365  1.00  0.00           C  \\nATOM      4  O   ALA A   1      -1.699  -3.470  -8.365  1.00  0.00           O  \\nATOM      5  CB  ALA A   1      -3.471  -1.235 -10.002  1.00  0.00           C  \\nATOM      6  H   ALA A   1      -2.900  -3.470 -11.942  1.00  0.00           H  \\nATOM      7  HA  ALA A   1      -1.389  -1.711 -10.135  1.00  0.00           H  \\nATOM      8 1HB  ALA A   1      -3.419  -0.931  -8.956  1.00  0.00           H  \\nATOM      9 2HB  ALA A   1      -3.372  -0.357 -10.640  1.00  0.00           H  \\nATOM     10 3HB  ALA A   1      -4.428  -1.717 -10.193  1.00  0.00           H  \\nATOM     11  N   ALA A   2      -3.229  -4.314  -9.663  1.00  0.00           N  \\nATOM     12  CA  ALA A   2      -3.278  -5.403  -8.695  1.00  0.00           C  \\nATOM     13  C   ALA A   2      -1.957  -6.161  -8.650  1.00  0.00           C  \\nATOM     14  O   ALA A   2      -1.487  -6.545  -7.580  1.00  0.00           O  \\nATOM     15  CB  ALA A   2      -4.421  -6.353  -9.022  1.00  0.00           C  \\nATOM     16  H   ALA A   2      -3.768  -4.231 -10.513  1.00  0.00           H  \\nATOM     17  HA  ALA A   2      -3.449  -4.972  -7.709  1.00  0.00           H  \\nATOM     18 1HB  ALA A   2      -4.445  -7.160  -8.291  1.00  0.00           H  \\nATOM     19 2HB  ALA A   2      -5.365  -5.809  -8.993  1.00  0.00           H  \\nATOM     20 3HB  ALA A   2      -4.274  -6.769 -10.017  1.00  0.00           H  \\nATOM     21  N   ALA A   3      -1.290  -6.437  -9.603  1.00  0.00           N  \\nATOM     22  CA  ALA A   3       0.013  -7.088  -9.642  1.00  0.00           C  \\nATOM     23  C   ALA A   3       1.108  -6.161  -9.131  1.00  0.00           C  \\nATOM     24  O   ALA A   3       1.988  -6.580  -8.378  1.00  0.00           O  \\nATOM     25  CB  ALA A   3       0.331  -7.552 -11.056  1.00  0.00           C  \\nATOM     26  H   ALA A   3      -1.633  -6.209 -10.526  1.00  0.00           H  \\nATOM     27  HA  ALA A   3      -0.022  -7.958  -8.985  1.00  0.00           H  \\nATOM     28 1HB  ALA A   3       1.308  -8.036 -11.070  1.00  0.00           H  \\nATOM     29 2HB  ALA A   3      -0.429  -8.260 -11.387  1.00  0.00           H  \\nATOM     30 3HB  ALA A   3       0.343  -6.694 -11.726  1.00  0.00           H  \\nATOM     31  N   ALA A   4       1.029  -4.998  -9.528  1.00  0.00           N  \\nATOM     32  CA  ALA A   4       1.955  -3.905  -9.258  1.00  0.00           C  \\nATOM     33  C   ALA A   4       1.890  -3.476  -7.798  1.00  0.00           C  \\nATOM     34  O   ALA A   4       2.918  -3.228  -7.167  1.00  0.00           O  \\nATOM     35  CB  ALA A   4       1.659  -2.723 -10.170  1.00  0.00           C  \\nATOM     36  H   ALA A   4       0.249  -4.729 -10.110  1.00  0.00           H  \\nATOM     37  HA  ALA A   4       2.966  -4.260  -9.457  1.00  0.00           H  \\nATOM     38 1HB  ALA A   4       2.358  -1.914  -9.957  1.00  0.00           H  \\nATOM     39 2HB  ALA A   4       1.766  -3.030 -11.210  1.00  0.00           H  \\nATOM     40 3HB  ALA A   4       0.641  -2.377  -9.998  1.00  0.00           H  \\nATOM     41  N   ALA A   5       0.601  -3.456  -7.476  1.00  0.00           N  \\nATOM     42  CA  ALA A   5       0.333  -2.884  -6.162  1.00  0.00           C  \\nATOM     43  C   ALA A   5       0.980  -3.710  -5.059  1.00  0.00           C  \\nATOM     44  O   ALA A   5       1.548  -3.164  -4.113  1.00  0.00           O  \\nATOM     45  CB  ALA A   5      -1.167  -2.773  -5.929  1.00  0.00           C  \\nATOM     46  H   ALA A   5      -0.075  -3.821  -8.132  1.00  0.00           H  \\nATOM     47  HA  ALA A   5       0.769  -1.885  -6.132  1.00  0.00           H  \\nATOM     48 1HB  ALA A   5      -1.352  -2.344  -4.944  1.00  0.00           H  \\nATOM     49 2HB  ALA A   5      -1.608  -2.131  -6.693  1.00  0.00           H  \\nATOM     50 3HB  ALA A   5      -1.618  -3.763  -5.984  1.00  0.00           H  \\nATOM     51  N   ALA A   6       0.878  -4.970  -5.208  1.00  0.00           N  \\nATOM     52  CA  ALA A   6       1.442  -5.704  -4.082  1.00  0.00           C  \\nATOM     53  C   ALA A   6       2.943  -5.471  -3.968  1.00  0.00           C  \\nATOM     54  O   ALA A   6       3.477  -5.325  -2.868  1.00  0.00           O  \\nATOM     55  CB  ALA A   6       1.147  -7.190  -4.217  1.00  0.00           C  \\nATOM     56  H   ALA A   6       0.442  -5.332  -6.044  1.00  0.00           H  \\nATOM     57  HA  ALA A   6       0.977  -5.333  -3.169  1.00  0.00           H  \\nATOM     58 1HB  ALA A   6       1.574  -7.725  -3.368  1.00  0.00           H  \\nATOM     59 2HB  ALA A   6       0.068  -7.347  -4.238  1.00  0.00           H  \\nATOM     60 3HB  ALA A   6       1.586  -7.566  -5.140  1.00  0.00           H  \\nATOM     61  N   ALA A   7       3.701  -5.411  -4.892  1.00  0.00           N  \\nATOM     62  CA  ALA A   7       5.127  -5.115  -4.831  1.00  0.00           C  \\nATOM     63  C   ALA A   7       5.372  -3.654  -4.477  1.00  0.00           C  \\nATOM     64  O   ALA A   7       6.251  -3.339  -3.675  1.00  0.00           O  \\nATOM     65  CB  ALA A   7       5.796  -5.457  -6.154  1.00  0.00           C  \\nATOM     66  H   ALA A   7       3.353  -5.577  -5.825  1.00  0.00           H  \\nATOM     67  HA  ALA A   7       5.567  -5.728  -4.044  1.00  0.00           H  \\nATOM     68 1HB  ALA A   7       6.861  -5.231  -6.093  1.00  0.00           H  \\nATOM     69 2HB  ALA A   7       5.662  -6.518  -6.366  1.00  0.00           H  \\nATOM     70 3HB  ALA A   7       5.345  -4.869  -6.952  1.00  0.00           H  \\nATOM     71  N   ALA A   8       4.663  -2.823  -5.019  1.00  0.00           N  \\nATOM     72  CA  ALA A   8       4.690  -1.370  -4.896  1.00  0.00           C  \\nATOM     73  C   ALA A   8       4.202  -0.925  -3.523  1.00  0.00           C  \\nATOM     74  O   ALA A   8       4.783  -0.032  -2.907  1.00  0.00           O  \\nATOM     75  CB  ALA A   8       3.848  -0.730  -5.990  1.00  0.00           C  \\nATOM     76  H   ALA A   8       3.940  -3.132  -5.653  1.00  0.00           H  \\nATOM     77  HA  ALA A   8       5.723  -1.040  -5.007  1.00  0.00           H  \\nATOM     78 1HB  ALA A   8       3.878   0.355  -5.884  1.00  0.00           H  \\nATOM     79 2HB  ALA A   8       4.245  -1.011  -6.966  1.00  0.00           H  \\nATOM     80 3HB  ALA A   8       2.819  -1.074  -5.904  1.00  0.00           H  \\nATOM     81  N   ALA A   9       3.148  -1.666  -3.248  1.00  0.00           N  \\nATOM     82  CA  ALA A   9       2.407  -1.285  -2.051  1.00  0.00           C  \\nATOM     83  C   ALA A   9       3.262  -1.443  -0.800  1.00  0.00           C  \\nATOM     84  O   ALA A   9       3.244  -0.590   0.087  1.00  0.00           O  \\nATOM     85  CB  ALA A   9       1.135  -2.112  -1.930  1.00  0.00           C  \\nATOM     86  H   ALA A   9       2.913  -2.435  -3.859  1.00  0.00           H  \\nATOM     87  HA  ALA A   9       2.136  -0.233  -2.140  1.00  0.00           H  \\nATOM     88 1HB  ALA A   9       0.593  -1.816  -1.031  1.00  0.00           H  \\nATOM     89 2HB  ALA A   9       0.507  -1.944  -2.805  1.00  0.00           H  \\nATOM     90 3HB  ALA A   9       1.393  -3.168  -1.866  1.00  0.00           H  \\nATOM     91  N   ALA A  10       3.960  -2.529  -0.809  1.00  0.00           N  \\nATOM     92  CA  ALA A  10       4.674  -2.702   0.450  1.00  0.00           C  \\nATOM     93  C   ALA A  10       5.684  -1.582   0.668  1.00  0.00           C  \\nATOM     94  O   ALA A  10       5.842  -1.084   1.783  1.00  0.00           O  \\nATOM     95  CB  ALA A  10       5.370  -4.055   0.483  1.00  0.00           C  \\nATOM     96  H   ALA A  10       3.946  -3.116  -1.631  1.00  0.00           H  \\nATOM     97  HA  ALA A  10       3.947  -2.658   1.260  1.00  0.00           H  \\nATOM     98 1HB  ALA A  10       5.899  -4.169   1.429  1.00  0.00           H  \\nATOM     99 2HB  ALA A  10       4.628  -4.848   0.385  1.00  0.00           H  \\nATOM    100 3HB  ALA A  10       6.080  -4.119  -0.339  1.00  0.00           H  \\nATOM    101  N   ALA A  11       6.377  -1.117  -0.193  1.00  0.00           N  \\nATOM    102  CA  ALA A  11       7.294  -0.000  -0.000  1.00  0.00           C  \\nATOM    103  C   ALA A  11       6.537   1.303   0.220  1.00  0.00           C  \\nATOM    104  O   ALA A  11       6.905   2.110   1.074  1.00  0.00           O  \\nATOM    105  CB  ALA A  11       8.232   0.128  -1.192  1.00  0.00           C  \\nATOM    106  H   ALA A  11       6.338  -1.510  -1.123  1.00  0.00           H  \\nATOM    107  HA  ALA A  11       7.885  -0.198   0.894  1.00  0.00           H  \\nATOM    108 1HB  ALA A  11       8.911   0.965  -1.032  1.00  0.00           H  \\nATOM    109 2HB  ALA A  11       8.809  -0.791  -1.301  1.00  0.00           H  \\nATOM    110 3HB  ALA A  11       7.650   0.299  -2.096  1.00  0.00           H  \\nATOM    111  N   ALA A  12       5.562   1.545  -0.456  1.00  0.00           N  \\nATOM    112  CA  ALA A  12       4.674   2.702  -0.450  1.00  0.00           C  \\nATOM    113  C   ALA A  12       3.807   2.724   0.803  1.00  0.00           C  \\nATOM    114  O   ALA A  12       3.616   3.773   1.418  1.00  0.00           O  \\nATOM    115  CB  ALA A  12       3.802   2.707  -1.697  1.00  0.00           C  \\nATOM    116  H   ALA A  12       5.268   0.875  -1.152  1.00  0.00           H  \\nATOM    117  HA  ALA A  12       5.290   3.601  -0.446  1.00  0.00           H  \\nATOM    118 1HB  ALA A  12       3.145   3.577  -1.677  1.00  0.00           H  \\nATOM    119 2HB  ALA A  12       4.435   2.750  -2.583  1.00  0.00           H  \\nATOM    120 3HB  ALA A  12       3.202   1.800  -1.724  1.00  0.00           H  \\nATOM    121  N   ALA A  13       3.394   1.503   1.000  1.00  0.00           N  \\nATOM    122  CA  ALA A  13       2.407   1.285   2.051  1.00  0.00           C  \\nATOM    123  C   ALA A  13       2.978   1.621   3.422  1.00  0.00           C  \\nATOM    124  O   ALA A  13       2.307   2.237   4.251  1.00  0.00           O  \\nATOM    125  CB  ALA A  13       1.912  -0.154   2.022  1.00  0.00           C  \\nATOM    126  H   ALA A  13       3.766   0.759   0.428  1.00  0.00           H  \\nATOM    127  HA  ALA A  13       1.564   1.952   1.867  1.00  0.00           H  \\nATOM    128 1HB  ALA A  13       1.175  -0.301   2.812  1.00  0.00           H  \\nATOM    129 2HB  ALA A  13       1.453  -0.362   1.056  1.00  0.00           H  \\nATOM    130 3HB  ALA A  13       2.751  -0.830   2.179  1.00  0.00           H  \\nATOM    131  N   ALA A  14       4.201   1.172   3.537  1.00  0.00           N  \\nATOM    132  CA  ALA A  14       4.690   1.370   4.896  1.00  0.00           C  \\nATOM    133  C   ALA A  14       4.752   2.851   5.249  1.00  0.00           C  \\nATOM    134  O   ALA A  14       4.410   3.248   6.363  1.00  0.00           O  \\nATOM    135  CB  ALA A  14       6.059   0.729   5.065  1.00  0.00           C  \\nATOM    136  H   ALA A  14       4.652   0.763   2.731  1.00  0.00           H  \\nATOM    137  HA  ALA A  14       3.989   0.893   5.581  1.00  0.00           H  \\nATOM    138 1HB  ALA A  14       6.411   0.885   6.085  1.00  0.00           H  \\nATOM    139 2HB  ALA A  14       5.987  -0.341   4.866  1.00  0.00           H  \\nATOM    140 3HB  ALA A  14       6.762   1.181   4.367  1.00  0.00           H  \\nATOM    141  N   ALA A  15       5.129   3.700   4.480  1.00  0.00           N  \\nATOM    142  CA  ALA A  15       5.127   5.115   4.831  1.00  0.00           C  \\nATOM    143  C   ALA A  15       3.707   5.653   4.946  1.00  0.00           C  \\nATOM    144  O   ALA A  15       3.396   6.425   5.853  1.00  0.00           O  \\nATOM    145  CB  ALA A  15       5.915   5.917   3.805  1.00  0.00           C  \\nATOM    146  H   ALA A  15       5.456   3.425   3.565  1.00  0.00           H  \\nATOM    147  HA  ALA A  15       5.605   5.223   5.805  1.00  0.00           H  \\nATOM    148 1HB  ALA A  15       5.904   6.971   4.081  1.00  0.00           H  \\nATOM    149 2HB  ALA A  15       6.944   5.559   3.776  1.00  0.00           H  \\nATOM    150 3HB  ALA A  15       5.461   5.795   2.823  1.00  0.00           H  \\nATOM    151  N   ALA A  16       2.852   5.339   4.157  1.00  0.00           N  \\nATOM    152  CA  ALA A  16       1.442   5.704   4.082  1.00  0.00           C  \\nATOM    153  C   ALA A  16       0.639   5.030   5.187  1.00  0.00           C  \\nATOM    154  O   ALA A  16      -0.215   5.655   5.815  1.00  0.00           O  \\nATOM    155  CB  ALA A  16       0.873   5.340   2.718  1.00  0.00           C  \\nATOM    156  H   ALA A  16       3.089   4.721   3.395  1.00  0.00           H  \\nATOM    157  HA  ALA A  16       1.363   6.782   4.221  1.00  0.00           H  \\nATOM    158 1HB  ALA A  16      -0.180   5.618   2.677  1.00  0.00           H  \\nATOM    159 2HB  ALA A  16       1.420   5.875   1.941  1.00  0.00           H  \\nATOM    160 3HB  ALA A  16       0.971   4.268   2.558  1.00  0.00           H  \\nATOM    161  N   ALA A  17       1.034   3.808   5.280  1.00  0.00           N  \\nATOM    162  CA  ALA A  17       0.333   2.884   6.162  1.00  0.00           C  \\nATOM    163  C   ALA A  17       0.485   3.293   7.622  1.00  0.00           C  \\nATOM    164  O   ALA A  17      -0.473   3.245   8.393  1.00  0.00           O  \\nATOM    165  CB  ALA A  17       0.841   1.464   5.955  1.00  0.00           C  \\nATOM    166  H   ALA A  17       1.829   3.505   4.735  1.00  0.00           H  \\nATOM    167  HA  ALA A  17      -0.729   2.918   5.915  1.00  0.00           H  \\nATOM    168 1HB  ALA A  17       0.308   0.786   6.621  1.00  0.00           H  \\nATOM    169 2HB  ALA A  17       0.672   1.165   4.921  1.00  0.00           H  \\nATOM    170 3HB  ALA A  17       1.907   1.424   6.174  1.00  0.00           H  \\nATOM    171  N   ALA A  18       1.735   3.665   7.837  1.00  0.00           N  \\nATOM    172  CA  ALA A  18       1.955   3.905   9.258  1.00  0.00           C  \\nATOM    173  C   ALA A  18       1.074   5.038   9.768  1.00  0.00           C  \\nATOM    174  O   ALA A  18       0.522   4.963  10.865  1.00  0.00           O  \\nATOM    175  CB  ALA A  18       3.421   4.213   9.523  1.00  0.00           C  \\nATOM    176  H   ALA A  18       2.367   3.741   7.053  1.00  0.00           H  \\nATOM    177  HA  ALA A  18       1.681   3.000   9.800  1.00  0.00           H  \\nATOM    178 1HB  ALA A  18       3.569   4.390  10.589  1.00  0.00           H  \\nATOM    179 2HB  ALA A  18       4.033   3.369   9.207  1.00  0.00           H  \\nATOM    180 3HB  ALA A  18       3.712   5.101   8.965  1.00  0.00           H  \\nATOM    181  N   ALA A  19       0.889   6.048   9.116  1.00  0.00           N  \\nATOM    182  CA  ALA A  19       0.013   7.088   9.642  1.00  0.00           C  \\nATOM    183  C   ALA A  19      -1.435   6.619   9.687  1.00  0.00           C  \\nATOM    184  O   ALA A  19      -2.156   6.892  10.647  1.00  0.00           O  \\nATOM    185  CB  ALA A  19       0.133   8.354   8.806  1.00  0.00           C  \\nATOM    186  H   ALA A  19       1.345   6.146   8.221  1.00  0.00           H  \\nATOM    187  HA  ALA A  19       0.325   7.309  10.663  1.00  0.00           H  \\nATOM    188 1HB  ALA A  19      -0.526   9.122   9.212  1.00  0.00           H  \\nATOM    189 2HB  ALA A  19       1.163   8.710   8.831  1.00  0.00           H  \\nATOM    190 3HB  ALA A  19      -0.151   8.139   7.778  1.00  0.00           H  \\nATOM    191  N   ALA A  20      -1.941   5.974   8.812  1.00  0.00           N  \\nATOM    192  CA  ALA A  20      -3.278   5.403   8.695  1.00  0.00           C  \\nATOM    193  C   ALA A  20      -3.455   4.218   9.635  1.00  0.00           C  \\nATOM    194  O   ALA A  20      -2.497   3.509   9.944  1.00  0.00           O  \\nATOM    195  CB  ALA A  20      -3.551   4.984   7.258  1.00  0.00           C  \\nATOM    196  H   ALA A  20      -1.401   5.754   7.987  1.00  0.00           H  \\nATOM    197  HA  ALA A  20      -3.999   6.168   8.982  1.00  0.00           H  \\nATOM    198 1HB  ALA A  20      -4.552   4.560   7.187  1.00  0.00           H  \\nATOM    199 2HB  ALA A  20      -3.478   5.853   6.605  1.00  0.00           H  \\nATOM    200 3HB  ALA A  20      -2.819   4.238   6.952  1.00  0.00           H  \\nTER                                                                             \\nATOM    202  N   ALA B  21       2.398   2.627 -11.699  1.00  0.00           N  \\nATOM    203  CA  ALA B  21       2.346   2.207 -10.303  1.00  0.00           C  \\nATOM    204  C   ALA B  21       2.420   3.405  -9.365  1.00  0.00           C  \\nATOM    205  O   ALA B  21       1.705   3.467  -8.365  1.00  0.00           O  \\nATOM    206  CB  ALA B  21       3.473   1.230 -10.002  1.00  0.00           C  \\nATOM    207  H   ALA B  21       2.905   3.466 -11.942  1.00  0.00           H  \\nATOM    208  HA  ALA B  21       1.392   1.709 -10.135  1.00  0.00           H  \\nATOM    209 1HB  ALA B  21       3.421   0.926  -8.956  1.00  0.00           H  \\nATOM    210 2HB  ALA B  21       3.373   0.352 -10.640  1.00  0.00           H  \\nATOM    211 3HB  ALA B  21       4.431   1.709 -10.193  1.00  0.00           H  \\nATOM    212  N   ALA B  22       3.236   4.309  -9.663  1.00  0.00           N  \\nATOM    213  CA  ALA B  22       3.286   5.398  -8.695  1.00  0.00           C  \\nATOM    214  C   ALA B  22       1.967   6.158  -8.650  1.00  0.00           C  \\nATOM    215  O   ALA B  22       1.497   6.543  -7.580  1.00  0.00           O  \\nATOM    216  CB  ALA B  22       4.431   6.346  -9.022  1.00  0.00           C  \\nATOM    217  H   ALA B  22       3.775   4.225 -10.513  1.00  0.00           H  \\nATOM    218  HA  ALA B  22       3.457   4.966  -7.709  1.00  0.00           H  \\nATOM    219 1HB  ALA B  22       4.456   7.153  -8.291  1.00  0.00           H  \\nATOM    220 2HB  ALA B  22       5.375   5.800  -8.993  1.00  0.00           H  \\nATOM    221 3HB  ALA B  22       4.284   6.763 -10.017  1.00  0.00           H  \\nATOM    222  N   ALA B  23       1.301   6.435  -9.603  1.00  0.00           N  \\nATOM    223  CA  ALA B  23      -0.002   7.088  -9.642  1.00  0.00           C  \\nATOM    224  C   ALA B  23      -1.098   6.163  -9.131  1.00  0.00           C  \\nATOM    225  O   ALA B  23      -1.977   6.583  -8.378  1.00  0.00           O  \\nATOM    226  CB  ALA B  23      -0.319   7.553 -11.056  1.00  0.00           C  \\nATOM    227  H   ALA B  23       1.643   6.207 -10.526  1.00  0.00           H  \\nATOM    228  HA  ALA B  23       0.035   7.958  -8.985  1.00  0.00           H  \\nATOM    229 1HB  ALA B  23      -1.295   8.039 -11.070  1.00  0.00           H  \\nATOM    230 2HB  ALA B  23       0.442   8.259 -11.387  1.00  0.00           H  \\nATOM    231 3HB  ALA B  23      -0.333   6.694 -11.726  1.00  0.00           H  \\nATOM    232  N   ALA B  24      -1.021   5.000  -9.528  1.00  0.00           N  \\nATOM    233  CA  ALA B  24      -1.948   3.908  -9.258  1.00  0.00           C  \\nATOM    234  C   ALA B  24      -1.884   3.479  -7.798  1.00  0.00           C  \\nATOM    235  O   ALA B  24      -2.912   3.233  -7.167  1.00  0.00           O  \\nATOM    236  CB  ALA B  24      -1.654   2.726 -10.170  1.00  0.00           C  \\nATOM    237  H   ALA B  24      -0.241   4.730 -10.110  1.00  0.00           H  \\nATOM    238  HA  ALA B  24      -2.959   4.264  -9.457  1.00  0.00           H  \\nATOM    239 1HB  ALA B  24      -2.355   1.918  -9.957  1.00  0.00           H  \\nATOM    240 2HB  ALA B  24      -1.761   3.033 -11.210  1.00  0.00           H  \\nATOM    241 3HB  ALA B  24      -0.637   2.378  -9.998  1.00  0.00           H  \\nATOM    242  N   ALA B  25      -0.596   3.457  -7.476  1.00  0.00           N  \\nATOM    243  CA  ALA B  25      -0.328   2.884  -6.162  1.00  0.00           C  \\nATOM    244  C   ALA B  25      -0.974   3.712  -5.059  1.00  0.00           C  \\nATOM    245  O   ALA B  25      -1.543   3.166  -4.113  1.00  0.00           O  \\nATOM    246  CB  ALA B  25       1.172   2.771  -5.929  1.00  0.00           C  \\nATOM    247  H   ALA B  25       0.081   3.821  -8.132  1.00  0.00           H  \\nATOM    248  HA  ALA B  25      -0.766   1.887  -6.132  1.00  0.00           H  \\nATOM    249 1HB  ALA B  25       1.355   2.342  -4.944  1.00  0.00           H  \\nATOM    250 2HB  ALA B  25       1.611   2.129  -6.693  1.00  0.00           H  \\nATOM    251 3HB  ALA B  25       1.624   3.760  -5.984  1.00  0.00           H  \\nATOM    252  N   ALA B  26      -0.871   4.971  -5.208  1.00  0.00           N  \\nATOM    253  CA  ALA B  26      -1.433   5.706  -4.082  1.00  0.00           C  \\nATOM    254  C   ALA B  26      -2.935   5.475  -3.968  1.00  0.00           C  \\nATOM    255  O   ALA B  26      -3.469   5.331  -2.868  1.00  0.00           O  \\nATOM    256  CB  ALA B  26      -1.135   7.192  -4.217  1.00  0.00           C  \\nATOM    257  H   ALA B  26      -0.434   5.333  -6.044  1.00  0.00           H  \\nATOM    258  HA  ALA B  26      -0.968   5.334  -3.169  1.00  0.00           H  \\nATOM    259 1HB  ALA B  26      -1.562   7.727  -3.368  1.00  0.00           H  \\nATOM    260 2HB  ALA B  26      -0.057   7.347  -4.238  1.00  0.00           H  \\nATOM    261 3HB  ALA B  26      -1.574   7.568  -5.140  1.00  0.00           H  \\nATOM    262  N   ALA B  27      -3.692   5.417  -4.892  1.00  0.00           N  \\nATOM    263  CA  ALA B  27      -5.119   5.123  -4.831  1.00  0.00           C  \\nATOM    264  C   ALA B  27      -5.366   3.663  -4.477  1.00  0.00           C  \\nATOM    265  O   ALA B  27      -6.246   3.349  -3.675  1.00  0.00           O  \\nATOM    266  CB  ALA B  27      -5.787   5.467  -6.154  1.00  0.00           C  \\nATOM    267  H   ALA B  27      -3.344   5.582  -5.825  1.00  0.00           H  \\nATOM    268  HA  ALA B  27      -5.558   5.737  -4.044  1.00  0.00           H  \\nATOM    269 1HB  ALA B  27      -6.852   5.242  -6.093  1.00  0.00           H  \\nATOM    270 2HB  ALA B  27      -5.652   6.527  -6.366  1.00  0.00           H  \\nATOM    271 3HB  ALA B  27      -5.338   4.878  -6.952  1.00  0.00           H  \\nATOM    272  N   ALA B  28      -4.659   2.830  -5.019  1.00  0.00           N  \\nATOM    273  CA  ALA B  28      -4.688   1.378  -4.896  1.00  0.00           C  \\nATOM    274  C   ALA B  28      -4.201   0.932  -3.523  1.00  0.00           C  \\nATOM    275  O   ALA B  28      -4.783   0.039  -2.907  1.00  0.00           O  \\nATOM    276  CB  ALA B  28      -3.847   0.736  -5.990  1.00  0.00           C  \\nATOM    277  H   ALA B  28      -3.935   3.138  -5.653  1.00  0.00           H  \\nATOM    278  HA  ALA B  28      -5.721   1.049  -5.007  1.00  0.00           H  \\nATOM    279 1HB  ALA B  28      -3.879  -0.348  -5.884  1.00  0.00           H  \\nATOM    280 2HB  ALA B  28      -4.243   1.017  -6.966  1.00  0.00           H  \\nATOM    281 3HB  ALA B  28      -2.817   1.078  -5.904  1.00  0.00           H  \\nATOM    282  N   ALA B  29      -3.145   1.671  -3.248  1.00  0.00           N  \\nATOM    283  CA  ALA B  29      -2.405   1.289  -2.051  1.00  0.00           C  \\nATOM    284  C   ALA B  29      -3.259   1.448  -0.800  1.00  0.00           C  \\nATOM    285  O   ALA B  29      -3.243   0.595   0.087  1.00  0.00           O  \\nATOM    286  CB  ALA B  29      -1.132   2.114  -1.930  1.00  0.00           C  \\nATOM    287  H   ALA B  29      -2.909   2.439  -3.859  1.00  0.00           H  \\nATOM    288  HA  ALA B  29      -2.136   0.236  -2.140  1.00  0.00           H  \\nATOM    289 1HB  ALA B  29      -0.590   1.817  -1.031  1.00  0.00           H  \\nATOM    290 2HB  ALA B  29      -0.504   1.944  -2.805  1.00  0.00           H  \\nATOM    291 3HB  ALA B  29      -1.388   3.170  -1.866  1.00  0.00           H  \\nATOM    292  N   ALA B  30      -3.956   2.536  -0.809  1.00  0.00           N  \\nATOM    293  CA  ALA B  30      -4.670   2.709   0.450  1.00  0.00           C  \\nATOM    294  C   ALA B  30      -5.682   1.591   0.668  1.00  0.00           C  \\nATOM    295  O   ALA B  30      -5.840   1.094   1.783  1.00  0.00           O  \\nATOM    296  CB  ALA B  30      -5.363   4.064   0.483  1.00  0.00           C  \\nATOM    297  H   ALA B  30      -3.941   3.122  -1.631  1.00  0.00           H  \\nATOM    298  HA  ALA B  30      -3.943   2.665   1.260  1.00  0.00           H  \\nATOM    299 1HB  ALA B  30      -5.892   4.178   1.429  1.00  0.00           H  \\nATOM    300 2HB  ALA B  30      -4.620   4.855   0.385  1.00  0.00           H  \\nATOM    301 3HB  ALA B  30      -6.073   4.129  -0.339  1.00  0.00           H  \\nATOM    302  N   ALA B  31      -6.375   1.127  -0.193  1.00  0.00           N  \\nATOM    303  CA  ALA B  31      -7.294   0.012  -0.000  1.00  0.00           C  \\nATOM    304  C   ALA B  31      -6.539  -1.293   0.220  1.00  0.00           C  \\nATOM    305  O   ALA B  31      -6.909  -2.099   1.074  1.00  0.00           O  \\nATOM    306  CB  ALA B  31      -8.232  -0.114  -1.192  1.00  0.00           C  \\nATOM    307  H   ALA B  31      -6.335   1.520  -1.123  1.00  0.00           H  \\nATOM    308  HA  ALA B  31      -7.885   0.211   0.894  1.00  0.00           H  \\nATOM    309 1HB  ALA B  31      -8.912  -0.951  -1.032  1.00  0.00           H  \\nATOM    310 2HB  ALA B  31      -8.808   0.805  -1.301  1.00  0.00           H  \\nATOM    311 3HB  ALA B  31      -7.650  -0.287  -2.096  1.00  0.00           H  \\nATOM    312  N   ALA B  32      -5.564  -1.536  -0.456  1.00  0.00           N  \\nATOM    313  CA  ALA B  32      -4.679  -2.694  -0.450  1.00  0.00           C  \\nATOM    314  C   ALA B  32      -3.812  -2.718   0.803  1.00  0.00           C  \\nATOM    315  O   ALA B  32      -3.622  -3.767   1.418  1.00  0.00           O  \\nATOM    316  CB  ALA B  32      -3.807  -2.701  -1.697  1.00  0.00           C  \\nATOM    317  H   ALA B  32      -5.269  -0.867  -1.152  1.00  0.00           H  \\nATOM    318  HA  ALA B  32      -5.296  -3.593  -0.446  1.00  0.00           H  \\nATOM    319 1HB  ALA B  32      -3.151  -3.572  -1.677  1.00  0.00           H  \\nATOM    320 2HB  ALA B  32      -4.440  -2.743  -2.583  1.00  0.00           H  \\nATOM    321 3HB  ALA B  32      -3.205  -1.795  -1.724  1.00  0.00           H  \\nATOM    322  N   ALA B  33      -3.396  -1.498   1.000  1.00  0.00           N  \\nATOM    323  CA  ALA B  33      -2.409  -1.281   2.051  1.00  0.00           C  \\nATOM    324  C   ALA B  33      -2.980  -1.617   3.422  1.00  0.00           C  \\nATOM    325  O   ALA B  33      -2.311  -2.233   4.251  1.00  0.00           O  \\nATOM    326  CB  ALA B  33      -1.912   0.157   2.022  1.00  0.00           C  \\nATOM    327  H   ALA B  33      -3.767  -0.753   0.428  1.00  0.00           H  \\nATOM    328  HA  ALA B  33      -1.567  -1.949   1.867  1.00  0.00           H  \\nATOM    329 1HB  ALA B  33      -1.175   0.303   2.812  1.00  0.00           H  \\nATOM    330 2HB  ALA B  33      -1.453   0.364   1.056  1.00  0.00           H  \\nATOM    331 3HB  ALA B  33      -2.750   0.834   2.179  1.00  0.00           H  \\nATOM    332  N   ALA B  34      -4.203  -1.166   3.537  1.00  0.00           N  \\nATOM    333  CA  ALA B  34      -4.692  -1.363   4.896  1.00  0.00           C  \\nATOM    334  C   ALA B  34      -4.757  -2.843   5.249  1.00  0.00           C  \\nATOM    335  O   ALA B  34      -4.416  -3.241   6.363  1.00  0.00           O  \\nATOM    336  CB  ALA B  34      -6.061  -0.719   5.065  1.00  0.00           C  \\nATOM    337  H   ALA B  34      -4.654  -0.755   2.731  1.00  0.00           H  \\nATOM    338  HA  ALA B  34      -3.991  -0.887   5.581  1.00  0.00           H  \\nATOM    339 1HB  ALA B  34      -6.412  -0.875   6.085  1.00  0.00           H  \\nATOM    340 2HB  ALA B  34      -5.987   0.350   4.866  1.00  0.00           H  \\nATOM    341 3HB  ALA B  34      -6.763  -1.170   4.367  1.00  0.00           H  \\nATOM    342  N   ALA B  35      -5.135  -3.692   4.480  1.00  0.00           N  \\nATOM    343  CA  ALA B  35      -5.135  -5.107   4.831  1.00  0.00           C  \\nATOM    344  C   ALA B  35      -3.716  -5.647   4.946  1.00  0.00           C  \\nATOM    345  O   ALA B  35      -3.406  -6.419   5.853  1.00  0.00           O  \\nATOM    346  CB  ALA B  35      -5.924  -5.907   3.805  1.00  0.00           C  \\nATOM    347  H   ALA B  35      -5.461  -3.416   3.565  1.00  0.00           H  \\nATOM    348  HA  ALA B  35      -5.613  -5.214   5.805  1.00  0.00           H  \\nATOM    349 1HB  ALA B  35      -5.915  -6.962   4.081  1.00  0.00           H  \\nATOM    350 2HB  ALA B  35      -6.953  -5.548   3.776  1.00  0.00           H  \\nATOM    351 3HB  ALA B  35      -5.470  -5.786   2.823  1.00  0.00           H  \\nATOM    352  N   ALA B  36      -2.861  -5.335   4.157  1.00  0.00           N  \\nATOM    353  CA  ALA B  36      -1.451  -5.701   4.082  1.00  0.00           C  \\nATOM    354  C   ALA B  36      -0.647  -5.029   5.187  1.00  0.00           C  \\nATOM    355  O   ALA B  36       0.206  -5.655   5.815  1.00  0.00           O  \\nATOM    356  CB  ALA B  36      -0.882  -5.339   2.718  1.00  0.00           C  \\nATOM    357  H   ALA B  36      -3.097  -4.716   3.395  1.00  0.00           H  \\nATOM    358  HA  ALA B  36      -1.374  -6.780   4.221  1.00  0.00           H  \\nATOM    359 1HB  ALA B  36       0.171  -5.619   2.677  1.00  0.00           H  \\nATOM    360 2HB  ALA B  36      -1.429  -5.872   1.941  1.00  0.00           H  \\nATOM    361 3HB  ALA B  36      -0.978  -4.266   2.558  1.00  0.00           H  \\nATOM    362  N   ALA B  37      -1.040  -3.807   5.280  1.00  0.00           N  \\nATOM    363  CA  ALA B  37      -0.337  -2.883   6.162  1.00  0.00           C  \\nATOM    364  C   ALA B  37      -0.490  -3.292   7.622  1.00  0.00           C  \\nATOM    365  O   ALA B  37       0.468  -3.245   8.393  1.00  0.00           O  \\nATOM    366  CB  ALA B  37      -0.843  -1.463   5.955  1.00  0.00           C  \\nATOM    367  H   ALA B  37      -1.834  -3.502   4.735  1.00  0.00           H  \\nATOM    368  HA  ALA B  37       0.724  -2.919   5.915  1.00  0.00           H  \\nATOM    369 1HB  ALA B  37      -0.309  -0.785   6.621  1.00  0.00           H  \\nATOM    370 2HB  ALA B  37      -0.674  -1.164   4.921  1.00  0.00           H  \\nATOM    371 3HB  ALA B  37      -1.909  -1.421   6.174  1.00  0.00           H  \\nATOM    372  N   ALA B  38      -1.741  -3.662   7.837  1.00  0.00           N  \\nATOM    373  CA  ALA B  38      -1.961  -3.902   9.258  1.00  0.00           C  \\nATOM    374  C   ALA B  38      -1.082  -5.037   9.768  1.00  0.00           C  \\nATOM    375  O   ALA B  38      -0.529  -4.962  10.865  1.00  0.00           O  \\nATOM    376  CB  ALA B  38      -3.428  -4.208   9.523  1.00  0.00           C  \\nATOM    377  H   ALA B  38      -2.373  -3.737   7.053  1.00  0.00           H  \\nATOM    378  HA  ALA B  38      -1.685  -2.997   9.800  1.00  0.00           H  \\nATOM    379 1HB  ALA B  38      -3.576  -4.384  10.589  1.00  0.00           H  \\nATOM    380 2HB  ALA B  38      -4.039  -3.362   9.207  1.00  0.00           H  \\nATOM    381 3HB  ALA B  38      -3.720  -5.096   8.965  1.00  0.00           H  \\nATOM    382  N   ALA B  39      -0.898  -6.046   9.116  1.00  0.00           N  \\nATOM    383  CA  ALA B  39      -0.025  -7.088   9.642  1.00  0.00           C  \\nATOM    384  C   ALA B  39       1.425  -6.622   9.687  1.00  0.00           C  \\nATOM    385  O   ALA B  39       2.145  -6.895  10.647  1.00  0.00           O  \\nATOM    386  CB  ALA B  39      -0.147  -8.354   8.806  1.00  0.00           C  \\nATOM    387  H   ALA B  39      -1.355  -6.144   8.221  1.00  0.00           H  \\nATOM    388  HA  ALA B  39      -0.337  -7.308  10.663  1.00  0.00           H  \\nATOM    389 1HB  ALA B  39       0.511  -9.123   9.212  1.00  0.00           H  \\nATOM    390 2HB  ALA B  39      -1.177  -8.709   8.831  1.00  0.00           H  \\nATOM    391 3HB  ALA B  39       0.138  -8.139   7.778  1.00  0.00           H  \\nATOM    392  N   ALA B  40       1.932  -5.977   8.812  1.00  0.00           N  \\nATOM    393  CA  ALA B  40       3.269  -5.408   8.695  1.00  0.00           C  \\nATOM    394  C   ALA B  40       3.448  -4.224   9.635  1.00  0.00           C  \\nATOM    395  O   ALA B  40       2.491  -3.513   9.944  1.00  0.00           O  \\nATOM    396  CB  ALA B  40       3.543  -4.989   7.258  1.00  0.00           C  \\nATOM    397  H   ALA B  40       1.392  -5.757   7.987  1.00  0.00           H  \\nATOM    398  HA  ALA B  40       3.989  -6.175   8.982  1.00  0.00           H  \\nATOM    399 1HB  ALA B  40       4.545  -4.567   7.187  1.00  0.00           H  \\nATOM    400 2HB  ALA B  40       3.468  -5.859   6.605  1.00  0.00           H  \\nATOM    401 3HB  ALA B  40       2.812  -4.243   6.952  1.00  0.00           H  \\nTER                                                                             \\nREMARK PDBinfo-LABEL:    1 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    2 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    3 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    4 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    5 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    6 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    7 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    8 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:    9 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   10 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   11 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   12 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   13 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   14 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   15 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   16 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   17 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   18 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   19 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   20 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   21 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   22 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   23 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   24 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   25 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   26 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   27 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   28 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   29 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   30 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   31 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   32 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   33 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   34 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   35 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   36 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   37 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   38 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   39 PEPTIDE_STUB_EXTENSION\\nREMARK PDBinfo-LABEL:   40 PEPTIDE_STUB_EXTENSION\\n\\n\",\"pdb\");\n",
              "\tviewer_16234210748847926.zoomTo();\n",
              "\tviewer_16234210748847926.setStyle({\"cartoon\": {\"color\": \"spectrum\"}, \"stick\": {\"colorscheme\": \"blackCarbon\", \"radius\": \"0.1\"}});\n",
              "viewer_16234210748847926.render();\n",
              "});\n",
              "</script>"
            ]
          },
          "metadata": {
            "tags": []
          }
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "419-nsfhmdlQ"
      },
      "source": [
        "## Conclusion\n",
        "\n",
        "This should give you an overview of how to use the MakeBundle and BundleGridSampler within PyRosetta.  Everything else is simply parameters.  Read up and you'll be ready to start designing your own helical bundles. "
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "NoVTLOyRmdlQ"
      },
      "source": [
        "## References:"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "wvop4ryrmdlQ"
      },
      "source": [
        "- **Bundle Tools**: Vikram Mulligan\n",
        "- Grigoryan, Gevorg and William F Degrado. Probing designability via a generalized model of helical bundle geometry Journal of molecular biology vol. 405,4 (2010): 1079-100. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3052747/\n",
        "- Huang, Po-Ssu et al. High thermodynamic stability of parametrically designed helical bundles Science (New York, N.Y.) vol. 346,6208 (2014): 481-485. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4612401/\n",
        "- Weitzner, Brian. \"Parametric disasters\" Published September 03, 2018. https://weitzner.github.io/posts/2018/08/parametric-disasters/"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "A_C4jYRumdlQ"
      },
      "source": [
        "# *De Novo* Protein Design with PyRosetta\n",
        "Keywords:\n",
        "\n",
        "## Overview"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "HwOz-3CsmdlQ"
      },
      "source": [
        "Before we begin, we must install the biopython package; it doesn't seem to require a \"restart notebook\" in order to be imported successfully in the cell after."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "T832jjV9mdlQ",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "d89457df-f3a9-47c4-a016-eed2a3ceda0f"
      },
      "source": [
        "!pip install biopython --user"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Collecting biopython\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/5a/42/de1ed545df624180b84c613e5e4de4848f72989ce5846a74af6baa0737b9/biopython-1.79-cp37-cp37m-manylinux_2_5_x86_64.manylinux1_x86_64.whl (2.3MB)\n",
            "\u001b[K     || 2.3MB 6.7MB/s \n",
            "\u001b[?25hRequirement already satisfied: numpy in /usr/local/lib/python3.7/dist-packages (from biopython) (1.19.5)\n",
            "Installing collected packages: biopython\n",
            "Successfully installed biopython-1.79\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "k_iXgy0AmdlQ"
      },
      "source": [
        "import Bio.Data.IUPACData as IUPACData\n",
        "import Bio.SeqUtils\n",
        "import logging\n",
        "logging.basicConfig(level=logging.INFO)\n",
        "import os\n",
        "import pyrosetta\n",
        "import pyrosetta.distributed\n",
        "import pyrosetta.distributed.viewer as viewer\n",
        "import site\n",
        "import sys"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "pkYOnh5SmdlR"
      },
      "source": [
        "Initialize PyRosetta:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "yJjuCv7_mdlR"
      },
      "source": [
        "flags = \"\"\"\n",
        "-linmem_ig 10\n",
        "-ignore_unrecognized_res 1\n",
        "-mute core.select.residue_selector.SecondaryStructureSelector\n",
        "-mute core.select.residue_selector.PrimarySequenceNeighborhoodSelector\n",
        "-mute protocols.DsspMover\n",
        "\"\"\"\n",
        "pyrosetta.distributed.init(flags)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "VGn8p8rSmdlR"
      },
      "source": [
        " Let's setup the pose of a *de novo* helical bundle from the PDB for downstream design: https://www.rcsb.org/structure/5J0J"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "3qU8arn7mdlR",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "95fd4f1d-68ee-4283-c5e8-ab4f61a66858"
      },
      "source": [
        "start_pose = pyrosetta.io.pose_from_file(\"inputs/5J0J.clean.pdb\")\n",
        "pose = start_pose.clone()"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:core.import_pose.import_pose: File 'inputs/5J0J.clean.pdb' automatically determined to be of type PDB\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  OXT on residue ALA:CtermProteinFull 70\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  OXT on residue ALA:CtermProteinFull 139\n",
            "INFO:rosetta:core.conformation.Conformation: [ WARNING ] missing heavyatom:  OXT on residue ALA:CtermProteinFull 210\n"
          ],
          "name": "stderr"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "TYaKl1o5mdlR"
      },
      "source": [
        "## Design Strategy \n",
        "\n",
        "Minimize the crystal structure coordinates, then convert chain \"A\" to poly-alanine, and then perform one-sided protein-protein interface design designing chain A while only re-packing the homotrimer interface residues of chains B and C! Therefore, we are designing a homotrimer into a heterotrimer. Furthermore, prevent backbone torsions from minimizing and only minimize the side-chains of the homotrimer interface and all of chain A using the `FastDesign` mover! After design, repack and minimize all side-chains."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "MgZG2xLpmdlR"
      },
      "source": [
        " Prior to and after design, we want to relax the protein with a scorefunction that packs the rotamers and minimizes the side-chain degrees of freedom while optimizing for realistic energies. If you were to allow backbone minimization (which you may optionally choose to), we would want use a Cartesian scorefunction (in this case, `ref2015_cart.wts`) which automatically sets `cart_bonded` scoreterm to a weight of 1.0, which helps to close chain breaks in the backbone. Similarly, you might want to also turn on the `coordinate_constraint` scoreterm to penalize deviations of the backbone coordinates from their initial coordinates during minimization. In this tutorial, we demonstrate that concept but will prevent backbone torsions from being minimized (however, feel free to turn on backbone minimization!):"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "ZUEILHySmdlR",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "b3d885de-5c01-4344-e0ba-65469d9e39c1"
      },
      "source": [
        "relax_scorefxn = pyrosetta.create_score_function(\"ref2015_cart.wts\")\n",
        "relax_scorefxn.set_weight(pyrosetta.rosetta.core.scoring.ScoreType.coordinate_constraint, 1.0)\n",
        "print(\"The starting pose total_score is {}\".format(relax_scorefxn(start_pose)))"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (210)\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "stream",
          "text": [
            "The starting pose total_score is 2113.581004929075\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "grsJVv6wmdlS"
      },
      "source": [
        " For *design*, if we allowed backbone minimzation we again would turn on the `coordinate_constraint` scoreterm to penalize deviations of the backbone coordinates from their initial coordinates during minimization. Additionally, we will use \"non-pairwise decomposable\" scoreterms to guide the packer trajectories (i.e. fixed backbone sequence design trajectories) in favor of our hypothetical design requirements to solve our biological problem. In this tutorial, we will make use of the following additional scoreterms during design:\n",
        "- `aa_composition`: This scoring term is intended for use during design, to penalize deviations from a desired residue type composition. Applies to any amino acid composition requirements specified by the user within a ResidueSelector. This scoreterm also applies to the `AddHelixSequenceConstraints` mover which sets up ideal sequence constraints for each helix in a pose or in a selection.\n",
        "- `voids_penalty`: This scoring term is intended for use during design, to penalize buried voids or cavities and to guide the packer to design solutions in which all buried volume is filled with side-chains.\n",
        "- `aa_repeat`: This wholebody scoring term is intended for use during design, to penalize long stretches in which the same residue type repeats over and over (e.g. poly-Q sequences). \n",
        "- `buried_unsatisfied_penalty`: This scoring term is intended for use during design, to provide a penalty for buried hydrogen bond donors or acceptors that are unsatisfied. \n",
        "- `netcharge`: This scoring term is intended for use during design, to penalize deviations from a desired net charge in a pose or in a selection.\n",
        "- `hbnet`: This scoring term is intended for use during design, to provide a bonus for hydrogen bond network formation."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "3KmAGvCSmdlS",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "f04aacd4-14e0-432c-f3ae-98bfe41ea806"
      },
      "source": [
        "design_scorefxn = pyrosetta.create_score_function(\"ref2015_cart.wts\")\n",
        "design_scorefxn.set_weight(pyrosetta.rosetta.core.scoring.ScoreType.coordinate_constraint, 10.0)\n",
        "design_scorefxn.set_weight(pyrosetta.rosetta.core.scoring.ScoreType.aa_composition, 1.0)\n",
        "design_scorefxn.set_weight(pyrosetta.rosetta.core.scoring.ScoreType.voids_penalty, 0.25)\n",
        "design_scorefxn.set_weight(pyrosetta.rosetta.core.scoring.ScoreType.aa_repeat, 1.0)\n",
        "design_scorefxn.set_weight(pyrosetta.rosetta.core.scoring.ScoreType.buried_unsatisfied_penalty, 1.0)\n",
        "design_scorefxn.set_weight(pyrosetta.rosetta.core.scoring.ScoreType.hbnet, 1.0)\n",
        "design_scorefxn.set_weight(pyrosetta.rosetta.core.scoring.ScoreType.netcharge, 1.0)\n",
        "print(\"The starting pose total_score is {}\".format(design_scorefxn(start_pose)))"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:basic.io.database: Database file opened: scoring/score_functions/aa_repeat_energy/default_repeat_penalty_table.rpt_pen\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "stream",
          "text": [
            "The starting pose total_score is 23016.581004929074\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "Rgv4TJ-hmdlS"
      },
      "source": [
        " By using the `relax_scorefxn` before and after the `design_scorefxn`, we ensure that these \"non-pairwise decomposable\" scoreterms are not forcing unrealistic rotamers that would otherwise not be held in place without these additional scoreterms."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "stQblJbGmdlS"
      },
      "source": [
        " Prior to any deviation from crystal structure coordinates, apply coordinate constraints to all of the backbone heavy atoms:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "b_yeMtv-mdlS"
      },
      "source": [
        "true_selector = pyrosetta.rosetta.core.select.residue_selector.TrueResidueSelector() # Select all residues\n",
        "\n",
        "# Apply a virtual root onto the pose to prevent large lever-arm effects while minimizing with coordinate constraints\n",
        "virtual_root = pyrosetta.rosetta.protocols.simple_moves.VirtualRootMover()\n",
        "virtual_root.set_removable(True)\n",
        "virtual_root.set_remove(False)\n",
        "virtual_root.apply(pose)\n",
        "\n",
        "# Construct the CoordinateConstraintGenerator\n",
        "coord_constraint_gen = pyrosetta.rosetta.protocols.constraint_generator.CoordinateConstraintGenerator()\n",
        "coord_constraint_gen.set_id(\"contrain_all_backbone_atoms!\")\n",
        "coord_constraint_gen.set_ambiguous_hnq(False)\n",
        "coord_constraint_gen.set_bounded(False)\n",
        "coord_constraint_gen.set_sidechain(False)\n",
        "coord_constraint_gen.set_sd(1.0) # Sets a standard deviation of contrained atoms to (an arbitrary) 1.0 Angstroms RMSD. Set higher or lower for different results.\n",
        "coord_constraint_gen.set_ca_only(False)\n",
        "coord_constraint_gen.set_residue_selector(true_selector)\n",
        "\n",
        "# Apply the CoordinateConstraintGenerator using the AddConstraints mover\n",
        "add_constraints = pyrosetta.rosetta.protocols.constraint_generator.AddConstraints()\n",
        "add_constraints.add_generator(coord_constraint_gen)\n",
        "add_constraints.apply(pose)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "BSajSgkEmdlT"
      },
      "source": [
        " Prior to design, minimize with the `FastRelax` mover to optimize the pose within the `relax_scorefxn` scorefunction. Note: this takes ~1min 10s"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "lE2QN9qRmdlT",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "01deceed-13c1-4b23-b3ef-3d3799e01496"
      },
      "source": [
        "tf = pyrosetta.rosetta.core.pack.task.TaskFactory()\n",
        "tf.push_back(pyrosetta.rosetta.core.pack.task.operation.InitializeFromCommandline())\n",
        "tf.push_back(pyrosetta.rosetta.core.pack.task.operation.IncludeCurrent())\n",
        "tf.push_back(pyrosetta.rosetta.core.pack.task.operation.NoRepackDisulfides())\n",
        "tf.push_back(pyrosetta.rosetta.core.pack.task.operation.OperateOnResidueSubset(\n",
        "    pyrosetta.rosetta.core.pack.task.operation.PreventRepackingRLT(), true_selector)) # Set to RestrictToRepackingRLT for slower/better results\n",
        "mm = pyrosetta.rosetta.core.kinematics.MoveMap()\n",
        "mm.set_bb(False) # Set to true if desired\n",
        "mm.set_chi(True)\n",
        "mm.set_jump(False)\n",
        "fast_relax = pyrosetta.rosetta.protocols.relax.FastRelax(scorefxn_in=relax_scorefxn, standard_repeats=1) # 2-5 repeats suggested for real applications\n",
        "fast_relax.cartesian(True)\n",
        "fast_relax.set_task_factory(tf)\n",
        "fast_relax.set_movemap(mm)\n",
        "fast_relax.minimize_bond_angles(True)\n",
        "fast_relax.minimize_bond_lengths(True)\n",
        "fast_relax.min_type(\"lbfgs_armijo_nonmonotone\")\n",
        "fast_relax.ramp_down_constraints(False)\n",
        "\n",
        "if not os.getenv(\"DEBUG\"):\n",
        "    %time fast_relax.apply(pose)\n",
        "\n",
        "# Optionally, instead of running this you could reload the saved pose from a previously run trajectory:\n",
        "#pose = pyrosetta.pose_from_file(\"minimized_start_pose.pdb\")"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (211)\n",
            "INFO:rosetta:core.energy_methods.CartesianBondedEnergy: Adding undefined angle VRT: X,ORIG,Y to DB with theta0 = 1.5708 , Ktheta = 80\n",
            "INFO:rosetta:core.energy_methods.CartesianBondedEnergy: Adding undefined length VRT: ORIG,X, to DB with d0 = 1 , Kd = 300\n",
            "INFO:rosetta:core.energy_methods.CartesianBondedEnergy: Adding undefined length VRT: ORIG,Y, to DB with d0 = 1 , Kd = 300\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: repeat  2113.58  0  0  0.55\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: coord_cst_weight  2113.58  0  0  0.55\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: scale:fa_rep  2001.9  0  0  0.022\n",
            "INFO:rosetta:core.pack.task: Packer task: initialize from command line()\n",
            "INFO:rosetta:core.pack.pack_rotamers: built 0 rotamers at 0 positions.\n",
            "INFO:rosetta:core.pack.interaction_graph.interaction_graph_factory: Instantiating DensePDInteractionGraph\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: repack  2001.9  0  0  0.022\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: scale:fa_rep  2003.18  0  0  0.02805\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: min  377.05  0  0  0.02805\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: coord_cst_weight  377.05  0  0  0.02805\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: scale:fa_rep  448.823  0  0  0.14575\n",
            "INFO:rosetta:core.pack.task: Packer task: initialize from command line()\n",
            "INFO:rosetta:core.pack.pack_rotamers: built 0 rotamers at 0 positions.\n",
            "INFO:rosetta:core.pack.interaction_graph.interaction_graph_factory: Instantiating DensePDInteractionGraph\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: repack  448.823  0  0  0.14575\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: scale:fa_rep  453.854  0  0  0.154\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: min  426.479  0  0  0.154\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: coord_cst_weight  426.479  0  0  0.154\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: scale:fa_rep  473.834  0  0  0.30745\n",
            "INFO:rosetta:core.pack.task: Packer task: initialize from command line()\n",
            "INFO:rosetta:core.pack.pack_rotamers: built 0 rotamers at 0 positions.\n",
            "INFO:rosetta:core.pack.interaction_graph.interaction_graph_factory: Instantiating DensePDInteractionGraph\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: repack  473.834  0  0  0.30745\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: scale:fa_rep  477.568  0  0  0.31955\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: min  469.781  0  0  0.31955\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: coord_cst_weight  469.781  0  0  0.31955\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: scale:fa_rep  523.488  0  0  0.55\n",
            "INFO:rosetta:core.pack.task: Packer task: initialize from command line()\n",
            "INFO:rosetta:core.pack.pack_rotamers: built 0 rotamers at 0 positions.\n",
            "INFO:rosetta:core.pack.interaction_graph.interaction_graph_factory: Instantiating DensePDInteractionGraph\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: repack  523.488  0  0  0.55\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: min  518.074  0  0  0.55\n",
            "INFO:rosetta:protocols.relax.FastRelax: MRP: 0  518.074  518.074  0  0\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: accept_to_best  518.074  0  0  0.55\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: endrepeat  518.074  0  0  0.55\n",
            "INFO:rosetta:protocols::checkpoint: Deleting checkpoints of FastRelax\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "stream",
          "text": [
            "CPU times: user 1min 27s, sys: 615 ms, total: 1min 28s\n",
            "Wall time: 1min 27s\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "sWO1dVNrmdlT"
      },
      "source": [
        " Let's check the delta `total_score` per residue after minimizing in the `relax_scorefxn` scorefunction:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "1XpitjyEmdlT",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "41eca4fa-925c-420c-f14b-6e8d185502b7"
      },
      "source": [
        "if not os.getenv(\"DEBUG\"):\n",
        "    initial_score_res = relax_scorefxn(start_pose)/start_pose.size()\n",
        "    final_score_res = relax_scorefxn(pose)/pose.size()\n",
        "    delta_total_score_res = final_score_res - initial_score_res\n",
        "    print(\"{0} kcal/(mol*res) - {1} kcal/(mol*res) = {2} kcal/(mol*res)\".format(final_score_res, initial_score_res, delta_total_score_res))"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "2.4553280277730765 kcal/(mol*res) - 10.064671452043214 kcal/(mol*res) = -7.609343424270138 kcal/(mol*res)\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "Kb0yWRjcmdlT"
      },
      "source": [
        " We can see that the crystal structure coordinates were not quite optimal according to the `relax_scorefxn` scorefunction. So which model is correct?\n",
        "\n",
        "By how many Angstroms RMSD did the backbone C atoms move?"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "nbgrader": {
          "grade": true,
          "grade_id": "cell-8da10817eefb8d88",
          "locked": false,
          "points": 0,
          "schema_version": 3,
          "solution": true
        },
        "id": "JkIpyhwbmdlT",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "e7302a16-dffc-473f-bee0-ac51164029ba"
      },
      "source": [
        "pyrosetta.rosetta.core.scoring.CA_rmsd(start_pose, pose)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "0.0"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 73
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "1lDlaHgfmdlU"
      },
      "source": [
        " For downstream analysis, we want to save the pose:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "A2Tz_ybOmdlU",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "8a512ffa-e088-4fc8-9730-1a68a6a6aca6"
      },
      "source": [
        "minimized_start_pose = pose.clone()\n",
        "pyrosetta.dump_pdb(minimized_start_pose, \"outputs/minimized_start_pose.pdb\")"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "True"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 75
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "wP-H9g72mdlU"
      },
      "source": [
        "## *De Novo* Protein Design\n",
        "\n",
        "Prior to designing chain A, first let's make chain A poly-alanine so that we can re-design the sidechains onto the backbone:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "WNKMDiR4mdlU"
      },
      "source": [
        "# Since we will do direct pose manipulation, first remove the constraints\n",
        "remove_constraints = pyrosetta.rosetta.protocols.constraint_generator.RemoveConstraints()\n",
        "remove_constraints.add_generator(coord_constraint_gen)\n",
        "remove_constraints.apply(pose)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "XD0YPoBdmdlU"
      },
      "source": [
        " Obtain chain A and convert it to poly-alanine"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "vQKEPt-ImdlU"
      },
      "source": [
        "keep_chA = pyrosetta.rosetta.protocols.grafting.simple_movers.KeepRegionMover(res_start=str(pose.chain_begin(1)), res_end=str(pose.chain_end(1)))\n",
        "keep_chA.apply(pose)\n",
        "polyA_chA = pyrosetta.rosetta.protocols.pose_creation.MakePolyXMover(aa=\"ALA\", keep_pro=0, keep_gly=0, keep_disulfide_cys=0)\n",
        "polyA_chA.apply(pose)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "eA7kc9e3mdlV"
      },
      "source": [
        " Obtain chains B and C"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "0XmH8WocmdlV"
      },
      "source": [
        "pose_chBC = minimized_start_pose.clone()\n",
        "keep_chBC = pyrosetta.rosetta.protocols.grafting.simple_movers.KeepRegionMover(res_start=str(pose_chBC.chain_begin(2)), res_end=str(pose_chBC.chain_end(3)-1))\n",
        "keep_chBC.apply(pose_chBC)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "clFbGu43mdlV"
      },
      "source": [
        " Append chains B and C onto the poly-alanine version of chain A"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "tzgakQKXmdlV"
      },
      "source": [
        "pyrosetta.rosetta.core.pose.append_pose_to_pose(pose1=pose, pose2=pose_chBC, new_chain=True)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "-2zrmcZXmdlV"
      },
      "source": [
        " Pose is now considered to have only 2 chains."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "xpUC4Fd1mdlV",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "e13ce6f1-d759-4a43-92c4-05f9d52f2bc8"
      },
      "source": [
        "pose.num_chains()"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "2"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 81
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "tjxrx8HvmdlW"
      },
      "source": [
        " Let's re-establish that there are 3 chains."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "yRApfpJcmdlW",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "b6ef4f72-1bf3-4d48-c3c6-635af6a2e4ae"
      },
      "source": [
        "switch_chains = pyrosetta.rosetta.protocols.simple_moves.SwitchChainOrderMover()\n",
        "switch_chains.chain_order(\"12\")\n",
        "switch_chains.apply(pose)\n",
        "\n",
        "print(pose.pdb_info())\n",
        "print(\"Now the number of chains = {}\".format(pose.num_chains()))"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:core.scoring.ScoreFunctionFactory: SCOREFUNCTION: ref2015\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "stream",
          "text": [
            "PDB file name: \n",
            " Pose Range  Chain    PDB Range  |   #Residues         #Atoms\n",
            "\n",
            "0001 -- 0070    A 0001  -- 0070  |   0070 residues;    00703 atoms\n",
            "0071 -- 0139    B 0071  -- 0139  |   0069 residues;    01187 atoms\n",
            "0140 -- 0209    C 0140  -- 0209  |   0070 residues;    01212 atoms\n",
            "                           TOTAL |   0209 residues;    03102 atoms\n",
            "\n",
            "Now the number of chains = 3\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "1BgttH3JmdlW"
      },
      "source": [
        " Re-apply backbone coordinate constraints:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "qcO9ygqomdlW"
      },
      "source": [
        "virtual_root.apply(pose)\n",
        "add_constraints.apply(pose)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "Zv9D657qmdlW"
      },
      "source": [
        " Have a look at the new pose, chain A in which is ready to be designed!"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 817
        },
        "id": "SJkNIRD_fQjn",
        "outputId": "34fa03a4-b986-4852-c703-3f5e26c77696"
      },
      "source": [
        "view_pose(pose)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_16234219519367425\"  style=\"position: relative; width: 1200px; height: 800px\">\n        <p id=\"3dmolwarning_16234219519367425\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n      resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n}\n\nvar viewer_16234219519367425 = null;\nvar warn = document.getElementById(\"3dmolwarning_16234219519367425\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_16234219519367425 = $3Dmol.createViewer($(\"#3dmolviewer_16234219519367425\"),{backgroundColor:\"white\"});\nviewer_16234219519367425.zoomTo();\n\tviewer_16234219519367425.addModels(\"HEADER                                            11-JUN-21   XXXX              \\nEXPDTA    THEORETICAL MODEL                                                     \\nREMARK 220                                                                      \\nREMARK 220 EXPERIMENTAL DETAILS                                                 \\nREMARK 220  EXPERIMENT TYPE                : THEORETICAL MODELLING              \\nREMARK 220  DATE OF DATA COLLECTION        : 11-JUN-21                          \\nREMARK 220                                                                      \\nREMARK 220 REMARK: MODEL GENERATED BY ROSETTA                                   \\nREMARK 220  VERSION 2020.50.post.dev+970.commits.3700df14560                    \\nATOM      1  N   ALA A   1      46.718  19.029  14.515  1.00  0.00           N  \\nATOM      2  CA  ALA A   1      45.711  17.981  14.626  1.00  0.00           C  \\nATOM      3  C   ALA A   1      46.287  16.744  15.312  1.00  0.00           C  \\nATOM      4  O   ALA A   1      45.575  16.034  16.023  1.00  0.00           O  \\nATOM      5  CB  ALA A   1      45.165  17.622  13.252  1.00  0.00           C  \\nATOM      6 1H   ALA A   1      46.320  19.828  14.064  1.00  0.00           H  \\nATOM      7 2H   ALA A   1      47.035  19.286  15.428  1.00  0.00           H  \\nATOM      8 3H   ALA A   1      47.491  18.693  13.977  1.00  0.00           H  \\nATOM      9  HA  ALA A   1      44.974  18.308  15.166  1.00  0.00           H  \\nATOM     10 1HB  ALA A   1      44.415  16.838  13.352  1.00  0.00           H  \\nATOM     11 2HB  ALA A   1      44.710  18.504  12.800  1.00  0.00           H  \\nATOM     12 3HB  ALA A   1      45.977  17.269  12.619  1.00  0.00           H  \\nATOM     13  N   ALA A   2      47.573  16.485  15.094  1.00  0.00           N  \\nATOM     14  CA  ALA A   2      48.260  15.426  15.824  1.00  0.00           C  \\nATOM     15  C   ALA A   2      48.210  15.740  17.310  1.00  0.00           C  \\nATOM     16  O   ALA A   2      48.033  14.850  18.140  1.00  0.00           O  \\nATOM     17  CB  ALA A   2      49.698  15.278  15.349  1.00  0.00           C  \\nATOM     18  H   ALA A   2      48.068  16.907  14.532  1.00  0.00           H  \\nATOM     19  HA  ALA A   2      47.807  14.582  15.672  1.00  0.00           H  \\nATOM     20 1HB  ALA A   2      50.193  14.496  15.925  1.00  0.00           H  \\nATOM     21 2HB  ALA A   2      49.707  15.011  14.293  1.00  0.00           H  \\nATOM     22 3HB  ALA A   2      50.226  16.220  15.490  1.00  0.00           H  \\nATOM     23  N   ALA A   3      48.363  17.021  17.631  1.00  0.00           N  \\nATOM     24  CA  ALA A   3      48.318  17.482  19.011  1.00  0.00           C  \\nATOM     25  C   ALA A   3      46.922  17.287  19.594  1.00  0.00           C  \\nATOM     26  O   ALA A   3      46.777  16.910  20.757  1.00  0.00           O  \\nATOM     27  CB  ALA A   3      48.735  18.943  19.099  1.00  0.00           C  \\nATOM     28  H   ALA A   3      48.495  17.650  17.059  1.00  0.00           H  \\nATOM     29  HA  ALA A   3      48.942  16.962  19.540  1.00  0.00           H  \\nATOM     30 1HB  ALA A   3      48.692  19.272  20.137  1.00  0.00           H  \\nATOM     31 2HB  ALA A   3      49.753  19.054  18.726  1.00  0.00           H  \\nATOM     32 3HB  ALA A   3      48.061  19.550  18.498  1.00  0.00           H  \\nATOM     33  N   ALA A   4      45.898  17.538  18.781  1.00  0.00           N  \\nATOM     34  CA  ALA A   4      44.513  17.367  19.215  1.00  0.00           C  \\nATOM     35  C   ALA A   4      44.255  15.920  19.614  1.00  0.00           C  \\nATOM     36  O   ALA A   4      43.737  15.652  20.696  1.00  0.00           O  \\nATOM     37  CB  ALA A   4      43.550  17.796  18.118  1.00  0.00           C  \\nATOM     38  H   ALA A   4      45.979  17.810  17.969  1.00  0.00           H  \\nATOM     39  HA  ALA A   4      44.352  17.927  19.990  1.00  0.00           H  \\nATOM     40 1HB  ALA A   4      42.524  17.654  18.458  1.00  0.00           H  \\nATOM     41 2HB  ALA A   4      43.712  18.848  17.883  1.00  0.00           H  \\nATOM     42 3HB  ALA A   4      43.723  17.195  17.226  1.00  0.00           H  \\nATOM     43  N   ALA A   5      44.611  14.995  18.729  1.00  0.00           N  \\nATOM     44  CA  ALA A   5      44.488  13.572  19.015  1.00  0.00           C  \\nATOM     45  C   ALA A   5      45.295  13.204  20.257  1.00  0.00           C  \\nATOM     46  O   ALA A   5      44.842  12.424  21.094  1.00  0.00           O  \\nATOM     47  CB  ALA A   5      44.941  12.747  17.819  1.00  0.00           C  \\nATOM     48  H   ALA A   5      44.928  15.169  17.949  1.00  0.00           H  \\nATOM     49  HA  ALA A   5      43.557  13.360  19.185  1.00  0.00           H  \\nATOM     50 1HB  ALA A   5      44.846  11.686  18.052  1.00  0.00           H  \\nATOM     51 2HB  ALA A   5      44.320  12.986  16.956  1.00  0.00           H  \\nATOM     52 3HB  ALA A   5      45.981  12.976  17.592  1.00  0.00           H  \\nATOM     53  N   ALA A   6      46.486  13.781  20.374  1.00  0.00           N  \\nATOM     54  CA  ALA A   6      47.366  13.501  21.502  1.00  0.00           C  \\nATOM     55  C   ALA A   6      46.771  14.033  22.801  1.00  0.00           C  \\nATOM     56  O   ALA A   6      46.837  13.370  23.836  1.00  0.00           O  \\nATOM     57  CB  ALA A   6      48.744  14.102  21.263  1.00  0.00           C  \\nATOM     58  H   ALA A   6      46.810  14.343  19.809  1.00  0.00           H  \\nATOM     59  HA  ALA A   6      47.470  12.541  21.591  1.00  0.00           H  \\nATOM     60 1HB  ALA A   6      49.388  13.885  22.116  1.00  0.00           H  \\nATOM     61 2HB  ALA A   6      49.178  13.670  20.361  1.00  0.00           H  \\nATOM     62 3HB  ALA A   6      48.654  15.180  21.142  1.00  0.00           H  \\nATOM     63  N   ALA A   7      46.190  15.228  22.743  1.00  0.00           N  \\nATOM     64  CA  ALA A   7      45.617  15.851  23.931  1.00  0.00           C  \\nATOM     65  C   ALA A   7      44.366  15.115  24.392  1.00  0.00           C  \\nATOM     66  O   ALA A   7      44.138  14.970  25.590  1.00  0.00           O  \\nATOM     67  CB  ALA A   7      45.299  17.314  23.661  1.00  0.00           C  \\nATOM     68  H   ALA A   7      46.114  15.698  22.027  1.00  0.00           H  \\nATOM     69  HA  ALA A   7      46.267  15.814  24.650  1.00  0.00           H  \\nATOM     70 1HB  ALA A   7      44.872  17.766  24.556  1.00  0.00           H  \\nATOM     71 2HB  ALA A   7      46.214  17.841  23.390  1.00  0.00           H  \\nATOM     72 3HB  ALA A   7      44.584  17.386  22.843  1.00  0.00           H  \\nATOM     73  N   ALA A   8      43.558  14.656  23.440  1.00  0.00           N  \\nATOM     74  CA  ALA A   8      42.375  13.868  23.763  1.00  0.00           C  \\nATOM     75  C   ALA A   8      42.787  12.599  24.490  1.00  0.00           C  \\nATOM     76  O   ALA A   8      42.157  12.197  25.469  1.00  0.00           O  \\nATOM     77  CB  ALA A   8      41.587  13.534  22.505  1.00  0.00           C  \\nATOM     78  H   ALA A   8      43.674  14.789  22.598  1.00  0.00           H  \\nATOM     79  HA  ALA A   8      41.798  14.382  24.350  1.00  0.00           H  \\nATOM     80 1HB  ALA A   8      40.716  12.934  22.770  1.00  0.00           H  \\nATOM     81 2HB  ALA A   8      41.259  14.457  22.025  1.00  0.00           H  \\nATOM     82 3HB  ALA A   8      42.218  12.973  21.818  1.00  0.00           H  \\nATOM     83  N   ALA A   9      43.856  11.977  24.006  1.00  0.00           N  \\nATOM     84  CA  ALA A   9      44.400  10.786  24.639  1.00  0.00           C  \\nATOM     85  C   ALA A   9      44.876  11.138  26.042  1.00  0.00           C  \\nATOM     86  O   ALA A   9      44.715  10.356  26.980  1.00  0.00           O  \\nATOM     87  CB  ALA A   9      45.535  10.202  23.811  1.00  0.00           C  \\nATOM     88  H   ALA A   9      44.287  12.228  23.305  1.00  0.00           H  \\nATOM     89  HA  ALA A   9      43.705  10.112  24.711  1.00  0.00           H  \\nATOM     90 1HB  ALA A   9      45.936   9.321  24.314  1.00  0.00           H  \\nATOM     91 2HB  ALA A   9      45.161   9.919  22.827  1.00  0.00           H  \\nATOM     92 3HB  ALA A   9      46.323  10.945  23.699  1.00  0.00           H  \\nATOM     93  N   ALA A  10      45.456  12.327  26.179  1.00  0.00           N  \\nATOM     94  CA  ALA A  10      45.928  12.800  27.473  1.00  0.00           C  \\nATOM     95  C   ALA A  10      44.723  13.115  28.353  1.00  0.00           C  \\nATOM     96  O   ALA A  10      44.672  12.731  29.523  1.00  0.00           O  \\nATOM     97  CB  ALA A  10      46.821  14.022  27.314  1.00  0.00           C  \\nATOM     98  H   ALA A  10      45.587  12.880  25.534  1.00  0.00           H  \\nATOM     99  HA  ALA A  10      46.445  12.100  27.902  1.00  0.00           H  \\nATOM    100 1HB  ALA A  10      47.145  14.366  28.296  1.00  0.00           H  \\nATOM    101 2HB  ALA A  10      47.693  13.760  26.715  1.00  0.00           H  \\nATOM    102 3HB  ALA A  10      46.265  14.816  26.817  1.00  0.00           H  \\nATOM    103  N   ALA A  11      43.746  13.798  27.767  1.00  0.00           N  \\nATOM    104  CA  ALA A  11      42.513  14.142  28.459  1.00  0.00           C  \\nATOM    105  C   ALA A  11      41.830  12.867  28.946  1.00  0.00           C  \\nATOM    106  O   ALA A  11      41.266  12.827  30.038  1.00  0.00           O  \\nATOM    107  CB  ALA A  11      41.588  14.939  27.551  1.00  0.00           C  \\nATOM    108  H   ALA A  11      43.775  14.077  26.954  1.00  0.00           H  \\nATOM    109  HA  ALA A  11      42.719  14.693  29.231  1.00  0.00           H  \\nATOM    110 1HB  ALA A  11      40.668  15.175  28.085  1.00  0.00           H  \\nATOM    111 2HB  ALA A  11      42.081  15.864  27.251  1.00  0.00           H  \\nATOM    112 3HB  ALA A  11      41.353  14.351  26.665  1.00  0.00           H  \\nATOM    113  N   ALA A  12      41.916  11.820  28.132  1.00  0.00           N  \\nATOM    114  CA  ALA A  12      41.329  10.527  28.460  1.00  0.00           C  \\nATOM    115  C   ALA A  12      42.018   9.891  29.669  1.00  0.00           C  \\nATOM    116  O   ALA A  12      41.355   9.346  30.550  1.00  0.00           O  \\nATOM    117  CB  ALA A  12      41.402   9.597  27.258  1.00  0.00           C  \\nATOM    118  H   ALA A  12      42.315  11.834  27.371  1.00  0.00           H  \\nATOM    119  HA  ALA A  12      40.393  10.650  28.679  1.00  0.00           H  \\nATOM    120 1HB  ALA A  12      40.956   8.636  27.514  1.00  0.00           H  \\nATOM    121 2HB  ALA A  12      40.858  10.038  26.423  1.00  0.00           H  \\nATOM    122 3HB  ALA A  12      42.443   9.449  26.975  1.00  0.00           H  \\nATOM    123  N   ALA A  13      43.346   9.967  29.708  1.00  0.00           N  \\nATOM    124  CA  ALA A  13      44.112   9.386  30.807  1.00  0.00           C  \\nATOM    125  C   ALA A  13      43.873  10.137  32.114  1.00  0.00           C  \\nATOM    126  O   ALA A  13      43.833   9.532  33.186  1.00  0.00           O  \\nATOM    127  CB  ALA A  13      45.596   9.376  30.468  1.00  0.00           C  \\nATOM    128  H   ALA A  13      43.828  10.351  29.108  1.00  0.00           H  \\nATOM    129  HA  ALA A  13      43.831   8.466  30.935  1.00  0.00           H  \\nATOM    130 1HB  ALA A  13      46.155   8.938  31.295  1.00  0.00           H  \\nATOM    131 2HB  ALA A  13      45.759   8.786  29.567  1.00  0.00           H  \\nATOM    132 3HB  ALA A  13      45.937  10.396  30.301  1.00  0.00           H  \\nATOM    133  N   ALA A  14      43.724  11.456  32.023  1.00  0.00           N  \\nATOM    134  CA  ALA A  14      43.447  12.281  33.195  1.00  0.00           C  \\nATOM    135  C   ALA A  14      42.111  11.897  33.818  1.00  0.00           C  \\nATOM    136  O   ALA A  14      41.958  11.886  35.037  1.00  0.00           O  \\nATOM    137  CB  ALA A  14      43.456  13.757  32.824  1.00  0.00           C  \\nATOM    138  H   ALA A  14      43.779  11.899  31.288  1.00  0.00           H  \\nATOM    139  HA  ALA A  14      44.142  12.139  33.857  1.00  0.00           H  \\nATOM    140 1HB  ALA A  14      43.242  14.356  33.709  1.00  0.00           H  \\nATOM    141 2HB  ALA A  14      44.436  14.027  32.431  1.00  0.00           H  \\nATOM    142 3HB  ALA A  14      42.697  13.946  32.067  1.00  0.00           H  \\nATOM    143  N   ALA A  15      41.147  11.585  32.961  1.00  0.00           N  \\nATOM    144  CA  ALA A  15      39.818  11.185  33.401  1.00  0.00           C  \\nATOM    145  C   ALA A  15      39.872   9.876  34.185  1.00  0.00           C  \\nATOM    146  O   ALA A  15      39.279   9.765  35.258  1.00  0.00           O  \\nATOM    147  CB  ALA A  15      38.883  11.052  32.208  1.00  0.00           C  \\nATOM    148  H   ALA A  15      41.240  11.598  32.106  1.00  0.00           H  \\nATOM    149  HA  ALA A  15      39.455  11.871  33.982  1.00  0.00           H  \\nATOM    150 1HB  ALA A  15      37.893  10.754  32.552  1.00  0.00           H  \\nATOM    151 2HB  ALA A  15      38.814  12.010  31.692  1.00  0.00           H  \\nATOM    152 3HB  ALA A  15      39.271  10.299  31.524  1.00  0.00           H  \\nATOM    153  N   ALA A  16      40.590   8.892  33.650  1.00  0.00           N  \\nATOM    154  CA  ALA A  16      40.716   7.594  34.307  1.00  0.00           C  \\nATOM    155  C   ALA A  16      41.311   7.758  35.696  1.00  0.00           C  \\nATOM    156  O   ALA A  16      40.893   7.094  36.646  1.00  0.00           O  \\nATOM    157  CB  ALA A  16      41.568   6.649  33.472  1.00  0.00           C  \\nATOM    158  H   ALA A  16      41.015   8.950  32.904  1.00  0.00           H  \\nATOM    159  HA  ALA A  16      39.835   7.198  34.400  1.00  0.00           H  \\nATOM    160 1HB  ALA A  16      41.657   5.691  33.983  1.00  0.00           H  \\nATOM    161 2HB  ALA A  16      41.099   6.500  32.499  1.00  0.00           H  \\nATOM    162 3HB  ALA A  16      42.559   7.080  33.334  1.00  0.00           H  \\nATOM    163  N   ALA A  17      42.288   8.649  35.808  1.00  0.00           N  \\nATOM    164  CA  ALA A  17      42.912   8.929  37.091  1.00  0.00           C  \\nATOM    165  C   ALA A  17      41.897   9.557  38.044  1.00  0.00           C  \\nATOM    166  O   ALA A  17      41.898   9.264  39.238  1.00  0.00           O  \\nATOM    167  CB  ALA A  17      44.117   9.840  36.911  1.00  0.00           C  \\nATOM    168  H   ALA A  17      42.608   9.106  35.154  1.00  0.00           H  \\nATOM    169  HA  ALA A  17      43.221   8.097  37.483  1.00  0.00           H  \\nATOM    170 1HB  ALA A  17      44.568  10.043  37.883  1.00  0.00           H  \\nATOM    171 2HB  ALA A  17      44.849   9.353  36.266  1.00  0.00           H  \\nATOM    172 3HB  ALA A  17      43.800  10.777  36.456  1.00  0.00           H  \\nATOM    173  N   ALA A  18      41.024  10.410  37.511  1.00  0.00           N  \\nATOM    174  CA  ALA A  18      39.999  11.059  38.327  1.00  0.00           C  \\nATOM    175  C   ALA A  18      39.078  10.027  38.968  1.00  0.00           C  \\nATOM    176  O   ALA A  18      38.849  10.057  40.177  1.00  0.00           O  \\nATOM    177  CB  ALA A  18      39.192  12.040  37.488  1.00  0.00           C  \\nATOM    178  H   ALA A  18      41.004  10.630  36.679  1.00  0.00           H  \\nATOM    179  HA  ALA A  18      40.431  11.559  39.037  1.00  0.00           H  \\nATOM    180 1HB  ALA A  18      38.432  12.511  38.111  1.00  0.00           H  \\nATOM    181 2HB  ALA A  18      39.856  12.804  37.084  1.00  0.00           H  \\nATOM    182 3HB  ALA A  18      38.712  11.508  36.669  1.00  0.00           H  \\nATOM    183  N   ALA A  19      38.551   9.119  38.152  1.00  0.00           N  \\nATOM    184  CA  ALA A  19      37.649   8.080  38.637  1.00  0.00           C  \\nATOM    185  C   ALA A  19      38.327   7.198  39.676  1.00  0.00           C  \\nATOM    186  O   ALA A  19      37.689   6.746  40.626  1.00  0.00           O  \\nATOM    187  CB  ALA A  19      37.143   7.235  37.477  1.00  0.00           C  \\nATOM    188  H   ALA A  19      38.702   9.083  37.306  1.00  0.00           H  \\nATOM    189  HA  ALA A  19      36.880   8.499  39.054  1.00  0.00           H  \\nATOM    190 1HB  ALA A  19      36.471   6.464  37.855  1.00  0.00           H  \\nATOM    191 2HB  ALA A  19      36.607   7.869  36.771  1.00  0.00           H  \\nATOM    192 3HB  ALA A  19      37.987   6.765  36.975  1.00  0.00           H  \\nATOM    193  N   ALA A  20      39.619   6.950  39.487  1.00  0.00           N  \\nATOM    194  CA  ALA A  20      40.389   6.158  40.438  1.00  0.00           C  \\nATOM    195  C   ALA A  20      40.457   6.878  41.780  1.00  0.00           C  \\nATOM    196  O   ALA A  20      40.299   6.264  42.835  1.00  0.00           O  \\nATOM    197  CB  ALA A  20      41.787   5.885  39.903  1.00  0.00           C  \\nATOM    198  H   ALA A  20      40.075   7.231  38.814  1.00  0.00           H  \\nATOM    199  HA  ALA A  20      39.948   5.305  40.573  1.00  0.00           H  \\nATOM    200 1HB  ALA A  20      42.348   5.301  40.633  1.00  0.00           H  \\nATOM    201 2HB  ALA A  20      41.717   5.328  38.969  1.00  0.00           H  \\nATOM    202 3HB  ALA A  20      42.299   6.829  39.725  1.00  0.00           H  \\nATOM    203  N   ALA A  21      40.691   8.186  41.729  1.00  0.00           N  \\nATOM    204  CA  ALA A  21      40.722   9.012  42.932  1.00  0.00           C  \\nATOM    205  C   ALA A  21      39.322   9.124  43.521  1.00  0.00           C  \\nATOM    206  O   ALA A  21      39.152   9.145  44.740  1.00  0.00           O  \\nATOM    207  CB  ALA A  21      41.286  10.391  42.625  1.00  0.00           C  \\nATOM    208  H   ALA A  21      40.837   8.623  41.003  1.00  0.00           H  \\nATOM    209  HA  ALA A  21      41.297   8.594  43.592  1.00  0.00           H  \\nATOM    210 1HB  ALA A  21      41.289  10.994  43.534  1.00  0.00           H  \\nATOM    211 2HB  ALA A  21      42.306  10.293  42.252  1.00  0.00           H  \\nATOM    212 3HB  ALA A  21      40.670  10.877  41.870  1.00  0.00           H  \\nATOM    213  N   ALA A  22      38.323   9.195  42.648  1.00  0.00           N  \\nATOM    214  CA  ALA A  22      36.938   9.321  43.084  1.00  0.00           C  \\nATOM    215  C   ALA A  22      36.511   8.076  43.857  1.00  0.00           C  \\nATOM    216  O   ALA A  22      35.848   8.173  44.890  1.00  0.00           O  \\nATOM    217  CB  ALA A  22      36.021   9.552  41.892  1.00  0.00           C  \\nATOM    218  H   ALA A  22      38.421   9.172  41.794  1.00  0.00           H  \\nATOM    219  HA  ALA A  22      36.859  10.086  43.675  1.00  0.00           H  \\nATOM    220 1HB  ALA A  22      34.990   9.638  42.237  1.00  0.00           H  \\nATOM    221 2HB  ALA A  22      36.311  10.472  41.383  1.00  0.00           H  \\nATOM    222 3HB  ALA A  22      36.104   8.715  41.202  1.00  0.00           H  \\nATOM    223  N   ALA A  23      36.902   6.909  43.352  1.00  0.00           N  \\nATOM    224  CA  ALA A  23      36.628   5.647  44.030  1.00  0.00           C  \\nATOM    225  C   ALA A  23      37.340   5.612  45.373  1.00  0.00           C  \\nATOM    226  O   ALA A  23      36.759   5.221  46.384  1.00  0.00           O  \\nATOM    227  CB  ALA A  23      37.054   4.468  43.168  1.00  0.00           C  \\nATOM    228  H   ALA A  23      37.331   6.822  42.612  1.00  0.00           H  \\nATOM    229  HA  ALA A  23      35.675   5.569  44.189  1.00  0.00           H  \\nATOM    230 1HB  ALA A  23      36.851   3.537  43.698  1.00  0.00           H  \\nATOM    231 2HB  ALA A  23      36.496   4.481  42.231  1.00  0.00           H  \\nATOM    232 3HB  ALA A  23      38.120   4.539  42.956  1.00  0.00           H  \\nATOM    233  N   ALA A  24      38.605   6.022  45.371  1.00  0.00           N  \\nATOM    234  CA  ALA A  24      39.387   6.108  46.596  1.00  0.00           C  \\nATOM    235  C   ALA A  24      38.691   7.053  47.566  1.00  0.00           C  \\nATOM    236  O   ALA A  24      38.549   6.755  48.751  1.00  0.00           O  \\nATOM    237  CB  ALA A  24      40.805   6.577  46.305  1.00  0.00           C  \\nATOM    238  H   ALA A  24      39.037   6.257  44.665  1.00  0.00           H  \\nATOM    239  HA  ALA A  24      39.442   5.231  47.008  1.00  0.00           H  \\nATOM    240 1HB  ALA A  24      41.363   6.650  47.238  1.00  0.00           H  \\nATOM    241 2HB  ALA A  24      41.296   5.863  45.643  1.00  0.00           H  \\nATOM    242 3HB  ALA A  24      40.773   7.554  45.825  1.00  0.00           H  \\nATOM    243  N   ALA A  25      38.241   8.188  47.041  1.00  0.00           N  \\nATOM    244  CA  ALA A  25      37.571   9.202  47.844  1.00  0.00           C  \\nATOM    245  C   ALA A  25      36.260   8.661  48.404  1.00  0.00           C  \\nATOM    246  O   ALA A  25      35.915   8.916  49.558  1.00  0.00           O  \\nATOM    247  CB  ALA A  25      37.322  10.457  47.020  1.00  0.00           C  \\nATOM    248  H   ALA A  25      38.314   8.397  46.210  1.00  0.00           H  \\nATOM    249  HA  ALA A  25      38.141   9.445  48.591  1.00  0.00           H  \\nATOM    250 1HB  ALA A  25      36.817  11.202  47.635  1.00  0.00           H  \\nATOM    251 2HB  ALA A  25      38.274  10.858  46.672  1.00  0.00           H  \\nATOM    252 3HB  ALA A  25      36.698  10.211  46.162  1.00  0.00           H  \\nATOM    253  N   ALA A  26      35.542   7.901  47.583  1.00  0.00           N  \\nATOM    254  CA  ALA A  26      34.261   7.331  47.988  1.00  0.00           C  \\nATOM    255  C   ALA A  26      34.442   6.235  49.032  1.00  0.00           C  \\nATOM    256  O   ALA A  26      33.568   6.018  49.871  1.00  0.00           O  \\nATOM    257  CB  ALA A  26      33.519   6.787  46.776  1.00  0.00           C  \\nATOM    258  H   ALA A  26      35.776   7.699  46.781  1.00  0.00           H  \\nATOM    259  HA  ALA A  26      33.715   8.030  48.380  1.00  0.00           H  \\nATOM    260 1HB  ALA A  26      32.564   6.367  47.093  1.00  0.00           H  \\nATOM    261 2HB  ALA A  26      33.342   7.594  46.065  1.00  0.00           H  \\nATOM    262 3HB  ALA A  26      34.117   6.010  46.302  1.00  0.00           H  \\nATOM    263  N   ALA A  27      35.574   5.540  48.974  1.00  0.00           N  \\nATOM    264  CA  ALA A  27      35.886   4.526  49.973  1.00  0.00           C  \\nATOM    265  C   ALA A  27      36.041   5.168  51.344  1.00  0.00           C  \\nATOM    266  O   ALA A  27      35.353   4.797  52.294  1.00  0.00           O  \\nATOM    267  CB  ALA A  27      37.149   3.767  49.592  1.00  0.00           C  \\nATOM    268  H   ALA A  27      36.176   5.637  48.368  1.00  0.00           H  \\nATOM    269  HA  ALA A  27      35.156   3.888  50.018  1.00  0.00           H  \\nATOM    270 1HB  ALA A  27      37.369   3.021  50.355  1.00  0.00           H  \\nATOM    271 2HB  ALA A  27      37.001   3.272  48.632  1.00  0.00           H  \\nATOM    272 3HB  ALA A  27      37.982   4.464  49.515  1.00  0.00           H  \\nATOM    273  N   ALA A  28      36.940   6.143  51.433  1.00  0.00           N  \\nATOM    274  CA  ALA A  28      37.251   6.792  52.700  1.00  0.00           C  \\nATOM    275  C   ALA A  28      36.052   7.536  53.281  1.00  0.00           C  \\nATOM    276  O   ALA A  28      36.018   7.830  54.475  1.00  0.00           O  \\nATOM    277  CB  ALA A  28      38.422   7.747  52.521  1.00  0.00           C  \\nATOM    278  H   ALA A  28      37.389   6.449  50.766  1.00  0.00           H  \\nATOM    279  HA  ALA A  28      37.529   6.113  53.335  1.00  0.00           H  \\nATOM    280 1HB  ALA A  28      38.656   8.219  53.476  1.00  0.00           H  \\nATOM    281 2HB  ALA A  28      39.293   7.193  52.168  1.00  0.00           H  \\nATOM    282 3HB  ALA A  28      38.160   8.512  51.793  1.00  0.00           H  \\nATOM    283  N   ALA A  29      35.071   7.838  52.438  1.00  0.00           N  \\nATOM    284  CA  ALA A  29      33.891   8.566  52.885  1.00  0.00           C  \\nATOM    285  C   ALA A  29      33.050   7.696  53.813  1.00  0.00           C  \\nATOM    286  O   ALA A  29      32.438   8.198  54.755  1.00  0.00           O  \\nATOM    287  CB  ALA A  29      33.066   9.030  51.693  1.00  0.00           C  \\nATOM    288  H   ALA A  29      35.064   7.632  51.603  1.00  0.00           H  \\nATOM    289  HA  ALA A  29      34.173   9.351  53.381  1.00  0.00           H  \\nATOM    290 1HB  ALA A  29      32.186   9.567  52.047  1.00  0.00           H  \\nATOM    291 2HB  ALA A  29      33.668   9.691  51.070  1.00  0.00           H  \\nATOM    292 3HB  ALA A  29      32.753   8.166  51.109  1.00  0.00           H  \\nATOM    293  N   ALA A  30      33.025   6.393  53.542  1.00  0.00           N  \\nATOM    294  CA  ALA A  30      32.366   5.436  54.426  1.00  0.00           C  \\nATOM    295  C   ALA A  30      33.394   4.808  55.370  1.00  0.00           C  \\nATOM    296  O   ALA A  30      33.299   3.633  55.724  1.00  0.00           O  \\nATOM    297  CB  ALA A  30      31.649   4.363  53.619  1.00  0.00           C  \\nATOM    298  H   ALA A  30      33.386   6.036  52.847  1.00  0.00           H  \\nATOM    299  HA  ALA A  30      31.708   5.902  54.964  1.00  0.00           H  \\nATOM    300 1HB  ALA A  30      31.173   3.655  54.297  1.00  0.00           H  \\nATOM    301 2HB  ALA A  30      30.891   4.828  52.988  1.00  0.00           H  \\nATOM    302 3HB  ALA A  30      32.369   3.838  52.993  1.00  0.00           H  \\nATOM    303  N   ALA A  31      34.385   5.606  55.758  1.00  0.00           N  \\nATOM    304  CA  ALA A  31      35.375   5.203  56.754  1.00  0.00           C  \\nATOM    305  C   ALA A  31      35.523   6.285  57.819  1.00  0.00           C  \\nATOM    306  O   ALA A  31      36.275   6.121  58.780  1.00  0.00           O  \\nATOM    307  CB  ALA A  31      36.716   4.916  56.093  1.00  0.00           C  \\nATOM    308  H   ALA A  31      34.508   6.401  55.453  1.00  0.00           H  \\nATOM    309  HA  ALA A  31      35.073   4.391  57.189  1.00  0.00           H  \\nATOM    310 1HB  ALA A  31      37.441   4.626  56.853  1.00  0.00           H  \\nATOM    311 2HB  ALA A  31      36.600   4.106  55.373  1.00  0.00           H  \\nATOM    312 3HB  ALA A  31      37.066   5.810  55.580  1.00  0.00           H  \\nATOM    313  N   ALA A  32      34.816   7.395  57.627  1.00  0.00           N  \\nATOM    314  CA  ALA A  32      34.634   8.394  58.669  1.00  0.00           C  \\nATOM    315  C   ALA A  32      33.207   8.290  59.192  1.00  0.00           C  \\nATOM    316  O   ALA A  32      32.893   8.793  60.267  1.00  0.00           O  \\nATOM    317  CB  ALA A  32      34.924   9.793  58.144  1.00  0.00           C  \\nATOM    318  H   ALA A  32      34.426   7.593  56.886  1.00  0.00           H  \\nATOM    319  HA  ALA A  32      35.245   8.215  59.401  1.00  0.00           H  \\nATOM    320 1HB  ALA A  32      34.759  10.521  58.939  1.00  0.00           H  \\nATOM    321 2HB  ALA A  32      35.960   9.849  57.811  1.00  0.00           H  \\nATOM    322 3HB  ALA A  32      34.262  10.012  57.308  1.00  0.00           H  \\nATOM    323  N   ALA A  33      32.345   7.639  58.415  1.00  0.00           N  \\nATOM    324  CA  ALA A  33      30.984   7.360  58.849  1.00  0.00           C  \\nATOM    325  C   ALA A  33      31.002   6.279  59.929  1.00  0.00           C  \\nATOM    326  O   ALA A  33      30.091   6.197  60.753  1.00  0.00           O  \\nATOM    327  CB  ALA A  33      30.120   6.934  57.671  1.00  0.00           C  \\nATOM    328  H   ALA A  33      32.527   7.348  57.627  1.00  0.00           H  \\nATOM    329  HA  ALA A  33      30.598   8.164  59.230  1.00  0.00           H  \\nATOM    330 1HB  ALA A  33      29.109   6.721  58.020  1.00  0.00           H  \\nATOM    331 2HB  ALA A  33      30.087   7.738  56.935  1.00  0.00           H  \\nATOM    332 3HB  ALA A  33      30.542   6.041  57.214  1.00  0.00           H  \\nATOM    333  N   ALA A  34      32.050   5.457  59.915  1.00  0.00           N  \\nATOM    334  CA  ALA A  34      32.263   4.437  60.937  1.00  0.00           C  \\nATOM    335  C   ALA A  34      32.992   5.028  62.141  1.00  0.00           C  \\nATOM    336  O   ALA A  34      32.755   4.629  63.282  1.00  0.00           O  \\nATOM    337  CB  ALA A  34      33.043   3.263  60.365  1.00  0.00           C  \\nATOM    338  H   ALA A  34      32.663   5.473  59.312  1.00  0.00           H  \\nATOM    339  HA  ALA A  34      31.404   4.099  61.237  1.00  0.00           H  \\nATOM    340 1HB  ALA A  34      33.197   2.514  61.142  1.00  0.00           H  \\nATOM    341 2HB  ALA A  34      32.483   2.821  59.541  1.00  0.00           H  \\nATOM    342 3HB  ALA A  34      34.009   3.611  60.001  1.00  0.00           H  \\nATOM    343  N   ALA A  35      33.881   5.979  61.868  1.00  0.00           N  \\nATOM    344  CA  ALA A  35      34.660   6.654  62.902  1.00  0.00           C  \\nATOM    345  C   ALA A  35      34.615   8.169  62.697  1.00  0.00           C  \\nATOM    346  O   ALA A  35      35.586   8.761  62.224  1.00  0.00           O  \\nATOM    347  CB  ALA A  35      36.098   6.155  62.899  1.00  0.00           C  \\nATOM    348  H   ALA A  35      34.055   6.257  61.073  1.00  0.00           H  \\nATOM    349  HA  ALA A  35      34.278   6.455  63.771  1.00  0.00           H  \\nATOM    350 1HB  ALA A  35      36.664   6.674  63.673  1.00  0.00           H  \\nATOM    351 2HB  ALA A  35      36.111   5.083  63.097  1.00  0.00           H  \\nATOM    352 3HB  ALA A  35      36.548   6.351  61.927  1.00  0.00           H  \\nATOM    353  N   ALA A  36      33.480   8.802  63.034  1.00  0.00           N  \\nATOM    354  CA  ALA A  36      33.349  10.257  62.895  1.00  0.00           C  \\nATOM    355  C   ALA A  36      34.242  11.024  63.870  1.00  0.00           C  \\nATOM    356  O   ALA A  36      33.740  11.731  64.744  1.00  0.00           O  \\nATOM    357  CB  ALA A  36      31.897  10.670  63.088  1.00  0.00           C  \\nATOM    358  H   ALA A  36      32.699   8.270  63.390  1.00  0.00           H  \\nATOM    359  HA  ALA A  36      33.552  10.530  61.987  1.00  0.00           H  \\nATOM    360 1HB  ALA A  36      31.808  11.751  62.973  1.00  0.00           H  \\nATOM    361 2HB  ALA A  36      31.276  10.174  62.342  1.00  0.00           H  \\nATOM    362 3HB  ALA A  36      31.567  10.383  64.085  1.00  0.00           H  \\nATOM    363  N   ALA A  37      35.554  10.871  63.721  1.00  0.00           N  \\nATOM    364  CA  ALA A  37      36.512  11.566  64.574  1.00  0.00           C  \\nATOM    365  C   ALA A  37      37.551  12.308  63.743  1.00  0.00           C  \\nATOM    366  O   ALA A  37      37.563  12.224  62.515  1.00  0.00           O  \\nATOM    367  CB  ALA A  37      37.192  10.584  65.517  1.00  0.00           C  \\nATOM    368  H   ALA A  37      35.918  10.367  63.127  1.00  0.00           H  \\nATOM    369  HA  ALA A  37      36.038  12.219  65.113  1.00  0.00           H  \\nATOM    370 1HB  ALA A  37      37.904  11.118  66.146  1.00  0.00           H  \\nATOM    371 2HB  ALA A  37      36.442  10.104  66.145  1.00  0.00           H  \\nATOM    372 3HB  ALA A  37      37.717   9.827  64.937  1.00  0.00           H  \\nATOM    373  N   ALA A  38      38.428  13.027  64.432  1.00  0.00           N  \\nATOM    374  CA  ALA A  38      39.416  13.881  63.788  1.00  0.00           C  \\nATOM    375  C   ALA A  38      40.503  13.079  63.080  1.00  0.00           C  \\nATOM    376  O   ALA A  38      40.668  11.885  63.324  1.00  0.00           O  \\nATOM    377  CB  ALA A  38      40.040  14.818  64.812  1.00  0.00           C  \\nATOM    378  H   ALA A  38      38.471  13.036  65.291  1.00  0.00           H  \\nATOM    379  HA  ALA A  38      38.971  14.432  63.125  1.00  0.00           H  \\nATOM    380 1HB  ALA A  38      40.770  15.461  64.320  1.00  0.00           H  \\nATOM    381 2HB  ALA A  38      39.262  15.432  65.265  1.00  0.00           H  \\nATOM    382 3HB  ALA A  38      40.536  14.232  65.585  1.00  0.00           H  \\nATOM    383  N   ALA A  39      41.166  13.758  62.155  1.00  0.00           N  \\nATOM    384  CA  ALA A  39      42.319  13.244  61.409  1.00  0.00           C  \\nATOM    385  C   ALA A  39      42.016  12.100  60.463  1.00  0.00           C  \\nATOM    386  O   ALA A  39      42.807  11.802  59.618  1.00  0.00           O  \\nATOM    387  CB  ALA A  39      43.402  12.808  62.385  1.00  0.00           C  \\nATOM    388  H   ALA A  39      40.955  14.558  61.929  1.00  0.00           H  \\nATOM    389  HA  ALA A  39      42.657  13.974  60.852  1.00  0.00           H  \\nATOM    390 1HB  ALA A  39      44.269  12.450  61.830  1.00  0.00           H  \\nATOM    391 2HB  ALA A  39      43.693  13.655  63.007  1.00  0.00           H  \\nATOM    392 3HB  ALA A  39      43.021  12.008  63.018  1.00  0.00           H  \\nATOM    393  N   ALA A  40      40.882  11.456  60.610  1.00  0.00           N  \\nATOM    394  CA  ALA A  40      40.291  10.669  59.535  1.00  0.00           C  \\nATOM    395  C   ALA A  40      39.620  11.649  58.582  1.00  0.00           C  \\nATOM    396  O   ALA A  40      39.622  11.462  57.366  1.00  0.00           O  \\nATOM    397  CB  ALA A  40      39.298   9.647  60.070  1.00  0.00           C  \\nATOM    398  H   ALA A  40      40.461  11.408  61.359  1.00  0.00           H  \\nATOM    399  HA  ALA A  40      40.989  10.196  59.054  1.00  0.00           H  \\nATOM    400 1HB  ALA A  40      38.851   9.104  59.238  1.00  0.00           H  \\nATOM    401 2HB  ALA A  40      39.815   8.947  60.726  1.00  0.00           H  \\nATOM    402 3HB  ALA A  40      38.517  10.159  60.630  1.00  0.00           H  \\nATOM    403  N   ALA A  41      39.041  12.696  59.162  1.00  0.00           N  \\nATOM    404  CA  ALA A  41      38.532  13.831  58.404  1.00  0.00           C  \\nATOM    405  C   ALA A  41      39.668  14.464  57.609  1.00  0.00           C  \\nATOM    406  O   ALA A  41      39.486  14.863  56.459  1.00  0.00           O  \\nATOM    407  CB  ALA A  41      37.885  14.852  59.328  1.00  0.00           C  \\nATOM    408  H   ALA A  41      38.929  12.772  60.011  1.00  0.00           H  \\nATOM    409  HA  ALA A  41      37.858  13.522  57.779  1.00  0.00           H  \\nATOM    410 1HB  ALA A  41      37.524  15.697  58.741  1.00  0.00           H  \\nATOM    411 2HB  ALA A  41      37.048  14.390  59.852  1.00  0.00           H  \\nATOM    412 3HB  ALA A  41      38.618  15.202  60.053  1.00  0.00           H  \\nATOM    413  N   ALA A  42      40.845  14.537  58.224  1.00  0.00           N  \\nATOM    414  CA  ALA A  42      42.022  15.105  57.573  1.00  0.00           C  \\nATOM    415  C   ALA A  42      42.475  14.215  56.418  1.00  0.00           C  \\nATOM    416  O   ALA A  42      42.906  14.714  55.380  1.00  0.00           O  \\nATOM    417  CB  ALA A  42      43.150  15.292  58.577  1.00  0.00           C  \\nATOM    418  H   ALA A  42      40.989  14.262  59.026  1.00  0.00           H  \\nATOM    419  HA  ALA A  42      41.794  15.975  57.211  1.00  0.00           H  \\nATOM    420 1HB  ALA A  42      44.021  15.711  58.072  1.00  0.00           H  \\nATOM    421 2HB  ALA A  42      42.826  15.971  59.366  1.00  0.00           H  \\nATOM    422 3HB  ALA A  42      43.412  14.330  59.013  1.00  0.00           H  \\nATOM    423  N   ALA A  43      42.373  12.901  56.603  1.00  0.00           N  \\nATOM    424  CA  ALA A  43      42.744  11.950  55.557  1.00  0.00           C  \\nATOM    425  C   ALA A  43      41.899  12.158  54.307  1.00  0.00           C  \\nATOM    426  O   ALA A  43      42.387  12.017  53.190  1.00  0.00           O  \\nATOM    427  CB  ALA A  43      42.601  10.521  56.061  1.00  0.00           C  \\nATOM    428  H   ALA A  43      42.091  12.533  57.327  1.00  0.00           H  \\nATOM    429  HA  ALA A  43      43.674  12.088  55.319  1.00  0.00           H  \\nATOM    430 1HB  ALA A  43      42.878   9.826  55.268  1.00  0.00           H  \\nATOM    431 2HB  ALA A  43      43.255  10.371  56.919  1.00  0.00           H  \\nATOM    432 3HB  ALA A  43      41.568  10.342  56.356  1.00  0.00           H  \\nATOM    433  N   ALA A  44      40.629  12.490  54.507  1.00  0.00           N  \\nATOM    434  CA  ALA A  44      39.711  12.720  53.398  1.00  0.00           C  \\nATOM    435  C   ALA A  44      40.003  14.050  52.719  1.00  0.00           C  \\nATOM    436  O   ALA A  44      40.039  14.130  51.491  1.00  0.00           O  \\nATOM    437  CB  ALA A  44      38.269  12.677  53.882  1.00  0.00           C  \\nATOM    438  H   ALA A  44      40.271  12.589  55.283  1.00  0.00           H  \\nATOM    439  HA  ALA A  44      39.825  12.015  52.741  1.00  0.00           H  \\nATOM    440 1HB  ALA A  44      37.598  12.854  53.042  1.00  0.00           H  \\nATOM    441 2HB  ALA A  44      38.059  11.699  54.315  1.00  0.00           H  \\nATOM    442 3HB  ALA A  44      38.116  13.447  54.637  1.00  0.00           H  \\nATOM    443  N   ALA A  45      40.211  15.089  53.522  1.00  0.00           N  \\nATOM    444  CA  ALA A  45      40.538  16.406  52.990  1.00  0.00           C  \\nATOM    445  C   ALA A  45      41.757  16.326  52.082  1.00  0.00           C  \\nATOM    446  O   ALA A  45      41.810  16.994  51.054  1.00  0.00           O  \\nATOM    447  CB  ALA A  45      40.780  17.395  54.121  1.00  0.00           C  \\nATOM    448  H   ALA A  45      40.169  15.057  54.381  1.00  0.00           H  \\nATOM    449  HA  ALA A  45      39.790  16.729  52.463  1.00  0.00           H  \\nATOM    450 1HB  ALA A  45      41.029  18.371  53.704  1.00  0.00           H  \\nATOM    451 2HB  ALA A  45      39.879  17.480  54.729  1.00  0.00           H  \\nATOM    452 3HB  ALA A  45      41.604  17.045  54.740  1.00  0.00           H  \\nATOM    453  N   ALA A  46      42.727  15.497  52.458  1.00  0.00           N  \\nATOM    454  CA  ALA A  46      43.905  15.287  51.625  1.00  0.00           C  \\nATOM    455  C   ALA A  46      43.507  14.737  50.260  1.00  0.00           C  \\nATOM    456  O   ALA A  46      44.031  15.165  49.233  1.00  0.00           O  \\nATOM    457  CB  ALA A  46      44.884  14.347  52.313  1.00  0.00           C  \\nATOM    458  H   ALA A  46      42.726  15.045  53.189  1.00  0.00           H  \\nATOM    459  HA  ALA A  46      44.353  16.137  51.489  1.00  0.00           H  \\nATOM    460 1HB  ALA A  46      45.755  14.199  51.675  1.00  0.00           H  \\nATOM    461 2HB  ALA A  46      45.199  14.781  53.262  1.00  0.00           H  \\nATOM    462 3HB  ALA A  46      44.401  13.389  52.496  1.00  0.00           H  \\nATOM    463  N   ALA A  47      42.575  13.789  50.257  1.00  0.00           N  \\nATOM    464  CA  ALA A  47      42.166  13.124  49.026  1.00  0.00           C  \\nATOM    465  C   ALA A  47      41.228  14.016  48.218  1.00  0.00           C  \\nATOM    466  O   ALA A  47      41.170  13.919  46.992  1.00  0.00           O  \\nATOM    467  CB  ALA A  47      41.499  11.792  49.337  1.00  0.00           C  \\nATOM    468  H   ALA A  47      42.163  13.512  50.959  1.00  0.00           H  \\nATOM    469  HA  ALA A  47      42.952  12.948  48.486  1.00  0.00           H  \\nATOM    470 1HB  ALA A  47      41.197  11.310  48.406  1.00  0.00           H  \\nATOM    471 2HB  ALA A  47      42.201  11.148  49.867  1.00  0.00           H  \\nATOM    472 3HB  ALA A  47      40.622  11.961  49.959  1.00  0.00           H  \\nATOM    473  N   ALA A  48      40.498  14.887  48.908  1.00  0.00           N  \\nATOM    474  CA  ALA A  48      39.616  15.837  48.240  1.00  0.00           C  \\nATOM    475  C   ALA A  48      40.446  16.839  47.437  1.00  0.00           C  \\nATOM    476  O   ALA A  48      40.168  17.084  46.262  1.00  0.00           O  \\nATOM    477  CB  ALA A  48      38.734  16.555  49.251  1.00  0.00           C  \\nATOM    478  H   ALA A  48      40.496  14.947  49.766  1.00  0.00           H  \\nATOM    479  HA  ALA A  48      39.044  15.357  47.621  1.00  0.00           H  \\nATOM    480 1HB  ALA A  48      38.091  17.266  48.733  1.00  0.00           H  \\nATOM    481 2HB  ALA A  48      38.119  15.826  49.779  1.00  0.00           H  \\nATOM    482 3HB  ALA A  48      39.360  17.087  49.966  1.00  0.00           H  \\nATOM    483  N   ALA A  49      41.475  17.402  48.063  1.00  0.00           N  \\nATOM    484  CA  ALA A  49      42.346  18.354  47.381  1.00  0.00           C  \\nATOM    485  C   ALA A  49      43.059  17.662  46.227  1.00  0.00           C  \\nATOM    486  O   ALA A  49      43.298  18.263  45.182  1.00  0.00           O  \\nATOM    487  CB  ALA A  49      43.352  18.957  48.350  1.00  0.00           C  \\nATOM    488  H   ALA A  49      41.690  17.251  48.882  1.00  0.00           H  \\nATOM    489  HA  ALA A  49      41.807  19.073  47.016  1.00  0.00           H  \\nATOM    490 1HB  ALA A  49      43.999  19.654  47.816  1.00  0.00           H  \\nATOM    491 2HB  ALA A  49      42.822  19.488  49.141  1.00  0.00           H  \\nATOM    492 3HB  ALA A  49      43.956  18.164  48.788  1.00  0.00           H  \\nATOM    493  N   ALA A  50      43.396  16.393  46.427  1.00  0.00           N  \\nATOM    494  CA  ALA A  50      44.003  15.586  45.379  1.00  0.00           C  \\nATOM    495  C   ALA A  50      43.038  15.459  44.211  1.00  0.00           C  \\nATOM    496  O   ALA A  50      43.424  15.625  43.055  1.00  0.00           O  \\nATOM    497  CB  ALA A  50      44.392  14.213  45.909  1.00  0.00           C  \\nATOM    498  H   ALA A  50      43.281  15.974  47.169  1.00  0.00           H  \\nATOM    499  HA  ALA A  50      44.809  16.024  45.063  1.00  0.00           H  \\nATOM    500 1HB  ALA A  50      44.830  13.624  45.104  1.00  0.00           H  \\nATOM    501 2HB  ALA A  50      45.118  14.326  46.714  1.00  0.00           H  \\nATOM    502 3HB  ALA A  50      43.506  13.706  46.289  1.00  0.00           H  \\nATOM    503  N   ALA A  51      41.778  15.167  44.517  1.00  0.00           N  \\nATOM    504  CA  ALA A  51      40.758  15.040  43.485  1.00  0.00           C  \\nATOM    505  C   ALA A  51      40.578  16.367  42.750  1.00  0.00           C  \\nATOM    506  O   ALA A  51      40.489  16.403  41.523  1.00  0.00           O  \\nATOM    507  CB  ALA A  51      39.440  14.579  44.091  1.00  0.00           C  \\nATOM    508  H   ALA A  51      41.488  15.039  45.317  1.00  0.00           H  \\nATOM    509  HA  ALA A  51      41.041  14.374  42.839  1.00  0.00           H  \\nATOM    510 1HB  ALA A  51      38.689  14.495  43.306  1.00  0.00           H  \\nATOM    511 2HB  ALA A  51      39.578  13.609  44.567  1.00  0.00           H  \\nATOM    512 3HB  ALA A  51      39.109  15.303  44.833  1.00  0.00           H  \\nATOM    513  N   ALA A  52      40.536  17.458  43.510  1.00  0.00           N  \\nATOM    514  CA  ALA A  52      40.361  18.784  42.933  1.00  0.00           C  \\nATOM    515  C   ALA A  52      41.565  19.158  42.079  1.00  0.00           C  \\nATOM    516  O   ALA A  52      41.408  19.694  40.985  1.00  0.00           O  \\nATOM    517  CB  ALA A  52      40.141  19.818  44.027  1.00  0.00           C  \\nATOM    518  H   ALA A  52      40.606  17.455  44.367  1.00  0.00           H  \\nATOM    519  HA  ALA A  52      39.579  18.777  42.359  1.00  0.00           H  \\nATOM    520 1HB  ALA A  52      40.018  20.803  43.578  1.00  0.00           H  \\nATOM    521 2HB  ALA A  52      39.246  19.563  44.595  1.00  0.00           H  \\nATOM    522 3HB  ALA A  52      41.002  19.830  44.695  1.00  0.00           H  \\nATOM    523  N   ALA A  53      42.764  18.868  42.573  1.00  0.00           N  \\nATOM    524  CA  ALA A  53      43.977  19.138  41.813  1.00  0.00           C  \\nATOM    525  C   ALA A  53      43.941  18.386  40.489  1.00  0.00           C  \\nATOM    526  O   ALA A  53      44.317  18.925  39.449  1.00  0.00           O  \\nATOM    527  CB  ALA A  53      45.211  18.753  42.615  1.00  0.00           C  \\nATOM    528  H   ALA A  53      42.901  18.516  43.346  1.00  0.00           H  \\nATOM    529  HA  ALA A  53      44.027  20.087  41.620  1.00  0.00           H  \\nATOM    530 1HB  ALA A  53      46.106  18.954  42.026  1.00  0.00           H  \\nATOM    531 2HB  ALA A  53      45.244  19.337  43.535  1.00  0.00           H  \\nATOM    532 3HB  ALA A  53      45.169  17.693  42.861  1.00  0.00           H  \\nATOM    533  N   ALA A  54      43.474  17.143  40.533  1.00  0.00           N  \\nATOM    534  CA  ALA A  54      43.344  16.329  39.331  1.00  0.00           C  \\nATOM    535  C   ALA A  54      42.245  16.881  38.434  1.00  0.00           C  \\nATOM    536  O   ALA A  54      42.347  16.834  37.208  1.00  0.00           O  \\nATOM    537  CB  ALA A  54      43.059  14.879  39.694  1.00  0.00           C  \\nATOM    538  H   ALA A  54      43.223  16.745  41.253  1.00  0.00           H  \\nATOM    539  HA  ALA A  54      44.179  16.356  38.837  1.00  0.00           H  \\nATOM    540 1HB  ALA A  54      42.958  14.289  38.783  1.00  0.00           H  \\nATOM    541 2HB  ALA A  54      43.881  14.485  40.292  1.00  0.00           H  \\nATOM    542 3HB  ALA A  54      42.135  14.822  40.267  1.00  0.00           H  \\nATOM    543  N   ALA A  55      41.194  17.412  39.052  1.00  0.00           N  \\nATOM    544  CA  ALA A  55      40.093  18.014  38.309  1.00  0.00           C  \\nATOM    545  C   ALA A  55      40.558  19.258  37.554  1.00  0.00           C  \\nATOM    546  O   ALA A  55      40.268  19.415  36.368  1.00  0.00           O  \\nATOM    547  CB  ALA A  55      38.946  18.359  39.248  1.00  0.00           C  \\nATOM    548  H   ALA A  55      41.095  17.435  39.906  1.00  0.00           H  \\nATOM    549  HA  ALA A  55      39.762  17.374  37.660  1.00  0.00           H  \\nATOM    550 1HB  ALA A  55      38.132  18.808  38.679  1.00  0.00           H  \\nATOM    551 2HB  ALA A  55      38.591  17.453  39.737  1.00  0.00           H  \\nATOM    552 3HB  ALA A  55      39.294  19.065  40.001  1.00  0.00           H  \\nATOM    553  N   ALA A  56      41.281  20.137  38.241  1.00  0.00           N  \\nATOM    554  CA  ALA A  56      41.777  21.367  37.626  1.00  0.00           C  \\nATOM    555  C   ALA A  56      42.772  21.056  36.510  1.00  0.00           C  \\nATOM    556  O   ALA A  56      42.778  21.714  35.470  1.00  0.00           O  \\nATOM    557  CB  ALA A  56      42.419  22.264  38.674  1.00  0.00           C  \\nATOM    558  H   ALA A  56      41.500  20.045  39.068  1.00  0.00           H  \\nATOM    559  HA  ALA A  56      41.030  21.848  37.236  1.00  0.00           H  \\nATOM    560 1HB  ALA A  56      42.785  23.175  38.199  1.00  0.00           H  \\nATOM    561 2HB  ALA A  56      41.680  22.523  39.433  1.00  0.00           H  \\nATOM    562 3HB  ALA A  56      43.251  21.740  39.142  1.00  0.00           H  \\nATOM    563  N   ALA A  57      43.614  20.052  36.732  1.00  0.00           N  \\nATOM    564  CA  ALA A  57      44.557  19.618  35.709  1.00  0.00           C  \\nATOM    565  C   ALA A  57      43.803  19.118  34.482  1.00  0.00           C  \\nATOM    566  O   ALA A  57      44.186  19.404  33.348  1.00  0.00           O  \\nATOM    567  CB  ALA A  57      45.478  18.535  36.252  1.00  0.00           C  \\nATOM    568  H   ALA A  57      43.660  19.607  37.466  1.00  0.00           H  \\nATOM    569  HA  ALA A  57      45.106  20.372  35.441  1.00  0.00           H  \\nATOM    570 1HB  ALA A  57      46.168  18.218  35.469  1.00  0.00           H  \\nATOM    571 2HB  ALA A  57      46.043  18.927  37.097  1.00  0.00           H  \\nATOM    572 3HB  ALA A  57      44.884  17.682  36.577  1.00  0.00           H  \\nATOM    573  N   ALA A  58      42.722  18.378  34.712  1.00  0.00           N  \\nATOM    574  CA  ALA A  58      41.902  17.876  33.614  1.00  0.00           C  \\nATOM    575  C   ALA A  58      41.286  19.027  32.826  1.00  0.00           C  \\nATOM    576  O   ALA A  58      41.209  18.977  31.600  1.00  0.00           O  \\nATOM    577  CB  ALA A  58      40.814  16.952  34.141  1.00  0.00           C  \\nATOM    578  H   ALA A  58      42.442  18.152  35.494  1.00  0.00           H  \\nATOM    579  HA  ALA A  58      42.463  17.365  33.010  1.00  0.00           H  \\nATOM    580 1HB  ALA A  58      40.211  16.589  33.310  1.00  0.00           H  \\nATOM    581 2HB  ALA A  58      41.273  16.105  34.654  1.00  0.00           H  \\nATOM    582 3HB  ALA A  58      40.180  17.497  34.839  1.00  0.00           H  \\nATOM    583  N   ALA A  59      40.859  20.067  33.535  1.00  0.00           N  \\nATOM    584  CA  ALA A  59      40.285  21.249  32.896  1.00  0.00           C  \\nATOM    585  C   ALA A  59      41.344  22.073  32.166  1.00  0.00           C  \\nATOM    586  O   ALA A  59      41.051  22.717  31.157  1.00  0.00           O  \\nATOM    587  CB  ALA A  59      39.571  22.109  33.929  1.00  0.00           C  \\nATOM    588  H   ALA A  59      40.890  20.114  34.393  1.00  0.00           H  \\nATOM    589  HA  ALA A  59      39.627  20.963  32.243  1.00  0.00           H  \\nATOM    590 1HB  ALA A  59      39.142  22.983  33.440  1.00  0.00           H  \\nATOM    591 2HB  ALA A  59      38.776  21.529  34.398  1.00  0.00           H  \\nATOM    592 3HB  ALA A  59      40.282  22.430  34.688  1.00  0.00           H  \\nATOM    593  N   ALA A  60      42.572  22.059  32.676  1.00  0.00           N  \\nATOM    594  CA  ALA A  60      43.666  22.768  32.019  1.00  0.00           C  \\nATOM    595  C   ALA A  60      43.936  22.123  30.666  1.00  0.00           C  \\nATOM    596  O   ALA A  60      43.986  22.801  29.640  1.00  0.00           O  \\nATOM    597  CB  ALA A  60      44.919  22.759  32.883  1.00  0.00           C  \\nATOM    598  H   ALA A  60      42.798  21.649  33.397  1.00  0.00           H  \\nATOM    599  HA  ALA A  60      43.408  23.692  31.872  1.00  0.00           H  \\nATOM    600 1HB  ALA A  60      45.724  23.277  32.362  1.00  0.00           H  \\nATOM    601 2HB  ALA A  60      44.714  23.264  33.827  1.00  0.00           H  \\nATOM    602 3HB  ALA A  60      45.217  21.730  33.079  1.00  0.00           H  \\nATOM    603  N   ALA A  61      44.087  20.803  30.675  1.00  0.00           N  \\nATOM    604  CA  ALA A  61      44.304  20.039  29.451  1.00  0.00           C  \\nATOM    605  C   ALA A  61      43.135  20.214  28.490  1.00  0.00           C  \\nATOM    606  O   ALA A  61      43.319  20.248  27.274  1.00  0.00           O  \\nATOM    607  CB  ALA A  61      44.512  18.566  29.772  1.00  0.00           C  \\nATOM    608  H   ALA A  61      44.068  20.320  31.387  1.00  0.00           H  \\nATOM    609  HA  ALA A  61      45.107  20.362  29.012  1.00  0.00           H  \\nATOM    610 1HB  ALA A  61      44.668  18.011  28.847  1.00  0.00           H  \\nATOM    611 2HB  ALA A  61      45.385  18.453  30.415  1.00  0.00           H  \\nATOM    612 3HB  ALA A  61      43.633  18.178  30.283  1.00  0.00           H  \\nATOM    613  N   ALA A  62      41.932  20.322  29.047  1.00  0.00           N  \\nATOM    614  CA  ALA A  62      40.722  20.473  28.246  1.00  0.00           C  \\nATOM    615  C   ALA A  62      40.705  21.801  27.493  1.00  0.00           C  \\nATOM    616  O   ALA A  62      40.435  21.835  26.294  1.00  0.00           O  \\nATOM    617  CB  ALA A  62      39.489  20.353  29.130  1.00  0.00           C  \\nATOM    618  H   ALA A  62      41.789  20.312  29.894  1.00  0.00           H  \\nATOM    619  HA  ALA A  62      40.688  19.758  27.591  1.00  0.00           H  \\nATOM    620 1HB  ALA A  62      38.592  20.462  28.520  1.00  0.00           H  \\nATOM    621 2HB  ALA A  62      39.482  19.376  29.614  1.00  0.00           H  \\nATOM    622 3HB  ALA A  62      39.508  21.134  29.888  1.00  0.00           H  \\nATOM    623  N   ALA A  63      40.990  22.890  28.203  1.00  0.00           N  \\nATOM    624  CA  ALA A  63      41.009  24.224  27.602  1.00  0.00           C  \\nATOM    625  C   ALA A  63      42.148  24.354  26.597  1.00  0.00           C  \\nATOM    626  O   ALA A  63      41.986  24.945  25.528  1.00  0.00           O  \\nATOM    627  CB  ALA A  63      41.130  25.292  28.679  1.00  0.00           C  \\nATOM    628  H   ALA A  63      41.177  22.884  29.042  1.00  0.00           H  \\nATOM    629  HA  ALA A  63      40.174  24.372  27.131  1.00  0.00           H  \\nATOM    630 1HB  ALA A  63      41.146  26.278  28.214  1.00  0.00           H  \\nATOM    631 2HB  ALA A  63      40.278  25.224  29.356  1.00  0.00           H  \\nATOM    632 3HB  ALA A  63      42.051  25.140  29.239  1.00  0.00           H  \\nATOM    633  N   ALA A  64      43.304  23.807  26.957  1.00  0.00           N  \\nATOM    634  CA  ALA A  64      44.451  23.767  26.059  1.00  0.00           C  \\nATOM    635  C   ALA A  64      44.063  23.125  24.732  1.00  0.00           C  \\nATOM    636  O   ALA A  64      44.505  23.557  23.669  1.00  0.00           O  \\nATOM    637  CB  ALA A  64      45.607  23.013  26.700  1.00  0.00           C  \\nATOM    638  H   ALA A  64      43.450  23.447  27.724  1.00  0.00           H  \\nATOM    639  HA  ALA A  64      44.749  24.673  25.881  1.00  0.00           H  \\nATOM    640 1HB  ALA A  64      46.452  22.988  26.012  1.00  0.00           H  \\nATOM    641 2HB  ALA A  64      45.903  23.516  27.621  1.00  0.00           H  \\nATOM    642 3HB  ALA A  64      45.295  21.995  26.928  1.00  0.00           H  \\nATOM    643  N   ALA A  65      43.222  22.099  24.806  1.00  0.00           N  \\nATOM    644  CA  ALA A  65      42.766  21.390  23.618  1.00  0.00           C  \\nATOM    645  C   ALA A  65      41.719  22.186  22.839  1.00  0.00           C  \\nATOM    646  O   ALA A  65      41.603  22.046  21.623  1.00  0.00           O  \\nATOM    647  CB  ALA A  65      42.209  20.028  24.006  1.00  0.00           C  \\nATOM    648  H   ALA A  65      42.898  21.791  25.541  1.00  0.00           H  \\nATOM    649  HA  ALA A  65      43.524  21.241  23.031  1.00  0.00           H  \\nATOM    650 1HB  ALA A  65      41.880  19.501  23.111  1.00  0.00           H  \\nATOM    651 2HB  ALA A  65      42.984  19.447  24.506  1.00  0.00           H  \\nATOM    652 3HB  ALA A  65      41.364  20.160  24.680  1.00  0.00           H  \\nATOM    653  N   ALA A  66      40.978  23.037  23.518  1.00  0.00           N  \\nATOM    654  CA  ALA A  66      39.980  23.885  22.899  1.00  0.00           C  \\nATOM    655  C   ALA A  66      40.596  25.055  22.179  1.00  0.00           C  \\nATOM    656  O   ALA A  66      40.085  25.540  21.207  1.00  0.00           O  \\nATOM    657  CB  ALA A  66      38.990  24.380  23.943  1.00  0.00           C  \\nATOM    658  H   ALA A  66      41.049  23.144  24.363  1.00  0.00           H  \\nATOM    659  HA  ALA A  66      39.493  23.362  22.239  1.00  0.00           H  \\nATOM    660 1HB  ALA A  66      38.242  25.012  23.465  1.00  0.00           H  \\nATOM    661 2HB  ALA A  66      38.498  23.527  24.411  1.00  0.00           H  \\nATOM    662 3HB  ALA A  66      39.519  24.955  24.702  1.00  0.00           H  \\nATOM    663  N   ALA A  67      41.732  25.479  22.675  1.00  0.00           N  \\nATOM    664  CA  ALA A  67      42.504  26.506  21.987  1.00  0.00           C  \\nATOM    665  C   ALA A  67      43.009  25.969  20.651  1.00  0.00           C  \\nATOM    666  O   ALA A  67      43.238  26.729  19.710  1.00  0.00           O  \\nATOM    667  CB  ALA A  67      43.665  26.976  22.852  1.00  0.00           C  \\nATOM    668  H   ALA A  67      42.092  25.195  23.403  1.00  0.00           H  \\nATOM    669  HA  ALA A  67      41.934  27.271  21.812  1.00  0.00           H  \\nATOM    670 1HB  ALA A  67      44.234  27.735  22.315  1.00  0.00           H  \\nATOM    671 2HB  ALA A  67      43.280  27.399  23.779  1.00  0.00           H  \\nATOM    672 3HB  ALA A  67      44.313  26.131  23.080  1.00  0.00           H  \\nATOM    673  N   ALA A  68      43.175  24.651  20.576  1.00  0.00           N  \\nATOM    674  CA  ALA A  68      43.667  24.000  19.368  1.00  0.00           C  \\nATOM    675  C   ALA A  68      42.547  23.785  18.354  1.00  0.00           C  \\nATOM    676  O   ALA A  68      42.754  23.937  17.150  1.00  0.00           O  \\nATOM    677  CB  ALA A  68      44.327  22.674  19.716  1.00  0.00           C  \\nATOM    678  H   ALA A  68      43.007  24.107  21.220  1.00  0.00           H  \\nATOM    679  HA  ALA A  68      44.339  24.566  18.956  1.00  0.00           H  \\nATOM    680 1HB  ALA A  68      44.693  22.199  18.805  1.00  0.00           H  \\nATOM    681 2HB  ALA A  68      45.162  22.849  20.394  1.00  0.00           H  \\nATOM    682 3HB  ALA A  68      43.600  22.021  20.197  1.00  0.00           H  \\nATOM    683  N   ALA A  69      41.363  23.430  18.842  1.00  0.00           N  \\nATOM    684  CA  ALA A  69      40.211  23.214  17.970  1.00  0.00           C  \\nATOM    685  C   ALA A  69      39.822  24.505  17.254  1.00  0.00           C  \\nATOM    686  O   ALA A  69      39.231  24.473  16.175  1.00  0.00           O  \\nATOM    687  CB  ALA A  69      39.033  22.674  18.768  1.00  0.00           C  \\nATOM    688  H   ALA A  69      41.199  23.307  19.677  1.00  0.00           H  \\nATOM    689  HA  ALA A  69      40.443  22.555  17.297  1.00  0.00           H  \\nATOM    690 1HB  ALA A  69      38.183  22.521  18.103  1.00  0.00           H  \\nATOM    691 2HB  ALA A  69      39.310  21.724  19.227  1.00  0.00           H  \\nATOM    692 3HB  ALA A  69      38.763  23.387  19.545  1.00  0.00           H  \\nATOM    693  N   ALA A  70      40.162  25.638  17.860  1.00  0.00           N  \\nATOM    694  CA  ALA A  70      39.855  26.942  17.284  1.00  0.00           C  \\nATOM    695  C   ALA A  70      40.852  27.303  16.187  1.00  0.00           C  \\nATOM    696  O   ALA A  70      42.065  27.204  16.377  1.00  0.00           O  \\nATOM    697  OXT ALA A  70      40.458  27.689  15.122  1.00  0.00           O  \\nATOM    698  CB  ALA A  70      39.846  28.011  18.367  1.00  0.00           C  \\nATOM    699  H   ALA A  70      40.576  25.679  18.612  1.00  0.00           H  \\nATOM    700  HA  ALA A  70      38.970  26.913  16.889  1.00  0.00           H  \\nATOM    701 1HB  ALA A  70      39.616  28.979  17.920  1.00  0.00           H  \\nATOM    702 2HB  ALA A  70      39.090  27.767  19.113  1.00  0.00           H  \\nATOM    703 3HB  ALA A  70      40.825  28.055  18.842  1.00  0.00           H  \\nTER                                                                             \\nATOM    705  N   TYR B  71      35.710  36.388  19.013  1.00  0.00           N  \\nATOM    706  CA  TYR B  71      34.402  36.488  18.434  1.00  0.00           C  \\nATOM    707  C   TYR B  71      33.573  35.299  18.771  1.00  0.00           C  \\nATOM    708  O   TYR B  71      33.315  34.433  18.000  1.00  0.00           O  \\nATOM    709  CB  TYR B  71      34.548  36.577  16.921  1.00  0.00           C  \\nATOM    710  CG  TYR B  71      35.147  35.286  16.337  1.00  0.00           C  \\nATOM    711  CD1 TYR B  71      34.314  34.280  15.860  1.00  0.00           C  \\nATOM    712  CD2 TYR B  71      36.529  35.103  16.315  1.00  0.00           C  \\nATOM    713  CE1 TYR B  71      34.854  33.115  15.345  1.00  0.00           C  \\nATOM    714  CE2 TYR B  71      37.068  33.929  15.806  1.00  0.00           C  \\nATOM    715  CZ  TYR B  71      36.236  32.942  15.317  1.00  0.00           C  \\nATOM    716  OH  TYR B  71      36.770  31.778  14.802  1.00  0.00           O  \\nATOM    717 1H   TYR B  71      36.245  37.198  18.772  1.00  0.00           H  \\nATOM    718 2H   TYR B  71      35.631  36.326  20.008  1.00  0.00           H  \\nATOM    719 3H   TYR B  71      36.168  35.570  18.664  1.00  0.00           H  \\nATOM    720  HA  TYR B  71      33.961  37.264  18.826  1.00  0.00           H  \\nATOM    721 1HB  TYR B  71      33.573  36.746  16.462  1.00  0.00           H  \\nATOM    722 2HB  TYR B  71      35.196  37.413  16.657  1.00  0.00           H  \\nATOM    723  HD1 TYR B  71      33.232  34.412  15.884  1.00  0.00           H  \\nATOM    724  HD2 TYR B  71      37.187  35.883  16.697  1.00  0.00           H  \\nATOM    725  HE1 TYR B  71      34.195  32.337  14.965  1.00  0.00           H  \\nATOM    726  HE2 TYR B  71      38.149  33.789  15.789  1.00  0.00           H  \\nATOM    727  HH  TYR B  71      37.701  31.720  15.032  1.00  0.00           H  \\nATOM    728  N   LYS B  72      33.117  35.298  19.982  1.00  0.00           N  \\nATOM    729  CA  LYS B  72      32.478  34.151  20.550  1.00  0.00           C  \\nATOM    730  C   LYS B  72      33.634  33.262  20.770  1.00  0.00           C  \\nATOM    731  O   LYS B  72      34.157  33.247  21.839  1.00  0.00           O  \\nATOM    732  CB  LYS B  72      31.342  33.622  19.660  1.00  0.00           C  \\nATOM    733  CG  LYS B  72      30.383  32.756  20.373  1.00  0.00           C  \\nATOM    734  CD  LYS B  72      29.266  32.286  19.437  1.00  0.00           C  \\nATOM    735  CE  LYS B  72      28.208  31.460  20.162  1.00  0.00           C  \\nATOM    736  NZ  LYS B  72      27.102  31.051  19.238  1.00  0.00           N  \\nATOM    737  H   LYS B  72      33.183  35.965  20.514  1.00  0.00           H  \\nATOM    738  HA  LYS B  72      32.084  34.379  21.413  1.00  0.00           H  \\nATOM    739 1HB  LYS B  72      30.799  34.449  19.203  1.00  0.00           H  \\nATOM    740 2HB  LYS B  72      31.722  32.987  18.865  1.00  0.00           H  \\nATOM    741 1HG  LYS B  72      30.903  31.988  20.830  1.00  0.00           H  \\nATOM    742 2HG  LYS B  72      29.922  33.345  21.163  1.00  0.00           H  \\nATOM    743 1HD  LYS B  72      28.788  33.155  18.984  1.00  0.00           H  \\nATOM    744 2HD  LYS B  72      29.701  31.679  18.641  1.00  0.00           H  \\nATOM    745 1HE  LYS B  72      28.668  30.567  20.579  1.00  0.00           H  \\nATOM    746 2HE  LYS B  72      27.789  32.050  20.976  1.00  0.00           H  \\nATOM    747 1HZ  LYS B  72      26.419  30.508  19.746  1.00  0.00           H  \\nATOM    748 2HZ  LYS B  72      26.659  31.876  18.856  1.00  0.00           H  \\nATOM    749 3HZ  LYS B  72      27.479  30.493  18.483  1.00  0.00           H  \\nATOM    750  N   ILE B  73      34.079  32.584  19.742  1.00  0.00           N  \\nATOM    751  CA  ILE B  73      35.171  31.658  19.903  1.00  0.00           C  \\nATOM    752  C   ILE B  73      36.358  32.336  20.561  1.00  0.00           C  \\nATOM    753  O   ILE B  73      36.745  31.991  21.645  1.00  0.00           O  \\nATOM    754  CB  ILE B  73      35.596  30.967  18.592  1.00  0.00           C  \\nATOM    755  CG1 ILE B  73      34.478  30.035  18.128  1.00  0.00           C  \\nATOM    756  CG2 ILE B  73      36.880  30.171  18.804  1.00  0.00           C  \\nATOM    757  CD1 ILE B  73      34.646  29.471  16.756  1.00  0.00           C  \\nATOM    758  H   ILE B  73      33.759  32.648  18.951  1.00  0.00           H  \\nATOM    759  HA  ILE B  73      34.887  30.943  20.499  1.00  0.00           H  \\nATOM    760  HB  ILE B  73      35.749  31.712  17.827  1.00  0.00           H  \\nATOM    761 1HG1 ILE B  73      34.394  29.207  18.836  1.00  0.00           H  \\nATOM    762 2HG1 ILE B  73      33.554  30.597  18.125  1.00  0.00           H  \\nATOM    763 1HG2 ILE B  73      37.160  29.688  17.868  1.00  0.00           H  \\nATOM    764 2HG2 ILE B  73      37.683  30.830  19.122  1.00  0.00           H  \\nATOM    765 3HG2 ILE B  73      36.712  29.410  19.572  1.00  0.00           H  \\nATOM    766 1HD1 ILE B  73      33.788  28.831  16.521  1.00  0.00           H  \\nATOM    767 2HD1 ILE B  73      34.702  30.275  16.035  1.00  0.00           H  \\nATOM    768 3HD1 ILE B  73      35.560  28.884  16.711  1.00  0.00           H  \\nATOM    769  N   LYS B  74      36.874  33.357  19.918  1.00  0.00           N  \\nATOM    770  CA  LYS B  74      37.966  34.129  20.458  1.00  0.00           C  \\nATOM    771  C   LYS B  74      37.682  34.728  21.827  1.00  0.00           C  \\nATOM    772  O   LYS B  74      38.529  34.736  22.665  1.00  0.00           O  \\nATOM    773  CB  LYS B  74      38.585  35.153  19.492  1.00  0.00           C  \\nATOM    774  CG  LYS B  74      39.826  35.837  20.036  1.00  0.00           C  \\nATOM    775  CD  LYS B  74      40.435  36.780  18.997  1.00  0.00           C  \\nATOM    776  CE  LYS B  74      41.659  37.502  19.546  1.00  0.00           C  \\nATOM    777  NZ  LYS B  74      42.255  38.425  18.541  1.00  0.00           N  \\nATOM    778  H   LYS B  74      36.597  33.608  19.149  1.00  0.00           H  \\nATOM    779  HA  LYS B  74      38.727  33.530  20.568  1.00  0.00           H  \\nATOM    780 1HB  LYS B  74      38.824  34.682  18.537  1.00  0.00           H  \\nATOM    781 2HB  LYS B  74      37.999  35.949  19.324  1.00  0.00           H  \\nATOM    782 1HG  LYS B  74      39.562  36.410  20.928  1.00  0.00           H  \\nATOM    783 2HG  LYS B  74      40.565  35.086  20.315  1.00  0.00           H  \\nATOM    784 1HD  LYS B  74      40.728  36.209  18.115  1.00  0.00           H  \\nATOM    785 2HD  LYS B  74      39.695  37.522  18.698  1.00  0.00           H  \\nATOM    786 1HE  LYS B  74      41.369  38.076  20.425  1.00  0.00           H  \\nATOM    787 2HE  LYS B  74      42.407  36.765  19.837  1.00  0.00           H  \\nATOM    788 1HZ  LYS B  74      41.570  39.119  18.274  1.00  0.00           H  \\nATOM    789 2HZ  LYS B  74      43.061  38.885  18.940  1.00  0.00           H  \\nATOM    790 3HZ  LYS B  74      42.536  37.900  17.725  1.00  0.00           H  \\nATOM    791  N   GLU B  75      36.494  35.236  22.053  1.00  0.00           N  \\nATOM    792  CA  GLU B  75      36.195  35.897  23.320  1.00  0.00           C  \\nATOM    793  C   GLU B  75      35.834  34.904  24.422  1.00  0.00           C  \\nATOM    794  O   GLU B  75      36.211  35.100  25.576  1.00  0.00           O  \\nATOM    795  CB  GLU B  75      35.029  36.854  23.074  1.00  0.00           C  \\nATOM    796  CG  GLU B  75      35.377  38.032  22.167  1.00  0.00           C  \\nATOM    797  CD  GLU B  75      34.164  38.748  21.626  1.00  0.00           C  \\nATOM    798  OE1 GLU B  75      33.083  38.243  21.804  1.00  0.00           O  \\nATOM    799  OE2 GLU B  75      34.327  39.758  20.992  1.00  0.00           O  \\nATOM    800  H   GLU B  75      35.837  35.220  21.498  1.00  0.00           H  \\nATOM    801  HA  GLU B  75      36.983  36.383  23.609  1.00  0.00           H  \\nATOM    802 1HB  GLU B  75      34.203  36.305  22.621  1.00  0.00           H  \\nATOM    803 2HB  GLU B  75      34.678  37.251  24.025  1.00  0.00           H  \\nATOM    804 1HG  GLU B  75      35.959  38.743  22.751  1.00  0.00           H  \\nATOM    805 2HG  GLU B  75      36.004  37.694  21.360  1.00  0.00           H  \\nATOM    806  N   THR B  76      35.111  33.866  24.054  1.00  0.00           N  \\nATOM    807  CA  THR B  76      34.785  32.804  24.978  1.00  0.00           C  \\nATOM    808  C   THR B  76      36.050  32.092  25.454  1.00  0.00           C  \\nATOM    809  O   THR B  76      36.195  31.740  26.589  1.00  0.00           O  \\nATOM    810  CB  THR B  76      33.755  31.751  24.462  1.00  0.00           C  \\nATOM    811  OG1 THR B  76      34.210  31.162  23.263  1.00  0.00           O  \\nATOM    812  CG2 THR B  76      32.367  32.349  24.258  1.00  0.00           C  \\nATOM    813  H   THR B  76      34.790  33.758  23.271  1.00  0.00           H  \\nATOM    814  HA  THR B  76      34.367  33.209  25.757  1.00  0.00           H  \\nATOM    815  HB  THR B  76      33.668  30.964  25.209  1.00  0.00           H  \\nATOM    816  HG1 THR B  76      34.608  31.847  22.702  1.00  0.00           H  \\nATOM    817 1HG2 THR B  76      31.686  31.564  23.923  1.00  0.00           H  \\nATOM    818 2HG2 THR B  76      32.012  32.759  25.203  1.00  0.00           H  \\nATOM    819 3HG2 THR B  76      32.400  33.133  23.524  1.00  0.00           H  \\nATOM    820  N   LEU B  77      36.976  31.938  24.544  1.00  0.00           N  \\nATOM    821  CA  LEU B  77      38.243  31.293  24.841  1.00  0.00           C  \\nATOM    822  C   LEU B  77      39.042  32.153  25.812  1.00  0.00           C  \\nATOM    823  O   LEU B  77      39.753  31.633  26.671  1.00  0.00           O  \\nATOM    824  CB  LEU B  77      38.996  31.042  23.534  1.00  0.00           C  \\nATOM    825  CG  LEU B  77      40.272  30.223  23.594  1.00  0.00           C  \\nATOM    826  CD1 LEU B  77      39.973  28.822  24.181  1.00  0.00           C  \\nATOM    827  CD2 LEU B  77      40.815  30.107  22.179  1.00  0.00           C  \\nATOM    828  H   LEU B  77      36.896  32.205  23.730  1.00  0.00           H  \\nATOM    829  HA  LEU B  77      38.074  30.436  25.263  1.00  0.00           H  \\nATOM    830 1HB  LEU B  77      38.320  30.519  22.859  1.00  0.00           H  \\nATOM    831 2HB  LEU B  77      39.235  32.009  23.086  1.00  0.00           H  \\nATOM    832  HG  LEU B  77      41.002  30.720  24.234  1.00  0.00           H  \\nATOM    833 1HD1 LEU B  77      39.241  28.311  23.554  1.00  0.00           H  \\nATOM    834 2HD1 LEU B  77      40.889  28.234  24.217  1.00  0.00           H  \\nATOM    835 3HD1 LEU B  77      39.574  28.932  25.186  1.00  0.00           H  \\nATOM    836 1HD2 LEU B  77      41.721  29.539  22.191  1.00  0.00           H  \\nATOM    837 2HD2 LEU B  77      40.077  29.608  21.546  1.00  0.00           H  \\nATOM    838 3HD2 LEU B  77      41.016  31.104  21.786  1.00  0.00           H  \\nATOM    839  N   LYS B  78      38.909  33.471  25.687  1.00  0.00           N  \\nATOM    840  CA  LYS B  78      39.591  34.390  26.594  1.00  0.00           C  \\nATOM    841  C   LYS B  78      38.963  34.340  27.984  1.00  0.00           C  \\nATOM    842  O   LYS B  78      39.668  34.400  28.992  1.00  0.00           O  \\nATOM    843  CB  LYS B  78      39.579  35.813  26.048  1.00  0.00           C  \\nATOM    844  CG  LYS B  78      40.454  36.790  26.843  1.00  0.00           C  \\nATOM    845  CD  LYS B  78      41.953  36.404  26.702  1.00  0.00           C  \\nATOM    846  CE  LYS B  78      42.883  37.437  27.342  1.00  0.00           C  \\nATOM    847  NZ  LYS B  78      44.339  37.051  27.212  1.00  0.00           N  \\nATOM    848  H   LYS B  78      38.431  33.858  25.086  1.00  0.00           H  \\nATOM    849  HA  LYS B  78      40.520  34.122  26.674  1.00  0.00           H  \\nATOM    850 1HB  LYS B  78      39.912  35.809  25.008  1.00  0.00           H  \\nATOM    851 2HB  LYS B  78      38.557  36.193  26.058  1.00  0.00           H  \\nATOM    852 1HG  LYS B  78      40.301  37.803  26.476  1.00  0.00           H  \\nATOM    853 2HG  LYS B  78      40.174  36.753  27.900  1.00  0.00           H  \\nATOM    854 1HD  LYS B  78      42.126  35.438  27.191  1.00  0.00           H  \\nATOM    855 2HD  LYS B  78      42.207  36.304  25.648  1.00  0.00           H  \\nATOM    856 1HE  LYS B  78      42.729  38.395  26.850  1.00  0.00           H  \\nATOM    857 2HE  LYS B  78      42.638  37.536  28.402  1.00  0.00           H  \\nATOM    858 1HZ  LYS B  78      44.914  37.756  27.637  1.00  0.00           H  \\nATOM    859 2HZ  LYS B  78      44.517  36.145  27.694  1.00  0.00           H  \\nATOM    860 3HZ  LYS B  78      44.584  36.959  26.241  1.00  0.00           H  \\nATOM    861  N   ARG B  79      37.638  34.239  28.035  1.00  0.00           N  \\nATOM    862  CA  ARG B  79      36.939  34.079  29.307  1.00  0.00           C  \\nATOM    863  C   ARG B  79      37.380  32.789  29.983  1.00  0.00           C  \\nATOM    864  O   ARG B  79      37.585  32.749  31.196  1.00  0.00           O  \\nATOM    865  CB  ARG B  79      35.440  34.081  29.118  1.00  0.00           C  \\nATOM    866  CG  ARG B  79      34.852  35.425  28.740  1.00  0.00           C  \\nATOM    867  CD  ARG B  79      33.411  35.310  28.542  1.00  0.00           C  \\nATOM    868  NE  ARG B  79      32.745  34.991  29.783  1.00  0.00           N  \\nATOM    869  CZ  ARG B  79      31.610  34.294  29.866  1.00  0.00           C  \\nATOM    870  NH1 ARG B  79      31.044  33.861  28.770  1.00  0.00           N  \\nATOM    871  NH2 ARG B  79      31.075  34.044  31.037  1.00  0.00           N  \\nATOM    872  H   ARG B  79      37.121  34.260  27.349  1.00  0.00           H  \\nATOM    873  HA  ARG B  79      37.161  34.821  29.890  1.00  0.00           H  \\nATOM    874 1HB  ARG B  79      35.173  33.372  28.339  1.00  0.00           H  \\nATOM    875 2HB  ARG B  79      34.957  33.753  30.035  1.00  0.00           H  \\nATOM    876 1HG  ARG B  79      35.039  36.145  29.535  1.00  0.00           H  \\nATOM    877 2HG  ARG B  79      35.305  35.776  27.816  1.00  0.00           H  \\nATOM    878 1HD  ARG B  79      33.012  36.252  28.168  1.00  0.00           H  \\nATOM    879 2HD  ARG B  79      33.204  34.514  27.825  1.00  0.00           H  \\nATOM    880  HE  ARG B  79      33.167  35.312  30.644  1.00  0.00           H  \\nATOM    881 1HH1 ARG B  79      31.464  34.058  27.879  1.00  0.00           H  \\nATOM    882 2HH1 ARG B  79      30.189  33.291  28.803  1.00  0.00           H  \\nATOM    883 1HH2 ARG B  79      31.523  34.376  31.880  1.00  0.00           H  \\nATOM    884 2HH2 ARG B  79      30.222  33.507  31.090  1.00  0.00           H  \\nATOM    885  N   LEU B  80      37.524  31.735  29.187  1.00  0.00           N  \\nATOM    886  CA  LEU B  80      37.970  30.445  29.697  1.00  0.00           C  \\nATOM    887  C   LEU B  80      39.362  30.561  30.318  1.00  0.00           C  \\nATOM    888  O   LEU B  80      39.635  29.971  31.366  1.00  0.00           O  \\nATOM    889  CB  LEU B  80      37.932  29.412  28.556  1.00  0.00           C  \\nATOM    890  CG  LEU B  80      38.245  27.973  28.918  1.00  0.00           C  \\nATOM    891  CD1 LEU B  80      37.246  27.514  29.973  1.00  0.00           C  \\nATOM    892  CD2 LEU B  80      38.125  27.100  27.642  1.00  0.00           C  \\nATOM    893  H   LEU B  80      37.369  31.742  28.341  1.00  0.00           H  \\nATOM    894  HA  LEU B  80      37.356  30.147  30.386  1.00  0.00           H  \\nATOM    895 1HB  LEU B  80      36.948  29.440  28.089  1.00  0.00           H  \\nATOM    896 2HB  LEU B  80      38.659  29.710  27.812  1.00  0.00           H  \\nATOM    897  HG  LEU B  80      39.252  27.896  29.333  1.00  0.00           H  \\nATOM    898 1HD1 LEU B  80      36.247  27.598  29.576  1.00  0.00           H  \\nATOM    899 2HD1 LEU B  80      37.447  26.480  30.246  1.00  0.00           H  \\nATOM    900 3HD1 LEU B  80      37.333  28.144  30.856  1.00  0.00           H  \\nATOM    901 1HD2 LEU B  80      38.326  26.060  27.886  1.00  0.00           H  \\nATOM    902 2HD2 LEU B  80      37.114  27.183  27.234  1.00  0.00           H  \\nATOM    903 3HD2 LEU B  80      38.839  27.439  26.897  1.00  0.00           H  \\nATOM    904  N   GLU B  81      40.231  31.335  29.672  1.00  0.00           N  \\nATOM    905  CA  GLU B  81      41.593  31.544  30.155  1.00  0.00           C  \\nATOM    906  C   GLU B  81      41.589  32.221  31.518  1.00  0.00           C  \\nATOM    907  O   GLU B  81      42.308  31.807  32.426  1.00  0.00           O  \\nATOM    908  CB  GLU B  81      42.377  32.408  29.149  1.00  0.00           C  \\nATOM    909  CG  GLU B  81      43.839  32.729  29.501  1.00  0.00           C  \\nATOM    910  CD  GLU B  81      44.493  33.612  28.426  1.00  0.00           C  \\nATOM    911  OE1 GLU B  81      44.480  33.242  27.280  1.00  0.00           O  \\nATOM    912  OE2 GLU B  81      44.967  34.685  28.755  1.00  0.00           O  \\nATOM    913  H   GLU B  81      40.053  31.756  28.943  1.00  0.00           H  \\nATOM    914  HA  GLU B  81      42.031  30.684  30.248  1.00  0.00           H  \\nATOM    915 1HB  GLU B  81      42.360  31.926  28.171  1.00  0.00           H  \\nATOM    916 2HB  GLU B  81      41.881  33.361  29.039  1.00  0.00           H  \\nATOM    917 1HG  GLU B  81      43.862  33.255  30.458  1.00  0.00           H  \\nATOM    918 2HG  GLU B  81      44.398  31.801  29.606  1.00  0.00           H  \\nATOM    919  N   ASP B  82      40.772  33.260  31.661  1.00  0.00           N  \\nATOM    920  CA  ASP B  82      40.694  33.995  32.918  1.00  0.00           C  \\nATOM    921  C   ASP B  82      40.091  33.126  34.013  1.00  0.00           C  \\nATOM    922  O   ASP B  82      40.580  33.110  35.143  1.00  0.00           O  \\nATOM    923  CB  ASP B  82      39.902  35.295  32.763  1.00  0.00           C  \\nATOM    924  CG  ASP B  82      40.691  36.378  32.014  1.00  0.00           C  \\nATOM    925  OD1 ASP B  82      41.875  36.189  31.843  1.00  0.00           O  \\nATOM    926  OD2 ASP B  82      40.120  37.380  31.647  1.00  0.00           O  \\nATOM    927  H   ASP B  82      40.251  33.559  31.045  1.00  0.00           H  \\nATOM    928  HA  ASP B  82      41.589  34.249  33.193  1.00  0.00           H  \\nATOM    929 1HB  ASP B  82      38.984  35.086  32.207  1.00  0.00           H  \\nATOM    930 2HB  ASP B  82      39.617  35.672  33.743  1.00  0.00           H  \\nATOM    931  N   SER B  83      39.029  32.403  33.672  1.00  0.00           N  \\nATOM    932  CA  SER B  83      38.361  31.532  34.631  1.00  0.00           C  \\nATOM    933  C   SER B  83      39.310  30.443  35.113  1.00  0.00           C  \\nATOM    934  O   SER B  83      39.338  30.112  36.295  1.00  0.00           O  \\nATOM    935  CB  SER B  83      37.127  30.935  34.025  1.00  0.00           C  \\nATOM    936  OG  SER B  83      36.461  30.165  34.973  1.00  0.00           O  \\nATOM    937  H   SER B  83      38.675  32.399  32.888  1.00  0.00           H  \\nATOM    938  HA  SER B  83      38.087  32.056  35.400  1.00  0.00           H  \\nATOM    939 1HB  SER B  83      36.489  31.727  33.664  1.00  0.00           H  \\nATOM    940 2HB  SER B  83      37.403  30.316  33.169  1.00  0.00           H  \\nATOM    941  HG  SER B  83      37.141  29.648  35.408  1.00  0.00           H  \\nATOM    942  N   LEU B  84      40.096  29.898  34.191  1.00  0.00           N  \\nATOM    943  CA  LEU B  84      41.055  28.856  34.529  1.00  0.00           C  \\nATOM    944  C   LEU B  84      42.145  29.407  35.446  1.00  0.00           C  \\nATOM    945  O   LEU B  84      42.570  28.737  36.390  1.00  0.00           O  \\nATOM    946  CB  LEU B  84      41.656  28.240  33.266  1.00  0.00           C  \\nATOM    947  CG  LEU B  84      42.638  27.082  33.495  1.00  0.00           C  \\nATOM    948  CD1 LEU B  84      41.946  25.929  34.253  1.00  0.00           C  \\nATOM    949  CD2 LEU B  84      43.168  26.620  32.161  1.00  0.00           C  \\nATOM    950  H   LEU B  84      40.093  30.116  33.359  1.00  0.00           H  \\nATOM    951  HA  LEU B  84      40.595  28.144  35.001  1.00  0.00           H  \\nATOM    952 1HB  LEU B  84      40.839  27.876  32.638  1.00  0.00           H  \\nATOM    953 2HB  LEU B  84      42.178  29.027  32.716  1.00  0.00           H  \\nATOM    954  HG  LEU B  84      43.444  27.429  34.093  1.00  0.00           H  \\nATOM    955 1HD1 LEU B  84      42.658  25.118  34.422  1.00  0.00           H  \\nATOM    956 2HD1 LEU B  84      41.585  26.296  35.210  1.00  0.00           H  \\nATOM    957 3HD1 LEU B  84      41.106  25.554  33.666  1.00  0.00           H  \\nATOM    958 1HD2 LEU B  84      43.878  25.831  32.322  1.00  0.00           H  \\nATOM    959 2HD2 LEU B  84      42.343  26.259  31.545  1.00  0.00           H  \\nATOM    960 3HD2 LEU B  84      43.661  27.453  31.660  1.00  0.00           H  \\nATOM    961  N   ARG B  85      42.587  30.632  35.169  1.00  0.00           N  \\nATOM    962  CA  ARG B  85      43.560  31.306  36.025  1.00  0.00           C  \\nATOM    963  C   ARG B  85      42.993  31.486  37.427  1.00  0.00           C  \\nATOM    964  O   ARG B  85      43.672  31.220  38.421  1.00  0.00           O  \\nATOM    965  CB  ARG B  85      43.916  32.665  35.443  1.00  0.00           C  \\nATOM    966  CG  ARG B  85      45.007  33.439  36.151  1.00  0.00           C  \\nATOM    967  CD  ARG B  85      45.153  34.794  35.547  1.00  0.00           C  \\nATOM    968  NE  ARG B  85      45.525  34.727  34.140  1.00  0.00           N  \\nATOM    969  CZ  ARG B  85      45.141  35.626  33.209  1.00  0.00           C  \\nATOM    970  NH1 ARG B  85      44.386  36.649  33.551  1.00  0.00           N  \\nATOM    971  NH2 ARG B  85      45.510  35.479  31.949  1.00  0.00           N  \\nATOM    972  H   ARG B  85      42.338  31.096  34.489  1.00  0.00           H  \\nATOM    973  HA  ARG B  85      44.360  30.762  36.088  1.00  0.00           H  \\nATOM    974 1HB  ARG B  85      44.208  32.545  34.399  1.00  0.00           H  \\nATOM    975 2HB  ARG B  85      43.036  33.298  35.450  1.00  0.00           H  \\nATOM    976 1HG  ARG B  85      44.746  33.553  37.205  1.00  0.00           H  \\nATOM    977 2HG  ARG B  85      45.956  32.909  36.062  1.00  0.00           H  \\nATOM    978 1HD  ARG B  85      44.201  35.322  35.621  1.00  0.00           H  \\nATOM    979 2HD  ARG B  85      45.922  35.352  36.079  1.00  0.00           H  \\nATOM    980  HE  ARG B  85      46.096  33.951  33.837  1.00  0.00           H  \\nATOM    981 1HH1 ARG B  85      44.086  36.763  34.504  1.00  0.00           H  \\nATOM    982 2HH1 ARG B  85      44.084  37.311  32.842  1.00  0.00           H  \\nATOM    983 1HH2 ARG B  85      46.075  34.693  31.667  1.00  0.00           H  \\nATOM    984 2HH2 ARG B  85      45.201  36.144  31.257  1.00  0.00           H  \\nATOM    985  N   GLU B  86      41.753  31.958  37.499  1.00  0.00           N  \\nATOM    986  CA  GLU B  86      41.075  32.134  38.777  1.00  0.00           C  \\nATOM    987  C   GLU B  86      41.061  30.815  39.546  1.00  0.00           C  \\nATOM    988  O   GLU B  86      41.308  30.780  40.750  1.00  0.00           O  \\nATOM    989  CB  GLU B  86      39.661  32.686  38.604  1.00  0.00           C  \\nATOM    990  CG  GLU B  86      38.863  32.822  39.913  1.00  0.00           C  \\nATOM    991  CD  GLU B  86      39.491  33.757  40.936  1.00  0.00           C  \\nATOM    992  OE1 GLU B  86      40.397  34.491  40.610  1.00  0.00           O  \\nATOM    993  OE2 GLU B  86      39.050  33.728  42.063  1.00  0.00           O  \\nATOM    994  H   GLU B  86      41.280  32.185  36.817  1.00  0.00           H  \\nATOM    995  HA  GLU B  86      41.555  32.790  39.307  1.00  0.00           H  \\nATOM    996 1HB  GLU B  86      39.709  33.664  38.126  1.00  0.00           H  \\nATOM    997 2HB  GLU B  86      39.098  32.034  37.945  1.00  0.00           H  \\nATOM    998 1HG  GLU B  86      37.864  33.183  39.674  1.00  0.00           H  \\nATOM    999 2HG  GLU B  86      38.761  31.830  40.356  1.00  0.00           H  \\nATOM   1000  N   LEU B  87      40.793  29.730  38.829  1.00  0.00           N  \\nATOM   1001  CA  LEU B  87      40.686  28.409  39.436  1.00  0.00           C  \\nATOM   1002  C   LEU B  87      42.037  27.927  39.957  1.00  0.00           C  \\nATOM   1003  O   LEU B  87      42.142  27.441  41.085  1.00  0.00           O  \\nATOM   1004  CB  LEU B  87      40.105  27.452  38.381  1.00  0.00           C  \\nATOM   1005  CG  LEU B  87      39.924  26.011  38.746  1.00  0.00           C  \\nATOM   1006  CD1 LEU B  87      39.017  25.894  39.942  1.00  0.00           C  \\nATOM   1007  CD2 LEU B  87      39.307  25.288  37.521  1.00  0.00           C  \\nATOM   1008  H   LEU B  87      40.667  29.732  37.978  1.00  0.00           H  \\nATOM   1009  HA  LEU B  87      40.074  28.455  40.188  1.00  0.00           H  \\nATOM   1010 1HB  LEU B  87      39.133  27.840  38.068  1.00  0.00           H  \\nATOM   1011 2HB  LEU B  87      40.755  27.472  37.518  1.00  0.00           H  \\nATOM   1012  HG  LEU B  87      40.886  25.575  38.999  1.00  0.00           H  \\nATOM   1013 1HD1 LEU B  87      38.876  24.847  40.202  1.00  0.00           H  \\nATOM   1014 2HD1 LEU B  87      39.452  26.421  40.792  1.00  0.00           H  \\nATOM   1015 3HD1 LEU B  87      38.075  26.329  39.693  1.00  0.00           H  \\nATOM   1016 1HD2 LEU B  87      39.971  25.382  36.669  1.00  0.00           H  \\nATOM   1017 2HD2 LEU B  87      39.149  24.237  37.734  1.00  0.00           H  \\nATOM   1018 3HD2 LEU B  87      38.349  25.746  37.279  1.00  0.00           H  \\nATOM   1019  N   ARG B  88      43.069  28.072  39.132  1.00  0.00           N  \\nATOM   1020  CA  ARG B  88      44.414  27.652  39.511  1.00  0.00           C  \\nATOM   1021  C   ARG B  88      44.926  28.441  40.712  1.00  0.00           C  \\nATOM   1022  O   ARG B  88      45.667  27.911  41.536  1.00  0.00           O  \\nATOM   1023  CB  ARG B  88      45.353  27.762  38.327  1.00  0.00           C  \\nATOM   1024  CG  ARG B  88      45.111  26.686  37.255  1.00  0.00           C  \\nATOM   1025  CD  ARG B  88      46.012  26.829  36.092  1.00  0.00           C  \\nATOM   1026  NE  ARG B  88      45.679  27.988  35.297  1.00  0.00           N  \\nATOM   1027  CZ  ARG B  88      46.387  28.422  34.239  1.00  0.00           C  \\nATOM   1028  NH1 ARG B  88      47.483  27.794  33.874  1.00  0.00           N  \\nATOM   1029  NH2 ARG B  88      45.983  29.480  33.566  1.00  0.00           N  \\nATOM   1030  H   ARG B  88      43.017  28.412  38.344  1.00  0.00           H  \\nATOM   1031  HA  ARG B  88      44.393  26.714  39.756  1.00  0.00           H  \\nATOM   1032 1HB  ARG B  88      45.220  28.738  37.855  1.00  0.00           H  \\nATOM   1033 2HB  ARG B  88      46.385  27.690  38.663  1.00  0.00           H  \\nATOM   1034 1HG  ARG B  88      45.280  25.704  37.691  1.00  0.00           H  \\nATOM   1035 2HG  ARG B  88      44.080  26.748  36.901  1.00  0.00           H  \\nATOM   1036 1HD  ARG B  88      47.039  26.933  36.436  1.00  0.00           H  \\nATOM   1037 2HD  ARG B  88      45.924  25.945  35.458  1.00  0.00           H  \\nATOM   1038  HE  ARG B  88      44.832  28.497  35.548  1.00  0.00           H  \\nATOM   1039 1HH1 ARG B  88      47.797  26.988  34.394  1.00  0.00           H  \\nATOM   1040 2HH1 ARG B  88      48.013  28.125  33.080  1.00  0.00           H  \\nATOM   1041 1HH2 ARG B  88      45.142  29.961  33.851  1.00  0.00           H  \\nATOM   1042 2HH2 ARG B  88      46.509  29.807  32.770  1.00  0.00           H  \\nATOM   1043  N   ARG B  89      44.521  29.704  40.817  1.00  0.00           N  \\nATOM   1044  CA  ARG B  89      44.936  30.545  41.935  1.00  0.00           C  \\nATOM   1045  C   ARG B  89      44.346  30.033  43.242  1.00  0.00           C  \\nATOM   1046  O   ARG B  89      45.046  29.917  44.248  1.00  0.00           O  \\nATOM   1047  CB  ARG B  89      44.442  31.975  41.722  1.00  0.00           C  \\nATOM   1048  CG  ARG B  89      44.833  32.977  42.795  1.00  0.00           C  \\nATOM   1049  CD  ARG B  89      44.139  34.311  42.614  1.00  0.00           C  \\nATOM   1050  NE  ARG B  89      42.678  34.245  42.877  1.00  0.00           N  \\nATOM   1051  CZ  ARG B  89      42.135  34.231  44.128  1.00  0.00           C  \\nATOM   1052  NH1 ARG B  89      42.922  34.262  45.189  1.00  0.00           N  \\nATOM   1053  NH2 ARG B  89      40.822  34.194  44.295  1.00  0.00           N  \\nATOM   1054  H   ARG B  89      44.006  30.098  40.253  1.00  0.00           H  \\nATOM   1055  HA  ARG B  89      45.903  30.517  42.011  1.00  0.00           H  \\nATOM   1056 1HB  ARG B  89      44.808  32.344  40.764  1.00  0.00           H  \\nATOM   1057 2HB  ARG B  89      43.354  31.975  41.666  1.00  0.00           H  \\nATOM   1058 1HG  ARG B  89      44.554  32.580  43.771  1.00  0.00           H  \\nATOM   1059 2HG  ARG B  89      45.910  33.141  42.763  1.00  0.00           H  \\nATOM   1060 1HD  ARG B  89      44.574  35.042  43.293  1.00  0.00           H  \\nATOM   1061 2HD  ARG B  89      44.278  34.645  41.586  1.00  0.00           H  \\nATOM   1062  HE  ARG B  89      42.020  34.244  42.069  1.00  0.00           H  \\nATOM   1063 1HH1 ARG B  89      43.925  34.289  45.076  1.00  0.00           H  \\nATOM   1064 2HH1 ARG B  89      42.518  34.251  46.117  1.00  0.00           H  \\nATOM   1065 1HH2 ARG B  89      40.198  34.162  43.470  1.00  0.00           H  \\nATOM   1066 2HH2 ARG B  89      40.430  34.184  45.220  1.00  0.00           H  \\nATOM   1067  N   ILE B  90      43.050  29.737  43.218  1.00  0.00           N  \\nATOM   1068  CA  ILE B  90      42.351  29.213  44.386  1.00  0.00           C  \\nATOM   1069  C   ILE B  90      42.933  27.861  44.779  1.00  0.00           C  \\nATOM   1070  O   ILE B  90      42.972  27.507  45.958  1.00  0.00           O  \\nATOM   1071  CB  ILE B  90      40.848  29.075  44.134  1.00  0.00           C  \\nATOM   1072  CG1 ILE B  90      40.233  30.451  43.982  1.00  0.00           C  \\nATOM   1073  CG2 ILE B  90      40.198  28.338  45.313  1.00  0.00           C  \\nATOM   1074  CD1 ILE B  90      38.849  30.409  43.454  1.00  0.00           C  \\nATOM   1075  H   ILE B  90      42.548  29.831  42.526  1.00  0.00           H  \\nATOM   1076  HA  ILE B  90      42.471  29.823  45.130  1.00  0.00           H  \\nATOM   1077  HB  ILE B  90      40.677  28.523  43.205  1.00  0.00           H  \\nATOM   1078 1HG1 ILE B  90      40.223  30.945  44.951  1.00  0.00           H  \\nATOM   1079 2HG1 ILE B  90      40.843  31.045  43.294  1.00  0.00           H  \\nATOM   1080 1HG2 ILE B  90      39.136  28.250  45.139  1.00  0.00           H  \\nATOM   1081 2HG2 ILE B  90      40.640  27.349  45.409  1.00  0.00           H  \\nATOM   1082 3HG2 ILE B  90      40.367  28.899  46.233  1.00  0.00           H  \\nATOM   1083 1HD1 ILE B  90      38.462  31.427  43.358  1.00  0.00           H  \\nATOM   1084 2HD1 ILE B  90      38.883  29.940  42.477  1.00  0.00           H  \\nATOM   1085 3HD1 ILE B  90      38.208  29.834  44.122  1.00  0.00           H  \\nATOM   1086  N   LEU B  91      43.392  27.109  43.786  1.00  0.00           N  \\nATOM   1087  CA  LEU B  91      43.991  25.809  44.047  1.00  0.00           C  \\nATOM   1088  C   LEU B  91      45.312  25.962  44.800  1.00  0.00           C  \\nATOM   1089  O   LEU B  91      45.623  25.158  45.673  1.00  0.00           O  \\nATOM   1090  CB  LEU B  91      44.228  25.034  42.751  1.00  0.00           C  \\nATOM   1091  CG  LEU B  91      44.785  23.616  42.927  1.00  0.00           C  \\nATOM   1092  CD1 LEU B  91      43.826  22.791  43.780  1.00  0.00           C  \\nATOM   1093  CD2 LEU B  91      44.965  22.996  41.568  1.00  0.00           C  \\nATOM   1094  H   LEU B  91      43.368  27.328  42.955  1.00  0.00           H  \\nATOM   1095  HA  LEU B  91      43.388  25.289  44.601  1.00  0.00           H  \\nATOM   1096 1HB  LEU B  91      43.290  24.975  42.197  1.00  0.00           H  \\nATOM   1097 2HB  LEU B  91      44.943  25.586  42.150  1.00  0.00           H  \\nATOM   1098  HG  LEU B  91      45.751  23.661  43.437  1.00  0.00           H  \\nATOM   1099 1HD1 LEU B  91      44.217  21.779  43.907  1.00  0.00           H  \\nATOM   1100 2HD1 LEU B  91      43.713  23.258  44.756  1.00  0.00           H  \\nATOM   1101 3HD1 LEU B  91      42.870  22.750  43.286  1.00  0.00           H  \\nATOM   1102 1HD2 LEU B  91      45.364  22.005  41.681  1.00  0.00           H  \\nATOM   1103 2HD2 LEU B  91      44.013  22.950  41.075  1.00  0.00           H  \\nATOM   1104 3HD2 LEU B  91      45.655  23.599  40.979  1.00  0.00           H  \\nATOM   1105  N   GLU B  92      46.086  26.990  44.459  1.00  0.00           N  \\nATOM   1106  CA  GLU B  92      47.334  27.270  45.167  1.00  0.00           C  \\nATOM   1107  C   GLU B  92      47.059  27.586  46.632  1.00  0.00           C  \\nATOM   1108  O   GLU B  92      47.791  27.152  47.518  1.00  0.00           O  \\nATOM   1109  CB  GLU B  92      48.119  28.405  44.502  1.00  0.00           C  \\nATOM   1110  CG  GLU B  92      48.755  28.035  43.151  1.00  0.00           C  \\nATOM   1111  CD  GLU B  92      49.882  27.021  43.270  1.00  0.00           C  \\nATOM   1112  OE1 GLU B  92      50.793  27.264  44.026  1.00  0.00           O  \\nATOM   1113  OE2 GLU B  92      49.827  26.008  42.607  1.00  0.00           O  \\nATOM   1114  H   GLU B  92      45.911  27.541  43.823  1.00  0.00           H  \\nATOM   1115  HA  GLU B  92      47.903  26.485  45.129  1.00  0.00           H  \\nATOM   1116 1HB  GLU B  92      47.463  29.253  44.336  1.00  0.00           H  \\nATOM   1117 2HB  GLU B  92      48.914  28.735  45.170  1.00  0.00           H  \\nATOM   1118 1HG  GLU B  92      47.983  27.617  42.511  1.00  0.00           H  \\nATOM   1119 2HG  GLU B  92      49.129  28.940  42.676  1.00  0.00           H  \\nATOM   1120  N   GLU B  93      45.999  28.350  46.878  1.00  0.00           N  \\nATOM   1121  CA  GLU B  93      45.582  28.670  48.238  1.00  0.00           C  \\nATOM   1122  C   GLU B  93      45.148  27.410  48.975  1.00  0.00           C  \\nATOM   1123  O   GLU B  93      45.513  27.195  50.130  1.00  0.00           O  \\nATOM   1124  CB  GLU B  93      44.432  29.691  48.257  1.00  0.00           C  \\nATOM   1125  CG  GLU B  93      43.964  30.089  49.679  1.00  0.00           C  \\nATOM   1126  CD  GLU B  93      42.902  31.187  49.703  1.00  0.00           C  \\nATOM   1127  OE1 GLU B  93      42.535  31.674  48.656  1.00  0.00           O  \\nATOM   1128  OE2 GLU B  93      42.469  31.537  50.776  1.00  0.00           O  \\nATOM   1129  H   GLU B  93      45.501  28.698  46.269  1.00  0.00           H  \\nATOM   1130  HA  GLU B  93      46.331  29.059  48.718  1.00  0.00           H  \\nATOM   1131 1HB  GLU B  93      44.734  30.592  47.726  1.00  0.00           H  \\nATOM   1132 2HB  GLU B  93      43.571  29.278  47.733  1.00  0.00           H  \\nATOM   1133 1HG  GLU B  93      43.556  29.202  50.168  1.00  0.00           H  \\nATOM   1134 2HG  GLU B  93      44.830  30.415  50.254  1.00  0.00           H  \\nATOM   1135  N   LEU B  94      44.360  26.583  48.295  1.00  0.00           N  \\nATOM   1136  CA  LEU B  94      43.876  25.330  48.862  1.00  0.00           C  \\nATOM   1137  C   LEU B  94      45.039  24.404  49.208  1.00  0.00           C  \\nATOM   1138  O   LEU B  94      45.004  23.690  50.209  1.00  0.00           O  \\nATOM   1139  CB  LEU B  94      42.908  24.651  47.878  1.00  0.00           C  \\nATOM   1140  CG  LEU B  94      42.280  23.347  48.345  1.00  0.00           C  \\nATOM   1141  CD1 LEU B  94      41.538  23.561  49.665  1.00  0.00           C  \\nATOM   1142  CD2 LEU B  94      41.291  22.843  47.247  1.00  0.00           C  \\nATOM   1143  H   LEU B  94      44.088  26.729  47.492  1.00  0.00           H  \\nATOM   1144  HA  LEU B  94      43.387  25.518  49.678  1.00  0.00           H  \\nATOM   1145 1HB  LEU B  94      42.126  25.346  47.604  1.00  0.00           H  \\nATOM   1146 2HB  LEU B  94      43.470  24.421  46.979  1.00  0.00           H  \\nATOM   1147  HG  LEU B  94      43.067  22.618  48.510  1.00  0.00           H  \\nATOM   1148 1HD1 LEU B  94      41.126  22.621  49.987  1.00  0.00           H  \\nATOM   1149 2HD1 LEU B  94      42.234  23.923  50.422  1.00  0.00           H  \\nATOM   1150 3HD1 LEU B  94      40.744  24.277  49.535  1.00  0.00           H  \\nATOM   1151 1HD2 LEU B  94      40.832  21.909  47.553  1.00  0.00           H  \\nATOM   1152 2HD2 LEU B  94      40.511  23.591  47.093  1.00  0.00           H  \\nATOM   1153 3HD2 LEU B  94      41.833  22.687  46.314  1.00  0.00           H  \\nATOM   1154  N   LYS B  95      46.072  24.431  48.373  1.00  0.00           N  \\nATOM   1155  CA  LYS B  95      47.251  23.596  48.574  1.00  0.00           C  \\nATOM   1156  C   LYS B  95      48.018  24.013  49.826  1.00  0.00           C  \\nATOM   1157  O   LYS B  95      48.537  23.169  50.556  1.00  0.00           O  \\nATOM   1158  CB  LYS B  95      48.090  23.600  47.296  1.00  0.00           C  \\nATOM   1159  CG  LYS B  95      49.190  22.550  47.229  1.00  0.00           C  \\nATOM   1160  CD  LYS B  95      49.746  22.477  45.794  1.00  0.00           C  \\nATOM   1161  CE  LYS B  95      50.760  21.348  45.615  1.00  0.00           C  \\nATOM   1162  NZ  LYS B  95      50.111  19.973  45.719  1.00  0.00           N  \\nATOM   1163  H   LYS B  95      46.115  24.930  47.674  1.00  0.00           H  \\nATOM   1164  HA  LYS B  95      46.972  22.674  48.684  1.00  0.00           H  \\nATOM   1165 1HB  LYS B  95      47.435  23.452  46.435  1.00  0.00           H  \\nATOM   1166 2HB  LYS B  95      48.554  24.583  47.178  1.00  0.00           H  \\nATOM   1167 1HG  LYS B  95      49.992  22.814  47.921  1.00  0.00           H  \\nATOM   1168 2HG  LYS B  95      48.787  21.577  47.511  1.00  0.00           H  \\nATOM   1169 1HD  LYS B  95      48.918  22.307  45.099  1.00  0.00           H  \\nATOM   1170 2HD  LYS B  95      50.218  23.428  45.537  1.00  0.00           H  \\nATOM   1171 1HE  LYS B  95      51.227  21.442  44.635  1.00  0.00           H  \\nATOM   1172 2HE  LYS B  95      51.528  21.434  46.383  1.00  0.00           H  \\nATOM   1173 1HZ  LYS B  95      49.689  19.870  46.623  1.00  0.00           H  \\nATOM   1174 2HZ  LYS B  95      49.370  19.829  44.988  1.00  0.00           H  \\nATOM   1175 3HZ  LYS B  95      50.814  19.264  45.596  1.00  0.00           H  \\nATOM   1176  N   GLU B  96      48.092  25.318  50.064  1.00  0.00           N  \\nATOM   1177  CA  GLU B  96      48.741  25.843  51.260  1.00  0.00           C  \\nATOM   1178  C   GLU B  96      47.933  25.548  52.517  1.00  0.00           C  \\nATOM   1179  O   GLU B  96      48.494  25.190  53.553  1.00  0.00           O  \\nATOM   1180  CB  GLU B  96      48.990  27.358  51.176  1.00  0.00           C  \\nATOM   1181  CG  GLU B  96      50.068  27.790  50.175  1.00  0.00           C  \\nATOM   1182  CD  GLU B  96      50.343  29.297  50.189  1.00  0.00           C  \\nATOM   1183  OE1 GLU B  96      49.669  30.011  50.902  1.00  0.00           O  \\nATOM   1184  OE2 GLU B  96      51.242  29.722  49.496  1.00  0.00           O  \\nATOM   1185  H   GLU B  96      47.772  25.925  49.545  1.00  0.00           H  \\nATOM   1186  HA  GLU B  96      49.610  25.421  51.357  1.00  0.00           H  \\nATOM   1187 1HB  GLU B  96      48.060  27.855  50.890  1.00  0.00           H  \\nATOM   1188 2HB  GLU B  96      49.271  27.734  52.159  1.00  0.00           H  \\nATOM   1189 1HG  GLU B  96      50.991  27.261  50.406  1.00  0.00           H  \\nATOM   1190 2HG  GLU B  96      49.754  27.492  49.177  1.00  0.00           H  \\nATOM   1191  N   MET B  97      46.617  25.711  52.425  1.00  0.00           N  \\nATOM   1192  CA  MET B  97      45.735  25.451  53.557  1.00  0.00           C  \\nATOM   1193  C   MET B  97      45.838  23.999  54.010  1.00  0.00           C  \\nATOM   1194  O   MET B  97      45.679  23.697  55.192  1.00  0.00           O  \\nATOM   1195  CB  MET B  97      44.283  25.739  53.197  1.00  0.00           C  \\nATOM   1196  CG  MET B  97      43.919  27.161  52.987  1.00  0.00           C  \\nATOM   1197  SD  MET B  97      44.065  28.158  54.457  1.00  0.00           S  \\nATOM   1198  CE  MET B  97      43.726  29.767  53.744  1.00  0.00           C  \\nATOM   1199  H   MET B  97      46.208  25.973  51.715  1.00  0.00           H  \\nATOM   1200  HA  MET B  97      45.994  26.027  54.293  1.00  0.00           H  \\nATOM   1201 1HB  MET B  97      44.012  25.183  52.298  1.00  0.00           H  \\nATOM   1202 2HB  MET B  97      43.656  25.402  53.988  1.00  0.00           H  \\nATOM   1203 1HG  MET B  97      44.534  27.601  52.227  1.00  0.00           H  \\nATOM   1204 2HG  MET B  97      42.902  27.186  52.658  1.00  0.00           H  \\nATOM   1205 1HE  MET B  97      43.771  30.529  54.522  1.00  0.00           H  \\nATOM   1206 2HE  MET B  97      44.466  29.989  52.971  1.00  0.00           H  \\nATOM   1207 3HE  MET B  97      42.732  29.769  53.292  1.00  0.00           H  \\nATOM   1208  N   LEU B  98      46.110  23.103  53.067  1.00  0.00           N  \\nATOM   1209  CA  LEU B  98      46.230  21.685  53.382  1.00  0.00           C  \\nATOM   1210  C   LEU B  98      47.504  21.417  54.178  1.00  0.00           C  \\nATOM   1211  O   LEU B  98      47.492  20.651  55.141  1.00  0.00           O  \\nATOM   1212  CB  LEU B  98      46.223  20.840  52.104  1.00  0.00           C  \\nATOM   1213  CG  LEU B  98      46.201  19.316  52.332  1.00  0.00           C  \\nATOM   1214  CD1 LEU B  98      44.911  18.956  53.106  1.00  0.00           C  \\nATOM   1215  CD2 LEU B  98      46.266  18.598  51.003  1.00  0.00           C  \\nATOM   1216  H   LEU B  98      46.229  23.291  52.236  1.00  0.00           H  \\nATOM   1217  HA  LEU B  98      45.473  21.416  53.925  1.00  0.00           H  \\nATOM   1218 1HB  LEU B  98      45.350  21.113  51.508  1.00  0.00           H  \\nATOM   1219 2HB  LEU B  98      47.118  21.082  51.524  1.00  0.00           H  \\nATOM   1220  HG  LEU B  98      47.059  19.026  52.941  1.00  0.00           H  \\nATOM   1221 1HD1 LEU B  98      44.871  17.890  53.296  1.00  0.00           H  \\nATOM   1222 2HD1 LEU B  98      44.901  19.491  54.053  1.00  0.00           H  \\nATOM   1223 3HD1 LEU B  98      44.040  19.246  52.516  1.00  0.00           H  \\nATOM   1224 1HD2 LEU B  98      46.252  17.524  51.159  1.00  0.00           H  \\nATOM   1225 2HD2 LEU B  98      45.419  18.887  50.412  1.00  0.00           H  \\nATOM   1226 3HD2 LEU B  98      47.184  18.876  50.486  1.00  0.00           H  \\nATOM   1227  N   GLU B  99      48.602  22.049  53.771  1.00  0.00           N  \\nATOM   1228  CA  GLU B  99      49.869  21.910  54.481  1.00  0.00           C  \\nATOM   1229  C   GLU B  99      49.745  22.430  55.910  1.00  0.00           C  \\nATOM   1230  O   GLU B  99      50.341  21.875  56.833  1.00  0.00           O  \\nATOM   1231  CB  GLU B  99      50.970  22.688  53.745  1.00  0.00           C  \\nATOM   1232  CG  GLU B  99      51.377  22.122  52.367  1.00  0.00           C  \\nATOM   1233  CD  GLU B  99      52.105  20.807  52.412  1.00  0.00           C  \\nATOM   1234  OE1 GLU B  99      53.092  20.723  53.103  1.00  0.00           O  \\nATOM   1235  OE2 GLU B  99      51.680  19.888  51.747  1.00  0.00           O  \\nATOM   1236  H   GLU B  99      48.639  22.565  53.084  1.00  0.00           H  \\nATOM   1237  HA  GLU B  99      50.108  20.971  54.522  1.00  0.00           H  \\nATOM   1238 1HB  GLU B  99      50.641  23.715  53.591  1.00  0.00           H  \\nATOM   1239 2HB  GLU B  99      51.862  22.724  54.368  1.00  0.00           H  \\nATOM   1240 1HG  GLU B  99      50.472  21.991  51.777  1.00  0.00           H  \\nATOM   1241 2HG  GLU B  99      51.998  22.859  51.858  1.00  0.00           H  \\nATOM   1242  N   ARG B 100      48.969  23.496  56.087  1.00  0.00           N  \\nATOM   1243  CA  ARG B 100      48.696  24.029  57.418  1.00  0.00           C  \\nATOM   1244  C   ARG B 100      47.926  23.017  58.257  1.00  0.00           C  \\nATOM   1245  O   ARG B 100      47.992  23.038  59.486  1.00  0.00           O  \\nATOM   1246  CB  ARG B 100      47.878  25.313  57.334  1.00  0.00           C  \\nATOM   1247  CG  ARG B 100      48.604  26.529  56.770  1.00  0.00           C  \\nATOM   1248  CD  ARG B 100      47.705  27.726  56.708  1.00  0.00           C  \\nATOM   1249  NE  ARG B 100      48.355  28.874  56.085  1.00  0.00           N  \\nATOM   1250  CZ  ARG B 100      49.140  29.760  56.733  1.00  0.00           C  \\nATOM   1251  NH1 ARG B 100      49.363  29.627  58.022  1.00  0.00           N  \\nATOM   1252  NH2 ARG B 100      49.686  30.765  56.070  1.00  0.00           N  \\nATOM   1253  H   ARG B 100      48.587  23.930  55.450  1.00  0.00           H  \\nATOM   1254  HA  ARG B 100      49.537  24.216  57.865  1.00  0.00           H  \\nATOM   1255 1HB  ARG B 100      47.006  25.145  56.716  1.00  0.00           H  \\nATOM   1256 2HB  ARG B 100      47.520  25.576  58.328  1.00  0.00           H  \\nATOM   1257 1HG  ARG B 100      49.456  26.768  57.402  1.00  0.00           H  \\nATOM   1258 2HG  ARG B 100      48.949  26.307  55.763  1.00  0.00           H  \\nATOM   1259 1HD  ARG B 100      46.821  27.478  56.123  1.00  0.00           H  \\nATOM   1260 2HD  ARG B 100      47.404  28.007  57.714  1.00  0.00           H  \\nATOM   1261  HE  ARG B 100      48.211  29.012  55.092  1.00  0.00           H  \\nATOM   1262 1HH1 ARG B 100      48.943  28.862  58.531  1.00  0.00           H  \\nATOM   1263 2HH1 ARG B 100      49.949  30.293  58.504  1.00  0.00           H  \\nATOM   1264 1HH2 ARG B 100      49.515  30.869  55.077  1.00  0.00           H  \\nATOM   1265 2HH2 ARG B 100      50.273  31.429  56.553  1.00  0.00           H  \\nATOM   1266  N   LEU B 101      47.192  22.135  57.586  1.00  0.00           N  \\nATOM   1267  CA  LEU B 101      46.382  21.138  58.272  1.00  0.00           C  \\nATOM   1268  C   LEU B 101      47.231  19.941  58.697  1.00  0.00           C  \\nATOM   1269  O   LEU B 101      46.868  19.214  59.620  1.00  0.00           O  \\nATOM   1270  CB  LEU B 101      45.239  20.654  57.344  1.00  0.00           C  \\nATOM   1271  CG  LEU B 101      44.181  19.715  57.940  1.00  0.00           C  \\nATOM   1272  CD1 LEU B 101      43.459  20.385  59.056  1.00  0.00           C  \\nATOM   1273  CD2 LEU B 101      43.203  19.303  56.839  1.00  0.00           C  \\nATOM   1274  H   LEU B 101      47.146  22.094  56.728  1.00  0.00           H  \\nATOM   1275  HA  LEU B 101      46.003  21.535  59.072  1.00  0.00           H  \\nATOM   1276 1HB  LEU B 101      44.734  21.528  56.930  1.00  0.00           H  \\nATOM   1277 2HB  LEU B 101      45.677  20.106  56.530  1.00  0.00           H  \\nATOM   1278  HG  LEU B 101      44.684  18.839  58.348  1.00  0.00           H  \\nATOM   1279 1HD1 LEU B 101      42.738  19.704  59.488  1.00  0.00           H  \\nATOM   1280 2HD1 LEU B 101      44.174  20.688  59.809  1.00  0.00           H  \\nATOM   1281 3HD1 LEU B 101      42.950  21.243  58.662  1.00  0.00           H  \\nATOM   1282 1HD2 LEU B 101      42.470  18.635  57.245  1.00  0.00           H  \\nATOM   1283 2HD2 LEU B 101      42.710  20.172  56.443  1.00  0.00           H  \\nATOM   1284 3HD2 LEU B 101      43.744  18.802  56.041  1.00  0.00           H  \\nATOM   1285  N   GLU B 102      48.357  19.746  58.019  1.00  0.00           N  \\nATOM   1286  CA  GLU B 102      49.255  18.639  58.327  1.00  0.00           C  \\nATOM   1287  C   GLU B 102      50.165  18.992  59.498  1.00  0.00           C  \\nATOM   1288  O   GLU B 102      50.822  18.122  60.070  1.00  0.00           O  \\nATOM   1289  CB  GLU B 102      50.067  18.220  57.098  1.00  0.00           C  \\nATOM   1290  CG  GLU B 102      50.811  16.886  57.301  1.00  0.00           C  \\nATOM   1291  CD  GLU B 102      51.425  16.308  56.043  1.00  0.00           C  \\nATOM   1292  OE1 GLU B 102      52.265  16.934  55.457  1.00  0.00           O  \\nATOM   1293  OE2 GLU B 102      51.016  15.223  55.668  1.00  0.00           O  \\nATOM   1294  H   GLU B 102      48.626  20.244  57.372  1.00  0.00           H  \\nATOM   1295  HA  GLU B 102      48.725  17.853  58.575  1.00  0.00           H  \\nATOM   1296 1HB  GLU B 102      49.404  18.125  56.236  1.00  0.00           H  \\nATOM   1297 2HB  GLU B 102      50.800  18.997  56.864  1.00  0.00           H  \\nATOM   1298 1HG  GLU B 102      51.603  17.043  58.033  1.00  0.00           H  \\nATOM   1299 2HG  GLU B 102      50.113  16.163  57.719  1.00  0.00           H  \\nATOM   1300  N   LYS B 103      50.128  20.274  59.813  1.00  0.00           N  \\nATOM   1301  CA  LYS B 103      50.838  20.829  60.920  1.00  0.00           C  \\nATOM   1302  C   LYS B 103      49.996  20.642  62.151  1.00  0.00           C  \\nATOM   1303  O   LYS B 103      50.320  19.842  63.010  1.00  0.00           O  \\nATOM   1304  CB  LYS B 103      51.182  22.299  60.697  1.00  0.00           C  \\nATOM   1305  CG  LYS B 103      52.255  22.523  59.640  1.00  0.00           C  \\nATOM   1306  CD  LYS B 103      52.559  23.997  59.442  1.00  0.00           C  \\nATOM   1307  CE  LYS B 103      53.617  24.195  58.362  1.00  0.00           C  \\nATOM   1308  NZ  LYS B 103      53.924  25.636  58.135  1.00  0.00           N  \\nATOM   1309  H   LYS B 103      49.659  20.843  59.375  1.00  0.00           H  \\nATOM   1310  HA  LYS B 103      51.691  20.367  61.033  1.00  0.00           H  \\nATOM   1311 1HB  LYS B 103      50.296  22.841  60.389  1.00  0.00           H  \\nATOM   1312 2HB  LYS B 103      51.522  22.739  61.633  1.00  0.00           H  \\nATOM   1313 1HG  LYS B 103      53.167  22.010  59.940  1.00  0.00           H  \\nATOM   1314 2HG  LYS B 103      51.923  22.101  58.699  1.00  0.00           H  \\nATOM   1315 1HD  LYS B 103      51.647  24.516  59.150  1.00  0.00           H  \\nATOM   1316 2HD  LYS B 103      52.916  24.424  60.378  1.00  0.00           H  \\nATOM   1317 1HE  LYS B 103      54.529  23.682  58.661  1.00  0.00           H  \\nATOM   1318 2HE  LYS B 103      53.254  23.758  57.428  1.00  0.00           H  \\nATOM   1319 1HZ  LYS B 103      54.626  25.722  57.414  1.00  0.00           H  \\nATOM   1320 2HZ  LYS B 103      53.085  26.117  57.846  1.00  0.00           H  \\nATOM   1321 3HZ  LYS B 103      54.270  26.047  58.991  1.00  0.00           H  \\nATOM   1322  N   ASN B 104      48.907  21.385  62.235  1.00  0.00           N  \\nATOM   1323  CA  ASN B 104      47.981  21.227  63.340  1.00  0.00           C  \\nATOM   1324  C   ASN B 104      46.570  21.109  62.884  1.00  0.00           C  \\nATOM   1325  O   ASN B 104      45.897  22.086  62.667  1.00  0.00           O  \\nATOM   1326  CB  ASN B 104      48.183  22.397  64.294  1.00  0.00           C  \\nATOM   1327  CG  ASN B 104      48.063  23.763  63.584  1.00  0.00           C  \\nATOM   1328  OD1 ASN B 104      48.875  24.010  62.680  1.00  0.00           O  \\nATOM   1329  ND2 ASN B 104      47.155  24.629  63.953  1.00  0.00           N  \\nATOM   1330  H   ASN B 104      48.687  21.991  61.669  1.00  0.00           H  \\nATOM   1331  HA  ASN B 104      48.203  20.411  63.832  1.00  0.00           H  \\nATOM   1332 1HB  ASN B 104      47.424  22.350  65.085  1.00  0.00           H  \\nATOM   1333 2HB  ASN B 104      49.160  22.323  64.767  1.00  0.00           H  \\nATOM   1334 1HD2 ASN B 104      46.481  24.437  64.692  1.00  0.00           H  \\nATOM   1335 2HD2 ASN B 104      47.095  25.508  63.489  1.00  0.00           H  \\nATOM   1336  N   PRO B 105      46.131  19.797  62.714  1.00  0.00           N  \\nATOM   1337  CA  PRO B 105      44.671  19.680  62.728  1.00  0.00           C  \\nATOM   1338  C   PRO B 105      44.019  20.367  63.924  1.00  0.00           C  \\nATOM   1339  O   PRO B 105      44.379  20.070  65.061  1.00  0.00           O  \\nATOM   1340  CB  PRO B 105      44.524  18.152  62.816  1.00  0.00           C  \\nATOM   1341  CG  PRO B 105      45.803  17.608  62.120  1.00  0.00           C  \\nATOM   1342  CD  PRO B 105      46.925  18.589  62.577  1.00  0.00           C  \\nATOM   1343  HA  PRO B 105      44.288  20.015  61.896  1.00  0.00           H  \\nATOM   1344 1HB  PRO B 105      44.449  17.843  63.865  1.00  0.00           H  \\nATOM   1345 2HB  PRO B 105      43.593  17.838  62.324  1.00  0.00           H  \\nATOM   1346 1HG  PRO B 105      45.988  16.559  62.411  1.00  0.00           H  \\nATOM   1347 2HG  PRO B 105      45.661  17.605  61.029  1.00  0.00           H  \\nATOM   1348 1HD  PRO B 105      47.336  18.278  63.548  1.00  0.00           H  \\nATOM   1349 2HD  PRO B 105      47.711  18.705  61.820  1.00  0.00           H  \\nATOM   1350  N   ASP B 106      43.074  21.257  63.660  1.00  0.00           N  \\nATOM   1351  CA  ASP B 106      42.354  21.987  64.688  1.00  0.00           C  \\nATOM   1352  C   ASP B 106      40.849  21.888  64.434  1.00  0.00           C  \\nATOM   1353  O   ASP B 106      40.373  20.900  63.925  1.00  0.00           O  \\nATOM   1354  CB  ASP B 106      42.802  23.450  64.822  1.00  0.00           C  \\nATOM   1355  CG  ASP B 106      44.215  23.663  65.447  1.00  0.00           C  \\nATOM   1356  OD1 ASP B 106      44.735  22.781  66.069  1.00  0.00           O  \\nATOM   1357  OD2 ASP B 106      44.749  24.726  65.280  1.00  0.00           O  \\nATOM   1358  H   ASP B 106      42.821  21.459  62.868  1.00  0.00           H  \\nATOM   1359  HA  ASP B 106      42.550  21.604  65.566  1.00  0.00           H  \\nATOM   1360 1HB  ASP B 106      42.749  23.910  63.904  1.00  0.00           H  \\nATOM   1361 2HB  ASP B 106      42.127  23.978  65.474  1.00  0.00           H  \\nATOM   1362  N   LYS B 107      40.098  22.920  64.791  1.00  0.00           N  \\nATOM   1363  CA  LYS B 107      38.690  22.982  64.474  1.00  0.00           C  \\nATOM   1364  C   LYS B 107      38.529  24.141  63.561  1.00  0.00           C  \\nATOM   1365  O   LYS B 107      37.928  24.036  62.513  1.00  0.00           O  \\nATOM   1366  CB  LYS B 107      37.799  23.133  65.707  1.00  0.00           C  \\nATOM   1367  CG  LYS B 107      36.288  23.083  65.364  1.00  0.00           C  \\nATOM   1368  CD  LYS B 107      35.397  23.179  66.598  1.00  0.00           C  \\nATOM   1369  CE  LYS B 107      35.427  21.897  67.410  1.00  0.00           C  \\nATOM   1370  NZ  LYS B 107      34.483  21.947  68.560  1.00  0.00           N  \\nATOM   1371  H   LYS B 107      40.373  23.604  65.225  1.00  0.00           H  \\nATOM   1372  HA  LYS B 107      38.402  22.166  64.024  1.00  0.00           H  \\nATOM   1373 1HB  LYS B 107      38.017  22.340  66.421  1.00  0.00           H  \\nATOM   1374 2HB  LYS B 107      38.001  24.089  66.194  1.00  0.00           H  \\nATOM   1375 1HG  LYS B 107      36.047  23.916  64.699  1.00  0.00           H  \\nATOM   1376 2HG  LYS B 107      36.062  22.153  64.840  1.00  0.00           H  \\nATOM   1377 1HD  LYS B 107      35.712  24.019  67.218  1.00  0.00           H  \\nATOM   1378 2HD  LYS B 107      34.369  23.346  66.279  1.00  0.00           H  \\nATOM   1379 1HE  LYS B 107      35.151  21.063  66.764  1.00  0.00           H  \\nATOM   1380 2HE  LYS B 107      36.436  21.735  67.787  1.00  0.00           H  \\nATOM   1381 1HZ  LYS B 107      33.538  22.087  68.227  1.00  0.00           H  \\nATOM   1382 2HZ  LYS B 107      34.532  21.079  69.070  1.00  0.00           H  \\nATOM   1383 3HZ  LYS B 107      34.737  22.710  69.168  1.00  0.00           H  \\nATOM   1384  N   ASP B 108      39.070  25.270  63.968  1.00  0.00           N  \\nATOM   1385  CA  ASP B 108      38.957  26.410  63.067  1.00  0.00           C  \\nATOM   1386  C   ASP B 108      39.748  26.165  61.787  1.00  0.00           C  \\nATOM   1387  O   ASP B 108      39.370  26.633  60.713  1.00  0.00           O  \\nATOM   1388  CB  ASP B 108      39.470  27.680  63.742  1.00  0.00           C  \\nATOM   1389  CG  ASP B 108      38.549  28.176  64.851  1.00  0.00           C  \\nATOM   1390  OD1 ASP B 108      37.447  27.687  64.948  1.00  0.00           O  \\nATOM   1391  OD2 ASP B 108      38.959  29.031  65.595  1.00  0.00           O  \\nATOM   1392  H   ASP B 108      39.499  25.438  64.694  1.00  0.00           H  \\nATOM   1393  HA  ASP B 108      38.016  26.540  62.826  1.00  0.00           H  \\nATOM   1394 1HB  ASP B 108      40.460  27.493  64.160  1.00  0.00           H  \\nATOM   1395 2HB  ASP B 108      39.578  28.469  62.992  1.00  0.00           H  \\nATOM   1396  N   VAL B 109      40.848  25.430  61.909  1.00  0.00           N  \\nATOM   1397  CA  VAL B 109      41.693  25.120  60.763  1.00  0.00           C  \\nATOM   1398  C   VAL B 109      40.925  24.300  59.732  1.00  0.00           C  \\nATOM   1399  O   VAL B 109      40.909  24.634  58.547  1.00  0.00           O  \\nATOM   1400  CB  VAL B 109      42.987  24.378  61.145  1.00  0.00           C  \\nATOM   1401  CG1 VAL B 109      43.711  23.909  59.897  1.00  0.00           C  \\nATOM   1402  CG2 VAL B 109      43.908  25.320  61.906  1.00  0.00           C  \\nATOM   1403  H   VAL B 109      41.128  25.096  62.650  1.00  0.00           H  \\nATOM   1404  HA  VAL B 109      41.982  25.955  60.339  1.00  0.00           H  \\nATOM   1405  HB  VAL B 109      42.736  23.507  61.730  1.00  0.00           H  \\nATOM   1406 1HG1 VAL B 109      44.624  23.373  60.181  1.00  0.00           H  \\nATOM   1407 2HG1 VAL B 109      43.067  23.267  59.343  1.00  0.00           H  \\nATOM   1408 3HG1 VAL B 109      43.974  24.768  59.279  1.00  0.00           H  \\nATOM   1409 1HG2 VAL B 109      44.826  24.786  62.175  1.00  0.00           H  \\nATOM   1410 2HG2 VAL B 109      44.154  26.170  61.270  1.00  0.00           H  \\nATOM   1411 3HG2 VAL B 109      43.431  25.686  62.808  1.00  0.00           H  \\nATOM   1412  N   ILE B 110      40.291  23.227  60.191  1.00  0.00           N  \\nATOM   1413  CA  ILE B 110      39.518  22.359  59.310  1.00  0.00           C  \\nATOM   1414  C   ILE B 110      38.450  23.156  58.570  1.00  0.00           C  \\nATOM   1415  O   ILE B 110      38.302  23.034  57.354  1.00  0.00           O  \\nATOM   1416  CB  ILE B 110      38.864  21.155  60.019  1.00  0.00           C  \\nATOM   1417  CG1 ILE B 110      39.952  20.148  60.448  1.00  0.00           C  \\nATOM   1418  CG2 ILE B 110      37.817  20.508  59.103  1.00  0.00           C  \\nATOM   1419  CD1 ILE B 110      39.460  19.052  61.367  1.00  0.00           C  \\nATOM   1420  H   ILE B 110      40.292  22.978  61.014  1.00  0.00           H  \\nATOM   1421  HA  ILE B 110      40.117  21.956  58.647  1.00  0.00           H  \\nATOM   1422  HB  ILE B 110      38.372  21.501  60.934  1.00  0.00           H  \\nATOM   1423 1HG1 ILE B 110      40.366  19.683  59.556  1.00  0.00           H  \\nATOM   1424 2HG1 ILE B 110      40.746  20.693  60.964  1.00  0.00           H  \\nATOM   1425 1HG2 ILE B 110      37.356  19.666  59.611  1.00  0.00           H  \\nATOM   1426 2HG2 ILE B 110      37.052  21.238  58.847  1.00  0.00           H  \\nATOM   1427 3HG2 ILE B 110      38.295  20.170  58.210  1.00  0.00           H  \\nATOM   1428 1HD1 ILE B 110      39.057  19.496  62.279  1.00  0.00           H  \\nATOM   1429 2HD1 ILE B 110      38.680  18.481  60.869  1.00  0.00           H  \\nATOM   1430 3HD1 ILE B 110      40.288  18.392  61.622  1.00  0.00           H  \\nATOM   1431  N   VAL B 111      37.709  23.973  59.311  1.00  0.00           N  \\nATOM   1432  CA  VAL B 111      36.656  24.794  58.726  1.00  0.00           C  \\nATOM   1433  C   VAL B 111      37.227  25.723  57.661  1.00  0.00           C  \\nATOM   1434  O   VAL B 111      36.581  25.993  56.648  1.00  0.00           O  \\nATOM   1435  CB  VAL B 111      35.935  25.583  59.817  1.00  0.00           C  \\nATOM   1436  CG1 VAL B 111      34.989  26.595  59.190  1.00  0.00           C  \\nATOM   1437  CG2 VAL B 111      35.171  24.597  60.705  1.00  0.00           C  \\nATOM   1438  H   VAL B 111      37.797  24.070  60.161  1.00  0.00           H  \\nATOM   1439  HA  VAL B 111      35.990  24.212  58.303  1.00  0.00           H  \\nATOM   1440  HB  VAL B 111      36.670  26.127  60.417  1.00  0.00           H  \\nATOM   1441 1HG1 VAL B 111      34.483  27.153  59.976  1.00  0.00           H  \\nATOM   1442 2HG1 VAL B 111      35.566  27.278  58.569  1.00  0.00           H  \\nATOM   1443 3HG1 VAL B 111      34.257  26.086  58.582  1.00  0.00           H  \\nATOM   1444 1HG2 VAL B 111      34.672  25.149  61.490  1.00  0.00           H  \\nATOM   1445 2HG2 VAL B 111      34.443  24.060  60.114  1.00  0.00           H  \\nATOM   1446 3HG2 VAL B 111      35.866  23.889  61.146  1.00  0.00           H  \\nATOM   1447  N   GLU B 112      38.441  26.209  57.897  1.00  0.00           N  \\nATOM   1448  CA  GLU B 112      39.104  27.107  56.957  1.00  0.00           C  \\nATOM   1449  C   GLU B 112      39.414  26.401  55.636  1.00  0.00           C  \\nATOM   1450  O   GLU B 112      39.383  27.024  54.576  1.00  0.00           O  \\nATOM   1451  CB  GLU B 112      40.375  27.771  57.519  1.00  0.00           C  \\nATOM   1452  CG  GLU B 112      40.118  28.885  58.553  1.00  0.00           C  \\nATOM   1453  CD  GLU B 112      41.387  29.472  59.159  1.00  0.00           C  \\nATOM   1454  OE1 GLU B 112      42.454  28.971  58.890  1.00  0.00           O  \\nATOM   1455  OE2 GLU B 112      41.277  30.432  59.889  1.00  0.00           O  \\nATOM   1456  H   GLU B 112      38.906  26.033  58.599  1.00  0.00           H  \\nATOM   1457  HA  GLU B 112      38.514  27.852  56.765  1.00  0.00           H  \\nATOM   1458 1HB  GLU B 112      41.000  27.027  57.992  1.00  0.00           H  \\nATOM   1459 2HB  GLU B 112      40.951  28.199  56.698  1.00  0.00           H  \\nATOM   1460 1HG  GLU B 112      39.592  29.688  58.040  1.00  0.00           H  \\nATOM   1461 2HG  GLU B 112      39.467  28.533  59.329  1.00  0.00           H  \\nATOM   1462  N   VAL B 113      39.706  25.104  55.700  1.00  0.00           N  \\nATOM   1463  CA  VAL B 113      40.001  24.326  54.498  1.00  0.00           C  \\nATOM   1464  C   VAL B 113      38.723  24.029  53.715  1.00  0.00           C  \\nATOM   1465  O   VAL B 113      38.716  24.089  52.487  1.00  0.00           O  \\nATOM   1466  CB  VAL B 113      40.678  22.975  54.877  1.00  0.00           C  \\nATOM   1467  CG1 VAL B 113      40.863  22.102  53.633  1.00  0.00           C  \\nATOM   1468  CG2 VAL B 113      42.025  23.223  55.557  1.00  0.00           C  \\nATOM   1469  H   VAL B 113      39.740  24.650  56.429  1.00  0.00           H  \\nATOM   1470  HA  VAL B 113      40.590  24.842  53.925  1.00  0.00           H  \\nATOM   1471  HB  VAL B 113      40.022  22.436  55.564  1.00  0.00           H  \\nATOM   1472 1HG1 VAL B 113      41.314  21.150  53.916  1.00  0.00           H  \\nATOM   1473 2HG1 VAL B 113      39.892  21.925  53.183  1.00  0.00           H  \\nATOM   1474 3HG1 VAL B 113      41.511  22.612  52.918  1.00  0.00           H  \\nATOM   1475 1HG2 VAL B 113      42.477  22.289  55.832  1.00  0.00           H  \\nATOM   1476 2HG2 VAL B 113      42.671  23.732  54.884  1.00  0.00           H  \\nATOM   1477 3HG2 VAL B 113      41.880  23.824  56.447  1.00  0.00           H  \\nATOM   1478  N   LEU B 114      37.645  23.709  54.426  1.00  0.00           N  \\nATOM   1479  CA  LEU B 114      36.368  23.418  53.782  1.00  0.00           C  \\nATOM   1480  C   LEU B 114      35.861  24.630  53.009  1.00  0.00           C  \\nATOM   1481  O   LEU B 114      35.326  24.495  51.910  1.00  0.00           O  \\nATOM   1482  CB  LEU B 114      35.355  23.012  54.860  1.00  0.00           C  \\nATOM   1483  CG  LEU B 114      35.626  21.655  55.540  1.00  0.00           C  \\nATOM   1484  CD1 LEU B 114      34.728  21.496  56.740  1.00  0.00           C  \\nATOM   1485  CD2 LEU B 114      35.364  20.536  54.554  1.00  0.00           C  \\nATOM   1486  H   LEU B 114      37.627  23.654  55.284  1.00  0.00           H  \\nATOM   1487  HA  LEU B 114      36.490  22.690  53.152  1.00  0.00           H  \\nATOM   1488 1HB  LEU B 114      35.354  23.778  55.638  1.00  0.00           H  \\nATOM   1489 2HB  LEU B 114      34.362  22.976  54.413  1.00  0.00           H  \\nATOM   1490  HG  LEU B 114      36.666  21.616  55.878  1.00  0.00           H  \\nATOM   1491 1HD1 LEU B 114      34.941  20.555  57.204  1.00  0.00           H  \\nATOM   1492 2HD1 LEU B 114      34.932  22.290  57.442  1.00  0.00           H  \\nATOM   1493 3HD1 LEU B 114      33.684  21.530  56.434  1.00  0.00           H  \\nATOM   1494 1HD2 LEU B 114      35.550  19.586  55.043  1.00  0.00           H  \\nATOM   1495 2HD2 LEU B 114      34.329  20.573  54.228  1.00  0.00           H  \\nATOM   1496 3HD2 LEU B 114      36.019  20.643  53.693  1.00  0.00           H  \\nATOM   1497  N   LYS B 115      36.038  25.814  53.586  1.00  0.00           N  \\nATOM   1498  CA  LYS B 115      35.582  27.043  52.948  1.00  0.00           C  \\nATOM   1499  C   LYS B 115      36.336  27.303  51.649  1.00  0.00           C  \\nATOM   1500  O   LYS B 115      35.747  27.749  50.672  1.00  0.00           O  \\nATOM   1501  CB  LYS B 115      35.724  28.213  53.929  1.00  0.00           C  \\nATOM   1502  CG  LYS B 115      34.692  28.188  55.101  1.00  0.00           C  \\nATOM   1503  CD  LYS B 115      34.997  29.256  56.163  1.00  0.00           C  \\nATOM   1504  CE  LYS B 115      34.562  30.663  55.750  1.00  0.00           C  \\nATOM   1505  NZ  LYS B 115      33.074  30.847  55.848  1.00  0.00           N  \\nATOM   1506  H   LYS B 115      36.420  25.933  54.347  1.00  0.00           H  \\nATOM   1507  HA  LYS B 115      34.641  26.954  52.733  1.00  0.00           H  \\nATOM   1508 1HB  LYS B 115      36.727  28.189  54.370  1.00  0.00           H  \\nATOM   1509 2HB  LYS B 115      35.620  29.154  53.396  1.00  0.00           H  \\nATOM   1510 1HG  LYS B 115      33.695  28.370  54.706  1.00  0.00           H  \\nATOM   1511 2HG  LYS B 115      34.697  27.207  55.570  1.00  0.00           H  \\nATOM   1512 1HD  LYS B 115      34.481  28.990  57.085  1.00  0.00           H  \\nATOM   1513 2HD  LYS B 115      36.070  29.257  56.363  1.00  0.00           H  \\nATOM   1514 1HE  LYS B 115      35.050  31.389  56.399  1.00  0.00           H  \\nATOM   1515 2HE  LYS B 115      34.873  30.844  54.722  1.00  0.00           H  \\nATOM   1516 1HZ  LYS B 115      32.601  30.188  55.246  1.00  0.00           H  \\nATOM   1517 2HZ  LYS B 115      32.782  30.692  56.808  1.00  0.00           H  \\nATOM   1518 3HZ  LYS B 115      32.827  31.788  55.575  1.00  0.00           H  \\nATOM   1519  N   VAL B 116      37.633  27.015  51.639  1.00  0.00           N  \\nATOM   1520  CA  VAL B 116      38.446  27.193  50.440  1.00  0.00           C  \\nATOM   1521  C   VAL B 116      38.081  26.141  49.394  1.00  0.00           C  \\nATOM   1522  O   VAL B 116      38.090  26.420  48.193  1.00  0.00           O  \\nATOM   1523  CB  VAL B 116      39.952  27.178  50.719  1.00  0.00           C  \\nATOM   1524  CG1 VAL B 116      40.706  27.238  49.387  1.00  0.00           C  \\nATOM   1525  CG2 VAL B 116      40.316  28.386  51.557  1.00  0.00           C  \\nATOM   1526  H   VAL B 116      38.069  26.712  52.316  1.00  0.00           H  \\nATOM   1527  HA  VAL B 116      38.265  28.068  50.062  1.00  0.00           H  \\nATOM   1528  HB  VAL B 116      40.225  26.253  51.236  1.00  0.00           H  \\nATOM   1529 1HG1 VAL B 116      41.772  27.235  49.576  1.00  0.00           H  \\nATOM   1530 2HG1 VAL B 116      40.433  26.390  48.777  1.00  0.00           H  \\nATOM   1531 3HG1 VAL B 116      40.441  28.155  48.860  1.00  0.00           H  \\nATOM   1532 1HG2 VAL B 116      41.371  28.404  51.738  1.00  0.00           H  \\nATOM   1533 2HG2 VAL B 116      40.040  29.295  51.021  1.00  0.00           H  \\nATOM   1534 3HG2 VAL B 116      39.783  28.350  52.503  1.00  0.00           H  \\nATOM   1535  N   ILE B 117      37.747  24.938  49.853  1.00  0.00           N  \\nATOM   1536  CA  ILE B 117      37.345  23.857  48.955  1.00  0.00           C  \\nATOM   1537  C   ILE B 117      36.060  24.222  48.215  1.00  0.00           C  \\nATOM   1538  O   ILE B 117      35.910  23.910  47.037  1.00  0.00           O  \\nATOM   1539  CB  ILE B 117      37.103  22.507  49.667  1.00  0.00           C  \\nATOM   1540  CG1 ILE B 117      38.398  21.897  50.150  1.00  0.00           C  \\nATOM   1541  CG2 ILE B 117      36.380  21.590  48.737  1.00  0.00           C  \\nATOM   1542  CD1 ILE B 117      38.218  20.731  51.089  1.00  0.00           C  \\nATOM   1543  H   ILE B 117      37.744  24.720  50.686  1.00  0.00           H  \\nATOM   1544  HA  ILE B 117      38.042  23.723  48.294  1.00  0.00           H  \\nATOM   1545  HB  ILE B 117      36.488  22.669  50.547  1.00  0.00           H  \\nATOM   1546 1HG1 ILE B 117      38.956  21.550  49.282  1.00  0.00           H  \\nATOM   1547 2HG1 ILE B 117      38.969  22.649  50.643  1.00  0.00           H  \\nATOM   1548 1HG2 ILE B 117      36.195  20.647  49.220  1.00  0.00           H  \\nATOM   1549 2HG2 ILE B 117      35.436  22.038  48.442  1.00  0.00           H  \\nATOM   1550 3HG2 ILE B 117      36.983  21.440  47.882  1.00  0.00           H  \\nATOM   1551 1HD1 ILE B 117      39.195  20.345  51.375  1.00  0.00           H  \\nATOM   1552 2HD1 ILE B 117      37.687  21.061  51.972  1.00  0.00           H  \\nATOM   1553 3HD1 ILE B 117      37.655  19.957  50.608  1.00  0.00           H  \\nATOM   1554  N   VAL B 118      35.142  24.895  48.902  1.00  0.00           N  \\nATOM   1555  CA  VAL B 118      33.877  25.292  48.293  1.00  0.00           C  \\nATOM   1556  C   VAL B 118      34.087  26.365  47.230  1.00  0.00           C  \\nATOM   1557  O   VAL B 118      33.445  26.337  46.181  1.00  0.00           O  \\nATOM   1558  CB  VAL B 118      32.853  25.731  49.350  1.00  0.00           C  \\nATOM   1559  CG1 VAL B 118      31.587  26.275  48.653  1.00  0.00           C  \\nATOM   1560  CG2 VAL B 118      32.506  24.522  50.227  1.00  0.00           C  \\nATOM   1561  H   VAL B 118      35.228  25.135  49.723  1.00  0.00           H  \\nATOM   1562  HA  VAL B 118      33.481  24.519  47.861  1.00  0.00           H  \\nATOM   1563  HB  VAL B 118      33.279  26.530  49.966  1.00  0.00           H  \\nATOM   1564 1HG1 VAL B 118      30.858  26.583  49.400  1.00  0.00           H  \\nATOM   1565 2HG1 VAL B 118      31.856  27.130  48.031  1.00  0.00           H  \\nATOM   1566 3HG1 VAL B 118      31.151  25.493  48.029  1.00  0.00           H  \\nATOM   1567 1HG2 VAL B 118      31.780  24.799  50.982  1.00  0.00           H  \\nATOM   1568 2HG2 VAL B 118      32.106  23.756  49.614  1.00  0.00           H  \\nATOM   1569 3HG2 VAL B 118      33.383  24.150  50.701  1.00  0.00           H  \\nATOM   1570  N   LYS B 119      34.984  27.309  47.498  1.00  0.00           N  \\nATOM   1571  CA  LYS B 119      35.297  28.345  46.522  1.00  0.00           C  \\nATOM   1572  C   LYS B 119      35.963  27.742  45.296  1.00  0.00           C  \\nATOM   1573  O   LYS B 119      35.760  28.216  44.182  1.00  0.00           O  \\nATOM   1574  CB  LYS B 119      36.249  29.394  47.105  1.00  0.00           C  \\nATOM   1575  CG  LYS B 119      35.675  30.300  48.188  1.00  0.00           C  \\nATOM   1576  CD  LYS B 119      34.593  31.220  47.619  1.00  0.00           C  \\nATOM   1577  CE  LYS B 119      34.237  32.361  48.584  1.00  0.00           C  \\nATOM   1578  NZ  LYS B 119      33.685  31.871  49.872  1.00  0.00           N  \\nATOM   1579  H   LYS B 119      35.423  27.371  48.234  1.00  0.00           H  \\nATOM   1580  HA  LYS B 119      34.474  28.772  46.238  1.00  0.00           H  \\nATOM   1581 1HB  LYS B 119      37.117  28.883  47.534  1.00  0.00           H  \\nATOM   1582 2HB  LYS B 119      36.617  30.029  46.302  1.00  0.00           H  \\nATOM   1583 1HG  LYS B 119      35.238  29.681  48.975  1.00  0.00           H  \\nATOM   1584 2HG  LYS B 119      36.471  30.901  48.625  1.00  0.00           H  \\nATOM   1585 1HD  LYS B 119      34.943  31.650  46.679  1.00  0.00           H  \\nATOM   1586 2HD  LYS B 119      33.693  30.635  47.412  1.00  0.00           H  \\nATOM   1587 1HE  LYS B 119      35.129  32.953  48.784  1.00  0.00           H  \\nATOM   1588 2HE  LYS B 119      33.492  32.998  48.105  1.00  0.00           H  \\nATOM   1589 1HZ  LYS B 119      33.460  32.659  50.462  1.00  0.00           H  \\nATOM   1590 2HZ  LYS B 119      32.853  31.330  49.700  1.00  0.00           H  \\nATOM   1591 3HZ  LYS B 119      34.374  31.295  50.330  1.00  0.00           H  \\nATOM   1592  N   ALA B 120      36.752  26.693  45.507  1.00  0.00           N  \\nATOM   1593  CA  ALA B 120      37.411  26.007  44.403  1.00  0.00           C  \\nATOM   1594  C   ALA B 120      36.373  25.323  43.527  1.00  0.00           C  \\nATOM   1595  O   ALA B 120      36.441  25.390  42.300  1.00  0.00           O  \\nATOM   1596  CB  ALA B 120      38.412  24.980  44.929  1.00  0.00           C  \\nATOM   1597  H   ALA B 120      36.922  26.359  46.281  1.00  0.00           H  \\nATOM   1598  HA  ALA B 120      37.886  26.656  43.861  1.00  0.00           H  \\nATOM   1599 1HB  ALA B 120      37.891  24.238  45.527  1.00  0.00           H  \\nATOM   1600 2HB  ALA B 120      38.902  24.488  44.092  1.00  0.00           H  \\nATOM   1601 3HB  ALA B 120      39.155  25.475  45.547  1.00  0.00           H  \\nATOM   1602  N   ILE B 121      35.410  24.664  44.165  1.00  0.00           N  \\nATOM   1603  CA  ILE B 121      34.331  24.005  43.442  1.00  0.00           C  \\nATOM   1604  C   ILE B 121      33.499  25.042  42.700  1.00  0.00           C  \\nATOM   1605  O   ILE B 121      33.155  24.852  41.536  1.00  0.00           O  \\nATOM   1606  CB  ILE B 121      33.441  23.177  44.381  1.00  0.00           C  \\nATOM   1607  CG1 ILE B 121      34.250  21.986  44.915  1.00  0.00           C  \\nATOM   1608  CG2 ILE B 121      32.192  22.698  43.627  1.00  0.00           C  \\nATOM   1609  CD1 ILE B 121      33.616  21.261  46.057  1.00  0.00           C  \\nATOM   1610  H   ILE B 121      35.359  24.585  45.020  1.00  0.00           H  \\nATOM   1611  HA  ILE B 121      34.713  23.400  42.786  1.00  0.00           H  \\nATOM   1612  HB  ILE B 121      33.144  23.792  45.234  1.00  0.00           H  \\nATOM   1613 1HG1 ILE B 121      34.397  21.275  44.104  1.00  0.00           H  \\nATOM   1614 2HG1 ILE B 121      35.226  22.345  45.237  1.00  0.00           H  \\nATOM   1615 1HG2 ILE B 121      31.563  22.122  44.300  1.00  0.00           H  \\nATOM   1616 2HG2 ILE B 121      31.642  23.558  43.260  1.00  0.00           H  \\nATOM   1617 3HG2 ILE B 121      32.490  22.077  42.783  1.00  0.00           H  \\nATOM   1618 1HD1 ILE B 121      34.272  20.461  46.336  1.00  0.00           H  \\nATOM   1619 2HD1 ILE B 121      33.488  21.946  46.893  1.00  0.00           H  \\nATOM   1620 3HD1 ILE B 121      32.650  20.862  45.760  1.00  0.00           H  \\nATOM   1621  N   GLU B 122      33.179  26.143  43.372  1.00  0.00           N  \\nATOM   1622  CA  GLU B 122      32.456  27.229  42.722  1.00  0.00           C  \\nATOM   1623  C   GLU B 122      33.227  27.712  41.496  1.00  0.00           C  \\nATOM   1624  O   GLU B 122      32.656  27.880  40.417  1.00  0.00           O  \\nATOM   1625  CB  GLU B 122      32.216  28.407  43.677  1.00  0.00           C  \\nATOM   1626  CG  GLU B 122      31.445  29.583  43.034  1.00  0.00           C  \\nATOM   1627  CD  GLU B 122      31.154  30.739  43.974  1.00  0.00           C  \\nATOM   1628  OE1 GLU B 122      31.511  30.674  45.134  1.00  0.00           O  \\nATOM   1629  OE2 GLU B 122      30.541  31.679  43.537  1.00  0.00           O  \\nATOM   1630  H   GLU B 122      33.367  26.286  44.199  1.00  0.00           H  \\nATOM   1631  HA  GLU B 122      31.591  26.904  42.427  1.00  0.00           H  \\nATOM   1632 1HB  GLU B 122      31.661  28.062  44.550  1.00  0.00           H  \\nATOM   1633 2HB  GLU B 122      33.171  28.787  44.029  1.00  0.00           H  \\nATOM   1634 1HG  GLU B 122      32.042  29.959  42.202  1.00  0.00           H  \\nATOM   1635 2HG  GLU B 122      30.510  29.216  42.623  1.00  0.00           H  \\nATOM   1636  N   ALA B 123      34.529  27.913  41.667  1.00  0.00           N  \\nATOM   1637  CA  ALA B 123      35.385  28.352  40.574  1.00  0.00           C  \\nATOM   1638  C   ALA B 123      35.407  27.321  39.448  1.00  0.00           C  \\nATOM   1639  O   ALA B 123      35.443  27.678  38.275  1.00  0.00           O  \\nATOM   1640  CB  ALA B 123      36.791  28.608  41.079  1.00  0.00           C  \\nATOM   1641  H   ALA B 123      34.944  27.801  42.413  1.00  0.00           H  \\nATOM   1642  HA  ALA B 123      35.037  29.183  40.214  1.00  0.00           H  \\nATOM   1643 1HB  ALA B 123      37.423  28.961  40.264  1.00  0.00           H  \\nATOM   1644 2HB  ALA B 123      36.734  29.361  41.854  1.00  0.00           H  \\nATOM   1645 3HB  ALA B 123      37.205  27.698  41.492  1.00  0.00           H  \\nATOM   1646  N   SER B 124      35.374  26.043  39.814  1.00  0.00           N  \\nATOM   1647  CA  SER B 124      35.434  24.956  38.840  1.00  0.00           C  \\nATOM   1648  C   SER B 124      34.142  24.842  38.037  1.00  0.00           C  \\nATOM   1649  O   SER B 124      34.174  24.635  36.825  1.00  0.00           O  \\nATOM   1650  CB  SER B 124      35.701  23.662  39.573  1.00  0.00           C  \\nATOM   1651  OG  SER B 124      35.771  22.567  38.709  1.00  0.00           O  \\nATOM   1652  H   SER B 124      35.317  25.776  40.630  1.00  0.00           H  \\nATOM   1653  HA  SER B 124      36.158  25.130  38.218  1.00  0.00           H  \\nATOM   1654 1HB  SER B 124      36.628  23.749  40.134  1.00  0.00           H  \\nATOM   1655 2HB  SER B 124      34.908  23.497  40.291  1.00  0.00           H  \\nATOM   1656  HG  SER B 124      35.784  21.800  39.283  1.00  0.00           H  \\nATOM   1657  N   VAL B 125      33.008  24.971  38.717  1.00  0.00           N  \\nATOM   1658  CA  VAL B 125      31.712  24.958  38.047  1.00  0.00           C  \\nATOM   1659  C   VAL B 125      31.604  26.144  37.087  1.00  0.00           C  \\nATOM   1660  O   VAL B 125      31.047  26.023  35.995  1.00  0.00           O  \\nATOM   1661  CB  VAL B 125      30.557  24.978  39.080  1.00  0.00           C  \\nATOM   1662  CG1 VAL B 125      29.206  25.147  38.370  1.00  0.00           C  \\nATOM   1663  CG2 VAL B 125      30.562  23.671  39.879  1.00  0.00           C  \\nATOM   1664  H   VAL B 125      32.961  25.067  39.570  1.00  0.00           H  \\nATOM   1665  HA  VAL B 125      31.630  24.142  37.529  1.00  0.00           H  \\nATOM   1666  HB  VAL B 125      30.699  25.827  39.758  1.00  0.00           H  \\nATOM   1667 1HG1 VAL B 125      28.407  25.166  39.112  1.00  0.00           H  \\nATOM   1668 2HG1 VAL B 125      29.208  26.083  37.809  1.00  0.00           H  \\nATOM   1669 3HG1 VAL B 125      29.046  24.311  37.686  1.00  0.00           H  \\nATOM   1670 1HG2 VAL B 125      29.756  23.685  40.615  1.00  0.00           H  \\nATOM   1671 2HG2 VAL B 125      30.423  22.848  39.203  1.00  0.00           H  \\nATOM   1672 3HG2 VAL B 125      31.505  23.552  40.386  1.00  0.00           H  \\nATOM   1673  N   GLU B 126      32.146  27.285  37.495  1.00  0.00           N  \\nATOM   1674  CA  GLU B 126      32.132  28.476  36.655  1.00  0.00           C  \\nATOM   1675  C   GLU B 126      33.007  28.268  35.421  1.00  0.00           C  \\nATOM   1676  O   GLU B 126      32.635  28.654  34.314  1.00  0.00           O  \\nATOM   1677  CB  GLU B 126      32.535  29.716  37.450  1.00  0.00           C  \\nATOM   1678  CG  GLU B 126      32.475  31.042  36.666  1.00  0.00           C  \\nATOM   1679  CD  GLU B 126      31.085  31.367  36.110  1.00  0.00           C  \\nATOM   1680  OE1 GLU B 126      30.144  30.665  36.412  1.00  0.00           O  \\nATOM   1681  OE2 GLU B 126      30.957  32.360  35.425  1.00  0.00           O  \\nATOM   1682  H   GLU B 126      32.529  27.397  38.257  1.00  0.00           H  \\nATOM   1683  HA  GLU B 126      31.225  28.643  36.356  1.00  0.00           H  \\nATOM   1684 1HB  GLU B 126      31.894  29.807  38.327  1.00  0.00           H  \\nATOM   1685 2HB  GLU B 126      33.556  29.590  37.811  1.00  0.00           H  \\nATOM   1686 1HG  GLU B 126      32.792  31.850  37.323  1.00  0.00           H  \\nATOM   1687 2HG  GLU B 126      33.186  30.984  35.840  1.00  0.00           H  \\nATOM   1688  N   ASN B 127      34.165  27.648  35.613  1.00  0.00           N  \\nATOM   1689  CA  ASN B 127      35.051  27.352  34.495  1.00  0.00           C  \\nATOM   1690  C   ASN B 127      34.420  26.353  33.530  1.00  0.00           C  \\nATOM   1691  O   ASN B 127      34.582  26.463  32.317  1.00  0.00           O  \\nATOM   1692  CB  ASN B 127      36.384  26.818  35.022  1.00  0.00           C  \\nATOM   1693  CG  ASN B 127      37.487  26.703  33.971  1.00  0.00           C  \\nATOM   1694  OD1 ASN B 127      37.926  27.698  33.393  1.00  0.00           O  \\nATOM   1695  ND2 ASN B 127      37.925  25.502  33.712  1.00  0.00           N  \\nATOM   1696  H   ASN B 127      34.461  27.389  36.378  1.00  0.00           H  \\nATOM   1697  HA  ASN B 127      35.227  28.171  34.006  1.00  0.00           H  \\nATOM   1698 1HB  ASN B 127      36.735  27.462  35.831  1.00  0.00           H  \\nATOM   1699 2HB  ASN B 127      36.218  25.830  35.458  1.00  0.00           H  \\nATOM   1700 1HD2 ASN B 127      37.538  24.711  34.221  1.00  0.00           H  \\nATOM   1701 2HD2 ASN B 127      38.626  25.364  33.015  1.00  0.00           H  \\nATOM   1702  N   GLN B 128      33.697  25.380  34.073  1.00  0.00           N  \\nATOM   1703  CA  GLN B 128      33.050  24.372  33.247  1.00  0.00           C  \\nATOM   1704  C   GLN B 128      31.886  24.974  32.465  1.00  0.00           C  \\nATOM   1705  O   GLN B 128      31.623  24.569  31.337  1.00  0.00           O  \\nATOM   1706  CB  GLN B 128      32.618  23.200  34.120  1.00  0.00           C  \\nATOM   1707  CG  GLN B 128      33.823  22.388  34.662  1.00  0.00           C  \\nATOM   1708  CD  GLN B 128      33.450  21.259  35.606  1.00  0.00           C  \\nATOM   1709  OE1 GLN B 128      32.773  20.322  35.185  1.00  0.00           O  \\nATOM   1710  NE2 GLN B 128      33.874  21.305  36.870  1.00  0.00           N  \\nATOM   1711  H   GLN B 128      33.566  25.282  34.917  1.00  0.00           H  \\nATOM   1712  HA  GLN B 128      33.694  24.032  32.606  1.00  0.00           H  \\nATOM   1713 1HB  GLN B 128      32.050  23.577  34.973  1.00  0.00           H  \\nATOM   1714 2HB  GLN B 128      31.976  22.557  33.557  1.00  0.00           H  \\nATOM   1715 1HG  GLN B 128      34.323  21.937  33.809  1.00  0.00           H  \\nATOM   1716 2HG  GLN B 128      34.519  23.039  35.159  1.00  0.00           H  \\nATOM   1717 1HE2 GLN B 128      33.670  20.540  37.488  1.00  0.00           H  \\nATOM   1718 2HE2 GLN B 128      34.440  22.064  37.232  1.00  0.00           H  \\nATOM   1719  N   ARG B 129      31.204  25.951  33.056  1.00  0.00           N  \\nATOM   1720  CA  ARG B 129      30.122  26.647  32.366  1.00  0.00           C  \\nATOM   1721  C   ARG B 129      30.669  27.468  31.202  1.00  0.00           C  \\nATOM   1722  O   ARG B 129      30.090  27.490  30.119  1.00  0.00           O  \\nATOM   1723  CB  ARG B 129      29.372  27.546  33.339  1.00  0.00           C  \\nATOM   1724  CG  ARG B 129      28.090  28.173  32.796  1.00  0.00           C  \\nATOM   1725  CD  ARG B 129      27.401  29.025  33.830  1.00  0.00           C  \\nATOM   1726  NE  ARG B 129      28.100  30.281  34.092  1.00  0.00           N  \\nATOM   1727  CZ  ARG B 129      27.944  31.389  33.332  1.00  0.00           C  \\nATOM   1728  NH1 ARG B 129      27.109  31.366  32.302  1.00  0.00           N  \\nATOM   1729  NH2 ARG B 129      28.611  32.495  33.617  1.00  0.00           N  \\nATOM   1730  H   ARG B 129      31.348  26.231  33.856  1.00  0.00           H  \\nATOM   1731  HA  ARG B 129      29.500  25.993  32.009  1.00  0.00           H  \\nATOM   1732 1HB  ARG B 129      29.114  26.973  34.230  1.00  0.00           H  \\nATOM   1733 2HB  ARG B 129      30.030  28.360  33.661  1.00  0.00           H  \\nATOM   1734 1HG  ARG B 129      28.338  28.806  31.939  1.00  0.00           H  \\nATOM   1735 2HG  ARG B 129      27.406  27.385  32.481  1.00  0.00           H  \\nATOM   1736 1HD  ARG B 129      26.394  29.263  33.491  1.00  0.00           H  \\nATOM   1737 2HD  ARG B 129      27.346  28.472  34.768  1.00  0.00           H  \\nATOM   1738  HE  ARG B 129      28.758  30.336  34.897  1.00  0.00           H  \\nATOM   1739 1HH1 ARG B 129      26.603  30.522  32.076  1.00  0.00           H  \\nATOM   1740 2HH1 ARG B 129      26.992  32.185  31.719  1.00  0.00           H  \\nATOM   1741 1HH2 ARG B 129      29.285  32.501  34.406  1.00  0.00           H  \\nATOM   1742 2HH2 ARG B 129      28.473  33.320  33.063  1.00  0.00           H  \\nATOM   1743  N   ILE B 130      31.784  28.148  31.438  1.00  0.00           N  \\nATOM   1744  CA  ILE B 130      32.424  28.947  30.404  1.00  0.00           C  \\nATOM   1745  C   ILE B 130      32.994  28.037  29.322  1.00  0.00           C  \\nATOM   1746  O   ILE B 130      32.967  28.375  28.140  1.00  0.00           O  \\nATOM   1747  CB  ILE B 130      33.504  29.852  30.991  1.00  0.00           C  \\nATOM   1748  CG1 ILE B 130      32.833  30.900  31.856  1.00  0.00           C  \\nATOM   1749  CG2 ILE B 130      34.307  30.512  29.883  1.00  0.00           C  \\nATOM   1750  CD1 ILE B 130      33.778  31.674  32.697  1.00  0.00           C  \\nATOM   1751  H   ILE B 130      32.193  28.162  32.195  1.00  0.00           H  \\nATOM   1752  HA  ILE B 130      31.762  29.525  29.994  1.00  0.00           H  \\nATOM   1753  HB  ILE B 130      34.169  29.262  31.629  1.00  0.00           H  \\nATOM   1754 1HG1 ILE B 130      32.302  31.581  31.209  1.00  0.00           H  \\nATOM   1755 2HG1 ILE B 130      32.110  30.412  32.515  1.00  0.00           H  \\nATOM   1756 1HG2 ILE B 130      35.051  31.145  30.338  1.00  0.00           H  \\nATOM   1757 2HG2 ILE B 130      34.790  29.750  29.277  1.00  0.00           H  \\nATOM   1758 3HG2 ILE B 130      33.648  31.112  29.255  1.00  0.00           H  \\nATOM   1759 1HD1 ILE B 130      33.224  32.404  33.290  1.00  0.00           H  \\nATOM   1760 2HD1 ILE B 130      34.286  30.979  33.364  1.00  0.00           H  \\nATOM   1761 3HD1 ILE B 130      34.503  32.186  32.068  1.00  0.00           H  \\nATOM   1762  N   SER B 131      33.500  26.876  29.725  1.00  0.00           N  \\nATOM   1763  CA  SER B 131      34.026  25.916  28.761  1.00  0.00           C  \\nATOM   1764  C   SER B 131      32.922  25.417  27.839  1.00  0.00           C  \\nATOM   1765  O   SER B 131      33.143  25.226  26.643  1.00  0.00           O  \\nATOM   1766  CB  SER B 131      34.628  24.751  29.532  1.00  0.00           C  \\nATOM   1767  OG  SER B 131      35.132  23.741  28.702  1.00  0.00           O  \\nATOM   1768  H   SER B 131      33.550  26.621  30.544  1.00  0.00           H  \\nATOM   1769  HA  SER B 131      34.700  26.351  28.216  1.00  0.00           H  \\nATOM   1770 1HB  SER B 131      35.414  25.115  30.185  1.00  0.00           H  \\nATOM   1771 2HB  SER B 131      33.863  24.331  30.166  1.00  0.00           H  \\nATOM   1772  HG  SER B 131      35.365  23.026  29.298  1.00  0.00           H  \\nATOM   1773  N   ALA B 132      31.734  25.206  28.397  1.00  0.00           N  \\nATOM   1774  CA  ALA B 132      30.608  24.713  27.611  1.00  0.00           C  \\nATOM   1775  C   ALA B 132      30.191  25.752  26.580  1.00  0.00           C  \\nATOM   1776  O   ALA B 132      29.869  25.412  25.445  1.00  0.00           O  \\nATOM   1777  CB  ALA B 132      29.448  24.376  28.521  1.00  0.00           C  \\nATOM   1778  H   ALA B 132      31.554  25.340  29.228  1.00  0.00           H  \\nATOM   1779  HA  ALA B 132      30.878  23.910  27.139  1.00  0.00           H  \\nATOM   1780 1HB  ALA B 132      29.135  25.273  29.057  1.00  0.00           H  \\nATOM   1781 2HB  ALA B 132      28.616  23.994  27.934  1.00  0.00           H  \\nATOM   1782 3HB  ALA B 132      29.772  23.621  29.233  1.00  0.00           H  \\nATOM   1783  N   GLU B 133      30.199  27.020  26.976  1.00  0.00           N  \\nATOM   1784  CA  GLU B 133      29.883  28.094  26.045  1.00  0.00           C  \\nATOM   1785  C   GLU B 133      30.924  28.139  24.931  1.00  0.00           C  \\nATOM   1786  O   GLU B 133      30.588  28.339  23.764  1.00  0.00           O  \\nATOM   1787  CB  GLU B 133      29.786  29.454  26.753  1.00  0.00           C  \\nATOM   1788  CG  GLU B 133      28.541  29.622  27.659  1.00  0.00           C  \\nATOM   1789  CD  GLU B 133      28.468  30.990  28.344  1.00  0.00           C  \\nATOM   1790  OE1 GLU B 133      29.429  31.717  28.258  1.00  0.00           O  \\nATOM   1791  OE2 GLU B 133      27.456  31.304  28.937  1.00  0.00           O  \\nATOM   1792  H   GLU B 133      30.383  27.283  27.774  1.00  0.00           H  \\nATOM   1793  HA  GLU B 133      29.018  27.920  25.641  1.00  0.00           H  \\nATOM   1794 1HB  GLU B 133      30.671  29.597  27.376  1.00  0.00           H  \\nATOM   1795 2HB  GLU B 133      29.778  30.251  26.010  1.00  0.00           H  \\nATOM   1796 1HG  GLU B 133      27.646  29.479  27.055  1.00  0.00           H  \\nATOM   1797 2HG  GLU B 133      28.560  28.842  28.420  1.00  0.00           H  \\nATOM   1798  N   ASN B 134      32.180  27.941  25.260  1.00  0.00           N  \\nATOM   1799  CA  ASN B 134      33.232  27.883  24.260  1.00  0.00           C  \\nATOM   1800  C   ASN B 134      33.014  26.704  23.305  1.00  0.00           C  \\nATOM   1801  O   ASN B 134      33.284  26.782  22.141  1.00  0.00           O  \\nATOM   1802  CB  ASN B 134      34.623  27.836  24.874  1.00  0.00           C  \\nATOM   1803  CG  ASN B 134      35.708  27.983  23.795  1.00  0.00           C  \\nATOM   1804  OD1 ASN B 134      35.798  28.989  23.093  1.00  0.00           O  \\nATOM   1805  ND2 ASN B 134      36.511  26.988  23.645  1.00  0.00           N  \\nATOM   1806  H   ASN B 134      32.454  27.837  26.060  1.00  0.00           H  \\nATOM   1807  HA  ASN B 134      33.192  28.697  23.729  1.00  0.00           H  \\nATOM   1808 1HB  ASN B 134      34.732  28.632  25.614  1.00  0.00           H  \\nATOM   1809 2HB  ASN B 134      34.761  26.887  25.391  1.00  0.00           H  \\nATOM   1810 1HD2 ASN B 134      37.216  27.026  22.940  1.00  0.00           H  \\nATOM   1811 2HD2 ASN B 134      36.408  26.174  24.235  1.00  0.00           H  \\nATOM   1812  N   GLN B 135      32.466  25.631  23.811  1.00  0.00           N  \\nATOM   1813  CA  GLN B 135      32.158  24.505  22.980  1.00  0.00           C  \\nATOM   1814  C   GLN B 135      30.944  24.761  22.111  1.00  0.00           C  \\nATOM   1815  O   GLN B 135      30.899  24.317  21.012  1.00  0.00           O  \\nATOM   1816  CB  GLN B 135      31.964  23.256  23.817  1.00  0.00           C  \\nATOM   1817  CG  GLN B 135      33.238  22.762  24.445  1.00  0.00           C  \\nATOM   1818  CD  GLN B 135      33.017  21.623  25.393  1.00  0.00           C  \\nATOM   1819  OE1 GLN B 135      32.181  20.741  25.163  1.00  0.00           O  \\nATOM   1820  NE2 GLN B 135      33.780  21.623  26.465  1.00  0.00           N  \\nATOM   1821  H   GLN B 135      32.253  25.527  24.630  1.00  0.00           H  \\nATOM   1822  HA  GLN B 135      32.916  24.341  22.397  1.00  0.00           H  \\nATOM   1823 1HB  GLN B 135      31.243  23.448  24.604  1.00  0.00           H  \\nATOM   1824 2HB  GLN B 135      31.567  22.489  23.194  1.00  0.00           H  \\nATOM   1825 1HG  GLN B 135      33.899  22.416  23.652  1.00  0.00           H  \\nATOM   1826 2HG  GLN B 135      33.717  23.573  24.982  1.00  0.00           H  \\nATOM   1827 1HE2 GLN B 135      33.721  20.880  27.137  1.00  0.00           H  \\nATOM   1828 2HE2 GLN B 135      34.449  22.362  26.592  1.00  0.00           H  \\nATOM   1829  N   LYS B 136      29.964  25.467  22.630  1.00  0.00           N  \\nATOM   1830  CA  LYS B 136      28.813  25.890  21.838  1.00  0.00           C  \\nATOM   1831  C   LYS B 136      29.265  26.658  20.603  1.00  0.00           C  \\nATOM   1832  O   LYS B 136      28.810  26.394  19.491  1.00  0.00           O  \\nATOM   1833  CB  LYS B 136      27.820  26.698  22.674  1.00  0.00           C  \\nATOM   1834  CG  LYS B 136      26.992  25.834  23.644  1.00  0.00           C  \\nATOM   1835  CD  LYS B 136      26.125  26.675  24.580  1.00  0.00           C  \\nATOM   1836  CE  LYS B 136      25.021  27.403  23.824  1.00  0.00           C  \\nATOM   1837  NZ  LYS B 136      24.136  28.175  24.743  1.00  0.00           N  \\nATOM   1838  H   LYS B 136      29.942  25.728  23.449  1.00  0.00           H  \\nATOM   1839  HA  LYS B 136      28.320  25.107  21.547  1.00  0.00           H  \\nATOM   1840 1HB  LYS B 136      28.347  27.441  23.253  1.00  0.00           H  \\nATOM   1841 2HB  LYS B 136      27.134  27.224  22.014  1.00  0.00           H  \\nATOM   1842 1HG  LYS B 136      26.350  25.168  23.072  1.00  0.00           H  \\nATOM   1843 2HG  LYS B 136      27.667  25.227  24.240  1.00  0.00           H  \\nATOM   1844 1HD  LYS B 136      25.678  26.027  25.335  1.00  0.00           H  \\nATOM   1845 2HD  LYS B 136      26.747  27.409  25.086  1.00  0.00           H  \\nATOM   1846 1HE  LYS B 136      25.473  28.089  23.110  1.00  0.00           H  \\nATOM   1847 2HE  LYS B 136      24.418  26.674  23.283  1.00  0.00           H  \\nATOM   1848 1HZ  LYS B 136      23.419  28.643  24.206  1.00  0.00           H  \\nATOM   1849 2HZ  LYS B 136      23.703  27.544  25.404  1.00  0.00           H  \\nATOM   1850 3HZ  LYS B 136      24.683  28.862  25.244  1.00  0.00           H  \\nATOM   1851  N   ALA B 137      30.166  27.611  20.813  1.00  0.00           N  \\nATOM   1852  CA  ALA B 137      30.680  28.437  19.731  1.00  0.00           C  \\nATOM   1853  C   ALA B 137      31.405  27.587  18.694  1.00  0.00           C  \\nATOM   1854  O   ALA B 137      31.184  27.744  17.495  1.00  0.00           O  \\nATOM   1855  CB  ALA B 137      31.642  29.424  20.343  1.00  0.00           C  \\nATOM   1856  H   ALA B 137      30.498  27.800  21.584  1.00  0.00           H  \\nATOM   1857  HA  ALA B 137      29.938  28.880  19.291  1.00  0.00           H  \\nATOM   1858 1HB  ALA B 137      32.021  30.085  19.597  1.00  0.00           H  \\nATOM   1859 2HB  ALA B 137      31.117  29.962  21.097  1.00  0.00           H  \\nATOM   1860 3HB  ALA B 137      32.467  28.890  20.815  1.00  0.00           H  \\nATOM   1861  N   LEU B 138      32.269  26.689  19.163  1.00  0.00           N  \\nATOM   1862  CA  LEU B 138      33.019  25.802  18.277  1.00  0.00           C  \\nATOM   1863  C   LEU B 138      32.081  25.013  17.374  1.00  0.00           C  \\nATOM   1864  O   LEU B 138      32.362  24.818  16.192  1.00  0.00           O  \\nATOM   1865  CB  LEU B 138      33.941  24.857  19.074  1.00  0.00           C  \\nATOM   1866  CG  LEU B 138      35.199  25.527  19.740  1.00  0.00           C  \\nATOM   1867  CD1 LEU B 138      35.921  24.512  20.649  1.00  0.00           C  \\nATOM   1868  CD2 LEU B 138      36.146  26.030  18.647  1.00  0.00           C  \\nATOM   1869  H   LEU B 138      32.440  26.572  19.998  1.00  0.00           H  \\nATOM   1870  HA  LEU B 138      33.600  26.336  17.713  1.00  0.00           H  \\nATOM   1871 1HB  LEU B 138      33.350  24.401  19.866  1.00  0.00           H  \\nATOM   1872 2HB  LEU B 138      34.290  24.069  18.410  1.00  0.00           H  \\nATOM   1873  HG  LEU B 138      34.883  26.370  20.348  1.00  0.00           H  \\nATOM   1874 1HD1 LEU B 138      36.791  24.986  21.111  1.00  0.00           H  \\nATOM   1875 2HD1 LEU B 138      35.239  24.174  21.428  1.00  0.00           H  \\nATOM   1876 3HD1 LEU B 138      36.243  23.663  20.055  1.00  0.00           H  \\nATOM   1877 1HD2 LEU B 138      37.010  26.502  19.110  1.00  0.00           H  \\nATOM   1878 2HD2 LEU B 138      36.481  25.192  18.031  1.00  0.00           H  \\nATOM   1879 3HD2 LEU B 138      35.627  26.754  18.024  1.00  0.00           H  \\nATOM   1880  N   ALA B 139      30.965  24.566  17.940  1.00  0.00           N  \\nATOM   1881  CA  ALA B 139      29.960  23.830  17.185  1.00  0.00           C  \\nATOM   1882  C   ALA B 139      29.252  24.748  16.192  1.00  0.00           C  \\nATOM   1883  O   ALA B 139      28.191  24.410  15.667  1.00  0.00           O  \\nATOM   1884  OXT ALA B 139      29.729  25.814  15.915  1.00  0.00           O  \\nATOM   1885  CB  ALA B 139      28.948  23.220  18.146  1.00  0.00           C  \\nATOM   1886  H   ALA B 139      30.765  24.677  18.769  1.00  0.00           H  \\nATOM   1887  HA  ALA B 139      30.395  23.122  16.683  1.00  0.00           H  \\nATOM   1888 1HB  ALA B 139      28.205  22.665  17.584  1.00  0.00           H  \\nATOM   1889 2HB  ALA B 139      29.451  22.560  18.845  1.00  0.00           H  \\nATOM   1890 3HB  ALA B 139      28.458  24.016  18.702  1.00  0.00           H  \\nTER                                                                             \\nATOM   1892  N   THR C 140      27.512  14.029  12.268  1.00  0.00           N  \\nATOM   1893  CA  THR C 140      28.049  14.350  13.620  1.00  0.00           C  \\nATOM   1894  C   THR C 140      27.258  15.481  14.277  1.00  0.00           C  \\nATOM   1895  O   THR C 140      27.635  15.974  15.340  1.00  0.00           O  \\nATOM   1896  CB  THR C 140      29.530  14.766  13.543  1.00  0.00           C  \\nATOM   1897  OG1 THR C 140      30.116  14.723  14.854  1.00  0.00           O  \\nATOM   1898  CG2 THR C 140      29.638  16.187  13.002  1.00  0.00           C  \\nATOM   1899 1H   THR C 140      28.048  13.288  11.863  1.00  0.00           H  \\nATOM   1900 2H   THR C 140      26.557  13.745  12.347  1.00  0.00           H  \\nATOM   1901 3H   THR C 140      27.571  14.839  11.684  1.00  0.00           H  \\nATOM   1902  HA  THR C 140      27.974  13.564  14.184  1.00  0.00           H  \\nATOM   1903  HB  THR C 140      30.064  14.078  12.889  1.00  0.00           H  \\nATOM   1904  HG1 THR C 140      30.096  15.607  15.239  1.00  0.00           H  \\nATOM   1905 1HG2 THR C 140      30.686  16.478  12.951  1.00  0.00           H  \\nATOM   1906 2HG2 THR C 140      29.197  16.234  12.007  1.00  0.00           H  \\nATOM   1907 3HG2 THR C 140      29.106  16.866  13.664  1.00  0.00           H  \\nATOM   1908  N   LYS C 141      26.158  15.882  13.645  1.00  0.00           N  \\nATOM   1909  CA  LYS C 141      25.350  16.991  14.142  1.00  0.00           C  \\nATOM   1910  C   LYS C 141      24.547  16.563  15.360  1.00  0.00           C  \\nATOM   1911  O   LYS C 141      24.509  17.274  16.363  1.00  0.00           O  \\nATOM   1912  CB  LYS C 141      24.435  17.544  13.051  1.00  0.00           C  \\nATOM   1913  CG  LYS C 141      24.895  18.897  12.440  1.00  0.00           C  \\nATOM   1914  CD  LYS C 141      26.339  18.862  11.924  1.00  0.00           C  \\nATOM   1915  CE  LYS C 141      27.315  19.561  12.892  1.00  0.00           C  \\nATOM   1916  NZ  LYS C 141      27.163  21.065  12.876  1.00  0.00           N  \\nATOM   1917  H   LYS C 141      25.857  15.527  12.922  1.00  0.00           H  \\nATOM   1918  HA  LYS C 141      25.936  17.716  14.407  1.00  0.00           H  \\nATOM   1919 1HB  LYS C 141      24.351  16.822  12.241  1.00  0.00           H  \\nATOM   1920 2HB  LYS C 141      23.435  17.691  13.462  1.00  0.00           H  \\nATOM   1921 1HG  LYS C 141      24.234  19.158  11.613  1.00  0.00           H  \\nATOM   1922 2HG  LYS C 141      24.813  19.674  13.200  1.00  0.00           H  \\nATOM   1923 1HD  LYS C 141      26.654  17.829  11.793  1.00  0.00           H  \\nATOM   1924 2HD  LYS C 141      26.388  19.362  10.956  1.00  0.00           H  \\nATOM   1925 1HE  LYS C 141      27.132  19.198  13.904  1.00  0.00           H  \\nATOM   1926 2HE  LYS C 141      28.336  19.308  12.607  1.00  0.00           H  \\nATOM   1927 1HZ  LYS C 141      27.826  21.473  13.523  1.00  0.00           H  \\nATOM   1928 2HZ  LYS C 141      26.225  21.320  13.152  1.00  0.00           H  \\nATOM   1929 3HZ  LYS C 141      27.344  21.417  11.947  1.00  0.00           H  \\nATOM   1930  N   TYR C 142      23.902  15.403  15.271  1.00  0.00           N  \\nATOM   1931  CA  TYR C 142      23.134  14.886  16.395  1.00  0.00           C  \\nATOM   1932  C   TYR C 142      24.038  14.667  17.598  1.00  0.00           C  \\nATOM   1933  O   TYR C 142      23.651  14.944  18.729  1.00  0.00           O  \\nATOM   1934  CB  TYR C 142      22.445  13.573  16.035  1.00  0.00           C  \\nATOM   1935  CG  TYR C 142      21.591  13.023  17.159  1.00  0.00           C  \\nATOM   1936  CD1 TYR C 142      20.331  13.554  17.396  1.00  0.00           C  \\nATOM   1937  CD2 TYR C 142      22.070  11.992  17.956  1.00  0.00           C  \\nATOM   1938  CE1 TYR C 142      19.551  13.052  18.419  1.00  0.00           C  \\nATOM   1939  CE2 TYR C 142      21.290  11.493  18.981  1.00  0.00           C  \\nATOM   1940  CZ  TYR C 142      20.034  12.019  19.212  1.00  0.00           C  \\nATOM   1941  OH  TYR C 142      19.252  11.519  20.228  1.00  0.00           O  \\nATOM   1942  H   TYR C 142      23.895  14.899  14.574  1.00  0.00           H  \\nATOM   1943  HA  TYR C 142      22.457  15.536  16.641  1.00  0.00           H  \\nATOM   1944 1HB  TYR C 142      21.810  13.720  15.161  1.00  0.00           H  \\nATOM   1945 2HB  TYR C 142      23.194  12.826  15.779  1.00  0.00           H  \\nATOM   1946  HD1 TYR C 142      19.955  14.364  16.771  1.00  0.00           H  \\nATOM   1947  HD2 TYR C 142      23.062  11.579  17.774  1.00  0.00           H  \\nATOM   1948  HE1 TYR C 142      18.562  13.469  18.604  1.00  0.00           H  \\nATOM   1949  HE2 TYR C 142      21.664  10.682  19.607  1.00  0.00           H  \\nATOM   1950  HH  TYR C 142      18.429  12.012  20.269  1.00  0.00           H  \\nATOM   1951  N   LYS C 143      25.242  14.166  17.344  1.00  0.00           N  \\nATOM   1952  CA  LYS C 143      26.176  13.844  18.414  1.00  0.00           C  \\nATOM   1953  C   LYS C 143      26.519  15.091  19.227  1.00  0.00           C  \\nATOM   1954  O   LYS C 143      26.355  15.108  20.443  1.00  0.00           O  \\nATOM   1955  CB  LYS C 143      27.395  13.132  17.820  1.00  0.00           C  \\nATOM   1956  CG  LYS C 143      28.465  12.712  18.805  1.00  0.00           C  \\nATOM   1957  CD  LYS C 143      29.434  11.685  18.160  1.00  0.00           C  \\nATOM   1958  CE  LYS C 143      30.161  12.225  16.936  1.00  0.00           C  \\nATOM   1959  NZ  LYS C 143      31.145  11.238  16.393  1.00  0.00           N  \\nATOM   1960  H   LYS C 143      25.543  14.004  16.555  1.00  0.00           H  \\nATOM   1961  HA  LYS C 143      25.762  13.201  19.011  1.00  0.00           H  \\nATOM   1962 1HB  LYS C 143      27.068  12.243  17.283  1.00  0.00           H  \\nATOM   1963 2HB  LYS C 143      27.873  13.794  17.092  1.00  0.00           H  \\nATOM   1964 1HG  LYS C 143      29.016  13.591  19.124  1.00  0.00           H  \\nATOM   1965 2HG  LYS C 143      27.998  12.263  19.680  1.00  0.00           H  \\nATOM   1966 1HD  LYS C 143      30.180  11.390  18.900  1.00  0.00           H  \\nATOM   1967 2HD  LYS C 143      28.873  10.798  17.867  1.00  0.00           H  \\nATOM   1968 1HE  LYS C 143      29.430  12.456  16.165  1.00  0.00           H  \\nATOM   1969 2HE  LYS C 143      30.683  13.121  17.194  1.00  0.00           H  \\nATOM   1970 1HZ  LYS C 143      31.836  11.027  17.096  1.00  0.00           H  \\nATOM   1971 2HZ  LYS C 143      30.664  10.390  16.127  1.00  0.00           H  \\nATOM   1972 3HZ  LYS C 143      31.600  11.637  15.583  1.00  0.00           H  \\nATOM   1973  N   ILE C 144      26.969  16.140  18.545  1.00  0.00           N  \\nATOM   1974  CA  ILE C 144      27.376  17.376  19.207  1.00  0.00           C  \\nATOM   1975  C   ILE C 144      26.234  17.997  20.009  1.00  0.00           C  \\nATOM   1976  O   ILE C 144      26.404  18.333  21.180  1.00  0.00           O  \\nATOM   1977  CB  ILE C 144      27.923  18.383  18.181  1.00  0.00           C  \\nATOM   1978  CG1 ILE C 144      29.240  17.858  17.614  1.00  0.00           C  \\nATOM   1979  CG2 ILE C 144      28.139  19.733  18.828  1.00  0.00           C  \\nATOM   1980  CD1 ILE C 144      29.731  18.587  16.397  1.00  0.00           C  \\nATOM   1981  H   ILE C 144      27.050  16.161  17.689  1.00  0.00           H  \\nATOM   1982  HA  ILE C 144      28.096  17.175  19.825  1.00  0.00           H  \\nATOM   1983  HB  ILE C 144      27.214  18.474  17.352  1.00  0.00           H  \\nATOM   1984 1HG1 ILE C 144      30.006  17.930  18.389  1.00  0.00           H  \\nATOM   1985 2HG1 ILE C 144      29.104  16.814  17.348  1.00  0.00           H  \\nATOM   1986 1HG2 ILE C 144      28.523  20.406  18.081  1.00  0.00           H  \\nATOM   1987 2HG2 ILE C 144      27.196  20.110  19.214  1.00  0.00           H  \\nATOM   1988 3HG2 ILE C 144      28.855  19.645  19.643  1.00  0.00           H  \\nATOM   1989 1HD1 ILE C 144      29.887  19.636  16.638  1.00  0.00           H  \\nATOM   1990 2HD1 ILE C 144      30.673  18.142  16.063  1.00  0.00           H  \\nATOM   1991 3HD1 ILE C 144      28.987  18.501  15.609  1.00  0.00           H  \\nATOM   1992  N   LYS C 145      25.071  18.143  19.385  1.00  0.00           N  \\nATOM   1993  CA  LYS C 145      23.953  18.817  20.035  1.00  0.00           C  \\nATOM   1994  C   LYS C 145      23.334  17.972  21.148  1.00  0.00           C  \\nATOM   1995  O   LYS C 145      22.797  18.512  22.116  1.00  0.00           O  \\nATOM   1996  CB  LYS C 145      22.897  19.147  18.987  1.00  0.00           C  \\nATOM   1997  CG  LYS C 145      23.328  20.217  17.996  1.00  0.00           C  \\nATOM   1998  CD  LYS C 145      22.317  20.370  16.867  1.00  0.00           C  \\nATOM   1999  CE  LYS C 145      21.022  21.007  17.355  1.00  0.00           C  \\nATOM   2000  NZ  LYS C 145      20.066  21.248  16.238  1.00  0.00           N  \\nATOM   2001  H   LYS C 145      24.904  17.864  18.589  1.00  0.00           H  \\nATOM   2002  HA  LYS C 145      24.277  19.638  20.436  1.00  0.00           H  \\nATOM   2003 1HB  LYS C 145      22.666  18.240  18.418  1.00  0.00           H  \\nATOM   2004 2HB  LYS C 145      21.982  19.473  19.476  1.00  0.00           H  \\nATOM   2005 1HG  LYS C 145      23.447  21.169  18.511  1.00  0.00           H  \\nATOM   2006 2HG  LYS C 145      24.290  19.928  17.568  1.00  0.00           H  \\nATOM   2007 1HD  LYS C 145      22.745  20.991  16.079  1.00  0.00           H  \\nATOM   2008 2HD  LYS C 145      22.099  19.385  16.448  1.00  0.00           H  \\nATOM   2009 1HE  LYS C 145      20.554  20.348  18.084  1.00  0.00           H  \\nATOM   2010 2HE  LYS C 145      21.251  21.959  17.834  1.00  0.00           H  \\nATOM   2011 1HZ  LYS C 145      20.487  21.868  15.560  1.00  0.00           H  \\nATOM   2012 2HZ  LYS C 145      19.838  20.369  15.793  1.00  0.00           H  \\nATOM   2013 3HZ  LYS C 145      19.223  21.669  16.601  1.00  0.00           H  \\nATOM   2014  N   GLU C 146      23.417  16.652  21.020  1.00  0.00           N  \\nATOM   2015  CA  GLU C 146      22.892  15.766  22.051  1.00  0.00           C  \\nATOM   2016  C   GLU C 146      23.785  15.817  23.285  1.00  0.00           C  \\nATOM   2017  O   GLU C 146      23.301  15.781  24.415  1.00  0.00           O  \\nATOM   2018  CB  GLU C 146      22.780  14.312  21.545  1.00  0.00           C  \\nATOM   2019  CG  GLU C 146      21.986  13.311  22.444  1.00  0.00           C  \\nATOM   2020  CD  GLU C 146      22.757  12.619  23.587  1.00  0.00           C  \\nATOM   2021  OE1 GLU C 146      23.965  12.450  23.512  1.00  0.00           O  \\nATOM   2022  OE2 GLU C 146      22.095  12.224  24.522  1.00  0.00           O  \\nATOM   2023  H   GLU C 146      23.771  16.247  20.350  1.00  0.00           H  \\nATOM   2024  HA  GLU C 146      22.006  16.065  22.306  1.00  0.00           H  \\nATOM   2025 1HB  GLU C 146      22.256  14.313  20.595  1.00  0.00           H  \\nATOM   2026 2HB  GLU C 146      23.779  13.907  21.366  1.00  0.00           H  \\nATOM   2027 1HG  GLU C 146      21.150  13.854  22.881  1.00  0.00           H  \\nATOM   2028 2HG  GLU C 146      21.569  12.543  21.801  1.00  0.00           H  \\nATOM   2029  N   THR C 147      25.092  15.906  23.062  1.00  0.00           N  \\nATOM   2030  CA  THR C 147      26.049  15.988  24.156  1.00  0.00           C  \\nATOM   2031  C   THR C 147      25.979  17.349  24.847  1.00  0.00           C  \\nATOM   2032  O   THR C 147      26.129  17.442  26.064  1.00  0.00           O  \\nATOM   2033  CB  THR C 147      27.518  15.732  23.739  1.00  0.00           C  \\nATOM   2034  OG1 THR C 147      27.912  16.658  22.719  1.00  0.00           O  \\nATOM   2035  CG2 THR C 147      27.745  14.302  23.298  1.00  0.00           C  \\nATOM   2036  H   THR C 147      25.450  15.920  22.281  1.00  0.00           H  \\nATOM   2037  HA  THR C 147      25.836  15.307  24.814  1.00  0.00           H  \\nATOM   2038  HB  THR C 147      28.158  15.906  24.604  1.00  0.00           H  \\nATOM   2039  HG1 THR C 147      27.136  16.967  22.205  1.00  0.00           H  \\nATOM   2040 1HG2 THR C 147      28.798  14.165  23.043  1.00  0.00           H  \\nATOM   2041 2HG2 THR C 147      27.482  13.630  24.113  1.00  0.00           H  \\nATOM   2042 3HG2 THR C 147      27.135  14.072  22.441  1.00  0.00           H  \\nATOM   2043  N   LEU C 148      25.741  18.400  24.070  1.00  0.00           N  \\nATOM   2044  CA  LEU C 148      25.580  19.736  24.635  1.00  0.00           C  \\nATOM   2045  C   LEU C 148      24.374  19.792  25.569  1.00  0.00           C  \\nATOM   2046  O   LEU C 148      24.414  20.459  26.601  1.00  0.00           O  \\nATOM   2047  CB  LEU C 148      25.500  20.805  23.532  1.00  0.00           C  \\nATOM   2048  CG  LEU C 148      26.849  21.106  22.765  1.00  0.00           C  \\nATOM   2049  CD1 LEU C 148      26.568  22.016  21.569  1.00  0.00           C  \\nATOM   2050  CD2 LEU C 148      27.845  21.779  23.725  1.00  0.00           C  \\nATOM   2051  H   LEU C 148      25.667  18.368  23.214  1.00  0.00           H  \\nATOM   2052  HA  LEU C 148      26.370  19.958  25.152  1.00  0.00           H  \\nATOM   2053 1HB  LEU C 148      24.769  20.475  22.796  1.00  0.00           H  \\nATOM   2054 2HB  LEU C 148      25.148  21.736  23.973  1.00  0.00           H  \\nATOM   2055  HG  LEU C 148      27.283  20.182  22.401  1.00  0.00           H  \\nATOM   2056 1HD1 LEU C 148      27.501  22.219  21.040  1.00  0.00           H  \\nATOM   2057 2HD1 LEU C 148      25.870  21.521  20.896  1.00  0.00           H  \\nATOM   2058 3HD1 LEU C 148      26.138  22.952  21.914  1.00  0.00           H  \\nATOM   2059 1HD2 LEU C 148      28.774  21.989  23.197  1.00  0.00           H  \\nATOM   2060 2HD2 LEU C 148      27.425  22.711  24.095  1.00  0.00           H  \\nATOM   2061 3HD2 LEU C 148      28.046  21.112  24.563  1.00  0.00           H  \\nATOM   2062  N   LYS C 149      23.305  19.089  25.209  1.00  0.00           N  \\nATOM   2063  CA  LYS C 149      22.102  19.068  26.033  1.00  0.00           C  \\nATOM   2064  C   LYS C 149      22.358  18.315  27.333  1.00  0.00           C  \\nATOM   2065  O   LYS C 149      21.866  18.703  28.391  1.00  0.00           O  \\nATOM   2066  CB  LYS C 149      20.947  18.421  25.270  1.00  0.00           C  \\nATOM   2067  CG  LYS C 149      19.600  18.478  25.989  1.00  0.00           C  \\nATOM   2068  CD  LYS C 149      18.484  17.889  25.127  1.00  0.00           C  \\nATOM   2069  CE  LYS C 149      18.165  18.785  23.936  1.00  0.00           C  \\nATOM   2070  NZ  LYS C 149      16.999  18.279  23.157  1.00  0.00           N  \\nATOM   2071  H   LYS C 149      23.250  18.616  24.493  1.00  0.00           H  \\nATOM   2072  HA  LYS C 149      21.855  19.979  26.257  1.00  0.00           H  \\nATOM   2073 1HB  LYS C 149      20.842  18.904  24.297  1.00  0.00           H  \\nATOM   2074 2HB  LYS C 149      21.187  17.370  25.082  1.00  0.00           H  \\nATOM   2075 1HG  LYS C 149      19.668  17.910  26.922  1.00  0.00           H  \\nATOM   2076 2HG  LYS C 149      19.360  19.513  26.234  1.00  0.00           H  \\nATOM   2077 1HD  LYS C 149      18.795  16.909  24.759  1.00  0.00           H  \\nATOM   2078 2HD  LYS C 149      17.585  17.763  25.731  1.00  0.00           H  \\nATOM   2079 1HE  LYS C 149      17.942  19.789  24.294  1.00  0.00           H  \\nATOM   2080 2HE  LYS C 149      19.036  18.828  23.280  1.00  0.00           H  \\nATOM   2081 1HZ  LYS C 149      16.820  18.897  22.378  1.00  0.00           H  \\nATOM   2082 2HZ  LYS C 149      17.201  17.351  22.810  1.00  0.00           H  \\nATOM   2083 3HZ  LYS C 149      16.185  18.245  23.753  1.00  0.00           H  \\nATOM   2084  N   ARG C 150      23.123  17.231  27.249  1.00  0.00           N  \\nATOM   2085  CA  ARG C 150      23.506  16.481  28.438  1.00  0.00           C  \\nATOM   2086  C   ARG C 150      24.492  17.283  29.277  1.00  0.00           C  \\nATOM   2087  O   ARG C 150      24.459  17.240  30.507  1.00  0.00           O  \\nATOM   2088  CB  ARG C 150      24.156  15.166  28.052  1.00  0.00           C  \\nATOM   2089  CG  ARG C 150      23.241  14.106  27.505  1.00  0.00           C  \\nATOM   2090  CD  ARG C 150      24.021  13.032  26.841  1.00  0.00           C  \\nATOM   2091  NE  ARG C 150      24.995  12.412  27.707  1.00  0.00           N  \\nATOM   2092  CZ  ARG C 150      26.094  11.787  27.239  1.00  0.00           C  \\nATOM   2093  NH1 ARG C 150      26.292  11.705  25.933  1.00  0.00           N  \\nATOM   2094  NH2 ARG C 150      26.983  11.272  28.082  1.00  0.00           N  \\nATOM   2095  H   ARG C 150      23.432  16.910  26.514  1.00  0.00           H  \\nATOM   2096  HA  ARG C 150      22.717  16.311  28.976  1.00  0.00           H  \\nATOM   2097 1HB  ARG C 150      24.923  15.352  27.300  1.00  0.00           H  \\nATOM   2098 2HB  ARG C 150      24.649  14.741  28.918  1.00  0.00           H  \\nATOM   2099 1HG  ARG C 150      22.645  13.671  28.305  1.00  0.00           H  \\nATOM   2100 2HG  ARG C 150      22.581  14.549  26.752  1.00  0.00           H  \\nATOM   2101 1HD  ARG C 150      23.339  12.258  26.521  1.00  0.00           H  \\nATOM   2102 2HD  ARG C 150      24.538  13.440  25.970  1.00  0.00           H  \\nATOM   2103  HE  ARG C 150      24.860  12.470  28.705  1.00  0.00           H  \\nATOM   2104 1HH1 ARG C 150      25.590  12.078  25.259  1.00  0.00           H  \\nATOM   2105 2HH1 ARG C 150      27.158  11.275  25.583  1.00  0.00           H  \\nATOM   2106 1HH2 ARG C 150      26.837  11.340  29.078  1.00  0.00           H  \\nATOM   2107 2HH2 ARG C 150      27.841  10.812  27.722  1.00  0.00           H  \\nATOM   2108  N   LEU C 151      25.372  18.012  28.601  1.00  0.00           N  \\nATOM   2109  CA  LEU C 151      26.342  18.860  29.274  1.00  0.00           C  \\nATOM   2110  C   LEU C 151      25.602  19.964  30.026  1.00  0.00           C  \\nATOM   2111  O   LEU C 151      25.968  20.323  31.144  1.00  0.00           O  \\nATOM   2112  CB  LEU C 151      27.305  19.483  28.232  1.00  0.00           C  \\nATOM   2113  CG  LEU C 151      28.513  20.260  28.755  1.00  0.00           C  \\nATOM   2114  CD1 LEU C 151      29.370  19.309  29.563  1.00  0.00           C  \\nATOM   2115  CD2 LEU C 151      29.311  20.847  27.569  1.00  0.00           C  \\nATOM   2116  H   LEU C 151      25.427  18.031  27.743  1.00  0.00           H  \\nATOM   2117  HA  LEU C 151      26.841  18.332  29.917  1.00  0.00           H  \\nATOM   2118 1HB  LEU C 151      27.678  18.698  27.589  1.00  0.00           H  \\nATOM   2119 2HB  LEU C 151      26.733  20.170  27.620  1.00  0.00           H  \\nATOM   2120  HG  LEU C 151      28.173  21.063  29.415  1.00  0.00           H  \\nATOM   2121 1HD1 LEU C 151      30.232  19.836  29.972  1.00  0.00           H  \\nATOM   2122 2HD1 LEU C 151      28.776  18.900  30.359  1.00  0.00           H  \\nATOM   2123 3HD1 LEU C 151      29.712  18.501  28.926  1.00  0.00           H  \\nATOM   2124 1HD2 LEU C 151      30.180  21.392  27.942  1.00  0.00           H  \\nATOM   2125 2HD2 LEU C 151      29.649  20.037  26.916  1.00  0.00           H  \\nATOM   2126 3HD2 LEU C 151      28.674  21.524  27.001  1.00  0.00           H  \\nATOM   2127  N   GLU C 152      24.544  20.481  29.410  1.00  0.00           N  \\nATOM   2128  CA  GLU C 152      23.701  21.494  30.035  1.00  0.00           C  \\nATOM   2129  C   GLU C 152      23.030  20.945  31.288  1.00  0.00           C  \\nATOM   2130  O   GLU C 152      23.015  21.602  32.330  1.00  0.00           O  \\nATOM   2131  CB  GLU C 152      22.630  22.058  29.091  1.00  0.00           C  \\nATOM   2132  CG  GLU C 152      21.787  23.187  29.759  1.00  0.00           C  \\nATOM   2133  CD  GLU C 152      20.744  23.858  28.864  1.00  0.00           C  \\nATOM   2134  OE1 GLU C 152      20.618  23.502  27.718  1.00  0.00           O  \\nATOM   2135  OE2 GLU C 152      20.073  24.735  29.357  1.00  0.00           O  \\nATOM   2136  H   GLU C 152      24.291  20.259  28.618  1.00  0.00           H  \\nATOM   2137  HA  GLU C 152      24.252  22.250  30.294  1.00  0.00           H  \\nATOM   2138 1HB  GLU C 152      23.100  22.452  28.191  1.00  0.00           H  \\nATOM   2139 2HB  GLU C 152      21.952  21.263  28.789  1.00  0.00           H  \\nATOM   2140 1HG  GLU C 152      21.274  22.756  30.620  1.00  0.00           H  \\nATOM   2141 2HG  GLU C 152      22.472  23.947  30.132  1.00  0.00           H  \\nATOM   2142  N   ASP C 153      22.467  19.746  31.182  1.00  0.00           N  \\nATOM   2143  CA  ASP C 153      21.804  19.123  32.322  1.00  0.00           C  \\nATOM   2144  C   ASP C 153      22.816  18.853  33.429  1.00  0.00           C  \\nATOM   2145  O   ASP C 153      22.550  19.111  34.601  1.00  0.00           O  \\nATOM   2146  CB  ASP C 153      21.132  17.794  31.937  1.00  0.00           C  \\nATOM   2147  CG  ASP C 153      19.803  17.890  31.108  1.00  0.00           C  \\nATOM   2148  OD1 ASP C 153      19.195  18.954  31.022  1.00  0.00           O  \\nATOM   2149  OD2 ASP C 153      19.385  16.867  30.616  1.00  0.00           O  \\nATOM   2150  H   ASP C 153      22.454  19.274  30.463  1.00  0.00           H  \\nATOM   2151  HA  ASP C 153      21.132  19.731  32.669  1.00  0.00           H  \\nATOM   2152 1HB  ASP C 153      21.843  17.207  31.354  1.00  0.00           H  \\nATOM   2153 2HB  ASP C 153      20.927  17.229  32.844  1.00  0.00           H  \\nATOM   2154  N   SER C 154      23.985  18.350  33.047  1.00  0.00           N  \\nATOM   2155  CA  SER C 154      25.066  18.115  33.994  1.00  0.00           C  \\nATOM   2156  C   SER C 154      25.452  19.391  34.742  1.00  0.00           C  \\nATOM   2157  O   SER C 154      25.744  19.357  35.936  1.00  0.00           O  \\nATOM   2158  CB  SER C 154      26.257  17.521  33.296  1.00  0.00           C  \\nATOM   2159  OG  SER C 154      27.255  17.220  34.219  1.00  0.00           O  \\nATOM   2160  H   SER C 154      24.177  18.134  32.237  1.00  0.00           H  \\nATOM   2161  HA  SER C 154      24.776  17.460  34.648  1.00  0.00           H  \\nATOM   2162 1HB  SER C 154      25.952  16.622  32.772  1.00  0.00           H  \\nATOM   2163 2HB  SER C 154      26.633  18.220  32.556  1.00  0.00           H  \\nATOM   2164  HG  SER C 154      26.875  16.565  34.806  1.00  0.00           H  \\nATOM   2165  N   LEU C 155      25.456  20.516  34.037  1.00  0.00           N  \\nATOM   2166  CA  LEU C 155      25.805  21.787  34.656  1.00  0.00           C  \\nATOM   2167  C   LEU C 155      24.724  22.225  35.635  1.00  0.00           C  \\nATOM   2168  O   LEU C 155      25.029  22.751  36.704  1.00  0.00           O  \\nATOM   2169  CB  LEU C 155      26.092  22.861  33.597  1.00  0.00           C  \\nATOM   2170  CG  LEU C 155      27.435  22.690  32.813  1.00  0.00           C  \\nATOM   2171  CD1 LEU C 155      27.478  23.634  31.636  1.00  0.00           C  \\nATOM   2172  CD2 LEU C 155      28.601  23.010  33.749  1.00  0.00           C  \\nATOM   2173  H   LEU C 155      25.262  20.570  33.201  1.00  0.00           H  \\nATOM   2174  HA  LEU C 155      26.632  21.679  35.151  1.00  0.00           H  \\nATOM   2175 1HB  LEU C 155      25.282  22.845  32.868  1.00  0.00           H  \\nATOM   2176 2HB  LEU C 155      26.105  23.832  34.080  1.00  0.00           H  \\nATOM   2177  HG  LEU C 155      27.529  21.668  32.452  1.00  0.00           H  \\nATOM   2178 1HD1 LEU C 155      28.421  23.501  31.121  1.00  0.00           H  \\nATOM   2179 2HD1 LEU C 155      26.654  23.405  30.959  1.00  0.00           H  \\nATOM   2180 3HD1 LEU C 155      27.392  24.659  31.983  1.00  0.00           H  \\nATOM   2181 1HD2 LEU C 155      29.536  22.891  33.206  1.00  0.00           H  \\nATOM   2182 2HD2 LEU C 155      28.519  24.035  34.108  1.00  0.00           H  \\nATOM   2183 3HD2 LEU C 155      28.582  22.336  34.582  1.00  0.00           H  \\nATOM   2184  N   ARG C 156      23.464  22.005  35.274  1.00  0.00           N  \\nATOM   2185  CA  ARG C 156      22.361  22.335  36.166  1.00  0.00           C  \\nATOM   2186  C   ARG C 156      22.489  21.544  37.463  1.00  0.00           C  \\nATOM   2187  O   ARG C 156      22.236  22.068  38.546  1.00  0.00           O  \\nATOM   2188  CB  ARG C 156      21.025  22.069  35.485  1.00  0.00           C  \\nATOM   2189  CG  ARG C 156      19.795  22.546  36.253  1.00  0.00           C  \\nATOM   2190  CD  ARG C 156      18.547  22.499  35.409  1.00  0.00           C  \\nATOM   2191  NE  ARG C 156      18.165  21.145  35.026  1.00  0.00           N  \\nATOM   2192  CZ  ARG C 156      18.302  20.615  33.790  1.00  0.00           C  \\nATOM   2193  NH1 ARG C 156      18.838  21.290  32.804  1.00  0.00           N  \\nATOM   2194  NH2 ARG C 156      17.904  19.391  33.541  1.00  0.00           N  \\nATOM   2195  H   ARG C 156      23.223  21.668  34.520  1.00  0.00           H  \\nATOM   2196  HA  ARG C 156      22.401  23.280  36.384  1.00  0.00           H  \\nATOM   2197 1HB  ARG C 156      21.021  22.541  34.502  1.00  0.00           H  \\nATOM   2198 2HB  ARG C 156      20.914  20.997  35.325  1.00  0.00           H  \\nATOM   2199 1HG  ARG C 156      19.646  21.903  37.121  1.00  0.00           H  \\nATOM   2200 2HG  ARG C 156      19.950  23.574  36.585  1.00  0.00           H  \\nATOM   2201 1HD  ARG C 156      17.720  22.939  35.965  1.00  0.00           H  \\nATOM   2202 2HD  ARG C 156      18.713  23.073  34.497  1.00  0.00           H  \\nATOM   2203  HE  ARG C 156      17.748  20.561  35.736  1.00  0.00           H  \\nATOM   2204 1HH1 ARG C 156      19.168  22.233  32.938  1.00  0.00           H  \\nATOM   2205 2HH1 ARG C 156      18.946  20.804  31.897  1.00  0.00           H  \\nATOM   2206 1HH2 ARG C 156      17.490  18.822  34.259  1.00  0.00           H  \\nATOM   2207 2HH2 ARG C 156      18.038  19.038  32.582  1.00  0.00           H  \\nATOM   2208  N   GLU C 157      22.900  20.285  37.350  1.00  0.00           N  \\nATOM   2209  CA  GLU C 157      23.081  19.437  38.521  1.00  0.00           C  \\nATOM   2210  C   GLU C 157      24.189  19.979  39.419  1.00  0.00           C  \\nATOM   2211  O   GLU C 157      24.058  19.984  40.639  1.00  0.00           O  \\nATOM   2212  CB  GLU C 157      23.369  17.989  38.114  1.00  0.00           C  \\nATOM   2213  CG  GLU C 157      23.654  17.018  39.278  1.00  0.00           C  \\nATOM   2214  CD  GLU C 157      22.529  16.878  40.283  1.00  0.00           C  \\nATOM   2215  OE1 GLU C 157      21.440  17.347  40.044  1.00  0.00           O  \\nATOM   2216  OE2 GLU C 157      22.766  16.273  41.305  1.00  0.00           O  \\nATOM   2217  H   GLU C 157      23.081  19.898  36.603  1.00  0.00           H  \\nATOM   2218  HA  GLU C 157      22.258  19.425  39.034  1.00  0.00           H  \\nATOM   2219 1HB  GLU C 157      22.525  17.600  37.544  1.00  0.00           H  \\nATOM   2220 2HB  GLU C 157      24.234  17.969  37.458  1.00  0.00           H  \\nATOM   2221 1HG  GLU C 157      23.875  16.037  38.862  1.00  0.00           H  \\nATOM   2222 2HG  GLU C 157      24.549  17.366  39.797  1.00  0.00           H  \\nATOM   2223  N   LEU C 158      25.277  20.439  38.809  1.00  0.00           N  \\nATOM   2224  CA  LEU C 158      26.393  21.002  39.561  1.00  0.00           C  \\nATOM   2225  C   LEU C 158      25.968  22.231  40.356  1.00  0.00           C  \\nATOM   2226  O   LEU C 158      26.339  22.383  41.519  1.00  0.00           O  \\nATOM   2227  CB  LEU C 158      27.574  21.366  38.649  1.00  0.00           C  \\nATOM   2228  CG  LEU C 158      28.418  20.208  38.210  1.00  0.00           C  \\nATOM   2229  CD1 LEU C 158      29.398  20.598  37.099  1.00  0.00           C  \\nATOM   2230  CD2 LEU C 158      29.190  19.799  39.434  1.00  0.00           C  \\nATOM   2231  H   LEU C 158      25.394  20.436  37.957  1.00  0.00           H  \\nATOM   2232  HA  LEU C 158      26.719  20.338  40.189  1.00  0.00           H  \\nATOM   2233 1HB  LEU C 158      27.186  21.850  37.758  1.00  0.00           H  \\nATOM   2234 2HB  LEU C 158      28.201  22.069  39.172  1.00  0.00           H  \\nATOM   2235  HG  LEU C 158      27.787  19.387  37.852  1.00  0.00           H  \\nATOM   2236 1HD1 LEU C 158      30.011  19.738  36.840  1.00  0.00           H  \\nATOM   2237 2HD1 LEU C 158      28.836  20.904  36.236  1.00  0.00           H  \\nATOM   2238 3HD1 LEU C 158      30.040  21.407  37.422  1.00  0.00           H  \\nATOM   2239 1HD2 LEU C 158      28.497  19.523  40.224  1.00  0.00           H  \\nATOM   2240 2HD2 LEU C 158      29.805  18.987  39.200  1.00  0.00           H  \\nATOM   2241 3HD2 LEU C 158      29.806  20.623  39.773  1.00  0.00           H  \\nATOM   2242  N   ARG C 159      25.193  23.108  39.727  1.00  0.00           N  \\nATOM   2243  CA  ARG C 159      24.737  24.323  40.390  1.00  0.00           C  \\nATOM   2244  C   ARG C 159      23.801  24.005  41.552  1.00  0.00           C  \\nATOM   2245  O   ARG C 159      23.774  24.727  42.546  1.00  0.00           O  \\nATOM   2246  CB  ARG C 159      24.042  25.246  39.402  1.00  0.00           C  \\nATOM   2247  CG  ARG C 159      24.977  25.891  38.370  1.00  0.00           C  \\nATOM   2248  CD  ARG C 159      24.322  27.020  37.642  1.00  0.00           C  \\nATOM   2249  NE  ARG C 159      23.170  26.613  36.830  1.00  0.00           N  \\nATOM   2250  CZ  ARG C 159      23.221  26.205  35.542  1.00  0.00           C  \\nATOM   2251  NH1 ARG C 159      24.368  26.133  34.908  1.00  0.00           N  \\nATOM   2252  NH2 ARG C 159      22.102  25.891  34.907  1.00  0.00           N  \\nATOM   2253  H   ARG C 159      24.918  23.023  38.917  1.00  0.00           H  \\nATOM   2254  HA  ARG C 159      25.506  24.794  40.747  1.00  0.00           H  \\nATOM   2255 1HB  ARG C 159      23.292  24.673  38.851  1.00  0.00           H  \\nATOM   2256 2HB  ARG C 159      23.525  26.038  39.938  1.00  0.00           H  \\nATOM   2257 1HG  ARG C 159      25.863  26.275  38.876  1.00  0.00           H  \\nATOM   2258 2HG  ARG C 159      25.277  25.140  37.639  1.00  0.00           H  \\nATOM   2259 1HD  ARG C 159      23.979  27.759  38.365  1.00  0.00           H  \\nATOM   2260 2HD  ARG C 159      25.052  27.482  36.978  1.00  0.00           H  \\nATOM   2261  HE  ARG C 159      22.260  26.651  37.269  1.00  0.00           H  \\nATOM   2262 1HH1 ARG C 159      25.224  26.365  35.391  1.00  0.00           H  \\nATOM   2263 2HH1 ARG C 159      24.383  25.850  33.938  1.00  0.00           H  \\nATOM   2264 1HH2 ARG C 159      21.216  25.959  35.388  1.00  0.00           H  \\nATOM   2265 2HH2 ARG C 159      22.136  25.594  33.940  1.00  0.00           H  \\nATOM   2266  N   ARG C 160      23.040  22.921  41.431  1.00  0.00           N  \\nATOM   2267  CA  ARG C 160      22.124  22.515  42.493  1.00  0.00           C  \\nATOM   2268  C   ARG C 160      22.911  22.026  43.704  1.00  0.00           C  \\nATOM   2269  O   ARG C 160      22.635  22.420  44.836  1.00  0.00           O  \\nATOM   2270  CB  ARG C 160      21.206  21.390  42.026  1.00  0.00           C  \\nATOM   2271  CG  ARG C 160      20.144  20.961  43.044  1.00  0.00           C  \\nATOM   2272  CD  ARG C 160      19.371  19.754  42.591  1.00  0.00           C  \\nATOM   2273  NE  ARG C 160      20.184  18.538  42.616  1.00  0.00           N  \\nATOM   2274  CZ  ARG C 160      20.428  17.814  43.738  1.00  0.00           C  \\nATOM   2275  NH1 ARG C 160      19.913  18.187  44.900  1.00  0.00           N  \\nATOM   2276  NH2 ARG C 160      21.180  16.735  43.674  1.00  0.00           N  \\nATOM   2277  H   ARG C 160      23.035  22.404  40.744  1.00  0.00           H  \\nATOM   2278  HA  ARG C 160      21.591  23.278  42.765  1.00  0.00           H  \\nATOM   2279 1HB  ARG C 160      20.701  21.691  41.108  1.00  0.00           H  \\nATOM   2280 2HB  ARG C 160      21.805  20.508  41.787  1.00  0.00           H  \\nATOM   2281 1HG  ARG C 160      20.637  20.714  43.986  1.00  0.00           H  \\nATOM   2282 2HG  ARG C 160      19.443  21.781  43.207  1.00  0.00           H  \\nATOM   2283 1HD  ARG C 160      18.510  19.603  43.240  1.00  0.00           H  \\nATOM   2284 2HD  ARG C 160      19.033  19.910  41.567  1.00  0.00           H  \\nATOM   2285  HE  ARG C 160      20.598  18.199  41.726  1.00  0.00           H  \\nATOM   2286 1HH1 ARG C 160      19.338  19.015  44.956  1.00  0.00           H  \\nATOM   2287 2HH1 ARG C 160      20.099  17.644  45.735  1.00  0.00           H  \\nATOM   2288 1HH2 ARG C 160      21.602  16.460  42.769  1.00  0.00           H  \\nATOM   2289 2HH2 ARG C 160      21.359  16.199  44.501  1.00  0.00           H  \\nATOM   2290  N   ILE C 161      23.895  21.168  43.457  1.00  0.00           N  \\nATOM   2291  CA  ILE C 161      24.725  20.619  44.525  1.00  0.00           C  \\nATOM   2292  C   ILE C 161      25.538  21.719  45.193  1.00  0.00           C  \\nATOM   2293  O   ILE C 161      25.772  21.684  46.403  1.00  0.00           O  \\nATOM   2294  CB  ILE C 161      25.709  19.551  44.020  1.00  0.00           C  \\nATOM   2295  CG1 ILE C 161      24.977  18.311  43.545  1.00  0.00           C  \\nATOM   2296  CG2 ILE C 161      26.643  19.169  45.160  1.00  0.00           C  \\nATOM   2297  CD1 ILE C 161      25.884  17.367  42.805  1.00  0.00           C  \\nATOM   2298  H   ILE C 161      24.104  20.885  42.672  1.00  0.00           H  \\nATOM   2299  HA  ILE C 161      24.152  20.217  45.197  1.00  0.00           H  \\nATOM   2300  HB  ILE C 161      26.284  19.946  43.178  1.00  0.00           H  \\nATOM   2301 1HG1 ILE C 161      24.550  17.791  44.401  1.00  0.00           H  \\nATOM   2302 2HG1 ILE C 161      24.162  18.605  42.872  1.00  0.00           H  \\nATOM   2303 1HG2 ILE C 161      27.338  18.415  44.822  1.00  0.00           H  \\nATOM   2304 2HG2 ILE C 161      27.190  20.048  45.500  1.00  0.00           H  \\nATOM   2305 3HG2 ILE C 161      26.052  18.771  45.976  1.00  0.00           H  \\nATOM   2306 1HD1 ILE C 161      25.313  16.497  42.474  1.00  0.00           H  \\nATOM   2307 2HD1 ILE C 161      26.290  17.887  41.938  1.00  0.00           H  \\nATOM   2308 3HD1 ILE C 161      26.695  17.046  43.452  1.00  0.00           H  \\nATOM   2309  N   LEU C 162      25.970  22.694  44.402  1.00  0.00           N  \\nATOM   2310  CA  LEU C 162      26.759  23.794  44.932  1.00  0.00           C  \\nATOM   2311  C   LEU C 162      25.890  24.668  45.828  1.00  0.00           C  \\nATOM   2312  O   LEU C 162      26.356  25.182  46.844  1.00  0.00           O  \\nATOM   2313  CB  LEU C 162      27.382  24.608  43.801  1.00  0.00           C  \\nATOM   2314  CG  LEU C 162      28.306  25.752  44.229  1.00  0.00           C  \\nATOM   2315  CD1 LEU C 162      29.467  25.215  45.096  1.00  0.00           C  \\nATOM   2316  CD2 LEU C 162      28.838  26.420  42.988  1.00  0.00           C  \\nATOM   2317  H   LEU C 162      25.819  22.741  43.557  1.00  0.00           H  \\nATOM   2318  HA  LEU C 162      27.483  23.437  45.469  1.00  0.00           H  \\nATOM   2319 1HB  LEU C 162      27.948  23.934  43.159  1.00  0.00           H  \\nATOM   2320 2HB  LEU C 162      26.572  25.037  43.209  1.00  0.00           H  \\nATOM   2321  HG  LEU C 162      27.741  26.471  44.817  1.00  0.00           H  \\nATOM   2322 1HD1 LEU C 162      30.123  26.036  45.392  1.00  0.00           H  \\nATOM   2323 2HD1 LEU C 162      29.065  24.740  45.985  1.00  0.00           H  \\nATOM   2324 3HD1 LEU C 162      30.031  24.497  44.531  1.00  0.00           H  \\nATOM   2325 1HD2 LEU C 162      29.472  27.239  43.284  1.00  0.00           H  \\nATOM   2326 2HD2 LEU C 162      29.409  25.700  42.398  1.00  0.00           H  \\nATOM   2327 3HD2 LEU C 162      28.008  26.799  42.393  1.00  0.00           H  \\nATOM   2328  N   GLU C 163      24.624  24.830  45.450  1.00  0.00           N  \\nATOM   2329  CA  GLU C 163      23.677  25.573  46.275  1.00  0.00           C  \\nATOM   2330  C   GLU C 163      23.535  24.886  47.629  1.00  0.00           C  \\nATOM   2331  O   GLU C 163      23.550  25.537  48.670  1.00  0.00           O  \\nATOM   2332  CB  GLU C 163      22.335  25.706  45.518  1.00  0.00           C  \\nATOM   2333  CG  GLU C 163      21.268  26.689  46.078  1.00  0.00           C  \\nATOM   2334  CD  GLU C 163      20.288  26.107  47.082  1.00  0.00           C  \\nATOM   2335  OE1 GLU C 163      20.022  24.931  47.032  1.00  0.00           O  \\nATOM   2336  OE2 GLU C 163      19.766  26.865  47.862  1.00  0.00           O  \\nATOM   2337  H   GLU C 163      24.289  24.519  44.722  1.00  0.00           H  \\nATOM   2338  HA  GLU C 163      24.018  26.469  46.424  1.00  0.00           H  \\nATOM   2339 1HB  GLU C 163      22.538  26.002  44.491  1.00  0.00           H  \\nATOM   2340 2HB  GLU C 163      21.860  24.731  45.473  1.00  0.00           H  \\nATOM   2341 1HG  GLU C 163      21.791  27.518  46.555  1.00  0.00           H  \\nATOM   2342 2HG  GLU C 163      20.708  27.094  45.237  1.00  0.00           H  \\nATOM   2343  N   GLU C 164      23.400  23.564  47.602  1.00  0.00           N  \\nATOM   2344  CA  GLU C 164      23.300  22.773  48.823  1.00  0.00           C  \\nATOM   2345  C   GLU C 164      24.591  22.854  49.630  1.00  0.00           C  \\nATOM   2346  O   GLU C 164      24.569  22.850  50.860  1.00  0.00           O  \\nATOM   2347  CB  GLU C 164      22.987  21.301  48.480  1.00  0.00           C  \\nATOM   2348  CG  GLU C 164      22.746  20.382  49.689  1.00  0.00           C  \\nATOM   2349  CD  GLU C 164      22.296  18.959  49.315  1.00  0.00           C  \\nATOM   2350  OE1 GLU C 164      22.184  18.663  48.146  1.00  0.00           O  \\nATOM   2351  OE2 GLU C 164      22.048  18.185  50.210  1.00  0.00           O  \\nATOM   2352  H   GLU C 164      23.364  23.097  46.881  1.00  0.00           H  \\nATOM   2353  HA  GLU C 164      22.579  23.121  49.370  1.00  0.00           H  \\nATOM   2354 1HB  GLU C 164      22.110  21.260  47.834  1.00  0.00           H  \\nATOM   2355 2HB  GLU C 164      23.820  20.881  47.918  1.00  0.00           H  \\nATOM   2356 1HG  GLU C 164      23.679  20.311  50.236  1.00  0.00           H  \\nATOM   2357 2HG  GLU C 164      22.007  20.839  50.345  1.00  0.00           H  \\nATOM   2358  N   LEU C 165      25.717  22.926  48.931  1.00  0.00           N  \\nATOM   2359  CA  LEU C 165      27.015  22.974  49.587  1.00  0.00           C  \\nATOM   2360  C   LEU C 165      27.197  24.301  50.312  1.00  0.00           C  \\nATOM   2361  O   LEU C 165      27.856  24.363  51.347  1.00  0.00           O  \\nATOM   2362  CB  LEU C 165      28.118  22.693  48.560  1.00  0.00           C  \\nATOM   2363  CG  LEU C 165      29.530  22.558  49.091  1.00  0.00           C  \\nATOM   2364  CD1 LEU C 165      29.597  21.467  50.178  1.00  0.00           C  \\nATOM   2365  CD2 LEU C 165      30.460  22.174  47.903  1.00  0.00           C  \\nATOM   2366  H   LEU C 165      25.756  22.948  48.072  1.00  0.00           H  \\nATOM   2367  HA  LEU C 165      27.065  22.262  50.243  1.00  0.00           H  \\nATOM   2368 1HB  LEU C 165      27.867  21.783  48.017  1.00  0.00           H  \\nATOM   2369 2HB  LEU C 165      28.122  23.514  47.855  1.00  0.00           H  \\nATOM   2370  HG  LEU C 165      29.833  23.502  49.530  1.00  0.00           H  \\nATOM   2371 1HD1 LEU C 165      30.613  21.401  50.545  1.00  0.00           H  \\nATOM   2372 2HD1 LEU C 165      28.933  21.728  51.004  1.00  0.00           H  \\nATOM   2373 3HD1 LEU C 165      29.302  20.514  49.767  1.00  0.00           H  \\nATOM   2374 1HD2 LEU C 165      31.481  22.068  48.249  1.00  0.00           H  \\nATOM   2375 2HD2 LEU C 165      30.130  21.224  47.476  1.00  0.00           H  \\nATOM   2376 3HD2 LEU C 165      30.417  22.948  47.137  1.00  0.00           H  \\nATOM   2377  N   LYS C 166      26.598  25.358  49.773  1.00  0.00           N  \\nATOM   2378  CA  LYS C 166      26.702  26.682  50.374  1.00  0.00           C  \\nATOM   2379  C   LYS C 166      25.693  26.866  51.500  1.00  0.00           C  \\nATOM   2380  O   LYS C 166      25.907  27.668  52.408  1.00  0.00           O  \\nATOM   2381  CB  LYS C 166      26.514  27.741  49.294  1.00  0.00           C  \\nATOM   2382  CG  LYS C 166      27.687  27.864  48.312  1.00  0.00           C  \\nATOM   2383  CD  LYS C 166      27.338  28.806  47.172  1.00  0.00           C  \\nATOM   2384  CE  LYS C 166      28.536  29.101  46.298  1.00  0.00           C  \\nATOM   2385  NZ  LYS C 166      29.420  30.096  46.938  1.00  0.00           N  \\nATOM   2386  H   LYS C 166      26.123  25.333  49.056  1.00  0.00           H  \\nATOM   2387  HA  LYS C 166      27.591  26.788  50.749  1.00  0.00           H  \\nATOM   2388 1HB  LYS C 166      25.619  27.496  48.712  1.00  0.00           H  \\nATOM   2389 2HB  LYS C 166      26.350  28.711  49.757  1.00  0.00           H  \\nATOM   2390 1HG  LYS C 166      28.566  28.240  48.835  1.00  0.00           H  \\nATOM   2391 2HG  LYS C 166      27.925  26.881  47.902  1.00  0.00           H  \\nATOM   2392 1HD  LYS C 166      26.561  28.341  46.559  1.00  0.00           H  \\nATOM   2393 2HD  LYS C 166      26.950  29.741  47.573  1.00  0.00           H  \\nATOM   2394 1HE  LYS C 166      29.102  28.186  46.141  1.00  0.00           H  \\nATOM   2395 2HE  LYS C 166      28.209  29.484  45.331  1.00  0.00           H  \\nATOM   2396 1HZ  LYS C 166      28.911  30.955  47.068  1.00  0.00           H  \\nATOM   2397 2HZ  LYS C 166      29.746  29.768  47.826  1.00  0.00           H  \\nATOM   2398 3HZ  LYS C 166      30.224  30.277  46.325  1.00  0.00           H  \\nATOM   2399  N   GLU C 167      24.592  26.124  51.436  1.00  0.00           N  \\nATOM   2400  CA  GLU C 167      23.594  26.149  52.497  1.00  0.00           C  \\nATOM   2401  C   GLU C 167      24.153  25.481  53.748  1.00  0.00           C  \\nATOM   2402  O   GLU C 167      23.964  25.965  54.864  1.00  0.00           O  \\nATOM   2403  CB  GLU C 167      22.275  25.501  52.076  1.00  0.00           C  \\nATOM   2404  CG  GLU C 167      21.429  26.354  51.126  1.00  0.00           C  \\nATOM   2405  CD  GLU C 167      20.896  27.623  51.800  1.00  0.00           C  \\nATOM   2406  OE1 GLU C 167      21.089  28.698  51.275  1.00  0.00           O  \\nATOM   2407  OE2 GLU C 167      20.294  27.502  52.847  1.00  0.00           O  \\nATOM   2408  H   GLU C 167      24.399  25.596  50.785  1.00  0.00           H  \\nATOM   2409  HA  GLU C 167      23.379  27.070  52.714  1.00  0.00           H  \\nATOM   2410 1HB  GLU C 167      22.484  24.555  51.574  1.00  0.00           H  \\nATOM   2411 2HB  GLU C 167      21.680  25.279  52.959  1.00  0.00           H  \\nATOM   2412 1HG  GLU C 167      22.036  26.651  50.278  1.00  0.00           H  \\nATOM   2413 2HG  GLU C 167      20.597  25.761  50.747  1.00  0.00           H  \\nATOM   2414  N   MET C 168      24.854  24.371  53.547  1.00  0.00           N  \\nATOM   2415  CA  MET C 168      25.421  23.608  54.650  1.00  0.00           C  \\nATOM   2416  C   MET C 168      26.681  24.258  55.203  1.00  0.00           C  \\nATOM   2417  O   MET C 168      27.083  23.993  56.336  1.00  0.00           O  \\nATOM   2418  CB  MET C 168      25.773  22.267  54.096  1.00  0.00           C  \\nATOM   2419  CG  MET C 168      24.617  21.459  53.719  1.00  0.00           C  \\nATOM   2420  SD  MET C 168      25.149  20.117  52.804  1.00  0.00           S  \\nATOM   2421  CE  MET C 168      26.159  19.396  53.983  1.00  0.00           C  \\nATOM   2422  H   MET C 168      25.016  24.035  52.772  1.00  0.00           H  \\nATOM   2423  HA  MET C 168      24.763  23.551  55.360  1.00  0.00           H  \\nATOM   2424 1HB  MET C 168      26.396  22.396  53.205  1.00  0.00           H  \\nATOM   2425 2HB  MET C 168      26.353  21.712  54.829  1.00  0.00           H  \\nATOM   2426 1HG  MET C 168      24.100  21.100  54.606  1.00  0.00           H  \\nATOM   2427 2HG  MET C 168      23.921  22.044  53.119  1.00  0.00           H  \\nATOM   2428 1HE  MET C 168      26.593  18.507  53.560  1.00  0.00           H  \\nATOM   2429 2HE  MET C 168      26.934  20.089  54.255  1.00  0.00           H  \\nATOM   2430 3HE  MET C 168      25.577  19.154  54.862  1.00  0.00           H  \\nATOM   2431  N   LEU C 169      27.308  25.100  54.393  1.00  0.00           N  \\nATOM   2432  CA  LEU C 169      28.496  25.820  54.820  1.00  0.00           C  \\nATOM   2433  C   LEU C 169      28.123  26.874  55.856  1.00  0.00           C  \\nATOM   2434  O   LEU C 169      28.907  27.181  56.754  1.00  0.00           O  \\nATOM   2435  CB  LEU C 169      29.198  26.463  53.612  1.00  0.00           C  \\nATOM   2436  CG  LEU C 169      30.626  26.992  53.847  1.00  0.00           C  \\nATOM   2437  CD1 LEU C 169      31.524  25.808  54.288  1.00  0.00           C  \\nATOM   2438  CD2 LEU C 169      31.152  27.628  52.560  1.00  0.00           C  \\nATOM   2439  H   LEU C 169      27.064  25.273  53.587  1.00  0.00           H  \\nATOM   2440  HA  LEU C 169      29.118  25.198  55.229  1.00  0.00           H  \\nATOM   2441 1HB  LEU C 169      29.257  25.729  52.819  1.00  0.00           H  \\nATOM   2442 2HB  LEU C 169      28.583  27.294  53.261  1.00  0.00           H  \\nATOM   2443  HG  LEU C 169      30.618  27.731  54.646  1.00  0.00           H  \\nATOM   2444 1HD1 LEU C 169      31.118  25.374  55.199  1.00  0.00           H  \\nATOM   2445 2HD1 LEU C 169      31.537  25.050  53.504  1.00  0.00           H  \\nATOM   2446 3HD1 LEU C 169      32.538  26.152  54.470  1.00  0.00           H  \\nATOM   2447 1HD2 LEU C 169      32.163  27.997  52.714  1.00  0.00           H  \\nATOM   2448 2HD2 LEU C 169      31.157  26.888  51.775  1.00  0.00           H  \\nATOM   2449 3HD2 LEU C 169      30.501  28.453  52.274  1.00  0.00           H  \\nATOM   2450  N   GLU C 170      26.920  27.424  55.726  1.00  0.00           N  \\nATOM   2451  CA  GLU C 170      26.431  28.425  56.666  1.00  0.00           C  \\nATOM   2452  C   GLU C 170      26.093  27.779  58.009  1.00  0.00           C  \\nATOM   2453  O   GLU C 170      26.188  28.422  59.054  1.00  0.00           O  \\nATOM   2454  CB  GLU C 170      25.178  29.072  56.059  1.00  0.00           C  \\nATOM   2455  CG  GLU C 170      24.466  30.130  56.887  1.00  0.00           C  \\nATOM   2456  CD  GLU C 170      25.205  31.402  57.127  1.00  0.00           C  \\nATOM   2457  OE1 GLU C 170      24.842  32.070  58.082  1.00  0.00           O  \\nATOM   2458  OE2 GLU C 170      26.081  31.734  56.375  1.00  0.00           O  \\nATOM   2459  H   GLU C 170      26.365  27.233  55.098  1.00  0.00           H  \\nATOM   2460  HA  GLU C 170      27.125  29.085  56.816  1.00  0.00           H  \\nATOM   2461 1HB  GLU C 170      25.441  29.523  55.101  1.00  0.00           H  \\nATOM   2462 2HB  GLU C 170      24.449  28.293  55.849  1.00  0.00           H  \\nATOM   2463 1HG  GLU C 170      23.522  30.366  56.398  1.00  0.00           H  \\nATOM   2464 2HG  GLU C 170      24.234  29.672  57.844  1.00  0.00           H  \\nATOM   2465  N   ARG C 171      25.694  26.509  57.975  1.00  0.00           N  \\nATOM   2466  CA  ARG C 171      25.462  25.748  59.200  1.00  0.00           C  \\nATOM   2467  C   ARG C 171      26.769  25.590  59.961  1.00  0.00           C  \\nATOM   2468  O   ARG C 171      26.818  25.729  61.183  1.00  0.00           O  \\nATOM   2469  CB  ARG C 171      24.893  24.367  58.904  1.00  0.00           C  \\nATOM   2470  CG  ARG C 171      23.465  24.352  58.386  1.00  0.00           C  \\nATOM   2471  CD  ARG C 171      22.990  22.958  58.143  1.00  0.00           C  \\nATOM   2472  NE  ARG C 171      21.649  22.925  57.568  1.00  0.00           N  \\nATOM   2473  CZ  ARG C 171      20.502  22.979  58.278  1.00  0.00           C  \\nATOM   2474  NH1 ARG C 171      20.540  23.063  59.590  1.00  0.00           N  \\nATOM   2475  NH2 ARG C 171      19.337  22.945  57.653  1.00  0.00           N  \\nATOM   2476  H   ARG C 171      25.550  26.064  57.253  1.00  0.00           H  \\nATOM   2477  HA  ARG C 171      24.835  26.228  59.763  1.00  0.00           H  \\nATOM   2478 1HB  ARG C 171      25.511  23.859  58.182  1.00  0.00           H  \\nATOM   2479 2HB  ARG C 171      24.916  23.771  59.817  1.00  0.00           H  \\nATOM   2480 1HG  ARG C 171      22.806  24.826  59.110  1.00  0.00           H  \\nATOM   2481 2HG  ARG C 171      23.425  24.900  57.439  1.00  0.00           H  \\nATOM   2482 1HD  ARG C 171      23.671  22.462  57.449  1.00  0.00           H  \\nATOM   2483 2HD  ARG C 171      22.972  22.410  59.084  1.00  0.00           H  \\nATOM   2484  HE  ARG C 171      21.573  22.864  56.561  1.00  0.00           H  \\nATOM   2485 1HH1 ARG C 171      21.429  23.088  60.069  1.00  0.00           H  \\nATOM   2486 2HH1 ARG C 171      19.681  23.102  60.118  1.00  0.00           H  \\nATOM   2487 1HH2 ARG C 171      19.306  22.880  56.645  1.00  0.00           H  \\nATOM   2488 2HH2 ARG C 171      18.479  22.985  58.183  1.00  0.00           H  \\nATOM   2489  N   LEU C 172      27.823  25.284  59.213  1.00  0.00           N  \\nATOM   2490  CA  LEU C 172      29.153  25.088  59.768  1.00  0.00           C  \\nATOM   2491  C   LEU C 172      29.622  26.320  60.537  1.00  0.00           C  \\nATOM   2492  O   LEU C 172      30.294  26.208  61.562  1.00  0.00           O  \\nATOM   2493  CB  LEU C 172      30.124  24.761  58.613  1.00  0.00           C  \\nATOM   2494  CG  LEU C 172      31.525  24.300  58.975  1.00  0.00           C  \\nATOM   2495  CD1 LEU C 172      31.442  22.981  59.724  1.00  0.00           C  \\nATOM   2496  CD2 LEU C 172      32.347  24.145  57.704  1.00  0.00           C  \\nATOM   2497  H   LEU C 172      27.790  25.183  58.359  1.00  0.00           H  \\nATOM   2498  HA  LEU C 172      29.136  24.337  60.381  1.00  0.00           H  \\nATOM   2499 1HB  LEU C 172      29.665  24.001  57.978  1.00  0.00           H  \\nATOM   2500 2HB  LEU C 172      30.252  25.655  58.022  1.00  0.00           H  \\nATOM   2501  HG  LEU C 172      31.988  25.037  59.633  1.00  0.00           H  \\nATOM   2502 1HD1 LEU C 172      32.440  22.667  59.998  1.00  0.00           H  \\nATOM   2503 2HD1 LEU C 172      30.849  23.098  60.605  1.00  0.00           H  \\nATOM   2504 3HD1 LEU C 172      30.983  22.226  59.085  1.00  0.00           H  \\nATOM   2505 1HD2 LEU C 172      33.342  23.824  57.960  1.00  0.00           H  \\nATOM   2506 2HD2 LEU C 172      31.880  23.407  57.051  1.00  0.00           H  \\nATOM   2507 3HD2 LEU C 172      32.399  25.103  57.191  1.00  0.00           H  \\nATOM   2508  N   GLU C 173      29.266  27.495  60.027  1.00  0.00           N  \\nATOM   2509  CA  GLU C 173      29.635  28.761  60.656  1.00  0.00           C  \\nATOM   2510  C   GLU C 173      28.926  28.968  61.992  1.00  0.00           C  \\nATOM   2511  O   GLU C 173      29.563  29.288  62.995  1.00  0.00           O  \\nATOM   2512  CB  GLU C 173      29.409  29.940  59.703  1.00  0.00           C  \\nATOM   2513  CG  GLU C 173      29.839  31.297  60.292  1.00  0.00           C  \\nATOM   2514  CD  GLU C 173      29.896  32.431  59.277  1.00  0.00           C  \\nATOM   2515  OE1 GLU C 173      29.873  33.567  59.686  1.00  0.00           O  \\nATOM   2516  OE2 GLU C 173      30.031  32.154  58.102  1.00  0.00           O  \\nATOM   2517  H   GLU C 173      28.804  27.588  59.308  1.00  0.00           H  \\nATOM   2518  HA  GLU C 173      30.590  28.763  60.821  1.00  0.00           H  \\nATOM   2519 1HB  GLU C 173      29.959  29.772  58.777  1.00  0.00           H  \\nATOM   2520 2HB  GLU C 173      28.349  30.002  59.446  1.00  0.00           H  \\nATOM   2521 1HG  GLU C 173      29.131  31.568  61.076  1.00  0.00           H  \\nATOM   2522 2HG  GLU C 173      30.818  31.181  60.753  1.00  0.00           H  \\nATOM   2523  N   LYS C 174      27.606  28.804  61.989  1.00  0.00           N  \\nATOM   2524  CA  LYS C 174      26.794  28.975  63.193  1.00  0.00           C  \\nATOM   2525  C   LYS C 174      27.275  28.090  64.340  1.00  0.00           C  \\nATOM   2526  O   LYS C 174      27.157  28.451  65.511  1.00  0.00           O  \\nATOM   2527  CB  LYS C 174      25.325  28.720  62.880  1.00  0.00           C  \\nATOM   2528  CG  LYS C 174      24.703  29.859  62.090  1.00  0.00           C  \\nATOM   2529  CD  LYS C 174      23.224  29.671  61.840  1.00  0.00           C  \\nATOM   2530  CE  LYS C 174      22.642  30.948  61.256  1.00  0.00           C  \\nATOM   2531  NZ  LYS C 174      23.312  31.295  59.995  1.00  0.00           N  \\nATOM   2532  H   LYS C 174      27.149  28.590  61.293  1.00  0.00           H  \\nATOM   2533  HA  LYS C 174      26.852  29.899  63.483  1.00  0.00           H  \\nATOM   2534 1HB  LYS C 174      25.235  27.800  62.292  1.00  0.00           H  \\nATOM   2535 2HB  LYS C 174      24.768  28.580  63.804  1.00  0.00           H  \\nATOM   2536 1HG  LYS C 174      24.857  30.796  62.622  1.00  0.00           H  \\nATOM   2537 2HG  LYS C 174      25.210  29.929  61.119  1.00  0.00           H  \\nATOM   2538 1HD  LYS C 174      23.078  28.851  61.133  1.00  0.00           H  \\nATOM   2539 2HD  LYS C 174      22.715  29.429  62.772  1.00  0.00           H  \\nATOM   2540 1HE  LYS C 174      21.578  30.815  61.068  1.00  0.00           H  \\nATOM   2541 2HE  LYS C 174      22.779  31.764  61.965  1.00  0.00           H  \\nATOM   2542 1HZ  LYS C 174      23.181  30.539  59.342  1.00  0.00           H  \\nATOM   2543 2HZ  LYS C 174      22.949  32.143  59.591  1.00  0.00           H  \\nATOM   2544 3HZ  LYS C 174      24.309  31.418  60.175  1.00  0.00           H  \\nATOM   2545  N   ASN C 175      27.816  26.928  63.990  1.00  0.00           N  \\nATOM   2546  CA  ASN C 175      28.257  25.949  64.975  1.00  0.00           C  \\nATOM   2547  C   ASN C 175      29.037  24.821  64.309  1.00  0.00           C  \\nATOM   2548  O   ASN C 175      28.447  23.835  63.869  1.00  0.00           O  \\nATOM   2549  CB  ASN C 175      27.102  25.364  65.779  1.00  0.00           C  \\nATOM   2550  CG  ASN C 175      26.110  24.586  64.964  1.00  0.00           C  \\nATOM   2551  OD1 ASN C 175      25.413  25.125  64.091  1.00  0.00           O  \\nATOM   2552  ND2 ASN C 175      26.033  23.311  65.235  1.00  0.00           N  \\nATOM   2553  H   ASN C 175      27.939  26.680  63.176  1.00  0.00           H  \\nATOM   2554  HA  ASN C 175      28.842  26.384  65.615  1.00  0.00           H  \\nATOM   2555 1HB  ASN C 175      27.511  24.688  66.535  1.00  0.00           H  \\nATOM   2556 2HB  ASN C 175      26.574  26.151  66.310  1.00  0.00           H  \\nATOM   2557 1HD2 ASN C 175      26.641  22.887  65.952  1.00  0.00           H  \\nATOM   2558 2HD2 ASN C 175      25.401  22.725  64.733  1.00  0.00           H  \\nATOM   2559  N   PRO C 176      30.371  24.958  64.231  1.00  0.00           N  \\nATOM   2560  CA  PRO C 176      31.144  23.927  63.536  1.00  0.00           C  \\nATOM   2561  C   PRO C 176      31.202  22.619  64.321  1.00  0.00           C  \\nATOM   2562  O   PRO C 176      32.216  22.320  64.951  1.00  0.00           O  \\nATOM   2563  CB  PRO C 176      32.501  24.622  63.399  1.00  0.00           C  \\nATOM   2564  CG  PRO C 176      32.583  25.555  64.614  1.00  0.00           C  \\nATOM   2565  CD  PRO C 176      31.138  26.060  64.820  1.00  0.00           C  \\nATOM   2566  HA  PRO C 176      30.778  23.762  62.653  1.00  0.00           H  \\nATOM   2567 1HB  PRO C 176      33.307  23.873  63.365  1.00  0.00           H  \\nATOM   2568 2HB  PRO C 176      32.526  25.171  62.446  1.00  0.00           H  \\nATOM   2569 1HG  PRO C 176      32.963  25.017  65.482  1.00  0.00           H  \\nATOM   2570 2HG  PRO C 176      33.292  26.374  64.412  1.00  0.00           H  \\nATOM   2571 1HD  PRO C 176      30.958  26.156  65.897  1.00  0.00           H  \\nATOM   2572 2HD  PRO C 176      30.949  27.001  64.265  1.00  0.00           H  \\nATOM   2573  N   ASP C 177      30.115  21.852  64.289  1.00  0.00           N  \\nATOM   2574  CA  ASP C 177      30.104  20.520  64.885  1.00  0.00           C  \\nATOM   2575  C   ASP C 177      31.015  19.597  64.084  1.00  0.00           C  \\nATOM   2576  O   ASP C 177      31.678  20.034  63.145  1.00  0.00           O  \\nATOM   2577  CB  ASP C 177      28.698  19.892  64.897  1.00  0.00           C  \\nATOM   2578  CG  ASP C 177      27.635  20.556  65.806  1.00  0.00           C  \\nATOM   2579  OD1 ASP C 177      27.932  21.451  66.574  1.00  0.00           O  \\nATOM   2580  OD2 ASP C 177      26.506  20.114  65.725  1.00  0.00           O  \\nATOM   2581  H   ASP C 177      29.370  22.081  63.926  1.00  0.00           H  \\nATOM   2582  HA  ASP C 177      30.444  20.571  65.792  1.00  0.00           H  \\nATOM   2583 1HB  ASP C 177      28.324  19.905  63.895  1.00  0.00           H  \\nATOM   2584 2HB  ASP C 177      28.764  18.854  65.195  1.00  0.00           H  \\nATOM   2585  N   LYS C 178      31.045  18.322  64.456  1.00  0.00           N  \\nATOM   2586  CA  LYS C 178      31.856  17.338  63.748  1.00  0.00           C  \\nATOM   2587  C   LYS C 178      30.991  16.476  62.835  1.00  0.00           C  \\nATOM   2588  O   LYS C 178      31.477  15.932  61.848  1.00  0.00           O  \\nATOM   2589  CB  LYS C 178      32.595  16.501  64.769  1.00  0.00           C  \\nATOM   2590  CG  LYS C 178      33.451  15.454  64.211  1.00  0.00           C  \\nATOM   2591  CD  LYS C 178      34.209  14.787  65.308  1.00  0.00           C  \\nATOM   2592  CE  LYS C 178      33.243  14.127  66.306  1.00  0.00           C  \\nATOM   2593  NZ  LYS C 178      33.897  13.059  67.087  1.00  0.00           N  \\nATOM   2594  H   LYS C 178      30.601  17.999  65.118  1.00  0.00           H  \\nATOM   2595  HA  LYS C 178      32.505  17.803  63.196  1.00  0.00           H  \\nATOM   2596 1HB  LYS C 178      33.220  17.149  65.383  1.00  0.00           H  \\nATOM   2597 2HB  LYS C 178      31.875  16.018  65.429  1.00  0.00           H  \\nATOM   2598 1HG  LYS C 178      32.841  14.707  63.696  1.00  0.00           H  \\nATOM   2599 2HG  LYS C 178      34.149  15.889  63.499  1.00  0.00           H  \\nATOM   2600 1HD  LYS C 178      34.851  14.047  64.876  1.00  0.00           H  \\nATOM   2601 2HD  LYS C 178      34.824  15.514  65.832  1.00  0.00           H  \\nATOM   2602 1HE  LYS C 178      32.879  14.888  66.992  1.00  0.00           H  \\nATOM   2603 2HE  LYS C 178      32.396  13.704  65.767  1.00  0.00           H  \\nATOM   2604 1HZ  LYS C 178      33.260  12.631  67.721  1.00  0.00           H  \\nATOM   2605 2HZ  LYS C 178      34.219  12.330  66.410  1.00  0.00           H  \\nATOM   2606 3HZ  LYS C 178      34.692  13.381  67.600  1.00  0.00           H  \\nATOM   2607  N   ASP C 179      29.708  16.361  63.165  1.00  0.00           N  \\nATOM   2608  CA  ASP C 179      28.761  15.631  62.326  1.00  0.00           C  \\nATOM   2609  C   ASP C 179      28.347  16.481  61.126  1.00  0.00           C  \\nATOM   2610  O   ASP C 179      28.050  15.954  60.054  1.00  0.00           O  \\nATOM   2611  CB  ASP C 179      27.523  15.240  63.143  1.00  0.00           C  \\nATOM   2612  CG  ASP C 179      26.632  14.175  62.472  1.00  0.00           C  \\nATOM   2613  OD1 ASP C 179      27.106  13.081  62.253  1.00  0.00           O  \\nATOM   2614  OD2 ASP C 179      25.491  14.469  62.200  1.00  0.00           O  \\nATOM   2615  H   ASP C 179      29.358  16.698  63.875  1.00  0.00           H  \\nATOM   2616  HA  ASP C 179      29.187  14.826  61.994  1.00  0.00           H  \\nATOM   2617 1HB  ASP C 179      27.838  14.862  64.115  1.00  0.00           H  \\nATOM   2618 2HB  ASP C 179      26.917  16.130  63.323  1.00  0.00           H  \\nATOM   2619  N   VAL C 180      28.323  17.798  61.313  1.00  0.00           N  \\nATOM   2620  CA  VAL C 180      28.053  18.720  60.216  1.00  0.00           C  \\nATOM   2621  C   VAL C 180      29.210  18.679  59.225  1.00  0.00           C  \\nATOM   2622  O   VAL C 180      28.997  18.640  58.015  1.00  0.00           O  \\nATOM   2623  CB  VAL C 180      27.887  20.172  60.721  1.00  0.00           C  \\nATOM   2624  CG1 VAL C 180      27.810  21.148  59.528  1.00  0.00           C  \\nATOM   2625  CG2 VAL C 180      26.618  20.280  61.552  1.00  0.00           C  \\nATOM   2626  H   VAL C 180      28.460  18.183  62.070  1.00  0.00           H  \\nATOM   2627  HA  VAL C 180      27.248  18.440  59.755  1.00  0.00           H  \\nATOM   2628  HB  VAL C 180      28.756  20.444  61.327  1.00  0.00           H  \\nATOM   2629 1HG1 VAL C 180      27.701  22.169  59.899  1.00  0.00           H  \\nATOM   2630 2HG1 VAL C 180      28.722  21.075  58.935  1.00  0.00           H  \\nATOM   2631 3HG1 VAL C 180      26.953  20.898  58.903  1.00  0.00           H  \\nATOM   2632 1HG2 VAL C 180      26.507  21.306  61.917  1.00  0.00           H  \\nATOM   2633 2HG2 VAL C 180      25.760  20.018  60.934  1.00  0.00           H  \\nATOM   2634 3HG2 VAL C 180      26.667  19.600  62.398  1.00  0.00           H  \\nATOM   2635  N   ILE C 181      30.434  18.686  59.748  1.00  0.00           N  \\nATOM   2636  CA  ILE C 181      31.631  18.544  58.923  1.00  0.00           C  \\nATOM   2637  C   ILE C 181      31.555  17.273  58.083  1.00  0.00           C  \\nATOM   2638  O   ILE C 181      31.911  17.279  56.906  1.00  0.00           O  \\nATOM   2639  CB  ILE C 181      32.950  18.525  59.714  1.00  0.00           C  \\nATOM   2640  CG1 ILE C 181      33.233  19.911  60.278  1.00  0.00           C  \\nATOM   2641  CG2 ILE C 181      34.089  18.006  58.818  1.00  0.00           C  \\nATOM   2642  CD1 ILE C 181      34.352  19.958  61.290  1.00  0.00           C  \\nATOM   2643  H   ILE C 181      30.599  18.773  60.588  1.00  0.00           H  \\nATOM   2644  HA  ILE C 181      31.691  19.301  58.319  1.00  0.00           H  \\nATOM   2645  HB  ILE C 181      32.847  17.856  60.568  1.00  0.00           H  \\nATOM   2646 1HG1 ILE C 181      33.492  20.571  59.455  1.00  0.00           H  \\nATOM   2647 2HG1 ILE C 181      32.327  20.282  60.751  1.00  0.00           H  \\nATOM   2648 1HG2 ILE C 181      35.014  17.982  59.385  1.00  0.00           H  \\nATOM   2649 2HG2 ILE C 181      33.852  16.999  58.473  1.00  0.00           H  \\nATOM   2650 3HG2 ILE C 181      34.206  18.646  57.966  1.00  0.00           H  \\nATOM   2651 1HD1 ILE C 181      34.482  20.986  61.639  1.00  0.00           H  \\nATOM   2652 2HD1 ILE C 181      34.101  19.320  62.136  1.00  0.00           H  \\nATOM   2653 3HD1 ILE C 181      35.272  19.610  60.834  1.00  0.00           H  \\nATOM   2654  N   VAL C 182      31.080  16.190  58.693  1.00  0.00           N  \\nATOM   2655  CA  VAL C 182      30.901  14.922  57.990  1.00  0.00           C  \\nATOM   2656  C   VAL C 182      29.944  15.088  56.814  1.00  0.00           C  \\nATOM   2657  O   VAL C 182      30.222  14.625  55.707  1.00  0.00           O  \\nATOM   2658  CB  VAL C 182      30.454  13.791  58.929  1.00  0.00           C  \\nATOM   2659  CG1 VAL C 182      30.019  12.585  58.115  1.00  0.00           C  \\nATOM   2660  CG2 VAL C 182      31.651  13.393  59.811  1.00  0.00           C  \\nATOM   2661  H   VAL C 182      30.853  16.164  59.522  1.00  0.00           H  \\nATOM   2662  HA  VAL C 182      31.757  14.628  57.643  1.00  0.00           H  \\nATOM   2663  HB  VAL C 182      29.622  14.116  59.541  1.00  0.00           H  \\nATOM   2664 1HG1 VAL C 182      29.719  11.779  58.787  1.00  0.00           H  \\nATOM   2665 2HG1 VAL C 182      29.179  12.870  57.496  1.00  0.00           H  \\nATOM   2666 3HG1 VAL C 182      30.821  12.255  57.492  1.00  0.00           H  \\nATOM   2667 1HG2 VAL C 182      31.358  12.588  60.480  1.00  0.00           H  \\nATOM   2668 2HG2 VAL C 182      32.470  13.069  59.190  1.00  0.00           H  \\nATOM   2669 3HG2 VAL C 182      31.975  14.229  60.384  1.00  0.00           H  \\nATOM   2670  N   GLU C 183      28.821  15.755  57.057  1.00  0.00           N  \\nATOM   2671  CA  GLU C 183      27.810  15.944  56.025  1.00  0.00           C  \\nATOM   2672  C   GLU C 183      28.353  16.809  54.892  1.00  0.00           C  \\nATOM   2673  O   GLU C 183      28.124  16.521  53.719  1.00  0.00           O  \\nATOM   2674  CB  GLU C 183      26.535  16.547  56.618  1.00  0.00           C  \\nATOM   2675  CG  GLU C 183      25.290  16.482  55.690  1.00  0.00           C  \\nATOM   2676  CD  GLU C 183      24.711  15.088  55.486  1.00  0.00           C  \\nATOM   2677  OE1 GLU C 183      24.976  14.210  56.275  1.00  0.00           O  \\nATOM   2678  OE2 GLU C 183      24.002  14.907  54.523  1.00  0.00           O  \\nATOM   2679  H   GLU C 183      28.620  16.109  57.815  1.00  0.00           H  \\nATOM   2680  HA  GLU C 183      27.570  15.080  55.655  1.00  0.00           H  \\nATOM   2681 1HB  GLU C 183      26.295  16.036  57.552  1.00  0.00           H  \\nATOM   2682 2HB  GLU C 183      26.716  17.596  56.862  1.00  0.00           H  \\nATOM   2683 1HG  GLU C 183      24.515  17.144  56.071  1.00  0.00           H  \\nATOM   2684 2HG  GLU C 183      25.580  16.825  54.719  1.00  0.00           H  \\nATOM   2685  N   VAL C 184      29.082  17.862  55.250  1.00  0.00           N  \\nATOM   2686  CA  VAL C 184      29.695  18.747  54.265  1.00  0.00           C  \\nATOM   2687  C   VAL C 184      30.661  17.976  53.375  1.00  0.00           C  \\nATOM   2688  O   VAL C 184      30.706  18.194  52.168  1.00  0.00           O  \\nATOM   2689  CB  VAL C 184      30.456  19.925  54.946  1.00  0.00           C  \\nATOM   2690  CG1 VAL C 184      31.281  20.686  53.909  1.00  0.00           C  \\nATOM   2691  CG2 VAL C 184      29.464  20.918  55.602  1.00  0.00           C  \\nATOM   2692  H   VAL C 184      29.238  18.087  56.065  1.00  0.00           H  \\nATOM   2693  HA  VAL C 184      29.000  19.120  53.700  1.00  0.00           H  \\nATOM   2694  HB  VAL C 184      31.131  19.517  55.700  1.00  0.00           H  \\nATOM   2695 1HG1 VAL C 184      31.822  21.497  54.396  1.00  0.00           H  \\nATOM   2696 2HG1 VAL C 184      31.987  20.002  53.449  1.00  0.00           H  \\nATOM   2697 3HG1 VAL C 184      30.620  21.098  53.147  1.00  0.00           H  \\nATOM   2698 1HG2 VAL C 184      30.017  21.726  56.080  1.00  0.00           H  \\nATOM   2699 2HG2 VAL C 184      28.812  21.335  54.841  1.00  0.00           H  \\nATOM   2700 3HG2 VAL C 184      28.867  20.406  56.346  1.00  0.00           H  \\nATOM   2701  N   LEU C 185      31.427  17.069  53.972  1.00  0.00           N  \\nATOM   2702  CA  LEU C 185      32.376  16.268  53.210  1.00  0.00           C  \\nATOM   2703  C   LEU C 185      31.649  15.388  52.200  1.00  0.00           C  \\nATOM   2704  O   LEU C 185      32.119  15.207  51.080  1.00  0.00           O  \\nATOM   2705  CB  LEU C 185      33.238  15.389  54.126  1.00  0.00           C  \\nATOM   2706  CG  LEU C 185      34.263  16.119  55.005  1.00  0.00           C  \\nATOM   2707  CD1 LEU C 185      34.846  15.141  55.972  1.00  0.00           C  \\nATOM   2708  CD2 LEU C 185      35.357  16.706  54.158  1.00  0.00           C  \\nATOM   2709  H   LEU C 185      31.416  16.898  54.815  1.00  0.00           H  \\nATOM   2710  HA  LEU C 185      32.967  16.861  52.720  1.00  0.00           H  \\nATOM   2711 1HB  LEU C 185      32.577  14.835  54.790  1.00  0.00           H  \\nATOM   2712 2HB  LEU C 185      33.778  14.671  53.510  1.00  0.00           H  \\nATOM   2713  HG  LEU C 185      33.779  16.908  55.548  1.00  0.00           H  \\nATOM   2714 1HD1 LEU C 185      35.343  14.356  55.420  1.00  0.00           H  \\nATOM   2715 2HD1 LEU C 185      35.544  15.643  56.586  1.00  0.00           H  \\nATOM   2716 3HD1 LEU C 185      34.052  14.720  56.589  1.00  0.00           H  \\nATOM   2717 1HD2 LEU C 185      35.857  15.908  53.623  1.00  0.00           H  \\nATOM   2718 2HD2 LEU C 185      34.938  17.419  53.452  1.00  0.00           H  \\nATOM   2719 3HD2 LEU C 185      36.071  17.206  54.804  1.00  0.00           H  \\nATOM   2720  N   LYS C 186      30.497  14.853  52.596  1.00  0.00           N  \\nATOM   2721  CA  LYS C 186      29.722  13.982  51.716  1.00  0.00           C  \\nATOM   2722  C   LYS C 186      29.235  14.729  50.477  1.00  0.00           C  \\nATOM   2723  O   LYS C 186      29.327  14.221  49.362  1.00  0.00           O  \\nATOM   2724  CB  LYS C 186      28.554  13.308  52.460  1.00  0.00           C  \\nATOM   2725  CG  LYS C 186      29.010  12.189  53.440  1.00  0.00           C  \\nATOM   2726  CD  LYS C 186      27.874  11.289  53.933  1.00  0.00           C  \\nATOM   2727  CE  LYS C 186      27.061  11.900  55.046  1.00  0.00           C  \\nATOM   2728  NZ  LYS C 186      26.047  10.914  55.579  1.00  0.00           N  \\nATOM   2729  H   LYS C 186      30.142  14.978  53.369  1.00  0.00           H  \\nATOM   2730  HA  LYS C 186      30.288  13.251  51.420  1.00  0.00           H  \\nATOM   2731 1HB  LYS C 186      28.007  14.055  53.036  1.00  0.00           H  \\nATOM   2732 2HB  LYS C 186      27.863  12.876  51.740  1.00  0.00           H  \\nATOM   2733 1HG  LYS C 186      29.715  11.561  52.952  1.00  0.00           H  \\nATOM   2734 2HG  LYS C 186      29.498  12.645  54.300  1.00  0.00           H  \\nATOM   2735 1HD  LYS C 186      27.207  11.064  53.102  1.00  0.00           H  \\nATOM   2736 2HD  LYS C 186      28.298  10.347  54.299  1.00  0.00           H  \\nATOM   2737 1HE  LYS C 186      27.724  12.191  55.847  1.00  0.00           H  \\nATOM   2738 2HE  LYS C 186      26.542  12.785  54.681  1.00  0.00           H  \\nATOM   2739 1HZ  LYS C 186      26.516  10.051  55.937  1.00  0.00           H  \\nATOM   2740 2HZ  LYS C 186      25.515  11.338  56.332  1.00  0.00           H  \\nATOM   2741 3HZ  LYS C 186      25.420  10.653  54.839  1.00  0.00           H  \\nATOM   2742  N   VAL C 187      28.720  15.937  50.677  1.00  0.00           N  \\nATOM   2743  CA  VAL C 187      28.226  16.748  49.570  1.00  0.00           C  \\nATOM   2744  C   VAL C 187      29.378  17.207  48.674  1.00  0.00           C  \\nATOM   2745  O   VAL C 187      29.207  17.346  47.463  1.00  0.00           O  \\nATOM   2746  CB  VAL C 187      27.372  17.912  50.071  1.00  0.00           C  \\nATOM   2747  CG1 VAL C 187      27.000  18.842  48.915  1.00  0.00           C  \\nATOM   2748  CG2 VAL C 187      26.115  17.314  50.671  1.00  0.00           C  \\nATOM   2749  H   VAL C 187      28.645  16.312  51.447  1.00  0.00           H  \\nATOM   2750  HA  VAL C 187      27.627  16.208  49.030  1.00  0.00           H  \\nATOM   2751  HB  VAL C 187      27.925  18.493  50.818  1.00  0.00           H  \\nATOM   2752 1HG1 VAL C 187      26.374  19.660  49.285  1.00  0.00           H  \\nATOM   2753 2HG1 VAL C 187      27.898  19.250  48.460  1.00  0.00           H  \\nATOM   2754 3HG1 VAL C 187      26.448  18.274  48.177  1.00  0.00           H  \\nATOM   2755 1HG2 VAL C 187      26.384  16.657  51.503  1.00  0.00           H  \\nATOM   2756 2HG2 VAL C 187      25.467  18.081  51.018  1.00  0.00           H  \\nATOM   2757 3HG2 VAL C 187      25.592  16.739  49.911  1.00  0.00           H  \\nATOM   2758  N   ILE C 188      30.552  17.422  49.263  1.00  0.00           N  \\nATOM   2759  CA  ILE C 188      31.732  17.781  48.481  1.00  0.00           C  \\nATOM   2760  C   ILE C 188      32.087  16.643  47.527  1.00  0.00           C  \\nATOM   2761  O   ILE C 188      32.427  16.884  46.368  1.00  0.00           O  \\nATOM   2762  CB  ILE C 188      32.976  18.113  49.344  1.00  0.00           C  \\nATOM   2763  CG1 ILE C 188      32.803  19.443  50.080  1.00  0.00           C  \\nATOM   2764  CG2 ILE C 188      34.213  18.108  48.487  1.00  0.00           C  \\nATOM   2765  CD1 ILE C 188      33.846  19.695  51.162  1.00  0.00           C  \\nATOM   2766  H   ILE C 188      30.691  17.367  50.109  1.00  0.00           H  \\nATOM   2767  HA  ILE C 188      31.530  18.565  47.948  1.00  0.00           H  \\nATOM   2768  HB  ILE C 188      33.086  17.350  50.109  1.00  0.00           H  \\nATOM   2769 1HG1 ILE C 188      32.870  20.249  49.351  1.00  0.00           H  \\nATOM   2770 2HG1 ILE C 188      31.829  19.468  50.521  1.00  0.00           H  \\nATOM   2771 1HG2 ILE C 188      35.069  18.319  49.100  1.00  0.00           H  \\nATOM   2772 2HG2 ILE C 188      34.329  17.134  48.022  1.00  0.00           H  \\nATOM   2773 3HG2 ILE C 188      34.126  18.849  47.735  1.00  0.00           H  \\nATOM   2774 1HD1 ILE C 188      33.659  20.663  51.627  1.00  0.00           H  \\nATOM   2775 2HD1 ILE C 188      33.778  18.917  51.911  1.00  0.00           H  \\nATOM   2776 3HD1 ILE C 188      34.834  19.689  50.734  1.00  0.00           H  \\nATOM   2777  N   VAL C 189      31.987  15.405  48.005  1.00  0.00           N  \\nATOM   2778  CA  VAL C 189      32.248  14.246  47.155  1.00  0.00           C  \\nATOM   2779  C   VAL C 189      31.178  14.121  46.075  1.00  0.00           C  \\nATOM   2780  O   VAL C 189      31.471  13.728  44.948  1.00  0.00           O  \\nATOM   2781  CB  VAL C 189      32.366  12.964  47.986  1.00  0.00           C  \\nATOM   2782  CG1 VAL C 189      32.520  11.741  47.049  1.00  0.00           C  \\nATOM   2783  CG2 VAL C 189      33.561  13.113  48.923  1.00  0.00           C  \\nATOM   2784  H   VAL C 189      31.771  15.210  48.814  1.00  0.00           H  \\nATOM   2785  HA  VAL C 189      33.105  14.364  46.716  1.00  0.00           H  \\nATOM   2786  HB  VAL C 189      31.455  12.822  48.572  1.00  0.00           H  \\nATOM   2787 1HG1 VAL C 189      32.603  10.830  47.640  1.00  0.00           H  \\nATOM   2788 2HG1 VAL C 189      31.648  11.672  46.396  1.00  0.00           H  \\nATOM   2789 3HG1 VAL C 189      33.420  11.856  46.442  1.00  0.00           H  \\nATOM   2790 1HG2 VAL C 189      33.678  12.223  49.530  1.00  0.00           H  \\nATOM   2791 2HG2 VAL C 189      34.434  13.266  48.346  1.00  0.00           H  \\nATOM   2792 3HG2 VAL C 189      33.419  13.962  49.561  1.00  0.00           H  \\nATOM   2793  N   LYS C 190      29.938  14.453  46.418  1.00  0.00           N  \\nATOM   2794  CA  LYS C 190      28.869  14.462  45.427  1.00  0.00           C  \\nATOM   2795  C   LYS C 190      29.154  15.518  44.362  1.00  0.00           C  \\nATOM   2796  O   LYS C 190      28.932  15.289  43.176  1.00  0.00           O  \\nATOM   2797  CB  LYS C 190      27.504  14.752  46.057  1.00  0.00           C  \\nATOM   2798  CG  LYS C 190      26.943  13.627  46.906  1.00  0.00           C  \\nATOM   2799  CD  LYS C 190      25.623  14.027  47.555  1.00  0.00           C  \\nATOM   2800  CE  LYS C 190      24.501  14.087  46.531  1.00  0.00           C  \\nATOM   2801  NZ  LYS C 190      24.139  12.727  46.028  1.00  0.00           N  \\nATOM   2802  H   LYS C 190      29.690  14.675  47.211  1.00  0.00           H  \\nATOM   2803  HA  LYS C 190      28.831  13.596  44.991  1.00  0.00           H  \\nATOM   2804 1HB  LYS C 190      27.567  15.640  46.668  1.00  0.00           H  \\nATOM   2805 2HB  LYS C 190      26.786  14.961  45.267  1.00  0.00           H  \\nATOM   2806 1HG  LYS C 190      26.786  12.747  46.282  1.00  0.00           H  \\nATOM   2807 2HG  LYS C 190      27.660  13.374  47.683  1.00  0.00           H  \\nATOM   2808 1HD  LYS C 190      25.366  13.308  48.334  1.00  0.00           H  \\nATOM   2809 2HD  LYS C 190      25.729  15.009  48.014  1.00  0.00           H  \\nATOM   2810 1HE  LYS C 190      23.623  14.541  46.991  1.00  0.00           H  \\nATOM   2811 2HE  LYS C 190      24.817  14.706  45.688  1.00  0.00           H  \\nATOM   2812 1HZ  LYS C 190      23.392  12.803  45.353  1.00  0.00           H  \\nATOM   2813 2HZ  LYS C 190      24.948  12.302  45.589  1.00  0.00           H  \\nATOM   2814 3HZ  LYS C 190      23.835  12.152  46.801  1.00  0.00           H  \\nATOM   2815  N   ALA C 191      29.655  16.672  44.793  1.00  0.00           N  \\nATOM   2816  CA  ALA C 191      29.996  17.745  43.866  1.00  0.00           C  \\nATOM   2817  C   ALA C 191      31.127  17.307  42.944  1.00  0.00           C  \\nATOM   2818  O   ALA C 191      31.067  17.521  41.737  1.00  0.00           O  \\nATOM   2819  CB  ALA C 191      30.403  19.000  44.627  1.00  0.00           C  \\nATOM   2820  H   ALA C 191      29.807  16.858  45.618  1.00  0.00           H  \\nATOM   2821  HA  ALA C 191      29.222  17.949  43.318  1.00  0.00           H  \\nATOM   2822 1HB  ALA C 191      30.639  19.789  43.920  1.00  0.00           H  \\nATOM   2823 2HB  ALA C 191      29.593  19.320  45.274  1.00  0.00           H  \\nATOM   2824 3HB  ALA C 191      31.276  18.787  45.232  1.00  0.00           H  \\nATOM   2825  N   ILE C 192      32.154  16.685  43.518  1.00  0.00           N  \\nATOM   2826  CA  ILE C 192      33.290  16.206  42.738  1.00  0.00           C  \\nATOM   2827  C   ILE C 192      32.846  15.113  41.771  1.00  0.00           C  \\nATOM   2828  O   ILE C 192      33.307  15.065  40.633  1.00  0.00           O  \\nATOM   2829  CB  ILE C 192      34.436  15.702  43.627  1.00  0.00           C  \\nATOM   2830  CG1 ILE C 192      35.055  16.902  44.358  1.00  0.00           C  \\nATOM   2831  CG2 ILE C 192      35.487  14.963  42.774  1.00  0.00           C  \\nATOM   2832  CD1 ILE C 192      35.993  16.544  45.451  1.00  0.00           C  \\nATOM   2833  H   ILE C 192      32.216  16.528  44.361  1.00  0.00           H  \\nATOM   2834  HA  ILE C 192      33.647  16.942  42.216  1.00  0.00           H  \\nATOM   2835  HB  ILE C 192      34.032  15.023  44.381  1.00  0.00           H  \\nATOM   2836 1HG1 ILE C 192      35.597  17.507  43.635  1.00  0.00           H  \\nATOM   2837 2HG1 ILE C 192      34.253  17.506  44.780  1.00  0.00           H  \\nATOM   2838 1HG2 ILE C 192      36.284  14.606  43.418  1.00  0.00           H  \\nATOM   2839 2HG2 ILE C 192      35.017  14.122  42.274  1.00  0.00           H  \\nATOM   2840 3HG2 ILE C 192      35.897  15.642  42.025  1.00  0.00           H  \\nATOM   2841 1HD1 ILE C 192      36.357  17.466  45.875  1.00  0.00           H  \\nATOM   2842 2HD1 ILE C 192      35.468  15.962  46.208  1.00  0.00           H  \\nATOM   2843 3HD1 ILE C 192      36.820  15.967  45.056  1.00  0.00           H  \\nATOM   2844  N   GLU C 193      31.952  14.237  42.224  1.00  0.00           N  \\nATOM   2845  CA  GLU C 193      31.357  13.232  41.348  1.00  0.00           C  \\nATOM   2846  C   GLU C 193      30.676  13.909  40.167  1.00  0.00           C  \\nATOM   2847  O   GLU C 193      30.931  13.574  39.011  1.00  0.00           O  \\nATOM   2848  CB  GLU C 193      30.385  12.310  42.093  1.00  0.00           C  \\nATOM   2849  CG  GLU C 193      29.789  11.206  41.199  1.00  0.00           C  \\nATOM   2850  CD  GLU C 193      28.956  10.181  41.934  1.00  0.00           C  \\nATOM   2851  OE1 GLU C 193      28.453  10.458  42.994  1.00  0.00           O  \\nATOM   2852  OE2 GLU C 193      28.886   9.068  41.447  1.00  0.00           O  \\nATOM   2853  H   GLU C 193      31.674  14.204  43.037  1.00  0.00           H  \\nATOM   2854  HA  GLU C 193      32.054  12.650  41.007  1.00  0.00           H  \\nATOM   2855 1HB  GLU C 193      30.898  11.838  42.931  1.00  0.00           H  \\nATOM   2856 2HB  GLU C 193      29.565  12.893  42.499  1.00  0.00           H  \\nATOM   2857 1HG  GLU C 193      29.155  11.689  40.455  1.00  0.00           H  \\nATOM   2858 2HG  GLU C 193      30.584  10.704  40.667  1.00  0.00           H  \\nATOM   2859  N   ALA C 194      29.803  14.862  40.471  1.00  0.00           N  \\nATOM   2860  CA  ALA C 194      29.082  15.592  39.442  1.00  0.00           C  \\nATOM   2861  C   ALA C 194      30.065  16.299  38.517  1.00  0.00           C  \\nATOM   2862  O   ALA C 194      29.849  16.373  37.309  1.00  0.00           O  \\nATOM   2863  CB  ALA C 194      28.124  16.572  40.084  1.00  0.00           C  \\nATOM   2864  H   ALA C 194      29.611  15.106  41.273  1.00  0.00           H  \\nATOM   2865  HA  ALA C 194      28.562  14.967  38.913  1.00  0.00           H  \\nATOM   2866 1HB  ALA C 194      27.567  17.112  39.317  1.00  0.00           H  \\nATOM   2867 2HB  ALA C 194      27.433  16.020  40.716  1.00  0.00           H  \\nATOM   2868 3HB  ALA C 194      28.684  17.267  40.692  1.00  0.00           H  \\nATOM   2869  N   SER C 195      31.155  16.802  39.088  1.00  0.00           N  \\nATOM   2870  CA  SER C 195      32.141  17.535  38.309  1.00  0.00           C  \\nATOM   2871  C   SER C 195      32.848  16.611  37.328  1.00  0.00           C  \\nATOM   2872  O   SER C 195      33.021  16.957  36.162  1.00  0.00           O  \\nATOM   2873  CB  SER C 195      33.136  18.198  39.224  1.00  0.00           C  \\nATOM   2874  OG  SER C 195      34.081  18.941  38.506  1.00  0.00           O  \\nATOM   2875  H   SER C 195      31.344  16.732  39.924  1.00  0.00           H  \\nATOM   2876  HA  SER C 195      31.691  18.227  37.800  1.00  0.00           H  \\nATOM   2877 1HB  SER C 195      32.608  18.846  39.924  1.00  0.00           H  \\nATOM   2878 2HB  SER C 195      33.643  17.440  39.804  1.00  0.00           H  \\nATOM   2879  HG  SER C 195      34.705  19.248  39.164  1.00  0.00           H  \\nATOM   2880  N   VAL C 196      33.256  15.438  37.801  1.00  0.00           N  \\nATOM   2881  CA  VAL C 196      33.914  14.461  36.941  1.00  0.00           C  \\nATOM   2882  C   VAL C 196      32.963  14.016  35.837  1.00  0.00           C  \\nATOM   2883  O   VAL C 196      33.370  13.853  34.688  1.00  0.00           O  \\nATOM   2884  CB  VAL C 196      34.408  13.238  37.760  1.00  0.00           C  \\nATOM   2885  CG1 VAL C 196      34.903  12.120  36.822  1.00  0.00           C  \\nATOM   2886  CG2 VAL C 196      35.565  13.668  38.683  1.00  0.00           C  \\nATOM   2887  H   VAL C 196      33.164  15.183  38.618  1.00  0.00           H  \\nATOM   2888  HA  VAL C 196      34.686  14.875  36.525  1.00  0.00           H  \\nATOM   2889  HB  VAL C 196      33.576  12.849  38.359  1.00  0.00           H  \\nATOM   2890 1HG1 VAL C 196      35.242  11.268  37.417  1.00  0.00           H  \\nATOM   2891 2HG1 VAL C 196      34.090  11.806  36.170  1.00  0.00           H  \\nATOM   2892 3HG1 VAL C 196      35.732  12.493  36.220  1.00  0.00           H  \\nATOM   2893 1HG2 VAL C 196      35.907  12.813  39.266  1.00  0.00           H  \\nATOM   2894 2HG2 VAL C 196      36.379  14.046  38.078  1.00  0.00           H  \\nATOM   2895 3HG2 VAL C 196      35.233  14.447  39.351  1.00  0.00           H  \\nATOM   2896  N   GLU C 197      31.693  13.832  36.185  1.00  0.00           N  \\nATOM   2897  CA  GLU C 197      30.694  13.422  35.205  1.00  0.00           C  \\nATOM   2898  C   GLU C 197      30.557  14.486  34.123  1.00  0.00           C  \\nATOM   2899  O   GLU C 197      30.467  14.168  32.937  1.00  0.00           O  \\nATOM   2900  CB  GLU C 197      29.324  13.223  35.874  1.00  0.00           C  \\nATOM   2901  CG  GLU C 197      28.147  12.835  34.929  1.00  0.00           C  \\nATOM   2902  CD  GLU C 197      28.176  11.432  34.373  1.00  0.00           C  \\nATOM   2903  OE1 GLU C 197      28.815  10.587  34.945  1.00  0.00           O  \\nATOM   2904  OE2 GLU C 197      27.540  11.206  33.371  1.00  0.00           O  \\nATOM   2905  H   GLU C 197      31.385  13.938  36.981  1.00  0.00           H  \\nATOM   2906  HA  GLU C 197      30.981  12.597  34.784  1.00  0.00           H  \\nATOM   2907 1HB  GLU C 197      29.408  12.453  36.641  1.00  0.00           H  \\nATOM   2908 2HB  GLU C 197      29.038  14.140  36.377  1.00  0.00           H  \\nATOM   2909 1HG  GLU C 197      27.216  12.968  35.478  1.00  0.00           H  \\nATOM   2910 2HG  GLU C 197      28.138  13.542  34.096  1.00  0.00           H  \\nATOM   2911  N   ASN C 198      30.533  15.749  34.515  1.00  0.00           N  \\nATOM   2912  CA  ASN C 198      30.490  16.833  33.563  1.00  0.00           C  \\nATOM   2913  C   ASN C 198      31.704  16.847  32.657  1.00  0.00           C  \\nATOM   2914  O   ASN C 198      31.606  17.049  31.503  1.00  0.00           O  \\nATOM   2915  CB  ASN C 198      30.342  18.170  34.264  1.00  0.00           C  \\nATOM   2916  CG  ASN C 198      30.052  19.277  33.285  1.00  0.00           C  \\nATOM   2917  OD1 ASN C 198      28.994  19.298  32.674  1.00  0.00           O  \\nATOM   2918  ND2 ASN C 198      30.973  20.159  33.097  1.00  0.00           N  \\nATOM   2919  H   ASN C 198      30.530  15.995  35.330  1.00  0.00           H  \\nATOM   2920  HA  ASN C 198      29.708  16.706  33.001  1.00  0.00           H  \\nATOM   2921 1HB  ASN C 198      29.540  18.116  34.999  1.00  0.00           H  \\nATOM   2922 2HB  ASN C 198      31.255  18.390  34.803  1.00  0.00           H  \\nATOM   2923 1HD2 ASN C 198      31.820  20.097  33.639  1.00  0.00           H  \\nATOM   2924 2HD2 ASN C 198      30.845  20.886  32.424  1.00  0.00           H  \\nATOM   2925  N   GLN C 199      32.858  16.625  33.209  1.00  0.00           N  \\nATOM   2926  CA  GLN C 199      34.076  16.648  32.415  1.00  0.00           C  \\nATOM   2927  C   GLN C 199      34.110  15.478  31.436  1.00  0.00           C  \\nATOM   2928  O   GLN C 199      34.723  15.573  30.375  1.00  0.00           O  \\nATOM   2929  CB  GLN C 199      35.260  16.645  33.369  1.00  0.00           C  \\nATOM   2930  CG  GLN C 199      35.356  17.961  34.181  1.00  0.00           C  \\nATOM   2931  CD  GLN C 199      36.447  17.978  35.231  1.00  0.00           C  \\nATOM   2932  OE1 GLN C 199      37.620  17.835  34.897  1.00  0.00           O  \\nATOM   2933  NE2 GLN C 199      36.106  18.143  36.509  1.00  0.00           N  \\nATOM   2934  H   GLN C 199      32.981  16.461  34.044  1.00  0.00           H  \\nATOM   2935  HA  GLN C 199      34.099  17.469  31.899  1.00  0.00           H  \\nATOM   2936 1HB  GLN C 199      35.159  15.814  34.069  1.00  0.00           H  \\nATOM   2937 2HB  GLN C 199      36.168  16.502  32.811  1.00  0.00           H  \\nATOM   2938 1HG  GLN C 199      35.563  18.771  33.485  1.00  0.00           H  \\nATOM   2939 2HG  GLN C 199      34.412  18.154  34.665  1.00  0.00           H  \\nATOM   2940 1HE2 GLN C 199      36.823  18.184  37.210  1.00  0.00           H  \\nATOM   2941 2HE2 GLN C 199      35.149  18.259  36.824  1.00  0.00           H  \\nATOM   2942  N   ARG C 200      33.432  14.385  31.777  1.00  0.00           N  \\nATOM   2943  CA  ARG C 200      33.352  13.245  30.871  1.00  0.00           C  \\nATOM   2944  C   ARG C 200      32.556  13.615  29.633  1.00  0.00           C  \\nATOM   2945  O   ARG C 200      32.976  13.342  28.511  1.00  0.00           O  \\nATOM   2946  CB  ARG C 200      32.730  12.030  31.547  1.00  0.00           C  \\nATOM   2947  CG  ARG C 200      33.671  11.383  32.453  1.00  0.00           C  \\nATOM   2948  CD  ARG C 200      33.236  10.084  33.053  1.00  0.00           C  \\nATOM   2949  NE  ARG C 200      32.380  10.250  34.198  1.00  0.00           N  \\nATOM   2950  CZ  ARG C 200      31.772   9.238  34.847  1.00  0.00           C  \\nATOM   2951  NH1 ARG C 200      31.932   8.001  34.433  1.00  0.00           N  \\nATOM   2952  NH2 ARG C 200      31.016   9.475  35.896  1.00  0.00           N  \\nATOM   2953  H   ARG C 200      33.013  14.280  32.521  1.00  0.00           H  \\nATOM   2954  HA  ARG C 200      34.248  13.000  30.591  1.00  0.00           H  \\nATOM   2955 1HB  ARG C 200      31.856  12.318  32.116  1.00  0.00           H  \\nATOM   2956 2HB  ARG C 200      32.417  11.307  30.797  1.00  0.00           H  \\nATOM   2957 1HG  ARG C 200      34.534  11.248  31.893  1.00  0.00           H  \\nATOM   2958 2HG  ARG C 200      33.888  12.053  33.259  1.00  0.00           H  \\nATOM   2959 1HD  ARG C 200      32.690   9.514  32.305  1.00  0.00           H  \\nATOM   2960 2HD  ARG C 200      34.112   9.520  33.365  1.00  0.00           H  \\nATOM   2961  HE  ARG C 200      32.232  11.189  34.544  1.00  0.00           H  \\nATOM   2962 1HH1 ARG C 200      32.500   7.808  33.626  1.00  0.00           H  \\nATOM   2963 2HH1 ARG C 200      31.468   7.247  34.920  1.00  0.00           H  \\nATOM   2964 1HH2 ARG C 200      30.869  10.423  36.214  1.00  0.00           H  \\nATOM   2965 2HH2 ARG C 200      30.548   8.719  36.365  1.00  0.00           H  \\nATOM   2966  N   ILE C 201      31.409  14.248  29.845  1.00  0.00           N  \\nATOM   2967  CA  ILE C 201      30.556  14.665  28.744  1.00  0.00           C  \\nATOM   2968  C   ILE C 201      31.278  15.712  27.900  1.00  0.00           C  \\nATOM   2969  O   ILE C 201      31.153  15.726  26.677  1.00  0.00           O  \\nATOM   2970  CB  ILE C 201      29.246  15.296  29.233  1.00  0.00           C  \\nATOM   2971  CG1 ILE C 201      28.391  14.286  29.941  1.00  0.00           C  \\nATOM   2972  CG2 ILE C 201      28.473  15.820  28.004  1.00  0.00           C  \\nATOM   2973  CD1 ILE C 201      27.247  14.926  30.675  1.00  0.00           C  \\nATOM   2974  H   ILE C 201      31.102  14.448  30.623  1.00  0.00           H  \\nATOM   2975  HA  ILE C 201      30.367  13.899  28.180  1.00  0.00           H  \\nATOM   2976  HB  ILE C 201      29.460  16.113  29.929  1.00  0.00           H  \\nATOM   2977 1HG1 ILE C 201      27.997  13.591  29.216  1.00  0.00           H  \\nATOM   2978 2HG1 ILE C 201      29.002  13.733  30.664  1.00  0.00           H  \\nATOM   2979 1HG2 ILE C 201      27.542  16.267  28.325  1.00  0.00           H  \\nATOM   2980 2HG2 ILE C 201      29.070  16.565  27.480  1.00  0.00           H  \\nATOM   2981 3HG2 ILE C 201      28.258  14.991  27.325  1.00  0.00           H  \\nATOM   2982 1HD1 ILE C 201      26.652  14.162  31.174  1.00  0.00           H  \\nATOM   2983 2HD1 ILE C 201      27.653  15.615  31.418  1.00  0.00           H  \\nATOM   2984 3HD1 ILE C 201      26.631  15.471  29.978  1.00  0.00           H  \\nATOM   2985  N   SER C 202      32.040  16.580  28.559  1.00  0.00           N  \\nATOM   2986  CA  SER C 202      32.816  17.601  27.861  1.00  0.00           C  \\nATOM   2987  C   SER C 202      33.842  16.968  26.923  1.00  0.00           C  \\nATOM   2988  O   SER C 202      33.967  17.371  25.765  1.00  0.00           O  \\nATOM   2989  CB  SER C 202      33.530  18.491  28.852  1.00  0.00           C  \\nATOM   2990  OG  SER C 202      34.315  19.480  28.209  1.00  0.00           O  \\nATOM   2991  H   SER C 202      32.125  16.599  29.414  1.00  0.00           H  \\nATOM   2992  HA  SER C 202      32.216  18.144  27.327  1.00  0.00           H  \\nATOM   2993 1HB  SER C 202      32.795  18.965  29.505  1.00  0.00           H  \\nATOM   2994 2HB  SER C 202      34.167  17.883  29.475  1.00  0.00           H  \\nATOM   2995  HG  SER C 202      34.748  19.956  28.922  1.00  0.00           H  \\nATOM   2996  N   ALA C 203      34.581  15.985  27.428  1.00  0.00           N  \\nATOM   2997  CA  ALA C 203      35.560  15.272  26.613  1.00  0.00           C  \\nATOM   2998  C   ALA C 203      34.879  14.603  25.421  1.00  0.00           C  \\nATOM   2999  O   ALA C 203      35.415  14.603  24.314  1.00  0.00           O  \\nATOM   3000  CB  ALA C 203      36.284  14.240  27.457  1.00  0.00           C  \\nATOM   3001  H   ALA C 203      34.535  15.711  28.242  1.00  0.00           H  \\nATOM   3002  HA  ALA C 203      36.209  15.906  26.271  1.00  0.00           H  \\nATOM   3003 1HB  ALA C 203      36.779  14.738  28.292  1.00  0.00           H  \\nATOM   3004 2HB  ALA C 203      35.559  13.520  27.843  1.00  0.00           H  \\nATOM   3005 3HB  ALA C 203      37.024  13.725  26.850  1.00  0.00           H  \\nATOM   3006  N   GLU C 204      33.695  14.041  25.657  1.00  0.00           N  \\nATOM   3007  CA  GLU C 204      32.887  13.460  24.586  1.00  0.00           C  \\nATOM   3008  C   GLU C 204      32.621  14.474  23.489  1.00  0.00           C  \\nATOM   3009  O   GLU C 204      32.772  14.179  22.305  1.00  0.00           O  \\nATOM   3010  CB  GLU C 204      31.512  12.902  25.020  1.00  0.00           C  \\nATOM   3011  CG  GLU C 204      31.494  11.613  25.847  1.00  0.00           C  \\nATOM   3012  CD  GLU C 204      30.044  11.139  26.121  1.00  0.00           C  \\nATOM   3013  OE1 GLU C 204      29.173  11.485  25.344  1.00  0.00           O  \\nATOM   3014  OE2 GLU C 204      29.806  10.473  27.106  1.00  0.00           O  \\nATOM   3015  H   GLU C 204      33.335  13.983  26.435  1.00  0.00           H  \\nATOM   3016  HA  GLU C 204      33.362  12.709  24.198  1.00  0.00           H  \\nATOM   3017 1HB  GLU C 204      30.975  13.639  25.578  1.00  0.00           H  \\nATOM   3018 2HB  GLU C 204      30.921  12.707  24.126  1.00  0.00           H  \\nATOM   3019 1HG  GLU C 204      32.032  10.834  25.309  1.00  0.00           H  \\nATOM   3020 2HG  GLU C 204      32.001  11.787  26.789  1.00  0.00           H  \\nATOM   3021  N   ASN C 205      32.264  15.675  23.875  1.00  0.00           N  \\nATOM   3022  CA  ASN C 205      31.927  16.712  22.932  1.00  0.00           C  \\nATOM   3023  C   ASN C 205      33.158  17.188  22.199  1.00  0.00           C  \\nATOM   3024  O   ASN C 205      33.099  17.588  21.094  1.00  0.00           O  \\nATOM   3025  CB  ASN C 205      31.178  17.845  23.584  1.00  0.00           C  \\nATOM   3026  CG  ASN C 205      30.609  18.787  22.539  1.00  0.00           C  \\nATOM   3027  OD1 ASN C 205      29.974  18.370  21.568  1.00  0.00           O  \\nATOM   3028  ND2 ASN C 205      30.813  20.043  22.724  1.00  0.00           N  \\nATOM   3029  H   ASN C 205      32.222  15.911  24.692  1.00  0.00           H  \\nATOM   3030  HA  ASN C 205      31.306  16.342  22.281  1.00  0.00           H  \\nATOM   3031 1HB  ASN C 205      30.367  17.450  24.200  1.00  0.00           H  \\nATOM   3032 2HB  ASN C 205      31.855  18.389  24.240  1.00  0.00           H  \\nATOM   3033 1HD2 ASN C 205      30.458  20.704  22.067  1.00  0.00           H  \\nATOM   3034 2HD2 ASN C 205      31.327  20.353  23.540  1.00  0.00           H  \\nATOM   3035  N   GLN C 206      34.289  17.113  22.836  1.00  0.00           N  \\nATOM   3036  CA  GLN C 206      35.535  17.516  22.194  1.00  0.00           C  \\nATOM   3037  C   GLN C 206      35.974  16.511  21.138  1.00  0.00           C  \\nATOM   3038  O   GLN C 206      36.583  16.889  20.139  1.00  0.00           O  \\nATOM   3039  CB  GLN C 206      36.641  17.714  23.224  1.00  0.00           C  \\nATOM   3040  CG  GLN C 206      36.407  18.902  24.121  1.00  0.00           C  \\nATOM   3041  CD  GLN C 206      37.453  19.071  25.194  1.00  0.00           C  \\nATOM   3042  OE1 GLN C 206      38.670  19.110  24.916  1.00  0.00           O  \\nATOM   3043  NE2 GLN C 206      37.008  19.195  26.452  1.00  0.00           N  \\nATOM   3044  H   GLN C 206      34.378  16.845  23.648  1.00  0.00           H  \\nATOM   3045  HA  GLN C 206      35.398  18.369  21.753  1.00  0.00           H  \\nATOM   3046 1HB  GLN C 206      36.730  16.822  23.843  1.00  0.00           H  \\nATOM   3047 2HB  GLN C 206      37.585  17.861  22.714  1.00  0.00           H  \\nATOM   3048 1HG  GLN C 206      36.424  19.797  23.502  1.00  0.00           H  \\nATOM   3049 2HG  GLN C 206      35.442  18.817  24.593  1.00  0.00           H  \\nATOM   3050 1HE2 GLN C 206      37.656  19.366  27.188  1.00  0.00           H  \\nATOM   3051 2HE2 GLN C 206      36.015  19.166  26.701  1.00  0.00           H  \\nATOM   3052  N   LYS C 207      35.676  15.234  21.365  1.00  0.00           N  \\nATOM   3053  CA  LYS C 207      35.926  14.204  20.361  1.00  0.00           C  \\nATOM   3054  C   LYS C 207      35.120  14.500  19.102  1.00  0.00           C  \\nATOM   3055  O   LYS C 207      35.652  14.488  17.991  1.00  0.00           O  \\nATOM   3056  CB  LYS C 207      35.600  12.811  20.923  1.00  0.00           C  \\nATOM   3057  CG  LYS C 207      36.620  12.249  21.982  1.00  0.00           C  \\nATOM   3058  CD  LYS C 207      36.137  10.874  22.527  1.00  0.00           C  \\nATOM   3059  CE  LYS C 207      37.058  10.252  23.631  1.00  0.00           C  \\nATOM   3060  NZ  LYS C 207      38.347   9.687  23.082  1.00  0.00           N  \\nATOM   3061  H   LYS C 207      35.327  14.938  22.093  1.00  0.00           H  \\nATOM   3062  HA  LYS C 207      36.868  14.206  20.128  1.00  0.00           H  \\nATOM   3063 1HB  LYS C 207      34.625  12.831  21.394  1.00  0.00           H  \\nATOM   3064 2HB  LYS C 207      35.547  12.099  20.103  1.00  0.00           H  \\nATOM   3065 1HG  LYS C 207      37.601  12.133  21.525  1.00  0.00           H  \\nATOM   3066 2HG  LYS C 207      36.703  12.954  22.809  1.00  0.00           H  \\nATOM   3067 1HD  LYS C 207      35.138  11.004  22.949  1.00  0.00           H  \\nATOM   3068 2HD  LYS C 207      36.066  10.171  21.699  1.00  0.00           H  \\nATOM   3069 1HE  LYS C 207      37.297  11.018  24.370  1.00  0.00           H  \\nATOM   3070 2HE  LYS C 207      36.510   9.449  24.121  1.00  0.00           H  \\nATOM   3071 1HZ  LYS C 207      38.930   9.257  23.849  1.00  0.00           H  \\nATOM   3072 2HZ  LYS C 207      38.135   8.969  22.408  1.00  0.00           H  \\nATOM   3073 3HZ  LYS C 207      38.876  10.418  22.637  1.00  0.00           H  \\nATOM   3074  N   ALA C 208      33.830  14.764  19.290  1.00  0.00           N  \\nATOM   3075  CA  ALA C 208      32.922  15.038  18.182  1.00  0.00           C  \\nATOM   3076  C   ALA C 208      33.345  16.279  17.406  1.00  0.00           C  \\nATOM   3077  O   ALA C 208      33.284  16.306  16.178  1.00  0.00           O  \\nATOM   3078  CB  ALA C 208      31.526  15.250  18.738  1.00  0.00           C  \\nATOM   3079  H   ALA C 208      33.452  14.791  20.062  1.00  0.00           H  \\nATOM   3080  HA  ALA C 208      32.933  14.285  17.571  1.00  0.00           H  \\nATOM   3081 1HB  ALA C 208      30.842  15.433  17.924  1.00  0.00           H  \\nATOM   3082 2HB  ALA C 208      31.221  14.366  19.287  1.00  0.00           H  \\nATOM   3083 3HB  ALA C 208      31.530  16.108  19.415  1.00  0.00           H  \\nATOM   3084  N   LEU C 209      33.772  17.308  18.129  1.00  0.00           N  \\nATOM   3085  CA  LEU C 209      34.193  18.553  17.503  1.00  0.00           C  \\nATOM   3086  C   LEU C 209      35.456  18.353  16.662  1.00  0.00           C  \\nATOM   3087  O   LEU C 209      35.676  19.069  15.686  1.00  0.00           O  \\nATOM   3088  CB  LEU C 209      34.429  19.653  18.562  1.00  0.00           C  \\nATOM   3089  CG  LEU C 209      33.154  20.224  19.280  1.00  0.00           C  \\nATOM   3090  CD1 LEU C 209      33.571  21.157  20.439  1.00  0.00           C  \\nATOM   3091  CD2 LEU C 209      32.324  20.985  18.275  1.00  0.00           C  \\nATOM   3092  H   LEU C 209      33.828  17.309  18.987  1.00  0.00           H  \\nATOM   3093  HA  LEU C 209      33.488  18.862  16.913  1.00  0.00           H  \\nATOM   3094 1HB  LEU C 209      35.080  19.238  19.333  1.00  0.00           H  \\nATOM   3095 2HB  LEU C 209      34.949  20.485  18.089  1.00  0.00           H  \\nATOM   3096  HG  LEU C 209      32.560  19.412  19.678  1.00  0.00           H  \\nATOM   3097 1HD1 LEU C 209      32.679  21.553  20.928  1.00  0.00           H  \\nATOM   3098 2HD1 LEU C 209      34.160  20.592  21.162  1.00  0.00           H  \\nATOM   3099 3HD1 LEU C 209      34.162  21.976  20.048  1.00  0.00           H  \\nATOM   3100 1HD2 LEU C 209      32.903  21.815  17.867  1.00  0.00           H  \\nATOM   3101 2HD2 LEU C 209      32.032  20.316  17.472  1.00  0.00           H  \\nATOM   3102 3HD2 LEU C 209      31.441  21.370  18.769  1.00  0.00           H  \\nTER                                                                             \\nVirtualRootMover_root 210\\n                                                      \\n\",\"pdb\");\n\tviewer_16234219519367425.setStyle({\"cartoon\": {\"color\": \"spectrum\"}, \"stick\": {\"colorscheme\": \"lightgreyCarbon\", \"radius\": 0.25}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 44.928, \"y\": 15.169, \"z\": 17.949}, \"end\": {\"x\": 45.575, \"y\": 16.034, \"z\": 16.023}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 46.81, \"y\": 14.343, \"z\": 19.809}, \"end\": {\"x\": 48.033, \"y\": 14.85, \"z\": 18.14}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 46.114, \"y\": 15.698, \"z\": 22.027}, \"end\": {\"x\": 46.777, \"y\": 16.91, \"z\": 20.757}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.674, \"y\": 14.789, \"z\": 22.598}, \"end\": {\"x\": 43.737, \"y\": 15.652, \"z\": 20.696}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 44.928, \"y\": 15.169, \"z\": 17.949}, \"end\": {\"x\": 45.575, \"y\": 16.034, \"z\": 16.023}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 44.287, \"y\": 12.228, \"z\": 23.305}, \"end\": {\"x\": 44.842, \"y\": 12.424, \"z\": 21.094}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 46.81, \"y\": 14.343, \"z\": 19.809}, \"end\": {\"x\": 48.033, \"y\": 14.85, \"z\": 18.14}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 45.587, \"y\": 12.88, \"z\": 25.534}, \"end\": {\"x\": 46.837, \"y\": 13.37, \"z\": 23.836}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 46.114, \"y\": 15.698, \"z\": 22.027}, \"end\": {\"x\": 46.777, \"y\": 16.91, \"z\": 20.757}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.775, \"y\": 14.077, \"z\": 26.954}, \"end\": {\"x\": 44.138, \"y\": 14.97, \"z\": 25.59}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.674, \"y\": 14.789, \"z\": 22.598}, \"end\": {\"x\": 43.737, \"y\": 15.652, \"z\": 20.696}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.315, \"y\": 11.834, \"z\": 27.371}, \"end\": {\"x\": 42.157, \"y\": 12.197, \"z\": 25.469}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 44.287, \"y\": 12.228, \"z\": 23.305}, \"end\": {\"x\": 44.842, \"y\": 12.424, \"z\": 21.094}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.828, \"y\": 10.351, \"z\": 29.108}, \"end\": {\"x\": 44.715, \"y\": 10.356, \"z\": 26.98}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 45.587, \"y\": 12.88, \"z\": 25.534}, \"end\": {\"x\": 46.837, \"y\": 13.37, \"z\": 23.836}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.779, \"y\": 11.899, \"z\": 31.288}, \"end\": {\"x\": 44.672, \"y\": 12.731, \"z\": 29.523}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.775, \"y\": 14.077, \"z\": 26.954}, \"end\": {\"x\": 44.138, \"y\": 14.97, \"z\": 25.59}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.24, \"y\": 11.598, \"z\": 32.106}, \"end\": {\"x\": 41.266, \"y\": 12.827, \"z\": 30.038}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.315, \"y\": 11.834, \"z\": 27.371}, \"end\": {\"x\": 42.157, \"y\": 12.197, \"z\": 25.469}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.015, \"y\": 8.95, \"z\": 32.904}, \"end\": {\"x\": 41.355, \"y\": 9.346, \"z\": 30.55}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.828, \"y\": 10.351, \"z\": 29.108}, \"end\": {\"x\": 44.715, \"y\": 10.356, \"z\": 26.98}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.608, \"y\": 9.106, \"z\": 35.154}, \"end\": {\"x\": 43.833, \"y\": 9.532, \"z\": 33.186}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.779, \"y\": 11.899, \"z\": 31.288}, \"end\": {\"x\": 44.672, \"y\": 12.731, \"z\": 29.523}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.004, \"y\": 10.63, \"z\": 36.679}, \"end\": {\"x\": 41.958, \"y\": 11.886, \"z\": 35.037}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.24, \"y\": 11.598, \"z\": 32.106}, \"end\": {\"x\": 41.266, \"y\": 12.827, \"z\": 30.038}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.702, \"y\": 9.083, \"z\": 37.306}, \"end\": {\"x\": 39.279, \"y\": 9.765, \"z\": 35.258}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.015, \"y\": 8.95, \"z\": 32.904}, \"end\": {\"x\": 41.355, \"y\": 9.346, \"z\": 30.55}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.075, \"y\": 7.231, \"z\": 38.814}, \"end\": {\"x\": 40.893, \"y\": 7.094, \"z\": 36.646}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.608, \"y\": 9.106, \"z\": 35.154}, \"end\": {\"x\": 43.833, \"y\": 9.532, \"z\": 33.186}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.837, \"y\": 8.623, \"z\": 41.003}, \"end\": {\"x\": 41.898, \"y\": 9.264, \"z\": 39.238}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.004, \"y\": 10.63, \"z\": 36.679}, \"end\": {\"x\": 41.958, \"y\": 11.886, \"z\": 35.037}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.421, \"y\": 9.172, \"z\": 41.794}, \"end\": {\"x\": 38.849, \"y\": 10.057, \"z\": 40.177}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.702, \"y\": 9.083, \"z\": 37.306}, \"end\": {\"x\": 39.279, \"y\": 9.765, \"z\": 35.258}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.331, \"y\": 6.822, \"z\": 42.612}, \"end\": {\"x\": 37.689, \"y\": 6.746, \"z\": 40.626}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.075, \"y\": 7.231, \"z\": 38.814}, \"end\": {\"x\": 40.893, \"y\": 7.094, \"z\": 36.646}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 39.037, \"y\": 6.257, \"z\": 44.665}, \"end\": {\"x\": 40.299, \"y\": 6.264, \"z\": 42.835}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.837, \"y\": 8.623, \"z\": 41.003}, \"end\": {\"x\": 41.898, \"y\": 9.264, \"z\": 39.238}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.314, \"y\": 8.397, \"z\": 46.21}, \"end\": {\"x\": 39.152, \"y\": 9.145, \"z\": 44.74}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.421, \"y\": 9.172, \"z\": 41.794}, \"end\": {\"x\": 38.849, \"y\": 10.057, \"z\": 40.177}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.776, \"y\": 7.699, \"z\": 46.781}, \"end\": {\"x\": 35.848, \"y\": 8.173, \"z\": 44.89}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.331, \"y\": 6.822, \"z\": 42.612}, \"end\": {\"x\": 37.689, \"y\": 6.746, \"z\": 40.626}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.176, \"y\": 5.637, \"z\": 48.368}, \"end\": {\"x\": 36.759, \"y\": 5.221, \"z\": 46.384}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 39.037, \"y\": 6.257, \"z\": 44.665}, \"end\": {\"x\": 40.299, \"y\": 6.264, \"z\": 42.835}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.389, \"y\": 6.449, \"z\": 50.766}, \"end\": {\"x\": 38.549, \"y\": 6.755, \"z\": 48.751}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.314, \"y\": 8.397, \"z\": 46.21}, \"end\": {\"x\": 39.152, \"y\": 9.145, \"z\": 44.74}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.776, \"y\": 7.699, \"z\": 46.781}, \"end\": {\"x\": 35.848, \"y\": 8.173, \"z\": 44.89}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.176, \"y\": 5.637, \"z\": 48.368}, \"end\": {\"x\": 36.759, \"y\": 5.221, \"z\": 46.384}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.386, \"y\": 6.036, \"z\": 52.847}, \"end\": {\"x\": 35.353, \"y\": 4.797, \"z\": 52.294}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.389, \"y\": 6.449, \"z\": 50.766}, \"end\": {\"x\": 38.549, \"y\": 6.755, \"z\": 48.751}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.508, \"y\": 6.401, \"z\": 55.453}, \"end\": {\"x\": 36.018, \"y\": 7.83, \"z\": 54.475}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.386, \"y\": 6.036, \"z\": 52.847}, \"end\": {\"x\": 35.353, \"y\": 4.797, \"z\": 52.294}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.508, \"y\": 6.401, \"z\": 55.453}, \"end\": {\"x\": 36.018, \"y\": 7.83, \"z\": 54.475}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.918, \"y\": 10.367, \"z\": 63.127}, \"end\": {\"x\": 35.586, \"y\": 8.761, \"z\": 62.224}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.21866693600841, \"y\": 12.330188316307208, \"z\": 66.40965255518343}, \"end\": {\"x\": 33.74, \"y\": 11.731, \"z\": 64.744}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.918, \"y\": 10.367, \"z\": 63.127}, \"end\": {\"x\": 35.586, \"y\": 8.761, \"z\": 62.224}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.091, \"y\": 12.533, \"z\": 57.327}, \"end\": {\"x\": 42.807, \"y\": 11.802, \"z\": 59.618}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.169, \"y\": 15.057, \"z\": 54.381}, \"end\": {\"x\": 39.486, \"y\": 14.863, \"z\": 56.459}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.726, \"y\": 15.045, \"z\": 53.189}, \"end\": {\"x\": 42.906, \"y\": 14.714, \"z\": 55.38}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.091, \"y\": 12.533, \"z\": 57.327}, \"end\": {\"x\": 42.807, \"y\": 11.802, \"z\": 59.618}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.496, \"y\": 14.947, \"z\": 49.766}, \"end\": {\"x\": 40.039, \"y\": 14.13, \"z\": 51.491}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.169, \"y\": 15.057, \"z\": 54.381}, \"end\": {\"x\": 39.486, \"y\": 14.863, \"z\": 56.459}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.69, \"y\": 17.251, \"z\": 48.882}, \"end\": {\"x\": 41.81, \"y\": 16.994, \"z\": 51.054}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.726, \"y\": 15.045, \"z\": 53.189}, \"end\": {\"x\": 42.906, \"y\": 14.714, \"z\": 55.38}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.281, \"y\": 15.974, \"z\": 47.169}, \"end\": {\"x\": 44.031, \"y\": 15.165, \"z\": 49.233}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.488, \"y\": 15.039, \"z\": 45.317}, \"end\": {\"x\": 41.17, \"y\": 13.919, \"z\": 46.992}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.496, \"y\": 14.947, \"z\": 49.766}, \"end\": {\"x\": 40.039, \"y\": 14.13, \"z\": 51.491}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.606, \"y\": 17.455, \"z\": 44.367}, \"end\": {\"x\": 40.168, \"y\": 17.084, \"z\": 46.262}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.69, \"y\": 17.251, \"z\": 48.882}, \"end\": {\"x\": 41.81, \"y\": 16.994, \"z\": 51.054}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.901, \"y\": 18.516, \"z\": 43.346}, \"end\": {\"x\": 43.298, \"y\": 18.263, \"z\": 45.182}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.281, \"y\": 15.974, \"z\": 47.169}, \"end\": {\"x\": 44.031, \"y\": 15.165, \"z\": 49.233}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.223, \"y\": 16.745, \"z\": 41.253}, \"end\": {\"x\": 43.424, \"y\": 15.625, \"z\": 43.055}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.488, \"y\": 15.039, \"z\": 45.317}, \"end\": {\"x\": 41.17, \"y\": 13.919, \"z\": 46.992}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.095, \"y\": 17.435, \"z\": 39.906}, \"end\": {\"x\": 40.489, \"y\": 16.403, \"z\": 41.523}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.606, \"y\": 17.455, \"z\": 44.367}, \"end\": {\"x\": 40.168, \"y\": 17.084, \"z\": 46.262}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.5, \"y\": 20.045, \"z\": 39.068}, \"end\": {\"x\": 41.408, \"y\": 19.694, \"z\": 40.985}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.901, \"y\": 18.516, \"z\": 43.346}, \"end\": {\"x\": 43.298, \"y\": 18.263, \"z\": 45.182}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.66, \"y\": 19.607, \"z\": 37.466}, \"end\": {\"x\": 44.317, \"y\": 18.925, \"z\": 39.449}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.223, \"y\": 16.745, \"z\": 41.253}, \"end\": {\"x\": 43.424, \"y\": 15.625, \"z\": 43.055}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.442, \"y\": 18.152, \"z\": 35.494}, \"end\": {\"x\": 42.347, \"y\": 16.834, \"z\": 37.208}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.095, \"y\": 17.435, \"z\": 39.906}, \"end\": {\"x\": 40.489, \"y\": 16.403, \"z\": 41.523}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.89, \"y\": 20.114, \"z\": 34.393}, \"end\": {\"x\": 40.268, \"y\": 19.415, \"z\": 36.368}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.5, \"y\": 20.045, \"z\": 39.068}, \"end\": {\"x\": 41.408, \"y\": 19.694, \"z\": 40.985}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.798, \"y\": 21.649, \"z\": 33.397}, \"end\": {\"x\": 42.778, \"y\": 21.714, \"z\": 35.47}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.66, \"y\": 19.607, \"z\": 37.466}, \"end\": {\"x\": 44.317, \"y\": 18.925, \"z\": 39.449}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 44.068, \"y\": 20.32, \"z\": 31.387}, \"end\": {\"x\": 44.186, \"y\": 19.404, \"z\": 33.348}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.442, \"y\": 18.152, \"z\": 35.494}, \"end\": {\"x\": 42.347, \"y\": 16.834, \"z\": 37.208}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.789, \"y\": 20.312, \"z\": 29.894}, \"end\": {\"x\": 41.209, \"y\": 18.977, \"z\": 31.6}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.89, \"y\": 20.114, \"z\": 34.393}, \"end\": {\"x\": 40.268, \"y\": 19.415, \"z\": 36.368}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.177, \"y\": 22.884, \"z\": 29.042}, \"end\": {\"x\": 41.051, \"y\": 22.717, \"z\": 31.157}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.798, \"y\": 21.649, \"z\": 33.397}, \"end\": {\"x\": 42.778, \"y\": 21.714, \"z\": 35.47}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.45, \"y\": 23.447, \"z\": 27.724}, \"end\": {\"x\": 43.986, \"y\": 22.801, \"z\": 29.64}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 44.068, \"y\": 20.32, \"z\": 31.387}, \"end\": {\"x\": 44.186, \"y\": 19.404, \"z\": 33.348}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.898, \"y\": 21.791, \"z\": 25.541}, \"end\": {\"x\": 43.319, \"y\": 20.248, \"z\": 27.274}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.789, \"y\": 20.312, \"z\": 29.894}, \"end\": {\"x\": 41.209, \"y\": 18.977, \"z\": 31.6}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.049, \"y\": 23.144, \"z\": 24.363}, \"end\": {\"x\": 40.435, \"y\": 21.835, \"z\": 26.294}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.177, \"y\": 22.884, \"z\": 29.042}, \"end\": {\"x\": 41.051, \"y\": 22.717, \"z\": 31.157}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.092, \"y\": 25.195, \"z\": 23.403}, \"end\": {\"x\": 41.986, \"y\": 24.945, \"z\": 25.528}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.45, \"y\": 23.447, \"z\": 27.724}, \"end\": {\"x\": 43.986, \"y\": 22.801, \"z\": 29.64}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.898, \"y\": 21.791, \"z\": 25.541}, \"end\": {\"x\": 43.319, \"y\": 20.248, \"z\": 27.274}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.199, \"y\": 23.307, \"z\": 19.677}, \"end\": {\"x\": 41.603, \"y\": 22.046, \"z\": 21.623}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.049, \"y\": 23.144, \"z\": 24.363}, \"end\": {\"x\": 40.435, \"y\": 21.835, \"z\": 26.294}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.092, \"y\": 25.195, \"z\": 23.403}, \"end\": {\"x\": 41.986, \"y\": 24.945, \"z\": 25.528}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.199, \"y\": 23.307, \"z\": 19.677}, \"end\": {\"x\": 41.603, \"y\": 22.046, \"z\": 21.623}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.759, \"y\": 32.648, \"z\": 18.951}, \"end\": {\"x\": 33.315, \"y\": 34.433, \"z\": 18.0}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.79, \"y\": 33.758, \"z\": 23.271}, \"end\": {\"x\": 34.157, \"y\": 33.247, \"z\": 21.839}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.607528040070385, \"y\": 31.84705697453721, \"z\": 22.70185412508183}, \"end\": {\"x\": 34.157, \"y\": 33.247, \"z\": 21.839}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.759, \"y\": 32.648, \"z\": 18.951}, \"end\": {\"x\": 33.315, \"y\": 34.433, \"z\": 18.0}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.896, \"y\": 32.205, \"z\": 23.73}, \"end\": {\"x\": 36.745, \"y\": 31.991, \"z\": 21.645}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.431, \"y\": 33.858, \"z\": 25.086}, \"end\": {\"x\": 38.529, \"y\": 34.736, \"z\": 22.665}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.121, \"y\": 34.26, \"z\": 27.349}, \"end\": {\"x\": 36.211, \"y\": 35.1, \"z\": 25.576}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.79, \"y\": 33.758, \"z\": 23.271}, \"end\": {\"x\": 34.157, \"y\": 33.247, \"z\": 21.839}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.607528040070385, \"y\": 31.84705697453721, \"z\": 22.70185412508183}, \"end\": {\"x\": 34.157, \"y\": 33.247, \"z\": 21.839}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.369, \"y\": 31.742, \"z\": 28.341}, \"end\": {\"x\": 36.195, \"y\": 31.74, \"z\": 26.589}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.896, \"y\": 32.205, \"z\": 23.73}, \"end\": {\"x\": 36.745, \"y\": 31.991, \"z\": 21.645}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.053, \"y\": 31.756, \"z\": 28.943}, \"end\": {\"x\": 39.753, \"y\": 31.633, \"z\": 26.671}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.431, \"y\": 33.858, \"z\": 25.086}, \"end\": {\"x\": 38.529, \"y\": 34.736, \"z\": 22.665}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 44.5168841824215, \"y\": 36.144802735412114, \"z\": 27.69370086875567}, \"end\": {\"x\": 44.96668824977388, \"y\": 34.68488058383626, \"z\": 28.75493233274207}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.251, \"y\": 33.559, \"z\": 31.045}, \"end\": {\"x\": 39.668, \"y\": 34.4, \"z\": 28.992}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.121, \"y\": 34.26, \"z\": 27.349}, \"end\": {\"x\": 36.211, \"y\": 35.1, \"z\": 25.576}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.189476346955757, \"y\": 33.290865925054334, \"z\": 28.803122703741924}, \"end\": {\"x\": 29.428845209954513, \"y\": 31.717333628843285, \"z\": 28.257713713651913}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.675, \"y\": 32.399, \"z\": 32.888}, \"end\": {\"x\": 37.585, \"y\": 32.749, \"z\": 31.196}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.369, \"y\": 31.742, \"z\": 28.341}, \"end\": {\"x\": 36.195, \"y\": 31.74, \"z\": 26.589}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.093, \"y\": 30.116, \"z\": 33.359}, \"end\": {\"x\": 39.635, \"y\": 29.971, \"z\": 31.366}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.053, \"y\": 31.756, \"z\": 28.943}, \"end\": {\"x\": 39.753, \"y\": 31.633, \"z\": 26.671}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 44.5168841824215, \"y\": 36.144802735412114, \"z\": 27.69370086875567}, \"end\": {\"x\": 44.96668824977388, \"y\": 34.68488058383626, \"z\": 28.75493233274207}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.338, \"y\": 31.096, \"z\": 34.489}, \"end\": {\"x\": 42.308, \"y\": 31.807, \"z\": 32.426}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.251, \"y\": 33.559, \"z\": 31.045}, \"end\": {\"x\": 39.668, \"y\": 34.4, \"z\": 28.992}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.28, \"y\": 32.185, \"z\": 36.817}, \"end\": {\"x\": 40.58, \"y\": 33.11, \"z\": 35.143}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.675, \"y\": 32.399, \"z\": 32.888}, \"end\": {\"x\": 37.585, \"y\": 32.749, \"z\": 31.196}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.667, \"y\": 29.732, \"z\": 37.978}, \"end\": {\"x\": 39.338, \"y\": 30.112, \"z\": 36.295}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.093, \"y\": 30.116, \"z\": 33.359}, \"end\": {\"x\": 39.635, \"y\": 29.971, \"z\": 31.366}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.017, \"y\": 28.412, \"z\": 38.344}, \"end\": {\"x\": 42.57, \"y\": 28.737, \"z\": 36.39}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 44.83161294124585, \"y\": 28.497463981683758, \"z\": 35.54834550802836}, \"end\": {\"x\": 42.57, \"y\": 28.737, \"z\": 36.39}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.338, \"y\": 31.096, \"z\": 34.489}, \"end\": {\"x\": 42.308, \"y\": 31.807, \"z\": 32.426}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 44.006, \"y\": 30.098, \"z\": 40.253}, \"end\": {\"x\": 43.672, \"y\": 31.22, \"z\": 38.421}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.28, \"y\": 32.185, \"z\": 36.817}, \"end\": {\"x\": 40.58, \"y\": 33.11, \"z\": 35.143}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.01972261255976, \"y\": 34.2435750163747, \"z\": 42.06894430177419}, \"end\": {\"x\": 40.396678011848536, \"y\": 34.49134644417415, \"z\": 40.60980465820651}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.19788003259319, \"y\": 34.16201625530438, \"z\": 43.47043935100746}, \"end\": {\"x\": 39.05025230668446, \"y\": 33.72767488786936, \"z\": 42.06272361365636}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.548, \"y\": 29.831, \"z\": 42.526}, \"end\": {\"x\": 41.308, \"y\": 30.78, \"z\": 40.75}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.667, \"y\": 29.732, \"z\": 37.978}, \"end\": {\"x\": 39.338, \"y\": 30.112, \"z\": 36.295}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.368, \"y\": 27.328, \"z\": 42.955}, \"end\": {\"x\": 42.142, \"y\": 27.441, \"z\": 41.085}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.017, \"y\": 28.412, \"z\": 38.344}, \"end\": {\"x\": 42.57, \"y\": 28.737, \"z\": 36.39}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 44.83161294124585, \"y\": 28.497463981683758, \"z\": 35.54834550802836}, \"end\": {\"x\": 42.57, \"y\": 28.737, \"z\": 36.39}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 45.911, \"y\": 27.541, \"z\": 43.823}, \"end\": {\"x\": 45.667, \"y\": 27.911, \"z\": 41.536}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 44.006, \"y\": 30.098, \"z\": 40.253}, \"end\": {\"x\": 43.672, \"y\": 31.22, \"z\": 38.421}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.01972261255976, \"y\": 34.2435750163747, \"z\": 42.06894430177419}, \"end\": {\"x\": 40.396678011848536, \"y\": 34.49134644417415, \"z\": 40.60980465820651}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.19788003259319, \"y\": 34.16201625530438, \"z\": 43.47043935100746}, \"end\": {\"x\": 39.05025230668446, \"y\": 33.72767488786936, \"z\": 42.06272361365636}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 45.501, \"y\": 28.698, \"z\": 46.269}, \"end\": {\"x\": 45.046, \"y\": 29.917, \"z\": 44.248}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.548, \"y\": 29.831, \"z\": 42.526}, \"end\": {\"x\": 41.308, \"y\": 30.78, \"z\": 40.75}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 44.088, \"y\": 26.729, \"z\": 47.492}, \"end\": {\"x\": 42.972, \"y\": 27.507, \"z\": 45.958}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.368, \"y\": 27.328, \"z\": 42.955}, \"end\": {\"x\": 42.142, \"y\": 27.441, \"z\": 41.085}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 46.115, \"y\": 24.93, \"z\": 47.674}, \"end\": {\"x\": 45.623, \"y\": 25.158, \"z\": 45.673}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 45.911, \"y\": 27.541, \"z\": 43.823}, \"end\": {\"x\": 45.667, \"y\": 27.911, \"z\": 41.536}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 47.772, \"y\": 25.925, \"z\": 49.545}, \"end\": {\"x\": 47.791, \"y\": 27.152, \"z\": 47.518}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 45.501, \"y\": 28.698, \"z\": 46.269}, \"end\": {\"x\": 45.046, \"y\": 29.917, \"z\": 44.248}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 46.208, \"y\": 25.973, \"z\": 51.715}, \"end\": {\"x\": 45.513, \"y\": 27.195, \"z\": 50.13}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 44.088, \"y\": 26.729, \"z\": 47.492}, \"end\": {\"x\": 42.972, \"y\": 27.507, \"z\": 45.958}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 46.229, \"y\": 23.291, \"z\": 52.236}, \"end\": {\"x\": 45.004, \"y\": 23.69, \"z\": 50.209}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 46.115, \"y\": 24.93, \"z\": 47.674}, \"end\": {\"x\": 45.623, \"y\": 25.158, \"z\": 45.673}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 48.639, \"y\": 22.565, \"z\": 53.084}, \"end\": {\"x\": 48.537, \"y\": 23.169, \"z\": 50.556}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 47.772, \"y\": 25.925, \"z\": 49.545}, \"end\": {\"x\": 47.791, \"y\": 27.152, \"z\": 47.518}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 48.587, \"y\": 23.93, \"z\": 55.45}, \"end\": {\"x\": 48.494, \"y\": 25.19, \"z\": 53.553}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 46.208, \"y\": 25.973, \"z\": 51.715}, \"end\": {\"x\": 45.513, \"y\": 27.195, \"z\": 50.13}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 46.229, \"y\": 23.291, \"z\": 52.236}, \"end\": {\"x\": 45.004, \"y\": 23.69, \"z\": 50.209}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 47.146, \"y\": 22.094, \"z\": 56.728}, \"end\": {\"x\": 47.492, \"y\": 20.651, \"z\": 55.141}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 48.626, \"y\": 20.244, \"z\": 57.372}, \"end\": {\"x\": 47.492, \"y\": 20.651, \"z\": 55.141}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 48.639, \"y\": 22.565, \"z\": 53.084}, \"end\": {\"x\": 48.537, \"y\": 23.169, \"z\": 50.556}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 48.587, \"y\": 23.93, \"z\": 55.45}, \"end\": {\"x\": 48.494, \"y\": 25.19, \"z\": 53.553}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 48.687, \"y\": 21.991, \"z\": 61.669}, \"end\": {\"x\": 47.992, \"y\": 23.038, \"z\": 59.486}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 47.146, \"y\": 22.094, \"z\": 56.728}, \"end\": {\"x\": 47.492, \"y\": 20.651, \"z\": 55.141}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 48.626, \"y\": 20.244, \"z\": 57.372}, \"end\": {\"x\": 47.492, \"y\": 20.651, \"z\": 55.141}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 48.687, \"y\": 21.991, \"z\": 61.669}, \"end\": {\"x\": 47.992, \"y\": 23.038, \"z\": 59.486}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 46.48112143287274, \"y\": 24.43688560293486, \"z\": 64.69180973353596}, \"end\": {\"x\": 44.74873557380715, \"y\": 24.72563549164821, \"z\": 65.28017034524638}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 46.48112143287274, \"y\": 24.43688560293486, \"z\": 64.69180973353596}, \"end\": {\"x\": 44.74873557380715, \"y\": 24.72563549164821, \"z\": 65.28017034524638}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.797, \"y\": 24.07, \"z\": 60.161}, \"end\": {\"x\": 37.928, \"y\": 24.036, \"z\": 62.513}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.906, \"y\": 26.033, \"z\": 58.599}, \"end\": {\"x\": 39.37, \"y\": 26.633, \"z\": 60.713}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 39.74, \"y\": 24.65, \"z\": 56.429}, \"end\": {\"x\": 40.909, \"y\": 24.634, \"z\": 58.547}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.627, \"y\": 23.654, \"z\": 55.284}, \"end\": {\"x\": 38.302, \"y\": 23.034, \"z\": 57.354}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.797, \"y\": 24.07, \"z\": 60.161}, \"end\": {\"x\": 37.928, \"y\": 24.036, \"z\": 62.513}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.42, \"y\": 25.933, \"z\": 54.347}, \"end\": {\"x\": 36.581, \"y\": 25.993, \"z\": 56.648}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.906, \"y\": 26.033, \"z\": 58.599}, \"end\": {\"x\": 39.37, \"y\": 26.633, \"z\": 60.713}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.069, \"y\": 26.712, \"z\": 52.316}, \"end\": {\"x\": 39.383, \"y\": 27.024, \"z\": 54.576}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 39.74, \"y\": 24.65, \"z\": 56.429}, \"end\": {\"x\": 40.909, \"y\": 24.634, \"z\": 58.547}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.744, \"y\": 24.72, \"z\": 50.686}, \"end\": {\"x\": 38.716, \"y\": 24.089, \"z\": 52.487}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.627, \"y\": 23.654, \"z\": 55.284}, \"end\": {\"x\": 38.302, \"y\": 23.034, \"z\": 57.354}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.228, \"y\": 25.135, \"z\": 49.723}, \"end\": {\"x\": 35.326, \"y\": 24.495, \"z\": 51.91}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.42, \"y\": 25.933, \"z\": 54.347}, \"end\": {\"x\": 36.581, \"y\": 25.993, \"z\": 56.648}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.423, \"y\": 27.371, \"z\": 48.234}, \"end\": {\"x\": 35.747, \"y\": 27.749, \"z\": 50.672}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.069, \"y\": 26.712, \"z\": 52.316}, \"end\": {\"x\": 39.383, \"y\": 27.024, \"z\": 54.576}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.922, \"y\": 26.359, \"z\": 46.281}, \"end\": {\"x\": 38.09, \"y\": 26.42, \"z\": 48.193}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.744, \"y\": 24.72, \"z\": 50.686}, \"end\": {\"x\": 38.716, \"y\": 24.089, \"z\": 52.487}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.359, \"y\": 24.585, \"z\": 45.02}, \"end\": {\"x\": 35.91, \"y\": 23.91, \"z\": 47.037}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.228, \"y\": 25.135, \"z\": 49.723}, \"end\": {\"x\": 35.326, \"y\": 24.495, \"z\": 51.91}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.367, \"y\": 26.286, \"z\": 44.199}, \"end\": {\"x\": 33.445, \"y\": 26.337, \"z\": 46.181}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.423, \"y\": 27.371, \"z\": 48.234}, \"end\": {\"x\": 35.747, \"y\": 27.749, \"z\": 50.672}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.944, \"y\": 27.801, \"z\": 42.413}, \"end\": {\"x\": 35.76, \"y\": 28.216, \"z\": 44.182}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.922, \"y\": 26.359, \"z\": 46.281}, \"end\": {\"x\": 38.09, \"y\": 26.42, \"z\": 48.193}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.317, \"y\": 25.776, \"z\": 40.63}, \"end\": {\"x\": 36.441, \"y\": 25.39, \"z\": 42.3}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.359, \"y\": 24.585, \"z\": 45.02}, \"end\": {\"x\": 35.91, \"y\": 23.91, \"z\": 47.037}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.961, \"y\": 25.067, \"z\": 39.57}, \"end\": {\"x\": 33.155, \"y\": 24.852, \"z\": 41.536}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.367, \"y\": 26.286, \"z\": 44.199}, \"end\": {\"x\": 33.445, \"y\": 26.337, \"z\": 46.181}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.224271632369327, \"y\": 30.276877354867583, \"z\": 46.32545610227355}, \"end\": {\"x\": 31.510631876663993, \"y\": 30.67397005123429, \"z\": 45.13401426878197}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.529, \"y\": 27.397, \"z\": 38.257}, \"end\": {\"x\": 32.656, \"y\": 27.88, \"z\": 40.417}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.944, \"y\": 27.801, \"z\": 42.413}, \"end\": {\"x\": 35.76, \"y\": 28.216, \"z\": 44.182}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.461, \"y\": 27.389, \"z\": 36.378}, \"end\": {\"x\": 35.443, \"y\": 27.678, \"z\": 38.275}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.317, \"y\": 25.776, \"z\": 40.63}, \"end\": {\"x\": 36.441, \"y\": 25.39, \"z\": 42.3}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.566, \"y\": 25.282, \"z\": 34.917}, \"end\": {\"x\": 34.174, \"y\": 24.635, \"z\": 36.825}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.44013511333774, \"y\": 22.064469540028885, \"z\": 37.23236983890624}, \"end\": {\"x\": 35.77134407051643, \"y\": 22.567117783120626, \"z\": 38.70923076436494}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.961, \"y\": 25.067, \"z\": 39.57}, \"end\": {\"x\": 33.155, \"y\": 24.852, \"z\": 41.536}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.348, \"y\": 26.231, \"z\": 33.856}, \"end\": {\"x\": 31.047, \"y\": 26.023, \"z\": 35.995}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.529, \"y\": 27.397, \"z\": 38.257}, \"end\": {\"x\": 32.656, \"y\": 27.88, \"z\": 40.417}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.757587951021616, \"y\": 30.335986356774868, \"z\": 34.896689777867465}, \"end\": {\"x\": 30.14420731503131, \"y\": 30.664761524126256, \"z\": 36.412283775164674}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.28530486774137, \"y\": 32.50091487463622, \"z\": 34.40550002704736}, \"end\": {\"x\": 30.95714299824809, \"y\": 32.35975413665545, \"z\": 35.42542876525172}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.193, \"y\": 28.162, \"z\": 32.195}, \"end\": {\"x\": 32.635, \"y\": 28.654, \"z\": 34.314}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.461, \"y\": 27.389, \"z\": 36.378}, \"end\": {\"x\": 35.443, \"y\": 27.678, \"z\": 38.275}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.55, \"y\": 26.621, \"z\": 30.544}, \"end\": {\"x\": 34.582, \"y\": 26.463, \"z\": 32.317}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.566, \"y\": 25.282, \"z\": 34.917}, \"end\": {\"x\": 34.174, \"y\": 24.635, \"z\": 36.825}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.44013511333774, \"y\": 22.064469540028885, \"z\": 37.23236983890624}, \"end\": {\"x\": 35.77134407051643, \"y\": 22.567117783120626, \"z\": 38.70923076436494}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.554, \"y\": 25.34, \"z\": 29.228}, \"end\": {\"x\": 31.623, \"y\": 24.569, \"z\": 31.337}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.819871868901842, \"y\": 20.097132435132995, \"z\": 33.63873775807972}, \"end\": {\"x\": 32.77328843911919, \"y\": 20.32190090795406, \"z\": 35.184731349317204}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.66974175102478, \"y\": 20.5402127688388, \"z\": 37.488125467948954}, \"end\": {\"x\": 34.08108314865374, \"y\": 18.941081444814955, \"z\": 38.50586644813445}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.348, \"y\": 26.231, \"z\": 33.856}, \"end\": {\"x\": 31.047, \"y\": 26.023, \"z\": 35.995}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.757587951021616, \"y\": 30.335986356774868, \"z\": 34.896689777867465}, \"end\": {\"x\": 30.14420731503131, \"y\": 30.664761524126256, \"z\": 36.412283775164674}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.28530486774137, \"y\": 32.50091487463622, \"z\": 34.40550002704736}, \"end\": {\"x\": 30.95714299824809, \"y\": 32.35975413665545, \"z\": 35.42542876525172}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.383, \"y\": 27.283, \"z\": 27.774}, \"end\": {\"x\": 30.09, \"y\": 27.49, \"z\": 30.119}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.193, \"y\": 28.162, \"z\": 32.195}, \"end\": {\"x\": 32.635, \"y\": 28.654, \"z\": 34.314}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.454, \"y\": 27.837, \"z\": 26.06}, \"end\": {\"x\": 32.967, \"y\": 28.375, \"z\": 28.14}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.55, \"y\": 26.621, \"z\": 30.544}, \"end\": {\"x\": 34.582, \"y\": 26.463, \"z\": 32.317}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.253, \"y\": 25.527, \"z\": 24.63}, \"end\": {\"x\": 33.143, \"y\": 25.226, \"z\": 26.643}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.554, \"y\": 25.34, \"z\": 29.228}, \"end\": {\"x\": 31.623, \"y\": 24.569, \"z\": 31.337}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.942, \"y\": 25.728, \"z\": 23.449}, \"end\": {\"x\": 29.869, \"y\": 25.412, \"z\": 25.445}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.189476346955757, \"y\": 33.290865925054334, \"z\": 28.803122703741924}, \"end\": {\"x\": 29.428845209954513, \"y\": 31.717333628843285, \"z\": 28.257713713651913}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.383, \"y\": 27.283, \"z\": 27.774}, \"end\": {\"x\": 30.09, \"y\": 27.49, \"z\": 30.119}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.498, \"y\": 27.8, \"z\": 21.584}, \"end\": {\"x\": 30.588, \"y\": 28.339, \"z\": 23.764}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.454, \"y\": 27.837, \"z\": 26.06}, \"end\": {\"x\": 32.967, \"y\": 28.375, \"z\": 28.14}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.44, \"y\": 26.572, \"z\": 19.998}, \"end\": {\"x\": 33.284, \"y\": 26.782, \"z\": 22.141}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.253, \"y\": 25.527, \"z\": 24.63}, \"end\": {\"x\": 33.143, \"y\": 25.226, \"z\": 26.643}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.765, \"y\": 24.677, \"z\": 18.769}, \"end\": {\"x\": 30.899, \"y\": 24.317, \"z\": 21.012}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.32683807501296, \"y\": 20.35260431802973, \"z\": 23.540226834555213}, \"end\": {\"x\": 32.18137603332113, \"y\": 20.741199970340038, \"z\": 25.16323827728038}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.72087825595617, \"y\": 20.880175444877075, \"z\": 27.136601848196545}, \"end\": {\"x\": 34.31508507317345, \"y\": 19.48007487537707, \"z\": 28.20927142849176}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.942, \"y\": 25.728, \"z\": 23.449}, \"end\": {\"x\": 29.869, \"y\": 25.412, \"z\": 25.445}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.498, \"y\": 27.8, \"z\": 21.584}, \"end\": {\"x\": 30.588, \"y\": 28.339, \"z\": 23.764}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.44, \"y\": 26.572, \"z\": 19.998}, \"end\": {\"x\": 33.284, \"y\": 26.782, \"z\": 22.141}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.765, \"y\": 24.677, \"z\": 18.769}, \"end\": {\"x\": 30.899, \"y\": 24.317, \"z\": 21.012}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.05, \"y\": 16.161, \"z\": 17.689}, \"end\": {\"x\": 27.635, \"y\": 15.974, \"z\": 15.34}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.904, \"y\": 17.864, \"z\": 18.589}, \"end\": {\"x\": 24.509, \"y\": 17.274, \"z\": 16.363}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.771, \"y\": 16.247, \"z\": 20.35}, \"end\": {\"x\": 23.651, \"y\": 14.944, \"z\": 18.729}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.45, \"y\": 15.92, \"z\": 22.281}, \"end\": {\"x\": 26.355, \"y\": 15.108, \"z\": 20.443}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.05, \"y\": 16.161, \"z\": 17.689}, \"end\": {\"x\": 27.635, \"y\": 15.974, \"z\": 15.34}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.136455906181823, \"y\": 16.967498478933162, \"z\": 22.205397136733485}, \"end\": {\"x\": 26.404, \"y\": 18.333, \"z\": 21.18}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.667, \"y\": 18.368, \"z\": 23.214}, \"end\": {\"x\": 26.404, \"y\": 18.333, \"z\": 21.18}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.904, \"y\": 17.864, \"z\": 18.589}, \"end\": {\"x\": 24.509, \"y\": 17.274, \"z\": 16.363}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.25, \"y\": 18.616, \"z\": 24.493}, \"end\": {\"x\": 22.797, \"y\": 18.512, \"z\": 22.116}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.771, \"y\": 16.247, \"z\": 20.35}, \"end\": {\"x\": 23.651, \"y\": 14.944, \"z\": 18.729}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.432, \"y\": 16.91, \"z\": 26.514}, \"end\": {\"x\": 23.301, \"y\": 15.781, \"z\": 24.415}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.589563925674486, \"y\": 12.078131537104372, \"z\": 25.25878934509002}, \"end\": {\"x\": 23.965311485529828, \"y\": 12.449716774199903, \"z\": 23.51205044150279}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.45, \"y\": 15.92, \"z\": 22.281}, \"end\": {\"x\": 26.355, \"y\": 15.108, \"z\": 20.443}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.136455906181823, \"y\": 16.967498478933162, \"z\": 22.205397136733485}, \"end\": {\"x\": 26.404, \"y\": 18.333, \"z\": 21.18}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.427, \"y\": 18.031, \"z\": 27.743}, \"end\": {\"x\": 26.129, \"y\": 17.442, \"z\": 26.064}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.667, \"y\": 18.368, \"z\": 23.214}, \"end\": {\"x\": 26.404, \"y\": 18.333, \"z\": 21.18}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.291, \"y\": 20.259, \"z\": 28.618}, \"end\": {\"x\": 24.414, \"y\": 20.459, \"z\": 26.601}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.25, \"y\": 18.616, \"z\": 24.493}, \"end\": {\"x\": 22.797, \"y\": 18.512, \"z\": 22.116}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.454, \"y\": 19.274, \"z\": 30.463}, \"end\": {\"x\": 21.866, \"y\": 18.703, \"z\": 28.391}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.432, \"y\": 16.91, \"z\": 26.514}, \"end\": {\"x\": 23.301, \"y\": 15.781, \"z\": 24.415}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.589563925674486, \"y\": 12.078131537104372, \"z\": 25.25878934509002}, \"end\": {\"x\": 23.965311485529828, \"y\": 12.449716774199903, \"z\": 23.51205044150279}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.157850437434107, \"y\": 11.27474139717846, \"z\": 25.58315812187468}, \"end\": {\"x\": 29.173031995488035, \"y\": 11.484988568726713, \"z\": 25.344087608353572}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.841190198975447, \"y\": 10.81166564254388, \"z\": 27.72207275072217}, \"end\": {\"x\": 29.806315435843562, \"y\": 10.472892507048423, \"z\": 27.105802516057818}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.177, \"y\": 18.134, \"z\": 32.237}, \"end\": {\"x\": 24.459, \"y\": 17.24, \"z\": 30.507}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.427, \"y\": 18.031, \"z\": 27.743}, \"end\": {\"x\": 26.129, \"y\": 17.442, \"z\": 26.064}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.262, \"y\": 20.57, \"z\": 33.201}, \"end\": {\"x\": 25.968, \"y\": 20.323, \"z\": 31.144}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.291, \"y\": 20.259, \"z\": 28.618}, \"end\": {\"x\": 24.414, \"y\": 20.459, \"z\": 26.601}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.223, \"y\": 21.668, \"z\": 34.52}, \"end\": {\"x\": 23.015, \"y\": 21.602, \"z\": 32.33}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.454, \"y\": 19.274, \"z\": 30.463}, \"end\": {\"x\": 21.866, \"y\": 18.703, \"z\": 28.391}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.94593277204187, \"y\": 20.80424270142054, \"z\": 31.897368431077453}, \"end\": {\"x\": 19.194710442396982, \"y\": 18.954379243328006, \"z\": 31.022428820684457}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.03790525184778, \"y\": 19.037638159609415, \"z\": 32.582111819737065}, \"end\": {\"x\": 19.194710442396982, \"y\": 18.954379243328006, \"z\": 31.022428820684457}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.081, \"y\": 19.898, \"z\": 36.603}, \"end\": {\"x\": 22.55, \"y\": 19.111, \"z\": 34.601}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.177, \"y\": 18.134, \"z\": 32.237}, \"end\": {\"x\": 24.459, \"y\": 17.24, \"z\": 30.507}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.394, \"y\": 20.436, \"z\": 37.957}, \"end\": {\"x\": 25.744, \"y\": 19.357, \"z\": 35.936}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.262, \"y\": 20.57, \"z\": 33.201}, \"end\": {\"x\": 25.968, \"y\": 20.323, \"z\": 31.144}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.918, \"y\": 23.023, \"z\": 38.917}, \"end\": {\"x\": 25.029, \"y\": 22.751, \"z\": 36.704}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.223, \"y\": 21.668, \"z\": 34.52}, \"end\": {\"x\": 23.015, \"y\": 21.602, \"z\": 32.33}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.94593277204187, \"y\": 20.80424270142054, \"z\": 31.897368431077453}, \"end\": {\"x\": 19.194710442396982, \"y\": 18.954379243328006, \"z\": 31.022428820684457}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.03790525184778, \"y\": 19.037638159609415, \"z\": 32.582111819737065}, \"end\": {\"x\": 19.194710442396982, \"y\": 18.954379243328006, \"z\": 31.022428820684457}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.035, \"y\": 22.404, \"z\": 40.744}, \"end\": {\"x\": 22.236, \"y\": 22.068, \"z\": 38.546}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.081, \"y\": 19.898, \"z\": 36.603}, \"end\": {\"x\": 22.55, \"y\": 19.111, \"z\": 34.601}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.59815944893826, \"y\": 18.199374880194412, \"z\": 41.726317548017576}, \"end\": {\"x\": 21.439824144373016, \"y\": 17.346597985060296, \"z\": 40.044344371162204}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.602348882858973, \"y\": 16.45989375526764, \"z\": 42.76911447950715}, \"end\": {\"x\": 22.76602818806995, \"y\": 16.27326641720404, \"z\": 41.30500076771362}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.104, \"y\": 20.885, \"z\": 42.672}, \"end\": {\"x\": 24.058, \"y\": 19.984, \"z\": 40.639}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.394, \"y\": 20.436, \"z\": 37.957}, \"end\": {\"x\": 25.744, \"y\": 19.357, \"z\": 35.936}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.819, \"y\": 22.741, \"z\": 43.557}, \"end\": {\"x\": 26.339, \"y\": 22.383, \"z\": 41.519}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.918, \"y\": 23.023, \"z\": 38.917}, \"end\": {\"x\": 25.029, \"y\": 22.751, \"z\": 36.704}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.289, \"y\": 24.519, \"z\": 44.722}, \"end\": {\"x\": 23.774, \"y\": 24.727, \"z\": 42.546}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.035, \"y\": 22.404, \"z\": 40.744}, \"end\": {\"x\": 22.236, \"y\": 22.068, \"z\": 38.546}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.59815944893826, \"y\": 18.199374880194412, \"z\": 41.726317548017576}, \"end\": {\"x\": 21.439824144373016, \"y\": 17.346597985060296, \"z\": 40.044344371162204}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.602348882858973, \"y\": 16.45989375526764, \"z\": 42.76911447950715}, \"end\": {\"x\": 22.76602818806995, \"y\": 16.27326641720404, \"z\": 41.30500076771362}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.364, \"y\": 23.097, \"z\": 46.881}, \"end\": {\"x\": 22.635, \"y\": 22.42, \"z\": 44.836}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.104, \"y\": 20.885, \"z\": 42.672}, \"end\": {\"x\": 24.058, \"y\": 19.984, \"z\": 40.639}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.756, \"y\": 22.948, \"z\": 48.072}, \"end\": {\"x\": 25.772, \"y\": 21.684, \"z\": 46.403}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.819, \"y\": 22.741, \"z\": 43.557}, \"end\": {\"x\": 26.339, \"y\": 22.383, \"z\": 41.519}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.123, \"y\": 25.333, \"z\": 49.056}, \"end\": {\"x\": 26.356, \"y\": 25.182, \"z\": 46.844}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.289, \"y\": 24.519, \"z\": 44.722}, \"end\": {\"x\": 23.774, \"y\": 24.727, \"z\": 42.546}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.399, \"y\": 25.596, \"z\": 50.785}, \"end\": {\"x\": 23.55, \"y\": 25.537, \"z\": 48.67}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.364, \"y\": 23.097, \"z\": 46.881}, \"end\": {\"x\": 22.635, \"y\": 22.42, \"z\": 44.836}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.016, \"y\": 24.035, \"z\": 52.772}, \"end\": {\"x\": 24.569, \"y\": 22.85, \"z\": 50.86}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.756, \"y\": 22.948, \"z\": 48.072}, \"end\": {\"x\": 25.772, \"y\": 21.684, \"z\": 46.403}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.064, \"y\": 25.273, \"z\": 53.587}, \"end\": {\"x\": 27.856, \"y\": 24.363, \"z\": 51.347}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.224271632369327, \"y\": 30.276877354867583, \"z\": 46.32545610227355}, \"end\": {\"x\": 31.510631876663993, \"y\": 30.67397005123429, \"z\": 45.13401426878197}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.123, \"y\": 25.333, \"z\": 49.056}, \"end\": {\"x\": 26.356, \"y\": 25.182, \"z\": 46.844}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.399, \"y\": 25.596, \"z\": 50.785}, \"end\": {\"x\": 23.55, \"y\": 25.537, \"z\": 48.67}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.016, \"y\": 24.035, \"z\": 52.772}, \"end\": {\"x\": 24.569, \"y\": 22.85, \"z\": 50.86}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.79, \"y\": 25.183, \"z\": 58.359}, \"end\": {\"x\": 27.083, \"y\": 23.993, \"z\": 56.336}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.064, \"y\": 25.273, \"z\": 53.587}, \"end\": {\"x\": 27.856, \"y\": 24.363, \"z\": 51.347}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.804, \"y\": 27.588, \"z\": 59.308}, \"end\": {\"x\": 28.907, \"y\": 27.181, \"z\": 56.754}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.149, \"y\": 28.59, \"z\": 61.293}, \"end\": {\"x\": 26.188, \"y\": 28.422, \"z\": 59.054}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.939, \"y\": 26.68, \"z\": 63.176}, \"end\": {\"x\": 26.818, \"y\": 25.729, \"z\": 61.183}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.79, \"y\": 25.183, \"z\": 58.359}, \"end\": {\"x\": 27.083, \"y\": 23.993, \"z\": 56.336}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.804, \"y\": 27.588, \"z\": 59.308}, \"end\": {\"x\": 28.907, \"y\": 27.181, \"z\": 56.754}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.149, \"y\": 28.59, \"z\": 61.293}, \"end\": {\"x\": 26.188, \"y\": 28.422, \"z\": 59.054}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.939, \"y\": 26.68, \"z\": 63.176}, \"end\": {\"x\": 26.818, \"y\": 25.729, \"z\": 61.183}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.64119334542987, \"y\": 22.88729805832813, \"z\": 65.95167460773206}, \"end\": {\"x\": 27.931636423541722, \"y\": 21.45143039853578, \"z\": 66.57403245473779}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.37, \"y\": 22.081, \"z\": 63.926}, \"end\": {\"x\": 28.447, \"y\": 23.835, \"z\": 63.869}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.64119334542987, \"y\": 22.88729805832813, \"z\": 65.95167460773206}, \"end\": {\"x\": 27.931636423541722, \"y\": 21.45143039853578, \"z\": 66.57403245473779}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.37, \"y\": 22.081, \"z\": 63.926}, \"end\": {\"x\": 28.447, \"y\": 23.835, \"z\": 63.869}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.21866693600841, \"y\": 12.330188316307208, \"z\": 66.40965255518343}, \"end\": {\"x\": 33.74, \"y\": 11.731, \"z\": 64.744}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.853, \"y\": 16.164, \"z\": 59.522}, \"end\": {\"x\": 31.477, \"y\": 15.932, \"z\": 61.848}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.62, \"y\": 16.109, \"z\": 57.815}, \"end\": {\"x\": 28.05, \"y\": 15.954, \"z\": 60.054}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.238, \"y\": 18.087, \"z\": 56.065}, \"end\": {\"x\": 28.997, \"y\": 18.64, \"z\": 58.015}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.416, \"y\": 16.898, \"z\": 54.815}, \"end\": {\"x\": 31.911, \"y\": 17.279, \"z\": 56.906}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.853, \"y\": 16.164, \"z\": 59.522}, \"end\": {\"x\": 31.477, \"y\": 15.932, \"z\": 61.848}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.142, \"y\": 14.978, \"z\": 53.369}, \"end\": {\"x\": 30.222, \"y\": 14.625, \"z\": 55.707}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.62, \"y\": 16.109, \"z\": 57.815}, \"end\": {\"x\": 28.05, \"y\": 15.954, \"z\": 60.054}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.645, \"y\": 16.312, \"z\": 51.447}, \"end\": {\"x\": 28.124, \"y\": 16.521, \"z\": 53.719}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.238, \"y\": 18.087, \"z\": 56.065}, \"end\": {\"x\": 28.997, \"y\": 18.64, \"z\": 58.015}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.691, \"y\": 17.367, \"z\": 50.109}, \"end\": {\"x\": 30.706, \"y\": 18.194, \"z\": 52.168}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.416, \"y\": 16.898, \"z\": 54.815}, \"end\": {\"x\": 31.911, \"y\": 17.279, \"z\": 56.906}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.771, \"y\": 15.21, \"z\": 48.814}, \"end\": {\"x\": 32.119, \"y\": 15.207, \"z\": 51.08}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.142, \"y\": 14.978, \"z\": 53.369}, \"end\": {\"x\": 30.222, \"y\": 14.625, \"z\": 55.707}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.69, \"y\": 14.675, \"z\": 47.211}, \"end\": {\"x\": 29.327, \"y\": 14.221, \"z\": 49.362}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.645, \"y\": 16.312, \"z\": 51.447}, \"end\": {\"x\": 28.124, \"y\": 16.521, \"z\": 53.719}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.807, \"y\": 16.858, \"z\": 45.618}, \"end\": {\"x\": 29.207, \"y\": 17.346, \"z\": 47.463}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.691, \"y\": 17.367, \"z\": 50.109}, \"end\": {\"x\": 30.706, \"y\": 18.194, \"z\": 52.168}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.216, \"y\": 16.528, \"z\": 44.361}, \"end\": {\"x\": 32.427, \"y\": 16.884, \"z\": 46.368}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.771, \"y\": 15.21, \"z\": 48.814}, \"end\": {\"x\": 32.119, \"y\": 15.207, \"z\": 51.08}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.674, \"y\": 14.204, \"z\": 43.037}, \"end\": {\"x\": 31.471, \"y\": 13.728, \"z\": 44.948}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.69, \"y\": 14.675, \"z\": 47.211}, \"end\": {\"x\": 29.327, \"y\": 14.221, \"z\": 49.362}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.611, \"y\": 15.106, \"z\": 41.273}, \"end\": {\"x\": 28.932, \"y\": 15.289, \"z\": 43.176}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.807, \"y\": 16.858, \"z\": 45.618}, \"end\": {\"x\": 29.207, \"y\": 17.346, \"z\": 47.463}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.344, \"y\": 16.732, \"z\": 39.924}, \"end\": {\"x\": 31.067, \"y\": 17.521, \"z\": 41.737}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.216, \"y\": 16.528, \"z\": 44.361}, \"end\": {\"x\": 32.427, \"y\": 16.884, \"z\": 46.368}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.164, \"y\": 15.183, \"z\": 38.618}, \"end\": {\"x\": 33.307, \"y\": 15.065, \"z\": 40.633}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.674, \"y\": 14.204, \"z\": 43.037}, \"end\": {\"x\": 31.471, \"y\": 13.728, \"z\": 44.948}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.385, \"y\": 13.938, \"z\": 36.981}, \"end\": {\"x\": 30.931, \"y\": 13.574, \"z\": 39.011}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.611, \"y\": 15.106, \"z\": 41.273}, \"end\": {\"x\": 28.932, \"y\": 15.289, \"z\": 43.176}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.53, \"y\": 15.995, \"z\": 35.33}, \"end\": {\"x\": 29.849, \"y\": 16.373, \"z\": 37.309}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.66974175102478, \"y\": 20.5402127688388, \"z\": 37.488125467948954}, \"end\": {\"x\": 34.08108314865374, \"y\": 18.941081444814955, \"z\": 38.50586644813445}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.344, \"y\": 16.732, \"z\": 39.924}, \"end\": {\"x\": 31.067, \"y\": 17.521, \"z\": 41.737}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.981, \"y\": 16.461, \"z\": 34.044}, \"end\": {\"x\": 33.021, \"y\": 16.957, \"z\": 36.162}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.14916747448213, \"y\": 18.2588541617528, \"z\": 36.823841674703004}, \"end\": {\"x\": 34.08108314865374, \"y\": 18.941081444814955, \"z\": 38.50586644813445}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.164, \"y\": 15.183, \"z\": 38.618}, \"end\": {\"x\": 33.307, \"y\": 15.065, \"z\": 40.633}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.013, \"y\": 14.28, \"z\": 32.521}, \"end\": {\"x\": 33.37, \"y\": 13.853, \"z\": 34.688}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.385, \"y\": 13.938, \"z\": 36.981}, \"end\": {\"x\": 30.931, \"y\": 13.574, \"z\": 39.011}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.102, \"y\": 14.448, \"z\": 30.623}, \"end\": {\"x\": 30.467, \"y\": 14.168, \"z\": 32.937}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.819871868901842, \"y\": 20.097132435132995, \"z\": 33.63873775807972}, \"end\": {\"x\": 32.77328843911919, \"y\": 20.32190090795406, \"z\": 35.184731349317204}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.53, \"y\": 15.995, \"z\": 35.33}, \"end\": {\"x\": 29.849, \"y\": 16.373, \"z\": 37.309}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.125, \"y\": 16.599, \"z\": 29.414}, \"end\": {\"x\": 31.606, \"y\": 17.049, \"z\": 31.503}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.981, \"y\": 16.461, \"z\": 34.044}, \"end\": {\"x\": 33.021, \"y\": 16.957, \"z\": 36.162}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.14916747448213, \"y\": 18.2588541617528, \"z\": 36.823841674703004}, \"end\": {\"x\": 34.08108314865374, \"y\": 18.941081444814955, \"z\": 38.50586644813445}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.535, \"y\": 15.711, \"z\": 28.242}, \"end\": {\"x\": 34.723, \"y\": 15.573, \"z\": 30.375}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.013, \"y\": 14.28, \"z\": 32.521}, \"end\": {\"x\": 33.37, \"y\": 13.853, \"z\": 34.688}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.335, \"y\": 13.983, \"z\": 26.435}, \"end\": {\"x\": 32.976, \"y\": 13.342, \"z\": 28.511}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.102, \"y\": 14.448, \"z\": 30.623}, \"end\": {\"x\": 30.467, \"y\": 14.168, \"z\": 32.937}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.222, \"y\": 15.911, \"z\": 24.692}, \"end\": {\"x\": 31.153, \"y\": 15.726, \"z\": 26.677}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.72087825595617, \"y\": 20.880175444877075, \"z\": 27.136601848196545}, \"end\": {\"x\": 34.31508507317345, \"y\": 19.48007487537707, \"z\": 28.20927142849176}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.125, \"y\": 16.599, \"z\": 29.414}, \"end\": {\"x\": 31.606, \"y\": 17.049, \"z\": 31.503}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.378, \"y\": 16.845, \"z\": 23.648}, \"end\": {\"x\": 33.967, \"y\": 17.371, \"z\": 25.765}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.014739213966315, \"y\": 19.16640831016724, \"z\": 26.700836371012787}, \"end\": {\"x\": 34.31508507317345, \"y\": 19.48007487537707, \"z\": 28.20927142849176}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.535, \"y\": 15.711, \"z\": 28.242}, \"end\": {\"x\": 34.723, \"y\": 15.573, \"z\": 30.375}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.327, \"y\": 14.938, \"z\": 22.093}, \"end\": {\"x\": 35.415, \"y\": 14.603, \"z\": 24.314}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.157850437434107, \"y\": 11.27474139717846, \"z\": 25.58315812187468}, \"end\": {\"x\": 29.173031995488035, \"y\": 11.484988568726713, \"z\": 25.344087608353572}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.841190198975447, \"y\": 10.81166564254388, \"z\": 27.72207275072217}, \"end\": {\"x\": 29.806315435843562, \"y\": 10.472892507048423, \"z\": 27.105802516057818}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.335, \"y\": 13.983, \"z\": 26.435}, \"end\": {\"x\": 32.976, \"y\": 13.342, \"z\": 28.511}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.452, \"y\": 14.791, \"z\": 20.062}, \"end\": {\"x\": 32.772, \"y\": 14.179, \"z\": 22.305}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.32683807501296, \"y\": 20.35260431802973, \"z\": 23.540226834555213}, \"end\": {\"x\": 32.18137603332113, \"y\": 20.741199970340038, \"z\": 25.16323827728038}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.222, \"y\": 15.911, \"z\": 24.692}, \"end\": {\"x\": 31.153, \"y\": 15.726, \"z\": 26.677}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.828, \"y\": 17.309, \"z\": 18.987}, \"end\": {\"x\": 33.099, \"y\": 17.588, \"z\": 21.094}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.378, \"y\": 16.845, \"z\": 23.648}, \"end\": {\"x\": 33.967, \"y\": 17.371, \"z\": 25.765}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.014739213966315, \"y\": 19.16640831016724, \"z\": 26.700836371012787}, \"end\": {\"x\": 34.31508507317345, \"y\": 19.48007487537707, \"z\": 28.20927142849176}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.327, \"y\": 14.938, \"z\": 22.093}, \"end\": {\"x\": 35.415, \"y\": 14.603, \"z\": 24.314}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.452, \"y\": 14.791, \"z\": 20.062}, \"end\": {\"x\": 32.772, \"y\": 14.179, \"z\": 22.305}});\n\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.828, \"y\": 17.309, \"z\": 18.987}, \"end\": {\"x\": 33.099, \"y\": 17.588, \"z\": 21.094}});\n\tviewer_16234219519367425.zoomTo();\nviewer_16234219519367425.render();\n});\n</script>",
            "text/html": [
              "<div id=\"3dmolviewer_16234219519367425\"  style=\"position: relative; width: 1200px; height: 800px\">\n",
              "        <p id=\"3dmolwarning_16234219519367425\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n",
              "        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n",
              "        </div>\n",
              "<script>\n",
              "\n",
              "var loadScriptAsync = function(uri){\n",
              "  return new Promise((resolve, reject) => {\n",
              "    var tag = document.createElement('script');\n",
              "    tag.src = uri;\n",
              "    tag.async = true;\n",
              "    tag.onload = () => {\n",
              "      resolve();\n",
              "    };\n",
              "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
              "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
              "});\n",
              "};\n",
              "\n",
              "if(typeof $3Dmolpromise === 'undefined') {\n",
              "$3Dmolpromise = null;\n",
              "  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n",
              "}\n",
              "\n",
              "var viewer_16234219519367425 = null;\n",
              "var warn = document.getElementById(\"3dmolwarning_16234219519367425\");\n",
              "if(warn) {\n",
              "    warn.parentNode.removeChild(warn);\n",
              "}\n",
              "$3Dmolpromise.then(function() {\n",
              "viewer_16234219519367425 = $3Dmol.createViewer($(\"#3dmolviewer_16234219519367425\"),{backgroundColor:\"white\"});\n",
              "viewer_16234219519367425.zoomTo();\n",
              "\tviewer_16234219519367425.addModels(\"HEADER                                            11-JUN-21   XXXX              \\nEXPDTA    THEORETICAL MODEL                                                     \\nREMARK 220                                                                      \\nREMARK 220 EXPERIMENTAL DETAILS                                                 \\nREMARK 220  EXPERIMENT TYPE                : THEORETICAL MODELLING              \\nREMARK 220  DATE OF DATA COLLECTION        : 11-JUN-21                          \\nREMARK 220                                                                      \\nREMARK 220 REMARK: MODEL GENERATED BY ROSETTA                                   \\nREMARK 220  VERSION 2020.50.post.dev+970.commits.3700df14560                    \\nATOM      1  N   ALA A   1      46.718  19.029  14.515  1.00  0.00           N  \\nATOM      2  CA  ALA A   1      45.711  17.981  14.626  1.00  0.00           C  \\nATOM      3  C   ALA A   1      46.287  16.744  15.312  1.00  0.00           C  \\nATOM      4  O   ALA A   1      45.575  16.034  16.023  1.00  0.00           O  \\nATOM      5  CB  ALA A   1      45.165  17.622  13.252  1.00  0.00           C  \\nATOM      6 1H   ALA A   1      46.320  19.828  14.064  1.00  0.00           H  \\nATOM      7 2H   ALA A   1      47.035  19.286  15.428  1.00  0.00           H  \\nATOM      8 3H   ALA A   1      47.491  18.693  13.977  1.00  0.00           H  \\nATOM      9  HA  ALA A   1      44.974  18.308  15.166  1.00  0.00           H  \\nATOM     10 1HB  ALA A   1      44.415  16.838  13.352  1.00  0.00           H  \\nATOM     11 2HB  ALA A   1      44.710  18.504  12.800  1.00  0.00           H  \\nATOM     12 3HB  ALA A   1      45.977  17.269  12.619  1.00  0.00           H  \\nATOM     13  N   ALA A   2      47.573  16.485  15.094  1.00  0.00           N  \\nATOM     14  CA  ALA A   2      48.260  15.426  15.824  1.00  0.00           C  \\nATOM     15  C   ALA A   2      48.210  15.740  17.310  1.00  0.00           C  \\nATOM     16  O   ALA A   2      48.033  14.850  18.140  1.00  0.00           O  \\nATOM     17  CB  ALA A   2      49.698  15.278  15.349  1.00  0.00           C  \\nATOM     18  H   ALA A   2      48.068  16.907  14.532  1.00  0.00           H  \\nATOM     19  HA  ALA A   2      47.807  14.582  15.672  1.00  0.00           H  \\nATOM     20 1HB  ALA A   2      50.193  14.496  15.925  1.00  0.00           H  \\nATOM     21 2HB  ALA A   2      49.707  15.011  14.293  1.00  0.00           H  \\nATOM     22 3HB  ALA A   2      50.226  16.220  15.490  1.00  0.00           H  \\nATOM     23  N   ALA A   3      48.363  17.021  17.631  1.00  0.00           N  \\nATOM     24  CA  ALA A   3      48.318  17.482  19.011  1.00  0.00           C  \\nATOM     25  C   ALA A   3      46.922  17.287  19.594  1.00  0.00           C  \\nATOM     26  O   ALA A   3      46.777  16.910  20.757  1.00  0.00           O  \\nATOM     27  CB  ALA A   3      48.735  18.943  19.099  1.00  0.00           C  \\nATOM     28  H   ALA A   3      48.495  17.650  17.059  1.00  0.00           H  \\nATOM     29  HA  ALA A   3      48.942  16.962  19.540  1.00  0.00           H  \\nATOM     30 1HB  ALA A   3      48.692  19.272  20.137  1.00  0.00           H  \\nATOM     31 2HB  ALA A   3      49.753  19.054  18.726  1.00  0.00           H  \\nATOM     32 3HB  ALA A   3      48.061  19.550  18.498  1.00  0.00           H  \\nATOM     33  N   ALA A   4      45.898  17.538  18.781  1.00  0.00           N  \\nATOM     34  CA  ALA A   4      44.513  17.367  19.215  1.00  0.00           C  \\nATOM     35  C   ALA A   4      44.255  15.920  19.614  1.00  0.00           C  \\nATOM     36  O   ALA A   4      43.737  15.652  20.696  1.00  0.00           O  \\nATOM     37  CB  ALA A   4      43.550  17.796  18.118  1.00  0.00           C  \\nATOM     38  H   ALA A   4      45.979  17.810  17.969  1.00  0.00           H  \\nATOM     39  HA  ALA A   4      44.352  17.927  19.990  1.00  0.00           H  \\nATOM     40 1HB  ALA A   4      42.524  17.654  18.458  1.00  0.00           H  \\nATOM     41 2HB  ALA A   4      43.712  18.848  17.883  1.00  0.00           H  \\nATOM     42 3HB  ALA A   4      43.723  17.195  17.226  1.00  0.00           H  \\nATOM     43  N   ALA A   5      44.611  14.995  18.729  1.00  0.00           N  \\nATOM     44  CA  ALA A   5      44.488  13.572  19.015  1.00  0.00           C  \\nATOM     45  C   ALA A   5      45.295  13.204  20.257  1.00  0.00           C  \\nATOM     46  O   ALA A   5      44.842  12.424  21.094  1.00  0.00           O  \\nATOM     47  CB  ALA A   5      44.941  12.747  17.819  1.00  0.00           C  \\nATOM     48  H   ALA A   5      44.928  15.169  17.949  1.00  0.00           H  \\nATOM     49  HA  ALA A   5      43.557  13.360  19.185  1.00  0.00           H  \\nATOM     50 1HB  ALA A   5      44.846  11.686  18.052  1.00  0.00           H  \\nATOM     51 2HB  ALA A   5      44.320  12.986  16.956  1.00  0.00           H  \\nATOM     52 3HB  ALA A   5      45.981  12.976  17.592  1.00  0.00           H  \\nATOM     53  N   ALA A   6      46.486  13.781  20.374  1.00  0.00           N  \\nATOM     54  CA  ALA A   6      47.366  13.501  21.502  1.00  0.00           C  \\nATOM     55  C   ALA A   6      46.771  14.033  22.801  1.00  0.00           C  \\nATOM     56  O   ALA A   6      46.837  13.370  23.836  1.00  0.00           O  \\nATOM     57  CB  ALA A   6      48.744  14.102  21.263  1.00  0.00           C  \\nATOM     58  H   ALA A   6      46.810  14.343  19.809  1.00  0.00           H  \\nATOM     59  HA  ALA A   6      47.470  12.541  21.591  1.00  0.00           H  \\nATOM     60 1HB  ALA A   6      49.388  13.885  22.116  1.00  0.00           H  \\nATOM     61 2HB  ALA A   6      49.178  13.670  20.361  1.00  0.00           H  \\nATOM     62 3HB  ALA A   6      48.654  15.180  21.142  1.00  0.00           H  \\nATOM     63  N   ALA A   7      46.190  15.228  22.743  1.00  0.00           N  \\nATOM     64  CA  ALA A   7      45.617  15.851  23.931  1.00  0.00           C  \\nATOM     65  C   ALA A   7      44.366  15.115  24.392  1.00  0.00           C  \\nATOM     66  O   ALA A   7      44.138  14.970  25.590  1.00  0.00           O  \\nATOM     67  CB  ALA A   7      45.299  17.314  23.661  1.00  0.00           C  \\nATOM     68  H   ALA A   7      46.114  15.698  22.027  1.00  0.00           H  \\nATOM     69  HA  ALA A   7      46.267  15.814  24.650  1.00  0.00           H  \\nATOM     70 1HB  ALA A   7      44.872  17.766  24.556  1.00  0.00           H  \\nATOM     71 2HB  ALA A   7      46.214  17.841  23.390  1.00  0.00           H  \\nATOM     72 3HB  ALA A   7      44.584  17.386  22.843  1.00  0.00           H  \\nATOM     73  N   ALA A   8      43.558  14.656  23.440  1.00  0.00           N  \\nATOM     74  CA  ALA A   8      42.375  13.868  23.763  1.00  0.00           C  \\nATOM     75  C   ALA A   8      42.787  12.599  24.490  1.00  0.00           C  \\nATOM     76  O   ALA A   8      42.157  12.197  25.469  1.00  0.00           O  \\nATOM     77  CB  ALA A   8      41.587  13.534  22.505  1.00  0.00           C  \\nATOM     78  H   ALA A   8      43.674  14.789  22.598  1.00  0.00           H  \\nATOM     79  HA  ALA A   8      41.798  14.382  24.350  1.00  0.00           H  \\nATOM     80 1HB  ALA A   8      40.716  12.934  22.770  1.00  0.00           H  \\nATOM     81 2HB  ALA A   8      41.259  14.457  22.025  1.00  0.00           H  \\nATOM     82 3HB  ALA A   8      42.218  12.973  21.818  1.00  0.00           H  \\nATOM     83  N   ALA A   9      43.856  11.977  24.006  1.00  0.00           N  \\nATOM     84  CA  ALA A   9      44.400  10.786  24.639  1.00  0.00           C  \\nATOM     85  C   ALA A   9      44.876  11.138  26.042  1.00  0.00           C  \\nATOM     86  O   ALA A   9      44.715  10.356  26.980  1.00  0.00           O  \\nATOM     87  CB  ALA A   9      45.535  10.202  23.811  1.00  0.00           C  \\nATOM     88  H   ALA A   9      44.287  12.228  23.305  1.00  0.00           H  \\nATOM     89  HA  ALA A   9      43.705  10.112  24.711  1.00  0.00           H  \\nATOM     90 1HB  ALA A   9      45.936   9.321  24.314  1.00  0.00           H  \\nATOM     91 2HB  ALA A   9      45.161   9.919  22.827  1.00  0.00           H  \\nATOM     92 3HB  ALA A   9      46.323  10.945  23.699  1.00  0.00           H  \\nATOM     93  N   ALA A  10      45.456  12.327  26.179  1.00  0.00           N  \\nATOM     94  CA  ALA A  10      45.928  12.800  27.473  1.00  0.00           C  \\nATOM     95  C   ALA A  10      44.723  13.115  28.353  1.00  0.00           C  \\nATOM     96  O   ALA A  10      44.672  12.731  29.523  1.00  0.00           O  \\nATOM     97  CB  ALA A  10      46.821  14.022  27.314  1.00  0.00           C  \\nATOM     98  H   ALA A  10      45.587  12.880  25.534  1.00  0.00           H  \\nATOM     99  HA  ALA A  10      46.445  12.100  27.902  1.00  0.00           H  \\nATOM    100 1HB  ALA A  10      47.145  14.366  28.296  1.00  0.00           H  \\nATOM    101 2HB  ALA A  10      47.693  13.760  26.715  1.00  0.00           H  \\nATOM    102 3HB  ALA A  10      46.265  14.816  26.817  1.00  0.00           H  \\nATOM    103  N   ALA A  11      43.746  13.798  27.767  1.00  0.00           N  \\nATOM    104  CA  ALA A  11      42.513  14.142  28.459  1.00  0.00           C  \\nATOM    105  C   ALA A  11      41.830  12.867  28.946  1.00  0.00           C  \\nATOM    106  O   ALA A  11      41.266  12.827  30.038  1.00  0.00           O  \\nATOM    107  CB  ALA A  11      41.588  14.939  27.551  1.00  0.00           C  \\nATOM    108  H   ALA A  11      43.775  14.077  26.954  1.00  0.00           H  \\nATOM    109  HA  ALA A  11      42.719  14.693  29.231  1.00  0.00           H  \\nATOM    110 1HB  ALA A  11      40.668  15.175  28.085  1.00  0.00           H  \\nATOM    111 2HB  ALA A  11      42.081  15.864  27.251  1.00  0.00           H  \\nATOM    112 3HB  ALA A  11      41.353  14.351  26.665  1.00  0.00           H  \\nATOM    113  N   ALA A  12      41.916  11.820  28.132  1.00  0.00           N  \\nATOM    114  CA  ALA A  12      41.329  10.527  28.460  1.00  0.00           C  \\nATOM    115  C   ALA A  12      42.018   9.891  29.669  1.00  0.00           C  \\nATOM    116  O   ALA A  12      41.355   9.346  30.550  1.00  0.00           O  \\nATOM    117  CB  ALA A  12      41.402   9.597  27.258  1.00  0.00           C  \\nATOM    118  H   ALA A  12      42.315  11.834  27.371  1.00  0.00           H  \\nATOM    119  HA  ALA A  12      40.393  10.650  28.679  1.00  0.00           H  \\nATOM    120 1HB  ALA A  12      40.956   8.636  27.514  1.00  0.00           H  \\nATOM    121 2HB  ALA A  12      40.858  10.038  26.423  1.00  0.00           H  \\nATOM    122 3HB  ALA A  12      42.443   9.449  26.975  1.00  0.00           H  \\nATOM    123  N   ALA A  13      43.346   9.967  29.708  1.00  0.00           N  \\nATOM    124  CA  ALA A  13      44.112   9.386  30.807  1.00  0.00           C  \\nATOM    125  C   ALA A  13      43.873  10.137  32.114  1.00  0.00           C  \\nATOM    126  O   ALA A  13      43.833   9.532  33.186  1.00  0.00           O  \\nATOM    127  CB  ALA A  13      45.596   9.376  30.468  1.00  0.00           C  \\nATOM    128  H   ALA A  13      43.828  10.351  29.108  1.00  0.00           H  \\nATOM    129  HA  ALA A  13      43.831   8.466  30.935  1.00  0.00           H  \\nATOM    130 1HB  ALA A  13      46.155   8.938  31.295  1.00  0.00           H  \\nATOM    131 2HB  ALA A  13      45.759   8.786  29.567  1.00  0.00           H  \\nATOM    132 3HB  ALA A  13      45.937  10.396  30.301  1.00  0.00           H  \\nATOM    133  N   ALA A  14      43.724  11.456  32.023  1.00  0.00           N  \\nATOM    134  CA  ALA A  14      43.447  12.281  33.195  1.00  0.00           C  \\nATOM    135  C   ALA A  14      42.111  11.897  33.818  1.00  0.00           C  \\nATOM    136  O   ALA A  14      41.958  11.886  35.037  1.00  0.00           O  \\nATOM    137  CB  ALA A  14      43.456  13.757  32.824  1.00  0.00           C  \\nATOM    138  H   ALA A  14      43.779  11.899  31.288  1.00  0.00           H  \\nATOM    139  HA  ALA A  14      44.142  12.139  33.857  1.00  0.00           H  \\nATOM    140 1HB  ALA A  14      43.242  14.356  33.709  1.00  0.00           H  \\nATOM    141 2HB  ALA A  14      44.436  14.027  32.431  1.00  0.00           H  \\nATOM    142 3HB  ALA A  14      42.697  13.946  32.067  1.00  0.00           H  \\nATOM    143  N   ALA A  15      41.147  11.585  32.961  1.00  0.00           N  \\nATOM    144  CA  ALA A  15      39.818  11.185  33.401  1.00  0.00           C  \\nATOM    145  C   ALA A  15      39.872   9.876  34.185  1.00  0.00           C  \\nATOM    146  O   ALA A  15      39.279   9.765  35.258  1.00  0.00           O  \\nATOM    147  CB  ALA A  15      38.883  11.052  32.208  1.00  0.00           C  \\nATOM    148  H   ALA A  15      41.240  11.598  32.106  1.00  0.00           H  \\nATOM    149  HA  ALA A  15      39.455  11.871  33.982  1.00  0.00           H  \\nATOM    150 1HB  ALA A  15      37.893  10.754  32.552  1.00  0.00           H  \\nATOM    151 2HB  ALA A  15      38.814  12.010  31.692  1.00  0.00           H  \\nATOM    152 3HB  ALA A  15      39.271  10.299  31.524  1.00  0.00           H  \\nATOM    153  N   ALA A  16      40.590   8.892  33.650  1.00  0.00           N  \\nATOM    154  CA  ALA A  16      40.716   7.594  34.307  1.00  0.00           C  \\nATOM    155  C   ALA A  16      41.311   7.758  35.696  1.00  0.00           C  \\nATOM    156  O   ALA A  16      40.893   7.094  36.646  1.00  0.00           O  \\nATOM    157  CB  ALA A  16      41.568   6.649  33.472  1.00  0.00           C  \\nATOM    158  H   ALA A  16      41.015   8.950  32.904  1.00  0.00           H  \\nATOM    159  HA  ALA A  16      39.835   7.198  34.400  1.00  0.00           H  \\nATOM    160 1HB  ALA A  16      41.657   5.691  33.983  1.00  0.00           H  \\nATOM    161 2HB  ALA A  16      41.099   6.500  32.499  1.00  0.00           H  \\nATOM    162 3HB  ALA A  16      42.559   7.080  33.334  1.00  0.00           H  \\nATOM    163  N   ALA A  17      42.288   8.649  35.808  1.00  0.00           N  \\nATOM    164  CA  ALA A  17      42.912   8.929  37.091  1.00  0.00           C  \\nATOM    165  C   ALA A  17      41.897   9.557  38.044  1.00  0.00           C  \\nATOM    166  O   ALA A  17      41.898   9.264  39.238  1.00  0.00           O  \\nATOM    167  CB  ALA A  17      44.117   9.840  36.911  1.00  0.00           C  \\nATOM    168  H   ALA A  17      42.608   9.106  35.154  1.00  0.00           H  \\nATOM    169  HA  ALA A  17      43.221   8.097  37.483  1.00  0.00           H  \\nATOM    170 1HB  ALA A  17      44.568  10.043  37.883  1.00  0.00           H  \\nATOM    171 2HB  ALA A  17      44.849   9.353  36.266  1.00  0.00           H  \\nATOM    172 3HB  ALA A  17      43.800  10.777  36.456  1.00  0.00           H  \\nATOM    173  N   ALA A  18      41.024  10.410  37.511  1.00  0.00           N  \\nATOM    174  CA  ALA A  18      39.999  11.059  38.327  1.00  0.00           C  \\nATOM    175  C   ALA A  18      39.078  10.027  38.968  1.00  0.00           C  \\nATOM    176  O   ALA A  18      38.849  10.057  40.177  1.00  0.00           O  \\nATOM    177  CB  ALA A  18      39.192  12.040  37.488  1.00  0.00           C  \\nATOM    178  H   ALA A  18      41.004  10.630  36.679  1.00  0.00           H  \\nATOM    179  HA  ALA A  18      40.431  11.559  39.037  1.00  0.00           H  \\nATOM    180 1HB  ALA A  18      38.432  12.511  38.111  1.00  0.00           H  \\nATOM    181 2HB  ALA A  18      39.856  12.804  37.084  1.00  0.00           H  \\nATOM    182 3HB  ALA A  18      38.712  11.508  36.669  1.00  0.00           H  \\nATOM    183  N   ALA A  19      38.551   9.119  38.152  1.00  0.00           N  \\nATOM    184  CA  ALA A  19      37.649   8.080  38.637  1.00  0.00           C  \\nATOM    185  C   ALA A  19      38.327   7.198  39.676  1.00  0.00           C  \\nATOM    186  O   ALA A  19      37.689   6.746  40.626  1.00  0.00           O  \\nATOM    187  CB  ALA A  19      37.143   7.235  37.477  1.00  0.00           C  \\nATOM    188  H   ALA A  19      38.702   9.083  37.306  1.00  0.00           H  \\nATOM    189  HA  ALA A  19      36.880   8.499  39.054  1.00  0.00           H  \\nATOM    190 1HB  ALA A  19      36.471   6.464  37.855  1.00  0.00           H  \\nATOM    191 2HB  ALA A  19      36.607   7.869  36.771  1.00  0.00           H  \\nATOM    192 3HB  ALA A  19      37.987   6.765  36.975  1.00  0.00           H  \\nATOM    193  N   ALA A  20      39.619   6.950  39.487  1.00  0.00           N  \\nATOM    194  CA  ALA A  20      40.389   6.158  40.438  1.00  0.00           C  \\nATOM    195  C   ALA A  20      40.457   6.878  41.780  1.00  0.00           C  \\nATOM    196  O   ALA A  20      40.299   6.264  42.835  1.00  0.00           O  \\nATOM    197  CB  ALA A  20      41.787   5.885  39.903  1.00  0.00           C  \\nATOM    198  H   ALA A  20      40.075   7.231  38.814  1.00  0.00           H  \\nATOM    199  HA  ALA A  20      39.948   5.305  40.573  1.00  0.00           H  \\nATOM    200 1HB  ALA A  20      42.348   5.301  40.633  1.00  0.00           H  \\nATOM    201 2HB  ALA A  20      41.717   5.328  38.969  1.00  0.00           H  \\nATOM    202 3HB  ALA A  20      42.299   6.829  39.725  1.00  0.00           H  \\nATOM    203  N   ALA A  21      40.691   8.186  41.729  1.00  0.00           N  \\nATOM    204  CA  ALA A  21      40.722   9.012  42.932  1.00  0.00           C  \\nATOM    205  C   ALA A  21      39.322   9.124  43.521  1.00  0.00           C  \\nATOM    206  O   ALA A  21      39.152   9.145  44.740  1.00  0.00           O  \\nATOM    207  CB  ALA A  21      41.286  10.391  42.625  1.00  0.00           C  \\nATOM    208  H   ALA A  21      40.837   8.623  41.003  1.00  0.00           H  \\nATOM    209  HA  ALA A  21      41.297   8.594  43.592  1.00  0.00           H  \\nATOM    210 1HB  ALA A  21      41.289  10.994  43.534  1.00  0.00           H  \\nATOM    211 2HB  ALA A  21      42.306  10.293  42.252  1.00  0.00           H  \\nATOM    212 3HB  ALA A  21      40.670  10.877  41.870  1.00  0.00           H  \\nATOM    213  N   ALA A  22      38.323   9.195  42.648  1.00  0.00           N  \\nATOM    214  CA  ALA A  22      36.938   9.321  43.084  1.00  0.00           C  \\nATOM    215  C   ALA A  22      36.511   8.076  43.857  1.00  0.00           C  \\nATOM    216  O   ALA A  22      35.848   8.173  44.890  1.00  0.00           O  \\nATOM    217  CB  ALA A  22      36.021   9.552  41.892  1.00  0.00           C  \\nATOM    218  H   ALA A  22      38.421   9.172  41.794  1.00  0.00           H  \\nATOM    219  HA  ALA A  22      36.859  10.086  43.675  1.00  0.00           H  \\nATOM    220 1HB  ALA A  22      34.990   9.638  42.237  1.00  0.00           H  \\nATOM    221 2HB  ALA A  22      36.311  10.472  41.383  1.00  0.00           H  \\nATOM    222 3HB  ALA A  22      36.104   8.715  41.202  1.00  0.00           H  \\nATOM    223  N   ALA A  23      36.902   6.909  43.352  1.00  0.00           N  \\nATOM    224  CA  ALA A  23      36.628   5.647  44.030  1.00  0.00           C  \\nATOM    225  C   ALA A  23      37.340   5.612  45.373  1.00  0.00           C  \\nATOM    226  O   ALA A  23      36.759   5.221  46.384  1.00  0.00           O  \\nATOM    227  CB  ALA A  23      37.054   4.468  43.168  1.00  0.00           C  \\nATOM    228  H   ALA A  23      37.331   6.822  42.612  1.00  0.00           H  \\nATOM    229  HA  ALA A  23      35.675   5.569  44.189  1.00  0.00           H  \\nATOM    230 1HB  ALA A  23      36.851   3.537  43.698  1.00  0.00           H  \\nATOM    231 2HB  ALA A  23      36.496   4.481  42.231  1.00  0.00           H  \\nATOM    232 3HB  ALA A  23      38.120   4.539  42.956  1.00  0.00           H  \\nATOM    233  N   ALA A  24      38.605   6.022  45.371  1.00  0.00           N  \\nATOM    234  CA  ALA A  24      39.387   6.108  46.596  1.00  0.00           C  \\nATOM    235  C   ALA A  24      38.691   7.053  47.566  1.00  0.00           C  \\nATOM    236  O   ALA A  24      38.549   6.755  48.751  1.00  0.00           O  \\nATOM    237  CB  ALA A  24      40.805   6.577  46.305  1.00  0.00           C  \\nATOM    238  H   ALA A  24      39.037   6.257  44.665  1.00  0.00           H  \\nATOM    239  HA  ALA A  24      39.442   5.231  47.008  1.00  0.00           H  \\nATOM    240 1HB  ALA A  24      41.363   6.650  47.238  1.00  0.00           H  \\nATOM    241 2HB  ALA A  24      41.296   5.863  45.643  1.00  0.00           H  \\nATOM    242 3HB  ALA A  24      40.773   7.554  45.825  1.00  0.00           H  \\nATOM    243  N   ALA A  25      38.241   8.188  47.041  1.00  0.00           N  \\nATOM    244  CA  ALA A  25      37.571   9.202  47.844  1.00  0.00           C  \\nATOM    245  C   ALA A  25      36.260   8.661  48.404  1.00  0.00           C  \\nATOM    246  O   ALA A  25      35.915   8.916  49.558  1.00  0.00           O  \\nATOM    247  CB  ALA A  25      37.322  10.457  47.020  1.00  0.00           C  \\nATOM    248  H   ALA A  25      38.314   8.397  46.210  1.00  0.00           H  \\nATOM    249  HA  ALA A  25      38.141   9.445  48.591  1.00  0.00           H  \\nATOM    250 1HB  ALA A  25      36.817  11.202  47.635  1.00  0.00           H  \\nATOM    251 2HB  ALA A  25      38.274  10.858  46.672  1.00  0.00           H  \\nATOM    252 3HB  ALA A  25      36.698  10.211  46.162  1.00  0.00           H  \\nATOM    253  N   ALA A  26      35.542   7.901  47.583  1.00  0.00           N  \\nATOM    254  CA  ALA A  26      34.261   7.331  47.988  1.00  0.00           C  \\nATOM    255  C   ALA A  26      34.442   6.235  49.032  1.00  0.00           C  \\nATOM    256  O   ALA A  26      33.568   6.018  49.871  1.00  0.00           O  \\nATOM    257  CB  ALA A  26      33.519   6.787  46.776  1.00  0.00           C  \\nATOM    258  H   ALA A  26      35.776   7.699  46.781  1.00  0.00           H  \\nATOM    259  HA  ALA A  26      33.715   8.030  48.380  1.00  0.00           H  \\nATOM    260 1HB  ALA A  26      32.564   6.367  47.093  1.00  0.00           H  \\nATOM    261 2HB  ALA A  26      33.342   7.594  46.065  1.00  0.00           H  \\nATOM    262 3HB  ALA A  26      34.117   6.010  46.302  1.00  0.00           H  \\nATOM    263  N   ALA A  27      35.574   5.540  48.974  1.00  0.00           N  \\nATOM    264  CA  ALA A  27      35.886   4.526  49.973  1.00  0.00           C  \\nATOM    265  C   ALA A  27      36.041   5.168  51.344  1.00  0.00           C  \\nATOM    266  O   ALA A  27      35.353   4.797  52.294  1.00  0.00           O  \\nATOM    267  CB  ALA A  27      37.149   3.767  49.592  1.00  0.00           C  \\nATOM    268  H   ALA A  27      36.176   5.637  48.368  1.00  0.00           H  \\nATOM    269  HA  ALA A  27      35.156   3.888  50.018  1.00  0.00           H  \\nATOM    270 1HB  ALA A  27      37.369   3.021  50.355  1.00  0.00           H  \\nATOM    271 2HB  ALA A  27      37.001   3.272  48.632  1.00  0.00           H  \\nATOM    272 3HB  ALA A  27      37.982   4.464  49.515  1.00  0.00           H  \\nATOM    273  N   ALA A  28      36.940   6.143  51.433  1.00  0.00           N  \\nATOM    274  CA  ALA A  28      37.251   6.792  52.700  1.00  0.00           C  \\nATOM    275  C   ALA A  28      36.052   7.536  53.281  1.00  0.00           C  \\nATOM    276  O   ALA A  28      36.018   7.830  54.475  1.00  0.00           O  \\nATOM    277  CB  ALA A  28      38.422   7.747  52.521  1.00  0.00           C  \\nATOM    278  H   ALA A  28      37.389   6.449  50.766  1.00  0.00           H  \\nATOM    279  HA  ALA A  28      37.529   6.113  53.335  1.00  0.00           H  \\nATOM    280 1HB  ALA A  28      38.656   8.219  53.476  1.00  0.00           H  \\nATOM    281 2HB  ALA A  28      39.293   7.193  52.168  1.00  0.00           H  \\nATOM    282 3HB  ALA A  28      38.160   8.512  51.793  1.00  0.00           H  \\nATOM    283  N   ALA A  29      35.071   7.838  52.438  1.00  0.00           N  \\nATOM    284  CA  ALA A  29      33.891   8.566  52.885  1.00  0.00           C  \\nATOM    285  C   ALA A  29      33.050   7.696  53.813  1.00  0.00           C  \\nATOM    286  O   ALA A  29      32.438   8.198  54.755  1.00  0.00           O  \\nATOM    287  CB  ALA A  29      33.066   9.030  51.693  1.00  0.00           C  \\nATOM    288  H   ALA A  29      35.064   7.632  51.603  1.00  0.00           H  \\nATOM    289  HA  ALA A  29      34.173   9.351  53.381  1.00  0.00           H  \\nATOM    290 1HB  ALA A  29      32.186   9.567  52.047  1.00  0.00           H  \\nATOM    291 2HB  ALA A  29      33.668   9.691  51.070  1.00  0.00           H  \\nATOM    292 3HB  ALA A  29      32.753   8.166  51.109  1.00  0.00           H  \\nATOM    293  N   ALA A  30      33.025   6.393  53.542  1.00  0.00           N  \\nATOM    294  CA  ALA A  30      32.366   5.436  54.426  1.00  0.00           C  \\nATOM    295  C   ALA A  30      33.394   4.808  55.370  1.00  0.00           C  \\nATOM    296  O   ALA A  30      33.299   3.633  55.724  1.00  0.00           O  \\nATOM    297  CB  ALA A  30      31.649   4.363  53.619  1.00  0.00           C  \\nATOM    298  H   ALA A  30      33.386   6.036  52.847  1.00  0.00           H  \\nATOM    299  HA  ALA A  30      31.708   5.902  54.964  1.00  0.00           H  \\nATOM    300 1HB  ALA A  30      31.173   3.655  54.297  1.00  0.00           H  \\nATOM    301 2HB  ALA A  30      30.891   4.828  52.988  1.00  0.00           H  \\nATOM    302 3HB  ALA A  30      32.369   3.838  52.993  1.00  0.00           H  \\nATOM    303  N   ALA A  31      34.385   5.606  55.758  1.00  0.00           N  \\nATOM    304  CA  ALA A  31      35.375   5.203  56.754  1.00  0.00           C  \\nATOM    305  C   ALA A  31      35.523   6.285  57.819  1.00  0.00           C  \\nATOM    306  O   ALA A  31      36.275   6.121  58.780  1.00  0.00           O  \\nATOM    307  CB  ALA A  31      36.716   4.916  56.093  1.00  0.00           C  \\nATOM    308  H   ALA A  31      34.508   6.401  55.453  1.00  0.00           H  \\nATOM    309  HA  ALA A  31      35.073   4.391  57.189  1.00  0.00           H  \\nATOM    310 1HB  ALA A  31      37.441   4.626  56.853  1.00  0.00           H  \\nATOM    311 2HB  ALA A  31      36.600   4.106  55.373  1.00  0.00           H  \\nATOM    312 3HB  ALA A  31      37.066   5.810  55.580  1.00  0.00           H  \\nATOM    313  N   ALA A  32      34.816   7.395  57.627  1.00  0.00           N  \\nATOM    314  CA  ALA A  32      34.634   8.394  58.669  1.00  0.00           C  \\nATOM    315  C   ALA A  32      33.207   8.290  59.192  1.00  0.00           C  \\nATOM    316  O   ALA A  32      32.893   8.793  60.267  1.00  0.00           O  \\nATOM    317  CB  ALA A  32      34.924   9.793  58.144  1.00  0.00           C  \\nATOM    318  H   ALA A  32      34.426   7.593  56.886  1.00  0.00           H  \\nATOM    319  HA  ALA A  32      35.245   8.215  59.401  1.00  0.00           H  \\nATOM    320 1HB  ALA A  32      34.759  10.521  58.939  1.00  0.00           H  \\nATOM    321 2HB  ALA A  32      35.960   9.849  57.811  1.00  0.00           H  \\nATOM    322 3HB  ALA A  32      34.262  10.012  57.308  1.00  0.00           H  \\nATOM    323  N   ALA A  33      32.345   7.639  58.415  1.00  0.00           N  \\nATOM    324  CA  ALA A  33      30.984   7.360  58.849  1.00  0.00           C  \\nATOM    325  C   ALA A  33      31.002   6.279  59.929  1.00  0.00           C  \\nATOM    326  O   ALA A  33      30.091   6.197  60.753  1.00  0.00           O  \\nATOM    327  CB  ALA A  33      30.120   6.934  57.671  1.00  0.00           C  \\nATOM    328  H   ALA A  33      32.527   7.348  57.627  1.00  0.00           H  \\nATOM    329  HA  ALA A  33      30.598   8.164  59.230  1.00  0.00           H  \\nATOM    330 1HB  ALA A  33      29.109   6.721  58.020  1.00  0.00           H  \\nATOM    331 2HB  ALA A  33      30.087   7.738  56.935  1.00  0.00           H  \\nATOM    332 3HB  ALA A  33      30.542   6.041  57.214  1.00  0.00           H  \\nATOM    333  N   ALA A  34      32.050   5.457  59.915  1.00  0.00           N  \\nATOM    334  CA  ALA A  34      32.263   4.437  60.937  1.00  0.00           C  \\nATOM    335  C   ALA A  34      32.992   5.028  62.141  1.00  0.00           C  \\nATOM    336  O   ALA A  34      32.755   4.629  63.282  1.00  0.00           O  \\nATOM    337  CB  ALA A  34      33.043   3.263  60.365  1.00  0.00           C  \\nATOM    338  H   ALA A  34      32.663   5.473  59.312  1.00  0.00           H  \\nATOM    339  HA  ALA A  34      31.404   4.099  61.237  1.00  0.00           H  \\nATOM    340 1HB  ALA A  34      33.197   2.514  61.142  1.00  0.00           H  \\nATOM    341 2HB  ALA A  34      32.483   2.821  59.541  1.00  0.00           H  \\nATOM    342 3HB  ALA A  34      34.009   3.611  60.001  1.00  0.00           H  \\nATOM    343  N   ALA A  35      33.881   5.979  61.868  1.00  0.00           N  \\nATOM    344  CA  ALA A  35      34.660   6.654  62.902  1.00  0.00           C  \\nATOM    345  C   ALA A  35      34.615   8.169  62.697  1.00  0.00           C  \\nATOM    346  O   ALA A  35      35.586   8.761  62.224  1.00  0.00           O  \\nATOM    347  CB  ALA A  35      36.098   6.155  62.899  1.00  0.00           C  \\nATOM    348  H   ALA A  35      34.055   6.257  61.073  1.00  0.00           H  \\nATOM    349  HA  ALA A  35      34.278   6.455  63.771  1.00  0.00           H  \\nATOM    350 1HB  ALA A  35      36.664   6.674  63.673  1.00  0.00           H  \\nATOM    351 2HB  ALA A  35      36.111   5.083  63.097  1.00  0.00           H  \\nATOM    352 3HB  ALA A  35      36.548   6.351  61.927  1.00  0.00           H  \\nATOM    353  N   ALA A  36      33.480   8.802  63.034  1.00  0.00           N  \\nATOM    354  CA  ALA A  36      33.349  10.257  62.895  1.00  0.00           C  \\nATOM    355  C   ALA A  36      34.242  11.024  63.870  1.00  0.00           C  \\nATOM    356  O   ALA A  36      33.740  11.731  64.744  1.00  0.00           O  \\nATOM    357  CB  ALA A  36      31.897  10.670  63.088  1.00  0.00           C  \\nATOM    358  H   ALA A  36      32.699   8.270  63.390  1.00  0.00           H  \\nATOM    359  HA  ALA A  36      33.552  10.530  61.987  1.00  0.00           H  \\nATOM    360 1HB  ALA A  36      31.808  11.751  62.973  1.00  0.00           H  \\nATOM    361 2HB  ALA A  36      31.276  10.174  62.342  1.00  0.00           H  \\nATOM    362 3HB  ALA A  36      31.567  10.383  64.085  1.00  0.00           H  \\nATOM    363  N   ALA A  37      35.554  10.871  63.721  1.00  0.00           N  \\nATOM    364  CA  ALA A  37      36.512  11.566  64.574  1.00  0.00           C  \\nATOM    365  C   ALA A  37      37.551  12.308  63.743  1.00  0.00           C  \\nATOM    366  O   ALA A  37      37.563  12.224  62.515  1.00  0.00           O  \\nATOM    367  CB  ALA A  37      37.192  10.584  65.517  1.00  0.00           C  \\nATOM    368  H   ALA A  37      35.918  10.367  63.127  1.00  0.00           H  \\nATOM    369  HA  ALA A  37      36.038  12.219  65.113  1.00  0.00           H  \\nATOM    370 1HB  ALA A  37      37.904  11.118  66.146  1.00  0.00           H  \\nATOM    371 2HB  ALA A  37      36.442  10.104  66.145  1.00  0.00           H  \\nATOM    372 3HB  ALA A  37      37.717   9.827  64.937  1.00  0.00           H  \\nATOM    373  N   ALA A  38      38.428  13.027  64.432  1.00  0.00           N  \\nATOM    374  CA  ALA A  38      39.416  13.881  63.788  1.00  0.00           C  \\nATOM    375  C   ALA A  38      40.503  13.079  63.080  1.00  0.00           C  \\nATOM    376  O   ALA A  38      40.668  11.885  63.324  1.00  0.00           O  \\nATOM    377  CB  ALA A  38      40.040  14.818  64.812  1.00  0.00           C  \\nATOM    378  H   ALA A  38      38.471  13.036  65.291  1.00  0.00           H  \\nATOM    379  HA  ALA A  38      38.971  14.432  63.125  1.00  0.00           H  \\nATOM    380 1HB  ALA A  38      40.770  15.461  64.320  1.00  0.00           H  \\nATOM    381 2HB  ALA A  38      39.262  15.432  65.265  1.00  0.00           H  \\nATOM    382 3HB  ALA A  38      40.536  14.232  65.585  1.00  0.00           H  \\nATOM    383  N   ALA A  39      41.166  13.758  62.155  1.00  0.00           N  \\nATOM    384  CA  ALA A  39      42.319  13.244  61.409  1.00  0.00           C  \\nATOM    385  C   ALA A  39      42.016  12.100  60.463  1.00  0.00           C  \\nATOM    386  O   ALA A  39      42.807  11.802  59.618  1.00  0.00           O  \\nATOM    387  CB  ALA A  39      43.402  12.808  62.385  1.00  0.00           C  \\nATOM    388  H   ALA A  39      40.955  14.558  61.929  1.00  0.00           H  \\nATOM    389  HA  ALA A  39      42.657  13.974  60.852  1.00  0.00           H  \\nATOM    390 1HB  ALA A  39      44.269  12.450  61.830  1.00  0.00           H  \\nATOM    391 2HB  ALA A  39      43.693  13.655  63.007  1.00  0.00           H  \\nATOM    392 3HB  ALA A  39      43.021  12.008  63.018  1.00  0.00           H  \\nATOM    393  N   ALA A  40      40.882  11.456  60.610  1.00  0.00           N  \\nATOM    394  CA  ALA A  40      40.291  10.669  59.535  1.00  0.00           C  \\nATOM    395  C   ALA A  40      39.620  11.649  58.582  1.00  0.00           C  \\nATOM    396  O   ALA A  40      39.622  11.462  57.366  1.00  0.00           O  \\nATOM    397  CB  ALA A  40      39.298   9.647  60.070  1.00  0.00           C  \\nATOM    398  H   ALA A  40      40.461  11.408  61.359  1.00  0.00           H  \\nATOM    399  HA  ALA A  40      40.989  10.196  59.054  1.00  0.00           H  \\nATOM    400 1HB  ALA A  40      38.851   9.104  59.238  1.00  0.00           H  \\nATOM    401 2HB  ALA A  40      39.815   8.947  60.726  1.00  0.00           H  \\nATOM    402 3HB  ALA A  40      38.517  10.159  60.630  1.00  0.00           H  \\nATOM    403  N   ALA A  41      39.041  12.696  59.162  1.00  0.00           N  \\nATOM    404  CA  ALA A  41      38.532  13.831  58.404  1.00  0.00           C  \\nATOM    405  C   ALA A  41      39.668  14.464  57.609  1.00  0.00           C  \\nATOM    406  O   ALA A  41      39.486  14.863  56.459  1.00  0.00           O  \\nATOM    407  CB  ALA A  41      37.885  14.852  59.328  1.00  0.00           C  \\nATOM    408  H   ALA A  41      38.929  12.772  60.011  1.00  0.00           H  \\nATOM    409  HA  ALA A  41      37.858  13.522  57.779  1.00  0.00           H  \\nATOM    410 1HB  ALA A  41      37.524  15.697  58.741  1.00  0.00           H  \\nATOM    411 2HB  ALA A  41      37.048  14.390  59.852  1.00  0.00           H  \\nATOM    412 3HB  ALA A  41      38.618  15.202  60.053  1.00  0.00           H  \\nATOM    413  N   ALA A  42      40.845  14.537  58.224  1.00  0.00           N  \\nATOM    414  CA  ALA A  42      42.022  15.105  57.573  1.00  0.00           C  \\nATOM    415  C   ALA A  42      42.475  14.215  56.418  1.00  0.00           C  \\nATOM    416  O   ALA A  42      42.906  14.714  55.380  1.00  0.00           O  \\nATOM    417  CB  ALA A  42      43.150  15.292  58.577  1.00  0.00           C  \\nATOM    418  H   ALA A  42      40.989  14.262  59.026  1.00  0.00           H  \\nATOM    419  HA  ALA A  42      41.794  15.975  57.211  1.00  0.00           H  \\nATOM    420 1HB  ALA A  42      44.021  15.711  58.072  1.00  0.00           H  \\nATOM    421 2HB  ALA A  42      42.826  15.971  59.366  1.00  0.00           H  \\nATOM    422 3HB  ALA A  42      43.412  14.330  59.013  1.00  0.00           H  \\nATOM    423  N   ALA A  43      42.373  12.901  56.603  1.00  0.00           N  \\nATOM    424  CA  ALA A  43      42.744  11.950  55.557  1.00  0.00           C  \\nATOM    425  C   ALA A  43      41.899  12.158  54.307  1.00  0.00           C  \\nATOM    426  O   ALA A  43      42.387  12.017  53.190  1.00  0.00           O  \\nATOM    427  CB  ALA A  43      42.601  10.521  56.061  1.00  0.00           C  \\nATOM    428  H   ALA A  43      42.091  12.533  57.327  1.00  0.00           H  \\nATOM    429  HA  ALA A  43      43.674  12.088  55.319  1.00  0.00           H  \\nATOM    430 1HB  ALA A  43      42.878   9.826  55.268  1.00  0.00           H  \\nATOM    431 2HB  ALA A  43      43.255  10.371  56.919  1.00  0.00           H  \\nATOM    432 3HB  ALA A  43      41.568  10.342  56.356  1.00  0.00           H  \\nATOM    433  N   ALA A  44      40.629  12.490  54.507  1.00  0.00           N  \\nATOM    434  CA  ALA A  44      39.711  12.720  53.398  1.00  0.00           C  \\nATOM    435  C   ALA A  44      40.003  14.050  52.719  1.00  0.00           C  \\nATOM    436  O   ALA A  44      40.039  14.130  51.491  1.00  0.00           O  \\nATOM    437  CB  ALA A  44      38.269  12.677  53.882  1.00  0.00           C  \\nATOM    438  H   ALA A  44      40.271  12.589  55.283  1.00  0.00           H  \\nATOM    439  HA  ALA A  44      39.825  12.015  52.741  1.00  0.00           H  \\nATOM    440 1HB  ALA A  44      37.598  12.854  53.042  1.00  0.00           H  \\nATOM    441 2HB  ALA A  44      38.059  11.699  54.315  1.00  0.00           H  \\nATOM    442 3HB  ALA A  44      38.116  13.447  54.637  1.00  0.00           H  \\nATOM    443  N   ALA A  45      40.211  15.089  53.522  1.00  0.00           N  \\nATOM    444  CA  ALA A  45      40.538  16.406  52.990  1.00  0.00           C  \\nATOM    445  C   ALA A  45      41.757  16.326  52.082  1.00  0.00           C  \\nATOM    446  O   ALA A  45      41.810  16.994  51.054  1.00  0.00           O  \\nATOM    447  CB  ALA A  45      40.780  17.395  54.121  1.00  0.00           C  \\nATOM    448  H   ALA A  45      40.169  15.057  54.381  1.00  0.00           H  \\nATOM    449  HA  ALA A  45      39.790  16.729  52.463  1.00  0.00           H  \\nATOM    450 1HB  ALA A  45      41.029  18.371  53.704  1.00  0.00           H  \\nATOM    451 2HB  ALA A  45      39.879  17.480  54.729  1.00  0.00           H  \\nATOM    452 3HB  ALA A  45      41.604  17.045  54.740  1.00  0.00           H  \\nATOM    453  N   ALA A  46      42.727  15.497  52.458  1.00  0.00           N  \\nATOM    454  CA  ALA A  46      43.905  15.287  51.625  1.00  0.00           C  \\nATOM    455  C   ALA A  46      43.507  14.737  50.260  1.00  0.00           C  \\nATOM    456  O   ALA A  46      44.031  15.165  49.233  1.00  0.00           O  \\nATOM    457  CB  ALA A  46      44.884  14.347  52.313  1.00  0.00           C  \\nATOM    458  H   ALA A  46      42.726  15.045  53.189  1.00  0.00           H  \\nATOM    459  HA  ALA A  46      44.353  16.137  51.489  1.00  0.00           H  \\nATOM    460 1HB  ALA A  46      45.755  14.199  51.675  1.00  0.00           H  \\nATOM    461 2HB  ALA A  46      45.199  14.781  53.262  1.00  0.00           H  \\nATOM    462 3HB  ALA A  46      44.401  13.389  52.496  1.00  0.00           H  \\nATOM    463  N   ALA A  47      42.575  13.789  50.257  1.00  0.00           N  \\nATOM    464  CA  ALA A  47      42.166  13.124  49.026  1.00  0.00           C  \\nATOM    465  C   ALA A  47      41.228  14.016  48.218  1.00  0.00           C  \\nATOM    466  O   ALA A  47      41.170  13.919  46.992  1.00  0.00           O  \\nATOM    467  CB  ALA A  47      41.499  11.792  49.337  1.00  0.00           C  \\nATOM    468  H   ALA A  47      42.163  13.512  50.959  1.00  0.00           H  \\nATOM    469  HA  ALA A  47      42.952  12.948  48.486  1.00  0.00           H  \\nATOM    470 1HB  ALA A  47      41.197  11.310  48.406  1.00  0.00           H  \\nATOM    471 2HB  ALA A  47      42.201  11.148  49.867  1.00  0.00           H  \\nATOM    472 3HB  ALA A  47      40.622  11.961  49.959  1.00  0.00           H  \\nATOM    473  N   ALA A  48      40.498  14.887  48.908  1.00  0.00           N  \\nATOM    474  CA  ALA A  48      39.616  15.837  48.240  1.00  0.00           C  \\nATOM    475  C   ALA A  48      40.446  16.839  47.437  1.00  0.00           C  \\nATOM    476  O   ALA A  48      40.168  17.084  46.262  1.00  0.00           O  \\nATOM    477  CB  ALA A  48      38.734  16.555  49.251  1.00  0.00           C  \\nATOM    478  H   ALA A  48      40.496  14.947  49.766  1.00  0.00           H  \\nATOM    479  HA  ALA A  48      39.044  15.357  47.621  1.00  0.00           H  \\nATOM    480 1HB  ALA A  48      38.091  17.266  48.733  1.00  0.00           H  \\nATOM    481 2HB  ALA A  48      38.119  15.826  49.779  1.00  0.00           H  \\nATOM    482 3HB  ALA A  48      39.360  17.087  49.966  1.00  0.00           H  \\nATOM    483  N   ALA A  49      41.475  17.402  48.063  1.00  0.00           N  \\nATOM    484  CA  ALA A  49      42.346  18.354  47.381  1.00  0.00           C  \\nATOM    485  C   ALA A  49      43.059  17.662  46.227  1.00  0.00           C  \\nATOM    486  O   ALA A  49      43.298  18.263  45.182  1.00  0.00           O  \\nATOM    487  CB  ALA A  49      43.352  18.957  48.350  1.00  0.00           C  \\nATOM    488  H   ALA A  49      41.690  17.251  48.882  1.00  0.00           H  \\nATOM    489  HA  ALA A  49      41.807  19.073  47.016  1.00  0.00           H  \\nATOM    490 1HB  ALA A  49      43.999  19.654  47.816  1.00  0.00           H  \\nATOM    491 2HB  ALA A  49      42.822  19.488  49.141  1.00  0.00           H  \\nATOM    492 3HB  ALA A  49      43.956  18.164  48.788  1.00  0.00           H  \\nATOM    493  N   ALA A  50      43.396  16.393  46.427  1.00  0.00           N  \\nATOM    494  CA  ALA A  50      44.003  15.586  45.379  1.00  0.00           C  \\nATOM    495  C   ALA A  50      43.038  15.459  44.211  1.00  0.00           C  \\nATOM    496  O   ALA A  50      43.424  15.625  43.055  1.00  0.00           O  \\nATOM    497  CB  ALA A  50      44.392  14.213  45.909  1.00  0.00           C  \\nATOM    498  H   ALA A  50      43.281  15.974  47.169  1.00  0.00           H  \\nATOM    499  HA  ALA A  50      44.809  16.024  45.063  1.00  0.00           H  \\nATOM    500 1HB  ALA A  50      44.830  13.624  45.104  1.00  0.00           H  \\nATOM    501 2HB  ALA A  50      45.118  14.326  46.714  1.00  0.00           H  \\nATOM    502 3HB  ALA A  50      43.506  13.706  46.289  1.00  0.00           H  \\nATOM    503  N   ALA A  51      41.778  15.167  44.517  1.00  0.00           N  \\nATOM    504  CA  ALA A  51      40.758  15.040  43.485  1.00  0.00           C  \\nATOM    505  C   ALA A  51      40.578  16.367  42.750  1.00  0.00           C  \\nATOM    506  O   ALA A  51      40.489  16.403  41.523  1.00  0.00           O  \\nATOM    507  CB  ALA A  51      39.440  14.579  44.091  1.00  0.00           C  \\nATOM    508  H   ALA A  51      41.488  15.039  45.317  1.00  0.00           H  \\nATOM    509  HA  ALA A  51      41.041  14.374  42.839  1.00  0.00           H  \\nATOM    510 1HB  ALA A  51      38.689  14.495  43.306  1.00  0.00           H  \\nATOM    511 2HB  ALA A  51      39.578  13.609  44.567  1.00  0.00           H  \\nATOM    512 3HB  ALA A  51      39.109  15.303  44.833  1.00  0.00           H  \\nATOM    513  N   ALA A  52      40.536  17.458  43.510  1.00  0.00           N  \\nATOM    514  CA  ALA A  52      40.361  18.784  42.933  1.00  0.00           C  \\nATOM    515  C   ALA A  52      41.565  19.158  42.079  1.00  0.00           C  \\nATOM    516  O   ALA A  52      41.408  19.694  40.985  1.00  0.00           O  \\nATOM    517  CB  ALA A  52      40.141  19.818  44.027  1.00  0.00           C  \\nATOM    518  H   ALA A  52      40.606  17.455  44.367  1.00  0.00           H  \\nATOM    519  HA  ALA A  52      39.579  18.777  42.359  1.00  0.00           H  \\nATOM    520 1HB  ALA A  52      40.018  20.803  43.578  1.00  0.00           H  \\nATOM    521 2HB  ALA A  52      39.246  19.563  44.595  1.00  0.00           H  \\nATOM    522 3HB  ALA A  52      41.002  19.830  44.695  1.00  0.00           H  \\nATOM    523  N   ALA A  53      42.764  18.868  42.573  1.00  0.00           N  \\nATOM    524  CA  ALA A  53      43.977  19.138  41.813  1.00  0.00           C  \\nATOM    525  C   ALA A  53      43.941  18.386  40.489  1.00  0.00           C  \\nATOM    526  O   ALA A  53      44.317  18.925  39.449  1.00  0.00           O  \\nATOM    527  CB  ALA A  53      45.211  18.753  42.615  1.00  0.00           C  \\nATOM    528  H   ALA A  53      42.901  18.516  43.346  1.00  0.00           H  \\nATOM    529  HA  ALA A  53      44.027  20.087  41.620  1.00  0.00           H  \\nATOM    530 1HB  ALA A  53      46.106  18.954  42.026  1.00  0.00           H  \\nATOM    531 2HB  ALA A  53      45.244  19.337  43.535  1.00  0.00           H  \\nATOM    532 3HB  ALA A  53      45.169  17.693  42.861  1.00  0.00           H  \\nATOM    533  N   ALA A  54      43.474  17.143  40.533  1.00  0.00           N  \\nATOM    534  CA  ALA A  54      43.344  16.329  39.331  1.00  0.00           C  \\nATOM    535  C   ALA A  54      42.245  16.881  38.434  1.00  0.00           C  \\nATOM    536  O   ALA A  54      42.347  16.834  37.208  1.00  0.00           O  \\nATOM    537  CB  ALA A  54      43.059  14.879  39.694  1.00  0.00           C  \\nATOM    538  H   ALA A  54      43.223  16.745  41.253  1.00  0.00           H  \\nATOM    539  HA  ALA A  54      44.179  16.356  38.837  1.00  0.00           H  \\nATOM    540 1HB  ALA A  54      42.958  14.289  38.783  1.00  0.00           H  \\nATOM    541 2HB  ALA A  54      43.881  14.485  40.292  1.00  0.00           H  \\nATOM    542 3HB  ALA A  54      42.135  14.822  40.267  1.00  0.00           H  \\nATOM    543  N   ALA A  55      41.194  17.412  39.052  1.00  0.00           N  \\nATOM    544  CA  ALA A  55      40.093  18.014  38.309  1.00  0.00           C  \\nATOM    545  C   ALA A  55      40.558  19.258  37.554  1.00  0.00           C  \\nATOM    546  O   ALA A  55      40.268  19.415  36.368  1.00  0.00           O  \\nATOM    547  CB  ALA A  55      38.946  18.359  39.248  1.00  0.00           C  \\nATOM    548  H   ALA A  55      41.095  17.435  39.906  1.00  0.00           H  \\nATOM    549  HA  ALA A  55      39.762  17.374  37.660  1.00  0.00           H  \\nATOM    550 1HB  ALA A  55      38.132  18.808  38.679  1.00  0.00           H  \\nATOM    551 2HB  ALA A  55      38.591  17.453  39.737  1.00  0.00           H  \\nATOM    552 3HB  ALA A  55      39.294  19.065  40.001  1.00  0.00           H  \\nATOM    553  N   ALA A  56      41.281  20.137  38.241  1.00  0.00           N  \\nATOM    554  CA  ALA A  56      41.777  21.367  37.626  1.00  0.00           C  \\nATOM    555  C   ALA A  56      42.772  21.056  36.510  1.00  0.00           C  \\nATOM    556  O   ALA A  56      42.778  21.714  35.470  1.00  0.00           O  \\nATOM    557  CB  ALA A  56      42.419  22.264  38.674  1.00  0.00           C  \\nATOM    558  H   ALA A  56      41.500  20.045  39.068  1.00  0.00           H  \\nATOM    559  HA  ALA A  56      41.030  21.848  37.236  1.00  0.00           H  \\nATOM    560 1HB  ALA A  56      42.785  23.175  38.199  1.00  0.00           H  \\nATOM    561 2HB  ALA A  56      41.680  22.523  39.433  1.00  0.00           H  \\nATOM    562 3HB  ALA A  56      43.251  21.740  39.142  1.00  0.00           H  \\nATOM    563  N   ALA A  57      43.614  20.052  36.732  1.00  0.00           N  \\nATOM    564  CA  ALA A  57      44.557  19.618  35.709  1.00  0.00           C  \\nATOM    565  C   ALA A  57      43.803  19.118  34.482  1.00  0.00           C  \\nATOM    566  O   ALA A  57      44.186  19.404  33.348  1.00  0.00           O  \\nATOM    567  CB  ALA A  57      45.478  18.535  36.252  1.00  0.00           C  \\nATOM    568  H   ALA A  57      43.660  19.607  37.466  1.00  0.00           H  \\nATOM    569  HA  ALA A  57      45.106  20.372  35.441  1.00  0.00           H  \\nATOM    570 1HB  ALA A  57      46.168  18.218  35.469  1.00  0.00           H  \\nATOM    571 2HB  ALA A  57      46.043  18.927  37.097  1.00  0.00           H  \\nATOM    572 3HB  ALA A  57      44.884  17.682  36.577  1.00  0.00           H  \\nATOM    573  N   ALA A  58      42.722  18.378  34.712  1.00  0.00           N  \\nATOM    574  CA  ALA A  58      41.902  17.876  33.614  1.00  0.00           C  \\nATOM    575  C   ALA A  58      41.286  19.027  32.826  1.00  0.00           C  \\nATOM    576  O   ALA A  58      41.209  18.977  31.600  1.00  0.00           O  \\nATOM    577  CB  ALA A  58      40.814  16.952  34.141  1.00  0.00           C  \\nATOM    578  H   ALA A  58      42.442  18.152  35.494  1.00  0.00           H  \\nATOM    579  HA  ALA A  58      42.463  17.365  33.010  1.00  0.00           H  \\nATOM    580 1HB  ALA A  58      40.211  16.589  33.310  1.00  0.00           H  \\nATOM    581 2HB  ALA A  58      41.273  16.105  34.654  1.00  0.00           H  \\nATOM    582 3HB  ALA A  58      40.180  17.497  34.839  1.00  0.00           H  \\nATOM    583  N   ALA A  59      40.859  20.067  33.535  1.00  0.00           N  \\nATOM    584  CA  ALA A  59      40.285  21.249  32.896  1.00  0.00           C  \\nATOM    585  C   ALA A  59      41.344  22.073  32.166  1.00  0.00           C  \\nATOM    586  O   ALA A  59      41.051  22.717  31.157  1.00  0.00           O  \\nATOM    587  CB  ALA A  59      39.571  22.109  33.929  1.00  0.00           C  \\nATOM    588  H   ALA A  59      40.890  20.114  34.393  1.00  0.00           H  \\nATOM    589  HA  ALA A  59      39.627  20.963  32.243  1.00  0.00           H  \\nATOM    590 1HB  ALA A  59      39.142  22.983  33.440  1.00  0.00           H  \\nATOM    591 2HB  ALA A  59      38.776  21.529  34.398  1.00  0.00           H  \\nATOM    592 3HB  ALA A  59      40.282  22.430  34.688  1.00  0.00           H  \\nATOM    593  N   ALA A  60      42.572  22.059  32.676  1.00  0.00           N  \\nATOM    594  CA  ALA A  60      43.666  22.768  32.019  1.00  0.00           C  \\nATOM    595  C   ALA A  60      43.936  22.123  30.666  1.00  0.00           C  \\nATOM    596  O   ALA A  60      43.986  22.801  29.640  1.00  0.00           O  \\nATOM    597  CB  ALA A  60      44.919  22.759  32.883  1.00  0.00           C  \\nATOM    598  H   ALA A  60      42.798  21.649  33.397  1.00  0.00           H  \\nATOM    599  HA  ALA A  60      43.408  23.692  31.872  1.00  0.00           H  \\nATOM    600 1HB  ALA A  60      45.724  23.277  32.362  1.00  0.00           H  \\nATOM    601 2HB  ALA A  60      44.714  23.264  33.827  1.00  0.00           H  \\nATOM    602 3HB  ALA A  60      45.217  21.730  33.079  1.00  0.00           H  \\nATOM    603  N   ALA A  61      44.087  20.803  30.675  1.00  0.00           N  \\nATOM    604  CA  ALA A  61      44.304  20.039  29.451  1.00  0.00           C  \\nATOM    605  C   ALA A  61      43.135  20.214  28.490  1.00  0.00           C  \\nATOM    606  O   ALA A  61      43.319  20.248  27.274  1.00  0.00           O  \\nATOM    607  CB  ALA A  61      44.512  18.566  29.772  1.00  0.00           C  \\nATOM    608  H   ALA A  61      44.068  20.320  31.387  1.00  0.00           H  \\nATOM    609  HA  ALA A  61      45.107  20.362  29.012  1.00  0.00           H  \\nATOM    610 1HB  ALA A  61      44.668  18.011  28.847  1.00  0.00           H  \\nATOM    611 2HB  ALA A  61      45.385  18.453  30.415  1.00  0.00           H  \\nATOM    612 3HB  ALA A  61      43.633  18.178  30.283  1.00  0.00           H  \\nATOM    613  N   ALA A  62      41.932  20.322  29.047  1.00  0.00           N  \\nATOM    614  CA  ALA A  62      40.722  20.473  28.246  1.00  0.00           C  \\nATOM    615  C   ALA A  62      40.705  21.801  27.493  1.00  0.00           C  \\nATOM    616  O   ALA A  62      40.435  21.835  26.294  1.00  0.00           O  \\nATOM    617  CB  ALA A  62      39.489  20.353  29.130  1.00  0.00           C  \\nATOM    618  H   ALA A  62      41.789  20.312  29.894  1.00  0.00           H  \\nATOM    619  HA  ALA A  62      40.688  19.758  27.591  1.00  0.00           H  \\nATOM    620 1HB  ALA A  62      38.592  20.462  28.520  1.00  0.00           H  \\nATOM    621 2HB  ALA A  62      39.482  19.376  29.614  1.00  0.00           H  \\nATOM    622 3HB  ALA A  62      39.508  21.134  29.888  1.00  0.00           H  \\nATOM    623  N   ALA A  63      40.990  22.890  28.203  1.00  0.00           N  \\nATOM    624  CA  ALA A  63      41.009  24.224  27.602  1.00  0.00           C  \\nATOM    625  C   ALA A  63      42.148  24.354  26.597  1.00  0.00           C  \\nATOM    626  O   ALA A  63      41.986  24.945  25.528  1.00  0.00           O  \\nATOM    627  CB  ALA A  63      41.130  25.292  28.679  1.00  0.00           C  \\nATOM    628  H   ALA A  63      41.177  22.884  29.042  1.00  0.00           H  \\nATOM    629  HA  ALA A  63      40.174  24.372  27.131  1.00  0.00           H  \\nATOM    630 1HB  ALA A  63      41.146  26.278  28.214  1.00  0.00           H  \\nATOM    631 2HB  ALA A  63      40.278  25.224  29.356  1.00  0.00           H  \\nATOM    632 3HB  ALA A  63      42.051  25.140  29.239  1.00  0.00           H  \\nATOM    633  N   ALA A  64      43.304  23.807  26.957  1.00  0.00           N  \\nATOM    634  CA  ALA A  64      44.451  23.767  26.059  1.00  0.00           C  \\nATOM    635  C   ALA A  64      44.063  23.125  24.732  1.00  0.00           C  \\nATOM    636  O   ALA A  64      44.505  23.557  23.669  1.00  0.00           O  \\nATOM    637  CB  ALA A  64      45.607  23.013  26.700  1.00  0.00           C  \\nATOM    638  H   ALA A  64      43.450  23.447  27.724  1.00  0.00           H  \\nATOM    639  HA  ALA A  64      44.749  24.673  25.881  1.00  0.00           H  \\nATOM    640 1HB  ALA A  64      46.452  22.988  26.012  1.00  0.00           H  \\nATOM    641 2HB  ALA A  64      45.903  23.516  27.621  1.00  0.00           H  \\nATOM    642 3HB  ALA A  64      45.295  21.995  26.928  1.00  0.00           H  \\nATOM    643  N   ALA A  65      43.222  22.099  24.806  1.00  0.00           N  \\nATOM    644  CA  ALA A  65      42.766  21.390  23.618  1.00  0.00           C  \\nATOM    645  C   ALA A  65      41.719  22.186  22.839  1.00  0.00           C  \\nATOM    646  O   ALA A  65      41.603  22.046  21.623  1.00  0.00           O  \\nATOM    647  CB  ALA A  65      42.209  20.028  24.006  1.00  0.00           C  \\nATOM    648  H   ALA A  65      42.898  21.791  25.541  1.00  0.00           H  \\nATOM    649  HA  ALA A  65      43.524  21.241  23.031  1.00  0.00           H  \\nATOM    650 1HB  ALA A  65      41.880  19.501  23.111  1.00  0.00           H  \\nATOM    651 2HB  ALA A  65      42.984  19.447  24.506  1.00  0.00           H  \\nATOM    652 3HB  ALA A  65      41.364  20.160  24.680  1.00  0.00           H  \\nATOM    653  N   ALA A  66      40.978  23.037  23.518  1.00  0.00           N  \\nATOM    654  CA  ALA A  66      39.980  23.885  22.899  1.00  0.00           C  \\nATOM    655  C   ALA A  66      40.596  25.055  22.179  1.00  0.00           C  \\nATOM    656  O   ALA A  66      40.085  25.540  21.207  1.00  0.00           O  \\nATOM    657  CB  ALA A  66      38.990  24.380  23.943  1.00  0.00           C  \\nATOM    658  H   ALA A  66      41.049  23.144  24.363  1.00  0.00           H  \\nATOM    659  HA  ALA A  66      39.493  23.362  22.239  1.00  0.00           H  \\nATOM    660 1HB  ALA A  66      38.242  25.012  23.465  1.00  0.00           H  \\nATOM    661 2HB  ALA A  66      38.498  23.527  24.411  1.00  0.00           H  \\nATOM    662 3HB  ALA A  66      39.519  24.955  24.702  1.00  0.00           H  \\nATOM    663  N   ALA A  67      41.732  25.479  22.675  1.00  0.00           N  \\nATOM    664  CA  ALA A  67      42.504  26.506  21.987  1.00  0.00           C  \\nATOM    665  C   ALA A  67      43.009  25.969  20.651  1.00  0.00           C  \\nATOM    666  O   ALA A  67      43.238  26.729  19.710  1.00  0.00           O  \\nATOM    667  CB  ALA A  67      43.665  26.976  22.852  1.00  0.00           C  \\nATOM    668  H   ALA A  67      42.092  25.195  23.403  1.00  0.00           H  \\nATOM    669  HA  ALA A  67      41.934  27.271  21.812  1.00  0.00           H  \\nATOM    670 1HB  ALA A  67      44.234  27.735  22.315  1.00  0.00           H  \\nATOM    671 2HB  ALA A  67      43.280  27.399  23.779  1.00  0.00           H  \\nATOM    672 3HB  ALA A  67      44.313  26.131  23.080  1.00  0.00           H  \\nATOM    673  N   ALA A  68      43.175  24.651  20.576  1.00  0.00           N  \\nATOM    674  CA  ALA A  68      43.667  24.000  19.368  1.00  0.00           C  \\nATOM    675  C   ALA A  68      42.547  23.785  18.354  1.00  0.00           C  \\nATOM    676  O   ALA A  68      42.754  23.937  17.150  1.00  0.00           O  \\nATOM    677  CB  ALA A  68      44.327  22.674  19.716  1.00  0.00           C  \\nATOM    678  H   ALA A  68      43.007  24.107  21.220  1.00  0.00           H  \\nATOM    679  HA  ALA A  68      44.339  24.566  18.956  1.00  0.00           H  \\nATOM    680 1HB  ALA A  68      44.693  22.199  18.805  1.00  0.00           H  \\nATOM    681 2HB  ALA A  68      45.162  22.849  20.394  1.00  0.00           H  \\nATOM    682 3HB  ALA A  68      43.600  22.021  20.197  1.00  0.00           H  \\nATOM    683  N   ALA A  69      41.363  23.430  18.842  1.00  0.00           N  \\nATOM    684  CA  ALA A  69      40.211  23.214  17.970  1.00  0.00           C  \\nATOM    685  C   ALA A  69      39.822  24.505  17.254  1.00  0.00           C  \\nATOM    686  O   ALA A  69      39.231  24.473  16.175  1.00  0.00           O  \\nATOM    687  CB  ALA A  69      39.033  22.674  18.768  1.00  0.00           C  \\nATOM    688  H   ALA A  69      41.199  23.307  19.677  1.00  0.00           H  \\nATOM    689  HA  ALA A  69      40.443  22.555  17.297  1.00  0.00           H  \\nATOM    690 1HB  ALA A  69      38.183  22.521  18.103  1.00  0.00           H  \\nATOM    691 2HB  ALA A  69      39.310  21.724  19.227  1.00  0.00           H  \\nATOM    692 3HB  ALA A  69      38.763  23.387  19.545  1.00  0.00           H  \\nATOM    693  N   ALA A  70      40.162  25.638  17.860  1.00  0.00           N  \\nATOM    694  CA  ALA A  70      39.855  26.942  17.284  1.00  0.00           C  \\nATOM    695  C   ALA A  70      40.852  27.303  16.187  1.00  0.00           C  \\nATOM    696  O   ALA A  70      42.065  27.204  16.377  1.00  0.00           O  \\nATOM    697  OXT ALA A  70      40.458  27.689  15.122  1.00  0.00           O  \\nATOM    698  CB  ALA A  70      39.846  28.011  18.367  1.00  0.00           C  \\nATOM    699  H   ALA A  70      40.576  25.679  18.612  1.00  0.00           H  \\nATOM    700  HA  ALA A  70      38.970  26.913  16.889  1.00  0.00           H  \\nATOM    701 1HB  ALA A  70      39.616  28.979  17.920  1.00  0.00           H  \\nATOM    702 2HB  ALA A  70      39.090  27.767  19.113  1.00  0.00           H  \\nATOM    703 3HB  ALA A  70      40.825  28.055  18.842  1.00  0.00           H  \\nTER                                                                             \\nATOM    705  N   TYR B  71      35.710  36.388  19.013  1.00  0.00           N  \\nATOM    706  CA  TYR B  71      34.402  36.488  18.434  1.00  0.00           C  \\nATOM    707  C   TYR B  71      33.573  35.299  18.771  1.00  0.00           C  \\nATOM    708  O   TYR B  71      33.315  34.433  18.000  1.00  0.00           O  \\nATOM    709  CB  TYR B  71      34.548  36.577  16.921  1.00  0.00           C  \\nATOM    710  CG  TYR B  71      35.147  35.286  16.337  1.00  0.00           C  \\nATOM    711  CD1 TYR B  71      34.314  34.280  15.860  1.00  0.00           C  \\nATOM    712  CD2 TYR B  71      36.529  35.103  16.315  1.00  0.00           C  \\nATOM    713  CE1 TYR B  71      34.854  33.115  15.345  1.00  0.00           C  \\nATOM    714  CE2 TYR B  71      37.068  33.929  15.806  1.00  0.00           C  \\nATOM    715  CZ  TYR B  71      36.236  32.942  15.317  1.00  0.00           C  \\nATOM    716  OH  TYR B  71      36.770  31.778  14.802  1.00  0.00           O  \\nATOM    717 1H   TYR B  71      36.245  37.198  18.772  1.00  0.00           H  \\nATOM    718 2H   TYR B  71      35.631  36.326  20.008  1.00  0.00           H  \\nATOM    719 3H   TYR B  71      36.168  35.570  18.664  1.00  0.00           H  \\nATOM    720  HA  TYR B  71      33.961  37.264  18.826  1.00  0.00           H  \\nATOM    721 1HB  TYR B  71      33.573  36.746  16.462  1.00  0.00           H  \\nATOM    722 2HB  TYR B  71      35.196  37.413  16.657  1.00  0.00           H  \\nATOM    723  HD1 TYR B  71      33.232  34.412  15.884  1.00  0.00           H  \\nATOM    724  HD2 TYR B  71      37.187  35.883  16.697  1.00  0.00           H  \\nATOM    725  HE1 TYR B  71      34.195  32.337  14.965  1.00  0.00           H  \\nATOM    726  HE2 TYR B  71      38.149  33.789  15.789  1.00  0.00           H  \\nATOM    727  HH  TYR B  71      37.701  31.720  15.032  1.00  0.00           H  \\nATOM    728  N   LYS B  72      33.117  35.298  19.982  1.00  0.00           N  \\nATOM    729  CA  LYS B  72      32.478  34.151  20.550  1.00  0.00           C  \\nATOM    730  C   LYS B  72      33.634  33.262  20.770  1.00  0.00           C  \\nATOM    731  O   LYS B  72      34.157  33.247  21.839  1.00  0.00           O  \\nATOM    732  CB  LYS B  72      31.342  33.622  19.660  1.00  0.00           C  \\nATOM    733  CG  LYS B  72      30.383  32.756  20.373  1.00  0.00           C  \\nATOM    734  CD  LYS B  72      29.266  32.286  19.437  1.00  0.00           C  \\nATOM    735  CE  LYS B  72      28.208  31.460  20.162  1.00  0.00           C  \\nATOM    736  NZ  LYS B  72      27.102  31.051  19.238  1.00  0.00           N  \\nATOM    737  H   LYS B  72      33.183  35.965  20.514  1.00  0.00           H  \\nATOM    738  HA  LYS B  72      32.084  34.379  21.413  1.00  0.00           H  \\nATOM    739 1HB  LYS B  72      30.799  34.449  19.203  1.00  0.00           H  \\nATOM    740 2HB  LYS B  72      31.722  32.987  18.865  1.00  0.00           H  \\nATOM    741 1HG  LYS B  72      30.903  31.988  20.830  1.00  0.00           H  \\nATOM    742 2HG  LYS B  72      29.922  33.345  21.163  1.00  0.00           H  \\nATOM    743 1HD  LYS B  72      28.788  33.155  18.984  1.00  0.00           H  \\nATOM    744 2HD  LYS B  72      29.701  31.679  18.641  1.00  0.00           H  \\nATOM    745 1HE  LYS B  72      28.668  30.567  20.579  1.00  0.00           H  \\nATOM    746 2HE  LYS B  72      27.789  32.050  20.976  1.00  0.00           H  \\nATOM    747 1HZ  LYS B  72      26.419  30.508  19.746  1.00  0.00           H  \\nATOM    748 2HZ  LYS B  72      26.659  31.876  18.856  1.00  0.00           H  \\nATOM    749 3HZ  LYS B  72      27.479  30.493  18.483  1.00  0.00           H  \\nATOM    750  N   ILE B  73      34.079  32.584  19.742  1.00  0.00           N  \\nATOM    751  CA  ILE B  73      35.171  31.658  19.903  1.00  0.00           C  \\nATOM    752  C   ILE B  73      36.358  32.336  20.561  1.00  0.00           C  \\nATOM    753  O   ILE B  73      36.745  31.991  21.645  1.00  0.00           O  \\nATOM    754  CB  ILE B  73      35.596  30.967  18.592  1.00  0.00           C  \\nATOM    755  CG1 ILE B  73      34.478  30.035  18.128  1.00  0.00           C  \\nATOM    756  CG2 ILE B  73      36.880  30.171  18.804  1.00  0.00           C  \\nATOM    757  CD1 ILE B  73      34.646  29.471  16.756  1.00  0.00           C  \\nATOM    758  H   ILE B  73      33.759  32.648  18.951  1.00  0.00           H  \\nATOM    759  HA  ILE B  73      34.887  30.943  20.499  1.00  0.00           H  \\nATOM    760  HB  ILE B  73      35.749  31.712  17.827  1.00  0.00           H  \\nATOM    761 1HG1 ILE B  73      34.394  29.207  18.836  1.00  0.00           H  \\nATOM    762 2HG1 ILE B  73      33.554  30.597  18.125  1.00  0.00           H  \\nATOM    763 1HG2 ILE B  73      37.160  29.688  17.868  1.00  0.00           H  \\nATOM    764 2HG2 ILE B  73      37.683  30.830  19.122  1.00  0.00           H  \\nATOM    765 3HG2 ILE B  73      36.712  29.410  19.572  1.00  0.00           H  \\nATOM    766 1HD1 ILE B  73      33.788  28.831  16.521  1.00  0.00           H  \\nATOM    767 2HD1 ILE B  73      34.702  30.275  16.035  1.00  0.00           H  \\nATOM    768 3HD1 ILE B  73      35.560  28.884  16.711  1.00  0.00           H  \\nATOM    769  N   LYS B  74      36.874  33.357  19.918  1.00  0.00           N  \\nATOM    770  CA  LYS B  74      37.966  34.129  20.458  1.00  0.00           C  \\nATOM    771  C   LYS B  74      37.682  34.728  21.827  1.00  0.00           C  \\nATOM    772  O   LYS B  74      38.529  34.736  22.665  1.00  0.00           O  \\nATOM    773  CB  LYS B  74      38.585  35.153  19.492  1.00  0.00           C  \\nATOM    774  CG  LYS B  74      39.826  35.837  20.036  1.00  0.00           C  \\nATOM    775  CD  LYS B  74      40.435  36.780  18.997  1.00  0.00           C  \\nATOM    776  CE  LYS B  74      41.659  37.502  19.546  1.00  0.00           C  \\nATOM    777  NZ  LYS B  74      42.255  38.425  18.541  1.00  0.00           N  \\nATOM    778  H   LYS B  74      36.597  33.608  19.149  1.00  0.00           H  \\nATOM    779  HA  LYS B  74      38.727  33.530  20.568  1.00  0.00           H  \\nATOM    780 1HB  LYS B  74      38.824  34.682  18.537  1.00  0.00           H  \\nATOM    781 2HB  LYS B  74      37.999  35.949  19.324  1.00  0.00           H  \\nATOM    782 1HG  LYS B  74      39.562  36.410  20.928  1.00  0.00           H  \\nATOM    783 2HG  LYS B  74      40.565  35.086  20.315  1.00  0.00           H  \\nATOM    784 1HD  LYS B  74      40.728  36.209  18.115  1.00  0.00           H  \\nATOM    785 2HD  LYS B  74      39.695  37.522  18.698  1.00  0.00           H  \\nATOM    786 1HE  LYS B  74      41.369  38.076  20.425  1.00  0.00           H  \\nATOM    787 2HE  LYS B  74      42.407  36.765  19.837  1.00  0.00           H  \\nATOM    788 1HZ  LYS B  74      41.570  39.119  18.274  1.00  0.00           H  \\nATOM    789 2HZ  LYS B  74      43.061  38.885  18.940  1.00  0.00           H  \\nATOM    790 3HZ  LYS B  74      42.536  37.900  17.725  1.00  0.00           H  \\nATOM    791  N   GLU B  75      36.494  35.236  22.053  1.00  0.00           N  \\nATOM    792  CA  GLU B  75      36.195  35.897  23.320  1.00  0.00           C  \\nATOM    793  C   GLU B  75      35.834  34.904  24.422  1.00  0.00           C  \\nATOM    794  O   GLU B  75      36.211  35.100  25.576  1.00  0.00           O  \\nATOM    795  CB  GLU B  75      35.029  36.854  23.074  1.00  0.00           C  \\nATOM    796  CG  GLU B  75      35.377  38.032  22.167  1.00  0.00           C  \\nATOM    797  CD  GLU B  75      34.164  38.748  21.626  1.00  0.00           C  \\nATOM    798  OE1 GLU B  75      33.083  38.243  21.804  1.00  0.00           O  \\nATOM    799  OE2 GLU B  75      34.327  39.758  20.992  1.00  0.00           O  \\nATOM    800  H   GLU B  75      35.837  35.220  21.498  1.00  0.00           H  \\nATOM    801  HA  GLU B  75      36.983  36.383  23.609  1.00  0.00           H  \\nATOM    802 1HB  GLU B  75      34.203  36.305  22.621  1.00  0.00           H  \\nATOM    803 2HB  GLU B  75      34.678  37.251  24.025  1.00  0.00           H  \\nATOM    804 1HG  GLU B  75      35.959  38.743  22.751  1.00  0.00           H  \\nATOM    805 2HG  GLU B  75      36.004  37.694  21.360  1.00  0.00           H  \\nATOM    806  N   THR B  76      35.111  33.866  24.054  1.00  0.00           N  \\nATOM    807  CA  THR B  76      34.785  32.804  24.978  1.00  0.00           C  \\nATOM    808  C   THR B  76      36.050  32.092  25.454  1.00  0.00           C  \\nATOM    809  O   THR B  76      36.195  31.740  26.589  1.00  0.00           O  \\nATOM    810  CB  THR B  76      33.755  31.751  24.462  1.00  0.00           C  \\nATOM    811  OG1 THR B  76      34.210  31.162  23.263  1.00  0.00           O  \\nATOM    812  CG2 THR B  76      32.367  32.349  24.258  1.00  0.00           C  \\nATOM    813  H   THR B  76      34.790  33.758  23.271  1.00  0.00           H  \\nATOM    814  HA  THR B  76      34.367  33.209  25.757  1.00  0.00           H  \\nATOM    815  HB  THR B  76      33.668  30.964  25.209  1.00  0.00           H  \\nATOM    816  HG1 THR B  76      34.608  31.847  22.702  1.00  0.00           H  \\nATOM    817 1HG2 THR B  76      31.686  31.564  23.923  1.00  0.00           H  \\nATOM    818 2HG2 THR B  76      32.012  32.759  25.203  1.00  0.00           H  \\nATOM    819 3HG2 THR B  76      32.400  33.133  23.524  1.00  0.00           H  \\nATOM    820  N   LEU B  77      36.976  31.938  24.544  1.00  0.00           N  \\nATOM    821  CA  LEU B  77      38.243  31.293  24.841  1.00  0.00           C  \\nATOM    822  C   LEU B  77      39.042  32.153  25.812  1.00  0.00           C  \\nATOM    823  O   LEU B  77      39.753  31.633  26.671  1.00  0.00           O  \\nATOM    824  CB  LEU B  77      38.996  31.042  23.534  1.00  0.00           C  \\nATOM    825  CG  LEU B  77      40.272  30.223  23.594  1.00  0.00           C  \\nATOM    826  CD1 LEU B  77      39.973  28.822  24.181  1.00  0.00           C  \\nATOM    827  CD2 LEU B  77      40.815  30.107  22.179  1.00  0.00           C  \\nATOM    828  H   LEU B  77      36.896  32.205  23.730  1.00  0.00           H  \\nATOM    829  HA  LEU B  77      38.074  30.436  25.263  1.00  0.00           H  \\nATOM    830 1HB  LEU B  77      38.320  30.519  22.859  1.00  0.00           H  \\nATOM    831 2HB  LEU B  77      39.235  32.009  23.086  1.00  0.00           H  \\nATOM    832  HG  LEU B  77      41.002  30.720  24.234  1.00  0.00           H  \\nATOM    833 1HD1 LEU B  77      39.241  28.311  23.554  1.00  0.00           H  \\nATOM    834 2HD1 LEU B  77      40.889  28.234  24.217  1.00  0.00           H  \\nATOM    835 3HD1 LEU B  77      39.574  28.932  25.186  1.00  0.00           H  \\nATOM    836 1HD2 LEU B  77      41.721  29.539  22.191  1.00  0.00           H  \\nATOM    837 2HD2 LEU B  77      40.077  29.608  21.546  1.00  0.00           H  \\nATOM    838 3HD2 LEU B  77      41.016  31.104  21.786  1.00  0.00           H  \\nATOM    839  N   LYS B  78      38.909  33.471  25.687  1.00  0.00           N  \\nATOM    840  CA  LYS B  78      39.591  34.390  26.594  1.00  0.00           C  \\nATOM    841  C   LYS B  78      38.963  34.340  27.984  1.00  0.00           C  \\nATOM    842  O   LYS B  78      39.668  34.400  28.992  1.00  0.00           O  \\nATOM    843  CB  LYS B  78      39.579  35.813  26.048  1.00  0.00           C  \\nATOM    844  CG  LYS B  78      40.454  36.790  26.843  1.00  0.00           C  \\nATOM    845  CD  LYS B  78      41.953  36.404  26.702  1.00  0.00           C  \\nATOM    846  CE  LYS B  78      42.883  37.437  27.342  1.00  0.00           C  \\nATOM    847  NZ  LYS B  78      44.339  37.051  27.212  1.00  0.00           N  \\nATOM    848  H   LYS B  78      38.431  33.858  25.086  1.00  0.00           H  \\nATOM    849  HA  LYS B  78      40.520  34.122  26.674  1.00  0.00           H  \\nATOM    850 1HB  LYS B  78      39.912  35.809  25.008  1.00  0.00           H  \\nATOM    851 2HB  LYS B  78      38.557  36.193  26.058  1.00  0.00           H  \\nATOM    852 1HG  LYS B  78      40.301  37.803  26.476  1.00  0.00           H  \\nATOM    853 2HG  LYS B  78      40.174  36.753  27.900  1.00  0.00           H  \\nATOM    854 1HD  LYS B  78      42.126  35.438  27.191  1.00  0.00           H  \\nATOM    855 2HD  LYS B  78      42.207  36.304  25.648  1.00  0.00           H  \\nATOM    856 1HE  LYS B  78      42.729  38.395  26.850  1.00  0.00           H  \\nATOM    857 2HE  LYS B  78      42.638  37.536  28.402  1.00  0.00           H  \\nATOM    858 1HZ  LYS B  78      44.914  37.756  27.637  1.00  0.00           H  \\nATOM    859 2HZ  LYS B  78      44.517  36.145  27.694  1.00  0.00           H  \\nATOM    860 3HZ  LYS B  78      44.584  36.959  26.241  1.00  0.00           H  \\nATOM    861  N   ARG B  79      37.638  34.239  28.035  1.00  0.00           N  \\nATOM    862  CA  ARG B  79      36.939  34.079  29.307  1.00  0.00           C  \\nATOM    863  C   ARG B  79      37.380  32.789  29.983  1.00  0.00           C  \\nATOM    864  O   ARG B  79      37.585  32.749  31.196  1.00  0.00           O  \\nATOM    865  CB  ARG B  79      35.440  34.081  29.118  1.00  0.00           C  \\nATOM    866  CG  ARG B  79      34.852  35.425  28.740  1.00  0.00           C  \\nATOM    867  CD  ARG B  79      33.411  35.310  28.542  1.00  0.00           C  \\nATOM    868  NE  ARG B  79      32.745  34.991  29.783  1.00  0.00           N  \\nATOM    869  CZ  ARG B  79      31.610  34.294  29.866  1.00  0.00           C  \\nATOM    870  NH1 ARG B  79      31.044  33.861  28.770  1.00  0.00           N  \\nATOM    871  NH2 ARG B  79      31.075  34.044  31.037  1.00  0.00           N  \\nATOM    872  H   ARG B  79      37.121  34.260  27.349  1.00  0.00           H  \\nATOM    873  HA  ARG B  79      37.161  34.821  29.890  1.00  0.00           H  \\nATOM    874 1HB  ARG B  79      35.173  33.372  28.339  1.00  0.00           H  \\nATOM    875 2HB  ARG B  79      34.957  33.753  30.035  1.00  0.00           H  \\nATOM    876 1HG  ARG B  79      35.039  36.145  29.535  1.00  0.00           H  \\nATOM    877 2HG  ARG B  79      35.305  35.776  27.816  1.00  0.00           H  \\nATOM    878 1HD  ARG B  79      33.012  36.252  28.168  1.00  0.00           H  \\nATOM    879 2HD  ARG B  79      33.204  34.514  27.825  1.00  0.00           H  \\nATOM    880  HE  ARG B  79      33.167  35.312  30.644  1.00  0.00           H  \\nATOM    881 1HH1 ARG B  79      31.464  34.058  27.879  1.00  0.00           H  \\nATOM    882 2HH1 ARG B  79      30.189  33.291  28.803  1.00  0.00           H  \\nATOM    883 1HH2 ARG B  79      31.523  34.376  31.880  1.00  0.00           H  \\nATOM    884 2HH2 ARG B  79      30.222  33.507  31.090  1.00  0.00           H  \\nATOM    885  N   LEU B  80      37.524  31.735  29.187  1.00  0.00           N  \\nATOM    886  CA  LEU B  80      37.970  30.445  29.697  1.00  0.00           C  \\nATOM    887  C   LEU B  80      39.362  30.561  30.318  1.00  0.00           C  \\nATOM    888  O   LEU B  80      39.635  29.971  31.366  1.00  0.00           O  \\nATOM    889  CB  LEU B  80      37.932  29.412  28.556  1.00  0.00           C  \\nATOM    890  CG  LEU B  80      38.245  27.973  28.918  1.00  0.00           C  \\nATOM    891  CD1 LEU B  80      37.246  27.514  29.973  1.00  0.00           C  \\nATOM    892  CD2 LEU B  80      38.125  27.100  27.642  1.00  0.00           C  \\nATOM    893  H   LEU B  80      37.369  31.742  28.341  1.00  0.00           H  \\nATOM    894  HA  LEU B  80      37.356  30.147  30.386  1.00  0.00           H  \\nATOM    895 1HB  LEU B  80      36.948  29.440  28.089  1.00  0.00           H  \\nATOM    896 2HB  LEU B  80      38.659  29.710  27.812  1.00  0.00           H  \\nATOM    897  HG  LEU B  80      39.252  27.896  29.333  1.00  0.00           H  \\nATOM    898 1HD1 LEU B  80      36.247  27.598  29.576  1.00  0.00           H  \\nATOM    899 2HD1 LEU B  80      37.447  26.480  30.246  1.00  0.00           H  \\nATOM    900 3HD1 LEU B  80      37.333  28.144  30.856  1.00  0.00           H  \\nATOM    901 1HD2 LEU B  80      38.326  26.060  27.886  1.00  0.00           H  \\nATOM    902 2HD2 LEU B  80      37.114  27.183  27.234  1.00  0.00           H  \\nATOM    903 3HD2 LEU B  80      38.839  27.439  26.897  1.00  0.00           H  \\nATOM    904  N   GLU B  81      40.231  31.335  29.672  1.00  0.00           N  \\nATOM    905  CA  GLU B  81      41.593  31.544  30.155  1.00  0.00           C  \\nATOM    906  C   GLU B  81      41.589  32.221  31.518  1.00  0.00           C  \\nATOM    907  O   GLU B  81      42.308  31.807  32.426  1.00  0.00           O  \\nATOM    908  CB  GLU B  81      42.377  32.408  29.149  1.00  0.00           C  \\nATOM    909  CG  GLU B  81      43.839  32.729  29.501  1.00  0.00           C  \\nATOM    910  CD  GLU B  81      44.493  33.612  28.426  1.00  0.00           C  \\nATOM    911  OE1 GLU B  81      44.480  33.242  27.280  1.00  0.00           O  \\nATOM    912  OE2 GLU B  81      44.967  34.685  28.755  1.00  0.00           O  \\nATOM    913  H   GLU B  81      40.053  31.756  28.943  1.00  0.00           H  \\nATOM    914  HA  GLU B  81      42.031  30.684  30.248  1.00  0.00           H  \\nATOM    915 1HB  GLU B  81      42.360  31.926  28.171  1.00  0.00           H  \\nATOM    916 2HB  GLU B  81      41.881  33.361  29.039  1.00  0.00           H  \\nATOM    917 1HG  GLU B  81      43.862  33.255  30.458  1.00  0.00           H  \\nATOM    918 2HG  GLU B  81      44.398  31.801  29.606  1.00  0.00           H  \\nATOM    919  N   ASP B  82      40.772  33.260  31.661  1.00  0.00           N  \\nATOM    920  CA  ASP B  82      40.694  33.995  32.918  1.00  0.00           C  \\nATOM    921  C   ASP B  82      40.091  33.126  34.013  1.00  0.00           C  \\nATOM    922  O   ASP B  82      40.580  33.110  35.143  1.00  0.00           O  \\nATOM    923  CB  ASP B  82      39.902  35.295  32.763  1.00  0.00           C  \\nATOM    924  CG  ASP B  82      40.691  36.378  32.014  1.00  0.00           C  \\nATOM    925  OD1 ASP B  82      41.875  36.189  31.843  1.00  0.00           O  \\nATOM    926  OD2 ASP B  82      40.120  37.380  31.647  1.00  0.00           O  \\nATOM    927  H   ASP B  82      40.251  33.559  31.045  1.00  0.00           H  \\nATOM    928  HA  ASP B  82      41.589  34.249  33.193  1.00  0.00           H  \\nATOM    929 1HB  ASP B  82      38.984  35.086  32.207  1.00  0.00           H  \\nATOM    930 2HB  ASP B  82      39.617  35.672  33.743  1.00  0.00           H  \\nATOM    931  N   SER B  83      39.029  32.403  33.672  1.00  0.00           N  \\nATOM    932  CA  SER B  83      38.361  31.532  34.631  1.00  0.00           C  \\nATOM    933  C   SER B  83      39.310  30.443  35.113  1.00  0.00           C  \\nATOM    934  O   SER B  83      39.338  30.112  36.295  1.00  0.00           O  \\nATOM    935  CB  SER B  83      37.127  30.935  34.025  1.00  0.00           C  \\nATOM    936  OG  SER B  83      36.461  30.165  34.973  1.00  0.00           O  \\nATOM    937  H   SER B  83      38.675  32.399  32.888  1.00  0.00           H  \\nATOM    938  HA  SER B  83      38.087  32.056  35.400  1.00  0.00           H  \\nATOM    939 1HB  SER B  83      36.489  31.727  33.664  1.00  0.00           H  \\nATOM    940 2HB  SER B  83      37.403  30.316  33.169  1.00  0.00           H  \\nATOM    941  HG  SER B  83      37.141  29.648  35.408  1.00  0.00           H  \\nATOM    942  N   LEU B  84      40.096  29.898  34.191  1.00  0.00           N  \\nATOM    943  CA  LEU B  84      41.055  28.856  34.529  1.00  0.00           C  \\nATOM    944  C   LEU B  84      42.145  29.407  35.446  1.00  0.00           C  \\nATOM    945  O   LEU B  84      42.570  28.737  36.390  1.00  0.00           O  \\nATOM    946  CB  LEU B  84      41.656  28.240  33.266  1.00  0.00           C  \\nATOM    947  CG  LEU B  84      42.638  27.082  33.495  1.00  0.00           C  \\nATOM    948  CD1 LEU B  84      41.946  25.929  34.253  1.00  0.00           C  \\nATOM    949  CD2 LEU B  84      43.168  26.620  32.161  1.00  0.00           C  \\nATOM    950  H   LEU B  84      40.093  30.116  33.359  1.00  0.00           H  \\nATOM    951  HA  LEU B  84      40.595  28.144  35.001  1.00  0.00           H  \\nATOM    952 1HB  LEU B  84      40.839  27.876  32.638  1.00  0.00           H  \\nATOM    953 2HB  LEU B  84      42.178  29.027  32.716  1.00  0.00           H  \\nATOM    954  HG  LEU B  84      43.444  27.429  34.093  1.00  0.00           H  \\nATOM    955 1HD1 LEU B  84      42.658  25.118  34.422  1.00  0.00           H  \\nATOM    956 2HD1 LEU B  84      41.585  26.296  35.210  1.00  0.00           H  \\nATOM    957 3HD1 LEU B  84      41.106  25.554  33.666  1.00  0.00           H  \\nATOM    958 1HD2 LEU B  84      43.878  25.831  32.322  1.00  0.00           H  \\nATOM    959 2HD2 LEU B  84      42.343  26.259  31.545  1.00  0.00           H  \\nATOM    960 3HD2 LEU B  84      43.661  27.453  31.660  1.00  0.00           H  \\nATOM    961  N   ARG B  85      42.587  30.632  35.169  1.00  0.00           N  \\nATOM    962  CA  ARG B  85      43.560  31.306  36.025  1.00  0.00           C  \\nATOM    963  C   ARG B  85      42.993  31.486  37.427  1.00  0.00           C  \\nATOM    964  O   ARG B  85      43.672  31.220  38.421  1.00  0.00           O  \\nATOM    965  CB  ARG B  85      43.916  32.665  35.443  1.00  0.00           C  \\nATOM    966  CG  ARG B  85      45.007  33.439  36.151  1.00  0.00           C  \\nATOM    967  CD  ARG B  85      45.153  34.794  35.547  1.00  0.00           C  \\nATOM    968  NE  ARG B  85      45.525  34.727  34.140  1.00  0.00           N  \\nATOM    969  CZ  ARG B  85      45.141  35.626  33.209  1.00  0.00           C  \\nATOM    970  NH1 ARG B  85      44.386  36.649  33.551  1.00  0.00           N  \\nATOM    971  NH2 ARG B  85      45.510  35.479  31.949  1.00  0.00           N  \\nATOM    972  H   ARG B  85      42.338  31.096  34.489  1.00  0.00           H  \\nATOM    973  HA  ARG B  85      44.360  30.762  36.088  1.00  0.00           H  \\nATOM    974 1HB  ARG B  85      44.208  32.545  34.399  1.00  0.00           H  \\nATOM    975 2HB  ARG B  85      43.036  33.298  35.450  1.00  0.00           H  \\nATOM    976 1HG  ARG B  85      44.746  33.553  37.205  1.00  0.00           H  \\nATOM    977 2HG  ARG B  85      45.956  32.909  36.062  1.00  0.00           H  \\nATOM    978 1HD  ARG B  85      44.201  35.322  35.621  1.00  0.00           H  \\nATOM    979 2HD  ARG B  85      45.922  35.352  36.079  1.00  0.00           H  \\nATOM    980  HE  ARG B  85      46.096  33.951  33.837  1.00  0.00           H  \\nATOM    981 1HH1 ARG B  85      44.086  36.763  34.504  1.00  0.00           H  \\nATOM    982 2HH1 ARG B  85      44.084  37.311  32.842  1.00  0.00           H  \\nATOM    983 1HH2 ARG B  85      46.075  34.693  31.667  1.00  0.00           H  \\nATOM    984 2HH2 ARG B  85      45.201  36.144  31.257  1.00  0.00           H  \\nATOM    985  N   GLU B  86      41.753  31.958  37.499  1.00  0.00           N  \\nATOM    986  CA  GLU B  86      41.075  32.134  38.777  1.00  0.00           C  \\nATOM    987  C   GLU B  86      41.061  30.815  39.546  1.00  0.00           C  \\nATOM    988  O   GLU B  86      41.308  30.780  40.750  1.00  0.00           O  \\nATOM    989  CB  GLU B  86      39.661  32.686  38.604  1.00  0.00           C  \\nATOM    990  CG  GLU B  86      38.863  32.822  39.913  1.00  0.00           C  \\nATOM    991  CD  GLU B  86      39.491  33.757  40.936  1.00  0.00           C  \\nATOM    992  OE1 GLU B  86      40.397  34.491  40.610  1.00  0.00           O  \\nATOM    993  OE2 GLU B  86      39.050  33.728  42.063  1.00  0.00           O  \\nATOM    994  H   GLU B  86      41.280  32.185  36.817  1.00  0.00           H  \\nATOM    995  HA  GLU B  86      41.555  32.790  39.307  1.00  0.00           H  \\nATOM    996 1HB  GLU B  86      39.709  33.664  38.126  1.00  0.00           H  \\nATOM    997 2HB  GLU B  86      39.098  32.034  37.945  1.00  0.00           H  \\nATOM    998 1HG  GLU B  86      37.864  33.183  39.674  1.00  0.00           H  \\nATOM    999 2HG  GLU B  86      38.761  31.830  40.356  1.00  0.00           H  \\nATOM   1000  N   LEU B  87      40.793  29.730  38.829  1.00  0.00           N  \\nATOM   1001  CA  LEU B  87      40.686  28.409  39.436  1.00  0.00           C  \\nATOM   1002  C   LEU B  87      42.037  27.927  39.957  1.00  0.00           C  \\nATOM   1003  O   LEU B  87      42.142  27.441  41.085  1.00  0.00           O  \\nATOM   1004  CB  LEU B  87      40.105  27.452  38.381  1.00  0.00           C  \\nATOM   1005  CG  LEU B  87      39.924  26.011  38.746  1.00  0.00           C  \\nATOM   1006  CD1 LEU B  87      39.017  25.894  39.942  1.00  0.00           C  \\nATOM   1007  CD2 LEU B  87      39.307  25.288  37.521  1.00  0.00           C  \\nATOM   1008  H   LEU B  87      40.667  29.732  37.978  1.00  0.00           H  \\nATOM   1009  HA  LEU B  87      40.074  28.455  40.188  1.00  0.00           H  \\nATOM   1010 1HB  LEU B  87      39.133  27.840  38.068  1.00  0.00           H  \\nATOM   1011 2HB  LEU B  87      40.755  27.472  37.518  1.00  0.00           H  \\nATOM   1012  HG  LEU B  87      40.886  25.575  38.999  1.00  0.00           H  \\nATOM   1013 1HD1 LEU B  87      38.876  24.847  40.202  1.00  0.00           H  \\nATOM   1014 2HD1 LEU B  87      39.452  26.421  40.792  1.00  0.00           H  \\nATOM   1015 3HD1 LEU B  87      38.075  26.329  39.693  1.00  0.00           H  \\nATOM   1016 1HD2 LEU B  87      39.971  25.382  36.669  1.00  0.00           H  \\nATOM   1017 2HD2 LEU B  87      39.149  24.237  37.734  1.00  0.00           H  \\nATOM   1018 3HD2 LEU B  87      38.349  25.746  37.279  1.00  0.00           H  \\nATOM   1019  N   ARG B  88      43.069  28.072  39.132  1.00  0.00           N  \\nATOM   1020  CA  ARG B  88      44.414  27.652  39.511  1.00  0.00           C  \\nATOM   1021  C   ARG B  88      44.926  28.441  40.712  1.00  0.00           C  \\nATOM   1022  O   ARG B  88      45.667  27.911  41.536  1.00  0.00           O  \\nATOM   1023  CB  ARG B  88      45.353  27.762  38.327  1.00  0.00           C  \\nATOM   1024  CG  ARG B  88      45.111  26.686  37.255  1.00  0.00           C  \\nATOM   1025  CD  ARG B  88      46.012  26.829  36.092  1.00  0.00           C  \\nATOM   1026  NE  ARG B  88      45.679  27.988  35.297  1.00  0.00           N  \\nATOM   1027  CZ  ARG B  88      46.387  28.422  34.239  1.00  0.00           C  \\nATOM   1028  NH1 ARG B  88      47.483  27.794  33.874  1.00  0.00           N  \\nATOM   1029  NH2 ARG B  88      45.983  29.480  33.566  1.00  0.00           N  \\nATOM   1030  H   ARG B  88      43.017  28.412  38.344  1.00  0.00           H  \\nATOM   1031  HA  ARG B  88      44.393  26.714  39.756  1.00  0.00           H  \\nATOM   1032 1HB  ARG B  88      45.220  28.738  37.855  1.00  0.00           H  \\nATOM   1033 2HB  ARG B  88      46.385  27.690  38.663  1.00  0.00           H  \\nATOM   1034 1HG  ARG B  88      45.280  25.704  37.691  1.00  0.00           H  \\nATOM   1035 2HG  ARG B  88      44.080  26.748  36.901  1.00  0.00           H  \\nATOM   1036 1HD  ARG B  88      47.039  26.933  36.436  1.00  0.00           H  \\nATOM   1037 2HD  ARG B  88      45.924  25.945  35.458  1.00  0.00           H  \\nATOM   1038  HE  ARG B  88      44.832  28.497  35.548  1.00  0.00           H  \\nATOM   1039 1HH1 ARG B  88      47.797  26.988  34.394  1.00  0.00           H  \\nATOM   1040 2HH1 ARG B  88      48.013  28.125  33.080  1.00  0.00           H  \\nATOM   1041 1HH2 ARG B  88      45.142  29.961  33.851  1.00  0.00           H  \\nATOM   1042 2HH2 ARG B  88      46.509  29.807  32.770  1.00  0.00           H  \\nATOM   1043  N   ARG B  89      44.521  29.704  40.817  1.00  0.00           N  \\nATOM   1044  CA  ARG B  89      44.936  30.545  41.935  1.00  0.00           C  \\nATOM   1045  C   ARG B  89      44.346  30.033  43.242  1.00  0.00           C  \\nATOM   1046  O   ARG B  89      45.046  29.917  44.248  1.00  0.00           O  \\nATOM   1047  CB  ARG B  89      44.442  31.975  41.722  1.00  0.00           C  \\nATOM   1048  CG  ARG B  89      44.833  32.977  42.795  1.00  0.00           C  \\nATOM   1049  CD  ARG B  89      44.139  34.311  42.614  1.00  0.00           C  \\nATOM   1050  NE  ARG B  89      42.678  34.245  42.877  1.00  0.00           N  \\nATOM   1051  CZ  ARG B  89      42.135  34.231  44.128  1.00  0.00           C  \\nATOM   1052  NH1 ARG B  89      42.922  34.262  45.189  1.00  0.00           N  \\nATOM   1053  NH2 ARG B  89      40.822  34.194  44.295  1.00  0.00           N  \\nATOM   1054  H   ARG B  89      44.006  30.098  40.253  1.00  0.00           H  \\nATOM   1055  HA  ARG B  89      45.903  30.517  42.011  1.00  0.00           H  \\nATOM   1056 1HB  ARG B  89      44.808  32.344  40.764  1.00  0.00           H  \\nATOM   1057 2HB  ARG B  89      43.354  31.975  41.666  1.00  0.00           H  \\nATOM   1058 1HG  ARG B  89      44.554  32.580  43.771  1.00  0.00           H  \\nATOM   1059 2HG  ARG B  89      45.910  33.141  42.763  1.00  0.00           H  \\nATOM   1060 1HD  ARG B  89      44.574  35.042  43.293  1.00  0.00           H  \\nATOM   1061 2HD  ARG B  89      44.278  34.645  41.586  1.00  0.00           H  \\nATOM   1062  HE  ARG B  89      42.020  34.244  42.069  1.00  0.00           H  \\nATOM   1063 1HH1 ARG B  89      43.925  34.289  45.076  1.00  0.00           H  \\nATOM   1064 2HH1 ARG B  89      42.518  34.251  46.117  1.00  0.00           H  \\nATOM   1065 1HH2 ARG B  89      40.198  34.162  43.470  1.00  0.00           H  \\nATOM   1066 2HH2 ARG B  89      40.430  34.184  45.220  1.00  0.00           H  \\nATOM   1067  N   ILE B  90      43.050  29.737  43.218  1.00  0.00           N  \\nATOM   1068  CA  ILE B  90      42.351  29.213  44.386  1.00  0.00           C  \\nATOM   1069  C   ILE B  90      42.933  27.861  44.779  1.00  0.00           C  \\nATOM   1070  O   ILE B  90      42.972  27.507  45.958  1.00  0.00           O  \\nATOM   1071  CB  ILE B  90      40.848  29.075  44.134  1.00  0.00           C  \\nATOM   1072  CG1 ILE B  90      40.233  30.451  43.982  1.00  0.00           C  \\nATOM   1073  CG2 ILE B  90      40.198  28.338  45.313  1.00  0.00           C  \\nATOM   1074  CD1 ILE B  90      38.849  30.409  43.454  1.00  0.00           C  \\nATOM   1075  H   ILE B  90      42.548  29.831  42.526  1.00  0.00           H  \\nATOM   1076  HA  ILE B  90      42.471  29.823  45.130  1.00  0.00           H  \\nATOM   1077  HB  ILE B  90      40.677  28.523  43.205  1.00  0.00           H  \\nATOM   1078 1HG1 ILE B  90      40.223  30.945  44.951  1.00  0.00           H  \\nATOM   1079 2HG1 ILE B  90      40.843  31.045  43.294  1.00  0.00           H  \\nATOM   1080 1HG2 ILE B  90      39.136  28.250  45.139  1.00  0.00           H  \\nATOM   1081 2HG2 ILE B  90      40.640  27.349  45.409  1.00  0.00           H  \\nATOM   1082 3HG2 ILE B  90      40.367  28.899  46.233  1.00  0.00           H  \\nATOM   1083 1HD1 ILE B  90      38.462  31.427  43.358  1.00  0.00           H  \\nATOM   1084 2HD1 ILE B  90      38.883  29.940  42.477  1.00  0.00           H  \\nATOM   1085 3HD1 ILE B  90      38.208  29.834  44.122  1.00  0.00           H  \\nATOM   1086  N   LEU B  91      43.392  27.109  43.786  1.00  0.00           N  \\nATOM   1087  CA  LEU B  91      43.991  25.809  44.047  1.00  0.00           C  \\nATOM   1088  C   LEU B  91      45.312  25.962  44.800  1.00  0.00           C  \\nATOM   1089  O   LEU B  91      45.623  25.158  45.673  1.00  0.00           O  \\nATOM   1090  CB  LEU B  91      44.228  25.034  42.751  1.00  0.00           C  \\nATOM   1091  CG  LEU B  91      44.785  23.616  42.927  1.00  0.00           C  \\nATOM   1092  CD1 LEU B  91      43.826  22.791  43.780  1.00  0.00           C  \\nATOM   1093  CD2 LEU B  91      44.965  22.996  41.568  1.00  0.00           C  \\nATOM   1094  H   LEU B  91      43.368  27.328  42.955  1.00  0.00           H  \\nATOM   1095  HA  LEU B  91      43.388  25.289  44.601  1.00  0.00           H  \\nATOM   1096 1HB  LEU B  91      43.290  24.975  42.197  1.00  0.00           H  \\nATOM   1097 2HB  LEU B  91      44.943  25.586  42.150  1.00  0.00           H  \\nATOM   1098  HG  LEU B  91      45.751  23.661  43.437  1.00  0.00           H  \\nATOM   1099 1HD1 LEU B  91      44.217  21.779  43.907  1.00  0.00           H  \\nATOM   1100 2HD1 LEU B  91      43.713  23.258  44.756  1.00  0.00           H  \\nATOM   1101 3HD1 LEU B  91      42.870  22.750  43.286  1.00  0.00           H  \\nATOM   1102 1HD2 LEU B  91      45.364  22.005  41.681  1.00  0.00           H  \\nATOM   1103 2HD2 LEU B  91      44.013  22.950  41.075  1.00  0.00           H  \\nATOM   1104 3HD2 LEU B  91      45.655  23.599  40.979  1.00  0.00           H  \\nATOM   1105  N   GLU B  92      46.086  26.990  44.459  1.00  0.00           N  \\nATOM   1106  CA  GLU B  92      47.334  27.270  45.167  1.00  0.00           C  \\nATOM   1107  C   GLU B  92      47.059  27.586  46.632  1.00  0.00           C  \\nATOM   1108  O   GLU B  92      47.791  27.152  47.518  1.00  0.00           O  \\nATOM   1109  CB  GLU B  92      48.119  28.405  44.502  1.00  0.00           C  \\nATOM   1110  CG  GLU B  92      48.755  28.035  43.151  1.00  0.00           C  \\nATOM   1111  CD  GLU B  92      49.882  27.021  43.270  1.00  0.00           C  \\nATOM   1112  OE1 GLU B  92      50.793  27.264  44.026  1.00  0.00           O  \\nATOM   1113  OE2 GLU B  92      49.827  26.008  42.607  1.00  0.00           O  \\nATOM   1114  H   GLU B  92      45.911  27.541  43.823  1.00  0.00           H  \\nATOM   1115  HA  GLU B  92      47.903  26.485  45.129  1.00  0.00           H  \\nATOM   1116 1HB  GLU B  92      47.463  29.253  44.336  1.00  0.00           H  \\nATOM   1117 2HB  GLU B  92      48.914  28.735  45.170  1.00  0.00           H  \\nATOM   1118 1HG  GLU B  92      47.983  27.617  42.511  1.00  0.00           H  \\nATOM   1119 2HG  GLU B  92      49.129  28.940  42.676  1.00  0.00           H  \\nATOM   1120  N   GLU B  93      45.999  28.350  46.878  1.00  0.00           N  \\nATOM   1121  CA  GLU B  93      45.582  28.670  48.238  1.00  0.00           C  \\nATOM   1122  C   GLU B  93      45.148  27.410  48.975  1.00  0.00           C  \\nATOM   1123  O   GLU B  93      45.513  27.195  50.130  1.00  0.00           O  \\nATOM   1124  CB  GLU B  93      44.432  29.691  48.257  1.00  0.00           C  \\nATOM   1125  CG  GLU B  93      43.964  30.089  49.679  1.00  0.00           C  \\nATOM   1126  CD  GLU B  93      42.902  31.187  49.703  1.00  0.00           C  \\nATOM   1127  OE1 GLU B  93      42.535  31.674  48.656  1.00  0.00           O  \\nATOM   1128  OE2 GLU B  93      42.469  31.537  50.776  1.00  0.00           O  \\nATOM   1129  H   GLU B  93      45.501  28.698  46.269  1.00  0.00           H  \\nATOM   1130  HA  GLU B  93      46.331  29.059  48.718  1.00  0.00           H  \\nATOM   1131 1HB  GLU B  93      44.734  30.592  47.726  1.00  0.00           H  \\nATOM   1132 2HB  GLU B  93      43.571  29.278  47.733  1.00  0.00           H  \\nATOM   1133 1HG  GLU B  93      43.556  29.202  50.168  1.00  0.00           H  \\nATOM   1134 2HG  GLU B  93      44.830  30.415  50.254  1.00  0.00           H  \\nATOM   1135  N   LEU B  94      44.360  26.583  48.295  1.00  0.00           N  \\nATOM   1136  CA  LEU B  94      43.876  25.330  48.862  1.00  0.00           C  \\nATOM   1137  C   LEU B  94      45.039  24.404  49.208  1.00  0.00           C  \\nATOM   1138  O   LEU B  94      45.004  23.690  50.209  1.00  0.00           O  \\nATOM   1139  CB  LEU B  94      42.908  24.651  47.878  1.00  0.00           C  \\nATOM   1140  CG  LEU B  94      42.280  23.347  48.345  1.00  0.00           C  \\nATOM   1141  CD1 LEU B  94      41.538  23.561  49.665  1.00  0.00           C  \\nATOM   1142  CD2 LEU B  94      41.291  22.843  47.247  1.00  0.00           C  \\nATOM   1143  H   LEU B  94      44.088  26.729  47.492  1.00  0.00           H  \\nATOM   1144  HA  LEU B  94      43.387  25.518  49.678  1.00  0.00           H  \\nATOM   1145 1HB  LEU B  94      42.126  25.346  47.604  1.00  0.00           H  \\nATOM   1146 2HB  LEU B  94      43.470  24.421  46.979  1.00  0.00           H  \\nATOM   1147  HG  LEU B  94      43.067  22.618  48.510  1.00  0.00           H  \\nATOM   1148 1HD1 LEU B  94      41.126  22.621  49.987  1.00  0.00           H  \\nATOM   1149 2HD1 LEU B  94      42.234  23.923  50.422  1.00  0.00           H  \\nATOM   1150 3HD1 LEU B  94      40.744  24.277  49.535  1.00  0.00           H  \\nATOM   1151 1HD2 LEU B  94      40.832  21.909  47.553  1.00  0.00           H  \\nATOM   1152 2HD2 LEU B  94      40.511  23.591  47.093  1.00  0.00           H  \\nATOM   1153 3HD2 LEU B  94      41.833  22.687  46.314  1.00  0.00           H  \\nATOM   1154  N   LYS B  95      46.072  24.431  48.373  1.00  0.00           N  \\nATOM   1155  CA  LYS B  95      47.251  23.596  48.574  1.00  0.00           C  \\nATOM   1156  C   LYS B  95      48.018  24.013  49.826  1.00  0.00           C  \\nATOM   1157  O   LYS B  95      48.537  23.169  50.556  1.00  0.00           O  \\nATOM   1158  CB  LYS B  95      48.090  23.600  47.296  1.00  0.00           C  \\nATOM   1159  CG  LYS B  95      49.190  22.550  47.229  1.00  0.00           C  \\nATOM   1160  CD  LYS B  95      49.746  22.477  45.794  1.00  0.00           C  \\nATOM   1161  CE  LYS B  95      50.760  21.348  45.615  1.00  0.00           C  \\nATOM   1162  NZ  LYS B  95      50.111  19.973  45.719  1.00  0.00           N  \\nATOM   1163  H   LYS B  95      46.115  24.930  47.674  1.00  0.00           H  \\nATOM   1164  HA  LYS B  95      46.972  22.674  48.684  1.00  0.00           H  \\nATOM   1165 1HB  LYS B  95      47.435  23.452  46.435  1.00  0.00           H  \\nATOM   1166 2HB  LYS B  95      48.554  24.583  47.178  1.00  0.00           H  \\nATOM   1167 1HG  LYS B  95      49.992  22.814  47.921  1.00  0.00           H  \\nATOM   1168 2HG  LYS B  95      48.787  21.577  47.511  1.00  0.00           H  \\nATOM   1169 1HD  LYS B  95      48.918  22.307  45.099  1.00  0.00           H  \\nATOM   1170 2HD  LYS B  95      50.218  23.428  45.537  1.00  0.00           H  \\nATOM   1171 1HE  LYS B  95      51.227  21.442  44.635  1.00  0.00           H  \\nATOM   1172 2HE  LYS B  95      51.528  21.434  46.383  1.00  0.00           H  \\nATOM   1173 1HZ  LYS B  95      49.689  19.870  46.623  1.00  0.00           H  \\nATOM   1174 2HZ  LYS B  95      49.370  19.829  44.988  1.00  0.00           H  \\nATOM   1175 3HZ  LYS B  95      50.814  19.264  45.596  1.00  0.00           H  \\nATOM   1176  N   GLU B  96      48.092  25.318  50.064  1.00  0.00           N  \\nATOM   1177  CA  GLU B  96      48.741  25.843  51.260  1.00  0.00           C  \\nATOM   1178  C   GLU B  96      47.933  25.548  52.517  1.00  0.00           C  \\nATOM   1179  O   GLU B  96      48.494  25.190  53.553  1.00  0.00           O  \\nATOM   1180  CB  GLU B  96      48.990  27.358  51.176  1.00  0.00           C  \\nATOM   1181  CG  GLU B  96      50.068  27.790  50.175  1.00  0.00           C  \\nATOM   1182  CD  GLU B  96      50.343  29.297  50.189  1.00  0.00           C  \\nATOM   1183  OE1 GLU B  96      49.669  30.011  50.902  1.00  0.00           O  \\nATOM   1184  OE2 GLU B  96      51.242  29.722  49.496  1.00  0.00           O  \\nATOM   1185  H   GLU B  96      47.772  25.925  49.545  1.00  0.00           H  \\nATOM   1186  HA  GLU B  96      49.610  25.421  51.357  1.00  0.00           H  \\nATOM   1187 1HB  GLU B  96      48.060  27.855  50.890  1.00  0.00           H  \\nATOM   1188 2HB  GLU B  96      49.271  27.734  52.159  1.00  0.00           H  \\nATOM   1189 1HG  GLU B  96      50.991  27.261  50.406  1.00  0.00           H  \\nATOM   1190 2HG  GLU B  96      49.754  27.492  49.177  1.00  0.00           H  \\nATOM   1191  N   MET B  97      46.617  25.711  52.425  1.00  0.00           N  \\nATOM   1192  CA  MET B  97      45.735  25.451  53.557  1.00  0.00           C  \\nATOM   1193  C   MET B  97      45.838  23.999  54.010  1.00  0.00           C  \\nATOM   1194  O   MET B  97      45.679  23.697  55.192  1.00  0.00           O  \\nATOM   1195  CB  MET B  97      44.283  25.739  53.197  1.00  0.00           C  \\nATOM   1196  CG  MET B  97      43.919  27.161  52.987  1.00  0.00           C  \\nATOM   1197  SD  MET B  97      44.065  28.158  54.457  1.00  0.00           S  \\nATOM   1198  CE  MET B  97      43.726  29.767  53.744  1.00  0.00           C  \\nATOM   1199  H   MET B  97      46.208  25.973  51.715  1.00  0.00           H  \\nATOM   1200  HA  MET B  97      45.994  26.027  54.293  1.00  0.00           H  \\nATOM   1201 1HB  MET B  97      44.012  25.183  52.298  1.00  0.00           H  \\nATOM   1202 2HB  MET B  97      43.656  25.402  53.988  1.00  0.00           H  \\nATOM   1203 1HG  MET B  97      44.534  27.601  52.227  1.00  0.00           H  \\nATOM   1204 2HG  MET B  97      42.902  27.186  52.658  1.00  0.00           H  \\nATOM   1205 1HE  MET B  97      43.771  30.529  54.522  1.00  0.00           H  \\nATOM   1206 2HE  MET B  97      44.466  29.989  52.971  1.00  0.00           H  \\nATOM   1207 3HE  MET B  97      42.732  29.769  53.292  1.00  0.00           H  \\nATOM   1208  N   LEU B  98      46.110  23.103  53.067  1.00  0.00           N  \\nATOM   1209  CA  LEU B  98      46.230  21.685  53.382  1.00  0.00           C  \\nATOM   1210  C   LEU B  98      47.504  21.417  54.178  1.00  0.00           C  \\nATOM   1211  O   LEU B  98      47.492  20.651  55.141  1.00  0.00           O  \\nATOM   1212  CB  LEU B  98      46.223  20.840  52.104  1.00  0.00           C  \\nATOM   1213  CG  LEU B  98      46.201  19.316  52.332  1.00  0.00           C  \\nATOM   1214  CD1 LEU B  98      44.911  18.956  53.106  1.00  0.00           C  \\nATOM   1215  CD2 LEU B  98      46.266  18.598  51.003  1.00  0.00           C  \\nATOM   1216  H   LEU B  98      46.229  23.291  52.236  1.00  0.00           H  \\nATOM   1217  HA  LEU B  98      45.473  21.416  53.925  1.00  0.00           H  \\nATOM   1218 1HB  LEU B  98      45.350  21.113  51.508  1.00  0.00           H  \\nATOM   1219 2HB  LEU B  98      47.118  21.082  51.524  1.00  0.00           H  \\nATOM   1220  HG  LEU B  98      47.059  19.026  52.941  1.00  0.00           H  \\nATOM   1221 1HD1 LEU B  98      44.871  17.890  53.296  1.00  0.00           H  \\nATOM   1222 2HD1 LEU B  98      44.901  19.491  54.053  1.00  0.00           H  \\nATOM   1223 3HD1 LEU B  98      44.040  19.246  52.516  1.00  0.00           H  \\nATOM   1224 1HD2 LEU B  98      46.252  17.524  51.159  1.00  0.00           H  \\nATOM   1225 2HD2 LEU B  98      45.419  18.887  50.412  1.00  0.00           H  \\nATOM   1226 3HD2 LEU B  98      47.184  18.876  50.486  1.00  0.00           H  \\nATOM   1227  N   GLU B  99      48.602  22.049  53.771  1.00  0.00           N  \\nATOM   1228  CA  GLU B  99      49.869  21.910  54.481  1.00  0.00           C  \\nATOM   1229  C   GLU B  99      49.745  22.430  55.910  1.00  0.00           C  \\nATOM   1230  O   GLU B  99      50.341  21.875  56.833  1.00  0.00           O  \\nATOM   1231  CB  GLU B  99      50.970  22.688  53.745  1.00  0.00           C  \\nATOM   1232  CG  GLU B  99      51.377  22.122  52.367  1.00  0.00           C  \\nATOM   1233  CD  GLU B  99      52.105  20.807  52.412  1.00  0.00           C  \\nATOM   1234  OE1 GLU B  99      53.092  20.723  53.103  1.00  0.00           O  \\nATOM   1235  OE2 GLU B  99      51.680  19.888  51.747  1.00  0.00           O  \\nATOM   1236  H   GLU B  99      48.639  22.565  53.084  1.00  0.00           H  \\nATOM   1237  HA  GLU B  99      50.108  20.971  54.522  1.00  0.00           H  \\nATOM   1238 1HB  GLU B  99      50.641  23.715  53.591  1.00  0.00           H  \\nATOM   1239 2HB  GLU B  99      51.862  22.724  54.368  1.00  0.00           H  \\nATOM   1240 1HG  GLU B  99      50.472  21.991  51.777  1.00  0.00           H  \\nATOM   1241 2HG  GLU B  99      51.998  22.859  51.858  1.00  0.00           H  \\nATOM   1242  N   ARG B 100      48.969  23.496  56.087  1.00  0.00           N  \\nATOM   1243  CA  ARG B 100      48.696  24.029  57.418  1.00  0.00           C  \\nATOM   1244  C   ARG B 100      47.926  23.017  58.257  1.00  0.00           C  \\nATOM   1245  O   ARG B 100      47.992  23.038  59.486  1.00  0.00           O  \\nATOM   1246  CB  ARG B 100      47.878  25.313  57.334  1.00  0.00           C  \\nATOM   1247  CG  ARG B 100      48.604  26.529  56.770  1.00  0.00           C  \\nATOM   1248  CD  ARG B 100      47.705  27.726  56.708  1.00  0.00           C  \\nATOM   1249  NE  ARG B 100      48.355  28.874  56.085  1.00  0.00           N  \\nATOM   1250  CZ  ARG B 100      49.140  29.760  56.733  1.00  0.00           C  \\nATOM   1251  NH1 ARG B 100      49.363  29.627  58.022  1.00  0.00           N  \\nATOM   1252  NH2 ARG B 100      49.686  30.765  56.070  1.00  0.00           N  \\nATOM   1253  H   ARG B 100      48.587  23.930  55.450  1.00  0.00           H  \\nATOM   1254  HA  ARG B 100      49.537  24.216  57.865  1.00  0.00           H  \\nATOM   1255 1HB  ARG B 100      47.006  25.145  56.716  1.00  0.00           H  \\nATOM   1256 2HB  ARG B 100      47.520  25.576  58.328  1.00  0.00           H  \\nATOM   1257 1HG  ARG B 100      49.456  26.768  57.402  1.00  0.00           H  \\nATOM   1258 2HG  ARG B 100      48.949  26.307  55.763  1.00  0.00           H  \\nATOM   1259 1HD  ARG B 100      46.821  27.478  56.123  1.00  0.00           H  \\nATOM   1260 2HD  ARG B 100      47.404  28.007  57.714  1.00  0.00           H  \\nATOM   1261  HE  ARG B 100      48.211  29.012  55.092  1.00  0.00           H  \\nATOM   1262 1HH1 ARG B 100      48.943  28.862  58.531  1.00  0.00           H  \\nATOM   1263 2HH1 ARG B 100      49.949  30.293  58.504  1.00  0.00           H  \\nATOM   1264 1HH2 ARG B 100      49.515  30.869  55.077  1.00  0.00           H  \\nATOM   1265 2HH2 ARG B 100      50.273  31.429  56.553  1.00  0.00           H  \\nATOM   1266  N   LEU B 101      47.192  22.135  57.586  1.00  0.00           N  \\nATOM   1267  CA  LEU B 101      46.382  21.138  58.272  1.00  0.00           C  \\nATOM   1268  C   LEU B 101      47.231  19.941  58.697  1.00  0.00           C  \\nATOM   1269  O   LEU B 101      46.868  19.214  59.620  1.00  0.00           O  \\nATOM   1270  CB  LEU B 101      45.239  20.654  57.344  1.00  0.00           C  \\nATOM   1271  CG  LEU B 101      44.181  19.715  57.940  1.00  0.00           C  \\nATOM   1272  CD1 LEU B 101      43.459  20.385  59.056  1.00  0.00           C  \\nATOM   1273  CD2 LEU B 101      43.203  19.303  56.839  1.00  0.00           C  \\nATOM   1274  H   LEU B 101      47.146  22.094  56.728  1.00  0.00           H  \\nATOM   1275  HA  LEU B 101      46.003  21.535  59.072  1.00  0.00           H  \\nATOM   1276 1HB  LEU B 101      44.734  21.528  56.930  1.00  0.00           H  \\nATOM   1277 2HB  LEU B 101      45.677  20.106  56.530  1.00  0.00           H  \\nATOM   1278  HG  LEU B 101      44.684  18.839  58.348  1.00  0.00           H  \\nATOM   1279 1HD1 LEU B 101      42.738  19.704  59.488  1.00  0.00           H  \\nATOM   1280 2HD1 LEU B 101      44.174  20.688  59.809  1.00  0.00           H  \\nATOM   1281 3HD1 LEU B 101      42.950  21.243  58.662  1.00  0.00           H  \\nATOM   1282 1HD2 LEU B 101      42.470  18.635  57.245  1.00  0.00           H  \\nATOM   1283 2HD2 LEU B 101      42.710  20.172  56.443  1.00  0.00           H  \\nATOM   1284 3HD2 LEU B 101      43.744  18.802  56.041  1.00  0.00           H  \\nATOM   1285  N   GLU B 102      48.357  19.746  58.019  1.00  0.00           N  \\nATOM   1286  CA  GLU B 102      49.255  18.639  58.327  1.00  0.00           C  \\nATOM   1287  C   GLU B 102      50.165  18.992  59.498  1.00  0.00           C  \\nATOM   1288  O   GLU B 102      50.822  18.122  60.070  1.00  0.00           O  \\nATOM   1289  CB  GLU B 102      50.067  18.220  57.098  1.00  0.00           C  \\nATOM   1290  CG  GLU B 102      50.811  16.886  57.301  1.00  0.00           C  \\nATOM   1291  CD  GLU B 102      51.425  16.308  56.043  1.00  0.00           C  \\nATOM   1292  OE1 GLU B 102      52.265  16.934  55.457  1.00  0.00           O  \\nATOM   1293  OE2 GLU B 102      51.016  15.223  55.668  1.00  0.00           O  \\nATOM   1294  H   GLU B 102      48.626  20.244  57.372  1.00  0.00           H  \\nATOM   1295  HA  GLU B 102      48.725  17.853  58.575  1.00  0.00           H  \\nATOM   1296 1HB  GLU B 102      49.404  18.125  56.236  1.00  0.00           H  \\nATOM   1297 2HB  GLU B 102      50.800  18.997  56.864  1.00  0.00           H  \\nATOM   1298 1HG  GLU B 102      51.603  17.043  58.033  1.00  0.00           H  \\nATOM   1299 2HG  GLU B 102      50.113  16.163  57.719  1.00  0.00           H  \\nATOM   1300  N   LYS B 103      50.128  20.274  59.813  1.00  0.00           N  \\nATOM   1301  CA  LYS B 103      50.838  20.829  60.920  1.00  0.00           C  \\nATOM   1302  C   LYS B 103      49.996  20.642  62.151  1.00  0.00           C  \\nATOM   1303  O   LYS B 103      50.320  19.842  63.010  1.00  0.00           O  \\nATOM   1304  CB  LYS B 103      51.182  22.299  60.697  1.00  0.00           C  \\nATOM   1305  CG  LYS B 103      52.255  22.523  59.640  1.00  0.00           C  \\nATOM   1306  CD  LYS B 103      52.559  23.997  59.442  1.00  0.00           C  \\nATOM   1307  CE  LYS B 103      53.617  24.195  58.362  1.00  0.00           C  \\nATOM   1308  NZ  LYS B 103      53.924  25.636  58.135  1.00  0.00           N  \\nATOM   1309  H   LYS B 103      49.659  20.843  59.375  1.00  0.00           H  \\nATOM   1310  HA  LYS B 103      51.691  20.367  61.033  1.00  0.00           H  \\nATOM   1311 1HB  LYS B 103      50.296  22.841  60.389  1.00  0.00           H  \\nATOM   1312 2HB  LYS B 103      51.522  22.739  61.633  1.00  0.00           H  \\nATOM   1313 1HG  LYS B 103      53.167  22.010  59.940  1.00  0.00           H  \\nATOM   1314 2HG  LYS B 103      51.923  22.101  58.699  1.00  0.00           H  \\nATOM   1315 1HD  LYS B 103      51.647  24.516  59.150  1.00  0.00           H  \\nATOM   1316 2HD  LYS B 103      52.916  24.424  60.378  1.00  0.00           H  \\nATOM   1317 1HE  LYS B 103      54.529  23.682  58.661  1.00  0.00           H  \\nATOM   1318 2HE  LYS B 103      53.254  23.758  57.428  1.00  0.00           H  \\nATOM   1319 1HZ  LYS B 103      54.626  25.722  57.414  1.00  0.00           H  \\nATOM   1320 2HZ  LYS B 103      53.085  26.117  57.846  1.00  0.00           H  \\nATOM   1321 3HZ  LYS B 103      54.270  26.047  58.991  1.00  0.00           H  \\nATOM   1322  N   ASN B 104      48.907  21.385  62.235  1.00  0.00           N  \\nATOM   1323  CA  ASN B 104      47.981  21.227  63.340  1.00  0.00           C  \\nATOM   1324  C   ASN B 104      46.570  21.109  62.884  1.00  0.00           C  \\nATOM   1325  O   ASN B 104      45.897  22.086  62.667  1.00  0.00           O  \\nATOM   1326  CB  ASN B 104      48.183  22.397  64.294  1.00  0.00           C  \\nATOM   1327  CG  ASN B 104      48.063  23.763  63.584  1.00  0.00           C  \\nATOM   1328  OD1 ASN B 104      48.875  24.010  62.680  1.00  0.00           O  \\nATOM   1329  ND2 ASN B 104      47.155  24.629  63.953  1.00  0.00           N  \\nATOM   1330  H   ASN B 104      48.687  21.991  61.669  1.00  0.00           H  \\nATOM   1331  HA  ASN B 104      48.203  20.411  63.832  1.00  0.00           H  \\nATOM   1332 1HB  ASN B 104      47.424  22.350  65.085  1.00  0.00           H  \\nATOM   1333 2HB  ASN B 104      49.160  22.323  64.767  1.00  0.00           H  \\nATOM   1334 1HD2 ASN B 104      46.481  24.437  64.692  1.00  0.00           H  \\nATOM   1335 2HD2 ASN B 104      47.095  25.508  63.489  1.00  0.00           H  \\nATOM   1336  N   PRO B 105      46.131  19.797  62.714  1.00  0.00           N  \\nATOM   1337  CA  PRO B 105      44.671  19.680  62.728  1.00  0.00           C  \\nATOM   1338  C   PRO B 105      44.019  20.367  63.924  1.00  0.00           C  \\nATOM   1339  O   PRO B 105      44.379  20.070  65.061  1.00  0.00           O  \\nATOM   1340  CB  PRO B 105      44.524  18.152  62.816  1.00  0.00           C  \\nATOM   1341  CG  PRO B 105      45.803  17.608  62.120  1.00  0.00           C  \\nATOM   1342  CD  PRO B 105      46.925  18.589  62.577  1.00  0.00           C  \\nATOM   1343  HA  PRO B 105      44.288  20.015  61.896  1.00  0.00           H  \\nATOM   1344 1HB  PRO B 105      44.449  17.843  63.865  1.00  0.00           H  \\nATOM   1345 2HB  PRO B 105      43.593  17.838  62.324  1.00  0.00           H  \\nATOM   1346 1HG  PRO B 105      45.988  16.559  62.411  1.00  0.00           H  \\nATOM   1347 2HG  PRO B 105      45.661  17.605  61.029  1.00  0.00           H  \\nATOM   1348 1HD  PRO B 105      47.336  18.278  63.548  1.00  0.00           H  \\nATOM   1349 2HD  PRO B 105      47.711  18.705  61.820  1.00  0.00           H  \\nATOM   1350  N   ASP B 106      43.074  21.257  63.660  1.00  0.00           N  \\nATOM   1351  CA  ASP B 106      42.354  21.987  64.688  1.00  0.00           C  \\nATOM   1352  C   ASP B 106      40.849  21.888  64.434  1.00  0.00           C  \\nATOM   1353  O   ASP B 106      40.373  20.900  63.925  1.00  0.00           O  \\nATOM   1354  CB  ASP B 106      42.802  23.450  64.822  1.00  0.00           C  \\nATOM   1355  CG  ASP B 106      44.215  23.663  65.447  1.00  0.00           C  \\nATOM   1356  OD1 ASP B 106      44.735  22.781  66.069  1.00  0.00           O  \\nATOM   1357  OD2 ASP B 106      44.749  24.726  65.280  1.00  0.00           O  \\nATOM   1358  H   ASP B 106      42.821  21.459  62.868  1.00  0.00           H  \\nATOM   1359  HA  ASP B 106      42.550  21.604  65.566  1.00  0.00           H  \\nATOM   1360 1HB  ASP B 106      42.749  23.910  63.904  1.00  0.00           H  \\nATOM   1361 2HB  ASP B 106      42.127  23.978  65.474  1.00  0.00           H  \\nATOM   1362  N   LYS B 107      40.098  22.920  64.791  1.00  0.00           N  \\nATOM   1363  CA  LYS B 107      38.690  22.982  64.474  1.00  0.00           C  \\nATOM   1364  C   LYS B 107      38.529  24.141  63.561  1.00  0.00           C  \\nATOM   1365  O   LYS B 107      37.928  24.036  62.513  1.00  0.00           O  \\nATOM   1366  CB  LYS B 107      37.799  23.133  65.707  1.00  0.00           C  \\nATOM   1367  CG  LYS B 107      36.288  23.083  65.364  1.00  0.00           C  \\nATOM   1368  CD  LYS B 107      35.397  23.179  66.598  1.00  0.00           C  \\nATOM   1369  CE  LYS B 107      35.427  21.897  67.410  1.00  0.00           C  \\nATOM   1370  NZ  LYS B 107      34.483  21.947  68.560  1.00  0.00           N  \\nATOM   1371  H   LYS B 107      40.373  23.604  65.225  1.00  0.00           H  \\nATOM   1372  HA  LYS B 107      38.402  22.166  64.024  1.00  0.00           H  \\nATOM   1373 1HB  LYS B 107      38.017  22.340  66.421  1.00  0.00           H  \\nATOM   1374 2HB  LYS B 107      38.001  24.089  66.194  1.00  0.00           H  \\nATOM   1375 1HG  LYS B 107      36.047  23.916  64.699  1.00  0.00           H  \\nATOM   1376 2HG  LYS B 107      36.062  22.153  64.840  1.00  0.00           H  \\nATOM   1377 1HD  LYS B 107      35.712  24.019  67.218  1.00  0.00           H  \\nATOM   1378 2HD  LYS B 107      34.369  23.346  66.279  1.00  0.00           H  \\nATOM   1379 1HE  LYS B 107      35.151  21.063  66.764  1.00  0.00           H  \\nATOM   1380 2HE  LYS B 107      36.436  21.735  67.787  1.00  0.00           H  \\nATOM   1381 1HZ  LYS B 107      33.538  22.087  68.227  1.00  0.00           H  \\nATOM   1382 2HZ  LYS B 107      34.532  21.079  69.070  1.00  0.00           H  \\nATOM   1383 3HZ  LYS B 107      34.737  22.710  69.168  1.00  0.00           H  \\nATOM   1384  N   ASP B 108      39.070  25.270  63.968  1.00  0.00           N  \\nATOM   1385  CA  ASP B 108      38.957  26.410  63.067  1.00  0.00           C  \\nATOM   1386  C   ASP B 108      39.748  26.165  61.787  1.00  0.00           C  \\nATOM   1387  O   ASP B 108      39.370  26.633  60.713  1.00  0.00           O  \\nATOM   1388  CB  ASP B 108      39.470  27.680  63.742  1.00  0.00           C  \\nATOM   1389  CG  ASP B 108      38.549  28.176  64.851  1.00  0.00           C  \\nATOM   1390  OD1 ASP B 108      37.447  27.687  64.948  1.00  0.00           O  \\nATOM   1391  OD2 ASP B 108      38.959  29.031  65.595  1.00  0.00           O  \\nATOM   1392  H   ASP B 108      39.499  25.438  64.694  1.00  0.00           H  \\nATOM   1393  HA  ASP B 108      38.016  26.540  62.826  1.00  0.00           H  \\nATOM   1394 1HB  ASP B 108      40.460  27.493  64.160  1.00  0.00           H  \\nATOM   1395 2HB  ASP B 108      39.578  28.469  62.992  1.00  0.00           H  \\nATOM   1396  N   VAL B 109      40.848  25.430  61.909  1.00  0.00           N  \\nATOM   1397  CA  VAL B 109      41.693  25.120  60.763  1.00  0.00           C  \\nATOM   1398  C   VAL B 109      40.925  24.300  59.732  1.00  0.00           C  \\nATOM   1399  O   VAL B 109      40.909  24.634  58.547  1.00  0.00           O  \\nATOM   1400  CB  VAL B 109      42.987  24.378  61.145  1.00  0.00           C  \\nATOM   1401  CG1 VAL B 109      43.711  23.909  59.897  1.00  0.00           C  \\nATOM   1402  CG2 VAL B 109      43.908  25.320  61.906  1.00  0.00           C  \\nATOM   1403  H   VAL B 109      41.128  25.096  62.650  1.00  0.00           H  \\nATOM   1404  HA  VAL B 109      41.982  25.955  60.339  1.00  0.00           H  \\nATOM   1405  HB  VAL B 109      42.736  23.507  61.730  1.00  0.00           H  \\nATOM   1406 1HG1 VAL B 109      44.624  23.373  60.181  1.00  0.00           H  \\nATOM   1407 2HG1 VAL B 109      43.067  23.267  59.343  1.00  0.00           H  \\nATOM   1408 3HG1 VAL B 109      43.974  24.768  59.279  1.00  0.00           H  \\nATOM   1409 1HG2 VAL B 109      44.826  24.786  62.175  1.00  0.00           H  \\nATOM   1410 2HG2 VAL B 109      44.154  26.170  61.270  1.00  0.00           H  \\nATOM   1411 3HG2 VAL B 109      43.431  25.686  62.808  1.00  0.00           H  \\nATOM   1412  N   ILE B 110      40.291  23.227  60.191  1.00  0.00           N  \\nATOM   1413  CA  ILE B 110      39.518  22.359  59.310  1.00  0.00           C  \\nATOM   1414  C   ILE B 110      38.450  23.156  58.570  1.00  0.00           C  \\nATOM   1415  O   ILE B 110      38.302  23.034  57.354  1.00  0.00           O  \\nATOM   1416  CB  ILE B 110      38.864  21.155  60.019  1.00  0.00           C  \\nATOM   1417  CG1 ILE B 110      39.952  20.148  60.448  1.00  0.00           C  \\nATOM   1418  CG2 ILE B 110      37.817  20.508  59.103  1.00  0.00           C  \\nATOM   1419  CD1 ILE B 110      39.460  19.052  61.367  1.00  0.00           C  \\nATOM   1420  H   ILE B 110      40.292  22.978  61.014  1.00  0.00           H  \\nATOM   1421  HA  ILE B 110      40.117  21.956  58.647  1.00  0.00           H  \\nATOM   1422  HB  ILE B 110      38.372  21.501  60.934  1.00  0.00           H  \\nATOM   1423 1HG1 ILE B 110      40.366  19.683  59.556  1.00  0.00           H  \\nATOM   1424 2HG1 ILE B 110      40.746  20.693  60.964  1.00  0.00           H  \\nATOM   1425 1HG2 ILE B 110      37.356  19.666  59.611  1.00  0.00           H  \\nATOM   1426 2HG2 ILE B 110      37.052  21.238  58.847  1.00  0.00           H  \\nATOM   1427 3HG2 ILE B 110      38.295  20.170  58.210  1.00  0.00           H  \\nATOM   1428 1HD1 ILE B 110      39.057  19.496  62.279  1.00  0.00           H  \\nATOM   1429 2HD1 ILE B 110      38.680  18.481  60.869  1.00  0.00           H  \\nATOM   1430 3HD1 ILE B 110      40.288  18.392  61.622  1.00  0.00           H  \\nATOM   1431  N   VAL B 111      37.709  23.973  59.311  1.00  0.00           N  \\nATOM   1432  CA  VAL B 111      36.656  24.794  58.726  1.00  0.00           C  \\nATOM   1433  C   VAL B 111      37.227  25.723  57.661  1.00  0.00           C  \\nATOM   1434  O   VAL B 111      36.581  25.993  56.648  1.00  0.00           O  \\nATOM   1435  CB  VAL B 111      35.935  25.583  59.817  1.00  0.00           C  \\nATOM   1436  CG1 VAL B 111      34.989  26.595  59.190  1.00  0.00           C  \\nATOM   1437  CG2 VAL B 111      35.171  24.597  60.705  1.00  0.00           C  \\nATOM   1438  H   VAL B 111      37.797  24.070  60.161  1.00  0.00           H  \\nATOM   1439  HA  VAL B 111      35.990  24.212  58.303  1.00  0.00           H  \\nATOM   1440  HB  VAL B 111      36.670  26.127  60.417  1.00  0.00           H  \\nATOM   1441 1HG1 VAL B 111      34.483  27.153  59.976  1.00  0.00           H  \\nATOM   1442 2HG1 VAL B 111      35.566  27.278  58.569  1.00  0.00           H  \\nATOM   1443 3HG1 VAL B 111      34.257  26.086  58.582  1.00  0.00           H  \\nATOM   1444 1HG2 VAL B 111      34.672  25.149  61.490  1.00  0.00           H  \\nATOM   1445 2HG2 VAL B 111      34.443  24.060  60.114  1.00  0.00           H  \\nATOM   1446 3HG2 VAL B 111      35.866  23.889  61.146  1.00  0.00           H  \\nATOM   1447  N   GLU B 112      38.441  26.209  57.897  1.00  0.00           N  \\nATOM   1448  CA  GLU B 112      39.104  27.107  56.957  1.00  0.00           C  \\nATOM   1449  C   GLU B 112      39.414  26.401  55.636  1.00  0.00           C  \\nATOM   1450  O   GLU B 112      39.383  27.024  54.576  1.00  0.00           O  \\nATOM   1451  CB  GLU B 112      40.375  27.771  57.519  1.00  0.00           C  \\nATOM   1452  CG  GLU B 112      40.118  28.885  58.553  1.00  0.00           C  \\nATOM   1453  CD  GLU B 112      41.387  29.472  59.159  1.00  0.00           C  \\nATOM   1454  OE1 GLU B 112      42.454  28.971  58.890  1.00  0.00           O  \\nATOM   1455  OE2 GLU B 112      41.277  30.432  59.889  1.00  0.00           O  \\nATOM   1456  H   GLU B 112      38.906  26.033  58.599  1.00  0.00           H  \\nATOM   1457  HA  GLU B 112      38.514  27.852  56.765  1.00  0.00           H  \\nATOM   1458 1HB  GLU B 112      41.000  27.027  57.992  1.00  0.00           H  \\nATOM   1459 2HB  GLU B 112      40.951  28.199  56.698  1.00  0.00           H  \\nATOM   1460 1HG  GLU B 112      39.592  29.688  58.040  1.00  0.00           H  \\nATOM   1461 2HG  GLU B 112      39.467  28.533  59.329  1.00  0.00           H  \\nATOM   1462  N   VAL B 113      39.706  25.104  55.700  1.00  0.00           N  \\nATOM   1463  CA  VAL B 113      40.001  24.326  54.498  1.00  0.00           C  \\nATOM   1464  C   VAL B 113      38.723  24.029  53.715  1.00  0.00           C  \\nATOM   1465  O   VAL B 113      38.716  24.089  52.487  1.00  0.00           O  \\nATOM   1466  CB  VAL B 113      40.678  22.975  54.877  1.00  0.00           C  \\nATOM   1467  CG1 VAL B 113      40.863  22.102  53.633  1.00  0.00           C  \\nATOM   1468  CG2 VAL B 113      42.025  23.223  55.557  1.00  0.00           C  \\nATOM   1469  H   VAL B 113      39.740  24.650  56.429  1.00  0.00           H  \\nATOM   1470  HA  VAL B 113      40.590  24.842  53.925  1.00  0.00           H  \\nATOM   1471  HB  VAL B 113      40.022  22.436  55.564  1.00  0.00           H  \\nATOM   1472 1HG1 VAL B 113      41.314  21.150  53.916  1.00  0.00           H  \\nATOM   1473 2HG1 VAL B 113      39.892  21.925  53.183  1.00  0.00           H  \\nATOM   1474 3HG1 VAL B 113      41.511  22.612  52.918  1.00  0.00           H  \\nATOM   1475 1HG2 VAL B 113      42.477  22.289  55.832  1.00  0.00           H  \\nATOM   1476 2HG2 VAL B 113      42.671  23.732  54.884  1.00  0.00           H  \\nATOM   1477 3HG2 VAL B 113      41.880  23.824  56.447  1.00  0.00           H  \\nATOM   1478  N   LEU B 114      37.645  23.709  54.426  1.00  0.00           N  \\nATOM   1479  CA  LEU B 114      36.368  23.418  53.782  1.00  0.00           C  \\nATOM   1480  C   LEU B 114      35.861  24.630  53.009  1.00  0.00           C  \\nATOM   1481  O   LEU B 114      35.326  24.495  51.910  1.00  0.00           O  \\nATOM   1482  CB  LEU B 114      35.355  23.012  54.860  1.00  0.00           C  \\nATOM   1483  CG  LEU B 114      35.626  21.655  55.540  1.00  0.00           C  \\nATOM   1484  CD1 LEU B 114      34.728  21.496  56.740  1.00  0.00           C  \\nATOM   1485  CD2 LEU B 114      35.364  20.536  54.554  1.00  0.00           C  \\nATOM   1486  H   LEU B 114      37.627  23.654  55.284  1.00  0.00           H  \\nATOM   1487  HA  LEU B 114      36.490  22.690  53.152  1.00  0.00           H  \\nATOM   1488 1HB  LEU B 114      35.354  23.778  55.638  1.00  0.00           H  \\nATOM   1489 2HB  LEU B 114      34.362  22.976  54.413  1.00  0.00           H  \\nATOM   1490  HG  LEU B 114      36.666  21.616  55.878  1.00  0.00           H  \\nATOM   1491 1HD1 LEU B 114      34.941  20.555  57.204  1.00  0.00           H  \\nATOM   1492 2HD1 LEU B 114      34.932  22.290  57.442  1.00  0.00           H  \\nATOM   1493 3HD1 LEU B 114      33.684  21.530  56.434  1.00  0.00           H  \\nATOM   1494 1HD2 LEU B 114      35.550  19.586  55.043  1.00  0.00           H  \\nATOM   1495 2HD2 LEU B 114      34.329  20.573  54.228  1.00  0.00           H  \\nATOM   1496 3HD2 LEU B 114      36.019  20.643  53.693  1.00  0.00           H  \\nATOM   1497  N   LYS B 115      36.038  25.814  53.586  1.00  0.00           N  \\nATOM   1498  CA  LYS B 115      35.582  27.043  52.948  1.00  0.00           C  \\nATOM   1499  C   LYS B 115      36.336  27.303  51.649  1.00  0.00           C  \\nATOM   1500  O   LYS B 115      35.747  27.749  50.672  1.00  0.00           O  \\nATOM   1501  CB  LYS B 115      35.724  28.213  53.929  1.00  0.00           C  \\nATOM   1502  CG  LYS B 115      34.692  28.188  55.101  1.00  0.00           C  \\nATOM   1503  CD  LYS B 115      34.997  29.256  56.163  1.00  0.00           C  \\nATOM   1504  CE  LYS B 115      34.562  30.663  55.750  1.00  0.00           C  \\nATOM   1505  NZ  LYS B 115      33.074  30.847  55.848  1.00  0.00           N  \\nATOM   1506  H   LYS B 115      36.420  25.933  54.347  1.00  0.00           H  \\nATOM   1507  HA  LYS B 115      34.641  26.954  52.733  1.00  0.00           H  \\nATOM   1508 1HB  LYS B 115      36.727  28.189  54.370  1.00  0.00           H  \\nATOM   1509 2HB  LYS B 115      35.620  29.154  53.396  1.00  0.00           H  \\nATOM   1510 1HG  LYS B 115      33.695  28.370  54.706  1.00  0.00           H  \\nATOM   1511 2HG  LYS B 115      34.697  27.207  55.570  1.00  0.00           H  \\nATOM   1512 1HD  LYS B 115      34.481  28.990  57.085  1.00  0.00           H  \\nATOM   1513 2HD  LYS B 115      36.070  29.257  56.363  1.00  0.00           H  \\nATOM   1514 1HE  LYS B 115      35.050  31.389  56.399  1.00  0.00           H  \\nATOM   1515 2HE  LYS B 115      34.873  30.844  54.722  1.00  0.00           H  \\nATOM   1516 1HZ  LYS B 115      32.601  30.188  55.246  1.00  0.00           H  \\nATOM   1517 2HZ  LYS B 115      32.782  30.692  56.808  1.00  0.00           H  \\nATOM   1518 3HZ  LYS B 115      32.827  31.788  55.575  1.00  0.00           H  \\nATOM   1519  N   VAL B 116      37.633  27.015  51.639  1.00  0.00           N  \\nATOM   1520  CA  VAL B 116      38.446  27.193  50.440  1.00  0.00           C  \\nATOM   1521  C   VAL B 116      38.081  26.141  49.394  1.00  0.00           C  \\nATOM   1522  O   VAL B 116      38.090  26.420  48.193  1.00  0.00           O  \\nATOM   1523  CB  VAL B 116      39.952  27.178  50.719  1.00  0.00           C  \\nATOM   1524  CG1 VAL B 116      40.706  27.238  49.387  1.00  0.00           C  \\nATOM   1525  CG2 VAL B 116      40.316  28.386  51.557  1.00  0.00           C  \\nATOM   1526  H   VAL B 116      38.069  26.712  52.316  1.00  0.00           H  \\nATOM   1527  HA  VAL B 116      38.265  28.068  50.062  1.00  0.00           H  \\nATOM   1528  HB  VAL B 116      40.225  26.253  51.236  1.00  0.00           H  \\nATOM   1529 1HG1 VAL B 116      41.772  27.235  49.576  1.00  0.00           H  \\nATOM   1530 2HG1 VAL B 116      40.433  26.390  48.777  1.00  0.00           H  \\nATOM   1531 3HG1 VAL B 116      40.441  28.155  48.860  1.00  0.00           H  \\nATOM   1532 1HG2 VAL B 116      41.371  28.404  51.738  1.00  0.00           H  \\nATOM   1533 2HG2 VAL B 116      40.040  29.295  51.021  1.00  0.00           H  \\nATOM   1534 3HG2 VAL B 116      39.783  28.350  52.503  1.00  0.00           H  \\nATOM   1535  N   ILE B 117      37.747  24.938  49.853  1.00  0.00           N  \\nATOM   1536  CA  ILE B 117      37.345  23.857  48.955  1.00  0.00           C  \\nATOM   1537  C   ILE B 117      36.060  24.222  48.215  1.00  0.00           C  \\nATOM   1538  O   ILE B 117      35.910  23.910  47.037  1.00  0.00           O  \\nATOM   1539  CB  ILE B 117      37.103  22.507  49.667  1.00  0.00           C  \\nATOM   1540  CG1 ILE B 117      38.398  21.897  50.150  1.00  0.00           C  \\nATOM   1541  CG2 ILE B 117      36.380  21.590  48.737  1.00  0.00           C  \\nATOM   1542  CD1 ILE B 117      38.218  20.731  51.089  1.00  0.00           C  \\nATOM   1543  H   ILE B 117      37.744  24.720  50.686  1.00  0.00           H  \\nATOM   1544  HA  ILE B 117      38.042  23.723  48.294  1.00  0.00           H  \\nATOM   1545  HB  ILE B 117      36.488  22.669  50.547  1.00  0.00           H  \\nATOM   1546 1HG1 ILE B 117      38.956  21.550  49.282  1.00  0.00           H  \\nATOM   1547 2HG1 ILE B 117      38.969  22.649  50.643  1.00  0.00           H  \\nATOM   1548 1HG2 ILE B 117      36.195  20.647  49.220  1.00  0.00           H  \\nATOM   1549 2HG2 ILE B 117      35.436  22.038  48.442  1.00  0.00           H  \\nATOM   1550 3HG2 ILE B 117      36.983  21.440  47.882  1.00  0.00           H  \\nATOM   1551 1HD1 ILE B 117      39.195  20.345  51.375  1.00  0.00           H  \\nATOM   1552 2HD1 ILE B 117      37.687  21.061  51.972  1.00  0.00           H  \\nATOM   1553 3HD1 ILE B 117      37.655  19.957  50.608  1.00  0.00           H  \\nATOM   1554  N   VAL B 118      35.142  24.895  48.902  1.00  0.00           N  \\nATOM   1555  CA  VAL B 118      33.877  25.292  48.293  1.00  0.00           C  \\nATOM   1556  C   VAL B 118      34.087  26.365  47.230  1.00  0.00           C  \\nATOM   1557  O   VAL B 118      33.445  26.337  46.181  1.00  0.00           O  \\nATOM   1558  CB  VAL B 118      32.853  25.731  49.350  1.00  0.00           C  \\nATOM   1559  CG1 VAL B 118      31.587  26.275  48.653  1.00  0.00           C  \\nATOM   1560  CG2 VAL B 118      32.506  24.522  50.227  1.00  0.00           C  \\nATOM   1561  H   VAL B 118      35.228  25.135  49.723  1.00  0.00           H  \\nATOM   1562  HA  VAL B 118      33.481  24.519  47.861  1.00  0.00           H  \\nATOM   1563  HB  VAL B 118      33.279  26.530  49.966  1.00  0.00           H  \\nATOM   1564 1HG1 VAL B 118      30.858  26.583  49.400  1.00  0.00           H  \\nATOM   1565 2HG1 VAL B 118      31.856  27.130  48.031  1.00  0.00           H  \\nATOM   1566 3HG1 VAL B 118      31.151  25.493  48.029  1.00  0.00           H  \\nATOM   1567 1HG2 VAL B 118      31.780  24.799  50.982  1.00  0.00           H  \\nATOM   1568 2HG2 VAL B 118      32.106  23.756  49.614  1.00  0.00           H  \\nATOM   1569 3HG2 VAL B 118      33.383  24.150  50.701  1.00  0.00           H  \\nATOM   1570  N   LYS B 119      34.984  27.309  47.498  1.00  0.00           N  \\nATOM   1571  CA  LYS B 119      35.297  28.345  46.522  1.00  0.00           C  \\nATOM   1572  C   LYS B 119      35.963  27.742  45.296  1.00  0.00           C  \\nATOM   1573  O   LYS B 119      35.760  28.216  44.182  1.00  0.00           O  \\nATOM   1574  CB  LYS B 119      36.249  29.394  47.105  1.00  0.00           C  \\nATOM   1575  CG  LYS B 119      35.675  30.300  48.188  1.00  0.00           C  \\nATOM   1576  CD  LYS B 119      34.593  31.220  47.619  1.00  0.00           C  \\nATOM   1577  CE  LYS B 119      34.237  32.361  48.584  1.00  0.00           C  \\nATOM   1578  NZ  LYS B 119      33.685  31.871  49.872  1.00  0.00           N  \\nATOM   1579  H   LYS B 119      35.423  27.371  48.234  1.00  0.00           H  \\nATOM   1580  HA  LYS B 119      34.474  28.772  46.238  1.00  0.00           H  \\nATOM   1581 1HB  LYS B 119      37.117  28.883  47.534  1.00  0.00           H  \\nATOM   1582 2HB  LYS B 119      36.617  30.029  46.302  1.00  0.00           H  \\nATOM   1583 1HG  LYS B 119      35.238  29.681  48.975  1.00  0.00           H  \\nATOM   1584 2HG  LYS B 119      36.471  30.901  48.625  1.00  0.00           H  \\nATOM   1585 1HD  LYS B 119      34.943  31.650  46.679  1.00  0.00           H  \\nATOM   1586 2HD  LYS B 119      33.693  30.635  47.412  1.00  0.00           H  \\nATOM   1587 1HE  LYS B 119      35.129  32.953  48.784  1.00  0.00           H  \\nATOM   1588 2HE  LYS B 119      33.492  32.998  48.105  1.00  0.00           H  \\nATOM   1589 1HZ  LYS B 119      33.460  32.659  50.462  1.00  0.00           H  \\nATOM   1590 2HZ  LYS B 119      32.853  31.330  49.700  1.00  0.00           H  \\nATOM   1591 3HZ  LYS B 119      34.374  31.295  50.330  1.00  0.00           H  \\nATOM   1592  N   ALA B 120      36.752  26.693  45.507  1.00  0.00           N  \\nATOM   1593  CA  ALA B 120      37.411  26.007  44.403  1.00  0.00           C  \\nATOM   1594  C   ALA B 120      36.373  25.323  43.527  1.00  0.00           C  \\nATOM   1595  O   ALA B 120      36.441  25.390  42.300  1.00  0.00           O  \\nATOM   1596  CB  ALA B 120      38.412  24.980  44.929  1.00  0.00           C  \\nATOM   1597  H   ALA B 120      36.922  26.359  46.281  1.00  0.00           H  \\nATOM   1598  HA  ALA B 120      37.886  26.656  43.861  1.00  0.00           H  \\nATOM   1599 1HB  ALA B 120      37.891  24.238  45.527  1.00  0.00           H  \\nATOM   1600 2HB  ALA B 120      38.902  24.488  44.092  1.00  0.00           H  \\nATOM   1601 3HB  ALA B 120      39.155  25.475  45.547  1.00  0.00           H  \\nATOM   1602  N   ILE B 121      35.410  24.664  44.165  1.00  0.00           N  \\nATOM   1603  CA  ILE B 121      34.331  24.005  43.442  1.00  0.00           C  \\nATOM   1604  C   ILE B 121      33.499  25.042  42.700  1.00  0.00           C  \\nATOM   1605  O   ILE B 121      33.155  24.852  41.536  1.00  0.00           O  \\nATOM   1606  CB  ILE B 121      33.441  23.177  44.381  1.00  0.00           C  \\nATOM   1607  CG1 ILE B 121      34.250  21.986  44.915  1.00  0.00           C  \\nATOM   1608  CG2 ILE B 121      32.192  22.698  43.627  1.00  0.00           C  \\nATOM   1609  CD1 ILE B 121      33.616  21.261  46.057  1.00  0.00           C  \\nATOM   1610  H   ILE B 121      35.359  24.585  45.020  1.00  0.00           H  \\nATOM   1611  HA  ILE B 121      34.713  23.400  42.786  1.00  0.00           H  \\nATOM   1612  HB  ILE B 121      33.144  23.792  45.234  1.00  0.00           H  \\nATOM   1613 1HG1 ILE B 121      34.397  21.275  44.104  1.00  0.00           H  \\nATOM   1614 2HG1 ILE B 121      35.226  22.345  45.237  1.00  0.00           H  \\nATOM   1615 1HG2 ILE B 121      31.563  22.122  44.300  1.00  0.00           H  \\nATOM   1616 2HG2 ILE B 121      31.642  23.558  43.260  1.00  0.00           H  \\nATOM   1617 3HG2 ILE B 121      32.490  22.077  42.783  1.00  0.00           H  \\nATOM   1618 1HD1 ILE B 121      34.272  20.461  46.336  1.00  0.00           H  \\nATOM   1619 2HD1 ILE B 121      33.488  21.946  46.893  1.00  0.00           H  \\nATOM   1620 3HD1 ILE B 121      32.650  20.862  45.760  1.00  0.00           H  \\nATOM   1621  N   GLU B 122      33.179  26.143  43.372  1.00  0.00           N  \\nATOM   1622  CA  GLU B 122      32.456  27.229  42.722  1.00  0.00           C  \\nATOM   1623  C   GLU B 122      33.227  27.712  41.496  1.00  0.00           C  \\nATOM   1624  O   GLU B 122      32.656  27.880  40.417  1.00  0.00           O  \\nATOM   1625  CB  GLU B 122      32.216  28.407  43.677  1.00  0.00           C  \\nATOM   1626  CG  GLU B 122      31.445  29.583  43.034  1.00  0.00           C  \\nATOM   1627  CD  GLU B 122      31.154  30.739  43.974  1.00  0.00           C  \\nATOM   1628  OE1 GLU B 122      31.511  30.674  45.134  1.00  0.00           O  \\nATOM   1629  OE2 GLU B 122      30.541  31.679  43.537  1.00  0.00           O  \\nATOM   1630  H   GLU B 122      33.367  26.286  44.199  1.00  0.00           H  \\nATOM   1631  HA  GLU B 122      31.591  26.904  42.427  1.00  0.00           H  \\nATOM   1632 1HB  GLU B 122      31.661  28.062  44.550  1.00  0.00           H  \\nATOM   1633 2HB  GLU B 122      33.171  28.787  44.029  1.00  0.00           H  \\nATOM   1634 1HG  GLU B 122      32.042  29.959  42.202  1.00  0.00           H  \\nATOM   1635 2HG  GLU B 122      30.510  29.216  42.623  1.00  0.00           H  \\nATOM   1636  N   ALA B 123      34.529  27.913  41.667  1.00  0.00           N  \\nATOM   1637  CA  ALA B 123      35.385  28.352  40.574  1.00  0.00           C  \\nATOM   1638  C   ALA B 123      35.407  27.321  39.448  1.00  0.00           C  \\nATOM   1639  O   ALA B 123      35.443  27.678  38.275  1.00  0.00           O  \\nATOM   1640  CB  ALA B 123      36.791  28.608  41.079  1.00  0.00           C  \\nATOM   1641  H   ALA B 123      34.944  27.801  42.413  1.00  0.00           H  \\nATOM   1642  HA  ALA B 123      35.037  29.183  40.214  1.00  0.00           H  \\nATOM   1643 1HB  ALA B 123      37.423  28.961  40.264  1.00  0.00           H  \\nATOM   1644 2HB  ALA B 123      36.734  29.361  41.854  1.00  0.00           H  \\nATOM   1645 3HB  ALA B 123      37.205  27.698  41.492  1.00  0.00           H  \\nATOM   1646  N   SER B 124      35.374  26.043  39.814  1.00  0.00           N  \\nATOM   1647  CA  SER B 124      35.434  24.956  38.840  1.00  0.00           C  \\nATOM   1648  C   SER B 124      34.142  24.842  38.037  1.00  0.00           C  \\nATOM   1649  O   SER B 124      34.174  24.635  36.825  1.00  0.00           O  \\nATOM   1650  CB  SER B 124      35.701  23.662  39.573  1.00  0.00           C  \\nATOM   1651  OG  SER B 124      35.771  22.567  38.709  1.00  0.00           O  \\nATOM   1652  H   SER B 124      35.317  25.776  40.630  1.00  0.00           H  \\nATOM   1653  HA  SER B 124      36.158  25.130  38.218  1.00  0.00           H  \\nATOM   1654 1HB  SER B 124      36.628  23.749  40.134  1.00  0.00           H  \\nATOM   1655 2HB  SER B 124      34.908  23.497  40.291  1.00  0.00           H  \\nATOM   1656  HG  SER B 124      35.784  21.800  39.283  1.00  0.00           H  \\nATOM   1657  N   VAL B 125      33.008  24.971  38.717  1.00  0.00           N  \\nATOM   1658  CA  VAL B 125      31.712  24.958  38.047  1.00  0.00           C  \\nATOM   1659  C   VAL B 125      31.604  26.144  37.087  1.00  0.00           C  \\nATOM   1660  O   VAL B 125      31.047  26.023  35.995  1.00  0.00           O  \\nATOM   1661  CB  VAL B 125      30.557  24.978  39.080  1.00  0.00           C  \\nATOM   1662  CG1 VAL B 125      29.206  25.147  38.370  1.00  0.00           C  \\nATOM   1663  CG2 VAL B 125      30.562  23.671  39.879  1.00  0.00           C  \\nATOM   1664  H   VAL B 125      32.961  25.067  39.570  1.00  0.00           H  \\nATOM   1665  HA  VAL B 125      31.630  24.142  37.529  1.00  0.00           H  \\nATOM   1666  HB  VAL B 125      30.699  25.827  39.758  1.00  0.00           H  \\nATOM   1667 1HG1 VAL B 125      28.407  25.166  39.112  1.00  0.00           H  \\nATOM   1668 2HG1 VAL B 125      29.208  26.083  37.809  1.00  0.00           H  \\nATOM   1669 3HG1 VAL B 125      29.046  24.311  37.686  1.00  0.00           H  \\nATOM   1670 1HG2 VAL B 125      29.756  23.685  40.615  1.00  0.00           H  \\nATOM   1671 2HG2 VAL B 125      30.423  22.848  39.203  1.00  0.00           H  \\nATOM   1672 3HG2 VAL B 125      31.505  23.552  40.386  1.00  0.00           H  \\nATOM   1673  N   GLU B 126      32.146  27.285  37.495  1.00  0.00           N  \\nATOM   1674  CA  GLU B 126      32.132  28.476  36.655  1.00  0.00           C  \\nATOM   1675  C   GLU B 126      33.007  28.268  35.421  1.00  0.00           C  \\nATOM   1676  O   GLU B 126      32.635  28.654  34.314  1.00  0.00           O  \\nATOM   1677  CB  GLU B 126      32.535  29.716  37.450  1.00  0.00           C  \\nATOM   1678  CG  GLU B 126      32.475  31.042  36.666  1.00  0.00           C  \\nATOM   1679  CD  GLU B 126      31.085  31.367  36.110  1.00  0.00           C  \\nATOM   1680  OE1 GLU B 126      30.144  30.665  36.412  1.00  0.00           O  \\nATOM   1681  OE2 GLU B 126      30.957  32.360  35.425  1.00  0.00           O  \\nATOM   1682  H   GLU B 126      32.529  27.397  38.257  1.00  0.00           H  \\nATOM   1683  HA  GLU B 126      31.225  28.643  36.356  1.00  0.00           H  \\nATOM   1684 1HB  GLU B 126      31.894  29.807  38.327  1.00  0.00           H  \\nATOM   1685 2HB  GLU B 126      33.556  29.590  37.811  1.00  0.00           H  \\nATOM   1686 1HG  GLU B 126      32.792  31.850  37.323  1.00  0.00           H  \\nATOM   1687 2HG  GLU B 126      33.186  30.984  35.840  1.00  0.00           H  \\nATOM   1688  N   ASN B 127      34.165  27.648  35.613  1.00  0.00           N  \\nATOM   1689  CA  ASN B 127      35.051  27.352  34.495  1.00  0.00           C  \\nATOM   1690  C   ASN B 127      34.420  26.353  33.530  1.00  0.00           C  \\nATOM   1691  O   ASN B 127      34.582  26.463  32.317  1.00  0.00           O  \\nATOM   1692  CB  ASN B 127      36.384  26.818  35.022  1.00  0.00           C  \\nATOM   1693  CG  ASN B 127      37.487  26.703  33.971  1.00  0.00           C  \\nATOM   1694  OD1 ASN B 127      37.926  27.698  33.393  1.00  0.00           O  \\nATOM   1695  ND2 ASN B 127      37.925  25.502  33.712  1.00  0.00           N  \\nATOM   1696  H   ASN B 127      34.461  27.389  36.378  1.00  0.00           H  \\nATOM   1697  HA  ASN B 127      35.227  28.171  34.006  1.00  0.00           H  \\nATOM   1698 1HB  ASN B 127      36.735  27.462  35.831  1.00  0.00           H  \\nATOM   1699 2HB  ASN B 127      36.218  25.830  35.458  1.00  0.00           H  \\nATOM   1700 1HD2 ASN B 127      37.538  24.711  34.221  1.00  0.00           H  \\nATOM   1701 2HD2 ASN B 127      38.626  25.364  33.015  1.00  0.00           H  \\nATOM   1702  N   GLN B 128      33.697  25.380  34.073  1.00  0.00           N  \\nATOM   1703  CA  GLN B 128      33.050  24.372  33.247  1.00  0.00           C  \\nATOM   1704  C   GLN B 128      31.886  24.974  32.465  1.00  0.00           C  \\nATOM   1705  O   GLN B 128      31.623  24.569  31.337  1.00  0.00           O  \\nATOM   1706  CB  GLN B 128      32.618  23.200  34.120  1.00  0.00           C  \\nATOM   1707  CG  GLN B 128      33.823  22.388  34.662  1.00  0.00           C  \\nATOM   1708  CD  GLN B 128      33.450  21.259  35.606  1.00  0.00           C  \\nATOM   1709  OE1 GLN B 128      32.773  20.322  35.185  1.00  0.00           O  \\nATOM   1710  NE2 GLN B 128      33.874  21.305  36.870  1.00  0.00           N  \\nATOM   1711  H   GLN B 128      33.566  25.282  34.917  1.00  0.00           H  \\nATOM   1712  HA  GLN B 128      33.694  24.032  32.606  1.00  0.00           H  \\nATOM   1713 1HB  GLN B 128      32.050  23.577  34.973  1.00  0.00           H  \\nATOM   1714 2HB  GLN B 128      31.976  22.557  33.557  1.00  0.00           H  \\nATOM   1715 1HG  GLN B 128      34.323  21.937  33.809  1.00  0.00           H  \\nATOM   1716 2HG  GLN B 128      34.519  23.039  35.159  1.00  0.00           H  \\nATOM   1717 1HE2 GLN B 128      33.670  20.540  37.488  1.00  0.00           H  \\nATOM   1718 2HE2 GLN B 128      34.440  22.064  37.232  1.00  0.00           H  \\nATOM   1719  N   ARG B 129      31.204  25.951  33.056  1.00  0.00           N  \\nATOM   1720  CA  ARG B 129      30.122  26.647  32.366  1.00  0.00           C  \\nATOM   1721  C   ARG B 129      30.669  27.468  31.202  1.00  0.00           C  \\nATOM   1722  O   ARG B 129      30.090  27.490  30.119  1.00  0.00           O  \\nATOM   1723  CB  ARG B 129      29.372  27.546  33.339  1.00  0.00           C  \\nATOM   1724  CG  ARG B 129      28.090  28.173  32.796  1.00  0.00           C  \\nATOM   1725  CD  ARG B 129      27.401  29.025  33.830  1.00  0.00           C  \\nATOM   1726  NE  ARG B 129      28.100  30.281  34.092  1.00  0.00           N  \\nATOM   1727  CZ  ARG B 129      27.944  31.389  33.332  1.00  0.00           C  \\nATOM   1728  NH1 ARG B 129      27.109  31.366  32.302  1.00  0.00           N  \\nATOM   1729  NH2 ARG B 129      28.611  32.495  33.617  1.00  0.00           N  \\nATOM   1730  H   ARG B 129      31.348  26.231  33.856  1.00  0.00           H  \\nATOM   1731  HA  ARG B 129      29.500  25.993  32.009  1.00  0.00           H  \\nATOM   1732 1HB  ARG B 129      29.114  26.973  34.230  1.00  0.00           H  \\nATOM   1733 2HB  ARG B 129      30.030  28.360  33.661  1.00  0.00           H  \\nATOM   1734 1HG  ARG B 129      28.338  28.806  31.939  1.00  0.00           H  \\nATOM   1735 2HG  ARG B 129      27.406  27.385  32.481  1.00  0.00           H  \\nATOM   1736 1HD  ARG B 129      26.394  29.263  33.491  1.00  0.00           H  \\nATOM   1737 2HD  ARG B 129      27.346  28.472  34.768  1.00  0.00           H  \\nATOM   1738  HE  ARG B 129      28.758  30.336  34.897  1.00  0.00           H  \\nATOM   1739 1HH1 ARG B 129      26.603  30.522  32.076  1.00  0.00           H  \\nATOM   1740 2HH1 ARG B 129      26.992  32.185  31.719  1.00  0.00           H  \\nATOM   1741 1HH2 ARG B 129      29.285  32.501  34.406  1.00  0.00           H  \\nATOM   1742 2HH2 ARG B 129      28.473  33.320  33.063  1.00  0.00           H  \\nATOM   1743  N   ILE B 130      31.784  28.148  31.438  1.00  0.00           N  \\nATOM   1744  CA  ILE B 130      32.424  28.947  30.404  1.00  0.00           C  \\nATOM   1745  C   ILE B 130      32.994  28.037  29.322  1.00  0.00           C  \\nATOM   1746  O   ILE B 130      32.967  28.375  28.140  1.00  0.00           O  \\nATOM   1747  CB  ILE B 130      33.504  29.852  30.991  1.00  0.00           C  \\nATOM   1748  CG1 ILE B 130      32.833  30.900  31.856  1.00  0.00           C  \\nATOM   1749  CG2 ILE B 130      34.307  30.512  29.883  1.00  0.00           C  \\nATOM   1750  CD1 ILE B 130      33.778  31.674  32.697  1.00  0.00           C  \\nATOM   1751  H   ILE B 130      32.193  28.162  32.195  1.00  0.00           H  \\nATOM   1752  HA  ILE B 130      31.762  29.525  29.994  1.00  0.00           H  \\nATOM   1753  HB  ILE B 130      34.169  29.262  31.629  1.00  0.00           H  \\nATOM   1754 1HG1 ILE B 130      32.302  31.581  31.209  1.00  0.00           H  \\nATOM   1755 2HG1 ILE B 130      32.110  30.412  32.515  1.00  0.00           H  \\nATOM   1756 1HG2 ILE B 130      35.051  31.145  30.338  1.00  0.00           H  \\nATOM   1757 2HG2 ILE B 130      34.790  29.750  29.277  1.00  0.00           H  \\nATOM   1758 3HG2 ILE B 130      33.648  31.112  29.255  1.00  0.00           H  \\nATOM   1759 1HD1 ILE B 130      33.224  32.404  33.290  1.00  0.00           H  \\nATOM   1760 2HD1 ILE B 130      34.286  30.979  33.364  1.00  0.00           H  \\nATOM   1761 3HD1 ILE B 130      34.503  32.186  32.068  1.00  0.00           H  \\nATOM   1762  N   SER B 131      33.500  26.876  29.725  1.00  0.00           N  \\nATOM   1763  CA  SER B 131      34.026  25.916  28.761  1.00  0.00           C  \\nATOM   1764  C   SER B 131      32.922  25.417  27.839  1.00  0.00           C  \\nATOM   1765  O   SER B 131      33.143  25.226  26.643  1.00  0.00           O  \\nATOM   1766  CB  SER B 131      34.628  24.751  29.532  1.00  0.00           C  \\nATOM   1767  OG  SER B 131      35.132  23.741  28.702  1.00  0.00           O  \\nATOM   1768  H   SER B 131      33.550  26.621  30.544  1.00  0.00           H  \\nATOM   1769  HA  SER B 131      34.700  26.351  28.216  1.00  0.00           H  \\nATOM   1770 1HB  SER B 131      35.414  25.115  30.185  1.00  0.00           H  \\nATOM   1771 2HB  SER B 131      33.863  24.331  30.166  1.00  0.00           H  \\nATOM   1772  HG  SER B 131      35.365  23.026  29.298  1.00  0.00           H  \\nATOM   1773  N   ALA B 132      31.734  25.206  28.397  1.00  0.00           N  \\nATOM   1774  CA  ALA B 132      30.608  24.713  27.611  1.00  0.00           C  \\nATOM   1775  C   ALA B 132      30.191  25.752  26.580  1.00  0.00           C  \\nATOM   1776  O   ALA B 132      29.869  25.412  25.445  1.00  0.00           O  \\nATOM   1777  CB  ALA B 132      29.448  24.376  28.521  1.00  0.00           C  \\nATOM   1778  H   ALA B 132      31.554  25.340  29.228  1.00  0.00           H  \\nATOM   1779  HA  ALA B 132      30.878  23.910  27.139  1.00  0.00           H  \\nATOM   1780 1HB  ALA B 132      29.135  25.273  29.057  1.00  0.00           H  \\nATOM   1781 2HB  ALA B 132      28.616  23.994  27.934  1.00  0.00           H  \\nATOM   1782 3HB  ALA B 132      29.772  23.621  29.233  1.00  0.00           H  \\nATOM   1783  N   GLU B 133      30.199  27.020  26.976  1.00  0.00           N  \\nATOM   1784  CA  GLU B 133      29.883  28.094  26.045  1.00  0.00           C  \\nATOM   1785  C   GLU B 133      30.924  28.139  24.931  1.00  0.00           C  \\nATOM   1786  O   GLU B 133      30.588  28.339  23.764  1.00  0.00           O  \\nATOM   1787  CB  GLU B 133      29.786  29.454  26.753  1.00  0.00           C  \\nATOM   1788  CG  GLU B 133      28.541  29.622  27.659  1.00  0.00           C  \\nATOM   1789  CD  GLU B 133      28.468  30.990  28.344  1.00  0.00           C  \\nATOM   1790  OE1 GLU B 133      29.429  31.717  28.258  1.00  0.00           O  \\nATOM   1791  OE2 GLU B 133      27.456  31.304  28.937  1.00  0.00           O  \\nATOM   1792  H   GLU B 133      30.383  27.283  27.774  1.00  0.00           H  \\nATOM   1793  HA  GLU B 133      29.018  27.920  25.641  1.00  0.00           H  \\nATOM   1794 1HB  GLU B 133      30.671  29.597  27.376  1.00  0.00           H  \\nATOM   1795 2HB  GLU B 133      29.778  30.251  26.010  1.00  0.00           H  \\nATOM   1796 1HG  GLU B 133      27.646  29.479  27.055  1.00  0.00           H  \\nATOM   1797 2HG  GLU B 133      28.560  28.842  28.420  1.00  0.00           H  \\nATOM   1798  N   ASN B 134      32.180  27.941  25.260  1.00  0.00           N  \\nATOM   1799  CA  ASN B 134      33.232  27.883  24.260  1.00  0.00           C  \\nATOM   1800  C   ASN B 134      33.014  26.704  23.305  1.00  0.00           C  \\nATOM   1801  O   ASN B 134      33.284  26.782  22.141  1.00  0.00           O  \\nATOM   1802  CB  ASN B 134      34.623  27.836  24.874  1.00  0.00           C  \\nATOM   1803  CG  ASN B 134      35.708  27.983  23.795  1.00  0.00           C  \\nATOM   1804  OD1 ASN B 134      35.798  28.989  23.093  1.00  0.00           O  \\nATOM   1805  ND2 ASN B 134      36.511  26.988  23.645  1.00  0.00           N  \\nATOM   1806  H   ASN B 134      32.454  27.837  26.060  1.00  0.00           H  \\nATOM   1807  HA  ASN B 134      33.192  28.697  23.729  1.00  0.00           H  \\nATOM   1808 1HB  ASN B 134      34.732  28.632  25.614  1.00  0.00           H  \\nATOM   1809 2HB  ASN B 134      34.761  26.887  25.391  1.00  0.00           H  \\nATOM   1810 1HD2 ASN B 134      37.216  27.026  22.940  1.00  0.00           H  \\nATOM   1811 2HD2 ASN B 134      36.408  26.174  24.235  1.00  0.00           H  \\nATOM   1812  N   GLN B 135      32.466  25.631  23.811  1.00  0.00           N  \\nATOM   1813  CA  GLN B 135      32.158  24.505  22.980  1.00  0.00           C  \\nATOM   1814  C   GLN B 135      30.944  24.761  22.111  1.00  0.00           C  \\nATOM   1815  O   GLN B 135      30.899  24.317  21.012  1.00  0.00           O  \\nATOM   1816  CB  GLN B 135      31.964  23.256  23.817  1.00  0.00           C  \\nATOM   1817  CG  GLN B 135      33.238  22.762  24.445  1.00  0.00           C  \\nATOM   1818  CD  GLN B 135      33.017  21.623  25.393  1.00  0.00           C  \\nATOM   1819  OE1 GLN B 135      32.181  20.741  25.163  1.00  0.00           O  \\nATOM   1820  NE2 GLN B 135      33.780  21.623  26.465  1.00  0.00           N  \\nATOM   1821  H   GLN B 135      32.253  25.527  24.630  1.00  0.00           H  \\nATOM   1822  HA  GLN B 135      32.916  24.341  22.397  1.00  0.00           H  \\nATOM   1823 1HB  GLN B 135      31.243  23.448  24.604  1.00  0.00           H  \\nATOM   1824 2HB  GLN B 135      31.567  22.489  23.194  1.00  0.00           H  \\nATOM   1825 1HG  GLN B 135      33.899  22.416  23.652  1.00  0.00           H  \\nATOM   1826 2HG  GLN B 135      33.717  23.573  24.982  1.00  0.00           H  \\nATOM   1827 1HE2 GLN B 135      33.721  20.880  27.137  1.00  0.00           H  \\nATOM   1828 2HE2 GLN B 135      34.449  22.362  26.592  1.00  0.00           H  \\nATOM   1829  N   LYS B 136      29.964  25.467  22.630  1.00  0.00           N  \\nATOM   1830  CA  LYS B 136      28.813  25.890  21.838  1.00  0.00           C  \\nATOM   1831  C   LYS B 136      29.265  26.658  20.603  1.00  0.00           C  \\nATOM   1832  O   LYS B 136      28.810  26.394  19.491  1.00  0.00           O  \\nATOM   1833  CB  LYS B 136      27.820  26.698  22.674  1.00  0.00           C  \\nATOM   1834  CG  LYS B 136      26.992  25.834  23.644  1.00  0.00           C  \\nATOM   1835  CD  LYS B 136      26.125  26.675  24.580  1.00  0.00           C  \\nATOM   1836  CE  LYS B 136      25.021  27.403  23.824  1.00  0.00           C  \\nATOM   1837  NZ  LYS B 136      24.136  28.175  24.743  1.00  0.00           N  \\nATOM   1838  H   LYS B 136      29.942  25.728  23.449  1.00  0.00           H  \\nATOM   1839  HA  LYS B 136      28.320  25.107  21.547  1.00  0.00           H  \\nATOM   1840 1HB  LYS B 136      28.347  27.441  23.253  1.00  0.00           H  \\nATOM   1841 2HB  LYS B 136      27.134  27.224  22.014  1.00  0.00           H  \\nATOM   1842 1HG  LYS B 136      26.350  25.168  23.072  1.00  0.00           H  \\nATOM   1843 2HG  LYS B 136      27.667  25.227  24.240  1.00  0.00           H  \\nATOM   1844 1HD  LYS B 136      25.678  26.027  25.335  1.00  0.00           H  \\nATOM   1845 2HD  LYS B 136      26.747  27.409  25.086  1.00  0.00           H  \\nATOM   1846 1HE  LYS B 136      25.473  28.089  23.110  1.00  0.00           H  \\nATOM   1847 2HE  LYS B 136      24.418  26.674  23.283  1.00  0.00           H  \\nATOM   1848 1HZ  LYS B 136      23.419  28.643  24.206  1.00  0.00           H  \\nATOM   1849 2HZ  LYS B 136      23.703  27.544  25.404  1.00  0.00           H  \\nATOM   1850 3HZ  LYS B 136      24.683  28.862  25.244  1.00  0.00           H  \\nATOM   1851  N   ALA B 137      30.166  27.611  20.813  1.00  0.00           N  \\nATOM   1852  CA  ALA B 137      30.680  28.437  19.731  1.00  0.00           C  \\nATOM   1853  C   ALA B 137      31.405  27.587  18.694  1.00  0.00           C  \\nATOM   1854  O   ALA B 137      31.184  27.744  17.495  1.00  0.00           O  \\nATOM   1855  CB  ALA B 137      31.642  29.424  20.343  1.00  0.00           C  \\nATOM   1856  H   ALA B 137      30.498  27.800  21.584  1.00  0.00           H  \\nATOM   1857  HA  ALA B 137      29.938  28.880  19.291  1.00  0.00           H  \\nATOM   1858 1HB  ALA B 137      32.021  30.085  19.597  1.00  0.00           H  \\nATOM   1859 2HB  ALA B 137      31.117  29.962  21.097  1.00  0.00           H  \\nATOM   1860 3HB  ALA B 137      32.467  28.890  20.815  1.00  0.00           H  \\nATOM   1861  N   LEU B 138      32.269  26.689  19.163  1.00  0.00           N  \\nATOM   1862  CA  LEU B 138      33.019  25.802  18.277  1.00  0.00           C  \\nATOM   1863  C   LEU B 138      32.081  25.013  17.374  1.00  0.00           C  \\nATOM   1864  O   LEU B 138      32.362  24.818  16.192  1.00  0.00           O  \\nATOM   1865  CB  LEU B 138      33.941  24.857  19.074  1.00  0.00           C  \\nATOM   1866  CG  LEU B 138      35.199  25.527  19.740  1.00  0.00           C  \\nATOM   1867  CD1 LEU B 138      35.921  24.512  20.649  1.00  0.00           C  \\nATOM   1868  CD2 LEU B 138      36.146  26.030  18.647  1.00  0.00           C  \\nATOM   1869  H   LEU B 138      32.440  26.572  19.998  1.00  0.00           H  \\nATOM   1870  HA  LEU B 138      33.600  26.336  17.713  1.00  0.00           H  \\nATOM   1871 1HB  LEU B 138      33.350  24.401  19.866  1.00  0.00           H  \\nATOM   1872 2HB  LEU B 138      34.290  24.069  18.410  1.00  0.00           H  \\nATOM   1873  HG  LEU B 138      34.883  26.370  20.348  1.00  0.00           H  \\nATOM   1874 1HD1 LEU B 138      36.791  24.986  21.111  1.00  0.00           H  \\nATOM   1875 2HD1 LEU B 138      35.239  24.174  21.428  1.00  0.00           H  \\nATOM   1876 3HD1 LEU B 138      36.243  23.663  20.055  1.00  0.00           H  \\nATOM   1877 1HD2 LEU B 138      37.010  26.502  19.110  1.00  0.00           H  \\nATOM   1878 2HD2 LEU B 138      36.481  25.192  18.031  1.00  0.00           H  \\nATOM   1879 3HD2 LEU B 138      35.627  26.754  18.024  1.00  0.00           H  \\nATOM   1880  N   ALA B 139      30.965  24.566  17.940  1.00  0.00           N  \\nATOM   1881  CA  ALA B 139      29.960  23.830  17.185  1.00  0.00           C  \\nATOM   1882  C   ALA B 139      29.252  24.748  16.192  1.00  0.00           C  \\nATOM   1883  O   ALA B 139      28.191  24.410  15.667  1.00  0.00           O  \\nATOM   1884  OXT ALA B 139      29.729  25.814  15.915  1.00  0.00           O  \\nATOM   1885  CB  ALA B 139      28.948  23.220  18.146  1.00  0.00           C  \\nATOM   1886  H   ALA B 139      30.765  24.677  18.769  1.00  0.00           H  \\nATOM   1887  HA  ALA B 139      30.395  23.122  16.683  1.00  0.00           H  \\nATOM   1888 1HB  ALA B 139      28.205  22.665  17.584  1.00  0.00           H  \\nATOM   1889 2HB  ALA B 139      29.451  22.560  18.845  1.00  0.00           H  \\nATOM   1890 3HB  ALA B 139      28.458  24.016  18.702  1.00  0.00           H  \\nTER                                                                             \\nATOM   1892  N   THR C 140      27.512  14.029  12.268  1.00  0.00           N  \\nATOM   1893  CA  THR C 140      28.049  14.350  13.620  1.00  0.00           C  \\nATOM   1894  C   THR C 140      27.258  15.481  14.277  1.00  0.00           C  \\nATOM   1895  O   THR C 140      27.635  15.974  15.340  1.00  0.00           O  \\nATOM   1896  CB  THR C 140      29.530  14.766  13.543  1.00  0.00           C  \\nATOM   1897  OG1 THR C 140      30.116  14.723  14.854  1.00  0.00           O  \\nATOM   1898  CG2 THR C 140      29.638  16.187  13.002  1.00  0.00           C  \\nATOM   1899 1H   THR C 140      28.048  13.288  11.863  1.00  0.00           H  \\nATOM   1900 2H   THR C 140      26.557  13.745  12.347  1.00  0.00           H  \\nATOM   1901 3H   THR C 140      27.571  14.839  11.684  1.00  0.00           H  \\nATOM   1902  HA  THR C 140      27.974  13.564  14.184  1.00  0.00           H  \\nATOM   1903  HB  THR C 140      30.064  14.078  12.889  1.00  0.00           H  \\nATOM   1904  HG1 THR C 140      30.096  15.607  15.239  1.00  0.00           H  \\nATOM   1905 1HG2 THR C 140      30.686  16.478  12.951  1.00  0.00           H  \\nATOM   1906 2HG2 THR C 140      29.197  16.234  12.007  1.00  0.00           H  \\nATOM   1907 3HG2 THR C 140      29.106  16.866  13.664  1.00  0.00           H  \\nATOM   1908  N   LYS C 141      26.158  15.882  13.645  1.00  0.00           N  \\nATOM   1909  CA  LYS C 141      25.350  16.991  14.142  1.00  0.00           C  \\nATOM   1910  C   LYS C 141      24.547  16.563  15.360  1.00  0.00           C  \\nATOM   1911  O   LYS C 141      24.509  17.274  16.363  1.00  0.00           O  \\nATOM   1912  CB  LYS C 141      24.435  17.544  13.051  1.00  0.00           C  \\nATOM   1913  CG  LYS C 141      24.895  18.897  12.440  1.00  0.00           C  \\nATOM   1914  CD  LYS C 141      26.339  18.862  11.924  1.00  0.00           C  \\nATOM   1915  CE  LYS C 141      27.315  19.561  12.892  1.00  0.00           C  \\nATOM   1916  NZ  LYS C 141      27.163  21.065  12.876  1.00  0.00           N  \\nATOM   1917  H   LYS C 141      25.857  15.527  12.922  1.00  0.00           H  \\nATOM   1918  HA  LYS C 141      25.936  17.716  14.407  1.00  0.00           H  \\nATOM   1919 1HB  LYS C 141      24.351  16.822  12.241  1.00  0.00           H  \\nATOM   1920 2HB  LYS C 141      23.435  17.691  13.462  1.00  0.00           H  \\nATOM   1921 1HG  LYS C 141      24.234  19.158  11.613  1.00  0.00           H  \\nATOM   1922 2HG  LYS C 141      24.813  19.674  13.200  1.00  0.00           H  \\nATOM   1923 1HD  LYS C 141      26.654  17.829  11.793  1.00  0.00           H  \\nATOM   1924 2HD  LYS C 141      26.388  19.362  10.956  1.00  0.00           H  \\nATOM   1925 1HE  LYS C 141      27.132  19.198  13.904  1.00  0.00           H  \\nATOM   1926 2HE  LYS C 141      28.336  19.308  12.607  1.00  0.00           H  \\nATOM   1927 1HZ  LYS C 141      27.826  21.473  13.523  1.00  0.00           H  \\nATOM   1928 2HZ  LYS C 141      26.225  21.320  13.152  1.00  0.00           H  \\nATOM   1929 3HZ  LYS C 141      27.344  21.417  11.947  1.00  0.00           H  \\nATOM   1930  N   TYR C 142      23.902  15.403  15.271  1.00  0.00           N  \\nATOM   1931  CA  TYR C 142      23.134  14.886  16.395  1.00  0.00           C  \\nATOM   1932  C   TYR C 142      24.038  14.667  17.598  1.00  0.00           C  \\nATOM   1933  O   TYR C 142      23.651  14.944  18.729  1.00  0.00           O  \\nATOM   1934  CB  TYR C 142      22.445  13.573  16.035  1.00  0.00           C  \\nATOM   1935  CG  TYR C 142      21.591  13.023  17.159  1.00  0.00           C  \\nATOM   1936  CD1 TYR C 142      20.331  13.554  17.396  1.00  0.00           C  \\nATOM   1937  CD2 TYR C 142      22.070  11.992  17.956  1.00  0.00           C  \\nATOM   1938  CE1 TYR C 142      19.551  13.052  18.419  1.00  0.00           C  \\nATOM   1939  CE2 TYR C 142      21.290  11.493  18.981  1.00  0.00           C  \\nATOM   1940  CZ  TYR C 142      20.034  12.019  19.212  1.00  0.00           C  \\nATOM   1941  OH  TYR C 142      19.252  11.519  20.228  1.00  0.00           O  \\nATOM   1942  H   TYR C 142      23.895  14.899  14.574  1.00  0.00           H  \\nATOM   1943  HA  TYR C 142      22.457  15.536  16.641  1.00  0.00           H  \\nATOM   1944 1HB  TYR C 142      21.810  13.720  15.161  1.00  0.00           H  \\nATOM   1945 2HB  TYR C 142      23.194  12.826  15.779  1.00  0.00           H  \\nATOM   1946  HD1 TYR C 142      19.955  14.364  16.771  1.00  0.00           H  \\nATOM   1947  HD2 TYR C 142      23.062  11.579  17.774  1.00  0.00           H  \\nATOM   1948  HE1 TYR C 142      18.562  13.469  18.604  1.00  0.00           H  \\nATOM   1949  HE2 TYR C 142      21.664  10.682  19.607  1.00  0.00           H  \\nATOM   1950  HH  TYR C 142      18.429  12.012  20.269  1.00  0.00           H  \\nATOM   1951  N   LYS C 143      25.242  14.166  17.344  1.00  0.00           N  \\nATOM   1952  CA  LYS C 143      26.176  13.844  18.414  1.00  0.00           C  \\nATOM   1953  C   LYS C 143      26.519  15.091  19.227  1.00  0.00           C  \\nATOM   1954  O   LYS C 143      26.355  15.108  20.443  1.00  0.00           O  \\nATOM   1955  CB  LYS C 143      27.395  13.132  17.820  1.00  0.00           C  \\nATOM   1956  CG  LYS C 143      28.465  12.712  18.805  1.00  0.00           C  \\nATOM   1957  CD  LYS C 143      29.434  11.685  18.160  1.00  0.00           C  \\nATOM   1958  CE  LYS C 143      30.161  12.225  16.936  1.00  0.00           C  \\nATOM   1959  NZ  LYS C 143      31.145  11.238  16.393  1.00  0.00           N  \\nATOM   1960  H   LYS C 143      25.543  14.004  16.555  1.00  0.00           H  \\nATOM   1961  HA  LYS C 143      25.762  13.201  19.011  1.00  0.00           H  \\nATOM   1962 1HB  LYS C 143      27.068  12.243  17.283  1.00  0.00           H  \\nATOM   1963 2HB  LYS C 143      27.873  13.794  17.092  1.00  0.00           H  \\nATOM   1964 1HG  LYS C 143      29.016  13.591  19.124  1.00  0.00           H  \\nATOM   1965 2HG  LYS C 143      27.998  12.263  19.680  1.00  0.00           H  \\nATOM   1966 1HD  LYS C 143      30.180  11.390  18.900  1.00  0.00           H  \\nATOM   1967 2HD  LYS C 143      28.873  10.798  17.867  1.00  0.00           H  \\nATOM   1968 1HE  LYS C 143      29.430  12.456  16.165  1.00  0.00           H  \\nATOM   1969 2HE  LYS C 143      30.683  13.121  17.194  1.00  0.00           H  \\nATOM   1970 1HZ  LYS C 143      31.836  11.027  17.096  1.00  0.00           H  \\nATOM   1971 2HZ  LYS C 143      30.664  10.390  16.127  1.00  0.00           H  \\nATOM   1972 3HZ  LYS C 143      31.600  11.637  15.583  1.00  0.00           H  \\nATOM   1973  N   ILE C 144      26.969  16.140  18.545  1.00  0.00           N  \\nATOM   1974  CA  ILE C 144      27.376  17.376  19.207  1.00  0.00           C  \\nATOM   1975  C   ILE C 144      26.234  17.997  20.009  1.00  0.00           C  \\nATOM   1976  O   ILE C 144      26.404  18.333  21.180  1.00  0.00           O  \\nATOM   1977  CB  ILE C 144      27.923  18.383  18.181  1.00  0.00           C  \\nATOM   1978  CG1 ILE C 144      29.240  17.858  17.614  1.00  0.00           C  \\nATOM   1979  CG2 ILE C 144      28.139  19.733  18.828  1.00  0.00           C  \\nATOM   1980  CD1 ILE C 144      29.731  18.587  16.397  1.00  0.00           C  \\nATOM   1981  H   ILE C 144      27.050  16.161  17.689  1.00  0.00           H  \\nATOM   1982  HA  ILE C 144      28.096  17.175  19.825  1.00  0.00           H  \\nATOM   1983  HB  ILE C 144      27.214  18.474  17.352  1.00  0.00           H  \\nATOM   1984 1HG1 ILE C 144      30.006  17.930  18.389  1.00  0.00           H  \\nATOM   1985 2HG1 ILE C 144      29.104  16.814  17.348  1.00  0.00           H  \\nATOM   1986 1HG2 ILE C 144      28.523  20.406  18.081  1.00  0.00           H  \\nATOM   1987 2HG2 ILE C 144      27.196  20.110  19.214  1.00  0.00           H  \\nATOM   1988 3HG2 ILE C 144      28.855  19.645  19.643  1.00  0.00           H  \\nATOM   1989 1HD1 ILE C 144      29.887  19.636  16.638  1.00  0.00           H  \\nATOM   1990 2HD1 ILE C 144      30.673  18.142  16.063  1.00  0.00           H  \\nATOM   1991 3HD1 ILE C 144      28.987  18.501  15.609  1.00  0.00           H  \\nATOM   1992  N   LYS C 145      25.071  18.143  19.385  1.00  0.00           N  \\nATOM   1993  CA  LYS C 145      23.953  18.817  20.035  1.00  0.00           C  \\nATOM   1994  C   LYS C 145      23.334  17.972  21.148  1.00  0.00           C  \\nATOM   1995  O   LYS C 145      22.797  18.512  22.116  1.00  0.00           O  \\nATOM   1996  CB  LYS C 145      22.897  19.147  18.987  1.00  0.00           C  \\nATOM   1997  CG  LYS C 145      23.328  20.217  17.996  1.00  0.00           C  \\nATOM   1998  CD  LYS C 145      22.317  20.370  16.867  1.00  0.00           C  \\nATOM   1999  CE  LYS C 145      21.022  21.007  17.355  1.00  0.00           C  \\nATOM   2000  NZ  LYS C 145      20.066  21.248  16.238  1.00  0.00           N  \\nATOM   2001  H   LYS C 145      24.904  17.864  18.589  1.00  0.00           H  \\nATOM   2002  HA  LYS C 145      24.277  19.638  20.436  1.00  0.00           H  \\nATOM   2003 1HB  LYS C 145      22.666  18.240  18.418  1.00  0.00           H  \\nATOM   2004 2HB  LYS C 145      21.982  19.473  19.476  1.00  0.00           H  \\nATOM   2005 1HG  LYS C 145      23.447  21.169  18.511  1.00  0.00           H  \\nATOM   2006 2HG  LYS C 145      24.290  19.928  17.568  1.00  0.00           H  \\nATOM   2007 1HD  LYS C 145      22.745  20.991  16.079  1.00  0.00           H  \\nATOM   2008 2HD  LYS C 145      22.099  19.385  16.448  1.00  0.00           H  \\nATOM   2009 1HE  LYS C 145      20.554  20.348  18.084  1.00  0.00           H  \\nATOM   2010 2HE  LYS C 145      21.251  21.959  17.834  1.00  0.00           H  \\nATOM   2011 1HZ  LYS C 145      20.487  21.868  15.560  1.00  0.00           H  \\nATOM   2012 2HZ  LYS C 145      19.838  20.369  15.793  1.00  0.00           H  \\nATOM   2013 3HZ  LYS C 145      19.223  21.669  16.601  1.00  0.00           H  \\nATOM   2014  N   GLU C 146      23.417  16.652  21.020  1.00  0.00           N  \\nATOM   2015  CA  GLU C 146      22.892  15.766  22.051  1.00  0.00           C  \\nATOM   2016  C   GLU C 146      23.785  15.817  23.285  1.00  0.00           C  \\nATOM   2017  O   GLU C 146      23.301  15.781  24.415  1.00  0.00           O  \\nATOM   2018  CB  GLU C 146      22.780  14.312  21.545  1.00  0.00           C  \\nATOM   2019  CG  GLU C 146      21.986  13.311  22.444  1.00  0.00           C  \\nATOM   2020  CD  GLU C 146      22.757  12.619  23.587  1.00  0.00           C  \\nATOM   2021  OE1 GLU C 146      23.965  12.450  23.512  1.00  0.00           O  \\nATOM   2022  OE2 GLU C 146      22.095  12.224  24.522  1.00  0.00           O  \\nATOM   2023  H   GLU C 146      23.771  16.247  20.350  1.00  0.00           H  \\nATOM   2024  HA  GLU C 146      22.006  16.065  22.306  1.00  0.00           H  \\nATOM   2025 1HB  GLU C 146      22.256  14.313  20.595  1.00  0.00           H  \\nATOM   2026 2HB  GLU C 146      23.779  13.907  21.366  1.00  0.00           H  \\nATOM   2027 1HG  GLU C 146      21.150  13.854  22.881  1.00  0.00           H  \\nATOM   2028 2HG  GLU C 146      21.569  12.543  21.801  1.00  0.00           H  \\nATOM   2029  N   THR C 147      25.092  15.906  23.062  1.00  0.00           N  \\nATOM   2030  CA  THR C 147      26.049  15.988  24.156  1.00  0.00           C  \\nATOM   2031  C   THR C 147      25.979  17.349  24.847  1.00  0.00           C  \\nATOM   2032  O   THR C 147      26.129  17.442  26.064  1.00  0.00           O  \\nATOM   2033  CB  THR C 147      27.518  15.732  23.739  1.00  0.00           C  \\nATOM   2034  OG1 THR C 147      27.912  16.658  22.719  1.00  0.00           O  \\nATOM   2035  CG2 THR C 147      27.745  14.302  23.298  1.00  0.00           C  \\nATOM   2036  H   THR C 147      25.450  15.920  22.281  1.00  0.00           H  \\nATOM   2037  HA  THR C 147      25.836  15.307  24.814  1.00  0.00           H  \\nATOM   2038  HB  THR C 147      28.158  15.906  24.604  1.00  0.00           H  \\nATOM   2039  HG1 THR C 147      27.136  16.967  22.205  1.00  0.00           H  \\nATOM   2040 1HG2 THR C 147      28.798  14.165  23.043  1.00  0.00           H  \\nATOM   2041 2HG2 THR C 147      27.482  13.630  24.113  1.00  0.00           H  \\nATOM   2042 3HG2 THR C 147      27.135  14.072  22.441  1.00  0.00           H  \\nATOM   2043  N   LEU C 148      25.741  18.400  24.070  1.00  0.00           N  \\nATOM   2044  CA  LEU C 148      25.580  19.736  24.635  1.00  0.00           C  \\nATOM   2045  C   LEU C 148      24.374  19.792  25.569  1.00  0.00           C  \\nATOM   2046  O   LEU C 148      24.414  20.459  26.601  1.00  0.00           O  \\nATOM   2047  CB  LEU C 148      25.500  20.805  23.532  1.00  0.00           C  \\nATOM   2048  CG  LEU C 148      26.849  21.106  22.765  1.00  0.00           C  \\nATOM   2049  CD1 LEU C 148      26.568  22.016  21.569  1.00  0.00           C  \\nATOM   2050  CD2 LEU C 148      27.845  21.779  23.725  1.00  0.00           C  \\nATOM   2051  H   LEU C 148      25.667  18.368  23.214  1.00  0.00           H  \\nATOM   2052  HA  LEU C 148      26.370  19.958  25.152  1.00  0.00           H  \\nATOM   2053 1HB  LEU C 148      24.769  20.475  22.796  1.00  0.00           H  \\nATOM   2054 2HB  LEU C 148      25.148  21.736  23.973  1.00  0.00           H  \\nATOM   2055  HG  LEU C 148      27.283  20.182  22.401  1.00  0.00           H  \\nATOM   2056 1HD1 LEU C 148      27.501  22.219  21.040  1.00  0.00           H  \\nATOM   2057 2HD1 LEU C 148      25.870  21.521  20.896  1.00  0.00           H  \\nATOM   2058 3HD1 LEU C 148      26.138  22.952  21.914  1.00  0.00           H  \\nATOM   2059 1HD2 LEU C 148      28.774  21.989  23.197  1.00  0.00           H  \\nATOM   2060 2HD2 LEU C 148      27.425  22.711  24.095  1.00  0.00           H  \\nATOM   2061 3HD2 LEU C 148      28.046  21.112  24.563  1.00  0.00           H  \\nATOM   2062  N   LYS C 149      23.305  19.089  25.209  1.00  0.00           N  \\nATOM   2063  CA  LYS C 149      22.102  19.068  26.033  1.00  0.00           C  \\nATOM   2064  C   LYS C 149      22.358  18.315  27.333  1.00  0.00           C  \\nATOM   2065  O   LYS C 149      21.866  18.703  28.391  1.00  0.00           O  \\nATOM   2066  CB  LYS C 149      20.947  18.421  25.270  1.00  0.00           C  \\nATOM   2067  CG  LYS C 149      19.600  18.478  25.989  1.00  0.00           C  \\nATOM   2068  CD  LYS C 149      18.484  17.889  25.127  1.00  0.00           C  \\nATOM   2069  CE  LYS C 149      18.165  18.785  23.936  1.00  0.00           C  \\nATOM   2070  NZ  LYS C 149      16.999  18.279  23.157  1.00  0.00           N  \\nATOM   2071  H   LYS C 149      23.250  18.616  24.493  1.00  0.00           H  \\nATOM   2072  HA  LYS C 149      21.855  19.979  26.257  1.00  0.00           H  \\nATOM   2073 1HB  LYS C 149      20.842  18.904  24.297  1.00  0.00           H  \\nATOM   2074 2HB  LYS C 149      21.187  17.370  25.082  1.00  0.00           H  \\nATOM   2075 1HG  LYS C 149      19.668  17.910  26.922  1.00  0.00           H  \\nATOM   2076 2HG  LYS C 149      19.360  19.513  26.234  1.00  0.00           H  \\nATOM   2077 1HD  LYS C 149      18.795  16.909  24.759  1.00  0.00           H  \\nATOM   2078 2HD  LYS C 149      17.585  17.763  25.731  1.00  0.00           H  \\nATOM   2079 1HE  LYS C 149      17.942  19.789  24.294  1.00  0.00           H  \\nATOM   2080 2HE  LYS C 149      19.036  18.828  23.280  1.00  0.00           H  \\nATOM   2081 1HZ  LYS C 149      16.820  18.897  22.378  1.00  0.00           H  \\nATOM   2082 2HZ  LYS C 149      17.201  17.351  22.810  1.00  0.00           H  \\nATOM   2083 3HZ  LYS C 149      16.185  18.245  23.753  1.00  0.00           H  \\nATOM   2084  N   ARG C 150      23.123  17.231  27.249  1.00  0.00           N  \\nATOM   2085  CA  ARG C 150      23.506  16.481  28.438  1.00  0.00           C  \\nATOM   2086  C   ARG C 150      24.492  17.283  29.277  1.00  0.00           C  \\nATOM   2087  O   ARG C 150      24.459  17.240  30.507  1.00  0.00           O  \\nATOM   2088  CB  ARG C 150      24.156  15.166  28.052  1.00  0.00           C  \\nATOM   2089  CG  ARG C 150      23.241  14.106  27.505  1.00  0.00           C  \\nATOM   2090  CD  ARG C 150      24.021  13.032  26.841  1.00  0.00           C  \\nATOM   2091  NE  ARG C 150      24.995  12.412  27.707  1.00  0.00           N  \\nATOM   2092  CZ  ARG C 150      26.094  11.787  27.239  1.00  0.00           C  \\nATOM   2093  NH1 ARG C 150      26.292  11.705  25.933  1.00  0.00           N  \\nATOM   2094  NH2 ARG C 150      26.983  11.272  28.082  1.00  0.00           N  \\nATOM   2095  H   ARG C 150      23.432  16.910  26.514  1.00  0.00           H  \\nATOM   2096  HA  ARG C 150      22.717  16.311  28.976  1.00  0.00           H  \\nATOM   2097 1HB  ARG C 150      24.923  15.352  27.300  1.00  0.00           H  \\nATOM   2098 2HB  ARG C 150      24.649  14.741  28.918  1.00  0.00           H  \\nATOM   2099 1HG  ARG C 150      22.645  13.671  28.305  1.00  0.00           H  \\nATOM   2100 2HG  ARG C 150      22.581  14.549  26.752  1.00  0.00           H  \\nATOM   2101 1HD  ARG C 150      23.339  12.258  26.521  1.00  0.00           H  \\nATOM   2102 2HD  ARG C 150      24.538  13.440  25.970  1.00  0.00           H  \\nATOM   2103  HE  ARG C 150      24.860  12.470  28.705  1.00  0.00           H  \\nATOM   2104 1HH1 ARG C 150      25.590  12.078  25.259  1.00  0.00           H  \\nATOM   2105 2HH1 ARG C 150      27.158  11.275  25.583  1.00  0.00           H  \\nATOM   2106 1HH2 ARG C 150      26.837  11.340  29.078  1.00  0.00           H  \\nATOM   2107 2HH2 ARG C 150      27.841  10.812  27.722  1.00  0.00           H  \\nATOM   2108  N   LEU C 151      25.372  18.012  28.601  1.00  0.00           N  \\nATOM   2109  CA  LEU C 151      26.342  18.860  29.274  1.00  0.00           C  \\nATOM   2110  C   LEU C 151      25.602  19.964  30.026  1.00  0.00           C  \\nATOM   2111  O   LEU C 151      25.968  20.323  31.144  1.00  0.00           O  \\nATOM   2112  CB  LEU C 151      27.305  19.483  28.232  1.00  0.00           C  \\nATOM   2113  CG  LEU C 151      28.513  20.260  28.755  1.00  0.00           C  \\nATOM   2114  CD1 LEU C 151      29.370  19.309  29.563  1.00  0.00           C  \\nATOM   2115  CD2 LEU C 151      29.311  20.847  27.569  1.00  0.00           C  \\nATOM   2116  H   LEU C 151      25.427  18.031  27.743  1.00  0.00           H  \\nATOM   2117  HA  LEU C 151      26.841  18.332  29.917  1.00  0.00           H  \\nATOM   2118 1HB  LEU C 151      27.678  18.698  27.589  1.00  0.00           H  \\nATOM   2119 2HB  LEU C 151      26.733  20.170  27.620  1.00  0.00           H  \\nATOM   2120  HG  LEU C 151      28.173  21.063  29.415  1.00  0.00           H  \\nATOM   2121 1HD1 LEU C 151      30.232  19.836  29.972  1.00  0.00           H  \\nATOM   2122 2HD1 LEU C 151      28.776  18.900  30.359  1.00  0.00           H  \\nATOM   2123 3HD1 LEU C 151      29.712  18.501  28.926  1.00  0.00           H  \\nATOM   2124 1HD2 LEU C 151      30.180  21.392  27.942  1.00  0.00           H  \\nATOM   2125 2HD2 LEU C 151      29.649  20.037  26.916  1.00  0.00           H  \\nATOM   2126 3HD2 LEU C 151      28.674  21.524  27.001  1.00  0.00           H  \\nATOM   2127  N   GLU C 152      24.544  20.481  29.410  1.00  0.00           N  \\nATOM   2128  CA  GLU C 152      23.701  21.494  30.035  1.00  0.00           C  \\nATOM   2129  C   GLU C 152      23.030  20.945  31.288  1.00  0.00           C  \\nATOM   2130  O   GLU C 152      23.015  21.602  32.330  1.00  0.00           O  \\nATOM   2131  CB  GLU C 152      22.630  22.058  29.091  1.00  0.00           C  \\nATOM   2132  CG  GLU C 152      21.787  23.187  29.759  1.00  0.00           C  \\nATOM   2133  CD  GLU C 152      20.744  23.858  28.864  1.00  0.00           C  \\nATOM   2134  OE1 GLU C 152      20.618  23.502  27.718  1.00  0.00           O  \\nATOM   2135  OE2 GLU C 152      20.073  24.735  29.357  1.00  0.00           O  \\nATOM   2136  H   GLU C 152      24.291  20.259  28.618  1.00  0.00           H  \\nATOM   2137  HA  GLU C 152      24.252  22.250  30.294  1.00  0.00           H  \\nATOM   2138 1HB  GLU C 152      23.100  22.452  28.191  1.00  0.00           H  \\nATOM   2139 2HB  GLU C 152      21.952  21.263  28.789  1.00  0.00           H  \\nATOM   2140 1HG  GLU C 152      21.274  22.756  30.620  1.00  0.00           H  \\nATOM   2141 2HG  GLU C 152      22.472  23.947  30.132  1.00  0.00           H  \\nATOM   2142  N   ASP C 153      22.467  19.746  31.182  1.00  0.00           N  \\nATOM   2143  CA  ASP C 153      21.804  19.123  32.322  1.00  0.00           C  \\nATOM   2144  C   ASP C 153      22.816  18.853  33.429  1.00  0.00           C  \\nATOM   2145  O   ASP C 153      22.550  19.111  34.601  1.00  0.00           O  \\nATOM   2146  CB  ASP C 153      21.132  17.794  31.937  1.00  0.00           C  \\nATOM   2147  CG  ASP C 153      19.803  17.890  31.108  1.00  0.00           C  \\nATOM   2148  OD1 ASP C 153      19.195  18.954  31.022  1.00  0.00           O  \\nATOM   2149  OD2 ASP C 153      19.385  16.867  30.616  1.00  0.00           O  \\nATOM   2150  H   ASP C 153      22.454  19.274  30.463  1.00  0.00           H  \\nATOM   2151  HA  ASP C 153      21.132  19.731  32.669  1.00  0.00           H  \\nATOM   2152 1HB  ASP C 153      21.843  17.207  31.354  1.00  0.00           H  \\nATOM   2153 2HB  ASP C 153      20.927  17.229  32.844  1.00  0.00           H  \\nATOM   2154  N   SER C 154      23.985  18.350  33.047  1.00  0.00           N  \\nATOM   2155  CA  SER C 154      25.066  18.115  33.994  1.00  0.00           C  \\nATOM   2156  C   SER C 154      25.452  19.391  34.742  1.00  0.00           C  \\nATOM   2157  O   SER C 154      25.744  19.357  35.936  1.00  0.00           O  \\nATOM   2158  CB  SER C 154      26.257  17.521  33.296  1.00  0.00           C  \\nATOM   2159  OG  SER C 154      27.255  17.220  34.219  1.00  0.00           O  \\nATOM   2160  H   SER C 154      24.177  18.134  32.237  1.00  0.00           H  \\nATOM   2161  HA  SER C 154      24.776  17.460  34.648  1.00  0.00           H  \\nATOM   2162 1HB  SER C 154      25.952  16.622  32.772  1.00  0.00           H  \\nATOM   2163 2HB  SER C 154      26.633  18.220  32.556  1.00  0.00           H  \\nATOM   2164  HG  SER C 154      26.875  16.565  34.806  1.00  0.00           H  \\nATOM   2165  N   LEU C 155      25.456  20.516  34.037  1.00  0.00           N  \\nATOM   2166  CA  LEU C 155      25.805  21.787  34.656  1.00  0.00           C  \\nATOM   2167  C   LEU C 155      24.724  22.225  35.635  1.00  0.00           C  \\nATOM   2168  O   LEU C 155      25.029  22.751  36.704  1.00  0.00           O  \\nATOM   2169  CB  LEU C 155      26.092  22.861  33.597  1.00  0.00           C  \\nATOM   2170  CG  LEU C 155      27.435  22.690  32.813  1.00  0.00           C  \\nATOM   2171  CD1 LEU C 155      27.478  23.634  31.636  1.00  0.00           C  \\nATOM   2172  CD2 LEU C 155      28.601  23.010  33.749  1.00  0.00           C  \\nATOM   2173  H   LEU C 155      25.262  20.570  33.201  1.00  0.00           H  \\nATOM   2174  HA  LEU C 155      26.632  21.679  35.151  1.00  0.00           H  \\nATOM   2175 1HB  LEU C 155      25.282  22.845  32.868  1.00  0.00           H  \\nATOM   2176 2HB  LEU C 155      26.105  23.832  34.080  1.00  0.00           H  \\nATOM   2177  HG  LEU C 155      27.529  21.668  32.452  1.00  0.00           H  \\nATOM   2178 1HD1 LEU C 155      28.421  23.501  31.121  1.00  0.00           H  \\nATOM   2179 2HD1 LEU C 155      26.654  23.405  30.959  1.00  0.00           H  \\nATOM   2180 3HD1 LEU C 155      27.392  24.659  31.983  1.00  0.00           H  \\nATOM   2181 1HD2 LEU C 155      29.536  22.891  33.206  1.00  0.00           H  \\nATOM   2182 2HD2 LEU C 155      28.519  24.035  34.108  1.00  0.00           H  \\nATOM   2183 3HD2 LEU C 155      28.582  22.336  34.582  1.00  0.00           H  \\nATOM   2184  N   ARG C 156      23.464  22.005  35.274  1.00  0.00           N  \\nATOM   2185  CA  ARG C 156      22.361  22.335  36.166  1.00  0.00           C  \\nATOM   2186  C   ARG C 156      22.489  21.544  37.463  1.00  0.00           C  \\nATOM   2187  O   ARG C 156      22.236  22.068  38.546  1.00  0.00           O  \\nATOM   2188  CB  ARG C 156      21.025  22.069  35.485  1.00  0.00           C  \\nATOM   2189  CG  ARG C 156      19.795  22.546  36.253  1.00  0.00           C  \\nATOM   2190  CD  ARG C 156      18.547  22.499  35.409  1.00  0.00           C  \\nATOM   2191  NE  ARG C 156      18.165  21.145  35.026  1.00  0.00           N  \\nATOM   2192  CZ  ARG C 156      18.302  20.615  33.790  1.00  0.00           C  \\nATOM   2193  NH1 ARG C 156      18.838  21.290  32.804  1.00  0.00           N  \\nATOM   2194  NH2 ARG C 156      17.904  19.391  33.541  1.00  0.00           N  \\nATOM   2195  H   ARG C 156      23.223  21.668  34.520  1.00  0.00           H  \\nATOM   2196  HA  ARG C 156      22.401  23.280  36.384  1.00  0.00           H  \\nATOM   2197 1HB  ARG C 156      21.021  22.541  34.502  1.00  0.00           H  \\nATOM   2198 2HB  ARG C 156      20.914  20.997  35.325  1.00  0.00           H  \\nATOM   2199 1HG  ARG C 156      19.646  21.903  37.121  1.00  0.00           H  \\nATOM   2200 2HG  ARG C 156      19.950  23.574  36.585  1.00  0.00           H  \\nATOM   2201 1HD  ARG C 156      17.720  22.939  35.965  1.00  0.00           H  \\nATOM   2202 2HD  ARG C 156      18.713  23.073  34.497  1.00  0.00           H  \\nATOM   2203  HE  ARG C 156      17.748  20.561  35.736  1.00  0.00           H  \\nATOM   2204 1HH1 ARG C 156      19.168  22.233  32.938  1.00  0.00           H  \\nATOM   2205 2HH1 ARG C 156      18.946  20.804  31.897  1.00  0.00           H  \\nATOM   2206 1HH2 ARG C 156      17.490  18.822  34.259  1.00  0.00           H  \\nATOM   2207 2HH2 ARG C 156      18.038  19.038  32.582  1.00  0.00           H  \\nATOM   2208  N   GLU C 157      22.900  20.285  37.350  1.00  0.00           N  \\nATOM   2209  CA  GLU C 157      23.081  19.437  38.521  1.00  0.00           C  \\nATOM   2210  C   GLU C 157      24.189  19.979  39.419  1.00  0.00           C  \\nATOM   2211  O   GLU C 157      24.058  19.984  40.639  1.00  0.00           O  \\nATOM   2212  CB  GLU C 157      23.369  17.989  38.114  1.00  0.00           C  \\nATOM   2213  CG  GLU C 157      23.654  17.018  39.278  1.00  0.00           C  \\nATOM   2214  CD  GLU C 157      22.529  16.878  40.283  1.00  0.00           C  \\nATOM   2215  OE1 GLU C 157      21.440  17.347  40.044  1.00  0.00           O  \\nATOM   2216  OE2 GLU C 157      22.766  16.273  41.305  1.00  0.00           O  \\nATOM   2217  H   GLU C 157      23.081  19.898  36.603  1.00  0.00           H  \\nATOM   2218  HA  GLU C 157      22.258  19.425  39.034  1.00  0.00           H  \\nATOM   2219 1HB  GLU C 157      22.525  17.600  37.544  1.00  0.00           H  \\nATOM   2220 2HB  GLU C 157      24.234  17.969  37.458  1.00  0.00           H  \\nATOM   2221 1HG  GLU C 157      23.875  16.037  38.862  1.00  0.00           H  \\nATOM   2222 2HG  GLU C 157      24.549  17.366  39.797  1.00  0.00           H  \\nATOM   2223  N   LEU C 158      25.277  20.439  38.809  1.00  0.00           N  \\nATOM   2224  CA  LEU C 158      26.393  21.002  39.561  1.00  0.00           C  \\nATOM   2225  C   LEU C 158      25.968  22.231  40.356  1.00  0.00           C  \\nATOM   2226  O   LEU C 158      26.339  22.383  41.519  1.00  0.00           O  \\nATOM   2227  CB  LEU C 158      27.574  21.366  38.649  1.00  0.00           C  \\nATOM   2228  CG  LEU C 158      28.418  20.208  38.210  1.00  0.00           C  \\nATOM   2229  CD1 LEU C 158      29.398  20.598  37.099  1.00  0.00           C  \\nATOM   2230  CD2 LEU C 158      29.190  19.799  39.434  1.00  0.00           C  \\nATOM   2231  H   LEU C 158      25.394  20.436  37.957  1.00  0.00           H  \\nATOM   2232  HA  LEU C 158      26.719  20.338  40.189  1.00  0.00           H  \\nATOM   2233 1HB  LEU C 158      27.186  21.850  37.758  1.00  0.00           H  \\nATOM   2234 2HB  LEU C 158      28.201  22.069  39.172  1.00  0.00           H  \\nATOM   2235  HG  LEU C 158      27.787  19.387  37.852  1.00  0.00           H  \\nATOM   2236 1HD1 LEU C 158      30.011  19.738  36.840  1.00  0.00           H  \\nATOM   2237 2HD1 LEU C 158      28.836  20.904  36.236  1.00  0.00           H  \\nATOM   2238 3HD1 LEU C 158      30.040  21.407  37.422  1.00  0.00           H  \\nATOM   2239 1HD2 LEU C 158      28.497  19.523  40.224  1.00  0.00           H  \\nATOM   2240 2HD2 LEU C 158      29.805  18.987  39.200  1.00  0.00           H  \\nATOM   2241 3HD2 LEU C 158      29.806  20.623  39.773  1.00  0.00           H  \\nATOM   2242  N   ARG C 159      25.193  23.108  39.727  1.00  0.00           N  \\nATOM   2243  CA  ARG C 159      24.737  24.323  40.390  1.00  0.00           C  \\nATOM   2244  C   ARG C 159      23.801  24.005  41.552  1.00  0.00           C  \\nATOM   2245  O   ARG C 159      23.774  24.727  42.546  1.00  0.00           O  \\nATOM   2246  CB  ARG C 159      24.042  25.246  39.402  1.00  0.00           C  \\nATOM   2247  CG  ARG C 159      24.977  25.891  38.370  1.00  0.00           C  \\nATOM   2248  CD  ARG C 159      24.322  27.020  37.642  1.00  0.00           C  \\nATOM   2249  NE  ARG C 159      23.170  26.613  36.830  1.00  0.00           N  \\nATOM   2250  CZ  ARG C 159      23.221  26.205  35.542  1.00  0.00           C  \\nATOM   2251  NH1 ARG C 159      24.368  26.133  34.908  1.00  0.00           N  \\nATOM   2252  NH2 ARG C 159      22.102  25.891  34.907  1.00  0.00           N  \\nATOM   2253  H   ARG C 159      24.918  23.023  38.917  1.00  0.00           H  \\nATOM   2254  HA  ARG C 159      25.506  24.794  40.747  1.00  0.00           H  \\nATOM   2255 1HB  ARG C 159      23.292  24.673  38.851  1.00  0.00           H  \\nATOM   2256 2HB  ARG C 159      23.525  26.038  39.938  1.00  0.00           H  \\nATOM   2257 1HG  ARG C 159      25.863  26.275  38.876  1.00  0.00           H  \\nATOM   2258 2HG  ARG C 159      25.277  25.140  37.639  1.00  0.00           H  \\nATOM   2259 1HD  ARG C 159      23.979  27.759  38.365  1.00  0.00           H  \\nATOM   2260 2HD  ARG C 159      25.052  27.482  36.978  1.00  0.00           H  \\nATOM   2261  HE  ARG C 159      22.260  26.651  37.269  1.00  0.00           H  \\nATOM   2262 1HH1 ARG C 159      25.224  26.365  35.391  1.00  0.00           H  \\nATOM   2263 2HH1 ARG C 159      24.383  25.850  33.938  1.00  0.00           H  \\nATOM   2264 1HH2 ARG C 159      21.216  25.959  35.388  1.00  0.00           H  \\nATOM   2265 2HH2 ARG C 159      22.136  25.594  33.940  1.00  0.00           H  \\nATOM   2266  N   ARG C 160      23.040  22.921  41.431  1.00  0.00           N  \\nATOM   2267  CA  ARG C 160      22.124  22.515  42.493  1.00  0.00           C  \\nATOM   2268  C   ARG C 160      22.911  22.026  43.704  1.00  0.00           C  \\nATOM   2269  O   ARG C 160      22.635  22.420  44.836  1.00  0.00           O  \\nATOM   2270  CB  ARG C 160      21.206  21.390  42.026  1.00  0.00           C  \\nATOM   2271  CG  ARG C 160      20.144  20.961  43.044  1.00  0.00           C  \\nATOM   2272  CD  ARG C 160      19.371  19.754  42.591  1.00  0.00           C  \\nATOM   2273  NE  ARG C 160      20.184  18.538  42.616  1.00  0.00           N  \\nATOM   2274  CZ  ARG C 160      20.428  17.814  43.738  1.00  0.00           C  \\nATOM   2275  NH1 ARG C 160      19.913  18.187  44.900  1.00  0.00           N  \\nATOM   2276  NH2 ARG C 160      21.180  16.735  43.674  1.00  0.00           N  \\nATOM   2277  H   ARG C 160      23.035  22.404  40.744  1.00  0.00           H  \\nATOM   2278  HA  ARG C 160      21.591  23.278  42.765  1.00  0.00           H  \\nATOM   2279 1HB  ARG C 160      20.701  21.691  41.108  1.00  0.00           H  \\nATOM   2280 2HB  ARG C 160      21.805  20.508  41.787  1.00  0.00           H  \\nATOM   2281 1HG  ARG C 160      20.637  20.714  43.986  1.00  0.00           H  \\nATOM   2282 2HG  ARG C 160      19.443  21.781  43.207  1.00  0.00           H  \\nATOM   2283 1HD  ARG C 160      18.510  19.603  43.240  1.00  0.00           H  \\nATOM   2284 2HD  ARG C 160      19.033  19.910  41.567  1.00  0.00           H  \\nATOM   2285  HE  ARG C 160      20.598  18.199  41.726  1.00  0.00           H  \\nATOM   2286 1HH1 ARG C 160      19.338  19.015  44.956  1.00  0.00           H  \\nATOM   2287 2HH1 ARG C 160      20.099  17.644  45.735  1.00  0.00           H  \\nATOM   2288 1HH2 ARG C 160      21.602  16.460  42.769  1.00  0.00           H  \\nATOM   2289 2HH2 ARG C 160      21.359  16.199  44.501  1.00  0.00           H  \\nATOM   2290  N   ILE C 161      23.895  21.168  43.457  1.00  0.00           N  \\nATOM   2291  CA  ILE C 161      24.725  20.619  44.525  1.00  0.00           C  \\nATOM   2292  C   ILE C 161      25.538  21.719  45.193  1.00  0.00           C  \\nATOM   2293  O   ILE C 161      25.772  21.684  46.403  1.00  0.00           O  \\nATOM   2294  CB  ILE C 161      25.709  19.551  44.020  1.00  0.00           C  \\nATOM   2295  CG1 ILE C 161      24.977  18.311  43.545  1.00  0.00           C  \\nATOM   2296  CG2 ILE C 161      26.643  19.169  45.160  1.00  0.00           C  \\nATOM   2297  CD1 ILE C 161      25.884  17.367  42.805  1.00  0.00           C  \\nATOM   2298  H   ILE C 161      24.104  20.885  42.672  1.00  0.00           H  \\nATOM   2299  HA  ILE C 161      24.152  20.217  45.197  1.00  0.00           H  \\nATOM   2300  HB  ILE C 161      26.284  19.946  43.178  1.00  0.00           H  \\nATOM   2301 1HG1 ILE C 161      24.550  17.791  44.401  1.00  0.00           H  \\nATOM   2302 2HG1 ILE C 161      24.162  18.605  42.872  1.00  0.00           H  \\nATOM   2303 1HG2 ILE C 161      27.338  18.415  44.822  1.00  0.00           H  \\nATOM   2304 2HG2 ILE C 161      27.190  20.048  45.500  1.00  0.00           H  \\nATOM   2305 3HG2 ILE C 161      26.052  18.771  45.976  1.00  0.00           H  \\nATOM   2306 1HD1 ILE C 161      25.313  16.497  42.474  1.00  0.00           H  \\nATOM   2307 2HD1 ILE C 161      26.290  17.887  41.938  1.00  0.00           H  \\nATOM   2308 3HD1 ILE C 161      26.695  17.046  43.452  1.00  0.00           H  \\nATOM   2309  N   LEU C 162      25.970  22.694  44.402  1.00  0.00           N  \\nATOM   2310  CA  LEU C 162      26.759  23.794  44.932  1.00  0.00           C  \\nATOM   2311  C   LEU C 162      25.890  24.668  45.828  1.00  0.00           C  \\nATOM   2312  O   LEU C 162      26.356  25.182  46.844  1.00  0.00           O  \\nATOM   2313  CB  LEU C 162      27.382  24.608  43.801  1.00  0.00           C  \\nATOM   2314  CG  LEU C 162      28.306  25.752  44.229  1.00  0.00           C  \\nATOM   2315  CD1 LEU C 162      29.467  25.215  45.096  1.00  0.00           C  \\nATOM   2316  CD2 LEU C 162      28.838  26.420  42.988  1.00  0.00           C  \\nATOM   2317  H   LEU C 162      25.819  22.741  43.557  1.00  0.00           H  \\nATOM   2318  HA  LEU C 162      27.483  23.437  45.469  1.00  0.00           H  \\nATOM   2319 1HB  LEU C 162      27.948  23.934  43.159  1.00  0.00           H  \\nATOM   2320 2HB  LEU C 162      26.572  25.037  43.209  1.00  0.00           H  \\nATOM   2321  HG  LEU C 162      27.741  26.471  44.817  1.00  0.00           H  \\nATOM   2322 1HD1 LEU C 162      30.123  26.036  45.392  1.00  0.00           H  \\nATOM   2323 2HD1 LEU C 162      29.065  24.740  45.985  1.00  0.00           H  \\nATOM   2324 3HD1 LEU C 162      30.031  24.497  44.531  1.00  0.00           H  \\nATOM   2325 1HD2 LEU C 162      29.472  27.239  43.284  1.00  0.00           H  \\nATOM   2326 2HD2 LEU C 162      29.409  25.700  42.398  1.00  0.00           H  \\nATOM   2327 3HD2 LEU C 162      28.008  26.799  42.393  1.00  0.00           H  \\nATOM   2328  N   GLU C 163      24.624  24.830  45.450  1.00  0.00           N  \\nATOM   2329  CA  GLU C 163      23.677  25.573  46.275  1.00  0.00           C  \\nATOM   2330  C   GLU C 163      23.535  24.886  47.629  1.00  0.00           C  \\nATOM   2331  O   GLU C 163      23.550  25.537  48.670  1.00  0.00           O  \\nATOM   2332  CB  GLU C 163      22.335  25.706  45.518  1.00  0.00           C  \\nATOM   2333  CG  GLU C 163      21.268  26.689  46.078  1.00  0.00           C  \\nATOM   2334  CD  GLU C 163      20.288  26.107  47.082  1.00  0.00           C  \\nATOM   2335  OE1 GLU C 163      20.022  24.931  47.032  1.00  0.00           O  \\nATOM   2336  OE2 GLU C 163      19.766  26.865  47.862  1.00  0.00           O  \\nATOM   2337  H   GLU C 163      24.289  24.519  44.722  1.00  0.00           H  \\nATOM   2338  HA  GLU C 163      24.018  26.469  46.424  1.00  0.00           H  \\nATOM   2339 1HB  GLU C 163      22.538  26.002  44.491  1.00  0.00           H  \\nATOM   2340 2HB  GLU C 163      21.860  24.731  45.473  1.00  0.00           H  \\nATOM   2341 1HG  GLU C 163      21.791  27.518  46.555  1.00  0.00           H  \\nATOM   2342 2HG  GLU C 163      20.708  27.094  45.237  1.00  0.00           H  \\nATOM   2343  N   GLU C 164      23.400  23.564  47.602  1.00  0.00           N  \\nATOM   2344  CA  GLU C 164      23.300  22.773  48.823  1.00  0.00           C  \\nATOM   2345  C   GLU C 164      24.591  22.854  49.630  1.00  0.00           C  \\nATOM   2346  O   GLU C 164      24.569  22.850  50.860  1.00  0.00           O  \\nATOM   2347  CB  GLU C 164      22.987  21.301  48.480  1.00  0.00           C  \\nATOM   2348  CG  GLU C 164      22.746  20.382  49.689  1.00  0.00           C  \\nATOM   2349  CD  GLU C 164      22.296  18.959  49.315  1.00  0.00           C  \\nATOM   2350  OE1 GLU C 164      22.184  18.663  48.146  1.00  0.00           O  \\nATOM   2351  OE2 GLU C 164      22.048  18.185  50.210  1.00  0.00           O  \\nATOM   2352  H   GLU C 164      23.364  23.097  46.881  1.00  0.00           H  \\nATOM   2353  HA  GLU C 164      22.579  23.121  49.370  1.00  0.00           H  \\nATOM   2354 1HB  GLU C 164      22.110  21.260  47.834  1.00  0.00           H  \\nATOM   2355 2HB  GLU C 164      23.820  20.881  47.918  1.00  0.00           H  \\nATOM   2356 1HG  GLU C 164      23.679  20.311  50.236  1.00  0.00           H  \\nATOM   2357 2HG  GLU C 164      22.007  20.839  50.345  1.00  0.00           H  \\nATOM   2358  N   LEU C 165      25.717  22.926  48.931  1.00  0.00           N  \\nATOM   2359  CA  LEU C 165      27.015  22.974  49.587  1.00  0.00           C  \\nATOM   2360  C   LEU C 165      27.197  24.301  50.312  1.00  0.00           C  \\nATOM   2361  O   LEU C 165      27.856  24.363  51.347  1.00  0.00           O  \\nATOM   2362  CB  LEU C 165      28.118  22.693  48.560  1.00  0.00           C  \\nATOM   2363  CG  LEU C 165      29.530  22.558  49.091  1.00  0.00           C  \\nATOM   2364  CD1 LEU C 165      29.597  21.467  50.178  1.00  0.00           C  \\nATOM   2365  CD2 LEU C 165      30.460  22.174  47.903  1.00  0.00           C  \\nATOM   2366  H   LEU C 165      25.756  22.948  48.072  1.00  0.00           H  \\nATOM   2367  HA  LEU C 165      27.065  22.262  50.243  1.00  0.00           H  \\nATOM   2368 1HB  LEU C 165      27.867  21.783  48.017  1.00  0.00           H  \\nATOM   2369 2HB  LEU C 165      28.122  23.514  47.855  1.00  0.00           H  \\nATOM   2370  HG  LEU C 165      29.833  23.502  49.530  1.00  0.00           H  \\nATOM   2371 1HD1 LEU C 165      30.613  21.401  50.545  1.00  0.00           H  \\nATOM   2372 2HD1 LEU C 165      28.933  21.728  51.004  1.00  0.00           H  \\nATOM   2373 3HD1 LEU C 165      29.302  20.514  49.767  1.00  0.00           H  \\nATOM   2374 1HD2 LEU C 165      31.481  22.068  48.249  1.00  0.00           H  \\nATOM   2375 2HD2 LEU C 165      30.130  21.224  47.476  1.00  0.00           H  \\nATOM   2376 3HD2 LEU C 165      30.417  22.948  47.137  1.00  0.00           H  \\nATOM   2377  N   LYS C 166      26.598  25.358  49.773  1.00  0.00           N  \\nATOM   2378  CA  LYS C 166      26.702  26.682  50.374  1.00  0.00           C  \\nATOM   2379  C   LYS C 166      25.693  26.866  51.500  1.00  0.00           C  \\nATOM   2380  O   LYS C 166      25.907  27.668  52.408  1.00  0.00           O  \\nATOM   2381  CB  LYS C 166      26.514  27.741  49.294  1.00  0.00           C  \\nATOM   2382  CG  LYS C 166      27.687  27.864  48.312  1.00  0.00           C  \\nATOM   2383  CD  LYS C 166      27.338  28.806  47.172  1.00  0.00           C  \\nATOM   2384  CE  LYS C 166      28.536  29.101  46.298  1.00  0.00           C  \\nATOM   2385  NZ  LYS C 166      29.420  30.096  46.938  1.00  0.00           N  \\nATOM   2386  H   LYS C 166      26.123  25.333  49.056  1.00  0.00           H  \\nATOM   2387  HA  LYS C 166      27.591  26.788  50.749  1.00  0.00           H  \\nATOM   2388 1HB  LYS C 166      25.619  27.496  48.712  1.00  0.00           H  \\nATOM   2389 2HB  LYS C 166      26.350  28.711  49.757  1.00  0.00           H  \\nATOM   2390 1HG  LYS C 166      28.566  28.240  48.835  1.00  0.00           H  \\nATOM   2391 2HG  LYS C 166      27.925  26.881  47.902  1.00  0.00           H  \\nATOM   2392 1HD  LYS C 166      26.561  28.341  46.559  1.00  0.00           H  \\nATOM   2393 2HD  LYS C 166      26.950  29.741  47.573  1.00  0.00           H  \\nATOM   2394 1HE  LYS C 166      29.102  28.186  46.141  1.00  0.00           H  \\nATOM   2395 2HE  LYS C 166      28.209  29.484  45.331  1.00  0.00           H  \\nATOM   2396 1HZ  LYS C 166      28.911  30.955  47.068  1.00  0.00           H  \\nATOM   2397 2HZ  LYS C 166      29.746  29.768  47.826  1.00  0.00           H  \\nATOM   2398 3HZ  LYS C 166      30.224  30.277  46.325  1.00  0.00           H  \\nATOM   2399  N   GLU C 167      24.592  26.124  51.436  1.00  0.00           N  \\nATOM   2400  CA  GLU C 167      23.594  26.149  52.497  1.00  0.00           C  \\nATOM   2401  C   GLU C 167      24.153  25.481  53.748  1.00  0.00           C  \\nATOM   2402  O   GLU C 167      23.964  25.965  54.864  1.00  0.00           O  \\nATOM   2403  CB  GLU C 167      22.275  25.501  52.076  1.00  0.00           C  \\nATOM   2404  CG  GLU C 167      21.429  26.354  51.126  1.00  0.00           C  \\nATOM   2405  CD  GLU C 167      20.896  27.623  51.800  1.00  0.00           C  \\nATOM   2406  OE1 GLU C 167      21.089  28.698  51.275  1.00  0.00           O  \\nATOM   2407  OE2 GLU C 167      20.294  27.502  52.847  1.00  0.00           O  \\nATOM   2408  H   GLU C 167      24.399  25.596  50.785  1.00  0.00           H  \\nATOM   2409  HA  GLU C 167      23.379  27.070  52.714  1.00  0.00           H  \\nATOM   2410 1HB  GLU C 167      22.484  24.555  51.574  1.00  0.00           H  \\nATOM   2411 2HB  GLU C 167      21.680  25.279  52.959  1.00  0.00           H  \\nATOM   2412 1HG  GLU C 167      22.036  26.651  50.278  1.00  0.00           H  \\nATOM   2413 2HG  GLU C 167      20.597  25.761  50.747  1.00  0.00           H  \\nATOM   2414  N   MET C 168      24.854  24.371  53.547  1.00  0.00           N  \\nATOM   2415  CA  MET C 168      25.421  23.608  54.650  1.00  0.00           C  \\nATOM   2416  C   MET C 168      26.681  24.258  55.203  1.00  0.00           C  \\nATOM   2417  O   MET C 168      27.083  23.993  56.336  1.00  0.00           O  \\nATOM   2418  CB  MET C 168      25.773  22.267  54.096  1.00  0.00           C  \\nATOM   2419  CG  MET C 168      24.617  21.459  53.719  1.00  0.00           C  \\nATOM   2420  SD  MET C 168      25.149  20.117  52.804  1.00  0.00           S  \\nATOM   2421  CE  MET C 168      26.159  19.396  53.983  1.00  0.00           C  \\nATOM   2422  H   MET C 168      25.016  24.035  52.772  1.00  0.00           H  \\nATOM   2423  HA  MET C 168      24.763  23.551  55.360  1.00  0.00           H  \\nATOM   2424 1HB  MET C 168      26.396  22.396  53.205  1.00  0.00           H  \\nATOM   2425 2HB  MET C 168      26.353  21.712  54.829  1.00  0.00           H  \\nATOM   2426 1HG  MET C 168      24.100  21.100  54.606  1.00  0.00           H  \\nATOM   2427 2HG  MET C 168      23.921  22.044  53.119  1.00  0.00           H  \\nATOM   2428 1HE  MET C 168      26.593  18.507  53.560  1.00  0.00           H  \\nATOM   2429 2HE  MET C 168      26.934  20.089  54.255  1.00  0.00           H  \\nATOM   2430 3HE  MET C 168      25.577  19.154  54.862  1.00  0.00           H  \\nATOM   2431  N   LEU C 169      27.308  25.100  54.393  1.00  0.00           N  \\nATOM   2432  CA  LEU C 169      28.496  25.820  54.820  1.00  0.00           C  \\nATOM   2433  C   LEU C 169      28.123  26.874  55.856  1.00  0.00           C  \\nATOM   2434  O   LEU C 169      28.907  27.181  56.754  1.00  0.00           O  \\nATOM   2435  CB  LEU C 169      29.198  26.463  53.612  1.00  0.00           C  \\nATOM   2436  CG  LEU C 169      30.626  26.992  53.847  1.00  0.00           C  \\nATOM   2437  CD1 LEU C 169      31.524  25.808  54.288  1.00  0.00           C  \\nATOM   2438  CD2 LEU C 169      31.152  27.628  52.560  1.00  0.00           C  \\nATOM   2439  H   LEU C 169      27.064  25.273  53.587  1.00  0.00           H  \\nATOM   2440  HA  LEU C 169      29.118  25.198  55.229  1.00  0.00           H  \\nATOM   2441 1HB  LEU C 169      29.257  25.729  52.819  1.00  0.00           H  \\nATOM   2442 2HB  LEU C 169      28.583  27.294  53.261  1.00  0.00           H  \\nATOM   2443  HG  LEU C 169      30.618  27.731  54.646  1.00  0.00           H  \\nATOM   2444 1HD1 LEU C 169      31.118  25.374  55.199  1.00  0.00           H  \\nATOM   2445 2HD1 LEU C 169      31.537  25.050  53.504  1.00  0.00           H  \\nATOM   2446 3HD1 LEU C 169      32.538  26.152  54.470  1.00  0.00           H  \\nATOM   2447 1HD2 LEU C 169      32.163  27.997  52.714  1.00  0.00           H  \\nATOM   2448 2HD2 LEU C 169      31.157  26.888  51.775  1.00  0.00           H  \\nATOM   2449 3HD2 LEU C 169      30.501  28.453  52.274  1.00  0.00           H  \\nATOM   2450  N   GLU C 170      26.920  27.424  55.726  1.00  0.00           N  \\nATOM   2451  CA  GLU C 170      26.431  28.425  56.666  1.00  0.00           C  \\nATOM   2452  C   GLU C 170      26.093  27.779  58.009  1.00  0.00           C  \\nATOM   2453  O   GLU C 170      26.188  28.422  59.054  1.00  0.00           O  \\nATOM   2454  CB  GLU C 170      25.178  29.072  56.059  1.00  0.00           C  \\nATOM   2455  CG  GLU C 170      24.466  30.130  56.887  1.00  0.00           C  \\nATOM   2456  CD  GLU C 170      25.205  31.402  57.127  1.00  0.00           C  \\nATOM   2457  OE1 GLU C 170      24.842  32.070  58.082  1.00  0.00           O  \\nATOM   2458  OE2 GLU C 170      26.081  31.734  56.375  1.00  0.00           O  \\nATOM   2459  H   GLU C 170      26.365  27.233  55.098  1.00  0.00           H  \\nATOM   2460  HA  GLU C 170      27.125  29.085  56.816  1.00  0.00           H  \\nATOM   2461 1HB  GLU C 170      25.441  29.523  55.101  1.00  0.00           H  \\nATOM   2462 2HB  GLU C 170      24.449  28.293  55.849  1.00  0.00           H  \\nATOM   2463 1HG  GLU C 170      23.522  30.366  56.398  1.00  0.00           H  \\nATOM   2464 2HG  GLU C 170      24.234  29.672  57.844  1.00  0.00           H  \\nATOM   2465  N   ARG C 171      25.694  26.509  57.975  1.00  0.00           N  \\nATOM   2466  CA  ARG C 171      25.462  25.748  59.200  1.00  0.00           C  \\nATOM   2467  C   ARG C 171      26.769  25.590  59.961  1.00  0.00           C  \\nATOM   2468  O   ARG C 171      26.818  25.729  61.183  1.00  0.00           O  \\nATOM   2469  CB  ARG C 171      24.893  24.367  58.904  1.00  0.00           C  \\nATOM   2470  CG  ARG C 171      23.465  24.352  58.386  1.00  0.00           C  \\nATOM   2471  CD  ARG C 171      22.990  22.958  58.143  1.00  0.00           C  \\nATOM   2472  NE  ARG C 171      21.649  22.925  57.568  1.00  0.00           N  \\nATOM   2473  CZ  ARG C 171      20.502  22.979  58.278  1.00  0.00           C  \\nATOM   2474  NH1 ARG C 171      20.540  23.063  59.590  1.00  0.00           N  \\nATOM   2475  NH2 ARG C 171      19.337  22.945  57.653  1.00  0.00           N  \\nATOM   2476  H   ARG C 171      25.550  26.064  57.253  1.00  0.00           H  \\nATOM   2477  HA  ARG C 171      24.835  26.228  59.763  1.00  0.00           H  \\nATOM   2478 1HB  ARG C 171      25.511  23.859  58.182  1.00  0.00           H  \\nATOM   2479 2HB  ARG C 171      24.916  23.771  59.817  1.00  0.00           H  \\nATOM   2480 1HG  ARG C 171      22.806  24.826  59.110  1.00  0.00           H  \\nATOM   2481 2HG  ARG C 171      23.425  24.900  57.439  1.00  0.00           H  \\nATOM   2482 1HD  ARG C 171      23.671  22.462  57.449  1.00  0.00           H  \\nATOM   2483 2HD  ARG C 171      22.972  22.410  59.084  1.00  0.00           H  \\nATOM   2484  HE  ARG C 171      21.573  22.864  56.561  1.00  0.00           H  \\nATOM   2485 1HH1 ARG C 171      21.429  23.088  60.069  1.00  0.00           H  \\nATOM   2486 2HH1 ARG C 171      19.681  23.102  60.118  1.00  0.00           H  \\nATOM   2487 1HH2 ARG C 171      19.306  22.880  56.645  1.00  0.00           H  \\nATOM   2488 2HH2 ARG C 171      18.479  22.985  58.183  1.00  0.00           H  \\nATOM   2489  N   LEU C 172      27.823  25.284  59.213  1.00  0.00           N  \\nATOM   2490  CA  LEU C 172      29.153  25.088  59.768  1.00  0.00           C  \\nATOM   2491  C   LEU C 172      29.622  26.320  60.537  1.00  0.00           C  \\nATOM   2492  O   LEU C 172      30.294  26.208  61.562  1.00  0.00           O  \\nATOM   2493  CB  LEU C 172      30.124  24.761  58.613  1.00  0.00           C  \\nATOM   2494  CG  LEU C 172      31.525  24.300  58.975  1.00  0.00           C  \\nATOM   2495  CD1 LEU C 172      31.442  22.981  59.724  1.00  0.00           C  \\nATOM   2496  CD2 LEU C 172      32.347  24.145  57.704  1.00  0.00           C  \\nATOM   2497  H   LEU C 172      27.790  25.183  58.359  1.00  0.00           H  \\nATOM   2498  HA  LEU C 172      29.136  24.337  60.381  1.00  0.00           H  \\nATOM   2499 1HB  LEU C 172      29.665  24.001  57.978  1.00  0.00           H  \\nATOM   2500 2HB  LEU C 172      30.252  25.655  58.022  1.00  0.00           H  \\nATOM   2501  HG  LEU C 172      31.988  25.037  59.633  1.00  0.00           H  \\nATOM   2502 1HD1 LEU C 172      32.440  22.667  59.998  1.00  0.00           H  \\nATOM   2503 2HD1 LEU C 172      30.849  23.098  60.605  1.00  0.00           H  \\nATOM   2504 3HD1 LEU C 172      30.983  22.226  59.085  1.00  0.00           H  \\nATOM   2505 1HD2 LEU C 172      33.342  23.824  57.960  1.00  0.00           H  \\nATOM   2506 2HD2 LEU C 172      31.880  23.407  57.051  1.00  0.00           H  \\nATOM   2507 3HD2 LEU C 172      32.399  25.103  57.191  1.00  0.00           H  \\nATOM   2508  N   GLU C 173      29.266  27.495  60.027  1.00  0.00           N  \\nATOM   2509  CA  GLU C 173      29.635  28.761  60.656  1.00  0.00           C  \\nATOM   2510  C   GLU C 173      28.926  28.968  61.992  1.00  0.00           C  \\nATOM   2511  O   GLU C 173      29.563  29.288  62.995  1.00  0.00           O  \\nATOM   2512  CB  GLU C 173      29.409  29.940  59.703  1.00  0.00           C  \\nATOM   2513  CG  GLU C 173      29.839  31.297  60.292  1.00  0.00           C  \\nATOM   2514  CD  GLU C 173      29.896  32.431  59.277  1.00  0.00           C  \\nATOM   2515  OE1 GLU C 173      29.873  33.567  59.686  1.00  0.00           O  \\nATOM   2516  OE2 GLU C 173      30.031  32.154  58.102  1.00  0.00           O  \\nATOM   2517  H   GLU C 173      28.804  27.588  59.308  1.00  0.00           H  \\nATOM   2518  HA  GLU C 173      30.590  28.763  60.821  1.00  0.00           H  \\nATOM   2519 1HB  GLU C 173      29.959  29.772  58.777  1.00  0.00           H  \\nATOM   2520 2HB  GLU C 173      28.349  30.002  59.446  1.00  0.00           H  \\nATOM   2521 1HG  GLU C 173      29.131  31.568  61.076  1.00  0.00           H  \\nATOM   2522 2HG  GLU C 173      30.818  31.181  60.753  1.00  0.00           H  \\nATOM   2523  N   LYS C 174      27.606  28.804  61.989  1.00  0.00           N  \\nATOM   2524  CA  LYS C 174      26.794  28.975  63.193  1.00  0.00           C  \\nATOM   2525  C   LYS C 174      27.275  28.090  64.340  1.00  0.00           C  \\nATOM   2526  O   LYS C 174      27.157  28.451  65.511  1.00  0.00           O  \\nATOM   2527  CB  LYS C 174      25.325  28.720  62.880  1.00  0.00           C  \\nATOM   2528  CG  LYS C 174      24.703  29.859  62.090  1.00  0.00           C  \\nATOM   2529  CD  LYS C 174      23.224  29.671  61.840  1.00  0.00           C  \\nATOM   2530  CE  LYS C 174      22.642  30.948  61.256  1.00  0.00           C  \\nATOM   2531  NZ  LYS C 174      23.312  31.295  59.995  1.00  0.00           N  \\nATOM   2532  H   LYS C 174      27.149  28.590  61.293  1.00  0.00           H  \\nATOM   2533  HA  LYS C 174      26.852  29.899  63.483  1.00  0.00           H  \\nATOM   2534 1HB  LYS C 174      25.235  27.800  62.292  1.00  0.00           H  \\nATOM   2535 2HB  LYS C 174      24.768  28.580  63.804  1.00  0.00           H  \\nATOM   2536 1HG  LYS C 174      24.857  30.796  62.622  1.00  0.00           H  \\nATOM   2537 2HG  LYS C 174      25.210  29.929  61.119  1.00  0.00           H  \\nATOM   2538 1HD  LYS C 174      23.078  28.851  61.133  1.00  0.00           H  \\nATOM   2539 2HD  LYS C 174      22.715  29.429  62.772  1.00  0.00           H  \\nATOM   2540 1HE  LYS C 174      21.578  30.815  61.068  1.00  0.00           H  \\nATOM   2541 2HE  LYS C 174      22.779  31.764  61.965  1.00  0.00           H  \\nATOM   2542 1HZ  LYS C 174      23.181  30.539  59.342  1.00  0.00           H  \\nATOM   2543 2HZ  LYS C 174      22.949  32.143  59.591  1.00  0.00           H  \\nATOM   2544 3HZ  LYS C 174      24.309  31.418  60.175  1.00  0.00           H  \\nATOM   2545  N   ASN C 175      27.816  26.928  63.990  1.00  0.00           N  \\nATOM   2546  CA  ASN C 175      28.257  25.949  64.975  1.00  0.00           C  \\nATOM   2547  C   ASN C 175      29.037  24.821  64.309  1.00  0.00           C  \\nATOM   2548  O   ASN C 175      28.447  23.835  63.869  1.00  0.00           O  \\nATOM   2549  CB  ASN C 175      27.102  25.364  65.779  1.00  0.00           C  \\nATOM   2550  CG  ASN C 175      26.110  24.586  64.964  1.00  0.00           C  \\nATOM   2551  OD1 ASN C 175      25.413  25.125  64.091  1.00  0.00           O  \\nATOM   2552  ND2 ASN C 175      26.033  23.311  65.235  1.00  0.00           N  \\nATOM   2553  H   ASN C 175      27.939  26.680  63.176  1.00  0.00           H  \\nATOM   2554  HA  ASN C 175      28.842  26.384  65.615  1.00  0.00           H  \\nATOM   2555 1HB  ASN C 175      27.511  24.688  66.535  1.00  0.00           H  \\nATOM   2556 2HB  ASN C 175      26.574  26.151  66.310  1.00  0.00           H  \\nATOM   2557 1HD2 ASN C 175      26.641  22.887  65.952  1.00  0.00           H  \\nATOM   2558 2HD2 ASN C 175      25.401  22.725  64.733  1.00  0.00           H  \\nATOM   2559  N   PRO C 176      30.371  24.958  64.231  1.00  0.00           N  \\nATOM   2560  CA  PRO C 176      31.144  23.927  63.536  1.00  0.00           C  \\nATOM   2561  C   PRO C 176      31.202  22.619  64.321  1.00  0.00           C  \\nATOM   2562  O   PRO C 176      32.216  22.320  64.951  1.00  0.00           O  \\nATOM   2563  CB  PRO C 176      32.501  24.622  63.399  1.00  0.00           C  \\nATOM   2564  CG  PRO C 176      32.583  25.555  64.614  1.00  0.00           C  \\nATOM   2565  CD  PRO C 176      31.138  26.060  64.820  1.00  0.00           C  \\nATOM   2566  HA  PRO C 176      30.778  23.762  62.653  1.00  0.00           H  \\nATOM   2567 1HB  PRO C 176      33.307  23.873  63.365  1.00  0.00           H  \\nATOM   2568 2HB  PRO C 176      32.526  25.171  62.446  1.00  0.00           H  \\nATOM   2569 1HG  PRO C 176      32.963  25.017  65.482  1.00  0.00           H  \\nATOM   2570 2HG  PRO C 176      33.292  26.374  64.412  1.00  0.00           H  \\nATOM   2571 1HD  PRO C 176      30.958  26.156  65.897  1.00  0.00           H  \\nATOM   2572 2HD  PRO C 176      30.949  27.001  64.265  1.00  0.00           H  \\nATOM   2573  N   ASP C 177      30.115  21.852  64.289  1.00  0.00           N  \\nATOM   2574  CA  ASP C 177      30.104  20.520  64.885  1.00  0.00           C  \\nATOM   2575  C   ASP C 177      31.015  19.597  64.084  1.00  0.00           C  \\nATOM   2576  O   ASP C 177      31.678  20.034  63.145  1.00  0.00           O  \\nATOM   2577  CB  ASP C 177      28.698  19.892  64.897  1.00  0.00           C  \\nATOM   2578  CG  ASP C 177      27.635  20.556  65.806  1.00  0.00           C  \\nATOM   2579  OD1 ASP C 177      27.932  21.451  66.574  1.00  0.00           O  \\nATOM   2580  OD2 ASP C 177      26.506  20.114  65.725  1.00  0.00           O  \\nATOM   2581  H   ASP C 177      29.370  22.081  63.926  1.00  0.00           H  \\nATOM   2582  HA  ASP C 177      30.444  20.571  65.792  1.00  0.00           H  \\nATOM   2583 1HB  ASP C 177      28.324  19.905  63.895  1.00  0.00           H  \\nATOM   2584 2HB  ASP C 177      28.764  18.854  65.195  1.00  0.00           H  \\nATOM   2585  N   LYS C 178      31.045  18.322  64.456  1.00  0.00           N  \\nATOM   2586  CA  LYS C 178      31.856  17.338  63.748  1.00  0.00           C  \\nATOM   2587  C   LYS C 178      30.991  16.476  62.835  1.00  0.00           C  \\nATOM   2588  O   LYS C 178      31.477  15.932  61.848  1.00  0.00           O  \\nATOM   2589  CB  LYS C 178      32.595  16.501  64.769  1.00  0.00           C  \\nATOM   2590  CG  LYS C 178      33.451  15.454  64.211  1.00  0.00           C  \\nATOM   2591  CD  LYS C 178      34.209  14.787  65.308  1.00  0.00           C  \\nATOM   2592  CE  LYS C 178      33.243  14.127  66.306  1.00  0.00           C  \\nATOM   2593  NZ  LYS C 178      33.897  13.059  67.087  1.00  0.00           N  \\nATOM   2594  H   LYS C 178      30.601  17.999  65.118  1.00  0.00           H  \\nATOM   2595  HA  LYS C 178      32.505  17.803  63.196  1.00  0.00           H  \\nATOM   2596 1HB  LYS C 178      33.220  17.149  65.383  1.00  0.00           H  \\nATOM   2597 2HB  LYS C 178      31.875  16.018  65.429  1.00  0.00           H  \\nATOM   2598 1HG  LYS C 178      32.841  14.707  63.696  1.00  0.00           H  \\nATOM   2599 2HG  LYS C 178      34.149  15.889  63.499  1.00  0.00           H  \\nATOM   2600 1HD  LYS C 178      34.851  14.047  64.876  1.00  0.00           H  \\nATOM   2601 2HD  LYS C 178      34.824  15.514  65.832  1.00  0.00           H  \\nATOM   2602 1HE  LYS C 178      32.879  14.888  66.992  1.00  0.00           H  \\nATOM   2603 2HE  LYS C 178      32.396  13.704  65.767  1.00  0.00           H  \\nATOM   2604 1HZ  LYS C 178      33.260  12.631  67.721  1.00  0.00           H  \\nATOM   2605 2HZ  LYS C 178      34.219  12.330  66.410  1.00  0.00           H  \\nATOM   2606 3HZ  LYS C 178      34.692  13.381  67.600  1.00  0.00           H  \\nATOM   2607  N   ASP C 179      29.708  16.361  63.165  1.00  0.00           N  \\nATOM   2608  CA  ASP C 179      28.761  15.631  62.326  1.00  0.00           C  \\nATOM   2609  C   ASP C 179      28.347  16.481  61.126  1.00  0.00           C  \\nATOM   2610  O   ASP C 179      28.050  15.954  60.054  1.00  0.00           O  \\nATOM   2611  CB  ASP C 179      27.523  15.240  63.143  1.00  0.00           C  \\nATOM   2612  CG  ASP C 179      26.632  14.175  62.472  1.00  0.00           C  \\nATOM   2613  OD1 ASP C 179      27.106  13.081  62.253  1.00  0.00           O  \\nATOM   2614  OD2 ASP C 179      25.491  14.469  62.200  1.00  0.00           O  \\nATOM   2615  H   ASP C 179      29.358  16.698  63.875  1.00  0.00           H  \\nATOM   2616  HA  ASP C 179      29.187  14.826  61.994  1.00  0.00           H  \\nATOM   2617 1HB  ASP C 179      27.838  14.862  64.115  1.00  0.00           H  \\nATOM   2618 2HB  ASP C 179      26.917  16.130  63.323  1.00  0.00           H  \\nATOM   2619  N   VAL C 180      28.323  17.798  61.313  1.00  0.00           N  \\nATOM   2620  CA  VAL C 180      28.053  18.720  60.216  1.00  0.00           C  \\nATOM   2621  C   VAL C 180      29.210  18.679  59.225  1.00  0.00           C  \\nATOM   2622  O   VAL C 180      28.997  18.640  58.015  1.00  0.00           O  \\nATOM   2623  CB  VAL C 180      27.887  20.172  60.721  1.00  0.00           C  \\nATOM   2624  CG1 VAL C 180      27.810  21.148  59.528  1.00  0.00           C  \\nATOM   2625  CG2 VAL C 180      26.618  20.280  61.552  1.00  0.00           C  \\nATOM   2626  H   VAL C 180      28.460  18.183  62.070  1.00  0.00           H  \\nATOM   2627  HA  VAL C 180      27.248  18.440  59.755  1.00  0.00           H  \\nATOM   2628  HB  VAL C 180      28.756  20.444  61.327  1.00  0.00           H  \\nATOM   2629 1HG1 VAL C 180      27.701  22.169  59.899  1.00  0.00           H  \\nATOM   2630 2HG1 VAL C 180      28.722  21.075  58.935  1.00  0.00           H  \\nATOM   2631 3HG1 VAL C 180      26.953  20.898  58.903  1.00  0.00           H  \\nATOM   2632 1HG2 VAL C 180      26.507  21.306  61.917  1.00  0.00           H  \\nATOM   2633 2HG2 VAL C 180      25.760  20.018  60.934  1.00  0.00           H  \\nATOM   2634 3HG2 VAL C 180      26.667  19.600  62.398  1.00  0.00           H  \\nATOM   2635  N   ILE C 181      30.434  18.686  59.748  1.00  0.00           N  \\nATOM   2636  CA  ILE C 181      31.631  18.544  58.923  1.00  0.00           C  \\nATOM   2637  C   ILE C 181      31.555  17.273  58.083  1.00  0.00           C  \\nATOM   2638  O   ILE C 181      31.911  17.279  56.906  1.00  0.00           O  \\nATOM   2639  CB  ILE C 181      32.950  18.525  59.714  1.00  0.00           C  \\nATOM   2640  CG1 ILE C 181      33.233  19.911  60.278  1.00  0.00           C  \\nATOM   2641  CG2 ILE C 181      34.089  18.006  58.818  1.00  0.00           C  \\nATOM   2642  CD1 ILE C 181      34.352  19.958  61.290  1.00  0.00           C  \\nATOM   2643  H   ILE C 181      30.599  18.773  60.588  1.00  0.00           H  \\nATOM   2644  HA  ILE C 181      31.691  19.301  58.319  1.00  0.00           H  \\nATOM   2645  HB  ILE C 181      32.847  17.856  60.568  1.00  0.00           H  \\nATOM   2646 1HG1 ILE C 181      33.492  20.571  59.455  1.00  0.00           H  \\nATOM   2647 2HG1 ILE C 181      32.327  20.282  60.751  1.00  0.00           H  \\nATOM   2648 1HG2 ILE C 181      35.014  17.982  59.385  1.00  0.00           H  \\nATOM   2649 2HG2 ILE C 181      33.852  16.999  58.473  1.00  0.00           H  \\nATOM   2650 3HG2 ILE C 181      34.206  18.646  57.966  1.00  0.00           H  \\nATOM   2651 1HD1 ILE C 181      34.482  20.986  61.639  1.00  0.00           H  \\nATOM   2652 2HD1 ILE C 181      34.101  19.320  62.136  1.00  0.00           H  \\nATOM   2653 3HD1 ILE C 181      35.272  19.610  60.834  1.00  0.00           H  \\nATOM   2654  N   VAL C 182      31.080  16.190  58.693  1.00  0.00           N  \\nATOM   2655  CA  VAL C 182      30.901  14.922  57.990  1.00  0.00           C  \\nATOM   2656  C   VAL C 182      29.944  15.088  56.814  1.00  0.00           C  \\nATOM   2657  O   VAL C 182      30.222  14.625  55.707  1.00  0.00           O  \\nATOM   2658  CB  VAL C 182      30.454  13.791  58.929  1.00  0.00           C  \\nATOM   2659  CG1 VAL C 182      30.019  12.585  58.115  1.00  0.00           C  \\nATOM   2660  CG2 VAL C 182      31.651  13.393  59.811  1.00  0.00           C  \\nATOM   2661  H   VAL C 182      30.853  16.164  59.522  1.00  0.00           H  \\nATOM   2662  HA  VAL C 182      31.757  14.628  57.643  1.00  0.00           H  \\nATOM   2663  HB  VAL C 182      29.622  14.116  59.541  1.00  0.00           H  \\nATOM   2664 1HG1 VAL C 182      29.719  11.779  58.787  1.00  0.00           H  \\nATOM   2665 2HG1 VAL C 182      29.179  12.870  57.496  1.00  0.00           H  \\nATOM   2666 3HG1 VAL C 182      30.821  12.255  57.492  1.00  0.00           H  \\nATOM   2667 1HG2 VAL C 182      31.358  12.588  60.480  1.00  0.00           H  \\nATOM   2668 2HG2 VAL C 182      32.470  13.069  59.190  1.00  0.00           H  \\nATOM   2669 3HG2 VAL C 182      31.975  14.229  60.384  1.00  0.00           H  \\nATOM   2670  N   GLU C 183      28.821  15.755  57.057  1.00  0.00           N  \\nATOM   2671  CA  GLU C 183      27.810  15.944  56.025  1.00  0.00           C  \\nATOM   2672  C   GLU C 183      28.353  16.809  54.892  1.00  0.00           C  \\nATOM   2673  O   GLU C 183      28.124  16.521  53.719  1.00  0.00           O  \\nATOM   2674  CB  GLU C 183      26.535  16.547  56.618  1.00  0.00           C  \\nATOM   2675  CG  GLU C 183      25.290  16.482  55.690  1.00  0.00           C  \\nATOM   2676  CD  GLU C 183      24.711  15.088  55.486  1.00  0.00           C  \\nATOM   2677  OE1 GLU C 183      24.976  14.210  56.275  1.00  0.00           O  \\nATOM   2678  OE2 GLU C 183      24.002  14.907  54.523  1.00  0.00           O  \\nATOM   2679  H   GLU C 183      28.620  16.109  57.815  1.00  0.00           H  \\nATOM   2680  HA  GLU C 183      27.570  15.080  55.655  1.00  0.00           H  \\nATOM   2681 1HB  GLU C 183      26.295  16.036  57.552  1.00  0.00           H  \\nATOM   2682 2HB  GLU C 183      26.716  17.596  56.862  1.00  0.00           H  \\nATOM   2683 1HG  GLU C 183      24.515  17.144  56.071  1.00  0.00           H  \\nATOM   2684 2HG  GLU C 183      25.580  16.825  54.719  1.00  0.00           H  \\nATOM   2685  N   VAL C 184      29.082  17.862  55.250  1.00  0.00           N  \\nATOM   2686  CA  VAL C 184      29.695  18.747  54.265  1.00  0.00           C  \\nATOM   2687  C   VAL C 184      30.661  17.976  53.375  1.00  0.00           C  \\nATOM   2688  O   VAL C 184      30.706  18.194  52.168  1.00  0.00           O  \\nATOM   2689  CB  VAL C 184      30.456  19.925  54.946  1.00  0.00           C  \\nATOM   2690  CG1 VAL C 184      31.281  20.686  53.909  1.00  0.00           C  \\nATOM   2691  CG2 VAL C 184      29.464  20.918  55.602  1.00  0.00           C  \\nATOM   2692  H   VAL C 184      29.238  18.087  56.065  1.00  0.00           H  \\nATOM   2693  HA  VAL C 184      29.000  19.120  53.700  1.00  0.00           H  \\nATOM   2694  HB  VAL C 184      31.131  19.517  55.700  1.00  0.00           H  \\nATOM   2695 1HG1 VAL C 184      31.822  21.497  54.396  1.00  0.00           H  \\nATOM   2696 2HG1 VAL C 184      31.987  20.002  53.449  1.00  0.00           H  \\nATOM   2697 3HG1 VAL C 184      30.620  21.098  53.147  1.00  0.00           H  \\nATOM   2698 1HG2 VAL C 184      30.017  21.726  56.080  1.00  0.00           H  \\nATOM   2699 2HG2 VAL C 184      28.812  21.335  54.841  1.00  0.00           H  \\nATOM   2700 3HG2 VAL C 184      28.867  20.406  56.346  1.00  0.00           H  \\nATOM   2701  N   LEU C 185      31.427  17.069  53.972  1.00  0.00           N  \\nATOM   2702  CA  LEU C 185      32.376  16.268  53.210  1.00  0.00           C  \\nATOM   2703  C   LEU C 185      31.649  15.388  52.200  1.00  0.00           C  \\nATOM   2704  O   LEU C 185      32.119  15.207  51.080  1.00  0.00           O  \\nATOM   2705  CB  LEU C 185      33.238  15.389  54.126  1.00  0.00           C  \\nATOM   2706  CG  LEU C 185      34.263  16.119  55.005  1.00  0.00           C  \\nATOM   2707  CD1 LEU C 185      34.846  15.141  55.972  1.00  0.00           C  \\nATOM   2708  CD2 LEU C 185      35.357  16.706  54.158  1.00  0.00           C  \\nATOM   2709  H   LEU C 185      31.416  16.898  54.815  1.00  0.00           H  \\nATOM   2710  HA  LEU C 185      32.967  16.861  52.720  1.00  0.00           H  \\nATOM   2711 1HB  LEU C 185      32.577  14.835  54.790  1.00  0.00           H  \\nATOM   2712 2HB  LEU C 185      33.778  14.671  53.510  1.00  0.00           H  \\nATOM   2713  HG  LEU C 185      33.779  16.908  55.548  1.00  0.00           H  \\nATOM   2714 1HD1 LEU C 185      35.343  14.356  55.420  1.00  0.00           H  \\nATOM   2715 2HD1 LEU C 185      35.544  15.643  56.586  1.00  0.00           H  \\nATOM   2716 3HD1 LEU C 185      34.052  14.720  56.589  1.00  0.00           H  \\nATOM   2717 1HD2 LEU C 185      35.857  15.908  53.623  1.00  0.00           H  \\nATOM   2718 2HD2 LEU C 185      34.938  17.419  53.452  1.00  0.00           H  \\nATOM   2719 3HD2 LEU C 185      36.071  17.206  54.804  1.00  0.00           H  \\nATOM   2720  N   LYS C 186      30.497  14.853  52.596  1.00  0.00           N  \\nATOM   2721  CA  LYS C 186      29.722  13.982  51.716  1.00  0.00           C  \\nATOM   2722  C   LYS C 186      29.235  14.729  50.477  1.00  0.00           C  \\nATOM   2723  O   LYS C 186      29.327  14.221  49.362  1.00  0.00           O  \\nATOM   2724  CB  LYS C 186      28.554  13.308  52.460  1.00  0.00           C  \\nATOM   2725  CG  LYS C 186      29.010  12.189  53.440  1.00  0.00           C  \\nATOM   2726  CD  LYS C 186      27.874  11.289  53.933  1.00  0.00           C  \\nATOM   2727  CE  LYS C 186      27.061  11.900  55.046  1.00  0.00           C  \\nATOM   2728  NZ  LYS C 186      26.047  10.914  55.579  1.00  0.00           N  \\nATOM   2729  H   LYS C 186      30.142  14.978  53.369  1.00  0.00           H  \\nATOM   2730  HA  LYS C 186      30.288  13.251  51.420  1.00  0.00           H  \\nATOM   2731 1HB  LYS C 186      28.007  14.055  53.036  1.00  0.00           H  \\nATOM   2732 2HB  LYS C 186      27.863  12.876  51.740  1.00  0.00           H  \\nATOM   2733 1HG  LYS C 186      29.715  11.561  52.952  1.00  0.00           H  \\nATOM   2734 2HG  LYS C 186      29.498  12.645  54.300  1.00  0.00           H  \\nATOM   2735 1HD  LYS C 186      27.207  11.064  53.102  1.00  0.00           H  \\nATOM   2736 2HD  LYS C 186      28.298  10.347  54.299  1.00  0.00           H  \\nATOM   2737 1HE  LYS C 186      27.724  12.191  55.847  1.00  0.00           H  \\nATOM   2738 2HE  LYS C 186      26.542  12.785  54.681  1.00  0.00           H  \\nATOM   2739 1HZ  LYS C 186      26.516  10.051  55.937  1.00  0.00           H  \\nATOM   2740 2HZ  LYS C 186      25.515  11.338  56.332  1.00  0.00           H  \\nATOM   2741 3HZ  LYS C 186      25.420  10.653  54.839  1.00  0.00           H  \\nATOM   2742  N   VAL C 187      28.720  15.937  50.677  1.00  0.00           N  \\nATOM   2743  CA  VAL C 187      28.226  16.748  49.570  1.00  0.00           C  \\nATOM   2744  C   VAL C 187      29.378  17.207  48.674  1.00  0.00           C  \\nATOM   2745  O   VAL C 187      29.207  17.346  47.463  1.00  0.00           O  \\nATOM   2746  CB  VAL C 187      27.372  17.912  50.071  1.00  0.00           C  \\nATOM   2747  CG1 VAL C 187      27.000  18.842  48.915  1.00  0.00           C  \\nATOM   2748  CG2 VAL C 187      26.115  17.314  50.671  1.00  0.00           C  \\nATOM   2749  H   VAL C 187      28.645  16.312  51.447  1.00  0.00           H  \\nATOM   2750  HA  VAL C 187      27.627  16.208  49.030  1.00  0.00           H  \\nATOM   2751  HB  VAL C 187      27.925  18.493  50.818  1.00  0.00           H  \\nATOM   2752 1HG1 VAL C 187      26.374  19.660  49.285  1.00  0.00           H  \\nATOM   2753 2HG1 VAL C 187      27.898  19.250  48.460  1.00  0.00           H  \\nATOM   2754 3HG1 VAL C 187      26.448  18.274  48.177  1.00  0.00           H  \\nATOM   2755 1HG2 VAL C 187      26.384  16.657  51.503  1.00  0.00           H  \\nATOM   2756 2HG2 VAL C 187      25.467  18.081  51.018  1.00  0.00           H  \\nATOM   2757 3HG2 VAL C 187      25.592  16.739  49.911  1.00  0.00           H  \\nATOM   2758  N   ILE C 188      30.552  17.422  49.263  1.00  0.00           N  \\nATOM   2759  CA  ILE C 188      31.732  17.781  48.481  1.00  0.00           C  \\nATOM   2760  C   ILE C 188      32.087  16.643  47.527  1.00  0.00           C  \\nATOM   2761  O   ILE C 188      32.427  16.884  46.368  1.00  0.00           O  \\nATOM   2762  CB  ILE C 188      32.976  18.113  49.344  1.00  0.00           C  \\nATOM   2763  CG1 ILE C 188      32.803  19.443  50.080  1.00  0.00           C  \\nATOM   2764  CG2 ILE C 188      34.213  18.108  48.487  1.00  0.00           C  \\nATOM   2765  CD1 ILE C 188      33.846  19.695  51.162  1.00  0.00           C  \\nATOM   2766  H   ILE C 188      30.691  17.367  50.109  1.00  0.00           H  \\nATOM   2767  HA  ILE C 188      31.530  18.565  47.948  1.00  0.00           H  \\nATOM   2768  HB  ILE C 188      33.086  17.350  50.109  1.00  0.00           H  \\nATOM   2769 1HG1 ILE C 188      32.870  20.249  49.351  1.00  0.00           H  \\nATOM   2770 2HG1 ILE C 188      31.829  19.468  50.521  1.00  0.00           H  \\nATOM   2771 1HG2 ILE C 188      35.069  18.319  49.100  1.00  0.00           H  \\nATOM   2772 2HG2 ILE C 188      34.329  17.134  48.022  1.00  0.00           H  \\nATOM   2773 3HG2 ILE C 188      34.126  18.849  47.735  1.00  0.00           H  \\nATOM   2774 1HD1 ILE C 188      33.659  20.663  51.627  1.00  0.00           H  \\nATOM   2775 2HD1 ILE C 188      33.778  18.917  51.911  1.00  0.00           H  \\nATOM   2776 3HD1 ILE C 188      34.834  19.689  50.734  1.00  0.00           H  \\nATOM   2777  N   VAL C 189      31.987  15.405  48.005  1.00  0.00           N  \\nATOM   2778  CA  VAL C 189      32.248  14.246  47.155  1.00  0.00           C  \\nATOM   2779  C   VAL C 189      31.178  14.121  46.075  1.00  0.00           C  \\nATOM   2780  O   VAL C 189      31.471  13.728  44.948  1.00  0.00           O  \\nATOM   2781  CB  VAL C 189      32.366  12.964  47.986  1.00  0.00           C  \\nATOM   2782  CG1 VAL C 189      32.520  11.741  47.049  1.00  0.00           C  \\nATOM   2783  CG2 VAL C 189      33.561  13.113  48.923  1.00  0.00           C  \\nATOM   2784  H   VAL C 189      31.771  15.210  48.814  1.00  0.00           H  \\nATOM   2785  HA  VAL C 189      33.105  14.364  46.716  1.00  0.00           H  \\nATOM   2786  HB  VAL C 189      31.455  12.822  48.572  1.00  0.00           H  \\nATOM   2787 1HG1 VAL C 189      32.603  10.830  47.640  1.00  0.00           H  \\nATOM   2788 2HG1 VAL C 189      31.648  11.672  46.396  1.00  0.00           H  \\nATOM   2789 3HG1 VAL C 189      33.420  11.856  46.442  1.00  0.00           H  \\nATOM   2790 1HG2 VAL C 189      33.678  12.223  49.530  1.00  0.00           H  \\nATOM   2791 2HG2 VAL C 189      34.434  13.266  48.346  1.00  0.00           H  \\nATOM   2792 3HG2 VAL C 189      33.419  13.962  49.561  1.00  0.00           H  \\nATOM   2793  N   LYS C 190      29.938  14.453  46.418  1.00  0.00           N  \\nATOM   2794  CA  LYS C 190      28.869  14.462  45.427  1.00  0.00           C  \\nATOM   2795  C   LYS C 190      29.154  15.518  44.362  1.00  0.00           C  \\nATOM   2796  O   LYS C 190      28.932  15.289  43.176  1.00  0.00           O  \\nATOM   2797  CB  LYS C 190      27.504  14.752  46.057  1.00  0.00           C  \\nATOM   2798  CG  LYS C 190      26.943  13.627  46.906  1.00  0.00           C  \\nATOM   2799  CD  LYS C 190      25.623  14.027  47.555  1.00  0.00           C  \\nATOM   2800  CE  LYS C 190      24.501  14.087  46.531  1.00  0.00           C  \\nATOM   2801  NZ  LYS C 190      24.139  12.727  46.028  1.00  0.00           N  \\nATOM   2802  H   LYS C 190      29.690  14.675  47.211  1.00  0.00           H  \\nATOM   2803  HA  LYS C 190      28.831  13.596  44.991  1.00  0.00           H  \\nATOM   2804 1HB  LYS C 190      27.567  15.640  46.668  1.00  0.00           H  \\nATOM   2805 2HB  LYS C 190      26.786  14.961  45.267  1.00  0.00           H  \\nATOM   2806 1HG  LYS C 190      26.786  12.747  46.282  1.00  0.00           H  \\nATOM   2807 2HG  LYS C 190      27.660  13.374  47.683  1.00  0.00           H  \\nATOM   2808 1HD  LYS C 190      25.366  13.308  48.334  1.00  0.00           H  \\nATOM   2809 2HD  LYS C 190      25.729  15.009  48.014  1.00  0.00           H  \\nATOM   2810 1HE  LYS C 190      23.623  14.541  46.991  1.00  0.00           H  \\nATOM   2811 2HE  LYS C 190      24.817  14.706  45.688  1.00  0.00           H  \\nATOM   2812 1HZ  LYS C 190      23.392  12.803  45.353  1.00  0.00           H  \\nATOM   2813 2HZ  LYS C 190      24.948  12.302  45.589  1.00  0.00           H  \\nATOM   2814 3HZ  LYS C 190      23.835  12.152  46.801  1.00  0.00           H  \\nATOM   2815  N   ALA C 191      29.655  16.672  44.793  1.00  0.00           N  \\nATOM   2816  CA  ALA C 191      29.996  17.745  43.866  1.00  0.00           C  \\nATOM   2817  C   ALA C 191      31.127  17.307  42.944  1.00  0.00           C  \\nATOM   2818  O   ALA C 191      31.067  17.521  41.737  1.00  0.00           O  \\nATOM   2819  CB  ALA C 191      30.403  19.000  44.627  1.00  0.00           C  \\nATOM   2820  H   ALA C 191      29.807  16.858  45.618  1.00  0.00           H  \\nATOM   2821  HA  ALA C 191      29.222  17.949  43.318  1.00  0.00           H  \\nATOM   2822 1HB  ALA C 191      30.639  19.789  43.920  1.00  0.00           H  \\nATOM   2823 2HB  ALA C 191      29.593  19.320  45.274  1.00  0.00           H  \\nATOM   2824 3HB  ALA C 191      31.276  18.787  45.232  1.00  0.00           H  \\nATOM   2825  N   ILE C 192      32.154  16.685  43.518  1.00  0.00           N  \\nATOM   2826  CA  ILE C 192      33.290  16.206  42.738  1.00  0.00           C  \\nATOM   2827  C   ILE C 192      32.846  15.113  41.771  1.00  0.00           C  \\nATOM   2828  O   ILE C 192      33.307  15.065  40.633  1.00  0.00           O  \\nATOM   2829  CB  ILE C 192      34.436  15.702  43.627  1.00  0.00           C  \\nATOM   2830  CG1 ILE C 192      35.055  16.902  44.358  1.00  0.00           C  \\nATOM   2831  CG2 ILE C 192      35.487  14.963  42.774  1.00  0.00           C  \\nATOM   2832  CD1 ILE C 192      35.993  16.544  45.451  1.00  0.00           C  \\nATOM   2833  H   ILE C 192      32.216  16.528  44.361  1.00  0.00           H  \\nATOM   2834  HA  ILE C 192      33.647  16.942  42.216  1.00  0.00           H  \\nATOM   2835  HB  ILE C 192      34.032  15.023  44.381  1.00  0.00           H  \\nATOM   2836 1HG1 ILE C 192      35.597  17.507  43.635  1.00  0.00           H  \\nATOM   2837 2HG1 ILE C 192      34.253  17.506  44.780  1.00  0.00           H  \\nATOM   2838 1HG2 ILE C 192      36.284  14.606  43.418  1.00  0.00           H  \\nATOM   2839 2HG2 ILE C 192      35.017  14.122  42.274  1.00  0.00           H  \\nATOM   2840 3HG2 ILE C 192      35.897  15.642  42.025  1.00  0.00           H  \\nATOM   2841 1HD1 ILE C 192      36.357  17.466  45.875  1.00  0.00           H  \\nATOM   2842 2HD1 ILE C 192      35.468  15.962  46.208  1.00  0.00           H  \\nATOM   2843 3HD1 ILE C 192      36.820  15.967  45.056  1.00  0.00           H  \\nATOM   2844  N   GLU C 193      31.952  14.237  42.224  1.00  0.00           N  \\nATOM   2845  CA  GLU C 193      31.357  13.232  41.348  1.00  0.00           C  \\nATOM   2846  C   GLU C 193      30.676  13.909  40.167  1.00  0.00           C  \\nATOM   2847  O   GLU C 193      30.931  13.574  39.011  1.00  0.00           O  \\nATOM   2848  CB  GLU C 193      30.385  12.310  42.093  1.00  0.00           C  \\nATOM   2849  CG  GLU C 193      29.789  11.206  41.199  1.00  0.00           C  \\nATOM   2850  CD  GLU C 193      28.956  10.181  41.934  1.00  0.00           C  \\nATOM   2851  OE1 GLU C 193      28.453  10.458  42.994  1.00  0.00           O  \\nATOM   2852  OE2 GLU C 193      28.886   9.068  41.447  1.00  0.00           O  \\nATOM   2853  H   GLU C 193      31.674  14.204  43.037  1.00  0.00           H  \\nATOM   2854  HA  GLU C 193      32.054  12.650  41.007  1.00  0.00           H  \\nATOM   2855 1HB  GLU C 193      30.898  11.838  42.931  1.00  0.00           H  \\nATOM   2856 2HB  GLU C 193      29.565  12.893  42.499  1.00  0.00           H  \\nATOM   2857 1HG  GLU C 193      29.155  11.689  40.455  1.00  0.00           H  \\nATOM   2858 2HG  GLU C 193      30.584  10.704  40.667  1.00  0.00           H  \\nATOM   2859  N   ALA C 194      29.803  14.862  40.471  1.00  0.00           N  \\nATOM   2860  CA  ALA C 194      29.082  15.592  39.442  1.00  0.00           C  \\nATOM   2861  C   ALA C 194      30.065  16.299  38.517  1.00  0.00           C  \\nATOM   2862  O   ALA C 194      29.849  16.373  37.309  1.00  0.00           O  \\nATOM   2863  CB  ALA C 194      28.124  16.572  40.084  1.00  0.00           C  \\nATOM   2864  H   ALA C 194      29.611  15.106  41.273  1.00  0.00           H  \\nATOM   2865  HA  ALA C 194      28.562  14.967  38.913  1.00  0.00           H  \\nATOM   2866 1HB  ALA C 194      27.567  17.112  39.317  1.00  0.00           H  \\nATOM   2867 2HB  ALA C 194      27.433  16.020  40.716  1.00  0.00           H  \\nATOM   2868 3HB  ALA C 194      28.684  17.267  40.692  1.00  0.00           H  \\nATOM   2869  N   SER C 195      31.155  16.802  39.088  1.00  0.00           N  \\nATOM   2870  CA  SER C 195      32.141  17.535  38.309  1.00  0.00           C  \\nATOM   2871  C   SER C 195      32.848  16.611  37.328  1.00  0.00           C  \\nATOM   2872  O   SER C 195      33.021  16.957  36.162  1.00  0.00           O  \\nATOM   2873  CB  SER C 195      33.136  18.198  39.224  1.00  0.00           C  \\nATOM   2874  OG  SER C 195      34.081  18.941  38.506  1.00  0.00           O  \\nATOM   2875  H   SER C 195      31.344  16.732  39.924  1.00  0.00           H  \\nATOM   2876  HA  SER C 195      31.691  18.227  37.800  1.00  0.00           H  \\nATOM   2877 1HB  SER C 195      32.608  18.846  39.924  1.00  0.00           H  \\nATOM   2878 2HB  SER C 195      33.643  17.440  39.804  1.00  0.00           H  \\nATOM   2879  HG  SER C 195      34.705  19.248  39.164  1.00  0.00           H  \\nATOM   2880  N   VAL C 196      33.256  15.438  37.801  1.00  0.00           N  \\nATOM   2881  CA  VAL C 196      33.914  14.461  36.941  1.00  0.00           C  \\nATOM   2882  C   VAL C 196      32.963  14.016  35.837  1.00  0.00           C  \\nATOM   2883  O   VAL C 196      33.370  13.853  34.688  1.00  0.00           O  \\nATOM   2884  CB  VAL C 196      34.408  13.238  37.760  1.00  0.00           C  \\nATOM   2885  CG1 VAL C 196      34.903  12.120  36.822  1.00  0.00           C  \\nATOM   2886  CG2 VAL C 196      35.565  13.668  38.683  1.00  0.00           C  \\nATOM   2887  H   VAL C 196      33.164  15.183  38.618  1.00  0.00           H  \\nATOM   2888  HA  VAL C 196      34.686  14.875  36.525  1.00  0.00           H  \\nATOM   2889  HB  VAL C 196      33.576  12.849  38.359  1.00  0.00           H  \\nATOM   2890 1HG1 VAL C 196      35.242  11.268  37.417  1.00  0.00           H  \\nATOM   2891 2HG1 VAL C 196      34.090  11.806  36.170  1.00  0.00           H  \\nATOM   2892 3HG1 VAL C 196      35.732  12.493  36.220  1.00  0.00           H  \\nATOM   2893 1HG2 VAL C 196      35.907  12.813  39.266  1.00  0.00           H  \\nATOM   2894 2HG2 VAL C 196      36.379  14.046  38.078  1.00  0.00           H  \\nATOM   2895 3HG2 VAL C 196      35.233  14.447  39.351  1.00  0.00           H  \\nATOM   2896  N   GLU C 197      31.693  13.832  36.185  1.00  0.00           N  \\nATOM   2897  CA  GLU C 197      30.694  13.422  35.205  1.00  0.00           C  \\nATOM   2898  C   GLU C 197      30.557  14.486  34.123  1.00  0.00           C  \\nATOM   2899  O   GLU C 197      30.467  14.168  32.937  1.00  0.00           O  \\nATOM   2900  CB  GLU C 197      29.324  13.223  35.874  1.00  0.00           C  \\nATOM   2901  CG  GLU C 197      28.147  12.835  34.929  1.00  0.00           C  \\nATOM   2902  CD  GLU C 197      28.176  11.432  34.373  1.00  0.00           C  \\nATOM   2903  OE1 GLU C 197      28.815  10.587  34.945  1.00  0.00           O  \\nATOM   2904  OE2 GLU C 197      27.540  11.206  33.371  1.00  0.00           O  \\nATOM   2905  H   GLU C 197      31.385  13.938  36.981  1.00  0.00           H  \\nATOM   2906  HA  GLU C 197      30.981  12.597  34.784  1.00  0.00           H  \\nATOM   2907 1HB  GLU C 197      29.408  12.453  36.641  1.00  0.00           H  \\nATOM   2908 2HB  GLU C 197      29.038  14.140  36.377  1.00  0.00           H  \\nATOM   2909 1HG  GLU C 197      27.216  12.968  35.478  1.00  0.00           H  \\nATOM   2910 2HG  GLU C 197      28.138  13.542  34.096  1.00  0.00           H  \\nATOM   2911  N   ASN C 198      30.533  15.749  34.515  1.00  0.00           N  \\nATOM   2912  CA  ASN C 198      30.490  16.833  33.563  1.00  0.00           C  \\nATOM   2913  C   ASN C 198      31.704  16.847  32.657  1.00  0.00           C  \\nATOM   2914  O   ASN C 198      31.606  17.049  31.503  1.00  0.00           O  \\nATOM   2915  CB  ASN C 198      30.342  18.170  34.264  1.00  0.00           C  \\nATOM   2916  CG  ASN C 198      30.052  19.277  33.285  1.00  0.00           C  \\nATOM   2917  OD1 ASN C 198      28.994  19.298  32.674  1.00  0.00           O  \\nATOM   2918  ND2 ASN C 198      30.973  20.159  33.097  1.00  0.00           N  \\nATOM   2919  H   ASN C 198      30.530  15.995  35.330  1.00  0.00           H  \\nATOM   2920  HA  ASN C 198      29.708  16.706  33.001  1.00  0.00           H  \\nATOM   2921 1HB  ASN C 198      29.540  18.116  34.999  1.00  0.00           H  \\nATOM   2922 2HB  ASN C 198      31.255  18.390  34.803  1.00  0.00           H  \\nATOM   2923 1HD2 ASN C 198      31.820  20.097  33.639  1.00  0.00           H  \\nATOM   2924 2HD2 ASN C 198      30.845  20.886  32.424  1.00  0.00           H  \\nATOM   2925  N   GLN C 199      32.858  16.625  33.209  1.00  0.00           N  \\nATOM   2926  CA  GLN C 199      34.076  16.648  32.415  1.00  0.00           C  \\nATOM   2927  C   GLN C 199      34.110  15.478  31.436  1.00  0.00           C  \\nATOM   2928  O   GLN C 199      34.723  15.573  30.375  1.00  0.00           O  \\nATOM   2929  CB  GLN C 199      35.260  16.645  33.369  1.00  0.00           C  \\nATOM   2930  CG  GLN C 199      35.356  17.961  34.181  1.00  0.00           C  \\nATOM   2931  CD  GLN C 199      36.447  17.978  35.231  1.00  0.00           C  \\nATOM   2932  OE1 GLN C 199      37.620  17.835  34.897  1.00  0.00           O  \\nATOM   2933  NE2 GLN C 199      36.106  18.143  36.509  1.00  0.00           N  \\nATOM   2934  H   GLN C 199      32.981  16.461  34.044  1.00  0.00           H  \\nATOM   2935  HA  GLN C 199      34.099  17.469  31.899  1.00  0.00           H  \\nATOM   2936 1HB  GLN C 199      35.159  15.814  34.069  1.00  0.00           H  \\nATOM   2937 2HB  GLN C 199      36.168  16.502  32.811  1.00  0.00           H  \\nATOM   2938 1HG  GLN C 199      35.563  18.771  33.485  1.00  0.00           H  \\nATOM   2939 2HG  GLN C 199      34.412  18.154  34.665  1.00  0.00           H  \\nATOM   2940 1HE2 GLN C 199      36.823  18.184  37.210  1.00  0.00           H  \\nATOM   2941 2HE2 GLN C 199      35.149  18.259  36.824  1.00  0.00           H  \\nATOM   2942  N   ARG C 200      33.432  14.385  31.777  1.00  0.00           N  \\nATOM   2943  CA  ARG C 200      33.352  13.245  30.871  1.00  0.00           C  \\nATOM   2944  C   ARG C 200      32.556  13.615  29.633  1.00  0.00           C  \\nATOM   2945  O   ARG C 200      32.976  13.342  28.511  1.00  0.00           O  \\nATOM   2946  CB  ARG C 200      32.730  12.030  31.547  1.00  0.00           C  \\nATOM   2947  CG  ARG C 200      33.671  11.383  32.453  1.00  0.00           C  \\nATOM   2948  CD  ARG C 200      33.236  10.084  33.053  1.00  0.00           C  \\nATOM   2949  NE  ARG C 200      32.380  10.250  34.198  1.00  0.00           N  \\nATOM   2950  CZ  ARG C 200      31.772   9.238  34.847  1.00  0.00           C  \\nATOM   2951  NH1 ARG C 200      31.932   8.001  34.433  1.00  0.00           N  \\nATOM   2952  NH2 ARG C 200      31.016   9.475  35.896  1.00  0.00           N  \\nATOM   2953  H   ARG C 200      33.013  14.280  32.521  1.00  0.00           H  \\nATOM   2954  HA  ARG C 200      34.248  13.000  30.591  1.00  0.00           H  \\nATOM   2955 1HB  ARG C 200      31.856  12.318  32.116  1.00  0.00           H  \\nATOM   2956 2HB  ARG C 200      32.417  11.307  30.797  1.00  0.00           H  \\nATOM   2957 1HG  ARG C 200      34.534  11.248  31.893  1.00  0.00           H  \\nATOM   2958 2HG  ARG C 200      33.888  12.053  33.259  1.00  0.00           H  \\nATOM   2959 1HD  ARG C 200      32.690   9.514  32.305  1.00  0.00           H  \\nATOM   2960 2HD  ARG C 200      34.112   9.520  33.365  1.00  0.00           H  \\nATOM   2961  HE  ARG C 200      32.232  11.189  34.544  1.00  0.00           H  \\nATOM   2962 1HH1 ARG C 200      32.500   7.808  33.626  1.00  0.00           H  \\nATOM   2963 2HH1 ARG C 200      31.468   7.247  34.920  1.00  0.00           H  \\nATOM   2964 1HH2 ARG C 200      30.869  10.423  36.214  1.00  0.00           H  \\nATOM   2965 2HH2 ARG C 200      30.548   8.719  36.365  1.00  0.00           H  \\nATOM   2966  N   ILE C 201      31.409  14.248  29.845  1.00  0.00           N  \\nATOM   2967  CA  ILE C 201      30.556  14.665  28.744  1.00  0.00           C  \\nATOM   2968  C   ILE C 201      31.278  15.712  27.900  1.00  0.00           C  \\nATOM   2969  O   ILE C 201      31.153  15.726  26.677  1.00  0.00           O  \\nATOM   2970  CB  ILE C 201      29.246  15.296  29.233  1.00  0.00           C  \\nATOM   2971  CG1 ILE C 201      28.391  14.286  29.941  1.00  0.00           C  \\nATOM   2972  CG2 ILE C 201      28.473  15.820  28.004  1.00  0.00           C  \\nATOM   2973  CD1 ILE C 201      27.247  14.926  30.675  1.00  0.00           C  \\nATOM   2974  H   ILE C 201      31.102  14.448  30.623  1.00  0.00           H  \\nATOM   2975  HA  ILE C 201      30.367  13.899  28.180  1.00  0.00           H  \\nATOM   2976  HB  ILE C 201      29.460  16.113  29.929  1.00  0.00           H  \\nATOM   2977 1HG1 ILE C 201      27.997  13.591  29.216  1.00  0.00           H  \\nATOM   2978 2HG1 ILE C 201      29.002  13.733  30.664  1.00  0.00           H  \\nATOM   2979 1HG2 ILE C 201      27.542  16.267  28.325  1.00  0.00           H  \\nATOM   2980 2HG2 ILE C 201      29.070  16.565  27.480  1.00  0.00           H  \\nATOM   2981 3HG2 ILE C 201      28.258  14.991  27.325  1.00  0.00           H  \\nATOM   2982 1HD1 ILE C 201      26.652  14.162  31.174  1.00  0.00           H  \\nATOM   2983 2HD1 ILE C 201      27.653  15.615  31.418  1.00  0.00           H  \\nATOM   2984 3HD1 ILE C 201      26.631  15.471  29.978  1.00  0.00           H  \\nATOM   2985  N   SER C 202      32.040  16.580  28.559  1.00  0.00           N  \\nATOM   2986  CA  SER C 202      32.816  17.601  27.861  1.00  0.00           C  \\nATOM   2987  C   SER C 202      33.842  16.968  26.923  1.00  0.00           C  \\nATOM   2988  O   SER C 202      33.967  17.371  25.765  1.00  0.00           O  \\nATOM   2989  CB  SER C 202      33.530  18.491  28.852  1.00  0.00           C  \\nATOM   2990  OG  SER C 202      34.315  19.480  28.209  1.00  0.00           O  \\nATOM   2991  H   SER C 202      32.125  16.599  29.414  1.00  0.00           H  \\nATOM   2992  HA  SER C 202      32.216  18.144  27.327  1.00  0.00           H  \\nATOM   2993 1HB  SER C 202      32.795  18.965  29.505  1.00  0.00           H  \\nATOM   2994 2HB  SER C 202      34.167  17.883  29.475  1.00  0.00           H  \\nATOM   2995  HG  SER C 202      34.748  19.956  28.922  1.00  0.00           H  \\nATOM   2996  N   ALA C 203      34.581  15.985  27.428  1.00  0.00           N  \\nATOM   2997  CA  ALA C 203      35.560  15.272  26.613  1.00  0.00           C  \\nATOM   2998  C   ALA C 203      34.879  14.603  25.421  1.00  0.00           C  \\nATOM   2999  O   ALA C 203      35.415  14.603  24.314  1.00  0.00           O  \\nATOM   3000  CB  ALA C 203      36.284  14.240  27.457  1.00  0.00           C  \\nATOM   3001  H   ALA C 203      34.535  15.711  28.242  1.00  0.00           H  \\nATOM   3002  HA  ALA C 203      36.209  15.906  26.271  1.00  0.00           H  \\nATOM   3003 1HB  ALA C 203      36.779  14.738  28.292  1.00  0.00           H  \\nATOM   3004 2HB  ALA C 203      35.559  13.520  27.843  1.00  0.00           H  \\nATOM   3005 3HB  ALA C 203      37.024  13.725  26.850  1.00  0.00           H  \\nATOM   3006  N   GLU C 204      33.695  14.041  25.657  1.00  0.00           N  \\nATOM   3007  CA  GLU C 204      32.887  13.460  24.586  1.00  0.00           C  \\nATOM   3008  C   GLU C 204      32.621  14.474  23.489  1.00  0.00           C  \\nATOM   3009  O   GLU C 204      32.772  14.179  22.305  1.00  0.00           O  \\nATOM   3010  CB  GLU C 204      31.512  12.902  25.020  1.00  0.00           C  \\nATOM   3011  CG  GLU C 204      31.494  11.613  25.847  1.00  0.00           C  \\nATOM   3012  CD  GLU C 204      30.044  11.139  26.121  1.00  0.00           C  \\nATOM   3013  OE1 GLU C 204      29.173  11.485  25.344  1.00  0.00           O  \\nATOM   3014  OE2 GLU C 204      29.806  10.473  27.106  1.00  0.00           O  \\nATOM   3015  H   GLU C 204      33.335  13.983  26.435  1.00  0.00           H  \\nATOM   3016  HA  GLU C 204      33.362  12.709  24.198  1.00  0.00           H  \\nATOM   3017 1HB  GLU C 204      30.975  13.639  25.578  1.00  0.00           H  \\nATOM   3018 2HB  GLU C 204      30.921  12.707  24.126  1.00  0.00           H  \\nATOM   3019 1HG  GLU C 204      32.032  10.834  25.309  1.00  0.00           H  \\nATOM   3020 2HG  GLU C 204      32.001  11.787  26.789  1.00  0.00           H  \\nATOM   3021  N   ASN C 205      32.264  15.675  23.875  1.00  0.00           N  \\nATOM   3022  CA  ASN C 205      31.927  16.712  22.932  1.00  0.00           C  \\nATOM   3023  C   ASN C 205      33.158  17.188  22.199  1.00  0.00           C  \\nATOM   3024  O   ASN C 205      33.099  17.588  21.094  1.00  0.00           O  \\nATOM   3025  CB  ASN C 205      31.178  17.845  23.584  1.00  0.00           C  \\nATOM   3026  CG  ASN C 205      30.609  18.787  22.539  1.00  0.00           C  \\nATOM   3027  OD1 ASN C 205      29.974  18.370  21.568  1.00  0.00           O  \\nATOM   3028  ND2 ASN C 205      30.813  20.043  22.724  1.00  0.00           N  \\nATOM   3029  H   ASN C 205      32.222  15.911  24.692  1.00  0.00           H  \\nATOM   3030  HA  ASN C 205      31.306  16.342  22.281  1.00  0.00           H  \\nATOM   3031 1HB  ASN C 205      30.367  17.450  24.200  1.00  0.00           H  \\nATOM   3032 2HB  ASN C 205      31.855  18.389  24.240  1.00  0.00           H  \\nATOM   3033 1HD2 ASN C 205      30.458  20.704  22.067  1.00  0.00           H  \\nATOM   3034 2HD2 ASN C 205      31.327  20.353  23.540  1.00  0.00           H  \\nATOM   3035  N   GLN C 206      34.289  17.113  22.836  1.00  0.00           N  \\nATOM   3036  CA  GLN C 206      35.535  17.516  22.194  1.00  0.00           C  \\nATOM   3037  C   GLN C 206      35.974  16.511  21.138  1.00  0.00           C  \\nATOM   3038  O   GLN C 206      36.583  16.889  20.139  1.00  0.00           O  \\nATOM   3039  CB  GLN C 206      36.641  17.714  23.224  1.00  0.00           C  \\nATOM   3040  CG  GLN C 206      36.407  18.902  24.121  1.00  0.00           C  \\nATOM   3041  CD  GLN C 206      37.453  19.071  25.194  1.00  0.00           C  \\nATOM   3042  OE1 GLN C 206      38.670  19.110  24.916  1.00  0.00           O  \\nATOM   3043  NE2 GLN C 206      37.008  19.195  26.452  1.00  0.00           N  \\nATOM   3044  H   GLN C 206      34.378  16.845  23.648  1.00  0.00           H  \\nATOM   3045  HA  GLN C 206      35.398  18.369  21.753  1.00  0.00           H  \\nATOM   3046 1HB  GLN C 206      36.730  16.822  23.843  1.00  0.00           H  \\nATOM   3047 2HB  GLN C 206      37.585  17.861  22.714  1.00  0.00           H  \\nATOM   3048 1HG  GLN C 206      36.424  19.797  23.502  1.00  0.00           H  \\nATOM   3049 2HG  GLN C 206      35.442  18.817  24.593  1.00  0.00           H  \\nATOM   3050 1HE2 GLN C 206      37.656  19.366  27.188  1.00  0.00           H  \\nATOM   3051 2HE2 GLN C 206      36.015  19.166  26.701  1.00  0.00           H  \\nATOM   3052  N   LYS C 207      35.676  15.234  21.365  1.00  0.00           N  \\nATOM   3053  CA  LYS C 207      35.926  14.204  20.361  1.00  0.00           C  \\nATOM   3054  C   LYS C 207      35.120  14.500  19.102  1.00  0.00           C  \\nATOM   3055  O   LYS C 207      35.652  14.488  17.991  1.00  0.00           O  \\nATOM   3056  CB  LYS C 207      35.600  12.811  20.923  1.00  0.00           C  \\nATOM   3057  CG  LYS C 207      36.620  12.249  21.982  1.00  0.00           C  \\nATOM   3058  CD  LYS C 207      36.137  10.874  22.527  1.00  0.00           C  \\nATOM   3059  CE  LYS C 207      37.058  10.252  23.631  1.00  0.00           C  \\nATOM   3060  NZ  LYS C 207      38.347   9.687  23.082  1.00  0.00           N  \\nATOM   3061  H   LYS C 207      35.327  14.938  22.093  1.00  0.00           H  \\nATOM   3062  HA  LYS C 207      36.868  14.206  20.128  1.00  0.00           H  \\nATOM   3063 1HB  LYS C 207      34.625  12.831  21.394  1.00  0.00           H  \\nATOM   3064 2HB  LYS C 207      35.547  12.099  20.103  1.00  0.00           H  \\nATOM   3065 1HG  LYS C 207      37.601  12.133  21.525  1.00  0.00           H  \\nATOM   3066 2HG  LYS C 207      36.703  12.954  22.809  1.00  0.00           H  \\nATOM   3067 1HD  LYS C 207      35.138  11.004  22.949  1.00  0.00           H  \\nATOM   3068 2HD  LYS C 207      36.066  10.171  21.699  1.00  0.00           H  \\nATOM   3069 1HE  LYS C 207      37.297  11.018  24.370  1.00  0.00           H  \\nATOM   3070 2HE  LYS C 207      36.510   9.449  24.121  1.00  0.00           H  \\nATOM   3071 1HZ  LYS C 207      38.930   9.257  23.849  1.00  0.00           H  \\nATOM   3072 2HZ  LYS C 207      38.135   8.969  22.408  1.00  0.00           H  \\nATOM   3073 3HZ  LYS C 207      38.876  10.418  22.637  1.00  0.00           H  \\nATOM   3074  N   ALA C 208      33.830  14.764  19.290  1.00  0.00           N  \\nATOM   3075  CA  ALA C 208      32.922  15.038  18.182  1.00  0.00           C  \\nATOM   3076  C   ALA C 208      33.345  16.279  17.406  1.00  0.00           C  \\nATOM   3077  O   ALA C 208      33.284  16.306  16.178  1.00  0.00           O  \\nATOM   3078  CB  ALA C 208      31.526  15.250  18.738  1.00  0.00           C  \\nATOM   3079  H   ALA C 208      33.452  14.791  20.062  1.00  0.00           H  \\nATOM   3080  HA  ALA C 208      32.933  14.285  17.571  1.00  0.00           H  \\nATOM   3081 1HB  ALA C 208      30.842  15.433  17.924  1.00  0.00           H  \\nATOM   3082 2HB  ALA C 208      31.221  14.366  19.287  1.00  0.00           H  \\nATOM   3083 3HB  ALA C 208      31.530  16.108  19.415  1.00  0.00           H  \\nATOM   3084  N   LEU C 209      33.772  17.308  18.129  1.00  0.00           N  \\nATOM   3085  CA  LEU C 209      34.193  18.553  17.503  1.00  0.00           C  \\nATOM   3086  C   LEU C 209      35.456  18.353  16.662  1.00  0.00           C  \\nATOM   3087  O   LEU C 209      35.676  19.069  15.686  1.00  0.00           O  \\nATOM   3088  CB  LEU C 209      34.429  19.653  18.562  1.00  0.00           C  \\nATOM   3089  CG  LEU C 209      33.154  20.224  19.280  1.00  0.00           C  \\nATOM   3090  CD1 LEU C 209      33.571  21.157  20.439  1.00  0.00           C  \\nATOM   3091  CD2 LEU C 209      32.324  20.985  18.275  1.00  0.00           C  \\nATOM   3092  H   LEU C 209      33.828  17.309  18.987  1.00  0.00           H  \\nATOM   3093  HA  LEU C 209      33.488  18.862  16.913  1.00  0.00           H  \\nATOM   3094 1HB  LEU C 209      35.080  19.238  19.333  1.00  0.00           H  \\nATOM   3095 2HB  LEU C 209      34.949  20.485  18.089  1.00  0.00           H  \\nATOM   3096  HG  LEU C 209      32.560  19.412  19.678  1.00  0.00           H  \\nATOM   3097 1HD1 LEU C 209      32.679  21.553  20.928  1.00  0.00           H  \\nATOM   3098 2HD1 LEU C 209      34.160  20.592  21.162  1.00  0.00           H  \\nATOM   3099 3HD1 LEU C 209      34.162  21.976  20.048  1.00  0.00           H  \\nATOM   3100 1HD2 LEU C 209      32.903  21.815  17.867  1.00  0.00           H  \\nATOM   3101 2HD2 LEU C 209      32.032  20.316  17.472  1.00  0.00           H  \\nATOM   3102 3HD2 LEU C 209      31.441  21.370  18.769  1.00  0.00           H  \\nTER                                                                             \\nVirtualRootMover_root 210\\n                                                      \\n\",\"pdb\");\n",
              "\tviewer_16234219519367425.setStyle({\"cartoon\": {\"color\": \"spectrum\"}, \"stick\": {\"colorscheme\": \"lightgreyCarbon\", \"radius\": 0.25}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 44.928, \"y\": 15.169, \"z\": 17.949}, \"end\": {\"x\": 45.575, \"y\": 16.034, \"z\": 16.023}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 46.81, \"y\": 14.343, \"z\": 19.809}, \"end\": {\"x\": 48.033, \"y\": 14.85, \"z\": 18.14}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 46.114, \"y\": 15.698, \"z\": 22.027}, \"end\": {\"x\": 46.777, \"y\": 16.91, \"z\": 20.757}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.674, \"y\": 14.789, \"z\": 22.598}, \"end\": {\"x\": 43.737, \"y\": 15.652, \"z\": 20.696}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 44.928, \"y\": 15.169, \"z\": 17.949}, \"end\": {\"x\": 45.575, \"y\": 16.034, \"z\": 16.023}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 44.287, \"y\": 12.228, \"z\": 23.305}, \"end\": {\"x\": 44.842, \"y\": 12.424, \"z\": 21.094}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 46.81, \"y\": 14.343, \"z\": 19.809}, \"end\": {\"x\": 48.033, \"y\": 14.85, \"z\": 18.14}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 45.587, \"y\": 12.88, \"z\": 25.534}, \"end\": {\"x\": 46.837, \"y\": 13.37, \"z\": 23.836}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 46.114, \"y\": 15.698, \"z\": 22.027}, \"end\": {\"x\": 46.777, \"y\": 16.91, \"z\": 20.757}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.775, \"y\": 14.077, \"z\": 26.954}, \"end\": {\"x\": 44.138, \"y\": 14.97, \"z\": 25.59}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.674, \"y\": 14.789, \"z\": 22.598}, \"end\": {\"x\": 43.737, \"y\": 15.652, \"z\": 20.696}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.315, \"y\": 11.834, \"z\": 27.371}, \"end\": {\"x\": 42.157, \"y\": 12.197, \"z\": 25.469}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 44.287, \"y\": 12.228, \"z\": 23.305}, \"end\": {\"x\": 44.842, \"y\": 12.424, \"z\": 21.094}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.828, \"y\": 10.351, \"z\": 29.108}, \"end\": {\"x\": 44.715, \"y\": 10.356, \"z\": 26.98}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 45.587, \"y\": 12.88, \"z\": 25.534}, \"end\": {\"x\": 46.837, \"y\": 13.37, \"z\": 23.836}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.779, \"y\": 11.899, \"z\": 31.288}, \"end\": {\"x\": 44.672, \"y\": 12.731, \"z\": 29.523}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.775, \"y\": 14.077, \"z\": 26.954}, \"end\": {\"x\": 44.138, \"y\": 14.97, \"z\": 25.59}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.24, \"y\": 11.598, \"z\": 32.106}, \"end\": {\"x\": 41.266, \"y\": 12.827, \"z\": 30.038}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.315, \"y\": 11.834, \"z\": 27.371}, \"end\": {\"x\": 42.157, \"y\": 12.197, \"z\": 25.469}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.015, \"y\": 8.95, \"z\": 32.904}, \"end\": {\"x\": 41.355, \"y\": 9.346, \"z\": 30.55}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.828, \"y\": 10.351, \"z\": 29.108}, \"end\": {\"x\": 44.715, \"y\": 10.356, \"z\": 26.98}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.608, \"y\": 9.106, \"z\": 35.154}, \"end\": {\"x\": 43.833, \"y\": 9.532, \"z\": 33.186}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.779, \"y\": 11.899, \"z\": 31.288}, \"end\": {\"x\": 44.672, \"y\": 12.731, \"z\": 29.523}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.004, \"y\": 10.63, \"z\": 36.679}, \"end\": {\"x\": 41.958, \"y\": 11.886, \"z\": 35.037}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.24, \"y\": 11.598, \"z\": 32.106}, \"end\": {\"x\": 41.266, \"y\": 12.827, \"z\": 30.038}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.702, \"y\": 9.083, \"z\": 37.306}, \"end\": {\"x\": 39.279, \"y\": 9.765, \"z\": 35.258}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.015, \"y\": 8.95, \"z\": 32.904}, \"end\": {\"x\": 41.355, \"y\": 9.346, \"z\": 30.55}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.075, \"y\": 7.231, \"z\": 38.814}, \"end\": {\"x\": 40.893, \"y\": 7.094, \"z\": 36.646}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.608, \"y\": 9.106, \"z\": 35.154}, \"end\": {\"x\": 43.833, \"y\": 9.532, \"z\": 33.186}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.837, \"y\": 8.623, \"z\": 41.003}, \"end\": {\"x\": 41.898, \"y\": 9.264, \"z\": 39.238}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.004, \"y\": 10.63, \"z\": 36.679}, \"end\": {\"x\": 41.958, \"y\": 11.886, \"z\": 35.037}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.421, \"y\": 9.172, \"z\": 41.794}, \"end\": {\"x\": 38.849, \"y\": 10.057, \"z\": 40.177}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.702, \"y\": 9.083, \"z\": 37.306}, \"end\": {\"x\": 39.279, \"y\": 9.765, \"z\": 35.258}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.331, \"y\": 6.822, \"z\": 42.612}, \"end\": {\"x\": 37.689, \"y\": 6.746, \"z\": 40.626}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.075, \"y\": 7.231, \"z\": 38.814}, \"end\": {\"x\": 40.893, \"y\": 7.094, \"z\": 36.646}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 39.037, \"y\": 6.257, \"z\": 44.665}, \"end\": {\"x\": 40.299, \"y\": 6.264, \"z\": 42.835}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.837, \"y\": 8.623, \"z\": 41.003}, \"end\": {\"x\": 41.898, \"y\": 9.264, \"z\": 39.238}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.314, \"y\": 8.397, \"z\": 46.21}, \"end\": {\"x\": 39.152, \"y\": 9.145, \"z\": 44.74}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.421, \"y\": 9.172, \"z\": 41.794}, \"end\": {\"x\": 38.849, \"y\": 10.057, \"z\": 40.177}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.776, \"y\": 7.699, \"z\": 46.781}, \"end\": {\"x\": 35.848, \"y\": 8.173, \"z\": 44.89}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.331, \"y\": 6.822, \"z\": 42.612}, \"end\": {\"x\": 37.689, \"y\": 6.746, \"z\": 40.626}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.176, \"y\": 5.637, \"z\": 48.368}, \"end\": {\"x\": 36.759, \"y\": 5.221, \"z\": 46.384}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 39.037, \"y\": 6.257, \"z\": 44.665}, \"end\": {\"x\": 40.299, \"y\": 6.264, \"z\": 42.835}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.389, \"y\": 6.449, \"z\": 50.766}, \"end\": {\"x\": 38.549, \"y\": 6.755, \"z\": 48.751}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.314, \"y\": 8.397, \"z\": 46.21}, \"end\": {\"x\": 39.152, \"y\": 9.145, \"z\": 44.74}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.776, \"y\": 7.699, \"z\": 46.781}, \"end\": {\"x\": 35.848, \"y\": 8.173, \"z\": 44.89}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.176, \"y\": 5.637, \"z\": 48.368}, \"end\": {\"x\": 36.759, \"y\": 5.221, \"z\": 46.384}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.386, \"y\": 6.036, \"z\": 52.847}, \"end\": {\"x\": 35.353, \"y\": 4.797, \"z\": 52.294}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.389, \"y\": 6.449, \"z\": 50.766}, \"end\": {\"x\": 38.549, \"y\": 6.755, \"z\": 48.751}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.508, \"y\": 6.401, \"z\": 55.453}, \"end\": {\"x\": 36.018, \"y\": 7.83, \"z\": 54.475}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.386, \"y\": 6.036, \"z\": 52.847}, \"end\": {\"x\": 35.353, \"y\": 4.797, \"z\": 52.294}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.508, \"y\": 6.401, \"z\": 55.453}, \"end\": {\"x\": 36.018, \"y\": 7.83, \"z\": 54.475}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.918, \"y\": 10.367, \"z\": 63.127}, \"end\": {\"x\": 35.586, \"y\": 8.761, \"z\": 62.224}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.21866693600841, \"y\": 12.330188316307208, \"z\": 66.40965255518343}, \"end\": {\"x\": 33.74, \"y\": 11.731, \"z\": 64.744}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.918, \"y\": 10.367, \"z\": 63.127}, \"end\": {\"x\": 35.586, \"y\": 8.761, \"z\": 62.224}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.091, \"y\": 12.533, \"z\": 57.327}, \"end\": {\"x\": 42.807, \"y\": 11.802, \"z\": 59.618}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.169, \"y\": 15.057, \"z\": 54.381}, \"end\": {\"x\": 39.486, \"y\": 14.863, \"z\": 56.459}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.726, \"y\": 15.045, \"z\": 53.189}, \"end\": {\"x\": 42.906, \"y\": 14.714, \"z\": 55.38}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.091, \"y\": 12.533, \"z\": 57.327}, \"end\": {\"x\": 42.807, \"y\": 11.802, \"z\": 59.618}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.496, \"y\": 14.947, \"z\": 49.766}, \"end\": {\"x\": 40.039, \"y\": 14.13, \"z\": 51.491}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.169, \"y\": 15.057, \"z\": 54.381}, \"end\": {\"x\": 39.486, \"y\": 14.863, \"z\": 56.459}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.69, \"y\": 17.251, \"z\": 48.882}, \"end\": {\"x\": 41.81, \"y\": 16.994, \"z\": 51.054}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.726, \"y\": 15.045, \"z\": 53.189}, \"end\": {\"x\": 42.906, \"y\": 14.714, \"z\": 55.38}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.281, \"y\": 15.974, \"z\": 47.169}, \"end\": {\"x\": 44.031, \"y\": 15.165, \"z\": 49.233}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.488, \"y\": 15.039, \"z\": 45.317}, \"end\": {\"x\": 41.17, \"y\": 13.919, \"z\": 46.992}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.496, \"y\": 14.947, \"z\": 49.766}, \"end\": {\"x\": 40.039, \"y\": 14.13, \"z\": 51.491}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.606, \"y\": 17.455, \"z\": 44.367}, \"end\": {\"x\": 40.168, \"y\": 17.084, \"z\": 46.262}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.69, \"y\": 17.251, \"z\": 48.882}, \"end\": {\"x\": 41.81, \"y\": 16.994, \"z\": 51.054}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.901, \"y\": 18.516, \"z\": 43.346}, \"end\": {\"x\": 43.298, \"y\": 18.263, \"z\": 45.182}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.281, \"y\": 15.974, \"z\": 47.169}, \"end\": {\"x\": 44.031, \"y\": 15.165, \"z\": 49.233}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.223, \"y\": 16.745, \"z\": 41.253}, \"end\": {\"x\": 43.424, \"y\": 15.625, \"z\": 43.055}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.488, \"y\": 15.039, \"z\": 45.317}, \"end\": {\"x\": 41.17, \"y\": 13.919, \"z\": 46.992}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.095, \"y\": 17.435, \"z\": 39.906}, \"end\": {\"x\": 40.489, \"y\": 16.403, \"z\": 41.523}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.606, \"y\": 17.455, \"z\": 44.367}, \"end\": {\"x\": 40.168, \"y\": 17.084, \"z\": 46.262}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.5, \"y\": 20.045, \"z\": 39.068}, \"end\": {\"x\": 41.408, \"y\": 19.694, \"z\": 40.985}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.901, \"y\": 18.516, \"z\": 43.346}, \"end\": {\"x\": 43.298, \"y\": 18.263, \"z\": 45.182}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.66, \"y\": 19.607, \"z\": 37.466}, \"end\": {\"x\": 44.317, \"y\": 18.925, \"z\": 39.449}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.223, \"y\": 16.745, \"z\": 41.253}, \"end\": {\"x\": 43.424, \"y\": 15.625, \"z\": 43.055}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.442, \"y\": 18.152, \"z\": 35.494}, \"end\": {\"x\": 42.347, \"y\": 16.834, \"z\": 37.208}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.095, \"y\": 17.435, \"z\": 39.906}, \"end\": {\"x\": 40.489, \"y\": 16.403, \"z\": 41.523}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.89, \"y\": 20.114, \"z\": 34.393}, \"end\": {\"x\": 40.268, \"y\": 19.415, \"z\": 36.368}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.5, \"y\": 20.045, \"z\": 39.068}, \"end\": {\"x\": 41.408, \"y\": 19.694, \"z\": 40.985}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.798, \"y\": 21.649, \"z\": 33.397}, \"end\": {\"x\": 42.778, \"y\": 21.714, \"z\": 35.47}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.66, \"y\": 19.607, \"z\": 37.466}, \"end\": {\"x\": 44.317, \"y\": 18.925, \"z\": 39.449}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 44.068, \"y\": 20.32, \"z\": 31.387}, \"end\": {\"x\": 44.186, \"y\": 19.404, \"z\": 33.348}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.442, \"y\": 18.152, \"z\": 35.494}, \"end\": {\"x\": 42.347, \"y\": 16.834, \"z\": 37.208}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.789, \"y\": 20.312, \"z\": 29.894}, \"end\": {\"x\": 41.209, \"y\": 18.977, \"z\": 31.6}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.89, \"y\": 20.114, \"z\": 34.393}, \"end\": {\"x\": 40.268, \"y\": 19.415, \"z\": 36.368}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.177, \"y\": 22.884, \"z\": 29.042}, \"end\": {\"x\": 41.051, \"y\": 22.717, \"z\": 31.157}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.798, \"y\": 21.649, \"z\": 33.397}, \"end\": {\"x\": 42.778, \"y\": 21.714, \"z\": 35.47}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.45, \"y\": 23.447, \"z\": 27.724}, \"end\": {\"x\": 43.986, \"y\": 22.801, \"z\": 29.64}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 44.068, \"y\": 20.32, \"z\": 31.387}, \"end\": {\"x\": 44.186, \"y\": 19.404, \"z\": 33.348}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.898, \"y\": 21.791, \"z\": 25.541}, \"end\": {\"x\": 43.319, \"y\": 20.248, \"z\": 27.274}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.789, \"y\": 20.312, \"z\": 29.894}, \"end\": {\"x\": 41.209, \"y\": 18.977, \"z\": 31.6}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.049, \"y\": 23.144, \"z\": 24.363}, \"end\": {\"x\": 40.435, \"y\": 21.835, \"z\": 26.294}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.177, \"y\": 22.884, \"z\": 29.042}, \"end\": {\"x\": 41.051, \"y\": 22.717, \"z\": 31.157}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.092, \"y\": 25.195, \"z\": 23.403}, \"end\": {\"x\": 41.986, \"y\": 24.945, \"z\": 25.528}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.45, \"y\": 23.447, \"z\": 27.724}, \"end\": {\"x\": 43.986, \"y\": 22.801, \"z\": 29.64}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.898, \"y\": 21.791, \"z\": 25.541}, \"end\": {\"x\": 43.319, \"y\": 20.248, \"z\": 27.274}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.199, \"y\": 23.307, \"z\": 19.677}, \"end\": {\"x\": 41.603, \"y\": 22.046, \"z\": 21.623}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.049, \"y\": 23.144, \"z\": 24.363}, \"end\": {\"x\": 40.435, \"y\": 21.835, \"z\": 26.294}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.092, \"y\": 25.195, \"z\": 23.403}, \"end\": {\"x\": 41.986, \"y\": 24.945, \"z\": 25.528}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.199, \"y\": 23.307, \"z\": 19.677}, \"end\": {\"x\": 41.603, \"y\": 22.046, \"z\": 21.623}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.759, \"y\": 32.648, \"z\": 18.951}, \"end\": {\"x\": 33.315, \"y\": 34.433, \"z\": 18.0}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.79, \"y\": 33.758, \"z\": 23.271}, \"end\": {\"x\": 34.157, \"y\": 33.247, \"z\": 21.839}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.607528040070385, \"y\": 31.84705697453721, \"z\": 22.70185412508183}, \"end\": {\"x\": 34.157, \"y\": 33.247, \"z\": 21.839}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.759, \"y\": 32.648, \"z\": 18.951}, \"end\": {\"x\": 33.315, \"y\": 34.433, \"z\": 18.0}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.896, \"y\": 32.205, \"z\": 23.73}, \"end\": {\"x\": 36.745, \"y\": 31.991, \"z\": 21.645}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.431, \"y\": 33.858, \"z\": 25.086}, \"end\": {\"x\": 38.529, \"y\": 34.736, \"z\": 22.665}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.121, \"y\": 34.26, \"z\": 27.349}, \"end\": {\"x\": 36.211, \"y\": 35.1, \"z\": 25.576}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.79, \"y\": 33.758, \"z\": 23.271}, \"end\": {\"x\": 34.157, \"y\": 33.247, \"z\": 21.839}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.607528040070385, \"y\": 31.84705697453721, \"z\": 22.70185412508183}, \"end\": {\"x\": 34.157, \"y\": 33.247, \"z\": 21.839}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.369, \"y\": 31.742, \"z\": 28.341}, \"end\": {\"x\": 36.195, \"y\": 31.74, \"z\": 26.589}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.896, \"y\": 32.205, \"z\": 23.73}, \"end\": {\"x\": 36.745, \"y\": 31.991, \"z\": 21.645}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.053, \"y\": 31.756, \"z\": 28.943}, \"end\": {\"x\": 39.753, \"y\": 31.633, \"z\": 26.671}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.431, \"y\": 33.858, \"z\": 25.086}, \"end\": {\"x\": 38.529, \"y\": 34.736, \"z\": 22.665}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 44.5168841824215, \"y\": 36.144802735412114, \"z\": 27.69370086875567}, \"end\": {\"x\": 44.96668824977388, \"y\": 34.68488058383626, \"z\": 28.75493233274207}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.251, \"y\": 33.559, \"z\": 31.045}, \"end\": {\"x\": 39.668, \"y\": 34.4, \"z\": 28.992}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.121, \"y\": 34.26, \"z\": 27.349}, \"end\": {\"x\": 36.211, \"y\": 35.1, \"z\": 25.576}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.189476346955757, \"y\": 33.290865925054334, \"z\": 28.803122703741924}, \"end\": {\"x\": 29.428845209954513, \"y\": 31.717333628843285, \"z\": 28.257713713651913}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.675, \"y\": 32.399, \"z\": 32.888}, \"end\": {\"x\": 37.585, \"y\": 32.749, \"z\": 31.196}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.369, \"y\": 31.742, \"z\": 28.341}, \"end\": {\"x\": 36.195, \"y\": 31.74, \"z\": 26.589}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.093, \"y\": 30.116, \"z\": 33.359}, \"end\": {\"x\": 39.635, \"y\": 29.971, \"z\": 31.366}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.053, \"y\": 31.756, \"z\": 28.943}, \"end\": {\"x\": 39.753, \"y\": 31.633, \"z\": 26.671}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 44.5168841824215, \"y\": 36.144802735412114, \"z\": 27.69370086875567}, \"end\": {\"x\": 44.96668824977388, \"y\": 34.68488058383626, \"z\": 28.75493233274207}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.338, \"y\": 31.096, \"z\": 34.489}, \"end\": {\"x\": 42.308, \"y\": 31.807, \"z\": 32.426}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.251, \"y\": 33.559, \"z\": 31.045}, \"end\": {\"x\": 39.668, \"y\": 34.4, \"z\": 28.992}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.28, \"y\": 32.185, \"z\": 36.817}, \"end\": {\"x\": 40.58, \"y\": 33.11, \"z\": 35.143}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.675, \"y\": 32.399, \"z\": 32.888}, \"end\": {\"x\": 37.585, \"y\": 32.749, \"z\": 31.196}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.667, \"y\": 29.732, \"z\": 37.978}, \"end\": {\"x\": 39.338, \"y\": 30.112, \"z\": 36.295}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.093, \"y\": 30.116, \"z\": 33.359}, \"end\": {\"x\": 39.635, \"y\": 29.971, \"z\": 31.366}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.017, \"y\": 28.412, \"z\": 38.344}, \"end\": {\"x\": 42.57, \"y\": 28.737, \"z\": 36.39}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 44.83161294124585, \"y\": 28.497463981683758, \"z\": 35.54834550802836}, \"end\": {\"x\": 42.57, \"y\": 28.737, \"z\": 36.39}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.338, \"y\": 31.096, \"z\": 34.489}, \"end\": {\"x\": 42.308, \"y\": 31.807, \"z\": 32.426}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 44.006, \"y\": 30.098, \"z\": 40.253}, \"end\": {\"x\": 43.672, \"y\": 31.22, \"z\": 38.421}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.28, \"y\": 32.185, \"z\": 36.817}, \"end\": {\"x\": 40.58, \"y\": 33.11, \"z\": 35.143}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.01972261255976, \"y\": 34.2435750163747, \"z\": 42.06894430177419}, \"end\": {\"x\": 40.396678011848536, \"y\": 34.49134644417415, \"z\": 40.60980465820651}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.19788003259319, \"y\": 34.16201625530438, \"z\": 43.47043935100746}, \"end\": {\"x\": 39.05025230668446, \"y\": 33.72767488786936, \"z\": 42.06272361365636}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.548, \"y\": 29.831, \"z\": 42.526}, \"end\": {\"x\": 41.308, \"y\": 30.78, \"z\": 40.75}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.667, \"y\": 29.732, \"z\": 37.978}, \"end\": {\"x\": 39.338, \"y\": 30.112, \"z\": 36.295}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.368, \"y\": 27.328, \"z\": 42.955}, \"end\": {\"x\": 42.142, \"y\": 27.441, \"z\": 41.085}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.017, \"y\": 28.412, \"z\": 38.344}, \"end\": {\"x\": 42.57, \"y\": 28.737, \"z\": 36.39}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 44.83161294124585, \"y\": 28.497463981683758, \"z\": 35.54834550802836}, \"end\": {\"x\": 42.57, \"y\": 28.737, \"z\": 36.39}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 45.911, \"y\": 27.541, \"z\": 43.823}, \"end\": {\"x\": 45.667, \"y\": 27.911, \"z\": 41.536}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 44.006, \"y\": 30.098, \"z\": 40.253}, \"end\": {\"x\": 43.672, \"y\": 31.22, \"z\": 38.421}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.01972261255976, \"y\": 34.2435750163747, \"z\": 42.06894430177419}, \"end\": {\"x\": 40.396678011848536, \"y\": 34.49134644417415, \"z\": 40.60980465820651}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.19788003259319, \"y\": 34.16201625530438, \"z\": 43.47043935100746}, \"end\": {\"x\": 39.05025230668446, \"y\": 33.72767488786936, \"z\": 42.06272361365636}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 45.501, \"y\": 28.698, \"z\": 46.269}, \"end\": {\"x\": 45.046, \"y\": 29.917, \"z\": 44.248}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.548, \"y\": 29.831, \"z\": 42.526}, \"end\": {\"x\": 41.308, \"y\": 30.78, \"z\": 40.75}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 44.088, \"y\": 26.729, \"z\": 47.492}, \"end\": {\"x\": 42.972, \"y\": 27.507, \"z\": 45.958}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.368, \"y\": 27.328, \"z\": 42.955}, \"end\": {\"x\": 42.142, \"y\": 27.441, \"z\": 41.085}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 46.115, \"y\": 24.93, \"z\": 47.674}, \"end\": {\"x\": 45.623, \"y\": 25.158, \"z\": 45.673}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 45.911, \"y\": 27.541, \"z\": 43.823}, \"end\": {\"x\": 45.667, \"y\": 27.911, \"z\": 41.536}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 47.772, \"y\": 25.925, \"z\": 49.545}, \"end\": {\"x\": 47.791, \"y\": 27.152, \"z\": 47.518}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 45.501, \"y\": 28.698, \"z\": 46.269}, \"end\": {\"x\": 45.046, \"y\": 29.917, \"z\": 44.248}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 46.208, \"y\": 25.973, \"z\": 51.715}, \"end\": {\"x\": 45.513, \"y\": 27.195, \"z\": 50.13}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 44.088, \"y\": 26.729, \"z\": 47.492}, \"end\": {\"x\": 42.972, \"y\": 27.507, \"z\": 45.958}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 46.229, \"y\": 23.291, \"z\": 52.236}, \"end\": {\"x\": 45.004, \"y\": 23.69, \"z\": 50.209}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 46.115, \"y\": 24.93, \"z\": 47.674}, \"end\": {\"x\": 45.623, \"y\": 25.158, \"z\": 45.673}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 48.639, \"y\": 22.565, \"z\": 53.084}, \"end\": {\"x\": 48.537, \"y\": 23.169, \"z\": 50.556}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 47.772, \"y\": 25.925, \"z\": 49.545}, \"end\": {\"x\": 47.791, \"y\": 27.152, \"z\": 47.518}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 48.587, \"y\": 23.93, \"z\": 55.45}, \"end\": {\"x\": 48.494, \"y\": 25.19, \"z\": 53.553}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 46.208, \"y\": 25.973, \"z\": 51.715}, \"end\": {\"x\": 45.513, \"y\": 27.195, \"z\": 50.13}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 46.229, \"y\": 23.291, \"z\": 52.236}, \"end\": {\"x\": 45.004, \"y\": 23.69, \"z\": 50.209}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 47.146, \"y\": 22.094, \"z\": 56.728}, \"end\": {\"x\": 47.492, \"y\": 20.651, \"z\": 55.141}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 48.626, \"y\": 20.244, \"z\": 57.372}, \"end\": {\"x\": 47.492, \"y\": 20.651, \"z\": 55.141}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 48.639, \"y\": 22.565, \"z\": 53.084}, \"end\": {\"x\": 48.537, \"y\": 23.169, \"z\": 50.556}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 48.587, \"y\": 23.93, \"z\": 55.45}, \"end\": {\"x\": 48.494, \"y\": 25.19, \"z\": 53.553}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 48.687, \"y\": 21.991, \"z\": 61.669}, \"end\": {\"x\": 47.992, \"y\": 23.038, \"z\": 59.486}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 47.146, \"y\": 22.094, \"z\": 56.728}, \"end\": {\"x\": 47.492, \"y\": 20.651, \"z\": 55.141}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 48.626, \"y\": 20.244, \"z\": 57.372}, \"end\": {\"x\": 47.492, \"y\": 20.651, \"z\": 55.141}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 48.687, \"y\": 21.991, \"z\": 61.669}, \"end\": {\"x\": 47.992, \"y\": 23.038, \"z\": 59.486}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 46.48112143287274, \"y\": 24.43688560293486, \"z\": 64.69180973353596}, \"end\": {\"x\": 44.74873557380715, \"y\": 24.72563549164821, \"z\": 65.28017034524638}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 46.48112143287274, \"y\": 24.43688560293486, \"z\": 64.69180973353596}, \"end\": {\"x\": 44.74873557380715, \"y\": 24.72563549164821, \"z\": 65.28017034524638}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.797, \"y\": 24.07, \"z\": 60.161}, \"end\": {\"x\": 37.928, \"y\": 24.036, \"z\": 62.513}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.906, \"y\": 26.033, \"z\": 58.599}, \"end\": {\"x\": 39.37, \"y\": 26.633, \"z\": 60.713}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 39.74, \"y\": 24.65, \"z\": 56.429}, \"end\": {\"x\": 40.909, \"y\": 24.634, \"z\": 58.547}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.627, \"y\": 23.654, \"z\": 55.284}, \"end\": {\"x\": 38.302, \"y\": 23.034, \"z\": 57.354}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.797, \"y\": 24.07, \"z\": 60.161}, \"end\": {\"x\": 37.928, \"y\": 24.036, \"z\": 62.513}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.42, \"y\": 25.933, \"z\": 54.347}, \"end\": {\"x\": 36.581, \"y\": 25.993, \"z\": 56.648}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.906, \"y\": 26.033, \"z\": 58.599}, \"end\": {\"x\": 39.37, \"y\": 26.633, \"z\": 60.713}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.069, \"y\": 26.712, \"z\": 52.316}, \"end\": {\"x\": 39.383, \"y\": 27.024, \"z\": 54.576}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 39.74, \"y\": 24.65, \"z\": 56.429}, \"end\": {\"x\": 40.909, \"y\": 24.634, \"z\": 58.547}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.744, \"y\": 24.72, \"z\": 50.686}, \"end\": {\"x\": 38.716, \"y\": 24.089, \"z\": 52.487}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.627, \"y\": 23.654, \"z\": 55.284}, \"end\": {\"x\": 38.302, \"y\": 23.034, \"z\": 57.354}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.228, \"y\": 25.135, \"z\": 49.723}, \"end\": {\"x\": 35.326, \"y\": 24.495, \"z\": 51.91}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.42, \"y\": 25.933, \"z\": 54.347}, \"end\": {\"x\": 36.581, \"y\": 25.993, \"z\": 56.648}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.423, \"y\": 27.371, \"z\": 48.234}, \"end\": {\"x\": 35.747, \"y\": 27.749, \"z\": 50.672}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.069, \"y\": 26.712, \"z\": 52.316}, \"end\": {\"x\": 39.383, \"y\": 27.024, \"z\": 54.576}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.922, \"y\": 26.359, \"z\": 46.281}, \"end\": {\"x\": 38.09, \"y\": 26.42, \"z\": 48.193}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.744, \"y\": 24.72, \"z\": 50.686}, \"end\": {\"x\": 38.716, \"y\": 24.089, \"z\": 52.487}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.359, \"y\": 24.585, \"z\": 45.02}, \"end\": {\"x\": 35.91, \"y\": 23.91, \"z\": 47.037}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.228, \"y\": 25.135, \"z\": 49.723}, \"end\": {\"x\": 35.326, \"y\": 24.495, \"z\": 51.91}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.367, \"y\": 26.286, \"z\": 44.199}, \"end\": {\"x\": 33.445, \"y\": 26.337, \"z\": 46.181}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.423, \"y\": 27.371, \"z\": 48.234}, \"end\": {\"x\": 35.747, \"y\": 27.749, \"z\": 50.672}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.944, \"y\": 27.801, \"z\": 42.413}, \"end\": {\"x\": 35.76, \"y\": 28.216, \"z\": 44.182}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.922, \"y\": 26.359, \"z\": 46.281}, \"end\": {\"x\": 38.09, \"y\": 26.42, \"z\": 48.193}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.317, \"y\": 25.776, \"z\": 40.63}, \"end\": {\"x\": 36.441, \"y\": 25.39, \"z\": 42.3}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.359, \"y\": 24.585, \"z\": 45.02}, \"end\": {\"x\": 35.91, \"y\": 23.91, \"z\": 47.037}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.961, \"y\": 25.067, \"z\": 39.57}, \"end\": {\"x\": 33.155, \"y\": 24.852, \"z\": 41.536}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.367, \"y\": 26.286, \"z\": 44.199}, \"end\": {\"x\": 33.445, \"y\": 26.337, \"z\": 46.181}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.224271632369327, \"y\": 30.276877354867583, \"z\": 46.32545610227355}, \"end\": {\"x\": 31.510631876663993, \"y\": 30.67397005123429, \"z\": 45.13401426878197}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.529, \"y\": 27.397, \"z\": 38.257}, \"end\": {\"x\": 32.656, \"y\": 27.88, \"z\": 40.417}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.944, \"y\": 27.801, \"z\": 42.413}, \"end\": {\"x\": 35.76, \"y\": 28.216, \"z\": 44.182}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.461, \"y\": 27.389, \"z\": 36.378}, \"end\": {\"x\": 35.443, \"y\": 27.678, \"z\": 38.275}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.317, \"y\": 25.776, \"z\": 40.63}, \"end\": {\"x\": 36.441, \"y\": 25.39, \"z\": 42.3}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.566, \"y\": 25.282, \"z\": 34.917}, \"end\": {\"x\": 34.174, \"y\": 24.635, \"z\": 36.825}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.44013511333774, \"y\": 22.064469540028885, \"z\": 37.23236983890624}, \"end\": {\"x\": 35.77134407051643, \"y\": 22.567117783120626, \"z\": 38.70923076436494}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.961, \"y\": 25.067, \"z\": 39.57}, \"end\": {\"x\": 33.155, \"y\": 24.852, \"z\": 41.536}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.348, \"y\": 26.231, \"z\": 33.856}, \"end\": {\"x\": 31.047, \"y\": 26.023, \"z\": 35.995}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.529, \"y\": 27.397, \"z\": 38.257}, \"end\": {\"x\": 32.656, \"y\": 27.88, \"z\": 40.417}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.757587951021616, \"y\": 30.335986356774868, \"z\": 34.896689777867465}, \"end\": {\"x\": 30.14420731503131, \"y\": 30.664761524126256, \"z\": 36.412283775164674}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.28530486774137, \"y\": 32.50091487463622, \"z\": 34.40550002704736}, \"end\": {\"x\": 30.95714299824809, \"y\": 32.35975413665545, \"z\": 35.42542876525172}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.193, \"y\": 28.162, \"z\": 32.195}, \"end\": {\"x\": 32.635, \"y\": 28.654, \"z\": 34.314}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.461, \"y\": 27.389, \"z\": 36.378}, \"end\": {\"x\": 35.443, \"y\": 27.678, \"z\": 38.275}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.55, \"y\": 26.621, \"z\": 30.544}, \"end\": {\"x\": 34.582, \"y\": 26.463, \"z\": 32.317}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.566, \"y\": 25.282, \"z\": 34.917}, \"end\": {\"x\": 34.174, \"y\": 24.635, \"z\": 36.825}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.44013511333774, \"y\": 22.064469540028885, \"z\": 37.23236983890624}, \"end\": {\"x\": 35.77134407051643, \"y\": 22.567117783120626, \"z\": 38.70923076436494}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.554, \"y\": 25.34, \"z\": 29.228}, \"end\": {\"x\": 31.623, \"y\": 24.569, \"z\": 31.337}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.819871868901842, \"y\": 20.097132435132995, \"z\": 33.63873775807972}, \"end\": {\"x\": 32.77328843911919, \"y\": 20.32190090795406, \"z\": 35.184731349317204}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.66974175102478, \"y\": 20.5402127688388, \"z\": 37.488125467948954}, \"end\": {\"x\": 34.08108314865374, \"y\": 18.941081444814955, \"z\": 38.50586644813445}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.348, \"y\": 26.231, \"z\": 33.856}, \"end\": {\"x\": 31.047, \"y\": 26.023, \"z\": 35.995}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.757587951021616, \"y\": 30.335986356774868, \"z\": 34.896689777867465}, \"end\": {\"x\": 30.14420731503131, \"y\": 30.664761524126256, \"z\": 36.412283775164674}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.28530486774137, \"y\": 32.50091487463622, \"z\": 34.40550002704736}, \"end\": {\"x\": 30.95714299824809, \"y\": 32.35975413665545, \"z\": 35.42542876525172}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.383, \"y\": 27.283, \"z\": 27.774}, \"end\": {\"x\": 30.09, \"y\": 27.49, \"z\": 30.119}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.193, \"y\": 28.162, \"z\": 32.195}, \"end\": {\"x\": 32.635, \"y\": 28.654, \"z\": 34.314}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.454, \"y\": 27.837, \"z\": 26.06}, \"end\": {\"x\": 32.967, \"y\": 28.375, \"z\": 28.14}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.55, \"y\": 26.621, \"z\": 30.544}, \"end\": {\"x\": 34.582, \"y\": 26.463, \"z\": 32.317}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.253, \"y\": 25.527, \"z\": 24.63}, \"end\": {\"x\": 33.143, \"y\": 25.226, \"z\": 26.643}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.554, \"y\": 25.34, \"z\": 29.228}, \"end\": {\"x\": 31.623, \"y\": 24.569, \"z\": 31.337}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.942, \"y\": 25.728, \"z\": 23.449}, \"end\": {\"x\": 29.869, \"y\": 25.412, \"z\": 25.445}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.189476346955757, \"y\": 33.290865925054334, \"z\": 28.803122703741924}, \"end\": {\"x\": 29.428845209954513, \"y\": 31.717333628843285, \"z\": 28.257713713651913}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.383, \"y\": 27.283, \"z\": 27.774}, \"end\": {\"x\": 30.09, \"y\": 27.49, \"z\": 30.119}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.498, \"y\": 27.8, \"z\": 21.584}, \"end\": {\"x\": 30.588, \"y\": 28.339, \"z\": 23.764}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.454, \"y\": 27.837, \"z\": 26.06}, \"end\": {\"x\": 32.967, \"y\": 28.375, \"z\": 28.14}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.44, \"y\": 26.572, \"z\": 19.998}, \"end\": {\"x\": 33.284, \"y\": 26.782, \"z\": 22.141}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.253, \"y\": 25.527, \"z\": 24.63}, \"end\": {\"x\": 33.143, \"y\": 25.226, \"z\": 26.643}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.765, \"y\": 24.677, \"z\": 18.769}, \"end\": {\"x\": 30.899, \"y\": 24.317, \"z\": 21.012}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.32683807501296, \"y\": 20.35260431802973, \"z\": 23.540226834555213}, \"end\": {\"x\": 32.18137603332113, \"y\": 20.741199970340038, \"z\": 25.16323827728038}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.72087825595617, \"y\": 20.880175444877075, \"z\": 27.136601848196545}, \"end\": {\"x\": 34.31508507317345, \"y\": 19.48007487537707, \"z\": 28.20927142849176}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.942, \"y\": 25.728, \"z\": 23.449}, \"end\": {\"x\": 29.869, \"y\": 25.412, \"z\": 25.445}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.498, \"y\": 27.8, \"z\": 21.584}, \"end\": {\"x\": 30.588, \"y\": 28.339, \"z\": 23.764}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.44, \"y\": 26.572, \"z\": 19.998}, \"end\": {\"x\": 33.284, \"y\": 26.782, \"z\": 22.141}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.765, \"y\": 24.677, \"z\": 18.769}, \"end\": {\"x\": 30.899, \"y\": 24.317, \"z\": 21.012}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.05, \"y\": 16.161, \"z\": 17.689}, \"end\": {\"x\": 27.635, \"y\": 15.974, \"z\": 15.34}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.904, \"y\": 17.864, \"z\": 18.589}, \"end\": {\"x\": 24.509, \"y\": 17.274, \"z\": 16.363}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.771, \"y\": 16.247, \"z\": 20.35}, \"end\": {\"x\": 23.651, \"y\": 14.944, \"z\": 18.729}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.45, \"y\": 15.92, \"z\": 22.281}, \"end\": {\"x\": 26.355, \"y\": 15.108, \"z\": 20.443}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.05, \"y\": 16.161, \"z\": 17.689}, \"end\": {\"x\": 27.635, \"y\": 15.974, \"z\": 15.34}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.136455906181823, \"y\": 16.967498478933162, \"z\": 22.205397136733485}, \"end\": {\"x\": 26.404, \"y\": 18.333, \"z\": 21.18}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.667, \"y\": 18.368, \"z\": 23.214}, \"end\": {\"x\": 26.404, \"y\": 18.333, \"z\": 21.18}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.904, \"y\": 17.864, \"z\": 18.589}, \"end\": {\"x\": 24.509, \"y\": 17.274, \"z\": 16.363}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.25, \"y\": 18.616, \"z\": 24.493}, \"end\": {\"x\": 22.797, \"y\": 18.512, \"z\": 22.116}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.771, \"y\": 16.247, \"z\": 20.35}, \"end\": {\"x\": 23.651, \"y\": 14.944, \"z\": 18.729}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.432, \"y\": 16.91, \"z\": 26.514}, \"end\": {\"x\": 23.301, \"y\": 15.781, \"z\": 24.415}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.589563925674486, \"y\": 12.078131537104372, \"z\": 25.25878934509002}, \"end\": {\"x\": 23.965311485529828, \"y\": 12.449716774199903, \"z\": 23.51205044150279}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.45, \"y\": 15.92, \"z\": 22.281}, \"end\": {\"x\": 26.355, \"y\": 15.108, \"z\": 20.443}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.136455906181823, \"y\": 16.967498478933162, \"z\": 22.205397136733485}, \"end\": {\"x\": 26.404, \"y\": 18.333, \"z\": 21.18}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.427, \"y\": 18.031, \"z\": 27.743}, \"end\": {\"x\": 26.129, \"y\": 17.442, \"z\": 26.064}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.667, \"y\": 18.368, \"z\": 23.214}, \"end\": {\"x\": 26.404, \"y\": 18.333, \"z\": 21.18}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.291, \"y\": 20.259, \"z\": 28.618}, \"end\": {\"x\": 24.414, \"y\": 20.459, \"z\": 26.601}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.25, \"y\": 18.616, \"z\": 24.493}, \"end\": {\"x\": 22.797, \"y\": 18.512, \"z\": 22.116}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.454, \"y\": 19.274, \"z\": 30.463}, \"end\": {\"x\": 21.866, \"y\": 18.703, \"z\": 28.391}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.432, \"y\": 16.91, \"z\": 26.514}, \"end\": {\"x\": 23.301, \"y\": 15.781, \"z\": 24.415}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.589563925674486, \"y\": 12.078131537104372, \"z\": 25.25878934509002}, \"end\": {\"x\": 23.965311485529828, \"y\": 12.449716774199903, \"z\": 23.51205044150279}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.157850437434107, \"y\": 11.27474139717846, \"z\": 25.58315812187468}, \"end\": {\"x\": 29.173031995488035, \"y\": 11.484988568726713, \"z\": 25.344087608353572}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.841190198975447, \"y\": 10.81166564254388, \"z\": 27.72207275072217}, \"end\": {\"x\": 29.806315435843562, \"y\": 10.472892507048423, \"z\": 27.105802516057818}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.177, \"y\": 18.134, \"z\": 32.237}, \"end\": {\"x\": 24.459, \"y\": 17.24, \"z\": 30.507}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.427, \"y\": 18.031, \"z\": 27.743}, \"end\": {\"x\": 26.129, \"y\": 17.442, \"z\": 26.064}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.262, \"y\": 20.57, \"z\": 33.201}, \"end\": {\"x\": 25.968, \"y\": 20.323, \"z\": 31.144}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.291, \"y\": 20.259, \"z\": 28.618}, \"end\": {\"x\": 24.414, \"y\": 20.459, \"z\": 26.601}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.223, \"y\": 21.668, \"z\": 34.52}, \"end\": {\"x\": 23.015, \"y\": 21.602, \"z\": 32.33}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.454, \"y\": 19.274, \"z\": 30.463}, \"end\": {\"x\": 21.866, \"y\": 18.703, \"z\": 28.391}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.94593277204187, \"y\": 20.80424270142054, \"z\": 31.897368431077453}, \"end\": {\"x\": 19.194710442396982, \"y\": 18.954379243328006, \"z\": 31.022428820684457}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.03790525184778, \"y\": 19.037638159609415, \"z\": 32.582111819737065}, \"end\": {\"x\": 19.194710442396982, \"y\": 18.954379243328006, \"z\": 31.022428820684457}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.081, \"y\": 19.898, \"z\": 36.603}, \"end\": {\"x\": 22.55, \"y\": 19.111, \"z\": 34.601}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.177, \"y\": 18.134, \"z\": 32.237}, \"end\": {\"x\": 24.459, \"y\": 17.24, \"z\": 30.507}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.394, \"y\": 20.436, \"z\": 37.957}, \"end\": {\"x\": 25.744, \"y\": 19.357, \"z\": 35.936}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.262, \"y\": 20.57, \"z\": 33.201}, \"end\": {\"x\": 25.968, \"y\": 20.323, \"z\": 31.144}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.918, \"y\": 23.023, \"z\": 38.917}, \"end\": {\"x\": 25.029, \"y\": 22.751, \"z\": 36.704}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.223, \"y\": 21.668, \"z\": 34.52}, \"end\": {\"x\": 23.015, \"y\": 21.602, \"z\": 32.33}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.94593277204187, \"y\": 20.80424270142054, \"z\": 31.897368431077453}, \"end\": {\"x\": 19.194710442396982, \"y\": 18.954379243328006, \"z\": 31.022428820684457}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.03790525184778, \"y\": 19.037638159609415, \"z\": 32.582111819737065}, \"end\": {\"x\": 19.194710442396982, \"y\": 18.954379243328006, \"z\": 31.022428820684457}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.035, \"y\": 22.404, \"z\": 40.744}, \"end\": {\"x\": 22.236, \"y\": 22.068, \"z\": 38.546}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.081, \"y\": 19.898, \"z\": 36.603}, \"end\": {\"x\": 22.55, \"y\": 19.111, \"z\": 34.601}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.59815944893826, \"y\": 18.199374880194412, \"z\": 41.726317548017576}, \"end\": {\"x\": 21.439824144373016, \"y\": 17.346597985060296, \"z\": 40.044344371162204}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.602348882858973, \"y\": 16.45989375526764, \"z\": 42.76911447950715}, \"end\": {\"x\": 22.76602818806995, \"y\": 16.27326641720404, \"z\": 41.30500076771362}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.104, \"y\": 20.885, \"z\": 42.672}, \"end\": {\"x\": 24.058, \"y\": 19.984, \"z\": 40.639}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.394, \"y\": 20.436, \"z\": 37.957}, \"end\": {\"x\": 25.744, \"y\": 19.357, \"z\": 35.936}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.819, \"y\": 22.741, \"z\": 43.557}, \"end\": {\"x\": 26.339, \"y\": 22.383, \"z\": 41.519}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.918, \"y\": 23.023, \"z\": 38.917}, \"end\": {\"x\": 25.029, \"y\": 22.751, \"z\": 36.704}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.289, \"y\": 24.519, \"z\": 44.722}, \"end\": {\"x\": 23.774, \"y\": 24.727, \"z\": 42.546}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.035, \"y\": 22.404, \"z\": 40.744}, \"end\": {\"x\": 22.236, \"y\": 22.068, \"z\": 38.546}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.59815944893826, \"y\": 18.199374880194412, \"z\": 41.726317548017576}, \"end\": {\"x\": 21.439824144373016, \"y\": 17.346597985060296, \"z\": 40.044344371162204}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.602348882858973, \"y\": 16.45989375526764, \"z\": 42.76911447950715}, \"end\": {\"x\": 22.76602818806995, \"y\": 16.27326641720404, \"z\": 41.30500076771362}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.364, \"y\": 23.097, \"z\": 46.881}, \"end\": {\"x\": 22.635, \"y\": 22.42, \"z\": 44.836}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.104, \"y\": 20.885, \"z\": 42.672}, \"end\": {\"x\": 24.058, \"y\": 19.984, \"z\": 40.639}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.756, \"y\": 22.948, \"z\": 48.072}, \"end\": {\"x\": 25.772, \"y\": 21.684, \"z\": 46.403}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.819, \"y\": 22.741, \"z\": 43.557}, \"end\": {\"x\": 26.339, \"y\": 22.383, \"z\": 41.519}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.123, \"y\": 25.333, \"z\": 49.056}, \"end\": {\"x\": 26.356, \"y\": 25.182, \"z\": 46.844}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.289, \"y\": 24.519, \"z\": 44.722}, \"end\": {\"x\": 23.774, \"y\": 24.727, \"z\": 42.546}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.399, \"y\": 25.596, \"z\": 50.785}, \"end\": {\"x\": 23.55, \"y\": 25.537, \"z\": 48.67}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.364, \"y\": 23.097, \"z\": 46.881}, \"end\": {\"x\": 22.635, \"y\": 22.42, \"z\": 44.836}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.016, \"y\": 24.035, \"z\": 52.772}, \"end\": {\"x\": 24.569, \"y\": 22.85, \"z\": 50.86}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.756, \"y\": 22.948, \"z\": 48.072}, \"end\": {\"x\": 25.772, \"y\": 21.684, \"z\": 46.403}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.064, \"y\": 25.273, \"z\": 53.587}, \"end\": {\"x\": 27.856, \"y\": 24.363, \"z\": 51.347}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.224271632369327, \"y\": 30.276877354867583, \"z\": 46.32545610227355}, \"end\": {\"x\": 31.510631876663993, \"y\": 30.67397005123429, \"z\": 45.13401426878197}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.123, \"y\": 25.333, \"z\": 49.056}, \"end\": {\"x\": 26.356, \"y\": 25.182, \"z\": 46.844}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.399, \"y\": 25.596, \"z\": 50.785}, \"end\": {\"x\": 23.55, \"y\": 25.537, \"z\": 48.67}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.016, \"y\": 24.035, \"z\": 52.772}, \"end\": {\"x\": 24.569, \"y\": 22.85, \"z\": 50.86}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.79, \"y\": 25.183, \"z\": 58.359}, \"end\": {\"x\": 27.083, \"y\": 23.993, \"z\": 56.336}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.064, \"y\": 25.273, \"z\": 53.587}, \"end\": {\"x\": 27.856, \"y\": 24.363, \"z\": 51.347}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.804, \"y\": 27.588, \"z\": 59.308}, \"end\": {\"x\": 28.907, \"y\": 27.181, \"z\": 56.754}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.149, \"y\": 28.59, \"z\": 61.293}, \"end\": {\"x\": 26.188, \"y\": 28.422, \"z\": 59.054}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.939, \"y\": 26.68, \"z\": 63.176}, \"end\": {\"x\": 26.818, \"y\": 25.729, \"z\": 61.183}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.79, \"y\": 25.183, \"z\": 58.359}, \"end\": {\"x\": 27.083, \"y\": 23.993, \"z\": 56.336}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.804, \"y\": 27.588, \"z\": 59.308}, \"end\": {\"x\": 28.907, \"y\": 27.181, \"z\": 56.754}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.149, \"y\": 28.59, \"z\": 61.293}, \"end\": {\"x\": 26.188, \"y\": 28.422, \"z\": 59.054}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.939, \"y\": 26.68, \"z\": 63.176}, \"end\": {\"x\": 26.818, \"y\": 25.729, \"z\": 61.183}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.64119334542987, \"y\": 22.88729805832813, \"z\": 65.95167460773206}, \"end\": {\"x\": 27.931636423541722, \"y\": 21.45143039853578, \"z\": 66.57403245473779}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.37, \"y\": 22.081, \"z\": 63.926}, \"end\": {\"x\": 28.447, \"y\": 23.835, \"z\": 63.869}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.64119334542987, \"y\": 22.88729805832813, \"z\": 65.95167460773206}, \"end\": {\"x\": 27.931636423541722, \"y\": 21.45143039853578, \"z\": 66.57403245473779}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.37, \"y\": 22.081, \"z\": 63.926}, \"end\": {\"x\": 28.447, \"y\": 23.835, \"z\": 63.869}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.21866693600841, \"y\": 12.330188316307208, \"z\": 66.40965255518343}, \"end\": {\"x\": 33.74, \"y\": 11.731, \"z\": 64.744}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.853, \"y\": 16.164, \"z\": 59.522}, \"end\": {\"x\": 31.477, \"y\": 15.932, \"z\": 61.848}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.62, \"y\": 16.109, \"z\": 57.815}, \"end\": {\"x\": 28.05, \"y\": 15.954, \"z\": 60.054}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.238, \"y\": 18.087, \"z\": 56.065}, \"end\": {\"x\": 28.997, \"y\": 18.64, \"z\": 58.015}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.416, \"y\": 16.898, \"z\": 54.815}, \"end\": {\"x\": 31.911, \"y\": 17.279, \"z\": 56.906}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.853, \"y\": 16.164, \"z\": 59.522}, \"end\": {\"x\": 31.477, \"y\": 15.932, \"z\": 61.848}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.142, \"y\": 14.978, \"z\": 53.369}, \"end\": {\"x\": 30.222, \"y\": 14.625, \"z\": 55.707}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.62, \"y\": 16.109, \"z\": 57.815}, \"end\": {\"x\": 28.05, \"y\": 15.954, \"z\": 60.054}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.645, \"y\": 16.312, \"z\": 51.447}, \"end\": {\"x\": 28.124, \"y\": 16.521, \"z\": 53.719}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.238, \"y\": 18.087, \"z\": 56.065}, \"end\": {\"x\": 28.997, \"y\": 18.64, \"z\": 58.015}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.691, \"y\": 17.367, \"z\": 50.109}, \"end\": {\"x\": 30.706, \"y\": 18.194, \"z\": 52.168}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.416, \"y\": 16.898, \"z\": 54.815}, \"end\": {\"x\": 31.911, \"y\": 17.279, \"z\": 56.906}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.771, \"y\": 15.21, \"z\": 48.814}, \"end\": {\"x\": 32.119, \"y\": 15.207, \"z\": 51.08}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.142, \"y\": 14.978, \"z\": 53.369}, \"end\": {\"x\": 30.222, \"y\": 14.625, \"z\": 55.707}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.69, \"y\": 14.675, \"z\": 47.211}, \"end\": {\"x\": 29.327, \"y\": 14.221, \"z\": 49.362}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.645, \"y\": 16.312, \"z\": 51.447}, \"end\": {\"x\": 28.124, \"y\": 16.521, \"z\": 53.719}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.807, \"y\": 16.858, \"z\": 45.618}, \"end\": {\"x\": 29.207, \"y\": 17.346, \"z\": 47.463}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.691, \"y\": 17.367, \"z\": 50.109}, \"end\": {\"x\": 30.706, \"y\": 18.194, \"z\": 52.168}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.216, \"y\": 16.528, \"z\": 44.361}, \"end\": {\"x\": 32.427, \"y\": 16.884, \"z\": 46.368}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.771, \"y\": 15.21, \"z\": 48.814}, \"end\": {\"x\": 32.119, \"y\": 15.207, \"z\": 51.08}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.674, \"y\": 14.204, \"z\": 43.037}, \"end\": {\"x\": 31.471, \"y\": 13.728, \"z\": 44.948}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.69, \"y\": 14.675, \"z\": 47.211}, \"end\": {\"x\": 29.327, \"y\": 14.221, \"z\": 49.362}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.611, \"y\": 15.106, \"z\": 41.273}, \"end\": {\"x\": 28.932, \"y\": 15.289, \"z\": 43.176}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.807, \"y\": 16.858, \"z\": 45.618}, \"end\": {\"x\": 29.207, \"y\": 17.346, \"z\": 47.463}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.344, \"y\": 16.732, \"z\": 39.924}, \"end\": {\"x\": 31.067, \"y\": 17.521, \"z\": 41.737}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.216, \"y\": 16.528, \"z\": 44.361}, \"end\": {\"x\": 32.427, \"y\": 16.884, \"z\": 46.368}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.164, \"y\": 15.183, \"z\": 38.618}, \"end\": {\"x\": 33.307, \"y\": 15.065, \"z\": 40.633}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.674, \"y\": 14.204, \"z\": 43.037}, \"end\": {\"x\": 31.471, \"y\": 13.728, \"z\": 44.948}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.385, \"y\": 13.938, \"z\": 36.981}, \"end\": {\"x\": 30.931, \"y\": 13.574, \"z\": 39.011}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.611, \"y\": 15.106, \"z\": 41.273}, \"end\": {\"x\": 28.932, \"y\": 15.289, \"z\": 43.176}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.53, \"y\": 15.995, \"z\": 35.33}, \"end\": {\"x\": 29.849, \"y\": 16.373, \"z\": 37.309}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.66974175102478, \"y\": 20.5402127688388, \"z\": 37.488125467948954}, \"end\": {\"x\": 34.08108314865374, \"y\": 18.941081444814955, \"z\": 38.50586644813445}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.344, \"y\": 16.732, \"z\": 39.924}, \"end\": {\"x\": 31.067, \"y\": 17.521, \"z\": 41.737}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.981, \"y\": 16.461, \"z\": 34.044}, \"end\": {\"x\": 33.021, \"y\": 16.957, \"z\": 36.162}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.14916747448213, \"y\": 18.2588541617528, \"z\": 36.823841674703004}, \"end\": {\"x\": 34.08108314865374, \"y\": 18.941081444814955, \"z\": 38.50586644813445}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.164, \"y\": 15.183, \"z\": 38.618}, \"end\": {\"x\": 33.307, \"y\": 15.065, \"z\": 40.633}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.013, \"y\": 14.28, \"z\": 32.521}, \"end\": {\"x\": 33.37, \"y\": 13.853, \"z\": 34.688}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.385, \"y\": 13.938, \"z\": 36.981}, \"end\": {\"x\": 30.931, \"y\": 13.574, \"z\": 39.011}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.102, \"y\": 14.448, \"z\": 30.623}, \"end\": {\"x\": 30.467, \"y\": 14.168, \"z\": 32.937}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.819871868901842, \"y\": 20.097132435132995, \"z\": 33.63873775807972}, \"end\": {\"x\": 32.77328843911919, \"y\": 20.32190090795406, \"z\": 35.184731349317204}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.53, \"y\": 15.995, \"z\": 35.33}, \"end\": {\"x\": 29.849, \"y\": 16.373, \"z\": 37.309}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.125, \"y\": 16.599, \"z\": 29.414}, \"end\": {\"x\": 31.606, \"y\": 17.049, \"z\": 31.503}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.981, \"y\": 16.461, \"z\": 34.044}, \"end\": {\"x\": 33.021, \"y\": 16.957, \"z\": 36.162}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.14916747448213, \"y\": 18.2588541617528, \"z\": 36.823841674703004}, \"end\": {\"x\": 34.08108314865374, \"y\": 18.941081444814955, \"z\": 38.50586644813445}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.535, \"y\": 15.711, \"z\": 28.242}, \"end\": {\"x\": 34.723, \"y\": 15.573, \"z\": 30.375}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.013, \"y\": 14.28, \"z\": 32.521}, \"end\": {\"x\": 33.37, \"y\": 13.853, \"z\": 34.688}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.335, \"y\": 13.983, \"z\": 26.435}, \"end\": {\"x\": 32.976, \"y\": 13.342, \"z\": 28.511}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.102, \"y\": 14.448, \"z\": 30.623}, \"end\": {\"x\": 30.467, \"y\": 14.168, \"z\": 32.937}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.222, \"y\": 15.911, \"z\": 24.692}, \"end\": {\"x\": 31.153, \"y\": 15.726, \"z\": 26.677}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.72087825595617, \"y\": 20.880175444877075, \"z\": 27.136601848196545}, \"end\": {\"x\": 34.31508507317345, \"y\": 19.48007487537707, \"z\": 28.20927142849176}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.125, \"y\": 16.599, \"z\": 29.414}, \"end\": {\"x\": 31.606, \"y\": 17.049, \"z\": 31.503}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.378, \"y\": 16.845, \"z\": 23.648}, \"end\": {\"x\": 33.967, \"y\": 17.371, \"z\": 25.765}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.014739213966315, \"y\": 19.16640831016724, \"z\": 26.700836371012787}, \"end\": {\"x\": 34.31508507317345, \"y\": 19.48007487537707, \"z\": 28.20927142849176}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.535, \"y\": 15.711, \"z\": 28.242}, \"end\": {\"x\": 34.723, \"y\": 15.573, \"z\": 30.375}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.327, \"y\": 14.938, \"z\": 22.093}, \"end\": {\"x\": 35.415, \"y\": 14.603, \"z\": 24.314}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.157850437434107, \"y\": 11.27474139717846, \"z\": 25.58315812187468}, \"end\": {\"x\": 29.173031995488035, \"y\": 11.484988568726713, \"z\": 25.344087608353572}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.841190198975447, \"y\": 10.81166564254388, \"z\": 27.72207275072217}, \"end\": {\"x\": 29.806315435843562, \"y\": 10.472892507048423, \"z\": 27.105802516057818}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.335, \"y\": 13.983, \"z\": 26.435}, \"end\": {\"x\": 32.976, \"y\": 13.342, \"z\": 28.511}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.452, \"y\": 14.791, \"z\": 20.062}, \"end\": {\"x\": 32.772, \"y\": 14.179, \"z\": 22.305}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.32683807501296, \"y\": 20.35260431802973, \"z\": 23.540226834555213}, \"end\": {\"x\": 32.18137603332113, \"y\": 20.741199970340038, \"z\": 25.16323827728038}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.222, \"y\": 15.911, \"z\": 24.692}, \"end\": {\"x\": 31.153, \"y\": 15.726, \"z\": 26.677}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.828, \"y\": 17.309, \"z\": 18.987}, \"end\": {\"x\": 33.099, \"y\": 17.588, \"z\": 21.094}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.378, \"y\": 16.845, \"z\": 23.648}, \"end\": {\"x\": 33.967, \"y\": 17.371, \"z\": 25.765}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.014739213966315, \"y\": 19.16640831016724, \"z\": 26.700836371012787}, \"end\": {\"x\": 34.31508507317345, \"y\": 19.48007487537707, \"z\": 28.20927142849176}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.327, \"y\": 14.938, \"z\": 22.093}, \"end\": {\"x\": 35.415, \"y\": 14.603, \"z\": 24.314}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.452, \"y\": 14.791, \"z\": 20.062}, \"end\": {\"x\": 32.772, \"y\": 14.179, \"z\": 22.305}});\n",
              "\tviewer_16234219519367425.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.828, \"y\": 17.309, \"z\": 18.987}, \"end\": {\"x\": 33.099, \"y\": 17.588, \"z\": 21.094}});\n",
              "\tviewer_16234219519367425.zoomTo();\n",
              "viewer_16234219519367425.render();\n",
              "});\n",
              "</script>"
            ]
          },
          "metadata": {
            "tags": []
          }
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "1xPr3_8rmdlW"
      },
      "source": [
        " Next, we need to apply certain movers with our design specifications to activate certain non-pairwise decomposable scoreterms in the `design_scorefxn` scorefunction, that will be implemented when we run the `FastDesign` mover (or any downstream mover that calls the packer).\n",
        "\n",
        " We will frequently be using the following residue selectors:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "20tPmJXJmdlX"
      },
      "source": [
        "chain_A = pyrosetta.rosetta.core.select.residue_selector.ChainSelector(\"A\")\n",
        "chain_BC = pyrosetta.rosetta.core.select.residue_selector.NotResidueSelector(chain_A)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "Xhn-zojamdlX"
      },
      "source": [
        " Apply the `AddCompositionConstraintMover` mover, which utilizes the `aa_composition` scoreterm. Applying the following mover to the pose imposes the design constraints that the ResidueSelector have 40% aliphatic or aromatic residues other than leucine (i.e. ALA, PHE, ILE, MET, PRO, VAL, TRP, or TYR), and 5% leucines. For documentation, see: https://www.rosettacommons.org/docs/latest/rosetta_basics/scoring/AACompositionEnergy"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "scrolled": true,
        "id": "KxR_7t6omdlY"
      },
      "source": [
        "add_composition_constraint = pyrosetta.rosetta.protocols.aa_composition.AddCompositionConstraintMover()\n",
        "add_composition_constraint.create_constraint_from_file_contents(\"\"\"\n",
        "PENALTY_DEFINITION\n",
        "OR_PROPERTIES AROMATIC ALIPHATIC\n",
        "NOT_TYPE LEU\n",
        "FRACT_DELTA_START -0.05\n",
        "FRACT_DELTA_END 0.05\n",
        "PENALTIES 1 0 1 # The above two lines mean that if we're 5% below or 5% above the desired content, we get a 1-point penalty.\n",
        "FRACTION 0.4 # Forty percent aromatic or aliphatic, but not leucine\n",
        "BEFORE_FUNCTION CONSTANT\n",
        "AFTER_FUNCTION CONSTANT\n",
        "END_PENALTY_DEFINITION\n",
        "\n",
        "PENALTY_DEFINITION\n",
        "TYPE LEU\n",
        "DELTA_START -1\n",
        "DELTA_END 1\n",
        "PENALTIES 1 0 1\n",
        "FRACTION 0.05 # Five percent leucine\n",
        "BEFORE_FUNCTION CONSTANT\n",
        "AFTER_FUNCTION CONSTANT\n",
        "END_PENALTY_DEFINITION\n",
        "\"\"\")\n",
        "add_composition_constraint.add_residue_selector(chain_A)\n",
        "add_composition_constraint.apply(pose)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "HutqlW_ZmdlY"
      },
      "source": [
        " Apply the `AddHelixSequenceConstraints` mover, which utilizes the `aa_composition` scoreterm. By default, this mover adds five types of sequence constraints to the designable residues in each alpha helix in the pose. Any of these behaviours may be disabled or modified by invoking advanced options, but no advanced options need be set in most cases. The five types of sequence constraints are:\n",
        "\n",
        "- A strong sequence constraint requiring at least two negatively-charged residues in the first (N-terminal) three residues of each alpha-helix.\n",
        "\n",
        "- A strong sequence constraint requiring at least two positively-charged residues in the last (C-terminal) three residues of each alpha-helix.\n",
        "\n",
        "- A weak but strongly ramping sequence constraint penalizing helix-disfavoring residue types (by default, Asn, Asp, Ser, Gly, Thr, and Val) throughout each helix. (A single such residue is sometimes tolerated, but the penalty for having more than one residue in this category increases quadratically with the count of helix-disfavouring residues.)\n",
        "\n",
        "- A weak sequence constraint coaxing the helix to have 10% alanine. Because this constraint is weak, deviations from this value are tolerated, but this should prevent an excessive abundance of alanine residues.\n",
        "\n",
        "- A weak sequence constraint coaxing the helix to have at least 25% hydrophobic content. This constraint is also weak, so slightly less hydrophobic helices will be tolerated to some degree. Note that alanine is not considered to be \"hydrophobic\" within Rosetta. \n",
        "\n",
        "### For documentation, see: https://www.rosettacommons.org/docs/latest/rosetta_basics/scoring/AACompositionEnergy"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "PgNr4DWPmdlY"
      },
      "source": [
        "add_helix_sequence_constraints = pyrosetta.rosetta.protocols.aa_composition.AddHelixSequenceConstraintsMover()\n",
        "add_helix_sequence_constraints.set_residue_selector(chain_A)\n",
        "add_helix_sequence_constraints.apply(pose)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "eC3j8qIxmdlY"
      },
      "source": [
        "Note: the `aa_repeat` scoreterm works out-of-the-box, and does not need to be applied to the pose to work, it just needs to have a weight of >0 in the scorefunction used by the packer. It imposes a penalty for each stretch of repeating amino acids, with the penalty value depending nonlinearly on the length of the repeating stretch. By default, 1- or 2-residue stretches incur no penalty, 3-residue stretches incur a penalty of +1, 4-residue stretches incur a penalty of +10, and 5-residue stretches or longer incur a penalty of +100. Since the term is sequence-based, it is really only useful for design -- that is, it will impose an identical penalty for a fixed-sequence pose, regardless its conformation. This also means that the term has no conformational derivatives: the minimizer ignores it completely. The term is not pairwise-decomposible, but has been made packer-compatible, so it can direct the sequence composition during a packer run. For documentation, see: https://www.rosettacommons.org/docs/latest/rosetta_basics/scoring/Repeat-stretch-energy"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "FrjVT6TsmdlZ"
      },
      "source": [
        " Similarly, the `voids_penalty` scoreterm does not need to be applied to the pose to work, it just needs to have a weight of >0 in the scorefunction used by the packer. For documentation, see: https://www.rosettacommons.org/docs/latest/rosetta_basics/scoring/VoidsPenaltyEnergy"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "38RUzNeymdlZ"
      },
      "source": [
        " Similarly, the `buried_unsatisfied_penalty` scoreterm does not need to be applied to the pose to work, it just needs to have a weight of >0 in the scorefunction used by the packer. For documentation, see: https://www.rosettacommons.org/docs/latest/rosetta_basics/scoring/BuriedUnsatPenalty"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "Ru1u-Qb2mdlZ"
      },
      "source": [
        " Similarly, the `hbnet` scoreterm does not need to be applied to the pose to work, it just needs to have a weight of >0 (ideally between 1.0 to 10.0) in the scorefunction used by the packer. For documentation, see: https://www.rosettacommons.org/docs/latest/rosetta_basics/scoring/HBNetEnergy"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "mV_W_pVFmdlZ"
      },
      "source": [
        " Apply the `AddNetChargeConstraintMover` mover, which utilizes the `netcharge` scoreterm. In this case, we require that the net charge in chain A must be exactly 0."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "iUxLYssfmdlZ"
      },
      "source": [
        "add_net_charge_constraint = pyrosetta.rosetta.protocols.aa_composition.AddNetChargeConstraintMover()\n",
        "add_net_charge_constraint.create_constraint_from_file_contents(\"\"\"\n",
        "DESIRED_CHARGE 0 #Desired net charge is zero.\n",
        "PENALTIES_CHARGE_RANGE -1 1 #Penalties are listed in the observed net charge range of -1 to +1.\n",
        "PENALTIES 1 0 1 #The penalties are 1 for an observed charge of -1, 0 for an observed charge of 0, and 1 for an observed charge of +1.\n",
        "BEFORE_FUNCTION QUADRATIC #Ramp quadratically for observed net charges of -2 or less.\n",
        "AFTER_FUNCTION QUADRATIC #Ramp quadratically for observed net charges of +2 or greater.\n",
        "\"\"\")\n",
        "add_net_charge_constraint.add_residue_selector(chain_A)\n",
        "add_net_charge_constraint.apply(pose)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "0gR-JTrdmdla"
      },
      "source": [
        " Specify a custom `relaxscript` that optimizes the `ramp_repack_min` weights to prevent too many alanines from being designed (demonstrated at Pre-RosettaCON 2018):"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "WW5pzgVTmdla"
      },
      "source": [
        " Specify TaskOperations to be applied to chain A. In this case, let's use the latest LayerDesign implementation (Note: this is still being actively developed) using the XmlObjects class."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "QpKP8P7Vmdla",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "6bff12a5-e1d8-44d9-a7aa-8742cf3182ad"
      },
      "source": [
        "layer_design_task = pyrosetta.rosetta.protocols.rosetta_scripts.XmlObjects.create_from_string(\"\"\"\n",
        "<RESIDUE_SELECTORS>\n",
        "  <Layer name=\"surface\" select_core=\"false\" select_boundary=\"false\" select_surface=\"true\" use_sidechain_neighbors=\"true\"/>\n",
        "  <Layer name=\"boundary\" select_core=\"false\" select_boundary=\"true\" select_surface=\"false\" use_sidechain_neighbors=\"true\"/>\n",
        "  <Layer name=\"core\" select_core=\"true\" select_boundary=\"false\" select_surface=\"false\" use_sidechain_neighbors=\"true\"/>\n",
        "  <SecondaryStructure name=\"sheet\" overlap=\"0\" minH=\"3\" minE=\"2\" include_terminal_loops=\"false\" use_dssp=\"true\" ss=\"E\"/>\n",
        "  <SecondaryStructure name=\"entire_loop\" overlap=\"0\" minH=\"3\" minE=\"2\" include_terminal_loops=\"true\" use_dssp=\"true\" ss=\"L\"/>\n",
        "  <SecondaryStructure name=\"entire_helix\" overlap=\"0\" minH=\"3\" minE=\"2\" include_terminal_loops=\"false\" use_dssp=\"true\" ss=\"H\"/>\n",
        "  <And name=\"helix_cap\" selectors=\"entire_loop\">\n",
        "   <PrimarySequenceNeighborhood lower=\"1\" upper=\"0\" selector=\"entire_helix\"/>\n",
        "  </And>\n",
        "  <And name=\"helix_start\" selectors=\"entire_helix\">\n",
        "    <PrimarySequenceNeighborhood lower=\"0\" upper=\"1\" selector=\"helix_cap\"/>\n",
        "  </And>\n",
        "  <And name=\"helix\" selectors=\"entire_helix\">\n",
        "    <Not selector=\"helix_start\"/>\n",
        "  </And>\n",
        "  <And name=\"loop\" selectors=\"entire_loop\">\n",
        "    <Not selector=\"helix_cap\"/>\n",
        "  </And>\n",
        "</RESIDUE_SELECTORS>\n",
        "<TASKOPERATIONS>\n",
        "  <DesignRestrictions name=\"layer_design\">\n",
        "    <Action selector_logic=\"surface AND helix_start\" aas=\"EHKPQR\"/>\n",
        "    <Action selector_logic=\"surface AND helix\" aas=\"EHKQR\"/>\n",
        "    <Action selector_logic=\"surface AND sheet\" aas=\"DEHKNQRST\"/>\n",
        "    <Action selector_logic=\"surface AND loop\" aas=\"DEGHKNPQRST\"/>\n",
        "    <Action selector_logic=\"boundary AND helix_start\" aas=\"ADEIKLMNPQRSTVWY\"/>\n",
        "    <Action selector_logic=\"boundary AND helix\" aas=\"ADEIKLMNQRSTVWY\"/>\n",
        "    <Action selector_logic=\"boundary AND sheet\" aas=\"DEFIKLNQRSTVWY\"/>\n",
        "    <Action selector_logic=\"boundary AND loop\" aas=\"ADEFGIKLMNPQRSTVWY\"/>\n",
        "    <Action selector_logic=\"core AND helix_start\" aas=\"AFILMPVWY\"/>\n",
        "    <Action selector_logic=\"core AND helix\" aas=\"AFILMVWY\"/>\n",
        "    <Action selector_logic=\"core AND sheet\" aas=\"FILVWY\"/>\n",
        "    <Action selector_logic=\"core AND loop\" aas=\"AFGILMPVWY\"/>\n",
        "    <Action selector_logic=\"helix_cap\" aas=\"DNST\"/>\n",
        "  </DesignRestrictions>\n",
        "</TASKOPERATIONS>\n",
        "\"\"\").get_task_operation(\"layer_design\")"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Generating XML Schema for rosetta_scripts...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Initializing schema validator...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Validating input script...\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "INFO:rosetta:protocols.rosetta_scripts.RosettaScriptsParser: Parsed script:\n",
            "<ROSETTASCRIPTS>\n",
            "\t<RESIDUE_SELECTORS>\n",
            "\t\t<Layer name=\"surface\" select_boundary=\"false\" select_core=\"false\" select_surface=\"true\" use_sidechain_neighbors=\"true\"/>\n",
            "\t\t<Layer name=\"boundary\" select_boundary=\"true\" select_core=\"false\" select_surface=\"false\" use_sidechain_neighbors=\"true\"/>\n",
            "\t\t<Layer name=\"core\" select_boundary=\"false\" select_core=\"true\" select_surface=\"false\" use_sidechain_neighbors=\"true\"/>\n",
            "\t\t<SecondaryStructure include_terminal_loops=\"false\" minE=\"2\" minH=\"3\" name=\"sheet\" overlap=\"0\" ss=\"E\" use_dssp=\"true\"/>\n",
            "\t\t<SecondaryStructure include_terminal_loops=\"true\" minE=\"2\" minH=\"3\" name=\"entire_loop\" overlap=\"0\" ss=\"L\" use_dssp=\"true\"/>\n",
            "\t\t<SecondaryStructure include_terminal_loops=\"false\" minE=\"2\" minH=\"3\" name=\"entire_helix\" overlap=\"0\" ss=\"H\" use_dssp=\"true\"/>\n",
            "\t\t<And name=\"helix_cap\" selectors=\"entire_loop\">\n",
            "\t\t\t<PrimarySequenceNeighborhood lower=\"1\" selector=\"entire_helix\" upper=\"0\"/>\n",
            "\t\t</And>\n",
            "\t\t<And name=\"helix_start\" selectors=\"entire_helix\">\n",
            "\t\t\t<PrimarySequenceNeighborhood lower=\"0\" selector=\"helix_cap\" upper=\"1\"/>\n",
            "\t\t</And>\n",
            "\t\t<And name=\"helix\" selectors=\"entire_helix\">\n",
            "\t\t\t<Not selector=\"helix_start\"/>\n",
            "\t\t</And>\n",
            "\t\t<And name=\"loop\" selectors=\"entire_loop\">\n",
            "\t\t\t<Not selector=\"helix_cap\"/>\n",
            "\t\t</And>\n",
            "\t</RESIDUE_SELECTORS>\n",
            "\t<TASKOPERATIONS>\n",
            "\t\t<DesignRestrictions name=\"layer_design\">\n",
            "\t\t\t<Action aas=\"EHKPQR\" selector_logic=\"surface AND helix_start\"/>\n",
            "\t\t\t<Action aas=\"EHKQR\" selector_logic=\"surface AND helix\"/>\n",
            "\t\t\t<Action aas=\"DEHKNQRST\" selector_logic=\"surface AND sheet\"/>\n",
            "\t\t\t<Action aas=\"DEGHKNPQRST\" selector_logic=\"surface AND loop\"/>\n",
            "\t\t\t<Action aas=\"ADEIKLMNPQRSTVWY\" selector_logic=\"boundary AND helix_start\"/>\n",
            "\t\t\t<Action aas=\"ADEIKLMNQRSTVWY\" selector_logic=\"boundary AND helix\"/>\n",
            "\t\t\t<Action aas=\"DEFIKLNQRSTVWY\" selector_logic=\"boundary AND sheet\"/>\n",
            "\t\t\t<Action aas=\"ADEFGIKLMNPQRSTVWY\" selector_logic=\"boundary AND loop\"/>\n",
            "\t\t\t<Action aas=\"AFILMPVWY\" selector_logic=\"core AND helix_start\"/>\n",
            "\t\t\t<Action aas=\"AFILMVWY\" selector_logic=\"core AND helix\"/>\n",
            "\t\t\t<Action aas=\"FILVWY\" selector_logic=\"core AND sheet\"/>\n",
            "\t\t\t<Action aas=\"AFGILMPVWY\" selector_logic=\"core AND loop\"/>\n",
            "\t\t\t<Action aas=\"DNST\" selector_logic=\"helix_cap\"/>\n",
            "\t\t</DesignRestrictions>\n",
            "\t</TASKOPERATIONS>\n",
            "\t<PROTOCOLS/>\n",
            "</ROSETTASCRIPTS>\n",
            "INFO:rosetta:core.scoring.ScoreFunctionFactory: SCOREFUNCTION: ref2015\n",
            "INFO:rosetta:protocols.rosetta_scripts.ParsedProtocol: ParsedProtocol mover with the following settings\n"
          ],
          "name": "stderr"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "esw8N4i1mdla"
      },
      "source": [
        " Also prepare a ResidueSelector for the heterotrimer interface within chains B and C, and the rest of chains B and C. We will use these with `RestrictAbsentCanonicalAASRLT`, `RestrictToRepackingRLT`, and `PreventRepackingRLT` TaskOperations:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "b19Ct9V8mdla"
      },
      "source": [
        "interface = pyrosetta.rosetta.core.select.residue_selector.InterGroupInterfaceByVectorSelector()\n",
        "interface.group1_selector(chain_A)\n",
        "interface.group2_selector(chain_BC)\n",
        "chain_BC_interface = pyrosetta.rosetta.core.select.residue_selector.AndResidueSelector(interface, chain_BC)\n",
        "not_chain_BC_interface = pyrosetta.rosetta.core.select.residue_selector.NotResidueSelector(chain_BC_interface)\n",
        "chain_BC_not_interface = pyrosetta.rosetta.core.select.residue_selector.AndResidueSelector(not_chain_BC_interface, chain_BC)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "wrT5FBrXmdlb"
      },
      "source": [
        " For minimization, we also need the following ResidueSelector:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "wWYRhQ9kmdlb"
      },
      "source": [
        "chain_A_and_BC_interface = pyrosetta.rosetta.core.select.residue_selector.OrResidueSelector(chain_A, chain_BC_interface)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "YNofyAwvmdlb"
      },
      "source": [
        " Make sure each ResidueSelector selects the regions as desired (in the following case, the `ResidueSelector` `interface` is visualized:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "DBGGYlW_mdlb",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 817
        },
        "outputId": "f044ce64-0087-480a-950d-56520a85f0eb"
      },
      "source": [
        "view = viewer.init(pose) \\\n",
        "    + viewer.setStyle() \\\n",
        "    + viewer.setStyle(residue_selector=interface, colorscheme=\"greyCarbon\")\n",
        "view()"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_16234223362302048\"  style=\"position: relative; width: 1200px; height: 800px\">\n        <p id=\"3dmolwarning_16234223362302048\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n      resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n}\n\nvar viewer_16234223362302048 = null;\nvar warn = document.getElementById(\"3dmolwarning_16234223362302048\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_16234223362302048 = $3Dmol.createViewer($(\"#3dmolviewer_16234223362302048\"),{backgroundColor:\"white\"});\nviewer_16234223362302048.zoomTo();\n\tviewer_16234223362302048.addModels(\"HEADER                                            11-JUN-21   XXXX              \\nEXPDTA    THEORETICAL MODEL                                                     \\nREMARK 220                                                                      \\nREMARK 220 EXPERIMENTAL DETAILS                                                 \\nREMARK 220  EXPERIMENT TYPE                : THEORETICAL MODELLING              \\nREMARK 220  DATE OF DATA COLLECTION        : 11-JUN-21                          \\nREMARK 220                                                                      \\nREMARK 220 REMARK: MODEL GENERATED BY ROSETTA                                   \\nREMARK 220  VERSION 2020.50.post.dev+970.commits.3700df14560                    \\nATOM      1  N   ALA A   1      46.718  19.029  14.515  1.00  0.00           N  \\nATOM      2  CA  ALA A   1      45.711  17.981  14.626  1.00  0.00           C  \\nATOM      3  C   ALA A   1      46.287  16.744  15.312  1.00  0.00           C  \\nATOM      4  O   ALA A   1      45.575  16.034  16.023  1.00  0.00           O  \\nATOM      5  CB  ALA A   1      45.165  17.622  13.252  1.00  0.00           C  \\nATOM      6 1H   ALA A   1      46.320  19.828  14.064  1.00  0.00           H  \\nATOM      7 2H   ALA A   1      47.035  19.286  15.428  1.00  0.00           H  \\nATOM      8 3H   ALA A   1      47.491  18.693  13.977  1.00  0.00           H  \\nATOM      9  HA  ALA A   1      44.974  18.308  15.166  1.00  0.00           H  \\nATOM     10 1HB  ALA A   1      44.415  16.838  13.352  1.00  0.00           H  \\nATOM     11 2HB  ALA A   1      44.710  18.504  12.800  1.00  0.00           H  \\nATOM     12 3HB  ALA A   1      45.977  17.269  12.619  1.00  0.00           H  \\nATOM     13  N   ALA A   2      47.573  16.485  15.094  1.00  0.00           N  \\nATOM     14  CA  ALA A   2      48.260  15.426  15.824  1.00  0.00           C  \\nATOM     15  C   ALA A   2      48.210  15.740  17.310  1.00  0.00           C  \\nATOM     16  O   ALA A   2      48.033  14.850  18.140  1.00  0.00           O  \\nATOM     17  CB  ALA A   2      49.698  15.278  15.349  1.00  0.00           C  \\nATOM     18  H   ALA A   2      48.068  16.907  14.532  1.00  0.00           H  \\nATOM     19  HA  ALA A   2      47.807  14.582  15.672  1.00  0.00           H  \\nATOM     20 1HB  ALA A   2      50.193  14.496  15.925  1.00  0.00           H  \\nATOM     21 2HB  ALA A   2      49.707  15.011  14.293  1.00  0.00           H  \\nATOM     22 3HB  ALA A   2      50.226  16.220  15.490  1.00  0.00           H  \\nATOM     23  N   ALA A   3      48.363  17.021  17.631  1.00  0.00           N  \\nATOM     24  CA  ALA A   3      48.318  17.482  19.011  1.00  0.00           C  \\nATOM     25  C   ALA A   3      46.922  17.287  19.594  1.00  0.00           C  \\nATOM     26  O   ALA A   3      46.777  16.910  20.757  1.00  0.00           O  \\nATOM     27  CB  ALA A   3      48.735  18.943  19.099  1.00  0.00           C  \\nATOM     28  H   ALA A   3      48.495  17.650  17.059  1.00  0.00           H  \\nATOM     29  HA  ALA A   3      48.942  16.962  19.540  1.00  0.00           H  \\nATOM     30 1HB  ALA A   3      48.692  19.272  20.137  1.00  0.00           H  \\nATOM     31 2HB  ALA A   3      49.753  19.054  18.726  1.00  0.00           H  \\nATOM     32 3HB  ALA A   3      48.061  19.550  18.498  1.00  0.00           H  \\nATOM     33  N   ALA A   4      45.898  17.538  18.781  1.00  0.00           N  \\nATOM     34  CA  ALA A   4      44.513  17.367  19.215  1.00  0.00           C  \\nATOM     35  C   ALA A   4      44.255  15.920  19.614  1.00  0.00           C  \\nATOM     36  O   ALA A   4      43.737  15.652  20.696  1.00  0.00           O  \\nATOM     37  CB  ALA A   4      43.550  17.796  18.118  1.00  0.00           C  \\nATOM     38  H   ALA A   4      45.979  17.810  17.969  1.00  0.00           H  \\nATOM     39  HA  ALA A   4      44.352  17.927  19.990  1.00  0.00           H  \\nATOM     40 1HB  ALA A   4      42.524  17.654  18.458  1.00  0.00           H  \\nATOM     41 2HB  ALA A   4      43.712  18.848  17.883  1.00  0.00           H  \\nATOM     42 3HB  ALA A   4      43.723  17.195  17.226  1.00  0.00           H  \\nATOM     43  N   ALA A   5      44.611  14.995  18.729  1.00  0.00           N  \\nATOM     44  CA  ALA A   5      44.488  13.572  19.015  1.00  0.00           C  \\nATOM     45  C   ALA A   5      45.295  13.204  20.257  1.00  0.00           C  \\nATOM     46  O   ALA A   5      44.842  12.424  21.094  1.00  0.00           O  \\nATOM     47  CB  ALA A   5      44.941  12.747  17.819  1.00  0.00           C  \\nATOM     48  H   ALA A   5      44.928  15.169  17.949  1.00  0.00           H  \\nATOM     49  HA  ALA A   5      43.557  13.360  19.185  1.00  0.00           H  \\nATOM     50 1HB  ALA A   5      44.846  11.686  18.052  1.00  0.00           H  \\nATOM     51 2HB  ALA A   5      44.320  12.986  16.956  1.00  0.00           H  \\nATOM     52 3HB  ALA A   5      45.981  12.976  17.592  1.00  0.00           H  \\nATOM     53  N   ALA A   6      46.486  13.781  20.374  1.00  0.00           N  \\nATOM     54  CA  ALA A   6      47.366  13.501  21.502  1.00  0.00           C  \\nATOM     55  C   ALA A   6      46.771  14.033  22.801  1.00  0.00           C  \\nATOM     56  O   ALA A   6      46.837  13.370  23.836  1.00  0.00           O  \\nATOM     57  CB  ALA A   6      48.744  14.102  21.263  1.00  0.00           C  \\nATOM     58  H   ALA A   6      46.810  14.343  19.809  1.00  0.00           H  \\nATOM     59  HA  ALA A   6      47.470  12.541  21.591  1.00  0.00           H  \\nATOM     60 1HB  ALA A   6      49.388  13.885  22.116  1.00  0.00           H  \\nATOM     61 2HB  ALA A   6      49.178  13.670  20.361  1.00  0.00           H  \\nATOM     62 3HB  ALA A   6      48.654  15.180  21.142  1.00  0.00           H  \\nATOM     63  N   ALA A   7      46.190  15.228  22.743  1.00  0.00           N  \\nATOM     64  CA  ALA A   7      45.617  15.851  23.931  1.00  0.00           C  \\nATOM     65  C   ALA A   7      44.366  15.115  24.392  1.00  0.00           C  \\nATOM     66  O   ALA A   7      44.138  14.970  25.590  1.00  0.00           O  \\nATOM     67  CB  ALA A   7      45.299  17.314  23.661  1.00  0.00           C  \\nATOM     68  H   ALA A   7      46.114  15.698  22.027  1.00  0.00           H  \\nATOM     69  HA  ALA A   7      46.267  15.814  24.650  1.00  0.00           H  \\nATOM     70 1HB  ALA A   7      44.872  17.766  24.556  1.00  0.00           H  \\nATOM     71 2HB  ALA A   7      46.214  17.841  23.390  1.00  0.00           H  \\nATOM     72 3HB  ALA A   7      44.584  17.386  22.843  1.00  0.00           H  \\nATOM     73  N   ALA A   8      43.558  14.656  23.440  1.00  0.00           N  \\nATOM     74  CA  ALA A   8      42.375  13.868  23.763  1.00  0.00           C  \\nATOM     75  C   ALA A   8      42.787  12.599  24.490  1.00  0.00           C  \\nATOM     76  O   ALA A   8      42.157  12.197  25.469  1.00  0.00           O  \\nATOM     77  CB  ALA A   8      41.587  13.534  22.505  1.00  0.00           C  \\nATOM     78  H   ALA A   8      43.674  14.789  22.598  1.00  0.00           H  \\nATOM     79  HA  ALA A   8      41.798  14.382  24.350  1.00  0.00           H  \\nATOM     80 1HB  ALA A   8      40.716  12.934  22.770  1.00  0.00           H  \\nATOM     81 2HB  ALA A   8      41.259  14.457  22.025  1.00  0.00           H  \\nATOM     82 3HB  ALA A   8      42.218  12.973  21.818  1.00  0.00           H  \\nATOM     83  N   ALA A   9      43.856  11.977  24.006  1.00  0.00           N  \\nATOM     84  CA  ALA A   9      44.400  10.786  24.639  1.00  0.00           C  \\nATOM     85  C   ALA A   9      44.876  11.138  26.042  1.00  0.00           C  \\nATOM     86  O   ALA A   9      44.715  10.356  26.980  1.00  0.00           O  \\nATOM     87  CB  ALA A   9      45.535  10.202  23.811  1.00  0.00           C  \\nATOM     88  H   ALA A   9      44.287  12.228  23.305  1.00  0.00           H  \\nATOM     89  HA  ALA A   9      43.705  10.112  24.711  1.00  0.00           H  \\nATOM     90 1HB  ALA A   9      45.936   9.321  24.314  1.00  0.00           H  \\nATOM     91 2HB  ALA A   9      45.161   9.919  22.827  1.00  0.00           H  \\nATOM     92 3HB  ALA A   9      46.323  10.945  23.699  1.00  0.00           H  \\nATOM     93  N   ALA A  10      45.456  12.327  26.179  1.00  0.00           N  \\nATOM     94  CA  ALA A  10      45.928  12.800  27.473  1.00  0.00           C  \\nATOM     95  C   ALA A  10      44.723  13.115  28.353  1.00  0.00           C  \\nATOM     96  O   ALA A  10      44.672  12.731  29.523  1.00  0.00           O  \\nATOM     97  CB  ALA A  10      46.821  14.022  27.314  1.00  0.00           C  \\nATOM     98  H   ALA A  10      45.587  12.880  25.534  1.00  0.00           H  \\nATOM     99  HA  ALA A  10      46.445  12.100  27.902  1.00  0.00           H  \\nATOM    100 1HB  ALA A  10      47.145  14.366  28.296  1.00  0.00           H  \\nATOM    101 2HB  ALA A  10      47.693  13.760  26.715  1.00  0.00           H  \\nATOM    102 3HB  ALA A  10      46.265  14.816  26.817  1.00  0.00           H  \\nATOM    103  N   ALA A  11      43.746  13.798  27.767  1.00  0.00           N  \\nATOM    104  CA  ALA A  11      42.513  14.142  28.459  1.00  0.00           C  \\nATOM    105  C   ALA A  11      41.830  12.867  28.946  1.00  0.00           C  \\nATOM    106  O   ALA A  11      41.266  12.827  30.038  1.00  0.00           O  \\nATOM    107  CB  ALA A  11      41.588  14.939  27.551  1.00  0.00           C  \\nATOM    108  H   ALA A  11      43.775  14.077  26.954  1.00  0.00           H  \\nATOM    109  HA  ALA A  11      42.719  14.693  29.231  1.00  0.00           H  \\nATOM    110 1HB  ALA A  11      40.668  15.175  28.085  1.00  0.00           H  \\nATOM    111 2HB  ALA A  11      42.081  15.864  27.251  1.00  0.00           H  \\nATOM    112 3HB  ALA A  11      41.353  14.351  26.665  1.00  0.00           H  \\nATOM    113  N   ALA A  12      41.916  11.820  28.132  1.00  0.00           N  \\nATOM    114  CA  ALA A  12      41.329  10.527  28.460  1.00  0.00           C  \\nATOM    115  C   ALA A  12      42.018   9.891  29.669  1.00  0.00           C  \\nATOM    116  O   ALA A  12      41.355   9.346  30.550  1.00  0.00           O  \\nATOM    117  CB  ALA A  12      41.402   9.597  27.258  1.00  0.00           C  \\nATOM    118  H   ALA A  12      42.315  11.834  27.371  1.00  0.00           H  \\nATOM    119  HA  ALA A  12      40.393  10.650  28.679  1.00  0.00           H  \\nATOM    120 1HB  ALA A  12      40.956   8.636  27.514  1.00  0.00           H  \\nATOM    121 2HB  ALA A  12      40.858  10.038  26.423  1.00  0.00           H  \\nATOM    122 3HB  ALA A  12      42.443   9.449  26.975  1.00  0.00           H  \\nATOM    123  N   ALA A  13      43.346   9.967  29.708  1.00  0.00           N  \\nATOM    124  CA  ALA A  13      44.112   9.386  30.807  1.00  0.00           C  \\nATOM    125  C   ALA A  13      43.873  10.137  32.114  1.00  0.00           C  \\nATOM    126  O   ALA A  13      43.833   9.532  33.186  1.00  0.00           O  \\nATOM    127  CB  ALA A  13      45.596   9.376  30.468  1.00  0.00           C  \\nATOM    128  H   ALA A  13      43.828  10.351  29.108  1.00  0.00           H  \\nATOM    129  HA  ALA A  13      43.831   8.466  30.935  1.00  0.00           H  \\nATOM    130 1HB  ALA A  13      46.155   8.938  31.295  1.00  0.00           H  \\nATOM    131 2HB  ALA A  13      45.759   8.786  29.567  1.00  0.00           H  \\nATOM    132 3HB  ALA A  13      45.937  10.396  30.301  1.00  0.00           H  \\nATOM    133  N   ALA A  14      43.724  11.456  32.023  1.00  0.00           N  \\nATOM    134  CA  ALA A  14      43.447  12.281  33.195  1.00  0.00           C  \\nATOM    135  C   ALA A  14      42.111  11.897  33.818  1.00  0.00           C  \\nATOM    136  O   ALA A  14      41.958  11.886  35.037  1.00  0.00           O  \\nATOM    137  CB  ALA A  14      43.456  13.757  32.824  1.00  0.00           C  \\nATOM    138  H   ALA A  14      43.779  11.899  31.288  1.00  0.00           H  \\nATOM    139  HA  ALA A  14      44.142  12.139  33.857  1.00  0.00           H  \\nATOM    140 1HB  ALA A  14      43.242  14.356  33.709  1.00  0.00           H  \\nATOM    141 2HB  ALA A  14      44.436  14.027  32.431  1.00  0.00           H  \\nATOM    142 3HB  ALA A  14      42.697  13.946  32.067  1.00  0.00           H  \\nATOM    143  N   ALA A  15      41.147  11.585  32.961  1.00  0.00           N  \\nATOM    144  CA  ALA A  15      39.818  11.185  33.401  1.00  0.00           C  \\nATOM    145  C   ALA A  15      39.872   9.876  34.185  1.00  0.00           C  \\nATOM    146  O   ALA A  15      39.279   9.765  35.258  1.00  0.00           O  \\nATOM    147  CB  ALA A  15      38.883  11.052  32.208  1.00  0.00           C  \\nATOM    148  H   ALA A  15      41.240  11.598  32.106  1.00  0.00           H  \\nATOM    149  HA  ALA A  15      39.455  11.871  33.982  1.00  0.00           H  \\nATOM    150 1HB  ALA A  15      37.893  10.754  32.552  1.00  0.00           H  \\nATOM    151 2HB  ALA A  15      38.814  12.010  31.692  1.00  0.00           H  \\nATOM    152 3HB  ALA A  15      39.271  10.299  31.524  1.00  0.00           H  \\nATOM    153  N   ALA A  16      40.590   8.892  33.650  1.00  0.00           N  \\nATOM    154  CA  ALA A  16      40.716   7.594  34.307  1.00  0.00           C  \\nATOM    155  C   ALA A  16      41.311   7.758  35.696  1.00  0.00           C  \\nATOM    156  O   ALA A  16      40.893   7.094  36.646  1.00  0.00           O  \\nATOM    157  CB  ALA A  16      41.568   6.649  33.472  1.00  0.00           C  \\nATOM    158  H   ALA A  16      41.015   8.950  32.904  1.00  0.00           H  \\nATOM    159  HA  ALA A  16      39.835   7.198  34.400  1.00  0.00           H  \\nATOM    160 1HB  ALA A  16      41.657   5.691  33.983  1.00  0.00           H  \\nATOM    161 2HB  ALA A  16      41.099   6.500  32.499  1.00  0.00           H  \\nATOM    162 3HB  ALA A  16      42.559   7.080  33.334  1.00  0.00           H  \\nATOM    163  N   ALA A  17      42.288   8.649  35.808  1.00  0.00           N  \\nATOM    164  CA  ALA A  17      42.912   8.929  37.091  1.00  0.00           C  \\nATOM    165  C   ALA A  17      41.897   9.557  38.044  1.00  0.00           C  \\nATOM    166  O   ALA A  17      41.898   9.264  39.238  1.00  0.00           O  \\nATOM    167  CB  ALA A  17      44.117   9.840  36.911  1.00  0.00           C  \\nATOM    168  H   ALA A  17      42.608   9.106  35.154  1.00  0.00           H  \\nATOM    169  HA  ALA A  17      43.221   8.097  37.483  1.00  0.00           H  \\nATOM    170 1HB  ALA A  17      44.568  10.043  37.883  1.00  0.00           H  \\nATOM    171 2HB  ALA A  17      44.849   9.353  36.266  1.00  0.00           H  \\nATOM    172 3HB  ALA A  17      43.800  10.777  36.456  1.00  0.00           H  \\nATOM    173  N   ALA A  18      41.024  10.410  37.511  1.00  0.00           N  \\nATOM    174  CA  ALA A  18      39.999  11.059  38.327  1.00  0.00           C  \\nATOM    175  C   ALA A  18      39.078  10.027  38.968  1.00  0.00           C  \\nATOM    176  O   ALA A  18      38.849  10.057  40.177  1.00  0.00           O  \\nATOM    177  CB  ALA A  18      39.192  12.040  37.488  1.00  0.00           C  \\nATOM    178  H   ALA A  18      41.004  10.630  36.679  1.00  0.00           H  \\nATOM    179  HA  ALA A  18      40.431  11.559  39.037  1.00  0.00           H  \\nATOM    180 1HB  ALA A  18      38.432  12.511  38.111  1.00  0.00           H  \\nATOM    181 2HB  ALA A  18      39.856  12.804  37.084  1.00  0.00           H  \\nATOM    182 3HB  ALA A  18      38.712  11.508  36.669  1.00  0.00           H  \\nATOM    183  N   ALA A  19      38.551   9.119  38.152  1.00  0.00           N  \\nATOM    184  CA  ALA A  19      37.649   8.080  38.637  1.00  0.00           C  \\nATOM    185  C   ALA A  19      38.327   7.198  39.676  1.00  0.00           C  \\nATOM    186  O   ALA A  19      37.689   6.746  40.626  1.00  0.00           O  \\nATOM    187  CB  ALA A  19      37.143   7.235  37.477  1.00  0.00           C  \\nATOM    188  H   ALA A  19      38.702   9.083  37.306  1.00  0.00           H  \\nATOM    189  HA  ALA A  19      36.880   8.499  39.054  1.00  0.00           H  \\nATOM    190 1HB  ALA A  19      36.471   6.464  37.855  1.00  0.00           H  \\nATOM    191 2HB  ALA A  19      36.607   7.869  36.771  1.00  0.00           H  \\nATOM    192 3HB  ALA A  19      37.987   6.765  36.975  1.00  0.00           H  \\nATOM    193  N   ALA A  20      39.619   6.950  39.487  1.00  0.00           N  \\nATOM    194  CA  ALA A  20      40.389   6.158  40.438  1.00  0.00           C  \\nATOM    195  C   ALA A  20      40.457   6.878  41.780  1.00  0.00           C  \\nATOM    196  O   ALA A  20      40.299   6.264  42.835  1.00  0.00           O  \\nATOM    197  CB  ALA A  20      41.787   5.885  39.903  1.00  0.00           C  \\nATOM    198  H   ALA A  20      40.075   7.231  38.814  1.00  0.00           H  \\nATOM    199  HA  ALA A  20      39.948   5.305  40.573  1.00  0.00           H  \\nATOM    200 1HB  ALA A  20      42.348   5.301  40.633  1.00  0.00           H  \\nATOM    201 2HB  ALA A  20      41.717   5.328  38.969  1.00  0.00           H  \\nATOM    202 3HB  ALA A  20      42.299   6.829  39.725  1.00  0.00           H  \\nATOM    203  N   ALA A  21      40.691   8.186  41.729  1.00  0.00           N  \\nATOM    204  CA  ALA A  21      40.722   9.012  42.932  1.00  0.00           C  \\nATOM    205  C   ALA A  21      39.322   9.124  43.521  1.00  0.00           C  \\nATOM    206  O   ALA A  21      39.152   9.145  44.740  1.00  0.00           O  \\nATOM    207  CB  ALA A  21      41.286  10.391  42.625  1.00  0.00           C  \\nATOM    208  H   ALA A  21      40.837   8.623  41.003  1.00  0.00           H  \\nATOM    209  HA  ALA A  21      41.297   8.594  43.592  1.00  0.00           H  \\nATOM    210 1HB  ALA A  21      41.289  10.994  43.534  1.00  0.00           H  \\nATOM    211 2HB  ALA A  21      42.306  10.293  42.252  1.00  0.00           H  \\nATOM    212 3HB  ALA A  21      40.670  10.877  41.870  1.00  0.00           H  \\nATOM    213  N   ALA A  22      38.323   9.195  42.648  1.00  0.00           N  \\nATOM    214  CA  ALA A  22      36.938   9.321  43.084  1.00  0.00           C  \\nATOM    215  C   ALA A  22      36.511   8.076  43.857  1.00  0.00           C  \\nATOM    216  O   ALA A  22      35.848   8.173  44.890  1.00  0.00           O  \\nATOM    217  CB  ALA A  22      36.021   9.552  41.892  1.00  0.00           C  \\nATOM    218  H   ALA A  22      38.421   9.172  41.794  1.00  0.00           H  \\nATOM    219  HA  ALA A  22      36.859  10.086  43.675  1.00  0.00           H  \\nATOM    220 1HB  ALA A  22      34.990   9.638  42.237  1.00  0.00           H  \\nATOM    221 2HB  ALA A  22      36.311  10.472  41.383  1.00  0.00           H  \\nATOM    222 3HB  ALA A  22      36.104   8.715  41.202  1.00  0.00           H  \\nATOM    223  N   ALA A  23      36.902   6.909  43.352  1.00  0.00           N  \\nATOM    224  CA  ALA A  23      36.628   5.647  44.030  1.00  0.00           C  \\nATOM    225  C   ALA A  23      37.340   5.612  45.373  1.00  0.00           C  \\nATOM    226  O   ALA A  23      36.759   5.221  46.384  1.00  0.00           O  \\nATOM    227  CB  ALA A  23      37.054   4.468  43.168  1.00  0.00           C  \\nATOM    228  H   ALA A  23      37.331   6.822  42.612  1.00  0.00           H  \\nATOM    229  HA  ALA A  23      35.675   5.569  44.189  1.00  0.00           H  \\nATOM    230 1HB  ALA A  23      36.851   3.537  43.698  1.00  0.00           H  \\nATOM    231 2HB  ALA A  23      36.496   4.481  42.231  1.00  0.00           H  \\nATOM    232 3HB  ALA A  23      38.120   4.539  42.956  1.00  0.00           H  \\nATOM    233  N   ALA A  24      38.605   6.022  45.371  1.00  0.00           N  \\nATOM    234  CA  ALA A  24      39.387   6.108  46.596  1.00  0.00           C  \\nATOM    235  C   ALA A  24      38.691   7.053  47.566  1.00  0.00           C  \\nATOM    236  O   ALA A  24      38.549   6.755  48.751  1.00  0.00           O  \\nATOM    237  CB  ALA A  24      40.805   6.577  46.305  1.00  0.00           C  \\nATOM    238  H   ALA A  24      39.037   6.257  44.665  1.00  0.00           H  \\nATOM    239  HA  ALA A  24      39.442   5.231  47.008  1.00  0.00           H  \\nATOM    240 1HB  ALA A  24      41.363   6.650  47.238  1.00  0.00           H  \\nATOM    241 2HB  ALA A  24      41.296   5.863  45.643  1.00  0.00           H  \\nATOM    242 3HB  ALA A  24      40.773   7.554  45.825  1.00  0.00           H  \\nATOM    243  N   ALA A  25      38.241   8.188  47.041  1.00  0.00           N  \\nATOM    244  CA  ALA A  25      37.571   9.202  47.844  1.00  0.00           C  \\nATOM    245  C   ALA A  25      36.260   8.661  48.404  1.00  0.00           C  \\nATOM    246  O   ALA A  25      35.915   8.916  49.558  1.00  0.00           O  \\nATOM    247  CB  ALA A  25      37.322  10.457  47.020  1.00  0.00           C  \\nATOM    248  H   ALA A  25      38.314   8.397  46.210  1.00  0.00           H  \\nATOM    249  HA  ALA A  25      38.141   9.445  48.591  1.00  0.00           H  \\nATOM    250 1HB  ALA A  25      36.817  11.202  47.635  1.00  0.00           H  \\nATOM    251 2HB  ALA A  25      38.274  10.858  46.672  1.00  0.00           H  \\nATOM    252 3HB  ALA A  25      36.698  10.211  46.162  1.00  0.00           H  \\nATOM    253  N   ALA A  26      35.542   7.901  47.583  1.00  0.00           N  \\nATOM    254  CA  ALA A  26      34.261   7.331  47.988  1.00  0.00           C  \\nATOM    255  C   ALA A  26      34.442   6.235  49.032  1.00  0.00           C  \\nATOM    256  O   ALA A  26      33.568   6.018  49.871  1.00  0.00           O  \\nATOM    257  CB  ALA A  26      33.519   6.787  46.776  1.00  0.00           C  \\nATOM    258  H   ALA A  26      35.776   7.699  46.781  1.00  0.00           H  \\nATOM    259  HA  ALA A  26      33.715   8.030  48.380  1.00  0.00           H  \\nATOM    260 1HB  ALA A  26      32.564   6.367  47.093  1.00  0.00           H  \\nATOM    261 2HB  ALA A  26      33.342   7.594  46.065  1.00  0.00           H  \\nATOM    262 3HB  ALA A  26      34.117   6.010  46.302  1.00  0.00           H  \\nATOM    263  N   ALA A  27      35.574   5.540  48.974  1.00  0.00           N  \\nATOM    264  CA  ALA A  27      35.886   4.526  49.973  1.00  0.00           C  \\nATOM    265  C   ALA A  27      36.041   5.168  51.344  1.00  0.00           C  \\nATOM    266  O   ALA A  27      35.353   4.797  52.294  1.00  0.00           O  \\nATOM    267  CB  ALA A  27      37.149   3.767  49.592  1.00  0.00           C  \\nATOM    268  H   ALA A  27      36.176   5.637  48.368  1.00  0.00           H  \\nATOM    269  HA  ALA A  27      35.156   3.888  50.018  1.00  0.00           H  \\nATOM    270 1HB  ALA A  27      37.369   3.021  50.355  1.00  0.00           H  \\nATOM    271 2HB  ALA A  27      37.001   3.272  48.632  1.00  0.00           H  \\nATOM    272 3HB  ALA A  27      37.982   4.464  49.515  1.00  0.00           H  \\nATOM    273  N   ALA A  28      36.940   6.143  51.433  1.00  0.00           N  \\nATOM    274  CA  ALA A  28      37.251   6.792  52.700  1.00  0.00           C  \\nATOM    275  C   ALA A  28      36.052   7.536  53.281  1.00  0.00           C  \\nATOM    276  O   ALA A  28      36.018   7.830  54.475  1.00  0.00           O  \\nATOM    277  CB  ALA A  28      38.422   7.747  52.521  1.00  0.00           C  \\nATOM    278  H   ALA A  28      37.389   6.449  50.766  1.00  0.00           H  \\nATOM    279  HA  ALA A  28      37.529   6.113  53.335  1.00  0.00           H  \\nATOM    280 1HB  ALA A  28      38.656   8.219  53.476  1.00  0.00           H  \\nATOM    281 2HB  ALA A  28      39.293   7.193  52.168  1.00  0.00           H  \\nATOM    282 3HB  ALA A  28      38.160   8.512  51.793  1.00  0.00           H  \\nATOM    283  N   ALA A  29      35.071   7.838  52.438  1.00  0.00           N  \\nATOM    284  CA  ALA A  29      33.891   8.566  52.885  1.00  0.00           C  \\nATOM    285  C   ALA A  29      33.050   7.696  53.813  1.00  0.00           C  \\nATOM    286  O   ALA A  29      32.438   8.198  54.755  1.00  0.00           O  \\nATOM    287  CB  ALA A  29      33.066   9.030  51.693  1.00  0.00           C  \\nATOM    288  H   ALA A  29      35.064   7.632  51.603  1.00  0.00           H  \\nATOM    289  HA  ALA A  29      34.173   9.351  53.381  1.00  0.00           H  \\nATOM    290 1HB  ALA A  29      32.186   9.567  52.047  1.00  0.00           H  \\nATOM    291 2HB  ALA A  29      33.668   9.691  51.070  1.00  0.00           H  \\nATOM    292 3HB  ALA A  29      32.753   8.166  51.109  1.00  0.00           H  \\nATOM    293  N   ALA A  30      33.025   6.393  53.542  1.00  0.00           N  \\nATOM    294  CA  ALA A  30      32.366   5.436  54.426  1.00  0.00           C  \\nATOM    295  C   ALA A  30      33.394   4.808  55.370  1.00  0.00           C  \\nATOM    296  O   ALA A  30      33.299   3.633  55.724  1.00  0.00           O  \\nATOM    297  CB  ALA A  30      31.649   4.363  53.619  1.00  0.00           C  \\nATOM    298  H   ALA A  30      33.386   6.036  52.847  1.00  0.00           H  \\nATOM    299  HA  ALA A  30      31.708   5.902  54.964  1.00  0.00           H  \\nATOM    300 1HB  ALA A  30      31.173   3.655  54.297  1.00  0.00           H  \\nATOM    301 2HB  ALA A  30      30.891   4.828  52.988  1.00  0.00           H  \\nATOM    302 3HB  ALA A  30      32.369   3.838  52.993  1.00  0.00           H  \\nATOM    303  N   ALA A  31      34.385   5.606  55.758  1.00  0.00           N  \\nATOM    304  CA  ALA A  31      35.375   5.203  56.754  1.00  0.00           C  \\nATOM    305  C   ALA A  31      35.523   6.285  57.819  1.00  0.00           C  \\nATOM    306  O   ALA A  31      36.275   6.121  58.780  1.00  0.00           O  \\nATOM    307  CB  ALA A  31      36.716   4.916  56.093  1.00  0.00           C  \\nATOM    308  H   ALA A  31      34.508   6.401  55.453  1.00  0.00           H  \\nATOM    309  HA  ALA A  31      35.073   4.391  57.189  1.00  0.00           H  \\nATOM    310 1HB  ALA A  31      37.441   4.626  56.853  1.00  0.00           H  \\nATOM    311 2HB  ALA A  31      36.600   4.106  55.373  1.00  0.00           H  \\nATOM    312 3HB  ALA A  31      37.066   5.810  55.580  1.00  0.00           H  \\nATOM    313  N   ALA A  32      34.816   7.395  57.627  1.00  0.00           N  \\nATOM    314  CA  ALA A  32      34.634   8.394  58.669  1.00  0.00           C  \\nATOM    315  C   ALA A  32      33.207   8.290  59.192  1.00  0.00           C  \\nATOM    316  O   ALA A  32      32.893   8.793  60.267  1.00  0.00           O  \\nATOM    317  CB  ALA A  32      34.924   9.793  58.144  1.00  0.00           C  \\nATOM    318  H   ALA A  32      34.426   7.593  56.886  1.00  0.00           H  \\nATOM    319  HA  ALA A  32      35.245   8.215  59.401  1.00  0.00           H  \\nATOM    320 1HB  ALA A  32      34.759  10.521  58.939  1.00  0.00           H  \\nATOM    321 2HB  ALA A  32      35.960   9.849  57.811  1.00  0.00           H  \\nATOM    322 3HB  ALA A  32      34.262  10.012  57.308  1.00  0.00           H  \\nATOM    323  N   ALA A  33      32.345   7.639  58.415  1.00  0.00           N  \\nATOM    324  CA  ALA A  33      30.984   7.360  58.849  1.00  0.00           C  \\nATOM    325  C   ALA A  33      31.002   6.279  59.929  1.00  0.00           C  \\nATOM    326  O   ALA A  33      30.091   6.197  60.753  1.00  0.00           O  \\nATOM    327  CB  ALA A  33      30.120   6.934  57.671  1.00  0.00           C  \\nATOM    328  H   ALA A  33      32.527   7.348  57.627  1.00  0.00           H  \\nATOM    329  HA  ALA A  33      30.598   8.164  59.230  1.00  0.00           H  \\nATOM    330 1HB  ALA A  33      29.109   6.721  58.020  1.00  0.00           H  \\nATOM    331 2HB  ALA A  33      30.087   7.738  56.935  1.00  0.00           H  \\nATOM    332 3HB  ALA A  33      30.542   6.041  57.214  1.00  0.00           H  \\nATOM    333  N   ALA A  34      32.050   5.457  59.915  1.00  0.00           N  \\nATOM    334  CA  ALA A  34      32.263   4.437  60.937  1.00  0.00           C  \\nATOM    335  C   ALA A  34      32.992   5.028  62.141  1.00  0.00           C  \\nATOM    336  O   ALA A  34      32.755   4.629  63.282  1.00  0.00           O  \\nATOM    337  CB  ALA A  34      33.043   3.263  60.365  1.00  0.00           C  \\nATOM    338  H   ALA A  34      32.663   5.473  59.312  1.00  0.00           H  \\nATOM    339  HA  ALA A  34      31.404   4.099  61.237  1.00  0.00           H  \\nATOM    340 1HB  ALA A  34      33.197   2.514  61.142  1.00  0.00           H  \\nATOM    341 2HB  ALA A  34      32.483   2.821  59.541  1.00  0.00           H  \\nATOM    342 3HB  ALA A  34      34.009   3.611  60.001  1.00  0.00           H  \\nATOM    343  N   ALA A  35      33.881   5.979  61.868  1.00  0.00           N  \\nATOM    344  CA  ALA A  35      34.660   6.654  62.902  1.00  0.00           C  \\nATOM    345  C   ALA A  35      34.615   8.169  62.697  1.00  0.00           C  \\nATOM    346  O   ALA A  35      35.586   8.761  62.224  1.00  0.00           O  \\nATOM    347  CB  ALA A  35      36.098   6.155  62.899  1.00  0.00           C  \\nATOM    348  H   ALA A  35      34.055   6.257  61.073  1.00  0.00           H  \\nATOM    349  HA  ALA A  35      34.278   6.455  63.771  1.00  0.00           H  \\nATOM    350 1HB  ALA A  35      36.664   6.674  63.673  1.00  0.00           H  \\nATOM    351 2HB  ALA A  35      36.111   5.083  63.097  1.00  0.00           H  \\nATOM    352 3HB  ALA A  35      36.548   6.351  61.927  1.00  0.00           H  \\nATOM    353  N   ALA A  36      33.480   8.802  63.034  1.00  0.00           N  \\nATOM    354  CA  ALA A  36      33.349  10.257  62.895  1.00  0.00           C  \\nATOM    355  C   ALA A  36      34.242  11.024  63.870  1.00  0.00           C  \\nATOM    356  O   ALA A  36      33.740  11.731  64.744  1.00  0.00           O  \\nATOM    357  CB  ALA A  36      31.897  10.670  63.088  1.00  0.00           C  \\nATOM    358  H   ALA A  36      32.699   8.270  63.390  1.00  0.00           H  \\nATOM    359  HA  ALA A  36      33.552  10.530  61.987  1.00  0.00           H  \\nATOM    360 1HB  ALA A  36      31.808  11.751  62.973  1.00  0.00           H  \\nATOM    361 2HB  ALA A  36      31.276  10.174  62.342  1.00  0.00           H  \\nATOM    362 3HB  ALA A  36      31.567  10.383  64.085  1.00  0.00           H  \\nATOM    363  N   ALA A  37      35.554  10.871  63.721  1.00  0.00           N  \\nATOM    364  CA  ALA A  37      36.512  11.566  64.574  1.00  0.00           C  \\nATOM    365  C   ALA A  37      37.551  12.308  63.743  1.00  0.00           C  \\nATOM    366  O   ALA A  37      37.563  12.224  62.515  1.00  0.00           O  \\nATOM    367  CB  ALA A  37      37.192  10.584  65.517  1.00  0.00           C  \\nATOM    368  H   ALA A  37      35.918  10.367  63.127  1.00  0.00           H  \\nATOM    369  HA  ALA A  37      36.038  12.219  65.113  1.00  0.00           H  \\nATOM    370 1HB  ALA A  37      37.904  11.118  66.146  1.00  0.00           H  \\nATOM    371 2HB  ALA A  37      36.442  10.104  66.145  1.00  0.00           H  \\nATOM    372 3HB  ALA A  37      37.717   9.827  64.937  1.00  0.00           H  \\nATOM    373  N   ALA A  38      38.428  13.027  64.432  1.00  0.00           N  \\nATOM    374  CA  ALA A  38      39.416  13.881  63.788  1.00  0.00           C  \\nATOM    375  C   ALA A  38      40.503  13.079  63.080  1.00  0.00           C  \\nATOM    376  O   ALA A  38      40.668  11.885  63.324  1.00  0.00           O  \\nATOM    377  CB  ALA A  38      40.040  14.818  64.812  1.00  0.00           C  \\nATOM    378  H   ALA A  38      38.471  13.036  65.291  1.00  0.00           H  \\nATOM    379  HA  ALA A  38      38.971  14.432  63.125  1.00  0.00           H  \\nATOM    380 1HB  ALA A  38      40.770  15.461  64.320  1.00  0.00           H  \\nATOM    381 2HB  ALA A  38      39.262  15.432  65.265  1.00  0.00           H  \\nATOM    382 3HB  ALA A  38      40.536  14.232  65.585  1.00  0.00           H  \\nATOM    383  N   ALA A  39      41.166  13.758  62.155  1.00  0.00           N  \\nATOM    384  CA  ALA A  39      42.319  13.244  61.409  1.00  0.00           C  \\nATOM    385  C   ALA A  39      42.016  12.100  60.463  1.00  0.00           C  \\nATOM    386  O   ALA A  39      42.807  11.802  59.618  1.00  0.00           O  \\nATOM    387  CB  ALA A  39      43.402  12.808  62.385  1.00  0.00           C  \\nATOM    388  H   ALA A  39      40.955  14.558  61.929  1.00  0.00           H  \\nATOM    389  HA  ALA A  39      42.657  13.974  60.852  1.00  0.00           H  \\nATOM    390 1HB  ALA A  39      44.269  12.450  61.830  1.00  0.00           H  \\nATOM    391 2HB  ALA A  39      43.693  13.655  63.007  1.00  0.00           H  \\nATOM    392 3HB  ALA A  39      43.021  12.008  63.018  1.00  0.00           H  \\nATOM    393  N   ALA A  40      40.882  11.456  60.610  1.00  0.00           N  \\nATOM    394  CA  ALA A  40      40.291  10.669  59.535  1.00  0.00           C  \\nATOM    395  C   ALA A  40      39.620  11.649  58.582  1.00  0.00           C  \\nATOM    396  O   ALA A  40      39.622  11.462  57.366  1.00  0.00           O  \\nATOM    397  CB  ALA A  40      39.298   9.647  60.070  1.00  0.00           C  \\nATOM    398  H   ALA A  40      40.461  11.408  61.359  1.00  0.00           H  \\nATOM    399  HA  ALA A  40      40.989  10.196  59.054  1.00  0.00           H  \\nATOM    400 1HB  ALA A  40      38.851   9.104  59.238  1.00  0.00           H  \\nATOM    401 2HB  ALA A  40      39.815   8.947  60.726  1.00  0.00           H  \\nATOM    402 3HB  ALA A  40      38.517  10.159  60.630  1.00  0.00           H  \\nATOM    403  N   ALA A  41      39.041  12.696  59.162  1.00  0.00           N  \\nATOM    404  CA  ALA A  41      38.532  13.831  58.404  1.00  0.00           C  \\nATOM    405  C   ALA A  41      39.668  14.464  57.609  1.00  0.00           C  \\nATOM    406  O   ALA A  41      39.486  14.863  56.459  1.00  0.00           O  \\nATOM    407  CB  ALA A  41      37.885  14.852  59.328  1.00  0.00           C  \\nATOM    408  H   ALA A  41      38.929  12.772  60.011  1.00  0.00           H  \\nATOM    409  HA  ALA A  41      37.858  13.522  57.779  1.00  0.00           H  \\nATOM    410 1HB  ALA A  41      37.524  15.697  58.741  1.00  0.00           H  \\nATOM    411 2HB  ALA A  41      37.048  14.390  59.852  1.00  0.00           H  \\nATOM    412 3HB  ALA A  41      38.618  15.202  60.053  1.00  0.00           H  \\nATOM    413  N   ALA A  42      40.845  14.537  58.224  1.00  0.00           N  \\nATOM    414  CA  ALA A  42      42.022  15.105  57.573  1.00  0.00           C  \\nATOM    415  C   ALA A  42      42.475  14.215  56.418  1.00  0.00           C  \\nATOM    416  O   ALA A  42      42.906  14.714  55.380  1.00  0.00           O  \\nATOM    417  CB  ALA A  42      43.150  15.292  58.577  1.00  0.00           C  \\nATOM    418  H   ALA A  42      40.989  14.262  59.026  1.00  0.00           H  \\nATOM    419  HA  ALA A  42      41.794  15.975  57.211  1.00  0.00           H  \\nATOM    420 1HB  ALA A  42      44.021  15.711  58.072  1.00  0.00           H  \\nATOM    421 2HB  ALA A  42      42.826  15.971  59.366  1.00  0.00           H  \\nATOM    422 3HB  ALA A  42      43.412  14.330  59.013  1.00  0.00           H  \\nATOM    423  N   ALA A  43      42.373  12.901  56.603  1.00  0.00           N  \\nATOM    424  CA  ALA A  43      42.744  11.950  55.557  1.00  0.00           C  \\nATOM    425  C   ALA A  43      41.899  12.158  54.307  1.00  0.00           C  \\nATOM    426  O   ALA A  43      42.387  12.017  53.190  1.00  0.00           O  \\nATOM    427  CB  ALA A  43      42.601  10.521  56.061  1.00  0.00           C  \\nATOM    428  H   ALA A  43      42.091  12.533  57.327  1.00  0.00           H  \\nATOM    429  HA  ALA A  43      43.674  12.088  55.319  1.00  0.00           H  \\nATOM    430 1HB  ALA A  43      42.878   9.826  55.268  1.00  0.00           H  \\nATOM    431 2HB  ALA A  43      43.255  10.371  56.919  1.00  0.00           H  \\nATOM    432 3HB  ALA A  43      41.568  10.342  56.356  1.00  0.00           H  \\nATOM    433  N   ALA A  44      40.629  12.490  54.507  1.00  0.00           N  \\nATOM    434  CA  ALA A  44      39.711  12.720  53.398  1.00  0.00           C  \\nATOM    435  C   ALA A  44      40.003  14.050  52.719  1.00  0.00           C  \\nATOM    436  O   ALA A  44      40.039  14.130  51.491  1.00  0.00           O  \\nATOM    437  CB  ALA A  44      38.269  12.677  53.882  1.00  0.00           C  \\nATOM    438  H   ALA A  44      40.271  12.589  55.283  1.00  0.00           H  \\nATOM    439  HA  ALA A  44      39.825  12.015  52.741  1.00  0.00           H  \\nATOM    440 1HB  ALA A  44      37.598  12.854  53.042  1.00  0.00           H  \\nATOM    441 2HB  ALA A  44      38.059  11.699  54.315  1.00  0.00           H  \\nATOM    442 3HB  ALA A  44      38.116  13.447  54.637  1.00  0.00           H  \\nATOM    443  N   ALA A  45      40.211  15.089  53.522  1.00  0.00           N  \\nATOM    444  CA  ALA A  45      40.538  16.406  52.990  1.00  0.00           C  \\nATOM    445  C   ALA A  45      41.757  16.326  52.082  1.00  0.00           C  \\nATOM    446  O   ALA A  45      41.810  16.994  51.054  1.00  0.00           O  \\nATOM    447  CB  ALA A  45      40.780  17.395  54.121  1.00  0.00           C  \\nATOM    448  H   ALA A  45      40.169  15.057  54.381  1.00  0.00           H  \\nATOM    449  HA  ALA A  45      39.790  16.729  52.463  1.00  0.00           H  \\nATOM    450 1HB  ALA A  45      41.029  18.371  53.704  1.00  0.00           H  \\nATOM    451 2HB  ALA A  45      39.879  17.480  54.729  1.00  0.00           H  \\nATOM    452 3HB  ALA A  45      41.604  17.045  54.740  1.00  0.00           H  \\nATOM    453  N   ALA A  46      42.727  15.497  52.458  1.00  0.00           N  \\nATOM    454  CA  ALA A  46      43.905  15.287  51.625  1.00  0.00           C  \\nATOM    455  C   ALA A  46      43.507  14.737  50.260  1.00  0.00           C  \\nATOM    456  O   ALA A  46      44.031  15.165  49.233  1.00  0.00           O  \\nATOM    457  CB  ALA A  46      44.884  14.347  52.313  1.00  0.00           C  \\nATOM    458  H   ALA A  46      42.726  15.045  53.189  1.00  0.00           H  \\nATOM    459  HA  ALA A  46      44.353  16.137  51.489  1.00  0.00           H  \\nATOM    460 1HB  ALA A  46      45.755  14.199  51.675  1.00  0.00           H  \\nATOM    461 2HB  ALA A  46      45.199  14.781  53.262  1.00  0.00           H  \\nATOM    462 3HB  ALA A  46      44.401  13.389  52.496  1.00  0.00           H  \\nATOM    463  N   ALA A  47      42.575  13.789  50.257  1.00  0.00           N  \\nATOM    464  CA  ALA A  47      42.166  13.124  49.026  1.00  0.00           C  \\nATOM    465  C   ALA A  47      41.228  14.016  48.218  1.00  0.00           C  \\nATOM    466  O   ALA A  47      41.170  13.919  46.992  1.00  0.00           O  \\nATOM    467  CB  ALA A  47      41.499  11.792  49.337  1.00  0.00           C  \\nATOM    468  H   ALA A  47      42.163  13.512  50.959  1.00  0.00           H  \\nATOM    469  HA  ALA A  47      42.952  12.948  48.486  1.00  0.00           H  \\nATOM    470 1HB  ALA A  47      41.197  11.310  48.406  1.00  0.00           H  \\nATOM    471 2HB  ALA A  47      42.201  11.148  49.867  1.00  0.00           H  \\nATOM    472 3HB  ALA A  47      40.622  11.961  49.959  1.00  0.00           H  \\nATOM    473  N   ALA A  48      40.498  14.887  48.908  1.00  0.00           N  \\nATOM    474  CA  ALA A  48      39.616  15.837  48.240  1.00  0.00           C  \\nATOM    475  C   ALA A  48      40.446  16.839  47.437  1.00  0.00           C  \\nATOM    476  O   ALA A  48      40.168  17.084  46.262  1.00  0.00           O  \\nATOM    477  CB  ALA A  48      38.734  16.555  49.251  1.00  0.00           C  \\nATOM    478  H   ALA A  48      40.496  14.947  49.766  1.00  0.00           H  \\nATOM    479  HA  ALA A  48      39.044  15.357  47.621  1.00  0.00           H  \\nATOM    480 1HB  ALA A  48      38.091  17.266  48.733  1.00  0.00           H  \\nATOM    481 2HB  ALA A  48      38.119  15.826  49.779  1.00  0.00           H  \\nATOM    482 3HB  ALA A  48      39.360  17.087  49.966  1.00  0.00           H  \\nATOM    483  N   ALA A  49      41.475  17.402  48.063  1.00  0.00           N  \\nATOM    484  CA  ALA A  49      42.346  18.354  47.381  1.00  0.00           C  \\nATOM    485  C   ALA A  49      43.059  17.662  46.227  1.00  0.00           C  \\nATOM    486  O   ALA A  49      43.298  18.263  45.182  1.00  0.00           O  \\nATOM    487  CB  ALA A  49      43.352  18.957  48.350  1.00  0.00           C  \\nATOM    488  H   ALA A  49      41.690  17.251  48.882  1.00  0.00           H  \\nATOM    489  HA  ALA A  49      41.807  19.073  47.016  1.00  0.00           H  \\nATOM    490 1HB  ALA A  49      43.999  19.654  47.816  1.00  0.00           H  \\nATOM    491 2HB  ALA A  49      42.822  19.488  49.141  1.00  0.00           H  \\nATOM    492 3HB  ALA A  49      43.956  18.164  48.788  1.00  0.00           H  \\nATOM    493  N   ALA A  50      43.396  16.393  46.427  1.00  0.00           N  \\nATOM    494  CA  ALA A  50      44.003  15.586  45.379  1.00  0.00           C  \\nATOM    495  C   ALA A  50      43.038  15.459  44.211  1.00  0.00           C  \\nATOM    496  O   ALA A  50      43.424  15.625  43.055  1.00  0.00           O  \\nATOM    497  CB  ALA A  50      44.392  14.213  45.909  1.00  0.00           C  \\nATOM    498  H   ALA A  50      43.281  15.974  47.169  1.00  0.00           H  \\nATOM    499  HA  ALA A  50      44.809  16.024  45.063  1.00  0.00           H  \\nATOM    500 1HB  ALA A  50      44.830  13.624  45.104  1.00  0.00           H  \\nATOM    501 2HB  ALA A  50      45.118  14.326  46.714  1.00  0.00           H  \\nATOM    502 3HB  ALA A  50      43.506  13.706  46.289  1.00  0.00           H  \\nATOM    503  N   ALA A  51      41.778  15.167  44.517  1.00  0.00           N  \\nATOM    504  CA  ALA A  51      40.758  15.040  43.485  1.00  0.00           C  \\nATOM    505  C   ALA A  51      40.578  16.367  42.750  1.00  0.00           C  \\nATOM    506  O   ALA A  51      40.489  16.403  41.523  1.00  0.00           O  \\nATOM    507  CB  ALA A  51      39.440  14.579  44.091  1.00  0.00           C  \\nATOM    508  H   ALA A  51      41.488  15.039  45.317  1.00  0.00           H  \\nATOM    509  HA  ALA A  51      41.041  14.374  42.839  1.00  0.00           H  \\nATOM    510 1HB  ALA A  51      38.689  14.495  43.306  1.00  0.00           H  \\nATOM    511 2HB  ALA A  51      39.578  13.609  44.567  1.00  0.00           H  \\nATOM    512 3HB  ALA A  51      39.109  15.303  44.833  1.00  0.00           H  \\nATOM    513  N   ALA A  52      40.536  17.458  43.510  1.00  0.00           N  \\nATOM    514  CA  ALA A  52      40.361  18.784  42.933  1.00  0.00           C  \\nATOM    515  C   ALA A  52      41.565  19.158  42.079  1.00  0.00           C  \\nATOM    516  O   ALA A  52      41.408  19.694  40.985  1.00  0.00           O  \\nATOM    517  CB  ALA A  52      40.141  19.818  44.027  1.00  0.00           C  \\nATOM    518  H   ALA A  52      40.606  17.455  44.367  1.00  0.00           H  \\nATOM    519  HA  ALA A  52      39.579  18.777  42.359  1.00  0.00           H  \\nATOM    520 1HB  ALA A  52      40.018  20.803  43.578  1.00  0.00           H  \\nATOM    521 2HB  ALA A  52      39.246  19.563  44.595  1.00  0.00           H  \\nATOM    522 3HB  ALA A  52      41.002  19.830  44.695  1.00  0.00           H  \\nATOM    523  N   ALA A  53      42.764  18.868  42.573  1.00  0.00           N  \\nATOM    524  CA  ALA A  53      43.977  19.138  41.813  1.00  0.00           C  \\nATOM    525  C   ALA A  53      43.941  18.386  40.489  1.00  0.00           C  \\nATOM    526  O   ALA A  53      44.317  18.925  39.449  1.00  0.00           O  \\nATOM    527  CB  ALA A  53      45.211  18.753  42.615  1.00  0.00           C  \\nATOM    528  H   ALA A  53      42.901  18.516  43.346  1.00  0.00           H  \\nATOM    529  HA  ALA A  53      44.027  20.087  41.620  1.00  0.00           H  \\nATOM    530 1HB  ALA A  53      46.106  18.954  42.026  1.00  0.00           H  \\nATOM    531 2HB  ALA A  53      45.244  19.337  43.535  1.00  0.00           H  \\nATOM    532 3HB  ALA A  53      45.169  17.693  42.861  1.00  0.00           H  \\nATOM    533  N   ALA A  54      43.474  17.143  40.533  1.00  0.00           N  \\nATOM    534  CA  ALA A  54      43.344  16.329  39.331  1.00  0.00           C  \\nATOM    535  C   ALA A  54      42.245  16.881  38.434  1.00  0.00           C  \\nATOM    536  O   ALA A  54      42.347  16.834  37.208  1.00  0.00           O  \\nATOM    537  CB  ALA A  54      43.059  14.879  39.694  1.00  0.00           C  \\nATOM    538  H   ALA A  54      43.223  16.745  41.253  1.00  0.00           H  \\nATOM    539  HA  ALA A  54      44.179  16.356  38.837  1.00  0.00           H  \\nATOM    540 1HB  ALA A  54      42.958  14.289  38.783  1.00  0.00           H  \\nATOM    541 2HB  ALA A  54      43.881  14.485  40.292  1.00  0.00           H  \\nATOM    542 3HB  ALA A  54      42.135  14.822  40.267  1.00  0.00           H  \\nATOM    543  N   ALA A  55      41.194  17.412  39.052  1.00  0.00           N  \\nATOM    544  CA  ALA A  55      40.093  18.014  38.309  1.00  0.00           C  \\nATOM    545  C   ALA A  55      40.558  19.258  37.554  1.00  0.00           C  \\nATOM    546  O   ALA A  55      40.268  19.415  36.368  1.00  0.00           O  \\nATOM    547  CB  ALA A  55      38.946  18.359  39.248  1.00  0.00           C  \\nATOM    548  H   ALA A  55      41.095  17.435  39.906  1.00  0.00           H  \\nATOM    549  HA  ALA A  55      39.762  17.374  37.660  1.00  0.00           H  \\nATOM    550 1HB  ALA A  55      38.132  18.808  38.679  1.00  0.00           H  \\nATOM    551 2HB  ALA A  55      38.591  17.453  39.737  1.00  0.00           H  \\nATOM    552 3HB  ALA A  55      39.294  19.065  40.001  1.00  0.00           H  \\nATOM    553  N   ALA A  56      41.281  20.137  38.241  1.00  0.00           N  \\nATOM    554  CA  ALA A  56      41.777  21.367  37.626  1.00  0.00           C  \\nATOM    555  C   ALA A  56      42.772  21.056  36.510  1.00  0.00           C  \\nATOM    556  O   ALA A  56      42.778  21.714  35.470  1.00  0.00           O  \\nATOM    557  CB  ALA A  56      42.419  22.264  38.674  1.00  0.00           C  \\nATOM    558  H   ALA A  56      41.500  20.045  39.068  1.00  0.00           H  \\nATOM    559  HA  ALA A  56      41.030  21.848  37.236  1.00  0.00           H  \\nATOM    560 1HB  ALA A  56      42.785  23.175  38.199  1.00  0.00           H  \\nATOM    561 2HB  ALA A  56      41.680  22.523  39.433  1.00  0.00           H  \\nATOM    562 3HB  ALA A  56      43.251  21.740  39.142  1.00  0.00           H  \\nATOM    563  N   ALA A  57      43.614  20.052  36.732  1.00  0.00           N  \\nATOM    564  CA  ALA A  57      44.557  19.618  35.709  1.00  0.00           C  \\nATOM    565  C   ALA A  57      43.803  19.118  34.482  1.00  0.00           C  \\nATOM    566  O   ALA A  57      44.186  19.404  33.348  1.00  0.00           O  \\nATOM    567  CB  ALA A  57      45.478  18.535  36.252  1.00  0.00           C  \\nATOM    568  H   ALA A  57      43.660  19.607  37.466  1.00  0.00           H  \\nATOM    569  HA  ALA A  57      45.106  20.372  35.441  1.00  0.00           H  \\nATOM    570 1HB  ALA A  57      46.168  18.218  35.469  1.00  0.00           H  \\nATOM    571 2HB  ALA A  57      46.043  18.927  37.097  1.00  0.00           H  \\nATOM    572 3HB  ALA A  57      44.884  17.682  36.577  1.00  0.00           H  \\nATOM    573  N   ALA A  58      42.722  18.378  34.712  1.00  0.00           N  \\nATOM    574  CA  ALA A  58      41.902  17.876  33.614  1.00  0.00           C  \\nATOM    575  C   ALA A  58      41.286  19.027  32.826  1.00  0.00           C  \\nATOM    576  O   ALA A  58      41.209  18.977  31.600  1.00  0.00           O  \\nATOM    577  CB  ALA A  58      40.814  16.952  34.141  1.00  0.00           C  \\nATOM    578  H   ALA A  58      42.442  18.152  35.494  1.00  0.00           H  \\nATOM    579  HA  ALA A  58      42.463  17.365  33.010  1.00  0.00           H  \\nATOM    580 1HB  ALA A  58      40.211  16.589  33.310  1.00  0.00           H  \\nATOM    581 2HB  ALA A  58      41.273  16.105  34.654  1.00  0.00           H  \\nATOM    582 3HB  ALA A  58      40.180  17.497  34.839  1.00  0.00           H  \\nATOM    583  N   ALA A  59      40.859  20.067  33.535  1.00  0.00           N  \\nATOM    584  CA  ALA A  59      40.285  21.249  32.896  1.00  0.00           C  \\nATOM    585  C   ALA A  59      41.344  22.073  32.166  1.00  0.00           C  \\nATOM    586  O   ALA A  59      41.051  22.717  31.157  1.00  0.00           O  \\nATOM    587  CB  ALA A  59      39.571  22.109  33.929  1.00  0.00           C  \\nATOM    588  H   ALA A  59      40.890  20.114  34.393  1.00  0.00           H  \\nATOM    589  HA  ALA A  59      39.627  20.963  32.243  1.00  0.00           H  \\nATOM    590 1HB  ALA A  59      39.142  22.983  33.440  1.00  0.00           H  \\nATOM    591 2HB  ALA A  59      38.776  21.529  34.398  1.00  0.00           H  \\nATOM    592 3HB  ALA A  59      40.282  22.430  34.688  1.00  0.00           H  \\nATOM    593  N   ALA A  60      42.572  22.059  32.676  1.00  0.00           N  \\nATOM    594  CA  ALA A  60      43.666  22.768  32.019  1.00  0.00           C  \\nATOM    595  C   ALA A  60      43.936  22.123  30.666  1.00  0.00           C  \\nATOM    596  O   ALA A  60      43.986  22.801  29.640  1.00  0.00           O  \\nATOM    597  CB  ALA A  60      44.919  22.759  32.883  1.00  0.00           C  \\nATOM    598  H   ALA A  60      42.798  21.649  33.397  1.00  0.00           H  \\nATOM    599  HA  ALA A  60      43.408  23.692  31.872  1.00  0.00           H  \\nATOM    600 1HB  ALA A  60      45.724  23.277  32.362  1.00  0.00           H  \\nATOM    601 2HB  ALA A  60      44.714  23.264  33.827  1.00  0.00           H  \\nATOM    602 3HB  ALA A  60      45.217  21.730  33.079  1.00  0.00           H  \\nATOM    603  N   ALA A  61      44.087  20.803  30.675  1.00  0.00           N  \\nATOM    604  CA  ALA A  61      44.304  20.039  29.451  1.00  0.00           C  \\nATOM    605  C   ALA A  61      43.135  20.214  28.490  1.00  0.00           C  \\nATOM    606  O   ALA A  61      43.319  20.248  27.274  1.00  0.00           O  \\nATOM    607  CB  ALA A  61      44.512  18.566  29.772  1.00  0.00           C  \\nATOM    608  H   ALA A  61      44.068  20.320  31.387  1.00  0.00           H  \\nATOM    609  HA  ALA A  61      45.107  20.362  29.012  1.00  0.00           H  \\nATOM    610 1HB  ALA A  61      44.668  18.011  28.847  1.00  0.00           H  \\nATOM    611 2HB  ALA A  61      45.385  18.453  30.415  1.00  0.00           H  \\nATOM    612 3HB  ALA A  61      43.633  18.178  30.283  1.00  0.00           H  \\nATOM    613  N   ALA A  62      41.932  20.322  29.047  1.00  0.00           N  \\nATOM    614  CA  ALA A  62      40.722  20.473  28.246  1.00  0.00           C  \\nATOM    615  C   ALA A  62      40.705  21.801  27.493  1.00  0.00           C  \\nATOM    616  O   ALA A  62      40.435  21.835  26.294  1.00  0.00           O  \\nATOM    617  CB  ALA A  62      39.489  20.353  29.130  1.00  0.00           C  \\nATOM    618  H   ALA A  62      41.789  20.312  29.894  1.00  0.00           H  \\nATOM    619  HA  ALA A  62      40.688  19.758  27.591  1.00  0.00           H  \\nATOM    620 1HB  ALA A  62      38.592  20.462  28.520  1.00  0.00           H  \\nATOM    621 2HB  ALA A  62      39.482  19.376  29.614  1.00  0.00           H  \\nATOM    622 3HB  ALA A  62      39.508  21.134  29.888  1.00  0.00           H  \\nATOM    623  N   ALA A  63      40.990  22.890  28.203  1.00  0.00           N  \\nATOM    624  CA  ALA A  63      41.009  24.224  27.602  1.00  0.00           C  \\nATOM    625  C   ALA A  63      42.148  24.354  26.597  1.00  0.00           C  \\nATOM    626  O   ALA A  63      41.986  24.945  25.528  1.00  0.00           O  \\nATOM    627  CB  ALA A  63      41.130  25.292  28.679  1.00  0.00           C  \\nATOM    628  H   ALA A  63      41.177  22.884  29.042  1.00  0.00           H  \\nATOM    629  HA  ALA A  63      40.174  24.372  27.131  1.00  0.00           H  \\nATOM    630 1HB  ALA A  63      41.146  26.278  28.214  1.00  0.00           H  \\nATOM    631 2HB  ALA A  63      40.278  25.224  29.356  1.00  0.00           H  \\nATOM    632 3HB  ALA A  63      42.051  25.140  29.239  1.00  0.00           H  \\nATOM    633  N   ALA A  64      43.304  23.807  26.957  1.00  0.00           N  \\nATOM    634  CA  ALA A  64      44.451  23.767  26.059  1.00  0.00           C  \\nATOM    635  C   ALA A  64      44.063  23.125  24.732  1.00  0.00           C  \\nATOM    636  O   ALA A  64      44.505  23.557  23.669  1.00  0.00           O  \\nATOM    637  CB  ALA A  64      45.607  23.013  26.700  1.00  0.00           C  \\nATOM    638  H   ALA A  64      43.450  23.447  27.724  1.00  0.00           H  \\nATOM    639  HA  ALA A  64      44.749  24.673  25.881  1.00  0.00           H  \\nATOM    640 1HB  ALA A  64      46.452  22.988  26.012  1.00  0.00           H  \\nATOM    641 2HB  ALA A  64      45.903  23.516  27.621  1.00  0.00           H  \\nATOM    642 3HB  ALA A  64      45.295  21.995  26.928  1.00  0.00           H  \\nATOM    643  N   ALA A  65      43.222  22.099  24.806  1.00  0.00           N  \\nATOM    644  CA  ALA A  65      42.766  21.390  23.618  1.00  0.00           C  \\nATOM    645  C   ALA A  65      41.719  22.186  22.839  1.00  0.00           C  \\nATOM    646  O   ALA A  65      41.603  22.046  21.623  1.00  0.00           O  \\nATOM    647  CB  ALA A  65      42.209  20.028  24.006  1.00  0.00           C  \\nATOM    648  H   ALA A  65      42.898  21.791  25.541  1.00  0.00           H  \\nATOM    649  HA  ALA A  65      43.524  21.241  23.031  1.00  0.00           H  \\nATOM    650 1HB  ALA A  65      41.880  19.501  23.111  1.00  0.00           H  \\nATOM    651 2HB  ALA A  65      42.984  19.447  24.506  1.00  0.00           H  \\nATOM    652 3HB  ALA A  65      41.364  20.160  24.680  1.00  0.00           H  \\nATOM    653  N   ALA A  66      40.978  23.037  23.518  1.00  0.00           N  \\nATOM    654  CA  ALA A  66      39.980  23.885  22.899  1.00  0.00           C  \\nATOM    655  C   ALA A  66      40.596  25.055  22.179  1.00  0.00           C  \\nATOM    656  O   ALA A  66      40.085  25.540  21.207  1.00  0.00           O  \\nATOM    657  CB  ALA A  66      38.990  24.380  23.943  1.00  0.00           C  \\nATOM    658  H   ALA A  66      41.049  23.144  24.363  1.00  0.00           H  \\nATOM    659  HA  ALA A  66      39.493  23.362  22.239  1.00  0.00           H  \\nATOM    660 1HB  ALA A  66      38.242  25.012  23.465  1.00  0.00           H  \\nATOM    661 2HB  ALA A  66      38.498  23.527  24.411  1.00  0.00           H  \\nATOM    662 3HB  ALA A  66      39.519  24.955  24.702  1.00  0.00           H  \\nATOM    663  N   ALA A  67      41.732  25.479  22.675  1.00  0.00           N  \\nATOM    664  CA  ALA A  67      42.504  26.506  21.987  1.00  0.00           C  \\nATOM    665  C   ALA A  67      43.009  25.969  20.651  1.00  0.00           C  \\nATOM    666  O   ALA A  67      43.238  26.729  19.710  1.00  0.00           O  \\nATOM    667  CB  ALA A  67      43.665  26.976  22.852  1.00  0.00           C  \\nATOM    668  H   ALA A  67      42.092  25.195  23.403  1.00  0.00           H  \\nATOM    669  HA  ALA A  67      41.934  27.271  21.812  1.00  0.00           H  \\nATOM    670 1HB  ALA A  67      44.234  27.735  22.315  1.00  0.00           H  \\nATOM    671 2HB  ALA A  67      43.280  27.399  23.779  1.00  0.00           H  \\nATOM    672 3HB  ALA A  67      44.313  26.131  23.080  1.00  0.00           H  \\nATOM    673  N   ALA A  68      43.175  24.651  20.576  1.00  0.00           N  \\nATOM    674  CA  ALA A  68      43.667  24.000  19.368  1.00  0.00           C  \\nATOM    675  C   ALA A  68      42.547  23.785  18.354  1.00  0.00           C  \\nATOM    676  O   ALA A  68      42.754  23.937  17.150  1.00  0.00           O  \\nATOM    677  CB  ALA A  68      44.327  22.674  19.716  1.00  0.00           C  \\nATOM    678  H   ALA A  68      43.007  24.107  21.220  1.00  0.00           H  \\nATOM    679  HA  ALA A  68      44.339  24.566  18.956  1.00  0.00           H  \\nATOM    680 1HB  ALA A  68      44.693  22.199  18.805  1.00  0.00           H  \\nATOM    681 2HB  ALA A  68      45.162  22.849  20.394  1.00  0.00           H  \\nATOM    682 3HB  ALA A  68      43.600  22.021  20.197  1.00  0.00           H  \\nATOM    683  N   ALA A  69      41.363  23.430  18.842  1.00  0.00           N  \\nATOM    684  CA  ALA A  69      40.211  23.214  17.970  1.00  0.00           C  \\nATOM    685  C   ALA A  69      39.822  24.505  17.254  1.00  0.00           C  \\nATOM    686  O   ALA A  69      39.231  24.473  16.175  1.00  0.00           O  \\nATOM    687  CB  ALA A  69      39.033  22.674  18.768  1.00  0.00           C  \\nATOM    688  H   ALA A  69      41.199  23.307  19.677  1.00  0.00           H  \\nATOM    689  HA  ALA A  69      40.443  22.555  17.297  1.00  0.00           H  \\nATOM    690 1HB  ALA A  69      38.183  22.521  18.103  1.00  0.00           H  \\nATOM    691 2HB  ALA A  69      39.310  21.724  19.227  1.00  0.00           H  \\nATOM    692 3HB  ALA A  69      38.763  23.387  19.545  1.00  0.00           H  \\nATOM    693  N   ALA A  70      40.162  25.638  17.860  1.00  0.00           N  \\nATOM    694  CA  ALA A  70      39.855  26.942  17.284  1.00  0.00           C  \\nATOM    695  C   ALA A  70      40.852  27.303  16.187  1.00  0.00           C  \\nATOM    696  O   ALA A  70      42.065  27.204  16.377  1.00  0.00           O  \\nATOM    697  OXT ALA A  70      40.458  27.689  15.122  1.00  0.00           O  \\nATOM    698  CB  ALA A  70      39.846  28.011  18.367  1.00  0.00           C  \\nATOM    699  H   ALA A  70      40.576  25.679  18.612  1.00  0.00           H  \\nATOM    700  HA  ALA A  70      38.970  26.913  16.889  1.00  0.00           H  \\nATOM    701 1HB  ALA A  70      39.616  28.979  17.920  1.00  0.00           H  \\nATOM    702 2HB  ALA A  70      39.090  27.767  19.113  1.00  0.00           H  \\nATOM    703 3HB  ALA A  70      40.825  28.055  18.842  1.00  0.00           H  \\nTER                                                                             \\nATOM    705  N   TYR B  71      35.710  36.388  19.013  1.00  0.00           N  \\nATOM    706  CA  TYR B  71      34.402  36.488  18.434  1.00  0.00           C  \\nATOM    707  C   TYR B  71      33.573  35.299  18.771  1.00  0.00           C  \\nATOM    708  O   TYR B  71      33.315  34.433  18.000  1.00  0.00           O  \\nATOM    709  CB  TYR B  71      34.548  36.577  16.921  1.00  0.00           C  \\nATOM    710  CG  TYR B  71      35.147  35.286  16.337  1.00  0.00           C  \\nATOM    711  CD1 TYR B  71      34.314  34.280  15.860  1.00  0.00           C  \\nATOM    712  CD2 TYR B  71      36.529  35.103  16.315  1.00  0.00           C  \\nATOM    713  CE1 TYR B  71      34.854  33.115  15.345  1.00  0.00           C  \\nATOM    714  CE2 TYR B  71      37.068  33.929  15.806  1.00  0.00           C  \\nATOM    715  CZ  TYR B  71      36.236  32.942  15.317  1.00  0.00           C  \\nATOM    716  OH  TYR B  71      36.770  31.778  14.802  1.00  0.00           O  \\nATOM    717 1H   TYR B  71      36.245  37.198  18.772  1.00  0.00           H  \\nATOM    718 2H   TYR B  71      35.631  36.326  20.008  1.00  0.00           H  \\nATOM    719 3H   TYR B  71      36.168  35.570  18.664  1.00  0.00           H  \\nATOM    720  HA  TYR B  71      33.961  37.264  18.826  1.00  0.00           H  \\nATOM    721 1HB  TYR B  71      33.573  36.746  16.462  1.00  0.00           H  \\nATOM    722 2HB  TYR B  71      35.196  37.413  16.657  1.00  0.00           H  \\nATOM    723  HD1 TYR B  71      33.232  34.412  15.884  1.00  0.00           H  \\nATOM    724  HD2 TYR B  71      37.187  35.883  16.697  1.00  0.00           H  \\nATOM    725  HE1 TYR B  71      34.195  32.337  14.965  1.00  0.00           H  \\nATOM    726  HE2 TYR B  71      38.149  33.789  15.789  1.00  0.00           H  \\nATOM    727  HH  TYR B  71      37.701  31.720  15.032  1.00  0.00           H  \\nATOM    728  N   LYS B  72      33.117  35.298  19.982  1.00  0.00           N  \\nATOM    729  CA  LYS B  72      32.478  34.151  20.550  1.00  0.00           C  \\nATOM    730  C   LYS B  72      33.634  33.262  20.770  1.00  0.00           C  \\nATOM    731  O   LYS B  72      34.157  33.247  21.839  1.00  0.00           O  \\nATOM    732  CB  LYS B  72      31.342  33.622  19.660  1.00  0.00           C  \\nATOM    733  CG  LYS B  72      30.383  32.756  20.373  1.00  0.00           C  \\nATOM    734  CD  LYS B  72      29.266  32.286  19.437  1.00  0.00           C  \\nATOM    735  CE  LYS B  72      28.208  31.460  20.162  1.00  0.00           C  \\nATOM    736  NZ  LYS B  72      27.102  31.051  19.238  1.00  0.00           N  \\nATOM    737  H   LYS B  72      33.183  35.965  20.514  1.00  0.00           H  \\nATOM    738  HA  LYS B  72      32.084  34.379  21.413  1.00  0.00           H  \\nATOM    739 1HB  LYS B  72      30.799  34.449  19.203  1.00  0.00           H  \\nATOM    740 2HB  LYS B  72      31.722  32.987  18.865  1.00  0.00           H  \\nATOM    741 1HG  LYS B  72      30.903  31.988  20.830  1.00  0.00           H  \\nATOM    742 2HG  LYS B  72      29.922  33.345  21.163  1.00  0.00           H  \\nATOM    743 1HD  LYS B  72      28.788  33.155  18.984  1.00  0.00           H  \\nATOM    744 2HD  LYS B  72      29.701  31.679  18.641  1.00  0.00           H  \\nATOM    745 1HE  LYS B  72      28.668  30.567  20.579  1.00  0.00           H  \\nATOM    746 2HE  LYS B  72      27.789  32.050  20.976  1.00  0.00           H  \\nATOM    747 1HZ  LYS B  72      26.419  30.508  19.746  1.00  0.00           H  \\nATOM    748 2HZ  LYS B  72      26.659  31.876  18.856  1.00  0.00           H  \\nATOM    749 3HZ  LYS B  72      27.479  30.493  18.483  1.00  0.00           H  \\nATOM    750  N   ILE B  73      34.079  32.584  19.742  1.00  0.00           N  \\nATOM    751  CA  ILE B  73      35.171  31.658  19.903  1.00  0.00           C  \\nATOM    752  C   ILE B  73      36.358  32.336  20.561  1.00  0.00           C  \\nATOM    753  O   ILE B  73      36.745  31.991  21.645  1.00  0.00           O  \\nATOM    754  CB  ILE B  73      35.596  30.967  18.592  1.00  0.00           C  \\nATOM    755  CG1 ILE B  73      34.478  30.035  18.128  1.00  0.00           C  \\nATOM    756  CG2 ILE B  73      36.880  30.171  18.804  1.00  0.00           C  \\nATOM    757  CD1 ILE B  73      34.646  29.471  16.756  1.00  0.00           C  \\nATOM    758  H   ILE B  73      33.759  32.648  18.951  1.00  0.00           H  \\nATOM    759  HA  ILE B  73      34.887  30.943  20.499  1.00  0.00           H  \\nATOM    760  HB  ILE B  73      35.749  31.712  17.827  1.00  0.00           H  \\nATOM    761 1HG1 ILE B  73      34.394  29.207  18.836  1.00  0.00           H  \\nATOM    762 2HG1 ILE B  73      33.554  30.597  18.125  1.00  0.00           H  \\nATOM    763 1HG2 ILE B  73      37.160  29.688  17.868  1.00  0.00           H  \\nATOM    764 2HG2 ILE B  73      37.683  30.830  19.122  1.00  0.00           H  \\nATOM    765 3HG2 ILE B  73      36.712  29.410  19.572  1.00  0.00           H  \\nATOM    766 1HD1 ILE B  73      33.788  28.831  16.521  1.00  0.00           H  \\nATOM    767 2HD1 ILE B  73      34.702  30.275  16.035  1.00  0.00           H  \\nATOM    768 3HD1 ILE B  73      35.560  28.884  16.711  1.00  0.00           H  \\nATOM    769  N   LYS B  74      36.874  33.357  19.918  1.00  0.00           N  \\nATOM    770  CA  LYS B  74      37.966  34.129  20.458  1.00  0.00           C  \\nATOM    771  C   LYS B  74      37.682  34.728  21.827  1.00  0.00           C  \\nATOM    772  O   LYS B  74      38.529  34.736  22.665  1.00  0.00           O  \\nATOM    773  CB  LYS B  74      38.585  35.153  19.492  1.00  0.00           C  \\nATOM    774  CG  LYS B  74      39.826  35.837  20.036  1.00  0.00           C  \\nATOM    775  CD  LYS B  74      40.435  36.780  18.997  1.00  0.00           C  \\nATOM    776  CE  LYS B  74      41.659  37.502  19.546  1.00  0.00           C  \\nATOM    777  NZ  LYS B  74      42.255  38.425  18.541  1.00  0.00           N  \\nATOM    778  H   LYS B  74      36.597  33.608  19.149  1.00  0.00           H  \\nATOM    779  HA  LYS B  74      38.727  33.530  20.568  1.00  0.00           H  \\nATOM    780 1HB  LYS B  74      38.824  34.682  18.537  1.00  0.00           H  \\nATOM    781 2HB  LYS B  74      37.999  35.949  19.324  1.00  0.00           H  \\nATOM    782 1HG  LYS B  74      39.562  36.410  20.928  1.00  0.00           H  \\nATOM    783 2HG  LYS B  74      40.565  35.086  20.315  1.00  0.00           H  \\nATOM    784 1HD  LYS B  74      40.728  36.209  18.115  1.00  0.00           H  \\nATOM    785 2HD  LYS B  74      39.695  37.522  18.698  1.00  0.00           H  \\nATOM    786 1HE  LYS B  74      41.369  38.076  20.425  1.00  0.00           H  \\nATOM    787 2HE  LYS B  74      42.407  36.765  19.837  1.00  0.00           H  \\nATOM    788 1HZ  LYS B  74      41.570  39.119  18.274  1.00  0.00           H  \\nATOM    789 2HZ  LYS B  74      43.061  38.885  18.940  1.00  0.00           H  \\nATOM    790 3HZ  LYS B  74      42.536  37.900  17.725  1.00  0.00           H  \\nATOM    791  N   GLU B  75      36.494  35.236  22.053  1.00  0.00           N  \\nATOM    792  CA  GLU B  75      36.195  35.897  23.320  1.00  0.00           C  \\nATOM    793  C   GLU B  75      35.834  34.904  24.422  1.00  0.00           C  \\nATOM    794  O   GLU B  75      36.211  35.100  25.576  1.00  0.00           O  \\nATOM    795  CB  GLU B  75      35.029  36.854  23.074  1.00  0.00           C  \\nATOM    796  CG  GLU B  75      35.377  38.032  22.167  1.00  0.00           C  \\nATOM    797  CD  GLU B  75      34.164  38.748  21.626  1.00  0.00           C  \\nATOM    798  OE1 GLU B  75      33.083  38.243  21.804  1.00  0.00           O  \\nATOM    799  OE2 GLU B  75      34.327  39.758  20.992  1.00  0.00           O  \\nATOM    800  H   GLU B  75      35.837  35.220  21.498  1.00  0.00           H  \\nATOM    801  HA  GLU B  75      36.983  36.383  23.609  1.00  0.00           H  \\nATOM    802 1HB  GLU B  75      34.203  36.305  22.621  1.00  0.00           H  \\nATOM    803 2HB  GLU B  75      34.678  37.251  24.025  1.00  0.00           H  \\nATOM    804 1HG  GLU B  75      35.959  38.743  22.751  1.00  0.00           H  \\nATOM    805 2HG  GLU B  75      36.004  37.694  21.360  1.00  0.00           H  \\nATOM    806  N   THR B  76      35.111  33.866  24.054  1.00  0.00           N  \\nATOM    807  CA  THR B  76      34.785  32.804  24.978  1.00  0.00           C  \\nATOM    808  C   THR B  76      36.050  32.092  25.454  1.00  0.00           C  \\nATOM    809  O   THR B  76      36.195  31.740  26.589  1.00  0.00           O  \\nATOM    810  CB  THR B  76      33.755  31.751  24.462  1.00  0.00           C  \\nATOM    811  OG1 THR B  76      34.210  31.162  23.263  1.00  0.00           O  \\nATOM    812  CG2 THR B  76      32.367  32.349  24.258  1.00  0.00           C  \\nATOM    813  H   THR B  76      34.790  33.758  23.271  1.00  0.00           H  \\nATOM    814  HA  THR B  76      34.367  33.209  25.757  1.00  0.00           H  \\nATOM    815  HB  THR B  76      33.668  30.964  25.209  1.00  0.00           H  \\nATOM    816  HG1 THR B  76      34.608  31.847  22.702  1.00  0.00           H  \\nATOM    817 1HG2 THR B  76      31.686  31.564  23.923  1.00  0.00           H  \\nATOM    818 2HG2 THR B  76      32.012  32.759  25.203  1.00  0.00           H  \\nATOM    819 3HG2 THR B  76      32.400  33.133  23.524  1.00  0.00           H  \\nATOM    820  N   LEU B  77      36.976  31.938  24.544  1.00  0.00           N  \\nATOM    821  CA  LEU B  77      38.243  31.293  24.841  1.00  0.00           C  \\nATOM    822  C   LEU B  77      39.042  32.153  25.812  1.00  0.00           C  \\nATOM    823  O   LEU B  77      39.753  31.633  26.671  1.00  0.00           O  \\nATOM    824  CB  LEU B  77      38.996  31.042  23.534  1.00  0.00           C  \\nATOM    825  CG  LEU B  77      40.272  30.223  23.594  1.00  0.00           C  \\nATOM    826  CD1 LEU B  77      39.973  28.822  24.181  1.00  0.00           C  \\nATOM    827  CD2 LEU B  77      40.815  30.107  22.179  1.00  0.00           C  \\nATOM    828  H   LEU B  77      36.896  32.205  23.730  1.00  0.00           H  \\nATOM    829  HA  LEU B  77      38.074  30.436  25.263  1.00  0.00           H  \\nATOM    830 1HB  LEU B  77      38.320  30.519  22.859  1.00  0.00           H  \\nATOM    831 2HB  LEU B  77      39.235  32.009  23.086  1.00  0.00           H  \\nATOM    832  HG  LEU B  77      41.002  30.720  24.234  1.00  0.00           H  \\nATOM    833 1HD1 LEU B  77      39.241  28.311  23.554  1.00  0.00           H  \\nATOM    834 2HD1 LEU B  77      40.889  28.234  24.217  1.00  0.00           H  \\nATOM    835 3HD1 LEU B  77      39.574  28.932  25.186  1.00  0.00           H  \\nATOM    836 1HD2 LEU B  77      41.721  29.539  22.191  1.00  0.00           H  \\nATOM    837 2HD2 LEU B  77      40.077  29.608  21.546  1.00  0.00           H  \\nATOM    838 3HD2 LEU B  77      41.016  31.104  21.786  1.00  0.00           H  \\nATOM    839  N   LYS B  78      38.909  33.471  25.687  1.00  0.00           N  \\nATOM    840  CA  LYS B  78      39.591  34.390  26.594  1.00  0.00           C  \\nATOM    841  C   LYS B  78      38.963  34.340  27.984  1.00  0.00           C  \\nATOM    842  O   LYS B  78      39.668  34.400  28.992  1.00  0.00           O  \\nATOM    843  CB  LYS B  78      39.579  35.813  26.048  1.00  0.00           C  \\nATOM    844  CG  LYS B  78      40.454  36.790  26.843  1.00  0.00           C  \\nATOM    845  CD  LYS B  78      41.953  36.404  26.702  1.00  0.00           C  \\nATOM    846  CE  LYS B  78      42.883  37.437  27.342  1.00  0.00           C  \\nATOM    847  NZ  LYS B  78      44.339  37.051  27.212  1.00  0.00           N  \\nATOM    848  H   LYS B  78      38.431  33.858  25.086  1.00  0.00           H  \\nATOM    849  HA  LYS B  78      40.520  34.122  26.674  1.00  0.00           H  \\nATOM    850 1HB  LYS B  78      39.912  35.809  25.008  1.00  0.00           H  \\nATOM    851 2HB  LYS B  78      38.557  36.193  26.058  1.00  0.00           H  \\nATOM    852 1HG  LYS B  78      40.301  37.803  26.476  1.00  0.00           H  \\nATOM    853 2HG  LYS B  78      40.174  36.753  27.900  1.00  0.00           H  \\nATOM    854 1HD  LYS B  78      42.126  35.438  27.191  1.00  0.00           H  \\nATOM    855 2HD  LYS B  78      42.207  36.304  25.648  1.00  0.00           H  \\nATOM    856 1HE  LYS B  78      42.729  38.395  26.850  1.00  0.00           H  \\nATOM    857 2HE  LYS B  78      42.638  37.536  28.402  1.00  0.00           H  \\nATOM    858 1HZ  LYS B  78      44.914  37.756  27.637  1.00  0.00           H  \\nATOM    859 2HZ  LYS B  78      44.517  36.145  27.694  1.00  0.00           H  \\nATOM    860 3HZ  LYS B  78      44.584  36.959  26.241  1.00  0.00           H  \\nATOM    861  N   ARG B  79      37.638  34.239  28.035  1.00  0.00           N  \\nATOM    862  CA  ARG B  79      36.939  34.079  29.307  1.00  0.00           C  \\nATOM    863  C   ARG B  79      37.380  32.789  29.983  1.00  0.00           C  \\nATOM    864  O   ARG B  79      37.585  32.749  31.196  1.00  0.00           O  \\nATOM    865  CB  ARG B  79      35.440  34.081  29.118  1.00  0.00           C  \\nATOM    866  CG  ARG B  79      34.852  35.425  28.740  1.00  0.00           C  \\nATOM    867  CD  ARG B  79      33.411  35.310  28.542  1.00  0.00           C  \\nATOM    868  NE  ARG B  79      32.745  34.991  29.783  1.00  0.00           N  \\nATOM    869  CZ  ARG B  79      31.610  34.294  29.866  1.00  0.00           C  \\nATOM    870  NH1 ARG B  79      31.044  33.861  28.770  1.00  0.00           N  \\nATOM    871  NH2 ARG B  79      31.075  34.044  31.037  1.00  0.00           N  \\nATOM    872  H   ARG B  79      37.121  34.260  27.349  1.00  0.00           H  \\nATOM    873  HA  ARG B  79      37.161  34.821  29.890  1.00  0.00           H  \\nATOM    874 1HB  ARG B  79      35.173  33.372  28.339  1.00  0.00           H  \\nATOM    875 2HB  ARG B  79      34.957  33.753  30.035  1.00  0.00           H  \\nATOM    876 1HG  ARG B  79      35.039  36.145  29.535  1.00  0.00           H  \\nATOM    877 2HG  ARG B  79      35.305  35.776  27.816  1.00  0.00           H  \\nATOM    878 1HD  ARG B  79      33.012  36.252  28.168  1.00  0.00           H  \\nATOM    879 2HD  ARG B  79      33.204  34.514  27.825  1.00  0.00           H  \\nATOM    880  HE  ARG B  79      33.167  35.312  30.644  1.00  0.00           H  \\nATOM    881 1HH1 ARG B  79      31.464  34.058  27.879  1.00  0.00           H  \\nATOM    882 2HH1 ARG B  79      30.189  33.291  28.803  1.00  0.00           H  \\nATOM    883 1HH2 ARG B  79      31.523  34.376  31.880  1.00  0.00           H  \\nATOM    884 2HH2 ARG B  79      30.222  33.507  31.090  1.00  0.00           H  \\nATOM    885  N   LEU B  80      37.524  31.735  29.187  1.00  0.00           N  \\nATOM    886  CA  LEU B  80      37.970  30.445  29.697  1.00  0.00           C  \\nATOM    887  C   LEU B  80      39.362  30.561  30.318  1.00  0.00           C  \\nATOM    888  O   LEU B  80      39.635  29.971  31.366  1.00  0.00           O  \\nATOM    889  CB  LEU B  80      37.932  29.412  28.556  1.00  0.00           C  \\nATOM    890  CG  LEU B  80      38.245  27.973  28.918  1.00  0.00           C  \\nATOM    891  CD1 LEU B  80      37.246  27.514  29.973  1.00  0.00           C  \\nATOM    892  CD2 LEU B  80      38.125  27.100  27.642  1.00  0.00           C  \\nATOM    893  H   LEU B  80      37.369  31.742  28.341  1.00  0.00           H  \\nATOM    894  HA  LEU B  80      37.356  30.147  30.386  1.00  0.00           H  \\nATOM    895 1HB  LEU B  80      36.948  29.440  28.089  1.00  0.00           H  \\nATOM    896 2HB  LEU B  80      38.659  29.710  27.812  1.00  0.00           H  \\nATOM    897  HG  LEU B  80      39.252  27.896  29.333  1.00  0.00           H  \\nATOM    898 1HD1 LEU B  80      36.247  27.598  29.576  1.00  0.00           H  \\nATOM    899 2HD1 LEU B  80      37.447  26.480  30.246  1.00  0.00           H  \\nATOM    900 3HD1 LEU B  80      37.333  28.144  30.856  1.00  0.00           H  \\nATOM    901 1HD2 LEU B  80      38.326  26.060  27.886  1.00  0.00           H  \\nATOM    902 2HD2 LEU B  80      37.114  27.183  27.234  1.00  0.00           H  \\nATOM    903 3HD2 LEU B  80      38.839  27.439  26.897  1.00  0.00           H  \\nATOM    904  N   GLU B  81      40.231  31.335  29.672  1.00  0.00           N  \\nATOM    905  CA  GLU B  81      41.593  31.544  30.155  1.00  0.00           C  \\nATOM    906  C   GLU B  81      41.589  32.221  31.518  1.00  0.00           C  \\nATOM    907  O   GLU B  81      42.308  31.807  32.426  1.00  0.00           O  \\nATOM    908  CB  GLU B  81      42.377  32.408  29.149  1.00  0.00           C  \\nATOM    909  CG  GLU B  81      43.839  32.729  29.501  1.00  0.00           C  \\nATOM    910  CD  GLU B  81      44.493  33.612  28.426  1.00  0.00           C  \\nATOM    911  OE1 GLU B  81      44.480  33.242  27.280  1.00  0.00           O  \\nATOM    912  OE2 GLU B  81      44.967  34.685  28.755  1.00  0.00           O  \\nATOM    913  H   GLU B  81      40.053  31.756  28.943  1.00  0.00           H  \\nATOM    914  HA  GLU B  81      42.031  30.684  30.248  1.00  0.00           H  \\nATOM    915 1HB  GLU B  81      42.360  31.926  28.171  1.00  0.00           H  \\nATOM    916 2HB  GLU B  81      41.881  33.361  29.039  1.00  0.00           H  \\nATOM    917 1HG  GLU B  81      43.862  33.255  30.458  1.00  0.00           H  \\nATOM    918 2HG  GLU B  81      44.398  31.801  29.606  1.00  0.00           H  \\nATOM    919  N   ASP B  82      40.772  33.260  31.661  1.00  0.00           N  \\nATOM    920  CA  ASP B  82      40.694  33.995  32.918  1.00  0.00           C  \\nATOM    921  C   ASP B  82      40.091  33.126  34.013  1.00  0.00           C  \\nATOM    922  O   ASP B  82      40.580  33.110  35.143  1.00  0.00           O  \\nATOM    923  CB  ASP B  82      39.902  35.295  32.763  1.00  0.00           C  \\nATOM    924  CG  ASP B  82      40.691  36.378  32.014  1.00  0.00           C  \\nATOM    925  OD1 ASP B  82      41.875  36.189  31.843  1.00  0.00           O  \\nATOM    926  OD2 ASP B  82      40.120  37.380  31.647  1.00  0.00           O  \\nATOM    927  H   ASP B  82      40.251  33.559  31.045  1.00  0.00           H  \\nATOM    928  HA  ASP B  82      41.589  34.249  33.193  1.00  0.00           H  \\nATOM    929 1HB  ASP B  82      38.984  35.086  32.207  1.00  0.00           H  \\nATOM    930 2HB  ASP B  82      39.617  35.672  33.743  1.00  0.00           H  \\nATOM    931  N   SER B  83      39.029  32.403  33.672  1.00  0.00           N  \\nATOM    932  CA  SER B  83      38.361  31.532  34.631  1.00  0.00           C  \\nATOM    933  C   SER B  83      39.310  30.443  35.113  1.00  0.00           C  \\nATOM    934  O   SER B  83      39.338  30.112  36.295  1.00  0.00           O  \\nATOM    935  CB  SER B  83      37.127  30.935  34.025  1.00  0.00           C  \\nATOM    936  OG  SER B  83      36.461  30.165  34.973  1.00  0.00           O  \\nATOM    937  H   SER B  83      38.675  32.399  32.888  1.00  0.00           H  \\nATOM    938  HA  SER B  83      38.087  32.056  35.400  1.00  0.00           H  \\nATOM    939 1HB  SER B  83      36.489  31.727  33.664  1.00  0.00           H  \\nATOM    940 2HB  SER B  83      37.403  30.316  33.169  1.00  0.00           H  \\nATOM    941  HG  SER B  83      37.141  29.648  35.408  1.00  0.00           H  \\nATOM    942  N   LEU B  84      40.096  29.898  34.191  1.00  0.00           N  \\nATOM    943  CA  LEU B  84      41.055  28.856  34.529  1.00  0.00           C  \\nATOM    944  C   LEU B  84      42.145  29.407  35.446  1.00  0.00           C  \\nATOM    945  O   LEU B  84      42.570  28.737  36.390  1.00  0.00           O  \\nATOM    946  CB  LEU B  84      41.656  28.240  33.266  1.00  0.00           C  \\nATOM    947  CG  LEU B  84      42.638  27.082  33.495  1.00  0.00           C  \\nATOM    948  CD1 LEU B  84      41.946  25.929  34.253  1.00  0.00           C  \\nATOM    949  CD2 LEU B  84      43.168  26.620  32.161  1.00  0.00           C  \\nATOM    950  H   LEU B  84      40.093  30.116  33.359  1.00  0.00           H  \\nATOM    951  HA  LEU B  84      40.595  28.144  35.001  1.00  0.00           H  \\nATOM    952 1HB  LEU B  84      40.839  27.876  32.638  1.00  0.00           H  \\nATOM    953 2HB  LEU B  84      42.178  29.027  32.716  1.00  0.00           H  \\nATOM    954  HG  LEU B  84      43.444  27.429  34.093  1.00  0.00           H  \\nATOM    955 1HD1 LEU B  84      42.658  25.118  34.422  1.00  0.00           H  \\nATOM    956 2HD1 LEU B  84      41.585  26.296  35.210  1.00  0.00           H  \\nATOM    957 3HD1 LEU B  84      41.106  25.554  33.666  1.00  0.00           H  \\nATOM    958 1HD2 LEU B  84      43.878  25.831  32.322  1.00  0.00           H  \\nATOM    959 2HD2 LEU B  84      42.343  26.259  31.545  1.00  0.00           H  \\nATOM    960 3HD2 LEU B  84      43.661  27.453  31.660  1.00  0.00           H  \\nATOM    961  N   ARG B  85      42.587  30.632  35.169  1.00  0.00           N  \\nATOM    962  CA  ARG B  85      43.560  31.306  36.025  1.00  0.00           C  \\nATOM    963  C   ARG B  85      42.993  31.486  37.427  1.00  0.00           C  \\nATOM    964  O   ARG B  85      43.672  31.220  38.421  1.00  0.00           O  \\nATOM    965  CB  ARG B  85      43.916  32.665  35.443  1.00  0.00           C  \\nATOM    966  CG  ARG B  85      45.007  33.439  36.151  1.00  0.00           C  \\nATOM    967  CD  ARG B  85      45.153  34.794  35.547  1.00  0.00           C  \\nATOM    968  NE  ARG B  85      45.525  34.727  34.140  1.00  0.00           N  \\nATOM    969  CZ  ARG B  85      45.141  35.626  33.209  1.00  0.00           C  \\nATOM    970  NH1 ARG B  85      44.386  36.649  33.551  1.00  0.00           N  \\nATOM    971  NH2 ARG B  85      45.510  35.479  31.949  1.00  0.00           N  \\nATOM    972  H   ARG B  85      42.338  31.096  34.489  1.00  0.00           H  \\nATOM    973  HA  ARG B  85      44.360  30.762  36.088  1.00  0.00           H  \\nATOM    974 1HB  ARG B  85      44.208  32.545  34.399  1.00  0.00           H  \\nATOM    975 2HB  ARG B  85      43.036  33.298  35.450  1.00  0.00           H  \\nATOM    976 1HG  ARG B  85      44.746  33.553  37.205  1.00  0.00           H  \\nATOM    977 2HG  ARG B  85      45.956  32.909  36.062  1.00  0.00           H  \\nATOM    978 1HD  ARG B  85      44.201  35.322  35.621  1.00  0.00           H  \\nATOM    979 2HD  ARG B  85      45.922  35.352  36.079  1.00  0.00           H  \\nATOM    980  HE  ARG B  85      46.096  33.951  33.837  1.00  0.00           H  \\nATOM    981 1HH1 ARG B  85      44.086  36.763  34.504  1.00  0.00           H  \\nATOM    982 2HH1 ARG B  85      44.084  37.311  32.842  1.00  0.00           H  \\nATOM    983 1HH2 ARG B  85      46.075  34.693  31.667  1.00  0.00           H  \\nATOM    984 2HH2 ARG B  85      45.201  36.144  31.257  1.00  0.00           H  \\nATOM    985  N   GLU B  86      41.753  31.958  37.499  1.00  0.00           N  \\nATOM    986  CA  GLU B  86      41.075  32.134  38.777  1.00  0.00           C  \\nATOM    987  C   GLU B  86      41.061  30.815  39.546  1.00  0.00           C  \\nATOM    988  O   GLU B  86      41.308  30.780  40.750  1.00  0.00           O  \\nATOM    989  CB  GLU B  86      39.661  32.686  38.604  1.00  0.00           C  \\nATOM    990  CG  GLU B  86      38.863  32.822  39.913  1.00  0.00           C  \\nATOM    991  CD  GLU B  86      39.491  33.757  40.936  1.00  0.00           C  \\nATOM    992  OE1 GLU B  86      40.397  34.491  40.610  1.00  0.00           O  \\nATOM    993  OE2 GLU B  86      39.050  33.728  42.063  1.00  0.00           O  \\nATOM    994  H   GLU B  86      41.280  32.185  36.817  1.00  0.00           H  \\nATOM    995  HA  GLU B  86      41.555  32.790  39.307  1.00  0.00           H  \\nATOM    996 1HB  GLU B  86      39.709  33.664  38.126  1.00  0.00           H  \\nATOM    997 2HB  GLU B  86      39.098  32.034  37.945  1.00  0.00           H  \\nATOM    998 1HG  GLU B  86      37.864  33.183  39.674  1.00  0.00           H  \\nATOM    999 2HG  GLU B  86      38.761  31.830  40.356  1.00  0.00           H  \\nATOM   1000  N   LEU B  87      40.793  29.730  38.829  1.00  0.00           N  \\nATOM   1001  CA  LEU B  87      40.686  28.409  39.436  1.00  0.00           C  \\nATOM   1002  C   LEU B  87      42.037  27.927  39.957  1.00  0.00           C  \\nATOM   1003  O   LEU B  87      42.142  27.441  41.085  1.00  0.00           O  \\nATOM   1004  CB  LEU B  87      40.105  27.452  38.381  1.00  0.00           C  \\nATOM   1005  CG  LEU B  87      39.924  26.011  38.746  1.00  0.00           C  \\nATOM   1006  CD1 LEU B  87      39.017  25.894  39.942  1.00  0.00           C  \\nATOM   1007  CD2 LEU B  87      39.307  25.288  37.521  1.00  0.00           C  \\nATOM   1008  H   LEU B  87      40.667  29.732  37.978  1.00  0.00           H  \\nATOM   1009  HA  LEU B  87      40.074  28.455  40.188  1.00  0.00           H  \\nATOM   1010 1HB  LEU B  87      39.133  27.840  38.068  1.00  0.00           H  \\nATOM   1011 2HB  LEU B  87      40.755  27.472  37.518  1.00  0.00           H  \\nATOM   1012  HG  LEU B  87      40.886  25.575  38.999  1.00  0.00           H  \\nATOM   1013 1HD1 LEU B  87      38.876  24.847  40.202  1.00  0.00           H  \\nATOM   1014 2HD1 LEU B  87      39.452  26.421  40.792  1.00  0.00           H  \\nATOM   1015 3HD1 LEU B  87      38.075  26.329  39.693  1.00  0.00           H  \\nATOM   1016 1HD2 LEU B  87      39.971  25.382  36.669  1.00  0.00           H  \\nATOM   1017 2HD2 LEU B  87      39.149  24.237  37.734  1.00  0.00           H  \\nATOM   1018 3HD2 LEU B  87      38.349  25.746  37.279  1.00  0.00           H  \\nATOM   1019  N   ARG B  88      43.069  28.072  39.132  1.00  0.00           N  \\nATOM   1020  CA  ARG B  88      44.414  27.652  39.511  1.00  0.00           C  \\nATOM   1021  C   ARG B  88      44.926  28.441  40.712  1.00  0.00           C  \\nATOM   1022  O   ARG B  88      45.667  27.911  41.536  1.00  0.00           O  \\nATOM   1023  CB  ARG B  88      45.353  27.762  38.327  1.00  0.00           C  \\nATOM   1024  CG  ARG B  88      45.111  26.686  37.255  1.00  0.00           C  \\nATOM   1025  CD  ARG B  88      46.012  26.829  36.092  1.00  0.00           C  \\nATOM   1026  NE  ARG B  88      45.679  27.988  35.297  1.00  0.00           N  \\nATOM   1027  CZ  ARG B  88      46.387  28.422  34.239  1.00  0.00           C  \\nATOM   1028  NH1 ARG B  88      47.483  27.794  33.874  1.00  0.00           N  \\nATOM   1029  NH2 ARG B  88      45.983  29.480  33.566  1.00  0.00           N  \\nATOM   1030  H   ARG B  88      43.017  28.412  38.344  1.00  0.00           H  \\nATOM   1031  HA  ARG B  88      44.393  26.714  39.756  1.00  0.00           H  \\nATOM   1032 1HB  ARG B  88      45.220  28.738  37.855  1.00  0.00           H  \\nATOM   1033 2HB  ARG B  88      46.385  27.690  38.663  1.00  0.00           H  \\nATOM   1034 1HG  ARG B  88      45.280  25.704  37.691  1.00  0.00           H  \\nATOM   1035 2HG  ARG B  88      44.080  26.748  36.901  1.00  0.00           H  \\nATOM   1036 1HD  ARG B  88      47.039  26.933  36.436  1.00  0.00           H  \\nATOM   1037 2HD  ARG B  88      45.924  25.945  35.458  1.00  0.00           H  \\nATOM   1038  HE  ARG B  88      44.832  28.497  35.548  1.00  0.00           H  \\nATOM   1039 1HH1 ARG B  88      47.797  26.988  34.394  1.00  0.00           H  \\nATOM   1040 2HH1 ARG B  88      48.013  28.125  33.080  1.00  0.00           H  \\nATOM   1041 1HH2 ARG B  88      45.142  29.961  33.851  1.00  0.00           H  \\nATOM   1042 2HH2 ARG B  88      46.509  29.807  32.770  1.00  0.00           H  \\nATOM   1043  N   ARG B  89      44.521  29.704  40.817  1.00  0.00           N  \\nATOM   1044  CA  ARG B  89      44.936  30.545  41.935  1.00  0.00           C  \\nATOM   1045  C   ARG B  89      44.346  30.033  43.242  1.00  0.00           C  \\nATOM   1046  O   ARG B  89      45.046  29.917  44.248  1.00  0.00           O  \\nATOM   1047  CB  ARG B  89      44.442  31.975  41.722  1.00  0.00           C  \\nATOM   1048  CG  ARG B  89      44.833  32.977  42.795  1.00  0.00           C  \\nATOM   1049  CD  ARG B  89      44.139  34.311  42.614  1.00  0.00           C  \\nATOM   1050  NE  ARG B  89      42.678  34.245  42.877  1.00  0.00           N  \\nATOM   1051  CZ  ARG B  89      42.135  34.231  44.128  1.00  0.00           C  \\nATOM   1052  NH1 ARG B  89      42.922  34.262  45.189  1.00  0.00           N  \\nATOM   1053  NH2 ARG B  89      40.822  34.194  44.295  1.00  0.00           N  \\nATOM   1054  H   ARG B  89      44.006  30.098  40.253  1.00  0.00           H  \\nATOM   1055  HA  ARG B  89      45.903  30.517  42.011  1.00  0.00           H  \\nATOM   1056 1HB  ARG B  89      44.808  32.344  40.764  1.00  0.00           H  \\nATOM   1057 2HB  ARG B  89      43.354  31.975  41.666  1.00  0.00           H  \\nATOM   1058 1HG  ARG B  89      44.554  32.580  43.771  1.00  0.00           H  \\nATOM   1059 2HG  ARG B  89      45.910  33.141  42.763  1.00  0.00           H  \\nATOM   1060 1HD  ARG B  89      44.574  35.042  43.293  1.00  0.00           H  \\nATOM   1061 2HD  ARG B  89      44.278  34.645  41.586  1.00  0.00           H  \\nATOM   1062  HE  ARG B  89      42.020  34.244  42.069  1.00  0.00           H  \\nATOM   1063 1HH1 ARG B  89      43.925  34.289  45.076  1.00  0.00           H  \\nATOM   1064 2HH1 ARG B  89      42.518  34.251  46.117  1.00  0.00           H  \\nATOM   1065 1HH2 ARG B  89      40.198  34.162  43.470  1.00  0.00           H  \\nATOM   1066 2HH2 ARG B  89      40.430  34.184  45.220  1.00  0.00           H  \\nATOM   1067  N   ILE B  90      43.050  29.737  43.218  1.00  0.00           N  \\nATOM   1068  CA  ILE B  90      42.351  29.213  44.386  1.00  0.00           C  \\nATOM   1069  C   ILE B  90      42.933  27.861  44.779  1.00  0.00           C  \\nATOM   1070  O   ILE B  90      42.972  27.507  45.958  1.00  0.00           O  \\nATOM   1071  CB  ILE B  90      40.848  29.075  44.134  1.00  0.00           C  \\nATOM   1072  CG1 ILE B  90      40.233  30.451  43.982  1.00  0.00           C  \\nATOM   1073  CG2 ILE B  90      40.198  28.338  45.313  1.00  0.00           C  \\nATOM   1074  CD1 ILE B  90      38.849  30.409  43.454  1.00  0.00           C  \\nATOM   1075  H   ILE B  90      42.548  29.831  42.526  1.00  0.00           H  \\nATOM   1076  HA  ILE B  90      42.471  29.823  45.130  1.00  0.00           H  \\nATOM   1077  HB  ILE B  90      40.677  28.523  43.205  1.00  0.00           H  \\nATOM   1078 1HG1 ILE B  90      40.223  30.945  44.951  1.00  0.00           H  \\nATOM   1079 2HG1 ILE B  90      40.843  31.045  43.294  1.00  0.00           H  \\nATOM   1080 1HG2 ILE B  90      39.136  28.250  45.139  1.00  0.00           H  \\nATOM   1081 2HG2 ILE B  90      40.640  27.349  45.409  1.00  0.00           H  \\nATOM   1082 3HG2 ILE B  90      40.367  28.899  46.233  1.00  0.00           H  \\nATOM   1083 1HD1 ILE B  90      38.462  31.427  43.358  1.00  0.00           H  \\nATOM   1084 2HD1 ILE B  90      38.883  29.940  42.477  1.00  0.00           H  \\nATOM   1085 3HD1 ILE B  90      38.208  29.834  44.122  1.00  0.00           H  \\nATOM   1086  N   LEU B  91      43.392  27.109  43.786  1.00  0.00           N  \\nATOM   1087  CA  LEU B  91      43.991  25.809  44.047  1.00  0.00           C  \\nATOM   1088  C   LEU B  91      45.312  25.962  44.800  1.00  0.00           C  \\nATOM   1089  O   LEU B  91      45.623  25.158  45.673  1.00  0.00           O  \\nATOM   1090  CB  LEU B  91      44.228  25.034  42.751  1.00  0.00           C  \\nATOM   1091  CG  LEU B  91      44.785  23.616  42.927  1.00  0.00           C  \\nATOM   1092  CD1 LEU B  91      43.826  22.791  43.780  1.00  0.00           C  \\nATOM   1093  CD2 LEU B  91      44.965  22.996  41.568  1.00  0.00           C  \\nATOM   1094  H   LEU B  91      43.368  27.328  42.955  1.00  0.00           H  \\nATOM   1095  HA  LEU B  91      43.388  25.289  44.601  1.00  0.00           H  \\nATOM   1096 1HB  LEU B  91      43.290  24.975  42.197  1.00  0.00           H  \\nATOM   1097 2HB  LEU B  91      44.943  25.586  42.150  1.00  0.00           H  \\nATOM   1098  HG  LEU B  91      45.751  23.661  43.437  1.00  0.00           H  \\nATOM   1099 1HD1 LEU B  91      44.217  21.779  43.907  1.00  0.00           H  \\nATOM   1100 2HD1 LEU B  91      43.713  23.258  44.756  1.00  0.00           H  \\nATOM   1101 3HD1 LEU B  91      42.870  22.750  43.286  1.00  0.00           H  \\nATOM   1102 1HD2 LEU B  91      45.364  22.005  41.681  1.00  0.00           H  \\nATOM   1103 2HD2 LEU B  91      44.013  22.950  41.075  1.00  0.00           H  \\nATOM   1104 3HD2 LEU B  91      45.655  23.599  40.979  1.00  0.00           H  \\nATOM   1105  N   GLU B  92      46.086  26.990  44.459  1.00  0.00           N  \\nATOM   1106  CA  GLU B  92      47.334  27.270  45.167  1.00  0.00           C  \\nATOM   1107  C   GLU B  92      47.059  27.586  46.632  1.00  0.00           C  \\nATOM   1108  O   GLU B  92      47.791  27.152  47.518  1.00  0.00           O  \\nATOM   1109  CB  GLU B  92      48.119  28.405  44.502  1.00  0.00           C  \\nATOM   1110  CG  GLU B  92      48.755  28.035  43.151  1.00  0.00           C  \\nATOM   1111  CD  GLU B  92      49.882  27.021  43.270  1.00  0.00           C  \\nATOM   1112  OE1 GLU B  92      50.793  27.264  44.026  1.00  0.00           O  \\nATOM   1113  OE2 GLU B  92      49.827  26.008  42.607  1.00  0.00           O  \\nATOM   1114  H   GLU B  92      45.911  27.541  43.823  1.00  0.00           H  \\nATOM   1115  HA  GLU B  92      47.903  26.485  45.129  1.00  0.00           H  \\nATOM   1116 1HB  GLU B  92      47.463  29.253  44.336  1.00  0.00           H  \\nATOM   1117 2HB  GLU B  92      48.914  28.735  45.170  1.00  0.00           H  \\nATOM   1118 1HG  GLU B  92      47.983  27.617  42.511  1.00  0.00           H  \\nATOM   1119 2HG  GLU B  92      49.129  28.940  42.676  1.00  0.00           H  \\nATOM   1120  N   GLU B  93      45.999  28.350  46.878  1.00  0.00           N  \\nATOM   1121  CA  GLU B  93      45.582  28.670  48.238  1.00  0.00           C  \\nATOM   1122  C   GLU B  93      45.148  27.410  48.975  1.00  0.00           C  \\nATOM   1123  O   GLU B  93      45.513  27.195  50.130  1.00  0.00           O  \\nATOM   1124  CB  GLU B  93      44.432  29.691  48.257  1.00  0.00           C  \\nATOM   1125  CG  GLU B  93      43.964  30.089  49.679  1.00  0.00           C  \\nATOM   1126  CD  GLU B  93      42.902  31.187  49.703  1.00  0.00           C  \\nATOM   1127  OE1 GLU B  93      42.535  31.674  48.656  1.00  0.00           O  \\nATOM   1128  OE2 GLU B  93      42.469  31.537  50.776  1.00  0.00           O  \\nATOM   1129  H   GLU B  93      45.501  28.698  46.269  1.00  0.00           H  \\nATOM   1130  HA  GLU B  93      46.331  29.059  48.718  1.00  0.00           H  \\nATOM   1131 1HB  GLU B  93      44.734  30.592  47.726  1.00  0.00           H  \\nATOM   1132 2HB  GLU B  93      43.571  29.278  47.733  1.00  0.00           H  \\nATOM   1133 1HG  GLU B  93      43.556  29.202  50.168  1.00  0.00           H  \\nATOM   1134 2HG  GLU B  93      44.830  30.415  50.254  1.00  0.00           H  \\nATOM   1135  N   LEU B  94      44.360  26.583  48.295  1.00  0.00           N  \\nATOM   1136  CA  LEU B  94      43.876  25.330  48.862  1.00  0.00           C  \\nATOM   1137  C   LEU B  94      45.039  24.404  49.208  1.00  0.00           C  \\nATOM   1138  O   LEU B  94      45.004  23.690  50.209  1.00  0.00           O  \\nATOM   1139  CB  LEU B  94      42.908  24.651  47.878  1.00  0.00           C  \\nATOM   1140  CG  LEU B  94      42.280  23.347  48.345  1.00  0.00           C  \\nATOM   1141  CD1 LEU B  94      41.538  23.561  49.665  1.00  0.00           C  \\nATOM   1142  CD2 LEU B  94      41.291  22.843  47.247  1.00  0.00           C  \\nATOM   1143  H   LEU B  94      44.088  26.729  47.492  1.00  0.00           H  \\nATOM   1144  HA  LEU B  94      43.387  25.518  49.678  1.00  0.00           H  \\nATOM   1145 1HB  LEU B  94      42.126  25.346  47.604  1.00  0.00           H  \\nATOM   1146 2HB  LEU B  94      43.470  24.421  46.979  1.00  0.00           H  \\nATOM   1147  HG  LEU B  94      43.067  22.618  48.510  1.00  0.00           H  \\nATOM   1148 1HD1 LEU B  94      41.126  22.621  49.987  1.00  0.00           H  \\nATOM   1149 2HD1 LEU B  94      42.234  23.923  50.422  1.00  0.00           H  \\nATOM   1150 3HD1 LEU B  94      40.744  24.277  49.535  1.00  0.00           H  \\nATOM   1151 1HD2 LEU B  94      40.832  21.909  47.553  1.00  0.00           H  \\nATOM   1152 2HD2 LEU B  94      40.511  23.591  47.093  1.00  0.00           H  \\nATOM   1153 3HD2 LEU B  94      41.833  22.687  46.314  1.00  0.00           H  \\nATOM   1154  N   LYS B  95      46.072  24.431  48.373  1.00  0.00           N  \\nATOM   1155  CA  LYS B  95      47.251  23.596  48.574  1.00  0.00           C  \\nATOM   1156  C   LYS B  95      48.018  24.013  49.826  1.00  0.00           C  \\nATOM   1157  O   LYS B  95      48.537  23.169  50.556  1.00  0.00           O  \\nATOM   1158  CB  LYS B  95      48.090  23.600  47.296  1.00  0.00           C  \\nATOM   1159  CG  LYS B  95      49.190  22.550  47.229  1.00  0.00           C  \\nATOM   1160  CD  LYS B  95      49.746  22.477  45.794  1.00  0.00           C  \\nATOM   1161  CE  LYS B  95      50.760  21.348  45.615  1.00  0.00           C  \\nATOM   1162  NZ  LYS B  95      50.111  19.973  45.719  1.00  0.00           N  \\nATOM   1163  H   LYS B  95      46.115  24.930  47.674  1.00  0.00           H  \\nATOM   1164  HA  LYS B  95      46.972  22.674  48.684  1.00  0.00           H  \\nATOM   1165 1HB  LYS B  95      47.435  23.452  46.435  1.00  0.00           H  \\nATOM   1166 2HB  LYS B  95      48.554  24.583  47.178  1.00  0.00           H  \\nATOM   1167 1HG  LYS B  95      49.992  22.814  47.921  1.00  0.00           H  \\nATOM   1168 2HG  LYS B  95      48.787  21.577  47.511  1.00  0.00           H  \\nATOM   1169 1HD  LYS B  95      48.918  22.307  45.099  1.00  0.00           H  \\nATOM   1170 2HD  LYS B  95      50.218  23.428  45.537  1.00  0.00           H  \\nATOM   1171 1HE  LYS B  95      51.227  21.442  44.635  1.00  0.00           H  \\nATOM   1172 2HE  LYS B  95      51.528  21.434  46.383  1.00  0.00           H  \\nATOM   1173 1HZ  LYS B  95      49.689  19.870  46.623  1.00  0.00           H  \\nATOM   1174 2HZ  LYS B  95      49.370  19.829  44.988  1.00  0.00           H  \\nATOM   1175 3HZ  LYS B  95      50.814  19.264  45.596  1.00  0.00           H  \\nATOM   1176  N   GLU B  96      48.092  25.318  50.064  1.00  0.00           N  \\nATOM   1177  CA  GLU B  96      48.741  25.843  51.260  1.00  0.00           C  \\nATOM   1178  C   GLU B  96      47.933  25.548  52.517  1.00  0.00           C  \\nATOM   1179  O   GLU B  96      48.494  25.190  53.553  1.00  0.00           O  \\nATOM   1180  CB  GLU B  96      48.990  27.358  51.176  1.00  0.00           C  \\nATOM   1181  CG  GLU B  96      50.068  27.790  50.175  1.00  0.00           C  \\nATOM   1182  CD  GLU B  96      50.343  29.297  50.189  1.00  0.00           C  \\nATOM   1183  OE1 GLU B  96      49.669  30.011  50.902  1.00  0.00           O  \\nATOM   1184  OE2 GLU B  96      51.242  29.722  49.496  1.00  0.00           O  \\nATOM   1185  H   GLU B  96      47.772  25.925  49.545  1.00  0.00           H  \\nATOM   1186  HA  GLU B  96      49.610  25.421  51.357  1.00  0.00           H  \\nATOM   1187 1HB  GLU B  96      48.060  27.855  50.890  1.00  0.00           H  \\nATOM   1188 2HB  GLU B  96      49.271  27.734  52.159  1.00  0.00           H  \\nATOM   1189 1HG  GLU B  96      50.991  27.261  50.406  1.00  0.00           H  \\nATOM   1190 2HG  GLU B  96      49.754  27.492  49.177  1.00  0.00           H  \\nATOM   1191  N   MET B  97      46.617  25.711  52.425  1.00  0.00           N  \\nATOM   1192  CA  MET B  97      45.735  25.451  53.557  1.00  0.00           C  \\nATOM   1193  C   MET B  97      45.838  23.999  54.010  1.00  0.00           C  \\nATOM   1194  O   MET B  97      45.679  23.697  55.192  1.00  0.00           O  \\nATOM   1195  CB  MET B  97      44.283  25.739  53.197  1.00  0.00           C  \\nATOM   1196  CG  MET B  97      43.919  27.161  52.987  1.00  0.00           C  \\nATOM   1197  SD  MET B  97      44.065  28.158  54.457  1.00  0.00           S  \\nATOM   1198  CE  MET B  97      43.726  29.767  53.744  1.00  0.00           C  \\nATOM   1199  H   MET B  97      46.208  25.973  51.715  1.00  0.00           H  \\nATOM   1200  HA  MET B  97      45.994  26.027  54.293  1.00  0.00           H  \\nATOM   1201 1HB  MET B  97      44.012  25.183  52.298  1.00  0.00           H  \\nATOM   1202 2HB  MET B  97      43.656  25.402  53.988  1.00  0.00           H  \\nATOM   1203 1HG  MET B  97      44.534  27.601  52.227  1.00  0.00           H  \\nATOM   1204 2HG  MET B  97      42.902  27.186  52.658  1.00  0.00           H  \\nATOM   1205 1HE  MET B  97      43.771  30.529  54.522  1.00  0.00           H  \\nATOM   1206 2HE  MET B  97      44.466  29.989  52.971  1.00  0.00           H  \\nATOM   1207 3HE  MET B  97      42.732  29.769  53.292  1.00  0.00           H  \\nATOM   1208  N   LEU B  98      46.110  23.103  53.067  1.00  0.00           N  \\nATOM   1209  CA  LEU B  98      46.230  21.685  53.382  1.00  0.00           C  \\nATOM   1210  C   LEU B  98      47.504  21.417  54.178  1.00  0.00           C  \\nATOM   1211  O   LEU B  98      47.492  20.651  55.141  1.00  0.00           O  \\nATOM   1212  CB  LEU B  98      46.223  20.840  52.104  1.00  0.00           C  \\nATOM   1213  CG  LEU B  98      46.201  19.316  52.332  1.00  0.00           C  \\nATOM   1214  CD1 LEU B  98      44.911  18.956  53.106  1.00  0.00           C  \\nATOM   1215  CD2 LEU B  98      46.266  18.598  51.003  1.00  0.00           C  \\nATOM   1216  H   LEU B  98      46.229  23.291  52.236  1.00  0.00           H  \\nATOM   1217  HA  LEU B  98      45.473  21.416  53.925  1.00  0.00           H  \\nATOM   1218 1HB  LEU B  98      45.350  21.113  51.508  1.00  0.00           H  \\nATOM   1219 2HB  LEU B  98      47.118  21.082  51.524  1.00  0.00           H  \\nATOM   1220  HG  LEU B  98      47.059  19.026  52.941  1.00  0.00           H  \\nATOM   1221 1HD1 LEU B  98      44.871  17.890  53.296  1.00  0.00           H  \\nATOM   1222 2HD1 LEU B  98      44.901  19.491  54.053  1.00  0.00           H  \\nATOM   1223 3HD1 LEU B  98      44.040  19.246  52.516  1.00  0.00           H  \\nATOM   1224 1HD2 LEU B  98      46.252  17.524  51.159  1.00  0.00           H  \\nATOM   1225 2HD2 LEU B  98      45.419  18.887  50.412  1.00  0.00           H  \\nATOM   1226 3HD2 LEU B  98      47.184  18.876  50.486  1.00  0.00           H  \\nATOM   1227  N   GLU B  99      48.602  22.049  53.771  1.00  0.00           N  \\nATOM   1228  CA  GLU B  99      49.869  21.910  54.481  1.00  0.00           C  \\nATOM   1229  C   GLU B  99      49.745  22.430  55.910  1.00  0.00           C  \\nATOM   1230  O   GLU B  99      50.341  21.875  56.833  1.00  0.00           O  \\nATOM   1231  CB  GLU B  99      50.970  22.688  53.745  1.00  0.00           C  \\nATOM   1232  CG  GLU B  99      51.377  22.122  52.367  1.00  0.00           C  \\nATOM   1233  CD  GLU B  99      52.105  20.807  52.412  1.00  0.00           C  \\nATOM   1234  OE1 GLU B  99      53.092  20.723  53.103  1.00  0.00           O  \\nATOM   1235  OE2 GLU B  99      51.680  19.888  51.747  1.00  0.00           O  \\nATOM   1236  H   GLU B  99      48.639  22.565  53.084  1.00  0.00           H  \\nATOM   1237  HA  GLU B  99      50.108  20.971  54.522  1.00  0.00           H  \\nATOM   1238 1HB  GLU B  99      50.641  23.715  53.591  1.00  0.00           H  \\nATOM   1239 2HB  GLU B  99      51.862  22.724  54.368  1.00  0.00           H  \\nATOM   1240 1HG  GLU B  99      50.472  21.991  51.777  1.00  0.00           H  \\nATOM   1241 2HG  GLU B  99      51.998  22.859  51.858  1.00  0.00           H  \\nATOM   1242  N   ARG B 100      48.969  23.496  56.087  1.00  0.00           N  \\nATOM   1243  CA  ARG B 100      48.696  24.029  57.418  1.00  0.00           C  \\nATOM   1244  C   ARG B 100      47.926  23.017  58.257  1.00  0.00           C  \\nATOM   1245  O   ARG B 100      47.992  23.038  59.486  1.00  0.00           O  \\nATOM   1246  CB  ARG B 100      47.878  25.313  57.334  1.00  0.00           C  \\nATOM   1247  CG  ARG B 100      48.604  26.529  56.770  1.00  0.00           C  \\nATOM   1248  CD  ARG B 100      47.705  27.726  56.708  1.00  0.00           C  \\nATOM   1249  NE  ARG B 100      48.355  28.874  56.085  1.00  0.00           N  \\nATOM   1250  CZ  ARG B 100      49.140  29.760  56.733  1.00  0.00           C  \\nATOM   1251  NH1 ARG B 100      49.363  29.627  58.022  1.00  0.00           N  \\nATOM   1252  NH2 ARG B 100      49.686  30.765  56.070  1.00  0.00           N  \\nATOM   1253  H   ARG B 100      48.587  23.930  55.450  1.00  0.00           H  \\nATOM   1254  HA  ARG B 100      49.537  24.216  57.865  1.00  0.00           H  \\nATOM   1255 1HB  ARG B 100      47.006  25.145  56.716  1.00  0.00           H  \\nATOM   1256 2HB  ARG B 100      47.520  25.576  58.328  1.00  0.00           H  \\nATOM   1257 1HG  ARG B 100      49.456  26.768  57.402  1.00  0.00           H  \\nATOM   1258 2HG  ARG B 100      48.949  26.307  55.763  1.00  0.00           H  \\nATOM   1259 1HD  ARG B 100      46.821  27.478  56.123  1.00  0.00           H  \\nATOM   1260 2HD  ARG B 100      47.404  28.007  57.714  1.00  0.00           H  \\nATOM   1261  HE  ARG B 100      48.211  29.012  55.092  1.00  0.00           H  \\nATOM   1262 1HH1 ARG B 100      48.943  28.862  58.531  1.00  0.00           H  \\nATOM   1263 2HH1 ARG B 100      49.949  30.293  58.504  1.00  0.00           H  \\nATOM   1264 1HH2 ARG B 100      49.515  30.869  55.077  1.00  0.00           H  \\nATOM   1265 2HH2 ARG B 100      50.273  31.429  56.553  1.00  0.00           H  \\nATOM   1266  N   LEU B 101      47.192  22.135  57.586  1.00  0.00           N  \\nATOM   1267  CA  LEU B 101      46.382  21.138  58.272  1.00  0.00           C  \\nATOM   1268  C   LEU B 101      47.231  19.941  58.697  1.00  0.00           C  \\nATOM   1269  O   LEU B 101      46.868  19.214  59.620  1.00  0.00           O  \\nATOM   1270  CB  LEU B 101      45.239  20.654  57.344  1.00  0.00           C  \\nATOM   1271  CG  LEU B 101      44.181  19.715  57.940  1.00  0.00           C  \\nATOM   1272  CD1 LEU B 101      43.459  20.385  59.056  1.00  0.00           C  \\nATOM   1273  CD2 LEU B 101      43.203  19.303  56.839  1.00  0.00           C  \\nATOM   1274  H   LEU B 101      47.146  22.094  56.728  1.00  0.00           H  \\nATOM   1275  HA  LEU B 101      46.003  21.535  59.072  1.00  0.00           H  \\nATOM   1276 1HB  LEU B 101      44.734  21.528  56.930  1.00  0.00           H  \\nATOM   1277 2HB  LEU B 101      45.677  20.106  56.530  1.00  0.00           H  \\nATOM   1278  HG  LEU B 101      44.684  18.839  58.348  1.00  0.00           H  \\nATOM   1279 1HD1 LEU B 101      42.738  19.704  59.488  1.00  0.00           H  \\nATOM   1280 2HD1 LEU B 101      44.174  20.688  59.809  1.00  0.00           H  \\nATOM   1281 3HD1 LEU B 101      42.950  21.243  58.662  1.00  0.00           H  \\nATOM   1282 1HD2 LEU B 101      42.470  18.635  57.245  1.00  0.00           H  \\nATOM   1283 2HD2 LEU B 101      42.710  20.172  56.443  1.00  0.00           H  \\nATOM   1284 3HD2 LEU B 101      43.744  18.802  56.041  1.00  0.00           H  \\nATOM   1285  N   GLU B 102      48.357  19.746  58.019  1.00  0.00           N  \\nATOM   1286  CA  GLU B 102      49.255  18.639  58.327  1.00  0.00           C  \\nATOM   1287  C   GLU B 102      50.165  18.992  59.498  1.00  0.00           C  \\nATOM   1288  O   GLU B 102      50.822  18.122  60.070  1.00  0.00           O  \\nATOM   1289  CB  GLU B 102      50.067  18.220  57.098  1.00  0.00           C  \\nATOM   1290  CG  GLU B 102      50.811  16.886  57.301  1.00  0.00           C  \\nATOM   1291  CD  GLU B 102      51.425  16.308  56.043  1.00  0.00           C  \\nATOM   1292  OE1 GLU B 102      52.265  16.934  55.457  1.00  0.00           O  \\nATOM   1293  OE2 GLU B 102      51.016  15.223  55.668  1.00  0.00           O  \\nATOM   1294  H   GLU B 102      48.626  20.244  57.372  1.00  0.00           H  \\nATOM   1295  HA  GLU B 102      48.725  17.853  58.575  1.00  0.00           H  \\nATOM   1296 1HB  GLU B 102      49.404  18.125  56.236  1.00  0.00           H  \\nATOM   1297 2HB  GLU B 102      50.800  18.997  56.864  1.00  0.00           H  \\nATOM   1298 1HG  GLU B 102      51.603  17.043  58.033  1.00  0.00           H  \\nATOM   1299 2HG  GLU B 102      50.113  16.163  57.719  1.00  0.00           H  \\nATOM   1300  N   LYS B 103      50.128  20.274  59.813  1.00  0.00           N  \\nATOM   1301  CA  LYS B 103      50.838  20.829  60.920  1.00  0.00           C  \\nATOM   1302  C   LYS B 103      49.996  20.642  62.151  1.00  0.00           C  \\nATOM   1303  O   LYS B 103      50.320  19.842  63.010  1.00  0.00           O  \\nATOM   1304  CB  LYS B 103      51.182  22.299  60.697  1.00  0.00           C  \\nATOM   1305  CG  LYS B 103      52.255  22.523  59.640  1.00  0.00           C  \\nATOM   1306  CD  LYS B 103      52.559  23.997  59.442  1.00  0.00           C  \\nATOM   1307  CE  LYS B 103      53.617  24.195  58.362  1.00  0.00           C  \\nATOM   1308  NZ  LYS B 103      53.924  25.636  58.135  1.00  0.00           N  \\nATOM   1309  H   LYS B 103      49.659  20.843  59.375  1.00  0.00           H  \\nATOM   1310  HA  LYS B 103      51.691  20.367  61.033  1.00  0.00           H  \\nATOM   1311 1HB  LYS B 103      50.296  22.841  60.389  1.00  0.00           H  \\nATOM   1312 2HB  LYS B 103      51.522  22.739  61.633  1.00  0.00           H  \\nATOM   1313 1HG  LYS B 103      53.167  22.010  59.940  1.00  0.00           H  \\nATOM   1314 2HG  LYS B 103      51.923  22.101  58.699  1.00  0.00           H  \\nATOM   1315 1HD  LYS B 103      51.647  24.516  59.150  1.00  0.00           H  \\nATOM   1316 2HD  LYS B 103      52.916  24.424  60.378  1.00  0.00           H  \\nATOM   1317 1HE  LYS B 103      54.529  23.682  58.661  1.00  0.00           H  \\nATOM   1318 2HE  LYS B 103      53.254  23.758  57.428  1.00  0.00           H  \\nATOM   1319 1HZ  LYS B 103      54.626  25.722  57.414  1.00  0.00           H  \\nATOM   1320 2HZ  LYS B 103      53.085  26.117  57.846  1.00  0.00           H  \\nATOM   1321 3HZ  LYS B 103      54.270  26.047  58.991  1.00  0.00           H  \\nATOM   1322  N   ASN B 104      48.907  21.385  62.235  1.00  0.00           N  \\nATOM   1323  CA  ASN B 104      47.981  21.227  63.340  1.00  0.00           C  \\nATOM   1324  C   ASN B 104      46.570  21.109  62.884  1.00  0.00           C  \\nATOM   1325  O   ASN B 104      45.897  22.086  62.667  1.00  0.00           O  \\nATOM   1326  CB  ASN B 104      48.183  22.397  64.294  1.00  0.00           C  \\nATOM   1327  CG  ASN B 104      48.063  23.763  63.584  1.00  0.00           C  \\nATOM   1328  OD1 ASN B 104      48.875  24.010  62.680  1.00  0.00           O  \\nATOM   1329  ND2 ASN B 104      47.155  24.629  63.953  1.00  0.00           N  \\nATOM   1330  H   ASN B 104      48.687  21.991  61.669  1.00  0.00           H  \\nATOM   1331  HA  ASN B 104      48.203  20.411  63.832  1.00  0.00           H  \\nATOM   1332 1HB  ASN B 104      47.424  22.350  65.085  1.00  0.00           H  \\nATOM   1333 2HB  ASN B 104      49.160  22.323  64.767  1.00  0.00           H  \\nATOM   1334 1HD2 ASN B 104      46.481  24.437  64.692  1.00  0.00           H  \\nATOM   1335 2HD2 ASN B 104      47.095  25.508  63.489  1.00  0.00           H  \\nATOM   1336  N   PRO B 105      46.131  19.797  62.714  1.00  0.00           N  \\nATOM   1337  CA  PRO B 105      44.671  19.680  62.728  1.00  0.00           C  \\nATOM   1338  C   PRO B 105      44.019  20.367  63.924  1.00  0.00           C  \\nATOM   1339  O   PRO B 105      44.379  20.070  65.061  1.00  0.00           O  \\nATOM   1340  CB  PRO B 105      44.524  18.152  62.816  1.00  0.00           C  \\nATOM   1341  CG  PRO B 105      45.803  17.608  62.120  1.00  0.00           C  \\nATOM   1342  CD  PRO B 105      46.925  18.589  62.577  1.00  0.00           C  \\nATOM   1343  HA  PRO B 105      44.288  20.015  61.896  1.00  0.00           H  \\nATOM   1344 1HB  PRO B 105      44.449  17.843  63.865  1.00  0.00           H  \\nATOM   1345 2HB  PRO B 105      43.593  17.838  62.324  1.00  0.00           H  \\nATOM   1346 1HG  PRO B 105      45.988  16.559  62.411  1.00  0.00           H  \\nATOM   1347 2HG  PRO B 105      45.661  17.605  61.029  1.00  0.00           H  \\nATOM   1348 1HD  PRO B 105      47.336  18.278  63.548  1.00  0.00           H  \\nATOM   1349 2HD  PRO B 105      47.711  18.705  61.820  1.00  0.00           H  \\nATOM   1350  N   ASP B 106      43.074  21.257  63.660  1.00  0.00           N  \\nATOM   1351  CA  ASP B 106      42.354  21.987  64.688  1.00  0.00           C  \\nATOM   1352  C   ASP B 106      40.849  21.888  64.434  1.00  0.00           C  \\nATOM   1353  O   ASP B 106      40.373  20.900  63.925  1.00  0.00           O  \\nATOM   1354  CB  ASP B 106      42.802  23.450  64.822  1.00  0.00           C  \\nATOM   1355  CG  ASP B 106      44.215  23.663  65.447  1.00  0.00           C  \\nATOM   1356  OD1 ASP B 106      44.735  22.781  66.069  1.00  0.00           O  \\nATOM   1357  OD2 ASP B 106      44.749  24.726  65.280  1.00  0.00           O  \\nATOM   1358  H   ASP B 106      42.821  21.459  62.868  1.00  0.00           H  \\nATOM   1359  HA  ASP B 106      42.550  21.604  65.566  1.00  0.00           H  \\nATOM   1360 1HB  ASP B 106      42.749  23.910  63.904  1.00  0.00           H  \\nATOM   1361 2HB  ASP B 106      42.127  23.978  65.474  1.00  0.00           H  \\nATOM   1362  N   LYS B 107      40.098  22.920  64.791  1.00  0.00           N  \\nATOM   1363  CA  LYS B 107      38.690  22.982  64.474  1.00  0.00           C  \\nATOM   1364  C   LYS B 107      38.529  24.141  63.561  1.00  0.00           C  \\nATOM   1365  O   LYS B 107      37.928  24.036  62.513  1.00  0.00           O  \\nATOM   1366  CB  LYS B 107      37.799  23.133  65.707  1.00  0.00           C  \\nATOM   1367  CG  LYS B 107      36.288  23.083  65.364  1.00  0.00           C  \\nATOM   1368  CD  LYS B 107      35.397  23.179  66.598  1.00  0.00           C  \\nATOM   1369  CE  LYS B 107      35.427  21.897  67.410  1.00  0.00           C  \\nATOM   1370  NZ  LYS B 107      34.483  21.947  68.560  1.00  0.00           N  \\nATOM   1371  H   LYS B 107      40.373  23.604  65.225  1.00  0.00           H  \\nATOM   1372  HA  LYS B 107      38.402  22.166  64.024  1.00  0.00           H  \\nATOM   1373 1HB  LYS B 107      38.017  22.340  66.421  1.00  0.00           H  \\nATOM   1374 2HB  LYS B 107      38.001  24.089  66.194  1.00  0.00           H  \\nATOM   1375 1HG  LYS B 107      36.047  23.916  64.699  1.00  0.00           H  \\nATOM   1376 2HG  LYS B 107      36.062  22.153  64.840  1.00  0.00           H  \\nATOM   1377 1HD  LYS B 107      35.712  24.019  67.218  1.00  0.00           H  \\nATOM   1378 2HD  LYS B 107      34.369  23.346  66.279  1.00  0.00           H  \\nATOM   1379 1HE  LYS B 107      35.151  21.063  66.764  1.00  0.00           H  \\nATOM   1380 2HE  LYS B 107      36.436  21.735  67.787  1.00  0.00           H  \\nATOM   1381 1HZ  LYS B 107      33.538  22.087  68.227  1.00  0.00           H  \\nATOM   1382 2HZ  LYS B 107      34.532  21.079  69.070  1.00  0.00           H  \\nATOM   1383 3HZ  LYS B 107      34.737  22.710  69.168  1.00  0.00           H  \\nATOM   1384  N   ASP B 108      39.070  25.270  63.968  1.00  0.00           N  \\nATOM   1385  CA  ASP B 108      38.957  26.410  63.067  1.00  0.00           C  \\nATOM   1386  C   ASP B 108      39.748  26.165  61.787  1.00  0.00           C  \\nATOM   1387  O   ASP B 108      39.370  26.633  60.713  1.00  0.00           O  \\nATOM   1388  CB  ASP B 108      39.470  27.680  63.742  1.00  0.00           C  \\nATOM   1389  CG  ASP B 108      38.549  28.176  64.851  1.00  0.00           C  \\nATOM   1390  OD1 ASP B 108      37.447  27.687  64.948  1.00  0.00           O  \\nATOM   1391  OD2 ASP B 108      38.959  29.031  65.595  1.00  0.00           O  \\nATOM   1392  H   ASP B 108      39.499  25.438  64.694  1.00  0.00           H  \\nATOM   1393  HA  ASP B 108      38.016  26.540  62.826  1.00  0.00           H  \\nATOM   1394 1HB  ASP B 108      40.460  27.493  64.160  1.00  0.00           H  \\nATOM   1395 2HB  ASP B 108      39.578  28.469  62.992  1.00  0.00           H  \\nATOM   1396  N   VAL B 109      40.848  25.430  61.909  1.00  0.00           N  \\nATOM   1397  CA  VAL B 109      41.693  25.120  60.763  1.00  0.00           C  \\nATOM   1398  C   VAL B 109      40.925  24.300  59.732  1.00  0.00           C  \\nATOM   1399  O   VAL B 109      40.909  24.634  58.547  1.00  0.00           O  \\nATOM   1400  CB  VAL B 109      42.987  24.378  61.145  1.00  0.00           C  \\nATOM   1401  CG1 VAL B 109      43.711  23.909  59.897  1.00  0.00           C  \\nATOM   1402  CG2 VAL B 109      43.908  25.320  61.906  1.00  0.00           C  \\nATOM   1403  H   VAL B 109      41.128  25.096  62.650  1.00  0.00           H  \\nATOM   1404  HA  VAL B 109      41.982  25.955  60.339  1.00  0.00           H  \\nATOM   1405  HB  VAL B 109      42.736  23.507  61.730  1.00  0.00           H  \\nATOM   1406 1HG1 VAL B 109      44.624  23.373  60.181  1.00  0.00           H  \\nATOM   1407 2HG1 VAL B 109      43.067  23.267  59.343  1.00  0.00           H  \\nATOM   1408 3HG1 VAL B 109      43.974  24.768  59.279  1.00  0.00           H  \\nATOM   1409 1HG2 VAL B 109      44.826  24.786  62.175  1.00  0.00           H  \\nATOM   1410 2HG2 VAL B 109      44.154  26.170  61.270  1.00  0.00           H  \\nATOM   1411 3HG2 VAL B 109      43.431  25.686  62.808  1.00  0.00           H  \\nATOM   1412  N   ILE B 110      40.291  23.227  60.191  1.00  0.00           N  \\nATOM   1413  CA  ILE B 110      39.518  22.359  59.310  1.00  0.00           C  \\nATOM   1414  C   ILE B 110      38.450  23.156  58.570  1.00  0.00           C  \\nATOM   1415  O   ILE B 110      38.302  23.034  57.354  1.00  0.00           O  \\nATOM   1416  CB  ILE B 110      38.864  21.155  60.019  1.00  0.00           C  \\nATOM   1417  CG1 ILE B 110      39.952  20.148  60.448  1.00  0.00           C  \\nATOM   1418  CG2 ILE B 110      37.817  20.508  59.103  1.00  0.00           C  \\nATOM   1419  CD1 ILE B 110      39.460  19.052  61.367  1.00  0.00           C  \\nATOM   1420  H   ILE B 110      40.292  22.978  61.014  1.00  0.00           H  \\nATOM   1421  HA  ILE B 110      40.117  21.956  58.647  1.00  0.00           H  \\nATOM   1422  HB  ILE B 110      38.372  21.501  60.934  1.00  0.00           H  \\nATOM   1423 1HG1 ILE B 110      40.366  19.683  59.556  1.00  0.00           H  \\nATOM   1424 2HG1 ILE B 110      40.746  20.693  60.964  1.00  0.00           H  \\nATOM   1425 1HG2 ILE B 110      37.356  19.666  59.611  1.00  0.00           H  \\nATOM   1426 2HG2 ILE B 110      37.052  21.238  58.847  1.00  0.00           H  \\nATOM   1427 3HG2 ILE B 110      38.295  20.170  58.210  1.00  0.00           H  \\nATOM   1428 1HD1 ILE B 110      39.057  19.496  62.279  1.00  0.00           H  \\nATOM   1429 2HD1 ILE B 110      38.680  18.481  60.869  1.00  0.00           H  \\nATOM   1430 3HD1 ILE B 110      40.288  18.392  61.622  1.00  0.00           H  \\nATOM   1431  N   VAL B 111      37.709  23.973  59.311  1.00  0.00           N  \\nATOM   1432  CA  VAL B 111      36.656  24.794  58.726  1.00  0.00           C  \\nATOM   1433  C   VAL B 111      37.227  25.723  57.661  1.00  0.00           C  \\nATOM   1434  O   VAL B 111      36.581  25.993  56.648  1.00  0.00           O  \\nATOM   1435  CB  VAL B 111      35.935  25.583  59.817  1.00  0.00           C  \\nATOM   1436  CG1 VAL B 111      34.989  26.595  59.190  1.00  0.00           C  \\nATOM   1437  CG2 VAL B 111      35.171  24.597  60.705  1.00  0.00           C  \\nATOM   1438  H   VAL B 111      37.797  24.070  60.161  1.00  0.00           H  \\nATOM   1439  HA  VAL B 111      35.990  24.212  58.303  1.00  0.00           H  \\nATOM   1440  HB  VAL B 111      36.670  26.127  60.417  1.00  0.00           H  \\nATOM   1441 1HG1 VAL B 111      34.483  27.153  59.976  1.00  0.00           H  \\nATOM   1442 2HG1 VAL B 111      35.566  27.278  58.569  1.00  0.00           H  \\nATOM   1443 3HG1 VAL B 111      34.257  26.086  58.582  1.00  0.00           H  \\nATOM   1444 1HG2 VAL B 111      34.672  25.149  61.490  1.00  0.00           H  \\nATOM   1445 2HG2 VAL B 111      34.443  24.060  60.114  1.00  0.00           H  \\nATOM   1446 3HG2 VAL B 111      35.866  23.889  61.146  1.00  0.00           H  \\nATOM   1447  N   GLU B 112      38.441  26.209  57.897  1.00  0.00           N  \\nATOM   1448  CA  GLU B 112      39.104  27.107  56.957  1.00  0.00           C  \\nATOM   1449  C   GLU B 112      39.414  26.401  55.636  1.00  0.00           C  \\nATOM   1450  O   GLU B 112      39.383  27.024  54.576  1.00  0.00           O  \\nATOM   1451  CB  GLU B 112      40.375  27.771  57.519  1.00  0.00           C  \\nATOM   1452  CG  GLU B 112      40.118  28.885  58.553  1.00  0.00           C  \\nATOM   1453  CD  GLU B 112      41.387  29.472  59.159  1.00  0.00           C  \\nATOM   1454  OE1 GLU B 112      42.454  28.971  58.890  1.00  0.00           O  \\nATOM   1455  OE2 GLU B 112      41.277  30.432  59.889  1.00  0.00           O  \\nATOM   1456  H   GLU B 112      38.906  26.033  58.599  1.00  0.00           H  \\nATOM   1457  HA  GLU B 112      38.514  27.852  56.765  1.00  0.00           H  \\nATOM   1458 1HB  GLU B 112      41.000  27.027  57.992  1.00  0.00           H  \\nATOM   1459 2HB  GLU B 112      40.951  28.199  56.698  1.00  0.00           H  \\nATOM   1460 1HG  GLU B 112      39.592  29.688  58.040  1.00  0.00           H  \\nATOM   1461 2HG  GLU B 112      39.467  28.533  59.329  1.00  0.00           H  \\nATOM   1462  N   VAL B 113      39.706  25.104  55.700  1.00  0.00           N  \\nATOM   1463  CA  VAL B 113      40.001  24.326  54.498  1.00  0.00           C  \\nATOM   1464  C   VAL B 113      38.723  24.029  53.715  1.00  0.00           C  \\nATOM   1465  O   VAL B 113      38.716  24.089  52.487  1.00  0.00           O  \\nATOM   1466  CB  VAL B 113      40.678  22.975  54.877  1.00  0.00           C  \\nATOM   1467  CG1 VAL B 113      40.863  22.102  53.633  1.00  0.00           C  \\nATOM   1468  CG2 VAL B 113      42.025  23.223  55.557  1.00  0.00           C  \\nATOM   1469  H   VAL B 113      39.740  24.650  56.429  1.00  0.00           H  \\nATOM   1470  HA  VAL B 113      40.590  24.842  53.925  1.00  0.00           H  \\nATOM   1471  HB  VAL B 113      40.022  22.436  55.564  1.00  0.00           H  \\nATOM   1472 1HG1 VAL B 113      41.314  21.150  53.916  1.00  0.00           H  \\nATOM   1473 2HG1 VAL B 113      39.892  21.925  53.183  1.00  0.00           H  \\nATOM   1474 3HG1 VAL B 113      41.511  22.612  52.918  1.00  0.00           H  \\nATOM   1475 1HG2 VAL B 113      42.477  22.289  55.832  1.00  0.00           H  \\nATOM   1476 2HG2 VAL B 113      42.671  23.732  54.884  1.00  0.00           H  \\nATOM   1477 3HG2 VAL B 113      41.880  23.824  56.447  1.00  0.00           H  \\nATOM   1478  N   LEU B 114      37.645  23.709  54.426  1.00  0.00           N  \\nATOM   1479  CA  LEU B 114      36.368  23.418  53.782  1.00  0.00           C  \\nATOM   1480  C   LEU B 114      35.861  24.630  53.009  1.00  0.00           C  \\nATOM   1481  O   LEU B 114      35.326  24.495  51.910  1.00  0.00           O  \\nATOM   1482  CB  LEU B 114      35.355  23.012  54.860  1.00  0.00           C  \\nATOM   1483  CG  LEU B 114      35.626  21.655  55.540  1.00  0.00           C  \\nATOM   1484  CD1 LEU B 114      34.728  21.496  56.740  1.00  0.00           C  \\nATOM   1485  CD2 LEU B 114      35.364  20.536  54.554  1.00  0.00           C  \\nATOM   1486  H   LEU B 114      37.627  23.654  55.284  1.00  0.00           H  \\nATOM   1487  HA  LEU B 114      36.490  22.690  53.152  1.00  0.00           H  \\nATOM   1488 1HB  LEU B 114      35.354  23.778  55.638  1.00  0.00           H  \\nATOM   1489 2HB  LEU B 114      34.362  22.976  54.413  1.00  0.00           H  \\nATOM   1490  HG  LEU B 114      36.666  21.616  55.878  1.00  0.00           H  \\nATOM   1491 1HD1 LEU B 114      34.941  20.555  57.204  1.00  0.00           H  \\nATOM   1492 2HD1 LEU B 114      34.932  22.290  57.442  1.00  0.00           H  \\nATOM   1493 3HD1 LEU B 114      33.684  21.530  56.434  1.00  0.00           H  \\nATOM   1494 1HD2 LEU B 114      35.550  19.586  55.043  1.00  0.00           H  \\nATOM   1495 2HD2 LEU B 114      34.329  20.573  54.228  1.00  0.00           H  \\nATOM   1496 3HD2 LEU B 114      36.019  20.643  53.693  1.00  0.00           H  \\nATOM   1497  N   LYS B 115      36.038  25.814  53.586  1.00  0.00           N  \\nATOM   1498  CA  LYS B 115      35.582  27.043  52.948  1.00  0.00           C  \\nATOM   1499  C   LYS B 115      36.336  27.303  51.649  1.00  0.00           C  \\nATOM   1500  O   LYS B 115      35.747  27.749  50.672  1.00  0.00           O  \\nATOM   1501  CB  LYS B 115      35.724  28.213  53.929  1.00  0.00           C  \\nATOM   1502  CG  LYS B 115      34.692  28.188  55.101  1.00  0.00           C  \\nATOM   1503  CD  LYS B 115      34.997  29.256  56.163  1.00  0.00           C  \\nATOM   1504  CE  LYS B 115      34.562  30.663  55.750  1.00  0.00           C  \\nATOM   1505  NZ  LYS B 115      33.074  30.847  55.848  1.00  0.00           N  \\nATOM   1506  H   LYS B 115      36.420  25.933  54.347  1.00  0.00           H  \\nATOM   1507  HA  LYS B 115      34.641  26.954  52.733  1.00  0.00           H  \\nATOM   1508 1HB  LYS B 115      36.727  28.189  54.370  1.00  0.00           H  \\nATOM   1509 2HB  LYS B 115      35.620  29.154  53.396  1.00  0.00           H  \\nATOM   1510 1HG  LYS B 115      33.695  28.370  54.706  1.00  0.00           H  \\nATOM   1511 2HG  LYS B 115      34.697  27.207  55.570  1.00  0.00           H  \\nATOM   1512 1HD  LYS B 115      34.481  28.990  57.085  1.00  0.00           H  \\nATOM   1513 2HD  LYS B 115      36.070  29.257  56.363  1.00  0.00           H  \\nATOM   1514 1HE  LYS B 115      35.050  31.389  56.399  1.00  0.00           H  \\nATOM   1515 2HE  LYS B 115      34.873  30.844  54.722  1.00  0.00           H  \\nATOM   1516 1HZ  LYS B 115      32.601  30.188  55.246  1.00  0.00           H  \\nATOM   1517 2HZ  LYS B 115      32.782  30.692  56.808  1.00  0.00           H  \\nATOM   1518 3HZ  LYS B 115      32.827  31.788  55.575  1.00  0.00           H  \\nATOM   1519  N   VAL B 116      37.633  27.015  51.639  1.00  0.00           N  \\nATOM   1520  CA  VAL B 116      38.446  27.193  50.440  1.00  0.00           C  \\nATOM   1521  C   VAL B 116      38.081  26.141  49.394  1.00  0.00           C  \\nATOM   1522  O   VAL B 116      38.090  26.420  48.193  1.00  0.00           O  \\nATOM   1523  CB  VAL B 116      39.952  27.178  50.719  1.00  0.00           C  \\nATOM   1524  CG1 VAL B 116      40.706  27.238  49.387  1.00  0.00           C  \\nATOM   1525  CG2 VAL B 116      40.316  28.386  51.557  1.00  0.00           C  \\nATOM   1526  H   VAL B 116      38.069  26.712  52.316  1.00  0.00           H  \\nATOM   1527  HA  VAL B 116      38.265  28.068  50.062  1.00  0.00           H  \\nATOM   1528  HB  VAL B 116      40.225  26.253  51.236  1.00  0.00           H  \\nATOM   1529 1HG1 VAL B 116      41.772  27.235  49.576  1.00  0.00           H  \\nATOM   1530 2HG1 VAL B 116      40.433  26.390  48.777  1.00  0.00           H  \\nATOM   1531 3HG1 VAL B 116      40.441  28.155  48.860  1.00  0.00           H  \\nATOM   1532 1HG2 VAL B 116      41.371  28.404  51.738  1.00  0.00           H  \\nATOM   1533 2HG2 VAL B 116      40.040  29.295  51.021  1.00  0.00           H  \\nATOM   1534 3HG2 VAL B 116      39.783  28.350  52.503  1.00  0.00           H  \\nATOM   1535  N   ILE B 117      37.747  24.938  49.853  1.00  0.00           N  \\nATOM   1536  CA  ILE B 117      37.345  23.857  48.955  1.00  0.00           C  \\nATOM   1537  C   ILE B 117      36.060  24.222  48.215  1.00  0.00           C  \\nATOM   1538  O   ILE B 117      35.910  23.910  47.037  1.00  0.00           O  \\nATOM   1539  CB  ILE B 117      37.103  22.507  49.667  1.00  0.00           C  \\nATOM   1540  CG1 ILE B 117      38.398  21.897  50.150  1.00  0.00           C  \\nATOM   1541  CG2 ILE B 117      36.380  21.590  48.737  1.00  0.00           C  \\nATOM   1542  CD1 ILE B 117      38.218  20.731  51.089  1.00  0.00           C  \\nATOM   1543  H   ILE B 117      37.744  24.720  50.686  1.00  0.00           H  \\nATOM   1544  HA  ILE B 117      38.042  23.723  48.294  1.00  0.00           H  \\nATOM   1545  HB  ILE B 117      36.488  22.669  50.547  1.00  0.00           H  \\nATOM   1546 1HG1 ILE B 117      38.956  21.550  49.282  1.00  0.00           H  \\nATOM   1547 2HG1 ILE B 117      38.969  22.649  50.643  1.00  0.00           H  \\nATOM   1548 1HG2 ILE B 117      36.195  20.647  49.220  1.00  0.00           H  \\nATOM   1549 2HG2 ILE B 117      35.436  22.038  48.442  1.00  0.00           H  \\nATOM   1550 3HG2 ILE B 117      36.983  21.440  47.882  1.00  0.00           H  \\nATOM   1551 1HD1 ILE B 117      39.195  20.345  51.375  1.00  0.00           H  \\nATOM   1552 2HD1 ILE B 117      37.687  21.061  51.972  1.00  0.00           H  \\nATOM   1553 3HD1 ILE B 117      37.655  19.957  50.608  1.00  0.00           H  \\nATOM   1554  N   VAL B 118      35.142  24.895  48.902  1.00  0.00           N  \\nATOM   1555  CA  VAL B 118      33.877  25.292  48.293  1.00  0.00           C  \\nATOM   1556  C   VAL B 118      34.087  26.365  47.230  1.00  0.00           C  \\nATOM   1557  O   VAL B 118      33.445  26.337  46.181  1.00  0.00           O  \\nATOM   1558  CB  VAL B 118      32.853  25.731  49.350  1.00  0.00           C  \\nATOM   1559  CG1 VAL B 118      31.587  26.275  48.653  1.00  0.00           C  \\nATOM   1560  CG2 VAL B 118      32.506  24.522  50.227  1.00  0.00           C  \\nATOM   1561  H   VAL B 118      35.228  25.135  49.723  1.00  0.00           H  \\nATOM   1562  HA  VAL B 118      33.481  24.519  47.861  1.00  0.00           H  \\nATOM   1563  HB  VAL B 118      33.279  26.530  49.966  1.00  0.00           H  \\nATOM   1564 1HG1 VAL B 118      30.858  26.583  49.400  1.00  0.00           H  \\nATOM   1565 2HG1 VAL B 118      31.856  27.130  48.031  1.00  0.00           H  \\nATOM   1566 3HG1 VAL B 118      31.151  25.493  48.029  1.00  0.00           H  \\nATOM   1567 1HG2 VAL B 118      31.780  24.799  50.982  1.00  0.00           H  \\nATOM   1568 2HG2 VAL B 118      32.106  23.756  49.614  1.00  0.00           H  \\nATOM   1569 3HG2 VAL B 118      33.383  24.150  50.701  1.00  0.00           H  \\nATOM   1570  N   LYS B 119      34.984  27.309  47.498  1.00  0.00           N  \\nATOM   1571  CA  LYS B 119      35.297  28.345  46.522  1.00  0.00           C  \\nATOM   1572  C   LYS B 119      35.963  27.742  45.296  1.00  0.00           C  \\nATOM   1573  O   LYS B 119      35.760  28.216  44.182  1.00  0.00           O  \\nATOM   1574  CB  LYS B 119      36.249  29.394  47.105  1.00  0.00           C  \\nATOM   1575  CG  LYS B 119      35.675  30.300  48.188  1.00  0.00           C  \\nATOM   1576  CD  LYS B 119      34.593  31.220  47.619  1.00  0.00           C  \\nATOM   1577  CE  LYS B 119      34.237  32.361  48.584  1.00  0.00           C  \\nATOM   1578  NZ  LYS B 119      33.685  31.871  49.872  1.00  0.00           N  \\nATOM   1579  H   LYS B 119      35.423  27.371  48.234  1.00  0.00           H  \\nATOM   1580  HA  LYS B 119      34.474  28.772  46.238  1.00  0.00           H  \\nATOM   1581 1HB  LYS B 119      37.117  28.883  47.534  1.00  0.00           H  \\nATOM   1582 2HB  LYS B 119      36.617  30.029  46.302  1.00  0.00           H  \\nATOM   1583 1HG  LYS B 119      35.238  29.681  48.975  1.00  0.00           H  \\nATOM   1584 2HG  LYS B 119      36.471  30.901  48.625  1.00  0.00           H  \\nATOM   1585 1HD  LYS B 119      34.943  31.650  46.679  1.00  0.00           H  \\nATOM   1586 2HD  LYS B 119      33.693  30.635  47.412  1.00  0.00           H  \\nATOM   1587 1HE  LYS B 119      35.129  32.953  48.784  1.00  0.00           H  \\nATOM   1588 2HE  LYS B 119      33.492  32.998  48.105  1.00  0.00           H  \\nATOM   1589 1HZ  LYS B 119      33.460  32.659  50.462  1.00  0.00           H  \\nATOM   1590 2HZ  LYS B 119      32.853  31.330  49.700  1.00  0.00           H  \\nATOM   1591 3HZ  LYS B 119      34.374  31.295  50.330  1.00  0.00           H  \\nATOM   1592  N   ALA B 120      36.752  26.693  45.507  1.00  0.00           N  \\nATOM   1593  CA  ALA B 120      37.411  26.007  44.403  1.00  0.00           C  \\nATOM   1594  C   ALA B 120      36.373  25.323  43.527  1.00  0.00           C  \\nATOM   1595  O   ALA B 120      36.441  25.390  42.300  1.00  0.00           O  \\nATOM   1596  CB  ALA B 120      38.412  24.980  44.929  1.00  0.00           C  \\nATOM   1597  H   ALA B 120      36.922  26.359  46.281  1.00  0.00           H  \\nATOM   1598  HA  ALA B 120      37.886  26.656  43.861  1.00  0.00           H  \\nATOM   1599 1HB  ALA B 120      37.891  24.238  45.527  1.00  0.00           H  \\nATOM   1600 2HB  ALA B 120      38.902  24.488  44.092  1.00  0.00           H  \\nATOM   1601 3HB  ALA B 120      39.155  25.475  45.547  1.00  0.00           H  \\nATOM   1602  N   ILE B 121      35.410  24.664  44.165  1.00  0.00           N  \\nATOM   1603  CA  ILE B 121      34.331  24.005  43.442  1.00  0.00           C  \\nATOM   1604  C   ILE B 121      33.499  25.042  42.700  1.00  0.00           C  \\nATOM   1605  O   ILE B 121      33.155  24.852  41.536  1.00  0.00           O  \\nATOM   1606  CB  ILE B 121      33.441  23.177  44.381  1.00  0.00           C  \\nATOM   1607  CG1 ILE B 121      34.250  21.986  44.915  1.00  0.00           C  \\nATOM   1608  CG2 ILE B 121      32.192  22.698  43.627  1.00  0.00           C  \\nATOM   1609  CD1 ILE B 121      33.616  21.261  46.057  1.00  0.00           C  \\nATOM   1610  H   ILE B 121      35.359  24.585  45.020  1.00  0.00           H  \\nATOM   1611  HA  ILE B 121      34.713  23.400  42.786  1.00  0.00           H  \\nATOM   1612  HB  ILE B 121      33.144  23.792  45.234  1.00  0.00           H  \\nATOM   1613 1HG1 ILE B 121      34.397  21.275  44.104  1.00  0.00           H  \\nATOM   1614 2HG1 ILE B 121      35.226  22.345  45.237  1.00  0.00           H  \\nATOM   1615 1HG2 ILE B 121      31.563  22.122  44.300  1.00  0.00           H  \\nATOM   1616 2HG2 ILE B 121      31.642  23.558  43.260  1.00  0.00           H  \\nATOM   1617 3HG2 ILE B 121      32.490  22.077  42.783  1.00  0.00           H  \\nATOM   1618 1HD1 ILE B 121      34.272  20.461  46.336  1.00  0.00           H  \\nATOM   1619 2HD1 ILE B 121      33.488  21.946  46.893  1.00  0.00           H  \\nATOM   1620 3HD1 ILE B 121      32.650  20.862  45.760  1.00  0.00           H  \\nATOM   1621  N   GLU B 122      33.179  26.143  43.372  1.00  0.00           N  \\nATOM   1622  CA  GLU B 122      32.456  27.229  42.722  1.00  0.00           C  \\nATOM   1623  C   GLU B 122      33.227  27.712  41.496  1.00  0.00           C  \\nATOM   1624  O   GLU B 122      32.656  27.880  40.417  1.00  0.00           O  \\nATOM   1625  CB  GLU B 122      32.216  28.407  43.677  1.00  0.00           C  \\nATOM   1626  CG  GLU B 122      31.445  29.583  43.034  1.00  0.00           C  \\nATOM   1627  CD  GLU B 122      31.154  30.739  43.974  1.00  0.00           C  \\nATOM   1628  OE1 GLU B 122      31.511  30.674  45.134  1.00  0.00           O  \\nATOM   1629  OE2 GLU B 122      30.541  31.679  43.537  1.00  0.00           O  \\nATOM   1630  H   GLU B 122      33.367  26.286  44.199  1.00  0.00           H  \\nATOM   1631  HA  GLU B 122      31.591  26.904  42.427  1.00  0.00           H  \\nATOM   1632 1HB  GLU B 122      31.661  28.062  44.550  1.00  0.00           H  \\nATOM   1633 2HB  GLU B 122      33.171  28.787  44.029  1.00  0.00           H  \\nATOM   1634 1HG  GLU B 122      32.042  29.959  42.202  1.00  0.00           H  \\nATOM   1635 2HG  GLU B 122      30.510  29.216  42.623  1.00  0.00           H  \\nATOM   1636  N   ALA B 123      34.529  27.913  41.667  1.00  0.00           N  \\nATOM   1637  CA  ALA B 123      35.385  28.352  40.574  1.00  0.00           C  \\nATOM   1638  C   ALA B 123      35.407  27.321  39.448  1.00  0.00           C  \\nATOM   1639  O   ALA B 123      35.443  27.678  38.275  1.00  0.00           O  \\nATOM   1640  CB  ALA B 123      36.791  28.608  41.079  1.00  0.00           C  \\nATOM   1641  H   ALA B 123      34.944  27.801  42.413  1.00  0.00           H  \\nATOM   1642  HA  ALA B 123      35.037  29.183  40.214  1.00  0.00           H  \\nATOM   1643 1HB  ALA B 123      37.423  28.961  40.264  1.00  0.00           H  \\nATOM   1644 2HB  ALA B 123      36.734  29.361  41.854  1.00  0.00           H  \\nATOM   1645 3HB  ALA B 123      37.205  27.698  41.492  1.00  0.00           H  \\nATOM   1646  N   SER B 124      35.374  26.043  39.814  1.00  0.00           N  \\nATOM   1647  CA  SER B 124      35.434  24.956  38.840  1.00  0.00           C  \\nATOM   1648  C   SER B 124      34.142  24.842  38.037  1.00  0.00           C  \\nATOM   1649  O   SER B 124      34.174  24.635  36.825  1.00  0.00           O  \\nATOM   1650  CB  SER B 124      35.701  23.662  39.573  1.00  0.00           C  \\nATOM   1651  OG  SER B 124      35.771  22.567  38.709  1.00  0.00           O  \\nATOM   1652  H   SER B 124      35.317  25.776  40.630  1.00  0.00           H  \\nATOM   1653  HA  SER B 124      36.158  25.130  38.218  1.00  0.00           H  \\nATOM   1654 1HB  SER B 124      36.628  23.749  40.134  1.00  0.00           H  \\nATOM   1655 2HB  SER B 124      34.908  23.497  40.291  1.00  0.00           H  \\nATOM   1656  HG  SER B 124      35.784  21.800  39.283  1.00  0.00           H  \\nATOM   1657  N   VAL B 125      33.008  24.971  38.717  1.00  0.00           N  \\nATOM   1658  CA  VAL B 125      31.712  24.958  38.047  1.00  0.00           C  \\nATOM   1659  C   VAL B 125      31.604  26.144  37.087  1.00  0.00           C  \\nATOM   1660  O   VAL B 125      31.047  26.023  35.995  1.00  0.00           O  \\nATOM   1661  CB  VAL B 125      30.557  24.978  39.080  1.00  0.00           C  \\nATOM   1662  CG1 VAL B 125      29.206  25.147  38.370  1.00  0.00           C  \\nATOM   1663  CG2 VAL B 125      30.562  23.671  39.879  1.00  0.00           C  \\nATOM   1664  H   VAL B 125      32.961  25.067  39.570  1.00  0.00           H  \\nATOM   1665  HA  VAL B 125      31.630  24.142  37.529  1.00  0.00           H  \\nATOM   1666  HB  VAL B 125      30.699  25.827  39.758  1.00  0.00           H  \\nATOM   1667 1HG1 VAL B 125      28.407  25.166  39.112  1.00  0.00           H  \\nATOM   1668 2HG1 VAL B 125      29.208  26.083  37.809  1.00  0.00           H  \\nATOM   1669 3HG1 VAL B 125      29.046  24.311  37.686  1.00  0.00           H  \\nATOM   1670 1HG2 VAL B 125      29.756  23.685  40.615  1.00  0.00           H  \\nATOM   1671 2HG2 VAL B 125      30.423  22.848  39.203  1.00  0.00           H  \\nATOM   1672 3HG2 VAL B 125      31.505  23.552  40.386  1.00  0.00           H  \\nATOM   1673  N   GLU B 126      32.146  27.285  37.495  1.00  0.00           N  \\nATOM   1674  CA  GLU B 126      32.132  28.476  36.655  1.00  0.00           C  \\nATOM   1675  C   GLU B 126      33.007  28.268  35.421  1.00  0.00           C  \\nATOM   1676  O   GLU B 126      32.635  28.654  34.314  1.00  0.00           O  \\nATOM   1677  CB  GLU B 126      32.535  29.716  37.450  1.00  0.00           C  \\nATOM   1678  CG  GLU B 126      32.475  31.042  36.666  1.00  0.00           C  \\nATOM   1679  CD  GLU B 126      31.085  31.367  36.110  1.00  0.00           C  \\nATOM   1680  OE1 GLU B 126      30.144  30.665  36.412  1.00  0.00           O  \\nATOM   1681  OE2 GLU B 126      30.957  32.360  35.425  1.00  0.00           O  \\nATOM   1682  H   GLU B 126      32.529  27.397  38.257  1.00  0.00           H  \\nATOM   1683  HA  GLU B 126      31.225  28.643  36.356  1.00  0.00           H  \\nATOM   1684 1HB  GLU B 126      31.894  29.807  38.327  1.00  0.00           H  \\nATOM   1685 2HB  GLU B 126      33.556  29.590  37.811  1.00  0.00           H  \\nATOM   1686 1HG  GLU B 126      32.792  31.850  37.323  1.00  0.00           H  \\nATOM   1687 2HG  GLU B 126      33.186  30.984  35.840  1.00  0.00           H  \\nATOM   1688  N   ASN B 127      34.165  27.648  35.613  1.00  0.00           N  \\nATOM   1689  CA  ASN B 127      35.051  27.352  34.495  1.00  0.00           C  \\nATOM   1690  C   ASN B 127      34.420  26.353  33.530  1.00  0.00           C  \\nATOM   1691  O   ASN B 127      34.582  26.463  32.317  1.00  0.00           O  \\nATOM   1692  CB  ASN B 127      36.384  26.818  35.022  1.00  0.00           C  \\nATOM   1693  CG  ASN B 127      37.487  26.703  33.971  1.00  0.00           C  \\nATOM   1694  OD1 ASN B 127      37.926  27.698  33.393  1.00  0.00           O  \\nATOM   1695  ND2 ASN B 127      37.925  25.502  33.712  1.00  0.00           N  \\nATOM   1696  H   ASN B 127      34.461  27.389  36.378  1.00  0.00           H  \\nATOM   1697  HA  ASN B 127      35.227  28.171  34.006  1.00  0.00           H  \\nATOM   1698 1HB  ASN B 127      36.735  27.462  35.831  1.00  0.00           H  \\nATOM   1699 2HB  ASN B 127      36.218  25.830  35.458  1.00  0.00           H  \\nATOM   1700 1HD2 ASN B 127      37.538  24.711  34.221  1.00  0.00           H  \\nATOM   1701 2HD2 ASN B 127      38.626  25.364  33.015  1.00  0.00           H  \\nATOM   1702  N   GLN B 128      33.697  25.380  34.073  1.00  0.00           N  \\nATOM   1703  CA  GLN B 128      33.050  24.372  33.247  1.00  0.00           C  \\nATOM   1704  C   GLN B 128      31.886  24.974  32.465  1.00  0.00           C  \\nATOM   1705  O   GLN B 128      31.623  24.569  31.337  1.00  0.00           O  \\nATOM   1706  CB  GLN B 128      32.618  23.200  34.120  1.00  0.00           C  \\nATOM   1707  CG  GLN B 128      33.823  22.388  34.662  1.00  0.00           C  \\nATOM   1708  CD  GLN B 128      33.450  21.259  35.606  1.00  0.00           C  \\nATOM   1709  OE1 GLN B 128      32.773  20.322  35.185  1.00  0.00           O  \\nATOM   1710  NE2 GLN B 128      33.874  21.305  36.870  1.00  0.00           N  \\nATOM   1711  H   GLN B 128      33.566  25.282  34.917  1.00  0.00           H  \\nATOM   1712  HA  GLN B 128      33.694  24.032  32.606  1.00  0.00           H  \\nATOM   1713 1HB  GLN B 128      32.050  23.577  34.973  1.00  0.00           H  \\nATOM   1714 2HB  GLN B 128      31.976  22.557  33.557  1.00  0.00           H  \\nATOM   1715 1HG  GLN B 128      34.323  21.937  33.809  1.00  0.00           H  \\nATOM   1716 2HG  GLN B 128      34.519  23.039  35.159  1.00  0.00           H  \\nATOM   1717 1HE2 GLN B 128      33.670  20.540  37.488  1.00  0.00           H  \\nATOM   1718 2HE2 GLN B 128      34.440  22.064  37.232  1.00  0.00           H  \\nATOM   1719  N   ARG B 129      31.204  25.951  33.056  1.00  0.00           N  \\nATOM   1720  CA  ARG B 129      30.122  26.647  32.366  1.00  0.00           C  \\nATOM   1721  C   ARG B 129      30.669  27.468  31.202  1.00  0.00           C  \\nATOM   1722  O   ARG B 129      30.090  27.490  30.119  1.00  0.00           O  \\nATOM   1723  CB  ARG B 129      29.372  27.546  33.339  1.00  0.00           C  \\nATOM   1724  CG  ARG B 129      28.090  28.173  32.796  1.00  0.00           C  \\nATOM   1725  CD  ARG B 129      27.401  29.025  33.830  1.00  0.00           C  \\nATOM   1726  NE  ARG B 129      28.100  30.281  34.092  1.00  0.00           N  \\nATOM   1727  CZ  ARG B 129      27.944  31.389  33.332  1.00  0.00           C  \\nATOM   1728  NH1 ARG B 129      27.109  31.366  32.302  1.00  0.00           N  \\nATOM   1729  NH2 ARG B 129      28.611  32.495  33.617  1.00  0.00           N  \\nATOM   1730  H   ARG B 129      31.348  26.231  33.856  1.00  0.00           H  \\nATOM   1731  HA  ARG B 129      29.500  25.993  32.009  1.00  0.00           H  \\nATOM   1732 1HB  ARG B 129      29.114  26.973  34.230  1.00  0.00           H  \\nATOM   1733 2HB  ARG B 129      30.030  28.360  33.661  1.00  0.00           H  \\nATOM   1734 1HG  ARG B 129      28.338  28.806  31.939  1.00  0.00           H  \\nATOM   1735 2HG  ARG B 129      27.406  27.385  32.481  1.00  0.00           H  \\nATOM   1736 1HD  ARG B 129      26.394  29.263  33.491  1.00  0.00           H  \\nATOM   1737 2HD  ARG B 129      27.346  28.472  34.768  1.00  0.00           H  \\nATOM   1738  HE  ARG B 129      28.758  30.336  34.897  1.00  0.00           H  \\nATOM   1739 1HH1 ARG B 129      26.603  30.522  32.076  1.00  0.00           H  \\nATOM   1740 2HH1 ARG B 129      26.992  32.185  31.719  1.00  0.00           H  \\nATOM   1741 1HH2 ARG B 129      29.285  32.501  34.406  1.00  0.00           H  \\nATOM   1742 2HH2 ARG B 129      28.473  33.320  33.063  1.00  0.00           H  \\nATOM   1743  N   ILE B 130      31.784  28.148  31.438  1.00  0.00           N  \\nATOM   1744  CA  ILE B 130      32.424  28.947  30.404  1.00  0.00           C  \\nATOM   1745  C   ILE B 130      32.994  28.037  29.322  1.00  0.00           C  \\nATOM   1746  O   ILE B 130      32.967  28.375  28.140  1.00  0.00           O  \\nATOM   1747  CB  ILE B 130      33.504  29.852  30.991  1.00  0.00           C  \\nATOM   1748  CG1 ILE B 130      32.833  30.900  31.856  1.00  0.00           C  \\nATOM   1749  CG2 ILE B 130      34.307  30.512  29.883  1.00  0.00           C  \\nATOM   1750  CD1 ILE B 130      33.778  31.674  32.697  1.00  0.00           C  \\nATOM   1751  H   ILE B 130      32.193  28.162  32.195  1.00  0.00           H  \\nATOM   1752  HA  ILE B 130      31.762  29.525  29.994  1.00  0.00           H  \\nATOM   1753  HB  ILE B 130      34.169  29.262  31.629  1.00  0.00           H  \\nATOM   1754 1HG1 ILE B 130      32.302  31.581  31.209  1.00  0.00           H  \\nATOM   1755 2HG1 ILE B 130      32.110  30.412  32.515  1.00  0.00           H  \\nATOM   1756 1HG2 ILE B 130      35.051  31.145  30.338  1.00  0.00           H  \\nATOM   1757 2HG2 ILE B 130      34.790  29.750  29.277  1.00  0.00           H  \\nATOM   1758 3HG2 ILE B 130      33.648  31.112  29.255  1.00  0.00           H  \\nATOM   1759 1HD1 ILE B 130      33.224  32.404  33.290  1.00  0.00           H  \\nATOM   1760 2HD1 ILE B 130      34.286  30.979  33.364  1.00  0.00           H  \\nATOM   1761 3HD1 ILE B 130      34.503  32.186  32.068  1.00  0.00           H  \\nATOM   1762  N   SER B 131      33.500  26.876  29.725  1.00  0.00           N  \\nATOM   1763  CA  SER B 131      34.026  25.916  28.761  1.00  0.00           C  \\nATOM   1764  C   SER B 131      32.922  25.417  27.839  1.00  0.00           C  \\nATOM   1765  O   SER B 131      33.143  25.226  26.643  1.00  0.00           O  \\nATOM   1766  CB  SER B 131      34.628  24.751  29.532  1.00  0.00           C  \\nATOM   1767  OG  SER B 131      35.132  23.741  28.702  1.00  0.00           O  \\nATOM   1768  H   SER B 131      33.550  26.621  30.544  1.00  0.00           H  \\nATOM   1769  HA  SER B 131      34.700  26.351  28.216  1.00  0.00           H  \\nATOM   1770 1HB  SER B 131      35.414  25.115  30.185  1.00  0.00           H  \\nATOM   1771 2HB  SER B 131      33.863  24.331  30.166  1.00  0.00           H  \\nATOM   1772  HG  SER B 131      35.365  23.026  29.298  1.00  0.00           H  \\nATOM   1773  N   ALA B 132      31.734  25.206  28.397  1.00  0.00           N  \\nATOM   1774  CA  ALA B 132      30.608  24.713  27.611  1.00  0.00           C  \\nATOM   1775  C   ALA B 132      30.191  25.752  26.580  1.00  0.00           C  \\nATOM   1776  O   ALA B 132      29.869  25.412  25.445  1.00  0.00           O  \\nATOM   1777  CB  ALA B 132      29.448  24.376  28.521  1.00  0.00           C  \\nATOM   1778  H   ALA B 132      31.554  25.340  29.228  1.00  0.00           H  \\nATOM   1779  HA  ALA B 132      30.878  23.910  27.139  1.00  0.00           H  \\nATOM   1780 1HB  ALA B 132      29.135  25.273  29.057  1.00  0.00           H  \\nATOM   1781 2HB  ALA B 132      28.616  23.994  27.934  1.00  0.00           H  \\nATOM   1782 3HB  ALA B 132      29.772  23.621  29.233  1.00  0.00           H  \\nATOM   1783  N   GLU B 133      30.199  27.020  26.976  1.00  0.00           N  \\nATOM   1784  CA  GLU B 133      29.883  28.094  26.045  1.00  0.00           C  \\nATOM   1785  C   GLU B 133      30.924  28.139  24.931  1.00  0.00           C  \\nATOM   1786  O   GLU B 133      30.588  28.339  23.764  1.00  0.00           O  \\nATOM   1787  CB  GLU B 133      29.786  29.454  26.753  1.00  0.00           C  \\nATOM   1788  CG  GLU B 133      28.541  29.622  27.659  1.00  0.00           C  \\nATOM   1789  CD  GLU B 133      28.468  30.990  28.344  1.00  0.00           C  \\nATOM   1790  OE1 GLU B 133      29.429  31.717  28.258  1.00  0.00           O  \\nATOM   1791  OE2 GLU B 133      27.456  31.304  28.937  1.00  0.00           O  \\nATOM   1792  H   GLU B 133      30.383  27.283  27.774  1.00  0.00           H  \\nATOM   1793  HA  GLU B 133      29.018  27.920  25.641  1.00  0.00           H  \\nATOM   1794 1HB  GLU B 133      30.671  29.597  27.376  1.00  0.00           H  \\nATOM   1795 2HB  GLU B 133      29.778  30.251  26.010  1.00  0.00           H  \\nATOM   1796 1HG  GLU B 133      27.646  29.479  27.055  1.00  0.00           H  \\nATOM   1797 2HG  GLU B 133      28.560  28.842  28.420  1.00  0.00           H  \\nATOM   1798  N   ASN B 134      32.180  27.941  25.260  1.00  0.00           N  \\nATOM   1799  CA  ASN B 134      33.232  27.883  24.260  1.00  0.00           C  \\nATOM   1800  C   ASN B 134      33.014  26.704  23.305  1.00  0.00           C  \\nATOM   1801  O   ASN B 134      33.284  26.782  22.141  1.00  0.00           O  \\nATOM   1802  CB  ASN B 134      34.623  27.836  24.874  1.00  0.00           C  \\nATOM   1803  CG  ASN B 134      35.708  27.983  23.795  1.00  0.00           C  \\nATOM   1804  OD1 ASN B 134      35.798  28.989  23.093  1.00  0.00           O  \\nATOM   1805  ND2 ASN B 134      36.511  26.988  23.645  1.00  0.00           N  \\nATOM   1806  H   ASN B 134      32.454  27.837  26.060  1.00  0.00           H  \\nATOM   1807  HA  ASN B 134      33.192  28.697  23.729  1.00  0.00           H  \\nATOM   1808 1HB  ASN B 134      34.732  28.632  25.614  1.00  0.00           H  \\nATOM   1809 2HB  ASN B 134      34.761  26.887  25.391  1.00  0.00           H  \\nATOM   1810 1HD2 ASN B 134      37.216  27.026  22.940  1.00  0.00           H  \\nATOM   1811 2HD2 ASN B 134      36.408  26.174  24.235  1.00  0.00           H  \\nATOM   1812  N   GLN B 135      32.466  25.631  23.811  1.00  0.00           N  \\nATOM   1813  CA  GLN B 135      32.158  24.505  22.980  1.00  0.00           C  \\nATOM   1814  C   GLN B 135      30.944  24.761  22.111  1.00  0.00           C  \\nATOM   1815  O   GLN B 135      30.899  24.317  21.012  1.00  0.00           O  \\nATOM   1816  CB  GLN B 135      31.964  23.256  23.817  1.00  0.00           C  \\nATOM   1817  CG  GLN B 135      33.238  22.762  24.445  1.00  0.00           C  \\nATOM   1818  CD  GLN B 135      33.017  21.623  25.393  1.00  0.00           C  \\nATOM   1819  OE1 GLN B 135      32.181  20.741  25.163  1.00  0.00           O  \\nATOM   1820  NE2 GLN B 135      33.780  21.623  26.465  1.00  0.00           N  \\nATOM   1821  H   GLN B 135      32.253  25.527  24.630  1.00  0.00           H  \\nATOM   1822  HA  GLN B 135      32.916  24.341  22.397  1.00  0.00           H  \\nATOM   1823 1HB  GLN B 135      31.243  23.448  24.604  1.00  0.00           H  \\nATOM   1824 2HB  GLN B 135      31.567  22.489  23.194  1.00  0.00           H  \\nATOM   1825 1HG  GLN B 135      33.899  22.416  23.652  1.00  0.00           H  \\nATOM   1826 2HG  GLN B 135      33.717  23.573  24.982  1.00  0.00           H  \\nATOM   1827 1HE2 GLN B 135      33.721  20.880  27.137  1.00  0.00           H  \\nATOM   1828 2HE2 GLN B 135      34.449  22.362  26.592  1.00  0.00           H  \\nATOM   1829  N   LYS B 136      29.964  25.467  22.630  1.00  0.00           N  \\nATOM   1830  CA  LYS B 136      28.813  25.890  21.838  1.00  0.00           C  \\nATOM   1831  C   LYS B 136      29.265  26.658  20.603  1.00  0.00           C  \\nATOM   1832  O   LYS B 136      28.810  26.394  19.491  1.00  0.00           O  \\nATOM   1833  CB  LYS B 136      27.820  26.698  22.674  1.00  0.00           C  \\nATOM   1834  CG  LYS B 136      26.992  25.834  23.644  1.00  0.00           C  \\nATOM   1835  CD  LYS B 136      26.125  26.675  24.580  1.00  0.00           C  \\nATOM   1836  CE  LYS B 136      25.021  27.403  23.824  1.00  0.00           C  \\nATOM   1837  NZ  LYS B 136      24.136  28.175  24.743  1.00  0.00           N  \\nATOM   1838  H   LYS B 136      29.942  25.728  23.449  1.00  0.00           H  \\nATOM   1839  HA  LYS B 136      28.320  25.107  21.547  1.00  0.00           H  \\nATOM   1840 1HB  LYS B 136      28.347  27.441  23.253  1.00  0.00           H  \\nATOM   1841 2HB  LYS B 136      27.134  27.224  22.014  1.00  0.00           H  \\nATOM   1842 1HG  LYS B 136      26.350  25.168  23.072  1.00  0.00           H  \\nATOM   1843 2HG  LYS B 136      27.667  25.227  24.240  1.00  0.00           H  \\nATOM   1844 1HD  LYS B 136      25.678  26.027  25.335  1.00  0.00           H  \\nATOM   1845 2HD  LYS B 136      26.747  27.409  25.086  1.00  0.00           H  \\nATOM   1846 1HE  LYS B 136      25.473  28.089  23.110  1.00  0.00           H  \\nATOM   1847 2HE  LYS B 136      24.418  26.674  23.283  1.00  0.00           H  \\nATOM   1848 1HZ  LYS B 136      23.419  28.643  24.206  1.00  0.00           H  \\nATOM   1849 2HZ  LYS B 136      23.703  27.544  25.404  1.00  0.00           H  \\nATOM   1850 3HZ  LYS B 136      24.683  28.862  25.244  1.00  0.00           H  \\nATOM   1851  N   ALA B 137      30.166  27.611  20.813  1.00  0.00           N  \\nATOM   1852  CA  ALA B 137      30.680  28.437  19.731  1.00  0.00           C  \\nATOM   1853  C   ALA B 137      31.405  27.587  18.694  1.00  0.00           C  \\nATOM   1854  O   ALA B 137      31.184  27.744  17.495  1.00  0.00           O  \\nATOM   1855  CB  ALA B 137      31.642  29.424  20.343  1.00  0.00           C  \\nATOM   1856  H   ALA B 137      30.498  27.800  21.584  1.00  0.00           H  \\nATOM   1857  HA  ALA B 137      29.938  28.880  19.291  1.00  0.00           H  \\nATOM   1858 1HB  ALA B 137      32.021  30.085  19.597  1.00  0.00           H  \\nATOM   1859 2HB  ALA B 137      31.117  29.962  21.097  1.00  0.00           H  \\nATOM   1860 3HB  ALA B 137      32.467  28.890  20.815  1.00  0.00           H  \\nATOM   1861  N   LEU B 138      32.269  26.689  19.163  1.00  0.00           N  \\nATOM   1862  CA  LEU B 138      33.019  25.802  18.277  1.00  0.00           C  \\nATOM   1863  C   LEU B 138      32.081  25.013  17.374  1.00  0.00           C  \\nATOM   1864  O   LEU B 138      32.362  24.818  16.192  1.00  0.00           O  \\nATOM   1865  CB  LEU B 138      33.941  24.857  19.074  1.00  0.00           C  \\nATOM   1866  CG  LEU B 138      35.199  25.527  19.740  1.00  0.00           C  \\nATOM   1867  CD1 LEU B 138      35.921  24.512  20.649  1.00  0.00           C  \\nATOM   1868  CD2 LEU B 138      36.146  26.030  18.647  1.00  0.00           C  \\nATOM   1869  H   LEU B 138      32.440  26.572  19.998  1.00  0.00           H  \\nATOM   1870  HA  LEU B 138      33.600  26.336  17.713  1.00  0.00           H  \\nATOM   1871 1HB  LEU B 138      33.350  24.401  19.866  1.00  0.00           H  \\nATOM   1872 2HB  LEU B 138      34.290  24.069  18.410  1.00  0.00           H  \\nATOM   1873  HG  LEU B 138      34.883  26.370  20.348  1.00  0.00           H  \\nATOM   1874 1HD1 LEU B 138      36.791  24.986  21.111  1.00  0.00           H  \\nATOM   1875 2HD1 LEU B 138      35.239  24.174  21.428  1.00  0.00           H  \\nATOM   1876 3HD1 LEU B 138      36.243  23.663  20.055  1.00  0.00           H  \\nATOM   1877 1HD2 LEU B 138      37.010  26.502  19.110  1.00  0.00           H  \\nATOM   1878 2HD2 LEU B 138      36.481  25.192  18.031  1.00  0.00           H  \\nATOM   1879 3HD2 LEU B 138      35.627  26.754  18.024  1.00  0.00           H  \\nATOM   1880  N   ALA B 139      30.965  24.566  17.940  1.00  0.00           N  \\nATOM   1881  CA  ALA B 139      29.960  23.830  17.185  1.00  0.00           C  \\nATOM   1882  C   ALA B 139      29.252  24.748  16.192  1.00  0.00           C  \\nATOM   1883  O   ALA B 139      28.191  24.410  15.667  1.00  0.00           O  \\nATOM   1884  OXT ALA B 139      29.729  25.814  15.915  1.00  0.00           O  \\nATOM   1885  CB  ALA B 139      28.948  23.220  18.146  1.00  0.00           C  \\nATOM   1886  H   ALA B 139      30.765  24.677  18.769  1.00  0.00           H  \\nATOM   1887  HA  ALA B 139      30.395  23.122  16.683  1.00  0.00           H  \\nATOM   1888 1HB  ALA B 139      28.205  22.665  17.584  1.00  0.00           H  \\nATOM   1889 2HB  ALA B 139      29.451  22.560  18.845  1.00  0.00           H  \\nATOM   1890 3HB  ALA B 139      28.458  24.016  18.702  1.00  0.00           H  \\nTER                                                                             \\nATOM   1892  N   THR C 140      27.512  14.029  12.268  1.00  0.00           N  \\nATOM   1893  CA  THR C 140      28.049  14.350  13.620  1.00  0.00           C  \\nATOM   1894  C   THR C 140      27.258  15.481  14.277  1.00  0.00           C  \\nATOM   1895  O   THR C 140      27.635  15.974  15.340  1.00  0.00           O  \\nATOM   1896  CB  THR C 140      29.530  14.766  13.543  1.00  0.00           C  \\nATOM   1897  OG1 THR C 140      30.116  14.723  14.854  1.00  0.00           O  \\nATOM   1898  CG2 THR C 140      29.638  16.187  13.002  1.00  0.00           C  \\nATOM   1899 1H   THR C 140      28.048  13.288  11.863  1.00  0.00           H  \\nATOM   1900 2H   THR C 140      26.557  13.745  12.347  1.00  0.00           H  \\nATOM   1901 3H   THR C 140      27.571  14.839  11.684  1.00  0.00           H  \\nATOM   1902  HA  THR C 140      27.974  13.564  14.184  1.00  0.00           H  \\nATOM   1903  HB  THR C 140      30.064  14.078  12.889  1.00  0.00           H  \\nATOM   1904  HG1 THR C 140      30.096  15.607  15.239  1.00  0.00           H  \\nATOM   1905 1HG2 THR C 140      30.686  16.478  12.951  1.00  0.00           H  \\nATOM   1906 2HG2 THR C 140      29.197  16.234  12.007  1.00  0.00           H  \\nATOM   1907 3HG2 THR C 140      29.106  16.866  13.664  1.00  0.00           H  \\nATOM   1908  N   LYS C 141      26.158  15.882  13.645  1.00  0.00           N  \\nATOM   1909  CA  LYS C 141      25.350  16.991  14.142  1.00  0.00           C  \\nATOM   1910  C   LYS C 141      24.547  16.563  15.360  1.00  0.00           C  \\nATOM   1911  O   LYS C 141      24.509  17.274  16.363  1.00  0.00           O  \\nATOM   1912  CB  LYS C 141      24.435  17.544  13.051  1.00  0.00           C  \\nATOM   1913  CG  LYS C 141      24.895  18.897  12.440  1.00  0.00           C  \\nATOM   1914  CD  LYS C 141      26.339  18.862  11.924  1.00  0.00           C  \\nATOM   1915  CE  LYS C 141      27.315  19.561  12.892  1.00  0.00           C  \\nATOM   1916  NZ  LYS C 141      27.163  21.065  12.876  1.00  0.00           N  \\nATOM   1917  H   LYS C 141      25.857  15.527  12.922  1.00  0.00           H  \\nATOM   1918  HA  LYS C 141      25.936  17.716  14.407  1.00  0.00           H  \\nATOM   1919 1HB  LYS C 141      24.351  16.822  12.241  1.00  0.00           H  \\nATOM   1920 2HB  LYS C 141      23.435  17.691  13.462  1.00  0.00           H  \\nATOM   1921 1HG  LYS C 141      24.234  19.158  11.613  1.00  0.00           H  \\nATOM   1922 2HG  LYS C 141      24.813  19.674  13.200  1.00  0.00           H  \\nATOM   1923 1HD  LYS C 141      26.654  17.829  11.793  1.00  0.00           H  \\nATOM   1924 2HD  LYS C 141      26.388  19.362  10.956  1.00  0.00           H  \\nATOM   1925 1HE  LYS C 141      27.132  19.198  13.904  1.00  0.00           H  \\nATOM   1926 2HE  LYS C 141      28.336  19.308  12.607  1.00  0.00           H  \\nATOM   1927 1HZ  LYS C 141      27.826  21.473  13.523  1.00  0.00           H  \\nATOM   1928 2HZ  LYS C 141      26.225  21.320  13.152  1.00  0.00           H  \\nATOM   1929 3HZ  LYS C 141      27.344  21.417  11.947  1.00  0.00           H  \\nATOM   1930  N   TYR C 142      23.902  15.403  15.271  1.00  0.00           N  \\nATOM   1931  CA  TYR C 142      23.134  14.886  16.395  1.00  0.00           C  \\nATOM   1932  C   TYR C 142      24.038  14.667  17.598  1.00  0.00           C  \\nATOM   1933  O   TYR C 142      23.651  14.944  18.729  1.00  0.00           O  \\nATOM   1934  CB  TYR C 142      22.445  13.573  16.035  1.00  0.00           C  \\nATOM   1935  CG  TYR C 142      21.591  13.023  17.159  1.00  0.00           C  \\nATOM   1936  CD1 TYR C 142      20.331  13.554  17.396  1.00  0.00           C  \\nATOM   1937  CD2 TYR C 142      22.070  11.992  17.956  1.00  0.00           C  \\nATOM   1938  CE1 TYR C 142      19.551  13.052  18.419  1.00  0.00           C  \\nATOM   1939  CE2 TYR C 142      21.290  11.493  18.981  1.00  0.00           C  \\nATOM   1940  CZ  TYR C 142      20.034  12.019  19.212  1.00  0.00           C  \\nATOM   1941  OH  TYR C 142      19.252  11.519  20.228  1.00  0.00           O  \\nATOM   1942  H   TYR C 142      23.895  14.899  14.574  1.00  0.00           H  \\nATOM   1943  HA  TYR C 142      22.457  15.536  16.641  1.00  0.00           H  \\nATOM   1944 1HB  TYR C 142      21.810  13.720  15.161  1.00  0.00           H  \\nATOM   1945 2HB  TYR C 142      23.194  12.826  15.779  1.00  0.00           H  \\nATOM   1946  HD1 TYR C 142      19.955  14.364  16.771  1.00  0.00           H  \\nATOM   1947  HD2 TYR C 142      23.062  11.579  17.774  1.00  0.00           H  \\nATOM   1948  HE1 TYR C 142      18.562  13.469  18.604  1.00  0.00           H  \\nATOM   1949  HE2 TYR C 142      21.664  10.682  19.607  1.00  0.00           H  \\nATOM   1950  HH  TYR C 142      18.429  12.012  20.269  1.00  0.00           H  \\nATOM   1951  N   LYS C 143      25.242  14.166  17.344  1.00  0.00           N  \\nATOM   1952  CA  LYS C 143      26.176  13.844  18.414  1.00  0.00           C  \\nATOM   1953  C   LYS C 143      26.519  15.091  19.227  1.00  0.00           C  \\nATOM   1954  O   LYS C 143      26.355  15.108  20.443  1.00  0.00           O  \\nATOM   1955  CB  LYS C 143      27.395  13.132  17.820  1.00  0.00           C  \\nATOM   1956  CG  LYS C 143      28.465  12.712  18.805  1.00  0.00           C  \\nATOM   1957  CD  LYS C 143      29.434  11.685  18.160  1.00  0.00           C  \\nATOM   1958  CE  LYS C 143      30.161  12.225  16.936  1.00  0.00           C  \\nATOM   1959  NZ  LYS C 143      31.145  11.238  16.393  1.00  0.00           N  \\nATOM   1960  H   LYS C 143      25.543  14.004  16.555  1.00  0.00           H  \\nATOM   1961  HA  LYS C 143      25.762  13.201  19.011  1.00  0.00           H  \\nATOM   1962 1HB  LYS C 143      27.068  12.243  17.283  1.00  0.00           H  \\nATOM   1963 2HB  LYS C 143      27.873  13.794  17.092  1.00  0.00           H  \\nATOM   1964 1HG  LYS C 143      29.016  13.591  19.124  1.00  0.00           H  \\nATOM   1965 2HG  LYS C 143      27.998  12.263  19.680  1.00  0.00           H  \\nATOM   1966 1HD  LYS C 143      30.180  11.390  18.900  1.00  0.00           H  \\nATOM   1967 2HD  LYS C 143      28.873  10.798  17.867  1.00  0.00           H  \\nATOM   1968 1HE  LYS C 143      29.430  12.456  16.165  1.00  0.00           H  \\nATOM   1969 2HE  LYS C 143      30.683  13.121  17.194  1.00  0.00           H  \\nATOM   1970 1HZ  LYS C 143      31.836  11.027  17.096  1.00  0.00           H  \\nATOM   1971 2HZ  LYS C 143      30.664  10.390  16.127  1.00  0.00           H  \\nATOM   1972 3HZ  LYS C 143      31.600  11.637  15.583  1.00  0.00           H  \\nATOM   1973  N   ILE C 144      26.969  16.140  18.545  1.00  0.00           N  \\nATOM   1974  CA  ILE C 144      27.376  17.376  19.207  1.00  0.00           C  \\nATOM   1975  C   ILE C 144      26.234  17.997  20.009  1.00  0.00           C  \\nATOM   1976  O   ILE C 144      26.404  18.333  21.180  1.00  0.00           O  \\nATOM   1977  CB  ILE C 144      27.923  18.383  18.181  1.00  0.00           C  \\nATOM   1978  CG1 ILE C 144      29.240  17.858  17.614  1.00  0.00           C  \\nATOM   1979  CG2 ILE C 144      28.139  19.733  18.828  1.00  0.00           C  \\nATOM   1980  CD1 ILE C 144      29.731  18.587  16.397  1.00  0.00           C  \\nATOM   1981  H   ILE C 144      27.050  16.161  17.689  1.00  0.00           H  \\nATOM   1982  HA  ILE C 144      28.096  17.175  19.825  1.00  0.00           H  \\nATOM   1983  HB  ILE C 144      27.214  18.474  17.352  1.00  0.00           H  \\nATOM   1984 1HG1 ILE C 144      30.006  17.930  18.389  1.00  0.00           H  \\nATOM   1985 2HG1 ILE C 144      29.104  16.814  17.348  1.00  0.00           H  \\nATOM   1986 1HG2 ILE C 144      28.523  20.406  18.081  1.00  0.00           H  \\nATOM   1987 2HG2 ILE C 144      27.196  20.110  19.214  1.00  0.00           H  \\nATOM   1988 3HG2 ILE C 144      28.855  19.645  19.643  1.00  0.00           H  \\nATOM   1989 1HD1 ILE C 144      29.887  19.636  16.638  1.00  0.00           H  \\nATOM   1990 2HD1 ILE C 144      30.673  18.142  16.063  1.00  0.00           H  \\nATOM   1991 3HD1 ILE C 144      28.987  18.501  15.609  1.00  0.00           H  \\nATOM   1992  N   LYS C 145      25.071  18.143  19.385  1.00  0.00           N  \\nATOM   1993  CA  LYS C 145      23.953  18.817  20.035  1.00  0.00           C  \\nATOM   1994  C   LYS C 145      23.334  17.972  21.148  1.00  0.00           C  \\nATOM   1995  O   LYS C 145      22.797  18.512  22.116  1.00  0.00           O  \\nATOM   1996  CB  LYS C 145      22.897  19.147  18.987  1.00  0.00           C  \\nATOM   1997  CG  LYS C 145      23.328  20.217  17.996  1.00  0.00           C  \\nATOM   1998  CD  LYS C 145      22.317  20.370  16.867  1.00  0.00           C  \\nATOM   1999  CE  LYS C 145      21.022  21.007  17.355  1.00  0.00           C  \\nATOM   2000  NZ  LYS C 145      20.066  21.248  16.238  1.00  0.00           N  \\nATOM   2001  H   LYS C 145      24.904  17.864  18.589  1.00  0.00           H  \\nATOM   2002  HA  LYS C 145      24.277  19.638  20.436  1.00  0.00           H  \\nATOM   2003 1HB  LYS C 145      22.666  18.240  18.418  1.00  0.00           H  \\nATOM   2004 2HB  LYS C 145      21.982  19.473  19.476  1.00  0.00           H  \\nATOM   2005 1HG  LYS C 145      23.447  21.169  18.511  1.00  0.00           H  \\nATOM   2006 2HG  LYS C 145      24.290  19.928  17.568  1.00  0.00           H  \\nATOM   2007 1HD  LYS C 145      22.745  20.991  16.079  1.00  0.00           H  \\nATOM   2008 2HD  LYS C 145      22.099  19.385  16.448  1.00  0.00           H  \\nATOM   2009 1HE  LYS C 145      20.554  20.348  18.084  1.00  0.00           H  \\nATOM   2010 2HE  LYS C 145      21.251  21.959  17.834  1.00  0.00           H  \\nATOM   2011 1HZ  LYS C 145      20.487  21.868  15.560  1.00  0.00           H  \\nATOM   2012 2HZ  LYS C 145      19.838  20.369  15.793  1.00  0.00           H  \\nATOM   2013 3HZ  LYS C 145      19.223  21.669  16.601  1.00  0.00           H  \\nATOM   2014  N   GLU C 146      23.417  16.652  21.020  1.00  0.00           N  \\nATOM   2015  CA  GLU C 146      22.892  15.766  22.051  1.00  0.00           C  \\nATOM   2016  C   GLU C 146      23.785  15.817  23.285  1.00  0.00           C  \\nATOM   2017  O   GLU C 146      23.301  15.781  24.415  1.00  0.00           O  \\nATOM   2018  CB  GLU C 146      22.780  14.312  21.545  1.00  0.00           C  \\nATOM   2019  CG  GLU C 146      21.986  13.311  22.444  1.00  0.00           C  \\nATOM   2020  CD  GLU C 146      22.757  12.619  23.587  1.00  0.00           C  \\nATOM   2021  OE1 GLU C 146      23.965  12.450  23.512  1.00  0.00           O  \\nATOM   2022  OE2 GLU C 146      22.095  12.224  24.522  1.00  0.00           O  \\nATOM   2023  H   GLU C 146      23.771  16.247  20.350  1.00  0.00           H  \\nATOM   2024  HA  GLU C 146      22.006  16.065  22.306  1.00  0.00           H  \\nATOM   2025 1HB  GLU C 146      22.256  14.313  20.595  1.00  0.00           H  \\nATOM   2026 2HB  GLU C 146      23.779  13.907  21.366  1.00  0.00           H  \\nATOM   2027 1HG  GLU C 146      21.150  13.854  22.881  1.00  0.00           H  \\nATOM   2028 2HG  GLU C 146      21.569  12.543  21.801  1.00  0.00           H  \\nATOM   2029  N   THR C 147      25.092  15.906  23.062  1.00  0.00           N  \\nATOM   2030  CA  THR C 147      26.049  15.988  24.156  1.00  0.00           C  \\nATOM   2031  C   THR C 147      25.979  17.349  24.847  1.00  0.00           C  \\nATOM   2032  O   THR C 147      26.129  17.442  26.064  1.00  0.00           O  \\nATOM   2033  CB  THR C 147      27.518  15.732  23.739  1.00  0.00           C  \\nATOM   2034  OG1 THR C 147      27.912  16.658  22.719  1.00  0.00           O  \\nATOM   2035  CG2 THR C 147      27.745  14.302  23.298  1.00  0.00           C  \\nATOM   2036  H   THR C 147      25.450  15.920  22.281  1.00  0.00           H  \\nATOM   2037  HA  THR C 147      25.836  15.307  24.814  1.00  0.00           H  \\nATOM   2038  HB  THR C 147      28.158  15.906  24.604  1.00  0.00           H  \\nATOM   2039  HG1 THR C 147      27.136  16.967  22.205  1.00  0.00           H  \\nATOM   2040 1HG2 THR C 147      28.798  14.165  23.043  1.00  0.00           H  \\nATOM   2041 2HG2 THR C 147      27.482  13.630  24.113  1.00  0.00           H  \\nATOM   2042 3HG2 THR C 147      27.135  14.072  22.441  1.00  0.00           H  \\nATOM   2043  N   LEU C 148      25.741  18.400  24.070  1.00  0.00           N  \\nATOM   2044  CA  LEU C 148      25.580  19.736  24.635  1.00  0.00           C  \\nATOM   2045  C   LEU C 148      24.374  19.792  25.569  1.00  0.00           C  \\nATOM   2046  O   LEU C 148      24.414  20.459  26.601  1.00  0.00           O  \\nATOM   2047  CB  LEU C 148      25.500  20.805  23.532  1.00  0.00           C  \\nATOM   2048  CG  LEU C 148      26.849  21.106  22.765  1.00  0.00           C  \\nATOM   2049  CD1 LEU C 148      26.568  22.016  21.569  1.00  0.00           C  \\nATOM   2050  CD2 LEU C 148      27.845  21.779  23.725  1.00  0.00           C  \\nATOM   2051  H   LEU C 148      25.667  18.368  23.214  1.00  0.00           H  \\nATOM   2052  HA  LEU C 148      26.370  19.958  25.152  1.00  0.00           H  \\nATOM   2053 1HB  LEU C 148      24.769  20.475  22.796  1.00  0.00           H  \\nATOM   2054 2HB  LEU C 148      25.148  21.736  23.973  1.00  0.00           H  \\nATOM   2055  HG  LEU C 148      27.283  20.182  22.401  1.00  0.00           H  \\nATOM   2056 1HD1 LEU C 148      27.501  22.219  21.040  1.00  0.00           H  \\nATOM   2057 2HD1 LEU C 148      25.870  21.521  20.896  1.00  0.00           H  \\nATOM   2058 3HD1 LEU C 148      26.138  22.952  21.914  1.00  0.00           H  \\nATOM   2059 1HD2 LEU C 148      28.774  21.989  23.197  1.00  0.00           H  \\nATOM   2060 2HD2 LEU C 148      27.425  22.711  24.095  1.00  0.00           H  \\nATOM   2061 3HD2 LEU C 148      28.046  21.112  24.563  1.00  0.00           H  \\nATOM   2062  N   LYS C 149      23.305  19.089  25.209  1.00  0.00           N  \\nATOM   2063  CA  LYS C 149      22.102  19.068  26.033  1.00  0.00           C  \\nATOM   2064  C   LYS C 149      22.358  18.315  27.333  1.00  0.00           C  \\nATOM   2065  O   LYS C 149      21.866  18.703  28.391  1.00  0.00           O  \\nATOM   2066  CB  LYS C 149      20.947  18.421  25.270  1.00  0.00           C  \\nATOM   2067  CG  LYS C 149      19.600  18.478  25.989  1.00  0.00           C  \\nATOM   2068  CD  LYS C 149      18.484  17.889  25.127  1.00  0.00           C  \\nATOM   2069  CE  LYS C 149      18.165  18.785  23.936  1.00  0.00           C  \\nATOM   2070  NZ  LYS C 149      16.999  18.279  23.157  1.00  0.00           N  \\nATOM   2071  H   LYS C 149      23.250  18.616  24.493  1.00  0.00           H  \\nATOM   2072  HA  LYS C 149      21.855  19.979  26.257  1.00  0.00           H  \\nATOM   2073 1HB  LYS C 149      20.842  18.904  24.297  1.00  0.00           H  \\nATOM   2074 2HB  LYS C 149      21.187  17.370  25.082  1.00  0.00           H  \\nATOM   2075 1HG  LYS C 149      19.668  17.910  26.922  1.00  0.00           H  \\nATOM   2076 2HG  LYS C 149      19.360  19.513  26.234  1.00  0.00           H  \\nATOM   2077 1HD  LYS C 149      18.795  16.909  24.759  1.00  0.00           H  \\nATOM   2078 2HD  LYS C 149      17.585  17.763  25.731  1.00  0.00           H  \\nATOM   2079 1HE  LYS C 149      17.942  19.789  24.294  1.00  0.00           H  \\nATOM   2080 2HE  LYS C 149      19.036  18.828  23.280  1.00  0.00           H  \\nATOM   2081 1HZ  LYS C 149      16.820  18.897  22.378  1.00  0.00           H  \\nATOM   2082 2HZ  LYS C 149      17.201  17.351  22.810  1.00  0.00           H  \\nATOM   2083 3HZ  LYS C 149      16.185  18.245  23.753  1.00  0.00           H  \\nATOM   2084  N   ARG C 150      23.123  17.231  27.249  1.00  0.00           N  \\nATOM   2085  CA  ARG C 150      23.506  16.481  28.438  1.00  0.00           C  \\nATOM   2086  C   ARG C 150      24.492  17.283  29.277  1.00  0.00           C  \\nATOM   2087  O   ARG C 150      24.459  17.240  30.507  1.00  0.00           O  \\nATOM   2088  CB  ARG C 150      24.156  15.166  28.052  1.00  0.00           C  \\nATOM   2089  CG  ARG C 150      23.241  14.106  27.505  1.00  0.00           C  \\nATOM   2090  CD  ARG C 150      24.021  13.032  26.841  1.00  0.00           C  \\nATOM   2091  NE  ARG C 150      24.995  12.412  27.707  1.00  0.00           N  \\nATOM   2092  CZ  ARG C 150      26.094  11.787  27.239  1.00  0.00           C  \\nATOM   2093  NH1 ARG C 150      26.292  11.705  25.933  1.00  0.00           N  \\nATOM   2094  NH2 ARG C 150      26.983  11.272  28.082  1.00  0.00           N  \\nATOM   2095  H   ARG C 150      23.432  16.910  26.514  1.00  0.00           H  \\nATOM   2096  HA  ARG C 150      22.717  16.311  28.976  1.00  0.00           H  \\nATOM   2097 1HB  ARG C 150      24.923  15.352  27.300  1.00  0.00           H  \\nATOM   2098 2HB  ARG C 150      24.649  14.741  28.918  1.00  0.00           H  \\nATOM   2099 1HG  ARG C 150      22.645  13.671  28.305  1.00  0.00           H  \\nATOM   2100 2HG  ARG C 150      22.581  14.549  26.752  1.00  0.00           H  \\nATOM   2101 1HD  ARG C 150      23.339  12.258  26.521  1.00  0.00           H  \\nATOM   2102 2HD  ARG C 150      24.538  13.440  25.970  1.00  0.00           H  \\nATOM   2103  HE  ARG C 150      24.860  12.470  28.705  1.00  0.00           H  \\nATOM   2104 1HH1 ARG C 150      25.590  12.078  25.259  1.00  0.00           H  \\nATOM   2105 2HH1 ARG C 150      27.158  11.275  25.583  1.00  0.00           H  \\nATOM   2106 1HH2 ARG C 150      26.837  11.340  29.078  1.00  0.00           H  \\nATOM   2107 2HH2 ARG C 150      27.841  10.812  27.722  1.00  0.00           H  \\nATOM   2108  N   LEU C 151      25.372  18.012  28.601  1.00  0.00           N  \\nATOM   2109  CA  LEU C 151      26.342  18.860  29.274  1.00  0.00           C  \\nATOM   2110  C   LEU C 151      25.602  19.964  30.026  1.00  0.00           C  \\nATOM   2111  O   LEU C 151      25.968  20.323  31.144  1.00  0.00           O  \\nATOM   2112  CB  LEU C 151      27.305  19.483  28.232  1.00  0.00           C  \\nATOM   2113  CG  LEU C 151      28.513  20.260  28.755  1.00  0.00           C  \\nATOM   2114  CD1 LEU C 151      29.370  19.309  29.563  1.00  0.00           C  \\nATOM   2115  CD2 LEU C 151      29.311  20.847  27.569  1.00  0.00           C  \\nATOM   2116  H   LEU C 151      25.427  18.031  27.743  1.00  0.00           H  \\nATOM   2117  HA  LEU C 151      26.841  18.332  29.917  1.00  0.00           H  \\nATOM   2118 1HB  LEU C 151      27.678  18.698  27.589  1.00  0.00           H  \\nATOM   2119 2HB  LEU C 151      26.733  20.170  27.620  1.00  0.00           H  \\nATOM   2120  HG  LEU C 151      28.173  21.063  29.415  1.00  0.00           H  \\nATOM   2121 1HD1 LEU C 151      30.232  19.836  29.972  1.00  0.00           H  \\nATOM   2122 2HD1 LEU C 151      28.776  18.900  30.359  1.00  0.00           H  \\nATOM   2123 3HD1 LEU C 151      29.712  18.501  28.926  1.00  0.00           H  \\nATOM   2124 1HD2 LEU C 151      30.180  21.392  27.942  1.00  0.00           H  \\nATOM   2125 2HD2 LEU C 151      29.649  20.037  26.916  1.00  0.00           H  \\nATOM   2126 3HD2 LEU C 151      28.674  21.524  27.001  1.00  0.00           H  \\nATOM   2127  N   GLU C 152      24.544  20.481  29.410  1.00  0.00           N  \\nATOM   2128  CA  GLU C 152      23.701  21.494  30.035  1.00  0.00           C  \\nATOM   2129  C   GLU C 152      23.030  20.945  31.288  1.00  0.00           C  \\nATOM   2130  O   GLU C 152      23.015  21.602  32.330  1.00  0.00           O  \\nATOM   2131  CB  GLU C 152      22.630  22.058  29.091  1.00  0.00           C  \\nATOM   2132  CG  GLU C 152      21.787  23.187  29.759  1.00  0.00           C  \\nATOM   2133  CD  GLU C 152      20.744  23.858  28.864  1.00  0.00           C  \\nATOM   2134  OE1 GLU C 152      20.618  23.502  27.718  1.00  0.00           O  \\nATOM   2135  OE2 GLU C 152      20.073  24.735  29.357  1.00  0.00           O  \\nATOM   2136  H   GLU C 152      24.291  20.259  28.618  1.00  0.00           H  \\nATOM   2137  HA  GLU C 152      24.252  22.250  30.294  1.00  0.00           H  \\nATOM   2138 1HB  GLU C 152      23.100  22.452  28.191  1.00  0.00           H  \\nATOM   2139 2HB  GLU C 152      21.952  21.263  28.789  1.00  0.00           H  \\nATOM   2140 1HG  GLU C 152      21.274  22.756  30.620  1.00  0.00           H  \\nATOM   2141 2HG  GLU C 152      22.472  23.947  30.132  1.00  0.00           H  \\nATOM   2142  N   ASP C 153      22.467  19.746  31.182  1.00  0.00           N  \\nATOM   2143  CA  ASP C 153      21.804  19.123  32.322  1.00  0.00           C  \\nATOM   2144  C   ASP C 153      22.816  18.853  33.429  1.00  0.00           C  \\nATOM   2145  O   ASP C 153      22.550  19.111  34.601  1.00  0.00           O  \\nATOM   2146  CB  ASP C 153      21.132  17.794  31.937  1.00  0.00           C  \\nATOM   2147  CG  ASP C 153      19.803  17.890  31.108  1.00  0.00           C  \\nATOM   2148  OD1 ASP C 153      19.195  18.954  31.022  1.00  0.00           O  \\nATOM   2149  OD2 ASP C 153      19.385  16.867  30.616  1.00  0.00           O  \\nATOM   2150  H   ASP C 153      22.454  19.274  30.463  1.00  0.00           H  \\nATOM   2151  HA  ASP C 153      21.132  19.731  32.669  1.00  0.00           H  \\nATOM   2152 1HB  ASP C 153      21.843  17.207  31.354  1.00  0.00           H  \\nATOM   2153 2HB  ASP C 153      20.927  17.229  32.844  1.00  0.00           H  \\nATOM   2154  N   SER C 154      23.985  18.350  33.047  1.00  0.00           N  \\nATOM   2155  CA  SER C 154      25.066  18.115  33.994  1.00  0.00           C  \\nATOM   2156  C   SER C 154      25.452  19.391  34.742  1.00  0.00           C  \\nATOM   2157  O   SER C 154      25.744  19.357  35.936  1.00  0.00           O  \\nATOM   2158  CB  SER C 154      26.257  17.521  33.296  1.00  0.00           C  \\nATOM   2159  OG  SER C 154      27.255  17.220  34.219  1.00  0.00           O  \\nATOM   2160  H   SER C 154      24.177  18.134  32.237  1.00  0.00           H  \\nATOM   2161  HA  SER C 154      24.776  17.460  34.648  1.00  0.00           H  \\nATOM   2162 1HB  SER C 154      25.952  16.622  32.772  1.00  0.00           H  \\nATOM   2163 2HB  SER C 154      26.633  18.220  32.556  1.00  0.00           H  \\nATOM   2164  HG  SER C 154      26.875  16.565  34.806  1.00  0.00           H  \\nATOM   2165  N   LEU C 155      25.456  20.516  34.037  1.00  0.00           N  \\nATOM   2166  CA  LEU C 155      25.805  21.787  34.656  1.00  0.00           C  \\nATOM   2167  C   LEU C 155      24.724  22.225  35.635  1.00  0.00           C  \\nATOM   2168  O   LEU C 155      25.029  22.751  36.704  1.00  0.00           O  \\nATOM   2169  CB  LEU C 155      26.092  22.861  33.597  1.00  0.00           C  \\nATOM   2170  CG  LEU C 155      27.435  22.690  32.813  1.00  0.00           C  \\nATOM   2171  CD1 LEU C 155      27.478  23.634  31.636  1.00  0.00           C  \\nATOM   2172  CD2 LEU C 155      28.601  23.010  33.749  1.00  0.00           C  \\nATOM   2173  H   LEU C 155      25.262  20.570  33.201  1.00  0.00           H  \\nATOM   2174  HA  LEU C 155      26.632  21.679  35.151  1.00  0.00           H  \\nATOM   2175 1HB  LEU C 155      25.282  22.845  32.868  1.00  0.00           H  \\nATOM   2176 2HB  LEU C 155      26.105  23.832  34.080  1.00  0.00           H  \\nATOM   2177  HG  LEU C 155      27.529  21.668  32.452  1.00  0.00           H  \\nATOM   2178 1HD1 LEU C 155      28.421  23.501  31.121  1.00  0.00           H  \\nATOM   2179 2HD1 LEU C 155      26.654  23.405  30.959  1.00  0.00           H  \\nATOM   2180 3HD1 LEU C 155      27.392  24.659  31.983  1.00  0.00           H  \\nATOM   2181 1HD2 LEU C 155      29.536  22.891  33.206  1.00  0.00           H  \\nATOM   2182 2HD2 LEU C 155      28.519  24.035  34.108  1.00  0.00           H  \\nATOM   2183 3HD2 LEU C 155      28.582  22.336  34.582  1.00  0.00           H  \\nATOM   2184  N   ARG C 156      23.464  22.005  35.274  1.00  0.00           N  \\nATOM   2185  CA  ARG C 156      22.361  22.335  36.166  1.00  0.00           C  \\nATOM   2186  C   ARG C 156      22.489  21.544  37.463  1.00  0.00           C  \\nATOM   2187  O   ARG C 156      22.236  22.068  38.546  1.00  0.00           O  \\nATOM   2188  CB  ARG C 156      21.025  22.069  35.485  1.00  0.00           C  \\nATOM   2189  CG  ARG C 156      19.795  22.546  36.253  1.00  0.00           C  \\nATOM   2190  CD  ARG C 156      18.547  22.499  35.409  1.00  0.00           C  \\nATOM   2191  NE  ARG C 156      18.165  21.145  35.026  1.00  0.00           N  \\nATOM   2192  CZ  ARG C 156      18.302  20.615  33.790  1.00  0.00           C  \\nATOM   2193  NH1 ARG C 156      18.838  21.290  32.804  1.00  0.00           N  \\nATOM   2194  NH2 ARG C 156      17.904  19.391  33.541  1.00  0.00           N  \\nATOM   2195  H   ARG C 156      23.223  21.668  34.520  1.00  0.00           H  \\nATOM   2196  HA  ARG C 156      22.401  23.280  36.384  1.00  0.00           H  \\nATOM   2197 1HB  ARG C 156      21.021  22.541  34.502  1.00  0.00           H  \\nATOM   2198 2HB  ARG C 156      20.914  20.997  35.325  1.00  0.00           H  \\nATOM   2199 1HG  ARG C 156      19.646  21.903  37.121  1.00  0.00           H  \\nATOM   2200 2HG  ARG C 156      19.950  23.574  36.585  1.00  0.00           H  \\nATOM   2201 1HD  ARG C 156      17.720  22.939  35.965  1.00  0.00           H  \\nATOM   2202 2HD  ARG C 156      18.713  23.073  34.497  1.00  0.00           H  \\nATOM   2203  HE  ARG C 156      17.748  20.561  35.736  1.00  0.00           H  \\nATOM   2204 1HH1 ARG C 156      19.168  22.233  32.938  1.00  0.00           H  \\nATOM   2205 2HH1 ARG C 156      18.946  20.804  31.897  1.00  0.00           H  \\nATOM   2206 1HH2 ARG C 156      17.490  18.822  34.259  1.00  0.00           H  \\nATOM   2207 2HH2 ARG C 156      18.038  19.038  32.582  1.00  0.00           H  \\nATOM   2208  N   GLU C 157      22.900  20.285  37.350  1.00  0.00           N  \\nATOM   2209  CA  GLU C 157      23.081  19.437  38.521  1.00  0.00           C  \\nATOM   2210  C   GLU C 157      24.189  19.979  39.419  1.00  0.00           C  \\nATOM   2211  O   GLU C 157      24.058  19.984  40.639  1.00  0.00           O  \\nATOM   2212  CB  GLU C 157      23.369  17.989  38.114  1.00  0.00           C  \\nATOM   2213  CG  GLU C 157      23.654  17.018  39.278  1.00  0.00           C  \\nATOM   2214  CD  GLU C 157      22.529  16.878  40.283  1.00  0.00           C  \\nATOM   2215  OE1 GLU C 157      21.440  17.347  40.044  1.00  0.00           O  \\nATOM   2216  OE2 GLU C 157      22.766  16.273  41.305  1.00  0.00           O  \\nATOM   2217  H   GLU C 157      23.081  19.898  36.603  1.00  0.00           H  \\nATOM   2218  HA  GLU C 157      22.258  19.425  39.034  1.00  0.00           H  \\nATOM   2219 1HB  GLU C 157      22.525  17.600  37.544  1.00  0.00           H  \\nATOM   2220 2HB  GLU C 157      24.234  17.969  37.458  1.00  0.00           H  \\nATOM   2221 1HG  GLU C 157      23.875  16.037  38.862  1.00  0.00           H  \\nATOM   2222 2HG  GLU C 157      24.549  17.366  39.797  1.00  0.00           H  \\nATOM   2223  N   LEU C 158      25.277  20.439  38.809  1.00  0.00           N  \\nATOM   2224  CA  LEU C 158      26.393  21.002  39.561  1.00  0.00           C  \\nATOM   2225  C   LEU C 158      25.968  22.231  40.356  1.00  0.00           C  \\nATOM   2226  O   LEU C 158      26.339  22.383  41.519  1.00  0.00           O  \\nATOM   2227  CB  LEU C 158      27.574  21.366  38.649  1.00  0.00           C  \\nATOM   2228  CG  LEU C 158      28.418  20.208  38.210  1.00  0.00           C  \\nATOM   2229  CD1 LEU C 158      29.398  20.598  37.099  1.00  0.00           C  \\nATOM   2230  CD2 LEU C 158      29.190  19.799  39.434  1.00  0.00           C  \\nATOM   2231  H   LEU C 158      25.394  20.436  37.957  1.00  0.00           H  \\nATOM   2232  HA  LEU C 158      26.719  20.338  40.189  1.00  0.00           H  \\nATOM   2233 1HB  LEU C 158      27.186  21.850  37.758  1.00  0.00           H  \\nATOM   2234 2HB  LEU C 158      28.201  22.069  39.172  1.00  0.00           H  \\nATOM   2235  HG  LEU C 158      27.787  19.387  37.852  1.00  0.00           H  \\nATOM   2236 1HD1 LEU C 158      30.011  19.738  36.840  1.00  0.00           H  \\nATOM   2237 2HD1 LEU C 158      28.836  20.904  36.236  1.00  0.00           H  \\nATOM   2238 3HD1 LEU C 158      30.040  21.407  37.422  1.00  0.00           H  \\nATOM   2239 1HD2 LEU C 158      28.497  19.523  40.224  1.00  0.00           H  \\nATOM   2240 2HD2 LEU C 158      29.805  18.987  39.200  1.00  0.00           H  \\nATOM   2241 3HD2 LEU C 158      29.806  20.623  39.773  1.00  0.00           H  \\nATOM   2242  N   ARG C 159      25.193  23.108  39.727  1.00  0.00           N  \\nATOM   2243  CA  ARG C 159      24.737  24.323  40.390  1.00  0.00           C  \\nATOM   2244  C   ARG C 159      23.801  24.005  41.552  1.00  0.00           C  \\nATOM   2245  O   ARG C 159      23.774  24.727  42.546  1.00  0.00           O  \\nATOM   2246  CB  ARG C 159      24.042  25.246  39.402  1.00  0.00           C  \\nATOM   2247  CG  ARG C 159      24.977  25.891  38.370  1.00  0.00           C  \\nATOM   2248  CD  ARG C 159      24.322  27.020  37.642  1.00  0.00           C  \\nATOM   2249  NE  ARG C 159      23.170  26.613  36.830  1.00  0.00           N  \\nATOM   2250  CZ  ARG C 159      23.221  26.205  35.542  1.00  0.00           C  \\nATOM   2251  NH1 ARG C 159      24.368  26.133  34.908  1.00  0.00           N  \\nATOM   2252  NH2 ARG C 159      22.102  25.891  34.907  1.00  0.00           N  \\nATOM   2253  H   ARG C 159      24.918  23.023  38.917  1.00  0.00           H  \\nATOM   2254  HA  ARG C 159      25.506  24.794  40.747  1.00  0.00           H  \\nATOM   2255 1HB  ARG C 159      23.292  24.673  38.851  1.00  0.00           H  \\nATOM   2256 2HB  ARG C 159      23.525  26.038  39.938  1.00  0.00           H  \\nATOM   2257 1HG  ARG C 159      25.863  26.275  38.876  1.00  0.00           H  \\nATOM   2258 2HG  ARG C 159      25.277  25.140  37.639  1.00  0.00           H  \\nATOM   2259 1HD  ARG C 159      23.979  27.759  38.365  1.00  0.00           H  \\nATOM   2260 2HD  ARG C 159      25.052  27.482  36.978  1.00  0.00           H  \\nATOM   2261  HE  ARG C 159      22.260  26.651  37.269  1.00  0.00           H  \\nATOM   2262 1HH1 ARG C 159      25.224  26.365  35.391  1.00  0.00           H  \\nATOM   2263 2HH1 ARG C 159      24.383  25.850  33.938  1.00  0.00           H  \\nATOM   2264 1HH2 ARG C 159      21.216  25.959  35.388  1.00  0.00           H  \\nATOM   2265 2HH2 ARG C 159      22.136  25.594  33.940  1.00  0.00           H  \\nATOM   2266  N   ARG C 160      23.040  22.921  41.431  1.00  0.00           N  \\nATOM   2267  CA  ARG C 160      22.124  22.515  42.493  1.00  0.00           C  \\nATOM   2268  C   ARG C 160      22.911  22.026  43.704  1.00  0.00           C  \\nATOM   2269  O   ARG C 160      22.635  22.420  44.836  1.00  0.00           O  \\nATOM   2270  CB  ARG C 160      21.206  21.390  42.026  1.00  0.00           C  \\nATOM   2271  CG  ARG C 160      20.144  20.961  43.044  1.00  0.00           C  \\nATOM   2272  CD  ARG C 160      19.371  19.754  42.591  1.00  0.00           C  \\nATOM   2273  NE  ARG C 160      20.184  18.538  42.616  1.00  0.00           N  \\nATOM   2274  CZ  ARG C 160      20.428  17.814  43.738  1.00  0.00           C  \\nATOM   2275  NH1 ARG C 160      19.913  18.187  44.900  1.00  0.00           N  \\nATOM   2276  NH2 ARG C 160      21.180  16.735  43.674  1.00  0.00           N  \\nATOM   2277  H   ARG C 160      23.035  22.404  40.744  1.00  0.00           H  \\nATOM   2278  HA  ARG C 160      21.591  23.278  42.765  1.00  0.00           H  \\nATOM   2279 1HB  ARG C 160      20.701  21.691  41.108  1.00  0.00           H  \\nATOM   2280 2HB  ARG C 160      21.805  20.508  41.787  1.00  0.00           H  \\nATOM   2281 1HG  ARG C 160      20.637  20.714  43.986  1.00  0.00           H  \\nATOM   2282 2HG  ARG C 160      19.443  21.781  43.207  1.00  0.00           H  \\nATOM   2283 1HD  ARG C 160      18.510  19.603  43.240  1.00  0.00           H  \\nATOM   2284 2HD  ARG C 160      19.033  19.910  41.567  1.00  0.00           H  \\nATOM   2285  HE  ARG C 160      20.598  18.199  41.726  1.00  0.00           H  \\nATOM   2286 1HH1 ARG C 160      19.338  19.015  44.956  1.00  0.00           H  \\nATOM   2287 2HH1 ARG C 160      20.099  17.644  45.735  1.00  0.00           H  \\nATOM   2288 1HH2 ARG C 160      21.602  16.460  42.769  1.00  0.00           H  \\nATOM   2289 2HH2 ARG C 160      21.359  16.199  44.501  1.00  0.00           H  \\nATOM   2290  N   ILE C 161      23.895  21.168  43.457  1.00  0.00           N  \\nATOM   2291  CA  ILE C 161      24.725  20.619  44.525  1.00  0.00           C  \\nATOM   2292  C   ILE C 161      25.538  21.719  45.193  1.00  0.00           C  \\nATOM   2293  O   ILE C 161      25.772  21.684  46.403  1.00  0.00           O  \\nATOM   2294  CB  ILE C 161      25.709  19.551  44.020  1.00  0.00           C  \\nATOM   2295  CG1 ILE C 161      24.977  18.311  43.545  1.00  0.00           C  \\nATOM   2296  CG2 ILE C 161      26.643  19.169  45.160  1.00  0.00           C  \\nATOM   2297  CD1 ILE C 161      25.884  17.367  42.805  1.00  0.00           C  \\nATOM   2298  H   ILE C 161      24.104  20.885  42.672  1.00  0.00           H  \\nATOM   2299  HA  ILE C 161      24.152  20.217  45.197  1.00  0.00           H  \\nATOM   2300  HB  ILE C 161      26.284  19.946  43.178  1.00  0.00           H  \\nATOM   2301 1HG1 ILE C 161      24.550  17.791  44.401  1.00  0.00           H  \\nATOM   2302 2HG1 ILE C 161      24.162  18.605  42.872  1.00  0.00           H  \\nATOM   2303 1HG2 ILE C 161      27.338  18.415  44.822  1.00  0.00           H  \\nATOM   2304 2HG2 ILE C 161      27.190  20.048  45.500  1.00  0.00           H  \\nATOM   2305 3HG2 ILE C 161      26.052  18.771  45.976  1.00  0.00           H  \\nATOM   2306 1HD1 ILE C 161      25.313  16.497  42.474  1.00  0.00           H  \\nATOM   2307 2HD1 ILE C 161      26.290  17.887  41.938  1.00  0.00           H  \\nATOM   2308 3HD1 ILE C 161      26.695  17.046  43.452  1.00  0.00           H  \\nATOM   2309  N   LEU C 162      25.970  22.694  44.402  1.00  0.00           N  \\nATOM   2310  CA  LEU C 162      26.759  23.794  44.932  1.00  0.00           C  \\nATOM   2311  C   LEU C 162      25.890  24.668  45.828  1.00  0.00           C  \\nATOM   2312  O   LEU C 162      26.356  25.182  46.844  1.00  0.00           O  \\nATOM   2313  CB  LEU C 162      27.382  24.608  43.801  1.00  0.00           C  \\nATOM   2314  CG  LEU C 162      28.306  25.752  44.229  1.00  0.00           C  \\nATOM   2315  CD1 LEU C 162      29.467  25.215  45.096  1.00  0.00           C  \\nATOM   2316  CD2 LEU C 162      28.838  26.420  42.988  1.00  0.00           C  \\nATOM   2317  H   LEU C 162      25.819  22.741  43.557  1.00  0.00           H  \\nATOM   2318  HA  LEU C 162      27.483  23.437  45.469  1.00  0.00           H  \\nATOM   2319 1HB  LEU C 162      27.948  23.934  43.159  1.00  0.00           H  \\nATOM   2320 2HB  LEU C 162      26.572  25.037  43.209  1.00  0.00           H  \\nATOM   2321  HG  LEU C 162      27.741  26.471  44.817  1.00  0.00           H  \\nATOM   2322 1HD1 LEU C 162      30.123  26.036  45.392  1.00  0.00           H  \\nATOM   2323 2HD1 LEU C 162      29.065  24.740  45.985  1.00  0.00           H  \\nATOM   2324 3HD1 LEU C 162      30.031  24.497  44.531  1.00  0.00           H  \\nATOM   2325 1HD2 LEU C 162      29.472  27.239  43.284  1.00  0.00           H  \\nATOM   2326 2HD2 LEU C 162      29.409  25.700  42.398  1.00  0.00           H  \\nATOM   2327 3HD2 LEU C 162      28.008  26.799  42.393  1.00  0.00           H  \\nATOM   2328  N   GLU C 163      24.624  24.830  45.450  1.00  0.00           N  \\nATOM   2329  CA  GLU C 163      23.677  25.573  46.275  1.00  0.00           C  \\nATOM   2330  C   GLU C 163      23.535  24.886  47.629  1.00  0.00           C  \\nATOM   2331  O   GLU C 163      23.550  25.537  48.670  1.00  0.00           O  \\nATOM   2332  CB  GLU C 163      22.335  25.706  45.518  1.00  0.00           C  \\nATOM   2333  CG  GLU C 163      21.268  26.689  46.078  1.00  0.00           C  \\nATOM   2334  CD  GLU C 163      20.288  26.107  47.082  1.00  0.00           C  \\nATOM   2335  OE1 GLU C 163      20.022  24.931  47.032  1.00  0.00           O  \\nATOM   2336  OE2 GLU C 163      19.766  26.865  47.862  1.00  0.00           O  \\nATOM   2337  H   GLU C 163      24.289  24.519  44.722  1.00  0.00           H  \\nATOM   2338  HA  GLU C 163      24.018  26.469  46.424  1.00  0.00           H  \\nATOM   2339 1HB  GLU C 163      22.538  26.002  44.491  1.00  0.00           H  \\nATOM   2340 2HB  GLU C 163      21.860  24.731  45.473  1.00  0.00           H  \\nATOM   2341 1HG  GLU C 163      21.791  27.518  46.555  1.00  0.00           H  \\nATOM   2342 2HG  GLU C 163      20.708  27.094  45.237  1.00  0.00           H  \\nATOM   2343  N   GLU C 164      23.400  23.564  47.602  1.00  0.00           N  \\nATOM   2344  CA  GLU C 164      23.300  22.773  48.823  1.00  0.00           C  \\nATOM   2345  C   GLU C 164      24.591  22.854  49.630  1.00  0.00           C  \\nATOM   2346  O   GLU C 164      24.569  22.850  50.860  1.00  0.00           O  \\nATOM   2347  CB  GLU C 164      22.987  21.301  48.480  1.00  0.00           C  \\nATOM   2348  CG  GLU C 164      22.746  20.382  49.689  1.00  0.00           C  \\nATOM   2349  CD  GLU C 164      22.296  18.959  49.315  1.00  0.00           C  \\nATOM   2350  OE1 GLU C 164      22.184  18.663  48.146  1.00  0.00           O  \\nATOM   2351  OE2 GLU C 164      22.048  18.185  50.210  1.00  0.00           O  \\nATOM   2352  H   GLU C 164      23.364  23.097  46.881  1.00  0.00           H  \\nATOM   2353  HA  GLU C 164      22.579  23.121  49.370  1.00  0.00           H  \\nATOM   2354 1HB  GLU C 164      22.110  21.260  47.834  1.00  0.00           H  \\nATOM   2355 2HB  GLU C 164      23.820  20.881  47.918  1.00  0.00           H  \\nATOM   2356 1HG  GLU C 164      23.679  20.311  50.236  1.00  0.00           H  \\nATOM   2357 2HG  GLU C 164      22.007  20.839  50.345  1.00  0.00           H  \\nATOM   2358  N   LEU C 165      25.717  22.926  48.931  1.00  0.00           N  \\nATOM   2359  CA  LEU C 165      27.015  22.974  49.587  1.00  0.00           C  \\nATOM   2360  C   LEU C 165      27.197  24.301  50.312  1.00  0.00           C  \\nATOM   2361  O   LEU C 165      27.856  24.363  51.347  1.00  0.00           O  \\nATOM   2362  CB  LEU C 165      28.118  22.693  48.560  1.00  0.00           C  \\nATOM   2363  CG  LEU C 165      29.530  22.558  49.091  1.00  0.00           C  \\nATOM   2364  CD1 LEU C 165      29.597  21.467  50.178  1.00  0.00           C  \\nATOM   2365  CD2 LEU C 165      30.460  22.174  47.903  1.00  0.00           C  \\nATOM   2366  H   LEU C 165      25.756  22.948  48.072  1.00  0.00           H  \\nATOM   2367  HA  LEU C 165      27.065  22.262  50.243  1.00  0.00           H  \\nATOM   2368 1HB  LEU C 165      27.867  21.783  48.017  1.00  0.00           H  \\nATOM   2369 2HB  LEU C 165      28.122  23.514  47.855  1.00  0.00           H  \\nATOM   2370  HG  LEU C 165      29.833  23.502  49.530  1.00  0.00           H  \\nATOM   2371 1HD1 LEU C 165      30.613  21.401  50.545  1.00  0.00           H  \\nATOM   2372 2HD1 LEU C 165      28.933  21.728  51.004  1.00  0.00           H  \\nATOM   2373 3HD1 LEU C 165      29.302  20.514  49.767  1.00  0.00           H  \\nATOM   2374 1HD2 LEU C 165      31.481  22.068  48.249  1.00  0.00           H  \\nATOM   2375 2HD2 LEU C 165      30.130  21.224  47.476  1.00  0.00           H  \\nATOM   2376 3HD2 LEU C 165      30.417  22.948  47.137  1.00  0.00           H  \\nATOM   2377  N   LYS C 166      26.598  25.358  49.773  1.00  0.00           N  \\nATOM   2378  CA  LYS C 166      26.702  26.682  50.374  1.00  0.00           C  \\nATOM   2379  C   LYS C 166      25.693  26.866  51.500  1.00  0.00           C  \\nATOM   2380  O   LYS C 166      25.907  27.668  52.408  1.00  0.00           O  \\nATOM   2381  CB  LYS C 166      26.514  27.741  49.294  1.00  0.00           C  \\nATOM   2382  CG  LYS C 166      27.687  27.864  48.312  1.00  0.00           C  \\nATOM   2383  CD  LYS C 166      27.338  28.806  47.172  1.00  0.00           C  \\nATOM   2384  CE  LYS C 166      28.536  29.101  46.298  1.00  0.00           C  \\nATOM   2385  NZ  LYS C 166      29.420  30.096  46.938  1.00  0.00           N  \\nATOM   2386  H   LYS C 166      26.123  25.333  49.056  1.00  0.00           H  \\nATOM   2387  HA  LYS C 166      27.591  26.788  50.749  1.00  0.00           H  \\nATOM   2388 1HB  LYS C 166      25.619  27.496  48.712  1.00  0.00           H  \\nATOM   2389 2HB  LYS C 166      26.350  28.711  49.757  1.00  0.00           H  \\nATOM   2390 1HG  LYS C 166      28.566  28.240  48.835  1.00  0.00           H  \\nATOM   2391 2HG  LYS C 166      27.925  26.881  47.902  1.00  0.00           H  \\nATOM   2392 1HD  LYS C 166      26.561  28.341  46.559  1.00  0.00           H  \\nATOM   2393 2HD  LYS C 166      26.950  29.741  47.573  1.00  0.00           H  \\nATOM   2394 1HE  LYS C 166      29.102  28.186  46.141  1.00  0.00           H  \\nATOM   2395 2HE  LYS C 166      28.209  29.484  45.331  1.00  0.00           H  \\nATOM   2396 1HZ  LYS C 166      28.911  30.955  47.068  1.00  0.00           H  \\nATOM   2397 2HZ  LYS C 166      29.746  29.768  47.826  1.00  0.00           H  \\nATOM   2398 3HZ  LYS C 166      30.224  30.277  46.325  1.00  0.00           H  \\nATOM   2399  N   GLU C 167      24.592  26.124  51.436  1.00  0.00           N  \\nATOM   2400  CA  GLU C 167      23.594  26.149  52.497  1.00  0.00           C  \\nATOM   2401  C   GLU C 167      24.153  25.481  53.748  1.00  0.00           C  \\nATOM   2402  O   GLU C 167      23.964  25.965  54.864  1.00  0.00           O  \\nATOM   2403  CB  GLU C 167      22.275  25.501  52.076  1.00  0.00           C  \\nATOM   2404  CG  GLU C 167      21.429  26.354  51.126  1.00  0.00           C  \\nATOM   2405  CD  GLU C 167      20.896  27.623  51.800  1.00  0.00           C  \\nATOM   2406  OE1 GLU C 167      21.089  28.698  51.275  1.00  0.00           O  \\nATOM   2407  OE2 GLU C 167      20.294  27.502  52.847  1.00  0.00           O  \\nATOM   2408  H   GLU C 167      24.399  25.596  50.785  1.00  0.00           H  \\nATOM   2409  HA  GLU C 167      23.379  27.070  52.714  1.00  0.00           H  \\nATOM   2410 1HB  GLU C 167      22.484  24.555  51.574  1.00  0.00           H  \\nATOM   2411 2HB  GLU C 167      21.680  25.279  52.959  1.00  0.00           H  \\nATOM   2412 1HG  GLU C 167      22.036  26.651  50.278  1.00  0.00           H  \\nATOM   2413 2HG  GLU C 167      20.597  25.761  50.747  1.00  0.00           H  \\nATOM   2414  N   MET C 168      24.854  24.371  53.547  1.00  0.00           N  \\nATOM   2415  CA  MET C 168      25.421  23.608  54.650  1.00  0.00           C  \\nATOM   2416  C   MET C 168      26.681  24.258  55.203  1.00  0.00           C  \\nATOM   2417  O   MET C 168      27.083  23.993  56.336  1.00  0.00           O  \\nATOM   2418  CB  MET C 168      25.773  22.267  54.096  1.00  0.00           C  \\nATOM   2419  CG  MET C 168      24.617  21.459  53.719  1.00  0.00           C  \\nATOM   2420  SD  MET C 168      25.149  20.117  52.804  1.00  0.00           S  \\nATOM   2421  CE  MET C 168      26.159  19.396  53.983  1.00  0.00           C  \\nATOM   2422  H   MET C 168      25.016  24.035  52.772  1.00  0.00           H  \\nATOM   2423  HA  MET C 168      24.763  23.551  55.360  1.00  0.00           H  \\nATOM   2424 1HB  MET C 168      26.396  22.396  53.205  1.00  0.00           H  \\nATOM   2425 2HB  MET C 168      26.353  21.712  54.829  1.00  0.00           H  \\nATOM   2426 1HG  MET C 168      24.100  21.100  54.606  1.00  0.00           H  \\nATOM   2427 2HG  MET C 168      23.921  22.044  53.119  1.00  0.00           H  \\nATOM   2428 1HE  MET C 168      26.593  18.507  53.560  1.00  0.00           H  \\nATOM   2429 2HE  MET C 168      26.934  20.089  54.255  1.00  0.00           H  \\nATOM   2430 3HE  MET C 168      25.577  19.154  54.862  1.00  0.00           H  \\nATOM   2431  N   LEU C 169      27.308  25.100  54.393  1.00  0.00           N  \\nATOM   2432  CA  LEU C 169      28.496  25.820  54.820  1.00  0.00           C  \\nATOM   2433  C   LEU C 169      28.123  26.874  55.856  1.00  0.00           C  \\nATOM   2434  O   LEU C 169      28.907  27.181  56.754  1.00  0.00           O  \\nATOM   2435  CB  LEU C 169      29.198  26.463  53.612  1.00  0.00           C  \\nATOM   2436  CG  LEU C 169      30.626  26.992  53.847  1.00  0.00           C  \\nATOM   2437  CD1 LEU C 169      31.524  25.808  54.288  1.00  0.00           C  \\nATOM   2438  CD2 LEU C 169      31.152  27.628  52.560  1.00  0.00           C  \\nATOM   2439  H   LEU C 169      27.064  25.273  53.587  1.00  0.00           H  \\nATOM   2440  HA  LEU C 169      29.118  25.198  55.229  1.00  0.00           H  \\nATOM   2441 1HB  LEU C 169      29.257  25.729  52.819  1.00  0.00           H  \\nATOM   2442 2HB  LEU C 169      28.583  27.294  53.261  1.00  0.00           H  \\nATOM   2443  HG  LEU C 169      30.618  27.731  54.646  1.00  0.00           H  \\nATOM   2444 1HD1 LEU C 169      31.118  25.374  55.199  1.00  0.00           H  \\nATOM   2445 2HD1 LEU C 169      31.537  25.050  53.504  1.00  0.00           H  \\nATOM   2446 3HD1 LEU C 169      32.538  26.152  54.470  1.00  0.00           H  \\nATOM   2447 1HD2 LEU C 169      32.163  27.997  52.714  1.00  0.00           H  \\nATOM   2448 2HD2 LEU C 169      31.157  26.888  51.775  1.00  0.00           H  \\nATOM   2449 3HD2 LEU C 169      30.501  28.453  52.274  1.00  0.00           H  \\nATOM   2450  N   GLU C 170      26.920  27.424  55.726  1.00  0.00           N  \\nATOM   2451  CA  GLU C 170      26.431  28.425  56.666  1.00  0.00           C  \\nATOM   2452  C   GLU C 170      26.093  27.779  58.009  1.00  0.00           C  \\nATOM   2453  O   GLU C 170      26.188  28.422  59.054  1.00  0.00           O  \\nATOM   2454  CB  GLU C 170      25.178  29.072  56.059  1.00  0.00           C  \\nATOM   2455  CG  GLU C 170      24.466  30.130  56.887  1.00  0.00           C  \\nATOM   2456  CD  GLU C 170      25.205  31.402  57.127  1.00  0.00           C  \\nATOM   2457  OE1 GLU C 170      24.842  32.070  58.082  1.00  0.00           O  \\nATOM   2458  OE2 GLU C 170      26.081  31.734  56.375  1.00  0.00           O  \\nATOM   2459  H   GLU C 170      26.365  27.233  55.098  1.00  0.00           H  \\nATOM   2460  HA  GLU C 170      27.125  29.085  56.816  1.00  0.00           H  \\nATOM   2461 1HB  GLU C 170      25.441  29.523  55.101  1.00  0.00           H  \\nATOM   2462 2HB  GLU C 170      24.449  28.293  55.849  1.00  0.00           H  \\nATOM   2463 1HG  GLU C 170      23.522  30.366  56.398  1.00  0.00           H  \\nATOM   2464 2HG  GLU C 170      24.234  29.672  57.844  1.00  0.00           H  \\nATOM   2465  N   ARG C 171      25.694  26.509  57.975  1.00  0.00           N  \\nATOM   2466  CA  ARG C 171      25.462  25.748  59.200  1.00  0.00           C  \\nATOM   2467  C   ARG C 171      26.769  25.590  59.961  1.00  0.00           C  \\nATOM   2468  O   ARG C 171      26.818  25.729  61.183  1.00  0.00           O  \\nATOM   2469  CB  ARG C 171      24.893  24.367  58.904  1.00  0.00           C  \\nATOM   2470  CG  ARG C 171      23.465  24.352  58.386  1.00  0.00           C  \\nATOM   2471  CD  ARG C 171      22.990  22.958  58.143  1.00  0.00           C  \\nATOM   2472  NE  ARG C 171      21.649  22.925  57.568  1.00  0.00           N  \\nATOM   2473  CZ  ARG C 171      20.502  22.979  58.278  1.00  0.00           C  \\nATOM   2474  NH1 ARG C 171      20.540  23.063  59.590  1.00  0.00           N  \\nATOM   2475  NH2 ARG C 171      19.337  22.945  57.653  1.00  0.00           N  \\nATOM   2476  H   ARG C 171      25.550  26.064  57.253  1.00  0.00           H  \\nATOM   2477  HA  ARG C 171      24.835  26.228  59.763  1.00  0.00           H  \\nATOM   2478 1HB  ARG C 171      25.511  23.859  58.182  1.00  0.00           H  \\nATOM   2479 2HB  ARG C 171      24.916  23.771  59.817  1.00  0.00           H  \\nATOM   2480 1HG  ARG C 171      22.806  24.826  59.110  1.00  0.00           H  \\nATOM   2481 2HG  ARG C 171      23.425  24.900  57.439  1.00  0.00           H  \\nATOM   2482 1HD  ARG C 171      23.671  22.462  57.449  1.00  0.00           H  \\nATOM   2483 2HD  ARG C 171      22.972  22.410  59.084  1.00  0.00           H  \\nATOM   2484  HE  ARG C 171      21.573  22.864  56.561  1.00  0.00           H  \\nATOM   2485 1HH1 ARG C 171      21.429  23.088  60.069  1.00  0.00           H  \\nATOM   2486 2HH1 ARG C 171      19.681  23.102  60.118  1.00  0.00           H  \\nATOM   2487 1HH2 ARG C 171      19.306  22.880  56.645  1.00  0.00           H  \\nATOM   2488 2HH2 ARG C 171      18.479  22.985  58.183  1.00  0.00           H  \\nATOM   2489  N   LEU C 172      27.823  25.284  59.213  1.00  0.00           N  \\nATOM   2490  CA  LEU C 172      29.153  25.088  59.768  1.00  0.00           C  \\nATOM   2491  C   LEU C 172      29.622  26.320  60.537  1.00  0.00           C  \\nATOM   2492  O   LEU C 172      30.294  26.208  61.562  1.00  0.00           O  \\nATOM   2493  CB  LEU C 172      30.124  24.761  58.613  1.00  0.00           C  \\nATOM   2494  CG  LEU C 172      31.525  24.300  58.975  1.00  0.00           C  \\nATOM   2495  CD1 LEU C 172      31.442  22.981  59.724  1.00  0.00           C  \\nATOM   2496  CD2 LEU C 172      32.347  24.145  57.704  1.00  0.00           C  \\nATOM   2497  H   LEU C 172      27.790  25.183  58.359  1.00  0.00           H  \\nATOM   2498  HA  LEU C 172      29.136  24.337  60.381  1.00  0.00           H  \\nATOM   2499 1HB  LEU C 172      29.665  24.001  57.978  1.00  0.00           H  \\nATOM   2500 2HB  LEU C 172      30.252  25.655  58.022  1.00  0.00           H  \\nATOM   2501  HG  LEU C 172      31.988  25.037  59.633  1.00  0.00           H  \\nATOM   2502 1HD1 LEU C 172      32.440  22.667  59.998  1.00  0.00           H  \\nATOM   2503 2HD1 LEU C 172      30.849  23.098  60.605  1.00  0.00           H  \\nATOM   2504 3HD1 LEU C 172      30.983  22.226  59.085  1.00  0.00           H  \\nATOM   2505 1HD2 LEU C 172      33.342  23.824  57.960  1.00  0.00           H  \\nATOM   2506 2HD2 LEU C 172      31.880  23.407  57.051  1.00  0.00           H  \\nATOM   2507 3HD2 LEU C 172      32.399  25.103  57.191  1.00  0.00           H  \\nATOM   2508  N   GLU C 173      29.266  27.495  60.027  1.00  0.00           N  \\nATOM   2509  CA  GLU C 173      29.635  28.761  60.656  1.00  0.00           C  \\nATOM   2510  C   GLU C 173      28.926  28.968  61.992  1.00  0.00           C  \\nATOM   2511  O   GLU C 173      29.563  29.288  62.995  1.00  0.00           O  \\nATOM   2512  CB  GLU C 173      29.409  29.940  59.703  1.00  0.00           C  \\nATOM   2513  CG  GLU C 173      29.839  31.297  60.292  1.00  0.00           C  \\nATOM   2514  CD  GLU C 173      29.896  32.431  59.277  1.00  0.00           C  \\nATOM   2515  OE1 GLU C 173      29.873  33.567  59.686  1.00  0.00           O  \\nATOM   2516  OE2 GLU C 173      30.031  32.154  58.102  1.00  0.00           O  \\nATOM   2517  H   GLU C 173      28.804  27.588  59.308  1.00  0.00           H  \\nATOM   2518  HA  GLU C 173      30.590  28.763  60.821  1.00  0.00           H  \\nATOM   2519 1HB  GLU C 173      29.959  29.772  58.777  1.00  0.00           H  \\nATOM   2520 2HB  GLU C 173      28.349  30.002  59.446  1.00  0.00           H  \\nATOM   2521 1HG  GLU C 173      29.131  31.568  61.076  1.00  0.00           H  \\nATOM   2522 2HG  GLU C 173      30.818  31.181  60.753  1.00  0.00           H  \\nATOM   2523  N   LYS C 174      27.606  28.804  61.989  1.00  0.00           N  \\nATOM   2524  CA  LYS C 174      26.794  28.975  63.193  1.00  0.00           C  \\nATOM   2525  C   LYS C 174      27.275  28.090  64.340  1.00  0.00           C  \\nATOM   2526  O   LYS C 174      27.157  28.451  65.511  1.00  0.00           O  \\nATOM   2527  CB  LYS C 174      25.325  28.720  62.880  1.00  0.00           C  \\nATOM   2528  CG  LYS C 174      24.703  29.859  62.090  1.00  0.00           C  \\nATOM   2529  CD  LYS C 174      23.224  29.671  61.840  1.00  0.00           C  \\nATOM   2530  CE  LYS C 174      22.642  30.948  61.256  1.00  0.00           C  \\nATOM   2531  NZ  LYS C 174      23.312  31.295  59.995  1.00  0.00           N  \\nATOM   2532  H   LYS C 174      27.149  28.590  61.293  1.00  0.00           H  \\nATOM   2533  HA  LYS C 174      26.852  29.899  63.483  1.00  0.00           H  \\nATOM   2534 1HB  LYS C 174      25.235  27.800  62.292  1.00  0.00           H  \\nATOM   2535 2HB  LYS C 174      24.768  28.580  63.804  1.00  0.00           H  \\nATOM   2536 1HG  LYS C 174      24.857  30.796  62.622  1.00  0.00           H  \\nATOM   2537 2HG  LYS C 174      25.210  29.929  61.119  1.00  0.00           H  \\nATOM   2538 1HD  LYS C 174      23.078  28.851  61.133  1.00  0.00           H  \\nATOM   2539 2HD  LYS C 174      22.715  29.429  62.772  1.00  0.00           H  \\nATOM   2540 1HE  LYS C 174      21.578  30.815  61.068  1.00  0.00           H  \\nATOM   2541 2HE  LYS C 174      22.779  31.764  61.965  1.00  0.00           H  \\nATOM   2542 1HZ  LYS C 174      23.181  30.539  59.342  1.00  0.00           H  \\nATOM   2543 2HZ  LYS C 174      22.949  32.143  59.591  1.00  0.00           H  \\nATOM   2544 3HZ  LYS C 174      24.309  31.418  60.175  1.00  0.00           H  \\nATOM   2545  N   ASN C 175      27.816  26.928  63.990  1.00  0.00           N  \\nATOM   2546  CA  ASN C 175      28.257  25.949  64.975  1.00  0.00           C  \\nATOM   2547  C   ASN C 175      29.037  24.821  64.309  1.00  0.00           C  \\nATOM   2548  O   ASN C 175      28.447  23.835  63.869  1.00  0.00           O  \\nATOM   2549  CB  ASN C 175      27.102  25.364  65.779  1.00  0.00           C  \\nATOM   2550  CG  ASN C 175      26.110  24.586  64.964  1.00  0.00           C  \\nATOM   2551  OD1 ASN C 175      25.413  25.125  64.091  1.00  0.00           O  \\nATOM   2552  ND2 ASN C 175      26.033  23.311  65.235  1.00  0.00           N  \\nATOM   2553  H   ASN C 175      27.939  26.680  63.176  1.00  0.00           H  \\nATOM   2554  HA  ASN C 175      28.842  26.384  65.615  1.00  0.00           H  \\nATOM   2555 1HB  ASN C 175      27.511  24.688  66.535  1.00  0.00           H  \\nATOM   2556 2HB  ASN C 175      26.574  26.151  66.310  1.00  0.00           H  \\nATOM   2557 1HD2 ASN C 175      26.641  22.887  65.952  1.00  0.00           H  \\nATOM   2558 2HD2 ASN C 175      25.401  22.725  64.733  1.00  0.00           H  \\nATOM   2559  N   PRO C 176      30.371  24.958  64.231  1.00  0.00           N  \\nATOM   2560  CA  PRO C 176      31.144  23.927  63.536  1.00  0.00           C  \\nATOM   2561  C   PRO C 176      31.202  22.619  64.321  1.00  0.00           C  \\nATOM   2562  O   PRO C 176      32.216  22.320  64.951  1.00  0.00           O  \\nATOM   2563  CB  PRO C 176      32.501  24.622  63.399  1.00  0.00           C  \\nATOM   2564  CG  PRO C 176      32.583  25.555  64.614  1.00  0.00           C  \\nATOM   2565  CD  PRO C 176      31.138  26.060  64.820  1.00  0.00           C  \\nATOM   2566  HA  PRO C 176      30.778  23.762  62.653  1.00  0.00           H  \\nATOM   2567 1HB  PRO C 176      33.307  23.873  63.365  1.00  0.00           H  \\nATOM   2568 2HB  PRO C 176      32.526  25.171  62.446  1.00  0.00           H  \\nATOM   2569 1HG  PRO C 176      32.963  25.017  65.482  1.00  0.00           H  \\nATOM   2570 2HG  PRO C 176      33.292  26.374  64.412  1.00  0.00           H  \\nATOM   2571 1HD  PRO C 176      30.958  26.156  65.897  1.00  0.00           H  \\nATOM   2572 2HD  PRO C 176      30.949  27.001  64.265  1.00  0.00           H  \\nATOM   2573  N   ASP C 177      30.115  21.852  64.289  1.00  0.00           N  \\nATOM   2574  CA  ASP C 177      30.104  20.520  64.885  1.00  0.00           C  \\nATOM   2575  C   ASP C 177      31.015  19.597  64.084  1.00  0.00           C  \\nATOM   2576  O   ASP C 177      31.678  20.034  63.145  1.00  0.00           O  \\nATOM   2577  CB  ASP C 177      28.698  19.892  64.897  1.00  0.00           C  \\nATOM   2578  CG  ASP C 177      27.635  20.556  65.806  1.00  0.00           C  \\nATOM   2579  OD1 ASP C 177      27.932  21.451  66.574  1.00  0.00           O  \\nATOM   2580  OD2 ASP C 177      26.506  20.114  65.725  1.00  0.00           O  \\nATOM   2581  H   ASP C 177      29.370  22.081  63.926  1.00  0.00           H  \\nATOM   2582  HA  ASP C 177      30.444  20.571  65.792  1.00  0.00           H  \\nATOM   2583 1HB  ASP C 177      28.324  19.905  63.895  1.00  0.00           H  \\nATOM   2584 2HB  ASP C 177      28.764  18.854  65.195  1.00  0.00           H  \\nATOM   2585  N   LYS C 178      31.045  18.322  64.456  1.00  0.00           N  \\nATOM   2586  CA  LYS C 178      31.856  17.338  63.748  1.00  0.00           C  \\nATOM   2587  C   LYS C 178      30.991  16.476  62.835  1.00  0.00           C  \\nATOM   2588  O   LYS C 178      31.477  15.932  61.848  1.00  0.00           O  \\nATOM   2589  CB  LYS C 178      32.595  16.501  64.769  1.00  0.00           C  \\nATOM   2590  CG  LYS C 178      33.451  15.454  64.211  1.00  0.00           C  \\nATOM   2591  CD  LYS C 178      34.209  14.787  65.308  1.00  0.00           C  \\nATOM   2592  CE  LYS C 178      33.243  14.127  66.306  1.00  0.00           C  \\nATOM   2593  NZ  LYS C 178      33.897  13.059  67.087  1.00  0.00           N  \\nATOM   2594  H   LYS C 178      30.601  17.999  65.118  1.00  0.00           H  \\nATOM   2595  HA  LYS C 178      32.505  17.803  63.196  1.00  0.00           H  \\nATOM   2596 1HB  LYS C 178      33.220  17.149  65.383  1.00  0.00           H  \\nATOM   2597 2HB  LYS C 178      31.875  16.018  65.429  1.00  0.00           H  \\nATOM   2598 1HG  LYS C 178      32.841  14.707  63.696  1.00  0.00           H  \\nATOM   2599 2HG  LYS C 178      34.149  15.889  63.499  1.00  0.00           H  \\nATOM   2600 1HD  LYS C 178      34.851  14.047  64.876  1.00  0.00           H  \\nATOM   2601 2HD  LYS C 178      34.824  15.514  65.832  1.00  0.00           H  \\nATOM   2602 1HE  LYS C 178      32.879  14.888  66.992  1.00  0.00           H  \\nATOM   2603 2HE  LYS C 178      32.396  13.704  65.767  1.00  0.00           H  \\nATOM   2604 1HZ  LYS C 178      33.260  12.631  67.721  1.00  0.00           H  \\nATOM   2605 2HZ  LYS C 178      34.219  12.330  66.410  1.00  0.00           H  \\nATOM   2606 3HZ  LYS C 178      34.692  13.381  67.600  1.00  0.00           H  \\nATOM   2607  N   ASP C 179      29.708  16.361  63.165  1.00  0.00           N  \\nATOM   2608  CA  ASP C 179      28.761  15.631  62.326  1.00  0.00           C  \\nATOM   2609  C   ASP C 179      28.347  16.481  61.126  1.00  0.00           C  \\nATOM   2610  O   ASP C 179      28.050  15.954  60.054  1.00  0.00           O  \\nATOM   2611  CB  ASP C 179      27.523  15.240  63.143  1.00  0.00           C  \\nATOM   2612  CG  ASP C 179      26.632  14.175  62.472  1.00  0.00           C  \\nATOM   2613  OD1 ASP C 179      27.106  13.081  62.253  1.00  0.00           O  \\nATOM   2614  OD2 ASP C 179      25.491  14.469  62.200  1.00  0.00           O  \\nATOM   2615  H   ASP C 179      29.358  16.698  63.875  1.00  0.00           H  \\nATOM   2616  HA  ASP C 179      29.187  14.826  61.994  1.00  0.00           H  \\nATOM   2617 1HB  ASP C 179      27.838  14.862  64.115  1.00  0.00           H  \\nATOM   2618 2HB  ASP C 179      26.917  16.130  63.323  1.00  0.00           H  \\nATOM   2619  N   VAL C 180      28.323  17.798  61.313  1.00  0.00           N  \\nATOM   2620  CA  VAL C 180      28.053  18.720  60.216  1.00  0.00           C  \\nATOM   2621  C   VAL C 180      29.210  18.679  59.225  1.00  0.00           C  \\nATOM   2622  O   VAL C 180      28.997  18.640  58.015  1.00  0.00           O  \\nATOM   2623  CB  VAL C 180      27.887  20.172  60.721  1.00  0.00           C  \\nATOM   2624  CG1 VAL C 180      27.810  21.148  59.528  1.00  0.00           C  \\nATOM   2625  CG2 VAL C 180      26.618  20.280  61.552  1.00  0.00           C  \\nATOM   2626  H   VAL C 180      28.460  18.183  62.070  1.00  0.00           H  \\nATOM   2627  HA  VAL C 180      27.248  18.440  59.755  1.00  0.00           H  \\nATOM   2628  HB  VAL C 180      28.756  20.444  61.327  1.00  0.00           H  \\nATOM   2629 1HG1 VAL C 180      27.701  22.169  59.899  1.00  0.00           H  \\nATOM   2630 2HG1 VAL C 180      28.722  21.075  58.935  1.00  0.00           H  \\nATOM   2631 3HG1 VAL C 180      26.953  20.898  58.903  1.00  0.00           H  \\nATOM   2632 1HG2 VAL C 180      26.507  21.306  61.917  1.00  0.00           H  \\nATOM   2633 2HG2 VAL C 180      25.760  20.018  60.934  1.00  0.00           H  \\nATOM   2634 3HG2 VAL C 180      26.667  19.600  62.398  1.00  0.00           H  \\nATOM   2635  N   ILE C 181      30.434  18.686  59.748  1.00  0.00           N  \\nATOM   2636  CA  ILE C 181      31.631  18.544  58.923  1.00  0.00           C  \\nATOM   2637  C   ILE C 181      31.555  17.273  58.083  1.00  0.00           C  \\nATOM   2638  O   ILE C 181      31.911  17.279  56.906  1.00  0.00           O  \\nATOM   2639  CB  ILE C 181      32.950  18.525  59.714  1.00  0.00           C  \\nATOM   2640  CG1 ILE C 181      33.233  19.911  60.278  1.00  0.00           C  \\nATOM   2641  CG2 ILE C 181      34.089  18.006  58.818  1.00  0.00           C  \\nATOM   2642  CD1 ILE C 181      34.352  19.958  61.290  1.00  0.00           C  \\nATOM   2643  H   ILE C 181      30.599  18.773  60.588  1.00  0.00           H  \\nATOM   2644  HA  ILE C 181      31.691  19.301  58.319  1.00  0.00           H  \\nATOM   2645  HB  ILE C 181      32.847  17.856  60.568  1.00  0.00           H  \\nATOM   2646 1HG1 ILE C 181      33.492  20.571  59.455  1.00  0.00           H  \\nATOM   2647 2HG1 ILE C 181      32.327  20.282  60.751  1.00  0.00           H  \\nATOM   2648 1HG2 ILE C 181      35.014  17.982  59.385  1.00  0.00           H  \\nATOM   2649 2HG2 ILE C 181      33.852  16.999  58.473  1.00  0.00           H  \\nATOM   2650 3HG2 ILE C 181      34.206  18.646  57.966  1.00  0.00           H  \\nATOM   2651 1HD1 ILE C 181      34.482  20.986  61.639  1.00  0.00           H  \\nATOM   2652 2HD1 ILE C 181      34.101  19.320  62.136  1.00  0.00           H  \\nATOM   2653 3HD1 ILE C 181      35.272  19.610  60.834  1.00  0.00           H  \\nATOM   2654  N   VAL C 182      31.080  16.190  58.693  1.00  0.00           N  \\nATOM   2655  CA  VAL C 182      30.901  14.922  57.990  1.00  0.00           C  \\nATOM   2656  C   VAL C 182      29.944  15.088  56.814  1.00  0.00           C  \\nATOM   2657  O   VAL C 182      30.222  14.625  55.707  1.00  0.00           O  \\nATOM   2658  CB  VAL C 182      30.454  13.791  58.929  1.00  0.00           C  \\nATOM   2659  CG1 VAL C 182      30.019  12.585  58.115  1.00  0.00           C  \\nATOM   2660  CG2 VAL C 182      31.651  13.393  59.811  1.00  0.00           C  \\nATOM   2661  H   VAL C 182      30.853  16.164  59.522  1.00  0.00           H  \\nATOM   2662  HA  VAL C 182      31.757  14.628  57.643  1.00  0.00           H  \\nATOM   2663  HB  VAL C 182      29.622  14.116  59.541  1.00  0.00           H  \\nATOM   2664 1HG1 VAL C 182      29.719  11.779  58.787  1.00  0.00           H  \\nATOM   2665 2HG1 VAL C 182      29.179  12.870  57.496  1.00  0.00           H  \\nATOM   2666 3HG1 VAL C 182      30.821  12.255  57.492  1.00  0.00           H  \\nATOM   2667 1HG2 VAL C 182      31.358  12.588  60.480  1.00  0.00           H  \\nATOM   2668 2HG2 VAL C 182      32.470  13.069  59.190  1.00  0.00           H  \\nATOM   2669 3HG2 VAL C 182      31.975  14.229  60.384  1.00  0.00           H  \\nATOM   2670  N   GLU C 183      28.821  15.755  57.057  1.00  0.00           N  \\nATOM   2671  CA  GLU C 183      27.810  15.944  56.025  1.00  0.00           C  \\nATOM   2672  C   GLU C 183      28.353  16.809  54.892  1.00  0.00           C  \\nATOM   2673  O   GLU C 183      28.124  16.521  53.719  1.00  0.00           O  \\nATOM   2674  CB  GLU C 183      26.535  16.547  56.618  1.00  0.00           C  \\nATOM   2675  CG  GLU C 183      25.290  16.482  55.690  1.00  0.00           C  \\nATOM   2676  CD  GLU C 183      24.711  15.088  55.486  1.00  0.00           C  \\nATOM   2677  OE1 GLU C 183      24.976  14.210  56.275  1.00  0.00           O  \\nATOM   2678  OE2 GLU C 183      24.002  14.907  54.523  1.00  0.00           O  \\nATOM   2679  H   GLU C 183      28.620  16.109  57.815  1.00  0.00           H  \\nATOM   2680  HA  GLU C 183      27.570  15.080  55.655  1.00  0.00           H  \\nATOM   2681 1HB  GLU C 183      26.295  16.036  57.552  1.00  0.00           H  \\nATOM   2682 2HB  GLU C 183      26.716  17.596  56.862  1.00  0.00           H  \\nATOM   2683 1HG  GLU C 183      24.515  17.144  56.071  1.00  0.00           H  \\nATOM   2684 2HG  GLU C 183      25.580  16.825  54.719  1.00  0.00           H  \\nATOM   2685  N   VAL C 184      29.082  17.862  55.250  1.00  0.00           N  \\nATOM   2686  CA  VAL C 184      29.695  18.747  54.265  1.00  0.00           C  \\nATOM   2687  C   VAL C 184      30.661  17.976  53.375  1.00  0.00           C  \\nATOM   2688  O   VAL C 184      30.706  18.194  52.168  1.00  0.00           O  \\nATOM   2689  CB  VAL C 184      30.456  19.925  54.946  1.00  0.00           C  \\nATOM   2690  CG1 VAL C 184      31.281  20.686  53.909  1.00  0.00           C  \\nATOM   2691  CG2 VAL C 184      29.464  20.918  55.602  1.00  0.00           C  \\nATOM   2692  H   VAL C 184      29.238  18.087  56.065  1.00  0.00           H  \\nATOM   2693  HA  VAL C 184      29.000  19.120  53.700  1.00  0.00           H  \\nATOM   2694  HB  VAL C 184      31.131  19.517  55.700  1.00  0.00           H  \\nATOM   2695 1HG1 VAL C 184      31.822  21.497  54.396  1.00  0.00           H  \\nATOM   2696 2HG1 VAL C 184      31.987  20.002  53.449  1.00  0.00           H  \\nATOM   2697 3HG1 VAL C 184      30.620  21.098  53.147  1.00  0.00           H  \\nATOM   2698 1HG2 VAL C 184      30.017  21.726  56.080  1.00  0.00           H  \\nATOM   2699 2HG2 VAL C 184      28.812  21.335  54.841  1.00  0.00           H  \\nATOM   2700 3HG2 VAL C 184      28.867  20.406  56.346  1.00  0.00           H  \\nATOM   2701  N   LEU C 185      31.427  17.069  53.972  1.00  0.00           N  \\nATOM   2702  CA  LEU C 185      32.376  16.268  53.210  1.00  0.00           C  \\nATOM   2703  C   LEU C 185      31.649  15.388  52.200  1.00  0.00           C  \\nATOM   2704  O   LEU C 185      32.119  15.207  51.080  1.00  0.00           O  \\nATOM   2705  CB  LEU C 185      33.238  15.389  54.126  1.00  0.00           C  \\nATOM   2706  CG  LEU C 185      34.263  16.119  55.005  1.00  0.00           C  \\nATOM   2707  CD1 LEU C 185      34.846  15.141  55.972  1.00  0.00           C  \\nATOM   2708  CD2 LEU C 185      35.357  16.706  54.158  1.00  0.00           C  \\nATOM   2709  H   LEU C 185      31.416  16.898  54.815  1.00  0.00           H  \\nATOM   2710  HA  LEU C 185      32.967  16.861  52.720  1.00  0.00           H  \\nATOM   2711 1HB  LEU C 185      32.577  14.835  54.790  1.00  0.00           H  \\nATOM   2712 2HB  LEU C 185      33.778  14.671  53.510  1.00  0.00           H  \\nATOM   2713  HG  LEU C 185      33.779  16.908  55.548  1.00  0.00           H  \\nATOM   2714 1HD1 LEU C 185      35.343  14.356  55.420  1.00  0.00           H  \\nATOM   2715 2HD1 LEU C 185      35.544  15.643  56.586  1.00  0.00           H  \\nATOM   2716 3HD1 LEU C 185      34.052  14.720  56.589  1.00  0.00           H  \\nATOM   2717 1HD2 LEU C 185      35.857  15.908  53.623  1.00  0.00           H  \\nATOM   2718 2HD2 LEU C 185      34.938  17.419  53.452  1.00  0.00           H  \\nATOM   2719 3HD2 LEU C 185      36.071  17.206  54.804  1.00  0.00           H  \\nATOM   2720  N   LYS C 186      30.497  14.853  52.596  1.00  0.00           N  \\nATOM   2721  CA  LYS C 186      29.722  13.982  51.716  1.00  0.00           C  \\nATOM   2722  C   LYS C 186      29.235  14.729  50.477  1.00  0.00           C  \\nATOM   2723  O   LYS C 186      29.327  14.221  49.362  1.00  0.00           O  \\nATOM   2724  CB  LYS C 186      28.554  13.308  52.460  1.00  0.00           C  \\nATOM   2725  CG  LYS C 186      29.010  12.189  53.440  1.00  0.00           C  \\nATOM   2726  CD  LYS C 186      27.874  11.289  53.933  1.00  0.00           C  \\nATOM   2727  CE  LYS C 186      27.061  11.900  55.046  1.00  0.00           C  \\nATOM   2728  NZ  LYS C 186      26.047  10.914  55.579  1.00  0.00           N  \\nATOM   2729  H   LYS C 186      30.142  14.978  53.369  1.00  0.00           H  \\nATOM   2730  HA  LYS C 186      30.288  13.251  51.420  1.00  0.00           H  \\nATOM   2731 1HB  LYS C 186      28.007  14.055  53.036  1.00  0.00           H  \\nATOM   2732 2HB  LYS C 186      27.863  12.876  51.740  1.00  0.00           H  \\nATOM   2733 1HG  LYS C 186      29.715  11.561  52.952  1.00  0.00           H  \\nATOM   2734 2HG  LYS C 186      29.498  12.645  54.300  1.00  0.00           H  \\nATOM   2735 1HD  LYS C 186      27.207  11.064  53.102  1.00  0.00           H  \\nATOM   2736 2HD  LYS C 186      28.298  10.347  54.299  1.00  0.00           H  \\nATOM   2737 1HE  LYS C 186      27.724  12.191  55.847  1.00  0.00           H  \\nATOM   2738 2HE  LYS C 186      26.542  12.785  54.681  1.00  0.00           H  \\nATOM   2739 1HZ  LYS C 186      26.516  10.051  55.937  1.00  0.00           H  \\nATOM   2740 2HZ  LYS C 186      25.515  11.338  56.332  1.00  0.00           H  \\nATOM   2741 3HZ  LYS C 186      25.420  10.653  54.839  1.00  0.00           H  \\nATOM   2742  N   VAL C 187      28.720  15.937  50.677  1.00  0.00           N  \\nATOM   2743  CA  VAL C 187      28.226  16.748  49.570  1.00  0.00           C  \\nATOM   2744  C   VAL C 187      29.378  17.207  48.674  1.00  0.00           C  \\nATOM   2745  O   VAL C 187      29.207  17.346  47.463  1.00  0.00           O  \\nATOM   2746  CB  VAL C 187      27.372  17.912  50.071  1.00  0.00           C  \\nATOM   2747  CG1 VAL C 187      27.000  18.842  48.915  1.00  0.00           C  \\nATOM   2748  CG2 VAL C 187      26.115  17.314  50.671  1.00  0.00           C  \\nATOM   2749  H   VAL C 187      28.645  16.312  51.447  1.00  0.00           H  \\nATOM   2750  HA  VAL C 187      27.627  16.208  49.030  1.00  0.00           H  \\nATOM   2751  HB  VAL C 187      27.925  18.493  50.818  1.00  0.00           H  \\nATOM   2752 1HG1 VAL C 187      26.374  19.660  49.285  1.00  0.00           H  \\nATOM   2753 2HG1 VAL C 187      27.898  19.250  48.460  1.00  0.00           H  \\nATOM   2754 3HG1 VAL C 187      26.448  18.274  48.177  1.00  0.00           H  \\nATOM   2755 1HG2 VAL C 187      26.384  16.657  51.503  1.00  0.00           H  \\nATOM   2756 2HG2 VAL C 187      25.467  18.081  51.018  1.00  0.00           H  \\nATOM   2757 3HG2 VAL C 187      25.592  16.739  49.911  1.00  0.00           H  \\nATOM   2758  N   ILE C 188      30.552  17.422  49.263  1.00  0.00           N  \\nATOM   2759  CA  ILE C 188      31.732  17.781  48.481  1.00  0.00           C  \\nATOM   2760  C   ILE C 188      32.087  16.643  47.527  1.00  0.00           C  \\nATOM   2761  O   ILE C 188      32.427  16.884  46.368  1.00  0.00           O  \\nATOM   2762  CB  ILE C 188      32.976  18.113  49.344  1.00  0.00           C  \\nATOM   2763  CG1 ILE C 188      32.803  19.443  50.080  1.00  0.00           C  \\nATOM   2764  CG2 ILE C 188      34.213  18.108  48.487  1.00  0.00           C  \\nATOM   2765  CD1 ILE C 188      33.846  19.695  51.162  1.00  0.00           C  \\nATOM   2766  H   ILE C 188      30.691  17.367  50.109  1.00  0.00           H  \\nATOM   2767  HA  ILE C 188      31.530  18.565  47.948  1.00  0.00           H  \\nATOM   2768  HB  ILE C 188      33.086  17.350  50.109  1.00  0.00           H  \\nATOM   2769 1HG1 ILE C 188      32.870  20.249  49.351  1.00  0.00           H  \\nATOM   2770 2HG1 ILE C 188      31.829  19.468  50.521  1.00  0.00           H  \\nATOM   2771 1HG2 ILE C 188      35.069  18.319  49.100  1.00  0.00           H  \\nATOM   2772 2HG2 ILE C 188      34.329  17.134  48.022  1.00  0.00           H  \\nATOM   2773 3HG2 ILE C 188      34.126  18.849  47.735  1.00  0.00           H  \\nATOM   2774 1HD1 ILE C 188      33.659  20.663  51.627  1.00  0.00           H  \\nATOM   2775 2HD1 ILE C 188      33.778  18.917  51.911  1.00  0.00           H  \\nATOM   2776 3HD1 ILE C 188      34.834  19.689  50.734  1.00  0.00           H  \\nATOM   2777  N   VAL C 189      31.987  15.405  48.005  1.00  0.00           N  \\nATOM   2778  CA  VAL C 189      32.248  14.246  47.155  1.00  0.00           C  \\nATOM   2779  C   VAL C 189      31.178  14.121  46.075  1.00  0.00           C  \\nATOM   2780  O   VAL C 189      31.471  13.728  44.948  1.00  0.00           O  \\nATOM   2781  CB  VAL C 189      32.366  12.964  47.986  1.00  0.00           C  \\nATOM   2782  CG1 VAL C 189      32.520  11.741  47.049  1.00  0.00           C  \\nATOM   2783  CG2 VAL C 189      33.561  13.113  48.923  1.00  0.00           C  \\nATOM   2784  H   VAL C 189      31.771  15.210  48.814  1.00  0.00           H  \\nATOM   2785  HA  VAL C 189      33.105  14.364  46.716  1.00  0.00           H  \\nATOM   2786  HB  VAL C 189      31.455  12.822  48.572  1.00  0.00           H  \\nATOM   2787 1HG1 VAL C 189      32.603  10.830  47.640  1.00  0.00           H  \\nATOM   2788 2HG1 VAL C 189      31.648  11.672  46.396  1.00  0.00           H  \\nATOM   2789 3HG1 VAL C 189      33.420  11.856  46.442  1.00  0.00           H  \\nATOM   2790 1HG2 VAL C 189      33.678  12.223  49.530  1.00  0.00           H  \\nATOM   2791 2HG2 VAL C 189      34.434  13.266  48.346  1.00  0.00           H  \\nATOM   2792 3HG2 VAL C 189      33.419  13.962  49.561  1.00  0.00           H  \\nATOM   2793  N   LYS C 190      29.938  14.453  46.418  1.00  0.00           N  \\nATOM   2794  CA  LYS C 190      28.869  14.462  45.427  1.00  0.00           C  \\nATOM   2795  C   LYS C 190      29.154  15.518  44.362  1.00  0.00           C  \\nATOM   2796  O   LYS C 190      28.932  15.289  43.176  1.00  0.00           O  \\nATOM   2797  CB  LYS C 190      27.504  14.752  46.057  1.00  0.00           C  \\nATOM   2798  CG  LYS C 190      26.943  13.627  46.906  1.00  0.00           C  \\nATOM   2799  CD  LYS C 190      25.623  14.027  47.555  1.00  0.00           C  \\nATOM   2800  CE  LYS C 190      24.501  14.087  46.531  1.00  0.00           C  \\nATOM   2801  NZ  LYS C 190      24.139  12.727  46.028  1.00  0.00           N  \\nATOM   2802  H   LYS C 190      29.690  14.675  47.211  1.00  0.00           H  \\nATOM   2803  HA  LYS C 190      28.831  13.596  44.991  1.00  0.00           H  \\nATOM   2804 1HB  LYS C 190      27.567  15.640  46.668  1.00  0.00           H  \\nATOM   2805 2HB  LYS C 190      26.786  14.961  45.267  1.00  0.00           H  \\nATOM   2806 1HG  LYS C 190      26.786  12.747  46.282  1.00  0.00           H  \\nATOM   2807 2HG  LYS C 190      27.660  13.374  47.683  1.00  0.00           H  \\nATOM   2808 1HD  LYS C 190      25.366  13.308  48.334  1.00  0.00           H  \\nATOM   2809 2HD  LYS C 190      25.729  15.009  48.014  1.00  0.00           H  \\nATOM   2810 1HE  LYS C 190      23.623  14.541  46.991  1.00  0.00           H  \\nATOM   2811 2HE  LYS C 190      24.817  14.706  45.688  1.00  0.00           H  \\nATOM   2812 1HZ  LYS C 190      23.392  12.803  45.353  1.00  0.00           H  \\nATOM   2813 2HZ  LYS C 190      24.948  12.302  45.589  1.00  0.00           H  \\nATOM   2814 3HZ  LYS C 190      23.835  12.152  46.801  1.00  0.00           H  \\nATOM   2815  N   ALA C 191      29.655  16.672  44.793  1.00  0.00           N  \\nATOM   2816  CA  ALA C 191      29.996  17.745  43.866  1.00  0.00           C  \\nATOM   2817  C   ALA C 191      31.127  17.307  42.944  1.00  0.00           C  \\nATOM   2818  O   ALA C 191      31.067  17.521  41.737  1.00  0.00           O  \\nATOM   2819  CB  ALA C 191      30.403  19.000  44.627  1.00  0.00           C  \\nATOM   2820  H   ALA C 191      29.807  16.858  45.618  1.00  0.00           H  \\nATOM   2821  HA  ALA C 191      29.222  17.949  43.318  1.00  0.00           H  \\nATOM   2822 1HB  ALA C 191      30.639  19.789  43.920  1.00  0.00           H  \\nATOM   2823 2HB  ALA C 191      29.593  19.320  45.274  1.00  0.00           H  \\nATOM   2824 3HB  ALA C 191      31.276  18.787  45.232  1.00  0.00           H  \\nATOM   2825  N   ILE C 192      32.154  16.685  43.518  1.00  0.00           N  \\nATOM   2826  CA  ILE C 192      33.290  16.206  42.738  1.00  0.00           C  \\nATOM   2827  C   ILE C 192      32.846  15.113  41.771  1.00  0.00           C  \\nATOM   2828  O   ILE C 192      33.307  15.065  40.633  1.00  0.00           O  \\nATOM   2829  CB  ILE C 192      34.436  15.702  43.627  1.00  0.00           C  \\nATOM   2830  CG1 ILE C 192      35.055  16.902  44.358  1.00  0.00           C  \\nATOM   2831  CG2 ILE C 192      35.487  14.963  42.774  1.00  0.00           C  \\nATOM   2832  CD1 ILE C 192      35.993  16.544  45.451  1.00  0.00           C  \\nATOM   2833  H   ILE C 192      32.216  16.528  44.361  1.00  0.00           H  \\nATOM   2834  HA  ILE C 192      33.647  16.942  42.216  1.00  0.00           H  \\nATOM   2835  HB  ILE C 192      34.032  15.023  44.381  1.00  0.00           H  \\nATOM   2836 1HG1 ILE C 192      35.597  17.507  43.635  1.00  0.00           H  \\nATOM   2837 2HG1 ILE C 192      34.253  17.506  44.780  1.00  0.00           H  \\nATOM   2838 1HG2 ILE C 192      36.284  14.606  43.418  1.00  0.00           H  \\nATOM   2839 2HG2 ILE C 192      35.017  14.122  42.274  1.00  0.00           H  \\nATOM   2840 3HG2 ILE C 192      35.897  15.642  42.025  1.00  0.00           H  \\nATOM   2841 1HD1 ILE C 192      36.357  17.466  45.875  1.00  0.00           H  \\nATOM   2842 2HD1 ILE C 192      35.468  15.962  46.208  1.00  0.00           H  \\nATOM   2843 3HD1 ILE C 192      36.820  15.967  45.056  1.00  0.00           H  \\nATOM   2844  N   GLU C 193      31.952  14.237  42.224  1.00  0.00           N  \\nATOM   2845  CA  GLU C 193      31.357  13.232  41.348  1.00  0.00           C  \\nATOM   2846  C   GLU C 193      30.676  13.909  40.167  1.00  0.00           C  \\nATOM   2847  O   GLU C 193      30.931  13.574  39.011  1.00  0.00           O  \\nATOM   2848  CB  GLU C 193      30.385  12.310  42.093  1.00  0.00           C  \\nATOM   2849  CG  GLU C 193      29.789  11.206  41.199  1.00  0.00           C  \\nATOM   2850  CD  GLU C 193      28.956  10.181  41.934  1.00  0.00           C  \\nATOM   2851  OE1 GLU C 193      28.453  10.458  42.994  1.00  0.00           O  \\nATOM   2852  OE2 GLU C 193      28.886   9.068  41.447  1.00  0.00           O  \\nATOM   2853  H   GLU C 193      31.674  14.204  43.037  1.00  0.00           H  \\nATOM   2854  HA  GLU C 193      32.054  12.650  41.007  1.00  0.00           H  \\nATOM   2855 1HB  GLU C 193      30.898  11.838  42.931  1.00  0.00           H  \\nATOM   2856 2HB  GLU C 193      29.565  12.893  42.499  1.00  0.00           H  \\nATOM   2857 1HG  GLU C 193      29.155  11.689  40.455  1.00  0.00           H  \\nATOM   2858 2HG  GLU C 193      30.584  10.704  40.667  1.00  0.00           H  \\nATOM   2859  N   ALA C 194      29.803  14.862  40.471  1.00  0.00           N  \\nATOM   2860  CA  ALA C 194      29.082  15.592  39.442  1.00  0.00           C  \\nATOM   2861  C   ALA C 194      30.065  16.299  38.517  1.00  0.00           C  \\nATOM   2862  O   ALA C 194      29.849  16.373  37.309  1.00  0.00           O  \\nATOM   2863  CB  ALA C 194      28.124  16.572  40.084  1.00  0.00           C  \\nATOM   2864  H   ALA C 194      29.611  15.106  41.273  1.00  0.00           H  \\nATOM   2865  HA  ALA C 194      28.562  14.967  38.913  1.00  0.00           H  \\nATOM   2866 1HB  ALA C 194      27.567  17.112  39.317  1.00  0.00           H  \\nATOM   2867 2HB  ALA C 194      27.433  16.020  40.716  1.00  0.00           H  \\nATOM   2868 3HB  ALA C 194      28.684  17.267  40.692  1.00  0.00           H  \\nATOM   2869  N   SER C 195      31.155  16.802  39.088  1.00  0.00           N  \\nATOM   2870  CA  SER C 195      32.141  17.535  38.309  1.00  0.00           C  \\nATOM   2871  C   SER C 195      32.848  16.611  37.328  1.00  0.00           C  \\nATOM   2872  O   SER C 195      33.021  16.957  36.162  1.00  0.00           O  \\nATOM   2873  CB  SER C 195      33.136  18.198  39.224  1.00  0.00           C  \\nATOM   2874  OG  SER C 195      34.081  18.941  38.506  1.00  0.00           O  \\nATOM   2875  H   SER C 195      31.344  16.732  39.924  1.00  0.00           H  \\nATOM   2876  HA  SER C 195      31.691  18.227  37.800  1.00  0.00           H  \\nATOM   2877 1HB  SER C 195      32.608  18.846  39.924  1.00  0.00           H  \\nATOM   2878 2HB  SER C 195      33.643  17.440  39.804  1.00  0.00           H  \\nATOM   2879  HG  SER C 195      34.705  19.248  39.164  1.00  0.00           H  \\nATOM   2880  N   VAL C 196      33.256  15.438  37.801  1.00  0.00           N  \\nATOM   2881  CA  VAL C 196      33.914  14.461  36.941  1.00  0.00           C  \\nATOM   2882  C   VAL C 196      32.963  14.016  35.837  1.00  0.00           C  \\nATOM   2883  O   VAL C 196      33.370  13.853  34.688  1.00  0.00           O  \\nATOM   2884  CB  VAL C 196      34.408  13.238  37.760  1.00  0.00           C  \\nATOM   2885  CG1 VAL C 196      34.903  12.120  36.822  1.00  0.00           C  \\nATOM   2886  CG2 VAL C 196      35.565  13.668  38.683  1.00  0.00           C  \\nATOM   2887  H   VAL C 196      33.164  15.183  38.618  1.00  0.00           H  \\nATOM   2888  HA  VAL C 196      34.686  14.875  36.525  1.00  0.00           H  \\nATOM   2889  HB  VAL C 196      33.576  12.849  38.359  1.00  0.00           H  \\nATOM   2890 1HG1 VAL C 196      35.242  11.268  37.417  1.00  0.00           H  \\nATOM   2891 2HG1 VAL C 196      34.090  11.806  36.170  1.00  0.00           H  \\nATOM   2892 3HG1 VAL C 196      35.732  12.493  36.220  1.00  0.00           H  \\nATOM   2893 1HG2 VAL C 196      35.907  12.813  39.266  1.00  0.00           H  \\nATOM   2894 2HG2 VAL C 196      36.379  14.046  38.078  1.00  0.00           H  \\nATOM   2895 3HG2 VAL C 196      35.233  14.447  39.351  1.00  0.00           H  \\nATOM   2896  N   GLU C 197      31.693  13.832  36.185  1.00  0.00           N  \\nATOM   2897  CA  GLU C 197      30.694  13.422  35.205  1.00  0.00           C  \\nATOM   2898  C   GLU C 197      30.557  14.486  34.123  1.00  0.00           C  \\nATOM   2899  O   GLU C 197      30.467  14.168  32.937  1.00  0.00           O  \\nATOM   2900  CB  GLU C 197      29.324  13.223  35.874  1.00  0.00           C  \\nATOM   2901  CG  GLU C 197      28.147  12.835  34.929  1.00  0.00           C  \\nATOM   2902  CD  GLU C 197      28.176  11.432  34.373  1.00  0.00           C  \\nATOM   2903  OE1 GLU C 197      28.815  10.587  34.945  1.00  0.00           O  \\nATOM   2904  OE2 GLU C 197      27.540  11.206  33.371  1.00  0.00           O  \\nATOM   2905  H   GLU C 197      31.385  13.938  36.981  1.00  0.00           H  \\nATOM   2906  HA  GLU C 197      30.981  12.597  34.784  1.00  0.00           H  \\nATOM   2907 1HB  GLU C 197      29.408  12.453  36.641  1.00  0.00           H  \\nATOM   2908 2HB  GLU C 197      29.038  14.140  36.377  1.00  0.00           H  \\nATOM   2909 1HG  GLU C 197      27.216  12.968  35.478  1.00  0.00           H  \\nATOM   2910 2HG  GLU C 197      28.138  13.542  34.096  1.00  0.00           H  \\nATOM   2911  N   ASN C 198      30.533  15.749  34.515  1.00  0.00           N  \\nATOM   2912  CA  ASN C 198      30.490  16.833  33.563  1.00  0.00           C  \\nATOM   2913  C   ASN C 198      31.704  16.847  32.657  1.00  0.00           C  \\nATOM   2914  O   ASN C 198      31.606  17.049  31.503  1.00  0.00           O  \\nATOM   2915  CB  ASN C 198      30.342  18.170  34.264  1.00  0.00           C  \\nATOM   2916  CG  ASN C 198      30.052  19.277  33.285  1.00  0.00           C  \\nATOM   2917  OD1 ASN C 198      28.994  19.298  32.674  1.00  0.00           O  \\nATOM   2918  ND2 ASN C 198      30.973  20.159  33.097  1.00  0.00           N  \\nATOM   2919  H   ASN C 198      30.530  15.995  35.330  1.00  0.00           H  \\nATOM   2920  HA  ASN C 198      29.708  16.706  33.001  1.00  0.00           H  \\nATOM   2921 1HB  ASN C 198      29.540  18.116  34.999  1.00  0.00           H  \\nATOM   2922 2HB  ASN C 198      31.255  18.390  34.803  1.00  0.00           H  \\nATOM   2923 1HD2 ASN C 198      31.820  20.097  33.639  1.00  0.00           H  \\nATOM   2924 2HD2 ASN C 198      30.845  20.886  32.424  1.00  0.00           H  \\nATOM   2925  N   GLN C 199      32.858  16.625  33.209  1.00  0.00           N  \\nATOM   2926  CA  GLN C 199      34.076  16.648  32.415  1.00  0.00           C  \\nATOM   2927  C   GLN C 199      34.110  15.478  31.436  1.00  0.00           C  \\nATOM   2928  O   GLN C 199      34.723  15.573  30.375  1.00  0.00           O  \\nATOM   2929  CB  GLN C 199      35.260  16.645  33.369  1.00  0.00           C  \\nATOM   2930  CG  GLN C 199      35.356  17.961  34.181  1.00  0.00           C  \\nATOM   2931  CD  GLN C 199      36.447  17.978  35.231  1.00  0.00           C  \\nATOM   2932  OE1 GLN C 199      37.620  17.835  34.897  1.00  0.00           O  \\nATOM   2933  NE2 GLN C 199      36.106  18.143  36.509  1.00  0.00           N  \\nATOM   2934  H   GLN C 199      32.981  16.461  34.044  1.00  0.00           H  \\nATOM   2935  HA  GLN C 199      34.099  17.469  31.899  1.00  0.00           H  \\nATOM   2936 1HB  GLN C 199      35.159  15.814  34.069  1.00  0.00           H  \\nATOM   2937 2HB  GLN C 199      36.168  16.502  32.811  1.00  0.00           H  \\nATOM   2938 1HG  GLN C 199      35.563  18.771  33.485  1.00  0.00           H  \\nATOM   2939 2HG  GLN C 199      34.412  18.154  34.665  1.00  0.00           H  \\nATOM   2940 1HE2 GLN C 199      36.823  18.184  37.210  1.00  0.00           H  \\nATOM   2941 2HE2 GLN C 199      35.149  18.259  36.824  1.00  0.00           H  \\nATOM   2942  N   ARG C 200      33.432  14.385  31.777  1.00  0.00           N  \\nATOM   2943  CA  ARG C 200      33.352  13.245  30.871  1.00  0.00           C  \\nATOM   2944  C   ARG C 200      32.556  13.615  29.633  1.00  0.00           C  \\nATOM   2945  O   ARG C 200      32.976  13.342  28.511  1.00  0.00           O  \\nATOM   2946  CB  ARG C 200      32.730  12.030  31.547  1.00  0.00           C  \\nATOM   2947  CG  ARG C 200      33.671  11.383  32.453  1.00  0.00           C  \\nATOM   2948  CD  ARG C 200      33.236  10.084  33.053  1.00  0.00           C  \\nATOM   2949  NE  ARG C 200      32.380  10.250  34.198  1.00  0.00           N  \\nATOM   2950  CZ  ARG C 200      31.772   9.238  34.847  1.00  0.00           C  \\nATOM   2951  NH1 ARG C 200      31.932   8.001  34.433  1.00  0.00           N  \\nATOM   2952  NH2 ARG C 200      31.016   9.475  35.896  1.00  0.00           N  \\nATOM   2953  H   ARG C 200      33.013  14.280  32.521  1.00  0.00           H  \\nATOM   2954  HA  ARG C 200      34.248  13.000  30.591  1.00  0.00           H  \\nATOM   2955 1HB  ARG C 200      31.856  12.318  32.116  1.00  0.00           H  \\nATOM   2956 2HB  ARG C 200      32.417  11.307  30.797  1.00  0.00           H  \\nATOM   2957 1HG  ARG C 200      34.534  11.248  31.893  1.00  0.00           H  \\nATOM   2958 2HG  ARG C 200      33.888  12.053  33.259  1.00  0.00           H  \\nATOM   2959 1HD  ARG C 200      32.690   9.514  32.305  1.00  0.00           H  \\nATOM   2960 2HD  ARG C 200      34.112   9.520  33.365  1.00  0.00           H  \\nATOM   2961  HE  ARG C 200      32.232  11.189  34.544  1.00  0.00           H  \\nATOM   2962 1HH1 ARG C 200      32.500   7.808  33.626  1.00  0.00           H  \\nATOM   2963 2HH1 ARG C 200      31.468   7.247  34.920  1.00  0.00           H  \\nATOM   2964 1HH2 ARG C 200      30.869  10.423  36.214  1.00  0.00           H  \\nATOM   2965 2HH2 ARG C 200      30.548   8.719  36.365  1.00  0.00           H  \\nATOM   2966  N   ILE C 201      31.409  14.248  29.845  1.00  0.00           N  \\nATOM   2967  CA  ILE C 201      30.556  14.665  28.744  1.00  0.00           C  \\nATOM   2968  C   ILE C 201      31.278  15.712  27.900  1.00  0.00           C  \\nATOM   2969  O   ILE C 201      31.153  15.726  26.677  1.00  0.00           O  \\nATOM   2970  CB  ILE C 201      29.246  15.296  29.233  1.00  0.00           C  \\nATOM   2971  CG1 ILE C 201      28.391  14.286  29.941  1.00  0.00           C  \\nATOM   2972  CG2 ILE C 201      28.473  15.820  28.004  1.00  0.00           C  \\nATOM   2973  CD1 ILE C 201      27.247  14.926  30.675  1.00  0.00           C  \\nATOM   2974  H   ILE C 201      31.102  14.448  30.623  1.00  0.00           H  \\nATOM   2975  HA  ILE C 201      30.367  13.899  28.180  1.00  0.00           H  \\nATOM   2976  HB  ILE C 201      29.460  16.113  29.929  1.00  0.00           H  \\nATOM   2977 1HG1 ILE C 201      27.997  13.591  29.216  1.00  0.00           H  \\nATOM   2978 2HG1 ILE C 201      29.002  13.733  30.664  1.00  0.00           H  \\nATOM   2979 1HG2 ILE C 201      27.542  16.267  28.325  1.00  0.00           H  \\nATOM   2980 2HG2 ILE C 201      29.070  16.565  27.480  1.00  0.00           H  \\nATOM   2981 3HG2 ILE C 201      28.258  14.991  27.325  1.00  0.00           H  \\nATOM   2982 1HD1 ILE C 201      26.652  14.162  31.174  1.00  0.00           H  \\nATOM   2983 2HD1 ILE C 201      27.653  15.615  31.418  1.00  0.00           H  \\nATOM   2984 3HD1 ILE C 201      26.631  15.471  29.978  1.00  0.00           H  \\nATOM   2985  N   SER C 202      32.040  16.580  28.559  1.00  0.00           N  \\nATOM   2986  CA  SER C 202      32.816  17.601  27.861  1.00  0.00           C  \\nATOM   2987  C   SER C 202      33.842  16.968  26.923  1.00  0.00           C  \\nATOM   2988  O   SER C 202      33.967  17.371  25.765  1.00  0.00           O  \\nATOM   2989  CB  SER C 202      33.530  18.491  28.852  1.00  0.00           C  \\nATOM   2990  OG  SER C 202      34.315  19.480  28.209  1.00  0.00           O  \\nATOM   2991  H   SER C 202      32.125  16.599  29.414  1.00  0.00           H  \\nATOM   2992  HA  SER C 202      32.216  18.144  27.327  1.00  0.00           H  \\nATOM   2993 1HB  SER C 202      32.795  18.965  29.505  1.00  0.00           H  \\nATOM   2994 2HB  SER C 202      34.167  17.883  29.475  1.00  0.00           H  \\nATOM   2995  HG  SER C 202      34.748  19.956  28.922  1.00  0.00           H  \\nATOM   2996  N   ALA C 203      34.581  15.985  27.428  1.00  0.00           N  \\nATOM   2997  CA  ALA C 203      35.560  15.272  26.613  1.00  0.00           C  \\nATOM   2998  C   ALA C 203      34.879  14.603  25.421  1.00  0.00           C  \\nATOM   2999  O   ALA C 203      35.415  14.603  24.314  1.00  0.00           O  \\nATOM   3000  CB  ALA C 203      36.284  14.240  27.457  1.00  0.00           C  \\nATOM   3001  H   ALA C 203      34.535  15.711  28.242  1.00  0.00           H  \\nATOM   3002  HA  ALA C 203      36.209  15.906  26.271  1.00  0.00           H  \\nATOM   3003 1HB  ALA C 203      36.779  14.738  28.292  1.00  0.00           H  \\nATOM   3004 2HB  ALA C 203      35.559  13.520  27.843  1.00  0.00           H  \\nATOM   3005 3HB  ALA C 203      37.024  13.725  26.850  1.00  0.00           H  \\nATOM   3006  N   GLU C 204      33.695  14.041  25.657  1.00  0.00           N  \\nATOM   3007  CA  GLU C 204      32.887  13.460  24.586  1.00  0.00           C  \\nATOM   3008  C   GLU C 204      32.621  14.474  23.489  1.00  0.00           C  \\nATOM   3009  O   GLU C 204      32.772  14.179  22.305  1.00  0.00           O  \\nATOM   3010  CB  GLU C 204      31.512  12.902  25.020  1.00  0.00           C  \\nATOM   3011  CG  GLU C 204      31.494  11.613  25.847  1.00  0.00           C  \\nATOM   3012  CD  GLU C 204      30.044  11.139  26.121  1.00  0.00           C  \\nATOM   3013  OE1 GLU C 204      29.173  11.485  25.344  1.00  0.00           O  \\nATOM   3014  OE2 GLU C 204      29.806  10.473  27.106  1.00  0.00           O  \\nATOM   3015  H   GLU C 204      33.335  13.983  26.435  1.00  0.00           H  \\nATOM   3016  HA  GLU C 204      33.362  12.709  24.198  1.00  0.00           H  \\nATOM   3017 1HB  GLU C 204      30.975  13.639  25.578  1.00  0.00           H  \\nATOM   3018 2HB  GLU C 204      30.921  12.707  24.126  1.00  0.00           H  \\nATOM   3019 1HG  GLU C 204      32.032  10.834  25.309  1.00  0.00           H  \\nATOM   3020 2HG  GLU C 204      32.001  11.787  26.789  1.00  0.00           H  \\nATOM   3021  N   ASN C 205      32.264  15.675  23.875  1.00  0.00           N  \\nATOM   3022  CA  ASN C 205      31.927  16.712  22.932  1.00  0.00           C  \\nATOM   3023  C   ASN C 205      33.158  17.188  22.199  1.00  0.00           C  \\nATOM   3024  O   ASN C 205      33.099  17.588  21.094  1.00  0.00           O  \\nATOM   3025  CB  ASN C 205      31.178  17.845  23.584  1.00  0.00           C  \\nATOM   3026  CG  ASN C 205      30.609  18.787  22.539  1.00  0.00           C  \\nATOM   3027  OD1 ASN C 205      29.974  18.370  21.568  1.00  0.00           O  \\nATOM   3028  ND2 ASN C 205      30.813  20.043  22.724  1.00  0.00           N  \\nATOM   3029  H   ASN C 205      32.222  15.911  24.692  1.00  0.00           H  \\nATOM   3030  HA  ASN C 205      31.306  16.342  22.281  1.00  0.00           H  \\nATOM   3031 1HB  ASN C 205      30.367  17.450  24.200  1.00  0.00           H  \\nATOM   3032 2HB  ASN C 205      31.855  18.389  24.240  1.00  0.00           H  \\nATOM   3033 1HD2 ASN C 205      30.458  20.704  22.067  1.00  0.00           H  \\nATOM   3034 2HD2 ASN C 205      31.327  20.353  23.540  1.00  0.00           H  \\nATOM   3035  N   GLN C 206      34.289  17.113  22.836  1.00  0.00           N  \\nATOM   3036  CA  GLN C 206      35.535  17.516  22.194  1.00  0.00           C  \\nATOM   3037  C   GLN C 206      35.974  16.511  21.138  1.00  0.00           C  \\nATOM   3038  O   GLN C 206      36.583  16.889  20.139  1.00  0.00           O  \\nATOM   3039  CB  GLN C 206      36.641  17.714  23.224  1.00  0.00           C  \\nATOM   3040  CG  GLN C 206      36.407  18.902  24.121  1.00  0.00           C  \\nATOM   3041  CD  GLN C 206      37.453  19.071  25.194  1.00  0.00           C  \\nATOM   3042  OE1 GLN C 206      38.670  19.110  24.916  1.00  0.00           O  \\nATOM   3043  NE2 GLN C 206      37.008  19.195  26.452  1.00  0.00           N  \\nATOM   3044  H   GLN C 206      34.378  16.845  23.648  1.00  0.00           H  \\nATOM   3045  HA  GLN C 206      35.398  18.369  21.753  1.00  0.00           H  \\nATOM   3046 1HB  GLN C 206      36.730  16.822  23.843  1.00  0.00           H  \\nATOM   3047 2HB  GLN C 206      37.585  17.861  22.714  1.00  0.00           H  \\nATOM   3048 1HG  GLN C 206      36.424  19.797  23.502  1.00  0.00           H  \\nATOM   3049 2HG  GLN C 206      35.442  18.817  24.593  1.00  0.00           H  \\nATOM   3050 1HE2 GLN C 206      37.656  19.366  27.188  1.00  0.00           H  \\nATOM   3051 2HE2 GLN C 206      36.015  19.166  26.701  1.00  0.00           H  \\nATOM   3052  N   LYS C 207      35.676  15.234  21.365  1.00  0.00           N  \\nATOM   3053  CA  LYS C 207      35.926  14.204  20.361  1.00  0.00           C  \\nATOM   3054  C   LYS C 207      35.120  14.500  19.102  1.00  0.00           C  \\nATOM   3055  O   LYS C 207      35.652  14.488  17.991  1.00  0.00           O  \\nATOM   3056  CB  LYS C 207      35.600  12.811  20.923  1.00  0.00           C  \\nATOM   3057  CG  LYS C 207      36.620  12.249  21.982  1.00  0.00           C  \\nATOM   3058  CD  LYS C 207      36.137  10.874  22.527  1.00  0.00           C  \\nATOM   3059  CE  LYS C 207      37.058  10.252  23.631  1.00  0.00           C  \\nATOM   3060  NZ  LYS C 207      38.347   9.687  23.082  1.00  0.00           N  \\nATOM   3061  H   LYS C 207      35.327  14.938  22.093  1.00  0.00           H  \\nATOM   3062  HA  LYS C 207      36.868  14.206  20.128  1.00  0.00           H  \\nATOM   3063 1HB  LYS C 207      34.625  12.831  21.394  1.00  0.00           H  \\nATOM   3064 2HB  LYS C 207      35.547  12.099  20.103  1.00  0.00           H  \\nATOM   3065 1HG  LYS C 207      37.601  12.133  21.525  1.00  0.00           H  \\nATOM   3066 2HG  LYS C 207      36.703  12.954  22.809  1.00  0.00           H  \\nATOM   3067 1HD  LYS C 207      35.138  11.004  22.949  1.00  0.00           H  \\nATOM   3068 2HD  LYS C 207      36.066  10.171  21.699  1.00  0.00           H  \\nATOM   3069 1HE  LYS C 207      37.297  11.018  24.370  1.00  0.00           H  \\nATOM   3070 2HE  LYS C 207      36.510   9.449  24.121  1.00  0.00           H  \\nATOM   3071 1HZ  LYS C 207      38.930   9.257  23.849  1.00  0.00           H  \\nATOM   3072 2HZ  LYS C 207      38.135   8.969  22.408  1.00  0.00           H  \\nATOM   3073 3HZ  LYS C 207      38.876  10.418  22.637  1.00  0.00           H  \\nATOM   3074  N   ALA C 208      33.830  14.764  19.290  1.00  0.00           N  \\nATOM   3075  CA  ALA C 208      32.922  15.038  18.182  1.00  0.00           C  \\nATOM   3076  C   ALA C 208      33.345  16.279  17.406  1.00  0.00           C  \\nATOM   3077  O   ALA C 208      33.284  16.306  16.178  1.00  0.00           O  \\nATOM   3078  CB  ALA C 208      31.526  15.250  18.738  1.00  0.00           C  \\nATOM   3079  H   ALA C 208      33.452  14.791  20.062  1.00  0.00           H  \\nATOM   3080  HA  ALA C 208      32.933  14.285  17.571  1.00  0.00           H  \\nATOM   3081 1HB  ALA C 208      30.842  15.433  17.924  1.00  0.00           H  \\nATOM   3082 2HB  ALA C 208      31.221  14.366  19.287  1.00  0.00           H  \\nATOM   3083 3HB  ALA C 208      31.530  16.108  19.415  1.00  0.00           H  \\nATOM   3084  N   LEU C 209      33.772  17.308  18.129  1.00  0.00           N  \\nATOM   3085  CA  LEU C 209      34.193  18.553  17.503  1.00  0.00           C  \\nATOM   3086  C   LEU C 209      35.456  18.353  16.662  1.00  0.00           C  \\nATOM   3087  O   LEU C 209      35.676  19.069  15.686  1.00  0.00           O  \\nATOM   3088  CB  LEU C 209      34.429  19.653  18.562  1.00  0.00           C  \\nATOM   3089  CG  LEU C 209      33.154  20.224  19.280  1.00  0.00           C  \\nATOM   3090  CD1 LEU C 209      33.571  21.157  20.439  1.00  0.00           C  \\nATOM   3091  CD2 LEU C 209      32.324  20.985  18.275  1.00  0.00           C  \\nATOM   3092  H   LEU C 209      33.828  17.309  18.987  1.00  0.00           H  \\nATOM   3093  HA  LEU C 209      33.488  18.862  16.913  1.00  0.00           H  \\nATOM   3094 1HB  LEU C 209      35.080  19.238  19.333  1.00  0.00           H  \\nATOM   3095 2HB  LEU C 209      34.949  20.485  18.089  1.00  0.00           H  \\nATOM   3096  HG  LEU C 209      32.560  19.412  19.678  1.00  0.00           H  \\nATOM   3097 1HD1 LEU C 209      32.679  21.553  20.928  1.00  0.00           H  \\nATOM   3098 2HD1 LEU C 209      34.160  20.592  21.162  1.00  0.00           H  \\nATOM   3099 3HD1 LEU C 209      34.162  21.976  20.048  1.00  0.00           H  \\nATOM   3100 1HD2 LEU C 209      32.903  21.815  17.867  1.00  0.00           H  \\nATOM   3101 2HD2 LEU C 209      32.032  20.316  17.472  1.00  0.00           H  \\nATOM   3102 3HD2 LEU C 209      31.441  21.370  18.769  1.00  0.00           H  \\nTER                                                                             \\nVirtualRootMover_root 210\\n                                                      \\n\",\"pdb\");\n\tviewer_16234223362302048.zoomTo();\n\tviewer_16234223362302048.setStyle({\"cartoon\": {\"color\": \"spectrum\"}, \"stick\": {\"colorscheme\": \"blackCarbon\", \"radius\": \"0.1\"}});\n\tviewer_16234223362302048.setStyle({\"resi\": [7, 8, 11, 12, 14, 15, 18, 19, 22, 25, 26, 29, 32, 33, 36, 37, 38, 39, 41, 42, 44, 45, 46, 48, 49, 51, 52, 53, 55, 56, 58, 59, 60, 62, 63, 65, 66, 67, 69, 70, 73, 77, 80, 84, 87, 88, 91, 94, 95, 98, 101, 102, 105, 110, 113, 117, 120, 123, 124, 127, 131, 134, 137, 138, 178, 179, 181, 182, 185, 186, 188, 189, 192, 193, 195, 196, 199, 200, 202, 203, 206, 207, 209, 1], \"chain\": [\"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"z\"]},{\"cartoon\": {\"color\": \"spectrum\"}, \"stick\": {\"colorscheme\": \"greyCarbon\", \"radius\": \"0.1\"}});\n\tviewer_16234223362302048.addResLabels({\"resi\": [7, 8, 11, 12, 14, 15, 18, 19, 22, 25, 26, 29, 32, 33, 36, 37, 38, 39, 41, 42, 44, 45, 46, 48, 49, 51, 52, 53, 55, 56, 58, 59, 60, 62, 63, 65, 66, 67, 69, 70, 73, 77, 80, 84, 87, 88, 91, 94, 95, 98, 101, 102, 105, 110, 113, 117, 120, 123, 124, 127, 131, 134, 137, 138, 178, 179, 181, 182, 185, 186, 188, 189, 192, 193, 195, 196, 199, 200, 202, 203, 206, 207, 209, 1], \"chain\": [\"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"z\"]},{\"fontSize\": 12, \"showBackground\": false, \"fontColor\": \"black\"});\nviewer_16234223362302048.render();\n});\n</script>",
            "text/html": [
              "<div id=\"3dmolviewer_16234223362302048\"  style=\"position: relative; width: 1200px; height: 800px\">\n",
              "        <p id=\"3dmolwarning_16234223362302048\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n",
              "        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n",
              "        </div>\n",
              "<script>\n",
              "\n",
              "var loadScriptAsync = function(uri){\n",
              "  return new Promise((resolve, reject) => {\n",
              "    var tag = document.createElement('script');\n",
              "    tag.src = uri;\n",
              "    tag.async = true;\n",
              "    tag.onload = () => {\n",
              "      resolve();\n",
              "    };\n",
              "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
              "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
              "});\n",
              "};\n",
              "\n",
              "if(typeof $3Dmolpromise === 'undefined') {\n",
              "$3Dmolpromise = null;\n",
              "  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n",
              "}\n",
              "\n",
              "var viewer_16234223362302048 = null;\n",
              "var warn = document.getElementById(\"3dmolwarning_16234223362302048\");\n",
              "if(warn) {\n",
              "    warn.parentNode.removeChild(warn);\n",
              "}\n",
              "$3Dmolpromise.then(function() {\n",
              "viewer_16234223362302048 = $3Dmol.createViewer($(\"#3dmolviewer_16234223362302048\"),{backgroundColor:\"white\"});\n",
              "viewer_16234223362302048.zoomTo();\n",
              "\tviewer_16234223362302048.addModels(\"HEADER                                            11-JUN-21   XXXX              \\nEXPDTA    THEORETICAL MODEL                                                     \\nREMARK 220                                                                      \\nREMARK 220 EXPERIMENTAL DETAILS                                                 \\nREMARK 220  EXPERIMENT TYPE                : THEORETICAL MODELLING              \\nREMARK 220  DATE OF DATA COLLECTION        : 11-JUN-21                          \\nREMARK 220                                                                      \\nREMARK 220 REMARK: MODEL GENERATED BY ROSETTA                                   \\nREMARK 220  VERSION 2020.50.post.dev+970.commits.3700df14560                    \\nATOM      1  N   ALA A   1      46.718  19.029  14.515  1.00  0.00           N  \\nATOM      2  CA  ALA A   1      45.711  17.981  14.626  1.00  0.00           C  \\nATOM      3  C   ALA A   1      46.287  16.744  15.312  1.00  0.00           C  \\nATOM      4  O   ALA A   1      45.575  16.034  16.023  1.00  0.00           O  \\nATOM      5  CB  ALA A   1      45.165  17.622  13.252  1.00  0.00           C  \\nATOM      6 1H   ALA A   1      46.320  19.828  14.064  1.00  0.00           H  \\nATOM      7 2H   ALA A   1      47.035  19.286  15.428  1.00  0.00           H  \\nATOM      8 3H   ALA A   1      47.491  18.693  13.977  1.00  0.00           H  \\nATOM      9  HA  ALA A   1      44.974  18.308  15.166  1.00  0.00           H  \\nATOM     10 1HB  ALA A   1      44.415  16.838  13.352  1.00  0.00           H  \\nATOM     11 2HB  ALA A   1      44.710  18.504  12.800  1.00  0.00           H  \\nATOM     12 3HB  ALA A   1      45.977  17.269  12.619  1.00  0.00           H  \\nATOM     13  N   ALA A   2      47.573  16.485  15.094  1.00  0.00           N  \\nATOM     14  CA  ALA A   2      48.260  15.426  15.824  1.00  0.00           C  \\nATOM     15  C   ALA A   2      48.210  15.740  17.310  1.00  0.00           C  \\nATOM     16  O   ALA A   2      48.033  14.850  18.140  1.00  0.00           O  \\nATOM     17  CB  ALA A   2      49.698  15.278  15.349  1.00  0.00           C  \\nATOM     18  H   ALA A   2      48.068  16.907  14.532  1.00  0.00           H  \\nATOM     19  HA  ALA A   2      47.807  14.582  15.672  1.00  0.00           H  \\nATOM     20 1HB  ALA A   2      50.193  14.496  15.925  1.00  0.00           H  \\nATOM     21 2HB  ALA A   2      49.707  15.011  14.293  1.00  0.00           H  \\nATOM     22 3HB  ALA A   2      50.226  16.220  15.490  1.00  0.00           H  \\nATOM     23  N   ALA A   3      48.363  17.021  17.631  1.00  0.00           N  \\nATOM     24  CA  ALA A   3      48.318  17.482  19.011  1.00  0.00           C  \\nATOM     25  C   ALA A   3      46.922  17.287  19.594  1.00  0.00           C  \\nATOM     26  O   ALA A   3      46.777  16.910  20.757  1.00  0.00           O  \\nATOM     27  CB  ALA A   3      48.735  18.943  19.099  1.00  0.00           C  \\nATOM     28  H   ALA A   3      48.495  17.650  17.059  1.00  0.00           H  \\nATOM     29  HA  ALA A   3      48.942  16.962  19.540  1.00  0.00           H  \\nATOM     30 1HB  ALA A   3      48.692  19.272  20.137  1.00  0.00           H  \\nATOM     31 2HB  ALA A   3      49.753  19.054  18.726  1.00  0.00           H  \\nATOM     32 3HB  ALA A   3      48.061  19.550  18.498  1.00  0.00           H  \\nATOM     33  N   ALA A   4      45.898  17.538  18.781  1.00  0.00           N  \\nATOM     34  CA  ALA A   4      44.513  17.367  19.215  1.00  0.00           C  \\nATOM     35  C   ALA A   4      44.255  15.920  19.614  1.00  0.00           C  \\nATOM     36  O   ALA A   4      43.737  15.652  20.696  1.00  0.00           O  \\nATOM     37  CB  ALA A   4      43.550  17.796  18.118  1.00  0.00           C  \\nATOM     38  H   ALA A   4      45.979  17.810  17.969  1.00  0.00           H  \\nATOM     39  HA  ALA A   4      44.352  17.927  19.990  1.00  0.00           H  \\nATOM     40 1HB  ALA A   4      42.524  17.654  18.458  1.00  0.00           H  \\nATOM     41 2HB  ALA A   4      43.712  18.848  17.883  1.00  0.00           H  \\nATOM     42 3HB  ALA A   4      43.723  17.195  17.226  1.00  0.00           H  \\nATOM     43  N   ALA A   5      44.611  14.995  18.729  1.00  0.00           N  \\nATOM     44  CA  ALA A   5      44.488  13.572  19.015  1.00  0.00           C  \\nATOM     45  C   ALA A   5      45.295  13.204  20.257  1.00  0.00           C  \\nATOM     46  O   ALA A   5      44.842  12.424  21.094  1.00  0.00           O  \\nATOM     47  CB  ALA A   5      44.941  12.747  17.819  1.00  0.00           C  \\nATOM     48  H   ALA A   5      44.928  15.169  17.949  1.00  0.00           H  \\nATOM     49  HA  ALA A   5      43.557  13.360  19.185  1.00  0.00           H  \\nATOM     50 1HB  ALA A   5      44.846  11.686  18.052  1.00  0.00           H  \\nATOM     51 2HB  ALA A   5      44.320  12.986  16.956  1.00  0.00           H  \\nATOM     52 3HB  ALA A   5      45.981  12.976  17.592  1.00  0.00           H  \\nATOM     53  N   ALA A   6      46.486  13.781  20.374  1.00  0.00           N  \\nATOM     54  CA  ALA A   6      47.366  13.501  21.502  1.00  0.00           C  \\nATOM     55  C   ALA A   6      46.771  14.033  22.801  1.00  0.00           C  \\nATOM     56  O   ALA A   6      46.837  13.370  23.836  1.00  0.00           O  \\nATOM     57  CB  ALA A   6      48.744  14.102  21.263  1.00  0.00           C  \\nATOM     58  H   ALA A   6      46.810  14.343  19.809  1.00  0.00           H  \\nATOM     59  HA  ALA A   6      47.470  12.541  21.591  1.00  0.00           H  \\nATOM     60 1HB  ALA A   6      49.388  13.885  22.116  1.00  0.00           H  \\nATOM     61 2HB  ALA A   6      49.178  13.670  20.361  1.00  0.00           H  \\nATOM     62 3HB  ALA A   6      48.654  15.180  21.142  1.00  0.00           H  \\nATOM     63  N   ALA A   7      46.190  15.228  22.743  1.00  0.00           N  \\nATOM     64  CA  ALA A   7      45.617  15.851  23.931  1.00  0.00           C  \\nATOM     65  C   ALA A   7      44.366  15.115  24.392  1.00  0.00           C  \\nATOM     66  O   ALA A   7      44.138  14.970  25.590  1.00  0.00           O  \\nATOM     67  CB  ALA A   7      45.299  17.314  23.661  1.00  0.00           C  \\nATOM     68  H   ALA A   7      46.114  15.698  22.027  1.00  0.00           H  \\nATOM     69  HA  ALA A   7      46.267  15.814  24.650  1.00  0.00           H  \\nATOM     70 1HB  ALA A   7      44.872  17.766  24.556  1.00  0.00           H  \\nATOM     71 2HB  ALA A   7      46.214  17.841  23.390  1.00  0.00           H  \\nATOM     72 3HB  ALA A   7      44.584  17.386  22.843  1.00  0.00           H  \\nATOM     73  N   ALA A   8      43.558  14.656  23.440  1.00  0.00           N  \\nATOM     74  CA  ALA A   8      42.375  13.868  23.763  1.00  0.00           C  \\nATOM     75  C   ALA A   8      42.787  12.599  24.490  1.00  0.00           C  \\nATOM     76  O   ALA A   8      42.157  12.197  25.469  1.00  0.00           O  \\nATOM     77  CB  ALA A   8      41.587  13.534  22.505  1.00  0.00           C  \\nATOM     78  H   ALA A   8      43.674  14.789  22.598  1.00  0.00           H  \\nATOM     79  HA  ALA A   8      41.798  14.382  24.350  1.00  0.00           H  \\nATOM     80 1HB  ALA A   8      40.716  12.934  22.770  1.00  0.00           H  \\nATOM     81 2HB  ALA A   8      41.259  14.457  22.025  1.00  0.00           H  \\nATOM     82 3HB  ALA A   8      42.218  12.973  21.818  1.00  0.00           H  \\nATOM     83  N   ALA A   9      43.856  11.977  24.006  1.00  0.00           N  \\nATOM     84  CA  ALA A   9      44.400  10.786  24.639  1.00  0.00           C  \\nATOM     85  C   ALA A   9      44.876  11.138  26.042  1.00  0.00           C  \\nATOM     86  O   ALA A   9      44.715  10.356  26.980  1.00  0.00           O  \\nATOM     87  CB  ALA A   9      45.535  10.202  23.811  1.00  0.00           C  \\nATOM     88  H   ALA A   9      44.287  12.228  23.305  1.00  0.00           H  \\nATOM     89  HA  ALA A   9      43.705  10.112  24.711  1.00  0.00           H  \\nATOM     90 1HB  ALA A   9      45.936   9.321  24.314  1.00  0.00           H  \\nATOM     91 2HB  ALA A   9      45.161   9.919  22.827  1.00  0.00           H  \\nATOM     92 3HB  ALA A   9      46.323  10.945  23.699  1.00  0.00           H  \\nATOM     93  N   ALA A  10      45.456  12.327  26.179  1.00  0.00           N  \\nATOM     94  CA  ALA A  10      45.928  12.800  27.473  1.00  0.00           C  \\nATOM     95  C   ALA A  10      44.723  13.115  28.353  1.00  0.00           C  \\nATOM     96  O   ALA A  10      44.672  12.731  29.523  1.00  0.00           O  \\nATOM     97  CB  ALA A  10      46.821  14.022  27.314  1.00  0.00           C  \\nATOM     98  H   ALA A  10      45.587  12.880  25.534  1.00  0.00           H  \\nATOM     99  HA  ALA A  10      46.445  12.100  27.902  1.00  0.00           H  \\nATOM    100 1HB  ALA A  10      47.145  14.366  28.296  1.00  0.00           H  \\nATOM    101 2HB  ALA A  10      47.693  13.760  26.715  1.00  0.00           H  \\nATOM    102 3HB  ALA A  10      46.265  14.816  26.817  1.00  0.00           H  \\nATOM    103  N   ALA A  11      43.746  13.798  27.767  1.00  0.00           N  \\nATOM    104  CA  ALA A  11      42.513  14.142  28.459  1.00  0.00           C  \\nATOM    105  C   ALA A  11      41.830  12.867  28.946  1.00  0.00           C  \\nATOM    106  O   ALA A  11      41.266  12.827  30.038  1.00  0.00           O  \\nATOM    107  CB  ALA A  11      41.588  14.939  27.551  1.00  0.00           C  \\nATOM    108  H   ALA A  11      43.775  14.077  26.954  1.00  0.00           H  \\nATOM    109  HA  ALA A  11      42.719  14.693  29.231  1.00  0.00           H  \\nATOM    110 1HB  ALA A  11      40.668  15.175  28.085  1.00  0.00           H  \\nATOM    111 2HB  ALA A  11      42.081  15.864  27.251  1.00  0.00           H  \\nATOM    112 3HB  ALA A  11      41.353  14.351  26.665  1.00  0.00           H  \\nATOM    113  N   ALA A  12      41.916  11.820  28.132  1.00  0.00           N  \\nATOM    114  CA  ALA A  12      41.329  10.527  28.460  1.00  0.00           C  \\nATOM    115  C   ALA A  12      42.018   9.891  29.669  1.00  0.00           C  \\nATOM    116  O   ALA A  12      41.355   9.346  30.550  1.00  0.00           O  \\nATOM    117  CB  ALA A  12      41.402   9.597  27.258  1.00  0.00           C  \\nATOM    118  H   ALA A  12      42.315  11.834  27.371  1.00  0.00           H  \\nATOM    119  HA  ALA A  12      40.393  10.650  28.679  1.00  0.00           H  \\nATOM    120 1HB  ALA A  12      40.956   8.636  27.514  1.00  0.00           H  \\nATOM    121 2HB  ALA A  12      40.858  10.038  26.423  1.00  0.00           H  \\nATOM    122 3HB  ALA A  12      42.443   9.449  26.975  1.00  0.00           H  \\nATOM    123  N   ALA A  13      43.346   9.967  29.708  1.00  0.00           N  \\nATOM    124  CA  ALA A  13      44.112   9.386  30.807  1.00  0.00           C  \\nATOM    125  C   ALA A  13      43.873  10.137  32.114  1.00  0.00           C  \\nATOM    126  O   ALA A  13      43.833   9.532  33.186  1.00  0.00           O  \\nATOM    127  CB  ALA A  13      45.596   9.376  30.468  1.00  0.00           C  \\nATOM    128  H   ALA A  13      43.828  10.351  29.108  1.00  0.00           H  \\nATOM    129  HA  ALA A  13      43.831   8.466  30.935  1.00  0.00           H  \\nATOM    130 1HB  ALA A  13      46.155   8.938  31.295  1.00  0.00           H  \\nATOM    131 2HB  ALA A  13      45.759   8.786  29.567  1.00  0.00           H  \\nATOM    132 3HB  ALA A  13      45.937  10.396  30.301  1.00  0.00           H  \\nATOM    133  N   ALA A  14      43.724  11.456  32.023  1.00  0.00           N  \\nATOM    134  CA  ALA A  14      43.447  12.281  33.195  1.00  0.00           C  \\nATOM    135  C   ALA A  14      42.111  11.897  33.818  1.00  0.00           C  \\nATOM    136  O   ALA A  14      41.958  11.886  35.037  1.00  0.00           O  \\nATOM    137  CB  ALA A  14      43.456  13.757  32.824  1.00  0.00           C  \\nATOM    138  H   ALA A  14      43.779  11.899  31.288  1.00  0.00           H  \\nATOM    139  HA  ALA A  14      44.142  12.139  33.857  1.00  0.00           H  \\nATOM    140 1HB  ALA A  14      43.242  14.356  33.709  1.00  0.00           H  \\nATOM    141 2HB  ALA A  14      44.436  14.027  32.431  1.00  0.00           H  \\nATOM    142 3HB  ALA A  14      42.697  13.946  32.067  1.00  0.00           H  \\nATOM    143  N   ALA A  15      41.147  11.585  32.961  1.00  0.00           N  \\nATOM    144  CA  ALA A  15      39.818  11.185  33.401  1.00  0.00           C  \\nATOM    145  C   ALA A  15      39.872   9.876  34.185  1.00  0.00           C  \\nATOM    146  O   ALA A  15      39.279   9.765  35.258  1.00  0.00           O  \\nATOM    147  CB  ALA A  15      38.883  11.052  32.208  1.00  0.00           C  \\nATOM    148  H   ALA A  15      41.240  11.598  32.106  1.00  0.00           H  \\nATOM    149  HA  ALA A  15      39.455  11.871  33.982  1.00  0.00           H  \\nATOM    150 1HB  ALA A  15      37.893  10.754  32.552  1.00  0.00           H  \\nATOM    151 2HB  ALA A  15      38.814  12.010  31.692  1.00  0.00           H  \\nATOM    152 3HB  ALA A  15      39.271  10.299  31.524  1.00  0.00           H  \\nATOM    153  N   ALA A  16      40.590   8.892  33.650  1.00  0.00           N  \\nATOM    154  CA  ALA A  16      40.716   7.594  34.307  1.00  0.00           C  \\nATOM    155  C   ALA A  16      41.311   7.758  35.696  1.00  0.00           C  \\nATOM    156  O   ALA A  16      40.893   7.094  36.646  1.00  0.00           O  \\nATOM    157  CB  ALA A  16      41.568   6.649  33.472  1.00  0.00           C  \\nATOM    158  H   ALA A  16      41.015   8.950  32.904  1.00  0.00           H  \\nATOM    159  HA  ALA A  16      39.835   7.198  34.400  1.00  0.00           H  \\nATOM    160 1HB  ALA A  16      41.657   5.691  33.983  1.00  0.00           H  \\nATOM    161 2HB  ALA A  16      41.099   6.500  32.499  1.00  0.00           H  \\nATOM    162 3HB  ALA A  16      42.559   7.080  33.334  1.00  0.00           H  \\nATOM    163  N   ALA A  17      42.288   8.649  35.808  1.00  0.00           N  \\nATOM    164  CA  ALA A  17      42.912   8.929  37.091  1.00  0.00           C  \\nATOM    165  C   ALA A  17      41.897   9.557  38.044  1.00  0.00           C  \\nATOM    166  O   ALA A  17      41.898   9.264  39.238  1.00  0.00           O  \\nATOM    167  CB  ALA A  17      44.117   9.840  36.911  1.00  0.00           C  \\nATOM    168  H   ALA A  17      42.608   9.106  35.154  1.00  0.00           H  \\nATOM    169  HA  ALA A  17      43.221   8.097  37.483  1.00  0.00           H  \\nATOM    170 1HB  ALA A  17      44.568  10.043  37.883  1.00  0.00           H  \\nATOM    171 2HB  ALA A  17      44.849   9.353  36.266  1.00  0.00           H  \\nATOM    172 3HB  ALA A  17      43.800  10.777  36.456  1.00  0.00           H  \\nATOM    173  N   ALA A  18      41.024  10.410  37.511  1.00  0.00           N  \\nATOM    174  CA  ALA A  18      39.999  11.059  38.327  1.00  0.00           C  \\nATOM    175  C   ALA A  18      39.078  10.027  38.968  1.00  0.00           C  \\nATOM    176  O   ALA A  18      38.849  10.057  40.177  1.00  0.00           O  \\nATOM    177  CB  ALA A  18      39.192  12.040  37.488  1.00  0.00           C  \\nATOM    178  H   ALA A  18      41.004  10.630  36.679  1.00  0.00           H  \\nATOM    179  HA  ALA A  18      40.431  11.559  39.037  1.00  0.00           H  \\nATOM    180 1HB  ALA A  18      38.432  12.511  38.111  1.00  0.00           H  \\nATOM    181 2HB  ALA A  18      39.856  12.804  37.084  1.00  0.00           H  \\nATOM    182 3HB  ALA A  18      38.712  11.508  36.669  1.00  0.00           H  \\nATOM    183  N   ALA A  19      38.551   9.119  38.152  1.00  0.00           N  \\nATOM    184  CA  ALA A  19      37.649   8.080  38.637  1.00  0.00           C  \\nATOM    185  C   ALA A  19      38.327   7.198  39.676  1.00  0.00           C  \\nATOM    186  O   ALA A  19      37.689   6.746  40.626  1.00  0.00           O  \\nATOM    187  CB  ALA A  19      37.143   7.235  37.477  1.00  0.00           C  \\nATOM    188  H   ALA A  19      38.702   9.083  37.306  1.00  0.00           H  \\nATOM    189  HA  ALA A  19      36.880   8.499  39.054  1.00  0.00           H  \\nATOM    190 1HB  ALA A  19      36.471   6.464  37.855  1.00  0.00           H  \\nATOM    191 2HB  ALA A  19      36.607   7.869  36.771  1.00  0.00           H  \\nATOM    192 3HB  ALA A  19      37.987   6.765  36.975  1.00  0.00           H  \\nATOM    193  N   ALA A  20      39.619   6.950  39.487  1.00  0.00           N  \\nATOM    194  CA  ALA A  20      40.389   6.158  40.438  1.00  0.00           C  \\nATOM    195  C   ALA A  20      40.457   6.878  41.780  1.00  0.00           C  \\nATOM    196  O   ALA A  20      40.299   6.264  42.835  1.00  0.00           O  \\nATOM    197  CB  ALA A  20      41.787   5.885  39.903  1.00  0.00           C  \\nATOM    198  H   ALA A  20      40.075   7.231  38.814  1.00  0.00           H  \\nATOM    199  HA  ALA A  20      39.948   5.305  40.573  1.00  0.00           H  \\nATOM    200 1HB  ALA A  20      42.348   5.301  40.633  1.00  0.00           H  \\nATOM    201 2HB  ALA A  20      41.717   5.328  38.969  1.00  0.00           H  \\nATOM    202 3HB  ALA A  20      42.299   6.829  39.725  1.00  0.00           H  \\nATOM    203  N   ALA A  21      40.691   8.186  41.729  1.00  0.00           N  \\nATOM    204  CA  ALA A  21      40.722   9.012  42.932  1.00  0.00           C  \\nATOM    205  C   ALA A  21      39.322   9.124  43.521  1.00  0.00           C  \\nATOM    206  O   ALA A  21      39.152   9.145  44.740  1.00  0.00           O  \\nATOM    207  CB  ALA A  21      41.286  10.391  42.625  1.00  0.00           C  \\nATOM    208  H   ALA A  21      40.837   8.623  41.003  1.00  0.00           H  \\nATOM    209  HA  ALA A  21      41.297   8.594  43.592  1.00  0.00           H  \\nATOM    210 1HB  ALA A  21      41.289  10.994  43.534  1.00  0.00           H  \\nATOM    211 2HB  ALA A  21      42.306  10.293  42.252  1.00  0.00           H  \\nATOM    212 3HB  ALA A  21      40.670  10.877  41.870  1.00  0.00           H  \\nATOM    213  N   ALA A  22      38.323   9.195  42.648  1.00  0.00           N  \\nATOM    214  CA  ALA A  22      36.938   9.321  43.084  1.00  0.00           C  \\nATOM    215  C   ALA A  22      36.511   8.076  43.857  1.00  0.00           C  \\nATOM    216  O   ALA A  22      35.848   8.173  44.890  1.00  0.00           O  \\nATOM    217  CB  ALA A  22      36.021   9.552  41.892  1.00  0.00           C  \\nATOM    218  H   ALA A  22      38.421   9.172  41.794  1.00  0.00           H  \\nATOM    219  HA  ALA A  22      36.859  10.086  43.675  1.00  0.00           H  \\nATOM    220 1HB  ALA A  22      34.990   9.638  42.237  1.00  0.00           H  \\nATOM    221 2HB  ALA A  22      36.311  10.472  41.383  1.00  0.00           H  \\nATOM    222 3HB  ALA A  22      36.104   8.715  41.202  1.00  0.00           H  \\nATOM    223  N   ALA A  23      36.902   6.909  43.352  1.00  0.00           N  \\nATOM    224  CA  ALA A  23      36.628   5.647  44.030  1.00  0.00           C  \\nATOM    225  C   ALA A  23      37.340   5.612  45.373  1.00  0.00           C  \\nATOM    226  O   ALA A  23      36.759   5.221  46.384  1.00  0.00           O  \\nATOM    227  CB  ALA A  23      37.054   4.468  43.168  1.00  0.00           C  \\nATOM    228  H   ALA A  23      37.331   6.822  42.612  1.00  0.00           H  \\nATOM    229  HA  ALA A  23      35.675   5.569  44.189  1.00  0.00           H  \\nATOM    230 1HB  ALA A  23      36.851   3.537  43.698  1.00  0.00           H  \\nATOM    231 2HB  ALA A  23      36.496   4.481  42.231  1.00  0.00           H  \\nATOM    232 3HB  ALA A  23      38.120   4.539  42.956  1.00  0.00           H  \\nATOM    233  N   ALA A  24      38.605   6.022  45.371  1.00  0.00           N  \\nATOM    234  CA  ALA A  24      39.387   6.108  46.596  1.00  0.00           C  \\nATOM    235  C   ALA A  24      38.691   7.053  47.566  1.00  0.00           C  \\nATOM    236  O   ALA A  24      38.549   6.755  48.751  1.00  0.00           O  \\nATOM    237  CB  ALA A  24      40.805   6.577  46.305  1.00  0.00           C  \\nATOM    238  H   ALA A  24      39.037   6.257  44.665  1.00  0.00           H  \\nATOM    239  HA  ALA A  24      39.442   5.231  47.008  1.00  0.00           H  \\nATOM    240 1HB  ALA A  24      41.363   6.650  47.238  1.00  0.00           H  \\nATOM    241 2HB  ALA A  24      41.296   5.863  45.643  1.00  0.00           H  \\nATOM    242 3HB  ALA A  24      40.773   7.554  45.825  1.00  0.00           H  \\nATOM    243  N   ALA A  25      38.241   8.188  47.041  1.00  0.00           N  \\nATOM    244  CA  ALA A  25      37.571   9.202  47.844  1.00  0.00           C  \\nATOM    245  C   ALA A  25      36.260   8.661  48.404  1.00  0.00           C  \\nATOM    246  O   ALA A  25      35.915   8.916  49.558  1.00  0.00           O  \\nATOM    247  CB  ALA A  25      37.322  10.457  47.020  1.00  0.00           C  \\nATOM    248  H   ALA A  25      38.314   8.397  46.210  1.00  0.00           H  \\nATOM    249  HA  ALA A  25      38.141   9.445  48.591  1.00  0.00           H  \\nATOM    250 1HB  ALA A  25      36.817  11.202  47.635  1.00  0.00           H  \\nATOM    251 2HB  ALA A  25      38.274  10.858  46.672  1.00  0.00           H  \\nATOM    252 3HB  ALA A  25      36.698  10.211  46.162  1.00  0.00           H  \\nATOM    253  N   ALA A  26      35.542   7.901  47.583  1.00  0.00           N  \\nATOM    254  CA  ALA A  26      34.261   7.331  47.988  1.00  0.00           C  \\nATOM    255  C   ALA A  26      34.442   6.235  49.032  1.00  0.00           C  \\nATOM    256  O   ALA A  26      33.568   6.018  49.871  1.00  0.00           O  \\nATOM    257  CB  ALA A  26      33.519   6.787  46.776  1.00  0.00           C  \\nATOM    258  H   ALA A  26      35.776   7.699  46.781  1.00  0.00           H  \\nATOM    259  HA  ALA A  26      33.715   8.030  48.380  1.00  0.00           H  \\nATOM    260 1HB  ALA A  26      32.564   6.367  47.093  1.00  0.00           H  \\nATOM    261 2HB  ALA A  26      33.342   7.594  46.065  1.00  0.00           H  \\nATOM    262 3HB  ALA A  26      34.117   6.010  46.302  1.00  0.00           H  \\nATOM    263  N   ALA A  27      35.574   5.540  48.974  1.00  0.00           N  \\nATOM    264  CA  ALA A  27      35.886   4.526  49.973  1.00  0.00           C  \\nATOM    265  C   ALA A  27      36.041   5.168  51.344  1.00  0.00           C  \\nATOM    266  O   ALA A  27      35.353   4.797  52.294  1.00  0.00           O  \\nATOM    267  CB  ALA A  27      37.149   3.767  49.592  1.00  0.00           C  \\nATOM    268  H   ALA A  27      36.176   5.637  48.368  1.00  0.00           H  \\nATOM    269  HA  ALA A  27      35.156   3.888  50.018  1.00  0.00           H  \\nATOM    270 1HB  ALA A  27      37.369   3.021  50.355  1.00  0.00           H  \\nATOM    271 2HB  ALA A  27      37.001   3.272  48.632  1.00  0.00           H  \\nATOM    272 3HB  ALA A  27      37.982   4.464  49.515  1.00  0.00           H  \\nATOM    273  N   ALA A  28      36.940   6.143  51.433  1.00  0.00           N  \\nATOM    274  CA  ALA A  28      37.251   6.792  52.700  1.00  0.00           C  \\nATOM    275  C   ALA A  28      36.052   7.536  53.281  1.00  0.00           C  \\nATOM    276  O   ALA A  28      36.018   7.830  54.475  1.00  0.00           O  \\nATOM    277  CB  ALA A  28      38.422   7.747  52.521  1.00  0.00           C  \\nATOM    278  H   ALA A  28      37.389   6.449  50.766  1.00  0.00           H  \\nATOM    279  HA  ALA A  28      37.529   6.113  53.335  1.00  0.00           H  \\nATOM    280 1HB  ALA A  28      38.656   8.219  53.476  1.00  0.00           H  \\nATOM    281 2HB  ALA A  28      39.293   7.193  52.168  1.00  0.00           H  \\nATOM    282 3HB  ALA A  28      38.160   8.512  51.793  1.00  0.00           H  \\nATOM    283  N   ALA A  29      35.071   7.838  52.438  1.00  0.00           N  \\nATOM    284  CA  ALA A  29      33.891   8.566  52.885  1.00  0.00           C  \\nATOM    285  C   ALA A  29      33.050   7.696  53.813  1.00  0.00           C  \\nATOM    286  O   ALA A  29      32.438   8.198  54.755  1.00  0.00           O  \\nATOM    287  CB  ALA A  29      33.066   9.030  51.693  1.00  0.00           C  \\nATOM    288  H   ALA A  29      35.064   7.632  51.603  1.00  0.00           H  \\nATOM    289  HA  ALA A  29      34.173   9.351  53.381  1.00  0.00           H  \\nATOM    290 1HB  ALA A  29      32.186   9.567  52.047  1.00  0.00           H  \\nATOM    291 2HB  ALA A  29      33.668   9.691  51.070  1.00  0.00           H  \\nATOM    292 3HB  ALA A  29      32.753   8.166  51.109  1.00  0.00           H  \\nATOM    293  N   ALA A  30      33.025   6.393  53.542  1.00  0.00           N  \\nATOM    294  CA  ALA A  30      32.366   5.436  54.426  1.00  0.00           C  \\nATOM    295  C   ALA A  30      33.394   4.808  55.370  1.00  0.00           C  \\nATOM    296  O   ALA A  30      33.299   3.633  55.724  1.00  0.00           O  \\nATOM    297  CB  ALA A  30      31.649   4.363  53.619  1.00  0.00           C  \\nATOM    298  H   ALA A  30      33.386   6.036  52.847  1.00  0.00           H  \\nATOM    299  HA  ALA A  30      31.708   5.902  54.964  1.00  0.00           H  \\nATOM    300 1HB  ALA A  30      31.173   3.655  54.297  1.00  0.00           H  \\nATOM    301 2HB  ALA A  30      30.891   4.828  52.988  1.00  0.00           H  \\nATOM    302 3HB  ALA A  30      32.369   3.838  52.993  1.00  0.00           H  \\nATOM    303  N   ALA A  31      34.385   5.606  55.758  1.00  0.00           N  \\nATOM    304  CA  ALA A  31      35.375   5.203  56.754  1.00  0.00           C  \\nATOM    305  C   ALA A  31      35.523   6.285  57.819  1.00  0.00           C  \\nATOM    306  O   ALA A  31      36.275   6.121  58.780  1.00  0.00           O  \\nATOM    307  CB  ALA A  31      36.716   4.916  56.093  1.00  0.00           C  \\nATOM    308  H   ALA A  31      34.508   6.401  55.453  1.00  0.00           H  \\nATOM    309  HA  ALA A  31      35.073   4.391  57.189  1.00  0.00           H  \\nATOM    310 1HB  ALA A  31      37.441   4.626  56.853  1.00  0.00           H  \\nATOM    311 2HB  ALA A  31      36.600   4.106  55.373  1.00  0.00           H  \\nATOM    312 3HB  ALA A  31      37.066   5.810  55.580  1.00  0.00           H  \\nATOM    313  N   ALA A  32      34.816   7.395  57.627  1.00  0.00           N  \\nATOM    314  CA  ALA A  32      34.634   8.394  58.669  1.00  0.00           C  \\nATOM    315  C   ALA A  32      33.207   8.290  59.192  1.00  0.00           C  \\nATOM    316  O   ALA A  32      32.893   8.793  60.267  1.00  0.00           O  \\nATOM    317  CB  ALA A  32      34.924   9.793  58.144  1.00  0.00           C  \\nATOM    318  H   ALA A  32      34.426   7.593  56.886  1.00  0.00           H  \\nATOM    319  HA  ALA A  32      35.245   8.215  59.401  1.00  0.00           H  \\nATOM    320 1HB  ALA A  32      34.759  10.521  58.939  1.00  0.00           H  \\nATOM    321 2HB  ALA A  32      35.960   9.849  57.811  1.00  0.00           H  \\nATOM    322 3HB  ALA A  32      34.262  10.012  57.308  1.00  0.00           H  \\nATOM    323  N   ALA A  33      32.345   7.639  58.415  1.00  0.00           N  \\nATOM    324  CA  ALA A  33      30.984   7.360  58.849  1.00  0.00           C  \\nATOM    325  C   ALA A  33      31.002   6.279  59.929  1.00  0.00           C  \\nATOM    326  O   ALA A  33      30.091   6.197  60.753  1.00  0.00           O  \\nATOM    327  CB  ALA A  33      30.120   6.934  57.671  1.00  0.00           C  \\nATOM    328  H   ALA A  33      32.527   7.348  57.627  1.00  0.00           H  \\nATOM    329  HA  ALA A  33      30.598   8.164  59.230  1.00  0.00           H  \\nATOM    330 1HB  ALA A  33      29.109   6.721  58.020  1.00  0.00           H  \\nATOM    331 2HB  ALA A  33      30.087   7.738  56.935  1.00  0.00           H  \\nATOM    332 3HB  ALA A  33      30.542   6.041  57.214  1.00  0.00           H  \\nATOM    333  N   ALA A  34      32.050   5.457  59.915  1.00  0.00           N  \\nATOM    334  CA  ALA A  34      32.263   4.437  60.937  1.00  0.00           C  \\nATOM    335  C   ALA A  34      32.992   5.028  62.141  1.00  0.00           C  \\nATOM    336  O   ALA A  34      32.755   4.629  63.282  1.00  0.00           O  \\nATOM    337  CB  ALA A  34      33.043   3.263  60.365  1.00  0.00           C  \\nATOM    338  H   ALA A  34      32.663   5.473  59.312  1.00  0.00           H  \\nATOM    339  HA  ALA A  34      31.404   4.099  61.237  1.00  0.00           H  \\nATOM    340 1HB  ALA A  34      33.197   2.514  61.142  1.00  0.00           H  \\nATOM    341 2HB  ALA A  34      32.483   2.821  59.541  1.00  0.00           H  \\nATOM    342 3HB  ALA A  34      34.009   3.611  60.001  1.00  0.00           H  \\nATOM    343  N   ALA A  35      33.881   5.979  61.868  1.00  0.00           N  \\nATOM    344  CA  ALA A  35      34.660   6.654  62.902  1.00  0.00           C  \\nATOM    345  C   ALA A  35      34.615   8.169  62.697  1.00  0.00           C  \\nATOM    346  O   ALA A  35      35.586   8.761  62.224  1.00  0.00           O  \\nATOM    347  CB  ALA A  35      36.098   6.155  62.899  1.00  0.00           C  \\nATOM    348  H   ALA A  35      34.055   6.257  61.073  1.00  0.00           H  \\nATOM    349  HA  ALA A  35      34.278   6.455  63.771  1.00  0.00           H  \\nATOM    350 1HB  ALA A  35      36.664   6.674  63.673  1.00  0.00           H  \\nATOM    351 2HB  ALA A  35      36.111   5.083  63.097  1.00  0.00           H  \\nATOM    352 3HB  ALA A  35      36.548   6.351  61.927  1.00  0.00           H  \\nATOM    353  N   ALA A  36      33.480   8.802  63.034  1.00  0.00           N  \\nATOM    354  CA  ALA A  36      33.349  10.257  62.895  1.00  0.00           C  \\nATOM    355  C   ALA A  36      34.242  11.024  63.870  1.00  0.00           C  \\nATOM    356  O   ALA A  36      33.740  11.731  64.744  1.00  0.00           O  \\nATOM    357  CB  ALA A  36      31.897  10.670  63.088  1.00  0.00           C  \\nATOM    358  H   ALA A  36      32.699   8.270  63.390  1.00  0.00           H  \\nATOM    359  HA  ALA A  36      33.552  10.530  61.987  1.00  0.00           H  \\nATOM    360 1HB  ALA A  36      31.808  11.751  62.973  1.00  0.00           H  \\nATOM    361 2HB  ALA A  36      31.276  10.174  62.342  1.00  0.00           H  \\nATOM    362 3HB  ALA A  36      31.567  10.383  64.085  1.00  0.00           H  \\nATOM    363  N   ALA A  37      35.554  10.871  63.721  1.00  0.00           N  \\nATOM    364  CA  ALA A  37      36.512  11.566  64.574  1.00  0.00           C  \\nATOM    365  C   ALA A  37      37.551  12.308  63.743  1.00  0.00           C  \\nATOM    366  O   ALA A  37      37.563  12.224  62.515  1.00  0.00           O  \\nATOM    367  CB  ALA A  37      37.192  10.584  65.517  1.00  0.00           C  \\nATOM    368  H   ALA A  37      35.918  10.367  63.127  1.00  0.00           H  \\nATOM    369  HA  ALA A  37      36.038  12.219  65.113  1.00  0.00           H  \\nATOM    370 1HB  ALA A  37      37.904  11.118  66.146  1.00  0.00           H  \\nATOM    371 2HB  ALA A  37      36.442  10.104  66.145  1.00  0.00           H  \\nATOM    372 3HB  ALA A  37      37.717   9.827  64.937  1.00  0.00           H  \\nATOM    373  N   ALA A  38      38.428  13.027  64.432  1.00  0.00           N  \\nATOM    374  CA  ALA A  38      39.416  13.881  63.788  1.00  0.00           C  \\nATOM    375  C   ALA A  38      40.503  13.079  63.080  1.00  0.00           C  \\nATOM    376  O   ALA A  38      40.668  11.885  63.324  1.00  0.00           O  \\nATOM    377  CB  ALA A  38      40.040  14.818  64.812  1.00  0.00           C  \\nATOM    378  H   ALA A  38      38.471  13.036  65.291  1.00  0.00           H  \\nATOM    379  HA  ALA A  38      38.971  14.432  63.125  1.00  0.00           H  \\nATOM    380 1HB  ALA A  38      40.770  15.461  64.320  1.00  0.00           H  \\nATOM    381 2HB  ALA A  38      39.262  15.432  65.265  1.00  0.00           H  \\nATOM    382 3HB  ALA A  38      40.536  14.232  65.585  1.00  0.00           H  \\nATOM    383  N   ALA A  39      41.166  13.758  62.155  1.00  0.00           N  \\nATOM    384  CA  ALA A  39      42.319  13.244  61.409  1.00  0.00           C  \\nATOM    385  C   ALA A  39      42.016  12.100  60.463  1.00  0.00           C  \\nATOM    386  O   ALA A  39      42.807  11.802  59.618  1.00  0.00           O  \\nATOM    387  CB  ALA A  39      43.402  12.808  62.385  1.00  0.00           C  \\nATOM    388  H   ALA A  39      40.955  14.558  61.929  1.00  0.00           H  \\nATOM    389  HA  ALA A  39      42.657  13.974  60.852  1.00  0.00           H  \\nATOM    390 1HB  ALA A  39      44.269  12.450  61.830  1.00  0.00           H  \\nATOM    391 2HB  ALA A  39      43.693  13.655  63.007  1.00  0.00           H  \\nATOM    392 3HB  ALA A  39      43.021  12.008  63.018  1.00  0.00           H  \\nATOM    393  N   ALA A  40      40.882  11.456  60.610  1.00  0.00           N  \\nATOM    394  CA  ALA A  40      40.291  10.669  59.535  1.00  0.00           C  \\nATOM    395  C   ALA A  40      39.620  11.649  58.582  1.00  0.00           C  \\nATOM    396  O   ALA A  40      39.622  11.462  57.366  1.00  0.00           O  \\nATOM    397  CB  ALA A  40      39.298   9.647  60.070  1.00  0.00           C  \\nATOM    398  H   ALA A  40      40.461  11.408  61.359  1.00  0.00           H  \\nATOM    399  HA  ALA A  40      40.989  10.196  59.054  1.00  0.00           H  \\nATOM    400 1HB  ALA A  40      38.851   9.104  59.238  1.00  0.00           H  \\nATOM    401 2HB  ALA A  40      39.815   8.947  60.726  1.00  0.00           H  \\nATOM    402 3HB  ALA A  40      38.517  10.159  60.630  1.00  0.00           H  \\nATOM    403  N   ALA A  41      39.041  12.696  59.162  1.00  0.00           N  \\nATOM    404  CA  ALA A  41      38.532  13.831  58.404  1.00  0.00           C  \\nATOM    405  C   ALA A  41      39.668  14.464  57.609  1.00  0.00           C  \\nATOM    406  O   ALA A  41      39.486  14.863  56.459  1.00  0.00           O  \\nATOM    407  CB  ALA A  41      37.885  14.852  59.328  1.00  0.00           C  \\nATOM    408  H   ALA A  41      38.929  12.772  60.011  1.00  0.00           H  \\nATOM    409  HA  ALA A  41      37.858  13.522  57.779  1.00  0.00           H  \\nATOM    410 1HB  ALA A  41      37.524  15.697  58.741  1.00  0.00           H  \\nATOM    411 2HB  ALA A  41      37.048  14.390  59.852  1.00  0.00           H  \\nATOM    412 3HB  ALA A  41      38.618  15.202  60.053  1.00  0.00           H  \\nATOM    413  N   ALA A  42      40.845  14.537  58.224  1.00  0.00           N  \\nATOM    414  CA  ALA A  42      42.022  15.105  57.573  1.00  0.00           C  \\nATOM    415  C   ALA A  42      42.475  14.215  56.418  1.00  0.00           C  \\nATOM    416  O   ALA A  42      42.906  14.714  55.380  1.00  0.00           O  \\nATOM    417  CB  ALA A  42      43.150  15.292  58.577  1.00  0.00           C  \\nATOM    418  H   ALA A  42      40.989  14.262  59.026  1.00  0.00           H  \\nATOM    419  HA  ALA A  42      41.794  15.975  57.211  1.00  0.00           H  \\nATOM    420 1HB  ALA A  42      44.021  15.711  58.072  1.00  0.00           H  \\nATOM    421 2HB  ALA A  42      42.826  15.971  59.366  1.00  0.00           H  \\nATOM    422 3HB  ALA A  42      43.412  14.330  59.013  1.00  0.00           H  \\nATOM    423  N   ALA A  43      42.373  12.901  56.603  1.00  0.00           N  \\nATOM    424  CA  ALA A  43      42.744  11.950  55.557  1.00  0.00           C  \\nATOM    425  C   ALA A  43      41.899  12.158  54.307  1.00  0.00           C  \\nATOM    426  O   ALA A  43      42.387  12.017  53.190  1.00  0.00           O  \\nATOM    427  CB  ALA A  43      42.601  10.521  56.061  1.00  0.00           C  \\nATOM    428  H   ALA A  43      42.091  12.533  57.327  1.00  0.00           H  \\nATOM    429  HA  ALA A  43      43.674  12.088  55.319  1.00  0.00           H  \\nATOM    430 1HB  ALA A  43      42.878   9.826  55.268  1.00  0.00           H  \\nATOM    431 2HB  ALA A  43      43.255  10.371  56.919  1.00  0.00           H  \\nATOM    432 3HB  ALA A  43      41.568  10.342  56.356  1.00  0.00           H  \\nATOM    433  N   ALA A  44      40.629  12.490  54.507  1.00  0.00           N  \\nATOM    434  CA  ALA A  44      39.711  12.720  53.398  1.00  0.00           C  \\nATOM    435  C   ALA A  44      40.003  14.050  52.719  1.00  0.00           C  \\nATOM    436  O   ALA A  44      40.039  14.130  51.491  1.00  0.00           O  \\nATOM    437  CB  ALA A  44      38.269  12.677  53.882  1.00  0.00           C  \\nATOM    438  H   ALA A  44      40.271  12.589  55.283  1.00  0.00           H  \\nATOM    439  HA  ALA A  44      39.825  12.015  52.741  1.00  0.00           H  \\nATOM    440 1HB  ALA A  44      37.598  12.854  53.042  1.00  0.00           H  \\nATOM    441 2HB  ALA A  44      38.059  11.699  54.315  1.00  0.00           H  \\nATOM    442 3HB  ALA A  44      38.116  13.447  54.637  1.00  0.00           H  \\nATOM    443  N   ALA A  45      40.211  15.089  53.522  1.00  0.00           N  \\nATOM    444  CA  ALA A  45      40.538  16.406  52.990  1.00  0.00           C  \\nATOM    445  C   ALA A  45      41.757  16.326  52.082  1.00  0.00           C  \\nATOM    446  O   ALA A  45      41.810  16.994  51.054  1.00  0.00           O  \\nATOM    447  CB  ALA A  45      40.780  17.395  54.121  1.00  0.00           C  \\nATOM    448  H   ALA A  45      40.169  15.057  54.381  1.00  0.00           H  \\nATOM    449  HA  ALA A  45      39.790  16.729  52.463  1.00  0.00           H  \\nATOM    450 1HB  ALA A  45      41.029  18.371  53.704  1.00  0.00           H  \\nATOM    451 2HB  ALA A  45      39.879  17.480  54.729  1.00  0.00           H  \\nATOM    452 3HB  ALA A  45      41.604  17.045  54.740  1.00  0.00           H  \\nATOM    453  N   ALA A  46      42.727  15.497  52.458  1.00  0.00           N  \\nATOM    454  CA  ALA A  46      43.905  15.287  51.625  1.00  0.00           C  \\nATOM    455  C   ALA A  46      43.507  14.737  50.260  1.00  0.00           C  \\nATOM    456  O   ALA A  46      44.031  15.165  49.233  1.00  0.00           O  \\nATOM    457  CB  ALA A  46      44.884  14.347  52.313  1.00  0.00           C  \\nATOM    458  H   ALA A  46      42.726  15.045  53.189  1.00  0.00           H  \\nATOM    459  HA  ALA A  46      44.353  16.137  51.489  1.00  0.00           H  \\nATOM    460 1HB  ALA A  46      45.755  14.199  51.675  1.00  0.00           H  \\nATOM    461 2HB  ALA A  46      45.199  14.781  53.262  1.00  0.00           H  \\nATOM    462 3HB  ALA A  46      44.401  13.389  52.496  1.00  0.00           H  \\nATOM    463  N   ALA A  47      42.575  13.789  50.257  1.00  0.00           N  \\nATOM    464  CA  ALA A  47      42.166  13.124  49.026  1.00  0.00           C  \\nATOM    465  C   ALA A  47      41.228  14.016  48.218  1.00  0.00           C  \\nATOM    466  O   ALA A  47      41.170  13.919  46.992  1.00  0.00           O  \\nATOM    467  CB  ALA A  47      41.499  11.792  49.337  1.00  0.00           C  \\nATOM    468  H   ALA A  47      42.163  13.512  50.959  1.00  0.00           H  \\nATOM    469  HA  ALA A  47      42.952  12.948  48.486  1.00  0.00           H  \\nATOM    470 1HB  ALA A  47      41.197  11.310  48.406  1.00  0.00           H  \\nATOM    471 2HB  ALA A  47      42.201  11.148  49.867  1.00  0.00           H  \\nATOM    472 3HB  ALA A  47      40.622  11.961  49.959  1.00  0.00           H  \\nATOM    473  N   ALA A  48      40.498  14.887  48.908  1.00  0.00           N  \\nATOM    474  CA  ALA A  48      39.616  15.837  48.240  1.00  0.00           C  \\nATOM    475  C   ALA A  48      40.446  16.839  47.437  1.00  0.00           C  \\nATOM    476  O   ALA A  48      40.168  17.084  46.262  1.00  0.00           O  \\nATOM    477  CB  ALA A  48      38.734  16.555  49.251  1.00  0.00           C  \\nATOM    478  H   ALA A  48      40.496  14.947  49.766  1.00  0.00           H  \\nATOM    479  HA  ALA A  48      39.044  15.357  47.621  1.00  0.00           H  \\nATOM    480 1HB  ALA A  48      38.091  17.266  48.733  1.00  0.00           H  \\nATOM    481 2HB  ALA A  48      38.119  15.826  49.779  1.00  0.00           H  \\nATOM    482 3HB  ALA A  48      39.360  17.087  49.966  1.00  0.00           H  \\nATOM    483  N   ALA A  49      41.475  17.402  48.063  1.00  0.00           N  \\nATOM    484  CA  ALA A  49      42.346  18.354  47.381  1.00  0.00           C  \\nATOM    485  C   ALA A  49      43.059  17.662  46.227  1.00  0.00           C  \\nATOM    486  O   ALA A  49      43.298  18.263  45.182  1.00  0.00           O  \\nATOM    487  CB  ALA A  49      43.352  18.957  48.350  1.00  0.00           C  \\nATOM    488  H   ALA A  49      41.690  17.251  48.882  1.00  0.00           H  \\nATOM    489  HA  ALA A  49      41.807  19.073  47.016  1.00  0.00           H  \\nATOM    490 1HB  ALA A  49      43.999  19.654  47.816  1.00  0.00           H  \\nATOM    491 2HB  ALA A  49      42.822  19.488  49.141  1.00  0.00           H  \\nATOM    492 3HB  ALA A  49      43.956  18.164  48.788  1.00  0.00           H  \\nATOM    493  N   ALA A  50      43.396  16.393  46.427  1.00  0.00           N  \\nATOM    494  CA  ALA A  50      44.003  15.586  45.379  1.00  0.00           C  \\nATOM    495  C   ALA A  50      43.038  15.459  44.211  1.00  0.00           C  \\nATOM    496  O   ALA A  50      43.424  15.625  43.055  1.00  0.00           O  \\nATOM    497  CB  ALA A  50      44.392  14.213  45.909  1.00  0.00           C  \\nATOM    498  H   ALA A  50      43.281  15.974  47.169  1.00  0.00           H  \\nATOM    499  HA  ALA A  50      44.809  16.024  45.063  1.00  0.00           H  \\nATOM    500 1HB  ALA A  50      44.830  13.624  45.104  1.00  0.00           H  \\nATOM    501 2HB  ALA A  50      45.118  14.326  46.714  1.00  0.00           H  \\nATOM    502 3HB  ALA A  50      43.506  13.706  46.289  1.00  0.00           H  \\nATOM    503  N   ALA A  51      41.778  15.167  44.517  1.00  0.00           N  \\nATOM    504  CA  ALA A  51      40.758  15.040  43.485  1.00  0.00           C  \\nATOM    505  C   ALA A  51      40.578  16.367  42.750  1.00  0.00           C  \\nATOM    506  O   ALA A  51      40.489  16.403  41.523  1.00  0.00           O  \\nATOM    507  CB  ALA A  51      39.440  14.579  44.091  1.00  0.00           C  \\nATOM    508  H   ALA A  51      41.488  15.039  45.317  1.00  0.00           H  \\nATOM    509  HA  ALA A  51      41.041  14.374  42.839  1.00  0.00           H  \\nATOM    510 1HB  ALA A  51      38.689  14.495  43.306  1.00  0.00           H  \\nATOM    511 2HB  ALA A  51      39.578  13.609  44.567  1.00  0.00           H  \\nATOM    512 3HB  ALA A  51      39.109  15.303  44.833  1.00  0.00           H  \\nATOM    513  N   ALA A  52      40.536  17.458  43.510  1.00  0.00           N  \\nATOM    514  CA  ALA A  52      40.361  18.784  42.933  1.00  0.00           C  \\nATOM    515  C   ALA A  52      41.565  19.158  42.079  1.00  0.00           C  \\nATOM    516  O   ALA A  52      41.408  19.694  40.985  1.00  0.00           O  \\nATOM    517  CB  ALA A  52      40.141  19.818  44.027  1.00  0.00           C  \\nATOM    518  H   ALA A  52      40.606  17.455  44.367  1.00  0.00           H  \\nATOM    519  HA  ALA A  52      39.579  18.777  42.359  1.00  0.00           H  \\nATOM    520 1HB  ALA A  52      40.018  20.803  43.578  1.00  0.00           H  \\nATOM    521 2HB  ALA A  52      39.246  19.563  44.595  1.00  0.00           H  \\nATOM    522 3HB  ALA A  52      41.002  19.830  44.695  1.00  0.00           H  \\nATOM    523  N   ALA A  53      42.764  18.868  42.573  1.00  0.00           N  \\nATOM    524  CA  ALA A  53      43.977  19.138  41.813  1.00  0.00           C  \\nATOM    525  C   ALA A  53      43.941  18.386  40.489  1.00  0.00           C  \\nATOM    526  O   ALA A  53      44.317  18.925  39.449  1.00  0.00           O  \\nATOM    527  CB  ALA A  53      45.211  18.753  42.615  1.00  0.00           C  \\nATOM    528  H   ALA A  53      42.901  18.516  43.346  1.00  0.00           H  \\nATOM    529  HA  ALA A  53      44.027  20.087  41.620  1.00  0.00           H  \\nATOM    530 1HB  ALA A  53      46.106  18.954  42.026  1.00  0.00           H  \\nATOM    531 2HB  ALA A  53      45.244  19.337  43.535  1.00  0.00           H  \\nATOM    532 3HB  ALA A  53      45.169  17.693  42.861  1.00  0.00           H  \\nATOM    533  N   ALA A  54      43.474  17.143  40.533  1.00  0.00           N  \\nATOM    534  CA  ALA A  54      43.344  16.329  39.331  1.00  0.00           C  \\nATOM    535  C   ALA A  54      42.245  16.881  38.434  1.00  0.00           C  \\nATOM    536  O   ALA A  54      42.347  16.834  37.208  1.00  0.00           O  \\nATOM    537  CB  ALA A  54      43.059  14.879  39.694  1.00  0.00           C  \\nATOM    538  H   ALA A  54      43.223  16.745  41.253  1.00  0.00           H  \\nATOM    539  HA  ALA A  54      44.179  16.356  38.837  1.00  0.00           H  \\nATOM    540 1HB  ALA A  54      42.958  14.289  38.783  1.00  0.00           H  \\nATOM    541 2HB  ALA A  54      43.881  14.485  40.292  1.00  0.00           H  \\nATOM    542 3HB  ALA A  54      42.135  14.822  40.267  1.00  0.00           H  \\nATOM    543  N   ALA A  55      41.194  17.412  39.052  1.00  0.00           N  \\nATOM    544  CA  ALA A  55      40.093  18.014  38.309  1.00  0.00           C  \\nATOM    545  C   ALA A  55      40.558  19.258  37.554  1.00  0.00           C  \\nATOM    546  O   ALA A  55      40.268  19.415  36.368  1.00  0.00           O  \\nATOM    547  CB  ALA A  55      38.946  18.359  39.248  1.00  0.00           C  \\nATOM    548  H   ALA A  55      41.095  17.435  39.906  1.00  0.00           H  \\nATOM    549  HA  ALA A  55      39.762  17.374  37.660  1.00  0.00           H  \\nATOM    550 1HB  ALA A  55      38.132  18.808  38.679  1.00  0.00           H  \\nATOM    551 2HB  ALA A  55      38.591  17.453  39.737  1.00  0.00           H  \\nATOM    552 3HB  ALA A  55      39.294  19.065  40.001  1.00  0.00           H  \\nATOM    553  N   ALA A  56      41.281  20.137  38.241  1.00  0.00           N  \\nATOM    554  CA  ALA A  56      41.777  21.367  37.626  1.00  0.00           C  \\nATOM    555  C   ALA A  56      42.772  21.056  36.510  1.00  0.00           C  \\nATOM    556  O   ALA A  56      42.778  21.714  35.470  1.00  0.00           O  \\nATOM    557  CB  ALA A  56      42.419  22.264  38.674  1.00  0.00           C  \\nATOM    558  H   ALA A  56      41.500  20.045  39.068  1.00  0.00           H  \\nATOM    559  HA  ALA A  56      41.030  21.848  37.236  1.00  0.00           H  \\nATOM    560 1HB  ALA A  56      42.785  23.175  38.199  1.00  0.00           H  \\nATOM    561 2HB  ALA A  56      41.680  22.523  39.433  1.00  0.00           H  \\nATOM    562 3HB  ALA A  56      43.251  21.740  39.142  1.00  0.00           H  \\nATOM    563  N   ALA A  57      43.614  20.052  36.732  1.00  0.00           N  \\nATOM    564  CA  ALA A  57      44.557  19.618  35.709  1.00  0.00           C  \\nATOM    565  C   ALA A  57      43.803  19.118  34.482  1.00  0.00           C  \\nATOM    566  O   ALA A  57      44.186  19.404  33.348  1.00  0.00           O  \\nATOM    567  CB  ALA A  57      45.478  18.535  36.252  1.00  0.00           C  \\nATOM    568  H   ALA A  57      43.660  19.607  37.466  1.00  0.00           H  \\nATOM    569  HA  ALA A  57      45.106  20.372  35.441  1.00  0.00           H  \\nATOM    570 1HB  ALA A  57      46.168  18.218  35.469  1.00  0.00           H  \\nATOM    571 2HB  ALA A  57      46.043  18.927  37.097  1.00  0.00           H  \\nATOM    572 3HB  ALA A  57      44.884  17.682  36.577  1.00  0.00           H  \\nATOM    573  N   ALA A  58      42.722  18.378  34.712  1.00  0.00           N  \\nATOM    574  CA  ALA A  58      41.902  17.876  33.614  1.00  0.00           C  \\nATOM    575  C   ALA A  58      41.286  19.027  32.826  1.00  0.00           C  \\nATOM    576  O   ALA A  58      41.209  18.977  31.600  1.00  0.00           O  \\nATOM    577  CB  ALA A  58      40.814  16.952  34.141  1.00  0.00           C  \\nATOM    578  H   ALA A  58      42.442  18.152  35.494  1.00  0.00           H  \\nATOM    579  HA  ALA A  58      42.463  17.365  33.010  1.00  0.00           H  \\nATOM    580 1HB  ALA A  58      40.211  16.589  33.310  1.00  0.00           H  \\nATOM    581 2HB  ALA A  58      41.273  16.105  34.654  1.00  0.00           H  \\nATOM    582 3HB  ALA A  58      40.180  17.497  34.839  1.00  0.00           H  \\nATOM    583  N   ALA A  59      40.859  20.067  33.535  1.00  0.00           N  \\nATOM    584  CA  ALA A  59      40.285  21.249  32.896  1.00  0.00           C  \\nATOM    585  C   ALA A  59      41.344  22.073  32.166  1.00  0.00           C  \\nATOM    586  O   ALA A  59      41.051  22.717  31.157  1.00  0.00           O  \\nATOM    587  CB  ALA A  59      39.571  22.109  33.929  1.00  0.00           C  \\nATOM    588  H   ALA A  59      40.890  20.114  34.393  1.00  0.00           H  \\nATOM    589  HA  ALA A  59      39.627  20.963  32.243  1.00  0.00           H  \\nATOM    590 1HB  ALA A  59      39.142  22.983  33.440  1.00  0.00           H  \\nATOM    591 2HB  ALA A  59      38.776  21.529  34.398  1.00  0.00           H  \\nATOM    592 3HB  ALA A  59      40.282  22.430  34.688  1.00  0.00           H  \\nATOM    593  N   ALA A  60      42.572  22.059  32.676  1.00  0.00           N  \\nATOM    594  CA  ALA A  60      43.666  22.768  32.019  1.00  0.00           C  \\nATOM    595  C   ALA A  60      43.936  22.123  30.666  1.00  0.00           C  \\nATOM    596  O   ALA A  60      43.986  22.801  29.640  1.00  0.00           O  \\nATOM    597  CB  ALA A  60      44.919  22.759  32.883  1.00  0.00           C  \\nATOM    598  H   ALA A  60      42.798  21.649  33.397  1.00  0.00           H  \\nATOM    599  HA  ALA A  60      43.408  23.692  31.872  1.00  0.00           H  \\nATOM    600 1HB  ALA A  60      45.724  23.277  32.362  1.00  0.00           H  \\nATOM    601 2HB  ALA A  60      44.714  23.264  33.827  1.00  0.00           H  \\nATOM    602 3HB  ALA A  60      45.217  21.730  33.079  1.00  0.00           H  \\nATOM    603  N   ALA A  61      44.087  20.803  30.675  1.00  0.00           N  \\nATOM    604  CA  ALA A  61      44.304  20.039  29.451  1.00  0.00           C  \\nATOM    605  C   ALA A  61      43.135  20.214  28.490  1.00  0.00           C  \\nATOM    606  O   ALA A  61      43.319  20.248  27.274  1.00  0.00           O  \\nATOM    607  CB  ALA A  61      44.512  18.566  29.772  1.00  0.00           C  \\nATOM    608  H   ALA A  61      44.068  20.320  31.387  1.00  0.00           H  \\nATOM    609  HA  ALA A  61      45.107  20.362  29.012  1.00  0.00           H  \\nATOM    610 1HB  ALA A  61      44.668  18.011  28.847  1.00  0.00           H  \\nATOM    611 2HB  ALA A  61      45.385  18.453  30.415  1.00  0.00           H  \\nATOM    612 3HB  ALA A  61      43.633  18.178  30.283  1.00  0.00           H  \\nATOM    613  N   ALA A  62      41.932  20.322  29.047  1.00  0.00           N  \\nATOM    614  CA  ALA A  62      40.722  20.473  28.246  1.00  0.00           C  \\nATOM    615  C   ALA A  62      40.705  21.801  27.493  1.00  0.00           C  \\nATOM    616  O   ALA A  62      40.435  21.835  26.294  1.00  0.00           O  \\nATOM    617  CB  ALA A  62      39.489  20.353  29.130  1.00  0.00           C  \\nATOM    618  H   ALA A  62      41.789  20.312  29.894  1.00  0.00           H  \\nATOM    619  HA  ALA A  62      40.688  19.758  27.591  1.00  0.00           H  \\nATOM    620 1HB  ALA A  62      38.592  20.462  28.520  1.00  0.00           H  \\nATOM    621 2HB  ALA A  62      39.482  19.376  29.614  1.00  0.00           H  \\nATOM    622 3HB  ALA A  62      39.508  21.134  29.888  1.00  0.00           H  \\nATOM    623  N   ALA A  63      40.990  22.890  28.203  1.00  0.00           N  \\nATOM    624  CA  ALA A  63      41.009  24.224  27.602  1.00  0.00           C  \\nATOM    625  C   ALA A  63      42.148  24.354  26.597  1.00  0.00           C  \\nATOM    626  O   ALA A  63      41.986  24.945  25.528  1.00  0.00           O  \\nATOM    627  CB  ALA A  63      41.130  25.292  28.679  1.00  0.00           C  \\nATOM    628  H   ALA A  63      41.177  22.884  29.042  1.00  0.00           H  \\nATOM    629  HA  ALA A  63      40.174  24.372  27.131  1.00  0.00           H  \\nATOM    630 1HB  ALA A  63      41.146  26.278  28.214  1.00  0.00           H  \\nATOM    631 2HB  ALA A  63      40.278  25.224  29.356  1.00  0.00           H  \\nATOM    632 3HB  ALA A  63      42.051  25.140  29.239  1.00  0.00           H  \\nATOM    633  N   ALA A  64      43.304  23.807  26.957  1.00  0.00           N  \\nATOM    634  CA  ALA A  64      44.451  23.767  26.059  1.00  0.00           C  \\nATOM    635  C   ALA A  64      44.063  23.125  24.732  1.00  0.00           C  \\nATOM    636  O   ALA A  64      44.505  23.557  23.669  1.00  0.00           O  \\nATOM    637  CB  ALA A  64      45.607  23.013  26.700  1.00  0.00           C  \\nATOM    638  H   ALA A  64      43.450  23.447  27.724  1.00  0.00           H  \\nATOM    639  HA  ALA A  64      44.749  24.673  25.881  1.00  0.00           H  \\nATOM    640 1HB  ALA A  64      46.452  22.988  26.012  1.00  0.00           H  \\nATOM    641 2HB  ALA A  64      45.903  23.516  27.621  1.00  0.00           H  \\nATOM    642 3HB  ALA A  64      45.295  21.995  26.928  1.00  0.00           H  \\nATOM    643  N   ALA A  65      43.222  22.099  24.806  1.00  0.00           N  \\nATOM    644  CA  ALA A  65      42.766  21.390  23.618  1.00  0.00           C  \\nATOM    645  C   ALA A  65      41.719  22.186  22.839  1.00  0.00           C  \\nATOM    646  O   ALA A  65      41.603  22.046  21.623  1.00  0.00           O  \\nATOM    647  CB  ALA A  65      42.209  20.028  24.006  1.00  0.00           C  \\nATOM    648  H   ALA A  65      42.898  21.791  25.541  1.00  0.00           H  \\nATOM    649  HA  ALA A  65      43.524  21.241  23.031  1.00  0.00           H  \\nATOM    650 1HB  ALA A  65      41.880  19.501  23.111  1.00  0.00           H  \\nATOM    651 2HB  ALA A  65      42.984  19.447  24.506  1.00  0.00           H  \\nATOM    652 3HB  ALA A  65      41.364  20.160  24.680  1.00  0.00           H  \\nATOM    653  N   ALA A  66      40.978  23.037  23.518  1.00  0.00           N  \\nATOM    654  CA  ALA A  66      39.980  23.885  22.899  1.00  0.00           C  \\nATOM    655  C   ALA A  66      40.596  25.055  22.179  1.00  0.00           C  \\nATOM    656  O   ALA A  66      40.085  25.540  21.207  1.00  0.00           O  \\nATOM    657  CB  ALA A  66      38.990  24.380  23.943  1.00  0.00           C  \\nATOM    658  H   ALA A  66      41.049  23.144  24.363  1.00  0.00           H  \\nATOM    659  HA  ALA A  66      39.493  23.362  22.239  1.00  0.00           H  \\nATOM    660 1HB  ALA A  66      38.242  25.012  23.465  1.00  0.00           H  \\nATOM    661 2HB  ALA A  66      38.498  23.527  24.411  1.00  0.00           H  \\nATOM    662 3HB  ALA A  66      39.519  24.955  24.702  1.00  0.00           H  \\nATOM    663  N   ALA A  67      41.732  25.479  22.675  1.00  0.00           N  \\nATOM    664  CA  ALA A  67      42.504  26.506  21.987  1.00  0.00           C  \\nATOM    665  C   ALA A  67      43.009  25.969  20.651  1.00  0.00           C  \\nATOM    666  O   ALA A  67      43.238  26.729  19.710  1.00  0.00           O  \\nATOM    667  CB  ALA A  67      43.665  26.976  22.852  1.00  0.00           C  \\nATOM    668  H   ALA A  67      42.092  25.195  23.403  1.00  0.00           H  \\nATOM    669  HA  ALA A  67      41.934  27.271  21.812  1.00  0.00           H  \\nATOM    670 1HB  ALA A  67      44.234  27.735  22.315  1.00  0.00           H  \\nATOM    671 2HB  ALA A  67      43.280  27.399  23.779  1.00  0.00           H  \\nATOM    672 3HB  ALA A  67      44.313  26.131  23.080  1.00  0.00           H  \\nATOM    673  N   ALA A  68      43.175  24.651  20.576  1.00  0.00           N  \\nATOM    674  CA  ALA A  68      43.667  24.000  19.368  1.00  0.00           C  \\nATOM    675  C   ALA A  68      42.547  23.785  18.354  1.00  0.00           C  \\nATOM    676  O   ALA A  68      42.754  23.937  17.150  1.00  0.00           O  \\nATOM    677  CB  ALA A  68      44.327  22.674  19.716  1.00  0.00           C  \\nATOM    678  H   ALA A  68      43.007  24.107  21.220  1.00  0.00           H  \\nATOM    679  HA  ALA A  68      44.339  24.566  18.956  1.00  0.00           H  \\nATOM    680 1HB  ALA A  68      44.693  22.199  18.805  1.00  0.00           H  \\nATOM    681 2HB  ALA A  68      45.162  22.849  20.394  1.00  0.00           H  \\nATOM    682 3HB  ALA A  68      43.600  22.021  20.197  1.00  0.00           H  \\nATOM    683  N   ALA A  69      41.363  23.430  18.842  1.00  0.00           N  \\nATOM    684  CA  ALA A  69      40.211  23.214  17.970  1.00  0.00           C  \\nATOM    685  C   ALA A  69      39.822  24.505  17.254  1.00  0.00           C  \\nATOM    686  O   ALA A  69      39.231  24.473  16.175  1.00  0.00           O  \\nATOM    687  CB  ALA A  69      39.033  22.674  18.768  1.00  0.00           C  \\nATOM    688  H   ALA A  69      41.199  23.307  19.677  1.00  0.00           H  \\nATOM    689  HA  ALA A  69      40.443  22.555  17.297  1.00  0.00           H  \\nATOM    690 1HB  ALA A  69      38.183  22.521  18.103  1.00  0.00           H  \\nATOM    691 2HB  ALA A  69      39.310  21.724  19.227  1.00  0.00           H  \\nATOM    692 3HB  ALA A  69      38.763  23.387  19.545  1.00  0.00           H  \\nATOM    693  N   ALA A  70      40.162  25.638  17.860  1.00  0.00           N  \\nATOM    694  CA  ALA A  70      39.855  26.942  17.284  1.00  0.00           C  \\nATOM    695  C   ALA A  70      40.852  27.303  16.187  1.00  0.00           C  \\nATOM    696  O   ALA A  70      42.065  27.204  16.377  1.00  0.00           O  \\nATOM    697  OXT ALA A  70      40.458  27.689  15.122  1.00  0.00           O  \\nATOM    698  CB  ALA A  70      39.846  28.011  18.367  1.00  0.00           C  \\nATOM    699  H   ALA A  70      40.576  25.679  18.612  1.00  0.00           H  \\nATOM    700  HA  ALA A  70      38.970  26.913  16.889  1.00  0.00           H  \\nATOM    701 1HB  ALA A  70      39.616  28.979  17.920  1.00  0.00           H  \\nATOM    702 2HB  ALA A  70      39.090  27.767  19.113  1.00  0.00           H  \\nATOM    703 3HB  ALA A  70      40.825  28.055  18.842  1.00  0.00           H  \\nTER                                                                             \\nATOM    705  N   TYR B  71      35.710  36.388  19.013  1.00  0.00           N  \\nATOM    706  CA  TYR B  71      34.402  36.488  18.434  1.00  0.00           C  \\nATOM    707  C   TYR B  71      33.573  35.299  18.771  1.00  0.00           C  \\nATOM    708  O   TYR B  71      33.315  34.433  18.000  1.00  0.00           O  \\nATOM    709  CB  TYR B  71      34.548  36.577  16.921  1.00  0.00           C  \\nATOM    710  CG  TYR B  71      35.147  35.286  16.337  1.00  0.00           C  \\nATOM    711  CD1 TYR B  71      34.314  34.280  15.860  1.00  0.00           C  \\nATOM    712  CD2 TYR B  71      36.529  35.103  16.315  1.00  0.00           C  \\nATOM    713  CE1 TYR B  71      34.854  33.115  15.345  1.00  0.00           C  \\nATOM    714  CE2 TYR B  71      37.068  33.929  15.806  1.00  0.00           C  \\nATOM    715  CZ  TYR B  71      36.236  32.942  15.317  1.00  0.00           C  \\nATOM    716  OH  TYR B  71      36.770  31.778  14.802  1.00  0.00           O  \\nATOM    717 1H   TYR B  71      36.245  37.198  18.772  1.00  0.00           H  \\nATOM    718 2H   TYR B  71      35.631  36.326  20.008  1.00  0.00           H  \\nATOM    719 3H   TYR B  71      36.168  35.570  18.664  1.00  0.00           H  \\nATOM    720  HA  TYR B  71      33.961  37.264  18.826  1.00  0.00           H  \\nATOM    721 1HB  TYR B  71      33.573  36.746  16.462  1.00  0.00           H  \\nATOM    722 2HB  TYR B  71      35.196  37.413  16.657  1.00  0.00           H  \\nATOM    723  HD1 TYR B  71      33.232  34.412  15.884  1.00  0.00           H  \\nATOM    724  HD2 TYR B  71      37.187  35.883  16.697  1.00  0.00           H  \\nATOM    725  HE1 TYR B  71      34.195  32.337  14.965  1.00  0.00           H  \\nATOM    726  HE2 TYR B  71      38.149  33.789  15.789  1.00  0.00           H  \\nATOM    727  HH  TYR B  71      37.701  31.720  15.032  1.00  0.00           H  \\nATOM    728  N   LYS B  72      33.117  35.298  19.982  1.00  0.00           N  \\nATOM    729  CA  LYS B  72      32.478  34.151  20.550  1.00  0.00           C  \\nATOM    730  C   LYS B  72      33.634  33.262  20.770  1.00  0.00           C  \\nATOM    731  O   LYS B  72      34.157  33.247  21.839  1.00  0.00           O  \\nATOM    732  CB  LYS B  72      31.342  33.622  19.660  1.00  0.00           C  \\nATOM    733  CG  LYS B  72      30.383  32.756  20.373  1.00  0.00           C  \\nATOM    734  CD  LYS B  72      29.266  32.286  19.437  1.00  0.00           C  \\nATOM    735  CE  LYS B  72      28.208  31.460  20.162  1.00  0.00           C  \\nATOM    736  NZ  LYS B  72      27.102  31.051  19.238  1.00  0.00           N  \\nATOM    737  H   LYS B  72      33.183  35.965  20.514  1.00  0.00           H  \\nATOM    738  HA  LYS B  72      32.084  34.379  21.413  1.00  0.00           H  \\nATOM    739 1HB  LYS B  72      30.799  34.449  19.203  1.00  0.00           H  \\nATOM    740 2HB  LYS B  72      31.722  32.987  18.865  1.00  0.00           H  \\nATOM    741 1HG  LYS B  72      30.903  31.988  20.830  1.00  0.00           H  \\nATOM    742 2HG  LYS B  72      29.922  33.345  21.163  1.00  0.00           H  \\nATOM    743 1HD  LYS B  72      28.788  33.155  18.984  1.00  0.00           H  \\nATOM    744 2HD  LYS B  72      29.701  31.679  18.641  1.00  0.00           H  \\nATOM    745 1HE  LYS B  72      28.668  30.567  20.579  1.00  0.00           H  \\nATOM    746 2HE  LYS B  72      27.789  32.050  20.976  1.00  0.00           H  \\nATOM    747 1HZ  LYS B  72      26.419  30.508  19.746  1.00  0.00           H  \\nATOM    748 2HZ  LYS B  72      26.659  31.876  18.856  1.00  0.00           H  \\nATOM    749 3HZ  LYS B  72      27.479  30.493  18.483  1.00  0.00           H  \\nATOM    750  N   ILE B  73      34.079  32.584  19.742  1.00  0.00           N  \\nATOM    751  CA  ILE B  73      35.171  31.658  19.903  1.00  0.00           C  \\nATOM    752  C   ILE B  73      36.358  32.336  20.561  1.00  0.00           C  \\nATOM    753  O   ILE B  73      36.745  31.991  21.645  1.00  0.00           O  \\nATOM    754  CB  ILE B  73      35.596  30.967  18.592  1.00  0.00           C  \\nATOM    755  CG1 ILE B  73      34.478  30.035  18.128  1.00  0.00           C  \\nATOM    756  CG2 ILE B  73      36.880  30.171  18.804  1.00  0.00           C  \\nATOM    757  CD1 ILE B  73      34.646  29.471  16.756  1.00  0.00           C  \\nATOM    758  H   ILE B  73      33.759  32.648  18.951  1.00  0.00           H  \\nATOM    759  HA  ILE B  73      34.887  30.943  20.499  1.00  0.00           H  \\nATOM    760  HB  ILE B  73      35.749  31.712  17.827  1.00  0.00           H  \\nATOM    761 1HG1 ILE B  73      34.394  29.207  18.836  1.00  0.00           H  \\nATOM    762 2HG1 ILE B  73      33.554  30.597  18.125  1.00  0.00           H  \\nATOM    763 1HG2 ILE B  73      37.160  29.688  17.868  1.00  0.00           H  \\nATOM    764 2HG2 ILE B  73      37.683  30.830  19.122  1.00  0.00           H  \\nATOM    765 3HG2 ILE B  73      36.712  29.410  19.572  1.00  0.00           H  \\nATOM    766 1HD1 ILE B  73      33.788  28.831  16.521  1.00  0.00           H  \\nATOM    767 2HD1 ILE B  73      34.702  30.275  16.035  1.00  0.00           H  \\nATOM    768 3HD1 ILE B  73      35.560  28.884  16.711  1.00  0.00           H  \\nATOM    769  N   LYS B  74      36.874  33.357  19.918  1.00  0.00           N  \\nATOM    770  CA  LYS B  74      37.966  34.129  20.458  1.00  0.00           C  \\nATOM    771  C   LYS B  74      37.682  34.728  21.827  1.00  0.00           C  \\nATOM    772  O   LYS B  74      38.529  34.736  22.665  1.00  0.00           O  \\nATOM    773  CB  LYS B  74      38.585  35.153  19.492  1.00  0.00           C  \\nATOM    774  CG  LYS B  74      39.826  35.837  20.036  1.00  0.00           C  \\nATOM    775  CD  LYS B  74      40.435  36.780  18.997  1.00  0.00           C  \\nATOM    776  CE  LYS B  74      41.659  37.502  19.546  1.00  0.00           C  \\nATOM    777  NZ  LYS B  74      42.255  38.425  18.541  1.00  0.00           N  \\nATOM    778  H   LYS B  74      36.597  33.608  19.149  1.00  0.00           H  \\nATOM    779  HA  LYS B  74      38.727  33.530  20.568  1.00  0.00           H  \\nATOM    780 1HB  LYS B  74      38.824  34.682  18.537  1.00  0.00           H  \\nATOM    781 2HB  LYS B  74      37.999  35.949  19.324  1.00  0.00           H  \\nATOM    782 1HG  LYS B  74      39.562  36.410  20.928  1.00  0.00           H  \\nATOM    783 2HG  LYS B  74      40.565  35.086  20.315  1.00  0.00           H  \\nATOM    784 1HD  LYS B  74      40.728  36.209  18.115  1.00  0.00           H  \\nATOM    785 2HD  LYS B  74      39.695  37.522  18.698  1.00  0.00           H  \\nATOM    786 1HE  LYS B  74      41.369  38.076  20.425  1.00  0.00           H  \\nATOM    787 2HE  LYS B  74      42.407  36.765  19.837  1.00  0.00           H  \\nATOM    788 1HZ  LYS B  74      41.570  39.119  18.274  1.00  0.00           H  \\nATOM    789 2HZ  LYS B  74      43.061  38.885  18.940  1.00  0.00           H  \\nATOM    790 3HZ  LYS B  74      42.536  37.900  17.725  1.00  0.00           H  \\nATOM    791  N   GLU B  75      36.494  35.236  22.053  1.00  0.00           N  \\nATOM    792  CA  GLU B  75      36.195  35.897  23.320  1.00  0.00           C  \\nATOM    793  C   GLU B  75      35.834  34.904  24.422  1.00  0.00           C  \\nATOM    794  O   GLU B  75      36.211  35.100  25.576  1.00  0.00           O  \\nATOM    795  CB  GLU B  75      35.029  36.854  23.074  1.00  0.00           C  \\nATOM    796  CG  GLU B  75      35.377  38.032  22.167  1.00  0.00           C  \\nATOM    797  CD  GLU B  75      34.164  38.748  21.626  1.00  0.00           C  \\nATOM    798  OE1 GLU B  75      33.083  38.243  21.804  1.00  0.00           O  \\nATOM    799  OE2 GLU B  75      34.327  39.758  20.992  1.00  0.00           O  \\nATOM    800  H   GLU B  75      35.837  35.220  21.498  1.00  0.00           H  \\nATOM    801  HA  GLU B  75      36.983  36.383  23.609  1.00  0.00           H  \\nATOM    802 1HB  GLU B  75      34.203  36.305  22.621  1.00  0.00           H  \\nATOM    803 2HB  GLU B  75      34.678  37.251  24.025  1.00  0.00           H  \\nATOM    804 1HG  GLU B  75      35.959  38.743  22.751  1.00  0.00           H  \\nATOM    805 2HG  GLU B  75      36.004  37.694  21.360  1.00  0.00           H  \\nATOM    806  N   THR B  76      35.111  33.866  24.054  1.00  0.00           N  \\nATOM    807  CA  THR B  76      34.785  32.804  24.978  1.00  0.00           C  \\nATOM    808  C   THR B  76      36.050  32.092  25.454  1.00  0.00           C  \\nATOM    809  O   THR B  76      36.195  31.740  26.589  1.00  0.00           O  \\nATOM    810  CB  THR B  76      33.755  31.751  24.462  1.00  0.00           C  \\nATOM    811  OG1 THR B  76      34.210  31.162  23.263  1.00  0.00           O  \\nATOM    812  CG2 THR B  76      32.367  32.349  24.258  1.00  0.00           C  \\nATOM    813  H   THR B  76      34.790  33.758  23.271  1.00  0.00           H  \\nATOM    814  HA  THR B  76      34.367  33.209  25.757  1.00  0.00           H  \\nATOM    815  HB  THR B  76      33.668  30.964  25.209  1.00  0.00           H  \\nATOM    816  HG1 THR B  76      34.608  31.847  22.702  1.00  0.00           H  \\nATOM    817 1HG2 THR B  76      31.686  31.564  23.923  1.00  0.00           H  \\nATOM    818 2HG2 THR B  76      32.012  32.759  25.203  1.00  0.00           H  \\nATOM    819 3HG2 THR B  76      32.400  33.133  23.524  1.00  0.00           H  \\nATOM    820  N   LEU B  77      36.976  31.938  24.544  1.00  0.00           N  \\nATOM    821  CA  LEU B  77      38.243  31.293  24.841  1.00  0.00           C  \\nATOM    822  C   LEU B  77      39.042  32.153  25.812  1.00  0.00           C  \\nATOM    823  O   LEU B  77      39.753  31.633  26.671  1.00  0.00           O  \\nATOM    824  CB  LEU B  77      38.996  31.042  23.534  1.00  0.00           C  \\nATOM    825  CG  LEU B  77      40.272  30.223  23.594  1.00  0.00           C  \\nATOM    826  CD1 LEU B  77      39.973  28.822  24.181  1.00  0.00           C  \\nATOM    827  CD2 LEU B  77      40.815  30.107  22.179  1.00  0.00           C  \\nATOM    828  H   LEU B  77      36.896  32.205  23.730  1.00  0.00           H  \\nATOM    829  HA  LEU B  77      38.074  30.436  25.263  1.00  0.00           H  \\nATOM    830 1HB  LEU B  77      38.320  30.519  22.859  1.00  0.00           H  \\nATOM    831 2HB  LEU B  77      39.235  32.009  23.086  1.00  0.00           H  \\nATOM    832  HG  LEU B  77      41.002  30.720  24.234  1.00  0.00           H  \\nATOM    833 1HD1 LEU B  77      39.241  28.311  23.554  1.00  0.00           H  \\nATOM    834 2HD1 LEU B  77      40.889  28.234  24.217  1.00  0.00           H  \\nATOM    835 3HD1 LEU B  77      39.574  28.932  25.186  1.00  0.00           H  \\nATOM    836 1HD2 LEU B  77      41.721  29.539  22.191  1.00  0.00           H  \\nATOM    837 2HD2 LEU B  77      40.077  29.608  21.546  1.00  0.00           H  \\nATOM    838 3HD2 LEU B  77      41.016  31.104  21.786  1.00  0.00           H  \\nATOM    839  N   LYS B  78      38.909  33.471  25.687  1.00  0.00           N  \\nATOM    840  CA  LYS B  78      39.591  34.390  26.594  1.00  0.00           C  \\nATOM    841  C   LYS B  78      38.963  34.340  27.984  1.00  0.00           C  \\nATOM    842  O   LYS B  78      39.668  34.400  28.992  1.00  0.00           O  \\nATOM    843  CB  LYS B  78      39.579  35.813  26.048  1.00  0.00           C  \\nATOM    844  CG  LYS B  78      40.454  36.790  26.843  1.00  0.00           C  \\nATOM    845  CD  LYS B  78      41.953  36.404  26.702  1.00  0.00           C  \\nATOM    846  CE  LYS B  78      42.883  37.437  27.342  1.00  0.00           C  \\nATOM    847  NZ  LYS B  78      44.339  37.051  27.212  1.00  0.00           N  \\nATOM    848  H   LYS B  78      38.431  33.858  25.086  1.00  0.00           H  \\nATOM    849  HA  LYS B  78      40.520  34.122  26.674  1.00  0.00           H  \\nATOM    850 1HB  LYS B  78      39.912  35.809  25.008  1.00  0.00           H  \\nATOM    851 2HB  LYS B  78      38.557  36.193  26.058  1.00  0.00           H  \\nATOM    852 1HG  LYS B  78      40.301  37.803  26.476  1.00  0.00           H  \\nATOM    853 2HG  LYS B  78      40.174  36.753  27.900  1.00  0.00           H  \\nATOM    854 1HD  LYS B  78      42.126  35.438  27.191  1.00  0.00           H  \\nATOM    855 2HD  LYS B  78      42.207  36.304  25.648  1.00  0.00           H  \\nATOM    856 1HE  LYS B  78      42.729  38.395  26.850  1.00  0.00           H  \\nATOM    857 2HE  LYS B  78      42.638  37.536  28.402  1.00  0.00           H  \\nATOM    858 1HZ  LYS B  78      44.914  37.756  27.637  1.00  0.00           H  \\nATOM    859 2HZ  LYS B  78      44.517  36.145  27.694  1.00  0.00           H  \\nATOM    860 3HZ  LYS B  78      44.584  36.959  26.241  1.00  0.00           H  \\nATOM    861  N   ARG B  79      37.638  34.239  28.035  1.00  0.00           N  \\nATOM    862  CA  ARG B  79      36.939  34.079  29.307  1.00  0.00           C  \\nATOM    863  C   ARG B  79      37.380  32.789  29.983  1.00  0.00           C  \\nATOM    864  O   ARG B  79      37.585  32.749  31.196  1.00  0.00           O  \\nATOM    865  CB  ARG B  79      35.440  34.081  29.118  1.00  0.00           C  \\nATOM    866  CG  ARG B  79      34.852  35.425  28.740  1.00  0.00           C  \\nATOM    867  CD  ARG B  79      33.411  35.310  28.542  1.00  0.00           C  \\nATOM    868  NE  ARG B  79      32.745  34.991  29.783  1.00  0.00           N  \\nATOM    869  CZ  ARG B  79      31.610  34.294  29.866  1.00  0.00           C  \\nATOM    870  NH1 ARG B  79      31.044  33.861  28.770  1.00  0.00           N  \\nATOM    871  NH2 ARG B  79      31.075  34.044  31.037  1.00  0.00           N  \\nATOM    872  H   ARG B  79      37.121  34.260  27.349  1.00  0.00           H  \\nATOM    873  HA  ARG B  79      37.161  34.821  29.890  1.00  0.00           H  \\nATOM    874 1HB  ARG B  79      35.173  33.372  28.339  1.00  0.00           H  \\nATOM    875 2HB  ARG B  79      34.957  33.753  30.035  1.00  0.00           H  \\nATOM    876 1HG  ARG B  79      35.039  36.145  29.535  1.00  0.00           H  \\nATOM    877 2HG  ARG B  79      35.305  35.776  27.816  1.00  0.00           H  \\nATOM    878 1HD  ARG B  79      33.012  36.252  28.168  1.00  0.00           H  \\nATOM    879 2HD  ARG B  79      33.204  34.514  27.825  1.00  0.00           H  \\nATOM    880  HE  ARG B  79      33.167  35.312  30.644  1.00  0.00           H  \\nATOM    881 1HH1 ARG B  79      31.464  34.058  27.879  1.00  0.00           H  \\nATOM    882 2HH1 ARG B  79      30.189  33.291  28.803  1.00  0.00           H  \\nATOM    883 1HH2 ARG B  79      31.523  34.376  31.880  1.00  0.00           H  \\nATOM    884 2HH2 ARG B  79      30.222  33.507  31.090  1.00  0.00           H  \\nATOM    885  N   LEU B  80      37.524  31.735  29.187  1.00  0.00           N  \\nATOM    886  CA  LEU B  80      37.970  30.445  29.697  1.00  0.00           C  \\nATOM    887  C   LEU B  80      39.362  30.561  30.318  1.00  0.00           C  \\nATOM    888  O   LEU B  80      39.635  29.971  31.366  1.00  0.00           O  \\nATOM    889  CB  LEU B  80      37.932  29.412  28.556  1.00  0.00           C  \\nATOM    890  CG  LEU B  80      38.245  27.973  28.918  1.00  0.00           C  \\nATOM    891  CD1 LEU B  80      37.246  27.514  29.973  1.00  0.00           C  \\nATOM    892  CD2 LEU B  80      38.125  27.100  27.642  1.00  0.00           C  \\nATOM    893  H   LEU B  80      37.369  31.742  28.341  1.00  0.00           H  \\nATOM    894  HA  LEU B  80      37.356  30.147  30.386  1.00  0.00           H  \\nATOM    895 1HB  LEU B  80      36.948  29.440  28.089  1.00  0.00           H  \\nATOM    896 2HB  LEU B  80      38.659  29.710  27.812  1.00  0.00           H  \\nATOM    897  HG  LEU B  80      39.252  27.896  29.333  1.00  0.00           H  \\nATOM    898 1HD1 LEU B  80      36.247  27.598  29.576  1.00  0.00           H  \\nATOM    899 2HD1 LEU B  80      37.447  26.480  30.246  1.00  0.00           H  \\nATOM    900 3HD1 LEU B  80      37.333  28.144  30.856  1.00  0.00           H  \\nATOM    901 1HD2 LEU B  80      38.326  26.060  27.886  1.00  0.00           H  \\nATOM    902 2HD2 LEU B  80      37.114  27.183  27.234  1.00  0.00           H  \\nATOM    903 3HD2 LEU B  80      38.839  27.439  26.897  1.00  0.00           H  \\nATOM    904  N   GLU B  81      40.231  31.335  29.672  1.00  0.00           N  \\nATOM    905  CA  GLU B  81      41.593  31.544  30.155  1.00  0.00           C  \\nATOM    906  C   GLU B  81      41.589  32.221  31.518  1.00  0.00           C  \\nATOM    907  O   GLU B  81      42.308  31.807  32.426  1.00  0.00           O  \\nATOM    908  CB  GLU B  81      42.377  32.408  29.149  1.00  0.00           C  \\nATOM    909  CG  GLU B  81      43.839  32.729  29.501  1.00  0.00           C  \\nATOM    910  CD  GLU B  81      44.493  33.612  28.426  1.00  0.00           C  \\nATOM    911  OE1 GLU B  81      44.480  33.242  27.280  1.00  0.00           O  \\nATOM    912  OE2 GLU B  81      44.967  34.685  28.755  1.00  0.00           O  \\nATOM    913  H   GLU B  81      40.053  31.756  28.943  1.00  0.00           H  \\nATOM    914  HA  GLU B  81      42.031  30.684  30.248  1.00  0.00           H  \\nATOM    915 1HB  GLU B  81      42.360  31.926  28.171  1.00  0.00           H  \\nATOM    916 2HB  GLU B  81      41.881  33.361  29.039  1.00  0.00           H  \\nATOM    917 1HG  GLU B  81      43.862  33.255  30.458  1.00  0.00           H  \\nATOM    918 2HG  GLU B  81      44.398  31.801  29.606  1.00  0.00           H  \\nATOM    919  N   ASP B  82      40.772  33.260  31.661  1.00  0.00           N  \\nATOM    920  CA  ASP B  82      40.694  33.995  32.918  1.00  0.00           C  \\nATOM    921  C   ASP B  82      40.091  33.126  34.013  1.00  0.00           C  \\nATOM    922  O   ASP B  82      40.580  33.110  35.143  1.00  0.00           O  \\nATOM    923  CB  ASP B  82      39.902  35.295  32.763  1.00  0.00           C  \\nATOM    924  CG  ASP B  82      40.691  36.378  32.014  1.00  0.00           C  \\nATOM    925  OD1 ASP B  82      41.875  36.189  31.843  1.00  0.00           O  \\nATOM    926  OD2 ASP B  82      40.120  37.380  31.647  1.00  0.00           O  \\nATOM    927  H   ASP B  82      40.251  33.559  31.045  1.00  0.00           H  \\nATOM    928  HA  ASP B  82      41.589  34.249  33.193  1.00  0.00           H  \\nATOM    929 1HB  ASP B  82      38.984  35.086  32.207  1.00  0.00           H  \\nATOM    930 2HB  ASP B  82      39.617  35.672  33.743  1.00  0.00           H  \\nATOM    931  N   SER B  83      39.029  32.403  33.672  1.00  0.00           N  \\nATOM    932  CA  SER B  83      38.361  31.532  34.631  1.00  0.00           C  \\nATOM    933  C   SER B  83      39.310  30.443  35.113  1.00  0.00           C  \\nATOM    934  O   SER B  83      39.338  30.112  36.295  1.00  0.00           O  \\nATOM    935  CB  SER B  83      37.127  30.935  34.025  1.00  0.00           C  \\nATOM    936  OG  SER B  83      36.461  30.165  34.973  1.00  0.00           O  \\nATOM    937  H   SER B  83      38.675  32.399  32.888  1.00  0.00           H  \\nATOM    938  HA  SER B  83      38.087  32.056  35.400  1.00  0.00           H  \\nATOM    939 1HB  SER B  83      36.489  31.727  33.664  1.00  0.00           H  \\nATOM    940 2HB  SER B  83      37.403  30.316  33.169  1.00  0.00           H  \\nATOM    941  HG  SER B  83      37.141  29.648  35.408  1.00  0.00           H  \\nATOM    942  N   LEU B  84      40.096  29.898  34.191  1.00  0.00           N  \\nATOM    943  CA  LEU B  84      41.055  28.856  34.529  1.00  0.00           C  \\nATOM    944  C   LEU B  84      42.145  29.407  35.446  1.00  0.00           C  \\nATOM    945  O   LEU B  84      42.570  28.737  36.390  1.00  0.00           O  \\nATOM    946  CB  LEU B  84      41.656  28.240  33.266  1.00  0.00           C  \\nATOM    947  CG  LEU B  84      42.638  27.082  33.495  1.00  0.00           C  \\nATOM    948  CD1 LEU B  84      41.946  25.929  34.253  1.00  0.00           C  \\nATOM    949  CD2 LEU B  84      43.168  26.620  32.161  1.00  0.00           C  \\nATOM    950  H   LEU B  84      40.093  30.116  33.359  1.00  0.00           H  \\nATOM    951  HA  LEU B  84      40.595  28.144  35.001  1.00  0.00           H  \\nATOM    952 1HB  LEU B  84      40.839  27.876  32.638  1.00  0.00           H  \\nATOM    953 2HB  LEU B  84      42.178  29.027  32.716  1.00  0.00           H  \\nATOM    954  HG  LEU B  84      43.444  27.429  34.093  1.00  0.00           H  \\nATOM    955 1HD1 LEU B  84      42.658  25.118  34.422  1.00  0.00           H  \\nATOM    956 2HD1 LEU B  84      41.585  26.296  35.210  1.00  0.00           H  \\nATOM    957 3HD1 LEU B  84      41.106  25.554  33.666  1.00  0.00           H  \\nATOM    958 1HD2 LEU B  84      43.878  25.831  32.322  1.00  0.00           H  \\nATOM    959 2HD2 LEU B  84      42.343  26.259  31.545  1.00  0.00           H  \\nATOM    960 3HD2 LEU B  84      43.661  27.453  31.660  1.00  0.00           H  \\nATOM    961  N   ARG B  85      42.587  30.632  35.169  1.00  0.00           N  \\nATOM    962  CA  ARG B  85      43.560  31.306  36.025  1.00  0.00           C  \\nATOM    963  C   ARG B  85      42.993  31.486  37.427  1.00  0.00           C  \\nATOM    964  O   ARG B  85      43.672  31.220  38.421  1.00  0.00           O  \\nATOM    965  CB  ARG B  85      43.916  32.665  35.443  1.00  0.00           C  \\nATOM    966  CG  ARG B  85      45.007  33.439  36.151  1.00  0.00           C  \\nATOM    967  CD  ARG B  85      45.153  34.794  35.547  1.00  0.00           C  \\nATOM    968  NE  ARG B  85      45.525  34.727  34.140  1.00  0.00           N  \\nATOM    969  CZ  ARG B  85      45.141  35.626  33.209  1.00  0.00           C  \\nATOM    970  NH1 ARG B  85      44.386  36.649  33.551  1.00  0.00           N  \\nATOM    971  NH2 ARG B  85      45.510  35.479  31.949  1.00  0.00           N  \\nATOM    972  H   ARG B  85      42.338  31.096  34.489  1.00  0.00           H  \\nATOM    973  HA  ARG B  85      44.360  30.762  36.088  1.00  0.00           H  \\nATOM    974 1HB  ARG B  85      44.208  32.545  34.399  1.00  0.00           H  \\nATOM    975 2HB  ARG B  85      43.036  33.298  35.450  1.00  0.00           H  \\nATOM    976 1HG  ARG B  85      44.746  33.553  37.205  1.00  0.00           H  \\nATOM    977 2HG  ARG B  85      45.956  32.909  36.062  1.00  0.00           H  \\nATOM    978 1HD  ARG B  85      44.201  35.322  35.621  1.00  0.00           H  \\nATOM    979 2HD  ARG B  85      45.922  35.352  36.079  1.00  0.00           H  \\nATOM    980  HE  ARG B  85      46.096  33.951  33.837  1.00  0.00           H  \\nATOM    981 1HH1 ARG B  85      44.086  36.763  34.504  1.00  0.00           H  \\nATOM    982 2HH1 ARG B  85      44.084  37.311  32.842  1.00  0.00           H  \\nATOM    983 1HH2 ARG B  85      46.075  34.693  31.667  1.00  0.00           H  \\nATOM    984 2HH2 ARG B  85      45.201  36.144  31.257  1.00  0.00           H  \\nATOM    985  N   GLU B  86      41.753  31.958  37.499  1.00  0.00           N  \\nATOM    986  CA  GLU B  86      41.075  32.134  38.777  1.00  0.00           C  \\nATOM    987  C   GLU B  86      41.061  30.815  39.546  1.00  0.00           C  \\nATOM    988  O   GLU B  86      41.308  30.780  40.750  1.00  0.00           O  \\nATOM    989  CB  GLU B  86      39.661  32.686  38.604  1.00  0.00           C  \\nATOM    990  CG  GLU B  86      38.863  32.822  39.913  1.00  0.00           C  \\nATOM    991  CD  GLU B  86      39.491  33.757  40.936  1.00  0.00           C  \\nATOM    992  OE1 GLU B  86      40.397  34.491  40.610  1.00  0.00           O  \\nATOM    993  OE2 GLU B  86      39.050  33.728  42.063  1.00  0.00           O  \\nATOM    994  H   GLU B  86      41.280  32.185  36.817  1.00  0.00           H  \\nATOM    995  HA  GLU B  86      41.555  32.790  39.307  1.00  0.00           H  \\nATOM    996 1HB  GLU B  86      39.709  33.664  38.126  1.00  0.00           H  \\nATOM    997 2HB  GLU B  86      39.098  32.034  37.945  1.00  0.00           H  \\nATOM    998 1HG  GLU B  86      37.864  33.183  39.674  1.00  0.00           H  \\nATOM    999 2HG  GLU B  86      38.761  31.830  40.356  1.00  0.00           H  \\nATOM   1000  N   LEU B  87      40.793  29.730  38.829  1.00  0.00           N  \\nATOM   1001  CA  LEU B  87      40.686  28.409  39.436  1.00  0.00           C  \\nATOM   1002  C   LEU B  87      42.037  27.927  39.957  1.00  0.00           C  \\nATOM   1003  O   LEU B  87      42.142  27.441  41.085  1.00  0.00           O  \\nATOM   1004  CB  LEU B  87      40.105  27.452  38.381  1.00  0.00           C  \\nATOM   1005  CG  LEU B  87      39.924  26.011  38.746  1.00  0.00           C  \\nATOM   1006  CD1 LEU B  87      39.017  25.894  39.942  1.00  0.00           C  \\nATOM   1007  CD2 LEU B  87      39.307  25.288  37.521  1.00  0.00           C  \\nATOM   1008  H   LEU B  87      40.667  29.732  37.978  1.00  0.00           H  \\nATOM   1009  HA  LEU B  87      40.074  28.455  40.188  1.00  0.00           H  \\nATOM   1010 1HB  LEU B  87      39.133  27.840  38.068  1.00  0.00           H  \\nATOM   1011 2HB  LEU B  87      40.755  27.472  37.518  1.00  0.00           H  \\nATOM   1012  HG  LEU B  87      40.886  25.575  38.999  1.00  0.00           H  \\nATOM   1013 1HD1 LEU B  87      38.876  24.847  40.202  1.00  0.00           H  \\nATOM   1014 2HD1 LEU B  87      39.452  26.421  40.792  1.00  0.00           H  \\nATOM   1015 3HD1 LEU B  87      38.075  26.329  39.693  1.00  0.00           H  \\nATOM   1016 1HD2 LEU B  87      39.971  25.382  36.669  1.00  0.00           H  \\nATOM   1017 2HD2 LEU B  87      39.149  24.237  37.734  1.00  0.00           H  \\nATOM   1018 3HD2 LEU B  87      38.349  25.746  37.279  1.00  0.00           H  \\nATOM   1019  N   ARG B  88      43.069  28.072  39.132  1.00  0.00           N  \\nATOM   1020  CA  ARG B  88      44.414  27.652  39.511  1.00  0.00           C  \\nATOM   1021  C   ARG B  88      44.926  28.441  40.712  1.00  0.00           C  \\nATOM   1022  O   ARG B  88      45.667  27.911  41.536  1.00  0.00           O  \\nATOM   1023  CB  ARG B  88      45.353  27.762  38.327  1.00  0.00           C  \\nATOM   1024  CG  ARG B  88      45.111  26.686  37.255  1.00  0.00           C  \\nATOM   1025  CD  ARG B  88      46.012  26.829  36.092  1.00  0.00           C  \\nATOM   1026  NE  ARG B  88      45.679  27.988  35.297  1.00  0.00           N  \\nATOM   1027  CZ  ARG B  88      46.387  28.422  34.239  1.00  0.00           C  \\nATOM   1028  NH1 ARG B  88      47.483  27.794  33.874  1.00  0.00           N  \\nATOM   1029  NH2 ARG B  88      45.983  29.480  33.566  1.00  0.00           N  \\nATOM   1030  H   ARG B  88      43.017  28.412  38.344  1.00  0.00           H  \\nATOM   1031  HA  ARG B  88      44.393  26.714  39.756  1.00  0.00           H  \\nATOM   1032 1HB  ARG B  88      45.220  28.738  37.855  1.00  0.00           H  \\nATOM   1033 2HB  ARG B  88      46.385  27.690  38.663  1.00  0.00           H  \\nATOM   1034 1HG  ARG B  88      45.280  25.704  37.691  1.00  0.00           H  \\nATOM   1035 2HG  ARG B  88      44.080  26.748  36.901  1.00  0.00           H  \\nATOM   1036 1HD  ARG B  88      47.039  26.933  36.436  1.00  0.00           H  \\nATOM   1037 2HD  ARG B  88      45.924  25.945  35.458  1.00  0.00           H  \\nATOM   1038  HE  ARG B  88      44.832  28.497  35.548  1.00  0.00           H  \\nATOM   1039 1HH1 ARG B  88      47.797  26.988  34.394  1.00  0.00           H  \\nATOM   1040 2HH1 ARG B  88      48.013  28.125  33.080  1.00  0.00           H  \\nATOM   1041 1HH2 ARG B  88      45.142  29.961  33.851  1.00  0.00           H  \\nATOM   1042 2HH2 ARG B  88      46.509  29.807  32.770  1.00  0.00           H  \\nATOM   1043  N   ARG B  89      44.521  29.704  40.817  1.00  0.00           N  \\nATOM   1044  CA  ARG B  89      44.936  30.545  41.935  1.00  0.00           C  \\nATOM   1045  C   ARG B  89      44.346  30.033  43.242  1.00  0.00           C  \\nATOM   1046  O   ARG B  89      45.046  29.917  44.248  1.00  0.00           O  \\nATOM   1047  CB  ARG B  89      44.442  31.975  41.722  1.00  0.00           C  \\nATOM   1048  CG  ARG B  89      44.833  32.977  42.795  1.00  0.00           C  \\nATOM   1049  CD  ARG B  89      44.139  34.311  42.614  1.00  0.00           C  \\nATOM   1050  NE  ARG B  89      42.678  34.245  42.877  1.00  0.00           N  \\nATOM   1051  CZ  ARG B  89      42.135  34.231  44.128  1.00  0.00           C  \\nATOM   1052  NH1 ARG B  89      42.922  34.262  45.189  1.00  0.00           N  \\nATOM   1053  NH2 ARG B  89      40.822  34.194  44.295  1.00  0.00           N  \\nATOM   1054  H   ARG B  89      44.006  30.098  40.253  1.00  0.00           H  \\nATOM   1055  HA  ARG B  89      45.903  30.517  42.011  1.00  0.00           H  \\nATOM   1056 1HB  ARG B  89      44.808  32.344  40.764  1.00  0.00           H  \\nATOM   1057 2HB  ARG B  89      43.354  31.975  41.666  1.00  0.00           H  \\nATOM   1058 1HG  ARG B  89      44.554  32.580  43.771  1.00  0.00           H  \\nATOM   1059 2HG  ARG B  89      45.910  33.141  42.763  1.00  0.00           H  \\nATOM   1060 1HD  ARG B  89      44.574  35.042  43.293  1.00  0.00           H  \\nATOM   1061 2HD  ARG B  89      44.278  34.645  41.586  1.00  0.00           H  \\nATOM   1062  HE  ARG B  89      42.020  34.244  42.069  1.00  0.00           H  \\nATOM   1063 1HH1 ARG B  89      43.925  34.289  45.076  1.00  0.00           H  \\nATOM   1064 2HH1 ARG B  89      42.518  34.251  46.117  1.00  0.00           H  \\nATOM   1065 1HH2 ARG B  89      40.198  34.162  43.470  1.00  0.00           H  \\nATOM   1066 2HH2 ARG B  89      40.430  34.184  45.220  1.00  0.00           H  \\nATOM   1067  N   ILE B  90      43.050  29.737  43.218  1.00  0.00           N  \\nATOM   1068  CA  ILE B  90      42.351  29.213  44.386  1.00  0.00           C  \\nATOM   1069  C   ILE B  90      42.933  27.861  44.779  1.00  0.00           C  \\nATOM   1070  O   ILE B  90      42.972  27.507  45.958  1.00  0.00           O  \\nATOM   1071  CB  ILE B  90      40.848  29.075  44.134  1.00  0.00           C  \\nATOM   1072  CG1 ILE B  90      40.233  30.451  43.982  1.00  0.00           C  \\nATOM   1073  CG2 ILE B  90      40.198  28.338  45.313  1.00  0.00           C  \\nATOM   1074  CD1 ILE B  90      38.849  30.409  43.454  1.00  0.00           C  \\nATOM   1075  H   ILE B  90      42.548  29.831  42.526  1.00  0.00           H  \\nATOM   1076  HA  ILE B  90      42.471  29.823  45.130  1.00  0.00           H  \\nATOM   1077  HB  ILE B  90      40.677  28.523  43.205  1.00  0.00           H  \\nATOM   1078 1HG1 ILE B  90      40.223  30.945  44.951  1.00  0.00           H  \\nATOM   1079 2HG1 ILE B  90      40.843  31.045  43.294  1.00  0.00           H  \\nATOM   1080 1HG2 ILE B  90      39.136  28.250  45.139  1.00  0.00           H  \\nATOM   1081 2HG2 ILE B  90      40.640  27.349  45.409  1.00  0.00           H  \\nATOM   1082 3HG2 ILE B  90      40.367  28.899  46.233  1.00  0.00           H  \\nATOM   1083 1HD1 ILE B  90      38.462  31.427  43.358  1.00  0.00           H  \\nATOM   1084 2HD1 ILE B  90      38.883  29.940  42.477  1.00  0.00           H  \\nATOM   1085 3HD1 ILE B  90      38.208  29.834  44.122  1.00  0.00           H  \\nATOM   1086  N   LEU B  91      43.392  27.109  43.786  1.00  0.00           N  \\nATOM   1087  CA  LEU B  91      43.991  25.809  44.047  1.00  0.00           C  \\nATOM   1088  C   LEU B  91      45.312  25.962  44.800  1.00  0.00           C  \\nATOM   1089  O   LEU B  91      45.623  25.158  45.673  1.00  0.00           O  \\nATOM   1090  CB  LEU B  91      44.228  25.034  42.751  1.00  0.00           C  \\nATOM   1091  CG  LEU B  91      44.785  23.616  42.927  1.00  0.00           C  \\nATOM   1092  CD1 LEU B  91      43.826  22.791  43.780  1.00  0.00           C  \\nATOM   1093  CD2 LEU B  91      44.965  22.996  41.568  1.00  0.00           C  \\nATOM   1094  H   LEU B  91      43.368  27.328  42.955  1.00  0.00           H  \\nATOM   1095  HA  LEU B  91      43.388  25.289  44.601  1.00  0.00           H  \\nATOM   1096 1HB  LEU B  91      43.290  24.975  42.197  1.00  0.00           H  \\nATOM   1097 2HB  LEU B  91      44.943  25.586  42.150  1.00  0.00           H  \\nATOM   1098  HG  LEU B  91      45.751  23.661  43.437  1.00  0.00           H  \\nATOM   1099 1HD1 LEU B  91      44.217  21.779  43.907  1.00  0.00           H  \\nATOM   1100 2HD1 LEU B  91      43.713  23.258  44.756  1.00  0.00           H  \\nATOM   1101 3HD1 LEU B  91      42.870  22.750  43.286  1.00  0.00           H  \\nATOM   1102 1HD2 LEU B  91      45.364  22.005  41.681  1.00  0.00           H  \\nATOM   1103 2HD2 LEU B  91      44.013  22.950  41.075  1.00  0.00           H  \\nATOM   1104 3HD2 LEU B  91      45.655  23.599  40.979  1.00  0.00           H  \\nATOM   1105  N   GLU B  92      46.086  26.990  44.459  1.00  0.00           N  \\nATOM   1106  CA  GLU B  92      47.334  27.270  45.167  1.00  0.00           C  \\nATOM   1107  C   GLU B  92      47.059  27.586  46.632  1.00  0.00           C  \\nATOM   1108  O   GLU B  92      47.791  27.152  47.518  1.00  0.00           O  \\nATOM   1109  CB  GLU B  92      48.119  28.405  44.502  1.00  0.00           C  \\nATOM   1110  CG  GLU B  92      48.755  28.035  43.151  1.00  0.00           C  \\nATOM   1111  CD  GLU B  92      49.882  27.021  43.270  1.00  0.00           C  \\nATOM   1112  OE1 GLU B  92      50.793  27.264  44.026  1.00  0.00           O  \\nATOM   1113  OE2 GLU B  92      49.827  26.008  42.607  1.00  0.00           O  \\nATOM   1114  H   GLU B  92      45.911  27.541  43.823  1.00  0.00           H  \\nATOM   1115  HA  GLU B  92      47.903  26.485  45.129  1.00  0.00           H  \\nATOM   1116 1HB  GLU B  92      47.463  29.253  44.336  1.00  0.00           H  \\nATOM   1117 2HB  GLU B  92      48.914  28.735  45.170  1.00  0.00           H  \\nATOM   1118 1HG  GLU B  92      47.983  27.617  42.511  1.00  0.00           H  \\nATOM   1119 2HG  GLU B  92      49.129  28.940  42.676  1.00  0.00           H  \\nATOM   1120  N   GLU B  93      45.999  28.350  46.878  1.00  0.00           N  \\nATOM   1121  CA  GLU B  93      45.582  28.670  48.238  1.00  0.00           C  \\nATOM   1122  C   GLU B  93      45.148  27.410  48.975  1.00  0.00           C  \\nATOM   1123  O   GLU B  93      45.513  27.195  50.130  1.00  0.00           O  \\nATOM   1124  CB  GLU B  93      44.432  29.691  48.257  1.00  0.00           C  \\nATOM   1125  CG  GLU B  93      43.964  30.089  49.679  1.00  0.00           C  \\nATOM   1126  CD  GLU B  93      42.902  31.187  49.703  1.00  0.00           C  \\nATOM   1127  OE1 GLU B  93      42.535  31.674  48.656  1.00  0.00           O  \\nATOM   1128  OE2 GLU B  93      42.469  31.537  50.776  1.00  0.00           O  \\nATOM   1129  H   GLU B  93      45.501  28.698  46.269  1.00  0.00           H  \\nATOM   1130  HA  GLU B  93      46.331  29.059  48.718  1.00  0.00           H  \\nATOM   1131 1HB  GLU B  93      44.734  30.592  47.726  1.00  0.00           H  \\nATOM   1132 2HB  GLU B  93      43.571  29.278  47.733  1.00  0.00           H  \\nATOM   1133 1HG  GLU B  93      43.556  29.202  50.168  1.00  0.00           H  \\nATOM   1134 2HG  GLU B  93      44.830  30.415  50.254  1.00  0.00           H  \\nATOM   1135  N   LEU B  94      44.360  26.583  48.295  1.00  0.00           N  \\nATOM   1136  CA  LEU B  94      43.876  25.330  48.862  1.00  0.00           C  \\nATOM   1137  C   LEU B  94      45.039  24.404  49.208  1.00  0.00           C  \\nATOM   1138  O   LEU B  94      45.004  23.690  50.209  1.00  0.00           O  \\nATOM   1139  CB  LEU B  94      42.908  24.651  47.878  1.00  0.00           C  \\nATOM   1140  CG  LEU B  94      42.280  23.347  48.345  1.00  0.00           C  \\nATOM   1141  CD1 LEU B  94      41.538  23.561  49.665  1.00  0.00           C  \\nATOM   1142  CD2 LEU B  94      41.291  22.843  47.247  1.00  0.00           C  \\nATOM   1143  H   LEU B  94      44.088  26.729  47.492  1.00  0.00           H  \\nATOM   1144  HA  LEU B  94      43.387  25.518  49.678  1.00  0.00           H  \\nATOM   1145 1HB  LEU B  94      42.126  25.346  47.604  1.00  0.00           H  \\nATOM   1146 2HB  LEU B  94      43.470  24.421  46.979  1.00  0.00           H  \\nATOM   1147  HG  LEU B  94      43.067  22.618  48.510  1.00  0.00           H  \\nATOM   1148 1HD1 LEU B  94      41.126  22.621  49.987  1.00  0.00           H  \\nATOM   1149 2HD1 LEU B  94      42.234  23.923  50.422  1.00  0.00           H  \\nATOM   1150 3HD1 LEU B  94      40.744  24.277  49.535  1.00  0.00           H  \\nATOM   1151 1HD2 LEU B  94      40.832  21.909  47.553  1.00  0.00           H  \\nATOM   1152 2HD2 LEU B  94      40.511  23.591  47.093  1.00  0.00           H  \\nATOM   1153 3HD2 LEU B  94      41.833  22.687  46.314  1.00  0.00           H  \\nATOM   1154  N   LYS B  95      46.072  24.431  48.373  1.00  0.00           N  \\nATOM   1155  CA  LYS B  95      47.251  23.596  48.574  1.00  0.00           C  \\nATOM   1156  C   LYS B  95      48.018  24.013  49.826  1.00  0.00           C  \\nATOM   1157  O   LYS B  95      48.537  23.169  50.556  1.00  0.00           O  \\nATOM   1158  CB  LYS B  95      48.090  23.600  47.296  1.00  0.00           C  \\nATOM   1159  CG  LYS B  95      49.190  22.550  47.229  1.00  0.00           C  \\nATOM   1160  CD  LYS B  95      49.746  22.477  45.794  1.00  0.00           C  \\nATOM   1161  CE  LYS B  95      50.760  21.348  45.615  1.00  0.00           C  \\nATOM   1162  NZ  LYS B  95      50.111  19.973  45.719  1.00  0.00           N  \\nATOM   1163  H   LYS B  95      46.115  24.930  47.674  1.00  0.00           H  \\nATOM   1164  HA  LYS B  95      46.972  22.674  48.684  1.00  0.00           H  \\nATOM   1165 1HB  LYS B  95      47.435  23.452  46.435  1.00  0.00           H  \\nATOM   1166 2HB  LYS B  95      48.554  24.583  47.178  1.00  0.00           H  \\nATOM   1167 1HG  LYS B  95      49.992  22.814  47.921  1.00  0.00           H  \\nATOM   1168 2HG  LYS B  95      48.787  21.577  47.511  1.00  0.00           H  \\nATOM   1169 1HD  LYS B  95      48.918  22.307  45.099  1.00  0.00           H  \\nATOM   1170 2HD  LYS B  95      50.218  23.428  45.537  1.00  0.00           H  \\nATOM   1171 1HE  LYS B  95      51.227  21.442  44.635  1.00  0.00           H  \\nATOM   1172 2HE  LYS B  95      51.528  21.434  46.383  1.00  0.00           H  \\nATOM   1173 1HZ  LYS B  95      49.689  19.870  46.623  1.00  0.00           H  \\nATOM   1174 2HZ  LYS B  95      49.370  19.829  44.988  1.00  0.00           H  \\nATOM   1175 3HZ  LYS B  95      50.814  19.264  45.596  1.00  0.00           H  \\nATOM   1176  N   GLU B  96      48.092  25.318  50.064  1.00  0.00           N  \\nATOM   1177  CA  GLU B  96      48.741  25.843  51.260  1.00  0.00           C  \\nATOM   1178  C   GLU B  96      47.933  25.548  52.517  1.00  0.00           C  \\nATOM   1179  O   GLU B  96      48.494  25.190  53.553  1.00  0.00           O  \\nATOM   1180  CB  GLU B  96      48.990  27.358  51.176  1.00  0.00           C  \\nATOM   1181  CG  GLU B  96      50.068  27.790  50.175  1.00  0.00           C  \\nATOM   1182  CD  GLU B  96      50.343  29.297  50.189  1.00  0.00           C  \\nATOM   1183  OE1 GLU B  96      49.669  30.011  50.902  1.00  0.00           O  \\nATOM   1184  OE2 GLU B  96      51.242  29.722  49.496  1.00  0.00           O  \\nATOM   1185  H   GLU B  96      47.772  25.925  49.545  1.00  0.00           H  \\nATOM   1186  HA  GLU B  96      49.610  25.421  51.357  1.00  0.00           H  \\nATOM   1187 1HB  GLU B  96      48.060  27.855  50.890  1.00  0.00           H  \\nATOM   1188 2HB  GLU B  96      49.271  27.734  52.159  1.00  0.00           H  \\nATOM   1189 1HG  GLU B  96      50.991  27.261  50.406  1.00  0.00           H  \\nATOM   1190 2HG  GLU B  96      49.754  27.492  49.177  1.00  0.00           H  \\nATOM   1191  N   MET B  97      46.617  25.711  52.425  1.00  0.00           N  \\nATOM   1192  CA  MET B  97      45.735  25.451  53.557  1.00  0.00           C  \\nATOM   1193  C   MET B  97      45.838  23.999  54.010  1.00  0.00           C  \\nATOM   1194  O   MET B  97      45.679  23.697  55.192  1.00  0.00           O  \\nATOM   1195  CB  MET B  97      44.283  25.739  53.197  1.00  0.00           C  \\nATOM   1196  CG  MET B  97      43.919  27.161  52.987  1.00  0.00           C  \\nATOM   1197  SD  MET B  97      44.065  28.158  54.457  1.00  0.00           S  \\nATOM   1198  CE  MET B  97      43.726  29.767  53.744  1.00  0.00           C  \\nATOM   1199  H   MET B  97      46.208  25.973  51.715  1.00  0.00           H  \\nATOM   1200  HA  MET B  97      45.994  26.027  54.293  1.00  0.00           H  \\nATOM   1201 1HB  MET B  97      44.012  25.183  52.298  1.00  0.00           H  \\nATOM   1202 2HB  MET B  97      43.656  25.402  53.988  1.00  0.00           H  \\nATOM   1203 1HG  MET B  97      44.534  27.601  52.227  1.00  0.00           H  \\nATOM   1204 2HG  MET B  97      42.902  27.186  52.658  1.00  0.00           H  \\nATOM   1205 1HE  MET B  97      43.771  30.529  54.522  1.00  0.00           H  \\nATOM   1206 2HE  MET B  97      44.466  29.989  52.971  1.00  0.00           H  \\nATOM   1207 3HE  MET B  97      42.732  29.769  53.292  1.00  0.00           H  \\nATOM   1208  N   LEU B  98      46.110  23.103  53.067  1.00  0.00           N  \\nATOM   1209  CA  LEU B  98      46.230  21.685  53.382  1.00  0.00           C  \\nATOM   1210  C   LEU B  98      47.504  21.417  54.178  1.00  0.00           C  \\nATOM   1211  O   LEU B  98      47.492  20.651  55.141  1.00  0.00           O  \\nATOM   1212  CB  LEU B  98      46.223  20.840  52.104  1.00  0.00           C  \\nATOM   1213  CG  LEU B  98      46.201  19.316  52.332  1.00  0.00           C  \\nATOM   1214  CD1 LEU B  98      44.911  18.956  53.106  1.00  0.00           C  \\nATOM   1215  CD2 LEU B  98      46.266  18.598  51.003  1.00  0.00           C  \\nATOM   1216  H   LEU B  98      46.229  23.291  52.236  1.00  0.00           H  \\nATOM   1217  HA  LEU B  98      45.473  21.416  53.925  1.00  0.00           H  \\nATOM   1218 1HB  LEU B  98      45.350  21.113  51.508  1.00  0.00           H  \\nATOM   1219 2HB  LEU B  98      47.118  21.082  51.524  1.00  0.00           H  \\nATOM   1220  HG  LEU B  98      47.059  19.026  52.941  1.00  0.00           H  \\nATOM   1221 1HD1 LEU B  98      44.871  17.890  53.296  1.00  0.00           H  \\nATOM   1222 2HD1 LEU B  98      44.901  19.491  54.053  1.00  0.00           H  \\nATOM   1223 3HD1 LEU B  98      44.040  19.246  52.516  1.00  0.00           H  \\nATOM   1224 1HD2 LEU B  98      46.252  17.524  51.159  1.00  0.00           H  \\nATOM   1225 2HD2 LEU B  98      45.419  18.887  50.412  1.00  0.00           H  \\nATOM   1226 3HD2 LEU B  98      47.184  18.876  50.486  1.00  0.00           H  \\nATOM   1227  N   GLU B  99      48.602  22.049  53.771  1.00  0.00           N  \\nATOM   1228  CA  GLU B  99      49.869  21.910  54.481  1.00  0.00           C  \\nATOM   1229  C   GLU B  99      49.745  22.430  55.910  1.00  0.00           C  \\nATOM   1230  O   GLU B  99      50.341  21.875  56.833  1.00  0.00           O  \\nATOM   1231  CB  GLU B  99      50.970  22.688  53.745  1.00  0.00           C  \\nATOM   1232  CG  GLU B  99      51.377  22.122  52.367  1.00  0.00           C  \\nATOM   1233  CD  GLU B  99      52.105  20.807  52.412  1.00  0.00           C  \\nATOM   1234  OE1 GLU B  99      53.092  20.723  53.103  1.00  0.00           O  \\nATOM   1235  OE2 GLU B  99      51.680  19.888  51.747  1.00  0.00           O  \\nATOM   1236  H   GLU B  99      48.639  22.565  53.084  1.00  0.00           H  \\nATOM   1237  HA  GLU B  99      50.108  20.971  54.522  1.00  0.00           H  \\nATOM   1238 1HB  GLU B  99      50.641  23.715  53.591  1.00  0.00           H  \\nATOM   1239 2HB  GLU B  99      51.862  22.724  54.368  1.00  0.00           H  \\nATOM   1240 1HG  GLU B  99      50.472  21.991  51.777  1.00  0.00           H  \\nATOM   1241 2HG  GLU B  99      51.998  22.859  51.858  1.00  0.00           H  \\nATOM   1242  N   ARG B 100      48.969  23.496  56.087  1.00  0.00           N  \\nATOM   1243  CA  ARG B 100      48.696  24.029  57.418  1.00  0.00           C  \\nATOM   1244  C   ARG B 100      47.926  23.017  58.257  1.00  0.00           C  \\nATOM   1245  O   ARG B 100      47.992  23.038  59.486  1.00  0.00           O  \\nATOM   1246  CB  ARG B 100      47.878  25.313  57.334  1.00  0.00           C  \\nATOM   1247  CG  ARG B 100      48.604  26.529  56.770  1.00  0.00           C  \\nATOM   1248  CD  ARG B 100      47.705  27.726  56.708  1.00  0.00           C  \\nATOM   1249  NE  ARG B 100      48.355  28.874  56.085  1.00  0.00           N  \\nATOM   1250  CZ  ARG B 100      49.140  29.760  56.733  1.00  0.00           C  \\nATOM   1251  NH1 ARG B 100      49.363  29.627  58.022  1.00  0.00           N  \\nATOM   1252  NH2 ARG B 100      49.686  30.765  56.070  1.00  0.00           N  \\nATOM   1253  H   ARG B 100      48.587  23.930  55.450  1.00  0.00           H  \\nATOM   1254  HA  ARG B 100      49.537  24.216  57.865  1.00  0.00           H  \\nATOM   1255 1HB  ARG B 100      47.006  25.145  56.716  1.00  0.00           H  \\nATOM   1256 2HB  ARG B 100      47.520  25.576  58.328  1.00  0.00           H  \\nATOM   1257 1HG  ARG B 100      49.456  26.768  57.402  1.00  0.00           H  \\nATOM   1258 2HG  ARG B 100      48.949  26.307  55.763  1.00  0.00           H  \\nATOM   1259 1HD  ARG B 100      46.821  27.478  56.123  1.00  0.00           H  \\nATOM   1260 2HD  ARG B 100      47.404  28.007  57.714  1.00  0.00           H  \\nATOM   1261  HE  ARG B 100      48.211  29.012  55.092  1.00  0.00           H  \\nATOM   1262 1HH1 ARG B 100      48.943  28.862  58.531  1.00  0.00           H  \\nATOM   1263 2HH1 ARG B 100      49.949  30.293  58.504  1.00  0.00           H  \\nATOM   1264 1HH2 ARG B 100      49.515  30.869  55.077  1.00  0.00           H  \\nATOM   1265 2HH2 ARG B 100      50.273  31.429  56.553  1.00  0.00           H  \\nATOM   1266  N   LEU B 101      47.192  22.135  57.586  1.00  0.00           N  \\nATOM   1267  CA  LEU B 101      46.382  21.138  58.272  1.00  0.00           C  \\nATOM   1268  C   LEU B 101      47.231  19.941  58.697  1.00  0.00           C  \\nATOM   1269  O   LEU B 101      46.868  19.214  59.620  1.00  0.00           O  \\nATOM   1270  CB  LEU B 101      45.239  20.654  57.344  1.00  0.00           C  \\nATOM   1271  CG  LEU B 101      44.181  19.715  57.940  1.00  0.00           C  \\nATOM   1272  CD1 LEU B 101      43.459  20.385  59.056  1.00  0.00           C  \\nATOM   1273  CD2 LEU B 101      43.203  19.303  56.839  1.00  0.00           C  \\nATOM   1274  H   LEU B 101      47.146  22.094  56.728  1.00  0.00           H  \\nATOM   1275  HA  LEU B 101      46.003  21.535  59.072  1.00  0.00           H  \\nATOM   1276 1HB  LEU B 101      44.734  21.528  56.930  1.00  0.00           H  \\nATOM   1277 2HB  LEU B 101      45.677  20.106  56.530  1.00  0.00           H  \\nATOM   1278  HG  LEU B 101      44.684  18.839  58.348  1.00  0.00           H  \\nATOM   1279 1HD1 LEU B 101      42.738  19.704  59.488  1.00  0.00           H  \\nATOM   1280 2HD1 LEU B 101      44.174  20.688  59.809  1.00  0.00           H  \\nATOM   1281 3HD1 LEU B 101      42.950  21.243  58.662  1.00  0.00           H  \\nATOM   1282 1HD2 LEU B 101      42.470  18.635  57.245  1.00  0.00           H  \\nATOM   1283 2HD2 LEU B 101      42.710  20.172  56.443  1.00  0.00           H  \\nATOM   1284 3HD2 LEU B 101      43.744  18.802  56.041  1.00  0.00           H  \\nATOM   1285  N   GLU B 102      48.357  19.746  58.019  1.00  0.00           N  \\nATOM   1286  CA  GLU B 102      49.255  18.639  58.327  1.00  0.00           C  \\nATOM   1287  C   GLU B 102      50.165  18.992  59.498  1.00  0.00           C  \\nATOM   1288  O   GLU B 102      50.822  18.122  60.070  1.00  0.00           O  \\nATOM   1289  CB  GLU B 102      50.067  18.220  57.098  1.00  0.00           C  \\nATOM   1290  CG  GLU B 102      50.811  16.886  57.301  1.00  0.00           C  \\nATOM   1291  CD  GLU B 102      51.425  16.308  56.043  1.00  0.00           C  \\nATOM   1292  OE1 GLU B 102      52.265  16.934  55.457  1.00  0.00           O  \\nATOM   1293  OE2 GLU B 102      51.016  15.223  55.668  1.00  0.00           O  \\nATOM   1294  H   GLU B 102      48.626  20.244  57.372  1.00  0.00           H  \\nATOM   1295  HA  GLU B 102      48.725  17.853  58.575  1.00  0.00           H  \\nATOM   1296 1HB  GLU B 102      49.404  18.125  56.236  1.00  0.00           H  \\nATOM   1297 2HB  GLU B 102      50.800  18.997  56.864  1.00  0.00           H  \\nATOM   1298 1HG  GLU B 102      51.603  17.043  58.033  1.00  0.00           H  \\nATOM   1299 2HG  GLU B 102      50.113  16.163  57.719  1.00  0.00           H  \\nATOM   1300  N   LYS B 103      50.128  20.274  59.813  1.00  0.00           N  \\nATOM   1301  CA  LYS B 103      50.838  20.829  60.920  1.00  0.00           C  \\nATOM   1302  C   LYS B 103      49.996  20.642  62.151  1.00  0.00           C  \\nATOM   1303  O   LYS B 103      50.320  19.842  63.010  1.00  0.00           O  \\nATOM   1304  CB  LYS B 103      51.182  22.299  60.697  1.00  0.00           C  \\nATOM   1305  CG  LYS B 103      52.255  22.523  59.640  1.00  0.00           C  \\nATOM   1306  CD  LYS B 103      52.559  23.997  59.442  1.00  0.00           C  \\nATOM   1307  CE  LYS B 103      53.617  24.195  58.362  1.00  0.00           C  \\nATOM   1308  NZ  LYS B 103      53.924  25.636  58.135  1.00  0.00           N  \\nATOM   1309  H   LYS B 103      49.659  20.843  59.375  1.00  0.00           H  \\nATOM   1310  HA  LYS B 103      51.691  20.367  61.033  1.00  0.00           H  \\nATOM   1311 1HB  LYS B 103      50.296  22.841  60.389  1.00  0.00           H  \\nATOM   1312 2HB  LYS B 103      51.522  22.739  61.633  1.00  0.00           H  \\nATOM   1313 1HG  LYS B 103      53.167  22.010  59.940  1.00  0.00           H  \\nATOM   1314 2HG  LYS B 103      51.923  22.101  58.699  1.00  0.00           H  \\nATOM   1315 1HD  LYS B 103      51.647  24.516  59.150  1.00  0.00           H  \\nATOM   1316 2HD  LYS B 103      52.916  24.424  60.378  1.00  0.00           H  \\nATOM   1317 1HE  LYS B 103      54.529  23.682  58.661  1.00  0.00           H  \\nATOM   1318 2HE  LYS B 103      53.254  23.758  57.428  1.00  0.00           H  \\nATOM   1319 1HZ  LYS B 103      54.626  25.722  57.414  1.00  0.00           H  \\nATOM   1320 2HZ  LYS B 103      53.085  26.117  57.846  1.00  0.00           H  \\nATOM   1321 3HZ  LYS B 103      54.270  26.047  58.991  1.00  0.00           H  \\nATOM   1322  N   ASN B 104      48.907  21.385  62.235  1.00  0.00           N  \\nATOM   1323  CA  ASN B 104      47.981  21.227  63.340  1.00  0.00           C  \\nATOM   1324  C   ASN B 104      46.570  21.109  62.884  1.00  0.00           C  \\nATOM   1325  O   ASN B 104      45.897  22.086  62.667  1.00  0.00           O  \\nATOM   1326  CB  ASN B 104      48.183  22.397  64.294  1.00  0.00           C  \\nATOM   1327  CG  ASN B 104      48.063  23.763  63.584  1.00  0.00           C  \\nATOM   1328  OD1 ASN B 104      48.875  24.010  62.680  1.00  0.00           O  \\nATOM   1329  ND2 ASN B 104      47.155  24.629  63.953  1.00  0.00           N  \\nATOM   1330  H   ASN B 104      48.687  21.991  61.669  1.00  0.00           H  \\nATOM   1331  HA  ASN B 104      48.203  20.411  63.832  1.00  0.00           H  \\nATOM   1332 1HB  ASN B 104      47.424  22.350  65.085  1.00  0.00           H  \\nATOM   1333 2HB  ASN B 104      49.160  22.323  64.767  1.00  0.00           H  \\nATOM   1334 1HD2 ASN B 104      46.481  24.437  64.692  1.00  0.00           H  \\nATOM   1335 2HD2 ASN B 104      47.095  25.508  63.489  1.00  0.00           H  \\nATOM   1336  N   PRO B 105      46.131  19.797  62.714  1.00  0.00           N  \\nATOM   1337  CA  PRO B 105      44.671  19.680  62.728  1.00  0.00           C  \\nATOM   1338  C   PRO B 105      44.019  20.367  63.924  1.00  0.00           C  \\nATOM   1339  O   PRO B 105      44.379  20.070  65.061  1.00  0.00           O  \\nATOM   1340  CB  PRO B 105      44.524  18.152  62.816  1.00  0.00           C  \\nATOM   1341  CG  PRO B 105      45.803  17.608  62.120  1.00  0.00           C  \\nATOM   1342  CD  PRO B 105      46.925  18.589  62.577  1.00  0.00           C  \\nATOM   1343  HA  PRO B 105      44.288  20.015  61.896  1.00  0.00           H  \\nATOM   1344 1HB  PRO B 105      44.449  17.843  63.865  1.00  0.00           H  \\nATOM   1345 2HB  PRO B 105      43.593  17.838  62.324  1.00  0.00           H  \\nATOM   1346 1HG  PRO B 105      45.988  16.559  62.411  1.00  0.00           H  \\nATOM   1347 2HG  PRO B 105      45.661  17.605  61.029  1.00  0.00           H  \\nATOM   1348 1HD  PRO B 105      47.336  18.278  63.548  1.00  0.00           H  \\nATOM   1349 2HD  PRO B 105      47.711  18.705  61.820  1.00  0.00           H  \\nATOM   1350  N   ASP B 106      43.074  21.257  63.660  1.00  0.00           N  \\nATOM   1351  CA  ASP B 106      42.354  21.987  64.688  1.00  0.00           C  \\nATOM   1352  C   ASP B 106      40.849  21.888  64.434  1.00  0.00           C  \\nATOM   1353  O   ASP B 106      40.373  20.900  63.925  1.00  0.00           O  \\nATOM   1354  CB  ASP B 106      42.802  23.450  64.822  1.00  0.00           C  \\nATOM   1355  CG  ASP B 106      44.215  23.663  65.447  1.00  0.00           C  \\nATOM   1356  OD1 ASP B 106      44.735  22.781  66.069  1.00  0.00           O  \\nATOM   1357  OD2 ASP B 106      44.749  24.726  65.280  1.00  0.00           O  \\nATOM   1358  H   ASP B 106      42.821  21.459  62.868  1.00  0.00           H  \\nATOM   1359  HA  ASP B 106      42.550  21.604  65.566  1.00  0.00           H  \\nATOM   1360 1HB  ASP B 106      42.749  23.910  63.904  1.00  0.00           H  \\nATOM   1361 2HB  ASP B 106      42.127  23.978  65.474  1.00  0.00           H  \\nATOM   1362  N   LYS B 107      40.098  22.920  64.791  1.00  0.00           N  \\nATOM   1363  CA  LYS B 107      38.690  22.982  64.474  1.00  0.00           C  \\nATOM   1364  C   LYS B 107      38.529  24.141  63.561  1.00  0.00           C  \\nATOM   1365  O   LYS B 107      37.928  24.036  62.513  1.00  0.00           O  \\nATOM   1366  CB  LYS B 107      37.799  23.133  65.707  1.00  0.00           C  \\nATOM   1367  CG  LYS B 107      36.288  23.083  65.364  1.00  0.00           C  \\nATOM   1368  CD  LYS B 107      35.397  23.179  66.598  1.00  0.00           C  \\nATOM   1369  CE  LYS B 107      35.427  21.897  67.410  1.00  0.00           C  \\nATOM   1370  NZ  LYS B 107      34.483  21.947  68.560  1.00  0.00           N  \\nATOM   1371  H   LYS B 107      40.373  23.604  65.225  1.00  0.00           H  \\nATOM   1372  HA  LYS B 107      38.402  22.166  64.024  1.00  0.00           H  \\nATOM   1373 1HB  LYS B 107      38.017  22.340  66.421  1.00  0.00           H  \\nATOM   1374 2HB  LYS B 107      38.001  24.089  66.194  1.00  0.00           H  \\nATOM   1375 1HG  LYS B 107      36.047  23.916  64.699  1.00  0.00           H  \\nATOM   1376 2HG  LYS B 107      36.062  22.153  64.840  1.00  0.00           H  \\nATOM   1377 1HD  LYS B 107      35.712  24.019  67.218  1.00  0.00           H  \\nATOM   1378 2HD  LYS B 107      34.369  23.346  66.279  1.00  0.00           H  \\nATOM   1379 1HE  LYS B 107      35.151  21.063  66.764  1.00  0.00           H  \\nATOM   1380 2HE  LYS B 107      36.436  21.735  67.787  1.00  0.00           H  \\nATOM   1381 1HZ  LYS B 107      33.538  22.087  68.227  1.00  0.00           H  \\nATOM   1382 2HZ  LYS B 107      34.532  21.079  69.070  1.00  0.00           H  \\nATOM   1383 3HZ  LYS B 107      34.737  22.710  69.168  1.00  0.00           H  \\nATOM   1384  N   ASP B 108      39.070  25.270  63.968  1.00  0.00           N  \\nATOM   1385  CA  ASP B 108      38.957  26.410  63.067  1.00  0.00           C  \\nATOM   1386  C   ASP B 108      39.748  26.165  61.787  1.00  0.00           C  \\nATOM   1387  O   ASP B 108      39.370  26.633  60.713  1.00  0.00           O  \\nATOM   1388  CB  ASP B 108      39.470  27.680  63.742  1.00  0.00           C  \\nATOM   1389  CG  ASP B 108      38.549  28.176  64.851  1.00  0.00           C  \\nATOM   1390  OD1 ASP B 108      37.447  27.687  64.948  1.00  0.00           O  \\nATOM   1391  OD2 ASP B 108      38.959  29.031  65.595  1.00  0.00           O  \\nATOM   1392  H   ASP B 108      39.499  25.438  64.694  1.00  0.00           H  \\nATOM   1393  HA  ASP B 108      38.016  26.540  62.826  1.00  0.00           H  \\nATOM   1394 1HB  ASP B 108      40.460  27.493  64.160  1.00  0.00           H  \\nATOM   1395 2HB  ASP B 108      39.578  28.469  62.992  1.00  0.00           H  \\nATOM   1396  N   VAL B 109      40.848  25.430  61.909  1.00  0.00           N  \\nATOM   1397  CA  VAL B 109      41.693  25.120  60.763  1.00  0.00           C  \\nATOM   1398  C   VAL B 109      40.925  24.300  59.732  1.00  0.00           C  \\nATOM   1399  O   VAL B 109      40.909  24.634  58.547  1.00  0.00           O  \\nATOM   1400  CB  VAL B 109      42.987  24.378  61.145  1.00  0.00           C  \\nATOM   1401  CG1 VAL B 109      43.711  23.909  59.897  1.00  0.00           C  \\nATOM   1402  CG2 VAL B 109      43.908  25.320  61.906  1.00  0.00           C  \\nATOM   1403  H   VAL B 109      41.128  25.096  62.650  1.00  0.00           H  \\nATOM   1404  HA  VAL B 109      41.982  25.955  60.339  1.00  0.00           H  \\nATOM   1405  HB  VAL B 109      42.736  23.507  61.730  1.00  0.00           H  \\nATOM   1406 1HG1 VAL B 109      44.624  23.373  60.181  1.00  0.00           H  \\nATOM   1407 2HG1 VAL B 109      43.067  23.267  59.343  1.00  0.00           H  \\nATOM   1408 3HG1 VAL B 109      43.974  24.768  59.279  1.00  0.00           H  \\nATOM   1409 1HG2 VAL B 109      44.826  24.786  62.175  1.00  0.00           H  \\nATOM   1410 2HG2 VAL B 109      44.154  26.170  61.270  1.00  0.00           H  \\nATOM   1411 3HG2 VAL B 109      43.431  25.686  62.808  1.00  0.00           H  \\nATOM   1412  N   ILE B 110      40.291  23.227  60.191  1.00  0.00           N  \\nATOM   1413  CA  ILE B 110      39.518  22.359  59.310  1.00  0.00           C  \\nATOM   1414  C   ILE B 110      38.450  23.156  58.570  1.00  0.00           C  \\nATOM   1415  O   ILE B 110      38.302  23.034  57.354  1.00  0.00           O  \\nATOM   1416  CB  ILE B 110      38.864  21.155  60.019  1.00  0.00           C  \\nATOM   1417  CG1 ILE B 110      39.952  20.148  60.448  1.00  0.00           C  \\nATOM   1418  CG2 ILE B 110      37.817  20.508  59.103  1.00  0.00           C  \\nATOM   1419  CD1 ILE B 110      39.460  19.052  61.367  1.00  0.00           C  \\nATOM   1420  H   ILE B 110      40.292  22.978  61.014  1.00  0.00           H  \\nATOM   1421  HA  ILE B 110      40.117  21.956  58.647  1.00  0.00           H  \\nATOM   1422  HB  ILE B 110      38.372  21.501  60.934  1.00  0.00           H  \\nATOM   1423 1HG1 ILE B 110      40.366  19.683  59.556  1.00  0.00           H  \\nATOM   1424 2HG1 ILE B 110      40.746  20.693  60.964  1.00  0.00           H  \\nATOM   1425 1HG2 ILE B 110      37.356  19.666  59.611  1.00  0.00           H  \\nATOM   1426 2HG2 ILE B 110      37.052  21.238  58.847  1.00  0.00           H  \\nATOM   1427 3HG2 ILE B 110      38.295  20.170  58.210  1.00  0.00           H  \\nATOM   1428 1HD1 ILE B 110      39.057  19.496  62.279  1.00  0.00           H  \\nATOM   1429 2HD1 ILE B 110      38.680  18.481  60.869  1.00  0.00           H  \\nATOM   1430 3HD1 ILE B 110      40.288  18.392  61.622  1.00  0.00           H  \\nATOM   1431  N   VAL B 111      37.709  23.973  59.311  1.00  0.00           N  \\nATOM   1432  CA  VAL B 111      36.656  24.794  58.726  1.00  0.00           C  \\nATOM   1433  C   VAL B 111      37.227  25.723  57.661  1.00  0.00           C  \\nATOM   1434  O   VAL B 111      36.581  25.993  56.648  1.00  0.00           O  \\nATOM   1435  CB  VAL B 111      35.935  25.583  59.817  1.00  0.00           C  \\nATOM   1436  CG1 VAL B 111      34.989  26.595  59.190  1.00  0.00           C  \\nATOM   1437  CG2 VAL B 111      35.171  24.597  60.705  1.00  0.00           C  \\nATOM   1438  H   VAL B 111      37.797  24.070  60.161  1.00  0.00           H  \\nATOM   1439  HA  VAL B 111      35.990  24.212  58.303  1.00  0.00           H  \\nATOM   1440  HB  VAL B 111      36.670  26.127  60.417  1.00  0.00           H  \\nATOM   1441 1HG1 VAL B 111      34.483  27.153  59.976  1.00  0.00           H  \\nATOM   1442 2HG1 VAL B 111      35.566  27.278  58.569  1.00  0.00           H  \\nATOM   1443 3HG1 VAL B 111      34.257  26.086  58.582  1.00  0.00           H  \\nATOM   1444 1HG2 VAL B 111      34.672  25.149  61.490  1.00  0.00           H  \\nATOM   1445 2HG2 VAL B 111      34.443  24.060  60.114  1.00  0.00           H  \\nATOM   1446 3HG2 VAL B 111      35.866  23.889  61.146  1.00  0.00           H  \\nATOM   1447  N   GLU B 112      38.441  26.209  57.897  1.00  0.00           N  \\nATOM   1448  CA  GLU B 112      39.104  27.107  56.957  1.00  0.00           C  \\nATOM   1449  C   GLU B 112      39.414  26.401  55.636  1.00  0.00           C  \\nATOM   1450  O   GLU B 112      39.383  27.024  54.576  1.00  0.00           O  \\nATOM   1451  CB  GLU B 112      40.375  27.771  57.519  1.00  0.00           C  \\nATOM   1452  CG  GLU B 112      40.118  28.885  58.553  1.00  0.00           C  \\nATOM   1453  CD  GLU B 112      41.387  29.472  59.159  1.00  0.00           C  \\nATOM   1454  OE1 GLU B 112      42.454  28.971  58.890  1.00  0.00           O  \\nATOM   1455  OE2 GLU B 112      41.277  30.432  59.889  1.00  0.00           O  \\nATOM   1456  H   GLU B 112      38.906  26.033  58.599  1.00  0.00           H  \\nATOM   1457  HA  GLU B 112      38.514  27.852  56.765  1.00  0.00           H  \\nATOM   1458 1HB  GLU B 112      41.000  27.027  57.992  1.00  0.00           H  \\nATOM   1459 2HB  GLU B 112      40.951  28.199  56.698  1.00  0.00           H  \\nATOM   1460 1HG  GLU B 112      39.592  29.688  58.040  1.00  0.00           H  \\nATOM   1461 2HG  GLU B 112      39.467  28.533  59.329  1.00  0.00           H  \\nATOM   1462  N   VAL B 113      39.706  25.104  55.700  1.00  0.00           N  \\nATOM   1463  CA  VAL B 113      40.001  24.326  54.498  1.00  0.00           C  \\nATOM   1464  C   VAL B 113      38.723  24.029  53.715  1.00  0.00           C  \\nATOM   1465  O   VAL B 113      38.716  24.089  52.487  1.00  0.00           O  \\nATOM   1466  CB  VAL B 113      40.678  22.975  54.877  1.00  0.00           C  \\nATOM   1467  CG1 VAL B 113      40.863  22.102  53.633  1.00  0.00           C  \\nATOM   1468  CG2 VAL B 113      42.025  23.223  55.557  1.00  0.00           C  \\nATOM   1469  H   VAL B 113      39.740  24.650  56.429  1.00  0.00           H  \\nATOM   1470  HA  VAL B 113      40.590  24.842  53.925  1.00  0.00           H  \\nATOM   1471  HB  VAL B 113      40.022  22.436  55.564  1.00  0.00           H  \\nATOM   1472 1HG1 VAL B 113      41.314  21.150  53.916  1.00  0.00           H  \\nATOM   1473 2HG1 VAL B 113      39.892  21.925  53.183  1.00  0.00           H  \\nATOM   1474 3HG1 VAL B 113      41.511  22.612  52.918  1.00  0.00           H  \\nATOM   1475 1HG2 VAL B 113      42.477  22.289  55.832  1.00  0.00           H  \\nATOM   1476 2HG2 VAL B 113      42.671  23.732  54.884  1.00  0.00           H  \\nATOM   1477 3HG2 VAL B 113      41.880  23.824  56.447  1.00  0.00           H  \\nATOM   1478  N   LEU B 114      37.645  23.709  54.426  1.00  0.00           N  \\nATOM   1479  CA  LEU B 114      36.368  23.418  53.782  1.00  0.00           C  \\nATOM   1480  C   LEU B 114      35.861  24.630  53.009  1.00  0.00           C  \\nATOM   1481  O   LEU B 114      35.326  24.495  51.910  1.00  0.00           O  \\nATOM   1482  CB  LEU B 114      35.355  23.012  54.860  1.00  0.00           C  \\nATOM   1483  CG  LEU B 114      35.626  21.655  55.540  1.00  0.00           C  \\nATOM   1484  CD1 LEU B 114      34.728  21.496  56.740  1.00  0.00           C  \\nATOM   1485  CD2 LEU B 114      35.364  20.536  54.554  1.00  0.00           C  \\nATOM   1486  H   LEU B 114      37.627  23.654  55.284  1.00  0.00           H  \\nATOM   1487  HA  LEU B 114      36.490  22.690  53.152  1.00  0.00           H  \\nATOM   1488 1HB  LEU B 114      35.354  23.778  55.638  1.00  0.00           H  \\nATOM   1489 2HB  LEU B 114      34.362  22.976  54.413  1.00  0.00           H  \\nATOM   1490  HG  LEU B 114      36.666  21.616  55.878  1.00  0.00           H  \\nATOM   1491 1HD1 LEU B 114      34.941  20.555  57.204  1.00  0.00           H  \\nATOM   1492 2HD1 LEU B 114      34.932  22.290  57.442  1.00  0.00           H  \\nATOM   1493 3HD1 LEU B 114      33.684  21.530  56.434  1.00  0.00           H  \\nATOM   1494 1HD2 LEU B 114      35.550  19.586  55.043  1.00  0.00           H  \\nATOM   1495 2HD2 LEU B 114      34.329  20.573  54.228  1.00  0.00           H  \\nATOM   1496 3HD2 LEU B 114      36.019  20.643  53.693  1.00  0.00           H  \\nATOM   1497  N   LYS B 115      36.038  25.814  53.586  1.00  0.00           N  \\nATOM   1498  CA  LYS B 115      35.582  27.043  52.948  1.00  0.00           C  \\nATOM   1499  C   LYS B 115      36.336  27.303  51.649  1.00  0.00           C  \\nATOM   1500  O   LYS B 115      35.747  27.749  50.672  1.00  0.00           O  \\nATOM   1501  CB  LYS B 115      35.724  28.213  53.929  1.00  0.00           C  \\nATOM   1502  CG  LYS B 115      34.692  28.188  55.101  1.00  0.00           C  \\nATOM   1503  CD  LYS B 115      34.997  29.256  56.163  1.00  0.00           C  \\nATOM   1504  CE  LYS B 115      34.562  30.663  55.750  1.00  0.00           C  \\nATOM   1505  NZ  LYS B 115      33.074  30.847  55.848  1.00  0.00           N  \\nATOM   1506  H   LYS B 115      36.420  25.933  54.347  1.00  0.00           H  \\nATOM   1507  HA  LYS B 115      34.641  26.954  52.733  1.00  0.00           H  \\nATOM   1508 1HB  LYS B 115      36.727  28.189  54.370  1.00  0.00           H  \\nATOM   1509 2HB  LYS B 115      35.620  29.154  53.396  1.00  0.00           H  \\nATOM   1510 1HG  LYS B 115      33.695  28.370  54.706  1.00  0.00           H  \\nATOM   1511 2HG  LYS B 115      34.697  27.207  55.570  1.00  0.00           H  \\nATOM   1512 1HD  LYS B 115      34.481  28.990  57.085  1.00  0.00           H  \\nATOM   1513 2HD  LYS B 115      36.070  29.257  56.363  1.00  0.00           H  \\nATOM   1514 1HE  LYS B 115      35.050  31.389  56.399  1.00  0.00           H  \\nATOM   1515 2HE  LYS B 115      34.873  30.844  54.722  1.00  0.00           H  \\nATOM   1516 1HZ  LYS B 115      32.601  30.188  55.246  1.00  0.00           H  \\nATOM   1517 2HZ  LYS B 115      32.782  30.692  56.808  1.00  0.00           H  \\nATOM   1518 3HZ  LYS B 115      32.827  31.788  55.575  1.00  0.00           H  \\nATOM   1519  N   VAL B 116      37.633  27.015  51.639  1.00  0.00           N  \\nATOM   1520  CA  VAL B 116      38.446  27.193  50.440  1.00  0.00           C  \\nATOM   1521  C   VAL B 116      38.081  26.141  49.394  1.00  0.00           C  \\nATOM   1522  O   VAL B 116      38.090  26.420  48.193  1.00  0.00           O  \\nATOM   1523  CB  VAL B 116      39.952  27.178  50.719  1.00  0.00           C  \\nATOM   1524  CG1 VAL B 116      40.706  27.238  49.387  1.00  0.00           C  \\nATOM   1525  CG2 VAL B 116      40.316  28.386  51.557  1.00  0.00           C  \\nATOM   1526  H   VAL B 116      38.069  26.712  52.316  1.00  0.00           H  \\nATOM   1527  HA  VAL B 116      38.265  28.068  50.062  1.00  0.00           H  \\nATOM   1528  HB  VAL B 116      40.225  26.253  51.236  1.00  0.00           H  \\nATOM   1529 1HG1 VAL B 116      41.772  27.235  49.576  1.00  0.00           H  \\nATOM   1530 2HG1 VAL B 116      40.433  26.390  48.777  1.00  0.00           H  \\nATOM   1531 3HG1 VAL B 116      40.441  28.155  48.860  1.00  0.00           H  \\nATOM   1532 1HG2 VAL B 116      41.371  28.404  51.738  1.00  0.00           H  \\nATOM   1533 2HG2 VAL B 116      40.040  29.295  51.021  1.00  0.00           H  \\nATOM   1534 3HG2 VAL B 116      39.783  28.350  52.503  1.00  0.00           H  \\nATOM   1535  N   ILE B 117      37.747  24.938  49.853  1.00  0.00           N  \\nATOM   1536  CA  ILE B 117      37.345  23.857  48.955  1.00  0.00           C  \\nATOM   1537  C   ILE B 117      36.060  24.222  48.215  1.00  0.00           C  \\nATOM   1538  O   ILE B 117      35.910  23.910  47.037  1.00  0.00           O  \\nATOM   1539  CB  ILE B 117      37.103  22.507  49.667  1.00  0.00           C  \\nATOM   1540  CG1 ILE B 117      38.398  21.897  50.150  1.00  0.00           C  \\nATOM   1541  CG2 ILE B 117      36.380  21.590  48.737  1.00  0.00           C  \\nATOM   1542  CD1 ILE B 117      38.218  20.731  51.089  1.00  0.00           C  \\nATOM   1543  H   ILE B 117      37.744  24.720  50.686  1.00  0.00           H  \\nATOM   1544  HA  ILE B 117      38.042  23.723  48.294  1.00  0.00           H  \\nATOM   1545  HB  ILE B 117      36.488  22.669  50.547  1.00  0.00           H  \\nATOM   1546 1HG1 ILE B 117      38.956  21.550  49.282  1.00  0.00           H  \\nATOM   1547 2HG1 ILE B 117      38.969  22.649  50.643  1.00  0.00           H  \\nATOM   1548 1HG2 ILE B 117      36.195  20.647  49.220  1.00  0.00           H  \\nATOM   1549 2HG2 ILE B 117      35.436  22.038  48.442  1.00  0.00           H  \\nATOM   1550 3HG2 ILE B 117      36.983  21.440  47.882  1.00  0.00           H  \\nATOM   1551 1HD1 ILE B 117      39.195  20.345  51.375  1.00  0.00           H  \\nATOM   1552 2HD1 ILE B 117      37.687  21.061  51.972  1.00  0.00           H  \\nATOM   1553 3HD1 ILE B 117      37.655  19.957  50.608  1.00  0.00           H  \\nATOM   1554  N   VAL B 118      35.142  24.895  48.902  1.00  0.00           N  \\nATOM   1555  CA  VAL B 118      33.877  25.292  48.293  1.00  0.00           C  \\nATOM   1556  C   VAL B 118      34.087  26.365  47.230  1.00  0.00           C  \\nATOM   1557  O   VAL B 118      33.445  26.337  46.181  1.00  0.00           O  \\nATOM   1558  CB  VAL B 118      32.853  25.731  49.350  1.00  0.00           C  \\nATOM   1559  CG1 VAL B 118      31.587  26.275  48.653  1.00  0.00           C  \\nATOM   1560  CG2 VAL B 118      32.506  24.522  50.227  1.00  0.00           C  \\nATOM   1561  H   VAL B 118      35.228  25.135  49.723  1.00  0.00           H  \\nATOM   1562  HA  VAL B 118      33.481  24.519  47.861  1.00  0.00           H  \\nATOM   1563  HB  VAL B 118      33.279  26.530  49.966  1.00  0.00           H  \\nATOM   1564 1HG1 VAL B 118      30.858  26.583  49.400  1.00  0.00           H  \\nATOM   1565 2HG1 VAL B 118      31.856  27.130  48.031  1.00  0.00           H  \\nATOM   1566 3HG1 VAL B 118      31.151  25.493  48.029  1.00  0.00           H  \\nATOM   1567 1HG2 VAL B 118      31.780  24.799  50.982  1.00  0.00           H  \\nATOM   1568 2HG2 VAL B 118      32.106  23.756  49.614  1.00  0.00           H  \\nATOM   1569 3HG2 VAL B 118      33.383  24.150  50.701  1.00  0.00           H  \\nATOM   1570  N   LYS B 119      34.984  27.309  47.498  1.00  0.00           N  \\nATOM   1571  CA  LYS B 119      35.297  28.345  46.522  1.00  0.00           C  \\nATOM   1572  C   LYS B 119      35.963  27.742  45.296  1.00  0.00           C  \\nATOM   1573  O   LYS B 119      35.760  28.216  44.182  1.00  0.00           O  \\nATOM   1574  CB  LYS B 119      36.249  29.394  47.105  1.00  0.00           C  \\nATOM   1575  CG  LYS B 119      35.675  30.300  48.188  1.00  0.00           C  \\nATOM   1576  CD  LYS B 119      34.593  31.220  47.619  1.00  0.00           C  \\nATOM   1577  CE  LYS B 119      34.237  32.361  48.584  1.00  0.00           C  \\nATOM   1578  NZ  LYS B 119      33.685  31.871  49.872  1.00  0.00           N  \\nATOM   1579  H   LYS B 119      35.423  27.371  48.234  1.00  0.00           H  \\nATOM   1580  HA  LYS B 119      34.474  28.772  46.238  1.00  0.00           H  \\nATOM   1581 1HB  LYS B 119      37.117  28.883  47.534  1.00  0.00           H  \\nATOM   1582 2HB  LYS B 119      36.617  30.029  46.302  1.00  0.00           H  \\nATOM   1583 1HG  LYS B 119      35.238  29.681  48.975  1.00  0.00           H  \\nATOM   1584 2HG  LYS B 119      36.471  30.901  48.625  1.00  0.00           H  \\nATOM   1585 1HD  LYS B 119      34.943  31.650  46.679  1.00  0.00           H  \\nATOM   1586 2HD  LYS B 119      33.693  30.635  47.412  1.00  0.00           H  \\nATOM   1587 1HE  LYS B 119      35.129  32.953  48.784  1.00  0.00           H  \\nATOM   1588 2HE  LYS B 119      33.492  32.998  48.105  1.00  0.00           H  \\nATOM   1589 1HZ  LYS B 119      33.460  32.659  50.462  1.00  0.00           H  \\nATOM   1590 2HZ  LYS B 119      32.853  31.330  49.700  1.00  0.00           H  \\nATOM   1591 3HZ  LYS B 119      34.374  31.295  50.330  1.00  0.00           H  \\nATOM   1592  N   ALA B 120      36.752  26.693  45.507  1.00  0.00           N  \\nATOM   1593  CA  ALA B 120      37.411  26.007  44.403  1.00  0.00           C  \\nATOM   1594  C   ALA B 120      36.373  25.323  43.527  1.00  0.00           C  \\nATOM   1595  O   ALA B 120      36.441  25.390  42.300  1.00  0.00           O  \\nATOM   1596  CB  ALA B 120      38.412  24.980  44.929  1.00  0.00           C  \\nATOM   1597  H   ALA B 120      36.922  26.359  46.281  1.00  0.00           H  \\nATOM   1598  HA  ALA B 120      37.886  26.656  43.861  1.00  0.00           H  \\nATOM   1599 1HB  ALA B 120      37.891  24.238  45.527  1.00  0.00           H  \\nATOM   1600 2HB  ALA B 120      38.902  24.488  44.092  1.00  0.00           H  \\nATOM   1601 3HB  ALA B 120      39.155  25.475  45.547  1.00  0.00           H  \\nATOM   1602  N   ILE B 121      35.410  24.664  44.165  1.00  0.00           N  \\nATOM   1603  CA  ILE B 121      34.331  24.005  43.442  1.00  0.00           C  \\nATOM   1604  C   ILE B 121      33.499  25.042  42.700  1.00  0.00           C  \\nATOM   1605  O   ILE B 121      33.155  24.852  41.536  1.00  0.00           O  \\nATOM   1606  CB  ILE B 121      33.441  23.177  44.381  1.00  0.00           C  \\nATOM   1607  CG1 ILE B 121      34.250  21.986  44.915  1.00  0.00           C  \\nATOM   1608  CG2 ILE B 121      32.192  22.698  43.627  1.00  0.00           C  \\nATOM   1609  CD1 ILE B 121      33.616  21.261  46.057  1.00  0.00           C  \\nATOM   1610  H   ILE B 121      35.359  24.585  45.020  1.00  0.00           H  \\nATOM   1611  HA  ILE B 121      34.713  23.400  42.786  1.00  0.00           H  \\nATOM   1612  HB  ILE B 121      33.144  23.792  45.234  1.00  0.00           H  \\nATOM   1613 1HG1 ILE B 121      34.397  21.275  44.104  1.00  0.00           H  \\nATOM   1614 2HG1 ILE B 121      35.226  22.345  45.237  1.00  0.00           H  \\nATOM   1615 1HG2 ILE B 121      31.563  22.122  44.300  1.00  0.00           H  \\nATOM   1616 2HG2 ILE B 121      31.642  23.558  43.260  1.00  0.00           H  \\nATOM   1617 3HG2 ILE B 121      32.490  22.077  42.783  1.00  0.00           H  \\nATOM   1618 1HD1 ILE B 121      34.272  20.461  46.336  1.00  0.00           H  \\nATOM   1619 2HD1 ILE B 121      33.488  21.946  46.893  1.00  0.00           H  \\nATOM   1620 3HD1 ILE B 121      32.650  20.862  45.760  1.00  0.00           H  \\nATOM   1621  N   GLU B 122      33.179  26.143  43.372  1.00  0.00           N  \\nATOM   1622  CA  GLU B 122      32.456  27.229  42.722  1.00  0.00           C  \\nATOM   1623  C   GLU B 122      33.227  27.712  41.496  1.00  0.00           C  \\nATOM   1624  O   GLU B 122      32.656  27.880  40.417  1.00  0.00           O  \\nATOM   1625  CB  GLU B 122      32.216  28.407  43.677  1.00  0.00           C  \\nATOM   1626  CG  GLU B 122      31.445  29.583  43.034  1.00  0.00           C  \\nATOM   1627  CD  GLU B 122      31.154  30.739  43.974  1.00  0.00           C  \\nATOM   1628  OE1 GLU B 122      31.511  30.674  45.134  1.00  0.00           O  \\nATOM   1629  OE2 GLU B 122      30.541  31.679  43.537  1.00  0.00           O  \\nATOM   1630  H   GLU B 122      33.367  26.286  44.199  1.00  0.00           H  \\nATOM   1631  HA  GLU B 122      31.591  26.904  42.427  1.00  0.00           H  \\nATOM   1632 1HB  GLU B 122      31.661  28.062  44.550  1.00  0.00           H  \\nATOM   1633 2HB  GLU B 122      33.171  28.787  44.029  1.00  0.00           H  \\nATOM   1634 1HG  GLU B 122      32.042  29.959  42.202  1.00  0.00           H  \\nATOM   1635 2HG  GLU B 122      30.510  29.216  42.623  1.00  0.00           H  \\nATOM   1636  N   ALA B 123      34.529  27.913  41.667  1.00  0.00           N  \\nATOM   1637  CA  ALA B 123      35.385  28.352  40.574  1.00  0.00           C  \\nATOM   1638  C   ALA B 123      35.407  27.321  39.448  1.00  0.00           C  \\nATOM   1639  O   ALA B 123      35.443  27.678  38.275  1.00  0.00           O  \\nATOM   1640  CB  ALA B 123      36.791  28.608  41.079  1.00  0.00           C  \\nATOM   1641  H   ALA B 123      34.944  27.801  42.413  1.00  0.00           H  \\nATOM   1642  HA  ALA B 123      35.037  29.183  40.214  1.00  0.00           H  \\nATOM   1643 1HB  ALA B 123      37.423  28.961  40.264  1.00  0.00           H  \\nATOM   1644 2HB  ALA B 123      36.734  29.361  41.854  1.00  0.00           H  \\nATOM   1645 3HB  ALA B 123      37.205  27.698  41.492  1.00  0.00           H  \\nATOM   1646  N   SER B 124      35.374  26.043  39.814  1.00  0.00           N  \\nATOM   1647  CA  SER B 124      35.434  24.956  38.840  1.00  0.00           C  \\nATOM   1648  C   SER B 124      34.142  24.842  38.037  1.00  0.00           C  \\nATOM   1649  O   SER B 124      34.174  24.635  36.825  1.00  0.00           O  \\nATOM   1650  CB  SER B 124      35.701  23.662  39.573  1.00  0.00           C  \\nATOM   1651  OG  SER B 124      35.771  22.567  38.709  1.00  0.00           O  \\nATOM   1652  H   SER B 124      35.317  25.776  40.630  1.00  0.00           H  \\nATOM   1653  HA  SER B 124      36.158  25.130  38.218  1.00  0.00           H  \\nATOM   1654 1HB  SER B 124      36.628  23.749  40.134  1.00  0.00           H  \\nATOM   1655 2HB  SER B 124      34.908  23.497  40.291  1.00  0.00           H  \\nATOM   1656  HG  SER B 124      35.784  21.800  39.283  1.00  0.00           H  \\nATOM   1657  N   VAL B 125      33.008  24.971  38.717  1.00  0.00           N  \\nATOM   1658  CA  VAL B 125      31.712  24.958  38.047  1.00  0.00           C  \\nATOM   1659  C   VAL B 125      31.604  26.144  37.087  1.00  0.00           C  \\nATOM   1660  O   VAL B 125      31.047  26.023  35.995  1.00  0.00           O  \\nATOM   1661  CB  VAL B 125      30.557  24.978  39.080  1.00  0.00           C  \\nATOM   1662  CG1 VAL B 125      29.206  25.147  38.370  1.00  0.00           C  \\nATOM   1663  CG2 VAL B 125      30.562  23.671  39.879  1.00  0.00           C  \\nATOM   1664  H   VAL B 125      32.961  25.067  39.570  1.00  0.00           H  \\nATOM   1665  HA  VAL B 125      31.630  24.142  37.529  1.00  0.00           H  \\nATOM   1666  HB  VAL B 125      30.699  25.827  39.758  1.00  0.00           H  \\nATOM   1667 1HG1 VAL B 125      28.407  25.166  39.112  1.00  0.00           H  \\nATOM   1668 2HG1 VAL B 125      29.208  26.083  37.809  1.00  0.00           H  \\nATOM   1669 3HG1 VAL B 125      29.046  24.311  37.686  1.00  0.00           H  \\nATOM   1670 1HG2 VAL B 125      29.756  23.685  40.615  1.00  0.00           H  \\nATOM   1671 2HG2 VAL B 125      30.423  22.848  39.203  1.00  0.00           H  \\nATOM   1672 3HG2 VAL B 125      31.505  23.552  40.386  1.00  0.00           H  \\nATOM   1673  N   GLU B 126      32.146  27.285  37.495  1.00  0.00           N  \\nATOM   1674  CA  GLU B 126      32.132  28.476  36.655  1.00  0.00           C  \\nATOM   1675  C   GLU B 126      33.007  28.268  35.421  1.00  0.00           C  \\nATOM   1676  O   GLU B 126      32.635  28.654  34.314  1.00  0.00           O  \\nATOM   1677  CB  GLU B 126      32.535  29.716  37.450  1.00  0.00           C  \\nATOM   1678  CG  GLU B 126      32.475  31.042  36.666  1.00  0.00           C  \\nATOM   1679  CD  GLU B 126      31.085  31.367  36.110  1.00  0.00           C  \\nATOM   1680  OE1 GLU B 126      30.144  30.665  36.412  1.00  0.00           O  \\nATOM   1681  OE2 GLU B 126      30.957  32.360  35.425  1.00  0.00           O  \\nATOM   1682  H   GLU B 126      32.529  27.397  38.257  1.00  0.00           H  \\nATOM   1683  HA  GLU B 126      31.225  28.643  36.356  1.00  0.00           H  \\nATOM   1684 1HB  GLU B 126      31.894  29.807  38.327  1.00  0.00           H  \\nATOM   1685 2HB  GLU B 126      33.556  29.590  37.811  1.00  0.00           H  \\nATOM   1686 1HG  GLU B 126      32.792  31.850  37.323  1.00  0.00           H  \\nATOM   1687 2HG  GLU B 126      33.186  30.984  35.840  1.00  0.00           H  \\nATOM   1688  N   ASN B 127      34.165  27.648  35.613  1.00  0.00           N  \\nATOM   1689  CA  ASN B 127      35.051  27.352  34.495  1.00  0.00           C  \\nATOM   1690  C   ASN B 127      34.420  26.353  33.530  1.00  0.00           C  \\nATOM   1691  O   ASN B 127      34.582  26.463  32.317  1.00  0.00           O  \\nATOM   1692  CB  ASN B 127      36.384  26.818  35.022  1.00  0.00           C  \\nATOM   1693  CG  ASN B 127      37.487  26.703  33.971  1.00  0.00           C  \\nATOM   1694  OD1 ASN B 127      37.926  27.698  33.393  1.00  0.00           O  \\nATOM   1695  ND2 ASN B 127      37.925  25.502  33.712  1.00  0.00           N  \\nATOM   1696  H   ASN B 127      34.461  27.389  36.378  1.00  0.00           H  \\nATOM   1697  HA  ASN B 127      35.227  28.171  34.006  1.00  0.00           H  \\nATOM   1698 1HB  ASN B 127      36.735  27.462  35.831  1.00  0.00           H  \\nATOM   1699 2HB  ASN B 127      36.218  25.830  35.458  1.00  0.00           H  \\nATOM   1700 1HD2 ASN B 127      37.538  24.711  34.221  1.00  0.00           H  \\nATOM   1701 2HD2 ASN B 127      38.626  25.364  33.015  1.00  0.00           H  \\nATOM   1702  N   GLN B 128      33.697  25.380  34.073  1.00  0.00           N  \\nATOM   1703  CA  GLN B 128      33.050  24.372  33.247  1.00  0.00           C  \\nATOM   1704  C   GLN B 128      31.886  24.974  32.465  1.00  0.00           C  \\nATOM   1705  O   GLN B 128      31.623  24.569  31.337  1.00  0.00           O  \\nATOM   1706  CB  GLN B 128      32.618  23.200  34.120  1.00  0.00           C  \\nATOM   1707  CG  GLN B 128      33.823  22.388  34.662  1.00  0.00           C  \\nATOM   1708  CD  GLN B 128      33.450  21.259  35.606  1.00  0.00           C  \\nATOM   1709  OE1 GLN B 128      32.773  20.322  35.185  1.00  0.00           O  \\nATOM   1710  NE2 GLN B 128      33.874  21.305  36.870  1.00  0.00           N  \\nATOM   1711  H   GLN B 128      33.566  25.282  34.917  1.00  0.00           H  \\nATOM   1712  HA  GLN B 128      33.694  24.032  32.606  1.00  0.00           H  \\nATOM   1713 1HB  GLN B 128      32.050  23.577  34.973  1.00  0.00           H  \\nATOM   1714 2HB  GLN B 128      31.976  22.557  33.557  1.00  0.00           H  \\nATOM   1715 1HG  GLN B 128      34.323  21.937  33.809  1.00  0.00           H  \\nATOM   1716 2HG  GLN B 128      34.519  23.039  35.159  1.00  0.00           H  \\nATOM   1717 1HE2 GLN B 128      33.670  20.540  37.488  1.00  0.00           H  \\nATOM   1718 2HE2 GLN B 128      34.440  22.064  37.232  1.00  0.00           H  \\nATOM   1719  N   ARG B 129      31.204  25.951  33.056  1.00  0.00           N  \\nATOM   1720  CA  ARG B 129      30.122  26.647  32.366  1.00  0.00           C  \\nATOM   1721  C   ARG B 129      30.669  27.468  31.202  1.00  0.00           C  \\nATOM   1722  O   ARG B 129      30.090  27.490  30.119  1.00  0.00           O  \\nATOM   1723  CB  ARG B 129      29.372  27.546  33.339  1.00  0.00           C  \\nATOM   1724  CG  ARG B 129      28.090  28.173  32.796  1.00  0.00           C  \\nATOM   1725  CD  ARG B 129      27.401  29.025  33.830  1.00  0.00           C  \\nATOM   1726  NE  ARG B 129      28.100  30.281  34.092  1.00  0.00           N  \\nATOM   1727  CZ  ARG B 129      27.944  31.389  33.332  1.00  0.00           C  \\nATOM   1728  NH1 ARG B 129      27.109  31.366  32.302  1.00  0.00           N  \\nATOM   1729  NH2 ARG B 129      28.611  32.495  33.617  1.00  0.00           N  \\nATOM   1730  H   ARG B 129      31.348  26.231  33.856  1.00  0.00           H  \\nATOM   1731  HA  ARG B 129      29.500  25.993  32.009  1.00  0.00           H  \\nATOM   1732 1HB  ARG B 129      29.114  26.973  34.230  1.00  0.00           H  \\nATOM   1733 2HB  ARG B 129      30.030  28.360  33.661  1.00  0.00           H  \\nATOM   1734 1HG  ARG B 129      28.338  28.806  31.939  1.00  0.00           H  \\nATOM   1735 2HG  ARG B 129      27.406  27.385  32.481  1.00  0.00           H  \\nATOM   1736 1HD  ARG B 129      26.394  29.263  33.491  1.00  0.00           H  \\nATOM   1737 2HD  ARG B 129      27.346  28.472  34.768  1.00  0.00           H  \\nATOM   1738  HE  ARG B 129      28.758  30.336  34.897  1.00  0.00           H  \\nATOM   1739 1HH1 ARG B 129      26.603  30.522  32.076  1.00  0.00           H  \\nATOM   1740 2HH1 ARG B 129      26.992  32.185  31.719  1.00  0.00           H  \\nATOM   1741 1HH2 ARG B 129      29.285  32.501  34.406  1.00  0.00           H  \\nATOM   1742 2HH2 ARG B 129      28.473  33.320  33.063  1.00  0.00           H  \\nATOM   1743  N   ILE B 130      31.784  28.148  31.438  1.00  0.00           N  \\nATOM   1744  CA  ILE B 130      32.424  28.947  30.404  1.00  0.00           C  \\nATOM   1745  C   ILE B 130      32.994  28.037  29.322  1.00  0.00           C  \\nATOM   1746  O   ILE B 130      32.967  28.375  28.140  1.00  0.00           O  \\nATOM   1747  CB  ILE B 130      33.504  29.852  30.991  1.00  0.00           C  \\nATOM   1748  CG1 ILE B 130      32.833  30.900  31.856  1.00  0.00           C  \\nATOM   1749  CG2 ILE B 130      34.307  30.512  29.883  1.00  0.00           C  \\nATOM   1750  CD1 ILE B 130      33.778  31.674  32.697  1.00  0.00           C  \\nATOM   1751  H   ILE B 130      32.193  28.162  32.195  1.00  0.00           H  \\nATOM   1752  HA  ILE B 130      31.762  29.525  29.994  1.00  0.00           H  \\nATOM   1753  HB  ILE B 130      34.169  29.262  31.629  1.00  0.00           H  \\nATOM   1754 1HG1 ILE B 130      32.302  31.581  31.209  1.00  0.00           H  \\nATOM   1755 2HG1 ILE B 130      32.110  30.412  32.515  1.00  0.00           H  \\nATOM   1756 1HG2 ILE B 130      35.051  31.145  30.338  1.00  0.00           H  \\nATOM   1757 2HG2 ILE B 130      34.790  29.750  29.277  1.00  0.00           H  \\nATOM   1758 3HG2 ILE B 130      33.648  31.112  29.255  1.00  0.00           H  \\nATOM   1759 1HD1 ILE B 130      33.224  32.404  33.290  1.00  0.00           H  \\nATOM   1760 2HD1 ILE B 130      34.286  30.979  33.364  1.00  0.00           H  \\nATOM   1761 3HD1 ILE B 130      34.503  32.186  32.068  1.00  0.00           H  \\nATOM   1762  N   SER B 131      33.500  26.876  29.725  1.00  0.00           N  \\nATOM   1763  CA  SER B 131      34.026  25.916  28.761  1.00  0.00           C  \\nATOM   1764  C   SER B 131      32.922  25.417  27.839  1.00  0.00           C  \\nATOM   1765  O   SER B 131      33.143  25.226  26.643  1.00  0.00           O  \\nATOM   1766  CB  SER B 131      34.628  24.751  29.532  1.00  0.00           C  \\nATOM   1767  OG  SER B 131      35.132  23.741  28.702  1.00  0.00           O  \\nATOM   1768  H   SER B 131      33.550  26.621  30.544  1.00  0.00           H  \\nATOM   1769  HA  SER B 131      34.700  26.351  28.216  1.00  0.00           H  \\nATOM   1770 1HB  SER B 131      35.414  25.115  30.185  1.00  0.00           H  \\nATOM   1771 2HB  SER B 131      33.863  24.331  30.166  1.00  0.00           H  \\nATOM   1772  HG  SER B 131      35.365  23.026  29.298  1.00  0.00           H  \\nATOM   1773  N   ALA B 132      31.734  25.206  28.397  1.00  0.00           N  \\nATOM   1774  CA  ALA B 132      30.608  24.713  27.611  1.00  0.00           C  \\nATOM   1775  C   ALA B 132      30.191  25.752  26.580  1.00  0.00           C  \\nATOM   1776  O   ALA B 132      29.869  25.412  25.445  1.00  0.00           O  \\nATOM   1777  CB  ALA B 132      29.448  24.376  28.521  1.00  0.00           C  \\nATOM   1778  H   ALA B 132      31.554  25.340  29.228  1.00  0.00           H  \\nATOM   1779  HA  ALA B 132      30.878  23.910  27.139  1.00  0.00           H  \\nATOM   1780 1HB  ALA B 132      29.135  25.273  29.057  1.00  0.00           H  \\nATOM   1781 2HB  ALA B 132      28.616  23.994  27.934  1.00  0.00           H  \\nATOM   1782 3HB  ALA B 132      29.772  23.621  29.233  1.00  0.00           H  \\nATOM   1783  N   GLU B 133      30.199  27.020  26.976  1.00  0.00           N  \\nATOM   1784  CA  GLU B 133      29.883  28.094  26.045  1.00  0.00           C  \\nATOM   1785  C   GLU B 133      30.924  28.139  24.931  1.00  0.00           C  \\nATOM   1786  O   GLU B 133      30.588  28.339  23.764  1.00  0.00           O  \\nATOM   1787  CB  GLU B 133      29.786  29.454  26.753  1.00  0.00           C  \\nATOM   1788  CG  GLU B 133      28.541  29.622  27.659  1.00  0.00           C  \\nATOM   1789  CD  GLU B 133      28.468  30.990  28.344  1.00  0.00           C  \\nATOM   1790  OE1 GLU B 133      29.429  31.717  28.258  1.00  0.00           O  \\nATOM   1791  OE2 GLU B 133      27.456  31.304  28.937  1.00  0.00           O  \\nATOM   1792  H   GLU B 133      30.383  27.283  27.774  1.00  0.00           H  \\nATOM   1793  HA  GLU B 133      29.018  27.920  25.641  1.00  0.00           H  \\nATOM   1794 1HB  GLU B 133      30.671  29.597  27.376  1.00  0.00           H  \\nATOM   1795 2HB  GLU B 133      29.778  30.251  26.010  1.00  0.00           H  \\nATOM   1796 1HG  GLU B 133      27.646  29.479  27.055  1.00  0.00           H  \\nATOM   1797 2HG  GLU B 133      28.560  28.842  28.420  1.00  0.00           H  \\nATOM   1798  N   ASN B 134      32.180  27.941  25.260  1.00  0.00           N  \\nATOM   1799  CA  ASN B 134      33.232  27.883  24.260  1.00  0.00           C  \\nATOM   1800  C   ASN B 134      33.014  26.704  23.305  1.00  0.00           C  \\nATOM   1801  O   ASN B 134      33.284  26.782  22.141  1.00  0.00           O  \\nATOM   1802  CB  ASN B 134      34.623  27.836  24.874  1.00  0.00           C  \\nATOM   1803  CG  ASN B 134      35.708  27.983  23.795  1.00  0.00           C  \\nATOM   1804  OD1 ASN B 134      35.798  28.989  23.093  1.00  0.00           O  \\nATOM   1805  ND2 ASN B 134      36.511  26.988  23.645  1.00  0.00           N  \\nATOM   1806  H   ASN B 134      32.454  27.837  26.060  1.00  0.00           H  \\nATOM   1807  HA  ASN B 134      33.192  28.697  23.729  1.00  0.00           H  \\nATOM   1808 1HB  ASN B 134      34.732  28.632  25.614  1.00  0.00           H  \\nATOM   1809 2HB  ASN B 134      34.761  26.887  25.391  1.00  0.00           H  \\nATOM   1810 1HD2 ASN B 134      37.216  27.026  22.940  1.00  0.00           H  \\nATOM   1811 2HD2 ASN B 134      36.408  26.174  24.235  1.00  0.00           H  \\nATOM   1812  N   GLN B 135      32.466  25.631  23.811  1.00  0.00           N  \\nATOM   1813  CA  GLN B 135      32.158  24.505  22.980  1.00  0.00           C  \\nATOM   1814  C   GLN B 135      30.944  24.761  22.111  1.00  0.00           C  \\nATOM   1815  O   GLN B 135      30.899  24.317  21.012  1.00  0.00           O  \\nATOM   1816  CB  GLN B 135      31.964  23.256  23.817  1.00  0.00           C  \\nATOM   1817  CG  GLN B 135      33.238  22.762  24.445  1.00  0.00           C  \\nATOM   1818  CD  GLN B 135      33.017  21.623  25.393  1.00  0.00           C  \\nATOM   1819  OE1 GLN B 135      32.181  20.741  25.163  1.00  0.00           O  \\nATOM   1820  NE2 GLN B 135      33.780  21.623  26.465  1.00  0.00           N  \\nATOM   1821  H   GLN B 135      32.253  25.527  24.630  1.00  0.00           H  \\nATOM   1822  HA  GLN B 135      32.916  24.341  22.397  1.00  0.00           H  \\nATOM   1823 1HB  GLN B 135      31.243  23.448  24.604  1.00  0.00           H  \\nATOM   1824 2HB  GLN B 135      31.567  22.489  23.194  1.00  0.00           H  \\nATOM   1825 1HG  GLN B 135      33.899  22.416  23.652  1.00  0.00           H  \\nATOM   1826 2HG  GLN B 135      33.717  23.573  24.982  1.00  0.00           H  \\nATOM   1827 1HE2 GLN B 135      33.721  20.880  27.137  1.00  0.00           H  \\nATOM   1828 2HE2 GLN B 135      34.449  22.362  26.592  1.00  0.00           H  \\nATOM   1829  N   LYS B 136      29.964  25.467  22.630  1.00  0.00           N  \\nATOM   1830  CA  LYS B 136      28.813  25.890  21.838  1.00  0.00           C  \\nATOM   1831  C   LYS B 136      29.265  26.658  20.603  1.00  0.00           C  \\nATOM   1832  O   LYS B 136      28.810  26.394  19.491  1.00  0.00           O  \\nATOM   1833  CB  LYS B 136      27.820  26.698  22.674  1.00  0.00           C  \\nATOM   1834  CG  LYS B 136      26.992  25.834  23.644  1.00  0.00           C  \\nATOM   1835  CD  LYS B 136      26.125  26.675  24.580  1.00  0.00           C  \\nATOM   1836  CE  LYS B 136      25.021  27.403  23.824  1.00  0.00           C  \\nATOM   1837  NZ  LYS B 136      24.136  28.175  24.743  1.00  0.00           N  \\nATOM   1838  H   LYS B 136      29.942  25.728  23.449  1.00  0.00           H  \\nATOM   1839  HA  LYS B 136      28.320  25.107  21.547  1.00  0.00           H  \\nATOM   1840 1HB  LYS B 136      28.347  27.441  23.253  1.00  0.00           H  \\nATOM   1841 2HB  LYS B 136      27.134  27.224  22.014  1.00  0.00           H  \\nATOM   1842 1HG  LYS B 136      26.350  25.168  23.072  1.00  0.00           H  \\nATOM   1843 2HG  LYS B 136      27.667  25.227  24.240  1.00  0.00           H  \\nATOM   1844 1HD  LYS B 136      25.678  26.027  25.335  1.00  0.00           H  \\nATOM   1845 2HD  LYS B 136      26.747  27.409  25.086  1.00  0.00           H  \\nATOM   1846 1HE  LYS B 136      25.473  28.089  23.110  1.00  0.00           H  \\nATOM   1847 2HE  LYS B 136      24.418  26.674  23.283  1.00  0.00           H  \\nATOM   1848 1HZ  LYS B 136      23.419  28.643  24.206  1.00  0.00           H  \\nATOM   1849 2HZ  LYS B 136      23.703  27.544  25.404  1.00  0.00           H  \\nATOM   1850 3HZ  LYS B 136      24.683  28.862  25.244  1.00  0.00           H  \\nATOM   1851  N   ALA B 137      30.166  27.611  20.813  1.00  0.00           N  \\nATOM   1852  CA  ALA B 137      30.680  28.437  19.731  1.00  0.00           C  \\nATOM   1853  C   ALA B 137      31.405  27.587  18.694  1.00  0.00           C  \\nATOM   1854  O   ALA B 137      31.184  27.744  17.495  1.00  0.00           O  \\nATOM   1855  CB  ALA B 137      31.642  29.424  20.343  1.00  0.00           C  \\nATOM   1856  H   ALA B 137      30.498  27.800  21.584  1.00  0.00           H  \\nATOM   1857  HA  ALA B 137      29.938  28.880  19.291  1.00  0.00           H  \\nATOM   1858 1HB  ALA B 137      32.021  30.085  19.597  1.00  0.00           H  \\nATOM   1859 2HB  ALA B 137      31.117  29.962  21.097  1.00  0.00           H  \\nATOM   1860 3HB  ALA B 137      32.467  28.890  20.815  1.00  0.00           H  \\nATOM   1861  N   LEU B 138      32.269  26.689  19.163  1.00  0.00           N  \\nATOM   1862  CA  LEU B 138      33.019  25.802  18.277  1.00  0.00           C  \\nATOM   1863  C   LEU B 138      32.081  25.013  17.374  1.00  0.00           C  \\nATOM   1864  O   LEU B 138      32.362  24.818  16.192  1.00  0.00           O  \\nATOM   1865  CB  LEU B 138      33.941  24.857  19.074  1.00  0.00           C  \\nATOM   1866  CG  LEU B 138      35.199  25.527  19.740  1.00  0.00           C  \\nATOM   1867  CD1 LEU B 138      35.921  24.512  20.649  1.00  0.00           C  \\nATOM   1868  CD2 LEU B 138      36.146  26.030  18.647  1.00  0.00           C  \\nATOM   1869  H   LEU B 138      32.440  26.572  19.998  1.00  0.00           H  \\nATOM   1870  HA  LEU B 138      33.600  26.336  17.713  1.00  0.00           H  \\nATOM   1871 1HB  LEU B 138      33.350  24.401  19.866  1.00  0.00           H  \\nATOM   1872 2HB  LEU B 138      34.290  24.069  18.410  1.00  0.00           H  \\nATOM   1873  HG  LEU B 138      34.883  26.370  20.348  1.00  0.00           H  \\nATOM   1874 1HD1 LEU B 138      36.791  24.986  21.111  1.00  0.00           H  \\nATOM   1875 2HD1 LEU B 138      35.239  24.174  21.428  1.00  0.00           H  \\nATOM   1876 3HD1 LEU B 138      36.243  23.663  20.055  1.00  0.00           H  \\nATOM   1877 1HD2 LEU B 138      37.010  26.502  19.110  1.00  0.00           H  \\nATOM   1878 2HD2 LEU B 138      36.481  25.192  18.031  1.00  0.00           H  \\nATOM   1879 3HD2 LEU B 138      35.627  26.754  18.024  1.00  0.00           H  \\nATOM   1880  N   ALA B 139      30.965  24.566  17.940  1.00  0.00           N  \\nATOM   1881  CA  ALA B 139      29.960  23.830  17.185  1.00  0.00           C  \\nATOM   1882  C   ALA B 139      29.252  24.748  16.192  1.00  0.00           C  \\nATOM   1883  O   ALA B 139      28.191  24.410  15.667  1.00  0.00           O  \\nATOM   1884  OXT ALA B 139      29.729  25.814  15.915  1.00  0.00           O  \\nATOM   1885  CB  ALA B 139      28.948  23.220  18.146  1.00  0.00           C  \\nATOM   1886  H   ALA B 139      30.765  24.677  18.769  1.00  0.00           H  \\nATOM   1887  HA  ALA B 139      30.395  23.122  16.683  1.00  0.00           H  \\nATOM   1888 1HB  ALA B 139      28.205  22.665  17.584  1.00  0.00           H  \\nATOM   1889 2HB  ALA B 139      29.451  22.560  18.845  1.00  0.00           H  \\nATOM   1890 3HB  ALA B 139      28.458  24.016  18.702  1.00  0.00           H  \\nTER                                                                             \\nATOM   1892  N   THR C 140      27.512  14.029  12.268  1.00  0.00           N  \\nATOM   1893  CA  THR C 140      28.049  14.350  13.620  1.00  0.00           C  \\nATOM   1894  C   THR C 140      27.258  15.481  14.277  1.00  0.00           C  \\nATOM   1895  O   THR C 140      27.635  15.974  15.340  1.00  0.00           O  \\nATOM   1896  CB  THR C 140      29.530  14.766  13.543  1.00  0.00           C  \\nATOM   1897  OG1 THR C 140      30.116  14.723  14.854  1.00  0.00           O  \\nATOM   1898  CG2 THR C 140      29.638  16.187  13.002  1.00  0.00           C  \\nATOM   1899 1H   THR C 140      28.048  13.288  11.863  1.00  0.00           H  \\nATOM   1900 2H   THR C 140      26.557  13.745  12.347  1.00  0.00           H  \\nATOM   1901 3H   THR C 140      27.571  14.839  11.684  1.00  0.00           H  \\nATOM   1902  HA  THR C 140      27.974  13.564  14.184  1.00  0.00           H  \\nATOM   1903  HB  THR C 140      30.064  14.078  12.889  1.00  0.00           H  \\nATOM   1904  HG1 THR C 140      30.096  15.607  15.239  1.00  0.00           H  \\nATOM   1905 1HG2 THR C 140      30.686  16.478  12.951  1.00  0.00           H  \\nATOM   1906 2HG2 THR C 140      29.197  16.234  12.007  1.00  0.00           H  \\nATOM   1907 3HG2 THR C 140      29.106  16.866  13.664  1.00  0.00           H  \\nATOM   1908  N   LYS C 141      26.158  15.882  13.645  1.00  0.00           N  \\nATOM   1909  CA  LYS C 141      25.350  16.991  14.142  1.00  0.00           C  \\nATOM   1910  C   LYS C 141      24.547  16.563  15.360  1.00  0.00           C  \\nATOM   1911  O   LYS C 141      24.509  17.274  16.363  1.00  0.00           O  \\nATOM   1912  CB  LYS C 141      24.435  17.544  13.051  1.00  0.00           C  \\nATOM   1913  CG  LYS C 141      24.895  18.897  12.440  1.00  0.00           C  \\nATOM   1914  CD  LYS C 141      26.339  18.862  11.924  1.00  0.00           C  \\nATOM   1915  CE  LYS C 141      27.315  19.561  12.892  1.00  0.00           C  \\nATOM   1916  NZ  LYS C 141      27.163  21.065  12.876  1.00  0.00           N  \\nATOM   1917  H   LYS C 141      25.857  15.527  12.922  1.00  0.00           H  \\nATOM   1918  HA  LYS C 141      25.936  17.716  14.407  1.00  0.00           H  \\nATOM   1919 1HB  LYS C 141      24.351  16.822  12.241  1.00  0.00           H  \\nATOM   1920 2HB  LYS C 141      23.435  17.691  13.462  1.00  0.00           H  \\nATOM   1921 1HG  LYS C 141      24.234  19.158  11.613  1.00  0.00           H  \\nATOM   1922 2HG  LYS C 141      24.813  19.674  13.200  1.00  0.00           H  \\nATOM   1923 1HD  LYS C 141      26.654  17.829  11.793  1.00  0.00           H  \\nATOM   1924 2HD  LYS C 141      26.388  19.362  10.956  1.00  0.00           H  \\nATOM   1925 1HE  LYS C 141      27.132  19.198  13.904  1.00  0.00           H  \\nATOM   1926 2HE  LYS C 141      28.336  19.308  12.607  1.00  0.00           H  \\nATOM   1927 1HZ  LYS C 141      27.826  21.473  13.523  1.00  0.00           H  \\nATOM   1928 2HZ  LYS C 141      26.225  21.320  13.152  1.00  0.00           H  \\nATOM   1929 3HZ  LYS C 141      27.344  21.417  11.947  1.00  0.00           H  \\nATOM   1930  N   TYR C 142      23.902  15.403  15.271  1.00  0.00           N  \\nATOM   1931  CA  TYR C 142      23.134  14.886  16.395  1.00  0.00           C  \\nATOM   1932  C   TYR C 142      24.038  14.667  17.598  1.00  0.00           C  \\nATOM   1933  O   TYR C 142      23.651  14.944  18.729  1.00  0.00           O  \\nATOM   1934  CB  TYR C 142      22.445  13.573  16.035  1.00  0.00           C  \\nATOM   1935  CG  TYR C 142      21.591  13.023  17.159  1.00  0.00           C  \\nATOM   1936  CD1 TYR C 142      20.331  13.554  17.396  1.00  0.00           C  \\nATOM   1937  CD2 TYR C 142      22.070  11.992  17.956  1.00  0.00           C  \\nATOM   1938  CE1 TYR C 142      19.551  13.052  18.419  1.00  0.00           C  \\nATOM   1939  CE2 TYR C 142      21.290  11.493  18.981  1.00  0.00           C  \\nATOM   1940  CZ  TYR C 142      20.034  12.019  19.212  1.00  0.00           C  \\nATOM   1941  OH  TYR C 142      19.252  11.519  20.228  1.00  0.00           O  \\nATOM   1942  H   TYR C 142      23.895  14.899  14.574  1.00  0.00           H  \\nATOM   1943  HA  TYR C 142      22.457  15.536  16.641  1.00  0.00           H  \\nATOM   1944 1HB  TYR C 142      21.810  13.720  15.161  1.00  0.00           H  \\nATOM   1945 2HB  TYR C 142      23.194  12.826  15.779  1.00  0.00           H  \\nATOM   1946  HD1 TYR C 142      19.955  14.364  16.771  1.00  0.00           H  \\nATOM   1947  HD2 TYR C 142      23.062  11.579  17.774  1.00  0.00           H  \\nATOM   1948  HE1 TYR C 142      18.562  13.469  18.604  1.00  0.00           H  \\nATOM   1949  HE2 TYR C 142      21.664  10.682  19.607  1.00  0.00           H  \\nATOM   1950  HH  TYR C 142      18.429  12.012  20.269  1.00  0.00           H  \\nATOM   1951  N   LYS C 143      25.242  14.166  17.344  1.00  0.00           N  \\nATOM   1952  CA  LYS C 143      26.176  13.844  18.414  1.00  0.00           C  \\nATOM   1953  C   LYS C 143      26.519  15.091  19.227  1.00  0.00           C  \\nATOM   1954  O   LYS C 143      26.355  15.108  20.443  1.00  0.00           O  \\nATOM   1955  CB  LYS C 143      27.395  13.132  17.820  1.00  0.00           C  \\nATOM   1956  CG  LYS C 143      28.465  12.712  18.805  1.00  0.00           C  \\nATOM   1957  CD  LYS C 143      29.434  11.685  18.160  1.00  0.00           C  \\nATOM   1958  CE  LYS C 143      30.161  12.225  16.936  1.00  0.00           C  \\nATOM   1959  NZ  LYS C 143      31.145  11.238  16.393  1.00  0.00           N  \\nATOM   1960  H   LYS C 143      25.543  14.004  16.555  1.00  0.00           H  \\nATOM   1961  HA  LYS C 143      25.762  13.201  19.011  1.00  0.00           H  \\nATOM   1962 1HB  LYS C 143      27.068  12.243  17.283  1.00  0.00           H  \\nATOM   1963 2HB  LYS C 143      27.873  13.794  17.092  1.00  0.00           H  \\nATOM   1964 1HG  LYS C 143      29.016  13.591  19.124  1.00  0.00           H  \\nATOM   1965 2HG  LYS C 143      27.998  12.263  19.680  1.00  0.00           H  \\nATOM   1966 1HD  LYS C 143      30.180  11.390  18.900  1.00  0.00           H  \\nATOM   1967 2HD  LYS C 143      28.873  10.798  17.867  1.00  0.00           H  \\nATOM   1968 1HE  LYS C 143      29.430  12.456  16.165  1.00  0.00           H  \\nATOM   1969 2HE  LYS C 143      30.683  13.121  17.194  1.00  0.00           H  \\nATOM   1970 1HZ  LYS C 143      31.836  11.027  17.096  1.00  0.00           H  \\nATOM   1971 2HZ  LYS C 143      30.664  10.390  16.127  1.00  0.00           H  \\nATOM   1972 3HZ  LYS C 143      31.600  11.637  15.583  1.00  0.00           H  \\nATOM   1973  N   ILE C 144      26.969  16.140  18.545  1.00  0.00           N  \\nATOM   1974  CA  ILE C 144      27.376  17.376  19.207  1.00  0.00           C  \\nATOM   1975  C   ILE C 144      26.234  17.997  20.009  1.00  0.00           C  \\nATOM   1976  O   ILE C 144      26.404  18.333  21.180  1.00  0.00           O  \\nATOM   1977  CB  ILE C 144      27.923  18.383  18.181  1.00  0.00           C  \\nATOM   1978  CG1 ILE C 144      29.240  17.858  17.614  1.00  0.00           C  \\nATOM   1979  CG2 ILE C 144      28.139  19.733  18.828  1.00  0.00           C  \\nATOM   1980  CD1 ILE C 144      29.731  18.587  16.397  1.00  0.00           C  \\nATOM   1981  H   ILE C 144      27.050  16.161  17.689  1.00  0.00           H  \\nATOM   1982  HA  ILE C 144      28.096  17.175  19.825  1.00  0.00           H  \\nATOM   1983  HB  ILE C 144      27.214  18.474  17.352  1.00  0.00           H  \\nATOM   1984 1HG1 ILE C 144      30.006  17.930  18.389  1.00  0.00           H  \\nATOM   1985 2HG1 ILE C 144      29.104  16.814  17.348  1.00  0.00           H  \\nATOM   1986 1HG2 ILE C 144      28.523  20.406  18.081  1.00  0.00           H  \\nATOM   1987 2HG2 ILE C 144      27.196  20.110  19.214  1.00  0.00           H  \\nATOM   1988 3HG2 ILE C 144      28.855  19.645  19.643  1.00  0.00           H  \\nATOM   1989 1HD1 ILE C 144      29.887  19.636  16.638  1.00  0.00           H  \\nATOM   1990 2HD1 ILE C 144      30.673  18.142  16.063  1.00  0.00           H  \\nATOM   1991 3HD1 ILE C 144      28.987  18.501  15.609  1.00  0.00           H  \\nATOM   1992  N   LYS C 145      25.071  18.143  19.385  1.00  0.00           N  \\nATOM   1993  CA  LYS C 145      23.953  18.817  20.035  1.00  0.00           C  \\nATOM   1994  C   LYS C 145      23.334  17.972  21.148  1.00  0.00           C  \\nATOM   1995  O   LYS C 145      22.797  18.512  22.116  1.00  0.00           O  \\nATOM   1996  CB  LYS C 145      22.897  19.147  18.987  1.00  0.00           C  \\nATOM   1997  CG  LYS C 145      23.328  20.217  17.996  1.00  0.00           C  \\nATOM   1998  CD  LYS C 145      22.317  20.370  16.867  1.00  0.00           C  \\nATOM   1999  CE  LYS C 145      21.022  21.007  17.355  1.00  0.00           C  \\nATOM   2000  NZ  LYS C 145      20.066  21.248  16.238  1.00  0.00           N  \\nATOM   2001  H   LYS C 145      24.904  17.864  18.589  1.00  0.00           H  \\nATOM   2002  HA  LYS C 145      24.277  19.638  20.436  1.00  0.00           H  \\nATOM   2003 1HB  LYS C 145      22.666  18.240  18.418  1.00  0.00           H  \\nATOM   2004 2HB  LYS C 145      21.982  19.473  19.476  1.00  0.00           H  \\nATOM   2005 1HG  LYS C 145      23.447  21.169  18.511  1.00  0.00           H  \\nATOM   2006 2HG  LYS C 145      24.290  19.928  17.568  1.00  0.00           H  \\nATOM   2007 1HD  LYS C 145      22.745  20.991  16.079  1.00  0.00           H  \\nATOM   2008 2HD  LYS C 145      22.099  19.385  16.448  1.00  0.00           H  \\nATOM   2009 1HE  LYS C 145      20.554  20.348  18.084  1.00  0.00           H  \\nATOM   2010 2HE  LYS C 145      21.251  21.959  17.834  1.00  0.00           H  \\nATOM   2011 1HZ  LYS C 145      20.487  21.868  15.560  1.00  0.00           H  \\nATOM   2012 2HZ  LYS C 145      19.838  20.369  15.793  1.00  0.00           H  \\nATOM   2013 3HZ  LYS C 145      19.223  21.669  16.601  1.00  0.00           H  \\nATOM   2014  N   GLU C 146      23.417  16.652  21.020  1.00  0.00           N  \\nATOM   2015  CA  GLU C 146      22.892  15.766  22.051  1.00  0.00           C  \\nATOM   2016  C   GLU C 146      23.785  15.817  23.285  1.00  0.00           C  \\nATOM   2017  O   GLU C 146      23.301  15.781  24.415  1.00  0.00           O  \\nATOM   2018  CB  GLU C 146      22.780  14.312  21.545  1.00  0.00           C  \\nATOM   2019  CG  GLU C 146      21.986  13.311  22.444  1.00  0.00           C  \\nATOM   2020  CD  GLU C 146      22.757  12.619  23.587  1.00  0.00           C  \\nATOM   2021  OE1 GLU C 146      23.965  12.450  23.512  1.00  0.00           O  \\nATOM   2022  OE2 GLU C 146      22.095  12.224  24.522  1.00  0.00           O  \\nATOM   2023  H   GLU C 146      23.771  16.247  20.350  1.00  0.00           H  \\nATOM   2024  HA  GLU C 146      22.006  16.065  22.306  1.00  0.00           H  \\nATOM   2025 1HB  GLU C 146      22.256  14.313  20.595  1.00  0.00           H  \\nATOM   2026 2HB  GLU C 146      23.779  13.907  21.366  1.00  0.00           H  \\nATOM   2027 1HG  GLU C 146      21.150  13.854  22.881  1.00  0.00           H  \\nATOM   2028 2HG  GLU C 146      21.569  12.543  21.801  1.00  0.00           H  \\nATOM   2029  N   THR C 147      25.092  15.906  23.062  1.00  0.00           N  \\nATOM   2030  CA  THR C 147      26.049  15.988  24.156  1.00  0.00           C  \\nATOM   2031  C   THR C 147      25.979  17.349  24.847  1.00  0.00           C  \\nATOM   2032  O   THR C 147      26.129  17.442  26.064  1.00  0.00           O  \\nATOM   2033  CB  THR C 147      27.518  15.732  23.739  1.00  0.00           C  \\nATOM   2034  OG1 THR C 147      27.912  16.658  22.719  1.00  0.00           O  \\nATOM   2035  CG2 THR C 147      27.745  14.302  23.298  1.00  0.00           C  \\nATOM   2036  H   THR C 147      25.450  15.920  22.281  1.00  0.00           H  \\nATOM   2037  HA  THR C 147      25.836  15.307  24.814  1.00  0.00           H  \\nATOM   2038  HB  THR C 147      28.158  15.906  24.604  1.00  0.00           H  \\nATOM   2039  HG1 THR C 147      27.136  16.967  22.205  1.00  0.00           H  \\nATOM   2040 1HG2 THR C 147      28.798  14.165  23.043  1.00  0.00           H  \\nATOM   2041 2HG2 THR C 147      27.482  13.630  24.113  1.00  0.00           H  \\nATOM   2042 3HG2 THR C 147      27.135  14.072  22.441  1.00  0.00           H  \\nATOM   2043  N   LEU C 148      25.741  18.400  24.070  1.00  0.00           N  \\nATOM   2044  CA  LEU C 148      25.580  19.736  24.635  1.00  0.00           C  \\nATOM   2045  C   LEU C 148      24.374  19.792  25.569  1.00  0.00           C  \\nATOM   2046  O   LEU C 148      24.414  20.459  26.601  1.00  0.00           O  \\nATOM   2047  CB  LEU C 148      25.500  20.805  23.532  1.00  0.00           C  \\nATOM   2048  CG  LEU C 148      26.849  21.106  22.765  1.00  0.00           C  \\nATOM   2049  CD1 LEU C 148      26.568  22.016  21.569  1.00  0.00           C  \\nATOM   2050  CD2 LEU C 148      27.845  21.779  23.725  1.00  0.00           C  \\nATOM   2051  H   LEU C 148      25.667  18.368  23.214  1.00  0.00           H  \\nATOM   2052  HA  LEU C 148      26.370  19.958  25.152  1.00  0.00           H  \\nATOM   2053 1HB  LEU C 148      24.769  20.475  22.796  1.00  0.00           H  \\nATOM   2054 2HB  LEU C 148      25.148  21.736  23.973  1.00  0.00           H  \\nATOM   2055  HG  LEU C 148      27.283  20.182  22.401  1.00  0.00           H  \\nATOM   2056 1HD1 LEU C 148      27.501  22.219  21.040  1.00  0.00           H  \\nATOM   2057 2HD1 LEU C 148      25.870  21.521  20.896  1.00  0.00           H  \\nATOM   2058 3HD1 LEU C 148      26.138  22.952  21.914  1.00  0.00           H  \\nATOM   2059 1HD2 LEU C 148      28.774  21.989  23.197  1.00  0.00           H  \\nATOM   2060 2HD2 LEU C 148      27.425  22.711  24.095  1.00  0.00           H  \\nATOM   2061 3HD2 LEU C 148      28.046  21.112  24.563  1.00  0.00           H  \\nATOM   2062  N   LYS C 149      23.305  19.089  25.209  1.00  0.00           N  \\nATOM   2063  CA  LYS C 149      22.102  19.068  26.033  1.00  0.00           C  \\nATOM   2064  C   LYS C 149      22.358  18.315  27.333  1.00  0.00           C  \\nATOM   2065  O   LYS C 149      21.866  18.703  28.391  1.00  0.00           O  \\nATOM   2066  CB  LYS C 149      20.947  18.421  25.270  1.00  0.00           C  \\nATOM   2067  CG  LYS C 149      19.600  18.478  25.989  1.00  0.00           C  \\nATOM   2068  CD  LYS C 149      18.484  17.889  25.127  1.00  0.00           C  \\nATOM   2069  CE  LYS C 149      18.165  18.785  23.936  1.00  0.00           C  \\nATOM   2070  NZ  LYS C 149      16.999  18.279  23.157  1.00  0.00           N  \\nATOM   2071  H   LYS C 149      23.250  18.616  24.493  1.00  0.00           H  \\nATOM   2072  HA  LYS C 149      21.855  19.979  26.257  1.00  0.00           H  \\nATOM   2073 1HB  LYS C 149      20.842  18.904  24.297  1.00  0.00           H  \\nATOM   2074 2HB  LYS C 149      21.187  17.370  25.082  1.00  0.00           H  \\nATOM   2075 1HG  LYS C 149      19.668  17.910  26.922  1.00  0.00           H  \\nATOM   2076 2HG  LYS C 149      19.360  19.513  26.234  1.00  0.00           H  \\nATOM   2077 1HD  LYS C 149      18.795  16.909  24.759  1.00  0.00           H  \\nATOM   2078 2HD  LYS C 149      17.585  17.763  25.731  1.00  0.00           H  \\nATOM   2079 1HE  LYS C 149      17.942  19.789  24.294  1.00  0.00           H  \\nATOM   2080 2HE  LYS C 149      19.036  18.828  23.280  1.00  0.00           H  \\nATOM   2081 1HZ  LYS C 149      16.820  18.897  22.378  1.00  0.00           H  \\nATOM   2082 2HZ  LYS C 149      17.201  17.351  22.810  1.00  0.00           H  \\nATOM   2083 3HZ  LYS C 149      16.185  18.245  23.753  1.00  0.00           H  \\nATOM   2084  N   ARG C 150      23.123  17.231  27.249  1.00  0.00           N  \\nATOM   2085  CA  ARG C 150      23.506  16.481  28.438  1.00  0.00           C  \\nATOM   2086  C   ARG C 150      24.492  17.283  29.277  1.00  0.00           C  \\nATOM   2087  O   ARG C 150      24.459  17.240  30.507  1.00  0.00           O  \\nATOM   2088  CB  ARG C 150      24.156  15.166  28.052  1.00  0.00           C  \\nATOM   2089  CG  ARG C 150      23.241  14.106  27.505  1.00  0.00           C  \\nATOM   2090  CD  ARG C 150      24.021  13.032  26.841  1.00  0.00           C  \\nATOM   2091  NE  ARG C 150      24.995  12.412  27.707  1.00  0.00           N  \\nATOM   2092  CZ  ARG C 150      26.094  11.787  27.239  1.00  0.00           C  \\nATOM   2093  NH1 ARG C 150      26.292  11.705  25.933  1.00  0.00           N  \\nATOM   2094  NH2 ARG C 150      26.983  11.272  28.082  1.00  0.00           N  \\nATOM   2095  H   ARG C 150      23.432  16.910  26.514  1.00  0.00           H  \\nATOM   2096  HA  ARG C 150      22.717  16.311  28.976  1.00  0.00           H  \\nATOM   2097 1HB  ARG C 150      24.923  15.352  27.300  1.00  0.00           H  \\nATOM   2098 2HB  ARG C 150      24.649  14.741  28.918  1.00  0.00           H  \\nATOM   2099 1HG  ARG C 150      22.645  13.671  28.305  1.00  0.00           H  \\nATOM   2100 2HG  ARG C 150      22.581  14.549  26.752  1.00  0.00           H  \\nATOM   2101 1HD  ARG C 150      23.339  12.258  26.521  1.00  0.00           H  \\nATOM   2102 2HD  ARG C 150      24.538  13.440  25.970  1.00  0.00           H  \\nATOM   2103  HE  ARG C 150      24.860  12.470  28.705  1.00  0.00           H  \\nATOM   2104 1HH1 ARG C 150      25.590  12.078  25.259  1.00  0.00           H  \\nATOM   2105 2HH1 ARG C 150      27.158  11.275  25.583  1.00  0.00           H  \\nATOM   2106 1HH2 ARG C 150      26.837  11.340  29.078  1.00  0.00           H  \\nATOM   2107 2HH2 ARG C 150      27.841  10.812  27.722  1.00  0.00           H  \\nATOM   2108  N   LEU C 151      25.372  18.012  28.601  1.00  0.00           N  \\nATOM   2109  CA  LEU C 151      26.342  18.860  29.274  1.00  0.00           C  \\nATOM   2110  C   LEU C 151      25.602  19.964  30.026  1.00  0.00           C  \\nATOM   2111  O   LEU C 151      25.968  20.323  31.144  1.00  0.00           O  \\nATOM   2112  CB  LEU C 151      27.305  19.483  28.232  1.00  0.00           C  \\nATOM   2113  CG  LEU C 151      28.513  20.260  28.755  1.00  0.00           C  \\nATOM   2114  CD1 LEU C 151      29.370  19.309  29.563  1.00  0.00           C  \\nATOM   2115  CD2 LEU C 151      29.311  20.847  27.569  1.00  0.00           C  \\nATOM   2116  H   LEU C 151      25.427  18.031  27.743  1.00  0.00           H  \\nATOM   2117  HA  LEU C 151      26.841  18.332  29.917  1.00  0.00           H  \\nATOM   2118 1HB  LEU C 151      27.678  18.698  27.589  1.00  0.00           H  \\nATOM   2119 2HB  LEU C 151      26.733  20.170  27.620  1.00  0.00           H  \\nATOM   2120  HG  LEU C 151      28.173  21.063  29.415  1.00  0.00           H  \\nATOM   2121 1HD1 LEU C 151      30.232  19.836  29.972  1.00  0.00           H  \\nATOM   2122 2HD1 LEU C 151      28.776  18.900  30.359  1.00  0.00           H  \\nATOM   2123 3HD1 LEU C 151      29.712  18.501  28.926  1.00  0.00           H  \\nATOM   2124 1HD2 LEU C 151      30.180  21.392  27.942  1.00  0.00           H  \\nATOM   2125 2HD2 LEU C 151      29.649  20.037  26.916  1.00  0.00           H  \\nATOM   2126 3HD2 LEU C 151      28.674  21.524  27.001  1.00  0.00           H  \\nATOM   2127  N   GLU C 152      24.544  20.481  29.410  1.00  0.00           N  \\nATOM   2128  CA  GLU C 152      23.701  21.494  30.035  1.00  0.00           C  \\nATOM   2129  C   GLU C 152      23.030  20.945  31.288  1.00  0.00           C  \\nATOM   2130  O   GLU C 152      23.015  21.602  32.330  1.00  0.00           O  \\nATOM   2131  CB  GLU C 152      22.630  22.058  29.091  1.00  0.00           C  \\nATOM   2132  CG  GLU C 152      21.787  23.187  29.759  1.00  0.00           C  \\nATOM   2133  CD  GLU C 152      20.744  23.858  28.864  1.00  0.00           C  \\nATOM   2134  OE1 GLU C 152      20.618  23.502  27.718  1.00  0.00           O  \\nATOM   2135  OE2 GLU C 152      20.073  24.735  29.357  1.00  0.00           O  \\nATOM   2136  H   GLU C 152      24.291  20.259  28.618  1.00  0.00           H  \\nATOM   2137  HA  GLU C 152      24.252  22.250  30.294  1.00  0.00           H  \\nATOM   2138 1HB  GLU C 152      23.100  22.452  28.191  1.00  0.00           H  \\nATOM   2139 2HB  GLU C 152      21.952  21.263  28.789  1.00  0.00           H  \\nATOM   2140 1HG  GLU C 152      21.274  22.756  30.620  1.00  0.00           H  \\nATOM   2141 2HG  GLU C 152      22.472  23.947  30.132  1.00  0.00           H  \\nATOM   2142  N   ASP C 153      22.467  19.746  31.182  1.00  0.00           N  \\nATOM   2143  CA  ASP C 153      21.804  19.123  32.322  1.00  0.00           C  \\nATOM   2144  C   ASP C 153      22.816  18.853  33.429  1.00  0.00           C  \\nATOM   2145  O   ASP C 153      22.550  19.111  34.601  1.00  0.00           O  \\nATOM   2146  CB  ASP C 153      21.132  17.794  31.937  1.00  0.00           C  \\nATOM   2147  CG  ASP C 153      19.803  17.890  31.108  1.00  0.00           C  \\nATOM   2148  OD1 ASP C 153      19.195  18.954  31.022  1.00  0.00           O  \\nATOM   2149  OD2 ASP C 153      19.385  16.867  30.616  1.00  0.00           O  \\nATOM   2150  H   ASP C 153      22.454  19.274  30.463  1.00  0.00           H  \\nATOM   2151  HA  ASP C 153      21.132  19.731  32.669  1.00  0.00           H  \\nATOM   2152 1HB  ASP C 153      21.843  17.207  31.354  1.00  0.00           H  \\nATOM   2153 2HB  ASP C 153      20.927  17.229  32.844  1.00  0.00           H  \\nATOM   2154  N   SER C 154      23.985  18.350  33.047  1.00  0.00           N  \\nATOM   2155  CA  SER C 154      25.066  18.115  33.994  1.00  0.00           C  \\nATOM   2156  C   SER C 154      25.452  19.391  34.742  1.00  0.00           C  \\nATOM   2157  O   SER C 154      25.744  19.357  35.936  1.00  0.00           O  \\nATOM   2158  CB  SER C 154      26.257  17.521  33.296  1.00  0.00           C  \\nATOM   2159  OG  SER C 154      27.255  17.220  34.219  1.00  0.00           O  \\nATOM   2160  H   SER C 154      24.177  18.134  32.237  1.00  0.00           H  \\nATOM   2161  HA  SER C 154      24.776  17.460  34.648  1.00  0.00           H  \\nATOM   2162 1HB  SER C 154      25.952  16.622  32.772  1.00  0.00           H  \\nATOM   2163 2HB  SER C 154      26.633  18.220  32.556  1.00  0.00           H  \\nATOM   2164  HG  SER C 154      26.875  16.565  34.806  1.00  0.00           H  \\nATOM   2165  N   LEU C 155      25.456  20.516  34.037  1.00  0.00           N  \\nATOM   2166  CA  LEU C 155      25.805  21.787  34.656  1.00  0.00           C  \\nATOM   2167  C   LEU C 155      24.724  22.225  35.635  1.00  0.00           C  \\nATOM   2168  O   LEU C 155      25.029  22.751  36.704  1.00  0.00           O  \\nATOM   2169  CB  LEU C 155      26.092  22.861  33.597  1.00  0.00           C  \\nATOM   2170  CG  LEU C 155      27.435  22.690  32.813  1.00  0.00           C  \\nATOM   2171  CD1 LEU C 155      27.478  23.634  31.636  1.00  0.00           C  \\nATOM   2172  CD2 LEU C 155      28.601  23.010  33.749  1.00  0.00           C  \\nATOM   2173  H   LEU C 155      25.262  20.570  33.201  1.00  0.00           H  \\nATOM   2174  HA  LEU C 155      26.632  21.679  35.151  1.00  0.00           H  \\nATOM   2175 1HB  LEU C 155      25.282  22.845  32.868  1.00  0.00           H  \\nATOM   2176 2HB  LEU C 155      26.105  23.832  34.080  1.00  0.00           H  \\nATOM   2177  HG  LEU C 155      27.529  21.668  32.452  1.00  0.00           H  \\nATOM   2178 1HD1 LEU C 155      28.421  23.501  31.121  1.00  0.00           H  \\nATOM   2179 2HD1 LEU C 155      26.654  23.405  30.959  1.00  0.00           H  \\nATOM   2180 3HD1 LEU C 155      27.392  24.659  31.983  1.00  0.00           H  \\nATOM   2181 1HD2 LEU C 155      29.536  22.891  33.206  1.00  0.00           H  \\nATOM   2182 2HD2 LEU C 155      28.519  24.035  34.108  1.00  0.00           H  \\nATOM   2183 3HD2 LEU C 155      28.582  22.336  34.582  1.00  0.00           H  \\nATOM   2184  N   ARG C 156      23.464  22.005  35.274  1.00  0.00           N  \\nATOM   2185  CA  ARG C 156      22.361  22.335  36.166  1.00  0.00           C  \\nATOM   2186  C   ARG C 156      22.489  21.544  37.463  1.00  0.00           C  \\nATOM   2187  O   ARG C 156      22.236  22.068  38.546  1.00  0.00           O  \\nATOM   2188  CB  ARG C 156      21.025  22.069  35.485  1.00  0.00           C  \\nATOM   2189  CG  ARG C 156      19.795  22.546  36.253  1.00  0.00           C  \\nATOM   2190  CD  ARG C 156      18.547  22.499  35.409  1.00  0.00           C  \\nATOM   2191  NE  ARG C 156      18.165  21.145  35.026  1.00  0.00           N  \\nATOM   2192  CZ  ARG C 156      18.302  20.615  33.790  1.00  0.00           C  \\nATOM   2193  NH1 ARG C 156      18.838  21.290  32.804  1.00  0.00           N  \\nATOM   2194  NH2 ARG C 156      17.904  19.391  33.541  1.00  0.00           N  \\nATOM   2195  H   ARG C 156      23.223  21.668  34.520  1.00  0.00           H  \\nATOM   2196  HA  ARG C 156      22.401  23.280  36.384  1.00  0.00           H  \\nATOM   2197 1HB  ARG C 156      21.021  22.541  34.502  1.00  0.00           H  \\nATOM   2198 2HB  ARG C 156      20.914  20.997  35.325  1.00  0.00           H  \\nATOM   2199 1HG  ARG C 156      19.646  21.903  37.121  1.00  0.00           H  \\nATOM   2200 2HG  ARG C 156      19.950  23.574  36.585  1.00  0.00           H  \\nATOM   2201 1HD  ARG C 156      17.720  22.939  35.965  1.00  0.00           H  \\nATOM   2202 2HD  ARG C 156      18.713  23.073  34.497  1.00  0.00           H  \\nATOM   2203  HE  ARG C 156      17.748  20.561  35.736  1.00  0.00           H  \\nATOM   2204 1HH1 ARG C 156      19.168  22.233  32.938  1.00  0.00           H  \\nATOM   2205 2HH1 ARG C 156      18.946  20.804  31.897  1.00  0.00           H  \\nATOM   2206 1HH2 ARG C 156      17.490  18.822  34.259  1.00  0.00           H  \\nATOM   2207 2HH2 ARG C 156      18.038  19.038  32.582  1.00  0.00           H  \\nATOM   2208  N   GLU C 157      22.900  20.285  37.350  1.00  0.00           N  \\nATOM   2209  CA  GLU C 157      23.081  19.437  38.521  1.00  0.00           C  \\nATOM   2210  C   GLU C 157      24.189  19.979  39.419  1.00  0.00           C  \\nATOM   2211  O   GLU C 157      24.058  19.984  40.639  1.00  0.00           O  \\nATOM   2212  CB  GLU C 157      23.369  17.989  38.114  1.00  0.00           C  \\nATOM   2213  CG  GLU C 157      23.654  17.018  39.278  1.00  0.00           C  \\nATOM   2214  CD  GLU C 157      22.529  16.878  40.283  1.00  0.00           C  \\nATOM   2215  OE1 GLU C 157      21.440  17.347  40.044  1.00  0.00           O  \\nATOM   2216  OE2 GLU C 157      22.766  16.273  41.305  1.00  0.00           O  \\nATOM   2217  H   GLU C 157      23.081  19.898  36.603  1.00  0.00           H  \\nATOM   2218  HA  GLU C 157      22.258  19.425  39.034  1.00  0.00           H  \\nATOM   2219 1HB  GLU C 157      22.525  17.600  37.544  1.00  0.00           H  \\nATOM   2220 2HB  GLU C 157      24.234  17.969  37.458  1.00  0.00           H  \\nATOM   2221 1HG  GLU C 157      23.875  16.037  38.862  1.00  0.00           H  \\nATOM   2222 2HG  GLU C 157      24.549  17.366  39.797  1.00  0.00           H  \\nATOM   2223  N   LEU C 158      25.277  20.439  38.809  1.00  0.00           N  \\nATOM   2224  CA  LEU C 158      26.393  21.002  39.561  1.00  0.00           C  \\nATOM   2225  C   LEU C 158      25.968  22.231  40.356  1.00  0.00           C  \\nATOM   2226  O   LEU C 158      26.339  22.383  41.519  1.00  0.00           O  \\nATOM   2227  CB  LEU C 158      27.574  21.366  38.649  1.00  0.00           C  \\nATOM   2228  CG  LEU C 158      28.418  20.208  38.210  1.00  0.00           C  \\nATOM   2229  CD1 LEU C 158      29.398  20.598  37.099  1.00  0.00           C  \\nATOM   2230  CD2 LEU C 158      29.190  19.799  39.434  1.00  0.00           C  \\nATOM   2231  H   LEU C 158      25.394  20.436  37.957  1.00  0.00           H  \\nATOM   2232  HA  LEU C 158      26.719  20.338  40.189  1.00  0.00           H  \\nATOM   2233 1HB  LEU C 158      27.186  21.850  37.758  1.00  0.00           H  \\nATOM   2234 2HB  LEU C 158      28.201  22.069  39.172  1.00  0.00           H  \\nATOM   2235  HG  LEU C 158      27.787  19.387  37.852  1.00  0.00           H  \\nATOM   2236 1HD1 LEU C 158      30.011  19.738  36.840  1.00  0.00           H  \\nATOM   2237 2HD1 LEU C 158      28.836  20.904  36.236  1.00  0.00           H  \\nATOM   2238 3HD1 LEU C 158      30.040  21.407  37.422  1.00  0.00           H  \\nATOM   2239 1HD2 LEU C 158      28.497  19.523  40.224  1.00  0.00           H  \\nATOM   2240 2HD2 LEU C 158      29.805  18.987  39.200  1.00  0.00           H  \\nATOM   2241 3HD2 LEU C 158      29.806  20.623  39.773  1.00  0.00           H  \\nATOM   2242  N   ARG C 159      25.193  23.108  39.727  1.00  0.00           N  \\nATOM   2243  CA  ARG C 159      24.737  24.323  40.390  1.00  0.00           C  \\nATOM   2244  C   ARG C 159      23.801  24.005  41.552  1.00  0.00           C  \\nATOM   2245  O   ARG C 159      23.774  24.727  42.546  1.00  0.00           O  \\nATOM   2246  CB  ARG C 159      24.042  25.246  39.402  1.00  0.00           C  \\nATOM   2247  CG  ARG C 159      24.977  25.891  38.370  1.00  0.00           C  \\nATOM   2248  CD  ARG C 159      24.322  27.020  37.642  1.00  0.00           C  \\nATOM   2249  NE  ARG C 159      23.170  26.613  36.830  1.00  0.00           N  \\nATOM   2250  CZ  ARG C 159      23.221  26.205  35.542  1.00  0.00           C  \\nATOM   2251  NH1 ARG C 159      24.368  26.133  34.908  1.00  0.00           N  \\nATOM   2252  NH2 ARG C 159      22.102  25.891  34.907  1.00  0.00           N  \\nATOM   2253  H   ARG C 159      24.918  23.023  38.917  1.00  0.00           H  \\nATOM   2254  HA  ARG C 159      25.506  24.794  40.747  1.00  0.00           H  \\nATOM   2255 1HB  ARG C 159      23.292  24.673  38.851  1.00  0.00           H  \\nATOM   2256 2HB  ARG C 159      23.525  26.038  39.938  1.00  0.00           H  \\nATOM   2257 1HG  ARG C 159      25.863  26.275  38.876  1.00  0.00           H  \\nATOM   2258 2HG  ARG C 159      25.277  25.140  37.639  1.00  0.00           H  \\nATOM   2259 1HD  ARG C 159      23.979  27.759  38.365  1.00  0.00           H  \\nATOM   2260 2HD  ARG C 159      25.052  27.482  36.978  1.00  0.00           H  \\nATOM   2261  HE  ARG C 159      22.260  26.651  37.269  1.00  0.00           H  \\nATOM   2262 1HH1 ARG C 159      25.224  26.365  35.391  1.00  0.00           H  \\nATOM   2263 2HH1 ARG C 159      24.383  25.850  33.938  1.00  0.00           H  \\nATOM   2264 1HH2 ARG C 159      21.216  25.959  35.388  1.00  0.00           H  \\nATOM   2265 2HH2 ARG C 159      22.136  25.594  33.940  1.00  0.00           H  \\nATOM   2266  N   ARG C 160      23.040  22.921  41.431  1.00  0.00           N  \\nATOM   2267  CA  ARG C 160      22.124  22.515  42.493  1.00  0.00           C  \\nATOM   2268  C   ARG C 160      22.911  22.026  43.704  1.00  0.00           C  \\nATOM   2269  O   ARG C 160      22.635  22.420  44.836  1.00  0.00           O  \\nATOM   2270  CB  ARG C 160      21.206  21.390  42.026  1.00  0.00           C  \\nATOM   2271  CG  ARG C 160      20.144  20.961  43.044  1.00  0.00           C  \\nATOM   2272  CD  ARG C 160      19.371  19.754  42.591  1.00  0.00           C  \\nATOM   2273  NE  ARG C 160      20.184  18.538  42.616  1.00  0.00           N  \\nATOM   2274  CZ  ARG C 160      20.428  17.814  43.738  1.00  0.00           C  \\nATOM   2275  NH1 ARG C 160      19.913  18.187  44.900  1.00  0.00           N  \\nATOM   2276  NH2 ARG C 160      21.180  16.735  43.674  1.00  0.00           N  \\nATOM   2277  H   ARG C 160      23.035  22.404  40.744  1.00  0.00           H  \\nATOM   2278  HA  ARG C 160      21.591  23.278  42.765  1.00  0.00           H  \\nATOM   2279 1HB  ARG C 160      20.701  21.691  41.108  1.00  0.00           H  \\nATOM   2280 2HB  ARG C 160      21.805  20.508  41.787  1.00  0.00           H  \\nATOM   2281 1HG  ARG C 160      20.637  20.714  43.986  1.00  0.00           H  \\nATOM   2282 2HG  ARG C 160      19.443  21.781  43.207  1.00  0.00           H  \\nATOM   2283 1HD  ARG C 160      18.510  19.603  43.240  1.00  0.00           H  \\nATOM   2284 2HD  ARG C 160      19.033  19.910  41.567  1.00  0.00           H  \\nATOM   2285  HE  ARG C 160      20.598  18.199  41.726  1.00  0.00           H  \\nATOM   2286 1HH1 ARG C 160      19.338  19.015  44.956  1.00  0.00           H  \\nATOM   2287 2HH1 ARG C 160      20.099  17.644  45.735  1.00  0.00           H  \\nATOM   2288 1HH2 ARG C 160      21.602  16.460  42.769  1.00  0.00           H  \\nATOM   2289 2HH2 ARG C 160      21.359  16.199  44.501  1.00  0.00           H  \\nATOM   2290  N   ILE C 161      23.895  21.168  43.457  1.00  0.00           N  \\nATOM   2291  CA  ILE C 161      24.725  20.619  44.525  1.00  0.00           C  \\nATOM   2292  C   ILE C 161      25.538  21.719  45.193  1.00  0.00           C  \\nATOM   2293  O   ILE C 161      25.772  21.684  46.403  1.00  0.00           O  \\nATOM   2294  CB  ILE C 161      25.709  19.551  44.020  1.00  0.00           C  \\nATOM   2295  CG1 ILE C 161      24.977  18.311  43.545  1.00  0.00           C  \\nATOM   2296  CG2 ILE C 161      26.643  19.169  45.160  1.00  0.00           C  \\nATOM   2297  CD1 ILE C 161      25.884  17.367  42.805  1.00  0.00           C  \\nATOM   2298  H   ILE C 161      24.104  20.885  42.672  1.00  0.00           H  \\nATOM   2299  HA  ILE C 161      24.152  20.217  45.197  1.00  0.00           H  \\nATOM   2300  HB  ILE C 161      26.284  19.946  43.178  1.00  0.00           H  \\nATOM   2301 1HG1 ILE C 161      24.550  17.791  44.401  1.00  0.00           H  \\nATOM   2302 2HG1 ILE C 161      24.162  18.605  42.872  1.00  0.00           H  \\nATOM   2303 1HG2 ILE C 161      27.338  18.415  44.822  1.00  0.00           H  \\nATOM   2304 2HG2 ILE C 161      27.190  20.048  45.500  1.00  0.00           H  \\nATOM   2305 3HG2 ILE C 161      26.052  18.771  45.976  1.00  0.00           H  \\nATOM   2306 1HD1 ILE C 161      25.313  16.497  42.474  1.00  0.00           H  \\nATOM   2307 2HD1 ILE C 161      26.290  17.887  41.938  1.00  0.00           H  \\nATOM   2308 3HD1 ILE C 161      26.695  17.046  43.452  1.00  0.00           H  \\nATOM   2309  N   LEU C 162      25.970  22.694  44.402  1.00  0.00           N  \\nATOM   2310  CA  LEU C 162      26.759  23.794  44.932  1.00  0.00           C  \\nATOM   2311  C   LEU C 162      25.890  24.668  45.828  1.00  0.00           C  \\nATOM   2312  O   LEU C 162      26.356  25.182  46.844  1.00  0.00           O  \\nATOM   2313  CB  LEU C 162      27.382  24.608  43.801  1.00  0.00           C  \\nATOM   2314  CG  LEU C 162      28.306  25.752  44.229  1.00  0.00           C  \\nATOM   2315  CD1 LEU C 162      29.467  25.215  45.096  1.00  0.00           C  \\nATOM   2316  CD2 LEU C 162      28.838  26.420  42.988  1.00  0.00           C  \\nATOM   2317  H   LEU C 162      25.819  22.741  43.557  1.00  0.00           H  \\nATOM   2318  HA  LEU C 162      27.483  23.437  45.469  1.00  0.00           H  \\nATOM   2319 1HB  LEU C 162      27.948  23.934  43.159  1.00  0.00           H  \\nATOM   2320 2HB  LEU C 162      26.572  25.037  43.209  1.00  0.00           H  \\nATOM   2321  HG  LEU C 162      27.741  26.471  44.817  1.00  0.00           H  \\nATOM   2322 1HD1 LEU C 162      30.123  26.036  45.392  1.00  0.00           H  \\nATOM   2323 2HD1 LEU C 162      29.065  24.740  45.985  1.00  0.00           H  \\nATOM   2324 3HD1 LEU C 162      30.031  24.497  44.531  1.00  0.00           H  \\nATOM   2325 1HD2 LEU C 162      29.472  27.239  43.284  1.00  0.00           H  \\nATOM   2326 2HD2 LEU C 162      29.409  25.700  42.398  1.00  0.00           H  \\nATOM   2327 3HD2 LEU C 162      28.008  26.799  42.393  1.00  0.00           H  \\nATOM   2328  N   GLU C 163      24.624  24.830  45.450  1.00  0.00           N  \\nATOM   2329  CA  GLU C 163      23.677  25.573  46.275  1.00  0.00           C  \\nATOM   2330  C   GLU C 163      23.535  24.886  47.629  1.00  0.00           C  \\nATOM   2331  O   GLU C 163      23.550  25.537  48.670  1.00  0.00           O  \\nATOM   2332  CB  GLU C 163      22.335  25.706  45.518  1.00  0.00           C  \\nATOM   2333  CG  GLU C 163      21.268  26.689  46.078  1.00  0.00           C  \\nATOM   2334  CD  GLU C 163      20.288  26.107  47.082  1.00  0.00           C  \\nATOM   2335  OE1 GLU C 163      20.022  24.931  47.032  1.00  0.00           O  \\nATOM   2336  OE2 GLU C 163      19.766  26.865  47.862  1.00  0.00           O  \\nATOM   2337  H   GLU C 163      24.289  24.519  44.722  1.00  0.00           H  \\nATOM   2338  HA  GLU C 163      24.018  26.469  46.424  1.00  0.00           H  \\nATOM   2339 1HB  GLU C 163      22.538  26.002  44.491  1.00  0.00           H  \\nATOM   2340 2HB  GLU C 163      21.860  24.731  45.473  1.00  0.00           H  \\nATOM   2341 1HG  GLU C 163      21.791  27.518  46.555  1.00  0.00           H  \\nATOM   2342 2HG  GLU C 163      20.708  27.094  45.237  1.00  0.00           H  \\nATOM   2343  N   GLU C 164      23.400  23.564  47.602  1.00  0.00           N  \\nATOM   2344  CA  GLU C 164      23.300  22.773  48.823  1.00  0.00           C  \\nATOM   2345  C   GLU C 164      24.591  22.854  49.630  1.00  0.00           C  \\nATOM   2346  O   GLU C 164      24.569  22.850  50.860  1.00  0.00           O  \\nATOM   2347  CB  GLU C 164      22.987  21.301  48.480  1.00  0.00           C  \\nATOM   2348  CG  GLU C 164      22.746  20.382  49.689  1.00  0.00           C  \\nATOM   2349  CD  GLU C 164      22.296  18.959  49.315  1.00  0.00           C  \\nATOM   2350  OE1 GLU C 164      22.184  18.663  48.146  1.00  0.00           O  \\nATOM   2351  OE2 GLU C 164      22.048  18.185  50.210  1.00  0.00           O  \\nATOM   2352  H   GLU C 164      23.364  23.097  46.881  1.00  0.00           H  \\nATOM   2353  HA  GLU C 164      22.579  23.121  49.370  1.00  0.00           H  \\nATOM   2354 1HB  GLU C 164      22.110  21.260  47.834  1.00  0.00           H  \\nATOM   2355 2HB  GLU C 164      23.820  20.881  47.918  1.00  0.00           H  \\nATOM   2356 1HG  GLU C 164      23.679  20.311  50.236  1.00  0.00           H  \\nATOM   2357 2HG  GLU C 164      22.007  20.839  50.345  1.00  0.00           H  \\nATOM   2358  N   LEU C 165      25.717  22.926  48.931  1.00  0.00           N  \\nATOM   2359  CA  LEU C 165      27.015  22.974  49.587  1.00  0.00           C  \\nATOM   2360  C   LEU C 165      27.197  24.301  50.312  1.00  0.00           C  \\nATOM   2361  O   LEU C 165      27.856  24.363  51.347  1.00  0.00           O  \\nATOM   2362  CB  LEU C 165      28.118  22.693  48.560  1.00  0.00           C  \\nATOM   2363  CG  LEU C 165      29.530  22.558  49.091  1.00  0.00           C  \\nATOM   2364  CD1 LEU C 165      29.597  21.467  50.178  1.00  0.00           C  \\nATOM   2365  CD2 LEU C 165      30.460  22.174  47.903  1.00  0.00           C  \\nATOM   2366  H   LEU C 165      25.756  22.948  48.072  1.00  0.00           H  \\nATOM   2367  HA  LEU C 165      27.065  22.262  50.243  1.00  0.00           H  \\nATOM   2368 1HB  LEU C 165      27.867  21.783  48.017  1.00  0.00           H  \\nATOM   2369 2HB  LEU C 165      28.122  23.514  47.855  1.00  0.00           H  \\nATOM   2370  HG  LEU C 165      29.833  23.502  49.530  1.00  0.00           H  \\nATOM   2371 1HD1 LEU C 165      30.613  21.401  50.545  1.00  0.00           H  \\nATOM   2372 2HD1 LEU C 165      28.933  21.728  51.004  1.00  0.00           H  \\nATOM   2373 3HD1 LEU C 165      29.302  20.514  49.767  1.00  0.00           H  \\nATOM   2374 1HD2 LEU C 165      31.481  22.068  48.249  1.00  0.00           H  \\nATOM   2375 2HD2 LEU C 165      30.130  21.224  47.476  1.00  0.00           H  \\nATOM   2376 3HD2 LEU C 165      30.417  22.948  47.137  1.00  0.00           H  \\nATOM   2377  N   LYS C 166      26.598  25.358  49.773  1.00  0.00           N  \\nATOM   2378  CA  LYS C 166      26.702  26.682  50.374  1.00  0.00           C  \\nATOM   2379  C   LYS C 166      25.693  26.866  51.500  1.00  0.00           C  \\nATOM   2380  O   LYS C 166      25.907  27.668  52.408  1.00  0.00           O  \\nATOM   2381  CB  LYS C 166      26.514  27.741  49.294  1.00  0.00           C  \\nATOM   2382  CG  LYS C 166      27.687  27.864  48.312  1.00  0.00           C  \\nATOM   2383  CD  LYS C 166      27.338  28.806  47.172  1.00  0.00           C  \\nATOM   2384  CE  LYS C 166      28.536  29.101  46.298  1.00  0.00           C  \\nATOM   2385  NZ  LYS C 166      29.420  30.096  46.938  1.00  0.00           N  \\nATOM   2386  H   LYS C 166      26.123  25.333  49.056  1.00  0.00           H  \\nATOM   2387  HA  LYS C 166      27.591  26.788  50.749  1.00  0.00           H  \\nATOM   2388 1HB  LYS C 166      25.619  27.496  48.712  1.00  0.00           H  \\nATOM   2389 2HB  LYS C 166      26.350  28.711  49.757  1.00  0.00           H  \\nATOM   2390 1HG  LYS C 166      28.566  28.240  48.835  1.00  0.00           H  \\nATOM   2391 2HG  LYS C 166      27.925  26.881  47.902  1.00  0.00           H  \\nATOM   2392 1HD  LYS C 166      26.561  28.341  46.559  1.00  0.00           H  \\nATOM   2393 2HD  LYS C 166      26.950  29.741  47.573  1.00  0.00           H  \\nATOM   2394 1HE  LYS C 166      29.102  28.186  46.141  1.00  0.00           H  \\nATOM   2395 2HE  LYS C 166      28.209  29.484  45.331  1.00  0.00           H  \\nATOM   2396 1HZ  LYS C 166      28.911  30.955  47.068  1.00  0.00           H  \\nATOM   2397 2HZ  LYS C 166      29.746  29.768  47.826  1.00  0.00           H  \\nATOM   2398 3HZ  LYS C 166      30.224  30.277  46.325  1.00  0.00           H  \\nATOM   2399  N   GLU C 167      24.592  26.124  51.436  1.00  0.00           N  \\nATOM   2400  CA  GLU C 167      23.594  26.149  52.497  1.00  0.00           C  \\nATOM   2401  C   GLU C 167      24.153  25.481  53.748  1.00  0.00           C  \\nATOM   2402  O   GLU C 167      23.964  25.965  54.864  1.00  0.00           O  \\nATOM   2403  CB  GLU C 167      22.275  25.501  52.076  1.00  0.00           C  \\nATOM   2404  CG  GLU C 167      21.429  26.354  51.126  1.00  0.00           C  \\nATOM   2405  CD  GLU C 167      20.896  27.623  51.800  1.00  0.00           C  \\nATOM   2406  OE1 GLU C 167      21.089  28.698  51.275  1.00  0.00           O  \\nATOM   2407  OE2 GLU C 167      20.294  27.502  52.847  1.00  0.00           O  \\nATOM   2408  H   GLU C 167      24.399  25.596  50.785  1.00  0.00           H  \\nATOM   2409  HA  GLU C 167      23.379  27.070  52.714  1.00  0.00           H  \\nATOM   2410 1HB  GLU C 167      22.484  24.555  51.574  1.00  0.00           H  \\nATOM   2411 2HB  GLU C 167      21.680  25.279  52.959  1.00  0.00           H  \\nATOM   2412 1HG  GLU C 167      22.036  26.651  50.278  1.00  0.00           H  \\nATOM   2413 2HG  GLU C 167      20.597  25.761  50.747  1.00  0.00           H  \\nATOM   2414  N   MET C 168      24.854  24.371  53.547  1.00  0.00           N  \\nATOM   2415  CA  MET C 168      25.421  23.608  54.650  1.00  0.00           C  \\nATOM   2416  C   MET C 168      26.681  24.258  55.203  1.00  0.00           C  \\nATOM   2417  O   MET C 168      27.083  23.993  56.336  1.00  0.00           O  \\nATOM   2418  CB  MET C 168      25.773  22.267  54.096  1.00  0.00           C  \\nATOM   2419  CG  MET C 168      24.617  21.459  53.719  1.00  0.00           C  \\nATOM   2420  SD  MET C 168      25.149  20.117  52.804  1.00  0.00           S  \\nATOM   2421  CE  MET C 168      26.159  19.396  53.983  1.00  0.00           C  \\nATOM   2422  H   MET C 168      25.016  24.035  52.772  1.00  0.00           H  \\nATOM   2423  HA  MET C 168      24.763  23.551  55.360  1.00  0.00           H  \\nATOM   2424 1HB  MET C 168      26.396  22.396  53.205  1.00  0.00           H  \\nATOM   2425 2HB  MET C 168      26.353  21.712  54.829  1.00  0.00           H  \\nATOM   2426 1HG  MET C 168      24.100  21.100  54.606  1.00  0.00           H  \\nATOM   2427 2HG  MET C 168      23.921  22.044  53.119  1.00  0.00           H  \\nATOM   2428 1HE  MET C 168      26.593  18.507  53.560  1.00  0.00           H  \\nATOM   2429 2HE  MET C 168      26.934  20.089  54.255  1.00  0.00           H  \\nATOM   2430 3HE  MET C 168      25.577  19.154  54.862  1.00  0.00           H  \\nATOM   2431  N   LEU C 169      27.308  25.100  54.393  1.00  0.00           N  \\nATOM   2432  CA  LEU C 169      28.496  25.820  54.820  1.00  0.00           C  \\nATOM   2433  C   LEU C 169      28.123  26.874  55.856  1.00  0.00           C  \\nATOM   2434  O   LEU C 169      28.907  27.181  56.754  1.00  0.00           O  \\nATOM   2435  CB  LEU C 169      29.198  26.463  53.612  1.00  0.00           C  \\nATOM   2436  CG  LEU C 169      30.626  26.992  53.847  1.00  0.00           C  \\nATOM   2437  CD1 LEU C 169      31.524  25.808  54.288  1.00  0.00           C  \\nATOM   2438  CD2 LEU C 169      31.152  27.628  52.560  1.00  0.00           C  \\nATOM   2439  H   LEU C 169      27.064  25.273  53.587  1.00  0.00           H  \\nATOM   2440  HA  LEU C 169      29.118  25.198  55.229  1.00  0.00           H  \\nATOM   2441 1HB  LEU C 169      29.257  25.729  52.819  1.00  0.00           H  \\nATOM   2442 2HB  LEU C 169      28.583  27.294  53.261  1.00  0.00           H  \\nATOM   2443  HG  LEU C 169      30.618  27.731  54.646  1.00  0.00           H  \\nATOM   2444 1HD1 LEU C 169      31.118  25.374  55.199  1.00  0.00           H  \\nATOM   2445 2HD1 LEU C 169      31.537  25.050  53.504  1.00  0.00           H  \\nATOM   2446 3HD1 LEU C 169      32.538  26.152  54.470  1.00  0.00           H  \\nATOM   2447 1HD2 LEU C 169      32.163  27.997  52.714  1.00  0.00           H  \\nATOM   2448 2HD2 LEU C 169      31.157  26.888  51.775  1.00  0.00           H  \\nATOM   2449 3HD2 LEU C 169      30.501  28.453  52.274  1.00  0.00           H  \\nATOM   2450  N   GLU C 170      26.920  27.424  55.726  1.00  0.00           N  \\nATOM   2451  CA  GLU C 170      26.431  28.425  56.666  1.00  0.00           C  \\nATOM   2452  C   GLU C 170      26.093  27.779  58.009  1.00  0.00           C  \\nATOM   2453  O   GLU C 170      26.188  28.422  59.054  1.00  0.00           O  \\nATOM   2454  CB  GLU C 170      25.178  29.072  56.059  1.00  0.00           C  \\nATOM   2455  CG  GLU C 170      24.466  30.130  56.887  1.00  0.00           C  \\nATOM   2456  CD  GLU C 170      25.205  31.402  57.127  1.00  0.00           C  \\nATOM   2457  OE1 GLU C 170      24.842  32.070  58.082  1.00  0.00           O  \\nATOM   2458  OE2 GLU C 170      26.081  31.734  56.375  1.00  0.00           O  \\nATOM   2459  H   GLU C 170      26.365  27.233  55.098  1.00  0.00           H  \\nATOM   2460  HA  GLU C 170      27.125  29.085  56.816  1.00  0.00           H  \\nATOM   2461 1HB  GLU C 170      25.441  29.523  55.101  1.00  0.00           H  \\nATOM   2462 2HB  GLU C 170      24.449  28.293  55.849  1.00  0.00           H  \\nATOM   2463 1HG  GLU C 170      23.522  30.366  56.398  1.00  0.00           H  \\nATOM   2464 2HG  GLU C 170      24.234  29.672  57.844  1.00  0.00           H  \\nATOM   2465  N   ARG C 171      25.694  26.509  57.975  1.00  0.00           N  \\nATOM   2466  CA  ARG C 171      25.462  25.748  59.200  1.00  0.00           C  \\nATOM   2467  C   ARG C 171      26.769  25.590  59.961  1.00  0.00           C  \\nATOM   2468  O   ARG C 171      26.818  25.729  61.183  1.00  0.00           O  \\nATOM   2469  CB  ARG C 171      24.893  24.367  58.904  1.00  0.00           C  \\nATOM   2470  CG  ARG C 171      23.465  24.352  58.386  1.00  0.00           C  \\nATOM   2471  CD  ARG C 171      22.990  22.958  58.143  1.00  0.00           C  \\nATOM   2472  NE  ARG C 171      21.649  22.925  57.568  1.00  0.00           N  \\nATOM   2473  CZ  ARG C 171      20.502  22.979  58.278  1.00  0.00           C  \\nATOM   2474  NH1 ARG C 171      20.540  23.063  59.590  1.00  0.00           N  \\nATOM   2475  NH2 ARG C 171      19.337  22.945  57.653  1.00  0.00           N  \\nATOM   2476  H   ARG C 171      25.550  26.064  57.253  1.00  0.00           H  \\nATOM   2477  HA  ARG C 171      24.835  26.228  59.763  1.00  0.00           H  \\nATOM   2478 1HB  ARG C 171      25.511  23.859  58.182  1.00  0.00           H  \\nATOM   2479 2HB  ARG C 171      24.916  23.771  59.817  1.00  0.00           H  \\nATOM   2480 1HG  ARG C 171      22.806  24.826  59.110  1.00  0.00           H  \\nATOM   2481 2HG  ARG C 171      23.425  24.900  57.439  1.00  0.00           H  \\nATOM   2482 1HD  ARG C 171      23.671  22.462  57.449  1.00  0.00           H  \\nATOM   2483 2HD  ARG C 171      22.972  22.410  59.084  1.00  0.00           H  \\nATOM   2484  HE  ARG C 171      21.573  22.864  56.561  1.00  0.00           H  \\nATOM   2485 1HH1 ARG C 171      21.429  23.088  60.069  1.00  0.00           H  \\nATOM   2486 2HH1 ARG C 171      19.681  23.102  60.118  1.00  0.00           H  \\nATOM   2487 1HH2 ARG C 171      19.306  22.880  56.645  1.00  0.00           H  \\nATOM   2488 2HH2 ARG C 171      18.479  22.985  58.183  1.00  0.00           H  \\nATOM   2489  N   LEU C 172      27.823  25.284  59.213  1.00  0.00           N  \\nATOM   2490  CA  LEU C 172      29.153  25.088  59.768  1.00  0.00           C  \\nATOM   2491  C   LEU C 172      29.622  26.320  60.537  1.00  0.00           C  \\nATOM   2492  O   LEU C 172      30.294  26.208  61.562  1.00  0.00           O  \\nATOM   2493  CB  LEU C 172      30.124  24.761  58.613  1.00  0.00           C  \\nATOM   2494  CG  LEU C 172      31.525  24.300  58.975  1.00  0.00           C  \\nATOM   2495  CD1 LEU C 172      31.442  22.981  59.724  1.00  0.00           C  \\nATOM   2496  CD2 LEU C 172      32.347  24.145  57.704  1.00  0.00           C  \\nATOM   2497  H   LEU C 172      27.790  25.183  58.359  1.00  0.00           H  \\nATOM   2498  HA  LEU C 172      29.136  24.337  60.381  1.00  0.00           H  \\nATOM   2499 1HB  LEU C 172      29.665  24.001  57.978  1.00  0.00           H  \\nATOM   2500 2HB  LEU C 172      30.252  25.655  58.022  1.00  0.00           H  \\nATOM   2501  HG  LEU C 172      31.988  25.037  59.633  1.00  0.00           H  \\nATOM   2502 1HD1 LEU C 172      32.440  22.667  59.998  1.00  0.00           H  \\nATOM   2503 2HD1 LEU C 172      30.849  23.098  60.605  1.00  0.00           H  \\nATOM   2504 3HD1 LEU C 172      30.983  22.226  59.085  1.00  0.00           H  \\nATOM   2505 1HD2 LEU C 172      33.342  23.824  57.960  1.00  0.00           H  \\nATOM   2506 2HD2 LEU C 172      31.880  23.407  57.051  1.00  0.00           H  \\nATOM   2507 3HD2 LEU C 172      32.399  25.103  57.191  1.00  0.00           H  \\nATOM   2508  N   GLU C 173      29.266  27.495  60.027  1.00  0.00           N  \\nATOM   2509  CA  GLU C 173      29.635  28.761  60.656  1.00  0.00           C  \\nATOM   2510  C   GLU C 173      28.926  28.968  61.992  1.00  0.00           C  \\nATOM   2511  O   GLU C 173      29.563  29.288  62.995  1.00  0.00           O  \\nATOM   2512  CB  GLU C 173      29.409  29.940  59.703  1.00  0.00           C  \\nATOM   2513  CG  GLU C 173      29.839  31.297  60.292  1.00  0.00           C  \\nATOM   2514  CD  GLU C 173      29.896  32.431  59.277  1.00  0.00           C  \\nATOM   2515  OE1 GLU C 173      29.873  33.567  59.686  1.00  0.00           O  \\nATOM   2516  OE2 GLU C 173      30.031  32.154  58.102  1.00  0.00           O  \\nATOM   2517  H   GLU C 173      28.804  27.588  59.308  1.00  0.00           H  \\nATOM   2518  HA  GLU C 173      30.590  28.763  60.821  1.00  0.00           H  \\nATOM   2519 1HB  GLU C 173      29.959  29.772  58.777  1.00  0.00           H  \\nATOM   2520 2HB  GLU C 173      28.349  30.002  59.446  1.00  0.00           H  \\nATOM   2521 1HG  GLU C 173      29.131  31.568  61.076  1.00  0.00           H  \\nATOM   2522 2HG  GLU C 173      30.818  31.181  60.753  1.00  0.00           H  \\nATOM   2523  N   LYS C 174      27.606  28.804  61.989  1.00  0.00           N  \\nATOM   2524  CA  LYS C 174      26.794  28.975  63.193  1.00  0.00           C  \\nATOM   2525  C   LYS C 174      27.275  28.090  64.340  1.00  0.00           C  \\nATOM   2526  O   LYS C 174      27.157  28.451  65.511  1.00  0.00           O  \\nATOM   2527  CB  LYS C 174      25.325  28.720  62.880  1.00  0.00           C  \\nATOM   2528  CG  LYS C 174      24.703  29.859  62.090  1.00  0.00           C  \\nATOM   2529  CD  LYS C 174      23.224  29.671  61.840  1.00  0.00           C  \\nATOM   2530  CE  LYS C 174      22.642  30.948  61.256  1.00  0.00           C  \\nATOM   2531  NZ  LYS C 174      23.312  31.295  59.995  1.00  0.00           N  \\nATOM   2532  H   LYS C 174      27.149  28.590  61.293  1.00  0.00           H  \\nATOM   2533  HA  LYS C 174      26.852  29.899  63.483  1.00  0.00           H  \\nATOM   2534 1HB  LYS C 174      25.235  27.800  62.292  1.00  0.00           H  \\nATOM   2535 2HB  LYS C 174      24.768  28.580  63.804  1.00  0.00           H  \\nATOM   2536 1HG  LYS C 174      24.857  30.796  62.622  1.00  0.00           H  \\nATOM   2537 2HG  LYS C 174      25.210  29.929  61.119  1.00  0.00           H  \\nATOM   2538 1HD  LYS C 174      23.078  28.851  61.133  1.00  0.00           H  \\nATOM   2539 2HD  LYS C 174      22.715  29.429  62.772  1.00  0.00           H  \\nATOM   2540 1HE  LYS C 174      21.578  30.815  61.068  1.00  0.00           H  \\nATOM   2541 2HE  LYS C 174      22.779  31.764  61.965  1.00  0.00           H  \\nATOM   2542 1HZ  LYS C 174      23.181  30.539  59.342  1.00  0.00           H  \\nATOM   2543 2HZ  LYS C 174      22.949  32.143  59.591  1.00  0.00           H  \\nATOM   2544 3HZ  LYS C 174      24.309  31.418  60.175  1.00  0.00           H  \\nATOM   2545  N   ASN C 175      27.816  26.928  63.990  1.00  0.00           N  \\nATOM   2546  CA  ASN C 175      28.257  25.949  64.975  1.00  0.00           C  \\nATOM   2547  C   ASN C 175      29.037  24.821  64.309  1.00  0.00           C  \\nATOM   2548  O   ASN C 175      28.447  23.835  63.869  1.00  0.00           O  \\nATOM   2549  CB  ASN C 175      27.102  25.364  65.779  1.00  0.00           C  \\nATOM   2550  CG  ASN C 175      26.110  24.586  64.964  1.00  0.00           C  \\nATOM   2551  OD1 ASN C 175      25.413  25.125  64.091  1.00  0.00           O  \\nATOM   2552  ND2 ASN C 175      26.033  23.311  65.235  1.00  0.00           N  \\nATOM   2553  H   ASN C 175      27.939  26.680  63.176  1.00  0.00           H  \\nATOM   2554  HA  ASN C 175      28.842  26.384  65.615  1.00  0.00           H  \\nATOM   2555 1HB  ASN C 175      27.511  24.688  66.535  1.00  0.00           H  \\nATOM   2556 2HB  ASN C 175      26.574  26.151  66.310  1.00  0.00           H  \\nATOM   2557 1HD2 ASN C 175      26.641  22.887  65.952  1.00  0.00           H  \\nATOM   2558 2HD2 ASN C 175      25.401  22.725  64.733  1.00  0.00           H  \\nATOM   2559  N   PRO C 176      30.371  24.958  64.231  1.00  0.00           N  \\nATOM   2560  CA  PRO C 176      31.144  23.927  63.536  1.00  0.00           C  \\nATOM   2561  C   PRO C 176      31.202  22.619  64.321  1.00  0.00           C  \\nATOM   2562  O   PRO C 176      32.216  22.320  64.951  1.00  0.00           O  \\nATOM   2563  CB  PRO C 176      32.501  24.622  63.399  1.00  0.00           C  \\nATOM   2564  CG  PRO C 176      32.583  25.555  64.614  1.00  0.00           C  \\nATOM   2565  CD  PRO C 176      31.138  26.060  64.820  1.00  0.00           C  \\nATOM   2566  HA  PRO C 176      30.778  23.762  62.653  1.00  0.00           H  \\nATOM   2567 1HB  PRO C 176      33.307  23.873  63.365  1.00  0.00           H  \\nATOM   2568 2HB  PRO C 176      32.526  25.171  62.446  1.00  0.00           H  \\nATOM   2569 1HG  PRO C 176      32.963  25.017  65.482  1.00  0.00           H  \\nATOM   2570 2HG  PRO C 176      33.292  26.374  64.412  1.00  0.00           H  \\nATOM   2571 1HD  PRO C 176      30.958  26.156  65.897  1.00  0.00           H  \\nATOM   2572 2HD  PRO C 176      30.949  27.001  64.265  1.00  0.00           H  \\nATOM   2573  N   ASP C 177      30.115  21.852  64.289  1.00  0.00           N  \\nATOM   2574  CA  ASP C 177      30.104  20.520  64.885  1.00  0.00           C  \\nATOM   2575  C   ASP C 177      31.015  19.597  64.084  1.00  0.00           C  \\nATOM   2576  O   ASP C 177      31.678  20.034  63.145  1.00  0.00           O  \\nATOM   2577  CB  ASP C 177      28.698  19.892  64.897  1.00  0.00           C  \\nATOM   2578  CG  ASP C 177      27.635  20.556  65.806  1.00  0.00           C  \\nATOM   2579  OD1 ASP C 177      27.932  21.451  66.574  1.00  0.00           O  \\nATOM   2580  OD2 ASP C 177      26.506  20.114  65.725  1.00  0.00           O  \\nATOM   2581  H   ASP C 177      29.370  22.081  63.926  1.00  0.00           H  \\nATOM   2582  HA  ASP C 177      30.444  20.571  65.792  1.00  0.00           H  \\nATOM   2583 1HB  ASP C 177      28.324  19.905  63.895  1.00  0.00           H  \\nATOM   2584 2HB  ASP C 177      28.764  18.854  65.195  1.00  0.00           H  \\nATOM   2585  N   LYS C 178      31.045  18.322  64.456  1.00  0.00           N  \\nATOM   2586  CA  LYS C 178      31.856  17.338  63.748  1.00  0.00           C  \\nATOM   2587  C   LYS C 178      30.991  16.476  62.835  1.00  0.00           C  \\nATOM   2588  O   LYS C 178      31.477  15.932  61.848  1.00  0.00           O  \\nATOM   2589  CB  LYS C 178      32.595  16.501  64.769  1.00  0.00           C  \\nATOM   2590  CG  LYS C 178      33.451  15.454  64.211  1.00  0.00           C  \\nATOM   2591  CD  LYS C 178      34.209  14.787  65.308  1.00  0.00           C  \\nATOM   2592  CE  LYS C 178      33.243  14.127  66.306  1.00  0.00           C  \\nATOM   2593  NZ  LYS C 178      33.897  13.059  67.087  1.00  0.00           N  \\nATOM   2594  H   LYS C 178      30.601  17.999  65.118  1.00  0.00           H  \\nATOM   2595  HA  LYS C 178      32.505  17.803  63.196  1.00  0.00           H  \\nATOM   2596 1HB  LYS C 178      33.220  17.149  65.383  1.00  0.00           H  \\nATOM   2597 2HB  LYS C 178      31.875  16.018  65.429  1.00  0.00           H  \\nATOM   2598 1HG  LYS C 178      32.841  14.707  63.696  1.00  0.00           H  \\nATOM   2599 2HG  LYS C 178      34.149  15.889  63.499  1.00  0.00           H  \\nATOM   2600 1HD  LYS C 178      34.851  14.047  64.876  1.00  0.00           H  \\nATOM   2601 2HD  LYS C 178      34.824  15.514  65.832  1.00  0.00           H  \\nATOM   2602 1HE  LYS C 178      32.879  14.888  66.992  1.00  0.00           H  \\nATOM   2603 2HE  LYS C 178      32.396  13.704  65.767  1.00  0.00           H  \\nATOM   2604 1HZ  LYS C 178      33.260  12.631  67.721  1.00  0.00           H  \\nATOM   2605 2HZ  LYS C 178      34.219  12.330  66.410  1.00  0.00           H  \\nATOM   2606 3HZ  LYS C 178      34.692  13.381  67.600  1.00  0.00           H  \\nATOM   2607  N   ASP C 179      29.708  16.361  63.165  1.00  0.00           N  \\nATOM   2608  CA  ASP C 179      28.761  15.631  62.326  1.00  0.00           C  \\nATOM   2609  C   ASP C 179      28.347  16.481  61.126  1.00  0.00           C  \\nATOM   2610  O   ASP C 179      28.050  15.954  60.054  1.00  0.00           O  \\nATOM   2611  CB  ASP C 179      27.523  15.240  63.143  1.00  0.00           C  \\nATOM   2612  CG  ASP C 179      26.632  14.175  62.472  1.00  0.00           C  \\nATOM   2613  OD1 ASP C 179      27.106  13.081  62.253  1.00  0.00           O  \\nATOM   2614  OD2 ASP C 179      25.491  14.469  62.200  1.00  0.00           O  \\nATOM   2615  H   ASP C 179      29.358  16.698  63.875  1.00  0.00           H  \\nATOM   2616  HA  ASP C 179      29.187  14.826  61.994  1.00  0.00           H  \\nATOM   2617 1HB  ASP C 179      27.838  14.862  64.115  1.00  0.00           H  \\nATOM   2618 2HB  ASP C 179      26.917  16.130  63.323  1.00  0.00           H  \\nATOM   2619  N   VAL C 180      28.323  17.798  61.313  1.00  0.00           N  \\nATOM   2620  CA  VAL C 180      28.053  18.720  60.216  1.00  0.00           C  \\nATOM   2621  C   VAL C 180      29.210  18.679  59.225  1.00  0.00           C  \\nATOM   2622  O   VAL C 180      28.997  18.640  58.015  1.00  0.00           O  \\nATOM   2623  CB  VAL C 180      27.887  20.172  60.721  1.00  0.00           C  \\nATOM   2624  CG1 VAL C 180      27.810  21.148  59.528  1.00  0.00           C  \\nATOM   2625  CG2 VAL C 180      26.618  20.280  61.552  1.00  0.00           C  \\nATOM   2626  H   VAL C 180      28.460  18.183  62.070  1.00  0.00           H  \\nATOM   2627  HA  VAL C 180      27.248  18.440  59.755  1.00  0.00           H  \\nATOM   2628  HB  VAL C 180      28.756  20.444  61.327  1.00  0.00           H  \\nATOM   2629 1HG1 VAL C 180      27.701  22.169  59.899  1.00  0.00           H  \\nATOM   2630 2HG1 VAL C 180      28.722  21.075  58.935  1.00  0.00           H  \\nATOM   2631 3HG1 VAL C 180      26.953  20.898  58.903  1.00  0.00           H  \\nATOM   2632 1HG2 VAL C 180      26.507  21.306  61.917  1.00  0.00           H  \\nATOM   2633 2HG2 VAL C 180      25.760  20.018  60.934  1.00  0.00           H  \\nATOM   2634 3HG2 VAL C 180      26.667  19.600  62.398  1.00  0.00           H  \\nATOM   2635  N   ILE C 181      30.434  18.686  59.748  1.00  0.00           N  \\nATOM   2636  CA  ILE C 181      31.631  18.544  58.923  1.00  0.00           C  \\nATOM   2637  C   ILE C 181      31.555  17.273  58.083  1.00  0.00           C  \\nATOM   2638  O   ILE C 181      31.911  17.279  56.906  1.00  0.00           O  \\nATOM   2639  CB  ILE C 181      32.950  18.525  59.714  1.00  0.00           C  \\nATOM   2640  CG1 ILE C 181      33.233  19.911  60.278  1.00  0.00           C  \\nATOM   2641  CG2 ILE C 181      34.089  18.006  58.818  1.00  0.00           C  \\nATOM   2642  CD1 ILE C 181      34.352  19.958  61.290  1.00  0.00           C  \\nATOM   2643  H   ILE C 181      30.599  18.773  60.588  1.00  0.00           H  \\nATOM   2644  HA  ILE C 181      31.691  19.301  58.319  1.00  0.00           H  \\nATOM   2645  HB  ILE C 181      32.847  17.856  60.568  1.00  0.00           H  \\nATOM   2646 1HG1 ILE C 181      33.492  20.571  59.455  1.00  0.00           H  \\nATOM   2647 2HG1 ILE C 181      32.327  20.282  60.751  1.00  0.00           H  \\nATOM   2648 1HG2 ILE C 181      35.014  17.982  59.385  1.00  0.00           H  \\nATOM   2649 2HG2 ILE C 181      33.852  16.999  58.473  1.00  0.00           H  \\nATOM   2650 3HG2 ILE C 181      34.206  18.646  57.966  1.00  0.00           H  \\nATOM   2651 1HD1 ILE C 181      34.482  20.986  61.639  1.00  0.00           H  \\nATOM   2652 2HD1 ILE C 181      34.101  19.320  62.136  1.00  0.00           H  \\nATOM   2653 3HD1 ILE C 181      35.272  19.610  60.834  1.00  0.00           H  \\nATOM   2654  N   VAL C 182      31.080  16.190  58.693  1.00  0.00           N  \\nATOM   2655  CA  VAL C 182      30.901  14.922  57.990  1.00  0.00           C  \\nATOM   2656  C   VAL C 182      29.944  15.088  56.814  1.00  0.00           C  \\nATOM   2657  O   VAL C 182      30.222  14.625  55.707  1.00  0.00           O  \\nATOM   2658  CB  VAL C 182      30.454  13.791  58.929  1.00  0.00           C  \\nATOM   2659  CG1 VAL C 182      30.019  12.585  58.115  1.00  0.00           C  \\nATOM   2660  CG2 VAL C 182      31.651  13.393  59.811  1.00  0.00           C  \\nATOM   2661  H   VAL C 182      30.853  16.164  59.522  1.00  0.00           H  \\nATOM   2662  HA  VAL C 182      31.757  14.628  57.643  1.00  0.00           H  \\nATOM   2663  HB  VAL C 182      29.622  14.116  59.541  1.00  0.00           H  \\nATOM   2664 1HG1 VAL C 182      29.719  11.779  58.787  1.00  0.00           H  \\nATOM   2665 2HG1 VAL C 182      29.179  12.870  57.496  1.00  0.00           H  \\nATOM   2666 3HG1 VAL C 182      30.821  12.255  57.492  1.00  0.00           H  \\nATOM   2667 1HG2 VAL C 182      31.358  12.588  60.480  1.00  0.00           H  \\nATOM   2668 2HG2 VAL C 182      32.470  13.069  59.190  1.00  0.00           H  \\nATOM   2669 3HG2 VAL C 182      31.975  14.229  60.384  1.00  0.00           H  \\nATOM   2670  N   GLU C 183      28.821  15.755  57.057  1.00  0.00           N  \\nATOM   2671  CA  GLU C 183      27.810  15.944  56.025  1.00  0.00           C  \\nATOM   2672  C   GLU C 183      28.353  16.809  54.892  1.00  0.00           C  \\nATOM   2673  O   GLU C 183      28.124  16.521  53.719  1.00  0.00           O  \\nATOM   2674  CB  GLU C 183      26.535  16.547  56.618  1.00  0.00           C  \\nATOM   2675  CG  GLU C 183      25.290  16.482  55.690  1.00  0.00           C  \\nATOM   2676  CD  GLU C 183      24.711  15.088  55.486  1.00  0.00           C  \\nATOM   2677  OE1 GLU C 183      24.976  14.210  56.275  1.00  0.00           O  \\nATOM   2678  OE2 GLU C 183      24.002  14.907  54.523  1.00  0.00           O  \\nATOM   2679  H   GLU C 183      28.620  16.109  57.815  1.00  0.00           H  \\nATOM   2680  HA  GLU C 183      27.570  15.080  55.655  1.00  0.00           H  \\nATOM   2681 1HB  GLU C 183      26.295  16.036  57.552  1.00  0.00           H  \\nATOM   2682 2HB  GLU C 183      26.716  17.596  56.862  1.00  0.00           H  \\nATOM   2683 1HG  GLU C 183      24.515  17.144  56.071  1.00  0.00           H  \\nATOM   2684 2HG  GLU C 183      25.580  16.825  54.719  1.00  0.00           H  \\nATOM   2685  N   VAL C 184      29.082  17.862  55.250  1.00  0.00           N  \\nATOM   2686  CA  VAL C 184      29.695  18.747  54.265  1.00  0.00           C  \\nATOM   2687  C   VAL C 184      30.661  17.976  53.375  1.00  0.00           C  \\nATOM   2688  O   VAL C 184      30.706  18.194  52.168  1.00  0.00           O  \\nATOM   2689  CB  VAL C 184      30.456  19.925  54.946  1.00  0.00           C  \\nATOM   2690  CG1 VAL C 184      31.281  20.686  53.909  1.00  0.00           C  \\nATOM   2691  CG2 VAL C 184      29.464  20.918  55.602  1.00  0.00           C  \\nATOM   2692  H   VAL C 184      29.238  18.087  56.065  1.00  0.00           H  \\nATOM   2693  HA  VAL C 184      29.000  19.120  53.700  1.00  0.00           H  \\nATOM   2694  HB  VAL C 184      31.131  19.517  55.700  1.00  0.00           H  \\nATOM   2695 1HG1 VAL C 184      31.822  21.497  54.396  1.00  0.00           H  \\nATOM   2696 2HG1 VAL C 184      31.987  20.002  53.449  1.00  0.00           H  \\nATOM   2697 3HG1 VAL C 184      30.620  21.098  53.147  1.00  0.00           H  \\nATOM   2698 1HG2 VAL C 184      30.017  21.726  56.080  1.00  0.00           H  \\nATOM   2699 2HG2 VAL C 184      28.812  21.335  54.841  1.00  0.00           H  \\nATOM   2700 3HG2 VAL C 184      28.867  20.406  56.346  1.00  0.00           H  \\nATOM   2701  N   LEU C 185      31.427  17.069  53.972  1.00  0.00           N  \\nATOM   2702  CA  LEU C 185      32.376  16.268  53.210  1.00  0.00           C  \\nATOM   2703  C   LEU C 185      31.649  15.388  52.200  1.00  0.00           C  \\nATOM   2704  O   LEU C 185      32.119  15.207  51.080  1.00  0.00           O  \\nATOM   2705  CB  LEU C 185      33.238  15.389  54.126  1.00  0.00           C  \\nATOM   2706  CG  LEU C 185      34.263  16.119  55.005  1.00  0.00           C  \\nATOM   2707  CD1 LEU C 185      34.846  15.141  55.972  1.00  0.00           C  \\nATOM   2708  CD2 LEU C 185      35.357  16.706  54.158  1.00  0.00           C  \\nATOM   2709  H   LEU C 185      31.416  16.898  54.815  1.00  0.00           H  \\nATOM   2710  HA  LEU C 185      32.967  16.861  52.720  1.00  0.00           H  \\nATOM   2711 1HB  LEU C 185      32.577  14.835  54.790  1.00  0.00           H  \\nATOM   2712 2HB  LEU C 185      33.778  14.671  53.510  1.00  0.00           H  \\nATOM   2713  HG  LEU C 185      33.779  16.908  55.548  1.00  0.00           H  \\nATOM   2714 1HD1 LEU C 185      35.343  14.356  55.420  1.00  0.00           H  \\nATOM   2715 2HD1 LEU C 185      35.544  15.643  56.586  1.00  0.00           H  \\nATOM   2716 3HD1 LEU C 185      34.052  14.720  56.589  1.00  0.00           H  \\nATOM   2717 1HD2 LEU C 185      35.857  15.908  53.623  1.00  0.00           H  \\nATOM   2718 2HD2 LEU C 185      34.938  17.419  53.452  1.00  0.00           H  \\nATOM   2719 3HD2 LEU C 185      36.071  17.206  54.804  1.00  0.00           H  \\nATOM   2720  N   LYS C 186      30.497  14.853  52.596  1.00  0.00           N  \\nATOM   2721  CA  LYS C 186      29.722  13.982  51.716  1.00  0.00           C  \\nATOM   2722  C   LYS C 186      29.235  14.729  50.477  1.00  0.00           C  \\nATOM   2723  O   LYS C 186      29.327  14.221  49.362  1.00  0.00           O  \\nATOM   2724  CB  LYS C 186      28.554  13.308  52.460  1.00  0.00           C  \\nATOM   2725  CG  LYS C 186      29.010  12.189  53.440  1.00  0.00           C  \\nATOM   2726  CD  LYS C 186      27.874  11.289  53.933  1.00  0.00           C  \\nATOM   2727  CE  LYS C 186      27.061  11.900  55.046  1.00  0.00           C  \\nATOM   2728  NZ  LYS C 186      26.047  10.914  55.579  1.00  0.00           N  \\nATOM   2729  H   LYS C 186      30.142  14.978  53.369  1.00  0.00           H  \\nATOM   2730  HA  LYS C 186      30.288  13.251  51.420  1.00  0.00           H  \\nATOM   2731 1HB  LYS C 186      28.007  14.055  53.036  1.00  0.00           H  \\nATOM   2732 2HB  LYS C 186      27.863  12.876  51.740  1.00  0.00           H  \\nATOM   2733 1HG  LYS C 186      29.715  11.561  52.952  1.00  0.00           H  \\nATOM   2734 2HG  LYS C 186      29.498  12.645  54.300  1.00  0.00           H  \\nATOM   2735 1HD  LYS C 186      27.207  11.064  53.102  1.00  0.00           H  \\nATOM   2736 2HD  LYS C 186      28.298  10.347  54.299  1.00  0.00           H  \\nATOM   2737 1HE  LYS C 186      27.724  12.191  55.847  1.00  0.00           H  \\nATOM   2738 2HE  LYS C 186      26.542  12.785  54.681  1.00  0.00           H  \\nATOM   2739 1HZ  LYS C 186      26.516  10.051  55.937  1.00  0.00           H  \\nATOM   2740 2HZ  LYS C 186      25.515  11.338  56.332  1.00  0.00           H  \\nATOM   2741 3HZ  LYS C 186      25.420  10.653  54.839  1.00  0.00           H  \\nATOM   2742  N   VAL C 187      28.720  15.937  50.677  1.00  0.00           N  \\nATOM   2743  CA  VAL C 187      28.226  16.748  49.570  1.00  0.00           C  \\nATOM   2744  C   VAL C 187      29.378  17.207  48.674  1.00  0.00           C  \\nATOM   2745  O   VAL C 187      29.207  17.346  47.463  1.00  0.00           O  \\nATOM   2746  CB  VAL C 187      27.372  17.912  50.071  1.00  0.00           C  \\nATOM   2747  CG1 VAL C 187      27.000  18.842  48.915  1.00  0.00           C  \\nATOM   2748  CG2 VAL C 187      26.115  17.314  50.671  1.00  0.00           C  \\nATOM   2749  H   VAL C 187      28.645  16.312  51.447  1.00  0.00           H  \\nATOM   2750  HA  VAL C 187      27.627  16.208  49.030  1.00  0.00           H  \\nATOM   2751  HB  VAL C 187      27.925  18.493  50.818  1.00  0.00           H  \\nATOM   2752 1HG1 VAL C 187      26.374  19.660  49.285  1.00  0.00           H  \\nATOM   2753 2HG1 VAL C 187      27.898  19.250  48.460  1.00  0.00           H  \\nATOM   2754 3HG1 VAL C 187      26.448  18.274  48.177  1.00  0.00           H  \\nATOM   2755 1HG2 VAL C 187      26.384  16.657  51.503  1.00  0.00           H  \\nATOM   2756 2HG2 VAL C 187      25.467  18.081  51.018  1.00  0.00           H  \\nATOM   2757 3HG2 VAL C 187      25.592  16.739  49.911  1.00  0.00           H  \\nATOM   2758  N   ILE C 188      30.552  17.422  49.263  1.00  0.00           N  \\nATOM   2759  CA  ILE C 188      31.732  17.781  48.481  1.00  0.00           C  \\nATOM   2760  C   ILE C 188      32.087  16.643  47.527  1.00  0.00           C  \\nATOM   2761  O   ILE C 188      32.427  16.884  46.368  1.00  0.00           O  \\nATOM   2762  CB  ILE C 188      32.976  18.113  49.344  1.00  0.00           C  \\nATOM   2763  CG1 ILE C 188      32.803  19.443  50.080  1.00  0.00           C  \\nATOM   2764  CG2 ILE C 188      34.213  18.108  48.487  1.00  0.00           C  \\nATOM   2765  CD1 ILE C 188      33.846  19.695  51.162  1.00  0.00           C  \\nATOM   2766  H   ILE C 188      30.691  17.367  50.109  1.00  0.00           H  \\nATOM   2767  HA  ILE C 188      31.530  18.565  47.948  1.00  0.00           H  \\nATOM   2768  HB  ILE C 188      33.086  17.350  50.109  1.00  0.00           H  \\nATOM   2769 1HG1 ILE C 188      32.870  20.249  49.351  1.00  0.00           H  \\nATOM   2770 2HG1 ILE C 188      31.829  19.468  50.521  1.00  0.00           H  \\nATOM   2771 1HG2 ILE C 188      35.069  18.319  49.100  1.00  0.00           H  \\nATOM   2772 2HG2 ILE C 188      34.329  17.134  48.022  1.00  0.00           H  \\nATOM   2773 3HG2 ILE C 188      34.126  18.849  47.735  1.00  0.00           H  \\nATOM   2774 1HD1 ILE C 188      33.659  20.663  51.627  1.00  0.00           H  \\nATOM   2775 2HD1 ILE C 188      33.778  18.917  51.911  1.00  0.00           H  \\nATOM   2776 3HD1 ILE C 188      34.834  19.689  50.734  1.00  0.00           H  \\nATOM   2777  N   VAL C 189      31.987  15.405  48.005  1.00  0.00           N  \\nATOM   2778  CA  VAL C 189      32.248  14.246  47.155  1.00  0.00           C  \\nATOM   2779  C   VAL C 189      31.178  14.121  46.075  1.00  0.00           C  \\nATOM   2780  O   VAL C 189      31.471  13.728  44.948  1.00  0.00           O  \\nATOM   2781  CB  VAL C 189      32.366  12.964  47.986  1.00  0.00           C  \\nATOM   2782  CG1 VAL C 189      32.520  11.741  47.049  1.00  0.00           C  \\nATOM   2783  CG2 VAL C 189      33.561  13.113  48.923  1.00  0.00           C  \\nATOM   2784  H   VAL C 189      31.771  15.210  48.814  1.00  0.00           H  \\nATOM   2785  HA  VAL C 189      33.105  14.364  46.716  1.00  0.00           H  \\nATOM   2786  HB  VAL C 189      31.455  12.822  48.572  1.00  0.00           H  \\nATOM   2787 1HG1 VAL C 189      32.603  10.830  47.640  1.00  0.00           H  \\nATOM   2788 2HG1 VAL C 189      31.648  11.672  46.396  1.00  0.00           H  \\nATOM   2789 3HG1 VAL C 189      33.420  11.856  46.442  1.00  0.00           H  \\nATOM   2790 1HG2 VAL C 189      33.678  12.223  49.530  1.00  0.00           H  \\nATOM   2791 2HG2 VAL C 189      34.434  13.266  48.346  1.00  0.00           H  \\nATOM   2792 3HG2 VAL C 189      33.419  13.962  49.561  1.00  0.00           H  \\nATOM   2793  N   LYS C 190      29.938  14.453  46.418  1.00  0.00           N  \\nATOM   2794  CA  LYS C 190      28.869  14.462  45.427  1.00  0.00           C  \\nATOM   2795  C   LYS C 190      29.154  15.518  44.362  1.00  0.00           C  \\nATOM   2796  O   LYS C 190      28.932  15.289  43.176  1.00  0.00           O  \\nATOM   2797  CB  LYS C 190      27.504  14.752  46.057  1.00  0.00           C  \\nATOM   2798  CG  LYS C 190      26.943  13.627  46.906  1.00  0.00           C  \\nATOM   2799  CD  LYS C 190      25.623  14.027  47.555  1.00  0.00           C  \\nATOM   2800  CE  LYS C 190      24.501  14.087  46.531  1.00  0.00           C  \\nATOM   2801  NZ  LYS C 190      24.139  12.727  46.028  1.00  0.00           N  \\nATOM   2802  H   LYS C 190      29.690  14.675  47.211  1.00  0.00           H  \\nATOM   2803  HA  LYS C 190      28.831  13.596  44.991  1.00  0.00           H  \\nATOM   2804 1HB  LYS C 190      27.567  15.640  46.668  1.00  0.00           H  \\nATOM   2805 2HB  LYS C 190      26.786  14.961  45.267  1.00  0.00           H  \\nATOM   2806 1HG  LYS C 190      26.786  12.747  46.282  1.00  0.00           H  \\nATOM   2807 2HG  LYS C 190      27.660  13.374  47.683  1.00  0.00           H  \\nATOM   2808 1HD  LYS C 190      25.366  13.308  48.334  1.00  0.00           H  \\nATOM   2809 2HD  LYS C 190      25.729  15.009  48.014  1.00  0.00           H  \\nATOM   2810 1HE  LYS C 190      23.623  14.541  46.991  1.00  0.00           H  \\nATOM   2811 2HE  LYS C 190      24.817  14.706  45.688  1.00  0.00           H  \\nATOM   2812 1HZ  LYS C 190      23.392  12.803  45.353  1.00  0.00           H  \\nATOM   2813 2HZ  LYS C 190      24.948  12.302  45.589  1.00  0.00           H  \\nATOM   2814 3HZ  LYS C 190      23.835  12.152  46.801  1.00  0.00           H  \\nATOM   2815  N   ALA C 191      29.655  16.672  44.793  1.00  0.00           N  \\nATOM   2816  CA  ALA C 191      29.996  17.745  43.866  1.00  0.00           C  \\nATOM   2817  C   ALA C 191      31.127  17.307  42.944  1.00  0.00           C  \\nATOM   2818  O   ALA C 191      31.067  17.521  41.737  1.00  0.00           O  \\nATOM   2819  CB  ALA C 191      30.403  19.000  44.627  1.00  0.00           C  \\nATOM   2820  H   ALA C 191      29.807  16.858  45.618  1.00  0.00           H  \\nATOM   2821  HA  ALA C 191      29.222  17.949  43.318  1.00  0.00           H  \\nATOM   2822 1HB  ALA C 191      30.639  19.789  43.920  1.00  0.00           H  \\nATOM   2823 2HB  ALA C 191      29.593  19.320  45.274  1.00  0.00           H  \\nATOM   2824 3HB  ALA C 191      31.276  18.787  45.232  1.00  0.00           H  \\nATOM   2825  N   ILE C 192      32.154  16.685  43.518  1.00  0.00           N  \\nATOM   2826  CA  ILE C 192      33.290  16.206  42.738  1.00  0.00           C  \\nATOM   2827  C   ILE C 192      32.846  15.113  41.771  1.00  0.00           C  \\nATOM   2828  O   ILE C 192      33.307  15.065  40.633  1.00  0.00           O  \\nATOM   2829  CB  ILE C 192      34.436  15.702  43.627  1.00  0.00           C  \\nATOM   2830  CG1 ILE C 192      35.055  16.902  44.358  1.00  0.00           C  \\nATOM   2831  CG2 ILE C 192      35.487  14.963  42.774  1.00  0.00           C  \\nATOM   2832  CD1 ILE C 192      35.993  16.544  45.451  1.00  0.00           C  \\nATOM   2833  H   ILE C 192      32.216  16.528  44.361  1.00  0.00           H  \\nATOM   2834  HA  ILE C 192      33.647  16.942  42.216  1.00  0.00           H  \\nATOM   2835  HB  ILE C 192      34.032  15.023  44.381  1.00  0.00           H  \\nATOM   2836 1HG1 ILE C 192      35.597  17.507  43.635  1.00  0.00           H  \\nATOM   2837 2HG1 ILE C 192      34.253  17.506  44.780  1.00  0.00           H  \\nATOM   2838 1HG2 ILE C 192      36.284  14.606  43.418  1.00  0.00           H  \\nATOM   2839 2HG2 ILE C 192      35.017  14.122  42.274  1.00  0.00           H  \\nATOM   2840 3HG2 ILE C 192      35.897  15.642  42.025  1.00  0.00           H  \\nATOM   2841 1HD1 ILE C 192      36.357  17.466  45.875  1.00  0.00           H  \\nATOM   2842 2HD1 ILE C 192      35.468  15.962  46.208  1.00  0.00           H  \\nATOM   2843 3HD1 ILE C 192      36.820  15.967  45.056  1.00  0.00           H  \\nATOM   2844  N   GLU C 193      31.952  14.237  42.224  1.00  0.00           N  \\nATOM   2845  CA  GLU C 193      31.357  13.232  41.348  1.00  0.00           C  \\nATOM   2846  C   GLU C 193      30.676  13.909  40.167  1.00  0.00           C  \\nATOM   2847  O   GLU C 193      30.931  13.574  39.011  1.00  0.00           O  \\nATOM   2848  CB  GLU C 193      30.385  12.310  42.093  1.00  0.00           C  \\nATOM   2849  CG  GLU C 193      29.789  11.206  41.199  1.00  0.00           C  \\nATOM   2850  CD  GLU C 193      28.956  10.181  41.934  1.00  0.00           C  \\nATOM   2851  OE1 GLU C 193      28.453  10.458  42.994  1.00  0.00           O  \\nATOM   2852  OE2 GLU C 193      28.886   9.068  41.447  1.00  0.00           O  \\nATOM   2853  H   GLU C 193      31.674  14.204  43.037  1.00  0.00           H  \\nATOM   2854  HA  GLU C 193      32.054  12.650  41.007  1.00  0.00           H  \\nATOM   2855 1HB  GLU C 193      30.898  11.838  42.931  1.00  0.00           H  \\nATOM   2856 2HB  GLU C 193      29.565  12.893  42.499  1.00  0.00           H  \\nATOM   2857 1HG  GLU C 193      29.155  11.689  40.455  1.00  0.00           H  \\nATOM   2858 2HG  GLU C 193      30.584  10.704  40.667  1.00  0.00           H  \\nATOM   2859  N   ALA C 194      29.803  14.862  40.471  1.00  0.00           N  \\nATOM   2860  CA  ALA C 194      29.082  15.592  39.442  1.00  0.00           C  \\nATOM   2861  C   ALA C 194      30.065  16.299  38.517  1.00  0.00           C  \\nATOM   2862  O   ALA C 194      29.849  16.373  37.309  1.00  0.00           O  \\nATOM   2863  CB  ALA C 194      28.124  16.572  40.084  1.00  0.00           C  \\nATOM   2864  H   ALA C 194      29.611  15.106  41.273  1.00  0.00           H  \\nATOM   2865  HA  ALA C 194      28.562  14.967  38.913  1.00  0.00           H  \\nATOM   2866 1HB  ALA C 194      27.567  17.112  39.317  1.00  0.00           H  \\nATOM   2867 2HB  ALA C 194      27.433  16.020  40.716  1.00  0.00           H  \\nATOM   2868 3HB  ALA C 194      28.684  17.267  40.692  1.00  0.00           H  \\nATOM   2869  N   SER C 195      31.155  16.802  39.088  1.00  0.00           N  \\nATOM   2870  CA  SER C 195      32.141  17.535  38.309  1.00  0.00           C  \\nATOM   2871  C   SER C 195      32.848  16.611  37.328  1.00  0.00           C  \\nATOM   2872  O   SER C 195      33.021  16.957  36.162  1.00  0.00           O  \\nATOM   2873  CB  SER C 195      33.136  18.198  39.224  1.00  0.00           C  \\nATOM   2874  OG  SER C 195      34.081  18.941  38.506  1.00  0.00           O  \\nATOM   2875  H   SER C 195      31.344  16.732  39.924  1.00  0.00           H  \\nATOM   2876  HA  SER C 195      31.691  18.227  37.800  1.00  0.00           H  \\nATOM   2877 1HB  SER C 195      32.608  18.846  39.924  1.00  0.00           H  \\nATOM   2878 2HB  SER C 195      33.643  17.440  39.804  1.00  0.00           H  \\nATOM   2879  HG  SER C 195      34.705  19.248  39.164  1.00  0.00           H  \\nATOM   2880  N   VAL C 196      33.256  15.438  37.801  1.00  0.00           N  \\nATOM   2881  CA  VAL C 196      33.914  14.461  36.941  1.00  0.00           C  \\nATOM   2882  C   VAL C 196      32.963  14.016  35.837  1.00  0.00           C  \\nATOM   2883  O   VAL C 196      33.370  13.853  34.688  1.00  0.00           O  \\nATOM   2884  CB  VAL C 196      34.408  13.238  37.760  1.00  0.00           C  \\nATOM   2885  CG1 VAL C 196      34.903  12.120  36.822  1.00  0.00           C  \\nATOM   2886  CG2 VAL C 196      35.565  13.668  38.683  1.00  0.00           C  \\nATOM   2887  H   VAL C 196      33.164  15.183  38.618  1.00  0.00           H  \\nATOM   2888  HA  VAL C 196      34.686  14.875  36.525  1.00  0.00           H  \\nATOM   2889  HB  VAL C 196      33.576  12.849  38.359  1.00  0.00           H  \\nATOM   2890 1HG1 VAL C 196      35.242  11.268  37.417  1.00  0.00           H  \\nATOM   2891 2HG1 VAL C 196      34.090  11.806  36.170  1.00  0.00           H  \\nATOM   2892 3HG1 VAL C 196      35.732  12.493  36.220  1.00  0.00           H  \\nATOM   2893 1HG2 VAL C 196      35.907  12.813  39.266  1.00  0.00           H  \\nATOM   2894 2HG2 VAL C 196      36.379  14.046  38.078  1.00  0.00           H  \\nATOM   2895 3HG2 VAL C 196      35.233  14.447  39.351  1.00  0.00           H  \\nATOM   2896  N   GLU C 197      31.693  13.832  36.185  1.00  0.00           N  \\nATOM   2897  CA  GLU C 197      30.694  13.422  35.205  1.00  0.00           C  \\nATOM   2898  C   GLU C 197      30.557  14.486  34.123  1.00  0.00           C  \\nATOM   2899  O   GLU C 197      30.467  14.168  32.937  1.00  0.00           O  \\nATOM   2900  CB  GLU C 197      29.324  13.223  35.874  1.00  0.00           C  \\nATOM   2901  CG  GLU C 197      28.147  12.835  34.929  1.00  0.00           C  \\nATOM   2902  CD  GLU C 197      28.176  11.432  34.373  1.00  0.00           C  \\nATOM   2903  OE1 GLU C 197      28.815  10.587  34.945  1.00  0.00           O  \\nATOM   2904  OE2 GLU C 197      27.540  11.206  33.371  1.00  0.00           O  \\nATOM   2905  H   GLU C 197      31.385  13.938  36.981  1.00  0.00           H  \\nATOM   2906  HA  GLU C 197      30.981  12.597  34.784  1.00  0.00           H  \\nATOM   2907 1HB  GLU C 197      29.408  12.453  36.641  1.00  0.00           H  \\nATOM   2908 2HB  GLU C 197      29.038  14.140  36.377  1.00  0.00           H  \\nATOM   2909 1HG  GLU C 197      27.216  12.968  35.478  1.00  0.00           H  \\nATOM   2910 2HG  GLU C 197      28.138  13.542  34.096  1.00  0.00           H  \\nATOM   2911  N   ASN C 198      30.533  15.749  34.515  1.00  0.00           N  \\nATOM   2912  CA  ASN C 198      30.490  16.833  33.563  1.00  0.00           C  \\nATOM   2913  C   ASN C 198      31.704  16.847  32.657  1.00  0.00           C  \\nATOM   2914  O   ASN C 198      31.606  17.049  31.503  1.00  0.00           O  \\nATOM   2915  CB  ASN C 198      30.342  18.170  34.264  1.00  0.00           C  \\nATOM   2916  CG  ASN C 198      30.052  19.277  33.285  1.00  0.00           C  \\nATOM   2917  OD1 ASN C 198      28.994  19.298  32.674  1.00  0.00           O  \\nATOM   2918  ND2 ASN C 198      30.973  20.159  33.097  1.00  0.00           N  \\nATOM   2919  H   ASN C 198      30.530  15.995  35.330  1.00  0.00           H  \\nATOM   2920  HA  ASN C 198      29.708  16.706  33.001  1.00  0.00           H  \\nATOM   2921 1HB  ASN C 198      29.540  18.116  34.999  1.00  0.00           H  \\nATOM   2922 2HB  ASN C 198      31.255  18.390  34.803  1.00  0.00           H  \\nATOM   2923 1HD2 ASN C 198      31.820  20.097  33.639  1.00  0.00           H  \\nATOM   2924 2HD2 ASN C 198      30.845  20.886  32.424  1.00  0.00           H  \\nATOM   2925  N   GLN C 199      32.858  16.625  33.209  1.00  0.00           N  \\nATOM   2926  CA  GLN C 199      34.076  16.648  32.415  1.00  0.00           C  \\nATOM   2927  C   GLN C 199      34.110  15.478  31.436  1.00  0.00           C  \\nATOM   2928  O   GLN C 199      34.723  15.573  30.375  1.00  0.00           O  \\nATOM   2929  CB  GLN C 199      35.260  16.645  33.369  1.00  0.00           C  \\nATOM   2930  CG  GLN C 199      35.356  17.961  34.181  1.00  0.00           C  \\nATOM   2931  CD  GLN C 199      36.447  17.978  35.231  1.00  0.00           C  \\nATOM   2932  OE1 GLN C 199      37.620  17.835  34.897  1.00  0.00           O  \\nATOM   2933  NE2 GLN C 199      36.106  18.143  36.509  1.00  0.00           N  \\nATOM   2934  H   GLN C 199      32.981  16.461  34.044  1.00  0.00           H  \\nATOM   2935  HA  GLN C 199      34.099  17.469  31.899  1.00  0.00           H  \\nATOM   2936 1HB  GLN C 199      35.159  15.814  34.069  1.00  0.00           H  \\nATOM   2937 2HB  GLN C 199      36.168  16.502  32.811  1.00  0.00           H  \\nATOM   2938 1HG  GLN C 199      35.563  18.771  33.485  1.00  0.00           H  \\nATOM   2939 2HG  GLN C 199      34.412  18.154  34.665  1.00  0.00           H  \\nATOM   2940 1HE2 GLN C 199      36.823  18.184  37.210  1.00  0.00           H  \\nATOM   2941 2HE2 GLN C 199      35.149  18.259  36.824  1.00  0.00           H  \\nATOM   2942  N   ARG C 200      33.432  14.385  31.777  1.00  0.00           N  \\nATOM   2943  CA  ARG C 200      33.352  13.245  30.871  1.00  0.00           C  \\nATOM   2944  C   ARG C 200      32.556  13.615  29.633  1.00  0.00           C  \\nATOM   2945  O   ARG C 200      32.976  13.342  28.511  1.00  0.00           O  \\nATOM   2946  CB  ARG C 200      32.730  12.030  31.547  1.00  0.00           C  \\nATOM   2947  CG  ARG C 200      33.671  11.383  32.453  1.00  0.00           C  \\nATOM   2948  CD  ARG C 200      33.236  10.084  33.053  1.00  0.00           C  \\nATOM   2949  NE  ARG C 200      32.380  10.250  34.198  1.00  0.00           N  \\nATOM   2950  CZ  ARG C 200      31.772   9.238  34.847  1.00  0.00           C  \\nATOM   2951  NH1 ARG C 200      31.932   8.001  34.433  1.00  0.00           N  \\nATOM   2952  NH2 ARG C 200      31.016   9.475  35.896  1.00  0.00           N  \\nATOM   2953  H   ARG C 200      33.013  14.280  32.521  1.00  0.00           H  \\nATOM   2954  HA  ARG C 200      34.248  13.000  30.591  1.00  0.00           H  \\nATOM   2955 1HB  ARG C 200      31.856  12.318  32.116  1.00  0.00           H  \\nATOM   2956 2HB  ARG C 200      32.417  11.307  30.797  1.00  0.00           H  \\nATOM   2957 1HG  ARG C 200      34.534  11.248  31.893  1.00  0.00           H  \\nATOM   2958 2HG  ARG C 200      33.888  12.053  33.259  1.00  0.00           H  \\nATOM   2959 1HD  ARG C 200      32.690   9.514  32.305  1.00  0.00           H  \\nATOM   2960 2HD  ARG C 200      34.112   9.520  33.365  1.00  0.00           H  \\nATOM   2961  HE  ARG C 200      32.232  11.189  34.544  1.00  0.00           H  \\nATOM   2962 1HH1 ARG C 200      32.500   7.808  33.626  1.00  0.00           H  \\nATOM   2963 2HH1 ARG C 200      31.468   7.247  34.920  1.00  0.00           H  \\nATOM   2964 1HH2 ARG C 200      30.869  10.423  36.214  1.00  0.00           H  \\nATOM   2965 2HH2 ARG C 200      30.548   8.719  36.365  1.00  0.00           H  \\nATOM   2966  N   ILE C 201      31.409  14.248  29.845  1.00  0.00           N  \\nATOM   2967  CA  ILE C 201      30.556  14.665  28.744  1.00  0.00           C  \\nATOM   2968  C   ILE C 201      31.278  15.712  27.900  1.00  0.00           C  \\nATOM   2969  O   ILE C 201      31.153  15.726  26.677  1.00  0.00           O  \\nATOM   2970  CB  ILE C 201      29.246  15.296  29.233  1.00  0.00           C  \\nATOM   2971  CG1 ILE C 201      28.391  14.286  29.941  1.00  0.00           C  \\nATOM   2972  CG2 ILE C 201      28.473  15.820  28.004  1.00  0.00           C  \\nATOM   2973  CD1 ILE C 201      27.247  14.926  30.675  1.00  0.00           C  \\nATOM   2974  H   ILE C 201      31.102  14.448  30.623  1.00  0.00           H  \\nATOM   2975  HA  ILE C 201      30.367  13.899  28.180  1.00  0.00           H  \\nATOM   2976  HB  ILE C 201      29.460  16.113  29.929  1.00  0.00           H  \\nATOM   2977 1HG1 ILE C 201      27.997  13.591  29.216  1.00  0.00           H  \\nATOM   2978 2HG1 ILE C 201      29.002  13.733  30.664  1.00  0.00           H  \\nATOM   2979 1HG2 ILE C 201      27.542  16.267  28.325  1.00  0.00           H  \\nATOM   2980 2HG2 ILE C 201      29.070  16.565  27.480  1.00  0.00           H  \\nATOM   2981 3HG2 ILE C 201      28.258  14.991  27.325  1.00  0.00           H  \\nATOM   2982 1HD1 ILE C 201      26.652  14.162  31.174  1.00  0.00           H  \\nATOM   2983 2HD1 ILE C 201      27.653  15.615  31.418  1.00  0.00           H  \\nATOM   2984 3HD1 ILE C 201      26.631  15.471  29.978  1.00  0.00           H  \\nATOM   2985  N   SER C 202      32.040  16.580  28.559  1.00  0.00           N  \\nATOM   2986  CA  SER C 202      32.816  17.601  27.861  1.00  0.00           C  \\nATOM   2987  C   SER C 202      33.842  16.968  26.923  1.00  0.00           C  \\nATOM   2988  O   SER C 202      33.967  17.371  25.765  1.00  0.00           O  \\nATOM   2989  CB  SER C 202      33.530  18.491  28.852  1.00  0.00           C  \\nATOM   2990  OG  SER C 202      34.315  19.480  28.209  1.00  0.00           O  \\nATOM   2991  H   SER C 202      32.125  16.599  29.414  1.00  0.00           H  \\nATOM   2992  HA  SER C 202      32.216  18.144  27.327  1.00  0.00           H  \\nATOM   2993 1HB  SER C 202      32.795  18.965  29.505  1.00  0.00           H  \\nATOM   2994 2HB  SER C 202      34.167  17.883  29.475  1.00  0.00           H  \\nATOM   2995  HG  SER C 202      34.748  19.956  28.922  1.00  0.00           H  \\nATOM   2996  N   ALA C 203      34.581  15.985  27.428  1.00  0.00           N  \\nATOM   2997  CA  ALA C 203      35.560  15.272  26.613  1.00  0.00           C  \\nATOM   2998  C   ALA C 203      34.879  14.603  25.421  1.00  0.00           C  \\nATOM   2999  O   ALA C 203      35.415  14.603  24.314  1.00  0.00           O  \\nATOM   3000  CB  ALA C 203      36.284  14.240  27.457  1.00  0.00           C  \\nATOM   3001  H   ALA C 203      34.535  15.711  28.242  1.00  0.00           H  \\nATOM   3002  HA  ALA C 203      36.209  15.906  26.271  1.00  0.00           H  \\nATOM   3003 1HB  ALA C 203      36.779  14.738  28.292  1.00  0.00           H  \\nATOM   3004 2HB  ALA C 203      35.559  13.520  27.843  1.00  0.00           H  \\nATOM   3005 3HB  ALA C 203      37.024  13.725  26.850  1.00  0.00           H  \\nATOM   3006  N   GLU C 204      33.695  14.041  25.657  1.00  0.00           N  \\nATOM   3007  CA  GLU C 204      32.887  13.460  24.586  1.00  0.00           C  \\nATOM   3008  C   GLU C 204      32.621  14.474  23.489  1.00  0.00           C  \\nATOM   3009  O   GLU C 204      32.772  14.179  22.305  1.00  0.00           O  \\nATOM   3010  CB  GLU C 204      31.512  12.902  25.020  1.00  0.00           C  \\nATOM   3011  CG  GLU C 204      31.494  11.613  25.847  1.00  0.00           C  \\nATOM   3012  CD  GLU C 204      30.044  11.139  26.121  1.00  0.00           C  \\nATOM   3013  OE1 GLU C 204      29.173  11.485  25.344  1.00  0.00           O  \\nATOM   3014  OE2 GLU C 204      29.806  10.473  27.106  1.00  0.00           O  \\nATOM   3015  H   GLU C 204      33.335  13.983  26.435  1.00  0.00           H  \\nATOM   3016  HA  GLU C 204      33.362  12.709  24.198  1.00  0.00           H  \\nATOM   3017 1HB  GLU C 204      30.975  13.639  25.578  1.00  0.00           H  \\nATOM   3018 2HB  GLU C 204      30.921  12.707  24.126  1.00  0.00           H  \\nATOM   3019 1HG  GLU C 204      32.032  10.834  25.309  1.00  0.00           H  \\nATOM   3020 2HG  GLU C 204      32.001  11.787  26.789  1.00  0.00           H  \\nATOM   3021  N   ASN C 205      32.264  15.675  23.875  1.00  0.00           N  \\nATOM   3022  CA  ASN C 205      31.927  16.712  22.932  1.00  0.00           C  \\nATOM   3023  C   ASN C 205      33.158  17.188  22.199  1.00  0.00           C  \\nATOM   3024  O   ASN C 205      33.099  17.588  21.094  1.00  0.00           O  \\nATOM   3025  CB  ASN C 205      31.178  17.845  23.584  1.00  0.00           C  \\nATOM   3026  CG  ASN C 205      30.609  18.787  22.539  1.00  0.00           C  \\nATOM   3027  OD1 ASN C 205      29.974  18.370  21.568  1.00  0.00           O  \\nATOM   3028  ND2 ASN C 205      30.813  20.043  22.724  1.00  0.00           N  \\nATOM   3029  H   ASN C 205      32.222  15.911  24.692  1.00  0.00           H  \\nATOM   3030  HA  ASN C 205      31.306  16.342  22.281  1.00  0.00           H  \\nATOM   3031 1HB  ASN C 205      30.367  17.450  24.200  1.00  0.00           H  \\nATOM   3032 2HB  ASN C 205      31.855  18.389  24.240  1.00  0.00           H  \\nATOM   3033 1HD2 ASN C 205      30.458  20.704  22.067  1.00  0.00           H  \\nATOM   3034 2HD2 ASN C 205      31.327  20.353  23.540  1.00  0.00           H  \\nATOM   3035  N   GLN C 206      34.289  17.113  22.836  1.00  0.00           N  \\nATOM   3036  CA  GLN C 206      35.535  17.516  22.194  1.00  0.00           C  \\nATOM   3037  C   GLN C 206      35.974  16.511  21.138  1.00  0.00           C  \\nATOM   3038  O   GLN C 206      36.583  16.889  20.139  1.00  0.00           O  \\nATOM   3039  CB  GLN C 206      36.641  17.714  23.224  1.00  0.00           C  \\nATOM   3040  CG  GLN C 206      36.407  18.902  24.121  1.00  0.00           C  \\nATOM   3041  CD  GLN C 206      37.453  19.071  25.194  1.00  0.00           C  \\nATOM   3042  OE1 GLN C 206      38.670  19.110  24.916  1.00  0.00           O  \\nATOM   3043  NE2 GLN C 206      37.008  19.195  26.452  1.00  0.00           N  \\nATOM   3044  H   GLN C 206      34.378  16.845  23.648  1.00  0.00           H  \\nATOM   3045  HA  GLN C 206      35.398  18.369  21.753  1.00  0.00           H  \\nATOM   3046 1HB  GLN C 206      36.730  16.822  23.843  1.00  0.00           H  \\nATOM   3047 2HB  GLN C 206      37.585  17.861  22.714  1.00  0.00           H  \\nATOM   3048 1HG  GLN C 206      36.424  19.797  23.502  1.00  0.00           H  \\nATOM   3049 2HG  GLN C 206      35.442  18.817  24.593  1.00  0.00           H  \\nATOM   3050 1HE2 GLN C 206      37.656  19.366  27.188  1.00  0.00           H  \\nATOM   3051 2HE2 GLN C 206      36.015  19.166  26.701  1.00  0.00           H  \\nATOM   3052  N   LYS C 207      35.676  15.234  21.365  1.00  0.00           N  \\nATOM   3053  CA  LYS C 207      35.926  14.204  20.361  1.00  0.00           C  \\nATOM   3054  C   LYS C 207      35.120  14.500  19.102  1.00  0.00           C  \\nATOM   3055  O   LYS C 207      35.652  14.488  17.991  1.00  0.00           O  \\nATOM   3056  CB  LYS C 207      35.600  12.811  20.923  1.00  0.00           C  \\nATOM   3057  CG  LYS C 207      36.620  12.249  21.982  1.00  0.00           C  \\nATOM   3058  CD  LYS C 207      36.137  10.874  22.527  1.00  0.00           C  \\nATOM   3059  CE  LYS C 207      37.058  10.252  23.631  1.00  0.00           C  \\nATOM   3060  NZ  LYS C 207      38.347   9.687  23.082  1.00  0.00           N  \\nATOM   3061  H   LYS C 207      35.327  14.938  22.093  1.00  0.00           H  \\nATOM   3062  HA  LYS C 207      36.868  14.206  20.128  1.00  0.00           H  \\nATOM   3063 1HB  LYS C 207      34.625  12.831  21.394  1.00  0.00           H  \\nATOM   3064 2HB  LYS C 207      35.547  12.099  20.103  1.00  0.00           H  \\nATOM   3065 1HG  LYS C 207      37.601  12.133  21.525  1.00  0.00           H  \\nATOM   3066 2HG  LYS C 207      36.703  12.954  22.809  1.00  0.00           H  \\nATOM   3067 1HD  LYS C 207      35.138  11.004  22.949  1.00  0.00           H  \\nATOM   3068 2HD  LYS C 207      36.066  10.171  21.699  1.00  0.00           H  \\nATOM   3069 1HE  LYS C 207      37.297  11.018  24.370  1.00  0.00           H  \\nATOM   3070 2HE  LYS C 207      36.510   9.449  24.121  1.00  0.00           H  \\nATOM   3071 1HZ  LYS C 207      38.930   9.257  23.849  1.00  0.00           H  \\nATOM   3072 2HZ  LYS C 207      38.135   8.969  22.408  1.00  0.00           H  \\nATOM   3073 3HZ  LYS C 207      38.876  10.418  22.637  1.00  0.00           H  \\nATOM   3074  N   ALA C 208      33.830  14.764  19.290  1.00  0.00           N  \\nATOM   3075  CA  ALA C 208      32.922  15.038  18.182  1.00  0.00           C  \\nATOM   3076  C   ALA C 208      33.345  16.279  17.406  1.00  0.00           C  \\nATOM   3077  O   ALA C 208      33.284  16.306  16.178  1.00  0.00           O  \\nATOM   3078  CB  ALA C 208      31.526  15.250  18.738  1.00  0.00           C  \\nATOM   3079  H   ALA C 208      33.452  14.791  20.062  1.00  0.00           H  \\nATOM   3080  HA  ALA C 208      32.933  14.285  17.571  1.00  0.00           H  \\nATOM   3081 1HB  ALA C 208      30.842  15.433  17.924  1.00  0.00           H  \\nATOM   3082 2HB  ALA C 208      31.221  14.366  19.287  1.00  0.00           H  \\nATOM   3083 3HB  ALA C 208      31.530  16.108  19.415  1.00  0.00           H  \\nATOM   3084  N   LEU C 209      33.772  17.308  18.129  1.00  0.00           N  \\nATOM   3085  CA  LEU C 209      34.193  18.553  17.503  1.00  0.00           C  \\nATOM   3086  C   LEU C 209      35.456  18.353  16.662  1.00  0.00           C  \\nATOM   3087  O   LEU C 209      35.676  19.069  15.686  1.00  0.00           O  \\nATOM   3088  CB  LEU C 209      34.429  19.653  18.562  1.00  0.00           C  \\nATOM   3089  CG  LEU C 209      33.154  20.224  19.280  1.00  0.00           C  \\nATOM   3090  CD1 LEU C 209      33.571  21.157  20.439  1.00  0.00           C  \\nATOM   3091  CD2 LEU C 209      32.324  20.985  18.275  1.00  0.00           C  \\nATOM   3092  H   LEU C 209      33.828  17.309  18.987  1.00  0.00           H  \\nATOM   3093  HA  LEU C 209      33.488  18.862  16.913  1.00  0.00           H  \\nATOM   3094 1HB  LEU C 209      35.080  19.238  19.333  1.00  0.00           H  \\nATOM   3095 2HB  LEU C 209      34.949  20.485  18.089  1.00  0.00           H  \\nATOM   3096  HG  LEU C 209      32.560  19.412  19.678  1.00  0.00           H  \\nATOM   3097 1HD1 LEU C 209      32.679  21.553  20.928  1.00  0.00           H  \\nATOM   3098 2HD1 LEU C 209      34.160  20.592  21.162  1.00  0.00           H  \\nATOM   3099 3HD1 LEU C 209      34.162  21.976  20.048  1.00  0.00           H  \\nATOM   3100 1HD2 LEU C 209      32.903  21.815  17.867  1.00  0.00           H  \\nATOM   3101 2HD2 LEU C 209      32.032  20.316  17.472  1.00  0.00           H  \\nATOM   3102 3HD2 LEU C 209      31.441  21.370  18.769  1.00  0.00           H  \\nTER                                                                             \\nVirtualRootMover_root 210\\n                                                      \\n\",\"pdb\");\n",
              "\tviewer_16234223362302048.zoomTo();\n",
              "\tviewer_16234223362302048.setStyle({\"cartoon\": {\"color\": \"spectrum\"}, \"stick\": {\"colorscheme\": \"blackCarbon\", \"radius\": \"0.1\"}});\n",
              "\tviewer_16234223362302048.setStyle({\"resi\": [7, 8, 11, 12, 14, 15, 18, 19, 22, 25, 26, 29, 32, 33, 36, 37, 38, 39, 41, 42, 44, 45, 46, 48, 49, 51, 52, 53, 55, 56, 58, 59, 60, 62, 63, 65, 66, 67, 69, 70, 73, 77, 80, 84, 87, 88, 91, 94, 95, 98, 101, 102, 105, 110, 113, 117, 120, 123, 124, 127, 131, 134, 137, 138, 178, 179, 181, 182, 185, 186, 188, 189, 192, 193, 195, 196, 199, 200, 202, 203, 206, 207, 209, 1], \"chain\": [\"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"z\"]},{\"cartoon\": {\"color\": \"spectrum\"}, \"stick\": {\"colorscheme\": \"greyCarbon\", \"radius\": \"0.1\"}});\n",
              "\tviewer_16234223362302048.addResLabels({\"resi\": [7, 8, 11, 12, 14, 15, 18, 19, 22, 25, 26, 29, 32, 33, 36, 37, 38, 39, 41, 42, 44, 45, 46, 48, 49, 51, 52, 53, 55, 56, 58, 59, 60, 62, 63, 65, 66, 67, 69, 70, 73, 77, 80, 84, 87, 88, 91, 94, 95, 98, 101, 102, 105, 110, 113, 117, 120, 123, 124, 127, 131, 134, 137, 138, 178, 179, 181, 182, 185, 186, 188, 189, 192, 193, 195, 196, 199, 200, 202, 203, 206, 207, 209, 1], \"chain\": [\"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"A\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"B\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"C\", \"z\"]},{\"fontSize\": 12, \"showBackground\": false, \"fontColor\": \"black\"});\n",
              "viewer_16234223362302048.render();\n",
              "});\n",
              "</script>"
            ]
          },
          "metadata": {
            "tags": []
          }
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "59cNPRfMmdlb"
      },
      "source": [
        " Create TaskFactory to be used with the `FastRelax` mover (We use this instead of the FastDesign mover as the constructor allows us to set a relax script):"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "_tTuhC3Ymdlb",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "f5e304c8-05c1-4f88-9f8f-9556093cf1cd"
      },
      "source": [
        "tf.clear()\n",
        "tf = pyrosetta.rosetta.core.pack.task.TaskFactory()\n",
        "\n",
        "tf.push_back(pyrosetta.rosetta.core.pack.task.operation.InitializeFromCommandline())\n",
        "tf.push_back(pyrosetta.rosetta.core.pack.task.operation.IncludeCurrent())\n",
        "tf.push_back(pyrosetta.rosetta.core.pack.task.operation.NoRepackDisulfides())\n",
        "\n",
        "# Prevent repacking on chain_BC_not_interface\n",
        "tf.push_back(pyrosetta.rosetta.core.pack.task.operation.OperateOnResidueSubset(\n",
        "    pyrosetta.rosetta.core.pack.task.operation.PreventRepackingRLT(), chain_BC_not_interface))\n",
        "\n",
        "# Repack only on chain_BC_interface\n",
        "tf.push_back(pyrosetta.rosetta.core.pack.task.operation.OperateOnResidueSubset(\n",
        "    pyrosetta.rosetta.core.pack.task.operation.RestrictToRepackingRLT(), chain_BC_interface))\n",
        "\n",
        "# Enable design on chain_A\n",
        "aa_to_design = pyrosetta.rosetta.core.pack.task.operation.RestrictAbsentCanonicalAASRLT()\n",
        "aa_to_design.aas_to_keep(\"ACDEFGHIKLMNPQRSTVWY\")\n",
        "tf.push_back(pyrosetta.rosetta.core.pack.task.operation.OperateOnResidueSubset(\n",
        "    aa_to_design, chain_A))\n",
        "\n",
        "# Apply layer design\n",
        "tf.push_back(layer_design_task)\n",
        "\n",
        "# Convert the task factory into a PackerTask\n",
        "packer_task = tf.create_task_and_apply_taskoperations(pose)\n",
        "# View the PackerTask\n",
        "print(packer_task)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:core.pack.task: Packer task: initialize from command line()\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "stream",
          "text": [
            "#Packer_Task\n",
            "\n",
            "Threads to request: ALL AVAILABLE\n",
            "\n",
            "resid\tpack?\tdesign?\tallowed_aas\n",
            "1\tTRUE\tTRUE\tASP:NtermProteinFull,ASN:NtermProteinFull,SER:NtermProteinFull,THR:NtermProteinFull\n",
            "2\tTRUE\tTRUE\tGLU,HIS,HIS_D,LYS,PRO,GLN,ARG\n",
            "3\tTRUE\tTRUE\tGLU,HIS,HIS_D,LYS,GLN,ARG\n",
            "4\tTRUE\tTRUE\tGLU,HIS,HIS_D,LYS,GLN,ARG\n",
            "5\tTRUE\tTRUE\tGLU,HIS,HIS_D,LYS,GLN,ARG\n",
            "6\tTRUE\tTRUE\tGLU,HIS,HIS_D,LYS,GLN,ARG\n",
            "7\tTRUE\tTRUE\tALA,ASP,GLU,ILE,LYS,LEU,MET,ASN,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "8\tTRUE\tTRUE\tALA,ASP,GLU,ILE,LYS,LEU,MET,ASN,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "9\tTRUE\tTRUE\tGLU,HIS,HIS_D,LYS,GLN,ARG\n",
            "10\tTRUE\tTRUE\tGLU,HIS,HIS_D,LYS,GLN,ARG\n",
            "11\tTRUE\tTRUE\tALA,PHE,ILE,LEU,MET,VAL,TRP,TYR\n",
            "12\tTRUE\tTRUE\tGLU,HIS,HIS_D,LYS,GLN,ARG\n",
            "13\tTRUE\tTRUE\tGLU,HIS,HIS_D,LYS,GLN,ARG\n",
            "14\tTRUE\tTRUE\tALA,PHE,ILE,LEU,MET,VAL,TRP,TYR\n",
            "15\tTRUE\tTRUE\tALA,ASP,GLU,ILE,LYS,LEU,MET,ASN,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "16\tTRUE\tTRUE\tGLU,HIS,HIS_D,LYS,GLN,ARG\n",
            "17\tTRUE\tTRUE\tGLU,HIS,HIS_D,LYS,GLN,ARG\n",
            "18\tTRUE\tTRUE\tALA,PHE,ILE,LEU,MET,VAL,TRP,TYR\n",
            "19\tTRUE\tTRUE\tGLU,HIS,HIS_D,LYS,GLN,ARG\n",
            "20\tTRUE\tTRUE\tGLU,HIS,HIS_D,LYS,GLN,ARG\n",
            "21\tTRUE\tTRUE\tALA,PHE,ILE,LEU,MET,VAL,TRP,TYR\n",
            "22\tTRUE\tTRUE\tALA,ASP,GLU,ILE,LYS,LEU,MET,ASN,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "23\tTRUE\tTRUE\tGLU,HIS,HIS_D,LYS,GLN,ARG\n",
            "24\tTRUE\tTRUE\tGLU,HIS,HIS_D,LYS,GLN,ARG\n",
            "25\tTRUE\tTRUE\tALA,PHE,ILE,LEU,MET,VAL,TRP,TYR\n",
            "26\tTRUE\tTRUE\tGLU,HIS,HIS_D,LYS,GLN,ARG\n",
            "27\tTRUE\tTRUE\tGLU,HIS,HIS_D,LYS,GLN,ARG\n",
            "28\tTRUE\tTRUE\tALA,ASP,GLU,ILE,LYS,LEU,MET,ASN,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "29\tTRUE\tTRUE\tALA,ASP,GLU,ILE,LYS,LEU,MET,ASN,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "30\tTRUE\tTRUE\tGLU,HIS,HIS_D,LYS,GLN,ARG\n",
            "31\tTRUE\tTRUE\tGLU,HIS,HIS_D,LYS,GLN,ARG\n",
            "32\tTRUE\tTRUE\tALA,PHE,ILE,LEU,MET,VAL,TRP,TYR\n",
            "33\tTRUE\tTRUE\tASP,GLU,GLY,HIS,HIS_D,LYS,ASN,PRO,GLN,ARG,SER,THR\n",
            "34\tTRUE\tTRUE\tASP,GLU,GLY,HIS,HIS_D,LYS,ASN,PRO,GLN,ARG,SER,THR\n",
            "35\tTRUE\tTRUE\tASP,GLU,GLY,HIS,HIS_D,LYS,ASN,PRO,GLN,ARG,SER,THR\n",
            "36\tTRUE\tTRUE\tASP,GLU,GLY,HIS,HIS_D,LYS,ASN,PRO,GLN,ARG,SER,THR\n",
            "37\tTRUE\tTRUE\tASP,GLU,GLY,HIS,HIS_D,LYS,ASN,PRO,GLN,ARG,SER,THR\n",
            "38\tTRUE\tTRUE\tASP,GLU,GLY,HIS,HIS_D,LYS,ASN,PRO,GLN,ARG,SER,THR\n",
            "39\tTRUE\tTRUE\tASP,ASN,SER,THR\n",
            "40\tTRUE\tTRUE\tALA,ASP,GLU,ILE,LYS,LEU,MET,ASN,PRO,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "41\tTRUE\tTRUE\tALA,ASP,GLU,ILE,LYS,LEU,MET,ASN,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "42\tTRUE\tTRUE\tALA,ASP,GLU,ILE,LYS,LEU,MET,ASN,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "43\tTRUE\tTRUE\tGLU,HIS,HIS_D,LYS,GLN,ARG\n",
            "44\tTRUE\tTRUE\tALA,ASP,GLU,ILE,LYS,LEU,MET,ASN,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "45\tTRUE\tTRUE\tALA,PHE,ILE,LEU,MET,VAL,TRP,TYR\n",
            "46\tTRUE\tTRUE\tGLU,HIS,HIS_D,LYS,GLN,ARG\n",
            "47\tTRUE\tTRUE\tALA,ASP,GLU,ILE,LYS,LEU,MET,ASN,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "48\tTRUE\tTRUE\tALA,ASP,GLU,ILE,LYS,LEU,MET,ASN,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "49\tTRUE\tTRUE\tALA,ASP,GLU,ILE,LYS,LEU,MET,ASN,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "50\tTRUE\tTRUE\tGLU,HIS,HIS_D,LYS,GLN,ARG\n",
            "51\tTRUE\tTRUE\tALA,PHE,ILE,LEU,MET,VAL,TRP,TYR\n",
            "52\tTRUE\tTRUE\tALA,PHE,ILE,LEU,MET,VAL,TRP,TYR\n",
            "53\tTRUE\tTRUE\tGLU,HIS,HIS_D,LYS,GLN,ARG\n",
            "54\tTRUE\tTRUE\tALA,ASP,GLU,ILE,LYS,LEU,MET,ASN,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "55\tTRUE\tTRUE\tALA,PHE,ILE,LEU,MET,VAL,TRP,TYR\n",
            "56\tTRUE\tTRUE\tALA,ASP,GLU,ILE,LYS,LEU,MET,ASN,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "57\tTRUE\tTRUE\tGLU,HIS,HIS_D,LYS,GLN,ARG\n",
            "58\tTRUE\tTRUE\tALA,PHE,ILE,LEU,MET,VAL,TRP,TYR\n",
            "59\tTRUE\tTRUE\tALA,PHE,ILE,LEU,MET,VAL,TRP,TYR\n",
            "60\tTRUE\tTRUE\tGLU,HIS,HIS_D,LYS,GLN,ARG\n",
            "61\tTRUE\tTRUE\tALA,ASP,GLU,ILE,LYS,LEU,MET,ASN,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "62\tTRUE\tTRUE\tALA,ASP,GLU,ILE,LYS,LEU,MET,ASN,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "63\tTRUE\tTRUE\tALA,ASP,GLU,ILE,LYS,LEU,MET,ASN,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "64\tTRUE\tTRUE\tGLU,HIS,HIS_D,LYS,GLN,ARG\n",
            "65\tTRUE\tTRUE\tALA,PHE,ILE,LEU,MET,VAL,TRP,TYR\n",
            "66\tTRUE\tTRUE\tALA,PHE,ILE,LEU,MET,VAL,TRP,TYR\n",
            "67\tTRUE\tTRUE\tGLU,HIS,HIS_D,LYS,GLN,ARG\n",
            "68\tTRUE\tTRUE\tALA,ASP,GLU,ILE,LYS,LEU,MET,ASN,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "69\tTRUE\tTRUE\tALA,ASP,GLU,ILE,LYS,LEU,MET,ASN,GLN,ARG,SER,THR,VAL,TRP,TYR\n",
            "70\tTRUE\tTRUE\tALA:CtermProteinFull,ASP:CtermProteinFull,GLU:CtermProteinFull,PHE:CtermProteinFull,GLY:CtermProteinFull,ILE:CtermProteinFull,LYS:CtermProteinFull,LEU:CtermProteinFull,MET:CtermProteinFull,ASN:CtermProteinFull,PRO:CtermProteinFull,GLN:CtermProteinFull,ARG:CtermProteinFull,SER:CtermProteinFull,THR:CtermProteinFull,VAL:CtermProteinFull,TRP:CtermProteinFull,TYR:CtermProteinFull\n",
            "71\tFALSE\tFALSE\t\n",
            "72\tFALSE\tFALSE\t\n",
            "73\tFALSE\tFALSE\t\n",
            "74\tFALSE\tFALSE\t\n",
            "75\tFALSE\tFALSE\t\n",
            "76\tFALSE\tFALSE\t\n",
            "77\tTRUE\tFALSE\tLEU\n",
            "78\tFALSE\tFALSE\t\n",
            "79\tFALSE\tFALSE\t\n",
            "80\tTRUE\tFALSE\tLEU\n",
            "81\tFALSE\tFALSE\t\n",
            "82\tFALSE\tFALSE\t\n",
            "83\tFALSE\tFALSE\t\n",
            "84\tTRUE\tFALSE\tLEU\n",
            "85\tFALSE\tFALSE\t\n",
            "86\tFALSE\tFALSE\t\n",
            "87\tTRUE\tFALSE\tLEU\n",
            "88\tTRUE\tFALSE\tARG\n",
            "89\tFALSE\tFALSE\t\n",
            "90\tFALSE\tFALSE\t\n",
            "91\tTRUE\tFALSE\tLEU\n",
            "92\tFALSE\tFALSE\t\n",
            "93\tFALSE\tFALSE\t\n",
            "94\tTRUE\tFALSE\tLEU\n",
            "95\tTRUE\tFALSE\tLYS\n",
            "96\tFALSE\tFALSE\t\n",
            "97\tFALSE\tFALSE\t\n",
            "98\tTRUE\tFALSE\tLEU\n",
            "99\tFALSE\tFALSE\t\n",
            "100\tFALSE\tFALSE\t\n",
            "101\tTRUE\tFALSE\tLEU\n",
            "102\tTRUE\tFALSE\tGLU\n",
            "103\tFALSE\tFALSE\t\n",
            "104\tFALSE\tFALSE\t\n",
            "105\tTRUE\tFALSE\tPRO\n",
            "106\tFALSE\tFALSE\t\n",
            "107\tFALSE\tFALSE\t\n",
            "108\tFALSE\tFALSE\t\n",
            "109\tFALSE\tFALSE\t\n",
            "110\tTRUE\tFALSE\tILE\n",
            "111\tFALSE\tFALSE\t\n",
            "112\tFALSE\tFALSE\t\n",
            "113\tTRUE\tFALSE\tVAL\n",
            "114\tFALSE\tFALSE\t\n",
            "115\tFALSE\tFALSE\t\n",
            "116\tFALSE\tFALSE\t\n",
            "117\tTRUE\tFALSE\tILE\n",
            "118\tFALSE\tFALSE\t\n",
            "119\tFALSE\tFALSE\t\n",
            "120\tTRUE\tFALSE\tALA\n",
            "121\tFALSE\tFALSE\t\n",
            "122\tFALSE\tFALSE\t\n",
            "123\tTRUE\tFALSE\tALA\n",
            "124\tFALSE\tFALSE\t\n",
            "125\tFALSE\tFALSE\t\n",
            "126\tFALSE\tFALSE\t\n",
            "127\tFALSE\tFALSE\t\n",
            "128\tFALSE\tFALSE\t\n",
            "129\tFALSE\tFALSE\t\n",
            "130\tFALSE\tFALSE\t\n",
            "131\tTRUE\tFALSE\tSER\n",
            "132\tFALSE\tFALSE\t\n",
            "133\tFALSE\tFALSE\t\n",
            "134\tFALSE\tFALSE\t\n",
            "135\tFALSE\tFALSE\t\n",
            "136\tFALSE\tFALSE\t\n",
            "137\tTRUE\tFALSE\tALA\n",
            "138\tTRUE\tFALSE\tLEU\n",
            "139\tFALSE\tFALSE\t\n",
            "140\tFALSE\tFALSE\t\n",
            "141\tFALSE\tFALSE\t\n",
            "142\tFALSE\tFALSE\t\n",
            "143\tFALSE\tFALSE\t\n",
            "144\tFALSE\tFALSE\t\n",
            "145\tFALSE\tFALSE\t\n",
            "146\tFALSE\tFALSE\t\n",
            "147\tFALSE\tFALSE\t\n",
            "148\tFALSE\tFALSE\t\n",
            "149\tFALSE\tFALSE\t\n",
            "150\tFALSE\tFALSE\t\n",
            "151\tFALSE\tFALSE\t\n",
            "152\tFALSE\tFALSE\t\n",
            "153\tFALSE\tFALSE\t\n",
            "154\tFALSE\tFALSE\t\n",
            "155\tFALSE\tFALSE\t\n",
            "156\tFALSE\tFALSE\t\n",
            "157\tFALSE\tFALSE\t\n",
            "158\tFALSE\tFALSE\t\n",
            "159\tFALSE\tFALSE\t\n",
            "160\tFALSE\tFALSE\t\n",
            "161\tFALSE\tFALSE\t\n",
            "162\tFALSE\tFALSE\t\n",
            "163\tFALSE\tFALSE\t\n",
            "164\tFALSE\tFALSE\t\n",
            "165\tFALSE\tFALSE\t\n",
            "166\tFALSE\tFALSE\t\n",
            "167\tFALSE\tFALSE\t\n",
            "168\tFALSE\tFALSE\t\n",
            "169\tFALSE\tFALSE\t\n",
            "170\tFALSE\tFALSE\t\n",
            "171\tFALSE\tFALSE\t\n",
            "172\tFALSE\tFALSE\t\n",
            "173\tFALSE\tFALSE\t\n",
            "174\tFALSE\tFALSE\t\n",
            "175\tFALSE\tFALSE\t\n",
            "176\tFALSE\tFALSE\t\n",
            "177\tFALSE\tFALSE\t\n",
            "178\tFALSE\tFALSE\t\n",
            "179\tFALSE\tFALSE\t\n",
            "180\tFALSE\tFALSE\t\n",
            "181\tTRUE\tFALSE\tILE\n",
            "182\tTRUE\tFALSE\tVAL\n",
            "183\tFALSE\tFALSE\t\n",
            "184\tFALSE\tFALSE\t\n",
            "185\tTRUE\tFALSE\tLEU\n",
            "186\tTRUE\tFALSE\tLYS\n",
            "187\tFALSE\tFALSE\t\n",
            "188\tTRUE\tFALSE\tILE\n",
            "189\tTRUE\tFALSE\tVAL\n",
            "190\tFALSE\tFALSE\t\n",
            "191\tFALSE\tFALSE\t\n",
            "192\tTRUE\tFALSE\tILE\n",
            "193\tTRUE\tFALSE\tGLU\n",
            "194\tFALSE\tFALSE\t\n",
            "195\tFALSE\tFALSE\t\n",
            "196\tTRUE\tFALSE\tVAL\n",
            "197\tFALSE\tFALSE\t\n",
            "198\tFALSE\tFALSE\t\n",
            "199\tFALSE\tFALSE\t\n",
            "200\tTRUE\tFALSE\tARG\n",
            "201\tFALSE\tFALSE\t\n",
            "202\tTRUE\tFALSE\tSER\n",
            "203\tTRUE\tFALSE\tALA\n",
            "204\tFALSE\tFALSE\t\n",
            "205\tFALSE\tFALSE\t\n",
            "206\tFALSE\tFALSE\t\n",
            "207\tTRUE\tFALSE\tLYS\n",
            "208\tFALSE\tFALSE\t\n",
            "209\tTRUE\tFALSE\tLEU\n",
            "210\tTRUE\tFALSE\tVRT\n",
            "\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "LzoYV-kHmdlc"
      },
      "source": [
        "The PackerTask looks as intended. Now setup the `MoveMapFactory`:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "EYH4pKJimdlc"
      },
      "source": [
        "# Set up a MoveMapFactory\n",
        "mmf = pyrosetta.rosetta.core.select.movemap.MoveMapFactory()\n",
        "mmf.all_bb(setting=False)  # Set to true if needed\n",
        "mmf.all_bondangles(setting=False)\n",
        "mmf.all_bondlengths(setting=False)\n",
        "mmf.all_chi(setting=True)\n",
        "mmf.all_jumps(setting=False)\n",
        "mmf.set_cartesian(setting=False)\n",
        "\n",
        "# Set movemap actions to turn on or off certain torsions, overriding the above defaults\n",
        "enable = pyrosetta.rosetta.core.select.movemap.move_map_action.mm_enable\n",
        "disable = pyrosetta.rosetta.core.select.movemap.move_map_action.mm_disable\n",
        "\n",
        "# Set custom minimizable torsions\n",
        "mmf.add_bondangles_action(action=enable, selector=chain_A_and_BC_interface)\n",
        "mmf.add_bondlengths_action(action=enable, selector=chain_A_and_BC_interface)\n",
        "mmf.add_chi_action(action=enable, selector=chain_A_and_BC_interface)\n",
        "\n",
        "mmf.add_bondangles_action(action=disable, selector=chain_BC_not_interface)\n",
        "mmf.add_bondlengths_action(action=disable, selector=chain_BC_not_interface)\n",
        "mmf.add_chi_action(action=disable, selector=chain_BC_not_interface)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "NM3K1tHkmdlc"
      },
      "source": [
        " Now let's double-check some more `pose` information to verify that we are ready for `FastDesign`:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "TQmyCTNOmdlc",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "4f423741-a715-40f6-aaba-ecb4f585407f"
      },
      "source": [
        "display_pose = pyrosetta.rosetta.protocols.fold_from_loops.movers.DisplayPoseLabelsMover()\n",
        "display_pose.tasks(tf)\n",
        "display_pose.movemap_factory(mmf)\n",
        "display_pose.apply(pose)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:protocols.DsspMover: LHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHLLLLLLLHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHLLLHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHLLLLHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHLLHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHLLLLHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHLL\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover: SEQUENCE       AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAYKIKETLKRLEDSLRELRRILEELKEMLERLEKNPDKDVIVEVLKVIVKAIEASVENQRISAENQKALATKYKIKETLKRLEDSLRELRRILEELKEMLERLEKNPDKDVIVEVLKVIVKAIEASVENQRISAENQKALX\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover: STRUCTURE      LHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHLLLLLLLHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHLLLHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHLLLLHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHLLHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHLLLLHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHLL\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover: MVMP_BB        ..................................................................................................................................................................................................................\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover: MVMP_CHI       **********************************************************************..*...*..*...*..**..*..**..*..**..*....*..*...*..*..**..*...*..*..**.......................................**.**..**.**..**.**..**.**..**.**\n",
            "INFO:rosetta:core.pack.task: Packer task: initialize from command line()\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover: DESIGN         **********************************************************************--------------------------------------------------------------------------------------------------------------------------------------------\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover: REPACK         **********************************************************************------*--*---*--**--*--**--*--**--*----*--*---*--*--*-------*-----**------------------------------------------**--**-**--**--*---*-**---*-**\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover: FOLDTREE       1********************************************************************C1****************************************************2************************************************************************************C2/C\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover:\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover: FOLD_TREE  EDGE 210 124 2  EDGE 124 71 -1  EDGE 124 209 -1  EDGE 71 1 1  EDGE 1 70 -1\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover: A[ALA:NtermProteinFull]AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA[ALA:CtermProteinFull]Y[TYR:NtermProteinFull]KIKETLKRLEDSLRELRRILEELKEMLERLEKNPDKDVIVEVLKVIVKAIEASVENQRISAENQKALA[ALA:CtermProteinFull]T[THR:NtermProteinFull]KYKIKETLKRLEDSLRELRRILEELKEMLERLEKNPDKDVIVEVLKVIVKAIEASVENQRISAENQKALX\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover: PDB file name:\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover:  Pose Range  Chain    PDB Range  |   #Residues         #Atoms\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover: 0001 -- 0070    A 0001  -- 0070  |   0070 residues;    00703 atoms\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover: 0071 -- 0139    B 0071  -- 0139  |   0069 residues;    01187 atoms\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover: 0140 -- 0209    C 0140  -- 0209  |   0070 residues;    01212 atoms\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover: 0210 -- 0210    z 0001  -- 0001  |   0001 residues;    00003 atoms\n",
            "INFO:rosetta:protocols.fold_from_loops.DisplayPoseLabelsMover:                            TOTAL |   0210 residues;    03105 atoms\n"
          ],
          "name": "stderr"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "7TaFdRytmdlc"
      },
      "source": [
        " We are ready to setup the `FastRelax` mover:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "rnvMp0YZmdlc",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "6a87c33f-8695-430c-9440-457dce8ae194"
      },
      "source": [
        "#fast_design = pyrosetta.rosetta.protocols.denovo_design.movers.FastDesign(scorefxn_in=design_scorefxn, standard_repeats=1) # 2-5 repeats suggested for real applications\n",
        "fast_design = pyrosetta.rosetta.protocols.relax.FastRelax(scorefxn_in=design_scorefxn, standard_repeats=1, script_file=\"KillA2019\")\n",
        "fast_design.cartesian(False)\n",
        "fast_design.set_task_factory(tf)\n",
        "fast_design.set_movemap_factory(mmf)\n",
        "fast_design.min_type(\"lbfgs_armijo_nonmonotone\")\n",
        "fast_design.ramp_down_constraints(False)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:protocols.relax.FastRelax: KillA2019 has been renamed to PolarDesign2019\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: score12 : 116.271\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: talaris2013 : 109.417\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: talaris2014 : 109.347\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: ref2015 : 106.875\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: beta_nov16 : 114.938\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: Looking for PolarDesign2019.ref2015.txt\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: ================== Reading script file: /usr/local/lib/python3.7/dist-packages/pyrosetta/database/sampling/relax_scripts/PolarDesign2019.ref2015.txt ==================\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: repeat %%nrepeats%%\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: reference -0.511909 2.0282 -3.29233 -3.74112 2.5917 -1.12843 -1.33724 3.25715 -1.56117 2.65488 2.88076 -2.80685 -1.5498 -2.69754 -1.62133 -1.86628 -0.0648395 2.6561 4.7344 1.37564\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: coord_cst_weight 1.0\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: scale:fa_rep 0.059\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: repack\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: scale:fa_rep 0.092\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: min 0.01\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: reference 0.335091 2.4652 -2.85533 -3.29412 1.6487 -1.27143 -1.07024 2.80415 -1.20417 2.08188 2.06776 -2.26985 -1.5128 -2.21054 -1.15433 -1.25928 0.392161 2.6131 2.9414 0.752641\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: coord_cst_weight 0.5\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: scale:fa_rep 0.280\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: repack\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: scale:fa_rep 0.323\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: min 0.01\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: reference 0.742591 2.7127 -2.62783 -3.12662 1.1962 -0.823929 -0.992739 2.55165 -1.09667 1.83938 1.72526 -2.04235 -1.6253 -2.10304 -1.06683 -0.951779 0.589661 2.6206 2.0389 0.440141\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: coord_cst_weight 0.0\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: scale:fa_rep 0.568\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: repack\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: scale:fa_rep 0.633\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: min 0.01\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: reset_reference\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: coord_cst_weight 0.0\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: scale:fa_rep 1\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: repack\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: min 0.00001\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: accept_to_best\n",
            "INFO:rosetta:protocols.relax.RelaxScriptManager: endrepeat\n"
          ],
          "name": "stderr"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "YqOEGsq7mdld"
      },
      "source": [
        "Note that this takes ~37min 13s:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "r-YaPIApmdld",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "467f70f0-b86f-4503-e2fe-33bcbd5665c4"
      },
      "source": [
        "if not os.getenv(\"DEBUG\"):\n",
        "    %time fast_design.apply(pose)\n",
        "    # Optionally, instead of running this you could reload the saved pose from a previously run trajectory:\n",
        "    pose = pyrosetta.pose_from_file(\"expected_outputs/designed_pose.pdb\")"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:rosetta:core.energy_methods.CartesianBondedEnergy: Creating new peptide-bonded energy container (210)\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: repeat  19150.4  0  0  0.55\n",
            "INFO:rosetta:basic.io.database: Database file opened: scoring/score_functions/aa_repeat_energy/default_repeat_penalty_table.rpt_pen\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: reference  18948.4  0  0  0.55\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: coord_cst_weight  18948.4  0  0  0.55\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: scale:fa_rep  18867.8  0  0  0.03245\n",
            "INFO:rosetta:core.pack.task: Packer task: initialize from command line()\n",
            "INFO:rosetta:core.pack.pack_rotamers: built 11290 rotamers at 105 positions.\n",
            "INFO:rosetta:core.pack.interaction_graph.interaction_graph_factory: Instantiating PDInteractionGraph\n",
            "INFO:rosetta:protocols.relax.FastRelax: CMD: repack  2534.98  0  0  0.03245\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "stream",
          "text": [
            "CPU times: user 28min 38s, sys: 7.14 s, total: 28min 45s\n",
            "Wall time: 28min 40s\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "error",
          "ename": "RuntimeError",
          "evalue": "ignored",
          "traceback": [
            "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
            "\u001b[0;31mRuntimeError\u001b[0m                              Traceback (most recent call last)",
            "\u001b[0;32m<ipython-input-99-1d144a6699ab>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      2\u001b[0m     \u001b[0mget_ipython\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mmagic\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'time fast_design.apply(pose)'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m     \u001b[0;31m# Optionally, instead of running this you could reload the saved pose from a previously run trajectory:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 4\u001b[0;31m     \u001b[0mpose\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mpyrosetta\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mpose_from_file\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"expected_outputs/designed_pose.pdb\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
            "\u001b[0;31mRuntimeError\u001b[0m: \n\nFile: /home/benchmark/rosetta/source/src/core/import_pose/import_pose.cc:362\n[ ERROR ] UtilityExitException\nERROR: Cannot open file \"expected_outputs/designed_pose.pdb\"\n\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "2iA-3I6-mdld"
      },
      "source": [
        " Save this pose for downstream reference:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "MBVvoaohmdld"
      },
      "source": [
        "if not os.getenv(\"DEBUG\"):\n",
        "    designed_pose = pose.clone()\n",
        "    #pyrosetta.dump_pdb(designed_pose, \"outputs/designed_pose.pdb\")"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "XCV7xghCmdle"
      },
      "source": [
        " Now that we have re-designed chain A, it is strongly recommended to use `FastRelax` with a Cartesian scorefunction to repack and minimize with a realistic scorefuction. Note: this takes ~6min 27s"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "VtoJ2xLgmdle"
      },
      "source": [
        "tf.clear()\n",
        "tf = pyrosetta.rosetta.core.pack.task.TaskFactory()\n",
        "tf.push_back(pyrosetta.rosetta.core.pack.task.operation.InitializeFromCommandline())\n",
        "tf.push_back(pyrosetta.rosetta.core.pack.task.operation.IncludeCurrent())\n",
        "tf.push_back(pyrosetta.rosetta.core.pack.task.operation.NoRepackDisulfides())\n",
        "tf.push_back(pyrosetta.rosetta.core.pack.task.operation.OperateOnResidueSubset(\n",
        "    pyrosetta.rosetta.core.pack.task.operation.RestrictToRepackingRLT(), true_selector))\n",
        "mm = pyrosetta.rosetta.core.kinematics.MoveMap()\n",
        "mm.set_bb(False) # Set to true if desired\n",
        "mm.set_chi(True)\n",
        "mm.set_jump(False)\n",
        "fast_relax = pyrosetta.rosetta.protocols.relax.FastRelax(scorefxn_in=relax_scorefxn, standard_repeats=1) # 2-5 repeats suggested for real applications\n",
        "fast_relax.cartesian(True)\n",
        "fast_relax.set_task_factory(tf)\n",
        "fast_relax.set_movemap(mm)\n",
        "fast_relax.minimize_bond_angles(True)\n",
        "fast_relax.minimize_bond_lengths(True)\n",
        "fast_relax.min_type(\"lbfgs_armijo_nonmonotone\") # Cartisian scorefunction\n",
        "fast_relax.ramp_down_constraints(False)\n",
        "\n",
        "# To run the FastRelax trajectory.\n",
        "if not os.getenv(\"DEBUG\"):\n",
        "    %time fast_relax.apply(pose)\n",
        "\n",
        "#Or for speed, we will load the pose from a previous trajectory.\n",
        "pose = pyrosetta.pose_from_file(\"expected_outputs/designed_relaxed_pose.pdb\")"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "ZHg1gbx9mdle"
      },
      "source": [
        " Save the pose for downstream analysis:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "8RAyfhoGmdle"
      },
      "source": [
        "designed_relaxed_pose = pose.clone()\n",
        "pyrosetta.dump_pdb(designed_relaxed_pose, \"expected_outputs/designed_relaxed_pose.pdb\")"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "YJ2BDSuXmdlf"
      },
      "source": [
        " Let's compare sequences to see what happened:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "6idXujAWmdlf"
      },
      "source": [
        "print(minimized_start_pose.chain_sequence(1))\n",
        "print(designed_relaxed_pose.chain_sequence(1))"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "m8Nbk7NVmdlf"
      },
      "source": [
        "View the resulting design!"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "_Z3tq7w_mdlf"
      },
      "source": [
        "chA = pyrosetta.rosetta.core.select.residue_selector.ChainSelector(\"A\")\n",
        "chB = pyrosetta.rosetta.core.select.residue_selector.ChainSelector(\"B\")\n",
        "chC = pyrosetta.rosetta.core.select.residue_selector.ChainSelector(\"C\")\n",
        "view = sum(\n",
        "    [\n",
        "        viewer.init(designed_relaxed_pose),\n",
        "        viewer.setStyle(cartoon_color=\"lightgrey\", radius=0.25),\n",
        "        viewer.setSurface(residue_selector=chA, colorscheme=\"orangeCarbon\", opacity=0.5, surface_type=\"VDW\"),\n",
        "        viewer.setSurface(residue_selector=chB, color=\"greenCarbon\", opacity=0.5, surface_type=\"VDW\"),\n",
        "        viewer.setSurface(residue_selector=chB, color=\"violetCarbon\", opacity=0.5, surface_type=\"VDW\"),\n",
        "        viewer.setDisulfides(radius=0.25),\n",
        "        viewer.setZoom(factor=1.5)\n",
        "    ]\n",
        ")\n",
        "view()"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "DlsxZh65mdlf"
      },
      "source": [
        "## Analysis:"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "528c6G7pmdlf"
      },
      "source": [
        " By how many Angstroms RMSD did the backbone C atoms move?"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "HiWEw_Kxmdlf"
      },
      "source": [
        "\n"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "UILAOCd2mdlf"
      },
      "source": [
        ""
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "8E-SvKGumdlg"
      },
      "source": [
        " What is the delta `total_score` from `minimized_start_pose` to `designed_relaxed_pose`?"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "nMut8RHCmdlg"
      },
      "source": [
        ""
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "GixVIo3emdlg"
      },
      "source": [
        " What is the per-residue energy difference for each mutated position between `minimized_start_pose` and `designed_relaxed_pose`?"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "EOmc2UeJmdlg"
      },
      "source": [
        ""
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "sL6I4pDPmdlg"
      },
      "source": [
        " Are the sequence constraints imposed by the `aa_repeat` scoreterm satisfied? Re-write the following python code that counts the number of residue types in chain A to check for the longest stretch of each residue type in the primary amino acid sequence in chain A:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "INlDwYIemdlg"
      },
      "source": [
        "for aa in IUPACData.protein_letters:\n",
        "    aa_selector = pyrosetta.rosetta.core.select.residue_selector.ResidueNameSelector(str.upper(Bio.SeqUtils.seq3(aa)))\n",
        "    aa_and_chain_A = pyrosetta.rosetta.core.select.residue_selector.AndResidueSelector(chain_A, aa_selector)\n",
        "    sel_res_count_metric = pyrosetta.rosetta.core.simple_metrics.metrics.SelectedResidueCountMetric()\n",
        "    sel_res_count_metric.set_residue_selector(aa_and_chain_A)\n",
        "    print(aa, int(sel_res_count_metric.calculate(designed_relaxed_pose)))"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "VI4yO69omdlg"
      },
      "source": [
        " Does chain A have 40% percent aromatic or aliphatic (but not leucine) and 5% leucine, satisfying the `aa_composition` scoreterm? For additional practice, re-write the following python implementation using PyRosetta ResidueSelectors and SimpleMetrics:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "md61jdRzmdlg"
      },
      "source": [
        "num_aro_ali = 0\n",
        "num_leucine = 0\n",
        "\n",
        "for aa in pose.chain_sequence(1):\n",
        "    if aa in \"WYFAVIMP\":\n",
        "        num_aro_ali += 1\n",
        "    if aa == \"L\":\n",
        "        num_leucine += 1\n",
        "        \n",
        "print(\"The % aromatic residues in chain A is {0}%\".format((num_aro_ali*100)/len(pose.chain_sequence(1))))\n",
        "print(\"The % leucine in chain A is {0}%\".format((num_leucine*100)/len(pose.chain_sequence(1))))"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "Xzq492h_mdlh"
      },
      "source": [
        " Are the sequence constraints imposed by the `AddHelixSequenceConstraints` mover with the `aa_composition` scoreterm satisfied?"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "BvPKwTnMmdlh"
      },
      "source": [
        " Uses sasa (solvent-accessible surface area) to asses whether there are there more or less voids in `designed_relaxed_pose` as compared to `minimized_start_pose`, satisfying the `voids_penalty` scoreterm."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "chXCQnzdmdlh"
      },
      "source": [
        "tf.clear()\n",
        "tf = pyrosetta.rosetta.core.pack.task.TaskFactory()\n",
        "tf.push_back(pyrosetta.rosetta.core.pack.task.operation.OperateOnResidueSubset(\n",
        "    pyrosetta.rosetta.core.pack.task.operation.RestrictToRepackingRLT(), true_selector))\n",
        "sasa = pyrosetta.rosetta.protocols.simple_filters.TaskAwareSASAFilter()\n",
        "sasa.task_factory(tf)\n",
        "print(\"The sasa of minimized_start_pose is {}\".format(sasa.score(minimized_start_pose)))\n",
        "print(\"The sasa of designed_relaxed_pose is {}\".format(sasa.score(designed_relaxed_pose)))"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "0gd1GRvsmdlh"
      },
      "source": [
        " Is the net charge of chain A equal to exactly zero, satisfying the `netcharge` scoreterm?"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "f0Hh_Ofrmdlh"
      },
      "source": [
        "# One method to calculate net charge of chain A\n",
        "num_negative = 0\n",
        "num_positive = 0\n",
        "for aa in pose.chain_sequence(1):\n",
        "    if aa in \"DE\":\n",
        "        num_negative += 1\n",
        "    if aa in \"KR\":\n",
        "        num_positive += 1\n",
        "print(\"The net charge of chain A = {0}\".format(num_positive - num_negative))\n",
        "\n",
        "# Another method to calculate net charge of chain A\n",
        "test_pose = pose.clone()\n",
        "keep_chA = pyrosetta.rosetta.protocols.grafting.simple_movers.KeepRegionMover(res_start=str(test_pose.chain_begin(1)), res_end=str(test_pose.chain_end(1)))\n",
        "keep_chA.apply(test_pose)\n",
        "net_charge = pyrosetta.rosetta.protocols.simple_filters.NetChargeFilter()\n",
        "print(\"The net charge of chain A = {0}\".format(net_charge.score(test_pose)))"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "6jwBBx6Cmdlh"
      },
      "source": [
        " Are there any buried unsatisfied polar atoms, satisfying the `buried_unsatisfied_penalty` scoreterm?"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "Oc54zIWWmdlh"
      },
      "source": [
        "uhb = pyrosetta.rosetta.protocols.rosetta_scripts.XmlObjects.create_from_string(\"\"\"\n",
        "<SCOREFXNS>\n",
        "  <ScoreFunction name=\"fa_default\" weights=\"ref2015\"/>\n",
        "</SCOREFXNS>\n",
        "<FILTERS>\n",
        "  <BuriedUnsatHbonds name=\"uhb_sc\" use_reporter_behavior=\"true\" use_hbnet_behavior=\"false\" scorefxn=\"fa_default\" report_bb_heavy_atom_unsats=\"false\" report_sc_heavy_atom_unsats=\"true\" cutoff=\"99999\" print_out_info_to_pdb=\"false\" ignore_surface_res=\"true\" residue_surface_cutoff=\"20.0\" ignore_bb_heavy_unsats=\"false\" confidence=\"0\"/>\n",
        "  <BuriedUnsatHbonds name=\"uhb_bb\" use_reporter_behavior=\"true\" use_hbnet_behavior=\"false\" scorefxn=\"fa_default\" report_bb_heavy_atom_unsats=\"true\" report_sc_heavy_atom_unsats=\"false\" cutoff=\"99999\" print_out_info_to_pdb=\"false\" ignore_surface_res=\"true\" residue_surface_cutoff=\"20.0\" ignore_bb_heavy_unsats=\"false\" confidence=\"0\"/>\n",
        "</FILTERS>\n",
        "\"\"\")"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "x3aunI7Hmdli"
      },
      "source": [
        "print(\"The number of unsatisfied side-chain heavy atoms in minimized_start_pose is {}\".format(uhb.get_filter(\"uhb_sc\").score(minimized_start_pose)))\n",
        "print(\"The number of unsatisfied backbone heavy atoms in minimized_start_pose is {}\".format(uhb.get_filter(\"uhb_bb\").score(minimized_start_pose)))\n",
        "\n",
        "print(\"The number of unsatisfied side-chain heavy atoms in designed_relaxed_pose is {}\".format(uhb.get_filter(\"uhb_sc\").score(designed_relaxed_pose)))\n",
        "print(\"The number of unsatisfied backbone heavy atoms in designed_relaxed_pose is {}\".format(uhb.get_filter(\"uhb_bb\").score(designed_relaxed_pose)))"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "d_buaJ5Hmdli"
      },
      "source": [
        " Inspect the tracer output from the `BuriedUnsatHbonds` filter, and inspect `designed_relaxed_pose` very closely in PyMol or py3Dmol. Would you agree with the `BuriedUnsatHbonds` filter? How does the number of buried unsatisfied heavy atoms compare to `minimized_start_pose`?"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "7JQ5SA3qmdli"
      },
      "source": [
        " Packer results are stochastic based on a random number generator, that is after running `pyrosetta.init()` you see:\n",
        ">rosetta:core.init.random: RandomGenerator:init: Normal mode, seed=937978431 RG_type=mt19937\n",
        "\n",
        "How do your results compare with your neighbors' results? Ideally you would run the same protocol hundreds of times, and filter them down using Rosetta filters that recaptiulate your design requirements to a number of designs that you can then experimentally validate to answer your biological question."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "sqe3TXxgmdli"
      },
      "source": [
        "# See Also\n",
        "\n",
        "Note these may not be available in PyRosetta through code or even by xml (remodel), but they are extremely useful tools when doing denovo protein design - and you should be aware of them.\n",
        "\n",
        "- **RosettaRemodel**\n",
        " - https://www.rosettacommons.org/docs/latest/application_documentation/design/rosettaremodel\n",
        "    \n",
        "    \n",
        "- **Sewing**\n",
        "\n",
        " - https://www.rosettacommons.org/docs/latest/scripting_documentation/RosettaScripts/composite_protocols/sewing/SEWING\n",
        " \n",
        " \n",
        "-  **Parametric Design**\n",
        " - Previous Workshop!"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "tfY7I5ms8Rvf"
      },
      "source": [
        "# **Point Mutation Scan**\n",
        "\n",
        "Keywords: FastRelax, ResidueSelector, NeighborhoodResidueSelector, TaskFactory, TaskOperation, RestrictToRepackingRLT, RestrictAbsentCanonicalAASRLT, NoRepackDisulfides, OperateOnResidueSubset, RigidBodyTransMover, ggplot2\n",
        "\n",
        "## Overview\n",
        "The purpose of this section is to create a protocol using what you have learned in the previous sections.  In this tutorial, we will prepare an antibody-antigen bound structure, make point mutations on the antibody, records changes in binding enregy, and generate a heatmap demonstrating energy differences. This method is widely used in antibody interface design for either improving binding affinity or expanding the binding range. \n",
        "\n",
        "\n",
        "\n",
        "## Protocol\n",
        "The whole protocol is broken down by eight steps.\n",
        "\n",
        "**Step 1.** Prepare the structure with FastRelax()\n",
        "\n",
        "**Step 2.** Write the function to perform the mutation PackMover()\n",
        "\n",
        "**Step 3.** Write the function to unbind the antibody-antigen bound structure unbind()\n",
        "\n",
        "**Step 4.** Write the function to get wildtype amino acid\n",
        "\n",
        "**Step 5.** Write the function to properly mutate and pack a specific residue and output energy metrics\n",
        "\n",
        "**Step 6.** Loop through interface positions mutating them into 20 amino acids with output files\n",
        "\n",
        "**Step 7.** Summarize all input files for binding energy analysis\n",
        "\n",
        "\n"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "i7Ft-iITfib-"
      },
      "source": [
        "## **Section Contributors:**\n",
        "Yuanhan Wu (The Wistar Institute)\n",
        "\n",
        "Daniel Kulp (The Wistar Institute)\n",
        "\n",
        "Jared Adolf-Bryfogle (Scripps; Institute for Protein Innovation)\n",
        "\n",
        "Ajasja Ljubeti (University of Washington)"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "JkMqBiLZ-i7K"
      },
      "source": [
        "### Loading structures "
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "CUe-0OB78Rv0"
      },
      "source": [
        "from pyrosetta import * \n",
        "init()\n",
        "pose = pose_from_pdb(\"inputs/1jhl.clean.pdb\")\n",
        "testPose = Pose()\n",
        "testPose.assign(pose)\n",
        "print(testPose)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "qTi7EVQn8Rv6"
      },
      "source": [
        "## **Step 1. Prepare the starting structure with FastRelax()**\n",
        "\n",
        "Properly relaxing a structure is crucial in design with Rosetta. A non relaxed structure may not overcome bad global energy well and therefore skew the following analysis on binding energy.\n",
        "\n",
        "A FastRelax() is used to relax the structrue. While we want to put the sturcture in its lowest energy state, we want to keep the backbone information from the crystal structure as much as possible (lowest RMSD). Therefore, we apply constrain_relax_to_start_coords(True) to FastRelax().\n",
        "\n",
        "Since FastRelax() is taking up a huge amount of resource, running it seems to crash the notebook, we commented out the \"apply\" part (the part that perform the relax) and print out the relaxMover() instead. We uploaded the relaxed structure to the input folder for furthre analysis.\n",
        "\n",
        "Coordinate constrained relax is a general prep.  For a more advanced version of this preparation method, see the [ pareto-optimal method by Nivon, Morreti, and Baker ](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0059004)\n",
        "\n"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "Fajoeayb8Rv7"
      },
      "source": [
        "from pyrosetta.rosetta.protocols.relax import FastRelax\n",
        "\n",
        "relax = FastRelax()\n",
        "scorefxn = get_fa_scorefxn()\n",
        "relax.set_scorefxn(scorefxn)\n",
        "relax = rosetta.protocols.relax.FastRelax()\n",
        "relax.constrain_relax_to_start_coords(True)\n",
        "print(relax)\n",
        "\n",
        "\n",
        "%time relax.apply(testPose)\n",
        "\n",
        "testPose.dump_pdb('test.relax.pdb')"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "-ZuDYWJECc3p"
      },
      "source": [
        "### Writing Function in Python\n",
        "\n",
        "Functions are good ways to organize your code. Starting from this section I am introducing serveral functions to facilitate the protocol.\n",
        "\n",
        "To define a function in python, a \"def\" key word is used. A function can either returns a value or simply executing code blocks. A defined function can be called in main function or other sections of code too."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "C5z5OqAc8RwP"
      },
      "source": [
        "## **Step 2. Write the function for mutation**\n",
        "\n",
        "This function utilizes the **ResidueSelectorMover()** as demonstrated by previous tutorials. Mutated position is allowed to be designed and repacked while the neighborhood residues are limited to repacked only. The final mutation will be performed with a **PackMover()**."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "VRirfC0d8RwQ"
      },
      "source": [
        "from pyrosetta.rosetta.core.pack.task import *\n",
        "from pyrosetta.rosetta.protocols import *\n",
        "from pyrosetta.rosetta.core.select import *\n",
        "\n",
        "def pack(pose, posi, amino, scorefxn):\n",
        "\n",
        "    # Select Mutate Position\n",
        "    mut_posi = pyrosetta.rosetta.core.select.residue_selector.ResidueIndexSelector()\n",
        "    mut_posi.set_index(posi)\n",
        "    #print(pyrosetta.rosetta.core.select.get_residues_from_subset(mut_posi.apply(pose)))\n",
        "\n",
        "    # Select Neighbor Position\n",
        "    nbr_selector = pyrosetta.rosetta.core.select.residue_selector.NeighborhoodResidueSelector()\n",
        "    nbr_selector.set_focus_selector(mut_posi)\n",
        "    nbr_selector.set_include_focus_in_subset(True)\n",
        "    #print(pyrosetta.rosetta.core.select.get_residues_from_subset(nbr_selector.apply(pose)))\n",
        "\n",
        "    # Select No Design Area\n",
        "    not_design = pyrosetta.rosetta.core.select.residue_selector.NotResidueSelector(mut_posi)\n",
        "    #print(pyrosetta.rosetta.core.select.get_residues_from_subset(not_design.apply(pose)))\n",
        "\n",
        "    # The task factory accepts all the task operations\n",
        "    tf = pyrosetta.rosetta.core.pack.task.TaskFactory()\n",
        "\n",
        "    # These are pretty standard\n",
        "    tf.push_back(pyrosetta.rosetta.core.pack.task.operation.InitializeFromCommandline())\n",
        "    tf.push_back(pyrosetta.rosetta.core.pack.task.operation.IncludeCurrent())\n",
        "    tf.push_back(pyrosetta.rosetta.core.pack.task.operation.NoRepackDisulfides())\n",
        "\n",
        "    # Disable Packing\n",
        "    prevent_repacking_rlt = pyrosetta.rosetta.core.pack.task.operation.PreventRepackingRLT()\n",
        "    prevent_subset_repacking = pyrosetta.rosetta.core.pack.task.operation.OperateOnResidueSubset(prevent_repacking_rlt, nbr_selector, True )\n",
        "    tf.push_back(prevent_subset_repacking)\n",
        "\n",
        "    # Disable design\n",
        "    tf.push_back(pyrosetta.rosetta.core.pack.task.operation.OperateOnResidueSubset(\n",
        "        pyrosetta.rosetta.core.pack.task.operation.RestrictToRepackingRLT(),not_design))\n",
        "\n",
        "    # Enable design\n",
        "    aa_to_design = pyrosetta.rosetta.core.pack.task.operation.RestrictAbsentCanonicalAASRLT()\n",
        "    aa_to_design.aas_to_keep(amino)\n",
        "    tf.push_back(pyrosetta.rosetta.core.pack.task.operation.OperateOnResidueSubset(aa_to_design, mut_posi))\n",
        "    \n",
        "    # Create Packer\n",
        "    packer = pyrosetta.rosetta.protocols.minimization_packing.PackRotamersMover()\n",
        "    packer.task_factory(tf)\n",
        "\n",
        "    #Perform The Move\n",
        "    if not os.getenv(\"DEBUG\"):\n",
        "      packer.apply(pose)\n",
        "\n",
        "#Load the previously-relaxed pose.\n",
        "relaxPose = pose_from_pdb(\"inputs/1jhl.relax.pdb\")\n",
        "\n",
        "#Clone it\n",
        "original = relaxPose.clone()\n",
        "scorefxn = get_score_function()\n",
        "print(\"\\nOld Energy:\", scorefxn(original),\"\\n\")\n",
        "pack(relaxPose, 130, 'A', scorefxn)\n",
        "print(\"\\nNew Energy:\", scorefxn(relaxPose),\"\\n\")\n",
        "\n",
        "#Set relaxPose back to original\n",
        "relaxPose = original.clone()\n"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "-scBuqg08RwW"
      },
      "source": [
        "### **Step 3. unbind()**\n",
        "This function is for binding energy analysis. To compute a binding energy, we need to score the total energy of a bound state structure, separate (unbind) the antigen and antibody and then score the unbound state total energy. The binding energy is given by **bound energy** - **unbound energy**.  You can also use the **InterfaceAnalyzerMover (IAM)** , located in `rosetta.protocols.analysis`.\n",
        "\n",
        "- Benefits to unbind over IAM:\n",
        " - Quicker\n",
        " - Simpler\n",
        " \n",
        "- Disadvantages to unbind over IAM:\n",
        " - Specific foldtree needs to used, with a specific jump. For IAM - it is now foldtree independent, so you only need to know the chainIDs on each side of the interface. "
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "iuqAC4i58RwX"
      },
      "source": [
        "from pyrosetta.rosetta.protocols import *\n",
        "\n",
        "def unbind(pose, partners):\n",
        "    STEP_SIZE = 100\n",
        "    JUMP = 2\n",
        "    docking.setup_foldtree(pose, partners, Vector1([-1,-1,-1]))\n",
        "    trans_mover = rigid.RigidBodyTransMover(pose,JUMP)\n",
        "    trans_mover.step_size(STEP_SIZE)\n",
        "    trans_mover.apply(pose)\n",
        "\n",
        "\n",
        "##Reset the original pose\n",
        "relaxPose = original.clone()\n",
        "\n",
        "scorefxn = get_score_function()\n",
        "bound_score = scorefxn(relaxPose)\n",
        "print(\"\\nBound State Score\",bound_score,\"\\n\")\n",
        "unbind(relaxPose, \"HL_A\")\n",
        "unbound_score = scorefxn(relaxPose)\n",
        "\n",
        "print(\"\\nUnbound State Score\", unbound_score,\"\\n\")\n",
        "print('dG', bound_score - unbound_score, 'Rosetta Energy Units (REU)')\n"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "KxTre0Bw8Rwg"
      },
      "source": [
        "## **Step 4. wildtype()**\n",
        "\n",
        "An important metrics for evaluating binding improvement is the ratio of mutant binding energy to wild type binding energy. This function returns the wild type amino acids in a given position. "
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "9jm9EKUe8Rwh"
      },
      "source": [
        "def wildtype(aatype = 'AA.aa_gly'):\n",
        "    AA = ['G','A','L','M','F','W','K','Q','E','S','P'\n",
        "            ,'V','I','C','Y','H','R','N','D','T']\n",
        "\n",
        "    AA_3 = ['AA.aa_gly','AA.aa_ala','AA.aa_leu','AA.aa_met','AA.aa_phe','AA.aa_trp'\n",
        "            ,'AA.aa_lys','AA.aa_gln','AA.aa_glu', 'AA.aa_ser','AA.aa_pro','AA.aa_val'\n",
        "            ,'AA.aa_ile','AA.aa_cys','AA.aa_tyr','AA.aa_his','AA.aa_arg','AA.aa_asn'\n",
        "            ,'AA.aa_asp','AA.aa_thr']\n",
        "\n",
        "    for i in range(0, len(AA_3)):\n",
        "        if(aatype == AA_3[i]):\n",
        "            return AA[i]\n",
        "\n",
        "print(wildtype(str(relaxPose.aa(130))))\n"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "gU_dHIlimdll"
      },
      "source": [
        "# Setp 4a.  Exploration. \n",
        "\n",
        "We can also use some utility functions instead of this function. \n",
        "\n",
        "We can use the sequence STRING returned.  Note that strings are indexed at 0 in Rosetta. "
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "wbDo27Ohmdll"
      },
      "source": [
        "\n",
        "print(relaxPose.sequence())\n",
        "print(\"\\n\",relaxPose.sequence()[129])"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "XMFsFs9Xmdll"
      },
      "source": [
        "Now lets get some information fromt the ResidueType object that holds chemical information. \n"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "dcHFBtX6mdlm"
      },
      "source": [
        "print(relaxPose.residue_type(130))"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "i8DCf_zPmdlm"
      },
      "source": [
        "Finally, lets get the residue from the type"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "u2ntqew4mdlm"
      },
      "source": [
        "resT = relaxPose.residue_type(130)\n",
        "\n",
        "print(resT.base_name())\n",
        "print(resT.name3())\n",
        "print(resT.name1())"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "HzVlvWse8Rwp"
      },
      "source": [
        "## **Step 5. Integrate functions for mutate and output**"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "RV4M9rco8Rwq"
      },
      "source": [
        "def mutate(pose, posi, amino, partners):\n",
        "    #main function for mutation\n",
        "    CSV_PREFIX = 'notec'\n",
        "    PDB_PREFIX = 'notep'\n",
        "\n",
        "    #Initiate test pose\n",
        "    testPose = Pose()\n",
        "    testPose.assign(pose)\n",
        "\n",
        "    #Initiate energy function\n",
        "    scorefxn = get_fa_scorefxn()\n",
        "    unbind(testPose, partners)\n",
        "    native_ub = scorefxn(testPose)\n",
        "    testPose.assign(pose)\n",
        "    \n",
        "    #Variables initiation\n",
        "    content = ''\n",
        "    name = CSV_PREFIX + str(posi)+str(amino) + '.csv'\n",
        "    pdbname = PDB_PREFIX + str(posi)+str(amino) + '.pdb'\n",
        "    wt = wildtype(str(pose.aa(posi)))\n",
        "\n",
        "    pack(testPose, posi, amino, scorefxn)\n",
        "    testPose.dump_pdb(pdbname)\n",
        "    bound = scorefxn(testPose)\n",
        "    unbind(testPose, partners)\n",
        "    unbound = scorefxn(testPose)\n",
        "    binding = unbound - bound\n",
        "    testPose.assign(pose)\n",
        "\n",
        "    if (wt == amino):\n",
        "        wt_energy = binding\n",
        "    else:\n",
        "        pack(testPose, posi, wt, scorefxn)\n",
        "        wtbound = scorefxn(testPose)\n",
        "        unbind(testPose, partners)\n",
        "        wtunbound = scorefxn(testPose)\n",
        "        wt_energy = wtunbound - wtbound\n",
        "        testPose.assign(pose)\n",
        "\n",
        "    content=(content+str(pose.pdb_info().pose2pdb(posi))+','+str(amino)+','\n",
        "              +str(native_ub)+','+str(bound)+','+str(unbound)+','+str(binding)+','\n",
        "              +str(wt_energy)+','+str(wt)+','+str(binding/wt_energy)+'\\n')\n",
        "\n",
        "    f = open(name,'w+')\n",
        "    f.write(content)\n",
        "    f.close()\n",
        "\n",
        "mutate(relaxPose, 130, 'A', 'HL_A')\n"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "6eaD1Qmo8Rwt"
      },
      "source": [
        "## **Step 6. Loop through interface positions**\n",
        "\n",
        "The following code loop through all heavy chain and light chain positions mutating them into all 20 amino acids. The actual run may again crash the notebook. \n",
        "\n",
        "You may output the data however you wish.  pose.scores() will have the data, maps and pandas Dataframes can also be used.  Here, we will simply output energy information of each mutation in form of csv files and cat them into one file for future analysis. An output file is uploaded to the input folder for the R analysis. \n",
        "\n",
        "In real work setting, parallelization can be achieve.  Please see the PyRosetta in Parellel chapter.  For this task, you not necessarily need to finish the whole loop. We have a finished version in the inputs folder.\n"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "xBR7u2b12th1"
      },
      "source": [
        "#NOTE - This takes a very long time!!  \n",
        "# You may skip this block to continue the tutorial with pre-configured outputs.\n",
        "if not os.getenv(\"DEBUG\"):\n",
        "  for i in [92,85,68,53,5,45,44,42,32,31,22,108,100]:\n",
        "    print(\"\\nMutating Position: \",str(i),\"\\n\")\n",
        "    for j in ['G','A','L','M','F','W','K','Q','E','S','P','V','I','C','Y','H','R','N','D','T']:\n",
        "      mutate(relaxPose, i, j, 'HL_A')"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "_t7PqsI2rtR4"
      },
      "source": [
        "Merging output csv files for binding energy analysis in R"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "iCJ5MTrX2zEt"
      },
      "source": [
        "## **Analysis of binding data**\n",
        "After gathering summarized binding energy information, we use pandas for filtering and visualization. We filtered out lower unbound energy structures, those that have higher unbound state total energy than native and make a heatmap from the filtered data. In case you don't want to finish the for loop at Step 8, we uploaded a finished version of merged output csv to the inputs folder named \"note_output.csv\"."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "0AgWCFvhxsnf"
      },
      "source": [
        "#import modules need for analysis\n",
        "import pandas as pd\n",
        "%matplotlib inline\n",
        "import matplotlib.pyplot as plt \n",
        "import seaborn as sns"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "sSuEU5sL3HM0"
      },
      "source": [
        "csv_file_name = 'inputs/note_output.csv'"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "Yhza0hnjxyUO"
      },
      "source": [
        "#Generating heatmaps\n",
        "\n",
        "UNBOUND_CUTOFF = -995\n",
        "RATIO_CUTOFF = 1.001\n",
        "\n",
        "#load the data into a pandas dataframe\n",
        "df = pd.read_csv(csv_file_name, names='Position Amino.Acid Native Bound Unbound Binding WT_Binding WT Ratio'.split(), index_col=False )\n",
        "#Add wildtype AA to position (for display)\n",
        "df['Position_WT_aa'] = df.Position + ' (' + df.WT  + ')' \n",
        "\n",
        "#filter values\n",
        "df = df.query(f'Unbound<{UNBOUND_CUTOFF} and Ratio>{RATIO_CUTOFF}')\n",
        "\n",
        "# convert from tidy format (https://en.wikipedia.org/wiki/Tidy_data) to a wider format\n",
        "df = pd.pivot_table(df, values='Ratio', \n",
        "                     index=['Position_WT_aa'], \n",
        "                     columns='Amino.Acid')\n",
        "\n",
        "#plot the heatmap\n",
        "f, ax = plt.subplots(figsize=(10, 10))\n",
        "sns.heatmap(df, cmap='coolwarm', linewidths= 1, linecolor='lightgray')\n",
        "plt.xlabel('Amino acid mutation');\n",
        "plt.ylabel('Position chain and wild type AA');\n",
        "sns.set_context(\"talk\") #make labels bigger"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "qwh88WH88Ry3"
      },
      "source": [
        "# See Also\n",
        "\n",
        "Note these may not be available in PyRosetta through code or even by xml (remodel), but they are extremely useful tools when doing denovo protein design - and you should be aware of them.\n",
        "\n",
        "- **RosettaRemodel**\n",
        " - https://www.rosettacommons.org/docs/latest/application_documentation/design/rosettaremodel\n",
        "    \n",
        "    \n",
        "- **Sewing**\n",
        "\n",
        " - https://www.rosettacommons.org/docs/latest/scripting_documentation/RosettaScripts/composite_protocols/sewing/SEWING\n",
        " \n",
        " \n",
        "-  **Parametric Design**\n",
        " - Previous Workshop!"
      ]
    }
  ]
}